{"cells":[{"cell_type":"markdown","source":["# Lab: exploring the CORD-19 dataset\n\nThis lab uses the CORD-19 dataset that you're using in this year's assignment. We'll replicate and extend the work done in\nthe demo in a way that'll hopefully indicate how to analyze the results that you get and how to extend the work (whether\nfor the assignment or for a future client). Obviously, you will not be allowed to submit this work as an extension of your\nassignments. We'll use the 2020-03-27 dataset in this lab."],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"042cceb6-e3cc-457e-91a6-bf6edd4ea2ae"}}},{"cell_type":"markdown","source":["### Naive sentiment analysis"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"812efa2b-6165-4ee6-b18d-b9f8e7546ffe"}}},{"cell_type":"markdown","source":["--> 1. If you have not yet loaded the 2020-03-27 CORD-19 dataset, you should create a table out of it (e.g. via the UI). You\nshouldn't need to do any pre-processing for this lab."],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"09721cb2-c9a2-4404-bf29-c62ae68c62b9"}}},{"cell_type":"code","source":["dbutils.fs.ls(\"dbfs:/FileStore/tables/CORD19/\")"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"b35949ed-8623-4a5f-87e4-cf8177722809"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">Out[14]: [FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/CountryKeys/&#39;, name=&#39;CountryKeys/&#39;, size=0),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/metadata_2020_03_27.csv&#39;, name=&#39;metadata_2020_03_27.csv&#39;, size=69689204),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/metadata_2020_05_01.csv&#39;, name=&#39;metadata_2020_05_01.csv&#39;, size=89290114),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/metadata_2020_07_01.csv&#39;, name=&#39;metadata_2020_07_01.csv&#39;, size=242867644),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/prd&#39;, name=&#39;prd&#39;, size=242867644),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/scimagojr_2019.csv&#39;, name=&#39;scimagojr_2019.csv&#39;, size=8170647)]</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">Out[14]: [FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/CountryKeys/&#39;, name=&#39;CountryKeys/&#39;, size=0),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/metadata_2020_03_27.csv&#39;, name=&#39;metadata_2020_03_27.csv&#39;, size=69689204),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/metadata_2020_05_01.csv&#39;, name=&#39;metadata_2020_05_01.csv&#39;, size=89290114),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/metadata_2020_07_01.csv&#39;, name=&#39;metadata_2020_07_01.csv&#39;, size=242867644),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/prd&#39;, name=&#39;prd&#39;, size=242867644),\n FileInfo(path=&#39;dbfs:/FileStore/tables/CORD19/scimagojr_2019.csv&#39;, name=&#39;scimagojr_2019.csv&#39;, size=8170647)]</div>"]}}],"execution_count":0},{"cell_type":"markdown","source":["--> 2. Create a dataframe from the table you created in Step 1"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"2631df94-1a72-4a1f-8bfa-3a1bb6cd0d9c"}}},{"cell_type":"code","source":["# File location and type\nfile_location = 'dbfs:/FileStore/tables/CORD19/metadata_2020_03_27.csv'\nfile_type = \"csv\"\n\n# CSV options\ninfer_schema = \"true\"\nfirst_row_is_header = \"true\"\ndelimiter = \",\"\n\n# The applied options are for CSV files. For other file types, these will be ignored.\ndf_cord = spark.read.format(file_type) \\\n               .option(\"inferSchema\", infer_schema) \\\n               .option(\"header\", first_row_is_header) \\\n               .option(\"quote\", \"\\\"\") \\\n               .option(\"escape\", \"\\\"\") \\\n               .option(\"sep\", delimiter) \\\n               .load(file_location)\n\n# Dataframe df_cord displayed\ndf_cord.printSchema()\ndisplay(df_cord)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"404a7f5c-0ffd-4f21-93eb-43a2d3ed9236"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">root\n |-- cord_uid: string (nullable = true)\n |-- sha: string (nullable = true)\n |-- source_x: string (nullable = true)\n |-- title: string (nullable = true)\n |-- doi: string (nullable = true)\n |-- pmcid: string (nullable = true)\n |-- pubmed_id: string (nullable = true)\n |-- license: string (nullable = true)\n |-- abstract: string (nullable = true)\n |-- publish_time: string (nullable = true)\n |-- authors: string (nullable = true)\n |-- journal: string (nullable = true)\n |-- Microsoft Academic Paper ID: string (nullable = true)\n |-- WHO #Covidence: string (nullable = true)\n |-- has_full_text: string (nullable = true)\n |-- full_text_file: string (nullable = true)\n |-- url: string (nullable = true)\n\n</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">root\n-- cord_uid: string (nullable = true)\n-- sha: string (nullable = true)\n-- source_x: string (nullable = true)\n-- title: string (nullable = true)\n-- doi: string (nullable = true)\n-- pmcid: string (nullable = true)\n-- pubmed_id: string (nullable = true)\n-- license: string (nullable = true)\n-- abstract: string (nullable = true)\n-- publish_time: string (nullable = true)\n-- authors: string (nullable = true)\n-- journal: string (nullable = true)\n-- Microsoft Academic Paper ID: string (nullable = true)\n-- WHO #Covidence: string (nullable = true)\n-- has_full_text: string (nullable = true)\n-- full_text_file: string (nullable = true)\n-- url: string (nullable = true)\n\n</div>"]}},{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"overflow":true,"datasetInfos":[],"data":[["vho70jcx","f056da9c64fbf00a4645ae326e8a4339d015d155","biorxiv","SIANN: Strain Identification by Alignment to Near Neighbors","10.1101/001727",null,null,"biorxiv","Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.","2014-01-10","Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/001727"],["i9tbix2v","daf32e013d325a6feb80e83d15aabc64a48fae33","biorxiv","Spatial epidemiology of networked metapopulation: An overview","10.1101/003889",null,null,"biorxiv","An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.","2014-06-04","Lin WANG; Xiang Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/003889"],["62gfisc6","f33c6d94b0efaa198f8f3f20e644625fa3fe10d2","biorxiv","Sequencing of the human IG light chain loci from a hydatidiform mole BAC library reveals locus-specific signatures of genetic diversity","10.1101/006866",null,null,"biorxiv","Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.","2014-07-03","Corey T Watson; Karyn Meltz Steinberg; Tina A Graves-Lindsay; Rene L Warren; Maika Malig; Jacqueline E Schein; Richard K Wilson; Rob Holt; Evan Eichler; Felix Breden",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/006866"],["058r9486","4da8a87e614373d56070ed272487451266dce919","biorxiv","Bayesian mixture analysis for metagenomic community profiling.","10.1101/007476",null,null,"biorxiv","Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix","2014-07-25","Sofia Morfopoulou; Vincent Plagnol",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/007476"],["wich35l7","eccef80cfbe078235df22398f195d5db462d8000","biorxiv","Mapping a viral phylogeny onto outbreak trees to improve host transmission inference","10.1101/010389",null,null,"biorxiv","Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.","2014-11-11","Stephen P Velsko; Jonathan E Allen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/010389"],["z3tgnzth","c41fdb2efd6d61384a92a84cbba3f8233629a41b","biorxiv","The infant airway microbiome in health and disease impacts later asthma development","10.1101/012070",null,null,"biorxiv","The nasopharynx (NP) is a reservoir for microbes associated with acute respiratory illnesses (ARI). The development of asthma is initiated during infancy, driven by airway inflammation associated with infections. Here, we report viral and bacterial community profiling of NP aspirates across a birth cohort, capturing all lower respiratory illnesses during their first year. Most infants were initially colonized with Staphylococcus or Corynebacterium before stable colonization with Alloiococcus or Moraxella, with transient incursions of Streptococcus, Moraxella or Haemophilus marking virus-associated ARIs. Our data identify the NP microbiome as a determinant for infection spread to the lower airways, severity of accompanying inflammatory symptoms, and risk for future asthma development. Early asymptomatic colonization with Streptococcus was a strong asthma predictor, and antibiotic usage disrupted asymptomatic colonization patterns.","2014-12-02","Shu Mei Teo; Danny Mok; Kym Pham; Merci Kusel; Michael Serralha; Niamh Troy; Barbara J Holt; Belinda J Hales; Michael L Walker; Elysia Hollams; Yury H Bochkov; Kristine Grindle; Sebastian L Johnston; James E Gern; Peter D Sly; Patrick G Holt; Kathryn E Holt; Michael Inouye",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/012070"],["1xxrnpg3","1dd898b5ca1ae70ec0e3cad89fc87a165002a99e","biorxiv","Using heterogeneity in the population structure of U.S. swine farms to compare transmission models for porcine epidemic diarrhoea","10.1101/017178",null,null,"biorxiv","ABSTRACTIn 2013, U.S. swine producers were confronted with the disruptive emergence of porcine epidemic diarrhoea (PED). Movement of animals among farms is hypothesised to have played a role in the spread of PED among farms. Via this or other mechanisms, the rate of spread may also depend on the geographic density of farms and climate. To evaluate such effects on a large scale, we analyse state-level counts of outbreaks with variables describing the distribution of farm sizes and types, aggregate flows of animals among farms, and an index of climate. Our first main finding is that it is possible for a correlation analysis to be sensitive to transmission model parameters. This finding is based on a global sensitivity analysis of correlations on simulated data that included a biased and noisy observation model based on the available PED data. Our second main finding is that flows are significantly associated with the reports of PED outbreaks. This finding is based on correlations of pairwise relationships and regression modeling of total and weekly outbreak counts. These findings illustrate how variation in population structure may be employed along with observational data to improve understanding of disease spread.","2015-03-27","Eamon B. O’Dea; Harry Snelson; Shweta Bansal",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/017178"],["8ilzm51q","33565294e6bc67fb7ee14dcae6cfdb08148f4ea5","biorxiv","Big city, small world: Density, contact rates, and transmission of dengue across Pakistan.","10.1101/018481",null,null,"biorxiv","Macroscopic descriptions of populations commonly assume that encounters between individuals are well mixed; i.e., each individual has an equal chance of coming into contact with any other individual. Relaxing this assumption can be challenging though, due to the difficulty of acquiring detailed knowledge about the non-random nature of encounters. Here, we fitted a mathematical model of dengue virus transmission to spatial time series data from Pakistan and compared maximum-likelihood estimates of “mixing parameters” when disaggregating data across an urban-rural gradient. We show that dynamics across this gradient are subject not only to differing transmission intensities but also to differing strengths of nonlinearity due to differences in mixing. We furthermore show that neglecting spatial variation in mixing can lead to substantial underestimates of the level of effort needed to control a pathogen with vaccines or other control efforts. We complement this analysis with relevant contemporary environmental drivers of dengue.","2015-04-27","Moritz U. G. Kraemer; T. Alex Perkins; Derek A.T. Cummings; Rubeena Zakar; Simon I. Hay; David L. Smith; Robert C. Reiner",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/018481"],["wafvnbdu","3461d71f6890f7e5ba53bf168be3945cdb16d901","biorxiv","MERS-CoV recombination: implications about the reservoir and potential for adaptation","10.1101/020834",null,null,"biorxiv","Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology.","2015-06-12","Gytis Dudas; Andrew Rambaut",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/020834"],["4xocqn6o","1f9d3f9a1a0e8db6a086e0a2b5ba50cf9f235dae","biorxiv","On the causes of evolutionary transition:transversion bias","10.1101/027722",null,null,"biorxiv","A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the \"conservative transitions\" hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.","2015-09-28","Arlin Stoltzfus; Ryan W. Norris",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/027722"],["q5iylws9","01e3b313e78a352593be2ff64927192af66619b5","biorxiv","Viruses are a dominant driver of protein adaptation in mammals","10.1101/029397",null,null,"biorxiv","Viruses interact with hundreds to thousands of proteins in mammals, yet adaptation against viruses has only been studied in a few proteins specialized in antiviral defense. Whether adaptation to viruses typically involves only specialized antiviral proteins or affects a broad array of proteins is unknown. Here, we analyze adaptation in ~1,300 virus-interacting proteins manually curated from a set of 9,900 proteins conserved across mammals. We show that viruses (i) use the more evolutionarily constrained proteins from the cellular functions they hijack and that (ii) despite this high constraint, virus-interacting proteins account for a high proportion of all protein adaptation in humans and other mammals. Adaptation is elevated in virus-interacting proteins across all functional categories, including both immune and non-immune functions. Our results demonstrate that viruses are one of the most dominant drivers of evolutionary change across mammalian and human proteomes.","2015-10-18","David Enard; Le Cai; Carina Gwenapp; Dmitri A Petrov",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/029397"],["nljskxut",null,"biorxiv","What's in my pot? Real-time species identification on the MinION","10.1101/030742",null,null,"biorxiv","Whole genome sequencing on next-generation instruments provides an unbiased way to identify the organisms present in complex metagenomic samples. However, the time-to-result can be protracted because of fixed-time sequencing runs and cumbersome bioinformatics workflows. This limits the utility of the approach in settings where rapid species identification is crucial, such as in the quality control of food-chain components, or in during an outbreak of an infectious disease. Here we present What′s in my Pot? (WIMP), a laboratory and analysis workflow in which, starting with an unprocessed sample, sequence data is generated and bacteria, viruses and fungi present in the sample are classified to subspecies and strain level in a quantitative manner, without prior knowledge of the sample composition, in approximately 3.5 hours. This workflow relies on the combination of Oxford Nanopore Technologies′ MinION ™ sensing device with a real-time species identification bioinformatics application.","2015-11-06","Sissel Juul; Fernando Izquierdo; Adam Hurst; Xiaoguang Dai; Amber Wright; Eugene Kulesha; Roger Pettett; Daniel J Turner",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/030742"],["lfm6erzy","3b73c8abb7528c958cec0bae1890828af007b599","biorxiv","How domain growth is implemented determines the long term behaviour of a cell population through its effect on spatial correlations","10.1101/041509",null,null,"biorxiv","AbstractDomain growth plays an important role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations.","2016-02-26","Robert J. H. Ross; R. E. Baker; C. A. Yates",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/041509"],["lcpp5fim","8cfb8dfc2f8b2c6e1d60ff48da02de6670694869","biorxiv","Productive infection of field strains of avian coronavirus infectious bronchitis virus in chicken peripheral blood-derived monocyte","10.1101/041558",null,null,"biorxiv","The avian coronavirus infectious bronchitis virus (IBV) typically infects the respiratory tract of chickens, but can also spread to other organs. However, the mechanisms of virus dissemination are presently unclear. We show that peripheral blood-derived monocytes/macrophages from chickens (chPBMC) are productively infected by clinical strains of IBV, accompanied by induction of apoptosis. Our data suggest that chPBMCs play a role in the dissemination of IBV, and may be important for viral pathogenesis.","2016-02-26","Yueting Zhang; Gary Whittaker",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/041558"],["gjxumrmm","6499f5c865c3db3bdfc1e969ea940a80d547a5bb","biorxiv","Zika virus outbreak in the Americas: Is Aedes albopictus an overlooked culprit?","10.1101/044594",null,null,"biorxiv","Summary / AbstractCodon usage patterns of viruses reflect a series of evolutionary changes that enable viruses to shape their survival rates and fitness toward the external environment and, most importantly, their hosts. In the present study, we employed multiple codon usage analysis indices to determine genotype specific codon usage patterns of Zika virus (ZIKV) strains from the current outbreak and those reported previously. Several genotype specific and common codon usage traits were noted in ZIKV coding sequences, indicative of independent evolutionary origins from a common ancestor. The overall influence of natural selection was found to be more profound than that of mutation pressure and acting on specific set of viral genes belonging to ZIKV strains of Asian genotype from the recent outbreak. Furthermore, an interplay of codon adaptation and deoptimization have been observed in ZIKV genomes. The collective findings of codon analysis in association with the geographical data of Aedes populations in the Americas suggests that ZIKV have evolved a dynamic set of codon usage patterns in order to maintain a successful replication and transmission chain within multiple hosts and vectors.","2016-03-30","Azeem Mehmood Butt; Izza Nasrullah; Raheel Qamar; Yigang Tong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/044594"],["csokkcqq","a59906b732bf4a489e282c3e4f499d4166c622e7","biorxiv","Evaluating performance of metagenomic characterization algorithms using in silico datasets generated with FASTQSim","10.1101/046532",null,null,"biorxiv","AbstractBackgroundIn silico bacterial, viral, and human truth datasets were generated to evaluate available metagenomics algorithms. Sequenced datasets include background organisms, creating ambiguity in the true source organism for each read. Bacterial and viral datasets were created with even and staggered coverage to evaluate organism identification, read mapping, and gene identification capabilities of available algorithms. These truth datasets are provided as a resource for the development and refinement of metagenomic algorithms. Algorithm performance on these truth datasets can inform decision makers on strengths and weaknesses of available algorithms and how the results may be best leveraged for bacterial and viral organism identification and characterization.Source organisms were selected to mirror communities described in the Human Microbiome Project as well as the emerging pathogens listed by the National Institute of Allergy and Infectious Diseases. The six in silico datasets were used to evaluate the performance of six leading metagenomics algorithms: MetaScope, Kraken, LMAT, MetaPhlAn, MetaCV, and MetaPhyler.ResultsAlgorithms were evaluated on runtime, true positive organisms identified to the genus and species levels, false positive organisms identified to genus and species level, read mapping, relative abundance estimation, and gene calling. No algorithm out performed the others in all categories, and the algorithm or algorithms of choice strongly depends on analysis goals. MetaPhlAn excels for bacteria and LMAT for viruses. The algorithms were ranked by overall performance using a normalized weighted sum of the above metrics, and MetaScope emerged as the overall winner, followed by Kraken and LMAT.ConclusionsSimulated FASTQ datasets with well-characterized truth data about microbial community composition reveal numerous insights about the relative strengths and weaknesses of the metagenomics algorithms evaluated. The simulated datasets are available to download from the Sequence Read Archive (SRP062063).","2016-03-31","Anna Shcherbina; Darrell O. Ricke; Nelson Chiu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/046532"],["08mep8hm","99dab788e984601ee4efcd1138926caf10c00dcf","biorxiv","Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension","10.1101/048801",null,null,"biorxiv","Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (fs) and monovalent salt concentrations (Cs). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log Cm)α with α = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation.","2016-04-15","Naoto Hori; Natalia A. Denesyuk; D. Thirumalai",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/048801"],["ejqq2wmi","97339c9d365437d54ea33be1a1b634d35f460208","biorxiv","Fractional Dosing of Yellow Fever Vaccine to Extend Supply: A Modeling Study","10.1101/053421",null,null,"biorxiv","BackgroundThe ongoing yellow fever (YF) epidemic in Angola strains the global vaccine supply, prompting WHO to adopt dose sparing for its vaccination campaign in Kinshasa in July-August 2016. Although a 5-fold fractional-dose vaccine is similar to standard-dose vaccine in safety and immunogenicity, efficacy is untested. There is an urgent need to ensure the robustness of fractional-dose vaccination by elucidating the conditions under which dose fractionation would reduce transmission.MethodsWe estimate the effective reproductive number for YF in Angola using disease natural history and case report data. With simple mathematical models of YF transmission, we calculate the infection attack rate (IAR, the proportion of population infected over the course of an epidemic) under varying levels of transmissibility and five-fold fractional-dose vaccine efficacy for two vaccination scenarios: (i) random vaccination in a hypothetical population that is completely susceptible; (ii) the Kinshasa vaccination campaign in July-August 2016 with different age cutoff for fractional-dose vaccines.FindingsWe estimate the effective reproductive number early in the Angola outbreak was between 5·2 and 7·1. If vaccine action is all-or-nothing (i.e. a proportion VE of vaccinees receives complete and the remainder receive no protection), n-fold fractionation can dramatically reduce IAR as long as efficacy VE exceeds 1/n. This benefit threshold becomes more stringent if vaccine action is leaky (i.e. the susceptibility of each vaccinee is reduced by a factor that is equal to the vaccine efficacy VE). The age cutoff for fractional-dose vaccines chosen by the WHO for the Kinshasa vaccination campaign (namely, 2 years) provides the largest reduction in IAR if the efficacy of five-fold fractional-dose vaccines exceeds 20%.InterpretationDose fractionation is a very effective strategy for reducing infection attack rate that would be robust with a large margin for error in case fractional-dose VE is lower than expected.FundingNIH-MIDAS, HMRF-Hong Kong","2016-05-14","Joseph T. Wu; Corey M. Peak; Gabriel M. Leung; Marc Lipsitch",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/053421"],["2eh0bhee","1c0b25c8e3c89243fd8797e2e964e12689bc8556","biorxiv","50-valent inactivated rhinovirus vaccine is broadly immunogenic in rhesus macaques","10.1101/053967",null,null,"biorxiv","AbstractAs the predominant etiological agent of the common cold, human rhinovirus (HRV) is the leading cause of human infectious disease. Early studies showed monovalent formalin-inactivated HRV vaccine can be protective, and virus-neutralizing antibodies (nAb) correlated with protection. However, co-circulation of many HRV types discouraged further vaccine efforts. We approached this problem straightforwardly. We tested the hypothesis that increasing virus input titers in polyvalent inactivated HRV vaccine will result in broad nAb responses. Here, we show that serum nAb against many rhinovirus types can be induced by polyvalent, inactivated HRVs plus alhydrogel (alum) adjuvant. Using formulations up to 25-valent in mice and 50-valent in rhesus macaques, HRV vaccine immunogenicity was related to sufficient quantity of input antigens, and valency was not a major factor for potency or breadth of the response. We for the first time generated a vaccine capable of inducing nAb responses to numerous and diverse HRV types.","2016-05-17","Sujin Lee; Minh Trang Nguyen; Michael G. Currier; Joe B. Jenkins; Elizabeth A. Strobert; Adriana E. Kajon; Ranjna Madan-Lala; Yury A. Bochkov; James E. Gern; Krishnendu Roy; Xiaoyan Lu; Dean D. Erdman; Paul Spearman; Martin L. Moore",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/053967"],["76i8yvb4","d1b26388670660f66249b55ece2beccee4c18d1a","biorxiv","Evidence for reassortment of highly divergent novel rotaviruses from bats in Cameroon, without evidence for human interspecies transmissions","10.1101/054072",null,null,"biorxiv","AbstractBats are an important reservoir for pathogenic human respiratory and hemorrhagic viruses but only little is known about bat viruses causing gastroenteritis in humans, including rotavirus A strains (RVA). Only three RVA strains have been reported in bats in Kenya (straw-colored fruit bat) and in China (lesser horseshoe and a stoliczka’s trident bat), being highly divergent from each other. To further elucidate the potential of bat RVAs to cause gastroenteritis in humans we started by investigating the genetic diversity of RVAs in fecal samples from 87 straw-colored fruit bats living in close contact with humans in Cameroon using metagenomics. Five samples contained significant numbers of RVA Illumina reads, sufficient to obtain their (near) complete genomes. A single RVA strain showed a close phylogenetic relationship with the Kenyan bat RVA strain in six gene segments, including VP7 (G25), whereas the other gene segments represented novel genotypes as ratified by the RCWG. The 4 other RVA strains were highly divergent from known strains (but very similar among each other) possessing all novel genotypes. Only the VP7 and VP4 genes showed a significant variability representing multiple novel G and P genotypes, indicating the frequent occurrence of reassortment events.Comparing these bat RVA strains with currently used human RVA screening primers indicated that several of the novel VP7 and VP4 segments would not be detected in routine epidemiological screening studies. Therefore, novel VP6 based screening primers matching both human and bat RVAs were developed and used to screen samples from 25 infants with gastroenteritis living in close proximity with the studied bat population. Although RVA infections were identified in 36% of the infants, Sanger sequencing did not indicate evidence of interspecies transmissions.This study identified multiple novel bat RVA strains, but further epidemiological studies in humans will have to assess if these viruses have the potential to cause gastroenteritis in humans.","2016-05-18","Claude Kwe Yinda; Mark Zeller; Nádia Conceição-Neto; Piet Maes; Ward Deboutte; Leen Beller; Elisabeth Heylen; Stephen Mbigha Ghogomu; Marc Van Ranst; Jelle Matthijnssens",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/054072"],["omtmy9z6",null,"biorxiv","The ATP synthase subunit β (ATP5B) is an entry factor for the hepatitis E virus","10.1101/060434",null,null,"biorxiv","AbstractHepatitis E occurs sporadically and as outbreaks due to contamination of drinking water. The causative agent, hepatitis E virus (HEV) is a hepatotropic non-enveloped RNA virus, which grows poorly in vitro. Consequently, many aspects of HEV biology are poorly characterized, including its cellular receptor and entry mechanism(s). Previous studies from our laboratory have shown that heparan sulfate proteoglycans (HSPGs) act as attachment factors for the virus. In the absence of purified high titer infectious virus, we have used hepatitis E virus-like particles (HEV-LPs) expressed and purified from E. coli to identify HEV entry factor(s) on liver cells in culture. Using a pull down and mass spectrometric approach, we identified the ATP synthase subunit β (ATP5B) to bind the HEV capsid protein. Its role in the entry of HEV was then validated using antibody and siRNA mediated approaches, and infectious HEV from the stools of a hepatitis E patient. Though ATP synthase is largely a mitochondrial protein, the cell surface expressed form of ATP5B is implicated in other viral infections.","2016-06-23","Zulfazal Ahmed; Prasida Holla; Imran Ahmad; Shahid Jameel",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/060434"],["2cerplno","53442eacc3f233078507fa37b78267399e8c1e3b","biorxiv","Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection","10.1101/061788",null,null,"biorxiv","ABSTRACTZika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes.","2016-07-01","Arunachalam Ramaiah; Deisy Contreras; Vineela Gangalapudi; Masumi Sameer Padhye; Jie Tang; Vaithilingaraja Arumugaswami",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/061788"],["t3tm6f9p","c04e10858012bdd666d50c0d69c3d1e7224ccbea","biorxiv","Design: An assay based on single-polypeptide-chain heterodimeric A2AR/D2R and non-oligomerized fusions for in vivo analysis of their allosteric receptor-receptor interactions","10.1101/065250",null,null,"biorxiv","AbstractBackgroundThe adenosine A2A receptor (A2AR) heteromerizes with the dopamine D2 receptor (D2R). In order to explore their functional interaction, we engineered previously stable single-polypeptide-chain (sc) A2AR/D2LR: whether the molecular entity of the striatal A2AR/D2R antagonism, i.e., scA2AR/D2Rs are just A2AR/D2R with the antagonism, remains unresolved.New MethodTo further clarify the heteromerization through the scA2AR/D2LR, we here designed supramolecularly ‘exclusive’ monomers and dimers, using the Cε2 domain of IgE-Fc or apoproteins of the bacterial light-harvesting antenna complex.ResultsA concept of the recptor protein assembly regulation, i.e., the selective monomer/non-obligate dimer formation was obtained. Although none of these new fusions were constructed or tested, we could aim at obtaining heterodimer-specific agents, using the scA2AR/D2R. Whether the resulting designs were explained feasibly and rationally was addressed. The structure and function of the non-obligate dimer were here discussed through scA2AR/D2R, focusing on the procedure of the membrane protein design and methods for transient protein-protein interactions.Summary and OutlookGiven that upon being expressed and allosteric regulation occurs regardless of specific signal to non-specific noise (S/N) ratio, the supramolecular designs, allowing us to express selectively monomer/non-obligate dimer of class A GPCR, are experimentally testable and will be used to confirm in vivo that such low S/N ratio interaction between A2AR and D2LR functions in the dopamine neurotransmission in the striatum.Abbreviations:A2ARadenosine A2A receptor3HA-A2ARA2AR tagged with a triple HA epitopeBchlbacteriochlorophyllBRETbioluminescence resonance energy transferCcarboxy-terminalCDcluster of differentiationD2LR and D2SRthe long and short form of dopamine D2 receptor, respectivelyFabantigen binding fragmentFcFc fragmentFcεRIhigh affinity receptor for IgEFRETfluorescence resonance energy transferG4San amino acid sequence consisting of a four-glycine-repeat followed by a serine residueGABAγ-aminobutyric acidGABABGABA type B receptorGPCRG protein-coupled receptorGttransducinHAhemagglutininHIVhuman immunodeficiency virusICintracellular loopsIgimmunoglobulinLHlight-harvesting antenna complexmAbmonoclonal antibodyMrmolecular weightNamino-terminalPDParkinson’s diseasePSphotosystemRCreaction centerRlucRenilla luciferasescsingle-chainTMtransmembrane3Dthree-dimensional","2016-07-26","Toshio Kamiya; Takashi Masuko; Dasiel Oscar Borroto-Escuela; Haruo Okado; Hiroyasu Nakata",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/065250"],["57sp9d9l","5570cb9ff2905fda60cfd6e94f3c32774b765265","biorxiv","The genome of the crustacean Parhyale hawaiensis: a model for animal development, regeneration, immunity and lignocellulose digestion","10.1101/065789",null,null,"biorxiv","ABSTRACTThe amphipod crustacean Parhyale hawaiensis is a blossoming model system for studies of developmental mechanisms and more recently regeneration. We have sequenced the genome allowing annotation of all key signaling pathways, transcription factors, and non-coding RNAs that will enhance ongoing functional studies. Parhyale is a member of the Malacostraca clade, which includes crustacean food crop species. We analysed the immunity related genes of Parhyale as an important comparative system for these species, where immunity related aquaculture problems have increased as farming has intensified. We also find that Parhyale and other species within Multicrustacea contain the enzyme sets necessary to perform lignocellulose digestion (“wood eating”), suggesting this ability may predate the diversification of this lineage. Our data provide an essential resource for further development of Parhyale as an experimental model. The first malacostracan genome will underpin ongoing comparative work in food crop species and research investigating lignocellulose as an energy source.","2016-07-25","Damian Kao; Alvina G. Lai; Evangelia Stamataki; Silvana Rosic; Nikolaos Konstantinides; Erin Jarvis; Alessia Di Donfrancesco; Natalia Pouchkina-Stantcheva; Marie Sémon; Marco Grillo; Heather Bruce; Suyash Kumar; Igor Siwanowicz; Andy Le; Andrew Lemire; Michael B. Eisen; Cassandra Extavour; William E. Browne; Carsten Wolff; Michalis Averof; Nipam H. Patel; Peter Sarkies; Anastasios Pavlopoulos; A. Aziz Aboobaker",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/065789"],["tiyjmih7","45f184e736c4a857bc0e3df0d55962f743c1493a","biorxiv","Similar ratios of introns to intergenic sequence across animal genomes","10.1101/068627",null,null,"biorxiv","AbstractOne central goal of genome biology is to understand how the usage of the genome differs between organisms. Our knowledge of genome composition, needed for downstream inferences, is critically dependent on gene annotations, yet problems associated with gene annotation and assembly errors are usually ignored in comparative genomics. Here we analyze the genomes of 68 species across 12 animal phyla and some single-cell eukaryotes for general trends in genome composition and transcription, taking into account problems of gene annotation. We show that, regardless of genome size, the ratio of introns to intergenic sequence is comparable across essentially all animals, with nearly all deviations dominated by increased intergenic sequence. Genomes of model organisms have ratios much closer to 1:1, suggesting that the majority of published genomes of non-model organisms are underannotated and consequently omit substantial numbers of genes, with likely negative impact on evolutionary interpretations. Finally, our results also indicate that most animals transcribe half or more of their genomes arguing against differences in genome usage between animal groups, and also suggesting that the transcribed portion is more dependent on genome size than previously thought.","2016-08-09","Warren R. Francis; Gert Wörheide",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/068627"],["30kwl4rj","ffbf8ea9948d73572fd052a74afa01b19e6758a3","biorxiv","Planning horizon affects prophylactic decision-making and epidemic dynamics","10.1101/069013",null,null,"biorxiv","AbstractHuman behavior can change the spread of infectious disease. There is limited understanding of how the time in the future over which individuals make a behavioral decision, their planning horizon, affects epidemic dynamics. We developed an agent-based model (along with an ODE analog) to explore the decision-making of self-interested individuals on adopting prophylactic behavior. The decision-making process incorporates prophylaxis efficacy and disease prevalence with individuals' payoffs and planning horizon. Our results show that for short and long planning horizons individuals do not consider engaging in prophylactic behavior. In contrast, individuals adopt prophylactic behavior when considering intermediate planning horizons. Such adoption, however, is not always monotonically associated with the prevalence of the disease, depending on the perceived protection efficacy and the disease parameters. Adoption of prophylactic behavior reduces the peak size while prolonging the epidemic and potentially generates secondary waves of infection. These effects can be made stronger by increasing the behavioral decision frequency or distorting an individual’s perceived risk of infection.","2016-08-12","Luis G. Nardin; Craig R. Miller; Benjamin J. Ridenhour; Stephen M. Krone; Paul Joyce; Bert O. Baumgaertner",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/069013"],["1znuxoj5","fd3fc2c49f5cc27e4262261d0c9045911d65cb6e","biorxiv","XRN1 is a Species-Specific Virus Restriction Factor in Yeasts","10.1101/069799",null,null,"biorxiv","AbstractIn eukaryotes, the degradation of cellular mRNAs is accomplished by Xrn1p and the cytoplasmic exosome. Because viral RNAs often lack canonical caps or poly-A tails, they can also be vulnerable to degradation by these host exonucleases. Yeast lack sophisticated mechanisms of innate and adaptive immunity, but do use RNA degradation as an antiviral defense mechanism. One model is that the RNA of yeast viruses is subject to degradation simply as a side effect of the intrinsic exonuclease activity of proteins involved in RNA metabolism. Contrary to this model, we find a highly refined, species-specific relationship between Xrn1p and the “L-A” totiviruses of different Saccharomyces yeast species. We show that the gene XRN1 has evolved rapidly under positive natural selection in Saccharomyces yeast, resulting in high levels of Xrn1p protein sequence divergence from one yeast species to the next. We also show that these sequence differences translate to differential interactions with the L-A virus, where Xrn1p from S. cerevisiae is most efficient at controlling the L-A virus that chronically infects S. cerevisiae, and Xrn1p from S. kudriavzevii is most efficient at controlling the L-A-like virus that we have discovered within S. kudriavzevii. All Xrn1p orthologs are equivalent in their interaction with another virus-like parasite, the Ty1 retrotransposon. Thus, the activity of Xrn1p against totiviruses is not simply an incidental consequence of the enzymatic activity of Xrn1p, but rather Xrn1p co-evolves with totiviruses to maintain its potent antiviral activity and limit viral propagation in Saccharomyces yeasts. Consistent with this, we demonstrated that Xrn1p physically interacts with the Gag protein encoded by the L-A virus, suggesting a host-virus interaction that is more complicated than just Xrn1p-mediated nucleolytic digestion of viral RNAs.","2016-08-16","Paul A. Rowley; Brandon Ho; Sarah Bushong; Arlen Johnson; Sara L. Sawyer",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/069799"],["pxpujrhx","44a440cc1c135c938d2216ea672f8ef4f9c01296","biorxiv","Genome-wide Prediction of microRNAs in Zika virus Genomes Reveals Possible Interactions with Human Genes Involved in the Nervous System Development","10.1101/070656",null,null,"biorxiv","AbstractZika virus (ZIKV) is a member of the family Flaviviridae. In 2015, ZIKV triggered a large epidemic in Brazil and spread across Latin America. In November of that year, the Brazilian Ministry of Health reported a 20-fold increase in cases of neonatal microcephaly, which corresponds geographically and temporally to the ZIKV outbreak. ZIKV was isolated from the brain tissue of a fetus diagnosed with microcephaly, and recent studies in mice models revealed that ZIKV infection may cause brain defects by influencing brain cell developments. Unfortunately, the mechanisms by which ZIKV alters neurophysiological development remain unknown. MicroRNAs (miRNAs) are small noncoding RNAs that regulate post-transcriptional gene expression by translational repression. In order to gain insight into the possible role of ZIKV-mediated miRNA signaling dysfunction in brain-tissue development, we computationally predicted new miRNAs encoded by the ZIKV genome and their effective hybridization with transcripts from human genes previously shown to be involved in microcephalia. The results of these studies suggest a possible role of these miRNAs on the expression of human genes associated with this disease. Besides, a new ZIKV miRNA was predicted in the 3’stem loop (3’ SL) of the 3’untranslated region (3’UTR) of the ZIKV genome, suggesting the role of the 3’UTR of flaviviruses as a source of miRNAs.","2016-08-21","Juan Cristina; Natalia Echeverría; Fabiana Gambaro; Alvaro Fajardo; Pilar Moreno",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/070656"],["2j4z5rp8","9b5f5119bbfbded3245acc37859cefde967458e7","biorxiv","Containing Emerging Epidemics: a Quantitative Comparison of Quarantine and Symptom Monitoring","10.1101/072652",null,null,"biorxiv","ABSTRACTStrategies for containing an emerging infectious disease outbreak must be non-pharmaceutical when drugs or vaccines for the pathogen do not yet exist or are unavailable. The success of these non-pharmaceutical strategies will depend not only on the effectiveness of quarantine or other isolation measures but also on the epidemiological characteristics of the infection. However, there is currently no systematic framework to assess the relationship between different containment strategies and the natural history and epidemiological dynamics of the pathogen. Here, we compare the effectiveness of quarantine and symptom monitoring, implemented via contact tracing, in controlling epidemics using an agent-based branching model. We examine the relationship between epidemic containment and the disease dynamics of symptoms and infectiousness for seven case study diseases with diverse natural histories including Ebola, Influenza A, and Severe Acute Respiratory Syndrome (SARS). We show that the comparative effectiveness of symptom monitoring and quarantine depends critically on the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. The benefit of quarantine over symptom monitoring is generally maximized for fast-course diseases, but we show the conditions under which symptom monitoring alone can control certain outbreaks. This quantitative framework can guide policy-makers on how best to use non-pharmaceutical interventions to contain emerging outbreaks and prioritize research during an outbreak of a novel pathogen.SIGNIFICANCEQuarantine and symptom monitoring of contacts with suspected exposure to an infectious disease are key interventions for the control of emerging epidemics; however, there does not yet exist a quantitative framework for comparing the control performance of each. Here, we use a mathematical model of seven case study diseases to show how the choice of intervention is influenced by the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. We use this information to identify the most important characteristics of the disease and setting that need to be characterized for an emerging pathogen in order to make an informed decision between quarantine and symptom monitoring.","2016-08-31","Corey M Peak; Lauren M Childs; Yonatan H Grad; Caroline O Buckee",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/072652"],["kxe65hd7",null,"biorxiv","Zika virus infection as a cause of congenital brain abnormalities and Guillain-Barré syndrome: systematic review","10.1101/073098",null,null,"biorxiv","AbstractBackgroundThe World Health Organization stated in March 2016 that there was scientific consensus that the mosquito-borne Zika virus was a cause of the neurological disorder Guillain-Barré syndrome and of microcephaly and other congenital brain abnormalities, based on rapid evidence assessments. Decisions about causality require systematic assessment to guide public health actions. The objectives of this study were: to update and re-assess the evidence for causality through a rapid and systematic review about links between Zika virus infection and a) congenital brain abnormalities, including microcephaly, in the foetuses and offspring of pregnant women and b) Guillain-Barré syndrome in any population; and to describe the process and outcomes of an expert assessment of the evidence about causality.Methods and findingsThe study had three linked components. First, in February 2016, we developed a causality framework that defined questions about the relationship between Zika virus infection and each of the two clinical outcomes in 10 dimensions; temporality, biological plausibility, strength of association, alternative explanations, cessation, dose-response, animal experiments, analogy, specificity and consistency. Second, we did a systematic review (protocol number CRD42016036693). We searched multiple online sources up to May 30, 2016 to find studies that directly addressed either outcome and any causality dimension, used methods to expedite study selection, data extraction and quality assessment, and summarised evidence descriptively. Third, a multidisciplinary panel of experts assessed the review findings and reached consensus on causality. We found 1091 unique items up to May 30, 2016. For congenital brain abnormalities, including microcephaly, we included 72 items; for eight of 10 causality dimensions (all except dose-response relationship and specificity) we found that more than half the relevant studies supported a causal association with Zika virus infection. For Guillain-Barré syndrome, we included 36 items, of which more than half the relevant studies supported a causal association in seven of ten dimensions (all except dose-response relationship, specificity and animal experimental evidence). Articles identified non-systematically from May 30-July 29, 2016 strengthened the review findings. The expert panel concluded that: a) the most likely explanation of available evidence from outbreaks of Zika virus infection and clusters of microcephaly is that Zika virus infection during pregnancy is a cause of congenital brain abnormalities including 61 microcephaly; and b) the most likely explanation of available evidence from outbreaks of Zika virus infection and Guillain-Barré syndrome is that Zika virus infection is a trigger of Guillain-Barré syndrome. The expert panel recognised that Zika virus alone may not be sufficient to cause either congenital brain abnormalities or Guillain-Barré syndrome but agreed that the evidence was sufficient to recommend increased public health measures. Weaknesses are the limited assessment of the role of dengue virus and other possible co-factors, the small number of comparative epidemiological studies, and the difficulty in keeping the review up to date with the pace of publication of new research.ConclusionsRapid and systematic reviews with frequent updating and open dissemination are now needed, both for appraisal of the evidence about Zika virus infection and for the next public health threats that will emerge. This rapid systematic review found sufficient evidence to say that Zika virus is a cause of congenital abnormalities and is a trigger of Guillain-Barré situation.","2016-09-02","Fabienne Krauer; Maurane Riesen; Ludovic Reveiz; Olufemi T Oladapo; Ruth Martínez-Vega; Teegwendé V. Porgo; Anina Haefliger; Nathalie J Broutet; Nicola Low",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/073098"],["8elmjre9","8f4b98da50277e6c9516aef0227f5cca9af6b5e8","biorxiv","Identification of quercetin from fruits to immediately fight Zika","10.1101/074559",null,null,"biorxiv","Zika virus is spread mainly by the bite of an infected mosquito, which can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika in the near future, but unfortunately so far there is no vaccine or medicine for Zika. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the key role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. In response to the current global health emergency triggered by the Zika outbreak, we successfully obtained several active forms of Zika NS2B-NS3pro and further attempted to discover its inhibitors from eatable plants and traditional herbal medicines to immediately fight Zika. Here, for the first time, we discovered that quercetin, a flavonoid extensively existing in many fruits and vegetables, effectively inhibits Zika NS2B-NS3pro. We further quantify its inhibitory activity with IC50 of 26 uM; and Ki of 23 uM. As quercetin has been extensively found in fruits, vegetables, leaves and grains, our discovery would benefit the public to immediately fight Zika.","2016-09-11","Amrita Roy; Liangzhong Lim; Jianxing Song",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/074559"],["l82aakrk","f9b108c052ded463773a9838a8157ac7d5ba6d1a","biorxiv","Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations","10.1101/078113",null,null,"biorxiv","ABSTRACTZika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short β-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors.","2016-09-28","Amrita Roy; Liangzhong Lim; Shagun Srivastava; Jianxing Song",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/078113"],["gfiukh90","5b27a16123f46b5aaeda3171d22e075dcbd36ee6","biorxiv","Identification of a Zika NS2B-NS3pro pocket susceptible to allosteric inhibition by small molecules including qucertin rich in edible plants","10.1101/078543",null,null,"biorxiv","ABSTRACTIt has been recently estimated that one-third of the world population will be infected by Zika virus, but unfortunately so far there is no vaccine or medicine available. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the irreplaceable role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. Very recently we identified two small molecules inhibit Zika NS2B-NS3pro in non-competitive mode, with Ki values of 0.57 and 2.02 µM respective for p-Nitrophenyl-p-guanidino benzoate and qucertin. Here, by molecular docking, we show that although one is designed compound while another is a natural product, both molecules bind to the same pocket on the back of the substrate-binding pocket of Zika NS2B-NS3pro. As the two inhibitors fundamentally differ from cn-716, the only known peptidomimetic boronic acid inhibitor in both structure scaffolds and inhibitory modes, our discovery might open up a new avenue for the future development of allosteric inhibitors, which is highly demanded to achieve therapeutic inhibition of flaviviral NS2B-NS3pro complexes. Furthermore, as qucertin is abundant in many vegetables and fruits such caper, lovage, tea and red onion, our results should benefit the public to immediately fight Zika virus.","2016-10-01","Liangzhong Lim; Amrita Roy; Jianxing Song",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/078543"],["mkoyc7wh",null,"biorxiv","Comparative analysis of variation and selection in the HCV genome","10.1101/078584",null,null,"biorxiv","AbstractGenotype 1 of the hepatitis C virus (HCV) is the most prevalent of the variants of this virus. Its two main subtypes, HCV-1a and HCV-1b, are associated to differences in epidemic features and risk groups, despite sharing similar features in most biological properties. We have analyzed the impact of positive selection on the evolution of these variants using complete genome coding regions, and compared the levels of genetic variability and the distribution of positively selected sites. We have also compared the distributions of positively selected and conserved sites considering different factors such as RNA secondary structure, the presence of different epitopes (antibody, CD4 and CD8), and secondary protein structure. Less than 10% of the genome was found to be under positive selection, and purifying selection was the main evolutionary force in both subtypes. We found differences in the number of positively selected sites between subtypes in several genes (Core, HVR2 in E2, P7, helicase in NS3 and NS4a). Heterozygosity values in positively selected sites and the rate of non-synonymous substitutions were significantly higher in subtype HCV-1b. Logistic regression analyses revealed that similar selective forces act at the genome level in both subtypes: RNA secondary structure and CD4 T-cell epitopes are associated with conservation, while CD8 T-cell epitopes are associated with positive selection in both subtypes. These results indicate that similar selective constraints are acting along HCV-1a and HCV-1b genomes, despite some differences in the distribution of positively selected sites at independent genes.","2016-09-30","Juan Ángel Patiño-Galindo; Fernando González-Candelas",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/078584"],["lj0ar2v5","28b107243576723248ad4053261000311a22f134","biorxiv","Genetic, transcriptome, proteomic and epidemiological evidence for blood brain barrier disruption and polymicrobial brain invasion as determinant factors in Alzheimer’s disease","10.1101/080333",null,null,"biorxiv","AbstractMultiple pathogens have been detected in Alzheimer’s disease (AD) brains. A bioinformatics approach was used to assess relationships between pathogens and AD genes (GWAS), the AD hippocampal transcriptome and plaque or tangle proteins. Host/pathogen interactomes (C.albicans, C.Neoformans, Bornavirus, B.Burgdorferri, cytomegalovirus, Ebola virus, HSV-1, HERV-W, HIV-1, Epstein-Barr, hepatitis C, influenza, C.Pneumoniae, P.Gingivalis, H.Pylori, T.Gondii, T.Cruzi) significantly overlap with misregulated AD hippocampal genes, with plaque and tangle proteins and, except Bornavirus, Ebola and HERV-W, with AD genes. Upregulated AD hippocampal genes match those upregulated by multiple bacteria, viruses, fungi or protozoa in immunocompetent blood cells. AD genes are enriched in bone marrow and immune locations and in GWAS datasets reflecting pathogen diversity, suggesting selection for pathogen resistance. The age of AD patients implies resistance to infections afflicting the younger. APOE4 protects against malaria and hepatitis C, and immune/inflammatory gain of function applies to APOE4, CR1, TREM2 and presenilin variants. 30/78 AD genes are expressed in the blood brain barrier (BBB), which is disrupted by AD risk factors (ageing, alcohol, aluminium, concussion, cerebral hypoperfusion, diabetes, homocysteine, hypercholesterolaemia, hypertension, obesity, pesticides, pollution, physical inactivity, sleep disruption and smoking). The BBB and AD benefit from statins, NSAIDs, oestrogen, melatonin and the Mediterranean diet. Polymicrobial involvement is supported by the upregulation of pathogen sensors/defenders (bacterial, fungal, viral) in the AD brain, blood or CSF. Cerebral pathogen invasion permitted by BBB inadequacy, activating a hyper-efficient immune/inflammatory system, betaamyloid and other antimicrobial defence may be responsible for AD which may respond to antibiotic, antifungal or antiviral therapy.","2016-10-12","C.J. Carter",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/080333"],["w49x7kca","e6fe63419fe399f9c77513c1c11a043db006a2f3","biorxiv","Accurate prediction of human essential genes using only nucleotide composition and association information","10.1101/084129",null,null,"biorxiv","AbstractThree groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a λ-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier’s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (https://cefg.uestc.edu.cn/Pheg), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the λ-interval Z curve method could be effectively extended to classification issues of other DNA elements.","2016-10-28","Feng-Biao Biao; Chuan Dong; Hong-Li Hua; Shuo Liu; Hao Luo; Hong-Wan Zhang; Yan-Ting Jin; Kai-Yue Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/084129"],["phmm596b","dff716159738acd6bd53b4be12af45358a58752e","biorxiv","MYCOPLASMA FAUCIUM AND BREAST CANCER","10.1101/089128",null,null,"biorxiv","Viruses and bacteria are the cause of a large number of different human diseases. It is believed that some of them may even contribute to the development of cancer. The present work is dedicated to the identification of mycoplasmas in patients with breast cancer. Mycoplasmas may participate in the development of several human diseases including chronic fatigue syndrome, acquired immunodeficiency syndrome, atypical pneumonia, etc. Moreover, there is a reason to believe that mycoplasma can participate in the development of cancer, leukemia and lymphoma. DNA samples from blood, saliva and tumor tissues of the Oncology Institute of Moldova patients diagnosed with breast cancer were analyzed. Mycoplasma testing was performed using nested PCR method. For Mycoplasma spp. detection, we used primers from the region of the 16S-23S RNA genes. The identification of Mycoplasma faucium, Mycoplasma salivarium and Mycoplasma orale was performed by nested PCR with primers for RNA polymerase beta subunit gene corresponding to mycoplasma. M.faucium and M.salivarius was found in saliva at about 100%, and M.orale at a frequency of about 50%. Only M.faucium was found with the frequency of about 60% in the tissue of the patients. Moreover, a fairly high rate of detection of mycoplasma is observed both in the cases when primers for RNA polymerase gene and primers for 16S-23S RNA were used. We found M.faucium in tumor tissues of patients diagnosed with breast cancer. It is known that mycoplasmas are able to stimulate the synthesis of certain cytokines, which act as mitogenes on the cell. We assume that M.faucium can stimulate the mitogenes synthesis in breast tissues (e.g., cytokines) which, in turn, stimulate cell division and thus participate in the initiation of breast cancer.","2016-11-22","Valentin Mitin; Lidia Tumanova; Natalia Botnariuc",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/089128"],["iedh762s","8c900409a679dc1f16ef13c73500cbb53605b683","biorxiv","Lung epithelial cells have virus-specific and shared gene expression responses to infection by diverse respiratory viruses","10.1101/090936",null,null,"biorxiv","AbstractThe severity and outcome of respiratory viral infections is partially determined by the cellular response mounted by infected lung epithelial cells. Disease prevention and treatment is dependent on our understanding of the shared and unique responses elicited by diverse viruses, yet few studies compare host responses to different viruses while controlling other experimental parameters. We compared changes in gene expression of murine lung epithelial cells infected individually by three respiratory viruses causing mild (rhinovirus, RV1B), moderate (coronavirus, MHV-1), and severe (influenza A virus, PR8) disease in mice. RV1B infection caused numerous gene expression changes, but the differential effect peaked at 12 hours post-infection. PR8 altered an intermediate number of genes whose expression continued to change through 24 hours. MHV-1 had comparatively few effects on host gene expression. The viruses elicited highly overlapping responses in antiviral genes, though MHV-1 induced a lower type I interferon response than the other two viruses. Signature genes were identified for each virus and included host defense genes for PR8, tissue remodeling genes for RV1B, and transcription factors for MHV-1. Our comparative approach identified universal and specific transcriptional signatures of virus infection that can be used to discover mechanisms of pathogenesis in the respiratory tract.","2016-12-02","James T. VanLeuven; Benjamin J. Ridenhour; Craig R. Miller; Tanya A. Miura",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/090936"],["2179jaes","6aa2716c00d89f4b760a9167b15d6203303ae6b9","biorxiv","Vitrification after multiple rounds of sample application and blotting improves particle density on cryo-electron microscopy grids","10.1101/094623",null,null,"biorxiv","Single particle cryo-electron microscopy (cryoEM) is becoming widely adopted as a tool for structural characterization of biomolecules at near-atomic resolution. Vitrification of the sample to obtain a dense distribution of particles within a single field of view remains a major bottleneck for the success of such experiments. Here, we describe a simple and cost-effective method to increase the density of frozen-hydrated particles on grids with holey carbon support films. It relies on performing multiple rounds of sample application and blotting prior to plunge freezing in liquid ethane. We show that this approach is generally applicable and significantly increases particle density for a range of samples, such as small protein complexes, viruses and filamentous assemblies. The method is versatile, easy to implement, minimizes sample requirements and can enable characterization of samples that would otherwise resist structural studies using single particle cryoEM.","2016-12-15","Joost Snijder; Andrew J. Borst; Annie Dosey; Alexandra C. Walls; Anika Burrell; Vijay S. Reddy; Justin M. Kollman; David Veesler",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/094623"],["ahdz5078","8eb99586e9103599ab2f58cc33ca94cd0b60ae53","biorxiv","Epidemiological and ecological modelling reveal diversity in upper respiratory tract microbial population structures from a cross-sectional community swabbing study","10.1101/099069",null,null,"biorxiv","ABSTRACTRespiratory tract infections (RTI) are responsible for over 4 million deaths per year worldwide with pathobiont carriage a required precursor to infection. Through a cross-sectional community-based nasal self-swabbing study we sought to determine carriage epidemiology for respiratory pathogens amongst bacteria (Streptococcus pneumoniae, Haemophilus influenza, Moraxella catarrhalis, Staphylococcus aureus, Pseudomonas aeruginosa and Neisseria meningitidis) and viruses (RSV, Influenza viruses A and B, Rhinovirus/Enterovirus, Coronavirus, Parainfluenza viruses 1-3 and Adenovirus (ADV)). Carriage of bacterial and viral species was shown to vary with participant age, recent RTI and the presence of other species. The spatial structure of microbial respiratory communities was less nested (more disordered) in the young (0-4 years) and those with recent RTI. Species frequency distributions were flatter than random expectation in young individuals (X2 = 20.42, p = 0.002), indicating spatial clumping of species consistent with facilitative relationships amongst them. Deviations from a neutral model of ecological niches were observed for samples collected in the summer and from older individuals (those aged 5-17, 18-64 and ≥65 years) but not in samples collected from winter, younger individuals (those aged 0-4 years), individuals with recent RTI and individuals without recent RTI, demonstrating the importance of both neutral and niche processes in respiratory community assembly. The application of epidemiological methods and ecological theory to sets of respiratory tract samples has yielded novel insights into the factors that drive microbial community composition, such as seasonality and age, as well as species patterns and interactions within the nose.","2017-01-09","Abigail L. Coughtrie; Denise E. Morris; Rebecca Anderson; Nelupha Begum; David W. Cleary; Saul N. Faust; Johanna M. Jefferies; Alex R. Kraaijeveld; Michael V. Moore; Mark A. Mullee; Paul J. Roderick; Andrew Tuck; Robert N. Whittaker; Ho Ming Yuen; C. Patrick Doncaster; Stuart C. Clarke",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/099069"],["pl7t3hca","e0fd364797eb02a280e25e2649dfe6d6c0d33e05","biorxiv","Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species","10.1101/101774",null,null,"biorxiv","AbstractTaxonomic classification for the Coronaviridae can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the Alphacoronavirus genus, there is a single species Alphacoronavirus 1 that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the Alphacoronavirus genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the Alphacoronavirus 1 species. Alphacoronavirus 1 clade A encompasses serotype I FCoV and CCoV, and Alphacoronavirus 1 clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed Alphacoronavirus 2 species, as an improved way to classify the diverse Alphacoronavirus genus.","2017-01-19","Gary R. Whittaker; Nicole M. André; Jean Kaoru Millet",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/101774"],["km0b1z64","d418cdee18b07a0adfb553d43b4a08f7736fa9f3","biorxiv","Metagenomic Sequencing Detects Respiratory Pathogens in Hematopoietic Cellular Transplant Patients","10.1101/102798",null,null,"biorxiv","ABSTRACTRATIONALECurrent microbiologic diagnostics often fail to identify the etiology of lower respiratory tract infections (LRTI) in hematopoietic cellular transplant recipients (HCT), which precludes the implementation of targeted therapies.OBJECTIVESTo address the need for improved LRTI diagnostics, we evaluated the utility of metagenomic next generation sequencing (mNGS) of bronchoalveolar lavage (BAL) to detect microbial pathogens in HCT patients with acute respiratory illnesses.METHODSWe enrolled 22 post-HCT adults ages 19-69 years with acute respiratory illnesses who underwent BAL at the University of Michigan between January 2012 and May 2013. mNGS was performed on BAL fluid to detect microbes and simultaneously assess the host transcriptional response. Results were compared against conventional microbiologic assays.MEASUREMENTS & MAIN RESULTSmNGS demonstrated 100% sensitivity for detecting respiratory microbes (human metapneumovirus, respiratory syncytial virus, Stenotrophomonas maltophilia, human herpesvirus 6 and cytomegalovirus) when compared to standard testing. Previously unrecognized LRTI pathogens were identified in six patients for whom standard testing was negative (human coronavirus 229E, human rhinovirus A, Corynebacterium propinquum and Streptococcus mitis); findings were confirmed by independent PCR and 16S rRNA sequencing. Relative to patients without infection, patients with infection had increased expression of immunity related genes (p=0.022) and significantly lower diversity of their respiratory microbiome (p=0.017).CONCLUSIONSCompared to conventional diagnostics, mNGS enhanced detection of pathogens in BAL fluid from HCT patients. Furthermore, our results suggest that combining unbiased microbial pathogen detection with assessment of host gene biomarkers of immune response may hold promise for enhancing the diagnosis of post-HCT respiratory infections.","2017-01-24","C Langelier; MS Zinter; K Kalantar; GA Yanik; S Christenson; B Odonovan; C White; M Wilson; A Sapru; CC Dvorak; S Miller; CY Chiu; JL DeRisi",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/102798"],["ois9fiuq",null,"biorxiv","The immunomodulatory CEA cell adhesion molecule 6 (CEACAM6/CD66c) is a candidate receptor for the influenza A virus","10.1101/104026",null,null,"biorxiv","AbstractTo establish a productive infection in host cells, viruses often use one or multiple host membrane glycoprotein as their receptors. For Influenza A virus (IAV) such a glycoprotein receptor has not been described, to date. Here we show that IAV is using the host membrane glycoprotein CD66c as a receptor for entry into human epithelial lung cells. Neuraminidase (NA), a viral spike protein binds to CD66c on the cell surface during IAV entry into the host cells. Lung cells overexpressing CD66c showed an increase in virus binding and subsequent entry into the cell. Upon comparison, CD66c demonstrated higher binding capacity than other membrane glycoproteins (EGFR and DC-SIGN) reported earlier to facilitate IAV entry into host cells. siRNA mediated knockdown of CD66c from lung cells inhibited virus binding on cell surface and entry into cells. Blocking CD66c by antibody on the cell surface resulted in decreased virus entry. We found CD66c is a specific glycoprotein receptor for influenza A virus that did not affect entry of non-IAV RNA virus (Hepatitis C virus). Finally, IAV pre-incubated with recombinant CD66c protein when administered intranasally in mice showed decreased cytopathic effects in mice lungs. This publication is the first to report CD66c (CEACAM6) as a glycoprotein receptor for Influenza A virus.Significance StatementCells are enclosed by a semipermeable membrane that allows selective exchange of biomolecules between cells and their surroundings. A set of specialized proteins in this semipermeable membrane, work like gatekeepers to the cell and regulate entry of these biomolecules. One class of such surface proteins is termed as receptors. Viruses bind to one or more of these receptors and manipulate gatekeepers for their own successful entry into host-cells. A membrane protein that influenza A virus (Flu virus) uses for entry into the cells was not discovered till date. This study reports for the first time, a receptor for influenza A virus, that was sought after by researchers for decades. The viral receptor is a promising target that can be used to inhibit virus entry into host cells.","2017-01-30","Shah Kamranur Rahman; Mairaj Ahmed Ansari; Pratibha Gaur; Imtiyaz Ahmad; Chandrani Chakravarty; Dileep Kumar Verma; Sanjay Chhibber; Naila Nehal; Shanmugaapriya Sellathanby; Dagmar Wirth; Gulam Waris; Sunil K. Lal",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/104026"],["9pa51d9x","95754ec84bdee0637765a5ee34983a5f6cc86eed","biorxiv","PGC-1α coordinates with Bcl-2 to control cell cycle in U251 cells through reducing ROS","10.1101/111567",null,null,"biorxiv","AbstractB-cell lymphoma 2 (Bcl-2) has a dual function, acting both as an oncogene and an anti-tumor gene. It is well known that Bcl-2 exerts its tumor promoting function through the mitochondrial pathway. However, the mechanism by which Bcl-2 suppresses tumor formation is not well understood. We have previously shown that Bcl-2 inhibits cell cycle progression from the G0/G1 to the S phase after serum starvation, and that quiescent Bcl-2 expressing cells maintained a significant lower level of mitochondrial reactive oxygen species (ROS) than the control cells. Based on the fact that ROS mediate cell cycle progression, and are controlled by peroxisome proliferator-activated receptor-γ co-activator 1α (PGC-1α), a key molecule induced by prolonged starvation and involved in mitochondrial metabolism, we hypothesized that PGC-1α might be related with the cell cycle function of Bcl-2. Here, we showed that PGC-1α was upregulated upon Bcl-2 overexpression and downregulated following Bcl-2 knockdown during serum starvation. Knockdown of PGC-1α activated Bcl-2 expression. Taken together, our results suggest that after serum depletion, PGC-la might coordinate with Bcl-2 to reduce ROS, which in turn delay cell cycle progression.Summary statementPGC-1α coordinate with Bcl-2 delay cell cycle progression to reduce ROS after serum depletion in human glioma U251 cells.","2017-02-24","Kun Yao; Xufeng Fu; Du Xing; Yan Li; Bing Han; Chen Zexi; Shanshan Yang; Ran Wei; Jiaqi Zhou; Qinghua Cui",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/111567"],["mudv1ejd","b4fc44eabb1f640124fff98990c85dc5c9984aee","biorxiv","The translational landscape of Zika virus during infection of mammalian and insect cells","10.1101/112904",null,null,"biorxiv","AbstractZika virus is a single-stranded, positive-sense RNA virus of the family Flaviviridae, which has recently undergone a rapid expansion among humans in the Western Hemisphere. Here, we report a high-resolution map of ribosomal occupancy of the Zika virus genome during infection of mammalian and insect cells, obtained by ribosome profiling. In contrast to some other flaviviruses such as West Nile, we find no evidence for substantial frameshift-induced ribosomal drop-off during translation of the viral polyprotein, indicating that Zika virus must use alternative mechanisms to downregulate levels of catalytically active viral polymerase. We also show that high levels of ribosome-protected fragments map in-frame to two previously overlooked upstream open reading frames (uORFs) initiating at CUG and UUG codons, with likely consequences for the efficiency of polyprotein expression. Curiously, in African isolates of Zika virus, the two uORFs are fused in-frame into a single uORF. A parallel RNA-Seq analysis reveals the 5′ end position of the subgenomic flavivirus RNA in mammalian and insect cells. Together, these provide the first analysis of flavivirus gene expression by ribosome profiling.Author SummaryRecent Zika virus outbreaks have been associated with congenital diseases and neurological complications. An enhanced understanding of the molecular biology of this pathogen may contribute towards the development of improved treatment and control methods. We present a single-codon resolution analysis of Zika virus translation in mammalian and mosquito cells using ribosome profiling. The analysis revealed two hitherto uncharacterized uORFs in the 5′ leader of Zika virus Brazilian isolate PE243, both of which are occupied by ribosomes during infection. In contrast, these two uORFs are fused into a single uORF in African isolates. This observation provides a new avenue for further investigations into potential factors involved in the emergence of Zika virus from a rarely detected pathogen into a major epidemic.","2017-03-02","Nerea Irigoyen; Adam M. Dinan; Luke W. Meredith; Ian Goodfellow; Ian Brierley; Andrew E. Firth",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/112904"],["d0y1hs88",null,"biorxiv","Myeloablation-associated deletion of ORF4 in a human coronavirus 229E infection","10.1101/113423",null,null,"biorxiv","AbstractWe describe metagenomic next-generation sequencing (mNGS) of a human coronavirus 229E from a patient with AML and persistent upper respiratory symptoms, who underwent hematopoietic cell transplantation (HCT). mNGS revealed a 548-nucleotide deletion, which comprised the near entirety of the ORF4 gene, and no minor allele variants were detected to suggest a mixed infection. As part of her pre-HCT conditioning regimen, the patient received myeloablative treatment with cyclophosphamide and 12 Gy total body irradiation. Iterative sequencing and RT-PCR confirmation of 4 respiratory samples over the 4-week peritransplant period revealed that the pre-conditioning strain contained an intact ORF4 gene, while the deletion strain appeared just after conditioning and persisted over a 2.5-week period. This sequence represents one of the largest genomic deletions detected in a human RNA virus and describes large-scale viral mutation associated with myeloablation for HCT.","2017-03-03","Alexander L. Greninger; Gregory Pepper; Ryan C. Shean; Anne Cent; Isabel Palileo; Jane M. Kuypers; Joshua T. Schiffer; Keith R. Jerome",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/113423"],["0abutzb3","d91120d80916351170b34715ea4ab141f02c51fb","biorxiv","Prophage genomics reveals patterns in phage genome organization and replication","10.1101/114819",null,null,"biorxiv","AbstractTemperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host’s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts’ translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution.","2017-03-07","Han Suh Kang; Katelyn McNair; Daniel A. Cuevas; Barbara A. Bailey; Anca M. Segall; Robert A. Edwards",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/114819"],["jopeo9gs","0cb9c296684ca5e71462d825cab2827854a01544","biorxiv","p53 is not necessary for DUX4 pathology","10.1101/118315",null,null,"biorxiv","AbstractsFSHD is a genetically dominant myopathy caused by mutations that cause expression of the normally silent DUX4 gene. This transcription factor has been shown to interfere with myogenesis when misexpressed at very low levels in myoblasts, and to cause cell death when overexpressed at high levels. A previous report using adeno-associated virus to deliver high levels of DUX4 to mouse skeletal muscle demonstrated severe pathology that was suppressed on a p53 knockout background, implying that DUX4 acted through the p53 pathway. Here, we investigate the p53-dependence of DUX4 using both in vitro cellular and the transgenic iDUX4[2.7] mouse models. We find that inhibiting p53 has no effect on the cytoxicity of DUX4 in vitro. When crossed onto the p53 null background, we find no suppression of the male-specific lethality or skin phenotypes of the DUX4 transgene, and find that primary myoblasts from this mouse are still killed by DUX4 expression. These data challenge the notion that the p53 pathway is central to the pathogenicity of DUX4.Summary StatementDUX4 is thought to mediate cytopathology through p53. Here, DUX4 is shown to kill primary myoblasts and promote pathological phenotypes in the iDUX4[2.7] mouse model on the p53-null background, calling into question this notion.","2017-03-19","Darko Bosnakovski; Erik A. Toso; Olivia O. Recht; Anja Cucak; Abhinav K Jain; Michelle C. Barton; Michael Kyba",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/118315"],["4it5c9n2","f122e125c2cd3580b1a38e27c780b99f671fd78f","biorxiv","An amino acid motif in HLA-DRβ1 distinguishes patients with uveitis in juvenile idiopathic arthritis","10.1101/140954",null,null,"biorxiv","AbstractUveitis is a visually-debilitating disorder that affects up to 30% of children with juvenile idiopathic arthritis (JIA). To identify genetic susceptibility loci for uveitis in JIA, we conducted a genome-wide association study comparing 192 JIA-associated uveitis cases with 330 JIA individuals without uveitis. Two cohorts of JIA patients underwent genotyping and quality control. We used an HLA-specific imputation panel to impute HLA-specific amino acids and HLA types, and identified the amino acid serine at position 11 (serine-11) in HLA- DRB1 as associated to increased risk of uveitis (OR = 2.60, p = 5.43×10−10). The signal at serine-11 was female-specific (interaction of sex and serine-11, p = 0.0096). Serine-11 resides in the YST-motif (positions 10-12) in the peptide binding groove of HLA-DRB1. Quantitative binding affinity predictions revealed peptide-binding preferences that distinguish HLA-DRB1 allotypes with the YST-motif. Our findings highlight a genetically distinct, sexually-dimorphic feature of JIA-associated uveitis.","2017-05-22","A.J.W. Haasnoot; M.W. Schilham; S.S.M. Kamphuis; P.C.E. Hissink Muller; A. Heiligenhaus; D. Foell; R.A. Ophoff; T.R.D.J. Radstake; A.I. Den Hollander; T.H.C.M. Reinards; S. Hiddingh; N. Schalij-Delfos; E.P.A.H. Hoppenreijs; M.A.J. van Rossum; C. Wouters; R.K. Saurenmann; N. Wulffraat; R. ten Cate; J.H. de Boer; S.L. Pulit; J.J.W. Kuiper",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/140954"],["pt0kes0s","88e3b43319399c339186c02c44d369228c35eef4","biorxiv","An anti-cancer binary system activated by bacteriophage HK022 Integrase","10.1101/147736",null,null,"biorxiv","ABSTRACTCancer gene therapy is a great promising tool for cancer therapeutics due to the specific targeting based on the cancerous gene expression background. Binary systems based on site-specific recombination are one of the most effective potential approaches for cancer gene therapy. In these systems, a cancer specific promoter expresses a site-specific recombinase/integrase that in turn controls the expression of a toxin gene. In the current study, we have developed a new HK022 bacteriophage Integrase (Int) based binary system activating a Diphtheria toxin (DTA) gene expression specifically in cancer cells. We have demonstrated the efficiency, and the high specificity of the system in vitro in cell cultures and in vivo in a lung cancer mouse model. Strikingly, different apoptotic and anti-apoptotic factors demonstrated a remarkable efficacy killing capability of the Int-based binary system compared to the conventional hTERT-DTA mono system in the LLC-Kat lung cancer mice model; we observed that the active hTERT promoter down regulation by the transcription factors Mad-1 is the cornerstone of this phenomenon. The new Int-based binary system offers advantages over already known counterparts and may therefore be developed into a safer and efficient cancer treatment technology.","2017-06-12","Amer Elias; Itay Spector; Natasha Gritsenko; Yael Zilberstein; Rena Gorovits; Gali Prag; Mikhail Kolot",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/147736"],["9e3z9xdd","0b4b4e5bb8d0d5167eec1e203b5dad283bd364a5","biorxiv","Doubling healthy lifespan using drug synergy","10.1101/153205",null,null,"biorxiv","SummaryPharmacological interventions that target human ageing would extend individual healthspan and result in dramatic economic benefits to rapidly ageing societies worldwide. For such interventions to be contemplated they need to comprise drugs that are efficacious when given to adults and for which extensive human safety data are available. Here we show that dramatic lifespan extension can be achieved in C.elegans by targeting multiple, evolutionarily conserved ageing pathways using drugs that are already in human use. By targeting multiple synergistic ageing pathways, we are able to slow ageing rate, double lifespan and improves healthspan while minimize developmental and fitness trade-offs. Moreover, we established that there is no synergistic benefit in a daf-2 or daf-7 background, implying the involvement of the TGFβ and IGF pathways in this synergy. Employing lipidomics and transcriptomics analysis we found lipid metabolism to be affected resulting in increased monounsaturated fatty acids (MUFA) and decrease membrane peroxidation index. Our best drug combination showed a conserved lifespan extension in fruit flies. To the best of our knowledge, this is the largest lifespan effect ever reported for any adult-onset drug treatment in C. elegans. This drug-repurposing approach, using drugs already approved for humans to target multiple conserved aging pathways simultaneously, could lead to interventions that prevent age-related diseases and overall frailty in a rapidly ageing population.","2017-06-21","Tesfahun Dessale; Krishna Chaithanya Batchu; Diogo Barardo; Ng Li Fang; Vanessa Yuk Man Lam; Linfan Xiao; Markus R. Wenk; Nicholas S. Tolwinski; Jan Gruber",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/153205"],["9y8r277c","90e28ff462882ca7a9329beb879f73c2e99430e4","biorxiv","Evolutionary proteomics uncovers ciliary signaling components","10.1101/153437",null,null,"biorxiv","ABSTRACTCilia are organelles specialized for movement and signaling. To infer when during animal evolution signaling pathways became associated with cilia, we characterized the proteomes of cilia from three organisms: sea urchins, sea anemones and choanoflagellates. From these ciliomes, we identified 437 high confidence ciliary candidate proteins conserved in mammals, including known regulators of Hh, GPCR and TRP channel signaling. The phylogenetic profiles of their ciliary association indicate that the Hh and GPCR pathways were linked to cilia before the origin of bilateria and TRP channels before the origin of animals. We demonstrated that some of the candidates not previously implicated in ciliary biology localized to cilia and further investigated ENKUR, a TRP channel-interacting protein that we identified in the cilia of all three organisms. In animals, ENKUR is expressed by cells with motile cilia, ENKUR localizes to cilia in diverse organisms and, in both Xenopus laevis and mice, ENKUR is required for patterning the left/right axis. Moreover, mutation of ENKUR causes situs inversus in humans. Thus, proteomic profiling of cilia from diverse eukaryotes defines a conserved ciliary proteome, reveals ancient connections to Hh, GPCR and TRP channel signaling, and uncovers a novel ciliary protein that controls vertebrate development and human disease.","2017-06-22","Monika Abedin Sigg; Tabea Menchen; Jeffery Johnson; Chanjae Lee; Semil P. Choksi; Galo Garcia; Henriette Busengdal; Gerard Dougherty; Petra Pennekamp; Claudius Werner; Fabian Rentzsch; Nevan Krogan; John B. Wallingford; Heymut Omran; Jeremy F. Reiter",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/153437"],["7xmn8yvc",null,"biorxiv","Mapping the drivers of within-host pathogen evolution using massive data sets","10.1101/155242",null,null,"biorxiv","AbstractDifferences among hosts, resulting from genetic variation in the immune system or heterogeneity in drug treatment, can impact within-host pathogen evolution. Identifying such interactions can potentially be achieved through genetic association studies. However, extensive and correlated genetic population structure in hosts and pathogens presents a substantial risk of confounding analyses. Moreover, the multiple testing burden of interaction scanning can potentially limit power. To address these problems, we have developed a Bayesian approach for detecting host influences on pathogen evolution that makes use of vast existing data sets of pathogen diversity to improve power and control for stratification. The approach models key processes, including recombination and selection, and identifies regions of the pathogen genome affected by host factors. Using simulations and empirical analysis of drug-induced selection on the HIV-1 genome we demonstrate the power of the method to recover known associations and show greatly improved precision-recall characteristics compared to other approaches. We build a high-resolution map of HLA-induced selection in the HIV-1 genome, identifying novel epitope-allele combinations.","2017-06-25","Duncan S. Palmer; Isaac Turner; Sarah Fidler; John Frater; Philip Goulder; Dominique Goedhals; Kuan-Hsiang Gary Huang; Annette Oxenius; Rodney Phillips; Roger Shapiro; Cloete van Vuuren; Angela R. McLean; Gil McVean",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/155242"],["jhwjh12k","7a51b191fd6575831585eee67a0ac67f209d8fcb","biorxiv","Nuclease escape elements protect messenger RNA against cleavage by multiple viral endonucleases","10.1101/155457",null,null,"biorxiv","ABSTRACTDuring lytic Kaposi’s sarcoma-associated herpesvirus (KSHV) infection, the viral endonu-clease SOX promotes widespread degradation of cytoplasmic messenger RNA (mRNA). However, select mRNAs, including the transcript encoding interleukin-6 (IL-6), escape SOX-induced cleavage. IL-6 escape is mediated through a 3’ UTR RNA regulatory element that overrides the SOX targeting mechanism. Here, we reveal that this protective RNA element functions to broadly restrict cleavage by a range of homologous and non-homologous viral endonucleases. However, it does not impede cleavage by cellular endonucleases. The IL-6 protective sequence may be representative of a larger class of nuclease escape elements, as we identified a similar protective element in the GADD45B mRNA. The IL-6 and GADD45B-derived elements display similarities in their sequence, putative structure, and several associated RNA binding proteins. However, the overall composition of their ribonucleoprotein complexes appears distinct, leading to differences in the breadth of nucleases restricted. These findings highlight how RNA elements can selectively control transcript abundance in the background of widespread virus-induced mRNA degradation.AUTHOR SUMMARYThe ability of viruses to control the host gene expression environment is crucial to promote viral infection. Many viruses express factors that reduce host gene expression through widespread mRNA decay. However, some mRNAs escape this fate, like the transcript encoding the immunoregulatory cytokine IL-6 during KSHV infection. IL-6 escape relies on an RNA regulatory element located in its 3’UTR and involves the recruitment of a protective protein complex. Here, we show that this escape extends beyond KSHV to a variety of related and unrelated viral endonucleases. However, the IL-6 element does not protect against cellular endonucleases, revealing for the first time a virus-specific nuclease escape element. We identified a related escape element in the GADD45B mRNA, which displays several similarities with the IL-6 element. However, these elements assemble a largely distinct complex of proteins, leading to differences in the breadth of their protective capacity. Collectively, these findings reveal how a putative new class of RNA elements function to control RNA fate in the background of widespread mRNA degradation by viral endonucleases.","2017-06-26","Mandy Muller; Britt A. Glaunsinger",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/155457"],["h89scli5","b5161b031c7f720562e94735a018d1c3c8be3ae5","biorxiv","Quantifying the Risk and Cost of Active Monitoring for Infectious Diseases","10.1101/156497",null,null,"biorxiv","AbstractDuring outbreaks of deadly emerging pathogens (e.g., Ebola, MERS-CoV) and bioterror threats (e.g., smallpox), actively monitoring potentially infected individuals aims to limit disease transmission and morbidity. Guidance issued by CDC on active monitoring was a cornerstone of its response to the West Africa Ebola outbreak. There are limited data on how to balance the costs and performance of this important public health activity. We present a framework that estimates the risks and costs of specific durations of active monitoring for pathogens of significant public health concern. We analyze data from New York City’s Ebola active monitoring program over a 16-month period in 2014-2016. For monitored individuals, we identified unique durations of active monitoring that minimize expected costs for those at “low (but not zero) risk” and “some or high risk”: 21 and 31 days, respectively. Extending our analysis to smallpox and MERS-CoV, we found that the optimal length of active monitoring relative to the median incubation period was reduced compared to Ebola due to less variable incubation periods. Active monitoring can save lives but is expensive. Resources can be most effectively allocated by using exposure-risk categories to modify the duration or intensity of active monitoring.","2017-06-28","Nicholas G Reich; Justin Lessler; Jay K Varma; Neil M Vora",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/156497"],["iisqysqm","138d4617f4f4984bec9cf2c7540fed533d9e4d6e","biorxiv","Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies","10.1101/157156",null,null,"biorxiv","AbstractHealth-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray® Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. www.syndromictrends.com presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.","2017-07-31","Lindsay Meyers; Christine C. Ginocchio; Aimie N. Faucett; Frederick S. Nolte; Per H. Gesteland; Amy Leber; Diane Janowiak; Virginia Donovan; Jennifer Dien Bard; Silvia Spitzer; Kathleen A. Stellrecht; Hossein Salimnia; Rangaraj Selvarangan; Stefan Juretschko; Judy A. Daly; Jeremy C. Wallentine; Kristy Lindsey; Franklin Moore; Sharon L. Reed; Maria Aguero-Rosenfeld; Paul D. Fey; Gregory A. Storch; Steve J. Melnick; Christine C. Robinson; Jennifer F. Meredith; Camille V. Cook; Robert K. Nelson; Jay D. Jones; Samuel V. Scarpino; Benjamin M. Althouse; Kirk M. Ririe; Bradley A. Malin; Mark A. Poritz",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/157156"],["j5gjipdn",null,"biorxiv","Self-assembled Star-shaped Chiroplasmonic Gold Nanoparticles for Ultrasensitive Chiro-immunosensor of Viruses","10.1101/162412",null,null,"biorxiv","Near field optics and optical tunneling light-matter interaction in the superstructure of chiral nanostructures and semiconductor quantum dots exhibit strong optical rotation activity that may open a new window for chiral-based bioanalytes detection. Herein we report an ultrasensitive, chiro-immunosensor using superstructure of chiral gold nanohybrids (CAu NPs) and quantum dots (QDs). Self-assembling techniques were employed to create asymmetric plasmonic chiral nanostructures for extending the spectral range of circular dichroism (CD) response for obtaining superior plasmonic resonant coupling with the QDs excitonic state; this may help to achieve lower the limit of detection (LOD) values. As a result, the designed probe exhibited avian influenza A (H5N1) viral concentration at picomolar level, a significant improvement in sensitivity in comparison to a non-assembled CAu NPs based chiroassay. The practicability of the proposed sensing system was successfully demonstrated on several virus cultures including, avian influenza A (H4N6) virus, fowl adenovirus and coronavirus in blood samples. The results of our study highlights that exciton-plasmon interaction changes chirality and the self-assembled nanostructures are an efficient strategy for enhancing the sensitivity of plasmonic nanosensors.","2017-07-12","Syed Rahin Ahmed; Éva Nagy; Suresh Neethirajan",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/162412"],["jzzhpw7h","3b22eecad8a582436c52284a4db2198a98a94e18","biorxiv","DUSP1 regulates apoptosis and cell migration, but not the JIP1-protected cytokine response, during Respiratory Syncytial Virus and Sendai Virus infection","10.1101/163360",null,null,"biorxiv","AbstractThe host antiviral response involves the induction of interferons and proinflammatory cytokines, but also the activation of cell death pathways, including apoptosis, to limit viral replication and spreading. This host defense is strictly regulated to eliminate the infection while limiting tissue damage that is associated with virus pathogenesis. Post-translational modifications, most notably phosphorylation, are key regulators of the antiviral defense implying an important role of protein phosphatases. Here, we investigated the role of the dual-specificity phosphatase 1 (DUSP1) in the host defense against human respiratory syncytial virus (RSV), a pathogenic virus of the Pneumoviridae family, and Sendai virus (SeV), a model virus being developed as a vector for anti-RSV vaccine. We found that DUSP1 is upregulated before being subjected to proteasomal degradation. DUSP1 does not inhibit the antiviral response, but negatively regulates virus-induced JNK/p38 MAPK phosphorylation. Interaction with the JNK-interacting protein 1 scaffold protein prevents dephosphorylation of JNK by DUSP1, likely explaining that AP-1 activation and downstream cytokine production are protected from DUSP1 inhibition. Importantly, DUSP1 promotes SeV-induced apoptosis and suppresses cell migration in RSV-infected cells. Collectively, our data unveil a previously unrecognized selective role of DUSP1 in the regulation of tissue damage and repair during infections by RSV and SeV.","2017-07-13","Alexa C. Robitaille; Elise Caron; Nicolas Zucchini; Espérance Mukawera; Damien Adam; Mélissa K. Mariani; Anaïs Gélinas; Audray Fortin; Emmanuelle Brochiero; Nathalie Grandvaux",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/163360"],["yqa7ggqg","5d9e490190986584d799e2828c6fd9679e5ae159","biorxiv","The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics","10.1101/167999",null,null,"biorxiv","AbstractBats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat Pteropus alecto (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that P. alecto ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.Significance StatementBats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection.","2017-07-24","Pamela C. De La Cruz-Rivera; Mohammed Kanchwala; Hanquan Liang; Ashwani Kumar; Lin-Fa Wang; Chao Xing; John W. Schoggins",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/167999"],["8xcplab3","fb7371ad31cd501854660876a282060402082537","biorxiv","MERS-CoV spillover at the camel-human interface","10.1101/173211",null,null,"biorxiv","AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans.","2017-08-10","Gytis Dudas; Luiz Max Carvalho; Andrew Rambaut; Trevor Bedford; Ali M. Somily; Mazin Barry; Sarah S. Al Subaie; Abdulaziz A. BinSaeed; Fahad A. Alzamil; Waleed Zaher; Theeb Al Qahtani; Khaldoon Al Jerian; Scott J.N. McNabb; Imad A. Al-Jahdali; Ahmed M. Alotaibi; Nahid A. Batarfi; Matthew Cotten; Simon J. Watson; Spela Binter; Paul Kellam",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/173211"],["v7asfaxc","656ef218f7dcea5555a59cdc29d0f9ec8d5c2a82","biorxiv","MERS-CoV NSP16 necessary for IFN resistance and viral pathogenesis","10.1101/173286",null,null,"biorxiv","AbstractCoronaviruses encode a mix of highly conserved and novel genes as well as genetic elements necessary for infection and pathogenesis, raising the possibility for common targets for attenuation and therapeutic design. In this study, we focus on the highly conserved nonstructural protein (NSP) 16, a viral 2’O methyl-transferase (MTase) that encodes critical functions in immune modulation and infection. Using reverse genetics, we disrupted a key motif in the conserved KDKE motif of MERS NSP16 (D130A) and evaluated the effect on viral infection and pathogenesis. While the absence of 2’O MTase activity had only marginal impact on propagation and replication in Vero cells, the MERS dNSP16 mutant demonstrated significant attenuation relative to control both in primary human airway cultures and in vivo. Further examination indicated the MERS dNSP16 mutant had a type I IFN based attenuation and was partially restored in the absence of IFIT molecules. Importantly, the robust attenuation permitted use of MERS dNSP16 as a live attenuated vaccine platform protecting from challenge with a mouse adapted MERS-CoV strain. These studies demonstrate the importance of the conserved 2’O MTase activity for CoV pathogenesis and highlight NSP16 as a conserved universal target for rapid live attenuated vaccine design in an expanding CoV outbreak setting.SignificanceCoronavirus emergence in both human and livestock represents a significant threat to global public health, as evidenced by the sudden emergence of SARS-CoV, MERS-CoV, PEDV and swine delta coronavirus in the 21st century. These studies describe an approach that effectively targets the highly conserved 2’O methyl-transferase activity of coronaviruses for attenuation. With clear understanding of the IFN/IFIT based mechanism, NSP16 mutants provide a suitable target for a live attenuated vaccine platform as well as therapeutic development for both current and future emergent CoV strains. Importantly, other approaches targeting other conserved pan-coronavirus functions have not yet proven effective against MERS-CoV, illustrating the broad applicability of targeting viral 2’O MTase function across coronaviruses.","2017-08-08","Vineet D. Menachery; Lisa E. Gralinski; Hugh D. Mitchell; Kenneth H. Dinnon; Sarah R. Leist; Boyd L. Yount; Rachel L. Graham; Eileen T. McAnarney; Kelly G. Stratton; Adam S. Cockrell; Kari Debbink; Amy C. Sims; Katrina M. Waters; Ralph S. Baric",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/173286"],["vqtx98xm","f126b7f9c8ba02a0f872b51cf95d08086e0bd4ac","biorxiv","Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations","10.1101/175562",null,null,"biorxiv","ABSTRACTThe coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3′→5′exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.IMPORTANCEUnique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness.","2017-08-11","Kevin W. Graepel; Xiaotao Lu; James Brett Case; Nicole R. Sexton; Everett Clinton Smith; Mark R. Denison",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/175562"],["pgfdy23h","7116ddab89281e7641fb659019c90ee3715f5039","biorxiv","The impact of persistent bacterial bronchitis on the pulmonary microbiome of children","10.1101/181982",null,null,"biorxiv","AbstractPersistent bacterial bronchitis is a leading cause of chronic wet cough in young children. This study aimed to characterise the respiratory bacterial microbiota of healthy children and to assess the impact of the changes associated with the development of persistent bacterial bronchitis.Blind, protected brushings were obtained from 20 healthy controls and 24 children with persistent bacterial bronchitis, with an additional directed sample obtained from persistent bacterial bronchitis patients. DNA was extracted, quantified using a 16S rRNA gene quantitative PCR assay prior to microbial community analysis by 16S rRNA gene sequencing.No significant difference in bacterial diversity or community composition (R2 = 0.01, P = 0.36) was observed between paired blind and non-blind brushes, showing that blind brushings are a valid means of accessing the airway microbiota. This has important implications for collecting lower respiratory samples from healthy children. A significant decrease in bacterial diversity (P < 0.001) and change in community composition (R2 = 0.08, P = 0.004) was observed between controls and patients. Bacterial communities within patients with PBB were dominated by Proteobacteria, and indicator species analysis showed that Haemophilus and Neisseria were significantly associated with the patient group. In 15 (52.9%) cases the dominant organism by sequencing was not identified by standard routine clinical culture.The bacteria present in the lungs of patients with persistent bacterial bronchitis were less diverse in terms of richness and evenness. The results validate the clinical diagnosis, and suggest that more attention to bacterial communities in children with chronic cough may lead to more rapid recognition of this condition with earlier treatment and reduction in disease burden.","2017-08-29","Leah Cuthbertson; Vanessa Craven; Lynne Bingle; William O.C.M. Cookson; Mark L. Everard; Miriam F. Moffatt",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/181982"],["p3b4yk5d","3d690de86ca6e200f4f9cac61b004e47ad753aae","biorxiv","Mouse hepatitis virus nsp14 exoribonuclease activity is required for resistance to innate immunity","10.1101/182196",null,null,"biorxiv","ABSTRACTCoronaviruses (CoV) are positive-sense RNA viruses that infect numerous mammalian and avian species and are capable of causing severe and lethal disease in humans. CoVs encode several innate immune antagonists that interact with the host innate immune response to facilitate efficient viral replication. CoV non-structural protein 14 (nsp14) encodes 3'-to-5' exoribonuclease activity (ExoN), which performs a proofreading function and is required for high-fidelity replication. Outside of the order Nidovirales, arenaviruses are the only RNA viruses that encode an ExoN, which functions to degrade dsRNA replication intermediates. In this study, we tested the hypothesis that CoV ExoN may also function to antagonize the innate immune response. We demonstrate that viruses lacking ExoN activity [ExoN(-)] are sensitive to cellular pretreatment with interferon beta (IFN-β) in a dose-dependent manner. In addition, ExoN(-) virus replication was attenuated in wild-type bone marrow-derived macrophages (BMMs) and partially restored in interferon alpha/beta receptor deficient (IFNAR-/-) BMMs. ExoN(-) virus replication did not result in IFN-β gene expression, and in the presence of an IFN-β-mediated antiviral state, ExoN(-) viral RNA levels were not substantially reduced relative to untreated. However, ExoN(-) virus generated from IFN-β pretreated cells had reduced specific infectivity and decreased relative fitness, suggesting that ExoN(-) virus generated during an antiviral state is less viable to establish a subsequent infection. Overall, our data suggest MHV ExoN activity is required for resistance to the innate immune response and antiviral mechanisms affecting the viral RNA sequence and/or an RNA modification act on viruses lacking ExoN activity.","2017-08-29","James Brett Case; Yize Li; Ruth Elliott; Xiaotao Lu; Kevin W. Graepel; Nicole R. Sexton; Everett Clinton Smith; Susan R. Weiss; Mark R. Denison",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/182196"],["g55d5ijx","08a22278486e12768ce186677a6a89663d24586f","biorxiv","Complemented palindrome small RNAs first discovered from SARS coronavirus","10.1101/185876",null,null,"biorxiv","AbstractIn this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes.","2017-09-07","Chang Liu; Ze Chen; Wenyuan Shen; Deshui Yu; Siyu Li; Yue Hu; Haishuo Ji; Wenjun Bu; Qingsong Wang; Shan Gao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/185876"],["vok1dni9","090b6c8b3df30bc248221869f673a2d970caa1b9","medrxiv","Quantifying the success of measles vaccination campaigns in the Rohingya refugee camps","10.1101/19008417",null,null,"medrvix","In the wake of the Rohingya population's mass migration from Myanmar, one of the world's largest refugee settlements was constructed in Cox's Bazar, Bangladesh to accommodate nearly 900,000 new refugees. Refugee populations are particularly vulnerable to infectious disease outbreaks due to many population and environmental factors. A large measles outbreak, with over 2,500 cases, occurred among the Rohingya population between September and December 2017. Here, we estimate key epidemiological parameters and use a dynamic mathematical model of measles transmission to evaluate the effectiveness of the reactive vaccination campaigns in the refugee camps. We also estimate the potential for subsequent outbreaks under different vaccination coverage scenarios. Our modeling results highlight the success of the vaccination campaigns in rapidly curbing transmission and emphasize the public health importance of maintaining high levels of vaccination in this population, where high birth rates and historically low vaccination coverage rates create suitable conditions for future measles outbreaks.","2019-10-05","Taylor Chin; Caroline O. Buckee; Ayesha S. Mahmud",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/19008417"],["jwesa12u","10525ac89e46be4cb9cb9fd1131d28411a902047","medrxiv","Using Digital Surveillance Tools for Near Real-Time Mapping of the Risk of International Infectious Disease Spread: Ebola as a Case Study","10.1101/19011940",null,null,"medrvix","In our increasingly interconnected world, it is crucial to understand the risk of an outbreak originating in one country or region and spreading to the rest of the world. Digital disease surveillance tools such as ProMED and HealthMap have the potential to serve as important early warning systems as well as complement the field surveillance during an ongoing outbreak. Here we present a flexible statistical model that uses data produced from digital surveillance tools (ProMED and HealthMap) to forecast short term incidence trends in a spatially explicit manner. The model was applied to data collected by ProMED and HealthMap during the 2013-2016 West African Ebola epidemic. The model was able to predict each instance of international spread 1 to 4 weeks in advance. Our study highlights the potential and limitations of using publicly available digital surveillance data for assessing outbreak dynamics in real-time.","2019-11-15","Sangeeta Bhatia; Britta Lassmann; Emily Cohn; Malwina Carrion; Moritz U.G. Kraemer; Mark Herringer; John Brownstein; Larry Madoff; Anne Cori; Pierre Nouvellet",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/19011940"],["npd2qf7m","44b6eea3629b4e13595dc49813ee39a0d20dc5f2","biorxiv","Evolutionary rate shifts suggest species-specific adaptation events in HIV-1 and SIV","10.1101/190769",null,null,"biorxiv","AbstractThe process of molecular adaptation following a cross-species virus transmission event is currently poorly understood. Here, we identified 137 protein sites that experienced deceleration in their rate of evolution along the HIV-1/SIV phylogeny, likely indicating gain-of-function and consequent adaptation. The majority of such events occurred in parallel to cross-species transmission events and varied between HIV-1 groups, indicating independent adaptation strategies. The evolutionary rate decelerations we found were particularly prominent in accessory proteins that counteract host antiviral restriction factors, suggesting that these factors are a major barrier to viral adaptation to a new host. Surprisingly, we observed that the non-pandemic HIV-1 group O, derived from gorillas, exhibited more rate deceleration events than the pandemic group M, derived from chimpanzees. We suggest that the species barrier is higher when the genetic distance of the hosts increases. Our approach paves the way for subsequent studies on cross-species transfers in other major pathogens.","2017-09-19","Maoz Gelbart; Adi Stern",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/190769"],["6riyqn4k","aa5a99e67d3e887b3fa88575fb0d4a662d0a60ce","biorxiv","Modeling Epidemics: A Primer and Numerus Software Implementation","10.1101/191601",null,null,"biorxiv","AbstractEpidemiological models are dominated by SEIR (Susceptible, Exposed, Infected and Removed) dynamical systems formulations and their elaborations. These formulations can be continuous or discrete, deterministic or stochastic, or spatially homogeneous or heterogeneous, the latter often embracing a network formulation. Here we review the continuous and discrete deterministic and discrete stochastic formulations of the SEIR dynamical systems models, and we outline how they can be easily and rapidly constructed using the Numerus Model Builder, a graphically-driven coding platform. We also demonstrate how to extend these models to a metapopulation setting using both the Numerus Model Builder network and geographical mapping tools.","2017-09-22","Wayne M. Getz; Richard Salter; Oliver Muellerklein; Hyun S. Yoon; Krti Tallam",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/191601"],["uw76y16g","9b17a9fa6be11d8b3dbed9f3dea2516020083bd3","biorxiv","Fast Virome Explorer: A Pipeline for Virus and Phage Identification and Abundance Profiling in Metagenomics Data","10.1101/196998",null,null,"biorxiv","AbstractIdentifying viruses and phages in a metagenomics sample has important implication in improving human health, preventing viral outbreaks, and developing personalized medicine. With the rapid increase in data files generated by next generation sequencing, existing tools for identifying and annotating viruses and phages in metagenomics samples suffer from expensive running time. In this paper, we developed a stand-alone pipeline, FastViromeExplorer, for rapid identification and abundance quantification of viruses and phages in big metagenomic data. Both real and simulated data validated FastViromeExplorer as a reliable tool to accurately identify viruses and their abundances in large data, as well as in a time efficient manner.","2017-10-02","Saima Sultana Tithi; Roderick V. Jensen; Liqing Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/196998"],["ifla4aix","7395de48b2402e5e1561a93b99e8aae8728c6ff1","biorxiv","RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain and is required for ubiquitination","10.1101/200410",null,null,"biorxiv","ABSTRACTTRIM25 is a novel RNA-binding protein and a member of the Tripartite Motif (TRIM) family of E3 ubiquitin ligases, which plays a pivotal role in the innate immune response. Almost nothing is known about its RNA-related roles in cell biology. Furthermore, its RNA-binding domain has not been characterized. Here, we reveal that RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain, which we postulate to be a novel RNA-binding domain. Using CLIP-seq and SILAC-based co-immunoprecipitation assays, we uncover TRIM25’s endogenous RNA targets and protein binding partners. Finally, we show that the RNA-binding activity of TRIM25 is important for its ubiquitin ligase function. These results reveal new insights into the molecular roles and characteristics of RNA-binding E3 ubiquitin ligases and demonstrate that RNA could be an essential factor for their biological functions.","2017-10-09","Nila Roy Choudhury; Gregory Heikel; Maryia Trubitsyna; Peter Kubik; Jakub Stanislaw Nowak; Shaun Webb; Sander Granneman; Christos Spanos; Juri Rappsilber; Alfredo Castello; Gracjan Michlewski",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/200410"],["v4spn75m",null,"biorxiv","IFITM proteins inhibit HIV-1 protein synthesis","10.1101/201525",null,null,"biorxiv","AbstractInterferon induced transmembrane proteins (IFITMs) inhibit the cellular entry of a broad range of viruses, but it has been suspected that for HIV-1 IFITMs may also inhibit a post-integration replicative step. We show that IFITM expression reduces HIV-1 viral protein synthesis by preferentially excluding viral mRNA transcripts from translation and thereby restricts viral production. Codon-optimization of proviral DNA rescues viral translation, implying that IFITM-mediated restriction requires recognition of viral RNA elements. In addition, we find that expression of the viral accessory protein Nef can help overcome the IFITM-mediated inhibition of virus production. Our studies identify a novel role for IFITMs in inhibiting HIV replication at the level of translation, but show that the effects can be overcome by the lentiviral protein Nef.","2017-10-11","Wing-Yiu Jason Lee; Chen Liang; Richard D Sloan",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/201525"],["43cxzudl",null,"biorxiv","Delving Below the Species Level To Characterize the Ecological Diversity within the Global Virome: An Exploration of West Nile Virus","10.1101/2019.12.12.874214",null,null,"biorxiv","SummaryEfforts to describe the diversity of viruses have largely focused on classifying viruses at the species level. However, substantial ecological diversity, both in virulence level and host range, is known within virus species. Here we demonstrate a proof of concept for easily discovering ecological diversity within a virus species taxon. We have focused on the West Nile Virus to take advantage of its broad host range in nature. We produced a genome-based phylogeny of world diversity of WNV and then used Ecotype Simulation 2 to hypothesize demarcation of genomes into 69 putative ecotypes (ecologically distinct populations), based only on clustering of genome sequences. Then we looked for evidence of ecological divergence among ecotypes based on differences in host bird associations within the Connecticut-New York region. Our results indicated significant heterogeneity among ecotypes for their associations with different bird hosts. Ecological diversity within other zoonotic viruses could be easily discovered using this approach. Opportunities for extending this line of research to human associations of virus ecotypes are limited by missing geographic metadata on human samples.","2019-12-13","Tong Kong; Kelly Mei; Ammie Wang; Danny Krizanc; Frederick M. Cohan",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.12.874214"],["2fcl6f4i","97027e08cb413795cf19064338efd049fe5d491b","biorxiv","Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts","10.1101/2019.12.13.876037",null,null,"biorxiv","AbstractAdenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes.","2019-12-13","Alexander M. Price; Katharina E. Hayer; Daniel P. Depledge; Angus C. Wilson; Matthew D. Weitzman",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.13.876037"],["bqwmx5dy","93f67ffe7803061de9b19c4dfa346b3aa97aa4eb","biorxiv","Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction","10.1101/2019.12.15.877092",null,null,"biorxiv","AbstractSingle particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive non-uniform refinement, an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, non-uniform refinement removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package and has already been used successfully in several notable structural studies.","2019-12-16","Ali Punjani; Haowei Zhang; David J. Fleet",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.15.877092"],["eohjkqc4",null,"biorxiv","Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses","10.1101/2019.12.16.875872",null,null,"biorxiv","AbstractRNA viruses rapidly mutate, which can result in increased virulence, increased escape from vaccine protection, and false negative detection results. Targeted detection methods have a limited ability to detect unknown viruses and often provide insufficient data to detect coinfections or identify antigenic variants. Random, deep sequencing is a method that can more fully detect and characterize RNA viruses and is often coupled with molecular techniques or culture methods for viral enrichment. Viral culture coupled with third-generation sequencing were tested for the ability to detect and characterize RNA viruses. Cultures of bovine viral diarrhea virus, canine distemper virus, epizootic hemorrhagic disease virus, infectious bronchitis virus, two influenza A viruses, and porcine respiratory and reproductive syndrome virus were sequenced on the MinION platform using a random, reverse primer in a strand-switching reaction, coupled with PCR-based barcoding. Reads were taxonomically classified and used for reference-based sequence building using a stock personal computer. This method accurately detected and identified complete coding sequence genomes with a minimum of 20× coverage depth for all seven viruses, including a sample containing two viruses. Each lineage-typing region had at least 26× coverage depth for all viruses. Furthermore, analyzing the canine distemper virus sample through a pipeline devoid of canine distemper virus reference sequences modeled the ability of this protocol to detect unknown viruses. These results show the ability of this technique to detect and characterize dsRNA, negative- and positive-sense ssRNA, nonsegmented, and segmented RNA viruses.","2019-12-16","Kelsey T. Young; Kevin K. Lahmers; Holly S. Sellers; David E. Stallknecht; Rebecca L. Poulson; Jerry T. Saliki; S. Mark Tompkins; Ian Padykula; Chris Siepker; Elizabeth W. Howerth; Michelle Todd; James B. Stanton",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.16.875872"],["t2azgyp8","b0899a264af548a89d649154aec569889717b295","medrxiv","Stochastic challenges to interrupting helminth transmission","10.1101/2019.12.17.19013490",null,null,"medrvix","Predicting the effect of different programmes designed to control both the morbidity induced by helminth infections and parasite transmission is greatly facilitated by the use of mathematical models of transmission and control impact. In such models, it is essential to account for as many sources of uncertainty -- natural, or otherwise -- to ensure robustness in prediction and to accurately depict variation around an expected outcome. In this paper, we investigate how well the standard deterministic models match the predictions made using individual-based stochastic simulations. We also explore how well concepts which derive from deterministic models, such as ‘breakpoints’ in transmission, apply in the stochastic world. Employing an individual based stochastic model framework we also investigate how transmission and control are affected by the migration of infected people into a defined community. To give our study focus we consider the control of soil-transmitted helminths (STH) by mass drug administration (MDA), though our methodology is readily applicable to the other helminth species such as the schistosome parasites and the filarial worms. We show it is possible to define a ‘stochastic breakpoint’ where much noise surrounds the expected deterministic breakpoint. We also discuss the concept of the ‘interruption of transmission’ independent of the ‘breakpoint’ concept where analyses of model behaviour illustrate the current limitations of deterministic models to account for the ‘fade-out’ or transmission extinction behaviour in simulations. The analyses based on migration confirm a relationship between the infected human migration rate per unit of time and the death rate of infective stages that are released into the free-living environment (eggs or larvae depending on the STH species) that create the reservoir of infection which in turn determines the likelihood that control activities aim at chemotherapeutic treatment of the human host will eliminate transmission. The development of a new stochastic simulation code for STH in the form of a publicly-available open-source python package which includes features to incorporate many population stratifications, different control interventions including mass drug administration (with defined frequency, coverage levels and compliance patterns) and inter-village human migration is also described.","2019-12-18","Robert J Hardwick; Marleen Werkman; James E Truscott; Roy M Anderson",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.17.19013490"],["bi49e7ne","a555fbeeffc44dfdb5087c83395f0f743654cbba","biorxiv","A novel totivirus alters gene expression and vacuolar morphology in Malassezia cells and induces a TLR3-mediated inflammatory immune response","10.1101/2019.12.17.880526",null,null,"biorxiv","AbstractMost fungal viruses have been identified in plant pathogens, whereas the presence of viral particles in human pathogenic fungi is less well studied. In the present study, we observed extrachromosomal double-stranded RNA (dsRNA) segments in various clinical isolates of Malassezia species. Malassezia is the most dominant fungal genus on the human skin surface, and species in this group are considered to be etiological factors in various skin diseases including dandruff, seborrheic dermatitis, and atopic dermatitis. We identified novel dsRNA segments and our sequencing results revealed that the virus, named MrV40, belongs to the Totiviridae family and contains an additional satellite dsRNA segment encoding a novel protein. The transcriptome of virus-infected M. restricta cells was compared to that of virus-free cells, and the results showed that transcripts involved in ribosomal biosynthesis were down regulated and those involved in energy production and programmed cell death were increased in abundance. Moreover, transmission electron microscopy revealed significantly larger vacuoles for virus-infected M. restricta cells, indicating that MrV40 infection dramatically altered M. restricta physiology. Our analysis also revealed that a viral nucleic acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells (BMDCs) and this result suggests that a viral element contributes to the pathogenesis of Malassezia.ImportanceMalassezia is the most dominant fungal genus on the human skin surface and is associated with various skin diseases including dandruff and seborrheic dermatitis. Among Malassezia species, M. restricta is the most widely observed species on the human skin. In the current study, we identified a novel dsRNA virus, named MrV40, in M. restricta and characterized the sequences and structure of the viral genome along with an independent satellite dsRNA viral segment. Moreover, we found altered expression of genes involved in ribosomal synthesis and programmed cell death, indicating that virus infection altered the physiology of the fungal host cells. Our data also showed that the viral nucleic acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells (BMDCs), indicating that a viral element likely contributes to the pathogenesis of Malassezia. This is the first study to identify and characterize a novel mycovirus in Malassezia.","2019-12-19","Minji Park; Yong-Joon Cho; Donggyu Kim; Chul-Su Yang; Shi Mun Lee; Thomas L. Dawson; Satoshi Nakamizo; Kenji Kabashima; Yang Won Lee; Won Hee Jung",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.17.880526"],["e5p4metz","8c223fd63ba6a924ecd3df8d32bcf0556758cb95","biorxiv","Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications","10.1101/2019.12.18.880849",null,null,"biorxiv","AbstractThe prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial (E. coli) and archaeal (H. volcanii, P. furiosus) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism.","2019-12-19","Felix Grünberger; Robert Knüppel; Michael Jüttner; Martin Fenk; Andreas Borst; Robert Reichelt; Winfried Hausner; Jörg Soppa; Sebastien Ferreira-Cerca; Dina Grohmann",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.18.880849"],["qw2tsrxm","73d80c8f5780d70bd8d343188c56e898e91557b6","biorxiv","Ca2+ ions promote fusion of Middle East Respiratory Syndrome coronavirus with host cells and increase infectivity","10.1101/2019.12.18.881391",null,null,"biorxiv","AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) is a major emerging zoonotic infectious disease. Since its first outbreak in 2012, the virus has repeatedly transmitted from camels to humans with 2,468 confirmed cases, causing 851 deaths. To date, there are no efficacious drugs and vaccines against MERS-CoV, increasing its potential to cause a public health emergency. A critical step in the life cycle of MERS-CoV is the fusion with the host cell with its spike (S) protein as main determinant of viral entry. Proteolytic cleavage of S exposes its fusion peptide (FP), which initiates membrane fusion. Previous studies on the related severe acute respiratory syndrome coronavirus (SARS-CoV) FP have shown that calcium (Ca2+) plays an important role for fusogenic activity via a Ca2+ binding pocket with conserved glutamic acid (E) and aspartic acid (D) residues. SARS-CoV and MERS-CoV FP share a high sequence homology and here, we investigated whether Ca2+ is required for MERS-CoV fusion by substituting E and D residues in the MERS-CoV FP with neutrally charged alanines. Upon verifying mutant cell surface expression and proteolytic cleavage, we tested the mutants ability to mediate infection of pseudo-particles (PPs) on host cells without and with Ca2+. Our results demonstrate that intracellular Ca2+ enhances MERS-CoV WT PPs infection by approximately two-fold and that E891 is a crucial residue for Ca2+ interaction. Electron spin resonance revealed that this enhancement could be attributed to Ca2+ increasing MERS-CoV FP fusion-relevant membrane ordering. Intriguingly, isothermal calorimetry titration showed that MERS-CoV FP binds one Ca2+, as opposed to SARS-CoV FP which binds two. Our data suggests that there are significant differences in FP-Ca2+ interactions of MERS-CoV and SARS-CoV FP despite their high sequence similarity and that the number of Ca2+ ions interacting with the FP has implications on the fusion dynamics of the virus.","2019-12-19","Marco R. Straus; Tiffany Tang; Alex L. Lai; Annkatrin Flegel; Miya Bidon; Jack H. Freed; Susan Daniel; Gary R. Whittaker",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.18.881391"],["7a5nxxar","63781701d3aac3b1640caae24b2c8c2a06c8e6ab","biorxiv","Comparison of alternative models of human movement and the spread of disease","10.1101/2019.12.19.882175",null,null,"biorxiv","AbstractPredictive models for the spatial spread of infectious diseases has received much attention in recent years as tools for the management of infectious diseas outbreaks. Prominently, various versions of the so-called gravity model, borrowed from transportation theory, have been used. However, the original literature suggests that the model has some potential misspecifications inasmuch as it fails to capture higher-order interactions among population centers. The fields of economics, geography and network sciences holds alternative formulations for the spatial coupling within and among conurbations. These includes Stouffer’s rank model, Fotheringham’s competing destinations model and the radiation model of Simini et al. Since the spread of infectious disease reflects mobility through the filter of age-specific susceptibility and infectivity and since, moreover, disease may alter spatial behavior, it is essential to confront with epidemiological data on spread. To study their relative merit we, accordingly, fit variants of these models to the uniquely detailed dataset of prevaccination measles in the 954 cities and towns of England and Wales over the years 1944-65 and compare them using a consistent likelihood framework. We find that while the gravity model is a reasonable first approximation, both Stouffer’s rank model, an extended version of the radiation model and the Fotheringham competing destinations model provide significantly better fits, Stouffer’s model being the best. Through a new method of spatially disaggregated likelihoods we identify areas of relatively poorer fit, and show that it is indeed in densely-populated conurbations that higher order spatial interactions are most important. Our main conclusion is that it is premature to narrow in on a single class of models for predicting spatial spread of infectious disease. The supplemental materials contain all code for reproducing the results and applying the methods to other data sets.Author summaryThe ability to predict how infectious disease will spread is of great importance in the face of the numerous emergent and re-emergent pathogens that currently threatening human well-being. We identified a variety of alternative models that predict human mobility as as a function of population distribution across a landscape. These consider some models that account for pair-wise interactions between population centers, as well as some that allow for higher-order interactions. We trained the models using a uniquely rich spatiotemporal data set on pre-vaccination measles in England and wales (1944-65), which comprises more than a million records from 954 cities and towns. Likelihood rankings of the different models reveal strong evidence for higher-order interactions in the form of competition among cities as destinations for travelers and, thus, dilution of spatial transmission. The currently most commonly used so-called ‘gravity’ models were far from the best in capturing spatial disease dynamics.","2019-12-19","Ottar N. Bjørnstad; Bryan T. Grenfell; Cecile Viboud; Aaron A. King",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.19.882175"],["8b2ookby","d83e3c028de1950c4f8dedae21eb90f90c4ed6c3","biorxiv","Translational profiling of macrophages infected with Leishmania donovani identifies mTOR- and eIF4A-sensitive immune-related transcripts","10.1101/2019.12.20.884338",null,null,"biorxiv","AbstractThe protozoan parasite Leishmania donovani (L. donovani) causes visceral leishmaniasis, a chronic infection which is fatal when untreated. While previous studies showed that L. donovani reprograms transcription to subvert host cell functions, it remains unclear whether the parasite also alters host mRNA translation to establish a successful infection. To assess this, we compared transcriptome-wide translation in primary mouse macrophages infected with L. donovani promastigotes or amastigotes using polysome-profiling. This identified ample selective changes in translation (3,127 transcripts) which were predicted to target central cellular functions by inducing synthesis of proteins related to chromatin remodeling and RNA metabolism while inhibiting those related to intracellular trafficking and antigen presentation. Parallel quantification of protein and mRNA levels for a set of transcripts whose translation was activated upon L. donovani infection (Papbpc1, Eif2ak2, and Tgfb) confirmed, as indicated by polysome-profiling, increased protein levels despite largely unaltered mRNA levels. Mechanistic in silico analyses suggested activated translation depending on the kinase mTOR (e.g. Pabpc1) and the RNA helicase eIF4A (e.g. Tgfb) during infection. Accordingly, treatment with mTOR inhibitors torin-1 or rapamycin reversed L. donovani-induced PABPC1 without affecting corresponding transcript levels. Similarly, the production of TGF-β decreased in presence of the eIF4A inhibitor silvestrol despite unaltered Tgfb mRNA levels. Consistent with parasite modulation of host eIF4A-sensitive translation to promote infection, silvestrol suppressed L. donovani replication within macrophages. In contrast, parasite survival was favored under mTOR inhibition. In summary, infection-associated changes in translation of mTOR- and eIF4A-sensitive mRNAs contribute to modulate mRNA metabolism and immune responses in L. donovani-infected macrophages. Although the net outcome of such translation programs favours parasite propagation, individual translation programs appear to have opposing roles during L. donovani infection, thereby suggesting their selective targeting as key for therapeutic effects.Author SummaryFine-tuning the efficiency of mRNA translation into proteins allows cells to tailor their responses to stress without the need for synthesizing new mRNA molecules. It is well established that the protozoan parasite Leishmania donovani alters transcription of specific genes to subvert host cell functions. However, discrepancies between transcriptomic and proteomic data suggest that post-transcriptional regulatory mechanisms also contribute to modulate host gene expression programs during L. donovani infection. Herein, we report that one third of protein-coding mRNAs expressed in macrophages are differentially translated upon infection with L. donovani. Our computational analyses reveal that subsets of mRNAs encoding functionally related proteins share the same directionality of translational regulation, which is likely to impact metabolic and microbicidal activity of infected cells. We also show that upregulated translation of transcripts that encode central regulators of mRNA metabolism and inflammation is sensitive to the activation of mTOR or eIF4A during infection. Finally, we observe that inhibition of eIF4A activity reduces parasite survival within macrophages while selective blockade of mTOR has the opposite effect. Thus, our study points to a dual role for translational control of host gene expression during L. donovani infection and suggests that novel regulatory nodes could be targeted for therapeutic intervention.","2019-12-20","Visnu Chaparro; Louis-Philippe Leroux; Laia Masvidal; Julie Lorent; Tyson E. Graber; Aude Zimmermann; Guillermo Arango Duque; Albert Descoteaux; Tommy Alain; Ola Larsson; Maritza Jaramillo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.20.884338"],["ahs2wdus","48df5b06710467cbe5b41dd851e946fc936ecf15","biorxiv","Low basal expression and slow induction of IFITM3 puts immune cells at risk of influenza A infection","10.1101/2019.12.20.885590",null,null,"biorxiv","AbstractThe interferon-induced transmembrane protein, IFITM3, has been shown to restrict influenza virus infection in murine and in vitro settings for ten years, but no explanation has been found to explain why this virus infection is so highly contagious and infects most individuals it comes in contact with. We confirm that the expression level of IFITM3 plays a role in determining the level of viral infection through manipulation of IFITM3 levels with interferon (IFN) stimulation and overexpression systems. Low basal expression may put some immune cells, including lymphocytes and lung-resident macrophages, at risk of influenza virus infection. Investigating the induction of IFITM3 by IFN, we find a strong preference for Type I IFN in IFITM3 induction in both cell lines and primary human cells. While myeloid cells can increase expression following stimulation by Type I IFN, lymphocytes show minimal induction of IFITM3 following IFN stimulation, suggesting that they are always at risk of viral infection. Surprisingly, we found that the time it takes for maximal induction of IFITM3 is relatively slow for an interferon-stimulated gene at around 36 hours. Low basal expression and slow induction of IFITM3 could increase the risk of influenza virus infection in selected immune cells.ImportanceInfluenza virus infection remains one of the top ten threats to global health, causing significant deaths and hospitalisations across the world each year. Understanding mechanisms for controlling influenza virus infection remain a priority. The interferon-induced transmembrane protein IFITM3 can restrict influenza infection by limiting replication of the virus. The precise mechanisms of how IFITM3 reduced replication of influenza are unknown, although it is predicted to prevent release of viral contents into the cytosol by preventing pore formation on the endosomal compartments where it is suggested to reside. Here we have shown that the expression level of IFITM3 is important in determining the control of influenza virus infection. We find an expression pattern for IFITM3 that varies based on cell type, tissue locality, differentiation state and cell naivety, all of which highlights cells that may be at the highest risk of influenza infection.","2019-12-23","Dannielle Wellington; Zixi Yin; Liwei Zhang; Jessica Forbester; Kerry Kite; Henry Laurenson-Schafer; Shokouh Makvandi-Nejad; Boquan Jin; Emma Bowes; Krishnageetha Manoharan; David Maldonado-Perez; Clare Verill; Ian Humphreys; Tao Dong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.20.885590"],["ts91ivwf","106f10eaae4fb67aca5b2bd737ce2bf1c27fa243","biorxiv","Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community","10.1101/2019.12.21.885921",null,null,"biorxiv","AbstractMost emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare—but detectable—bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens.","2019-12-23","Daniel J. Becker; Kelly A. Speer; Alexis M. Brown; M. Brock Fenton; Alex D. Washburne; Sonia Altizer; Daniel G. Streicker; Raina K. Plowright; Vladimir E. Chizhikov; Nancy B. Simmons; Dmitriy V. Volokhov",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2019.12.21.885921"],["2vecg9op","09c9fcabc66a106e01ef42247cbd86b6d85bd67f","biorxiv","Mono-ADP-ribosylation by ARTD10 restricts Chikungunya virus replication by interfering with the proteolytic activity of nsP2","10.1101/2020.01.07.896977",null,null,"biorxiv","AbstractA subset of intracellular mono-ADP-ribosyltransferases diphtheria toxin-like (ARTDs, aka mono-PARPs) is induced by type I interferons. Some of these mono-ARTDs feature antiviral activity while certain RNA viruses, including Chikungunya virus (CHIKV), encode mono-ADP-ribosylhydrolases, suggesting a role for mono-ADP-ribosylation (MARylation) in host-virus conflicts. CHIKV expresses four non-structural proteins (nsP1-nsP4), with nsP3 containing a macrodomain that hydrolyzes and thereby reverses protein MARylation in vitro and in cells. This de-MARylation activity is essential as hydrolase inactivating mutations result in replication defective virus. However, the substrates of MARylation during CHIKV infection are unknown and thus it is unclear how the macrodomain contributes to virus replication and how mono-ARTD-dependent MARylation confers antiviral immunity. We identified ARTD10 and ARTD12 as restriction factors for CHIKV replication in a catalytic activity-dependent manner. CHIKV replication requires processing of the non-structural polyprotein nsP1-4 by the nsP2-encoded protease and the assembly of the four individual nsPs into a functional replication complex. Expression of ARTD10 and ARTD12 resulted in a reduction of processed nsPs. Similarly, MAR hydrolase inactive CHIKV replicon mutants revealed a decrease in processed nsPs, comparable to an nsP2 protease defective mutant. This suggested that the macrodomain contributes to nsP2 protease activity. In support, a hydrolase-deficient virus was complemented by a protease-deficient virus. We hypothesized that MARylation regulates the proteolytic function of nsP2. Indeed, we found that nsP2 is MARylated by ARTD10. This inhibited nsP2 protease activity, thereby preventing polyprotein processing and consequently virus replication. This inhibition was antagonized by the MAR hydrolase activity of nsP3. Together, our findings provide a mechanistic explanation for the need of the viral MAR hydrolase for efficient replication of CHIKV.Author SummaryInfectious diseases still pose major health threats. Especially fast evolving viruses find ever new strategies to manipulate the immune response. With climate warming and increased human mobility vector-borne pathogens like Chikungunya virus (CHIKV) spread and cause world-wide epidemics. Beyond the acute phase, CHIKV patients regularly suffer from chronic rheumatism. This entails a decline in life quality and an economic burden. To date no drugs are approved and the mode of pathogenesis remains elusive. Here we describe a mechanistic function of the CHIKV nsP3 macrodomain. We found that the viral nsP2 is mono-ADP-ribosylated interfering with its auto-proteolytic function. The nsP3 macrodomain removes this modification and restores the protease activity that is essential for replication. Because macrodomains are highly conserved they might represent broad antiviral targets.","2020-01-08","Sarah Krieg; Fabian Pott; Laura Eckei; Maud Verheirstraeten; Mareike Bütepage; Barbara Lippok; Christine Goffinet; Bernhard Lüscher; Patricia Verheugd",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.07.896977"],["2f9nc2to","b80d9de7750d627a0189fcd2dfd82c7efcb1c079","biorxiv","An interaction network of RNA-binding proteins involved in Drosophila oogenesis","10.1101/2020.01.08.899146",null,null,"biorxiv","AbstractDuring Drosophila oogenesis, the localization and translational regulation of maternal transcripts relies on RNA-binding proteins (RBPs). Many of these RBPs localize several mRNAs and may have additional direct interaction partners to regulate their functions. Using immunoprecipitation from whole Drosophila ovaries coupled to mass spectrometry, we examined protein-protein associations of 6 GFP-tagged RBPs expressed at physiological levels. Analysis of the interaction network and further validation in human cells allowed us to identify 26 previously unknown associations, besides recovering several well characterized interactions. We identified interactions between RBPs and several splicing factors, providing links between nuclear and cytoplasmic events of mRNA regulation. Additionally, components of the translational and RNA decay machineries were selectively co-purified with some baits, suggesting a mechanism for how RBPs may regulate maternal transcripts. Given the evolutionary conservation of the studied RBPs, the interaction network presented here provides the foundation for future functional and structural studies of mRNA localization across metazoans.","2020-01-09","Prashali Bansal; Johannes Madlung; Kristina Schaaf; Boris Macek; Fulvia Bono",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.08.899146"],["8iet67u3","e18620d5cff022d18aa23da2fa4634331fa54860","biorxiv","Altered polarization of PAR-2 signaling during airway epithelial remodeling","10.1101/2020.01.09.900555",null,null,"biorxiv","ABSTRACTBackgroundProtease-activated receptor 2 (PAR-2) is activated by proteases involved in allergy and triggers airway epithelial secretion and inflammation. PAR-2 is normally expressed basolaterally in differentiated nasal ciliated cells.ObjectiveWe tested if epithelial remodeling during diseases characterized by loss of cilia and squamous metaplasia may alter PAR-2 polarization.MethodsEndogenous PAR-2 responses were measured by live cell calcium and cilia imaging, measurement of fluid secretion, and quantification of cytokines. We utilized airway squamous cell lines, primary differentiated air-liquid interface cultures, and tissue explants. Cells were exposed to disease-related modifiers that alter epithelial morphology, including IL-13, cigarette smoke condensate, and retinoic acid deficiency. We used concentrations and exposure times that altered epithelial morphology without causing breakdown of the epithelial barrier, likely reflecting early disease states.ResultsPAR-2 signaling in airway squamous cells activated calcium and inflammatory responses. Squamous cells cultured at air liquid interface (ALI) responded to PAR-2 agonists applied both apically and basolaterally. Primary well-differentiated nasal epithelial ALI cultures responded only to basolateral PAR-2 stimulation. Primary cultures exposed to IL-13, cigarette smoke condensate, or reduced retinoic acid responded to both apical and basolateral PAR-2 stimulation. Nasal polyp tissue, but not control middle turbinate, exhibited apical calcium responses to PAR-2 stimulation. However, isolated ciliated cells from both polyp and turbinate maintained basolateral PAR-2 polarization.ConclusionsSquamous metaplasia and/or loss of cilia enhances apical PAR-2 responses. Altered PAR-2 polarization in dedifferentiated or remodeled epithelia may contribute to increased sensitivity to inhaled protease allergens in inflammatory airway diseases.","2020-01-09","Ryan M. Carey; Jenna R. Freund; Benjamin M. Hariri; Nithin D. Adappa; James N. Palmer; Robert J. Lee",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.09.900555"],["wtarrhf0","0624a12abfe85c8b5070850d912a2db4cd453236","biorxiv","TALC: Transcription Aware Long Read Correction","10.1101/2020.01.10.901728",null,null,"biorxiv","ABSTRACTMotivationLong-read sequencing technologies are invaluable for determining complex RNA transcript architectures but are error-prone. Numerous “hybrid correction” algorithms have been developed for genomic data that correct long reads by exploiting the accuracy and depth of short reads sequenced from the same sample. These algorithms are not suited for correcting more complex transcriptome sequencing data.ResultsWe have created a novel algorithm called TALC (Transcription Aware Long Read Correction) which models changes in RNA expression and isoform representation in a weighted De-Bruijn graph to correct long reads from transcriptome studies. We show that transcription aware correction by TALC improves the accuracy of the whole spectrum of downstream RNA-seq applications and is thus necessary for transcriptome analyses that use long read technology.Availability and ImplementationTALC is implemented in C++ and available at https://gitlab.igh.cnrs.fr/lbroseus/TALC.Contactwilliam.ritchie@igh.cnrs.fr","2020-01-11","Lucile Broseus; Aubin Thomas; Andrew J. Oldfield; Dany Severac; Emeric Dubois; William Ritchie",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.10.901728"],["ahrrphm8","0015023cc06b5362d332b3baf348d11567ca2fbb","biorxiv","The RNA pseudoknots in foot-and-mouth disease virus are dispensable for genome replication but essential for the production of infectious virus","10.1101/2020.01.10.901801",null,null,"biorxiv","AbstractThe positive stranded RNA genomes of picornaviruses comprise a single large open reading frame flanked by 5′ and 3′ untranslated regions (UTRs). Foot-and-mouth disease virus (FMDV) has an unusually large 5′ UTR (1.3 kb) containing five structural domains. These include the internal ribosome entry site (IRES), which facilitates initiation of translation, and the cis-acting replication element (cre). Less well characterised structures are a 5′ terminal 360 nucleotide stem-loop, a variable length poly-C-tract of approximately 100-200 nucleotides and a series of two to four tandemly repeated pseudoknots (PKs). We investigated the structures of the PKs by selective 2′ hydroxyl acetylation analysed by primer extension (SHAPE) analysis and determined their contribution to genome replication by mutation and deletion experiments. SHAPE and mutation experiments confirmed the importance of the previously predicted PK structures for their function. Deletion experiments showed that although PKs are not essential for replication, they provide genomes with a competitive advantage. However, although replicons and full-length genomes lacking all PKs were replication competent, no infectious virus was rescued from genomes containing less than one PK copy. This is consistent with our earlier report describing the presence of putative packaging signals in the PK region.","2020-01-11","Joseph C. Ward; Lidia Lasecka-Dykes; Chris Neil; Oluwapelumi Adeyemi; Sarah Gold; Niall McLean; Caroline Wright; Morgan R. Herod; David Kealy; Emma Warner; Donald P. King; Tobias J. Tuthill; David J. Rowlands; Nicola J. Stonehouse",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.10.901801"],["8kced06y","f8e8a9c6f5662c1739be031e6134284ef08ea8b2","biorxiv","Programmable low-cost DNA-based platform for viral RNA detection","10.1101/2020.01.12.902452",null,null,"biorxiv","AbstractViral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ∼24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.","2020-01-16","Lifeng Zhou; Arun Richard Chandrasekaran; Jibin Abraham Punnoose; Gaston Bonenfant; Stephon Charles; Oksana Levchenko; Pheonah Badu; Cassandra Cavaliere; Cara T. Pager; Ken Halvorsen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.12.902452"],["sjicr24s","de3ae7971891058db2f4b36cd216531e6f533332","biorxiv","Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family","10.1101/2020.01.15.908608",null,null,"biorxiv","ABSTRACTThe viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I-β2m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and β2m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.IMPORTANCEUrsidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines.","2020-01-16","Hongyu Yuan; Lizhen Ma; Lijie Zhang; Xiaoying Li; Chun Xia",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.15.908608"],["bf4qpsy7","26cb6703ca72bf97887abbc29a40b1bd9d7890f4","biorxiv","Hotspots of aberrant enhancer activity in fibrolamellar carcinoma reveal molecular mechanisms of oncogenesis and intrinsic drug resistance","10.1101/2020.01.18.911297",null,null,"biorxiv","AbstractFibrolamellar carcinoma (FLC) is a rare, therapeutically intractable liver cancer that disproportionately affects youth. Although FLC tumors exhibit a distinct gene expression profile, the causative transcriptional mechanisms remain unclear. Here we used chromatin run-on sequencing to discover approximately 7,000 enhancers and 141 enhancer hotspots activated in FLC relative to non-malignant liver. Detailed bioinformatic analyses revealed aberrant ERK/MEK signaling and candidate master transcriptional regulators. We also defined the genes most strongly associated with aberrant FLC enhancer activity, including CA12 and SLC16A14. Treatment of FLC cell models with inhibitors of CA12 or SLC16A14 independently reduced cell viability and/or significantly enhanced the effect of MEK inhibitor cobimetinib. These findings highlight new molecular targets for drug development as well as novel drug combination approaches.","2020-01-18","Timothy A. Dinh; Ramja Sritharan; F. Donelson Smith; Adam B. Francisco; Rosanna K. Ma; Rodica P. Bunaciu; Matt Kanke; Charles G. Danko; Andrew P. Massa; John D. Scott; Praveen Sethupathy",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.18.911297"],["v4mbry22","8b88d01563538e4abd29c3ba493c534af30664cc","biorxiv","A mathematical model for simulating the transmission of Wuhan novel Coronavirus","10.1101/2020.01.19.911669",null,null,"biorxiv","AbstractAs reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.","2020-01-19","Tianmu Chen; Jia Rui; Qiupeng Wang; Zeyu Zhao; Jing-An Cui; Ling Yin",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.19.911669"],["3qds5j1p",null,"biorxiv","Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China","10.1101/2020.01.20.913368",null,null,"biorxiv","AbstractDetailed genomic and structure-based analysis of a new coronavirus, namely 2019-nCoV, showed that the new virus is a new type of bat coronavirus and is genetically fairly distant from the human SARS coronavirus. Structure analysis of the spike (S) protein of this new virus showed that its S protein only binds weakly to the ACE2 receptor on human cells whereas the human SARS coronavirus exhibits strongly affinity to the ACE receptor. These findings suggest that the new virus does not readily transmit between humans and should theoretically not able to cause very serious human infection. These data are important to guide design of infection control policy and inform the public on the nature of threat imposed by 2019-nCov when results of direct laboratory tests on this virus are not expected to be available in the near future.","2020-01-21","Ning Dong; Xuemei Yang; Lianwei Ye; Kaichao Chen; Edward Wai-Chi Chan; Mengsu Yang; Sheng Chen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.20.913368"],["t9bt70f2","a05179c57d5e1cdddc839ff3c0b70a614efbb6cb","biorxiv","Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm","10.1101/2020.01.21.914044",null,null,"biorxiv","AbstractThe recent outbreak of pneumonia in Wuhan, China caused by the 2019 Novel Coronavirus (2019-nCoV) emphasizes the importance of detecting novel viruses and predicting their risks of infecting people. In this report, we introduced the VHP (Virus Host Prediction) to predict the potential hosts of viruses using deep learning algorithm. Our prediction suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially the severe acute respiratory syndrome coronavirus (SARS-CoV), Bat SARS-like Coronaviruses and the Middle East respiratory syndrome coronavirus (MERS-CoV). Based on our prediction, compared to the Coronaviruses infecting other vertebrates, bat coronaviruses are assigned with more similar infectivity patterns with 2019-nCoVs. Furthermore, by comparing the infectivity patterns of all viruses hosted on vertebrates, we found mink viruses show a closer infectivity pattern to 2019-nCov. These consequences of infectivity pattern analysis illustrate that bat and mink may be two candidate reservoirs of 2019-nCov.These results warn us to beware of 2019-nCoV and guide us to further explore the properties and reservoir of it.One Sentence SummaryIt is of great value to identify whether a newly discovered virus has the risk of infecting human. Guo et al. proposed a virus host prediction method based on deep learning to detect what kind of host a virus can infect with DNA sequence as input. Applied to the Wuhan 2019 Novel Coronavirus, our prediction demonstrated that several vertebrate-infectious coronaviruses have strong potential to infect human. This method will be helpful in future viral analysis and early prevention and control of viral pathogens.","2020-01-24","Qian Guo; Mo Li; Chunhui Wang; Peihong Wang; Zhencheng Fang; Jie tan; Shufang Wu; Yonghong Xiao; Huaiqiu Zhu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.21.914044"],["f1ixbzx8","403d885a33958731c4607ea632453e25a89e2b57","biorxiv","Novel ionophores active against La Crosse virus identified through rapid antiviral screening","10.1101/2020.01.21.914929",null,null,"biorxiv","AbstractBunyaviruses are significant human pathogens, causing diseases ranging from hemorrhagic fevers to encephalitis. Among these viruses, La Crosse virus (LACV), a member of the California serogroup, circulates in the eastern and midwestern United States. While LACV infection is often asymptomatic, dozens of cases of encephalitis are reported yearly. Unfortunately, no antivirals have been approved to treat LACV infection. Here, we developed a method to rapidly test potential antivirals against LACV infection. From this screen, we identified several potential antiviral molecules, including known antivirals. Additionally, we identified many novel antivirals that exhibited antiviral activity without affecting cellular viability. Valinomycin, a potassium ionophore, was among our top targets. We found that valinomycin exhibited potent anti-LACV activity in multiple cell types in a dose-dependent manner. Valinomycin did not affect particle stability or infectivity, suggesting that it may preclude virus replication by altering cellular potassium ions, a known determinant of LACV entry. We extended these results to other ionophores and found that the antiviral activity of valinomycin extended to other viral families including bunyaviruses (Rift Valley fever virus, Keystone virus), enteroviruses (Coxsackievirus, rhinovirus), flavirivuses (Zika), and coronaviruses (229E and MERS-CoV). In all viral infections, we observed significant reductions in virus titer in valinomycin-treated cells. In sum, we demonstrate the importance of potassium ions to virus infection, suggesting a potential therapeutic target to disrupt virus replication.ImportanceNo antivirals are approved for the treatment of bunyavirus infection. The ability to rapidly screen compounds and identify novel antivirals is one means to accelerate drug discovery for viruses with no approved treatments. We used this approach to screen hundreds of compounds against La Crosse virus, an emerging bunyavirus that causes significant disease, including encephalitis. We identified several known and previously unidentified antivirals. We focused on a potassium ionophore, valinomycin, due to its promising in vitro antiviral activity. We demonstrate that valinomycin, as well as a selection of other ionophores, exhibits activity against La Crosse virus as well as several other distantly related bunyaviruses. We finally observe that valinomycin has activity against a wide array of human viral pathogens, suggesting that disrupting potassium ion homeostasis with valinomycin may be a potent host pathway to target to quell virus infection.","2020-01-23","Zachary J. Sandler; Michelle N. Vu; Vineet D. Menachery; Bryan C. Mounce",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.21.914929"],["o47v5vgw",null,"biorxiv","Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin","10.1101/2020.01.22.914952",null,null,"biorxiv","Since the SARS outbreak 18 years ago, a large number of severe acute respiratory syndrome related coronaviruses (SARSr-CoV) have been discovered in their natural reservoir host, bats1-4. Previous studies indicated that some of those bat SARSr-CoVs have the potential to infect humans5-7. Here we report the identification and characterization of a novel coronavirus (nCoV-2019) which caused an epidemic of acute respiratory syndrome in humans, in Wuhan, China. The epidemic, started from December 12th, 2019, has caused 198 laboratory confirmed infections with three fatal cases by January 20th, 2020. Full-length genome sequences were obtained from five patients at the early stage of the outbreak. They are almost identical to each other and share 79.5% sequence identify to SARS-CoV. Furthermore, it was found that nCoV-2019 is 96% identical at the whole genome level to a bat coronavirus. The pairwise protein sequence analysis of seven conserved non-structural proteins show that this virus belongs to the species of SARSr-CoV. The nCoV-2019 virus was then isolated from the bronchoalveolar lavage fluid of a critically ill patient, which can be neutralized by sera from several patients. Importantly, we have confirmed that this novel CoV uses the same cell entry receptor, ACE2, as SARS-CoV.","2020-01-23","Peng Zhou; Xing-Lou Yang; Xian-Guang Wang; Ben Hu; Lei Zhang; Wei Zhang; Hao-Rui Si; Yan Zhu; Bei Li; Chao-Lin Huang; Hui-Dong Chen; Jing Chen; Yun Luo; Hua Guo; Ren-Di Jiang; Mei-Qin Liu; Ying Chen; Xu-Rui Shen; Xi Wang; Xiao-Shuang Zheng; Kai Zhao; Quan-Jiao Chen; Fei Deng; Lin-Lin Liu; Bing Yan; Fa-Xian Zhan; Yan-Yi Wang; Geng-Fu Xiao; Zheng-Li Shi",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.22.914952"],["tyhwtt7j","9daed5e675ac113d1939056dcebbb1f32ec0d7ec","biorxiv","Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV","10.1101/2020.01.22.915660",null,null,"biorxiv","AbstractOver the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.","2020-01-22","Michael Letko; Vincent Munster",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.22.915660"],["mt8h4rk5","6cb02c7565f6f74a3d165e14196de5e9e87d2d04","medrxiv","Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions","10.1101/2020.01.23.20018549",null,null,"medrvix","Since first identified, the epidemic scale of the recently emerged novel coronavirus (2019-nCoV) in Wuhan, China, has increased rapidly, with cases arising across China and other countries and regions. using a transmission model, we estimate a basic reproductive number of 3.11 (95%CI, 2.39-4.13); 58-76% of transmissions must be prevented to stop increasing; Wuhan case ascertainment of 5.0% (3.6-7.4); 21022 (11090-33490) total infections in Wuhan 1 to 22 January.","2020-01-24","Jonathan M Read; Jessica RE Bridgen; Derek AT Cummings; Antonia Ho; Chris P Jewell",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.23.20018549"],["px70bft2","24ff82a421f1a33db3ebaa22652bd62e0cb77c57","biorxiv","Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak","10.1101/2020.01.23.916395",null,null,"biorxiv","AbstractBackgroundsAn ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number, R0, of 2019-nCoV in the early phase of the outbreak.MethodsAccounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (γ), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI.FindingsThe early outbreak data largely follows the exponential growth. We estimated that the mean R0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0.ConclusionThe mean estimate of R0 for the 2019-nCoV ranges from 2.24 to 3.58, and significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.","2020-01-24","Shi Zhao; Qianyin Lin; Jinjun Ran; Salihu S Musa; Guangpu Yang; Weiming Wang; Yijun Lou; Daozhou Gao; Lin Yang; Daihai He; Maggie H Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.23.916395"],["59rcogfc","fe66306680e5be9820e2cc3742b663a782eb1b31","biorxiv","Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China","10.1101/2020.01.23.916726",null,null,"biorxiv","We present a timely evaluation of the Chinese 2019-nCov epidemic in its initial phase, where 2019-nCov demonstrates comparable transmissibility but lower fatality rates than SARS and MERS. A quick diagnosis that leads to case isolation and integrated interventions will have a major impact on its future trend. Nevertheless, as China is facing its Spring Festival travel rush and the epidemic has spread beyond its borders, further investigation on its potential spatiotemporal transmission pattern and novel intervention strategies are warranted.","2020-01-25","Mingwang Shen; Zhihang Peng; Yanni Xiao; Lei Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.23.916726"],["jjkf5gg3","671b44ae9bec89cacf7ae9e7b2a79649a9b66773","biorxiv","Pattern of early human-to-human transmission of Wuhan 2019-nCoV","10.1101/2020.01.23.917351",null,null,"biorxiv","ABSTRACTOn December 31, 2019, the World Health Organization was notified about a cluster of pneumonia of unknown aetiology in the city of Wuhan, China. Chinese authorities later identified a new coronavirus (2019-nCoV) as the causative agent of the outbreak. As of January 23, 2020, 655 cases have been confirmed in China and several other countries. Understanding the transmission characteristics and the potential for sustained human-to-human transmission of 2019-nCoV is critically important for coordinating current screening and containment strategies, and determining whether the outbreak constitutes a public health emergency of international concern (PHEIC). We performed stochastic simulations of early outbreak trajectories that are consistent with the epidemiological findings to date. We found the basic reproduction number, R0, to be around 2.2 (90% high density interval 1.4—3.8), indicating the potential for sustained human-to-human transmission. Transmission characteristics appear to be of a similar magnitude to severe acute respiratory syndrome-related coronavirus (SARS-CoV) and the 1918 pandemic influenza. These findings underline the importance of heightened screening, surveillance and control efforts, particularly at airports and other travel hubs, in order to prevent further international spread of 2019-nCoV.","2020-01-24","Julien Riou; Christian L. Althaus",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.23.917351"],["j99qg2a0","e4cb4af2f4ec6f023096afcf7b3d4410584e1393","biorxiv","Designing Effective small interfering RNA for Post-Transcriptional Silencing of Human GREM1: A Comprehensive Bioinformatics Approach","10.1101/2020.01.23.917559",null,null,"biorxiv","ABSTRACTHuman gremlin-1 is a physiologically versatile signaling molecule that has been associated with several human diseases including cancer. The ability of gremlin-1 to induce fibrosis in organs and transduce angiogenesis makes it a target for cancer therapy. RNAi-based therapy has proven to be very efficient and specific in tumor growth inhibition. The efficacy and specificity of siRNA-mediated gene silencing depends on the designing approaches. Here, empirical guidelines for siRNA design and comprehensive target site availability analysis were used to select effective siRNA from a plethora of potential candidates designed using several computation algorithms. Then, the selected siRNA candidates were subjected to stringent similarity searches in order to obtain siRNA candidates with reduced off-target effects (high specificity). The best candidates were compared to experimentally successful gremlin-1 siRNAs in order to predict the silencing potency of the selected siRNAs. siRNA-6 (sense strand: 5’-CCAAGAAAUUCACUACCAU-3’), siRNA-7 (sense strand: 5’-CCAUGAUGGUCACACUCAA-3’) and siRNA-47 (sense strand: 5’-GGCCCAGCACAAUGACUCA-3’) were predicted to be highly effective siRNA candidates for gremlin-1 silencing. These siRNAs can be considered for RNAi-based therapy because off-target effects are predicted to be minimal.","2020-01-24","Asiedu Ebenezer",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.23.917559"],["u2w2bb2p","a3116a1ea2e157c8855023d968cc0f0bd6c9846f","biorxiv","The 2019-new coronavirus epidemic: evidence for virus evolution","10.1101/2020.01.24.915157",null,null,"biorxiv","AbstractThere is concern about a new coronavirus, the 2019-nCoV, as a global public health threat. In this article, we provide a preliminary evolutionary and molecular epidemiological analysis of this new virus. A phylogenetic tree has been built using the 15 available whole genome sequence of 2019-nCoV and 12 whole genome sequences highly similar sequences available in gene bank (5 from SARS, 2 from MERS and 5 from Bat SARS-like Coronavirus). FUBAR analysis shows that the Nucleocapsid and the Spike Glycoprotein has some sites under positive pressure while homology modelling helped to explain some molecular and structural differences between the viruses. The phylogenetic tree showed that 2019.nCoV significantly clustered with Bat SARS-like Coronavirus sequence isolated in 2015, whereas structural analysis revealed mutation in S and nucleocapsid proteins. From these results, 2019nCoV could be considered a coronavirus distinct from SARS virus, probably transmitted from bats or another host where mutations conferred upon it the ability to infect humans.","2020-01-24","Domenico Benvenuto; Marta Giovannetti; Alessandra Ciccozzi; Silvia Spoto; Silvia Angeletti; Massimo Ciccozzi",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.24.915157"],["qpaig5tm","b6e92eda6c26b456165422d4e5552b81754c4a82","biorxiv","2019-20 Wuhan coronavirus outbreak: Intense surveillance is vital for preventing sustained transmission in new locations","10.1101/2020.01.24.919159",null,null,"biorxiv","ABSTRACTThe outbreak of pneumonia originating in Wuhan, China, has generated 830 confirmed cases, including 26 deaths, as of 24 January 2020. The virus (2019-nCoV) has spread elsewhere in China and to other countries, including South Korea, Thailand, Japan and USA. Fortunately, there has not yet been evidence of sustained human-to-human transmission outside of China. Here we assess the risk of sustained transmission whenever the coronavirus arrives in other countries. Data describing the times from symptom onset to hospitalisation for 47 patients infected in the current outbreak are used to generate an estimate for the probability that an imported case is followed by sustained human-to-human transmission. Under the assumptions that the imported case is representative of the patients in China, and that the 2019-nCoV is similarly transmissible to the SARS coronavirus, the probability that an imported case is followed by sustained human-to-human transmission is 0.37. However, if the mean time from symptom onset to hospitalisation can be halved by intense surveillance, then the probability that an imported case leads to sustained transmission is only 0.005. This emphasises the importance of current surveillance efforts in countries around the world, to ensure that the ongoing outbreak will not become a large global epidemic.","2020-01-25","R.N. Thompson",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.24.919159"],["5jai745w","bb97f90ff5bfd41f2f1148c5cbdb257fdba535df","biorxiv","Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China","10.1101/2020.01.24.919183",null,null,"biorxiv","Emerging and re-emerging infectious diseases, such as SARS, MERS, Zika and highly pathogenic influenza present a major threat to public health1–3. Despite intense research effort, how, when and where novel diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city of Wuhan, Hubei province, China. At the time of writing, at least 62 suspected cases have been reported since the first patient was hospitalized on December 12nd 2019. Epidemiological investigation by the local Center for Disease Control and Prevention (CDC) suggested that the outbreak was associated with a sea food market in Wuhan. We studied seven patients who were workers at the market, and collected bronchoalveolar lavage fluid (BALF) from one patient who exhibited a severe respiratory syndrome including fever, dizziness and cough, and who was admitted to Wuhan Central Hospital on December 26th 2019. Next generation metagenomic RNA sequencing4 identified a novel RNA virus from the family Coronaviridae designed WH-Human-1 coronavirus (WHCV).Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that WHCV was most closely related (89.1% nucleotide similarity similarity) to a group of Severe Acute Respiratory Syndrome (SARS)-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China and that have a history of genomic recombination. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.","2020-01-25","Fan Wu; Su Zhao; Bin Yu; Yan-Mei Chen; Wen Wang; Yi Hu; Zhi-Gang Song; Zhao-Wu Tao; Jun-Hua Tian; Yuan-Yuan Pei; Ming-Li Yuan; Yu-Ling Zhang; Fa-Hui Dai; Yi Liu; Qi-Min Wang; Jiao-Jiao Zheng; Lin Xu; Edward C. Holmes; Yong-Zhen Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.24.919183"],["hab8h5gz",null,"biorxiv","From SARS-CoV to Wuhan 2019-nCoV Outbreak: Similarity of Early Epidemic and Prediction of Future Trends","10.1101/2020.01.24.919241",null,null,"biorxiv","This manuscript has been withdrawn as it was submitted without the full consent of all the authors. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.","2020-01-25","Zeliang Chen; Wenjun Zhang; Yi Lu; Cheng Guo; Zhongmin Guo; Conghui Liao; Xi Zhang; Yi Zhang; Xiaohu Han; Qianlin Li; W. Ian Lipkin; Jiahai Lu",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.24.919241"],["9d8k7b23","e29b287d326be9c2c0a7dcd7a81cb49cc2125360","biorxiv","Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes","10.1101/2020.01.24.919282",null,null,"biorxiv","AbstractThe incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, Aedes aegypti, and its microbiome, which has been shown to influence virus transmission success. We examined the ‘metavirome’ of four populations of Ae. aegypti mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in Ae. aegypti that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.ImportanceSince 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks.","2020-01-25","Sean M. Boyles; Carla N. Mavian; Esteban Finol; Maria Ukhanova; Caroline J. Stephenson; Gabriela Hamerlinck; Seokyoung Kang; Caleb Baumgartner; Mary Geesey; Israel Stinton; Kate Williams; Derrick K. Mathias; Mattia Prosperi; Volker Mai; Marco Salemi; Eva A. Buckner; John A. Lednicky; Adam R. Rivers; Rhoel R. Dinglasan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.24.919282"],["t1iagum7",null,"biorxiv","MRCA time and epidemic dynamics of the 2019 novel coronavirus","10.1101/2020.01.25.919688",null,null,"biorxiv","AbstractThe 2019 novel coronavirus (2019-nCoV) have emerged from Wuhan, China. Studying the epidemic dynamics is crucial for further surveillance and control of the outbreak. We employed a Bayesian framework to infer the time-calibrated phylogeny and the epidemic dynamics represented by the effective reproductive number (Re) changing over time from 33 genomic sequences available from GISAID. The time of the most recent common ancestor (MRCA) was December 17, 2019 (95% HPD: December 7, 2019 – December 23, 2019). The median estimate of Re shifted from 1.6 to 1.1 on around January 1, 2020. This study provides an early insight of the 2019-nCoV epidemic. However, due to limited amount of data, one should be cautious when interpreting the results at this stage.","2020-01-26","Chi Zhang; Mei Wang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.25.919688"],["3e2soc6w","abcfffafab399149d4adadd6bb458c4994e2025d","biorxiv","Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China","10.1101/2020.01.25.919787",null,null,"biorxiv","ABSTRACTRationaleSeveral studies have estimated basic production number of novel coronavirus pneumonia (NCP). However, the time-varying transmission dynamics of NCP during the outbreak remain unclear.ObjectivesWe aimed to estimate the basic and time-varying transmission dynamics of NCP across China, and compared them with SARS.MethodsData on NCP cases by February 7, 2020 were collected from epidemiological investigations or official websites. Data on severe acute respiratory syndrome (SARS) cases in Guangdong Province, Beijing and Hong Kong during 2002-2003 were also obtained. We estimated the doubling time, basic reproduction number (R0) and time-varying reproduction number (Rt) of NCP and SARS.Measurements and main resultsAs of February 7, 2020, 34,598 NCP cases were identified in China, and daily confirmed cases decreased after February 4. The doubling time of NCP nationwide was 2.4 days which was shorter than that of SARS in Guangdong (14.3 days), Hong Kong (5.7 days) and Beijing (12.4 days). The R0 of NCP cases nationwide and in Wuhan were 4.5 and 4.4 respectively, which were higher than R0 of SARS in Guangdong (R0=2.3), Hongkong (R0=2.3), and Beijing (R0=2.6). The Rt for NCP continuously decreased especially after January 16 nationwide and in Wuhan. The R0 for secondary NCP cases in Guangdong was 0.6, and the Rt values were less than 1 during the epidemic.ConclusionsNCP may have a higher transmissibility than SARS, and the efforts of containing the outbreak are effective. However, the efforts are needed to persist in for reducing time-varying reproduction number below one.At a Glance CommentaryScientific Knowledge on the SubjectSince December 29, 2019, pneumonia infection with 2019-nCoV, now named as Novel Coronavirus Pneumonia (NCP), occurred in Wuhan, Hubei Province, China. The disease has rapidly spread from Wuhan to other areas. As a novel virus, the time-varying transmission dynamics of NCP remain unclear, and it is also important to compare it with SARS.What This Study Adds to the FieldWe compared the transmission dynamics of NCP with SARS, and found that NCP has a higher transmissibility than SARS. Time-varying production number indicates that rigorous control measures taken by governments are effective across China, and persistent efforts are needed to be taken for reducing instantaneous reproduction number below one.","2020-01-26","Tao Liu; Jianxiong Hu; Jianpeng Xiao; Guanhao He; Min Kang; Zuhua Rong; Lifeng Lin; Haojie Zhong; Qiong Huang; Aiping Deng; Weilin Zeng; Xiaohua Tan; Siqing Zeng; Zhihua Zhu; Jiansen Li; Dexin Gong; Donghua Wan; Shaowei Chen; Lingchuan Guo; Yan Li; Limei Sun; Wenjia Liang; Tie Song; Jianfeng He; Wenjun Ma",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.25.919787"],["ykofrn9i","cbc05d14c57b91081970a232ab83bc993f998fe2","medrxiv","Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data","10.1101/2020.01.26.20018754",null,null,"medrvix","The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.","2020-01-28","Natalie M. Linton; Tetsuro Kobayashi; Yichi Yang; Katsuma Hayashi; Andrei R. Akhmetzhanov; Sung-mok Jung; Baoyin Yuan; Ryo Kinoshita; Hiroshi Nishiura",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.26.20018754"],["n93cm6ws",null,"medrxiv","Epidemiological identification of a novel infectious disease in real time: Analysis of the atypical pneumonia outbreak in Wuhan, China, 2019-20","10.1101/2020.01.26.20018887",null,null,"medrvix","Objective: Virological tests indicate that a novel coronavirus is the most likely explanation for the 2019-20 pneumonia outbreak in Wuhan, China. We demonstrate that non-virological descriptive characteristics could have determined that the outbreak is caused by a novel pathogen in advance of virological testing. Methods: Characteristics of the ongoing outbreak were collected in real time from two medical social media sites. These were compared against characteristics of ten existing pathogens that can induce atypical pneumonia. The probability that the current outbreak is due to \"Disease X\" (i.e., previously unknown etiology) as opposed to one of the known pathogens was inferred, and this estimate was updated as the outbreak continued. Results: The probability that Disease X is driving the outbreak was assessed as over 32% on 31 December 2019, one week before virus identification. After some specific pathogens were ruled out by laboratory tests on 5 Jan 2020, the inferred probability of Disease X was over 59%. Conclusions: We showed quantitatively that the emerging outbreak of atypical pneumonia cases is consistent with causation by a novel pathogen. The proposed approach, that uses only routinely-observed non-virological data, can aid ongoing risk assessments even before virological test results become available. Keywords: Epidemic; Causation; Bayes' theorem; Diagnosis; Prediction; Statistical model","2020-01-28","Sung-mok Jung; Ryo Kinoshita; Robin N. Thompson; Katsuma Hayashi; Natalie M. Linton; Yichi Yang; Andrei R. Akhmetzhanov; Hiroshi Nishiura",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.26.20018887"],["4pgish5x","68e509d42a7d3349550512d80eca4e2b2123613f","biorxiv","Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCov","10.1101/2020.01.26.919985",null,null,"biorxiv","AbstractA novel coronavirus (2019-nCov) was identified in Wuhan, Hubei Province, China in December of 2019. This new coronavirus has resulted in thousands of cases of lethal disease in China, with additional patients being identified in a rapidly growing number internationally. 2019-nCov was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-Cov. Here based on the public database and the state-of-the-art single-cell RNA-Seq technique, we analyzed the ACE2 RNA expression profile in the normal human lungs. The result indicates that the ACE2 virus receptor expression is concentrated in a small population of type II alveolar cells (AT2). Surprisingly, we found that this population of ACE2-expressing AT2 also highly expressed many other genes that positively regulating viral reproduction and transmission. A comparison between eight individual samples demonstrated that the Asian male one has an extremely large number of ACE2-expressing cells in the lung. This study provides a biological background for the epidemic investigation of the 2019-nCov infection disease, and could be informative for future anti-ACE2 therapeutic strategy development.","2020-01-26","Yu Zhao; Zixian Zhao; Yujia Wang; Yueqing Zhou; Yu Ma; Wei Zuo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.26.919985"],["k2qj6o7j",null,"biorxiv","Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event","10.1101/2020.01.26.920249",null,null,"biorxiv","AbstractBackgroundA novel coronavirus (2019-nCoV) associated with human to human transmission and severe human infection has been recently reported from the city of Wuhan in China. Our objectives were to characterize the genetic relationships of the 2019-nCoV and to search for putative recombination within the subgenus of sarbecovirus.MethodsPutative recombination was investigated by RDP4 and Simplot v3.5.1 and discordant phylogenetic clustering in individual genomic fragments was confirmed by phylogenetic analysis using maximum likelihood and Bayesian methods.ResultsOur analysis suggests that the 2019-nCoV although closely related to BatCoV RaTG13 sequence throughout the genome (sequence similarity 96.3%), shows discordant clustering with the Bat-SARS-like coronavirus sequences. Specifically, in the 5’-part spanning the first 11,498 nucleotides and the last 3’-part spanning 24,341-30,696 positions, 2019-nCoV and RaTG13 formed a single cluster with Bat-SARS-like coronavirus sequences, whereas in the middle region spanning the 3’-end of ORF1a, the ORF1b and almost half of the spike regions, 2019-nCoV and RaTG13 grouped in a separate distant lineage within the sarbecovirus branch.ConclusionsThe levels of genetic similarity between the 2019-nCoV and RaTG13 suggest that the latter does not provide the exact variant that caused the outbreak in humans, but the hypothesis that 2019-nCoV has originated from bats is very likely. We show evidence that the novel coronavirus (2019-nCov) is not-mosaic consisting in almost half of its genome of a distinct lineage within the betacoronavirus. These genomic features and their potential association with virus characteristics and virulence in humans need further attention.","2020-01-27","D. Paraskevis; E.G. Kostaki; G. Magiorkinis; G. Panayiotakopoulos; S. Tsiodras",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.26.920249"],["u8goc7io","12fac9aedb1a09a3922a3c084ce4723708e463d6","medrxiv","The incubation period of 2019-nCoV infections among travellers from Wuhan, China","10.1101/2020.01.27.20018986",null,null,"medrvix","Currently, a novel coronavirus 2019-nCoV causes an outbreak of viral pneumonia in Wuhan, China. Little is known about its epidemiological characteristics. Using the travel history and symptom onset of 88 confirmed cases that were detected outside Wuhan, we estimate the mean incubation period to be 6.4 (5.6 - 7.7, 95% CI) days, ranging from 2.1 to 11.1 days (2.5th to 97.5th percentile). These values help to inform case definitions for 2019-nCoV and appropriate durations for quarantine.","2020-01-28","Jantien A. Backer; Don Klinkenberg; Jacco Wallinga",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.27.20018986"],["2ygtr69i","029c1c588047f1d612a219ee15494d2d19ff7439","biorxiv","Protective Population Behavior Change in Outbreaks of Emerging Infectious Disease","10.1101/2020.01.27.921536",null,null,"biorxiv","ABSTRACTDuring outbreaks of emerging infections, the lack of effective drugs and vaccines increases reliance on non-pharmacologic public health interventions and behavior change to limit human-to-human transmission. Interventions that increase the speed with which infected individuals remove themselves from the susceptible population are paramount, particularly isolation and hospitalization. Ebola virus disease (EVD), Severe Acute Respiratory Syndrome (SARS), and Middle East Respiratory Syndrome (MERS) are zoonotic viruses that have caused significant recent outbreaks with sustained human-to-human transmission. This investigation quantified changing mean removal rates (MRR) and days from symptom onset to hospitalization (DSOH) of infected individuals from the population in seven different outbreaks of EVD, SARS, and MERS, to test for statistically significant differences in these metrics between outbreaks. We found that epidemic week and viral serial interval were correlated with the speed with which populations developed and maintained health behaviors in each outbreak.","2020-01-28","Evans K. Lodge; Annakate M. Schatz; John M. Drake",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.27.921536"],["zgdbvfi4","06a1002f9fbea7179ac3572843f66b14568af6e4","biorxiv","Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation","10.1101/2020.01.27.921627",null,null,"biorxiv","Abstract2019-nCov has caused more than 80 deaths as of 27 January 2020 in China, and infection cases have been reported in more than 10 countries. However, there is no approved drug to treat the disease. 2019-nCov Mpro is a potential drug target to combat the virus. We built homology models based on SARS Mpro structures, and docked 1903 small molecule drugs to the models. Based on the docking score and the 3D similarity of the binding mode to the known Mpro ligands, 4 drugs were selected for binding free energy calculations. Both MM/GBSA and SIE methods voted for nelfinavir, with the binding free energy of −24.69±0.52 kcal/mol and −9.42±0.04 kcal/mol, respectively. Therefore, we suggested that nelfinavir might be a potential inhibitor against 2019-nCov Mpro.","2020-01-28","Zhijian Xu; Cheng Peng; Yulong Shi; Zhengdan Zhu; Kaijie Mu; Xiaoyu Wang; Weiliang Zhu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.27.921627"],["ely200x3","aef0734eb2538d3624d650a4ae20c3f66cace5f2","biorxiv","Chronic hM3Dq signaling in microglia ameliorates neuroinflammation in male mice","10.1101/2020.01.27.921809",null,null,"biorxiv","AbstractMicroglia express muscarinic G protein-coupled receptors (GPCRs) that sense cholinergic activity and are activated by acetylcholine to potentially regulate microglial functions. Knowledge about how distinct types of muscarinic GPCR signaling regulate microglia function in vitro and in vivo is still poor, partly due to the fact that some of these receptors are also present in astrocytes and neurons. We generated mice expressing the hM3Dq Designer Receptor Exclusively Activated by Designer Drugs (DREADD) selectively in microglia to investigate the role of muscarinic M3Gq-linked signaling. We show that activation of hM3Dq using clozapine N-oxide (CNO) elevated intracellular calcium levels and increased phagocytosis of FluoSpheres in vitro. Acute treatment with CNO in vivo did not affect male mouse behavior, however chronic CNO treatment decreased sickness behavior triggered by lipopolysaccharide (LPS) treatment. Interestingly, whereas acute treatment with CNO increased synthesis of cytokine mRNA, chronic treatment attenuated LPS-induced cytokine mRNA changes in the brain, likely explaining the improvement in sickness behavior by chronic hM3Dq activation. No effect of CNO was observed in DREADD-negative mice. These results suggest that chronic activation of M3 muscarinic receptors (the hM3Dq progenitor) in microglia, and potentially other Gq-coupled GPCRs, preconditions microglia to decrease their response to further immunological challenges. Our results indicate that hM3Dq can be a useful tool to modulate neuroinflammation and study microglial immunological memory in vivo, which may be applicable for manipulations of neuroinflammation in neurodegenerative and psychiatric diseases.HighlightsMicroglial function was manipulated by specific activation of hM3Dq signaling.Chronic hM3Dq activation prevented LPS-induced sickness behavior in mice.Microglial hM3Dq signaling modulated expression of pro-inflammatory cytokines.","2020-01-28","William Binning; Aja E. Hogan-Cann; Diana Yae Sakae; Matthew Maksoud; Valeriy Ostapchenko; Mohammed Al-Onaizi; Sara Matovic; Wei-Yang Lu; Marco A. M. Prado; Wataru Inoue; Vania F. Prado",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.27.921809"],["si486lqi","754315299d847600d6c5d414665c728d40bf731d","biorxiv","Breaking down of the healthcare system: Mathematical modelling for controlling the novel coronavirus (2019-nCoV) outbreak in Wuhan, China","10.1101/2020.01.27.922443",null,null,"biorxiv","AbstractA novel coronavirus pneumonia initially identified in Wuhan, China and provisionally named 2019-nCoV has surged in the public. In anticipation of substantial burdens on healthcare system following this human-to-human spread, we aim to scrutinise the currently available information and evaluate the burden of healthcare systems during this outbreak in Wuhan. We applied a modified SIR model to project the actual number of infected cases and the specific burdens on isolation wards and intensive care units, given the scenarios of different diagnosis rates as well as different public health intervention efficacy. Our estimates suggest the actual number of infected cases could be much higher than the reported, with estimated 26,701 cases (as of 28th January 2020) assuming 50% diagnosis rate if no public health interventions were implemented. The estimated burdens on healthcare system could be largely reduced if at least 70% efficacy of public health intervention is achieved.","2020-01-28","Wai-Kit Ming; Jian Huang; Casper J. P. Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.27.922443"],["x22rc60j","30b5c7b8faf95265f67ae59f4686eaf9b2772893","medrxiv","Estimated effectiveness of traveller screening to prevent international spread of 2019 novel coronavirus (2019-nCoV)","10.1101/2020.01.28.20019224",null,null,"medrvix","Traveller screening is being used to limit further global spread of 2019 novel coronavirus (nCoV) following its recent emergence. Here, we analyze the expected impact of different travel screening programs given remaining uncertainty around the values of key nCoV life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss around half of infected travellers. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. These findings emphasize the need for measures to track travellers who become ill after being missed by a travel screening program. We make our model available for interactive use so stakeholders can explore scenarios of interest using the most up-to-date information. We hope these findings contribute to evidence-based policy to combat the spread of nCoV, and to prospective planning to mitigate future emerging pathogens.","2020-01-30","Katelyn Gostic; Ana C. R. Gomez; Riley O. Mummah; Adam J. Kucharski; James O. Lloyd-Smith",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.28.20019224"],["mbdj3r0m","957e71389292e0881dff2bbe0e87fe6bf8b2d381","medrxiv","Risk for Transportation of 2019 Novel Coronavirus (COVID-19) from Wuhan to Cities in China","10.1101/2020.01.28.20019299",null,null,"medrvix","On January 23, 2020, China quarantined Wuhan to contain an emerging coronavirus (COVID-19). We estimated the probability of transportation of COVID-19 from Wuhan to 369 cities in China before the quarantine. The expected risk is >50% in 130 (95% CI 89-190) cities and >99% in the 4 largest metropolitan areas of China.","2020-01-30","Zhanwei Du; Ling Wang; Simon Cauchemez; Xiaoke Xu; Xianwen Wang; Benjamin J Cowling; Lauren Ancel Meyers",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.28.20019299"],["635lbedk","d49f8ba7212183bc1b7f2cb6dd709dd197303b3d","biorxiv","Spreading predictability in complex networks","10.1101/2020.01.28.922757",null,null,"biorxiv","AbstractSpreading dynamics analysis is an important and interesting topic since it has many applications such as rumor or disease controlling, viral marketing and information recommending. Many state-of-the-art researches focus on predicting infection scale or threshold. Few researchers pay attention to the predicting of infection nodes from a snapshot. With developing of precision marketing, recommending and, controlling, how to predict infection nodes precisely from snapshot becomes a key issue in spreading dynamics analysis. In this paper, a probability based prediction model is presented so as to estimate the infection nodes from a snapshot of spreading. Experimental results on synthetic and real networks demonstrate that the model proposed could predict the infection nodes precisely in the sense of probability.","2020-01-28","Na Zhao; Jian Wang; Yong Yu; Jun-Yan Zhao; Duan-Bing Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.28.922757"],["iu2ziccm","c6ec1380338cb2467dc6fc14add09ca5bea18411","biorxiv","Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening","10.1101/2020.01.28.922922",null,null,"biorxiv","Abstract2019 Novel Coronavirus (2019-nCoV) is a virus identified as the cause of the outbreak of pneumonia first detected in Wuhan, China. Investigations on the transmissibility, severity, and other features associated with this virus are ongoing. Currently, there is no vaccine or therapeutic antibody to prevent the infection, and more time is required to develop an effective immune strategy against the pathogen. In contrast, specific inhibitors targeting the key protease involved in replication and proliferation of the virus are the most effective means to alleviate the epidemic. The main protease of SARS-CoV is essential for the life cycle of the virus, which showed 96.1% of similarity with the main proteaseof 2019-nCoV, is considered to be an attractive target for drug development. In this study, we have identified 4 small molecular drugs with high binding capacity with SARS-CoV main protease by high-throughput screening based on the 8,000 clinical drug libraries, all these drugs have been widely used in clinical applications with guaranteed safety, which may serve as promising candidates to treat the infection of 2019-nCoV.","2020-01-29","Yan Li; Jinyong Zhang; Ning Wang; Haibo Li; Yun Shi; Gang Guo; Kaiyun Liu; Hao Zeng; Quanming Zou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.28.922922"],["lzda8kyn",null,"biorxiv","Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody","10.1101/2020.01.28.923011",null,null,"biorxiv","ABSTRACTThe newly identified 2019 novel coronavirus (2019-nCoV) has caused more than 800 laboratory-confirmed human infections, including 25 deaths, posing a serious threat to human health. Currently, however, there is no specific antiviral treatment or vaccine. Considering the relatively high identity of receptor binding domain (RBD) in 2019-nCoV and SARS-CoV, it is urgent to assess the cross-reactivity of anti-SARS-CoV antibodies with 2019-nCoV spike protein, which could have important implications for rapid development of vaccines and therapeutic antibodies against 2019-nCoV. Here, we report for the first time that a SARS-CoV-specific human monoclonal antibody, CR3022, could bind potently with 2019-nCoV RBD (KD of 6.3 nM). The epitope of CR3022 does not overlap with the ACE2 binding site within 2019-nCoV RBD. Therefore, CR3022 has the potential to be developed as candidate therapeutics, alone or in combination with other neutralizing antibodies, for the prevention and treatment of 2019-nCoV infections. Interestingly, some of the most potent SARS-CoV-specific neutralizing antibodies (e.g., m396, CR3014) that target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike protein, indicating that the difference in the RBD of SARS-CoV and 2019-nCoV has a critical impact for the cross-reactivity of neutralizing antibodies, and that it is still necessary to develop novel monoclonal antibodies that could bind specifically to 2019-nCoV RBD.","2020-01-28","Xiaolong Tian; Cheng Li; Ailing Huang; Shuai Xia; Sicong Lu; Zhengli Shi; Lu Lu; Shibo Jiang; Zhenlin Yang; Yanling Wu; Tianlei Ying",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.28.923011"],["x443k65a","f24d3b4b4af138be06b7452b7acefc8948bc1056","biorxiv","Beware of asymptomatic transmission: Study on 2019-nCoV prevention and control measures based on extended SEIR model","10.1101/2020.01.28.923169",null,null,"biorxiv","SummaryBackgroundThe 2019 new coronavirus, “2019-nCoV”, was discovered from Wuhan Viral Pneumonia cases in December 2019, and was named by the World Health Organization on January 12, 2020. In the early stage, people knows little about the 2019-nCoV virus was not clear, and the spread period was encountering China’s annual spring migration, which made the epidemic spread rapidly from Wuhan to almost all provinces in China.MethodsThis study builds a SEIRD model that considers the movement of people across regions, revealing the effects of three measures on controlling the spread of the epidemic.Based on MATLAB R2017a, computational experiments were performed to simulate the epidemic prevention and control measures.FindingsThe research results show that current prevention and control measures in China are very necessary. This study further validates the concerns of international and domestic experts regarding asymptomatic transmission (E-status).InterpretationThe results of this study are applicable to explore the impact of the implementation of relevant measures on the prevention and control of epidemic spread, and to identify key individuals that may exist during the spread of the epidemic.","2020-01-28","Peng Shao; Yingji Shan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.28.923169"],["rr5qhsam","2afff5fe89a3e3ec270a41c2b929dfeec68d7d50","medrxiv","Real time estimation of the risk of death from novel coronavirus (2019-nCoV) infection: Inference using exported cases","10.1101/2020.01.29.20019547",null,null,"medrvix","The exported cases of 2019 novel coronavirus (2019-nCoV) infection who were confirmed in other countries provide a chance to estimate the cumulative incidence and confirmed case fatality risk (cCFR) in China. Knowledge of the cCFR is critical to characterize the severity and understand pandemic potential of 2019-nCoV in the early stage of epidemic. Using the exponential growth rate of the incidence, the present study statistically estimated the cCFR and the basic reproduction number, i.e., the average number of secondary cases generated by a single primary case in a naive population. As of 24 January 2020, with 23 exported cases, and estimating the growth rate from 8 December 2019 (scenario 1) and using the data since growth of exported cases (scenario 2), the cumulative incidence in China was estimated at 5433 cases (95% confidence interval (CI): 3883, 7160) and 17780 cases (95% CI: 9646, 28724), respectively. The latest estimates of the cCFR were 4.6% (95% CI: 3.1-6.6) for scenario 1 and 7.7% (95% CI: 4.9-11.3%) for scenario 2, respectively. The basic reproduction number was estimated to be 2.2 (95% CI: 2.1, 2.3) and 3.7 (95% CI: 3.1, 4.3) for scenarios 1 and 2, respectively. Based on the results, we note that current 2019-nCoV epidemic has a substation potential to cause a pandemic. The proposed approach can provide insights into early risk assessment using only publicly available data.","2020-02-02","Sung-mok Jung; Andrei R. Akhmetzhanov; Katsuma Hayashi; Natalie M. Linton; Yichi Yang; Baoyin Yuan; Tetsuro Kobayashi; Ryo Kinoshita; Hiroshi Nishiura",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.29.20019547"],["yd7tu7u4","c04c7fb330a409a00f67040dde0f83b3da88eacb","biorxiv","Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines","10.1101/2020.01.29.924100",null,null,"biorxiv","AbstractStarting from December 2019, a novel coronavirus, named 2019-nCoV, was found to cause Severe Acute Respiratory (SARI) symptoms and rapid pandemic in China. With the hope to identify candidate drugs for 2019-nCoV, we adopted a computational approach to screen for available commercial medicines which may function as inhibitors for the Mpro of 2019-nCoV. Up to 10 commercial medicines that may form hydrogen bounds to key residues within the binding pocket of 2019-nCoV Mpro were identified, which may have higher mutation tolerance than lopinavir/ritonavir and may also function as inhibitors for other coronaviruses with similar Mpro binding sites and pocket structures.","2020-01-29","Xin Liu; Xiu-Jie Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.29.924100"],["ac00tai9","b9dbb79c9e06164c3b7fefce67c11c5d6caf1fcd","biorxiv","Interpretable detection of novel human viruses from genome sequencing data","10.1101/2020.01.29.925354",null,null,"biorxiv","Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020.","2020-01-30","Jakub M Bartoszewicz; Anja Seidel; Bernhard Y Renard",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.29.925354"],["xxcx7hg5","acb0e11e5763dbf7588ec435751a2ab705a783f1","biorxiv","Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development","10.1101/2020.01.29.925867",null,null,"biorxiv","ABSTRACTNovel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARS-like bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.","2020-01-30","Arunachalam Ramaiah; Vaithilingaraja Arumugaswami",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.29.925867"],["90tccg03","a6af28a7f0a0d8d1e4d5b06f5ae38412298bbcb4","medrxiv","Direct Measurement of Rates of Asymptomatic Infection and Clinical Care-Seeking for Seasonal Coronavirus","10.1101/2020.01.30.20019612",null,null,"medrvix","The pandemic potential of the novel coronavirus (nCoV) that emerged in Wuhan, China, during December 2019 is strongly tied to the number and contagiousness of undocumented human infections. Here we present findings from a proactive longitudinal sampling study of acute viral respiratory infections that documents rates of asymptomatic infection and clinical care seeking for seasonal coronavirus. We find that the majority of infections are asymptomatic by most symptom definitions and that only 4% of individuals experiencing a seasonal coronavirus infection episode sought medical care for their symptoms. These numbers indicate that a very high percentage of seasonal coronavirus infections are undocumented and provide a reference for understanding the spread of the emergent nCoV.","2020-02-03","Jeffrey Shaman; Marta Galanti",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.20019612"],["81j4347d",null,"medrxiv","Modelling the epidemic trend of the 2019-nCOV outbreak in Hubei Province, China","10.1101/2020.01.30.20019828",null,null,"medrvix","As of 8am 30th January (Beijing Time) 2020, Approximate 8000 cases across the world have been confirmed. It's necessary to simulate epidemic trend of the 2019-nCOV outbreak in Hubei Province, the hardest-hit area. By SEIR simulation, the predicted epidemic peak in Hubei will be within 28th January 2020 to 7th February 2020, up to 7000-9000 infectious cases in total. The estimate above was based on some assumptions and limitations exited.","2020-02-02","Lizhe Ai",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.20019828"],["mm4ngrla","f17c6ac4986aecba2dc22f67b32f2d16b369117b","medrxiv","Transmission and epidemiological characteristics of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infected Pneumonia (COVID-19): preliminary evidence obtained in comparison with 2003-SARS","10.1101/2020.01.30.20019836",null,null,"medrvix","Objectives: Latest epidemic data of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infected Pneumonia (COVID-19) was collected and a detailed statistical analysis was carried out to make comparison with 2003-SARS in order to provide scientific reference for the prevention and control of COVID-19. Methods: The information of COVID-19 and 2003-SARS from websites of NHCPRC and the World Health Organization was collected, and then the transmission dynamics of the two kinds of infectious diseases were analyzed. The information of 853 confirmed COVID-19 patients obtained from the website of health committees of 18 provinces. A descriptive epidemiological analysis method was employed to carefully analyze the epidemic characteristics. Subsequently, the COVID-19 epidemic data in Wuhan and other inland regions of China was analyzed separately and compared. A multivariate function model was constructed based on the confirmed COVID-19 case data. Results: The growth rate of new cases and deaths of COVID-19 were significantly faster than those of 2003-SARS. The number of confirmed cases in Wuhan and other inland areas both showed increasing trends. 853 confirmed COVID-19 cases aged 1 months to 94 years and the average age was (45.05 ± 17.22) years. The gender ratio (M: F) was 1.12: 1. Conclusions: The fatality rate of COVID-19 is lower than that of 2003-SARS and the cure rate is higher. The age of COVID-19 patients is mainly concentrated in the 30-50 years old (60.61%). The harm of the first-generation COVID-19 patients is higher than that of secondary cases.","2020-02-02","Rongqiang Zhang; Hui Liu; Fengying Li; Bei Zhang; Qiling Liu; Xiangwen Li; Limei Luo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.20019836"],["e85xdrcw",null,"medrxiv","The impact of transmission control measures during the first 50 days of the COVID-19 epidemic in China","10.1101/2020.01.30.20019844",null,null,"medrvix","Respiratory illness caused by a novel coronavirus (COVID-19) appeared in China during December 2019. Attempting to contain infection, China banned travel to and from Wuhan city on 23 January and implemented a national emergency response. Here we evaluate the spread and control of the epidemic based on a unique synthesis of data including case reports, human movement and public health interventions. The Wuhan shutdown slowed the dispersal of infection to other cities by an estimated 2.91 days (95%CI: 2.54-3.29), delaying epidemic growth elsewhere in China. Other cities that implemented control measures pre-emptively reported 33.3% (11.1-44.4%) fewer cases in the first week of their outbreaks (13.0; 7.1-18.8) compared with cities that started control later (20.6; 14.5-26.8). Among interventions investigated here, the most effective were suspending intra-city public transport, closing entertainment venues and banning public gatherings. The national emergency response delayed the growth and limited the size of the COVID-19 epidemic and, by 19 February (day 50), had averted hundreds of thousands of cases across China.","2020-02-02","Huaiyu Tian; Yonghong Liu; Yidan Li; Chieh-Hsi Wu; Bin Chen; Moritz U. G. Kraemer; Bingying Li; Jun Cai; Bo Xu; Qiqi Yang; Ben Wang; Peng Yang; Yujun Cui; Yimeng Song; Pai Zheng; Quanyi Wang; Ottar N Bjornstad; Ruifu Yang; Bryan Grenfell; Oliver Pybus; Christopher Dye",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.20019844"],["p5aj5k2g","0ff4ad5359ee2df3568605385a8616d8da66bf2c","medrxiv","Reconciling early-outbreak estimates of the basic reproductive number and its uncertainty: framework and applications to the novel coronavirus (SARS-CoV-2) outbreak","10.1101/2020.01.30.20019877",null,null,"medrvix","A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number Ro -- the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of Ro vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of Ro across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate $r$, the mean generation interval $\\bar G$, and the generation-interval dispersion $\\kappa$. We then apply our framework to early estimates of Ro for the SARS-CoV-2 outbreak. We show that many early Ro estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of Ro, including the shape of the generation-interval distribution, in efforts to estimate Ro at the outset of an epidemic.","2020-02-02","Sang Woo Park; Benjamin M. Bolker; David Champredon; David J.D. Earn; Michael Li; Joshua S. Weitz; Bryan T. Grenfell; Jonathan Dushoff",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.20019877"],["ua6aeg5a","18dc6a67045d2687d2b5c11c85797f42824ed243","biorxiv","Evolution and variation of 2019-novel coronavirus","10.1101/2020.01.30.926477",null,null,"biorxiv","AbstractBackgroundThe current outbreak caused by novel coronavirus (2019-nCoV) in China has become a worldwide concern. As of 28 January 2020, there were 4631 confirmed cases and 106 deaths, and 11 countries or regions were affected.MethodsWe downloaded the genomes of 2019-nCoVs and similar isolates from the Global Initiative on Sharing Avian Influenza Database (GISAID and nucleotide database of the National Center for Biotechnology Information (NCBI). Lasergene 7.0 and MEGA 6.0 softwares were used to calculate genetic distances of the sequences, to construct phylogenetic trees, and to align amino acid sequences. Bayesian coalescent phylogenetic analysis, implemented in the BEAST software package, was used to calculate the molecular clock related characteristics such as the nucleotide substitution rate and the most recent common ancestor (tMRCA) of 2019-nCoVs.ResultsAn isolate numbered EPI_ISL_403928 showed different phylogenetic trees and genetic distances of the whole length genome, the coding sequences (CDS) of ployprotein (P), spike protein (S), and nucleoprotein (N) from other 2019-nCoVs. There are 22, 4, 2 variations in P, S, and N at the level of amino acid residues. The nucleotide substitution rates from high to low are 1·05 × 10−2 (nucleotide substitutions/site/year, with 95% HPD interval being 6.27 × 10−4 to 2.72 × 10−2) for N, 5.34 × 10−3 (5.10 × 10−4, 1.28 × 10−2) for S, 1.69 × 10−3 (3.94 × 10−4, 3.60 × 10−3) for P, 1.65 × 10−3 (4.47 × 10−4, 3.24 × 10−3) for the whole genome, respectively. At this nucleotide substitution rate, the most recent common ancestor (tMRCA) of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.ConclusionOur analysis suggests that at least two different viral strains of 2019-nCoV are involved in this outbreak that might occur a few months earlier before it was officially reported.","2020-01-30","Chenglong Xiong; Lufang Jiang; Yue Chen; Qingwu Jiang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.926477"],["nfpw89fu",null,"biorxiv","Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes","10.1101/2020.01.30.926881",null,null,"biorxiv","AbstractDeubiquitinating enzymes (DUBs) are responsible for removing ubiquitin (Ub) from its protein conjugates. DUBs have been implicated as attractive therapeutic targets in the treatment of viral diseases, neurodegenerative disorders and cancer. The lack of selective chemical tools for the exploration of these enzymes significantly impairs the determination of their roles in both normal and pathological states. Commercially available fluorogenic substrates are based on the C-terminal Ub motif or contain Ub coupled to a fluorophore (Z-LRGG-AMC, Ub-AMC); therefore, these substrates suffer from lack of selectivity. By using a hybrid combinatorial substrate library (HyCoSuL) and a defined P2 library containing a wide variety of nonproteinogenic amino acids, we established a full substrate specificity profile for two DUBs—MERS PLpro and human UCH-L3. Based on these results, we designed and synthesized Ub-based substrates and activity-based probes (ABPs) containing selected unnatural amino acids located in the C-terminal Ub motif. Biochemical analysis and cell-based experiments confirmed the activity and selectivity of engineered Ub-based substrates and probes. Using this approach, we propose that for any protease that recognizes Ub and Ub-like substrates, a highly active and selective unnatural substrate or probe can be engineered.","2020-01-31","Wioletta Rut; Mikołaj Żmudziński; Scott J. Snipas; Miklos Bekes; Tony T. Huang; Marcin Drag",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.926881"],["dpcpg5c1","ffc9bc4ac6548cb1c03c2c6f088ab1c1c2067c6f","biorxiv","Nucleotide Analogues as Inhibitors of Viral Polymerases","10.1101/2020.01.30.927574",null,null,"biorxiv","SummaryCoronaviruses such as the newly discovered virus from Wuhan, China, 2019-nCoV, and the viruses that cause SARS and MERS, have resulted in regional and global public health emergencies. Based on our molecular insight that the hepatitis C virus and the coronavirus use a similar viral genome replication mechanism, we reasoned that the FDA-approved drug EPCLUSA (Sofosbuvir/Velpatasvir) for the treatment of hepatitis C will also inhibit the above coronaviruses, including 2019-nCoV. To develop broad spectrum anti-viral agents, we further describe a novel strategy to design and synthesize viral polymerase inhibitors, by combining the ProTide Prodrug approach used in the development of Sofosbuvir with the use of 3’-blocking groups that we have previously built into nucleotide analogues that function as polymerase terminators.","2020-01-31","Jingyue Ju; Shiv Kumar; Xiaoxu Li; Steffen Jockusch; James J. Russo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.927574"],["vpcx2t3w",null,"biorxiv","The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes","10.1101/2020.01.30.927806",null,null,"biorxiv","AbstractSince December 2019, a newly identified coronavirus (2019 novel coronavirus, 2019-nCov) is causing outbreak of pneumonia in one of largest cities, Wuhan, in Hubei province of China and has draw significant public health attention. The same as severe acute respiratory syndrome coronavirus (SARS-CoV), 2019-nCov enters into host cells via cell receptor angiotensin converting enzyme II (ACE2). In order to dissect the ACE2-expressing cell composition and proportion and explore a potential route of the 2019-nCov infection in digestive system infection, 4 datasets with single-cell transcriptomes of lung, esophagus, gastric, ileum and colon were analyzed. The data showed that ACE2 was not only highly expressed in the lung AT2 cells, esophagus upper and stratified epithelial cells but also in absorptive enterocytes from ileum and colon. These results indicated along with respiratory systems, digestive system is a potential routes for 2019-nCov infection. In conclusion, this study has provided the bioinformatics evidence of the potential route for infection of 2019-nCov in digestive system along with respiratory tract and may have significant impact for our healthy policy setting regards to prevention of 2019-nCoV infection.","2020-01-31","Hao Zhang; Zijian Kang; Haiyi Gong; Da Xu; Jing Wang; Zifu Li; Xingang Cui; Jianru Xiao; Tong Meng; Wang Zhou; Jianmin Liu; Huji Xu",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.927806"],["1qniriu0","6ae20454d1a9f228864de24660c2460becbc8151","biorxiv","Machine intelligence design of 2019-nCoV drugs","10.1101/2020.01.30.927889",null,null,"biorxiv","AbstractWuhan coronavirus, called 2019-nCoV, is a newly emerged virus that infected more than 9692 people and leads to more than 213 fatalities by January 30, 2020. Currently, there is no effective treatment for this epidemic. However, the viral protease of a coronavirus is well-known to be essential for its replication and thus is an effective drug target. Fortunately, the sequence identity of the 2019-nCoV protease and that of severe-acute respiratory syndrome virus (SARS-CoV) is as high as 96.1%. We show that the protease inhibitor binding sites of 2019-nCoV and SARS-CoV are almost identical, which means all potential anti-SARS-CoV chemotherapies are also potential 2019-nCoV drugs. Here, we report a family of potential 2019-nCoV drugs generated by a machine intelligence-based generative network complex (GNC). The potential effectiveness of treating 2019-nCoV by using some existing HIV drugs is also analyzed.","2020-02-04","Kaifu Gao; Duc Duy Nguyen; Rui Wang; Guo-Wei Wei",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.30.927889"],["2wshgzjk","a01672035e0b9d21f9a934b9c9071610513325f2","medrxiv","Effectiveness of airport screening at detecting travellers infected with 2019-nCoV","10.1101/2020.01.31.20019265",null,null,"medrvix","As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available.","2020-02-02","Billy Quilty; Sam Clifford; Stefan Flasche; Rosalind M Eggo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.31.20019265"],["gmi1ewc2","a44334e676e43c1889d282b0f0a8365f0f1e0c52","medrxiv","Early dynamics of transmission and control of COVID-19: a mathematical modelling study","10.1101/2020.01.31.20019901",null,null,"medrvix","Background: An outbreak of the novel coronavirus SARS-CoV-2 has led to 46,997 confirmed cases as of 13th February 2020. Understanding the early transmission dynamics of the infection and evaluating the effectiveness of control measures is crucial for assessing the potential for sustained transmission to occur in new areas. Methods: We combined a stochastic transmission model with data on cases of novel coronavirus disease (COVID-19) in Wuhan and international cases that originated in Wuhan to estimate how transmission had varied over time during January and February 2020. Based on these estimates, we then calculated the probability that newly introduced cases might generate outbreaks in other areas. Findings: We estimated that the median daily reproduction number, Rt , declined from 2.35 (95% CI: 1.15-4.77) one week before travel restrictions were introduced on 23rd January to 1.05 (95% CI: 0.413-2.39) one week after. Based on our estimates of Rt,we calculated that in locations with similar transmission potential as Wuhan in early January, once there are at least four independently introduced cases, there is a more than 50% chance the infection will establish within that population. Interpretation: Our results show that COVID-19 transmission likely declined in Wuhan during late January 2020, coinciding with the introduction of control measures. As more cases arrive in international locations with similar transmission potential to Wuhan pre-control, it is likely many chains of transmission will fail to establish initially, but may still cause new outbreaks eventually.","2020-02-02","Adam J Kucharski; Timothy W Russell; Charlie Diamond; Yang Liu; John Edmunds; Sebastian Funk; Rosalind M Eggo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.31.20019901"],["gxstlzuk","0c27c0ddaa4761f6155838df81a88d24619720f8","biorxiv","Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes","10.1101/2020.01.31.928796",null,null,"biorxiv","ABSTRACTSummaryGenome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.AvailabilityAvailable online: https://www.genomedetective.com/app/typingtool/cov*Contactkoen@emweb.be and deoliveira@ukzn.ac.zaSupplementary informationSupplementary data is available online.","2020-02-02","Sara Cleemput; Wim Dumon; Vagner Fonseca; Wasim Abdool Karim; Marta Giovanetti; Luiz Carlos Alcantara; Koen Deforche; Tulio de Oliveira",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.31.928796"],["rv26qfmf","eae11a3f01b79e76a968bfb84886432f04584596","biorxiv","The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells","10.1101/2020.01.31.929042",null,null,"biorxiv","AbstractThe emergence of a novel, highly pathogenic coronavirus, 2019-nCoV, in China, and its rapid national and international spread pose a global health emergency. Coronaviruses use their spike proteins to select and enter target cells and insights into nCoV-2019 spike (S)-driven entry might facilitate assessment of pandemic potential and reveal therapeutic targets. Here, we demonstrate that 2019-nCoV-S uses the SARS-coronavirus receptor, ACE2, for entry and the cellular protease TMPRSS2 for 2019-nCoV-S priming. A TMPRSS2 inhibitor blocked entry and might constitute a treatment option. Finally, we show that the serum form a convalescent SARS patient neutralized 2019-nCoV-S-driven entry. Our results reveal important commonalities between 2019-nCoV and SARS-coronavirus infection, which might translate into similar transmissibility and disease pathogenesis. Moreover, they identify a target for antiviral intervention.One sentence summaryThe novel 2019 coronavirus and the SARS-coronavirus share central biological properties which can guide risk assessment and intervention.","2020-01-31","Markus Hoffmann; Hannah Kleine-Weber; Nadine Krüger; Marcel Müller; Christian Drosten; Stefan Pöhlmann",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.31.929042"],["a1mcnta5",null,"biorxiv","Highly Distinguished Amino Acid Sequences of 2019-nCoV (Wuhan Coronavirus)","10.1101/2020.01.31.929497",null,null,"biorxiv","AbstractUsing a method for pathogen screening in DNA synthesis orders, we have identified a number of amino acid sequences that distinguish 2019-nCoV (Wuhan Coronavirus) from all other known viruses in Coronaviridae. We find three main regions of unique sequence: two in the 1ab polyprotein QHO60603.1, one in surface glycoprotein QHO60594.1.","2020-02-02","Jacob Beal; Thomas Mitchell; Daniel Wyschogrod; Jeff Manthey; Adam Clore",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.31.929497"],["cszqykpu","ff54e3e961a72eb1d2500166809b3651b2f98cf6","biorxiv","Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model","10.1101/2020.01.31.929547",null,null,"biorxiv","AbstractThe infection of a novel coronavirus found in Wuhan of China (2019-nCoV) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for 2019-nCoV, various strategies are being tested in China, including drug repurposing. In this study, we used our pretrained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd < 1000 nM. In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for 2019-nCoV.","2020-02-02","Bo Ram Beck; Bonggun Shin; Yoonjung Choi; Sungsoo Park; Keunsoo Kang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.31.929547"],["vbgf50os","9e94f9379fd74fcacc4f3a57e03cbe9035efee8e","biorxiv","Molecular Modeling Evaluation of the Binding Effect of Ritonavir, Lopinavir and Darunavir to Severe Acute Respiratory Syndrome Coronavirus 2 Proteases","10.1101/2020.01.31.929695",null,null,"biorxiv","AbstractThree anti-HIV drugs, ritonavir, lopinavir and darunavir, might have therapeutic effect on coronavirus disease 2019 (COVID-19). In this study, the structure models of two severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) proteases, coronavirus endopeptidase C30 (CEP_C30) and papain like viral protease (PLVP), were built by homology modeling. Ritonavir, lopinavir and darunavir were then docked to the models, respectively, followed by energy minimization of the protease-drug complexes. In the simulations, ritonavir can bind to CEP_C30 most suitably, and induce significant conformation changes of CEP_C30; lopinavir can also bind to CEP_C30 suitably, and induce significant conformation changes of CEP_C30; darunavir can bind to PLVP suitably with slight conformation changes of PLVP. It is suggested that the therapeutic effect of ritonavir and lopinavir on COVID-19 may be mainly due to their inhibitory effect on CEP_C30, while ritonavir may have stronger efficacy; the inhibitory effect of darunavir on SARS-CoV-2 and its potential therapeutic effect may be mainly due to its inhibitory effect on PLVP.","2020-02-03","Shen Lin; Runnan Shen; Jingdong He; Xinhao Li; Xushun Guo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.01.31.929695"],["aeuy92bx","a8e53c77bc72f467723a7d5b8020875c0e96f8aa","biorxiv","Potent neutralization of 2019 novel coronavirus by recombinant ACE2-Ig","10.1101/2020.02.01.929976",null,null,"biorxiv","Abstract2019-nCoV, which is a novel coronavirus emerged in Wuhan, China, at the end of 2019, has caused at least infected 11,844 as of Feb 1, 2020. However, there is no specific antiviral treatment or vaccine currently. Very recently report had suggested that novel CoV would use the same cell entry receptor, ACE2, as the SARS-CoV. In this report, we generated a novel recombinant protein by connecting the extracellular domain of human ACE2 to the Fc region of the human immunoglobulin IgG1. An ACE2 mutant with low catalytic activity was also used in the study. The fusion proteins were then characterized. Both fusion proteins has high affinity binding to the receptor-binding domain (RBD) of SARS-CoV and 2019-nCoV and exerted desired pharmacological properties. Moreover, fusion proteins potently neutralized SARS-CoV and 2019-nCoV in vitro. As these fusion proteins exhibit cross-reactivity against coronaviruses, they could have potential applications for diagnosis, prophylaxis, and treatment of 2019-nCoV.","2020-02-02","Changhai Lei; Wenyan Fu; Kewen Qian; Tian Li; Sheng Zhang; Min Ding; Shi Hu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.01.929976"],["eecyduzq","836190d366ba8e34c3a5df8a24d17a36fd503505","biorxiv","Fast assessment of human receptor-binding capability of 2019 novel coronavirus (2019-nCoV)","10.1101/2020.02.01.930537",null,null,"biorxiv","AbstractThe outbreaks of 2002/2003 SARS, 2012/2015 MERS and 2019/2020 Wuhan respiratory syndrome clearly indicate that genome evolution of an animal coronavirus (CoV) may enable it to acquire human transmission ability, and thereby to cause serious threats to global public health. It is widely accepted that CoV human transmission is driven by the interactions of its spike protein (S-protein) with human receptor on host cell surface; so, quantitative evaluation of these interactions may be used to assess the human transmission capability of CoVs. However, quantitative methods directly using viral genome data are still lacking. Here, we perform large-scale protein-protein docking to quantify the interactions of 2019-nCoV S-protein receptor-binding domain (S-RBD) with human receptor ACE2, based on experimental SARS-CoV S-RBD-ACE2 complex structure. By sampling a large number of thermodynamically probable binding conformations with Monte Carlo algorithm, this approach successfully identified the experimental complex structure as the lowest-energy receptor-binding conformations, and hence established an experiment-based strength reference for evaluating the receptor-binding affinity of 2019-nCoV via comparison with SARS-CoV. Our results show that this binding affinity is about 73% of that of SARS-CoV, supporting that 2019-nCoV may cause human transmission similar to that of SARS-CoV. Thus, this study presents a method for rapidly assessing the human transmission capability of a newly emerged CoV and its mutant strains, and demonstrates that post-genome analysis of protein-protein interactions may provide early scientific guidance for viral prevention and control.","2020-02-03","Qiang Huang; Andreas Herrmann",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.01.930537"],["uqls3p01",null,"biorxiv","Genomic variance of the 2019-nCoV coronavirus","10.1101/2020.02.02.931162",null,null,"biorxiv","AbstractThere is rising global concern for the recently emerged novel Coronavirus (2019-nCov). Full genomic sequences have been released by the worldwide scientific community in the last few weeks in order to understand the evolutionary origin and molecular characteristics of this virus. Taking advantage of all the genomic information currently available, we constructed a phylogenetic tree including also representatives of other coronaviridae, such as Bat coronavirus (BCoV) and SARS. We confirm high sequence similarity (>99%) between all sequenced 2019-nCoVs genomes available, with the closest BCoV sequence sharing 96.2% sequence identity, confirming the notion of a zoonotic origin of 2019-nCoV. Despite the low heterogeneity of the 2019-nCoV genomes, we could identify at least two hyper-variable genomic hotspots, one of which is responsible for a Serine/Leucine variation in the viral ORF8-encoded protein. Finally, we perform a full proteomic comparison with other coronaviridae, identifying key aminoacidic differences to be considered for antiviral strategies deriving from previous anti-coronavirus approaches.","2020-02-04","Carmine Ceraolo; Federico M. Giorgi",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.02.931162"],["rj9k9awy",null,"medrxiv","Evidence and characteristics of human-to-human transmission of SARS-CoV-2","10.1101/2020.02.03.20019141",null,null,"medrvix","Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.","2020-02-05","Min Kang; Jie Wu; Wenjun Ma; Jianfeng He; Jing Lu; Tao Liu; Baisheng Li; Shujiang Mei; Feng Ruan; Lifeng Lin; Lirong Zou; Changwen Ke; Haojie Zhong; Yingtao Zhang; Xuguang Chen; Zhe Liu; Qi Zhu; Jianpeng Xiao; Jianxiang Yu; Jianxiong Hu; Weilin Zeng; Xing Li; Yuhuang Liao; Xiujuan Tang; Songjian Xiao; Ying Wang; Yingchao Song; Xue Zhuang; Lijun Liang; Siqing Zeng; Guanhao He; Peng Lin; Huihong Deng; Tie Song",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.20019141"],["vspnuxz9","a1bff76ce360e8990b0a4ee2a5228a6e6e63d9c1","medrxiv","Serial interval of novel coronavirus (2019-nCoV) infections","10.1101/2020.02.03.20019497",null,null,"medrvix","Objective: To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods: We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n=28) and a subset of pairs with highest certainty in reporting (n=18). In addition, we adjusting for right truncation of the data as the epidemic is still in its growth phase. Results: Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions: The serial interval of COVID-19 is shorter than its median incubation period. This suggests that a substantial proportion of secondary transmission may occur prior to illness onset. The COVID-19 serial interval is also shorter than the serial interval of severe acute respiratory syndrome (SARS), indicating that calculations made using the SARS serial interval may introduce bias.","2020-02-13","Hiroshi Nishiura; Natalie M Linton; Andrei R. Akhmetzhanov",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.20019497"],["mjhez5im","5d3612cd95331a2f0c43ef9d03ce583b5ff8c995","medrxiv","Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV","10.1101/2020.02.03.20020206",null,null,"medrvix","The 2019-nCoV is reported to share the same entry (ACE2) as SARS-CoV according to the updated findings. Analyzing the distribution and expression level of the route of coronavirus may help reveal underlying mechanisms of viral susceptibility and post-infection modulation. In this study, we found that the expression of ACE2 in healthy populations and patients with underlying diseases was not significantly different, suggesting relatively similar susceptibility, which was consistent with current clinical observations. Moreover, based on the expression of ACE2 in smoking individuals, we inferred that long-term smoking might be a risk factor for 2019-nCoV. Analyzing the ACE2 in SARS-CoV infected cells suggested that ACE2 was more than just a receptor but also participated in post-infection regulation, including immune response, cytokine secretion, and viral genome replication. We also constructed Protein-protein interaction (PPI) networks and identified hub genes in viral activity and cytokine secretion. Our findings could explain the clinical symptoms so far and help clinicians and researchers understand the pathogenesis and design therapeutic strategies for 2019-nCoV.","2020-02-05","Xiang He; Lei Zhang; Qin Ran; Anying Xiong; Junyi Wang; Dehong Wu; Feng Chen; Guoping Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.20020206"],["f3h74j1n",null,"medrxiv","Estimation of the asymptomatic ratio of novel coronavirus infections (COVID-19)","10.1101/2020.02.03.20020248",null,null,"medrvix","A total of 565 Japanese citizens were evacuated from Wuhan, China to Japan. All passengers were screened for symptoms and also undertook reverse transcription polymerase chain reaction testing, identifying 5 asymptomatic and 7 symptomatic passengers testing positive for 2019-nCoV. We show that the screening result is suggestive of the asymptomatic ratio at 41.6%.","2020-02-11","Hiroshi Nishiura; Tetsuro Kobayashi; Takeshi Miyama; Ayako Suzuki; Sungmok Jung; Katsuma Hayashi; Ryo Kinoshita; Yichi Yang; Baoyin Yuan; Andrei R. Akhmetzhanov; Natalie M Linton",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.20020248"],["b4mdiont","21100aba41a4bfb48d7dc37f1bf5dbb38bf3867a","medrxiv","Network-based Drug Repurposing for Human Coronavirus","10.1101/2020.02.03.20020263",null,null,"medrvix","Human Coronaviruses (HCoVs), including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle east respiratory syndrome coronavirus (MERS-CoV), and 2019 novel coronavirus (2019-nCoV), lead global epidemics with high morbidity and mortality. However, there are currently no effective drugs targeting 2019-nCoV. Drug repurposing, represented as an effective drug discovery strategy from existing drugs, could shorten the time and reduce the cost compared to de novo drug discovery. In this study, we present an integrative, antiviral drug repurposing methodology implementing a systems pharmacology-based network medicine platform, quantifying the interplay between the HCoV-host interactome and drug targets in the human protein-protein interaction network. Phylogenetic analyses of 15 HCoV whole genomes reveal that 2019-nCoV has the highest nucleotide sequence identity with SARS-CoV (79.7%) among the six other known pathogenic HCoVs. Specifically, the envelope and nucleocapsid proteins of 2019-nCoV are two evolutionarily conserved regions, having the sequence identities of 96% and 89.6%, respectively, compared to SARS-CoV. Using network proximity analyses of drug targets and known HCoV-host interactions in the human protein-protein interactome, we computationally identified 135 putative repurposable drugs for the potential prevention and treatment of HCoVs. In addition, we prioritized 16 potential anti-HCoV repurposable drugs (including melatonin, mercaptopurine, and sirolimus) that were further validated by enrichment analyses of drug-gene signatures and HCoV-induced transcriptomics data in human cell lines. Finally, we showcased three potential drug combinations (including sirolimus plus dactinomycin, mercaptopurine plus melatonin, and toremifene plus emodin) captured by the Complementary Exposure pattern: the targets of the drugs both hit the HCoV-host subnetwork, but target separate neighborhoods in the human protein-protein interactome network. In summary, this study offers powerful network-based methodologies for rapid identification of candidate repurposable drugs and potential drug combinations toward future clinical trials for HCoVs.","2020-02-05","Yadi Zhou; Yuan Hou; Jiayu Shen; Yin Huang; William Martin; Feixiong Cheng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.20020263"],["4sw1hrnf","5c525cc04075b7c82e52b1e80d96bb394b1a0907","medrxiv","Getting to zero quickly in the 2019-nCov epidemic with vaccines or rapid testing","10.1101/2020.02.03.20020271",null,null,"medrvix","Any plan for stopping the ongoing 2019-nCov epidemic must be based on a quantitative understanding of the proportion of the at-risk population that needs to be protected by effective control measures in order for transmission to decline sufficiently and quickly enough for the epidemic to end. Using an SEIR-type transmission model, we contrasted two alternate strategies by modeling the proportion of the population that needs to be protected from infection by one-time vaccination (assuming 100% effectiveness) or by testing with isolation and treatment of individuals within six, 24, or 48 hours of symptom onset. If R is currently 2.2, vaccination at the herd immunity coverage of 55% would drive R just below 1, but transmission could persist for years. Over 80% of coverage is required to end the epidemic in 6 months with population-wide vaccination. The epidemic could be ended in just under a year if testing with isolation and treatment reached 80% of symptomatically infected patients within 24 hours of symptom onset (assuming 10% asymptomatic transmission). The epidemic could be ended in six months if testing with isolation and treatment reached 90% of symptomatic patients. If 90% of symptomatic patients could be tested within six hours of symptoms appearing, the epidemic could be ended in under four months.","2020-02-05","Gerardo Chowell; Ranu Dhillon; Devabhaktuni Srikrishna",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.20020271"],["bnv38la3","b51d1c7287ac75790a52752489577d33d522898b","medrxiv","Diarrhea may be underestimated: a missing link in 2019 novel coronavirus","10.1101/2020.02.03.20020289",null,null,"medrvix","The outbreak of pneumonia caused by the 2019 Novel Coronavirus (2019-nCoV) was reported in Wuhan City, China. However, the clinical symptoms varied in different reports. Based on the results of inter-group difference test, we found that the incidence of diarrhea differed in three recent reports. As 2019-nCoV utilizes the same cell entry receptor ACE2 as severe acute respiratory syndrome coronavirus (SARS-CoV) and ACE2 tightly controls intestinal inflammation, to trace the route of infection mediated by 2019-nCoV, we used the single-cell RNA sequencing data for analysis. We found that the ACE2 mRNA was highly expressed in the healthy human small intestine rather than the lung. Besides, single-cell RNA sequencing data showed that ACE2 was significantly elevated in the proximal and distal enterocytes, where the small intestinal epithelium is exposed to the foreign pathogen. Thus, we suspect that ACE2-expressing small intestinal epithelium cells might be vulnerable to 2019-nCoV infection when people eat infected wild animals and diarrhea may serve as an indicator for infection, suggesting that clinicians should pay more attention to patients with diarrhea during the outbreak of pneumonia.","2020-02-11","Weicheng Liang; Zhijie Feng; Shitao Rao; Cuicui Xiao; Zexiao Lin; Qi Zhang; Wei Qi",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.20020289"],["xo5u02d7","6c91b00faa16142426820016ad3a7847fc77c8e4","biorxiv","Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection","10.1101/2020.02.03.931766",null,null,"biorxiv","AbstractA newly identified coronavirus, 2019-nCoV, has been posing significant threats to public health since December 2019. ACE2, the host cell receptor for severe acute respiratory syndrome coronavirus (SARS), has recently been demonstrated in mediating 2019-nCoV infection. Interestingly, besides the respiratory system, substantial proportion of SARS and 2019-nCoV patients showed signs of various degrees of liver damage, the mechanism and implication of which have not yet been determined. Here, we performed an unbiased evaluation of cell type specific expression of ACE2 in healthy liver tissues using single cell RNA-seq data of two independent cohorts, and identified specific expression in cholangiocytes. The results indicated that virus might directly bind to ACE2 positive cholangiocytes but not necessarily hepatocytes. This finding suggested the liver abnormalities of SARS and 2019-nCoV patients may not be due to hepatocyte damage, but cholangiocyte dysfunction and other causes such as drug induced and systemic inflammatory response induced liver injury. Our findings indicate that special care of liver dysfunction should be installed in treating 2019-nCoV patients during the hospitalization and shortly after cure.","2020-02-04","Xiaoqiang Chai; Longfei Hu; Yan Zhang; Weiyu Han; Zhou Lu; Aiwu Ke; Jian Zhou; Guoming Shi; Nan Fang; Jia Fan; Jiabin Cai; Jue Fan; Fei Lan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.931766"],["cetdqgff","bb8e3d331bb2975e1c644c8729b842797da2d626","biorxiv","Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study","10.1101/2020.02.03.932350",null,null,"biorxiv","AbstractAs of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus, within Betacoronavirus. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.","2020-02-04","Gurjit S. Randhawa; Maximillian P.M. Soltysiak; Hadi El Roz; Camila P.E. de Souza; Kathleen A. Hill; Lila Kari",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.932350"],["7i52vltp","181b7b57851e6f58a601b68e613d10c10616f774","biorxiv","Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies","10.1101/2020.02.03.933226",null,null,"biorxiv","AbstractThe beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.","2020-02-04","Syed Faraz Ahmed; Ahmed A. Quadeer; Matthew R. McKay",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.03.933226"],["f5yjz0zp","1b531d266b4933bf8040739ed199914655b075d6","medrxiv","Population movement, city closure and spatial transmission of the 2019-nCoV infection in China","10.1101/2020.02.04.20020339",null,null,"medrvix","The outbreak of pneumonia caused by a novel coronavirus (2019-nCoV) in Wuhan City of China obtained global concern, the population outflow from Wuhan has contributed to spatial expansion in other parts of China. We examined the effects of population outflow from Wuhan on the 2019-nCoV transmission in other provinces and cities of China, as well as the impacts of the city closure in Wuhan. We observed a significantly positive association between population movement and the number of cases. Further analysis revealed that if the city closure policy was implemented two days earlier, 1420 (95% CI: 1059, 1833) cases could be prevented, and if two days later, 1462 (95% CI: 1090, 1886) more cases would be possible. Our findings suggest that population movement might be one important trigger of the 2019-nCoV infection transmission in China, and the policy of city closure is effective to prevent the epidemic.","2020-02-05","Siqi Ai; Guanghu Zhu; Fei Tian; Huan Li; Yuan Gao; Yinglin Wu; Qiyong Liu; Hualiang Lin",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.20020339"],["xv42jvd6","e3f19f88e7e3d1279b4d93e4db041e44a8d35dbb","medrxiv","Epidemiological parameter review and comparative dynamics of influenza, respiratory syncytial virus, rhinovirus, human coronavirus, and adenovirus","10.1101/2020.02.04.20020404",null,null,"medrvix","Influenza-like illness (ILI) accounts for a large burden of annual morbidity and mortality worldwide. A finer-grained knowledge of the parameters and dynamics of the viruses commonly underlying ILI is needed for modeling, diagnostic, and intervention efforts. We conducted an extensive literature review for epidemiological parameter values for influenza, respiratory syncytial virus (RSV), rhinovirus, human coronavirus (HCoV), and adenovirus. We also developed a deterministic SEIR model for ILI, and derived an expression for R0. We here report ranges and means for parameters for these five common viruses.","2020-02-05","Julie Spencer; Deborah P Shutt; Sarah K Moser; Hannah Clegg; Helen J Wearing; Harshini Mukundan; Carrie A Manore",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.20020404"],["kctgr529","c9533866dd77deaa65ee1015b57bca5486f93326","medrxiv","The impact of traffic isolation in Wuhan on the spread of 2019-nCov","10.1101/2020.02.04.20020438",null,null,"medrvix","The 2019-nCoV outbreak occurred near the Chinese Spring Festival transport period in Wuhan. As an important transportation center, the migration of Wuhan accelerated the spread of 2019-nCoV across mainland China. Based on the cumulative Baidu migration index (CBMI), we first analyzed the proportion of Wuhan's migrant population to other cities. Our results confirm that there is a significant correlation between the export population of Wuhan and reported cases in various regions. We subsequently found that the mortality rate in Hubei Province was much higher than that in other regions of mainland China, while the investigation of potential cases in Wuhan was far behind other provinces in Mainland China, which indicates the effectiveness of early isolation.","2020-02-05","Gehui Jin; Jiayu Yu; Liyuan Han; Shiwei Duan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.20020438"],["8kdtpwbv","164b0678afc42f1923aa100b624ca969a53fb3b2","medrxiv","Forecasting the Wuhan coronavirus (2019-nCoV) epidemics using a simple (simplistic) model - update (Feb. 8, 2020)","10.1101/2020.02.04.20020461",null,null,"medrvix","Confirmed infection cases in mainland China were analyzed using the data up to January 28, 2020 (first 13 days of reliable confirmed cases). For the first period the cumulative number of cases followed an exponential function. However, from January 28, we discerned a downward deviation from the exponential growth. This slower-than-exponential growth was also confirmed by a steady decline of the effective reproduction number. A backtrend analysis suggested the original basic reproduction number R0 to be about 2.4 to 2.5. As data become available, we subsequently analyzed them during three consecutive periods obtaining a sequence of model predictions. All available data up were processed the same way. We used a simple logistic growth model that fitted very well with all data. Using this model and the three sets of data, we estimated maximum cases as about 21,000, 28,000 and 35,000 cases refining these predictions in near-real time. With slightly different approach (linearization in time) the estimate of maximum cases was even higher (about 65,000). Although the estimates of maximum cases increase as more data were reported all models show reaching a peak in mid-February in contrast to the unconfined exponential growth. These predictions do not account for any possible other secondary sources of infection.","2020-02-05","Slav W Hermanowicz",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.20020461"],["iat80b4l","013d9d1cba8a54d5d3718c229b812d7cf91b6c89","medrxiv","Assessing spread risk of Wuhan novel coronavirus within and beyond China, January-April 2020: a travel network-based modelling study","10.1101/2020.02.04.20020479",null,null,"medrvix","Background: A novel coronavirus (2019-nCoV) emerged in Wuhan City, China, at the end of 2019 and has caused an outbreak of human-to-human transmission with a Public Health Emergency of International Concern declared by the World Health Organization on January 30, 2020. Aim: We aimed to estimate the potential risk and geographic range of Wuhan novel coronavirus (2019-nCoV) spread within and beyond China from January through to April, 2020. Methods: A series of domestic and international travel network-based connectivity and risk analyses were performed, by using de-identified and aggregated mobile phone data, air passenger itinerary data, and case reports. Results: The cordon sanitaire of Wuhan is likely to have occurred during the latter stages of peak population numbers leaving the city before Lunar New Year (LNY), with travellers departing into neighbouring cities and other megacities in China. We estimated that 59,912 air passengers, of which 834 (95% UI: 478 - 1349) had 2019-nCoV infection, travelled from Wuhan to 382 cities outside of mainland China during the two weeks prior to the lockdown of Wuhan. The majority of these cities were in Asia, but major hubs in Europe, the US and Australia were also prominent, with strong correlation seen between predicted importation risks and reported cases. Because significant spread has already occurred, a large number of airline travellers (3.3 million under the scenario of 75% travel reduction from normal volumes) may be required to be screened at origin high-risk cities in China and destinations across the globe for the following three months of February to April, 2020 to effectively limit spread beyond its current extent. Conclusion: Further spread of 2019-nCoV within China and international exportation is likely to occur. All countries, especially vulnerable regions, should be prepared for efforts to contain the 2019-nCoV infection.","2020-02-05","Shengjie Lai; Isaac Bogoch; Nick Ruktanonchai; Alexander Watts; Xin Lu; Weizhong Yang; Hongjie Yu; Kamran Khan; Andrew J Tatem",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.20020479"],["9fd5a49o","158e546658e42ab0a89d174bc9facb437a034df6","medrxiv","Using predicted imports of 2019-nCoV cases to determine locations that may not be identifying all imported cases","10.1101/2020.02.04.20020495",null,null,"medrvix","Cases from the ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (2019-nCoV) exported from mainland China can lead to self-sustained outbreaks in other populations. Internationally imported cases are currently being reported in several different locations. Early detection of imported cases is critical for containment of the virus. Based on air travel volume estimates from Wuhan to international destinations and using a generalized linear regression model we identify locations which may potentially have undetected internationally imported cases.","2020-02-05","Pablo M De Salazar; Rene Niehus; Aimee Taylor; Caroline O Buckee; Marc Lipsitch",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.20020495"],["ecu579el","136ee862ad53b606c8f2bea917b9705f5079e88e","medrxiv","Risk assessment of novel coronavirus COVID-19 outbreaks outside China","10.1101/2020.02.04.20020503",null,null,"medrvix","We developed a computational tool to assess the risk of novel coronavirus outbreaks outside China. We estimate the dependence of the risk of a major outbreak in a country from imported cases on key parameters such as: (i) the evolution of the cumulative number of cases in mainland China outside the closed areas; (ii) the connectivity of the destination country with China, including baseline travel frequencies, the effect of travel restrictions, and the efficacy of entry screening at destination; (iii) the efficacy of control measures in the destination country (expressed by the local reproduction number Rloc). We found that in countries with low connectivity to China but with relatively high Rloc, the most beneficial control measure to reduce the risk of outbreaks is a further reduction in their importation number either by entry screening or travel restrictions. Countries with high connectivity but low Rloc benefit the most from policies that further reduce Rloc. Countries in the middle should consider a combination of such policies. Risk assessments were illustrated for selected groups of countries from America, Asia and Europe, and we investigated how their risks depend on those parameters, and how the risk is increasing in time as the number of cases in China is growing.","2020-02-05","Peter Boldog; Tamas Tekeli; Zsolt Vizi; Attila Denes; Ferenc Bartha; Gergely Rost",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.20020503"],["mtv80pjo","8a1fde8c65e439496ac5810504de23ef77312f28","biorxiv","Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1","10.1101/2020.02.04.933135",null,null,"biorxiv","AbstractAs the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji et al. (https://doi.org/10.1002/jmv.25682) and by Pradhan et al. (https://doi.org/10.1101/2020.01.30.927871), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the “novel insertions” observed in the spike protein are naturally evolved from bat coronaviruses.","2020-02-08","Chengxin Zhang; Wei Zheng; Xiaoqiang Huang; Eric W. Bell; Xiaogen Zhou; Yang Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.933135"],["6ojmmmuj","74b00f19c3af87d1081644f02490ba250f57b7ca","biorxiv","Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach","10.1101/2020.02.04.934232",null,null,"biorxiv","AbstractBackgroundNew endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a “public-health emergency of international concern” due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.MethodSeveral techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.ResultsExtensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.ConclusionT cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.","2020-02-11","Miyssa I. Abdelmageed; Abdelrahman H. Abdelmoneim; Mujahed I. Mustafa; Nafisa M. Elfadol; Naseem S. Murshed; Shaza W. Shantier; Abdelrafie M. Makhawi",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.04.934232"],["uqkcbs2s","59eab95c43fdea01481fdbf9bae45dfe28ffc693","medrxiv","Bulk and single-cell transcriptomics identify tobacco-use disparity in lung gene expression of ACE2, the receptor of 2019-nCov","10.1101/2020.02.05.20020107",null,null,"medrvix","In current severe global emergency situation of 2019-nCov outbreak, it is imperative to identify vulnerable and susceptible groups for effective protection and care. Recently, studies found that 2019-nCov and SARS-nCov share the same receptor, ACE2. In this study, we analyzed five large-scale bulk transcriptomic datasets of normal lung tissue and two single-cell transcriptomic datasets to investigate the disparities related to race, age, gender and smoking status in ACE2 gene expression and its distribution among cell types. We didn't find significant disparities in ACE2 gene expression between racial groups (Asian vs Caucasian), age groups (>60 vs <60) or gender groups (male vs female). However, we observed significantly higher ACE2 gene expression in former smoker's lung compared to non-smoker's lung. Also, we found higher ACE2 gene expression in Asian current smokers compared to non-smokers but not in Caucasian current smokers, which may indicate an existence of gene-smoking interaction. In addition, we found that ACE2 gene is expressed in specific cell types related to smoking history and location. In bronchial epithelium, ACE2 is actively expressed in goblet cells of current smokers and club cells of non-smokers. In alveoli, ACE2 is actively expressed in remodelled AT2 cells of former smokers. Together, this study indicates that smokers especially former smokers may be more susceptible to 2019-nCov and have infection paths different with non-smokers. Thus, smoking history may provide valuable information in identifying susceptible population and standardizing treatment regimen.","2020-02-11","Guoshuai Cai",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.05.20020107"],["xtmozkf6","f5d9ab798adf8face1655d476644ebb75fbfef18","medrxiv","ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism","10.1101/2020.02.05.20020545",null,null,"medrvix","Respiratory disease caused by the 2019 novel coronavirus (2019-nCoV) pneumonia first emerged in Wuhan, Hubei Province, China, in December 2019 and spread rapidly to other provinces and other countries. Angiotensin-converting enzyme 2 (ACE2) is the receptor for SARS-CoV and has been suggested to be also the receptor for 2019-nCoV. Paradoxically, ACE2 expression in the lung protects mice from SARS-CoV spike protein induced lung injury by attenuating the renin-angiotensin system. In the intestine, ACE2 also suppresses intestinal inflammation by maintaining amino acid homeostasis, antimicrobial peptide expression and ecology of the gut microbiome. Upon analysis of single cell-RNA sequencing data from control subjects and those with colitis or inflammatory bowel disease (IBD), we found that ACE2 expression in the colonocytes was positively associated with genes regulating viral infection, innate and cellular immunity, but was negatively associated with viral transcription, protein translation, humoral immunity, phagocytosis and complement activation. In summary, we suggest that ACE2 may play dual roles in mediating the susceptibility and immunity of 2019-nCoV infection.","2020-02-07","Jun Wang; Shanmeizi Zhao; Ming Liu; Zhiyao Zhao; Yiping Xu; Ping Wang; Meng Lin; Yanhui Xu; Bing Huang; Xiaoyu Zuo; Zhanghua Chen; Fan Bai; Jun Cui; Andrew M Lew; Jincun Zhao; Yan Zhang; Haibin Luo; Yuxia Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.05.20020545"],["vpouyyev","5f6a410a4ba086b296100a8a5a10df59eb337c90","medrxiv","Epidemic doubling time of the COVID-19 epidemic by Chinese province","10.1101/2020.02.05.20020750",null,null,"medrvix","COVID-19 epidemic doubling time by Chinese province was increasing from January 20 through February 9, 2020. Yet, the harmonic mean doubling time was relatively short, ranging from 1.4 (Hunan, 95% CI, 1.2-2.0) to 3.0 (Xinjiang, 95% CI, 2.0-4.9) days, with an estimate of 2.5 days (95% CI, 2.4-2.7) for Hubei.","2020-02-06","Kamalich Muniz-Rodriguez; Gerardo Chowell; Chi-Hin Cheung; Dongyu Jia; Po-Ying Lai; Yiseul Lee; Manyun Liu; Sylvia K. Ofori; Kimberlyn M. Roosa; Lone Simonsen; Cecile G. Viboud; Isaac Chun-Hai Fung",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.05.20020750"],["4i97fpzc","8762adc9311e5803d1de438493772269acfe7fa3","medrxiv","Preparedness and vulnerability of African countries against introductions of 2019-nCoV","10.1101/2020.02.05.20020792",null,null,"medrvix","The novel coronavirus (2019-nCoV) epidemic has spread to 23 countries from China. Local cycles of transmission already occurred in 7 countries following case importation. No African country has reported cases yet. The management and control of 2019-nCoV introductions heavily relies on the public health capacity of a country. Here we evaluate the preparedness and vulnerability of African countries against their risk of importation of 2019-nCoV. We used data on air travel volumes departing from airports in the infected provinces in China and directed to Africa to estimate the risk of introduction per country. We determined the countries capacity to detect and respond to cases with two indicators: preparedness, using the WHO International Health Regulation Monitoring and Evaluation Framework; and vulnerability, with the Infectious Disease Vulnerability Index. Countries were clustered according to the Chinese regions contributing the most to their risk. Findings: Countries at the highest importation risk (Egypt, Algeria, Republic of South Africa) have moderate to high capacity to respond to outbreaks. Countries at moderate risk (Nigeria, Ethiopia, Sudan, Angola, Tanzania, Ghana, Kenya) have variable capacity and high vulnerability. Three clusters of countries are identified that share the same exposure to the risk originating from the provinces of Guangdong, Fujian, and Beijing, respectively. Interpretation: Several countries in Africa are stepping up their preparedness to detect and cope with 2019-nCoV importations. Resources and intensified surveillance and capacity capacity should be urgently prioritized towards countries at moderate risk that may be ill-prepared to face the importation and to limit onward transmission.","2020-02-07","Marius Gilbert; Giulia Pullano; Francesco Pinotti; Eugenio Valdano; Chiara Poletto; Pierre-Yves Boelle; Eric D'Ortenzio; Yazdan Yazdanpanah; Serge Paul Eholie; Mathias Altmann; Bernardo Gutierrez; Moritz U. G. Kraemer; Vittoria Colizza",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.05.20020792"],["jawwrsa5",null,"biorxiv","A database resource for Genome-wide dynamics analysis of Coronaviruses on a historical and global scale","10.1101/2020.02.05.920009",null,null,"biorxiv","AbstractThe recent outbreak of a new zoonotic origin Coronavirus has ring the bell for the potential spread of epidemic Coronavirus crossing the species. With the urgent needs to assist the control of the Coronavirus spread and to provide valuable scientific information, we developed a coronavirus database (CoVdb), an online genomics and proteomics analysis platform. Based on public available coronavirus genomic information, the database annotates the genome of every strain and identifies 780 possible ORFs of all strains available in Genebank. In addition, the comprehensive evaluation of all the published genomes of Coronavirus strains, including population genetics analysis, functional analysis and structural analysis on a historical and global scale were presented in the CoVdb. In the database, the researcher can easily obtain the basic information of a Coronavirus gene with the distribution of the gene among strains, conserved or high mutation regions, possible subcellular location and topology of the gene. Moreover, sliding windows for population genetics analysis results is provided, thereby facilitating genetics and evolutional analysis at the genomic level. CoVdb can be accessed freely at http://covdb.popgenetics.net.","2020-02-07","Zhenglin Zhu; Kaiwen Meng; Geng Meng",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.05.920009"],["7vrm081c","73c8af41cfdbf52c0dfba37727e3b94cb56b495e","biorxiv","The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19","10.1101/2020.02.05.935072",null,null,"biorxiv","AbstractWuhan Novel Coronavirus disease (COVID-19) outbreak has become a global outbreak which has raised the concern of scientific community to design and discover a definitive cure against this deadly virus which has caused deaths of numerous infected people upon infection and spreading. To date, no antiviral therapy or vaccine is available which can effectively combat the infection caused by this virus. This study was conducted to design possible epitope-based subunit vaccines against the SARS-CoV-2 virus using the approaches of reverse vaccinology and immunoinformatics. Upon continual computational experimentation three possible vaccine constructs were designed and one vaccine construct was selected as the best vaccine based on molecular docking study which is supposed to effectively act against SARS-CoV-2. Later, molecular dynamics simulation and in silico codon adaptation experiments were carried out in order to check biological stability and find effective mass production strategy of the selected vaccine. Hopefully, this study will contribute to uphold the present efforts of the researches to secure a definitive treatment against this lethal virus.","2020-02-11","Bishajit Sarkar; Md. Asad Ullah; Fatema Tuz Johora; Masuma Afrin Taniya; Yusha Araf",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.05.935072"],["py6qu4tl","dca8ced82157924ed86c698a7dd482be81b4b266","biorxiv","Teicoplanin potently blocks the cell entry of 2019-nCoV","10.1101/2020.02.05.935387",null,null,"biorxiv","AbstractSince December 2019, the outbreak of a new coronavirus, named 2019-nCoV, has greatly threatened the public health in China and raised great concerns worldwide. No specific treatment for this infection is currently available. We previously reported that teicoplanin, a glycopeptide antibiotic which has routinely been used in the clinic to treat bacterial infection with low toxicity, significantly inhibits the invasion of cells by Ebola virus, SARS-CoV and MERS-CoV, via specifically inhibiting the activity of cathepsin L. Here, we tested the efficacy of teicoplanin against 2019-nCoV virus infection and found that teicoplanin potently prevents the entrance of 2019-nCoV-Spike-pseudoviruses into the cytoplasm, with an IC50 of 1.66 μM. Although the inhibitory effect upon the replication of wildtype viruses ex vivo and in vivo remains to be determined, our preliminary result indicates that the potential antiviral activity of teicoplanin could be applied for the treatment of 2019-nCoV virus infection.","2020-02-13","Junsong Zhang; Xiancai Ma; Fei Yu; Jun Liu; Fan Zou; Ting Pan; Hui Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.05.935387"],["g5wpa2ee","0aff00101d5ccc6592987185ab833f95d842f98b","biorxiv","Potentially highly potent drugs for 2019-nCoV","10.1101/2020.02.05.936013",null,null,"biorxiv","AbstractThe World Health Organization (WHO) has declared the 2019 novel coronavirus (2019-nCoV) infection outbreak a global health emergency. Currently, there is no effective anti-2019-nCoV medication. The sequence identity of the 3CL proteases of 2019-nCoV and SARS is 96%, which provides a sound foundation for structural-based drug repositioning (SBDR). Based on a SARS 3CL protease X-ray crystal structure, we construct a 3D homology structure of 2019-nCoV 3CL protease. Based on this structure and existing experimental datasets for SARS 3CL protease inhibitors, we develop an SBDR model based on machine learning and mathematics to screen 1465 drugs in the DrugBank that have been approved by the U.S. Food and Drug Administration (FDA). We found that many FDA approved drugs are potentially highly potent to 2019-nCoV.","2020-02-13","Duc Duy Nguyen; Kaifu Gao; Jiahui Chen; Rui Wang; Guo-Wei Wei",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.05.936013"],["ra3t6kmm","c85f571a674c7fed0ccb9176e9cf9f3d3659ca32","medrxiv","Analysis of the epidemic growth of the early 2019-nCoV outbreak using internationally confirmed cases","10.1101/2020.02.06.20020941",null,null,"medrvix","Background: On January 23, 2020, a quarantine was imposed on travel in and out of Wuhan, where the 2019 novel coronavirus (2019-nCoV) outbreak originated from. Previous analyses estimated the basic epidemiological parameters using symptom onset dates of the confirmed cases in Wuhan and outside China. Methods: We obtained information on the 46 coronavirus cases who traveled from Wuhan before January 23 and have been subsequently confirmed in Hong Kong, Japan, Korea, Macau, Singapore, and Taiwan as of February 5, 2020. Most cases have detailed travel history and disease progress. Compared to previous analyses, an important distinction is that we used this data to informatively simulate the infection time of each case using the symptom onset time, previously reported incubation interval, and travel history. We then fitted a simple exponential growth model with adjustment for the January 23 travel ban to the distribution of the simulated infection time. We used a Bayesian analysis with diffuse priors to quantify the uncertainty of the estimated epidemiological parameters. We performed sensitivity analysis to different choices of incubation interval and the hyperparameters in the prior specification. Results: We found that our model provides good fit to the distribution of the infection time. Assuming the travel rate to the selected countries and regions is constant over the study period, we found that the epidemic was doubling in size every 2.9 days (95% credible interval [CrI], 2 days--4.1 days). Using previously reported serial interval for 2019-nCoV, the estimated basic reproduction number is 5.7 (95% CrI, 3.4--9.2). The estimates did not change substantially if we assumed the travel rate doubled in the last 3 days before January 23, when we used previously reported incubation interval for severe acute respiratory syndrome (SARS), or when we changed the hyperparameters in our prior specification. Conclusions: Our estimated epidemiological parameters are higher than an earlier report using confirmed cases in Wuhan. This indicates the 2019-nCoV could have been spreading faster than previous estimates.","2020-02-09","Qingyuan Zhao; Yang Chen; Dylan S Small",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.06.20020941"],["tovfd9lw","dfb0fedbeed56bd2b795a67faab28295afc14c96","medrxiv","Clinical characteristics of 2019 novel coronavirus infection in China","10.1101/2020.02.06.20020974",null,null,"medrvix","Background: Since December 2019, acute respiratory disease (ARD) due to 2019 novel coronavirus (2019-nCoV) emerged in Wuhan city and rapidly spread throughout China. We sought to delineate the clinical characteristics of these cases. Methods: We extracted the data on 1,099 patients with laboratory-confirmed 2019-nCoV ARD from 552 hospitals in 31 provinces/provincial municipalities through January 29th, 2020. Results: The median age was 47.0 years, and 41.90% were females. Only 1.18% of patients had a direct contact with wildlife, whereas 31.30% had been to Wuhan and 71.80% had contacted with people from Wuhan. Fever (87.9%) and cough (67.7%) were the most common symptoms. Diarrhea is uncommon. The median incubation period was 3.0 days (range, 0 to 24.0 days). On admission, ground-glass opacity was the typical radiological finding on chest computed tomography (50.00%). Significantly more severe cases were diagnosed by symptoms plus reverse-transcriptase polymerase-chain-reaction without abnormal radiological findings than non-severe cases (23.87% vs. 5.20%, P<0.001). Lymphopenia was observed in 82.1% of patients. 55 patients (5.00%) were admitted to intensive care unit and 15 (1.36%) succumbed. Severe pneumonia was independently associated with either the admission to intensive care unit, mechanical ventilation, or death in multivariate competing-risk model (sub-distribution hazards ratio, 9.80; 95% confidence interval, 4.06 to 23.67). Conclusions: The 2019-nCoV epidemic spreads rapidly by human-to-human transmission. Normal radiologic findings are present among some patients with 2019-nCoV infection. The disease severity (including oxygen saturation, respiratory rate, blood leukocyte/lymphocyte count and chest X-ray/CT manifestations) predict poor clinical outcomes.","2020-02-09","Wei-jie Guan; Zheng-yi Ni; Yu Hu; Wen-hua Liang; Chun-quan Ou; Jian-xing He; Lei Liu; Hong Shan; Chun-liang Lei; David SC Hui; Bin Du; Lan-juan Li; Guang Zeng; Kowk-Yung Yuen; Ru-chong Chen; Chun-li Tang; Tao Wang; Ping-yan Chen; Jie Xiang; Shi-yue Li; Jin-lin Wang; Zi-jing Liang; Yi-xiang Peng; Li Wei; Yong Liu; Ya-hua Hu; Peng Peng; Jian-ming Wang; Ji-yang Liu; Zhong Chen; Gang Li; Zhi-jian Zheng; Shao-qin Qiu; Jie Luo; Chang-jiang Ye; Shao-yong Zhu; Nan-shan Zhong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.06.20020974"],["77nsidzc","4c24ee4a86de1d5207d93ce39132f7b660cf82e0","biorxiv","Freshwater monitoring by nanopore sequencing","10.1101/2020.02.06.936302",null,null,"biorxiv","ABSTRACTClean freshwater lies at the heart of human society and monitoring its quality is paramount. In addition to chemical controls, traditional microbiological water tests focus on the detection of specific bacterial pathogens. The direct tracing of all aquatic DNA poses a more profound alternative. Yet, this has hitherto been underused due to challenges in cost and logistics. Here we present a simple, fast, inexpensive and comprehensive freshwater diagnostics workflow centred around portable nanopore DNA sequencing. Using defined bacterial compositions and spatiotemporal microbiota from surface water of an example river in Cambridge (UK), our study shows how nanopore sequencing can be readily integrated for the assessment of aquatic bacterial diversity and pollution. We provide a computational benchmark that features more than ten taxonomic classification tools to derive guidelines for bacterial DNA analyses with nanopore data. Through complementary physicochemical measurements, we find that nanopore metagenomics can depict fine temporal gradients along the main hydrological axis of an urban-rural interface and yield high-resolution pathogen maps that address concerns of public health.","2020-02-07","Lara Urban; Andre Holzer; J Jotautas Baronas; Michael Hall; Philipp Braeuninger-Weimer; Michael J Scherm; Daniel J Kunz; Surangi N Perera; Daniel E Martin-Herranz; Edward T Tipper; Susannah J Salter; Maximilian R Stammnitz",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.06.936302"],["q14x0i2c","ac35eeb6eaaf382fe0e123719f7dc409046e5d87","medrxiv","Incorporating Human Movement Data to Improve Epidemiological Estimates for 2019-nCoV","10.1101/2020.02.07.20021071",null,null,"medrvix","Estimating the key epidemiological features of the novel coronavirus (2019-nCoV) epidemic proves to be challenging, given incompleteness and delays in early data reporting, in particular, the severe under-reporting bias in the epicenter, Wuhan, Hubei Province, China. As a result, the current literature reports widely varying estimates. We developed an alternative geo-stratified debiasing estimation framework by incorporating human mobility with case reporting data in three stratified zones, i.e., Wuhan, Hubei Province excluding Wuhan, and mainland China excluding Hubei. We estimated the latent infection ratio to be around 0.12% (18,556 people) and the basic reproduction number to be 3.24 in Wuhan before the city's lockdown on January 23, 2020. The findings based on this debiasing framework have important implications to prioritization of control and prevention efforts.","2020-02-09","Zhidong Cao; Qingpeng Zhang; Xin Lu; Dirk Pfeiffer; Lei Wang; Hongbing Song; Tao Pei; Zhongwei Jia; Daniel Dajun Zeng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.20021071"],["kevrp8rg","d5f14e63b86239f491c9dd1749cff01dc4d633cb","medrxiv","Networks of information token recurrences derived from genomic sequences may reveal hidden patterns in epidemic outbreaks: A case study of the 2019-nCoV coronavirus.","10.1101/2020.02.07.20021139",null,null,"medrvix","Profiling the genetic evolution and dynamic spreading of viruses is a crucial task when responding to epidemic outbreaks. We aim to devise novel ways to model, visualise and analyse the temporal dynamics of epidemic outbreaks in order to help researchers and other people involved in crisis response to make well-informed and targeted decisions about from which geographical locations and time periods more genetic samples may be required to fully understand the outbreak. Our approach relies on the application of Transcendental Information Cascades to a set of temporally ordered nucleotide sequences and we apply it to real-world data that was collected during the currently ongoing outbreak of the novel 2019-nCoV coronavirus. We assess information-theoretic and network-theoretic measures that characterise the resulting complex network and suggest touching points and temporal pathways that are of interest for deeper investigation by geneticists and epidemiologists.","2020-02-11","Markus Luczak-Roesch",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.20021139"],["45g12waw","36a5f6d55d7c5f67d4344e36da0a72856ad3dda0","medrxiv","The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated","10.1101/2020.02.07.20021154",null,null,"medrvix","The novel coronavirus (2019-nCoV) is a recently emerged human pathogen that has spread widely since January 2020. Initially, the basic reproductive number, R0, was estimated to be 2.2 to 2.7. Here we provide a new estimate of this quantity. We collected extensive individual case reports and estimated key epidemiology parameters, including the incubation period. Integrating these estimates and high-resolution real-time human travel and infection data with mathematical models, we estimated that the number of infected individuals during early epidemic double every 2.4 days, and the R0 value is likely to be between 4.7 and 6.6. We further show that quarantine and contact tracing of symptomatic individuals alone may not be effective and early, strong control measures are needed to stop transmission of the virus.","2020-02-11","Steven Sanche; Yen Ting Lin; Chonggang Xu; Ethan Romero-Severson; Nick Hengartner; Ruian Ke",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.20021154"],["7axgotby","ba92c589f077b55e9cd13264d619c2ffe9d3ce7a","medrxiv","Transmission Dynamics of 2019-nCoV in Malaysia","10.1101/2020.02.07.20021188",null,null,"medrvix","This paper focuses on the formulation of a deterministic 2019-nCov transmission model by considering the exposed and recovered populations with immunity. The scenario of the simulation is depicted based on the patient zero in Malaysia. The transmission model is found to be able to predict the next confirmed case given a single case introduced in a fully susceptible population. The mathematical model is developed based on the SEIR model which has susceptible, exposed, infectious and recovered populations. The system of equations which were obtained were solved numerically and the simulation results were analyzed. The analysis includes the impact of the disease if no control is taken.","2020-02-11","Jane Labadin; Boon Hao Hong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.20021188"],["y4we4rrk",null,"medrxiv","Tracking the spread of novel coronavirus (2019-nCoV) based on big data","10.1101/2020.02.07.20021196",null,null,"medrvix","The novel coronavirus (2019-nCoV) appeared in Wuhan in late 2019 have infected 34,598 people, and killed 723 among them until 8th February 2020. The new virus has spread to at least 316 cities (until 1st February 2020) in China. We used the traffic flow data from Baidu Map, and number of air passengers who left Wuhan from 1st January to 26th January, to quantify the potential infectious people. We developed multiple linear models with local population and air passengers as predicted variables to explain the variance of confirmed cases in every city across China. We found the contribution of air passengers from Wuhan was decreasing gradually, but the effect of local population was increasing, indicating the trend of local transmission. However, the increase of local transmission is slow during the early stage of novel coronavirus, due to the super strict control measures carried out by government agents and communities.","2020-02-11","Xumao Zhao; Xiang Liu; Xinhai Li",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.20021196"],["ptbk69rk","3e5633a9a76a8c03e4ec57e4659872a63f045117","biorxiv","Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group","10.1101/2020.02.07.937862",null,null,"biorxiv","AbstractThe present outbreak of lower respiratory tract infections, including respiratory distress syndrome, is the third spillover, in only two decades, of an animal coronavirus to humans resulting in a major epidemic. Here, the Coronavirus Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the official classification of viruses and taxa naming (taxonomy) of the Coronaviridae family, assessed the novelty of the human pathogen tentatively named 2019-nCoV. Based on phylogeny, taxonomy and established practice, the CSG formally recognizes this virus as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus and designates it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). To facilitate communication, the CSG further proposes to use the following naming convention for individual isolates: SARS-CoV-2/Isolate/Host/Date/Location. The spectrum of clinical manifestations associated with SARS-CoV-2 infections in humans remains to be determined. The independent zoonotic transmission of SARS-CoV and SARS-CoV-2 highlights the need for studying the entire (virus) species to complement research focused on individual pathogenic viruses of immediate significance. This research will improve our understanding of virus-host interactions in an ever-changing environment and enhance our preparedness for future outbreaks.","2020-02-11","Alexander E. Gorbalenya; Susan C. Baker; Ralph S. Baric; Raoul J. de Groot; Christian Drosten; Anastasia A. Gulyaeva; Bart L. Haagmans; Chris Lauber; Andrey M Leontovich; Benjamin W. Neuman; Dmitry Penzar; Stanley Perlman; Leo L.M. Poon; Dmitry Samborskiy; Igor A. Sidorov; Isabel Sola; John Ziebuhr",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.937862"],["kh06egig",null,"biorxiv","Exploring the coronavirus epidemic using the new WashU Virus Genome Browser","10.1101/2020.02.07.939124",null,null,"biorxiv","AbstractSince its debut in mid-December, 2019, the novel coronavirus (2019-nCoV) has rapidly spread from its origin in Wuhan, China, to several countries across the globe, leading to a global health crisis. As of February 7, 2020, 44 strains of the virus have been sequenced and uploaded to NCBI’s GenBank [1], providing insight into the virus’s evolutionary history and pathogenesis. Here, we present the WashU Virus Genome Browser, a web-based portal for viewing virus genomic data. The browser is home to 16 complete 2019-nCoV genome sequences, together with hundreds of related viral sequences including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and Ebola virus. In addition, the browser features unique customizability, supporting user-provided upload of novel viral sequences in various formats. Sequences can be viewed in both a track-based representation as well as a phylogenetic tree-based view, allowing the user to easily compare sequence features across multiple strains. The WashU Virus Genome Browser inherited many features and track types from the WashU Epigenome Browser, and additionally incorporated a new type of SNV track to address the specific needs of viral research. Our Virus Browser portal can be accessed at https://virusgateway.wustl.edu, and documentation is available at https://virusgateway.readthedocs.io/.","2020-02-11","Jennifer A. Flynn; Deepak Purushotham; Mayank NK Choudhary; Xiaoyu Zhuo; Changxu Fan; Gavriel Matt; Daofeng Li; Ting Wang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.939124"],["dnxhtbxn","adb09e4e5c7331b2aa661b3d3bb0a643e00d11bc","biorxiv","Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019","10.1101/2020.02.07.939207",null,null,"biorxiv","SUMMARYA novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019.","2020-02-13","Matthew C. Wong; Sara J. Javornik Cregeen; Nadim J. Ajami; Joseph F. Petrosino",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.939207"],["25uy4lrw",null,"biorxiv","The Pathogenicity of SARS-CoV-2 in hACE2 Transgenic Mice","10.1101/2020.02.07.939389",null,null,"biorxiv","AbstractSevere acute respiratory syndrome CoV-2 (SARS-CoV-2) caused the Corona Virus Disease 2019 (COVID-19) cases in China has become a public health emergency of international concern (PHEIC). Based on angiotensin converting enzyme 2 (ACE2) as cell entry receptor of SARS-CoV, we used the hACE2 transgenic mice infected with SARS-CoV-2 to study the pathogenicity of the virus. Weight loss and virus replication in lung were observed in hACE2 mice infected with SARS-CoV-2. The typical histopathology was interstitial pneumonia with infiltration of significant lymphocytes and monocytes in alveolar interstitium, and accumulation of macrophages in alveolar cavities. Viral antigens were observed in the bronchial epithelial cells, alveolar macrophages and alveolar epithelia. The phenomenon was not found in wild type mice with SARS-CoV-2 infection. The pathogenicity of SARS-CoV-2 in hACE2 mice was clarified and the Koch’s postulates were fulfilled as well, and the mouse model may facilitate the development of therapeutics and vaccines against SARS-CoV-2.","2020-02-11","Linlin Bao; Wei Deng; Baoying Huang; Hong Gao; Jiangning Liu; Lili Ren; Qiang Wei; Pin Yu; Yanfeng Xu; Feifei Qi; Yajin Qu; Fengdi Li; Qi Lv; Wenling Wang; Jing Xue; Shuran Gong; Mingya Liu; Guanpeng Wang; Shunyi Wang; Zhiqi Song; Linna Zhao; Peipei Liu; Li Zhao; Fei Ye; Huijuan Wang; Weimin Zhou; Na Zhu; Wei Zhen; Haisheng Yu; Xiaojuan Zhang; Li Guo; Lan Chen; Conghui Wang; Ying Wang; Xinming Wang; Yan Xiao; Qiangming Sun; Hongqi Liu; Fanli Zhu; Chunxia Ma; Lingmei Yan; Mengli Yang; Jun Han; Wenbo Xu; Wenjie Tan; Xiaozhong Peng; Qi Jin; Guizhen Wu; Chuan Qin",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.07.939389"],["a8ig607t","2cc809ed4d5c3640ee2351fdb1876e8ff6c01b51","medrxiv","Feasibility of controlling 2019-nCoV outbreaks by isolation of cases and contacts","10.1101/2020.02.08.20021162",null,null,"medrvix","Background: To assess the viability of isolation and contact tracing to control onwards transmission from imported cases of 2019-nCoV. Methods: We developed a stochastic transmission model, parameterised to the 2019-nCoV outbreak. We used the model to quantify the potential effectiveness of contact tracing and isolation of cases at controlling a 2019 nCoV-like pathogen. We considered scenarios that varied in: the number of initial cases; the basic reproduction number R0; the delay from symptom onset to isolation; the probability contacts were traced; the proportion of transmission that occurred before symptom onset, and the proportion of subclinical infections. We assumed isolation prevented all further transmission in the model. Outbreaks were deemed controlled if transmission ended within 12 weeks or before 5000 cases in total. We measured the success of controlling outbreaks using isolation and contact tracing, and quantified the weekly maximum number of cases traced to measure feasibility of public health effort. Findings: While simulated outbreaks starting with only 5 initial cases, R0 of 1.5 and little transmission before symptom onset could be controlled even with low contact tracing probability, the prospects of controlling an outbreak dramatically dropped with the number of initial cases, with higher R0, and with more transmission before symptom onset. Across different initial numbers of cases, the majority of scenarios with an R0 of 1.5 were controllable with under 50% of contacts successfully traced. For R0 of 2.5 and 3.5, more than 70% and 90% of contacts respectively had to be traced to control the majority of outbreaks. The delay between symptom onset and isolation played the largest role in determining whether an outbreak was controllable for lower values of R0. For higher values of R0 and a large initial number of cases, contact tracing and isolation was only potentially feasible when less than 1% of transmission occurred before symptom onset. Interpretation: We found that in most scenarios contact tracing and case isolation alone is unlikely to control a new outbreak of 2019-nCov within three months. The probability of control decreases with longer delays from symptom onset to isolation, fewer cases ascertained by contact tracing, and increasing transmission before symptoms. This model can be modified to reflect updated transmission characteristics and more specific definitions of outbreak control to assess the potential success of local response efforts.","2020-02-11","Joel Hellewell; Sam Abbott; Amy Gimma; Nikos I Bosse; Christopher I Jarvis; Timothy W Russell; James D Munday; Adam J Kucharski; W John Edmunds; CMMID nCoV working group; Sebastian Funk; Rosalind M Eggo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.20021162"],["0qwwycnc",null,"medrxiv","Caution on Kidney Dysfunctions of 2019-nCoV Patients","10.1101/2020.02.08.20021212",null,null,"medrvix","Until 24:00 of February 7th 2020, 31774 laboratory-confirmed cases of novel coronavirus (2019-nCoV) infection have been reported, including 6101 severe cases in critical conditions and 722 deaths. The critical and urgent need at this moment is to find an effective treatment strategy with available means to prevent these thousands of severe inpatients from worsening and dying. It has been recently known that the 2019-nCoV shares a common cellular mechanism with the severe acute respiratory syndrome-associated coronavirus (SARS-CoV). Thus, we surveyed a previous retrospective case study on SARS which showed that acute renal impairment was uncommon in SARS but carried a formidably high mortality (91.7%, 33 of 36 cases). Here we report an ongoing case study on kidney functions in 59 patients infected by 2019-nCoV (including 28 diagnosed as severe cases and 3 deaths). 63% (32/51) of the patients exhibited proteinuria, indicative of renal impairment. 19% (11/59) and 27% (16/59) of the patients had an elevated level of plasma creatinine and urea nitrogen respectively. The computerized tomography (CT) scan showed radiographic abnormalities of the kidneys in 100% (27/27) of the patients. Together, these multiple lines of evidence point to the idea that renal impairment is common in 2019-nCov patients, which may be one of the major causes of the illness by the virus infection and also may contribute to multi-organ failure and death eventually. Therefore, we strongly suggest exercising a high degree of caution in monitoring the kidney functions of 2019-nCoV patients and, very importantly, that applying potential interventions including continuous renal replacement therapies (CRRT) for protecting kidney functions as early as possible, particular for those with plasma creatinine rising, is key to preventing fatality.","2020-02-12","Zhen Li; Ming Wu; Jie Guo; Jiwei Yao; Xiang Liao; Siji Song; Min Han; Jiali Li; Guangjie Duan; Yuanxiu Zhou; Xiaojun Wu; Zhansong Zhou; Taojiao Wang; Ming Hu; Xianxiang Chen; Yu Fu; Chong Lei; Hailong Dong; Yi Zhou; Hongbo Jia; Xiaowei Chen; Junan Yan",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.20021212"],["juiomzb3","5bbc428de7fc2f10c24d8439edda26d84b319687","medrxiv","Estimation of the Time-Varying Reproduction Number of COVID-19 Outbreak in China","10.1101/2020.02.08.20021253",null,null,"medrvix","Background: The 2019-nCoV outbreak in Wuhan, China has attracted world-wide attention. As of February 11, 2020, a total of 44730 cases of novel coronavirus-infected pneumonia associated with COVID-19 were confirmed by the National Health Commission of China. Methods: Three approaches, namely Poisson likelihood-based method (ML), exponential growth rate-based method (EGR) and stochastic Susceptible-Infected-Removed dynamic model-based method (SIR), were implemented to estimate the basic and controlled reproduction numbers. Results: A total of 71 chains of transmission together with dates of symptoms onset and 67 dates of infections were identified among 5405 confirmed cases outside Hubei as reported by February 2, 2020. Based on this information, we find the serial interval having an average of 4.41 days with a standard deviation of 3.17 days and the infectious period having an average of 10.91 days with a standard deviation of 3.95 days. Conclusions: The controlled reproduction number is declining. It is lower than one in most regions of China, but is still larger than one in Hubei Province. Sustained efforts are needed to further reduce the Rc to below one in order to end the current epidemic.","2020-02-11","Chong You; Yuhao Deng; Wenjie Hu; Jiarui Sun; Qiushi Lin; Feng Zhou; Cheng Heng Pang; Yuan Zhang; Zhengchao Chen; Xiao-Hua Zhou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.20021253"],["2g8rfzf4","5cfb44ecdfb61b2898b6cf7315b1e810874a9560","medrxiv","Assessing the plausibility of subcritical transmission of 2019-nCoV in the United States","10.1101/2020.02.08.20021311",null,null,"medrvix","Abstract: The 2019-nCoV outbreak has raised concern of global spread. While person-to-person transmission within the Wuhan district has led to a large outbreak, the transmission potential outside of the region remains unclear. Here we present a simple approach for determining whether the upper limit of the confidence interval for the reproduction number exceeds one for transmission in the United States, which would allow endemic transmission. As of February 7, 2020, the number of cases in the United states support subcritical transmission, rather than ongoing transmission. However, this conclusion can change if pre-symptomatic cases resulting from human-to-human transmission have not yet been identified.","2020-02-11","Seth Blumberg; Thomas M Lietman; Travis C Porco",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.20021311"],["pm8usc55",null,"biorxiv","The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS","10.1101/2020.02.08.926006",null,null,"biorxiv","AbstractAt the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China1, even more spread than SARS-CoV infection2. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion3,4. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear5,6. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.","2020-02-11","Tong Meng; Hao Cao; Hao Zhang; Zijian Kang; Da Xu; Haiyi Gong; Jing Wang; Zifu Li; Xingang Cui; Huji Xu; Haifeng Wei; Xiuwu Pan; Rongrong Zhu; Jianru Xiao; Wang Zhou; Liming Cheng; Jianmin Liu",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.926006"],["hwvr0v8c","329dfdd2fbe4b3c510c5329219bfd33764ab1710","biorxiv","Domestication reprogrammed the budding yeast life cycle","10.1101/2020.02.08.939314",null,null,"biorxiv","AbstractDomestication of plants and animals is the foundation for feeding the world population. We report that domestication of the model yeast S. cerevisiae reprogrammed its life cycle entirely. We tracked growth, gamete formation and cell survival across many environments for nearly 1000 genome sequenced isolates and found a remarkable dichotomy between domesticated and wild yeasts. Wild yeasts near uniformly trigger meiosis and sporulate when encountering nutrient depletions, whereas domestication relaxed selection on sexual reproduction and favoured survival as quiescent cells. Domestication also systematically enhanced fermentative over respiratory traits while decreasing stress tolerance. We show that this yeast domestication syndrome was driven by aneuploidies and gene function losses that emerged independently in multiple domesticated lineages during the specie’s recent evolutionary history. We found domestication to be the most dramatic event in budding yeast evolution, raising questions on how much domestication has distorted our understanding of this key model species.","2020-02-09","Matteo De Chiara; Benjamin Barré; Karl Persson; Amadi Onyetuga Chioma; Agurtzane Irizar; Joseph Schacherer; Jonas Warringer; Gianni Liti",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.939314"],["xetzg7gp","17307ab6eda64e1538da204c02114dd36a9b7fd6","biorxiv","The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2","10.1101/2020.02.08.939553",null,null,"biorxiv","The outbreak of the 2019 Novel Coronavirus (SARS-CoV-2) rapidly spread from Wuhan, China to more than 150 countries, areas or territories, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of SARS-CoV-2, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development, is lacking. In this study, we investigated the potential of the SARS-CoV-2 viral proteins to induce class I and II MHC presentation and to form linear antibody epitopes. We created an online database to broadly share the predictions as a resource for the research community. Using this resource, we showed that genetic variations in SARS-CoV-2, though still few for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus. Importantly, we discovered evidence that SARS-CoV-2, along with related coronaviruses, used mutations to evade attack from the human immune system. Overall, we present an immunological resource for SARS-CoV-2 that could promote both therapeutic development and mechanistic research.","2020-02-12","James Zhu; Jiwoong Kim; Xue Xiao; Yunguan Wang; Danni Luo; Ran Chen; Lin Xu; He Zhang; Guanghua Xiao; John Schoggins; Xiaowei Zhan; Tao Wang; Yang Xie",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.939553"],["emr0eh0i","a22ceac8d5e1cfaf2e79557d3c794458d0e1cdbe","biorxiv","Identification of a Pangolin Niche for a 2019-nCoV-like Coronavirus via an Extensive Meta-metagenomic Search","10.1101/2020.02.08.939660",null,null,"biorxiv","AbstractIn numerous instances, tracking the biological significance of a nucleic acid sequence can be augmented through the identification of environmental niches in which the sequence of interest is present. Many metagenomic datasets are now available, with deep sequencing of samples from diverse biological niches. While any individual metagenomic dataset can be readily queried using web-based tools, meta-searches through all such datasets are less accessible. In this brief communication, we demonstrate such a meta-meta-genomic approach, examining close matches to the Wuhan coronavirus 2019-nCoV in all high-throughput sequencing datasets in the NCBI Sequence Read Archive accessible with the keyword “virome”. In addition to the homology to bat coronaviruses observed in descriptions of the 2019-nCoV sequence (F. Wu et al. 2020, Nature, doi.org/10.1038/s41586-020-2008-3; P. Zhou et al. 2020, Nature, doi.org/10.1038/s41586-020-2012-7), we note a strong homology to numerous sequence reads in a metavirome dataset generated from the lungs of deceased Pangolins reported by Liu et al. (Viruses 11:11, 2019, http://doi.org/10.3390/v11110979). Our observations are relevant to discussions of the derivation of 2019-nCoV and illustrate the utility and limitations of meta-metagenomic search tools in effective and rapid characterization of potentially significant nucleic acid sequences.ImportanceMeta-metagenomic searches allow for high-speed, low-cost identification of potentially significant biological niches for sequences of interest.","2020-02-14","Lamia Wahba; Nimit Jain; Andrew Z. Fire; Massa J. Shoura; Karen L. Artiles; Matthew J. McCoy; Dae-Eun Jeong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.939660"],["mm2sp48e","b970b7a807f37678f91d4b989ffdb7d7606d754d","biorxiv","Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection","10.1101/2020.02.08.939892",null,null,"biorxiv","AbstractSince December 2019, a novel coronavirus named 2019 coronavirus (2019-nCoV) has emerged in Wuhan of China and spread to several countries worldwide within just one month. Apart from fever and respiratory complications, acute kidney injury has been observed in some patients with 2019-nCoV. In a short period of time, angiotensin converting enzyme II (ACE2), have been proposed to serve as the receptor for the entry of 2019-nCoV, which is the same for severe acute respiratory syndrome coronavirus (SARS). To investigate the possible cause of kidney damage in 2019-nCoV patients, we used both published kidney and bladder cell atlas data and an independent unpublished kidney single cell RNA-Seq data generated in-house to evaluate ACE2 gene expressions in all cell types in healthy kidneys and bladders.Our results showed the enriched expression of all subtypes of proximal tubule cells of kidney and low but detectable levels of expression in bladder epithelial cells. These results indicated the urinary system is a potential route for 2019-nCoV infection, along with the respiratory system and digestion system. Our findings suggested the kidney abnormalities of SARS and 2019-nCoV patients may be due to proximal tubule cells damage and subsequent systematic inflammatory response induced kidney injury. Beyond that, laboratory tests of viruses and related indicators in urine may be needed in some special patients of 2019-nCoV.","2020-02-18","Wei Lin; Longfei Hu; Yan Zhang; Joshua D. Ooi; Ting Meng; Peng Jin; Xiang Ding; Longkai Peng; Lei Song; Zhou Xiao; Xiang Ao; Xiangcheng Xiao; Qiaoling Zhou; Ping Xiao; Jue Fan; Yong Zhong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.08.939892"],["f87h5qh6","5c31897d01f3edc7f58a0f03fceec2373fcfdc3d","medrxiv","The effect of travel restrictions on the spread of the 2019 novel coronavirus (2019-nCoV) outbreak","10.1101/2020.02.09.20021261",null,null,"medrvix","Motivated by the rapid spread of a novel coronavirus (2019-nCoV) in Mainland China, we use a global metapopulation disease transmission model to project the impact of both domestic and international travel limitations on the national and international spread of the epidemic. The model is calibrated on the evidence of internationally imported cases before the implementation of the travel quarantine of Wuhan. By assuming a generation time of 7.5 days, the reproduction number is estimated to be 2.4 [90% CI 2.2-2.6]. The median estimate for number of cases before the travel ban implementation on January 23, 2020 is 58,956 [90% CI 40,759 - 87,471] in Wuhan and 3,491 [90% CI 1,924 - 7,360] in other locations in Mainland China. The model shows that as of January 23, most Chinese cities had already received a considerable number of infected cases, and the travel quarantine delays the overall epidemic progression by only 3 to 5 days. The travel quarantine has a more marked effect at the international scale, where we estimate the number of case importations to be reduced by 80% until the end of February. Modeling results also indicate that sustained 90% travel restrictions to and from Mainland China only modestly affect the epidemic trajectory unless combined with a 50% or higher reduction of transmission in the community.","2020-02-11","Matteo Chinazzi; Jessica T. Davis; Marco Ajelli; Corrado Gioannini; Maria Litvinova; Stefano Merler; Ana Pastore y Piontti; Luca Rossi; Kaiyuan Sun; Cécile Viboud; Xinyue Xiong; Hongjie Yu; M. Elizabeth Halloran; Ira M. Longini; Alessandro Vespignani",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.09.20021261"],["6u9q0ox9","2913d91f13fd59c698f68ba63008d8e0550c0607","medrxiv","Spatially Explicit Modeling of 2019-nCoV Epidemic Trend based on Mobile Phone Data in Mainland China","10.1101/2020.02.09.20021360",null,null,"medrvix","IAs of February 11, 2020, all prefecture-level cities in mainland China have reported confirmed cases of 2019 novel coronavirus (2019-nCoV), but the city-level epidemical dynamics is unknown. The aim of this study is to model the current dynamics of 2019-nCoV at city level and predict the trend in the next 30 days under three possible scenarios in mainland China. We developed a spatially explicit epidemic model to consider the unique characteristics of the virus transmission in individual cities. Our model considered that the rate of virus transmission among local residents is different from those with Wuhan travel history due to the self-isolation policy. We introduced a decay rate to quantify the effort of each city to gradually control the disease spreading. We used mobile phone data to obtain the number of individuals in each city who have travel history to Wuhan. This city-level model was trained using confirmed cases up to February 10, 2020 and validated by new confirmed cases on February 11, 2020. We used the trained model to predict the future dynamics up to March 12, 2020 under different scenarios: the current trend maintained, control efforts expanded, and person-to-person contact increased due to work resuming. We estimated that the total infections in mainland China would be 72172, 54348, and 149774 by March 12, 2020 under each scenario respectively. Under the current trend, all cities will show the peak point of daily new infections by February 21. This date can be advanced to February 14 with control efforts expanded or postponed to February 26 under pressure of work resuming. Except Wuhan that cannot eliminate the disease by March 12, our model predicts that 95.4%, 100%, and 75.7% cities will have no new infections by the end of February under three scenarios. The spatial pattern of our prediction could help the government allocate resources to cities that have a more serious epidemic in the next 30 days.","2020-02-11","Xiaolin Zhu; Aiyin Zhang; Shuai Xu; Pengfei Jia; Xiaoyue Tan; Jiaqi Tian; Tao Wei; Zhenxian Quan; Jiali Yu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.09.20021360"],["5dd89gnm","d2609c4d8837558f4b8f32170422ce1cd12425a6","medrxiv","Assessing the Tendency of 2019-nCoV (COVID-19) Outbreak in China","10.1101/2020.02.09.20021444",null,null,"medrvix","Since December 8, 2019, the spread of COVID-19 is increasing every day. It is particularly important to predict the trend of the epidemic for the timely adjustment of the economy and industries. We proposed a Flow-SEHIR model in this paper, based on which we further analyzed the trends of 2019-nCoV (COVID-19) in China. The results show that the basic reproductive numbers R0 of COVID-19 is 3.56 (95% CI: 2.31-4.81). The number of daily confirmed new cases reaches the inflection point on Feb. 6-10 outside Hubei. For the maximum of infected cases number, the predicted peak value in China except Hubei was estimated to be 13806 (95% CI: 11926-15845). The peak arrival time is on March 3-9. The temporal number of patients in most areas of China outside Hubei will peak from March 12 to March 15. The peak values of more than 73.5% provinces or regions in China will be controlled within 1000. According to Flow-SEHIR model and estimations from the data of evacuation of nationals from Wuhan, the peak cumulative number of patients in Hubei was estimated to be 403481 (95% CI: 143284-1166936).","2020-02-11","Qinghe Liu; Zhicheng Liu; Deqiang Li; Zefei Gao; Junkai Zhu; Junyan Yang; Qiao Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.09.20021444"],["uqgrjjwp","dcd7a1235ea74e3ef71d051103bf8a64c3c8f457","medrxiv","The lockdown of Hubei Province causing different transmission dynamics of the novel coronavirus (2019-nCoV) in Wuhan and Beijing","10.1101/2020.02.09.20021477",null,null,"medrvix","Background: After the outbreak of novel coronavirus (2019-nCoV) starting in late 2019, a number of researchers have reported the predicted the virus transmission dynamics. However, under the strict control policy the novel coronavirus does not spread naturally outside Hubei Province, and none of the prediction closes to the real situation. Methods and findings: We used the traditional SEIR model, fully estimated the effect of control measures, to predict the virus transmission in Wuhan, the capital city of Hubei Province, and Beijing. We forecast that the outbreak of 2019-nCoV would reach its peak around March 6±10 in Wuhan and March 20±16 in Beijing, respectively. The infectious population in Beijing would be much less (only 0.3%) than those in Wuhan at the peak of this transmission wave. The number of confirmed cases in cities inside Hubei Province grow exponentially, whereas those in cities outside the province increase linearly. Conclusions: The unprecedented province lockdown substantially suspends the national and global outbreak of 2019-nCoV.","2020-02-11","Xinhai Li; Xumao Zhao; Yuehua Sun",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.09.20021477"],["er3zmcz2","815745bf1b522d33fd7371cc9a6561a2a93ef87e","medrxiv","Simulating the infected population and spread trend of 2019-nCov under different policy by EIR model","10.1101/2020.02.10.20021519",null,null,"medrvix","Chinese government has taken strong measures in response to the epidemic of new coronavirus (2019-nCoV) from Jan.23, 2020. The number of confirmed infected individuals are still increasing rapidly. Estimating the accurate infected population and the future trend of epidemic spreading under control measures is significant and urgent. There have been reports external icon of spread from an infected patient with no symptoms to a close contact, which means the incubation individuals may has the possibility of infectiousness. However, the traditional transmission model, Susceptible-Exposed-Infectious-Recovered (SEIR) model, assumes that the exposed individual is being infected but without infectiousness. Thus, the estimating infected populations based on SEIR model from the existing literatures seems too far more than the official reported data. Here, we inferred that the epidemic could be spread by exposed (incubation) individuals. Then, we provide a new Exposed-identified-Recovered (EIR) model, and simulated the epidemic spreading processes from free propagation phase to extremely control phase. Then, we estimate of the size of the epidemic and forecast the future development of the epidemics under strong prevention interventions. According to the spread characters of 2019-nCov, we construct a novel EIR compartment system dynamics model. This model integrates two phases of the epidemic spreading: before intervention and after intervention. We assume that 2019-nCov is firstly spread without intervention then the government started to take strong quarantine measures. Use the latest reported official data, we estimate the basic parameters of the model and the basic reproduction number of 2019-nCov. Then, based on this model, we simulate the future spread of the epidemics. Both the infected population and the spreading trend of 2019-nCov under different prevention policy scenarios are estimated. The epidemic spreading trends under different quarantine rate and action starting date of prevention policy are simulated and compared.","2020-02-12","Hao Xiong; Huili Yan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.20021519"],["pd70i3d8","48656efc59191537073975938f25f201524971af","medrxiv","Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage","10.1101/2020.02.10.20021584",null,null,"medrvix","Background: Severe ill patients with 2019 novel coronavirus (2019-nCoV) infection progressed rapidly to acute respiratory failure. We aimed to select the most useful prognostic factor for severe illness incidence. Methods: The study prospectively included 61 patients with 2019-nCoV infection treated at Beijing Ditan Hospital from January 13, 2020 to January 31, 2020. Prognostic factor of severe illness was selected by the LASSO COX regression analyses, to predict the severe illness probability of 2019-CoV pneumonia. The predictive accuracy was evaluated by concordance index, calibration curve, decision curve and clinical impact curve. Results: The neutrophil-to-lymphocyte ratio (NLR) was identified as the independent risk factor for severe illness in patients with 2019-nCoV infection. The NLR had a c-index of 0.807 (95% confidence interval, 0.676-0.38), the calibration curves fitted well, and the decision curve and clinical impact curve showed that the NLR had superior standardized net benefit. In addition, the incidence of severe illness was 9.1% in age ≥ 50 and NLR < 3.13 patients, and half of patients with age ≥ 50 and NLR ≥ 3.13 would develop severe illness. Based on the risk stratification of NLR with age, the study developed a 2019-nCoV pneumonia management process. Conclusions: The NLR was the early identification of risk factors for 2019-nCoV severe illness. Patients with age ≥ 50 and NLR ≥ 3.13 facilitated severe illness, and they should rapidly access to intensive care unit if necessary.","2020-02-12","Jingyuan Liu; Yao Liu; Pan Xiang; Lin Pu; Haofeng Xiong; Chuansheng Li; Ming Zhang; Jianbo Tan; Yanli Xu; Rui Song; Meihua Song; Lin Wang; Wei Zhang; Bing Han; Li Yang; Xiaojing Wang; Guiqin Zhou; Ting Zhang; Ben Li; Yanbin Wang; Zhihai Chen; Xianbo Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.20021584"],["172tl3fb",null,"medrxiv","Epidemiological and clinical features of the 2019 novel coronavirus outbreak in China","10.1101/2020.02.10.20021675",null,null,"medrvix","Our manuscript was based on surveillance cases of COVID-19 identified before January 26, 2020. As of February 20, 2020, the total number of confirmed cases in mainland China has reached 18 times of the number in our manuscript. While the methods and the main conclusions in our original analyses remain solid, we decided to withdraw this preprint for the time being, and will replace it with a more up-to-date version shortly. Should you have any comments or suggestions, please feel free to contact the corresponding author.","2020-02-11","Yang Yang; Qingbin Lu; Mingjin Liu; Yixing Wang; Anran Zhang; Neda Jalali; Natalie Dean; Ira Longini; M. Elizabeth Halloran; Bo Xu; Xiaoai Zhang; Liping Wang; Wei Liu; Liqun Fang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.20021675"],["zbqbogwu","06c1b3535b83251cf92c01258b5048beeab7a460","medrxiv","Beyond R0: the importance of contact tracing when predicting epidemics","10.1101/2020.02.10.20021725",null,null,"medrvix","The basic reproductive number --- R0 --- is one of the most common and most commonly misapplied numbers in public health. Nevertheless, estimating R0 for every transmissible pathogen, emerging or endemic, remains a priority for epidemiologists the world over. Although often used to compare outbreaks and forecast pandemic risk, this single number belies the complexity that two different pathogens can exhibit, even when they have the same R0. Here, we show how predicting outbreak size requires both an estimate of R0 and an estimate of the heterogeneity in the number of secondary infections. To facilitate rapid determination of outbreak risk, we propose a reformulation of a classic result from random network theory that relies on contact tracing data to simultaneously determine the first moment (R0) and the higher moments (representing the heterogeneity) in the distribution of secondary infections. Further, we show how this framework is robust in the face of the typically limited amount of data for emerging pathogens. Lastly, we demonstrate that without data on the heterogeneity in secondary infections for emerging pathogens like 2019-nCoV, the uncertainty in outbreak size ranges dramatically, in the case of 2019-nCoV from 5-40% of susceptible individuals. Taken together, our work highlights the critical need for contact tracing during emerging infectious disease outbreaks and the need to look beyond R0 when predicting epidemic size.","2020-02-12","Laurent Hébert-Dufresne; Benjamin M. Althouse; Samuel V. Scarpino; Antoine Allard",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.20021725"],["5td4p2gd",null,"medrxiv","Statistical Inference for Coronavirus Infected Patients in Wuhan","10.1101/2020.02.10.20021774",null,null,"medrvix","Importance: The new coronavirus outbreak has seriously affected the quality of life in China. Wuhan is the disaster area, where the number of cases has increased rapidly. However, the current measures of infected patients in Wuhan are still underestimated. Objective: To estimate the overall infected patients in Wuhan from several sampled data. The correct estimated infected patients can be helpful for the government to arrange the needed beds in hospital wards to meet the actual needs. Design: We proposed to use the sampling survey to estimate the overall infected patients in Wuhan. The sampling survey is a kind of non-comprehensive survey. It selected some units from all the survey objects to carry out the survey and made the estimation and inference to all the survey objects. Sampling surveys can obtain information that reflects the overall situation, although it is not a comprehensive survey. Setting: We estimated the overall infection rate in Wenzhou city, which has a better data collection system. Simultaneously, another different samples of Wuhan tourists to Singapore will be used to validate the infection rate in Wenzhou city. Combined these two samples, we give the estimation of the number of infected patients in Wuhan and other prefecture-level cities in Hubei Province. Participants: The number of people who returned from Wuhan to Wenzhou was selected from the daily notification of the pneumonia epidemic caused by a new coronavirus infection in the city. Exposures for observational studies: The daily rate of the pneumonia epidemic caused by the new coronavirus infection in Wenzhou City. The numerator is the number of people diagnosed and whether each person diagnosed had a history of living in Wuhan. The denominator is the total number of people returning to Wenzhou from Wuhan. Based on this rate, it is reasonable to predict the number of the infected patients.","2020-02-12","Yongdao Zhou; Jianghu Dong",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.20021774"],["o7pz0rst","eec5b4ce2de8d9d00be664e6af79cac476991bbf","medrxiv","Distribution of the 2019-nCoV Epidemic and Correlation with Population Emigration from Wuhan, China","10.1101/2020.02.10.20021824",null,null,"medrvix","BACKGROUNDS: The ongoing new coronavirus (2019-nCoV) pneumonia outbreak is spreading in China and has not reached its peak. Five millions of people had emigrated from Wuhan before the city lockdown, which potentially represent a source of virus spreaders. Case distribution and its correlation with population emigration from Wuhan in early epidemic are of great importance for early warning and prevention of future outbreak. METHODS: The officially reported cases of 2019-nCoV pneumonia were collected as of January 30, 2020. Time and location information of these cases were extracted analyzed with ArcGIS and WinBUGS. Population migration data of Wuhan City and Hubei province were extracted from Baidu Qianxi and analyzed for their correlation with case number. FINDINGS: The 2019-nCoV pneumonia cases were predominantly distributed in Hubei and other provinces of South China. Hot spot provinces included Sichuan and Yunnan provinces that are adjacent to Hubei. While Wuhan city has the highest number of cases, the time risk is relatively stable. Numbers of cases in some cities are relatively low, but the time risks are continuously rising. The case numbers of different provinces and cities of Hubei province were highly correlated with the emigrated populations from Wuhan. Lockdown of 19 cities of Hubei province, and implementation of nationwide control measures efficiently prevented the exponential growth of case number. INTERPRETATION: Population emigrated from Wuhan was the main infection source for other cities and provinces. Some cities with low case number but were in rapid increase. Due to the upcoming Spring Festival return transport wave, understanding of the trends of risks in different regions is of great significance for preparedness for both individuals and institutions.","2020-02-12","Zeliang Chen; Qi Zhang; Yi Lu; Xi Zhang; Wenjun Zhang; Cheng Guo; Conghui Liao; Qianlin Li; Xiaohu Han; Jiahai Lu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.20021824"],["qng05jry","15ad534bb2ec7bc4b0ee83ddac11a1b837d743c6","medrxiv","Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)","10.1101/2020.02.10.20021832",null,null,"medrvix","Background: To explore the cellular immunity and cytokines status of NCP patients and to predict the correlation between the cellular immunity levels, cytokines and the severity of patients. Methods: 123 NCP patients were divided into mild and severe groups. Peripheral blood was collected, lymphocyte subsets and cytokines were detected. Correlation analysis was performed on the lymphocyte subsets and cytokines, and the differences between the indexes of the two groups were analyzed. Results: 102 mild and 21 severe patients were included. Lymphocyte subsets were reduced in two groups. The proportion of CD8 + T reduction in the mild and severe group was 28.43% and 61.9%, respectively; The proportion of B cell reduction was 25.49% and 28.57%; The proportion of NK cell reduction was 34.31% and 47.62%; The detection value of IL-6 was 0 in 55.88% of the mild group, mild group has a significantly lower proportion of patients with IL-6 higher than normal than severe group; There was no significant linear correlation between the lymphocyte subsets and cytokines, while significant differences were noticed between the two groups in CD4 + T, CD8 + T, IL-6 and IL-10. Conclusions: Low levels of CD4+T and CD8+T are common in severe NCP. IL-6 and IL-10 levels were higher in severe patients. T cell subsets and cytokines can be used as one of the basis for predicting the transition from mild to severe. Large number of samples are still needed to confirm the \"warning value\" of CD4 + T, CD8 + T IL-6 and IL-10.","2020-02-12","Suxin Wan; Qingjie Yi; Shibing Fan; Jinglong Lv; Xianxiang Zhang; Lian Guo; Chunhui Lang; Qing Xiao; Kaihu Xiao; Zhengjun Yi; Mao Qiang; Jianglin Xiang; Bangshuo Zhang; Yongping Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.20021832"],["7n8p9okf","83323659de2f2d5ed3ca6ada7cf137a8a17bb014","biorxiv","Alpha-ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication","10.1101/2020.02.10.936898",null,null,"biorxiv","ABSTRACTThe main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate. Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs. In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases. Six crystal structures of protease:inhibitor complexes were determined as part of this study. Compounds synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic. Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera. The best near-equipotent inhibitors, 11u (P2 = cyclopentylmethyl) and 11r (P2 = cyclohexylmethyl), display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures. In Huh7 cells, 11r exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.","2020-02-10","Linlin Zhang; Daizong Lin; Yuri Kusov; Yong Nian; Qingjun Ma; Jiang Wang; Albrecht von Brunn; Pieter Leyssen; Kristina Lanko; Johan Neyts; Adriaan de Wilde; Eric J. Snijder; Hong Liu; Rolf Hilgenfeld",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.936898"],["z3x0c37y","7f8c673d136b9bc0aeda55e49cc72aacaa5688de","biorxiv","Structural genomics and interactomics of 2019 Wuhan novel coronavirus, 2019-nCoV, indicate evolutionary conserved functional regions of viral proteins","10.1101/2020.02.10.942136",null,null,"biorxiv","AbstractDuring its first month, the recently emerged 2019 Wuhan novel coronavirus (2019-nCoV) has already infected many thousands of people in mainland China and worldwide and took hundreds of lives. However, the swiftly spreading virus also caused an unprecedentedly rapid response from the research community facing the unknown health challenge of potentially enormous proportions. Unfortunately, the experimental research to understand the molecular mechanisms behind the viral infection and to design a vaccine or antivirals is costly and takes months to develop. To expedite the advancement of our knowledge we leverage the data about the related coronaviruses that is readily available in public databases, and integrate these data into a single computational pipeline. As a result, we provide a comprehensive structural genomics and interactomics road-maps of 2019-nCoV and use these information to infer the possible functional differences and similarities with the related SARS coronavirus. All data are made publicly available to the research community at http://korkinlab.org/wuhan","2020-02-14","Hongzhu Cui; Ziyang Gao; Ming Liu; Senbao Lu; Sun Mo; Winnie Mkandawire; Oleksandr Narykov; Suhas Srinivasan; Dmitry Korkin",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.942136"],["xa6kwguo","7ab9f9fcea519ebce527c3ede8091beedbb26ad9","biorxiv","Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses","10.1101/2020.02.10.942185",null,null,"biorxiv","AbstractThe 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be proteolytically-sensitive. We suggest this loop confers fusion activation and entry properties more in line with MERS-CoV and other coronaviruses, and that the presence of this structural loop in 2019-nCoV may affect virus stability and transmission.","2020-02-18","Javier A. Jaimes; Nicole M. André; Jean K. Millet; Gary R. Whittaker",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.942185"],["bgdcm1i1","8c210df7072bc88fa7d2da72b25c84430c16f5e8","biorxiv","Recombination and lineage-specific mutations led to the emergence of SARS-CoV-2","10.1101/2020.02.10.942748",null,null,"biorxiv","The recent outbreak of a new coronavirus (SARS-CoV-2) in Wuhan, China, underscores the need for understanding the evolutionary processes that drive the emergence and adaptation of zoonotic viruses in humans. Here, we show that recombination in betacoronaviruses, including human-infecting viruses like SARS-CoV and MERS-CoV, frequently encompasses the Receptor Binding Domain (RBD) in the Spike gene. We find that this common process likely led to a recombination event at least 11 years ago in an ancestor of the SARS-CoV-2 involving the RBD. As a result of this recombination event, SARS-CoV and SARS-CoV-2 share a similar genotype in RBD, including two insertions (positions 432-436 and 460-472), and alleles 427N and 436Y. Both 427N and 436Y belong to a helix that interacts with the human ACE2 receptor. Ancestral state analyses revealed that SARS-CoV-2 differentiated from its most recent common ancestor with RaTG13 by accumulating a significant number of amino acid changes in the RBD. In sum, we propose a two-hit scenario in the emergence of the SARS-CoV-2 virus whereby the SARS-CoV-2 ancestors in bats first acquired genetic characteristics of SARS-CoV by incorporation of a SARS-like RBD through recombination before 2009, and subsequently, the lineage that led to SARS-CoV-2 accumulated further, unique changes specifically in the RBD.","2020-02-18","Juan Angel Patino-Galindo; Ioan Filip; Mohammed AlQuraishi; Raul Rabadan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.942748"],["mpb4fy16","35ac99bfeb665ce0a2ae361dba1161e69eb8b7d5","biorxiv","ΦX174 Attenuation by Whole Genome Codon Deoptimization","10.1101/2020.02.10.942847",null,null,"biorxiv","AbstractNatural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its E. coli host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence.","2020-02-11","James T. Van Leuven; Martina M. Ederer; Katelyn Burleigh; LuAnn Scott; Randall A. Hughes; Vlad Codrea; Andrew D. Ellington; Holly Wichman; Craig Miller",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.10.942847"],["6r6zwfoy","2cfe7a4cc1fceed0ad7ffbd6244a61e0b2c3ac8b","medrxiv","Facemask shortage and the coronavirus disease (COVID-19) outbreak: Reflection on public health measures","10.1101/2020.02.11.20020735",null,null,"medrvix","To the best of our knowledge, this is the first study to investigate the facemask shortage during the novel coronavirus pneumonia (COVID-19) outbreak in China. We have summarized in detail the management strategies implemented by the Chinese governments during the outbreaks. By considering three scenarios for the outbreak development, we simulated the facemasks availability from late-December 2019 to late-April 2020 and estimated the duration of sufficient facemask supplies. Our findings showed that if the COVID-19 outbreak occurred only in Wuhan city or Hubei province, facemask shortage would not appear with the existing public health measures. However, if the outbreak occurred in the whole of China, a shortage of facemask could be substantial assuming no alternative public health measures. Supplies of facemasks in the whole of China would have been sufficient for both healthcare workers and the general population if the COVID-19 outbreak only occurred in Wuhan city or Hubei province. However, if the outbreak occurred in the whole of China, facemask supplies in China could last for 5 days if under the existing public health measures and a shortage of 853 million facemasks is expected by 30 Apr 2020. Assuming a gradually decreased import volume, we estimated that dramatic increase in productivity (42.7 times of the usual level) is needed to mitigate the facemask crisis by the end of April. In light of the COVID-19 outbreak in China, a shortage of facemasks and other medical resources can considerably compromise the efficacy of public health measures. Effective public health measures should also consider the adequacy and affordability of medical resources. Global collaboration should be strengthened to prevent the development of a global pandemic from a regional epidemic via easing the medical resources crisis in the affected countries.","2020-02-12","Huailiang Wu; Jian Huang; Casper JP Zhang; Zonglin He; Wai-kit Ming",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20020735"],["5xki1ulf","8b88901d0bb033af3a18ebcdf7f2aa7110b21504","medrxiv","Evaluating the accuracy of different respiratory specimens in the laboratory diagnosis and monitoring the viral shedding of 2019-nCoV infections","10.1101/2020.02.11.20021493",null,null,"medrvix","Background: The outbreak of novel coronavirus pneumonia (NCP) caused by 2019-nCoV spread rapidly, and elucidation the diagnostic accuracy of different respiratory specimens is crucial for the control and treatment of this diseases. Methods: Respiratory samples including nasal swabs, throat swabs, sputum and bronchoalveolar lavage fluid (BALF) were collected from Guangdong CDC confirmed NCP patients, and viral RNAs were detected using a CFDA approved detection kit. Results were analyzed in combination with sample collection date and clinical information. Finding: Except for BALF, the sputum possessed the highest positive rate (74.4%~88.9%), followed by nasal swabs (53.6%~73.3%) for both severe and mild cases during the first 14 days after illness onset (d.a.o). For samples collected ≥ 15 d.a.o, sputum and nasal swabs still possessed a high positive rate ranging from 42.9%~61.1%. The positive rate of throat swabs collected ≥ 8 d.a.o was low, especially in samples from mild cases. Viral RNAs could be detected in all the lower respiratory tract of severe cases, but not the mild cases. CT scan of cases 02, 07 and 13 showed typical viral pneumonia with ground glass opacity, while no viral RNAs were detected in first three or all the upper respiratory samples. Interpretation: Sputum is most accurate for laboratory diagnosis of NCP, followed by nasal swabs. Detection of viral RNAs in BLAF is necessary for diagnosis and monitoring of viruses in severe cases. CT scan could serve as an important make up for the diagnosis of NCP. Funding National Science and Technology Major Project, Sanming Project of Medicine and China Postdoctoral Science Foundation.","2020-02-12","Yang Yang; Minghui Yang; Chenguang Shen; Fuxiang Wang; Jing Yuan; Jinxiu Li; Mingxia Zhang; Zhaoqin Wang; Li Xing; Jinli Wei; Ling Peng; Gary Wong; Haixia Zheng; Mingfeng Liao; Kai Feng; Jianming Li; Qianting Yang; Juanjuan Zhao; Zheng Zhang; Lei Liu; Yingxia Liu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20021493"],["w5t1ni4l",null,"medrxiv","Ophthalmologic evidence against the interpersonal transmission of 2019 novel coronavirus through conjunctiva","10.1101/2020.02.11.20021956",null,null,"medrvix","Background: The emerging 2019 novel coronavirus (2019−nCoV) has pushed several countries into state of emergency all over the world. The possible transmission of 2019−nCoV by conjunctiva is controversial and has substantial public health implications. Methods: A retrospective cohort study was initiated to investigate the possible transmission of 2019−nCoV through aerosol contact with conjunctiva. We enrolled 67 cases of confirmed or suspected cases of novel coronavirus pneumonia (NCP) during 17−28 Jan 2020. Nasopharyngeal and conjunctival swabs were collected for real time RT-PCR analysis to detect 2019−nCoV. Results: 63 patients were identified as laboratory−confirmed NCP and the remaining four were suspected NCP. Conjunctival swab samples from one NCP patient yielded positive PCR results and two NCP patients yielded probable positive PCR results. None of the three patients had ocular symptoms. The only one NCP patient with conjunctivitis as the first symptom had negative conjunctival sac 2019−nCoV test. Conjunctival swab samples from the four suspected cases of NCIP were negative. Conclusion: 2019−nCoV can be detected in the conjunctival sac of patients with NCP. Through clinical analysis, viral transmission via the conjunctival route was not supported by the data. Good clinical protection can effectively cut off the transmission path.","2020-02-12","Yunyun Zhou; Yuyang Zeng; Yongqing Tong; ChangZheng Chen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20021956"],["w4zad1zt",null,"medrxiv","Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China","10.1101/2020.02.11.20022053",null,null,"medrvix","An increasing number of cases of novel coronavirus pneumonia (NCP) infected with 2019-nCoV have been identified in Wuhan and other cities in China, since December 2019. We analyzed data on the 17 confirmed cases in Dazhou to provide the epidemiologic characteristics of NCP outside Wuhan. Among them, 12 patients were still quarantined in the hospital, 5 patients were discharged NCP patients according to the national standards. Compared with non-discharged NCP patients, the discharged NCP patients had younger ages. Moreover, discharged NCP patients had higher heart rate, lymphocytes levels and monocytes levels than non-discharged NCP patients on admission to the hospital. Notably, all of 17 patients had abnormal increased C-reactive protein levels, and 16 patients had abnormal computed tomography images. This study provided some information that younger age, higher lymphocytes levels and monocytes levels at the diagnoses of 2019-nCoV may contributed to faster recovery and better therapeutic outcome.","2020-02-12","Jie Li; Shilin Li; Yurui Cai; Qin Liu; Xue Li; Zhaoping Zeng; Yanpeng Chu; Fangcheng Zhu; Fanxin Zeng",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20022053"],["1f5hn3ig",null,"medrxiv","Primary Care Practitioners' Response to 2019 Novel Coronavirus Outbreak in China","10.1101/2020.02.11.20022095",null,null,"medrvix","None","2020-02-12","Zhijie Xu; Yi Qian; Lizheng Fang; Mi Yao",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20022095"],["qbemfymp","004f0f8bb66cf446678dc13cf2701feec4f36d76","medrxiv","Healthcare-resource-adjusted vulnerabilities towards the 2019-nCoV epidemic across China","10.1101/2020.02.11.20022111",null,null,"medrvix","We integrate the human movement and healthcare resource data to identify cities with high vulnerability towards the 2019-nCoV epidemic with respect to available health resources. The results inform public health responses in multiple ways.","2020-02-12","Hanchu Zhou; Jianan Yang; Kaichen Tang; Qingpeng Zhang; zhidong cao; Dirk Pfeiffer; Daniel Dajun Zeng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20022111"],["ssdx8xwi","8bcbb998e7067bab377471601a3e72e50c2fe24f","medrxiv","Data-Based Analysis, Modelling and Forecasting of the COVID-19 outbreak","10.1101/2020.02.11.20022186",null,null,"medrvix","Since the first suspected case of coronavirus disease-2019 (COVID-19) on December 1st, 2019, in Wuhan, Hubei Province, China, a total of 40,235 confirmed cases and 909 deaths have been reported in China up to February 10, 2020, evoking fear locally and internationally. Here, based on the publicly available epidemiological data for Hubei, China from January 11 to February 10, 2020, we provide estimates of the main epidemiological parameters. In particular, we provide an estimation of the case fatality and case recovery ratios, along with their 90% confidence intervals as the outbreak evolves. On the basis of a Susceptible-Infected-Recovered-Dead (SIDR) model, we provide estimations of the basic reproduction number (R0), and the per day infection mortality and recovery rates. By calibrating the parameters of the SIRD model to the reported data, we also attempt to forecast the evolution of the of the outbreak at the epicenter three weeks ahead, i.e. until February 29. As the number of infected individuals, especially of those with asymptomatic or mild courses, is suspected to be much higher than the official numbers, which can be considered only as a subset of the actual numbers of infected and recovered cases in the total population, we have repeated the calculations under a second scenario that considers twenty times the number of confirmed infected cases and forty times the number of recovered, leaving the number of deaths unchanged. Based on the reported data, the expected value of R0 as computed considering the period from the 11th of January until the 18th of January, using the official counts of confirmed cases was found to be ~4.6, while the one computed under the second scenario was found to be ~3.2. Thus, based on the SIRD simulations, the estimated average value of R0 was found to be ~2.6 based on confirmed cases and ~2 based on the second scenario. Our forecasting flashes a note of caution for the presently unfolding outbreak in China. Based on the official counts for confirmed cases, the simulations suggest that the cumulative number of infected could reach 180,000 (with lower bound of 45,000) by February 29. Regarding the number of deaths, simulations forecast that on the basis of the up to the 10th of February reported data, the death toll might exceed 2,700 (as a lower bound) by February 29. Our analysis further reveals a significant decline of the case fatality ratio from January 26 to which various factors may have contributed, such as the severe control measures taken in Hubei, China (e.g. quarantine and hospitalization of infected individuals), but mainly because of the fact that the actual cumulative numbers of infected and recovered cases in the population most likely are much higher than the reported ones. Thus, in a scenario where we have taken twenty times the confirmed number of infected and forty times the confirmed number of recovered cases, the case fatality ratio is around 0.15% in the total population. Importantly, based on this scenario, simulations suggest a slow down of the outbreak in Hubei at the end of February.","2020-02-13","Cleo Anastassopoulou; Lucia Russo; Athanasios Tsakris; Constantinos Siettos",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20022186"],["dbwfn27p","fbc41a8e025cd6eb2d7c156aad3b6923af2349c4","medrxiv","Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCoV, in the nasal tissue","10.1101/2020.02.11.20022228",null,null,"medrvix","A novel coronavirus (2019-nCoV) was first identified in Wuhan, Hubei Province, and then spreads to the other Provinces of China. WHO decides to determine a Public Health Emergency of International Concern (PHEIC) of 2019-nCoV. 2019-nCov was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-Cov. Here based on the public single-cell RNA-Seq datasets, we analyzed the ACE2 RNA expression profile in the tissues at different locations of the respiratory tract. The result indicates that the ACE2 expression appears in nasal epithelial cells. We found that the size of this population of ACE2-expressing nasal epithelial cells is comparable with the size of the population of ACE2-expression type II alveolar cells (AT2) in the Asian sample reported by Yu Zhao et al. We further detected 2019-nCoV by polymerase chain reaction (PCR) from the nasal-swab and throat-swab of seven suspected cases. We found that 2019-nCoV tends to have a higher concentration in the nasal-swab comparing to the throat-swab, which could attribute to the ACE2-expressing nasal epithelial cells. We hope this study could be informative for virus-prevention strategy development, especially the treatment of nasal mucus.","2020-02-13","CHAO WU; Shufa Zheng; Yu Chen; Min Zheng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20022228"],["tzdgc3dw","a726e1844e86c7c17b8b4461dd4c29ad3094c792","medrxiv","Lockdown may partially halt the spread of 2019 novel coronavirus in Hubei province, China","10.1101/2020.02.11.20022236",null,null,"medrvix","We present a timely evaluation of the impact of lockdown on the 2019-nCov epidemic in Hubei province, China. The implementation appears to be effective in reducing about 60% of new infections and deaths, and its effect also appears to be sustainable even after its removal. Delaying its implementation reduces its effectiveness. However, the direct economic cost of such a lockdown remains to be seen and whether the model is replicable in other Chinese regions remains a matter of further investigation.","2020-02-13","Mingwang Shen; Zhihang Peng; Yuming Guo; Yanni Xiao; Lei Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.20022236"],["prqeufwg","fc15f433048e4d9464964930c36a1e7057ba1088","biorxiv","Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation","10.1101/2020.02.11.944462",null,null,"biorxiv","AbstractThe outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 Å-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis.","2020-02-15","Daniel Wrapp; Nianshuang Wang; Kizzmekia S. Corbett; Jory A. Goldsmith; Ching-Lin Hsieh; Olubukola Abiona; Barney S. Graham; Jason S. McLellan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.944462"],["cv36vc8i","4d1090e239baa1ea54af521f4b8bd8b5ab51fd35","biorxiv","A Generalized Discrete Dynamic Model for Human Epidemics","10.1101/2020.02.11.944728",null,null,"biorxiv","AbstractA discrete dynamic model for human epidemics was developed in present study. The model included major parameters as transmission strength and its decline parameters, mean incubation period, hospitalization time, non-hospitalization daily mortality, non-hospitalization daily recovery rate, and hospitalization proportion, etc. Sensitivity analysis of the model indicated the total cumulative cases significantly increased with initial transmission strength, hospitalization time. The total cumulative cases significantly decreased with transmission strength’s decline and hospitalization proportion, and linearly decreased with non-hospitalization daily mortality and non-hospitalization daily recovery rate. In a certain range, the total cumulative cases significantly increased with mean incubation period. Sensitivity analysis demonstrated that dynamic change of transmission strength is one of the most important and controllable factors. In addition, reducing the delay for hospitalization is much effective in weakening disease epidemic. Non-hospitalization recovery rate is of importance for enhancing immunity to recover from the disease.","2020-02-12","Wenjun Zhang; Zeliang Chen; Yi Lu; Zhongmin Guo; Yanhong Qi; Guoling Wang; Jiahai Lu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.11.944728"],["7t41wvw3","d3c2e2839498c613ee95739dce7052109750362c","medrxiv","Long-Term Persistence of IgG Antibodies in SARS-CoV Infected Healthcare Workers","10.1101/2020.02.12.20021386",null,null,"medrvix","BACKGROUND: The ongoing worldwide outbreak of the 2019-nCoV is markedly similar to the severe acute respiratory syndrome (SARS) outbreak 17 years ago. During the 2002-2003 SARS outbreak, healthcare workers formed a special population of patients. Although virus-specific IgG play important roles in virus neutralization and prevention against future infection, limited information is available regarding the long term persistence of IgG after infection with SARS-like coronavirus. METHODS: A long-term prospective cohort study followed 34 SARS-CoV-infected healthcare workers from a hospital with clustered infected cases during the 2002-2003 SARS outbreak in Guangzhou, China, with a 13-year follow-up. Serum samples were collected annually from 2003-2015. Twenty SARS-CoV-infected and 40 non-infected healthcare workers were enrolled in 2015, and their serum samples were collected. All sera were tested for IgG antibodies with ELISA using whole virus and a recombinant nucleocapsid protein of SARS-CoV, as a diagnostic antigen. RESULTS: Anti SARS-CoV IgG was found to persist for up to 12 years. IgG titers typically peaked in 2004, declining rapidly from 2004-2006, and then continued to decline at a slower rate. IgG titers in SARS-CoV-infected healthcare workers remained at a significantly high level until 2015. Patients treated with corticosteroids at the time of infection were found to have lower IgG titers than those without. CONCLUSIONS: IgG antibodies against SARS-CoV can persist for at least 12 years. The presence of SARS-CoV IgG might provide protection against SARS-CoV and other betacoronavirus. This study provides valuable information regarding humoral immune responses against SARS-CoV and the 2019-nCoV.","2020-02-14","Xiaoqin Guo; Zhongmin Guo; Chaohui Duan; Zeliang chen; Guoling Wang; Yi Lu; Mengfeng Li; Jiahai Lu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20021386"],["bhm2un6v","d34a47b592e9c6e84b5b4115474349acbc0fb925","medrxiv","Statistics based predictions of coronavirus 2019-nCoV spreading in mainland China","10.1101/2020.02.12.20021931",null,null,"medrvix","Background. The epidemic outbreak cased by coronavirus 2019-nCoV is of great interest to researches because of the high rate of spread of the infection and the significant number of fatalities. A detailed scientific analysis of the phenomenon is yet to come, but the public is already interested in the questions of the duration of the epidemic, the expected number of patients and deaths. For long time predictions, the complicated mathematical models are necessary which need many efforts for unknown parameters identification and calculations. In this article, some preliminary estimates will be presented. Objective. Since the reliable long time data are available only for mainland China, we will try to predict the epidemic characteristics only in this area. We will estimate some of the epidemic characteristics and present the most reliable dependences for victim numbers, infected and removed persons versus time. Methods. In this study we use the known SIR model for the dynamics of an epidemic, the known exact solution of the linear equations and statistical approach developed before for investigation of the children disease, which occurred in Chernivtsi (Ukraine) in 1988-1989. Results. The optimal values of the SIR model parameters were identified with the use of statistical approach. The numbers of infected, susceptible and removed persons versus time were predicted. Conclusions. Simple mathematical model was used to predict the characteristics of the epidemic caused by coronavirus 2019-nCoV in mainland China. The further research should focus on updating the predictions with the use of fresh data and using more complicated mathematical models.","2020-02-13","Igor Nesteruk",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20021931"],["lvjveaob","5fb2cce255dd56b944ae52cec664bd7f4f222c66","medrxiv","Estimating the daily trend in the size of the COVID-19 infected population in Wuhan","10.1101/2020.02.12.20022277",null,null,"medrvix","There has been an outbreak of coronavirus disease (COVID-19) in Wuhan city, Hubei province, China since December 2019. Cases have been exported to other parts of China and more than 20 countries. We provide estimates of the daily trend in the size of the epidemic in Wuhan based on detailed information of 10,940 confirmed cases outside Hubei province.","2020-02-13","Qiushi Lin; Taojun Hu; Xiao-Hua Zhou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022277"],["j60j7e9b","5585d267f9b505b068f9f72bff5f12c248472f0a","medrxiv","Epidemic size of novel coronavirus-infected pneumonia in the Epicenter Wuhan: using data of five-countries' evacuation action","10.1101/2020.02.12.20022285",null,null,"medrvix","Background: Since late December 2019, novel coronavirus-infected pneumonia (NCP) emerged in Wuhan, Hubei province, China. Meanwhile, NCP rapidly spread from China to other countries, and several countries' government rush to evacuate their citizens from Wuhan. We analyzed the infection rate of the evacuees and extrapolated the results in Wuhan's NCP incidence estimation. Methods: We collected the total number and confirmed cases of 2019-nCov infection in the evacuation of Korea, Japan, Germany, Singapore, and France and estimated the infection rate of the 2019 novel coronavirus (2019-nCov) among people who were evacuated from Wuhan with a meta-analysis. NCP incidence of Wuhan was indirectly estimated based on data of evacuation. Results: From Jan 29 to Feb 2, 2020, 1916 people have been evacuated from Wuhan, among them 17 have been confirmed 2019-nCov infected. The infection rate is estimated to be 1.1% (95% CI 0.4%-3.1%) using one group meta-analysis method with random effect model. We then estimated that almost 110,000 (95% CI: 40,000-310,000) people were infected with 2019-nCov in Wuhan around Feb 2, 2020, assuming the infection risk of evacuees is close to Chinese citizens in Wuhan. Conclusions: At the beginning of the outbreak, incidence of NCP may be vastly underestimated. Our result emphasizes that 2019-nCov has proposed a huge public health threats in Wuhan. We need to respond more rapidly, take large-scale public health interventions and draconian measures to limiting population mobility and control the epidemic.","2020-02-13","Hongxin Zhao; Sailimai Man; Bo Wang; Yi Ning",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022285"],["78qyuk5x",null,"medrxiv","Clinical diagnosis of 8274 samples with 2019-novel coronavirus in Wuhan","10.1101/2020.02.12.20022327",null,null,"medrvix","Background: 2019-Novel coronavirus (2019-nCoV) outbreaks create challenges for hospital laboratories because thousands of samples must be evaluated each day. Sample types, interpretation methods, and corresponding laboratory standards must be established. The possibility of other infections should be assessed to provide a basis for clinical classification, isolation, and treatment. Accordingly, in the present study, we evaluated the testing methods for 2019-nCoV and co-infections. Methods: We used a fluorescence-based quantitative PCR kit urgently distributed by the Chinese CDC to detect 8274 close contacts in the Wuhan region against two loci on the 2019-nCoV genome. We also analyzed 613 patients with fever who underwent multiple tests for 13 respiratory pathogens; 316 subjects were also tested for 2019-nCoV. Findings: Among the 8274 subjects, 2745 (33.2%) had 2019-nCoV infection; 5277 (63.8%) subjects showed negative results in the 2019-nCoV nucleic acid test (non-019-nCoV); and 252 cases (3.0%) because only one target was positive, the diagnosis was not definitive. Sixteen patients who originally had only one positive target were re-examined a few days later; 14 patients (87.5%) were finally defined as 2019-nCoV-positive, and 2 (12.5%) were finally defined as negative. The positive rates of nCoV-NP and nCovORF1ab were 34.7% and 34.7%, respectively. nCoV-NP-positive only and nCovORF1ab-positive cases accounted for 1.5% and 1.5%, respectively. In the 316 patients with multiple respiratory pathogens, 104 were positive for 2019-nCov and 6/104 had co-infection with coronavirus (3/104), influenza A virus (2/104), rhinovirus (2/104), and influenza A H3N2 (1/104); the remaining 212 patients had influenza A virus (11/202), influenza A H3N2 (11/202), rhinovirus (10/202), respiratory syncytial virus (7/202), influenza B virus (6/202), metapneumovirus (4/202), and coronavirus (2/202). Interpretation: Clinical testing methods for 2019-nCoV require improvement. Importantly, 5.8% of 2019-nCoV infected and 18.4% of non-2019-nCoV-infected patients had other pathogen infections. It is important to treat combined infections and perform rapid screening to avoid cross-contamination of patients. A test that quickly and simultaneously screens as many pathogens as possible is needed.","2020-02-13","Ming Wang; Qing Wu; Wanzhou Xu; Bin Qiao; Jingwei Wang; Hongyun Zheng; Shupeng Jiang; Junchi Mei; Zegang Wu; Yayun Deng; Fangyuan Zhou; Wei Wu; Yan Zhang; Zhihua Lv; Jingtao Huang; Xiaoqian Guo; Lina Feng; Zunen Xia; Di Li; Zhiliang Xu; Tiangang Liu; Pingan Zhang; Yongqing Tong; Yan Li",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022327"],["vv1wjen0","409c431edf54d022463f40f2d31a0dfab42f5324","medrxiv","Can Search Query Forecast successfully in China's 2019-nCov pneumonia?","10.1101/2020.02.12.20022400",null,null,"medrvix","Recently the novel coronavirus (2019-nCov) pneumonia outbreak in China then the world, and the Number of infections and death continues to increases. Search Query performs well in forecasting the epidemics. It is still a question whether search engine data can forecast the drift and the inflexion in 2019-nCov pneumonia. Based on the Baidu Search Index, we propose three prediction models: composite Index, composite Index with filtering and suspected NCP(Novel Coronavirus Pneumonia). The result demonstrates that the predictive model of composite index with filtering performs the best while the model of suspected NCP has the highest forecast error. We further predict the out-of-the-set NCP confirmed cases and monitor that the next peak of new diagnoses will occur on February 16th and 17th.","2020-02-18","Li Xiaoxuan; Wu Qi; Lv Benfu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022400"],["s5w8rf6t","c672f5b0d7c0a40f7cc1bdf079066d72eb8f64ec","medrxiv","ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection","10.1101/2020.02.12.20022418",null,null,"medrvix","In December 2019 and January 2020, novel coronavirus (2019-nCoV) - infected pneumonia (NCIP) occurred in Wuhan, and has already posed a serious threat to public health. ACE2 (Angiotensin Converting Enzyme 2) has been shown to be one of the major receptors that mediate the entry of 2019-nCoV into human cells, which also happens in severe acute respiratory syndrome coronavirus (SARS). Several researches have indicated that some patients have abnormal renal function or even kidney damage in addition to injury in respiratory system, and the related mechanism is unknown. This arouses our interest in whether coronavirus infection will affect the urinary and male reproductive systems. Here in this study, we used the online datasets to analyze ACE2 expression in different human organs. The results indicate that ACE2 highly expresses in renal tubular cells, Leydig cells and cells in seminiferous ducts in testis. Therefore, virus might directly bind to such ACE2 positive cells and damage the kidney and testicular tissue of patients. Our results indicate that renal function evaluation and special care should be performed in 2019-nCoV patients during clinical work, because of the kidney damage caused by virus and antiviral drugs with certain renal toxicity. In addition, due to the potential pathogenicity of the virus to testicular tissues, clinicians should pay attention to the risk of testicular lesions in patients during hospitalization and later clinical follow-up, especially the assessment and appropriate intervention in young patients' fertility.","2020-02-13","Caibin Fan; Kai Li; Yanhong Ding; Wei Lu Lu; Jianqing Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022418"],["y7d3ux28","11f96d05db6854ef95312aa3a4736724ce1f02d6","medrxiv","Interventions targeting air travellers early in the pandemic may delay local outbreaks of SARS-CoV-2","10.1101/2020.02.12.20022426",null,null,"medrvix","Background: We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country. Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival & traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% & 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low.","2020-02-13","Samuel J Clifford; Carl A B Pearson; Petra Klepac; Kevin Van Zandvoort; Billy J Quilty; Rosalind M Eggo; Stefan Flasche",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022426"],["kfsuiz5t",null,"medrxiv","Early epidemiological assessment of the transmission potential and virulence of coronavirus disease 2019 (COVID-19) in Wuhan City: China, January-February, 2020","10.1101/2020.02.12.20022434",null,null,"medrvix","Background: Since the first cluster of cases was identified in Wuhan City, China, in December, 2019, coronavirus disease 2019 (COVID-19) has rapidly spread across China, causing multiple introductions in 109 countries/territories/areas as of March 10th. Despite the scarcity of publicly available data, scientists around the world have made strides in estimating the magnitude of the epidemic, the basic reproduction number, and transmission patterns. Recently more evidence suggests that a substantial fraction of the infected individuals with the novel coronavirus show little if any symptoms, which suggest the need to reassess the transmission potential of this emerging disease. In this study, we derive estimates of the transmissibility and virulence of COVID-19 in Wuhan City, China, by reconstructing the underlying transmission dynamics using multiple data sources. Methods: We employ statistical methods and publicly available epidemiological datasets to jointly derive estimates of transmissibility and severity associated with the novel coronavirus. For this purpose, the daily series of laboratory-confirmed COVID-19 cases and deaths in Wuhan City and epidemiological data of Japanese evacuees from Wuhan City on board government-chartered flights were integrated into our analysis. Results: Our posterior estimates of basic reproduction number (R) in Wuhan City, China in 2019-2020 reached values as high as 5.20 (95%CrI: 5.04-5.47) and the enhanced public health intervention after January 23rd in 2020 was associated with a declined R at 0.58 (95%CrI: 0.51-0.64), with the total number of infections (i.e. cumulative infections) estimated at 1905526 (95%CrI: 1350283-2655936) in Wuhan City, raising the proportion of infected individuals to 19.1% (95%CrI: 13.5-26.6%). We also found that most recent crude infection fatality ratio (IFR) and time-delay adjusted IFR is estimated to be 0.04% (95% CrI: 0.03-0.06%) and 0.12% (95%CrI: 0.08-0.17%), which is several orders of magnitude smaller than the crude CFR estimated at 4.19% Conclusions: We have estimated key epidemiological parameters of the transmissibility and virulence of COVID-19 in Wuhan, China during January-February, 2020 using an ecological modelling approach. The power of our approach lies in the ability to infer epidemiological parameters with quantified uncertainty from partial observations collected by surveillance systems.","2020-02-13","Kenji Mizumoto; Katsushi Kagaya; Gerardo Chowell",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022434"],["zvngy7zz","90b5ecf991032f3918ad43b252e17d1171b4ea63","medrxiv","The role of absolute humidity on transmission rates of the COVID-19 outbreak","10.1101/2020.02.12.20022467",null,null,"medrvix","A novel coronavirus (COVID-19) was identified in Wuhan, Hubei Province, China, in December 2019 and has caused over 40,000 cases worldwide to date. Previous studies have supported an epidemiological hypothesis that cold and dry (low absolute humidity) environments facilitate the survival and spread of droplet-mediated viral diseases, and warm and humid (high absolute humidity) environments see attenuated viral transmission (i.e., influenza). However, the role of absolute humidity in transmission of COVID-19 has not yet been established. Here, we examine province-level variability of the basic reproductive numbers of COVID-19 across China and find that changes in weather alone (i.e., increase of temperature and humidity as spring and summer months arrive in the North Hemisphere) will not necessarily lead to declines in COVID-19 case counts without the implementation of extensive public health interventions.","2020-02-17","Wei Luo; Maimuna S Majumder; Dianbo Liu; Canelle Poirier; Kenneth D Mandl; Marc Lipsitch; Mauricio Santillana",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022467"],["x4qdiln9","05d99c07db59b6948e39bfa62c2cbbf62944059a","medrxiv","A spatial model of CoVID-19 transmission in England and Wales: early spread and peak timing","10.1101/2020.02.12.20022566",null,null,"medrvix","Background: An outbreak of a novel coronavirus, named CoVID-19, was first reported in China on 31 December 2019. As of 9 February 2020, cases have been reported in 25 countries, including probable cases of human-to-human transmission in England. Methods: We adapted an existing national-scale metapopulation model to capture the spread of CoVID-19 in England and Wales. We used 2011 census data to capture population sizes and population movement, together with parameter estimates from the current outbreak in China. Results: We predict that a CoVID-19 outbreak will peak 126 to 147 days (~4 months) after the start of person-to-person transmission in England and Wales in the absence of controls, assuming biological parameters remain unchanged. Therefore, if person-to-person transmission persists from February, we predict the epidemic peak would occur in June. The starting location has minimal impact on peak timing, and model stochasticity varies peak timing by 10 days. Incorporating realistic parameter uncertainty leads to estimates of peak time ranging from 78 days to 241 days after person-to-person transmission has been established. Seasonal changes in transmission rate substantially impact the timing and size of the epidemic peak, as well as the total attack rate. Discussion: We provide initial estimates of the potential course of CoVID-19 in England and Wales in the absence of control measures. These results can be refined with improved estimates of epidemiological parameters, and permit investigation of control measures and cost effectiveness analyses. Seasonal changes in transmission rate could shift the timing of the peak into winter months, which will have important implications for healthcare capacity planning.","2020-02-14","Leon Danon; Ellen Brooks-Pollock; Mick Bailey; Matt J Keeling",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.20022566"],["anls8gri","3579a2b0053bef55f93716e444173727a6c94c6d","biorxiv","Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus","10.1101/2020.02.12.945576",null,null,"biorxiv","Pathogenic human coronavirus infections, such as severe acute respiratory syndrome CoV (SARS-CoV) and Middle East respiratory syndrome CoV (MERS-CoV), cause high morbidity and mortality 1,2. Recently, a severe pneumonia-associated respiratory syndrome caused by a new coronavirus was reported at December 2019 (2019-nCoV) in the city Wuhan, Hubei province, China3–5, which was also named as pneumonia-associated respiratory syndrome (PARS)6. Up to 9th of February 2020, at least 37, 251 cases have been reported with 812 fatal cases according to the report from China CDC. However, the immune mechanism that potential orchestrated acute mortality from patients of 2019-nCoV is still unknown. Here we show that after the 2019-nCoV infection, CD4+T lymphocytes are rapidly activated to become pathogenic T helper (Th) 1 cells and generate GM-CSF etc. The cytokines environment induces inflammatory CD14+CD16+ monocytes with high expression of IL-6 and accelerates the inflammation. These aberrant and excessive immune cells may enter the pulmonary circulation in huge numbers and play an immune damaging role to causing lung functional disability and quick mortality. Our results demonstrate that excessive non-effective host immune responses by pathogenic T cells and inflammatory monocytes may associate with severe lung pathology. Therefore, we suggest that monoclonal antibody that targets the GM-CSF or interleukin 6 receptor may potentially curb immunopathology caused by 2019-nCoV and consequently win more time for virus clearance.","2020-02-20","Yonggang Zhou; Binqing Fu; Xiaohu Zheng; Dongsheng Wang; Changcheng Zhao; Yingjie qi; Rui Sun; Zhigang Tian; Xiaoling Xu; Haiming Wei",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.945576"],["8p1agcm2","f6f1ec12291f743de1504003929ea654217f2af6","biorxiv","Candidate targets for immune responses to 2019-Novel Coronavirus (nCoV): sequence homology- and bioinformatic-based predictions","10.1101/2020.02.12.946087",null,null,"biorxiv","ABSTRACTEffective countermeasures against the recent emergence and rapid expansion of the 2019-Novel Coronavirus (2019-nCoV) require the development of data and tools to understand and monitor viral spread and immune responses. However, little information about the targets of immune responses to 2019-nCoV is available. We used the Immune Epitope Database and Analysis Resource (IEDB) resource to catalog available data related to other coronaviruses, including SARS-CoV, which has high sequence similarity to 2019-nCoV, and is the best-characterized coronavirus in terms of epitope responses. We identified multiple specific regions in 2019-nCoV that have high homology to SARS virus. Parallel bionformatic predictions identified a priori potential B and T cell epitopes for 2019-nCoV. The independent identification of the same regions using two approaches reflects the high probability that these regions are targets for immune recognition of 2019-nCoV.ONE SENTENCE SUMMARYWe identified potential targets for immune responses to 2019-nCoV and provide essential information for understanding human immune responses to this virus and evaluation of diagnostic and vaccine candidates.","2020-02-20","Alba Grifoni; John Sidney; Yun Zhang; Richard H Scheuermann; Bjoern Peters; Alessandro Sette",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.946087"],["er6lz09f","4b56993a4d0f091e70ef5dec2e5a61c3043f752e","biorxiv","DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes","10.1101/2020.02.12.946772",null,null,"biorxiv","AbstractMotivationAccurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm.ResultsWe trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours.AvailabilityThe repository of this project will be published.Contactdongsi@uw.eduSupplementary informationSupplementary data will be available at Bioinformatics online.","2020-02-13","Jonas Pfab; Dong Si",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.12.946772"],["z8ix4kz7","38bab3f17b186bd8ee289e5d135bb7d500ef500b","medrxiv","Probabilistic reconstruction of measles transmission clusters from routinely collected surveillance data","10.1101/2020.02.13.20020891",null,null,"medrvix","Pockets of susceptibility resulting from spatial or social heterogeneity in vaccine coverage can drive measles outbreaks, as cases imported into such pockets are likely to cause further transmission and lead to large transmission clusters. Characterising the dynamics of transmission is essential for identifying which individuals and regions might be most at risk. As data from detailed contact tracing investigations are not available in many settings, we combined age, location, genotype, and onset date of cases in order to probabilistically reconstruct the importation status and transmission clusters within a newly developed R package called o2geosocial. We compared our inferred cluster size distributions to 737 transmission clusters identified through detailed contact-tracing in the United States between 2001 and 2016. We were able to reconstruct the importation status of the cases and found good agreement between the inferred and reference clusters. The results were improved when the contact-tracing investigations were used to set the importation status before running the model. Spatial heterogeneity in vaccine coverage is difficult to measure directly. Our approach was able to highlight areas with potential for local transmission using a minimal number of variables and could be applied to assess the intensity of ongoing transmission in a region.","2020-02-15","Alexis Robert; Adam J Kucharski; Paul A Gastañaduy; Prabasaj Paul; Sebastian Funk",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.20020891"],["lmoekp1m","7ed2c0786eef3d39fa0435b461aa34ab454a4753","medrxiv","Understanding the present status and forecasting of COVID―19 in Wuhan","10.1101/2020.02.13.20022251",null,null,"medrvix","The present status of COVID―19 is analyzed and the end of the disease is forecasted. The peak of the epidemic is different in three regions, Wuhan, Hubei province except Wuhan, and mainland China except Hubei. In two regions except Wuhan, the peak of the epidemic passed ten days ago. If the trend until February 11 does not change, the disease may end by the end of February. In Wuhan, the epidemic reached a peak but the reported number of newly infected patients fluctuates largely. We need to know the reason for the big fluctuation to forecast the end of the disease.","2020-02-14","Toshihisa Tomie",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.20022251"],["nqxo3f45","8b71ce92ec03e066eca7792bfd17e2f210d50ce3","medrxiv","Optimizing diagnostic strategy for novel coronavirus pneumonia, a multi-center study in Eastern China","10.1101/2020.02.13.20022673",null,null,"medrvix","COVID-19 caused by a novel coronavirus SARS-CoV-2 emerged in Wuhan, Hubei province since December 2019, and caused a rapid outbreak throughout China and globally. Cities outside Hubei are also facing great challenge and require implementing of effective and feasible strategy in precision diagnosing novel coronavirus pneumonia (NCP). We described a multicenter prospective study on diagnostic strategy of suspected NCP patients from January 22nd to February 9th, 2020 in Eastern China cities. Nasopharyngeal swabs were collected from the patients. The epidemiological characteristics, clinical symptoms, laboratory assessments, and computed tomographic (CT) scans were obtained. Pathogen screen were performed including RT-PCR, multiplex PCR, rapid flu antigen tests and mNGS. We enrolled 53 suspected NCP patients, among whom 20 were laboratory-confirmed. Fourteen (70%) and 3 (15%) patients were positive for the first and second SARS-CoV-2 RT-PCR test, respectively. All NCP patients were positive for mNGS. Chest CT images and the symptoms of early stage NCP patients were similar to other viral pneumonia patients. We identified 11 of 20 co-infections in NCP cases, including regular respiratory virus, fungi and bacteria synchronously. Genomic analysis showed that 8 of 10 cases had no mutation in virus genome, while 2 cases had only one single mutation in N gene. Our study discovered that a combination of chest CT, SARS-CoV-2 RT-PCR and multi-plex PCR is recommended in regions outside Hubei province. Co-infection of other pathogens with SARS-CoV-2 exists and should be acknowledged. Repeated sampling, change of specimen type or metagenomics sequencing could further facilitate during critical clinical cases.","2020-02-17","Jing-Wen Ai; Hao-Cheng Zhang; Teng Xu; Jing Wu; Mengqi Zhu; Yi-Qi Yu; Han-Yue Zhang; Zhongliang Shen; Yang Li; Xian Zhou; Guo-Qing Zang; Jie Xu; Wen-Jing Chen; Yong-Jun Li; De-Sheng Xie; Ming-Zhe Zhou; Jing-Ying Sun; Jia-Zhen Chen; Wen-Hong Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.20022673"],["h0h0d93t","adb3a6501ad731eda95d0a1a182a793a0dcd58b0","medrxiv","Quantifying bias of COVID-19 prevalence and severity estimates in Wuhan, China that depend on reported cases in international travelers","10.1101/2020.02.13.20022707",null,null,"medrvix","Risk of COVID-19 infection in Wuhan has been estimated using imported case counts of international travelers, often under the assumption that all cases in travelers are ascertained. Recent work indicates variation among countries in detection capacity for imported cases. Singapore has historically had very strong epidemiological surveillance and contact-tracing capacity and has shown in the COVID-19 epidemic evidence of a high sensitivity of case detection. We therefore used a Bayesian modeling approach to estimate the relative imported case detection capacity for other countries compared to that of Singapore. We estimate that the global ability to detect imported cases is 38% (95% HPDI 22% - 64%) of Singapore′s capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 - 4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore. Using the second component of the Global Health Security index to stratify likely case-detection capacities, we found that the ability to detect imported cases relative to Singapore among high surveillance locations is 40% (95% HPDI 22% - 67%), among intermediate surveillance locations it is 37% (95% HPDI 18% - 68%), and among low surveillance locations it is 11% (95% HPDI 0% - 42%). Using a simple mathematical model, we further find that treating all travelers as if they were residents (rather than accounting for the brief stay of some of these travelers in Wuhan) can modestly contribute to underestimation of prevalence as well. We conclude that estimates of case counts in Wuhan based on assumptions of perfect detection in travelers may be underestimated by several fold, and severity correspondingly overestimated by several fold. Undetected cases are likely in countries around the world, with greater risk in countries of low detection capacity and high connectivity to the epicenter of the outbreak.","2020-02-14","Rene Niehus; Pablo M De Salazar; Aimee Taylor; Marc Lipsitch",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.20022707"],["60qmiwjm",null,"medrxiv","Analysis of meteorological conditions and prediction of epidemic trend of 2019-nCoV infection in 2020","10.1101/2020.02.13.20022715",null,null,"medrvix","Objective: To investigate the meteorological condition for incidence and spread of 2019-nCoV infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease. Methods: The meteorological factors during the outbreak period of the novel coronavirus pneumonia in Wuhan in 2019 were collected and analyzed, and were confirmed with those of Severe Acute Respiratory Syndrome (SARS) in China in 2003. Data of patients infected with 2019-nCoV and SARS coronavirus were collected from WHO website and other public sources. Results: This study found that the suitable temperature range for 2019-nCoV coronavirus survival is (13-24 degree Celsius), among which 19 degree Celsius lasting about 60 days is conducive to the spread between the vector and humans; the humidity range is 50%-80%, of which about 75% humidity is conducive to the survival of the coronavirus; the suitable precipitation range is below 30 mm/ month. Cold air and continuous low temperature over one week are helpful for the elimination of the virus. The prediction results show that with the approach of spring, the temperature in north China gradually rises, and the coronavirus spreads to middle and high latitudes along the temperature line of 13-18 degree Celsius. The population of new coronavirus infections is concentrated in Beijing, Tianjin, Hebei, Jiangsu, Zhejiang, Shanghai and other urban agglomerations. Starting from May 2020, the Beijing-Tianjin-Hebei urban agglomeration, the Central China Zhengzhou-Wuhan urban agglomeration, the eastern Jiangsu-Zhejiang-Shanghai urban agglomeration, and the southern Pearl River Delta urban agglomeration are all under a high temperature above 24 degree Celsius, which is not conducive to the survival and reproduction of coronaviruses, so the epidemic is expected to end. Conclusions: A wide range of continuous warm and dry weather is conducive to the survival of 2019-nCoV. The coming of spring, in addition to the original Wuhan-Zhengzhou urban agglomeration in central China, means that the prevention and control measures in big cities located in mid-latitude should be strengthened, especially the monitoring of transportation hubs. The Pearl River Delta urban agglomeration is a concentrated area of population in south China, with a faster temperature rise than those in mid-high latitudes, and thus the prevention in this area should be prioritized. From a global perspective, cities with a mean temperature below 24 degree Celsius are all high-risk cities for 2019-nCoV transmission before June.","2020-02-18","Jin Bu; Dong-Dong Peng; Hui Xiao; Qian Yue; Yan Han; Yu Lin; Gang Hu; Jing Chen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.20022715"],["3p2dl8yf","1218f278a4f8d83dac14b23c8f698062812ef9d5","medrxiv","Potential impact of seasonal forcing on a SARS-CoV-2 pandemic","10.1101/2020.02.13.20022806",null,null,"medrvix","A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing more than 80,000 confirmed infections and 2,700 fatalities (as of Feb 27, 2020). Imported cases and transmission clusters of various sizes have been reported globally suggesting a pandemic is likely. Here, we explore how seasonal variation in transmissibility could modulate a SARS-CoV-2 pandemic. Data from routine diagnostics show a strong and consistent seasonal variation of the four endemic coronaviruses (229E, HKU1, NL63, OC43) and we parameterize our model for SARS-CoV-2 using these data. The model allows for many subpopulations of different size with variable parameters. Simulations of different scenarios show that plausible parameters result in a small peak in early 2020 in temperate regions of the Northern Hemisphere and a larger peak in winter 2020/2021. Variation in transmission and migration rates can result in substantial variation in prevalence between regions. While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. The likely aggregated effect of seasonal variation, infection control measures and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.","2020-02-17","Richard A Neher; Robert Dyrdak; Valentin Druelle; Emma B Hodcroft; Jan Albert",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.20022806"],["ibx89gqw",null,"medrxiv","Estimating the distribution of the incubation period of 2019 novel coronavirus (COVID-19) infection between travelers to Hubei, China and non-travelers","10.1101/2020.02.13.20022822",null,null,"medrvix","Objectives: Amid the continuing spread of the novel coronavirus (COVID-19), the incubation period of COVID-19 should be regularly re-assessed as more information is available upon the increase in reported cases. The present work estimated the distribution of incubation periods of patients infected in and outside Hubei province of China. Methods: Clinical data were collected from the individual cases reported by the media as they were not fully available on the official pages of the Chinese health authorities. MLE was used to estimate the distributions of the incubation period. Results: It was found that the incubation period of patients with no travel history to Hubei was longer and more volatile. Conclusion: It is recommended that the duration of quarantine should be extended to at least 3 weeks.","2020-02-18","Char Leung",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.20022822"],["hlhfx3fm","47b07518dc0f608e53198bb5f8a95caa1c164fe5","medrxiv","A simple laboratory parameter facilitates early identification of COVID-19 patients","10.1101/2020.02.13.20022830",null,null,"medrvix","The total number of COVID-19 patients since the outbreak of this infection in Wuhan, China has reached 40000 and are still growing. To facilitate triage or identification of the large number of COVID-19 patients from other patients with similar symptoms in designated fever clinics, we set to identify a practical marker that could be conveniently utilized by first-line health-care workers in clinics. To do so, we performed a case-control study by analyzing clinical and laboratory findings between PCR-confirmed SARS-CoV-2 positive patients (n=52) and SARS-CoV-2 negative patients (n=53). The patients in two cohorts all had similar symptoms, mainly fever and respiratory symptoms. The rates of patients with leukocyte counts (normal or decreased number) or lymphopenia (two parameters suggested by current National and WHO COVID-19 guidelines) had no differences between these two cohorts, while the rate of eosinopenia (decreased number of eosinophils) in SARS-CoV-2 positive patients (79%) was much higher than that in SARS-CoV-2 negative patients (36%). When the symptoms were combined with eosinopenia, this combination led to a diagnosis sensitivity and specificity of 79% and 64%, respectively, much higher than 48% and 53% when symptoms were combined with leukocyte counts (normal or decreased number) and/ or lymphopenia. Thus, our analysis reveals that eosinopenia may be a potentially more reliable laboratory predictor for SARS-CoV-2 infection than leukocyte counts and lymphopenia recommended by the current guidelines.","2020-02-17","Qilin Li; Xiuli Ding; Geqing Xia; Zhi Geng; Fenghua Chen; Lin Wang; Zheng Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.20022830"],["5f42du0b","3319392ca8bd2b8f2e2d00bccdb80deaa8a77c2e","biorxiv","Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China","10.1101/2020.02.13.945485",null,null,"biorxiv","AbstractThe ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins (Manis javanica) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission.","2020-02-18","Tommy Tsan-Yuk Lam; Marcus Ho-Hin Shum; Hua-Chen Zhu; Yi-Gang Tong; Xue-Bing Ni; Yun-Shi Liao; Wei Wei; William Yiu-Man Cheung; Wen-Juan Li; Lian-Feng Li; Gabriel M Leung; Edward C. Holmes; Yan-Ling Hu; Yi Guan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.13.945485"],["rdgyej4p",null,"medrxiv","Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis","10.1101/2020.02.14.20021535",null,null,"medrvix","Background:A Novel pneumonia associated with the 2019 coronavirus infected pneumonia (NCIP) suddenly broke out in Wuhan, China in December 2019. 37287 confirmed cases and 813 death case in China (Until 8th/Feb/2019) have been reported in just fortnight. Although this risky pneumonia with high infection rates and high mortality rates need to be resolved immediately, major gaps in our knowledge of clinical characters of it were still not be established. The aim of this study is to summaries and analysis the clinical characteristics of 2019-nCoV pneumonia. Methods: Literature have been systematically performed a search on PubMed, Embase, Web of Science, GreyNet International, and The Cochrane Library from inception up to February 8, 2020. The Newcastle-Ottawa Scale was used to assess quality, and publication bias was analyzed by Egger test. In the single-arm meta-analysis, A fix-effects model was used to obtain a pooled incidence rate. We conducted subgroup analysis according to geographic region and research scale. Results: A total of nine studies including 356 patients were included in this study, the mean age was 52.4 years and 221 (62.1%) were male. The pooled incidences rate of symptoms as follows: pharyngalgia (12.2%, 95% CI: 0.087-0.167), diarrhea (9.2%, 95% CI: 0.062-0.133) and headache (8.9%, 95% CI: 0.063-0.125). Meanwhile, 5.7% (95% CI: 0.027-0.114) of patients were found without any symptoms although they were diagnosed by RT-PCR. In the terms of CT imaging examination, the most of patients showed bilateral mottling or ground-glass opacity, 8.6% (95% CI: 0.048-0.148) of patients with crazy-paving pattern, and 11.5% (95% CI: 0.064-0.197) of patients without obvious CT imaging presentations. The pooled incidence of mortality was 8.9% (95% CI: 0.062-0.126). Conclusions: To our knowledge, this is the first evidence-based medicine research to further elaborate the clinical characteristics of NCIP, which is beneficial to the next step of prevention and treatment.","2020-02-17","Kai Qian; Yi Deng; Yonghang Tai; Jun Peng; Hao Peng; Lihong Jiang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.14.20021535"],["j1yqn9xg","f4d76fb8161f4cee6987d7d412798e1276348858","medrxiv","Assessing the impact of reduced travel on exportation dynamics of novel coronavirus infection (COVID-19)","10.1101/2020.02.14.20022897",null,null,"medrvix","The impact of the drastic reduction in travel volume within mainland China in January and February 2020 was quantified with respect to reports of novel coronavirus (COVID-19) infections outside China. Data on confirmed cases diagnosed outside China were analyzed using statistical models to estimate the impact of travel reduction on three epidemiological outcome measures: (i) the number of exported cases, (ii) the probability of a major epidemic, and (iii) the time delay to a major epidemic. From 28 January to 7 February 2020, we estimated that 226 exported cases (95% confidence interval: 86, 449) were prevented, corresponding to a 70.4% reduction in incidence compared to the counterfactual scenario. The reduced probability of a major epidemic ranged from 7% to 20% in Japan, which resulted in a median time delay to a major epidemic of two days. Depending on the scenario, the estimated delay may be less than one day. As the delay is small, the decision to control travel volume through restrictions on freedom of movement should be balanced between the resulting estimated epidemiological impact and predicted economic fallout.","2020-02-17","Asami Anzai; Tetsuro Kobayashi; Natalie M. Linton; Ryo Kinoshita; Katsuma Hayashi; Ayako Suzuki; Yichi Yang; Sungmok Jung; Takeshi Miyama; Andrei R. Akhmetzhanov; Hiroshi Nishiura",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.14.20022897"],["6f6t558e","b1b9d81726e56c2781ce6b63ae5015cf2f314c6f","medrxiv","Estimating the Efficacy of Traffic Blockage and Quarantine for the Epidemic Caused by 2019-nCoV (COVID-19)","10.1101/2020.02.14.20022913",null,null,"medrvix","Background: Since the 2019-nCoV (COVID-19) outbreaks in Wuhan, China, the cumulative number of confirmed cases is increasing every day, and a large number of populations all over the world are at risk. The quarantine and traffic blockage can alleviate the risk of the epidemic and the infections, henceforth evaluating the efficacy of such actions is essential to inform policy makers and raise the public awareness of the importance of self-isolation and quarantine. Method: We collected confirmed case data and the migration data, and introduced the quarantine factor and traffic blockage factor to the Flow-SEIR model. By varying the quarantine factor and traffic blockage factor, we simulated the change of the peak number and arrival time of infections, then the efficacy of these two intervation measures can be analyzed in our simulation. In our study, the self-protection at home is also included in quarantine. Results: In the simulated results, the quarantine and traffic blockage are effective for epidemic control. For Hubei province, the current quarantine factor is estimaed to be 0.405, which means around 40.5% of suceptibles who are close contacting with are in quarantine, and the current traffic blockage factor is estimaed to be 0.66, which indicates around 34% of suceptibles who had flowed out from Hubei. For the other provinces outside Hubei, the current quarantine factor is estimated to be 0.285, and the current traffic blockage factor is estimated to be 0.26. With the quarantine and traffic blockage factor increasing, the number of infections decrease dramatically. We also simulated the start dates of quarantine and traffic blockage at four time points, the simulated results show that the early of warning is also effective for epidemic containing. However, provincial level traffic blockage can only alleviate 21.06% - 22.38% of the peak number of infections. In general, the quarantine is much more effective than the traffic blockage control. Conclusion: Both of quarantine and traffic blockage are effective ways to control the spread of COVID-19. However, the eff icacy of quarantine is found to be much stronger than that of traffic blockage. Considering traffic blockage may also cause huge losses of economy, we propose to gradually deregulate the traffic blockage, and improve quarantine instead. Also, there might be a large number of asymptomatic carriers of COVID-19, the quarantine should be continued for a long time until the epidemic is totally under control.","2020-02-17","Deqiang Li; Zhicheng Liu; Qinghe Liu; Zefei Gao; Junkai Zhu; Junyan Yang; Qiao Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.14.20022913"],["4ah705nc","f4f4b045dce8ff5ff1f5aaa48a6bdbaaa7b05179","medrxiv","A new transmission route for the propagation of the SARS-CoV-2 coronavirus","10.1101/2020.02.14.20022939",null,null,"medrvix","We explore here how variation in the SARS-CoV-2 virus tropism could influence epidemic spread. We use a compartmental model fit to the existing data. The model indicates that Wuhan quarantine measures were effective but that alternative virus forms (gut tropism) and a second propagation route (through environment) was present. For Singapore and Shenzhen region the secondary route does not seem to be active yet. Adequate prevention measures taking into account both routes should be implemented.","2020-02-18","Antoine Danchin; Tuen Wai Patrick Ng; Gabriel TURINICI",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.14.20022939"],["u2uswgy3","080660f20f078c10524f6186bca263327094acbb","medrxiv","A deep learning algorithm using CT images to screen for Corona Virus Disease (COVID-19)","10.1101/2020.02.14.20023028",null,null,"medrvix","To control the spread of Corona Virus Disease (COVID-19), screening large numbers of suspected cases for appropriate quarantine and treatment is a priority. Pathogenic laboratory testing is the diagnostic gold standard but it is time consuming with significant false negative results. Fast and accurate diagnostic methods are urgently needed to combat the disease. Based on COVID-19 radiographical changes in CT images, we aimed to develop a deep learning method that could extract COVID-19's graphical features in order to provide a clinical diagnosis ahead of the pathogenic test, thus saving critical time for disease control. Methods:We collected 1,119 CT images of pathogen-confirmed COVID-19 cases along with those previously diagnosed with typical viral pneumonia. We modified the Inception transfer-learning model to establish the algorithm, followed by internal and external validation. Results: The internal validation achieved a total accuracy of 89.5% with specificity of 0.88 and sensitivity of 0.87. The external testing dataset showed a total accuracy of 79.3% with specificity of 0.83 and sensitivity of 0.67. In addition, in 54 COVID-19 images that first two nucleic acid test results were negative, 46 were predicted as COVID-19 positive by the algorithm, with the accuracy of 85.2%. Conclusion: These results demonstrate the proof-of-principle for using artificial intelligence to extract radiological features for timely and accurate COVID-19 diagnosis.","2020-02-17","Shuai Wang; Bo Kang; Jinlu Ma; Xianjun Zeng; Mingming Xiao; Jia Guo; Mengjiao Cai; Jingyi Yang; Yaodong Li; Xiangfei Meng; Bo Xu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.14.20023028"],["rgzmpoxv","f7891683eecd12b3908862fc7ef1d341ca7a293c","medrxiv","The Efficacy of Contact Tracing for the Containment of the 2019 Novel Coronavirus (COVID-19).","10.1101/2020.02.14.20023036",null,null,"medrvix","Contact tracing is a central public health response to infectious disease outbreaks, especially in the early stages of an outbreak when specific treatments are limited. Importation of novel Coronavirus (COVID-19) from China and elsewhere into the United Kingdom highlights the need to understand the impact of contact tracing as a control measure. Using detailed survey information on social encounters coupled to predictive models, we investigate the likely efficacy of the current UK definition of a close contact (within 2 meters for 15 minutes or more) and the distribution of secondary cases that may go untraced. Taking recent estimates for COVID-19 transmission, we show that less than 1 in 5 cases will generate any subsequent untraced cases, although this comes at a high logistical burden with an average of 36.1 individuals (95th percentiles 0-182) traced per case. Changes to the definition of a close contact can reduce this burden, but with increased risk of untraced cases; we estimate that any definition where close contact requires more than 4 hours of contact is likely to lead to uncontrolled spread.","2020-02-17","Matt J Keeling; T. Deirdre Hollingsworth; Jonathan M Read",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.14.20023036"],["3nipr212","0eda8331214ca028350b07e2953702f3078a105e","medrxiv","Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (COVID-19)","10.1101/2020.02.14.20023127",null,null,"medrvix","Background Estimation of the fraction and contagiousness of undocumented novel coronavirus (COVID-19) infections is critical for understanding the overall prevalence and pandemic potential of this disease. Many mild infections are typically not reported and, depending on their contagiousness, may support stealth transmission and the spread of documented infection. Methods Here we use observations of reported infection and spread within China in conjunction with mobility data, a networked dynamic metapopulation model and Bayesian inference, to infer critical epidemiological characteristics associated with the emerging coronavirus, including the fraction of undocumented infections and their contagiousness. Results We estimate 86% of all infections were undocumented (95% CI: [82%-90%]) prior to the Wuhan travel shutdown (January 23, 2020). Per person, these undocumented infections were 52% as contagious as documented infections ([44%-69%]) and were the source of infection for two-thirds of documented cases. Our estimate of the reproductive number (2.23; [1.77-3.00]) aligns with earlier findings; however, after travel restrictions and control measures were imposed this number falls considerably. Conclusions A majority of COVID-19 infections were undocumented prior to implementation of control measures on January 23, and these undocumented infections substantially contributed to virus transmission. These findings explain the rapid geographic spread of COVID-19 and indicate containment of this virus will be particularly challenging. Our findings also indicate that heightened awareness of the outbreak, increased use of personal protective measures, and travel restriction have been associated with reductions of the overall force of infection; however, it is unclear whether this reduction will be sufficient to stem the virus spread.","2020-02-17","Ruiyun Li; Sen Pei; Bin Chen; Yimeng Song; Tao Zhang; Wan Yang; Jeffrey Shaman",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.14.20023127"],["rcbw54xc","57e01ad2a4961cd5cc6a3733f5f8c013a8946f3c","medrxiv","A model simulation study on effects of intervention measures in Wuhan COVID-19 epidemic","10.1101/2020.02.14.20023168",null,null,"medrvix","Background: In the beginning of January 2020, new unknown virus pneumonia cases started to emerge in local hospitals in Wuhan, China. This virus epidemic quickly became a public health emergency of international concern by the WHO. Enormous amount of medical supplies as well as healthcare personals from other provinces were mobilized to support Wuhan. This current work tent to help people understanding how infectious disease spread and the purpose and consequences of various efforts based on simulation model. Method: a simulation model was created using known parameters. R0 set to 3 and mean incubation time to be 7.5days. the epidemic was divided to 3 periods. Simulation would run 50 times to mimic different patient0 status. Personal activity index was used to mimic different level of control measures. 141427709 simulated patients were created. Cumulation number of patients at the end of period 1 (day50) is 2868.7 ± 1739.0. Total infected patients could be 913396.5 ± 559099.9 by the end of period 2 (day70) in free transmission state. And at day90, total patients number is 913396.5 ± 559099.9. Conclusion: COVID-19 is a novel severe respiratory disease. This will put great burden on the shoulder of healthcare workers as well as on medical hardware and supplements. Current strict control measures help to contain disease from spreading. An early detecting, reporting and fast reacting system needs to be setup to prevent future unknown infectious disease.","2020-02-18","Guopeng ZHOU; Chunhua CHI",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.14.20023168"],["y9sel25v",null,"medrxiv","Estimate number of individuals infected with the 2019-novel coronavirus in South Korea due to the influx of international students from countries with virus risk: a simulation study","10.1101/2020.02.15.20023234",null,null,"medrvix","Background: In March 2020, overall, 37,000 international students from the country at risk of the 2019-novel coronavirus (COVID-19) infection will arrive in Seoul, South Korea. Individuals from the country at risk of COVID-19 infection have been included in a home-quarantine program, but the efficacy of the program is uncertain. Methods: To estimate the possible number of infected individuals within the large influx of international students, we used a deterministic compartmental model for epidemic and perform a simulation-based search of different rates of compliance with home-quarantine. Results: Under the home-quarantine program, the total number of the infected individuals would reach 24-53 from March 17-March 20, 50-86 from March 18-March 16, and 234-343 from March 4-March 23 with the arrival of 0.1%, 0.2%, and 1% of pre-infectious individuals, in Seoul, South Korea, respectively. Our findings indicated when incoming international students showed strict compliance with quarantine, epidemics were less likely to occur in Seoul, South Korea. Conclusion: To mitigate possible epidemics, additional efforts to improve the compliance of home-quarantine are warranted along with other containment policies.","2020-02-19","Sukhyun Ryu; Sheikh Taslim Ali; Jun-sik Lim; Byung Chul Chun",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.15.20023234"],["1rq612hm","91098f6fe46a21565bf0cb06fe960cdb2c3f5e38","medrxiv","The role of institutional trust in preventive and treatment-seeking behaviors during the 2019 novel coronavirus (2019-nCoV) outbreak among residents in Hubei, China","10.1101/2020.02.15.20023333",null,null,"medrvix","Background Since December 2019, pneumonia associated with the 2019 novel coronavirus (2019-nCoV) has emerged in Wuhan, China. The exponential increase of the confirmed number of cases of 2019n-CoV is of great concern to the global community. The fears and panic among residents in the epicenters have prompted diverse responses, which are understudied. During such a crisis, community trust and support for the government and health authorities are important to contain the outbreak. We aimed to investigate the influence of institutional trust on public responses to the 2019-nCoV outbreak. Methods An anonymous Internet-based, cross-sectional survey was administered on January 29, 2020. The study population comprised all residents currently residing or working in the province of Hubei, where Wuhan is the capital city. The level of trust in information provision and preventive instructions, individual preventive behaviors and treatment-seeking behaviors were queried. Findings The majority of the participants expressed a great extent of trust in the information and preventive instructions provided by the central government than by the local government. A high uptake of 2019-nCoV preventive measures was found, particularly among people who had been placed under quarantine. Being under quarantine (adjusted odds ratio [OR] = 2.35, 95% confidence interval [CI] 1.80 to 3.08) and having a high institutional trust score (OR = 2.23, 95% CI 1.96 to 2.53) were both strong and significant determinants of higher preventive behavior scores. The majority of study participants (85.7%, n = 3,640) reported that they would seek hospital treatment if they suspected themselves to have been infected with 2019 n-CoV. Few of the participants from Wuhan (16.6%, n = 475) and those participants who were under quarantine (13.8%, n = 550) expressed an unwillingness to seek hospital treatment. Similarly, being under quarantine (OR = 2.36, 95% CI 1.80 to 3.09) and having a high institutional trust score (OR = 2.20, 95% CI 1.96 to 2.49) were two strong significant determinants of hospital treatment-seeking. Interpretation The results of this study suggest that institutional trust is an important factor influencing adequate preventive behavior and seeking formal medical care during an outbreak. In view of the 2019-nCoV being highly pathogenic and extremely contagious, our findings also underscore the importance of public health intervention to reach individuals with poor adherence to preventive measures and who are reluctant to seek treatment at formal health services. Funding National Key R&D Program of China, Ningbo Health Branding Subject Fund, Sanming Project of Medicine in Shenzhen, K.C. Wong Magna Fund in Ningbo University, National Natural Science Foundation of China, Fundamental Research Funds for the Central Universities, China Postdoctoral Science Foundation, and Natural Science Basic Research Program of Shanxi Province. Keywords: 2019-nCoV; institutional trust; preventive behaviors","2020-02-21","Li Ping Wong; Qun hong Wu; Xi Chen; Zhuo Chen; Haridah Alias; Mingwang Shen; Jing cen Hu; Shiwei Duan; Jin jie Zhang; Liyuan Han",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.15.20023333"],["ehm4fgos","317943dec06b88c1c62ef0ecd832f0d644b1d417","medrxiv","Evaluating new evidence in the early dynamics of the novel coronavirus COVID-19 outbreak in Wuhan, China with real time domestic traffic and potential asymptomatic transmissions","10.1101/2020.02.15.20023440",null,null,"medrvix","The novel coronavirus (COVID-19), first detected in Wuhan, China in December 2019, has spread to 28 countries/regions with over 43,000 confirmed cases. Much about this outbreak is still unknown. At this early stage of the epidemic, it is important to investigate alternative sources of information to understand its dynamics and spread. With updated real time domestic traffic, this study aims to integrate recent evidence of international evacuees extracted from Wuhan between Jan. 29 and Feb. 2, 2020 to infer the dynamics of the COVD-19 outbreak in Wuhan. In addition, a modified SEIR model was used to evaluate the empirical support for the presence of asymptomatic transmissions. Based on the data examined, this study found little evidence for the presence of asymptomatic transmissions. However, it is still too early to rule out its presence conclusively due to sample size and other limitations. The updated basic reproductive number was found to be 2.12 on average with a 95% credible interval of [2.04, 2.18]. It is smaller than previous estimates probably because the new estimate factors in the social and non-pharmaceutical mitigation implemented in Wuhan through the evacuee dataset. Detailed predictions of infected individuals exported both domestically and internationally were produced. The estimated case confirmation rate has been low but has increased steadily to 23.37% on average. The findings of this study depend on the validity of the underlying assumptions, and continuing work is needed, especially in monitoring the current infection status of Wuhan residents.","2020-02-18","Can Zhou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.15.20023440"],["3r0nqzgu","73998a3f565b661816d48aa7c20fa5ae617864d5","medrxiv","Profiling ACE2 expression in colon tissue of healthy adults and colorectal cancer patients by single-cell transcriptome analysis","10.1101/2020.02.15.20023457",null,null,"medrvix","A newly identified novel coronavirus (2019-nCoV) has caused numerous acute respiratory syndrome cases in Wuhan China from December 2019 to Feb 2020. Its fast spreading to other provinces in China and overseas is very likely causing a pandemic. Since the novel coronavirus has been reported to be capable of endangering thousands of lives, it is extremely important to find out how the coronavirus is transmitted in human organs. Apart from fever and respiratory complications, gastrointestinal symptoms are observed in some patients with 2019-nCoV but the significance remains undetermined. The cell receptor angiotensin covering enzyme II (ACE2), which is the major receptor of SARS-nCoV, has been reported to be a cellular entry receptor of 2019-nCoV as well. Here, to more precisely explore the potential pathogen transmission route of the 2019-nCoV infections in the gastrointestinal tract, we analyzed the ACE2 RNA expression profile in the colon tissue of healthy adults and colorectal cancer patients of our cohort and other databases. The data indicates that ACE2 is mainly expressed in epithelial cells of the colon. The expression of ACE2 is gradually increased from healthy control, adenoma to colorectal cancer patients in our cohort as well as in the external Asian datasets. According to the expression profile of ACE2 in colon epithelial cells, we speculate adenoma and colorectal cancer patients are more likely to be infected with 2019-nCoV than healthy people. Our data may provide a theoretical basis for the classification and management of future 2019-nCoV susceptibility people in clinical application.","2020-02-23","Haoyan Chen; Baoqin Xuan; Yuqing Yan; Xiaoqiang Zhu; Chaoqin Shen; Gang Zhao; Linhua Ji; Danhua Xu; Hua Xiong; TaChung Yu; Xiaobo Li; Qiang Liu; Yingxuan Chen; Yun Cui; Jie Hong; Jing-Yuan Fang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.15.20023457"],["b4kzgubs","f9c6a656d0ce352cc1fa0df8d83e6d4a1f53d7c2","medrxiv","Estimating the Case Fatality Risk of COVID-19 using Cases from Outside China","10.1101/2020.02.15.20023499",null,null,"medrvix","There is large uncertainty around the case fatality risk (CFR) for COVID-19 in China. Therefore, we considered symptomatic cases outside of China (countries/settings with 20+ cases) and the proportion who are in intensive care units (4.0%, 14/349 on 13 February 2020). Given what is known about CFRs for ICU patients with severe respiratory conditions from a meta-analysis, we estimated a CFR of 1.37% (95%CI: 0.57% to 3.22%) for COVID-19 cases outside of China.","2020-02-18","Nick Wilson; Amanda Kvalsvig; Lucy Telfar Barnard; Michael Baker",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.15.20023499"],["9aegg5sd","1df9ce60e358f6cef0be3f6e15fe2a9c6f93f39c","biorxiv","Multivariate Analyses of Codon Usage of SARS-CoV-2 and other betacoronaviruses","10.1101/2020.02.15.950568",null,null,"biorxiv","AbstractCoronavirus disease 2019 (COVID-19) is a global health concern as it continues to spread within China and beyond. The causative agent of this disease, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belongs to the genus Betacoronavirus which also includes severe acute respiratory syndrome related coronavirus (SARSr-CoV) and Middle East respiratory syndrome related coronavirus (MERSr-CoV). Codon usage of viral genes are believed to be subjected to different selection pressures in different host environments. Previous studies on codon usage of influenza A viruses can help identify viral host origins and evolution trends, however, similar studies on coronaviruses are lacking. In this study, global correspondence analysis (CA), within-group correspondence analysis (WCA) and between-group correspondence analysis (BCA) were performed among different genes in coronavirus viral sequences. The amino acid usage pattern of SARS-CoV-2 was generally found similar to bat and human SARSr-CoVs. However, we found greater synonymous codon usage differences between SARS-CoV-2 and its phylogenetic relatives on spike and membrane genes, suggesting these two genes of SARS-CoV-2 are subjected to different evolutionary pressures.","2020-02-20","Haogao Gu; Daniel Chu; Malik Peiris; Leo L.M. Poon",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.15.950568"],["m87tapjp","10138816c67a562b55bc9c2b882b62c81fc4b062","medrxiv","Epidemic analysis of COVID-19 in China by dynamical modeling","10.1101/2020.02.16.20023465",null,null,"medrvix","The outbreak of the novel coronavirus (2019-nCoV) epidemic has attracted world- wide attention. Herein, we propose a mathematical model to analyzes this epidemic, based on a dynamic mechanism that incorporating the intrinsic impact of hidden la- tent and infectious cases on the entire process of transmission. Meanwhile, this model is validated by data correlation analysis, predicting the recent public data, and back- tracking, as well as sensitivity analysis. The dynamical model reveals the impact of various measures on the key parameters of the epidemic. According to the public data of NHCs from 01/20 to 02/09, we predict the epidemic peak and possible end time for 5 different regions. The epidemic in Beijing and Shanghai, Mainland/Hubei and Hubei/Wuhan, are expected to end before the end of February, and before mid- March respectively. The model indicates that, the outbreak in Wuhan is predicted to be ended in the early April. As a result, more effective policies and more efforts on clinical research are demanded. Moreover, through the backtracking simulation, we infer that the outbreak of the epidemic in Mainland/Hubei, Hubei/Wuhan, and Wuhan can be dated back to the end of December 2019 or the beginning of January 2020.","2020-02-18","Liangrong Peng; Wuyue Yang; Dongyan Zhang; Changjing Zhuge; Liu Hong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023465"],["9j5yda3k","b117016a887f05386acc4387dab5bf8c769cf90e","medrxiv","Simulating and Forecasting the Cumulative Confirmed Cases of SARS-CoV-2 in China by Boltzmann Function-based Regression Analyses","10.1101/2020.02.16.20023564",null,null,"medrvix","An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential total number of confirmed cases in mainland China by applying Boltzmann-function based regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well fitted with the Boltzmann function (R2 being close to 0.999). The potential total number of confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases of SARS-CoV-2 in mainland China (including Hubei Province) will become minimal between Feb 28 and Mar 10, 2020, with 95% CI. In addition, we found that the data of cumulative confirmed cases of 2003 SARS-CoV in China and Worldwide were also well fitted to the Boltzmann function. To our knowledge this is the first study revealing that the Boltzmann function is suitable to simulate epidemics. The estimated potential total number of confirmed cases and key dates for the SARS-CoV-2 outbreak may provide certain guidance for governments, organizations and citizens to optimize preparedness and response efforts.","2020-02-18","Xinmiao Fu; Qi Ying; Tieyong Zeng; Tao Long; Yan Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023564"],["ndnkhj42","8bd8158e4a47c65b12a462b49b4b8b4af1488dc3","medrxiv","Estimation of the final size of the coronavirus epidemic by the logistic model","10.1101/2020.02.16.20023606",null,null,"medrvix","In the note, the logistic growth regression model is used for the estimation of the final size and its peak time of the coronavirus epidemic. Based on available data estimation the final size will be about 90 000 cases and the peak time was on 10 Feb 2020.","2020-02-18","milan batista",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023606"],["oknxic4q","d069dfb7f0aefcdc2c890a1bbe773ebd26b01a55","medrxiv","Utilize State Transition Matrix Model to Predict the Novel Corona Virus Infection Peak and Patient Distribution","10.1101/2020.02.16.20023614",null,null,"medrvix","Background: Since December 2019, a pneumonia caused by the 2019 novel coronavirus (2019-nCoV) has broken out in Wuhan, Hubei province, China. The continuous rising of infected cases has imposed overwhelming pressure on public health decision and medical resource allocation in China. We managed to forecast the infection peak time in Hubei province and the severe and critical case distribution. Methods: We used data resource according to cases reported by the National Health Commission of the People's Republic of China (Jan 25, 2019, to Feb 28, 2020) as the training set to deduce the arrival of the peak infection time and the number of severe and critical cases in Wuhan on subsequent days. Medical observation, discharge, infected, non-Severe, infected and severe, cure and death data were collected and analyzed. Using this state transition matrix model, we will be able predict when the inflection peak time (the maximum open infection cases) in Hubei Province will occur. Also, we can use this model to predict the patient distribution (severe, non-severe) to better allocate medical resource. Under relative pessimistic scenario, the inflection peak time is April 6-April 14. The numbers of critically ill and critically ill patients will lie between 8300-9800 and 2200-2700, respectively. Results: In very optimistic scenarios (daily NCC decay rate of -10%), the peak time of open inflection cases will arrive around February 23-February 26. At the same time, there will be a peak in the numbers of severely ill and critically ill patients, between 6800-7200 and 1800-2000, respectively. In a relative optimistic scenario (daily NCC decay rate of -5%), the inflection case peak time will arrive around February 28-March 2. The numbers of critically ill and critically ill patients will lie between 7100-7800 and 1900-2200, respectively. In a relatively pessimistic scenario (daily NCC decay rate of -1%), the inflection peak time does not arrive around the end of March. Estimated time is April 6-April 14. The numbers of critically ill and critically ill patients will lie between 8300-9800 and 2200-2700, respectively. We are using the diagnosis rate, mortality rate, cure rate as the 2/8 data. There should be room for improvement, if these metrics continue to improve. In that case, the peak time will arrive earlier than our estimation. Also, the severe and critical case ratios are likely to decline as the virus becomes less toxic and medical conditions improve. If that happens, the peak numbers will be lower than predicted above. Conclusion: We can infer that we are still not close to the end of this outbreak and the number of critically ill patients is still climbing. Assisting critical care resources in Hubei province requires the government to consider further tilt, and it is vital to make reasonable management of doctors and medical assistance systems to curb the transmission trend.","2020-02-19","Ke Wu; Junhua Zheng; Jian Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023614"],["c800ynvc","3da1a03cb675f463b0d8e30744aa33512241dd29","medrxiv","SEIR Transmission dynamics model of 2019 nCoV coronavirus with considering the weak infectious ability and changes in latency duration","10.1101/2020.02.16.20023655",null,null,"medrvix","Pneumonia patients of 2019-ncov in latent period are not easy to be effectively quarantined, but there is evidence that they have strong infectious ability. Here, the infectious ability of patients during the latent period is slightly less than that of the infected patients was assumed. We established a new SEIR propagation dynamics model, that considered the weak transmission ability of the incubation period, the variation of the incubation period length, and the government intervention measures to track and isolate comprehensively. Based on the raw epidemic data of China from January 23, 2020 to February 10, 2020, the dynamic parameters of the new present SEIR model are fitted. Through the Euler integration algorithm to solve the model, the effect of infectious ability of incubation patients on the theoretical estimation of the present SEIR model was analyzed, and the occurrence time of peak number in China was predicted.","2020-02-20","Pengpeng Shi; Shengli Cao; Peihua Feng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023655"],["9xmeyrtu","7325363d061f0f4699e29bfa544438e14e7c207d","medrxiv","Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients","10.1101/2020.02.16.20023671",null,null,"medrvix","Background: The dynamic changes of lymphocyte subsets and cytokines profiles of patients with novel coronavirus disease (COVID-19) and their correlation with the disease severity remain unclear. Method: Peripheral blood samples were longitudinally collected from 40 confirmed COVID-19 patients and examined for lymphocyte subsets by flow cytometry and cytokine profiles by specific immunoassays. Findings: Of the 40 COVID-19 patients enrolled, 13 severe cases showed significant and sustained decreases in lymphocyte counts but increases in neutrophil counts than 27 mild cases. Further analysis demonstrated significant decreases in the counts of T cells, especially CD8 + T cells, as well as increases in IL-6, IL-10, IL-2 and IFN-γ levels in the peripheral blood in the severe cases compared to those in the mild cases. T cell counts and cytokine levels in severe COVID-19 patients who survived the disease gradually recovered at later time points to levels that were comparable to those of the mild cases. Moreover, the neutrophil-to-CD8+ T cell ratio (N8R) were identified as the most powerful prognostic factor affecting the prognosis for severe COVID-19. Conclusion: The degree of lymphopenia and a proinflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity. N8R may serve as a useful prognostic factor for early identification of severe COVID-19 cases.","2020-02-18","Jing Liu; Sumeng Li; Jia Liu; Boyun Liang; Xiaobei Wang; Hua Wang; Wei Li; Qiaoxia Tong; Jianhua Yi; Lei Zhao; Lijuan Xiong; Chunxia Guo; Jin Tian; Jinzhuo Luo; Jinghong Yao; Ran Pang; Hui Shen; Cheng Peng; Ting Liu; Qian Zhang; Jun Wu; Ling Xu; Sihong Lu; Baoju Wang; Zhihong Weng; Chunrong Han; Huabing Zhu; Ruxia Zhou; Helong Zhou; Xiliu Chen; Pian Ye; Bin Zhu; Shengsong He; Yongwen He; Shenghua Jie; Ping Wei; Jianao Zhang; Yinping Lu; Weixian Wang; Li Zhang; Ling Li; Fengqin Zhou; Jun Wang; Ulf Dittmer; Mengji Lu; Yu Hu; Dongliang Yang; Xin Zheng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023671"],["fugb778l","6445688d3f59cb7e02f2a8b28450cdf118a8a373","medrxiv","Contacts in context: large-scale setting-specific social mixing matrices from the BBC Pandemic project","10.1101/2020.02.16.20023754",null,null,"medrvix","Social mixing patterns are crucial in driving transmission of infectious diseases and informing public health interventions to contain their spread. Age-specific social mixing is often inferred from surveys of self-recorded contacts which by design often have a very limited number of participants. In addition, such surveys are rare, so public health interventions are often evaluated by considering only one such study. Here we report detailed population contact patterns for United Kingdom based self-reported contact data from over 36,000 volunteers that participated in the massive citizen science project BBC Pandemic. The amount of data collected allows us generate fine-scale age-specific population contact matrices by context (home, work, school, other) and type (conversational or physical) of contact that took place. These matrices are highly relevant for informing prevention and control of new outbreaks, and evaluating strategies that reduce the amount of mixing in the population (such as school closures, social distancing, or working from home). In addition, they finally provide the possibility to use multiple sources of social mixing data to evaluate the uncertainty that stems from social mixing when designing public health interventions.","2020-02-19","Petra Klepac; Adam J Kucharski; Andrew JK Conlan; Stephen Kissler; Maria Tang; Hannah Fry; Julia R Gog",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023754"],["asyrijyg",null,"medrxiv","Study on SARS-COV-2 transmission and the effects of control measures in China","10.1101/2020.02.16.20023770",null,null,"medrvix","Objective To reconstruct the transmission trajectory of SARS-COV-2 and analyze the effects of control measures in China. Methods Python 3.7.1 was used to write a SEIR class to model the epidemic procedure and a back propagation class to estimate the initial true infected number. The epidemic area in China was divided into three parts, Wuhan city, Hubei province (except Wuhan) and China (except Hubei) based on the different transmission pattern. A limitation factor for the medical resource was imposed to model the infected but not quarantined. Credible data source from Baidu Qianxi were used to assess the number of infected cases migrated from Wuhan to other areas. Results Basic reproduction number, R0, was 3.6 in the very early stage. The true infected number was 4508 in our model in Wuhan before January 22, 2020. By January 22 2020, it was estimated that 1764 infected cases migrated from Wuhan to other cities in Hubei province. Effective reproductive number, R, gradually decreased from 3.6 (Wuhan, stage 1), 3.4 (Hubei except Wuhan, stage 1) and 3.3 (China except Hubei, stage 1) to 0.67 (Wuhan, stage 4), 0.83 (Hubei except Wuhan, stage 2) and 0.63 (China except Hubei, stage 2), respectively. Especially after January 23, 2020 when Wuhan City was closed, the infected number showed a turning point in Wuhan. By early April, there would be 42073, 21342 and 13384 infected cases in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively, and there would be 2179, 633 and 107 death in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively. Conclusion A series of control measures in China have effectively prevented the spread of COVID-19, and the epidemic will end in early April.","2020-02-18","Bo Zhang; Hongwei Zhou; Fang Zhou",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023770"],["32vs98ya","e62e453735dc2022f4d81aa9b22769de30d0bb90","medrxiv","When will the battle against novel coronavirus end in Wuhan: a SEIR modeling analysis","10.1101/2020.02.16.20023804",null,null,"medrvix","Recent outbreak of 2019-nCoV in Wuhan raised serious public health concerns. By February 15, 2020 in Wuhan, the total number of confirmed infection cases has reached 37,914, and the number of deaths has reached 1123, accounting for 56.9% of the total confirmed cases and 73.7% of the total deaths in China. People are eager to know when the epidemic will be completely controlled and when people's work and life will be on the right track. In this study we analyzed the epidemic dynamics and trend of 2019-nCoV in Wuhan by using the data after the closure of Wuhan city till February 12, 2020 based on the SEIR modeling method. The optimal parameters were estimated as R0=1.44 (interquartile range: 1.40-1.47)，TI=14 (interquartile range: 14-14) and TE=3.0 (interquartile range: 2.8-3.1). Based on these parameters, the number of infected individuals in Wuhan city may reach the peak around February 19 at about 45,000 people. Once entering March, the epidemic would gradually decline, and end around the late March. It is worth noting that the above prediction is based on the assumption that the number of susceptible population N = 200,000 will not increase. If the epidemic situation is not properly controlled, the peak of infected number can be further increased and the peak time will be a little postponed. It was expected that the epidemic would subside in early March, and disappear gradually towards the late March.","2020-02-18","lianglu zhang; kangkang wan; jing chen; changming lu; lanlan dong; zhicheng wu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023804"],["jljjqs6m","33bd85ea9ca61c4d0674c43ae5b901b271b46f1e","medrxiv","Fractal kinetics of COVID-19 pandemic","10.1101/2020.02.16.20023820",null,null,"medrvix","We give an update to the original paper posted on 2/17/20 -- now (as of 3/1/20) the China deaths are rapidly decreasing, and we find an exponential decline to the power law similar to the that predicted by the network model of \\citet{vazquez_polynomial_2006}. At the same time, we see non-China deaths increasing rapidly, and similar to the early behavior of the China statistics. Thus, we see three stages of the spread of the disease in terms of number of deaths: exponential growth, power-law behavior, and then exponential decline in the daily rate. (Original abstract) The novel coronavirus (COVID-19) continues to grow rapidly in China and is spreading in other parts of the world. The classic epidemiological approach in studying this growth is to quantify a reproduction number and infection time, and this is the approach followed by many studies on the epidemiology of this disease. However, this assumption leads to exponential growth, and while the growth rate is high, it is not following exponential behavior. One approach that is being used is to simply keep adjusting the reproduction number to match the dynamics. Other approaches use rate equations such as the SEIR and logistical models. Here we show that the current growth closely follows power-law kinetics, indicative of an underlying fractal or small-world network of connections between susceptible and infected individuals. Positive deviations from this growth law might indicate either a failure of the current containment efforts while negative deviations might indicate the beginnings of the end of the pandemic. We cannot predict the ultimate extent of the pandemic but can get an estimate of the growth of the disease.","2020-02-20","Anna L. Ziff; Robert M. Ziff",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023820"],["z5i8hyja","fbebe4b66073c44cface2e842754bce26e3e2913","medrxiv","Risk map of the novel coronavirus (2019-nCoV) in China: proportionate control is needed","10.1101/2020.02.16.20023838",null,null,"medrvix","Background China is running a national level antivirus campaign against the novel coronavirus (2019-nCoV). Strict control measures are being enforced in either the populated areas and remote regions. While the virus is closed to be under control, tremendous economic loss has been caused. Methods and findings We assessed the pandemic risk of 2019-nCoV for all cities/regions in China using the random forest algorithm, taking into account the effect of five factors: the accumulative and increased numbers of confirmed cases, total population, population density, and GDP. We defined four levels of the risk, corresponding to the four response levels to public health emergencies in China. The classification system has good consistency among cities in China, as the error rate of the confusion matrix is 1.58%. Conclusions The pandemic risk of 2019-nCoV is dramatically different among the 442 cities/regions. We recommend to adopt proportionate control policy according to the risk level to reduce unnecessary economic loss.","2020-02-18","Xinhai Li; Xumao Zhao; Yingqiang Lou; Yuehua Sun",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023838"],["bsgqsdbx","d4cfb1fc4fc53abbc1d4b0ca60276c6af6632c3c","medrxiv","Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019","10.1101/2020.02.16.20023903",null,null,"medrvix","Background Since late December, 2019, an outbreak of pneumonia cases caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, and continued to spread throughout China and across the globe. To date, few data on immunologic features of Coronavirus Disease 2019 (COVID-19) have been reported. Methods In this single-centre retrospective study, a total of 21 patients with pneumonia who were laboratory-confirmed to be infected with SARS-CoV-2 in Wuhan Tongji hospital were included from Dec 19, 2019 to Jan 27, 2020. The immunologic characteristics as well as their clinical, laboratory, radiological features were compared between 11 severe cases and 10 moderate cases. Results Of the 21 patients with COVID-19, only 4 (19%) had a history of exposure to the Huanan seafood market. 7 (33.3%) patients had underlying conditions. The average age of severe and moderate cases was 63.9 and 51.4 years, 10 (90.9%) severe cases and 7 (70.0%) moderate cases were male. Common clinical manifestations including fever (100%, 100%), cough (70%, 90%), fatigue (100%, 70%) and myalgia (50%, 30%) in severe cases and moderate cases. PaO2/FiO2 ratio was significantly lower in severe cases (122.9) than moderate cases (366.2). Lymphocyte counts were significantly lower in severe cases (7000 million/L) than moderate cases (11000 million/L). Alanine aminotransferase, lactate dehydrogenase levels, high-sensitivity C-reactive protein and ferritin were significantly higher in severe cases (41.4 U/L, 567.2 U/L, 135.2 mg/L and 1734.4 ug/L) than moderate cases (17.6 U/L, 234.4 U/L, 51.4 mg/L and 880.2 ug /L). IL-2R, TNF-α and IL-10 concentrations on admission were significantly higher in severe cases (1202.4 pg/mL, 10.9 pg/mL and 10.9 pg/mL) than moderate cases (441.7 pg/mL, 7.5 pg/mL and 6.6 pg/mL). Absolute number of total T lymphocytes, CD4+T cells and CD8+T cells decreased in nearly all the patients, and were significantly lower in severe cases (332.5, 185.6 and 124.3 million/L) than moderate cases (676.5, 359.2 and 272.0 million/L). The expressions of IFN-γ by CD4+T cells tended to be lower in severe cases (14.6%) than moderate cases (23.6%). Conclusion The SARS-CoV-2 infection may affect primarily T lymphocytes, particularly CD4+T cells, resulting in significant decrease in number as well as IFN-γ production, which may be associated with disease severity. Together with clinical characteristics, early immunologic indicators including diminished T lymphocytes and elevated cytokines may serve as potential markers for prognosis in COVID-19.","2020-02-19","Guang Chen; Di Wu; Wei Guo; Yong Cao; Da Huang; Hongwu Wang; Tao Wang; Xiaoyun Zhang; Huilong Chen; Haijing Yu; Xiaoping Zhang; Minxia Zhang; Shiji Wu; Jianxin Song; Tao Chen; Meifang Han; Shusheng Li; Xiaoping Luo; Jianping Zhao; Qin Ning",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.20023903"],["tjxtifao","56a81431a148344381a1da80c877885f3f3d6d59","biorxiv","Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites","10.1101/2020.02.16.946699",null,null,"biorxiv","AbstractAlternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of Toxoplasma gondii and Plasmodium falciparum. We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies.","2020-02-17","V Vern Lee; Louise M. Judd; Aaron R. Jex; Kathryn E. Holt; Christopher J. Tonkin; Stuart A. Ralph",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.946699"],["nhq0oq8y","a388cca261e8af7936cf3b79c35f43442f65706b","biorxiv","SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development","10.1101/2020.02.16.951723",null,null,"biorxiv","AbstractSARS-CoV-2 and SARS-CoV share a common human receptor ACE2. Protein-protein interaction structure modeling indicates that spike-RBD of the two viruses also has similar overall binding conformation and binding free energy to ACE2. In vitro assays using recombinant ACE2 proteins and ACE2 expressing cells confirmed the two coronaviruses’ similar binding affinities to ACE2. The above studies provide experimental supporting evidences and possible explanation for the high transmissibility observed in the SARS-CoV-2 outbreak. Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2. ACE2-non-blocking SARS-CoV RBD antibodies, though with weaker neutralizing activities against SARS-CoV, showed positive cross-neutralizing activities to SARS-CoV-2 with an unknown mechanism. These findings suggest a trade-off between the efficacy and spectrum for therapeutic antibodies to different coronaviruses, and hence highlight the possibilities and challenges in developing broadly protecting antibodies and vaccines against SARS-CoV-2 and its future mutants.","2020-02-20","Chunyun Sun; Long Chen; Ji Yang; Chunxia Luo; Yanjing Zhang; Jing Li; Jiahui Yang; Jie Zhang; Liangzhi Xie",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.951723"],["rymj6lkj","22f3be8bbf6d792549b7a6088df7440e425cb34f","biorxiv","Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses","10.1101/2020.02.16.951913",null,null,"biorxiv","AbstractThe new coronavirus (2019-nCoV) outbreak from December 2019 in Wuhan, Hubei, China, has been declared a global public health emergency. Angiotensin I converting enzyme 2 (ACE2), is the host receptor by 2019-nCov to infect human cells. Although ACE2 is reported to be expressed in lung, liver, stomach, ileum, kidney and colon, its expressing levels are rather low, especially in the lung. 2019-nCoV may use co-receptors/auxiliary proteins as ACE2 partner to facilitate the virus entry. To identify the potential candidates, we explored the single cell gene expression atlas including 119 cell types of 13 human tissues and analyzed the single cell co-expression spectrum of 51 reported RNA virus receptors and 400 other membrane proteins. Consistent with other recent reports, we confirmed that ACE2 was mainly expressed in lung AT2, liver cholangiocyte, colon colonocytes, esophagus keratinocytes, ileum ECs, rectum ECs, stomach epithelial cells, and kidney proximal tubules. Intriguingly, we found that the candidate co-receptors, manifesting the most similar expression patterns with ACE2 across 13 human tissues, are all peptidases, including ANPEP, DPP4 and ENPEP. Among them, ANPEP and DPP4 are the known receptors for human CoVs, suggesting ENPEP as another potential receptor for human CoVs. We also conducted “CellPhoneDB” analysis to understand the cell crosstalk between CoV-targets and their surrounding cells across different tissues. We found that macrophages frequently communicate with the CoVs targets through chemokine and phagocytosis signaling, highlighting the importance of tissue macrophages in immune defense and immune pathogenesis.","2020-02-21","Furong Qi; Shen Qian; Shuye Zhang; Zheng Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.16.951913"],["x2bl5lcn",null,"medrxiv","Estimating the case fatality ratio of the COVID-19 epidemic in China","10.1101/2020.02.17.20023630",null,null,"medrvix","Background: Corona Virus Disease 2019 (COVID-19) due to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan city and rapidly spread throughout China since late December 2019. Crude case fatality ratio (CFR) with dividing the number of known deaths by the number of confirmed cases does not represent the true CFR and might be off by orders of magnitude. We aim to provide a precise estimate of the CFR of COVID-19 using statistical models at the early stage of the epidemic. Methods: We extracted data from the daily released epidemic report published by the National Health Commission P. R. China from 20 Jan 2020, to 1 March 2020. Competing risk model was used to obtain the cumulative hazards for death, cure, and cure-death hazard ratio. Then the CFR was estimated based on the slope of the last piece in joinpoint regression model, which reflected the most recent trend of the epidemic. Results: As of 1 March 2020, totally 80,369 cases were diagnosed as COVID-19 in China. The CFR of COVID-19 were estimated to be 70.9% (95% CI: 66.8%-75.6%) during Jan 20-Feb 2, 20.2% (18.6%-22.1%) during Feb 3-14, 6.9% (6.4%-7.4%) during Feb 15-23, 1.5% (1.4%-1.6%) during Feb 24-March 1 in Hubei province, and 20.3% (17.0%-25.3%) during Jan 20-28, 1.9% (1.8%-2.1%) during Jan 29-Feb 12, 0.9% (0.8%-1.1%) during Feb 13-18, 0.4% (0.4%-0.5%) during Feb 19-March 1 in other areas of China, respectively. Conclusions: Based on analyses of public data, we found that the CFR in Hubei was much higher than that of other regions in China, over 3 times in all estimation. The CFR would follow a downwards trend based on our estimation from recently released data. Nevertheless, at early stage of outbreak, CFR estimates should be viewed cautiously because of limited data source on true onset and recovery time.","2020-02-20","Xing Wang; Zihui Ma; Yi Ning; Chen Chen; Rujin Chen; Qiwen Chen; Heng Zhang; Chunming Li; Yan He; Tao Wang; Cheng Tong; Junqing Wu; Yuyan Li; Handong Ma; Shaodian Zhang; Hongxin Zhao",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20023630"],["7odpslba","fe685aa676e739bd52ba2585a7e5b27c55e2d0d6","medrxiv","Evidence for gastrointestinal infection of SARS-CoV-2","10.1101/2020.02.17.20023721",null,null,"medrvix","The new coronavirus (SARS-CoV-2) outbreak originating from Wuhan, China, poses a threat to global health. While it's evident that the virus invades respiratory tract and transmits from human to human through airway, other viral tropisms and transmission routes remain unknown. We tested viral RNA in stool from 73 SARS-CoV-2-infected hospitalized patients using rRT-PCR. 53.42% of the patients tested positive in stool. 23.29% of the patients remained positive in feces even after the viral RNA decreased to undetectable level in respiratory tract. The viral RNA was also detected in gastrointestinal tissues. Furthermore, gastric, duodenal and rectal epithelia showed positive immunofluorescent staining of viral host receptor ACE2 and viral nucleocapsid protein in a case of SARS-CoV-2 infection. Our results provide evidence for gastrointestinal infection of SARS-CoV-2, highlighting its potential fecal-oral transmission route.","2020-02-20","Fei Xiao; Meiwen Tang; Xiaobin Zheng; Chunna Li; Jianzhong He; Zhongsi Hong; Siwen Huang; Zhenyi Zhang; Xianqi Lin; Zhaoxiong Fang; Renxu Lai; Shoudeng Chen; Jing Liu; Jin Huang; Jinyu Xia; Zhonghe Li; Guanmin Jiang; Ye Liu; Xiaofeng Li; Hong Shan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20023721"],["q6pz6l2b","0cad40f50e41144e8b64744251515eca8cfac5f7","medrxiv","The reproductive number R0 of COVID-19 based on estimate of a statistical time delay dynamical system","10.1101/2020.02.17.20023747",null,null,"medrvix","In this paper, we estimate the reproductive number R0 of COVID-19 based on Wallinga and Lipsitch framework {11} and a novel statistical time delay dynamic system. We use the observed data reported in CCDC's paper to estimate distribution of the generation interval of the infection and apply the simulation results from the time delay dynamic system as well as released data from CCDC to fit the growth rate. The conclusion is: Based our Fudan-CCDC model, the growth rate r of COVID-19 is almost in [0.30, 0.32] which is larger than the growth rate 0.1 estimated by CCDC {9}, and the reproductive number R0 of COVID-19 is estimated by 3.25≤ R0 ≤3.4 if we simply use R=1+r*Tc with Tc=7.5, which is bigger than that of SARS. Some evolutions and predictions are listed.","2020-02-20","Nian Shao; Jin Cheng; Wenbin Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20023747"],["nav88l73",null,"medrxiv","COVID-19 in a Designated Infectious Diseases HospitalOutside Hubei Province,China","10.1101/2020.02.17.20024018",null,null,"medrvix","Background A new type of novel coronavirus infection (COVID-19) occurred in Wuhan, Hubei Province. Previous investigations reported patients in Wuhan city often progressed into severe or critical and had a high mortality rate.The clinical characteristics of affected patients outside the epicenter of Hubei province are less well understood. Methods All confirmed COVID-19 case treated in the Third People's Hospital of Shenzhen,from January 11, 2020 to February 6, 2020, were included in this study. We analyzed the epidemiological and clinical features of these cases to better inform patient management in normal hospital settings. Results Among the 298 confirmed cases, 233(81.5%) had been to Hubei while 42(14%) had not clear epidemiological history. Only 192(64%) cases presented with fever as initial symptom. The lymphocyte count decreased in 38% patients after admission. The number (percent) of cases classified as non-severe and severe was 240(80.6%) and 58(19.4%) respectively. Thirty-two patients (10.7%) needed ICU care. Compared to the non-severe cases, severe cases were associated with older age, underlying diseases, as well as higher levels of CRP, IL-6 and ESR. The median (IRQ) duration of positive viral test were 14(10-19). Slower clearance of virus was associated with higher risk of progression to severe clinical condition. As of February 14, 2020, 66(22.1%) patients were discharged and the overall mortality rate remains 0. Conclusions In a designated hospital outside the Hubei Province, COVID-19 patients were mainly characterized by mild symptoms and could be effectively manage by properly using the existing hospital system.","2020-02-19","Qingxian Cai; Deliang Huang; Pengcheng Ou; Hong Yu; Zhibin Zhu; Zhang Xia; Yinan Su; Zhenghua Ma; Yiming Zhang; Zhiwei Li; Qing He; Yang Fu; Lei Liu; Jun Chen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20024018"],["s12vwz8m","dac1b1607ae72b9509ab26367e0d55016e8132a8","medrxiv","Epidemic Situation of Novel Coronavirus Pneumonia in China mainland","10.1101/2020.02.17.20024034",null,null,"medrvix","[Objective] Analyze the occurrence of novel coronavirus pneumonia(NCP) in China mainland, explore the epidemiological rules, and evaluate the effect of prevention and control. [Methods] From December 1, 2019 to February 14, 2020, Analysis of 66,492 confirmed cases of NCP in China mainland. [Results] From December 1, 2019 to February 14, 2020, a total of 66,492 cases of NCP were confirmed in China mainland, a total of 54,406 cases were confirmed in Hubei Province, a total of 37,914 cases were confirmed in Wuhan city. On February 5, 2020, the number of suspected cases of NCP in China mainland reached a maximum of 5,328. Since then, the suspected cases have shown a significant downward trend. On February 4, 2020, the number of confirmed cases has gradually decreased in China mainland since then. On February 12, 2020, the number of confirmed cases of NCP in China mainland increased explosively to 15,152, and then began to decline. From February 3, 2020, except for Hubei Province, In China mainland, the number of newly confirmed cases of NCP has continued to decline; From December 1, 2019 to February 14, 2020, a total of 1,523 cases of NCP deaths in China mainland, a cumulative cure of 8096 cases, and mortality and cure rate were 2.29% (1523/66492) and 12.18% (8096/66492) respectively; Starting from January 27, 2020, The spread index of NCP gradually declined, and the extinction index of NCP rose little by little from January 29, 2020. [Conclusion] Starting from February 5, 2020, the number of suspected cases of the NCP is gradually decrease in China mainland. From February 3, 2020, the number of newly confirmed cases of NCP is continuous decline in China mainland except Hubei Province, This shows that the control of the transmission of NCP has achieved. Judged from the number of confirmed cases of NCP, controlling the outbreak in China mainland is to control the epidemic in Hubei province, and the key to controlling the outbreak in Hubei province is to control the epidemic in Wuhan city; Judging from the gradual increase in the number and cure rate of the NCP, and the gradual decline in the number of deaths and mortality, the trend of outbreak control and treatment is getting better; A decrease of the NCP spread index indicates a slowdown in the spread of the virus, but the cumulative confirmed cases is still increase, so the epidemic will continue for some time. However, the turning point of the epidemic in mainland China has not yet occurred. On February 12, 2020, the turning point of the epidemic in China mainland except Hubei province, has turned up.","2020-02-18","Liu youbin; Li bao hong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20024034"],["nnkholfe","541e7d329fa466fe7b961263666386dddec75894","medrxiv","A quantitative framework to define the end of an outbreak: application to Ebola Virus Disease","10.1101/2020.02.17.20024042",null,null,"medrvix","Declaring the end of an outbreak is an important step in controlling infectious disease outbreaks. An objective estimation of the probability of cases arising in the future is important to reduce the risk of post-declaration flare-ups. We developed a simulation-based model to quantify that probability. We tested it on simulated Ebola Virus Disease (EVD) data and found this probability was most sensitive to the instantaneous reproduction number, the reporting rate, and the delay between symptom onset and recovery or death of the last detected case. For EVD, our results suggest that the current WHO criterion of 42 days since the outcome of the last detected case is too short and very sensitive to underreporting. The 90 days of enhanced surveillance period after the end-of-outbreak declaration is therefore crucial to capture potential flare-ups of cases. Hence, we suggest a shift to a preliminary end-of-outbreak declaration after 63 days from the symptom onset day of the last detected case. This should be followed by a 90-day enhanced surveillance, after which the official end-of-outbreak can be declared. This corresponds to less than 5% probability of flare ups in most of the scenarios examined. Our quantitative framework could be adapted to define end-of-outbreak criteria for other infectious diseases.","2020-02-20","Bimandra A Djaafara; Natsuko Imai; Esther Hamblion; Benido Impouma; Christl A Donnelly; Anne Cori",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20024042"],["2w3bx6p8","a6d2875c8b70a41ea815511417546a0689816964","medrxiv","Estimating number of global importations of COVID-19 from Wuhan, risk of transmission outside mainland China and COVID-19 introduction index between countries outside mainland China","10.1101/2020.02.17.20024075",null,null,"medrvix","Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China. Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity. Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country. Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.","2020-02-20","Haoyang Sun; Borame Lee Dickens; Mark Chen; Alex Richard Cook; Hannah Eleanor Clapham",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20024075"],["mlkxcn5k","f0e75c4697317cbd0e3a5cdd722fe0526595db64","medrxiv","Clinical features and progression of acute respiratory distress syndrome in coronavirus disease 2019","10.1101/2020.02.17.20024166",null,null,"medrvix","Background: The outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) results in a cluster of coronavirus disease 2019 (COVID-19). We reported the clinical characteristics of COVID-19 patients with acute respiratory distress syndrome (ARDS), and further investigated the treatment and progression of ARDS in COVID-19. Methods: This study enrolled 109 patients with COVID-19 admitted to the Central Hospital of Wuhan, a designated hospital in Wuhan, China, from January 2 to February 1, 2020. Patients were followed up to February 12, 2020. The clinical data were collected from the electronic medical records. The differences in the treatment and progression with the time and the severity of ARDS were determined. Findings: Among 109 patients, mean age was 55 years, and 59 patients were male. With a median 15 days (range, 4 to 30 days) follow-up period, 31 patients (28.4%) died, while 78 (71.6%) survived and discharged. Of all patients, 53 (48.6%) developed ARDS. Compared to non-ARDS patients, ARDS patients were elder (mean age, 61 years vs. 49 years), and more likely to have the coexistent conditions, including diabetes (20.8% vs. 1.8%), cerebrovascular disease (11.3% vs. 0%), and chronic kidney disease (15.1% vs. 3.6%). Compared to mild ARDS patients, those with moderate and severe ARDS had higher mortality rates. No significant effect of antivirus, glucocorticoid, or immunoglobulin treatment on survival was observed in patients with ARDS. Interpretation: The mortality rate increased with the severity of ARDS in COVID-19, and the effects of current therapies on the survival for these patients were not satisfactory, which needs more attention from clinicians. Funding: Health and Family Planning Commission of Wuhan Municipality.","2020-02-20","Yanli Liu; Wenwu Sun; Jia Li; Liangkai Chen; Yujun Wang; Lijuan Zhang; Li Yu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20024166"],["t2x4yfrv","341fe593ea84b66223d50924bddbba252c37bc4e","medrxiv","Tracking and Predicting COVID-19 Epidemic in China Mainland","10.1101/2020.02.17.20024257",null,null,"medrvix","By proposing a varying coefficient Susceptible-Infected-Removal model (vSIR), we track the epidemic of COVID-19 in 30 provinces in China and 15 cities in Hubei province, the epicenter of the outbreak. It is found that the spread of COVID-19 has been significantly slowing down within the two weeks from January 27 to February 10th with 87.0% and 84.3% reductions in the reproduction number R0 among the 30 provinces and 15 Hubei cities, respectively. This suggests the extreme control measures implemented since January 23, which include cutting off Wuhan and many other cities and towns, a great public awareness and high level of self isolation at home, have contributed to a substantial decline in the reproductivity of the COVID-19 in China. We predict that Hubei province will reach its peak between February 20 and 22, 2020, and if the removal rate can be increased to 0.1, the epidemic outside Hubei province will end in May 2020, and inside Hubei in early June.","2020-02-20","Haoxuan Sun; Yumou Qiu; Han Yan; Yaxuan Huang; Yuru Zhu; Song Xi Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.20024257"],["t9f87kk0","a8be6ba53508e988c2c8c153478ff615b0d77687","biorxiv","Modulation of metabolic functions through Cas13d-mediated gene knockdown in liver","10.1101/2020.02.17.945014",null,null,"biorxiv","AbstractRNA knockdown in vivo carries significant potential for disease modelings and therapies. Despite the emerging approaches of CRISPR/Cas9-mediated permanent knock out of targeted genes, strategies targeting RNA for disruption are advantageous in the treatment of acquired metabolic disorders when permanent modification of the genome DNA is not appropriate, and RNA virus infection diseases when pathogenic DNA is not available (such as SARS-Cov-2 and MERS infections). Recently, Cas13d, a family of RNA-targeting CRISPR effectors, has been shown to accomplish robust down-regulation of cellular RNAs in mammalian cells in vitro. Among the various Cas13d subtypes, CasRx (RfxCas13d) showed the most potent RNA knockdown efficiency in HEK293T cells. However, the RNA-targeting activity of Cas13d still needs to be verified in vivo. In this study, the CasRx system was demonstrated to efficiently and functionally knock down genes related to metabolism functions, including Pten, Pcsk9 and lncLstr, in mouse hepatocytes. CasRx-mediated simultaneous knockdown of multiple genes was also achieved by sgRNA arrays, providing a useful strategy to modulate complex metabolism networks. Moreover, the AAV (adeno-associated virus)-mediated delivery of CasRx and Pcsk9 sgRNAs into mouse liver successfully decreased serum PCSK9, resulting in significant reduction of serum cholesterol levels. Importantly, CasRx-mediated knockdown of Pcsk9 is reversible and Pcsk9 could be repeatedly down-regulated, providing an effective strategy to reversibly modulate metabolic genes. The present work supplies a successful proof-of-concept trial that suggests efficient and regulatory knockdown of target metabolic genes for a designed metabolism modulation in the liver.","2020-02-19","Bingbing He; Wenbo Peng; Jia Huang; Hang Zhang; Yingsi Zhou; Xiali Yang; Jing Liu; Zhijie Li; Chunlong Xu; Mingxing Xue; Hui Yang; Pengyu Huang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.945014"],["mw0wb8a8",null,"biorxiv","Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins","10.1101/2020.02.17.951335",null,null,"biorxiv","AbstractThe outbreak of 2019-nCoV in the central Chinese city of Wuhan at the end of 2019 poses unprecedent public health challenges to both China and the rest world1. The new coronavirus shares high sequence identity to SARS-CoV and a newly identified bat coronavirus2. While bats may be the reservoir host for various coronaviruses, whether 2019-nCoV has other hosts is still ambiguous. In this study, one coronavirus isolated from Malayan pangolins showed 100%, 98.2%, 96.7% and 90.4% amino acid identity with 2019-nCoV in the E, M, N and S genes, respectively. In particular, the receptor-binding domain of the S protein of the Pangolin-CoV is virtually identical to that of 2019-nCoV, with one amino acid difference. Comparison of available genomes suggests 2019-nCoV might have originated from the recombination of a Pangolin-CoV-like virus with a Bat-CoV-RaTG13-like virus. Infected pangolins showed clinical signs and histopathological changes, and the circulating antibodies reacted with the S protein of 2019-nCoV. The isolation of a coronavirus that is highly related to 2019-nCoV in the pangolins suggests that these animals have the potential to act as the intermediate host of 2019-nCoV. The newly identified coronavirus in the most-trafficked mammal could represent a continuous threat to public health if wildlife trade is not effectively controlled.","2020-02-20","Kangpeng Xiao; Junqiong Zhai; Yaoyu Feng; Niu Zhou; Xu Zhang; Jie-Jian Zou; Na Li; Yaqiong Guo; Xiaobing Li; Xuejuan Shen; Zhipeng Zhang; Fanfan Shu; Wanyi Huang; Yu Li; Ziding Zhang; Rui-Ai Chen; Ya-Jiang Wu; Shi-Ming Peng; Mian Huang; Wei-Jun Xie; Qin-Hui Cai; Fang-Hui Hou; Yahong Liu; Wu Chen; Lihua Xiao; Yongyi Shen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.951335"],["sty7myg4","b705170c7527f5bc6842d90bf443fa9f9ee00bf9","biorxiv","Structure of dimeric full-length human ACE2 in complex with B0AT1","10.1101/2020.02.17.951848",null,null,"biorxiv","AbstractAngiotensin-converting enzyme 2 (ACE2) is the surface receptor for SARS coronavirus (SARS-CoV), directly interacting with the spike glycoprotein (S protein). ACE2 is also suggested to be the receptor for the new coronavirus (2019-nCoV), which is causing a serious epidemic in China manifested with severe respiratory syndrome. B0AT1 (SLC6A19) is a neutral amino acid transporter whose surface expression in intestinal cells requires ACE2. Here we present the 2.9 Å resolution cryo-EM structure of full-length human ACE2 in complex with B0AT1. The complex, assembled as a dimer of ACE2-B0AT1 heterodimers, exhibits open and closed conformations due to the shifts of the peptidase domains (PDs) of ACE2. A newly resolved Collectrin-like domain (CLD) on ACE2 mediates homo-dimerization. Structural modelling suggests that the ACE2-B0AT1 complex can bind two S proteins simultaneously, providing important clues to the molecular basis for coronavirus recognition and infection.","2020-02-18","Renhong Yan; Yuanyuan Zhang; Yaning Li; Lu Xia; Qiang Zhou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.951848"],["os758lsd",null,"biorxiv","Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine","10.1101/2020.02.17.951939",null,null,"biorxiv","A recombinant adenovirus vaccine against the SARS Coronavirus (SARS-CoV) was constructed, which contains fragments from the S, N, and Orf8 genes. Rhesus Macaques immunized with the recombinant adenovirus generated antigen-specific humoral and cellular response. Furthermore, the vaccine provided significant protection against subsequent live SARS-CoV challenge. In contrast, three out of four monkeys immunized with placebo suffered severe alveolar damage and pulmonary destruction.","2020-02-21","Yiyou Chen; Qiang Wei; Ruobing Li; Hong Gao; Hua Zhu; Wei Deng; Linlin Bao; Wei Tong; Zhe Cong; Hong Jiang; Chuan Qin",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.951939"],["xhachzby","cc5e36a8d22f6b1708b1c43764806b1722aa1754","biorxiv","X-ray Structure of Main Protease of the Novel Coronavirus SARS-CoV-2 Enables Design of α-Ketoamide Inhibitors","10.1101/2020.02.17.952879",null,null,"biorxiv","AbstractA novel coronavirus has been identified as the causative agent of a massive outbreak of atypical pneumonia originating at Wuhan, Hubei province, China. Involved in the formation of the coronavirus replication complex, the viral main protease (Mpro, also called 3CLpro) represents an attractive target for therapy. We determined the crystal structure of the unliganded Mpro at 1.75 Å resolution and used this structure to guide optimization of a series of alpha-ketoamide inhibitors. The main goal of the optimization efforts was improvement of the pharmacokinetic properties of the compounds. We further describe 1.95- and 2.20-Å crystal structures of the complex between the enzyme and the most potent alpha-ketoamide optimized this way. These structures will form the basis for further development of these compounds to antiviral drugs.","2020-02-20","Linlin Zhang; Daizong Lin; Xinyuanyuan Sun; Katharina Rox; Rolf Hilgenfeld",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.952879"],["p4q64ksa","1b4ab28a9391d516e1753874c73d2d244104f825","biorxiv","Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019","10.1101/2020.02.17.952895",null,null,"biorxiv","AbstractThe spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells1. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar2, but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1–specific neutralizing antibodies to inhibit SARS-CoV-23. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS)4-6, the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection7,8. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride9,10) or the PBM (SB2035805), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo. Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi-identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed.","2020-02-21","Intikhab alam; Allan Kamau; Maxat Kulmanov; Stefan T. Arold; Arnab Pain; Takashi Gojobori; Carlos M. Duarte",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.952895"],["hicqy5sj",null,"biorxiv","Molecular mechanism of evolution and human infection with the novel coronavirus (2019-nCoV)","10.1101/2020.02.17.952903",null,null,"biorxiv","AbstractSince December, 2019, an outbreak of pneumonia caused by the new coronavirus (2019-nCoV) has hit the city of Wuhan in the Hubei Province. With the continuous development of the epidemic, it has become a national public health crisis and calls for urgent antiviral treatments or vaccines. The spike protein on the coronavirus envelope is critical for host cell infection and virus vitality. Previous studies showed that 2019-nCoV is highly homologous to human SARS-CoV and attaches host cells though the binding of the spike receptor binding domain (RBD) domain to the angiotensin-converting enzyme II (ACE2). However, the molecular mechanisms of 2019-nCoV binding to human ACE2 and evolution of 2019-nCoV remain unclear. In this study, we have extensively studied the RBD-ACE2 complex, spike protein, and free RBD systems of 2019-nCoV and SARS-CoV using protein-protein docking and molecular dynamics (MD) simulations. It was shown that the RBD-ACE2 binding free energy for 2019-nCoV is significantly lower than that for SARS-CoV, which is consistent with the fact that 2019-nCoV is much more infectious than SARS-CoV. In addition, the spike protein of 2019-nCoV shows a significantly lower free energy than that of SARS-CoV, suggesting that 2019-nCoV is more stable and able to survive a higher temperature than SARS-CoV. This may also provide insights into the evolution of 2019-nCoV because SARS-like coronaviruses are thought to have originated in bats that are known to have a higher body-temperature than humans. It was also revealed that the RBD of 2019-nCoV is much more flexible especially near the binding site and thus will have a higher entropy penalty upon binding ACE2, compared to the RBD of SARS-CoV. That means that 2019-nCoV will be much more temperature-sensitive in terms of human infection than SARS-CoV. With the rising temperature, 2019-nCoV is expected to decrease its infection ability much faster than SARS-CoV, and get controlled more easily. The present findings are expected to be helpful for the disease prevention and control as well as drug and vaccine development of 2019-nCoV.","2020-02-21","Jiahua He; Huanyu Tao; Yumeng Yan; Sheng-You Huang; Yi Xiao",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.17.952903"],["ic2h01gm","6186828135d9e5e7e50f66567505d02e910b3ee2","medrxiv","Association between 2019-nCoV transmission and N95 respirator use","10.1101/2020.02.18.20021881",null,null,"medrvix","2019-nCoV had caused pneumonia outbreak in Wuhan. Existing evidence have confirmed the human-to-human transmission of 2019-nCoV. We retrospectively collected infection data from 2 January to 22 January at six departments from Zhongnan Hospital of Wuhan University. In our study, we found N95 respirators, disinfection and hand washing can help to reduce the risk of 2019-nCoV infection in medical staffs. Our results call for re-emphasizing strict occupational protection code in battling this novel contagious disease. The risk of 2019-nCoV infection was higher in the open area than in the quarantined area. N95 may be more effective for 2019-nCoV infections.","2020-02-19","Xinghuan Wang; Zhenyu Pan; Zhenshun Cheng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20021881"],["ahxnhutv","db142fe07544dcf0e454f8a9f39452f0aa92a6ca","medrxiv","Kidney impairment is associated with in-hospital death of COVID-19 patients","10.1101/2020.02.18.20023242",null,null,"medrvix","Background: Information on kidney impairment in patients with coronavirus disease 2019 (COVID-19) is limited. This study aims to assess the prevalence and impact of abnormal urine analysis and kidney dysfunction in hospitalized COVID-19 patients in Wuhan. Method: We conducted a consecutive cohort study of COVID-19 patients admitted in a tertiary teaching hospital with 3 branches following a major outbreak in Wuhan in 2020. Hematuria, proteinuria, serum creatinine concentration and other clinical parameters were extracted from the electronic hospitalization databases and laboratory databases. Incidence rate for acute kidney injury (AKI) was examined during the study period. Association between kidney impairment and in-hospital death was analyzed. Results: We included 710 consecutive COVID19 patients, 89 (12.3%) of whom died in hospital. The median age of the patients was 63 years (inter quartile range, 51-71), including 374 men and 336 women. On admission, 44% of patients have proteinuria hematuria and 26.9% have hematuria, and the prevalence of elevated serum creatinine and blood urea nitrogen were 15.5% and 14.1% respectively. During the study period, AKI occurred in 3.2% patients. Kaplan-Meier analysis demonstrated that patients with kidney impairment have higher risk for in-hospital death. Cox proportional hazard regression confirmed that elevated serum creatinine, elevated urea nitrogen, AKI, proteinuria and hematuria was an independent risk factor for in-hospital death after adjusting for age, sex, disease severity, leukocyte count and lymphocyte count. Conclusion: The prevalence of kidney impairment (hematuria, proteinuria and kidney dysfunction) in hospitalized COVID-19 patients was high. After adjustment for confounders, kidney impairment indicators were associated with higher risk of in-hospital death. Clinicians should increase their awareness of kidney impairment in hospitalized COVID-19 patients.","2020-02-20","Yichun Cheng; Ran Luo; Kun Wang; Meng Zhang; Zhixiang Wang; Lei Dong; Junhua Li; Ying Yao; Shuwang Ge; Gang Xu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20023242"],["uo7fb10k","b9548530d954b407b35fb266e2b4192402c982a1","medrxiv","Phase adjusted estimation of the number of 2019 novel coronavirus cases in Wuhan, China","10.1101/2020.02.18.20024281",null,null,"medrvix","An outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV / SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R0 of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of Rt. In the first assumption, Rt was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with Rt = 1.9, 2.6 or 3.1. The number of infections would reach 11,044, 70,258 and 227,989, respectively, by 29 February 2020. In the second assumption, Rt was assumed to gradually decrease at different phases from high level of transmission (Rt = 3.1, 2.6 and 1.9) to below 1 (Rt = 0.9 or 0.5) owing to increasingly implemented public heath intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077-84,520 or 55,869-81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce Rt to an ideal level and control the infection.","2020-02-23","Huwen Wang; Zezhou Wang; Yinqiao Dong; Ruijie Chang; Chen Xu; Xiaoyue Yu; Shuxian Zhang; Lhakpa Tsamlag; Meili Shang; Jinyan Huang; Ying Wang; Gang Xu; Tian Shen; Xinxin Zhang; Yong Cai",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024281"],["nzynerfu","36521caf90f471c9da1a4e84f8562440d73ead9a","medrxiv","Estimation of the epidemic properties of the 2019 novel coronavirus: A mathematical modeling study","10.1101/2020.02.18.20024315",null,null,"medrvix","Background The 2019 novel Coronavirus (COVID-19) emerged in Wuhan, China in December 2019 and has been spreading rapidly in China. Decisions about its pandemic threat and the appropriate level of public health response depend heavily on estimates of its basic reproduction number and assessments of interventions conducted in the early stages of the epidemic. Methods We conducted a mathematical modeling study using five independent methods to assess the basic reproduction number (R0) of COVID-19, using data on confirmed cases obtained from the China National Health Commission for the period 10th January to 8th February. We analyzed the data for the period before the closure of Wuhan city (10th January to 23rd January) and the post-closure period (23rd January to 8th February) and for the whole period, to assess both the epidemic risk of the virus and the effectiveness of the closure of Wuhan city on spread of COVID-19. Findings Before the closure of Wuhan city the basic reproduction number of COVID-19 was 4.38 (95% CI: 3.63-5.13), dropping to 3.41 (95% CI: 3.16-3.65) after the closure of Wuhan city. Over the entire epidemic period COVID-19 had a basic reproduction number of 3.39 (95% CI: 3.09-3.70), indicating it has a very high transmissibility. Interpretation COVID-19 is a highly transmissible virus with a very high risk of epidemic outbreak once it emerges in metropolitan areas. The closure of Wuhan city was effective in reducing the severity of the epidemic, but even after closure of the city and the subsequent expansion of that closure to other parts of Hubei the virus remained extremely infectious. Emergency planners in other cities should consider this high infectiousness when considering responses to this virus.","2020-02-20","Jinghua Li; Yijing Wang; Stuart Gilmour; Mengying Wang; Daisuke Yoneoka; Ying Wang; Xinyi You; Jing Gu; Chun Hao; Liping Peng; Zhicheng Du; Dong Roman Xu; Yuantao Hao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024315"],["yj25zkx6","4609a2dd3568a7241548b7303e0d836e33d4d5ca","medrxiv","Reduction and Functional Exhaustion of T Cells in Patients with Coronavirus Disease 2019 (COVID-19)","10.1101/2020.02.18.20024364",null,null,"medrvix","BACKGROUND The outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has posed great threat to human health, which has been declared a public health emergency of international concern (PHEIC) by the WHO. T cells play a critical role in antiviral immunity but their numbers and functional state in COVID-19 patients remain largely unclear. METHODS We retrospectively reviewed the counts of total T cells, CD4+, CD8+ T cell subsets, and serum cytokine concentration from inpatient data of 522 patients with laboratory-confirmed COVID-19, admitted into two hospitals in Wuhan from December 2019 to January 2020, and 40 healthy controls, who came to the hospitals for routine physical examination. In addition, the expression of T cell exhaustion markers PD-1 and Tim-3 were measured by flow cytometry in the peripheral blood of 14 COVID-19 cases. RESULTS The number of total T cells, CD4+ and CD8+ T cells were dramatically reduced in COVID-19 patients, especially among elderly patients (≥60 years of age) and in patients requiring Intensive Care Unit (ICU) care. Counts of total T cells, CD8+T cells or CD4+T cells lower than 800/μL, 300/μL, or 400/μL, respectively, are negatively correlated with patient survival. Statistical analysis demonstrated that T cell numbers are negatively correlated to serum IL-6, IL-10 and TNF-α concentration, with patients in decline period showing reduced IL-6, IL-10 and TNF-α concentrations and restored T cell counts. Finally, T cells from COVID-19 patients have significantly higher levels of the exhausted marker PD-1 as compared to health controls. Moreover, increasing PD-1 and Tim-3 expression on T cells could be seen as patients progressed from prodromal to overtly symptomatic stages, further indicative of T cell exhaustion. CONCLUSIONS T cell counts are reduced significantly in COVID-19 patients, and the surviving T cells appear functionally exhausted. Non-ICU patients, with total T cells, CD8+T cells CD4+T cells counts lower than 800/μL, 300/μL, and 400/μL, respectively, may still require aggressive intervention even in the immediate absence of more severe symptoms due to a high risk for further deterioration in condition.","2020-02-20","Bo Diao; Chenhui Wang; Yingjun Tan; Xiewan Chen; Ying Liu; Lifeng Ning; Li Chen; Min Li; Yueping Liu; Gang Wang; Zilin Yuan; Zeqing Feng; Yuzhang Wu; Yongwen Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024364"],["j0nm444m","fc7a6b5d1852c5ecce2d20fd0d73d5f957ed7055","medrxiv","Effective containment explains sub-exponential growth in confirmed cases of recent COVID-19 outbreak in Mainland China","10.1101/2020.02.18.20024414",null,null,"medrvix","The recent outbreak of COVID-19 in Mainland China is characterized by a distinctive algebraic, sub-exponential increase of confirmed cases with time during the early phase of the epidemic, contrasting an initial exponential growth expected for an unconstrained outbreak with sufficiently large reproduction rate. Although case counts vary significantly between affected provinces in Mainland China, the scaling law t^μ is surprisingly universal, with a range of exponents μ = 2.1 ± 0.3. The universality of this behavior indicates that, in spite of social, regional, demographical, geographical, and socio-economical heterogeneities of affected Chinese provinces, this outbreak is dominated by fundamental mechanisms that are not captured by standard epidemiological models. We show that the observed scaling law is a direct consequence of containment policies that effectively deplete the susceptible population. To this end we introduce a parsimonious model that captures both, quarantine of symptomatic infected individuals as well as population wide isolation in response to mitigation policies or behavioral changes. For a wide range of parameters, the model reproduces the observed scaling law in confirmed cases and explains the observed exponents. Quantitative fits to empirical data permit the identification of peak times in the number of asymptomatic or oligo-symptomatic, unidentified infected individuals, as well as estimates of local variations in the basic reproduction number. The model implies that the observed scaling law in confirmed cases is a direct signature of effective contaiment strategies and/or systematic behavioral changes that affect a substantial fraction of the susceptible population. These insights may aid the implementation of containment strategies in potential export induced COVID-19 secondary outbreaks elsewhere or similar future outbreaks of other emergent infectious diseases.","2020-02-20","Benjamin F Maier; Dirk Brockmann",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024414"],["28ddm9um","ba1c4c0de19352d0b9ed7db6199803b621f0a8f2","medrxiv","Psychological responses, behavioral changes and public perceptions during the early phase of the COVID-19 outbreak in China: a population based cross-sectional survey","10.1101/2020.02.18.20024448",null,null,"medrvix","Objective: To investigate psychological and behavioral responses to the threat of SARS-CoV-2 infections and their associations with public perceptions in China Design: Cross sectional population-based telephone survey via random digital dialing between 1 and 10 February, 2020 Setting: Wuhan (the epicentre and quarantined city), and Shanghai (a typical major city with close transportation link with Wuhan) Participants: Random sample of 510 residents in Wuhan and 501 residents in Shanghai aged above 18 Main outcome measures: Anxiety (measured by the 7-item generalized anxiety disorder [GAD-7] scale), recommended and avoidance behaviors (engaged in all six behaviors such as increasing surface cleaning and reducing going out). Results: The prevalence rates of moderate or severe anxiety (score ≥10 on GAD-7) were 32.7% (n=167) among Wuhan participants and 20.4% (n=102) among Shanghai participants. 78.6% (n=401) of Wuhan participants and 63.9% (n=320) of Shanghai participants had carried out all six precautionary behaviors. For both measures, Wuhan participants were more responsive to the outbreak (p<0.001). Controlling for personal characteristics, logistic regression results suggested that risks of moderate or severe anxiety were positively associated with perceived susceptibility (odds ratio 1.6, 95% confidence interval 1.3-1.8) and severity of the disease (1.6, 1.4-1.9) and confusion about information reliability (1.6, 1.5-1.9). Having confidence in taking measures to protect oneself against the disease was associated with a lower risk (0.6, 0.5-0.7). The strongest predictor of behavioral change was perceived severity (1.2, 1.1-1.4), followed by confusion about information reliability (1.1, 1.0-1.3). Conclusions: Psychological and behavioral responses to COVID-19 have been dramatic during the rising phase of the outbreak. Our results support efforts for timely dissemination of accurate and reliable information to address the high anxiety level.","2020-02-20","Mengcen Qian; Qianhui Wu; Peng Wu; Zhiyuan Hou; Yuxia Liang; Benjamin J Cowling; Hongjie Yu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024448"],["od8s0zhm","537009c0181c3269f40753f75ddca4d17ae8abae","medrxiv","Estimating the cure rate and case fatality rate of the ongoing epidemic COVID-19","10.1101/2020.02.18.20024513",null,null,"medrvix","The epidemic caused by the novel coronavirus COVID-19 in Wuhan at the end of 2019 has become an urgent public event of worldwide concern. However, due to the changing data of the epidemic, there is no scientific estimate of the cure rate and case fatality rate of the epidemic. This study proposes a method to estimate the cure rate and case fatality rate of COVID-19. The ratio of cumulative discharges on a given day to the sum of cumulative discharges on a given day and cumulative deaths before j days is used to estimate the cure rate. Moreover, the case fatality ratio can also be estimated. After simulation calculations, j is statistically appropriate when it is 8-10, and it is also clinically appropriate. When j is 9, based on the available data, it is inferred that the cure rate of this epidemic is about 93% and the case fatality rate is about 7%. This method of estimating the cure rate can be used to evaluate the effectiveness of treatment in different medical schemes and different regions, and has great value and significance for decision-making in the epidemic.","2020-02-20","Ying Diao; Xiaoyun Liu; Tao Wang; Xiaofei Zeng; Chen Dong; Changlong Zhou; Yuanming Zhang; Xuan She; Dingfu Liu; Zhongli Hu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024513"],["aoi4iqkf","9bf1677e5f822110e736076a4ba6960a80f2f348","medrxiv","Clinical characteristics of 50466 patients with 2019-nCoV infection","10.1101/2020.02.18.20024539",null,null,"medrvix","Objective: We aim to summarize reliable evidences of evidence-based medicine for the treatment and prevention of the 2019 novel coronavirus (2019-nCoV) by analyzing all the published studies on the clinical characteristics of patients with 2019-nCoV. Methods: PubMed, Cochrane Library, Embase, and other databases were searched. Several studies on the clinical characteristics of 2019-nCoV infection were collected for Meta-analysis. Results: Ten studies were included in Meta-analysis, including a total number of 50466 patients with 2019-nCoV infection. Meta-analysis shows that, among these patients, the incidence of fever was 89.1%, the incidence of cough was 72.2%, and the incidence of muscle soreness or fatigue was 42.5%. The incidence of acute respiratory distress syndrome (ARDS) was 14.8%, the incidence of abnormal chest computer tomography (CT) was 96.6%, the percentage of severe cases in all infected cases was 18.1%, and the case fatality rate of patients with 2019-nCoV infection was 4.3%. Conclusion: Fever and cough are the most common symptoms in patients with 2019-nCoV infection, and most of these patients have abnormal chest CT examination. Several people have muscle soreness or fatigue as well as ARDS. Diarrhea, hemoptysis, headache, sore throat, shock, and other symptoms only occur in a small number of patients. The case fatality rate of patients with 2019-nCoV infection is lower than that of Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS).","2020-02-23","Pengfei Sun; Shuyan Qie; Zongjan Liu; Jizhen Ren; Jianing Jianing Xi",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024539"],["tc8eru1w","672ca9b6e50e26c49fc7be2ef3b4f685badf9db1","medrxiv","Modeling and Prediction of the 2019 Coronavirus Disease Spreading in China Incorporating Human Migration Data","10.1101/2020.02.18.20024570",null,null,"medrvix","This study integrates the daily intercity migration data with the classic Susceptible-Exposed-Infected-Removed (SEIR) model to construct a new model suitable for describing the dynamics of epidemic spreading of Coronavirus Disease 2019 (COVID-19) in China. Daily intercity migration data for 367 cities in China are collected from Baidu Migration, a mobile-app based human migration tracking data system. Historical data of infected, recovered and death cases from official source are used for model fitting. The set of model parameters obtained from best data fitting using a constrained nonlinear optimization procedure is used for estimation of the dynamics of epidemic spreading in the coming weeks. Our results show that the number of infections in most cities in China will peak between mid February to early March 2020, with about 0.8%, less than 0.1% and less than 0.01% of the population eventually infected in Wuhan, Hubei Province and the rest of China, respectively.","2020-02-20","Choujun Zhan; Chi K. Tse; Yuxia Fu; Zhikang Lai; Haijun Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024570"],["toizrp89","a09c1ad0f05bbcc256cd667ad7411e6690134370","medrxiv","Association of Population Migration and Coronavirus Disease 2019 Epidemic Control","10.1101/2020.02.18.20024661",null,null,"medrvix","Background and Objective To analyze the impact of different patterns of migration flow in two cities, Hefei and Shenzhen, on the epidemic and disease control of Coronavirus Disease 2019 (COVID-19), in order to provide insight for making differentiated controlling policies. Methods We collected demographic and epidemiological information of confirmed COVID-19 cases in Hefei and Shenzhen between January 19 and February 11, 2020, from data officially published by the provincial and municipal Centers for Disease Control and Prevention (CDC). From these data we calculated basic reproduction number R0 to reflect the rate of spread of COVID-19 in these cities. Aggregated data of population migration during the same period was extracted from Baidu Migration. The change of R0 in the two cites were analyzed and compared. Spearman correlation analysis between R0 and population inflow from epidemic focus were performed. Results A total of 157 confirmed cases was identified in Hefei by 24:00 February 11, 2020, with an average age of 44.4±15.6 years, 74 female (47.1%) and 386 confirmed cases were identified in Shenzhen, with an average age of 45.15±17.99 years, 202 female (52.3%). Significant difference in the proportion of imported cases between the two cities was observed (Hefei vs Shenzhen, 24.2% vs 74.9%, p=0.000). Before January 31 2020, during the initial stage of the Level 1 Response to Major Public Health Emergencies, there was no significant association observed in Shenzhen between R0 and the proportion of population inflow from the epidemic focus (P =0.260, r=-0.452); meanwhile in Hefei, such association was strong (P =0.000, r=1.0). However, after the initial stage of response, the situation reversed. A weak association was observed in Shenzhen between be R0 and the proportion of population inflow from the epidemic focus (P=0.073, r=0.536) but not in Hefei (P =0.498, r=0.217). Conclusion Following Level 1 Response, consistent decline of R0 of COVID-19 was observed in both Hefei and Shenzhen. Different patterns of disease spread were observed in the two cities, driven by different patterns of population migration. This indicated that population migration should be taken into consideration when we set controlling policy of a novel infectious disease.","2020-02-20","Yu Ding; Sihui Luo; Xueying Zheng; Ping Ling; Tong Yue; Zhirong Liu; Jianping Weng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.20024661"],["2inlyd0t","cb3974f33a1d54369d630e62052d603148bb12e7","biorxiv","Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ?","10.1101/2020.02.18.954628",null,null,"biorxiv","AbstractThe outbreak of 2019-nCoV pneumonia (COVID-19) in the city of Wuhan, China has resulted in more than 70,000 laboratory confirmed cases, and recent studies showed that 2019-nCoV (SARS-CoV-2) could be of bat origin but involve other potential intermediate hosts. In this study, we assembled the genomes of coronaviruses identified in sick pangolins. The molecular and phylogenetic analyses showed that pangolin Coronaviruses (pangolin-CoV) are genetically related to both the 2019-nCoV and bat Coronaviruses but do not support the 2019-nCoV arose directly from the pangolin-CoV. Our study also suggested that pangolin be natural host of Betacoronavirus, with a potential to infect humans. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. Conservation of wildlife and limits of the exposures of humans to wildlife will be important to minimize the spillover risks of coronaviruses from wild animals to humans.","2020-02-20","Ping Liu; Jing-Zhe Jiang; Xiu-Feng Wan; Yan Hua; Xiaohu Wang; Fanghui Hou; Jing Chen; Jiejian Zou; Jinping Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.954628"],["ukh84gvu","c2af4db8591a8ef84229eb08359c0a50efa24544","biorxiv","Structure and immune recognition of the porcine epidemic diarrhea virus spike protein","10.1101/2020.02.18.955195",null,null,"biorxiv","AbstractPorcine epidemic diarrhea virus is an alphacoronavirus responsible for significant morbidity and mortality in pigs. A key determinant of viral tropism and entry, the PEDV spike protein is a key target for the host antibody response and a good candidate for a protein-based vaccine immunogen. We used electron microscopy to evaluate the PEDV spike structure, as well as pig polyclonal antibody responses to viral infection. The structure of the PEDV spike reveals a configuration similar to that of HuCoV-NL63. Several PEDV protein-protein interfaces are mediated by non-protein components including a glycan at Asn264 and two bound palmitoleic acid molecules. The polyclonal antibody response to PEDV infection shows a dominance of epitopes in the S1 region. This structural and immune characterization provides new insights into coronavirus spike stability determinants and explores the immune landscape of viral spike proteins.","2020-02-19","Robert N. Kirchdoerfer; Mahesh Bhandari; Olnita Martini; Leigh M. Sewall; Sandhya Bangaru; Kyoung-Jin Yoon; Andrew B. Ward",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.955195"],["deu7lrxq",null,"biorxiv","CasRx-mediated RNA targeting prevents choroidal neovascularization in a mouse model of age-related macular degeneration","10.1101/2020.02.18.955286",null,null,"biorxiv","AbstractThe smallest Cas13 family protein, CasRx, has a high cleavage activity and targeting specificity, offering attractive opportunity for therapeutic applications. Here we report that delivery of CasRx by adeno-associated virus via intravitreal injection could efficiently knockdown Vegfa transcripts and significantly reduce the area of laser-induced choroidal neovascularization in a mouse model of age-related macular degeneration. Thus, RNA-targeting CRISPR system could be used for in vivo gene therapy.","2020-02-20","Changyang Zhou; Xinde Hu; Cheng Tang; Wenjia Liu; Shaoran Wang; Yingsi Zhou; Qimeng Zhao; Qiyu Bo; Linyu Shi; Xiaodong Sun; Haibo Zhou; Hui Yang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.18.955286"],["chhemr7s",null,"medrxiv","Early Phylogenetic Estimate Of The Effective Reproduction Number Of 2019-nCoV","10.1101/2020.02.19.20024851",null,null,"medrvix","To reconstruct the evolutionary dynamics of the 2019 novel coronavirus, 52 2019−nCOV genomes available on 04 February 2020 at GISAID were analysed. The two models used to estimate the reproduction number (coalescent−based exponential growth and a birth−death skyline method) indicated an estimated mean evolutionary rate of 7.8 x 10−4 subs/site/year (range 1.1x10−4−15x10−4). The estimated R value was 2.6 (range 2.1−5.1), and increased from 0.8 to 2.4 in December 2019. The estimated mean doubling time of the epidemic was between 3.6 and 4.1 days. This study proves the usefulness of phylogeny in supporting the surveillance of emerging new infections even as the epidemic is growing.","2020-02-23","Alessia Lai; Annalisa Bergna; Carla Acciarri; Massimo Galli; Gianguglielmo Zehender",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20024851"],["5ge7ozpd",null,"medrxiv","Comparative study of the lymphocyte change between COVID-19 and non-COVID-19 pneumonia cases suggesting uncontrolled inflammation might not be the main reason of tissue injury","10.1101/2020.02.19.20024885",null,null,"medrvix","The authors have withdrawn this manuscript because the statistical methods in our manuscript need to be modified and we are going to improve the statistical methods and try to give more precise model. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.","2020-02-23","Yishan Zheng; Zhen Huang; Guoping Ying; Xia Zhang; Wei Ye; Zhiliang Hu; Chunmei Hu; Hongxia Wei; Yi Zeng; Yun Chi; Cong Cheng; Feishen Lin; Hu Lu; Lingyan Xiao; Yan Song; Chunming Wang; Yongxiang Yi; Lei Dong",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20024885"],["fmymklz6","f3ff1ecae96700f41b83d2a034a3a959428388b0","medrxiv","The cross-sectional study of hospitalized coronavirus disease 2019 patients in Xiangyang, Hubei province","10.1101/2020.02.19.20025023",null,null,"medrvix","Summary Objective To describe the epidemiological and clinical characteristics of the Coronavirus Disease 2019 (COVID-19) hospitalized patients and to offer suggestions to the urgent needs of COVID-19 prevention, diagnosis and treatment. Methods We included 102 confirmed COVID-19 cases hospitalized in Xiangyang No.1 peoples hospital, Hubei, China until Feb 9th, 2020. Demographic data, laboratory findings and chest computed tomographic (CT) images were obtained and analyzed. Findings All cases were confirmed by real-time RT-PCR, including 52 males and 50 females with a mean age of 50.38 years (SD 16.86). Incubation time ranged from one to twenty days with a mean period of 8.09 days (SD 4.99). Fever (86[84.3%] of 102 patients), cough (58[57%]), fatigue (28[27%]), shortness of breath (24[23%]), diarrhea (15[15%]), expectoration (13[12%]), inappetence (11[10%]) were common clinical manifestations. We observed a decreased blood leukocyte count and lymphopenia in 21 (20.6%) and 56 (54.9%) patients, respectively. There were 66 (68%) of 97 patients with elevated C-reactive protein levels and 49 (57.6%) of 85 with increased erythrocytes sedimentation rate. Higher levels of procalcitonin and ferritin were observed in 19 (25.3%) of 75 and 12 (92.3%) of 13 patients, respectively. Eight patients were admitted to intensive care unit (ICU), six developed respiratory failure, three had multiple organ failure and three died. The cumulative positivity rate over three rounds of real-time RT-PCR was 96%. One-hundred patients were found with typical radiological abnormalities in two rounds of chest CT scans, indicating a 98% consistency with real-time RT-PCR results. Interpretation Most COVID-19 patients in Xiangyang were secondary cases without sex difference, and the rate of severe cases and death was low. Middle-to-old-age individuals were more susceptible to the virus infection and the subsequent development of severe/fatal consequences. The average incubation period was longer among our patients. We recommend prolonging the quarantine period to three weeks. Three times real-time RT-PCR plus two times CT scans is a practical clinical diagnosis strategy at present and should be used to increase the accuracy of diagnosis, thereby controlling the source of infection more effectively. Key Words SARS-CoV-2; COVID-19; epidemiological and clinical features; diagnosis","2020-02-23","Jinwei Ai; Junwen Chen; Yong Wang; Xiaoyun Liu; Wufeng Fan; Gaojing Qu; Meiling Zhang; Shengduo Polo Pei; Bowen Tang; Shuai Yuan; Yang Li; Lisha Wang; Guoxin Huang; Bin Pei",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025023"],["dbzrd23n","eb8ac60527db35b10881cb4fd86b8a6e21983d02","medrxiv","A descriptive study of the impact of diseases control and prevention on the epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai, lessons learned for metropolis epidemics prevention","10.1101/2020.02.19.20025031",null,null,"medrvix","Objective: To describe and evaluate the impact of diseases control and prevention on epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai. Design: A retrospective descriptive study Setting: China Participants: Epidemiology information was collected from publicly accessible database. 265 patients admitted to Shanghai Public Health Center with confirmed COVID-19 were enrolled for clinical features analysis. Main outcome measure: Prevention and control measures taken by Shanghai government, epidemiological, demographic, clinical, laboratory and radiology data were collected. Weibull distribution, Chi-square test, Fisher's exact test, t test or Mann-Whitney U test were used in statistical analysis. Results: COVID-19 transmission rate within Shanghai had reduced over 99% than previous speculated, and the exponential growth has been stopped so far. Epidemic was characterized by the first stage mainly composed of imported cases and the second stage where >50% of cases were local. The incubation period was 6.4 (95% CI 5.3 to 7.6) days and the mean onset-admission interval was 5.5 days (95% CI, 5.1 to 5.9). Median time for COVID-19 progressed to severe diseases were 8.5 days (IQR: 4.8-11.0 days). By February 11th, proportion of patients being mild, moderate, severe and critically ill were 1.9%(5/265), 89.8%(238/265), 3.8%(10/265), 4.5%(12/265), respectively; 47 people in our cohort were discharged, and 1 patient died. Conclusion: Strict controlling of the transmission rate at the early stage of an epidemic in metropolis can quickly prohibit the spread of the diseases. Controlling local clusters is the key to prevent outbreaks from imported cases. Most COVID-19 severe cases progressed within 14 days of disease onset. Multiple systemic laboratory abnormalities had been observed before significant respiratory dysfunction. Keyword: COVID-19, SARS-CoV-2, epidemics dynamics, diseases control, clinical features","2020-02-23","Hongzhou Lu; Jingwen Ai; Yinzhong Shen; Yang Li; Tao Li; Xian Zhou; Haocheng Zhang; Qiran Zhang; Yun Ling; Sheng Wang; Hongping Qu; Yuan Gao; Yingchuan Li; Kanglong Yu; Duming Zhu; Hecheng Zhu; Rui Tian; Mei Zeng; Qiang Li; Yuanlin Song; Xiangyang Li; Jinfu Xu; Jie Xu; Enqiang Mao; Bijie Hu; Xin Li; Lei Zhu; Wenhong Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025031"],["hpid3fzc",null,"medrxiv","Trends and prediction in daily incidence of novel coronavirus infection in China, Hubei Province and Wuhan City: an application of Farr law","10.1101/2020.02.19.20025148",null,null,"medrvix","Background: The recent outbreak of novel coronavirus (2019-nCoV) has infected tens of thousands of patients in China. Studies have forecasted future trends of the incidence of 2019-nCoV infection, but appeared unsuccessful. Farr law is a classic epidemiology theory/practice for predicting epidemics. Therefore, we used and validated a model based on Farr law to predict the daily-incidence of 2019-nCoV infection in China and 2 regions of high-incidence. Methods: We extracted the 2019-nCoV incidence data of China, Hubei Province and Wuhan City from websites of the Chinese and Hubei health commissions. A model based on Farr law was developed using the data available on Feb. 8, 2020, and used to predict daily-incidence of 2019-nCoV infection in China, Hubei Province and Wuhan City afterward. Results: We observed 50,995 (37001 on or before Feb. 8) incident cases in China from January 16 to February 15, 2020. The daily-incidence has peaked in China, Hubei Providence and Wuhan City, but with different downward slopes. If no major changes occur, our model shows that the daily-incidence of 2019-nCoV will drop to single-digit by February 25 for China and Hubei Province, but by March 8 for Wuhan city. However, predicted 75% confidence intervals of daily-incidence in all 3 regions of interest had an upward trend. The predicted trends overall match the prospectively-collected data, confirming usefulness of these models. Conclusions: This study shows the daily-incidence of 2019-nCoV in China, Hubei Province and Wuhan City has reached the peak and was decreasing. However, there is a possibility of upward trend.","2020-02-23","Jie Xu; Yajiao Cheng; Xiaoling Yuan; Wei V. Li; Lanjing Zhang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025148"],["6kpgt70s","712f1a74a5083d690832eed5fd9ff16a04434d8d","medrxiv","Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification","10.1101/2020.02.19.20025155",null,null,"medrvix","Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed individuals or potentially aid with screening efforts in the field and potential ports of entry.","2020-02-24","Laura E Lamb; Sarah N Bartolone; Elijah Ward; Michael B Chancellor",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025155"],["02qvv8le",null,"medrxiv","Estimating the risk of 2019 Novel Coronavirus death during the course of the outbreak in China, 2020","10.1101/2020.02.19.20025163",null,null,"medrvix","Since the first case of Novel Coronavirus (2019-nCov) was identified in December 2019 in Wuhan City, China, the number of cases continues to grow across China and multiple cases have been exported to other countries. The cumulative number of reported deaths is at 637 as of February 7, 2020. Here we statistically estimated the time-delay adjusted death risk for Wuhan as well as for China excluding Wuhan to interpret the current severity of the epidemic in China. We found that the latest estimates of the death risk in Wuhan could be as high as 20% in the epicenter of the epidemic whereas we estimate it ~1% in the relatively mildly-affected areas. Because the elevated death risk estimates are likely associated with a breakdown of the medical/health system, enhanced public health interventions including social distancing and movement restrictions should be effectively implemented to bring the epidemic under control.","2020-02-23","Kenji Mizumoto; Gerardo Chowell",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025163"],["qg2o0ug4","3c94f582610453b6c1389c3a5d6ad57bcf0ba033","medrxiv","Clinical characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China","10.1101/2020.02.19.20025239",null,null,"medrvix","Summary Background The pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is a highly infectious disease, which was occurred in Wuhan, Hubei Province, China in December 2019. As of February 13, 2020, a total of 59883 cases of COVID-19 in China have been confirmed and 1368 patients have died from the disease. However, the clinical characteristics of the dyed patients were still not clearly clarified. This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality. Methods The clinical records, laboratory findings and radiologic assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data. Findings The mean age of the dead was 71.48 years, the average course of the disease was 10.56 days, all patients eventually died of respiratory failure. All of those who died had underlying diseases, the most common of which was hypertension (16/25, 64%), followed by diabetes (10/25, 40%), heart diseases (8/25, 32%), kidney diseases (5/25, 20%), cerebral infarction (4/25, 16%), chronic obstructive pulmonary disease (COPD, 2/25, 8%), malignant tumors (2/25, 8%) and acute pancreatitis (1/25, 4%). The most common organ damage outside the lungs was the heart, followed by kidney and liver. In the patients' last examination before death, white blood cell and neutrophil counts were elevated in 17 patients (17/25, 68%) and 18 patients (18/25, 72%), lymphocyte counts were decreased in 22 patients (22/25, 88%). Most patients' PCT, CRP and SAA levels were elevated, the percentages were 90.5% (19/21), 85% (19/20) and 100% (21/21) respectively. The levels of the last test of neutrophils (15/16, 93.8%), PCT (11/11, 100%), CRP (11/13, 84.6%), cTnI (8/9, 88.9%), D-Dimer (11/12, 91.6%) and LDH (9/9, 100%) were increased as compared to the first test, while the levels of lymphocytes were decreased (14/16, 87.5%). Interpretation The age and underlying diseases (hypertension, diabetes, etc.) were the most important risk factors for death of COVID-19 pneumonia. Bacterial infections may play an important role in promoting the death of patients. Malnutrition was common to severe patients. Multiple organ dysfunction can be observed, the most common organ damage was lung, followed by heart, kidney and liver. The rising of neutrophils, SAA, PCT, CRP, cTnI, D-Dimer and LDH levels can be used as indicators of disease progression, as well as the decline of lymphocytes counts.","2020-02-25","Xun Li; Luwen Wang; Shaonan Yan; Fan Yang; Longkui Xiang; Jiling Zhu; Bo Shen; Zuojiong Gong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025239"],["goqgv2cc","b463aa228141ea5c920755e57bf9c49a55adf141","medrxiv","A Note on COVID-19 Diagnosis Number Prediction Model in China","10.1101/2020.02.19.20025262",null,null,"medrvix","Importance: To predict the diagnosed COVID-19 patients and the trend of the epidemic in China. It may give the public some scientific information to ease the fear of the epidemic. Objective: In December 2019, pneumonia infected with the novel coronavirus burst in Wuhan, China. We aimed to use a mathematical model to predict number of diagnosed patients in future to ease anxiety on the emergent situation. Design: According to all diagnosis number from WHO website and combining with the transmission mode of infectious diseases, the mathematical model was fitted to predict future trend of outbreak. Setting: Our model was based on the epidemic situation in China, which could provide referential significance for disease prediction in other countries, and provide clues for prevention and intervention of relevant health authorities. Participants: In this retrospective, all diagnosis number from Jan 21 to Feb 10, 2020 reported from China was included and downloaded from WHO website. Main Outcome(s) and Measure(s): We develop a simple but accurate formula to predict the next day diagnosis number:N_i/N_(i-1) =[(N_(i-1)/N_(i-2) )]^α，where Ni is the total diagnosed patient till the ith day, and α was estimated as 0.904 at Feb 10. Results: Based on this model, it is predicted that the rate of disease infection will decrease exponentially. The total number of infected people is limited; thus, the disease will have limited impact. However, new diagnosis will last to March. Conclusions and Relevance: Through the establishment of our model, we can better predict the trend of the epidemic in China.","2020-02-23","Yi Li; Xianhong Yin; Meng Liang; Xiaoyu Liu; Meng Hao; Yi Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025262"],["368pz09b","1e45cd5feb7928bbc8bcd524d9c6b7adeae08a6b","medrxiv","SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China","10.1101/2020.02.19.20025288",null,null,"medrvix","Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.","2020-02-23","Lunwen Wang; Xun Li; Hui Chen; Shaonan Yan; Yan Li; Dong Li; Zuojiong Gong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025288"],["j0ufth5d","4e2bc97a6164191ba3d53abfa49ccd973aa80326","medrxiv","Early Prediction of Disease Progression in 2019 Novel Coronavirus Pneumonia Patients Outside Wuhan with CT and Clinical Characteristics","10.1101/2020.02.19.20025296",null,null,"medrvix","Objective: To determine the predictive value of CT and clinical characteristics for short-term disease progression in patients with 2019 novel coronavirus pneumonia (NCP). Materials and Methods: 224 patients with confirmed 2019 novel coronavirus (COVID-19) infection outside Wuhan who had chest CT examinations were retrospectively screened. Clinical data were obtained from electronic medical records. CT images were reviewed and scored for lesion distribution, lobe and segment involvement, ground-glass opacities, consolidation, and interstitial thickening. All included patients with moderate NCP were observed for at least 14 days from admission to determine whether they exacerbated to severe NCP (progressive group) or not (stable group). CT and clinical characteristics between the two groups were compared, and multivariate logistic regression and sensitivity analyses were performed to identify the risk factors for developing severe NCP. Results: A total of 141 patients with moderate NCP were included, of which 15 (10.6%) patients developed severe NCP during hospitalization and assigned to the progressive group. Multivariate logistic regression analysis showed that higher neutrophil-to-lymphocyte ratio (NLR) (odds ratio [OR] and 95% confidence interval [CI], 1.26 [1.04-1.53]; P = 0.018) and CT severity score (OR and 95% CI, 1.25 [1.08-1.46]; P = 0.004) on admission were independent predictors for progression to severe NCP, and sensitivity analysis confirmed the consistent results in nonimported patients but not in imported patients. However, no significant difference in lung involvement was found on CT between imported and nonimported patients (all P > 0.05). Patients who were admitted more than 4 days from symptom onset tended to have more severe lung involvement. Spearman correlation analysis showed the close association between CT severity score and inflammatory indexes (r = 0.17~0.47, all P < 0.05). Conclusion: CT severity score was associated with inflammatory levels and higher NLR and CT severity score on admission were independent risk factors for short-term progression in patients with NCP outside Wuhan. Furthermore, early admission and surveillance by CT should be recommended to improve clinical outcomes.","2020-02-23","Zhichao Feng; Qizhi Yu; Shanhu Yao; Lei Luo; Junhong Duan; Zhimin Yan; Min Yang; Hongpei Tan; Mengtian Ma; Ting Li; Dali Yi; Ze Mi; Huafei Zhao; Yi Jiang; Zhenhu He; Huiling Li; Wei Nie; Yin Liu; Jing Zhao; Muqing Luo; Xuanhui Liu; Pengfei Rong; Wei Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025296"],["8o3l3rsf",null,"medrxiv","Effectiveness of control strategies for Coronavirus Disease 2019: a SEIR dynamic modeling study","10.1101/2020.02.19.20025387",null,null,"medrvix","Background: Since its first cases occurrence in Wuhan, China, the Coronavirus Disease 2019 (COVID-19) has been spreading rapidly to other provinces and neighboring countries. A series of intervention strategies have been implemented, but didn't stop its spread. Methods: Two mathematical models have been developed to simulate the current epidemic situation in the city of Wuhan and in other parts of China. Special considerations were given to the mobility of people for the estimation and forecast the number of asymptomatic infections, symptomatic infections, and the infections of super-spreading events (Isse). Findings: The basic reproductive number (R0) was calculated for the period between 18 January 2020 and 16 February 2020: R0 declined from 5.75 to 1.69 in Wuhan and from 6.22 to 1.67 in the entire country (not including the Wuhan area). At the same time, Wuhan is estimated to observe a peak in the number of confirmed cases around 6 February 2020. The number of infected individuals in the entire country (not including the Wuhan area) peaked around February 3. The results also show that the peak of new asymptomatic cases per day in Wuhan occurred on February 6, and the peak of new symptomatic infections have occurred on February 3. Concurrently, while the number of confirmed cases nationwide would continue to decline, the number of real-time COVID-19 inpatients in Wuhan has reached a peak of 13,030 on February 14 before it decreases. The model further shows that the COVID-19 cases will gradually wane by the end of April 2020, both in Wuhan and the other parts of China. The number of confirmed cases would reach the single digit on March 27 in Wuhan and March 19 in the entire country. The five cities with top risk index in China with the exclusion of Wuhan are: Huanggang, Xiaogan, Jingzhou, Chongqing, and Xiangyang city. Interpretations: Although the national peak time has been reached, a significant proportion of asymptomatic patients and the infections of super-spreading events (Isse) still exist in the population, indicating the potential difficulty for the prevention and control of the disease. As the Return-to-Work tide is approaching and upgrading, further measures (e.g., escalatory quarantine, mask wearing when going out, and sit apart when taking vehicles) will be particularly crucial to stop the COVID-19 in other cities outside of Wuhan.","2020-02-23","Jinhua Pan; Ye Yao; Zhixi Liu; Mengyi Li; Ying Wang; Weizhen Dong; Haidong Kan; Weibing Wang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025387"],["h5pit94p",null,"medrxiv","Generalized anxiety disorder, depressive symptoms and sleep quality during COVID-19 epidemic in China: a web-based cross-sectional survey","10.1101/2020.02.19.20025395",null,null,"medrvix","Background: China has been severely affected by COVID-19 (Corona Virus Disease 2019) since December, 2019. This study aimed to assess the population mental health burden during the epidemic, and to explore the potential influence factors. Methods: Using a web-based cross-sectional survey, we collected data from 7,236 self-selected volunteers assessed with demographic information, COVID-19 related knowledge, Generalized Anxiety Disorder-7 (GAD-7), Center for Epidemiology Scale for Depression (CES-D), and Pittsburgh Sleep Quality Index (PSQI). Logistic regressions were used to identify influence factors associated with mental health problem. Results: Of the total sample analyzed, the overall prevalence of GAD, depressive symptoms, and sleep quality were 35.1%, 20.1%, and 18.2%, respectively. Young people reported a higher prevalence of GAD and depressive symptoms than older people (P<0.001). Compared with other occupational group, healthcare workers have the highest rate of poor sleep quality (P<0.001). Multivariate logistic regression showed that age (< 35 years) and times to focus on the COVID-19 (≥ 3 hours per day) were associated with GAD, and healthcare workers were associated with poor sleep quality. Conclusions: Our study identified a major mental health burden of the public during COVID-19 epidemic in China. Young people, people who spent too much time on the epidemic, and healthcare workers were at high risk for mental illness. Continuous surveillance and monitoring of the psychological consequences for outbreaks should become routine as part of preparedness efforts worldwide.","2020-02-23","Yeen Huang; Ning Zhao",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025395"],["st5vs6gq","7bfc745e4edcef81527e38d69589e7bcb5f725c2","medrxiv","The serial interval of COVID-19 from publicly reported confirmed cases","10.1101/2020.02.19.20025452",null,null,"medrvix","As a novel coronavirus (COVID-19) continues to emerge throughout China and threaten the globe, its transmission characteristics remain uncertain. Here, we analyze the serial intervals-the time period between the onset of symptoms in an index (infector) case and the onset of symptoms in a secondary (infectee) case-of 468 infector-infectee pairs with confirmed COVID-19 cases reported by health departments in 18 Chinese provinces between January 21, 2020, and February 8, 2020. The reported serial intervals range from -11 days to 20 days, with a mean of 3.96 days (95% confidence interval: 3.53-4.39), a standard deviation of 4.75 days (95% confidence interval: 4.46-5.07), and 12.6% of reports indicating pre-symptomatic transmission.","2020-02-23","Zhanwei Du; Xiaoke Xu; Ye Wu; Lin Wang; Benjamin J Cowling; Lauren Ancel Meyers",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.20025452"],["ao7bkcv5","aa507fcd3876f8c422993c8dd13847b1b8e77471","biorxiv","Pangolin homology associated with 2019-nCoV","10.1101/2020.02.19.950253",null,null,"biorxiv","AbstractTo explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV.","2020-02-20","Tao Zhang; Qunfu Wu; Zhigang Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.950253"],["j6y806qu","9f5625d9287d8c6729215a20add99c8cd4ba08cb","biorxiv","Potential T-cell and B-cell Epitopes of 2019-nCoV","10.1101/2020.02.19.955484",null,null,"biorxiv","As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data.Significance StatementThe novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV.","2020-02-21","Ethan Fast; Russ B. Altman; Binbin Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.955484"],["dc8mbllr",null,"biorxiv","Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor","10.1101/2020.02.19.956235",null,null,"biorxiv","AbstractA novel and highly pathogenic coronavirus (2019-nCoV) has caused an outbreak in Wuhan city, Hubei province of China since December 2019, and soon spread nationwide and spilled over to other countries around the world. To better understand the initial step of infection at atomic-level, we determined the crystal structure of the 2019-nCoV spike receptor-binding domain (RBD) bound with the cell receptor ACE2 at 2.45 Å resolution. The overall ACE2-binding mode of the 2019-nCoV RBD is nearly identical to that of the SARS-CoV RBD, which also utilizes ACE2 as the cell receptor. Structural analysis identified residues in 2019-nCoV RBD critical for ACE2 binding, and majority of which are either highly conserved or shared similar side chain properties with those in the SARS-CoV RBD. Such similarity in structure and sequence strongly argue for a convergent evolution between 2019-nCoV and SARS-CoV RBD for improved binding to ACE2 despite of being segregated in different genetic lineages in the betacoronavirus genus. The epitopes of two SARS-CoV antibodies targeting the RBD are also analyzed with the 2019-nCoV RBD, providing insights into future identification of cross-reactive antibodies.","2020-02-20","Jun Lan; Jiwan Ge; Jinfang Yu; Sisi Shan; Huan Zhou; Shilong Fan; Qi Zhang; Xuanling Shi; Qisheng Wang; Linqi Zhang; Xinquan Wang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.956235"],["yaow639d",null,"biorxiv","Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein","10.1101/2020.02.19.956581",null,null,"biorxiv","SUMMARYThe recent emergence of a novel coronavirus associated with an ongoing outbreak of pneumonia (Covid-2019) resulted in infections of more than 72,000 people and claimed over 1,800 lives. Coronavirus spike (S) glycoprotein trimers promote entry into cells and are the main target of the humoral immune response. We show here that SARS-CoV-2 S mediates entry in VeroE6 cells and in BHK cells transiently transfected with human ACE2, establishing ACE2 as a functional receptor for this novel coronavirus. We further demonstrate that the receptor-binding domains of SARS-CoV-2 S and SARS-CoV S bind with similar affinities to human ACE2, which correlates with the efficient spread of SARS-CoV-2 among humans. We found that the SARS-CoV-2 S glycoprotein harbors a furin cleavage site at the boundary between the S1/S2 subunits, which is processed during biogenesis and sets this virus apart from SARS-CoV and other SARS-related CoVs. We determined a cryo-electron microscopy structure of the SARS-CoV-2 S ectodomain trimer, demonstrating spontaneous opening of the receptor-binding domain, and providing a blueprint for the design of vaccines and inhibitors of viral entry. Finally, we demonstrate that SARS-CoV S murine polyclonal sera potently inhibited SARS-CoV-2 S-mediated entry into target cells, thereby indicating that cross-neutralizing antibodies targeting conserved S epitopes can be elicited upon vaccination.","2020-02-20","Alexandra C. Walls; Young-Jun Park; M. Alexandra Tortorici; Abigail Wall; Andrew T. McGuire; David Veesler",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.956581"],["elo7kfun","3a6544f6f247366d4c12167f7fba743e9d981c63","biorxiv","Structural basis for the recognition of the 2019-nCoV by human ACE2","10.1101/2020.02.19.956946",null,null,"biorxiv","AbstractAngiotensin-converting enzyme 2 (ACE2) has been suggested to be the cellular receptor for the new coronavirus (2019-nCoV) that is causing the coronavirus disease 2019 (COVID-19). Like other coronaviruses such as the SARS-CoV, the 2019-nCoV uses the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) to engage ACE2. We most recently determined the structure of the full-length human ACE2 in complex with a neutral amino acid transporter B0AT1. Here we report the cryo-EM structure of the full-length human ACE2 bound to the RBD of the 2019-nCoV at an overall resolution of 2.9 Å in the presence of B0AT1. The local resolution at the ACE2-RBD interface is 3.5 Å, allowing analysis of the detailed interactions between the RBD and the receptor. Similar to that for the SARS-CoV, the RBD of the 2019-nCoV is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues. Pairwise comparison reveals a number of variations that may determine the different affinities between ACE2 and the RBDs from these two related viruses.","2020-02-20","Renhong Yan; Yuanyuan Zhang; Yingying Guo; Lu Xia; Qiang Zhou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.956946"],["99sbhbuo","70c7260fc0702bd4a554421ac68972c991313e56","biorxiv","Mucin 4 Protects Female Mice from Coronavirus Pathogenesis","10.1101/2020.02.19.957118",null,null,"biorxiv","AbstractUsing incipient lines of the Collaborative Cross (CC), a murine genetic reference population, we previously identified a quantitative trait loci (QTL) associated with low SARS-CoV titer. In this study, we integrated sequence information and RNA expression of genes within the QTL to identify mucin 4 (Muc4) as a high priority candidate for controlling SARS-CoV titer in the lung. To test this hypothesis, we infected Muc4-/- mice and found that female, but not male, Muc4-/- mice developed more weight loss and disease following infection with SARS-CoV. Female Muc4-/- mice also had more difficulty breathing despite reduced lung pathology; however, no change in viral titers was observed. Comparing across viral families, studies with chikungunya virus, a mosquito-borne arthralgic virus, suggests that Muc4’s impact on viral pathogenesis may be widespread. Although not confirming the original titer QTL, our data identifies a role for Muc4 in the SARS-CoV disease and viral pathogenesis.ImportanceGiven the recent emergence of SARS-CoV-2, this work suggest that Muc4 expression plays a protective role in female mice not conserved in male mice following SARS-CoV infection. With the SARS-CoV-2 outbreak continuing, treatments that modulate or enhance Muc4 activity may provide an avenue for treatment and improved outcomes. In addition, the work highlights the importance of studying host factors including host genetics and biological sex as key parameters influencing infection and disease outcomes.","2020-02-20","Jessica A. Plante; Kenneth S. Plante; Lisa E. Gralinski; Anne Beall; Martin T. Ferris; Daniel Bottomly; Richard Green; Shannon K. McWeeney; Mark T. Heise; Ralph S. Baric; Vineet D. Menachery",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.19.957118"],["bojhhrvb",null,"medrxiv","Novel Coronavirus 2019 (Covid-19) epidemic scale estimation: topological network-based infection dynamic model","10.1101/2020.02.20.20023572",null,null,"medrvix","Backgrounds: An ongoing outbreak of novel coronavirus pneumonia (Covid-19) hit Wuhan and hundreds of cities, 29 territories globally. We present a method for scale estimation in dynamic while most of the researchers used static parameters. Methods: We use historical data and the SEIR model for important parameters assumption. And according to the timeline, we use dynamic parameters for infection topology network building. Also, the migration data is used for the Non-Wuhan area estimation which can be cross-validated for the Wuhan model. All data are from the public. Results: The estimated number of infections is 61,596 (95%CI: 58,344.02-64,847.98) by 25 Jan in Wuhan. And the estimation number of the imported cases from Wuhan of Guangzhou was 170 (95%CI: 161.27-179.26), infection scale in Guangzhou is 315 (95%CI: 109.20-520.79), while the imported cases are 168 and the scale of the infection is 339 published by the authority. Conclusions: Using dynamic network models and dynamic parameters for different time periods is an effective way of infection scale modeling.","2020-02-23","Keke Tang; Yining Huang; Meilian Chen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20023572"],["bxzprauu","373b3b78e296f0046af3506452c6934b71084b58","medrxiv","COVID-19 in Wuhan: Immediate Psychological Impact on 5062 Health Workers","10.1101/2020.02.20.20025338",null,null,"medrvix","BACKGROUND: The outbreak of COVID-19 has laid unprecedented psychological stress on health workers (HWs). We aimed to assess the immediate psychological impact on HWs at Tongji Hospital in Wuhan, China. METHODS: We conducted a single-center, cross-sectional survey of HWs via online questionnaires between February 8th and 10th, 2020. We evaluated stress, depression and anxiety by Impact of Event Scale-Revised (IES-R), Patient Health Questionnaire-9 (PHQ-9), and Generalized Anxiety Disorder 7-item (GAD-7), respectively. We also designed a questionnaire to assess the effect of psychological protective measures taken by Tongji Hospital. Multivariate logistic regression was used to identify predictors of acute stress, depression, and anxiety. RESULTS: We received 5062 completed questionnaires (response rate, 77.1 percent). 1509 (29.8 percent), 681 (13.5 percent) and 1218 (24.1 percent) HWs reported stress, depression and anxiety symptoms. Women (hazard ratio[HR], 1.31; P=0.032), years of working> 10 years (HR, 2.02; P<0.001), concomitant chronic diseases (HR, 1.51; P<0.001), history of mental disorders (HR, 3.27; P<0.001), and family members or relatives confirmed or suspected (HR, 1.23; P=0.030) were risk factors for stress, whereas care provided by hospital and department administrators(odds ratio [OR], 0.76; P=0.024) and full coverage of all departments with protective measures (OR, 0.69; P=0.004) were protective factors. CONCLUSIONS: Women and those who have more than 10 years of working, concomitant chronic diseases, history of mental disorders, and family members or relatives confirmed or suspected are susceptible to stress, depression and anxiety among HWs during the COVID-19 pandemic. Psychological protective measures implemented by the hospital could be helpful.","2020-02-23","Zhou Zhu; Shabei Xu; Hui Wang; Zheng Liu; Jianhong Wu; Guo Li; Jinfeng Miao; Chenyan Zhang; Yuan Yang; Wenzhe Sun; Suiqiang Zhu; Yebin Fan; Junbo Hu; Jihong Liu; Wei Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025338"],["eu3rry7p","f854d7b750541d65c48df953a65028a50ce57881","medrxiv","ACP risk grade: a simple mortality index for patients with confirmed or suspected severe acute respiratory syndrome coronavirus 2 disease (COVID-19) during the early stage of outbreak in Wuhan, China","10.1101/2020.02.20.20025510",null,null,"medrvix","Background: Since the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) disease (COVID-19) outbreaks in Wuhan, China, healthcare systems capacities in highly endemic areas have been overwhelmed. Approaches to efficient management are urgently needed and key to a quicker control of the outbreaks and casualties. We aimed to characterize the clinical features of hospitalized patients with confirmed or suspected COVID-19, and develop a mortality risk index for COVID-19 patients. Methods: In this retrospective one-centre cohort study, we included all the confirmed or suspected COVID-19 patients hospitalized in a COVID-19-designated hospital from January 21 to February 5, 2020. Demographic, clinical, laboratory, radiological and clinical outcome data were collected from the hospital information system, nursing records and laboratory reports. Results: Of 577 patients with at least one post-admission evaluation, the median age was 55 years (interquartile range [IQR], 39 - 66); 254 (44.0%) were men; 22.8% (100/438) were severe pneumonia on admission, and 37.7% (75/199) patients were SARS-CoV-2 positive. The clinical, laboratory and radiological data were comparable between positive and negative SARS-CoV-2 patients. During a median follow-up of 8.4 days (IQR, 5.8 - 12.0), 39 patients died with a 12-day cumulative mortality of 8.7% (95% CI, 5.9% to 11.5%). A simple mortality risk index (called ACP index), composed of Age and C-reactive Protein, was developed. By applying the ACP index, patients were categorized into three grades. The 12-day cumulative mortality in grade three (age ≥ 60 years and CRP ≥ 34 mg/L) was 33.2% (95% CI, 19.8% to 44.3%), which was significantly higher than those of grade two (age ≥ 60 years and CRP < 34 mg/L; age < 60 years and CRP ≥ 34 mg/L; 5.6% [95% CI, 0 to 11.3%]) and grade one (age < 60 years and CRP < 34 mg/L, 0%) (P <0.001), respectively. Conclusion: The ACP index can predict COVID-19 related short-term mortality, which may be a useful and convenient tool for quickly establishing a COVID-19 hierarchical management system that can greatly reduce the medical burden and therefore mortality in highly endemic areas.","2020-02-23","Jiatao Lu; Shufang Hu; Rong Fan; Zhihong Liu; Xueru Yin; Qiongya Wang; Qingquan Lv; Zhifang Cai; Haijun Li; Yuhai Hu; Ying Han; Hongping Hu; Wenyong Gao; Shibo Feng; Qiongfang Liu; Hui Li; Jian Sun; Jie Peng; Xuefeng Yi; Zixiao Zhou; Yabing Guo; Jinlin Hou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025510"],["yqiggzk9",null,"medrxiv","Clinical characteristics of 51 patients discharged from hospital with COVID-19 in Chongqing，China","10.1101/2020.02.20.20025536",null,null,"medrvix","Abstract Background：Since December 2019, Severe acute respiratory syndrome coronavirus 2（SARS-CoV-2）-infected disease (Coronavirus Disease 2019，COVID-19) emerged in Wuhan , China，and rapidly spread throughout China，even throughout the world. We try to describe the epidemiological and clinical characteristics of COVID-19 in non-Wuhan area，and explore its effective treatment. Methods：Retrospective, single-center case series of the 51 hospitalized patients with confirmed COVID-19 at Chongqing University Three Gorges Hospital in Chongqing, China, from January 20 to February 3, 2020；The discharge time was from January 29 to February 11, 2020. The main results and indicators of epidemiology, demography, clinical manifestation, laboratory examination, imaging data and treatment data of 51 patients with covid-19 were collected and analyzed. The changes of blood routine and biochemical indexes at discharge and admission were compared. Compare the clinical characteristics of severe patients (including severe and critical patients) and non- severe patients (general patients). Results： Of 51 hospitalized patients with COVID-19, the median age was 45 years (interquartile range, 34-51; range, 16-68 years) and 32 (62.7%) were men.43（84.3%）patients had been to Wuhan or Other Hubei areas outside Wuhan，and 4（7.7%） patients had a clear contact history of COVID-19 patients before the onset of the disease, and 4（7.7%） patients had no clear epidemiological history of COVID-19.Common symptoms included fever (43 [84.3%]), cough (38 [74.5%]) and fatigue (22 [43.1%]). Lymphopenia was observed in 26 patients (51.0%), and elevated C-reactive protein level in 32 patients (62.7%). Ground-glass opacity was the typical radiological finding on chest computed tomography (41 [80.4%])，Local consolidation of pneumonia in some patients(17 [33.3%]).Most of the patients were treated with traditional Chinese medicine decoction (28 [54.9%])，all of them received aerosol inhalation of recombinant human interferon a-1b for injection and oral antiviral therapy with Lopinavir and Ritonavir tablets (51 [100%]); Most of the patients were given Bacillus licheniformis capsules regulated intestinal flora treatment (44 [86.3%]). 10 patients (19.6%) received short-term (3-5 days) glucocorticoid treatment. Compared with non-severe patients (n = 44), severe patients (n = 7) were older (median age, 52 years vs 44 years), had a higher proportion of diabetes mellitus (4 [57.1%] vs 0 [0.0%]), most of them needed antibiotic treatment (7 [100%] vs 4 [9.1%], most of them needed nutritional diet (6 [85.7%) vs 0 [0.0%], and were more likely to have dyspnea (6 [85.7%] vs 5 [11.4%])，most of them needed noninvasive mechanical ventilation (6 [85.7%] vs 0 [0.0%]). Except one patient died, the remaining 50 patients were discharged according to the discharge standard, the common clinical symptoms disappeared basically, the lymphocyte increased significantly (P=0.008), CRP decreased significantly (P <0.001). The median length of stay was 12 days (IQR, 9-13). Conclusion：In 51 single center cases confirmed as COVID-19 and discharged from the hospital, 13.7% of the patients were severe. The main clinical symptoms of patients with COVID-19 were fever, cough and asthenia，Some patients had obvious dyspnea. They had clinical laboratory and radiologic characteristics. There is no specific drug treatment for the disease. For the treatment of COVID-19, in addition to oxygen inhalation and antiviral treatment, attention should be paid to the dialectical treatment of traditional Chinese medicine, regulation of intestinal flora, nutritional support treatment and other comprehensive treatment.","2020-02-23","liu lei; Gao Jian-ya",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025536"],["4pe9o59g","8d6d3be5f9353eb3358b44e646c83f1a90861b4d","medrxiv","The efficacy of convalescent plasma for the treatment of severe influenza","10.1101/2020.02.20.20025593",null,null,"medrvix","Background. Administration of convalescent plasma may be of clinical benefit for treatment of severe acute viral respiratory infections. However, no clear evidence exists to support or oppose convalescent plasma use in clinical practice. We conducted a systematic review and meta-analysis to assess the evidence of randomized controlled trials (RCTs) in the convalescent plasma for the treatment of severe influenza. Methods. Healthcare databases were searched in February 2020. All records were screened against the eligibility criteria. Data extraction and risk of bias assessments were undertaken. The primary outcome was case fatality rates by influenza. Results. We identified 5 RCTs of severe influenza. The pooled analyses showed no evidence for a reduction in mortality (Odds Ratio (OR) 1.06; p = 0.87; I2 = 35%). We also found non significant reductions in days in ICU and hospital, and days on mechanical ventilation. There seemed to have a biological benefit of increasing HAI titer levels and decreasing influenza B virus loads and cytokines after convalescent plasma treatment. No serious adverse events was reported between two groups. Studies were commonly of low risk of bias with high quality. Conclusions. Convalescent plasma appears safe but may not reduce mortality in severe influenza. This therapy should be studied within the context of a well designed clinical trial for treatment of SARS Cov 2 infection.","2020-02-23","Zhiheng Xu; Jianmeng Zhou; Yongbo Huang; Xuesong Liu; Yonghao Xu; Sibei Chen; Dongdong Liu; Zhimin Lin; Xiaoqing Liu; Yimin Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025593"],["rmia5w3x","6fcf4019b0be24071210f97eda4baba53a6a79de","medrxiv","Clinical and pathological characteristics of 2019 novel coronavirus disease (COVID-19): a systematic review","10.1101/2020.02.20.20025601",null,null,"medrvix","Importance: In 2002-2003, a severe pulmonary infectious disease occurred in Guangdong, China. The disease was caused by SARS-CoV, 17 years apart, also happen in China, and SARS-CoV-2, this epidemic has posed a significant hazard to people health both China and the whole world. Objective: Summarized the latest epidemiological changes, clinical manifestations, auxiliary examination and pathological characteristics of COVID-19. Evidence Review: PubMed database were searched from 2019 to 2020 using the index terms novel coronavirus or COVID-19 or 2019-nCoV or SARS-CoV-2 and synonyms. Articles that reported clinical characteristics, laboratory results, imageological diagnosis and pathologic condition were included and were retrospectively reviewed for these cases. This paper adopts the method of descriptive statistics. Results: 34 COVID-19-related articles were eligible for this systematic review;Four of the articles were related to pathology. We found that Fever (86.0%), cough (63.9%) and Malaise/Fatigue (34.7%) were the most common symptoms in COVID-19. But in general, the clinical symptoms and signs of COVID-19 were not obvious. Compared with SARS, COVID-19 was transmitted in a more diverse way. The mortality rates of COVID-19 were 2.5%, and the overall infection rate of healthcare worker of COVID-19 was 3.9%. We also found that the pathological features of COVID-19 have greatly similar with SARS, which manifested as ARDS. But the latest pathological examination of COVID-19 revealed the obvious mucinous secretions in the lungs. Interpretation: The clinical and pathological characteristics of SARS and COVID-19 in China are very similar, but also difference. The latest finds of pathological examination on COVID-19 may upend existing treatment schemes, so the early recognition of disease by healthcare worker is very important.","2020-02-25","mao yaqian; Wei Lin; Junping Wen; Gang Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025601"],["ofoqk100","a36bfdd1c9a1666401269fb0c08a3c2922bd6812","medrxiv","Clinical Characteristics of 24 Asymptomatic Infections with COVID-19 Screened among Close Contacts in Nanjing, China","10.1101/2020.02.20.20025619",null,null,"medrvix","Background: Previous studies have showed clinical characteristics of patients with the 2019 novel coronavirus disease (COVID-19) and the evidence of person-to-person transmission. Limited data are available for asymptomatic infections. This study aims to present the clinical characteristics of 24 cases with asymptomatic infection screened from close contacts and to show the transmission potential of asymptomatic COVID-19 virus carriers. Methods: Epidemiological investigations were conducted among all close contacts of COVID-19 patients (or suspected patients) in Nanjing, Jiangsu Province, China, from Jan 28 to Feb 9, 2020, both in clinic and in community. Asymptomatic carriers were laboratory-confirmed positive for the COVID-19 virus by testing the nucleic acid of the pharyngeal swab samples. Their clinical records, laboratory assessments, and chest CT scans were reviewed. Findings: None of the 24 asymptomatic cases presented any obvious symptoms before nucleic acid screening. Five cases (20.8%) developed symptoms (fever, cough, fatigue and etc.) during hospitalization. Twelve (50.0%) cases showed typical CT images of ground-glass chest and five (20.8%) presented stripe shadowing in the lungs. The remaining seven (29.2%) cases showed normal CT image and had no symptoms during hospitalization. These seven cases were younger (median age: 14.0 years; P = 0.012) than the rest. None of the 24 cases developed severe COVID-19 pneumonia or died. The median communicable period, defined as the interval from the first day of positive nucleic acid tests to the first day of continuous negative tests, was 9.5 days (up to 21 days among the 24 asymptomatic cases). Through epidemiological investigation, we observed a typical asymptomatic transmission to the cohabiting family members, which even caused severe COVID-19 pneumonia. Interpretation: The asymptomatic carriers identified from close contacts were prone to be mildly ill during hospitalization. However, the communicable period could be up to three weeks and the communicated patients could develop severe illness. These results highlighted the importance of close contact tracing and longitudinally surveillance via virus nucleic acid tests. Further isolation recommendation and continuous nucleic acid tests may also be recommended to the patients discharged.","2020-02-23","Zhiliang Hu; Ci Song; Chuanjun Xu; Guangfu Jin; Yaling Chen; Xin Xu; Hongxia Ma; Wei Chen; Yuan Lin; Yishan Zheng; Jianming Wang; zhibin hu; Yongxiang Yi; Hongbing Shen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025619"],["kqanoog7",null,"medrxiv","Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19","10.1101/2020.02.20.20025841",null,null,"medrvix","We report the kinetics of the immune response in relation to clinical and virological features of a patient with mild-to-moderate coronavirus disease-19 (COVID-19) requiring hospitalisation. Increased antibody-secreting cells, follicular T-helper cells, activated CD4+ and CD8+ T-cells and IgM/IgG SARS-CoV-2-binding antibodies were detected in blood, prior to symptomatic recovery. These immunological changes persisted for at least 7 days following full resolution of symptoms, indicating substantial anti-viral immunity in this non-severe COVID-19.","2020-02-23","Irani Thevarajan; Thi HO Nguyen; Marios Koutsakos; Julian Druce; Leon Caly; Carolien E van de Sandt; Xiaoxiao Jia; Suellen Nicholson; Mike Catton; Benjamin Cowie; Steven Tong; Sharon Lewin; Katherine Kedzierska",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025841"],["atnz63pk","207dcb6e3cd43cdea91d29a15be1ee34068bd54e","medrxiv","Estimating the Asymptomatic Proportion of 2019 Novel Coronavirus onboard the Princess Cruises Ship, 2020","10.1101/2020.02.20.20025866",null,null,"medrvix","The potential infectiousness of asymptomatic COVID-19 cases together with a substantial fraction of asymptomatic infections among all infections, have been highlighted in clinical studies. We conducted statistical modeling analysis to derive the delay-adjusted asymptomatic proportion of the positive COVID-19 infections onboard the Princess Cruises ship along with the timeline of infections. We estimated the asymptomatic proportion at 17.9% (95% CrI: 15.5%-20.2%), with most of the infections occurring before the start of the 2-week quarantine.","2020-02-23","Kenji Mizumoto; Katsushi Kagaya; Alexander Zarebski; Gerardo Chowell",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025866"],["s7uqawbd","bd8f57ea4aadeda075d557f566a1909d68e658c4","medrxiv","Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO","10.1101/2020.02.20.20025874",null,null,"medrvix","The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.","2020-02-24","Lin Yu; Shanshan Wu; Xiaowen Hao; Xuelong Li; Xiyang Liu; Shenglong Ye; Heng Han; Xue Dong; Xin Li; Jiyao Li; Jianmin Liu; Na Liu; Wanzhong Zhang; Vicent Pelechano; Wei-Hua Chen; Xiushan Yin",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025874"],["86opxdjd","ba3efcd6b74e55327fd7db470d824fc18943f30e","medrxiv","Evaluating the impact of international airline suspensions on the early global spread of COVID-19","10.1101/2020.02.20.20025882",null,null,"medrvix","Global airline networks play a key role in the global importation of emerging infectious diseases. Detailed information on air traffic between international airports has been demonstrated to be useful in retrospectively validating and prospectively predicting case emergence in other countries. In this paper, we use a well-established metric known as effective distance on the global air traffic data from IATA to quantify risk of emergence for different countries as a consequence of direct importation from China, and compare it against arrival times for the first 24 countries. Using this model trained on official first reports from WHO, we estimate time of arrival (ToA) for all other countries. We then incorporate data on airline suspensions to recompute the effective distance and assess the effect of such cancellations in delaying the estimated arrival time for all other countries. Finally we use the infectious disease vulnerability indices to explain some of the estimated reporting delays.","2020-02-23","Aniruddha Adiga; Srinivasan Venkatramanan; James Schlitt; Akhil Peddireddy; Allan Dickerman; Andrei Bura; Andrew Warren; Brian D Klahn; Chunhong Mao; Dawen Xie; Dustin Machi; Erin Raymond; Fanchao Meng; Golda Barrow; Henning Mortveit; Jiangzhuo Chen; Jim Walke; Joshua Goldstein; Mandy L Wilson; Mark Orr; Przemyslaw Porebski; Pyrros A Telionis; Richard Beckman; Stefan Hoops; Stephen Eubank; Young Yun Baek; Bryan Lewis; Madhav Marathe; Chris Barrett",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025882"],["w9a0jopu","9896bc65559e6406d0d3cb35e9b01b953d61959d","medrxiv","From Isolation to Coordination: How Can Telemedicine Help Combat the COVID-19 Outbreak?","10.1101/2020.02.20.20025957",null,null,"medrvix","The rapid spread of Coronavirus disease 2019 (COVID-19) presents China with a critical challenge. As normal capacity of the Chinese hospitals is exceeded, healthcare professionals struggling to manage this unprecedented crisis face the difficult question of how best to coordinate the medical resources used in highly separated locations. Responding rapidly to this crisis, the National Telemedicine Center of China (NTCC), located in Zhengzhou, Henan Province, has established the Emergency Telemedicine Consultation System (ETCS), a telemedicine-enabled outbreak alert and response network. ETCS is built upon a doctor-to-doctor (D2D) approach, in which health services can be accessed remotely through terminals across hospitals. The system architecture of ETCS comprises three major architectural layers: (1) telemedicine service platform layer, (2) telemedicine cloud layer, and (3) telemedicine service application layer. Our ETCS has demonstrated substantial benefits in terms of the effectiveness of consultations and remote patient monitoring, multidisciplinary care, and prevention education and training.","2020-02-23","Yunkai Zhai; Yichuan Wang; Minhao Zhang; Jody Hoffer Gittell; Shuai Jiang; Baozhan Chen; Fangfang Cui; Xianying He; Jie Zhao; Xiaojun Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025957"],["4gaa14ly","ae6fc64042b050df93ebb8f8045892952f18510f","medrxiv","Assessing the impact of a symptom-based mass screening and testing intervention during a novel infectious disease outbreak: The case of COVID-19","10.1101/2020.02.20.20025973",null,null,"medrvix","A symptom-based mass screening and testing intervention (MSTI) can identify a large fraction of infected individuals during an infectious disease outbreak. China is currently using this strategy for the COVID-19 outbreak. However, MSTI might lead to increased transmission if not properly implemented. We investigate under which conditions MSTI is beneficial.","2020-02-23","Yang Ge; Brian Kenneth McKay; Shengzhi Sun; Feng Zhang; Andreas Handel",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025973"],["1s0exznp",null,"medrxiv","Generation of antibodies against COVID-19 virus for development of diagnostic tools","10.1101/2020.02.20.20025999",null,null,"medrvix","The COVID-19 China coronavirus started in Dec 2019 was challenged by the lack of accurate serological diagnostic tool for this deadly disease to quickly identify and isolate the infected patients. The generation of COVID-19-specific antibodies is essential for such tasks. Here we report that polyclonal and monoclonal antibodies were generated by immunizing animals with synthetic peptides corresponding to different areas of Nucleoprotein (N) of COVID-19. The specificities of the COVID-19 antibodies were assessed by Western Blot analysis against NPs from COVID-19, MERS and SARS. Antibodies were used for immunohistochemistry staining of the tissue sections from COVID-19 infected patient, as a potential diagnostic tool. A Sandwich ELISA kit was quickly assembled for quantitation of the virus/NP of COVID-19 concentrations in the vaccine preparations. Development of POCT is also aggressively undergoing.","2020-02-25","Maohua Li; Ronghua Jin; Ya Peng; Cuiyan Wang; Wenlin Ren; Fudong Lv; Sitao Gong; Feng Fang; Qianyun Wang; Jianli Li; Tong Shen; Hunter Sun; Lei Zhou; Yali Cui; Hao Song; Le Sun",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.20025999"],["bnnt05fn","d01e580a45c457c7c200cd884e746b4d836e2ea9","biorxiv","Vulnerabilities in coronavirus glycan shields despite extensive glycosylation","10.1101/2020.02.20.957472",null,null,"biorxiv","AbstractSevere acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) coronaviruses (CoVs) are zoonotic pathogens with high fatality rates and pandemic potential. Vaccine development has focussed on the principal target of the neutralizing humoral immune response, the spike (S) glycoprotein, which mediates receptor recognition and membrane fusion. Coronavirus S proteins are extensively glycosylated viral fusion proteins, encoding around 69-87 N-linked glycosylation sites per trimeric spike. Using a multifaceted structural approach, we reveal a specific area of high glycan density on MERS S that results in the formation of under-processed oligomannose-type glycan clusters, which was absent on SARS and HKU1 CoVs. We provide a comparison of the global glycan density of coronavirus spikes with other viral proteins including HIV-1 envelope, Lassa virus glycoprotein complex, and influenza hemagglutinin, where glycosylation plays a known role in shielding immunogenic epitopes. Consistent with the ability of the antibody-mediated immune response to effectively target and neutralize coronaviruses, we demonstrate that the glycans of coronavirus spikes are not able to form an efficacious high-density global shield to thwart the humoral immune response. Overall, our data reveal how differential organisation of viral glycosylation across class I viral fusion proteins influence not only individual glycan compositions but also the immunological pressure across the viral protein surface.","2020-02-21","Yasunori Watanabe; Zachary T. Berndsen; Jayna Raghwani; Gemma E. Seabright; Joel D. Allen; Jason S. McLellan; Ian A. Wilson; Thomas A. Bowden; Andrew B. Ward; Max Crispin",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.957472"],["535lw99y","78e49fdb6f0aa9924a5b510341d52b618fff0ca6","biorxiv","A Multiscale and Comparative Model for Receptor Binding of 2019 Novel Coronavirus and the Implication of its Life Cycle in Host Cells","10.1101/2020.02.20.958272",null,null,"biorxiv","ABSTRACTThe respiratory syndrome caused by a new type of coronavirus has been emerging from China and caused more than 1000 death globally since December 2019. This new virus, called 2019 novel coronavirus (2019-nCoV) uses the same receptor called Angiotensinconverting enzyme 2 (ACE2) to attack humans as the coronavirus that caused the severe acute respiratory syndrome (SARS) seventeen years ago. Both viruses recognize ACE2 through the spike proteins (S-protein) on their surfaces. It was found that the S-protein from the SARS coronavirus (SARS-CoV) bind stronger to ACE2 than 2019-nCoV. However, function of a bio-system is often under kinetic, rather than thermodynamic, control. To address this issue, we constructed a structural model for complex formed between ACE2 and the S-protein from 2019-nCoV, so that the rate of their association can be estimated and compared with the binding of S-protein from SARS-CoV by a multiscale simulation method. Our simulation results suggest that the association of new virus to the receptor is slower than SARS, which is consistent with the experimental data obtained very recently. We further integrated this difference of association rate between virus and receptor into a mathematical model which describes the life cycle of virus in host cells and its interplay with the innate immune system. Interestingly, we found that the slower association between virus and receptor can result in longer incubation period, while still maintaining a relatively higher level of viral concentration in human body. Our computational study therefore provides, from the molecular level, one possible explanation that the new disease by far spread much faster than SARS.","2020-02-21","Zhaoqian Su; Yinghao Wu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.958272"],["0vjs2w3l","9aef9e1c22896809852ec6c94e413aff33b3d9cf","biorxiv","Inactivating porcine coronavirus before nuclei acid isolation with the temperature higher than 56 °C damages its genome integrity seriously","10.1101/2020.02.20.958785",null,null,"biorxiv","Abstract2019-Novel Coronavirus (2019-nCoV) is the pathogen of Corona Virus Disease 2019. Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis. However, the positive rate is only 30-50%. Currently, fluorescent quantitative RT-PCR technology is mainly used to detect 2019-nCoV. According to “The Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)” issued by National Health and Commission of China and “The Experts’ Consensus on Nucleic Acid Detection of 2019-nCoV” released by Chinese Society of Laboratory Medicine, the human samples must be placed under 56°C or higher to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated as the template of qRT-PCR. In this study, we demonstrated that the virus inactivation treatment disrupts its genome integrity seriously when using porcine epidemic diarrhea virus (vaccine), a kind of coronavirus, as a model. Our results showed that only 50.11% of the detectable viral templates left after the inactivation of 56 °C for 30 minutes and only 3.36% left after the inactivation of 92 °C for 5 minutes when the samples were preserved by Hank’s solution, one of an isotonic salt solutions currently suggested. However, the detectable templates of viral nucleic acids can be unchanged after the samples were incubated at 56 °C or higher if the samples were preserved with an optimized solution to protect the RNA from being disrupted. We therefore highly recommend to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV genome and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage.","2020-02-22","Qingxin Zhang; Qingshun Zhao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.20.958785"],["u6b8iwr0","233978852b45429db49d2aabe1c9c9bfe73b1884","medrxiv","CoVID-19 in Japan: What could happen in the future?","10.1101/2020.02.21.20026070",null,null,"medrvix","COVID-19 has been impacting on the whole world critically and constantly Since December 2019. We have independently developed a novel statistical time delay dynamic model on the basis of the distribution models from CCDC. Based only on the numbers of confirmed cases in different regions in China, the model can clearly reveal that the containment of the epidemic highly depends on early and effective isolation. We apply the model on the epidemic in Japan and conclude that there could be a rapid outbreak in Japan if no effective quarantine measures are carried out immediately.","2020-02-23","Nian Shao; Hanshuang Pan; Xingjie Li; Weijia Li; Shufen Wang; Yan Xuan; Yue Yan; Yu Jiang; Keji Liu; Yu Chen; Boxi Xu; Xinyue Luo; Christopher Y. Shen; Min Zhong; Xiang Xu; Xu Chen; Shuai Lu; Guanghong Ding; Jin Cheng; Wenbin Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026070"],["1mxjklgx","091a8e9a61e19e88caeb039f0e3888d111b20439","medrxiv","Epidemiological characteristics of 1212 COVID-19 patients in Henan, China","10.1101/2020.02.21.20026112",null,null,"medrvix","Based on publicly released data for 1212 patients, we investigated the epidemiological characteristics of COVID-19 in Henan of China. The following findings are obtained: 1) COVID-19 patients in Henan show gender (55% vs 45%) and age (81% aged between 21 and 60) preferences, possible causes were explored; 2) Statistical analysis on 483 patients reveals that the estimated average, mode and median incubation periods are 7.4, 4 and 7 days; Incubation periods of 92% patients were no more than 14 days; 3) The epidemic of COVID-19 in Henan has undergone three stages and showed high correlations with the numbers of patients that recently return from Wuhan; 4) Network analysis on the aggregate outbreak phenomena of COVID-19 revealed that 208 cases were clustering infected, and various people's Hospital are the main force in treating patients. The related investigations have potential implications for the prevention and control of COVID-19.","2020-02-23","Pei Wang; Junan Lu; Yanyu Jin; Mengfan Zhu; Lingling Wang; Shunjie Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026112"],["aoyyk5fl","eaf0b485f290fa884dd18de71be79d86de20eb31","medrxiv","Public Exposure to Live Animals, Behavioural Change, and Support in Containment Measures in response to COVID-19 Outbreak: a population-based cross sectional survey in China","10.1101/2020.02.21.20026146",null,null,"medrvix","Background In response to the COVID-19 outbreak, we aimed to investigate behavioural change on exposure to live animals before and during the outbreak, and public support and confidence for governmental containment measures. Methods A population-based cross-sectional telephone survey via random dialing was conducted in Wuhan (the epicentre) and Shanghai (an affected city with imported cases) between 1 and 10 February, 2020. 510 residents in Wuhan and 501 residents in Shanghai were randomly sampled. Differences of outcome measures were compared before and during the outbreak, and between two cities. Findings Proportion of respondents visiting wet markets at usual was 23.3% (119/510) in Wuhan and 20.4% (102/501) in Shanghai. During the outbreak, it decreased to 3.1% (16) in Wuhan (p<0.001), and 4.4% (22) in Shanghai (p<0.001). Proportion of those consuming wild animal products declined from 10.2% (52) to 0.6% (3) in Wuhan (p<0.001), and from 5.2% (26) to 0.8% (4) in Shanghai (p<0.001). 79.0% (403) of respondents in Wuhan and 66.9% (335) of respondents in Shanghai supported permanent closure of wet markets (P<0.001). 95% and 92% of respondents supported banning wild animal trade and quarantining Wuhan, and 75% were confident towards containment measures. Females and the more educated were more supportive for the above containment measures. Interpretation The public responded quickly to the outbreak, and reduced exposure to live animals, especially in Wuhan. With high public support in containment measures, better regulation of wet markets and healthy diets should be promoted.","2020-02-23","Zhiyuan Hou; Leesa Lin; Liang Lu; Fanxing Du; Mengcen Qian; Yuxia Liang; Juanjuan Zhang; Hongjie Yu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026146"],["t05ummop","d12b31daa935e9043a9f526a2547b6216ada7cde","medrxiv","2019 Novel Coronavirus can be detected in urine, blood, anal swabs and oropharyngeal swabs samples","10.1101/2020.02.21.20026179",null,null,"medrvix","We tested samples collected from nine patients diagnosed with coronavirus disease 2019 (COVID-19). The virus was found in urine, blood, anal swabs and oropharyngeal swabs. It is the first time for SARS-CoV-2 found in urine, though no urinary irritation was found.","2020-02-25","Liang Peng; Jing Liu; Wenxiong Xu; Qiumin Luo; Keji Deng; Bingliang Lin; Zhiliang Gao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026179"],["z4u017sa",null,"medrxiv","Comparison of throat swabs and sputum specimens for viral nucleic acid detection in 52 cases of novel coronavirus (SARS-Cov-2) infected pneumonia (COVID-19)","10.1101/2020.02.21.20026187",null,null,"medrvix","Abstract Background: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Diagnostic test based on real-time reverse transcription polymerase chain reaction assay (qRT-PCR) was the main means of confirmation, and sample collection was mostly throat swabs, which was easy to miss the diagnosis. It is necessary to seek specimen types with higher detection efficiency and accuracy. Methods: Paired specimens of throat swabs and sputum were obtained from 54 cases, and RNA was extracted and tested for 2019-nCoV (equated with SARS-CoV-2) by qRT-PCR assay. Results: The positive rates of 2019-nCoV from sputum specimens and throat swabs were 76.9% and 44.2%, respectively. Sputum specimens showed a significantly higher positive rate than throat swabs in detecting viral nucleic acid using qRT-PCR assay (P=0.001). Conclusions: The detection rates of 2019-nCoV from sputum specimens are significantly higher than throat swabs. We suggest that sputum would benefit for the detection of 2019-nCoV in patients who produce sputum. The results can facilitate the selection of specimens and increase the accuracy of diagnosis.","2020-02-23","Chenyao Lin; Jie Xiang; Mingzhe Yan; Hongze Li; Shuang Huang; Changxin Shen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026187"],["2vvqczj7",null,"medrxiv","Epidemiological Development of Novel Coronavirus Pneumonia in China and Its Forecast","10.1101/2020.02.21.20026229",null,null,"medrvix","ABSTRACT BACKGROUND AND OBJECTIVE The novel coronavirus (SARS-Cov-2) infected coronavirus disease 2019 (COVID-19) was broken out in Wuhan and Hubei province for more than a month. It severely threats people's health of thousands in Chin and even other countries. In order to prevent its wide spread, it is necessary to understand the development of the epidemic with precise mathematical language. METHODS The various data of novel coronavirus pneumonia were collected from the official websites of the National Health Committee of the People's Republic of China. According to epidemic and administrative division, three groups were divided to analyze the data, Hubei Province (including Wuhan), nationwide without Hubei and Henan Province. With classic SIR models, the fitting epidemiological curves of incidence have made, and basic reproduction number (R0) was also calculated as well. Therefore the disease's infection intensity, peak time and the epidemiological end time can be deduced.","2020-02-26","Shan shan Wu; Pan pan Sun; Rui ling Li; Liang Zhao; Yan li Wang; Li fang Jiang; Jin Bo Deng",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026229"],["ragcpbl6","87b89b038b28d2ba0bcfd2203845bf15f922c1b5","medrxiv","Evolving epidemiology of novel coronavirus diseases 2019 and possible interruption of local transmission outside Hubei Province in China: a descriptive and modeling study","10.1101/2020.02.21.20026328",null,null,"medrvix","Background The COVID-19 epidemic originated in Wuhan City of Hubei Province in December 2019 and has spread throughout China. Understanding the fast evolving epidemiology and transmission dynamics of the outbreak beyond Hubei would provide timely information to guide intervention policy. Methods We collected individual information on 8,579 laboratory-confirmed cases from official publically sources reported outside Hubei in mainland China, as of February 17, 2020. We estimated the temporal variation of the demographic characteristics of cases and key time-to-event intervals. We used a Bayesian approach to estimate the dynamics of the net reproduction number (Rt) at the provincial level. Results The median age of the cases was 44 years, with an increasing of cases in younger age groups and the elderly as the epidemic progressed. The delay from symptom onset to hospital admission decreased from 4.4 days (95%CI: 0.0-14.0) until January 27 to 2.6 days (0.0-9.0) from January 28 to February 17. The mean incubation period was estimated at 5.2 days (1.8-12.4) and the mean serial interval at 5.1 days (1.3-11.6). The epidemic dynamics in provinces outside Hubei was highly variable, but consistently included a mix of case importations and local transmission. We estimate that the epidemic was self-sustained for less than three weeks with Rt reaching peaks between 1.40 (1.04-1.85) in Shenzhen City of Guangdong Province and 2.17 (1.69-2.76) in Shandong Province. In all the analyzed locations (n=10) Rt was estimated to be below the epidemic threshold since the end of January. Conclusion Our findings suggest that the strict containment measures and movement restrictions in place may contribute to the interruption of local COVID-19 transmission outside Hubei Province. The shorter serial interval estimated here implies that transmissibility is not as high as initial estimates suggested.","2020-02-23","Juanjuan Zhang; Maria Litvinova; Wei Wang; Yan Wang; Xiaowei Deng; Xinghui Chen; Mei Li; Wen Zheng; Lan Yi; Xinhua Chen; Qianhui Wu; Yuxia Liang; Xiling Wang; Juan Yang; Kaiyuan Sun; Ira M. Longini; M. Elizabeth Halloran; Peng Wu; Benjamin J. Cowling; Stefano Merler; Cecile Viboud; Alessandro Vespignani; Marco Ajelli; Hongjie Yu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026328"],["vw3h1m5y","b842a645116edb86df997aa960699a7513f2196a","medrxiv","Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020","10.1101/2020.02.21.20026435",null,null,"medrvix","The ongoing COVID-19 epidemic that spread widely in China since December 2019 is now generating local transmission in multiple countries including Singapore as of March 5, 2020. This highlights the need to monitor in real time the transmission potential of COVID-19. In Singapore, four major COVID-19 case clusters have emerged thus far. Here we estimate the effective reproduction number, Rt, of COVID-19 in Singapore from the publicly available daily case series of imported and autochthonous cases by date of symptoms onset, after adjusting the local cases for reporting delays. We also derive the reproduction number from the distribution of cluster sizes using a branching process analysis. The effective reproduction number peaked with a mean value ~1.0 around February 6-12, 2020 and declined thereafter. As of March 5, 2020, our most recent estimate of Rt is at 0.9 (95% CI: 0.7,1.0) while an estimate of the overall R based on cluster size distribution is at 0.7 (95% CI: 0.5, 1.0). The trajectory of the reproduction number in Singapore underscore the significant effects of containment efforts in Singapore while at the same time suggest the need to sustain social distancing and active case finding efforts to stomp out all active chains of transmission.","2020-02-25","Amna Tariq; Yiseul Lee; Kimberlyn Roosa; Seth Blumberg; Ping Yan; Stefan Ma; Gerardo Chowell",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026435"],["eedk54xf","29bc34aad802ab6b35074f23c631540a84b16d0d","medrxiv","Trends in Transmissibility of 2019 Novel Coronavirus-infected Pneumonia in Wuhan and 29 Provinces in China","10.1101/2020.02.21.20026468",null,null,"medrvix","Background: The 2019 novel coronavirus infected pneumonia (COVID- 19) represents a significant public health threat. The COVID-19 emerged in December 2019 in Wuhan, China and rapidly spread to other regions and countries. The variation in transmission patterns and disease spread in regard to time or among different locations, partially reflecting the public health in- tervention effects, remains to be quantified. As most transmissibility-related epidemic parameters are unknown, we sought, with minimal assumptions, to estimate real-time transmissibility and forecast new cases using dynamic modelling. Method: Using the cases reported from the National Health Commission of China and transportation data, including the total number of travelling hours through railway, airplane, and car outbound from Wuhan, we have built a time-series model to estimate real-time underlying transmission rates of newly generated cases sequentially from January 20, 2020 to Feb 13, 2020 in Wuhan, Hubei province and other 28 provinces in China. We quantified the instantaneous transmission rate and relative reproduction number (R_t) of COVID-19, and evaluated whether public health intervention affected the case transmissibility in each province. Based on the current estimates, we have predicted the trend of disease spread with a high level of certainty. Findings: We estimated that R_t declined from the range of 4 to 5 towards 1 and remained below unity, while there was an initial growth followed by a decline in a shorter period in Hubei and other provinces. The ratio of transmission rates decreased dramatically from January 23 to 27 likely due to the rigorous public health intervention implemented by the government beginning on January 23, 2020. The mean duration of the infectious period was 6 to 9 days. We have predicted the trend of infection sizes which be- came stable in provinces around February 19 to 24, 2020, and the date of containment would be one-week later in Wuhan. Interpretation: Public health interventions implemented at both the so- cial and personal levels are effective in preventing outbreaks of COVID-19 in Wuhan and other provinces. Model prediction results suggested that COVID- 19 will be contained around the end of February 2020 in China. Keywords: Coronavirus, COVID-19, transmissibility, dynamic reproduction number R, statistical modelling, pneumonia outbreak","2020-02-25","Huazhen Lin; Wei Liu; Hong Gao; Jinyu Nie; Qiao Fan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026468"],["jydxk6j1",null,"medrxiv","Estimating the serial interval of the novel coronavirus disease (COVID-19): A statistical analysis using the public data in Hong Kong from January 16 to February 15, 2020","10.1101/2020.02.21.20026559",null,null,"medrvix","Backgrounds: The emerging virus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has caused a large outbreak of novel coronavirus disease (COVID-19) in Wuhan, China since December 2019. Based on the publicly available surveillance data, we identified 21 transmission chains in Hong Kong and estimated the serial interval (SI) of COVID-19. Methods: Index cases were identified and reported after symptoms onset, and contact tracing was conducted to collect the data of the associated secondary cases. An interval censored likelihood framework is adopted to fit a Gamma distribution function to govern the SI of COVID-19. Findings: Assuming a Gamma distributed model, we estimated the mean of SI at 4.4 days (95%CI: 2.9−6.7) and SD of SI at 3.0 days (95%CI: 1.8−5.8) by using the information of all 21 transmission chains in Hong Kong. Conclusion: The SI of COVID-19 may be shorter than the preliminary estimates in previous works. Given the likelihood that SI could be shorter than the incubation period, pre-symptomatic transmission may occur, and extra efforts on timely contact tracing and quarantine are recommended in combating the COVID-19 outbreak.","2020-02-25","Shi Zhao; Daozhou Gao; Zian Zhuang; Marc Chong; Yongli Cai; Jinjun Ran; Peihua Cao; Kai Wang; Yijun Lou; Weiming Wang; Lin Yang; Daihai He; Maggie Wang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.20026559"],["if9lar1c",null,"biorxiv","Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform","10.1101/2020.02.21.959817",null,null,"biorxiv","AbstractReverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in E. coli hosts due to size and occasional instability1-3. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the Coronaviridae, Flaviviridae and Paramyxoviridae families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in Saccharomyces cerevisiae using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-24 in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak.","2020-02-21","Tran Thi Nhu Thao; Fabien Labroussaa; Nadine Ebert; Philip V’kovski; Hanspeter Stalder; Jasmine Portmann; Jenna Kelly; Silvio Steiner; Melle Holwerda; Annika Kratzel; Mitra Gultom; Laura Laloli; Linda Hüsser; Manon Wider; Stephanie Pfaender; Dagny Hirt; Valentina Cippà; Silvia Crespo-Pomar; Simon Schröder; Doreen Muth; Daniela Niemeyer; Marcel A. Müller; Christian Drosten; Ronald Dijkman; Joerg Jores; Volker Thiel",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.959817"],["v0yzur5h","cc41c51293ed24a57ddf467e462ec9399463ee33","biorxiv","No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics","10.1101/2020.02.21.959973",null,null,"biorxiv","The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.","2020-02-22","Anton Nekrutenko; Sergei L Kosakovsky Pond",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.959973"],["ppl3xnm9",null,"biorxiv","Molecular Survey for Selected Viral Pathogens in Wild Leopard Cats (Prionailurus bengalensis) in Taiwan with an Emphasis on the Spatial and Temporal Dynamics of Carnivore Protoparvovirus 1","10.1101/2020.02.21.960492",null,null,"biorxiv","ABSTRACTThe leopard cat (Prionailurus bengalensis) has been listed as an endangered species under the Wildlife Conservation Act in Taiwan since 2009. In this study, we targeted viral pathogens, included carnivore protoparvovirus 1 (CPPV-1), feline leukemia virus (FeLV), feline immunodeficiency virus (FIV), coronavirus (CoV), and canine morbillivirus (CMV), using molecular screening. The spatial and temporal dynamics of the target pathogens were evaluated. Through sequencing and phylogenetic analysis, we aimed to clarify the phylogenetic relationship of isolated viral pathogens between leopard cats and domestic carnivores. Samples from 23 and 29 leopard cats that were live-trapped and found dead, respectively, were collected from Miaoli County from 2015 to 2019 in northwestern Taiwan. CPPV-1 and coronavirus were detected in leopard cats. The prevalence (95% confidence interval) of CPPV-1, and CoV was 63.5% (50.4%–76.6%) and 8.8% (0%–18.4%), respectively. The majority of sequences of each CPPV-1 strain amplified from Taiwanese leopard cats and domestic carnivores were identical. All the amplified CoV sequences from leopard cats were identified as feline coronavirus. The spatial and temporal aggregation of CPPV-1 infection in leopard cats was not determined in the sampling area, which indicated a wide distribution of CPPV-1 in the leopard cat habitat. We consider sympatric domestic carnivores to be the probable primary reservoir for the pathogens identified. We strongly recommend establishing efforts to manage CPPV-1 and FCoV in the leopard cat habitat, with an emphasis on vaccination programs and population control measures for free-roaming dogs and cats.IMPORTANCEThe leopard cat (Prionailurus bengalensis) is an endangered species in Taiwan. The effects of infectious diseases on the wildlife population have increasingly been recognized. In this study, we targeted highly pathogenic viral pathogens in wild cat species, included carnivore protoparvovirus 1 (CPPV-1), feline leukemia virus (FeLV), feline immunodeficiency virus (FIV), coronavirus (CoV), and canine morbillivirus (CMV), using molecular screening. Furthermore, we collected the epidemiological and phylogenetic data to understand the spatial and temporal dynamics of the target pathogens in the wild leopard cat population and identified the possible origin of target pathogens. Based on our study, we consider sympatric domestic carnivores to be the probable primary reservoir for the pathogens identified. Our study provides a deeper understanding related to the distribution of target viral pathogens in the wild leopard cats. The information is essential for leopard cat conservation and pathogen management.","2020-02-23","Chen-Chih Chen; Ai-Mei Chang; Wan-Jhen Chen; Po-Jen Chang; Yu-Ching Lai; Hsu-Hsun Lee",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.21.960492"],["o3hytzwu","54d2fd5689ac3255e8041c36b5fb333a3bd3f9b2","medrxiv","A Novel Method for the Estimation of a Dynamic Effective Reproduction Number (Dynamic-R) in the CoViD-19 Outbreak","10.1101/2020.02.22.20023267",null,null,"medrvix","The CoViD-19 outbreak has escalated to a pandemic in the last few weeks, with no signs of stopping. Pharmaceutical solutions based upon virologic studies, at this point, remain inconclusive. In contrast, this paper looks towards epidemiological models during this phase of viral growth, in particular, by providing a responsive, timely model of the R value based on the previous few days' results. Such an R value, although bearing less statistical precision due to limited sampling, could allow R to become a more effective, responsive standalone measure of infectious transmission. It demonstrates that the R value can be used as a dynamic, time-dependent indicator without the use of curve-fitting, and also estimates the most recent R-value of the CoViD-19 outbreak to be about 4.29, based on the data from the previous 3 days.","2020-02-25","Yi Chen Chong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.20023267"],["y24qtqjj","4f8d24c531d2c334969e09e4b5aed66dcc925c4b","medrxiv","Association of radiologic findings with mortality of patients infected with 2019 novel coronavirus in Wuhan, China","10.1101/2020.02.22.20024927",null,null,"medrvix","Radiologic characteristics of 2019 novel coronavirus (2019-nCoV) infected pneumonia (NCIP) which had not been fully understood are especially important for diagnosing and predicting prognosis. We retrospective studied 27 consecutive patients who were confirmed NCIP, the clinical characteristics and CT image findings were collected, and the association of radiologic findings with mortality of patients was evaluated. 27 patients included 12 men and 15 women, with median age of 60 years (IQR 47-69). 17 patients discharged in recovered condition and 10 patients died in hospital. The median age of mortality group was higher compared to survival group (68 (IQR 63-73) vs 55 (IQR 35-60), P = 0.003). The comorbidity rate in mortality group was significantly higher than in survival group (80% vs 29%, P = 0.018). The predominant CT characteristics consisted of ground glass opacity (67%), bilateral sides involved (86%), both peripheral and central distribution (74%), and lower zone involvement (96%). The median CT score of mortality group was higher compared to survival group (30 (IQR 7-13) vs 12 (IQR 11-43), P = 0.021), with more frequency of consolidation (40% vs 6%, P = 0.047) and air bronchogram (60% vs 12%, P = 0.025). An optimal cutoff value of a CT score of 24.5 had a sensitivity of 85.6% and a specificity of 84.5% for the prediction of mortality. 2019-nCoV was more likely to infect elderly people with chronic comorbidities. CT findings of NCIP were featured by predominant ground glass opacities mixed with consolidations, mainly peripheral or combined peripheral and central distributions, bilateral and lower lung zones being mostly involved. A simple CT scoring method was capable to predict mortality.","2020-02-23","Mingli Yuan; Wen Yin; Zhaowu Tao; Weijun Tan; Yi Hu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.20024927"],["pi5uq90z","5a9164999237a46d3d6aed53bd59b17d91a9e6d0","medrxiv","SARS-CoV-2 transmission in cancer patients of a tertiary hospital in Wuhan","10.1101/2020.02.22.20025320",null,null,"medrvix","In December 2019, an outbreak of atypical pneumonia known as 2019 novel coronavirus disease (COVID-19) occurred in Wuhan, China. This new type of pneumonia is characterized by rapid human-to-human transmission. Among the different disease types, cancer patients are often recalled to the hospital for treatment and disease surveillance, and the majority of cancer treatments such as chemotherapy and radiotherapy are immunosuppressive. This prompts us to consider if cancer patients were at an elevated risk of SARS-CoV-2 infection. A total of 1,524 cancer patients who were managed at our tertiary cancer institution-Zhongnan hospital of Wuhan University were reviewed during the period of Dec 30, 2019 to Feb 17, 2020. It was found that cancer patients had an estimated 2-fold increased risk of COVID-19 than the general population. We identified twelve patients who were infected with SARS-CoV-2, with two recorded deaths (16.7%), albeit one patient passed away from a COVID-19 unrelated cause. Interestingly, only five of these patients were ongoing treatment at the time of contracting the virus, suggesting that hospital visitation was the likely factor contributing to the elevated incidence in cancer patients. Moreover, we also observed that the incidence of severe COVID-19 was not higher than in the general population. Consequently, for cancer patients who require treatment, proper isolation protocols must be in place to mitigate the risk of SARS-CoV-2 infection.","2020-02-25","Jing Yu; Wen Ouyang; Melvin L.K. Chua; Conghua Xie",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.20025320"],["hth8f5sn","04030bba3035a58c7725ae267973206f6eb6c0b4","medrxiv","Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus","10.1101/2020.02.22.20025460",null,null,"medrvix","Background: The recent outbreak of infections by the 2019 novel coronavirus (2019-nCoV), the third zoonotic CoV has raised great public health concern. The demand for rapid and accurate diagnosis of this novel pathogen brought significant clinical and technological challenges. Currently, metagenomic next-generation sequencing (mNGS) and reverse-transcription PCR (RT-PCR) are the most widely used molecular diagnostics. Methods: 2019-nCoV infections were confirmed in 52 specimens by mNGS. Genomic information was analyzed and used for the design and development of an isothermal, CRISPR-based diagnostic for this novel virus. The diagnostic performance of CRISPR-nCoV was assessed and compared across three technology platforms (mNGS, RT-PCR and CRISPR). Results: 2019-nCoVs sequenced in our study were conserved with the Wuhan strain, and shared certain genetic similarity with SARS-CoV. A high degree of variation in the level of viral RNA was observed in clinical specimens. CRISPR-nCoV demonstrated a near single-copy sensitivity and great clinical sensitivity with a shorter turn-around time than RT-PCR. Conclusion: CRISPR-nCoV presents as a promising diagnostic option for the emerging pathogen.","2020-02-23","Tieying Hou; Weiqi Zeng; Minling Yang; Wenjing Chen; Lili Ren; Jingwen Ai; Ji Wu; Yalong Liao; Xuejing Gou; Yongjun Li; Xiaorui Wang; Hang Su; Jianwei Wang; Bing Gu; Teng Xu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.20025460"],["qc1mpzli",null,"medrxiv","Temperature significant change COVID-19 Transmission in 429 cities","10.1101/2020.02.22.20025791",null,null,"medrvix","Background There is no evidence supporting that temperature changes COVID-19 transmission. Methods We collected the cumulative number of confirmed cases of all cities and regions affected by COVID-19 in the world from January 20 to February 4, 2020, and calculated the daily means of the average, minimum and maximum temperatures in January. Then, restricted cubic spline function and generalized linear mixture model were used to analyze the relationships. Results There were in total 24,232 confirmed cases in China and 26 overseas countries. In total, 16,480 cases (68.01%) were from Hubei Province. The lgN rose as the average temperature went up to a peak of 8.72℃ and then slowly declined. The apexes of the minimum temperature and the maximum temperature were 6.70℃ and 12.42℃ respectively. The curves shared similar shapes. Under the circumstance of lower temperature, every 1℃ increase in average, minimum and maximum temperatures led to an increase of the cumulative number of cases by 0.83, 0.82 and 0.83 respectively. In the single-factor model of the higher-temperature group, every 1℃ increase in the minimum temperature led to a decrease of the cumulative number of cases by 0.86. Conclusion The study found that, to certain extent, temperature could significant change COVID-19 transmission, and there might be a best temperature for the viral transmission, which may partly explain why it first broke out in Wuhan. It is suggested that countries and regions with a lower temperature in the world adopt the strictest control measures to prevent future reversal.","2020-02-25","Mao Wang; Aili Jiang; Lijuan Gong; Lina Luo; Wenbin Guo; Chuyi Li; Jing Zheng; Chaoyong Li; Bixing Yang; Jietong Zeng; Youping Chen; Ke Zheng; Hongyan Li",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.20025791"],["niw61l9r","7fd337f16780aba8f1e599ce7516dc9b1d80a546","medrxiv","Neurological Manifestations of Hospitalized Patients with COVID-19 in Wuhan, China: a retrospective case series study","10.1101/2020.02.22.20026500",null,null,"medrvix","OBJECTIVE: To study the neurological manifestations of patients with coronavirus disease 2019 (COVID-19). DESIGN: Retrospective case series SETTING: Three designated COVID-19 care hospitals of the Union Hospital of Huazhong University of Science and Technology in Wuhan, China. PARTICIPANTS: Two hundred fourteen hospitalized patients with laboratory confirmed diagnosis of severe acute respiratory syndrome from coronavirus 2 (SARS-CoV-2) infection. Data were collected from 16 January 2020 to 19 February 2020. MAIN OUTCOME MEASURES: Clinical data were extracted from electronic medical records and reviewed by a trained team of physicians. Neurological symptoms fall into three categories: central nervous system (CNS) symptoms or diseases (headache, dizziness, impaired consciousness, ataxia, acute cerebrovascular disease, and epilepsy), peripheral nervous system (PNS) symptoms (hypogeusia, hyposmia, hypopsia, and neuralgia), and skeletal muscular symptoms. Data of all neurological symptoms were checked by two trained neurologists. RESULTS: Of 214 patients studied, 88 (41.1%) were severe and 126 (58.9%) were non-severe patients. Compared with non-severe patients, severe patients were older (58.7 ± 15.0 years vs 48.9 ± 14.7 years), had more underlying disorders (42 [47.7%] vs 41 [32.5%]), especially hypertension (32 [36.4%] vs 19 [15.1%]), and showed less typical symptoms such as fever (40 [45.5%] vs 92 [73%]) and cough (30 [34.1%] vs 77 [61.1%]). Seventy-eight (36.4%) patients had neurologic manifestations. More severe patients were likely to have neurologic symptoms (40 [45.5%] vs 38 [30.2%]), such as acute cerebrovascular diseases (5 [5.7%] vs 1 [0.8%]), impaired consciousness (13 [14.8%] vs 3 [2.4%]) and skeletal muscle injury (17 [19.3%] vs 6 [4.8%]). CONCLUSION: Compared with non-severe patients with COVID-19, severe patients commonly had neurologic symptoms manifested as acute cerebrovascular diseases, consciousness impairment and skeletal muscle symptoms.","2020-02-25","Ling Mao; Mengdie Wang; Shanghai Chen; Quanwei He; Jiang Chang; Candong Hong; Yifan Zhou; David Wang; Yanan Li; Huijuan Jin; Bo Hu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.20026500"],["tb37l9yp",null,"medrxiv","A Peptide-based Magnetic Chemiluminescence Enzyme Immunoassay for Serological Diagnosis of Corona Virus Disease 2019 (COVID-19)","10.1101/2020.02.22.20026617",null,null,"medrvix","A respiratory illness has been spreading rapidly in China, since its outbreak in Wuhan city, Hubei province in December 2019. The illness was caused by a novel coronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Clinical manifestations related to SARS-CoV-2 infection ranged from no symptom to fatal pneumonia. World Health Organization (WHO) named the diseases associated with SARS-CoV-2 infection as COVID-19. Real time RT-PCR is the only laboratory test available till now to confirm the infection. However, the accuracy of real time RT-PCR depends on many factors, including sampling location and of methods, quality of RNA extraction and training of operators etc. Variations in these factors might significantly lower the sensitivity of the detection. We developed a peptide-based luminescent immunoassay to detect IgG and IgM. Cut-off value of this assay was determined by the detection of 200 healthy sera and 167 sera from patients infected with other pathogens than SARS-CoV-2. To evaluate the performance of this assay, we detected IgG and IgM in the 276 sera from confirmed patients. The positive rate of IgG and IgM were 71.4% (197/276) and 57.2% (158/276) respectively. By combining with real time RT-PCR detection, this assay might help to enhance the accuracy of diagnosis of SARS-CoV-2 infection.","2020-02-25","Xuefei Cai; Juan Chen; Jieli Hu; Quanxin Long; Haijun Deng; Kai Fan; Pu Liao; Beizhong Liu; Guicheng Wu; Yaokai Chen; Zhijie Li; Kun Wang; Xiaoli Zhang; Wenguang Tian; Jianglin Xiang; Hongxin Du; Jing Wang; Yuan Hu; Ni Tang; Yong Lin; Jihua Ren; Luyi Huang; Jie Wei; Chunyang Gan; Yanmeng Chen; Qingzhu Gao; Amei Chen; Changlong He; Daoxin Wang; Peng Hu; Fachun Zhou; Ailong Huang; Ping Liu; Deqiang Wang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.20026617"],["sjod3q3j","eeae21c913759e8cf4c345e561457fe7ef84480f","medrxiv","Applying chemical reaction transition theory to predict the latent transmission dynamics of coronavirus outbreak in China","10.1101/2020.02.22.20026815",null,null,"medrvix","The recent outbreak of the Covid-19 suggests a rather long latent phase that precludes public health officials to predict the pandemic transmission on time. Here we apply mass action laws and chemical transition theory to propose a kinetic model that accounts for viral transmission dynamics at the latent phase. This model is useful for authorities to make early preventions and control measurements that stop the spread of a deadly new virus.","2020-02-25","Peng Xu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.20026815"],["dqxfcwyu","eb7fc5004191e8fce0d6e4ff98de6865872e665f","biorxiv","Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2","10.1101/2020.02.22.951178",null,null,"biorxiv","AbstractCoronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the β-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2.","2020-02-27","Tamina Park; Sang-Yeop Lee; Seil Kim; Mi Jeong Kim; Hong Gi Kim; Sangmi Jun; Seung Il Kim; Bum Tae Kim; Edmond Changkyun Park; Daeui Park",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.951178"],["nbu6bsb3","7207df5b61e93fa5a2b6e720c91aed05270e81be","biorxiv","A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2","10.1101/2020.02.22.961268",null,null,"biorxiv","1AbstractThe ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.","2020-02-27","Zhen Zhao; Haodong Cui; Wenxing Song; Xiaoling Ru; Wenhua Zhou; Xuefeng Yu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.22.961268"],["ne0qwv56","8b2e50eb3ea84225580fbdbccccb4fcd3f062feb","medrxiv","Intrinsic growth rules of patients infected, dead and cured with 2019 novel coronavirus in mainland China","10.1101/2020.02.23.20024802",null,null,"medrvix","Background An outbreak of a novel coronavirus (SARS-CoV-2)-infected pneumonia (COVID-19) was first occurred in Wuhan, China, in December 2019 and then spread rapidly to other regions. It has been declared that at least one confirmed case infected by SARS-CoV-2 was found in each province of China by late January 2020. Methods We collected the time series data of the cumulative number of confirmed infected, dead, and cured cases from the health commissions in 31 provinces in mainland China. A simple descriptive model in a logistic form was formulated to infer the intrinsic epidemic rules of COVID-19. Furthermore, we compared the intrinsic epidemic rules of COVID-19 in Hubei with that of the severe acute respiratory syndrome (SARS) in Beijing, which was obtained from the Ministry of Public Health of China in 2003. Results The feature of data of three population groups in all the provinces could be well captured (R-square>0.95) by the proposed descriptive model with slight variation among different provinces. By comparing fitted parameters, we found that the fitted curve of the infected case is the earliest to be saturated and has the lowest semi-saturation period in both COVID-19 and SARS. While all three population groups of SARS are later to saturate and have a longer semi-saturation period than that of COVID-19. Conclusions Our model is robust and stable to depict the intrinsic growth rule for the cumulative number of confirmed infected, dead, and cured cases in 30 provinces, which could depict the epidemic situation in mainland China very well. Despite the virus caused SARS (SARS-CoV) and the virus caused COVID-19 (SARS-CoV-2) are homologous, their epidemiologic characteristics are quite different. The duration of the outbreak would be shorter for COVID-19.","2020-02-25","Chuanliang Han; Yimeng Liu; Saini Yang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.20024802"],["oly7a3c2",null,"medrxiv","Conjunctival polymerase chain reaction-tests of 2019 novel coronavirus in patients in Shenyang,China","10.1101/2020.02.23.20024935",null,null,"medrvix","Purpose: The 2019 novel coronavirus(COVID-19) mainly transmitted by person-to-person through inhalation of respiratory droplets. We report the laboratory results of conjunctival PCR-tests and some clinical features of these patients in shenyang China. Design: This is a cross-sectional non-randomized study. Subjects: The study include 14 confirmly diagnosed cases, 16 suspected cases and some medical observed patients. Methods: All patients with diagnosed and suspected COVID-19 were admitted to a designated hospital in Shenyang,China. We collected conjunctival samples of these patients to do the laboratory tests by real time RT-PCR.Medical observed patients were enrolled if they had clinical symptoms. Then we analysed the PCR results and clinical data from eletronic medical records in order to find some relationships. Main Outcome Measures: Clinical condition and PCR results of conjunctival swabs compared with other specimens. Results: One of the identified case coverted from suspected case without typical clinical symptoms. Twenty-two medical observed cases were removed because none of them converted to identified cases.One of the suspected converted to identified case recently.The included cases in our study are imported cases with less underlying diseases and the severity of their infection was relatively moderate. All the conjunctival results of PCR-test were negative. Two cases had typical clinical symptoms but were finally confirmed by repeated pharynx swab tests. Conclusion: Conjunctiva may be a transmission way of COVID-19. And ocular conjunctival swabs in combination with PCR test could be a non-invasive,convenient and feasible diagnostic method for identifying the infection of COVID-19. Emphasis on the false-negative results is vital.","2020-02-25","li Xu; Xinyue Zhang; Wei Song; Baijun Sun; Jinping Mu; Xue Dong; Bing Wang",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.20024935"],["8l1vfsbc","bbba5ff9ab9eccef99a0bbc6ea557a373158bbcd","medrxiv","The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing","10.1101/2020.02.23.20026690",null,null,"medrvix","The novel coronavirus SARS-CoV-2, etiological agent of recently named Coronavirus infected disease (COVID-19) by WHO, has caused more than 2, 000 deaths worldwide since its emergency in Wuhan City, Hubei province, China, in December, 2019. The symptoms of COVID-19 varied from modest, mild to acute respiratory distress syndrome (ARDS), and the latter of which is generally associated with deregulated immune cytokine production; however, we currently know little as to the interplay between the extent of clinical symptoms and the compositions of lung immune microenvironment. Here, we comprehensively characterized the lung immune microenvironment with the bronchoalveolar lavage fluid (BALF) from 3 severe and 3 mild COVID-19 patients and 8 previously reported healthy lung controls through single-cell RNA sequence (scRNA-seq) combined with TCR-seq. Our data shows that monocyte-derived FCN1+ macrophages, whereas notFABP4+ alveolar macrophages that represent a predominant macrophage subset in BALF from patients with mild diseases, overwhelm in the severely damaged lungs from patients with ARDS. These cells are highly inflammatory and enormous chemokine producers implicated in cytokine storm. Furthermore, the formation of tissue resident, highly expanded clonal CD8+ T cells in the lung microenvironment of mild symptom patients suggests a robust adaptive immune response connected to a better control of COVID-19. This study first reported the cellular atlas of lung bronchoalveolar immune microenvironment in COVID-19 patients at the single-cell resolution, and unveiled the potential immune mechanisms underlying disease progression and protection in COVID-19.","2020-02-26","Minfeng Liao; Yang Liu; Jin Yuan; Yanling Wen; Gang Xu; Juanjuan Zhao; Lin Chen; Jinxiu Li; Xin Wang; Fuxiang Wang; Lei Liu; Shuye Zhang; Zheng Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.20026690"],["jxtch47t","a9c58042555c0d56b8e94bc5558db0c3dd697dcc","medrxiv","Epidemiologic and Clinical Characteristics of 91 Hospitalized Patients with COVID-19 in Zhejiang, China: A retrospective, multi-centre case series","10.1101/2020.02.23.20026856",null,null,"medrvix","Background Recent studies have focused initial clinical and Epidemiologic characteristics on the COVID-19, mainly revealing situation in Wuhan, Hubei. Aim To reveal more data on the epidemiologic and clinical characteristics of COVID-19 patients outside of Wuhan, in Zhejiang, China. Design Retrospective case series. Methods 88 cases of laboratory-confirmed and 3 cases of clinical-confirmed COVID-19 were admitted to five hospitals in Zhejiang province, China. Data were collected from 20 January 2020 to 11 February 2020. Results Of all 91 patients, 88 (96.70%) were laboratory-confirmed COVID-19 with throat swab samples that tested positive for SARS-Cov-2 while 3 (3.30%) were clinical-diagnosed COVID-19 cases. The median age of the patients was 50 (36.5-57) years, and female accounted for 59.34%. In this sample 40 (43.96%) patients had contracted the diseases from local cases, 31 (34.07%) patients had been to Wuhan/Hubei, 8 (8.79%) cases had contacted with people from Wuhan, 11 (12.09%) cases were confirmed aircraft transmission. In particular within the city of Ningbo, 60.52% cases can be traced back to an event held in a temple. The most common symptoms were fever (71.43%), cough (60.44%) and fatigue (43.96%). The median of incubation period was 6 (IQR, 3-8) days and the median time from first visit to a doctor to confirmed diagnosis was 1 (1-2) days. According to the Chest computed tomography scans, 67.03% cases had bilateral pneumonia. Conclusions Social activity cluster, family cluster and travel by airplane were how COVID-19 patients get transmitted and could be rapidly diagnosed COVID-19 in Zhejiang.","2020-02-25","Guo-Qing Qian; Nai-Bin Yang; Feng Ding; Ada Hoi Yan Ma; Zong-Yi Wang; Yue-Fei Shen; Chun-Wei Shi; Xiang Lian; Jin-Guo Chu; Lei Chen; Zhi-Yu Wang; Da-Wei Ren; Guo-Xiang Li; Xue-Qin Chen; Hua-Jiang Shen; Xiao-Min Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.20026856"],["oeqf7uk9","bf39a81c9aab9e0569f33d93d5e203d320dac024","medrxiv","Gender differences in patients with COVID-19: Focus on severity and mortality","10.1101/2020.02.23.20026864",null,null,"medrvix","Importance: The recent outbreak of Novel Coronavirus (SARS-CoV-2) Disease (COVID-19) has put the world on alert, that is reminiscent of the SARS outbreak seventeen years ago. Objective: We aim to compare the severity and mortality between male and female patients with both COVID-19 and SARS, to explore the most useful prognostic factors for individualized assessment. Design, Setting, and Participants: We extracted the data from a case series of 43 hospitalized patients we treated, a public data set of the first 37 cases died of COVID-19 in Wuhan city and 1019 survived patients from six cities in China. We also analyzed the data of 524 patients with SARS, including 139 deaths, from Beijing city in early 2003. Main Outcomes and Measures: Severity and mortality. Results: Older age and high number of comorbidities were associated with higher severity and mortality in patients with both COVID-19 and SARS. The percentages of older age (≥65 years) were much higher in the deceased group than in the survived group in patients with both COVID-19 (83.8 vs. 13.2, P<0.001) and SARS (37.4 vs. 4.9, P<0.001). In the case series, men tend to be more serious than women (P=0.035), although age was comparable between men and women. In the public data set, age was also comparable between men and women in the deceased group or the survived group in patients with COVID-19. Meanwhile, gender distribution was exactly symmetrical in the 1019 survivors of COVID-19. However, the percentage of male were higher in the deceased group than in the survived group (70.3 vs. 50.0, P=0.015). The gender role in mortality was also observed in SARS patients. Survival analysis showed that men (hazard ratio [95% CI] 1.47 [1.05-2.06, P= 0.025) had a significantly higher mortality rate than women in patients with SARS. Conclusions and Relevance: Older age and male gender are risk factors for worse outcome in patients with COVID. While men and women have the same susceptibility to both SARS-CoV-2 and SARS-CoV, men may be more prone to have higher severity and mortality independent of age and susceptibility.","2020-02-25","Jian-Min Jin; Peng Bai; Wei He; Fei Wu; Xiao-Fang Liu; De-Min Han; Shi Liu; Jin-Kui Yang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.20026864"],["mlyg4hj0",null,"medrxiv","Mental health status and coping strategy of medical workers in China during The COVID-19 outbreak","10.1101/2020.02.23.20026872",null,null,"medrvix","The authors have withdrawn their manuscript whilst they perform additional experiments to test some of their conclusions further. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.","2020-02-25","Chen Siyu; Min Xia; Weiping Wen; Liqian Cui; Weiqiang Yang; Shaokun Liu; Jiahua Fan Fan; Huijun Yue; Shangqing Tang; Bingjie Tang; Xiaoling Li; Lin Chen; Zili Qin; Kexing Lv; Xueqin Guo; Yu Lin; Yihui Wen; Wenxiang Gao; Ying Zheng; Wei Xu; Yun Li; Yang Xu; Li Ling; Wenbin Lei",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.20026872"],["zqsy6r4i","6bdd6867205d9b09f833f01c65f1bbfdae469164","medrxiv","Deep learning Enables Accurate Diagnosis of Novel Coronavirus (COVID-19) with CT images","10.1101/2020.02.23.20026930",null,null,"medrvix","Background A novel coronavirus (COVID-19) has emerged recently as an acute respiratory syndrome. The outbreak was originally reported in Wuhan, China, but has subsequently been spread world-widely. As the COVID-19 continues to spread rapidly across the world, computed tomography (CT) has become essentially important for fast diagnoses. Thus, it is urgent to develop an accurate computer-aided method to assist clinicians to identify COVID-19-infected patients by CT images. Materials and Methods We collected chest CT scans of 88 patients diagnosed with the COVID-19 from hospitals of two provinces in China, 101 patients infected with bacteria pneumonia, and 86 healthy persons for comparison and modeling. Based on the collected dataset, a deep learning-based CT diagnosis system (DeepPneumonia) was developed to identify patients with COVID-19. Results The experimental results showed that our model can accurately identify the COVID-19 patients from others with an excellent AUC of 0.99 and recall (sensitivity) of 0.93. In addition, our model was capable of discriminating the COVID-19 infected patients and bacteria pneumonia-infected patients with an AUC of 0.95, recall (sensitivity) of 0.96. Moreover, our model could localize the main lesion features, especially the ground-glass opacity (GGO) that is of great help to assist doctors in diagnosis. The diagnosis for a patient could be finished in 30 seconds, and the implementation on Tianhe-2 supercompueter enables a parallel executions of thousands of tasks simultaneously. An online server is available for online diagnoses with CT images by http://biomed.nscc-gz.cn/server/Ncov2019. Conclusions The established models can achieve a rapid and accurate identification of COVID-19 in human samples, thereby allowing identification of patients.","2020-02-25","Ying Song; Shuangjia Zheng; Liang Li; Xiang Zhang; Xiaodong Zhang; Ziwang Huang; Jianwen Chen; Huiying Zhao; Yusheng Jie; Ruixuan Wang; Yutian Chong; Jun Shen; Yunfei Zha; Yuedong Yang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.20026930"],["jtsxpjhc","e4701bdf5d414ea8ca38a4d8d3f69540805293b0","medrxiv","Clinical features and laboratory inspection of novel coronavirus pneumonia (COVID-19) in Xiangyang, Hubei","10.1101/2020.02.23.20026963",null,null,"medrvix","Background: Since December 2019, a novel coronavirus pneumonia (COVID-19) rapidly spread in China, reached multiple continents currently.We aimed to reveal the infectious characteristics of COVID-19 that provide more information for the research of novel coronavirus. Methods: We performed a retrospective study on the clinical characteristics of 128 COVID-19 cases with laboratory-confirmed from Xiangyang No 1 Hospitalad during January 2020 to 16 February 2020. Results: Female patients account for 53.1%. The aged below 20 years that accounts for 1.6% of overall patients. The aged in 21~50, 51~65, over 66 years were accounts for 44.5%, 35.1%,18.8%, respectively. In the difference age spectrum, all severe groups compared with non-severe groups were difference significantly ( P < 0.01 ). Fever ( 89.8% ) and Cough ( 67.2% ) were common clinical symptoms. The rate of patients with sore throats (14.1%) was rare. The rate of chest computed tomography scan showing ground glass opacity in overall, non-severe, severe groups were 63.3%, 60.7%, 76.2%, respectively. White blood cell counts in the normal range of overall patients, but severe group patients were increased significantly ( P < 0.01). Lymphocytes of overall patients were decreased. Alanine transaminase (ALT) and aspartate transaminase (AST) in the normal range of overall patients, but its were elevated in the severe group. Creatinine (CR) and blood urea nitrogen (BUN) of overall patients in the normal range. C-reactive protein (CRP) level of all patients were increased markedly, but it in the severe group was significantly higher than that in the non-severe group ( P < 0.01 ). Conclusions: Our data provide more information that advanced age, lower lymphocytes levels at the diagnosed COVID-19 patients may be a risk factor for unfavourable prognosis. The white blood cells and C-reactive protein level elevated in severe COVID-19 patients may be accompanying bacterial infection. 2019-nCov may be carries a risk factor of impaired liver and kidney function.","2020-02-25","Weiliang Cao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.20026963"],["25va2cvt","09b6706748f0c1ae0da436ac2dfac9052b84e4ea","biorxiv","Cryo-EM structures of HKU2 and SADS-CoV spike glycoproteins and insights into coronavirus evolution","10.1101/2020.02.23.961912",null,null,"biorxiv","AbstractA new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.","2020-02-24","Jinfang Yu; Shuyuan Qiao; Runyu Guo; Xinquan Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.23.961912"],["nqn1qzcu","1d882d90109fb4496088342cf2bcaa4fec40cf9f","medrxiv","Inferring transmission trees to guide targeting of interventions against visceral leishmaniasis and post-kala-azar dermal leishmaniasis","10.1101/2020.02.24.20023325",null,null,"medrvix","Understanding of spatiotemporal transmission of infectious diseases has improved significantly in recent years. Advances in Bayesian inference methods for individual-level geo-located epidemiological data have enabled reconstruction of transmission trees and quantification of disease spread in space and time, while accounting for uncertainty in missing data. However, these methods have rarely been applied to endemic diseases or ones in which asymptomatic infection plays a role, for which novel estimation methods are required. Here, we develop such methods to analyse longitudinal incidence data on visceral leishmaniasis (VL), and its sequela, post-kala-azar dermal leishmaniasis (PKDL), in a highly endemic community in Bangladesh. Incorporating recent data on infectiousness of VL and PKDL, we show that while VL cases drive transmission when incidence is high, the contribution of PKDL increases significantly as VL incidence declines (reaching 55% in this setting). Transmission is highly focal: >85% of mean distances from inferred infectors to their secondary VL cases were <300m, and estimated average times from infector onset to secondary case infection were <4 months for 90% of VL infectors, but up to 2.75yrs for PKDL infectors. Estimated numbers of secondary VL cases per VL and PKDL case varied from 0-6 and were strongly correlated with the infector's duration of symptoms. Counterfactual simulations suggest that prevention of PKDL could have reduced VL incidence by up to a quarter. These results highlight the need for prompt detection and treatment of PKDL to achieve VL elimination in the Indian subcontinent and provide quantitative estimates to guide spatiotemporally-targeted interventions against VL.","2020-02-25","Lloyd A. C. Chapman; Simon E. F. Spencer; Timothy M. Pollington; Chris P. Jewell; Dinesh Mondal; Jorge Alvar; T. Deirdre Hollingsworth; Mary M. Cameron; Caryn Bern; Graham F. Medley",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20023325"],["f21dknmb","01d162d7fae6aaba8e6e60e563ef4c2fca7b0e18","medrxiv","TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection","10.1101/2020.02.24.20025437",null,null,"medrvix","Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the \"coronavirus\" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.","2020-02-26","Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Yiqing Xue; Chunli Zhang; Guangming Tan; Gang Niu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20025437"],["r31pgnu0","47f65bff75b7752a38ccbb540fbd74955b3addea","medrxiv","Demonstration of four entities of appendicitis in China through studying cluster/outbreak","10.1101/2020.02.24.20026021",null,null,"medrvix","Objective: To differentiate different entities of appendicitis through studying cluster/outbreak, ascertain common setting of cluster/outbreak, and provide the epidemiological evidences of infectious etiology of appendicitis. Background: Differential diagnosis and management for perforated appendicitis and non-perforated appendicitis and the infectious etiology of appendicitis are current hot topics. Methods: Field investigation for Tibetan students were carried out and reports published in English and Chinese medical journals were reviewed. Results: The literature review included 473 patients in 7 cluster/outbreaks of appendicitis in 6 provinces and autonomous regions. All the clusters/outbreaks occurred in group living units. We found two classic entities of appendicitis with natural history from non perforated appendicitis to perforated appendicitis and two entities of non-perforated appendicitis . In classic entities, one may represent majority of sporadic patients and the other may represent partial sporadic patients with obvious gastrointestinal manifestation. In entities of non-perforated appendicitis, one was identical to features of sporadic non-perforated appendicitis and the other one is identical to the following Tibetan students and associated with Fusobacterium. The field investigation for 120 Tibetan students with appendicitis showed that the resected appendices exhibited diffuse or focal hemorrhages and infiltration by eosinophils and by lymphocytes. Most patients had normal body temperature, white blood cell count and neutrophil count. This is a new entity of appendicitis. The clusters/outbreaks of appendicitis showed the features of infectious disease in epidemiology. The entity of perforated appendicitis was not found. Conclusion: Studying cluster/outbreak is a good method to differentiate different entities of appendicitis and infectious etiology.","2020-02-26","Yi-Tian Guo; De-qiang Ye; Gui-fang Yang; Guo-zhen Liu; Xiao-chen Cui; Shi-Yun Tan; Yi Guo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20026021"],["g3asp1ps","99a46ba28f5811c112876ec77f570d6c10778db6","medrxiv","WeChat, a Chinese social media, may early detect the SARS-CoV-2 outbreak in 2019","10.1101/2020.02.24.20026682",null,null,"medrvix","We plotted daily data on the frequencies of keywords related to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from WeChat, a Chinese social media. Using 'Feidian', Chinese abbreviation for SARS, may detect the SARS-CoV-2 outbreak in 2019 two weeks earlier. WeChat offered a new approach to early detect disease outbreaks.","2020-02-26","Wenjun Wang; Yikai Wang; Xin Zhang; Yaping Li; Xiaoli Jia; Shuangsuo Dang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20026682"],["rr7r7ef0","0fd300aefb704c20f32152b97b6194015f1c74e7","medrxiv","Characterizing the transmission and identifying the control strategy for COVID-19 through epidemiological modeling","10.1101/2020.02.24.20026773",null,null,"medrvix","The outbreak of the novel coronavirus disease, COVID-19, originating from Wuhan, China in early December, has infected more than 70,000 people in China and other countries and has caused more than 2,000 deaths. As the disease continues to spread, the biomedical society urgently began identifying effective approaches to prevent further outbreaks. Through rigorous epidemiological analysis, we characterized the fast transmission of COVID-19 with a basic reproductive number 5.6 and proved a sole zoonotic source to originate in Wuhan. No changes in transmission have been noted across generations. By evaluating different control strategies through predictive modeling and Monte carlo simulations, a comprehensive quarantine in hospitals and quarantine stations has been found to be the most effective approach. Government action to immediately enforce this quarantine is highly recommended.","2020-02-25","Ke K. Zhang; Linglin Xie; Lauren Lawless; Huijuan Zhou; Guannan Gao; Chengbin Xue",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20026773"],["tkvgzbuk","5b1a2b3207e3f674b94f6f271b92c29f3a8a6754","medrxiv","Clinical and radiographic features of cardiac injury in patients with 2019 novel coronavirus pneumonia","10.1101/2020.02.24.20027052",null,null,"medrvix","Objective: To investigate the correlation between clinical characteristics and cardiac injury of COVID-2019 pneumonia. Methods: In this retrospective, single-center study, 41 consecutive corona virus disease 2019 (COVID-2019) patients (including 2 deaths) of COVID-2019 in Beijing Youan Hospital, China Jan 21 to Feb 03, 2020, were involved in this study. The high risk factors of cardiac injury in different COVID-2019 patients were analyzed. Computed tomographic (CT) imaging of epicardial adipose tissue (EAT) has been used to demonstrate the cardiac inflammation of COVID-2019. Results：Of the 41 COVID-2019 patients, 2 (4.88%), 32 (78.05%), 4 (9.75%) and 3 (7.32%) patients were clinically diagnosed as light, mild, severe and critical cases, according to the 6th guidance issued by the National Health Commission of China. 10 (24.4%) patients had underlying complications, such as hypertension, CAD, type 2 diabetes mellites and tumor. The peak value of TnI in critical patients is 40-fold more than normal value. 2 patients in the critical group had the onset of atrial fibrillation, and the peak heart rates reached up to 160 bpm. CT scan showed low EAT density in severe and critical patients. Conclusion: Our results indicated that cardiac injury of COVID-2019 was rare in light and mild patients, while common in severe and critical patients. Therefore, the monitoring of the heart functions of COVID-2019 patients and applying potential interventions for those with abnormal cardiac injury related characteristics, is vital to prevent the fatality.","2020-02-27","Hui Hui; Yingqian Zhang; Xin Yang; Xi Wang; Bingxi He; Li Li; Hongjun Li; Jie Tian; Yundai Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20027052"],["b2znv6pa","d9eeeb81d17be6a722b11081f13576de197d249f","medrxiv","2019 novel coronavirus disease in hemodialysis (HD) patients: Report from one HD center in Wuhan, China","10.1101/2020.02.24.20027201",null,null,"medrvix","The outbreak of COVID-19 originated in Wuhan has become a global epidemic of contagious diseases, which poses a serious threat to human life and health, especially for those with underlined diseases. However, Impacts of COVID-19 epidemic on HD center and HD patients are still unknown. In this report, we reviewed the whole course of the epidemic emerged in the HD center of Renmin Hospital, Wuhan University from January 14, 2020, the day the first case was confirmed, to February 17, 2020, the day the epidemic extinction. There are totally 37 cases among 230 HD patients and 4 cases among 33 staff being diagnosed with COVID-19. The epidemiology, clinical presentation and immune profile of dialysis patients contracted COVID-19 were further studied. We found that the two key measures we took in response to the epidemic, one was upgrading level of prevention and protection on January 21 and the other one starting universal screening, isolating, and distributing the infected cases on February 4, were effective in the epidemic control. No new COVID-19 case had been diagnosed since February 13. During the epidemic, 7 HD patients died, including 6 with COVID-19 and 1 without COVID-19. The presumed causes of death were not directly related to pneumonia, but due to cardiovascular and cerebrovascular diseases, hyperkalemia, etc. Most of the leukocytes in the peripheral blood of the HD patients infected with COVID-19 decreased, and the CT images of the chest mostly showed the ground glass like changes on the right side. The symptoms of most of the patients were mild, and there were no cases admitted to ICU. The frequency of lymphocytes in PBMCs and the serum level of inflammatory cytokines were assessed in HD patients contracted COVID-19 or not, non-HD COVID-19 patients, as well as healthy volunteers. The results showed that lymphocytes of T cell, Th cells, killer T cells, as well as NK cells in PBMCs of HD patients decreased significantly than other groups. HD patients with COVID-19 also displayed remarkable lower serum level of inflammatory cytokines than other COVID-19 patients. Our study indicates that HD patients are the highly susceptible population and HD centers are high risk area in the outbreak of COVID-19 epidemic. Measures of prevention, protection, screening, isolation, and distribution are essential in the epidemic management and should be taken in the early stage. HD Patients with COVID-19 are mostly clinical mild and unlikely progress to severe pneumonia due to the impaired cellular immune function and incapability of mounting cytokines storm. More attention should be paid to prevent cardiovascular events, which may be the collateral impacts of COVID-19 epidemic on HD patients.","2020-02-25","Yiqiong Ma; Bo Diao; Xifeng Lv; Jili Zhu; Wei Liang; Lei Liu; Wenduo Bu; Huiling Cheng; Sihao Zhang; Lianhua Yang; Ming Shi; Guohua Ding; Bo Shen; Huiming Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20027201"],["9tgnx7du","6f53b9dff58818839c45410d11b58f0f3e327d8a","medrxiv","Estimation of risk factors for COVID-19 mortality - preliminary results","10.1101/2020.02.24.20027268",null,null,"medrvix","Since late December 2019 a new epidemic outbreak has emerged from Whuhan, China. Rapidly the new coronavirus has spread worldwide. China CDC has reported results of a descriptive exploratory analysis of all cases diagnosed until the 11th February 2020, presenting the epidemiologic curves and geo-temporal spread of COVID-19 along with case fatality rate according to some baseline characteristics, such as age, gender and several well-established high prevalence comorbidities. Despite this, we intend to increase even further the predictive value of that manuscript by presenting the odds ratio for mortality due to COVID-19 adjusted for the presence of those comorbidities and baseline characteristics such as age and gender. Besides, we present a way to determine the risk of each particular patient, given his characteristics. We found that age is the variable that presents higher risk of COVID-19 mortality, where 60 or older patients have an OR = 18.8161 (CI95%[7.1997; 41.5517]). Regarding comorbidities, cardiovascular disease appears to be the riskiest (OR=12.8328 CI95%[10.2736; 15.8643], along with chronic respiratory disease (OR=7.7925 CI95%[5.5446; 10.4319]). Males are more likely to die from COVID-19 (OR=1.8518 (CI95%[1.5996; 2.1270]). Some limitations such as the lack of information about the correct prevalence of gender per age or about comorbidities per age and gender or the assumption of independence between risk factors are expected to have a small impact on results. A final point of paramount importance is that the equation presented here can be used to determine the probability of dying from COVID-19 for a particular patient, given its age interval, gender and comorbidities associated.","2020-02-25","Francisco Caramelo; Nuno Ferreira; Barbara Oliveiros",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20027268"],["ad3tct5b","53326420d7eb607538a248c0da27d3b092f651e2","medrxiv","Lessons learnt from 288 COVID-19 international cases: importations over time, effect of interventions, underdetection of imported cases","10.1101/2020.02.24.20027326",null,null,"medrvix","288 cases have been confirmed out of China from January 3 to February 13, 2020. We collected and synthesized all available information on these cases from official sources and media. We analyzed importations that were successfully isolated and those leading to onward transmission. We modeled their number over time, in relation to the origin of travel (Hubei province, other Chinese provinces, other countries) and interventions. We characterized importations timeline to assess the rapidity of isolation, and epidemiologically linked clusters to estimate the rate of detection. We found a rapid exponential growth of importations from Hubei, combined with a slower growth from the other areas. We predicted a rebound of importations from South East Asia in the upcoming weeks. Time from travel to detection has considerably decreased since the first importation, however 6 cases out of 10 were estimated to go undetected. Countries outside China should be prepared for the possible emergence of several undetected clusters of chains of local transmissions.","2020-02-25","Francesco Pinotti; Laura Di Domenico; Ernesto Ortega; Marco Mancastroppa; Giulia Pullano; Eugenio Valdano; Pierre-Yves Boelle; Chiara Poletto; Vittoria Colizza",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20027326"],["of9wlhga","ffbd7555a337706238c211197b221795e4e35146","medrxiv","Estimation of COVID-2019 burden and potential for international dissemination of infection from Iran","10.1101/2020.02.24.20027375",null,null,"medrvix","The Coronavirus Disease 2019 (COVID-19) epidemic began in Wuhan, China in late 2019 and continues to spread globally, with exported cases confirmed in 28 countries at the time of writing. During the interval between February 19 and 23, 2020, Iran reported its first 43 cases with eight deaths. Three exported cases originating in Iran were identified, suggesting a underlying burden of disease in that country than is indicated by reported cases. A large epidemic in Iran could further fuel global dissemination of COVID-19. We sought to estimate COVID-19 outbreak size in Iran based on known exported case counts and air travel links between Iran and other countries, and to anticipate where infections originating in Iran may spread to next. We assessed interconnectivity between Iran and other countries using using International Air Transport Association (IATA) data. We used the methods of Fraser et al. to estimate the size of the underlying epidemic that would result in cases being observed in the United Arab Emirates (UAE), Lebanon, and Canada. Time at risk estimates were based on a presumed 6 week epidemic age, and length of stay data for visitors to Iran derived from the United Nations World Tourism Organization (UNWTO). We evaluated the relationship between the strength of travel links with Iran, and destination country rankings on the Infectious Disease Vulnerability Index (IDVI), a validated metric that estimates the capacity of a country to respond to an infectious disease outbreak . Scores range between 0-1, with higher scores reflecting greater capacity to manage infectious outbreaks. UAE, Lebanon, and Canada ranked 3rd, 21st, and 31st, respectively, for outbound air travel volume from Iran in February 2019. We estimated that 18,300 (95% confidence interval: 3770 to 53,470) COVID-19 cases would have had to occur in Iran, assuming an outbreak duration of 1.5 months in the country, in order to observe these three internationally exported cases reported at the time of writing. Results were robust under varying assumptions about undiagnosed case numbers in Syria, Azerbaijan and Iraq. Even if it were assumed that all cases were identified in all countries with certainty, the \"best case\" outbreak size was substantial (1820, 95% CI: 380-5320 cases), and far higher than reported case counts. Given the low volumes of air travel to countries with identified cases of COVID-19 with origin in Iran (such as Canada), it is likely that Iran is currently experiencing a COVID-19 epidemic of significant size for such exportations to be occurring. This is concerning, both for public health in Iran itself, and because of the high likelihood for outward dissemination of the epidemic to neighbouring countries with lower capacity to respond to infectious diseases epidemics.","2020-02-25","Ashleigh R. Tuite; Isaac Bogoch; Ryan Sherbo; Alexander Watts; David N. Fisman; Kamran Khan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20027375"],["vkgnwxzc","c1ae608c7ffb926a0f50a6a34c0780983274ea74","medrxiv","Estimate the incubation period of coronavirus 2019 (COVID-19)","10.1101/2020.02.24.20027474",null,null,"medrvix","Motivation: Wuhan pneumonia is an acute infectious disease caused by the 2019 novel coronavirus (COVID-19). It is being treated as a Class A infectious disease though it was classified as Class B according to the Infectious Disease Prevention Act of China. Accurate estimation of the incubation period of the coronavirus is essential to the prevention and control. However, it remains unclear about its exact incubation period though it is believed that symptoms of COVID-19 can appear in as few as 2 days or as long as 14 or even more after exposure. The accurate incubation period calculation requires original chain-of-infection data that may not be fully available in the Wuhan regions. In this study, we aim to accurately calculate the incubation period of COVID-19 by taking advantage of the chain-of-infection data, which is well-documented and epidemiologically informative, outside the Wuhan regions. Methods: We acquired and collected officially reported COVID-19 data from 10 regions in China except for Hubei province. To achieve the accurate calculation of the incubation period, we only involved the officially confirmed cases with a clear history of exposure and time of onset. We excluded those without relevant epidemiological descriptions, working or living in Wuhan for a long time, or hard to determine the possible exposure time. We proposed a Monte Caro simulation approach to estimate the incubation of COVID-19 as well as employed nonparametric ways. We also employed manifold learning and related statistical analysis to decipher the incubation relationships between different age/gender groups. Result: The incubation period of COVID-19 did not follow general incubation distributions such as lognormal, Weibull, and Gamma distributions. We estimated that the mean and median of its incubation were 5.84 and 5.0 days via bootstrap and proposed Monte Carlo simulations. We found that the incubation periods of the groups with age>=40 years and age<40 years demonstrated a statistically significant difference. The former group had a longer incubation period and a larger variance than the latter. It further suggested that different quarantine time should be applied to the groups for their different incubation periods. Our machine learning analysis also showed that the two groups were linearly separable. incubation of COVID-19 along with previous statistical analysis. Our results further indicated that the incubation difference between males and females did not demonstrate a statistical significance.","2020-02-29","Henry Han",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20027474"],["mopydeq5","7b5be48b7769ccad2f44a8c54efcea5281a6c633","medrxiv","Spread and control of COVID-19 in China and their associations with population movement, public health emergency measures, and medical resources","10.1101/2020.02.24.20027623",null,null,"medrvix","ABSTRACT BACKGROUND The COVID-19 epidemic, first emerged in Wuhan during December 2019, has spread globally. While the mass population movement for Chinese New Year has significantly influenced spreading the disease, little direct evidence exists about the relevance to epidemic and its control of population movement from Wuhan, local emergency response, and medical resources in China. METHODS Spearman's correlation analysis was performed between official data of confirmed COVID-19 cases from Jan 20th to Feb 19th, 2020 and real-time travel data and health resources data. RESULTS There were 74,675 confirmed COVID-19 cases in China by Feb 19th, 2020. The overall fatality rate was 2.84%, much higher in Hubei than in other regions (3.27% vs 0.73%). The index of population inflow from Hubei was positively correlated with total (Provincial r=0.9159, p<0.001; City r=0.6311, p<0.001) and primary cases (Provincial r=0.8702, p<0.001; City r=0.6358, p<0.001). The local health emergency measures (eg, city lockdown and traffic control) were associated with reduced infections nationwide. Moreover, the number of public health employees per capita was inversely correlated with total cases (r=-0.6295, p<0.001) and infection rates (r=-0.4912, p<0.01). Similarly, cities with less medical resources had higher fatality (r=-0.4791, p<0.01) and lower cure rates (r=0.5286, p<0.01) among the confirmed cases. CONCLUSIONS The spread of the COVID-19 in China in its early phase was attributed primarily to population movement from Hubei, and effective governmental health emergency measures and adequate medical resources played important roles in subsequent control of epidemic and improved prognosis of affected individuals.","2020-02-27","Songmin Ying; Fei Li; Xinwei Geng; Zhouyang Li; Xufei Du; Haixia Chen; Sisi Chen; Min Zhang; Zhehua Shao; Yinfang Wu; Madiha Zahra Syeda; Fugui Yan; Luanqing Che; Bin Zhang; Jian Lou; Shaobin Wang; Zhengming Chen; Wen Li; Ye Shen; Zhihua Chen; Huahao Shen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20027623"],["eoxrh2sl","d7701711485f822c0f43df58b48faf6a433028c4","medrxiv","Transmission potential of the novel coronavirus (COVID-19) onboard the Diamond Princess Cruises Ship, 2020","10.1101/2020.02.24.20027649",null,null,"medrvix","An outbreak of COVID-19 developed aboard the Princess Cruises Ship during January-February 2020. Using mathematical modeling and time-series incidence data describing the trajectory of the outbreak among passengers and crew members, we characterize how the transmission potential varied over the course of the outbreak. Our estimate of the mean reproduction number in the confined setting reached values as high as ~11, which is higher than mean estimates reported from community-level transmission dynamics in China and Singapore (approximate range: 1.1-7). Our findings suggest that Rt decreased substantially compared to values during the early phase after the Japanese government implemented an enhanced quarantine control. Most recent estimates of Rt reached values largely below the epidemic threshold, indicating that a secondary outbreak of the novel coronavirus was unlikely to occur aboard the Diamond Princess Ship.","2020-02-27","Kenji Mizumoto; Gerardo Chowell",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.20027649"],["0n5bxp93",null,"biorxiv","IFN-λ4 increases the risk of gastrointestinal infections and malaria in Malian children","10.1101/2020.02.24.962688",null,null,"biorxiv","ABSTRACTGenetic polymorphisms within the IFNL3/IFNL4 genomic region encoding type III interferons have been strongly associated with impaired clearance of hepatitis C virus (HCV) infection. We hypothesized that this association might extend to the immune response to other pathogens as well. In a cohort of 914 Malian children enrolled at birth, we analyzed episodes of malaria, gastrointestinal and respiratory infections in relation to two genetic polymorphisms functionally affecting type III interferons - rs368234815 (IFN-λ4) and rs4803217 (IFN-λ3), using information for 30,626 clinic visits from birth through 1-5 years of follow-up. Compared to children with the rs368234815-TT/TT genotype (IFN-λ4-Null), each copy of the rs368234815-dG allele was associated with an earlier first episode of a gastrointestinal infection (p=0.004), malaria (p=0.044) or respiratory infection (p=0.045). The risk of ever experiencing an infection during the follow-up was also significantly increased with each copy of the rs368234815-dG allele – for gastrointestinal infections (OR=1.5, 95%CI (1.11-2.03), p=0.0079) and malaria (OR=1.32, 95%CI (1.04-1.68, p=0.022), but not respiratory infections (p=0.58). IFNL4-rs368234815 and IFNL3-rs4803217 were in moderate linkage disequilibrium in this population (r2=0.78). All the associations for rs4803217 were weaker and lost significance after adjusting for rs368234815, implicating IFN-λ4 and not IFN-λ3 as the primary cause of these associations. Thus, our results suggest the role of IFN-λ4 in several infections in young children.","2020-02-25","Ludmila Prokunina-Olsson; Robert L. Morrison; Adeola Obajemu; Almahamoudou Mahamar; Sungduk Kim; Oumar Attaher; Oscar Florez-Vargas; Youssoufa Sidibe; Olusegun O. Onabajo; Amy A. Hutchinson; Michelle Manning; Jennifer Kwan; Nathan Brand; Alassane Dicko; Michal Fried; Paul S. Albert; Sam M. Mbulaiteye; Patrick E. Duffy",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.962688"],["epy7774j","c8b6f2752a842dd9eb9c50a82112748ef10ba259","biorxiv","Increasing Host Cellular Receptor—Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection","10.1101/2020.02.24.963348",null,null,"biorxiv","AbstractThe ongoing outbreak of a new coronavirus (2019-nCoV) causes an epidemic of acute respiratory syndrome in humans. 2019-nCoV rapidly spread to national regions and multiple other countries, thus, pose a serious threat to public health. Recent studies show that spike (S) proteins of 2019-nCoV and SARS-CoV may use the same host cell receptor called angiotensin-converting enzyme 2 (ACE2) for entering into host cells. The affinity between ACE2 and 2019-nCoV S is much higher than ACE2 binding to SARS-CoV S protein, explaining that why 2019-nCoV seems to be more readily transmitted from the human to human. Here, we reported that ACE2 can be significantly upregulated after infection of various viruses including SARS-CoV and MERS-CoV. Basing on findings here, we propose that coronavirus infection can positively induce its cellular entry receptor to accelerate their replication and spread, thus drugs targeting ACE2 expression may be prepared for the future emerging infectious diseases caused by this cluster of viruses.","2020-02-27","Pei-Hui Wang; Yun Cheng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.24.963348"],["z4q0eyjo",null,"medrxiv","Deep learning-based model for detecting 2019 novel coronavirus pneumonia on high-resolution computed tomography: a prospective study","10.1101/2020.02.25.20021568",null,null,"medrvix","Background: Computed tomography (CT) is the preferred imaging method for diagnosing 2019 novel coronavirus (COVID19) pneumonia. Our research aimed to construct a system based on deep learning for detecting COVID-19 pneumonia on high resolution CT, relieve working pressure of radiologists and contribute to the control of the epidemic. Methods: For model development and validation, 46,096 anonymous images from 106 admitted patients, including 51 patients of laboratory confirmed COVID-19 pneumonia and 55 control patients of other diseases in Renmin Hospital of Wuhan University (Wuhan, Hubei province, China) were retrospectively collected and processed. Twenty-seven consecutive patients undergoing CT scans in Feb, 5, 2020 in Renmin Hospital of Wuhan University were prospectively collected to evaluate and compare the efficiency of radiologists against 2019-CoV pneumonia with that of the model. Findings: The model achieved a per-patient sensitivity of 100%, specificity of 93.55%, accuracy of 95.24%, PPV of 84.62%, and NPV of 100%; a per-image sensitivity of 94.34%, specificity of 99.16%, accuracy of 98.85%, PPV of 88.37%, and NPV of 99.61% in retrospective dataset. For 27 prospective patients, the model achieved a comparable performance to that of expert radiologist. With the assistance of the model, the reading time of radiologists was greatly decreased by 65%. Conclusion: The deep learning model showed a comparable performance with expert radiologist, and greatly improve the efficiency of radiologists in clinical practice. It holds great potential to relieve the pressure of frontline radiologists, improve early diagnosis, isolation and treatment, and thus contribute to the control of the epidemic.","2020-02-26","Jun Chen; Lianlian Wu; Jun Zhang; Liang Zhang; Dexin Gong; Yilin Zhao; Shan Hu; Yonggui Wang; Xiao Hu; Biqing Zheng; Kuo Zhang; Huiling Wu; Zehua Dong; Youming Xu; Yijie Zhu; Xi Chen; Lilei Yu; Honggang Yu",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20021568"],["acmfhbtm","da5ecffba8c5d1b7afee057ff61149fb976501ce","medrxiv","Highland of COVID-19 outside Hubei: epidemic characteristics, control and projections of Wenzhou, China","10.1101/2020.02.25.20024398",null,null,"medrvix","In late December 2019, Chinese authorities reported a cluster of pneumonia cases of unknown aetiology in Wuhan, China. Wenzhou, as a southeast coastal city with the most cases outside Hubei Province, its policy control and epidemic projections have certain references for national and worldwide epidemic prevention and control. The maximum number of illness onset in Wenzhou appeared on January 26 (38 cases). Since January 27, the incidence per day in Wenzhou has continued to decline is likely to be due to isolation of people return to Wenzhou, implementation of region quarantine and suspension of public transportation system. Under the current control efforts, we estimate that the daily incidence by March 3-9, 2020 will drop to 0 in Wenzhou using SEIR model. The total number of affected people is 538.","2020-02-29","Liangde Xu; Jian Yuan; Yaru Zhang; Guosi Zhang; Fan Lu; Jianzhong Su; Jia Qu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20024398"],["dy28eo89","bca15ad98e1d08c6a62c9c0486de84bc15a9f661","medrxiv","Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?","10.1101/2020.02.25.20024711",null,null,"medrvix","Background. The clinical presentation of 2019 Novel Coronavirus (2019-nCov) infected pneumonia (NCIP) resembles that of other etiologies of community-acquired pneumonia (CAP). We aimed to identify clinical laboratory features to distinguish NCIP from CAP. Methods. We compared the ability of the hematological and biochemical features of 84 patients with NCIP at hospital admission and 316 patients with CAP. Parameters independently predictive of NCIP were calculated by multivariate logistic regression. The receiver operating characteristic (ROC) curves were generated and the area under the ROC curve (AUC) was measured to evaluate the discriminative ability. Results. Most hematological and biochemical indexes of patients with NCIP were significantly different from patients with CAP. Nine laboratory parameters were identified to be highly predictive of a diagnosis of NCIP by multivariate analysis. The AUCs demonstrated good discriminatory ability for red cell distribution width (RDW) with an AUC of 0.88 and Hemoglobin (HGB) with an AUC of 0.82. Red blood cell (RBC), albumin (ALB), eosinophil (EO), hematocrit (HCT), alkaline phosphatase (ALP), and white blood cell (WBC) had fair discriminatory ability. Combinations of any two parameters performed better than did the RDW alone. Conclusions. Routine laboratory examinations may be helpful for the diagnosis of NCIP. Application of laboratory tests may help to optimize the use of isolation rooms for patients when they present with unexplained febrile respiratory illnesses.","2020-02-25","Yunbao Pan; Guangming Ye; Xiantao Zeng; Guohong Liu; Xiaojiao Zeng; Xianghu Jiang; Jin Zhao; Liangjun Chen; Shuang Guo; Qiaoling Deng; Xiaoyue Hong; Ying Yang; Yirong Li; Xinghuan Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20024711"],["370fqkat","c8d206a4f9af0709b6e9ee90c4d854d482cb0784","medrxiv","Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia","10.1101/2020.02.25.20025643",null,null,"medrvix","Aim: The new coronavirus pneumonia (COVID-19) outbreaking at the end of 2019 is highly contagious. Crude mortality rate reached 49% in critical patients. Inflammation matters on disease progression. This study analyzed blood inflammation indicators among mild, severe and critical patients, helping to identify severe or critical patients early. Methods: In this cross-sectional study, 100 patients were included and divided to mild, severe or critical groups. Correlation of peripheral blood inflammation-related indicators with disease criticality was analyzed. Cut-off values for critically ill patients were speculated through the ROC curve. Results：Significantly, disease severity were associated with age (R=-0.564, P<0.001), interleukin-2 receptor (IL2R) (R=-0.534, P<0.001), interleukin-6 (IL-6) (R=-0.535, P<0.001), interleukin-8 (IL-8) (R=-0.308, P<0.001), interleukin-10 (IL-10) (R=-0.422, P<0.001), tumor necrosis factor α (TNFα) (R=-0.322, P<0.001), C-reactive protein (CRP) (R=-0.604, P<0.001), ferroprotein (R=-0.508, P<0.001), procalcitonin (R=-0.650, P<0.001), white cell counts (WBC) (R=-0.54, P<0.001), lymphocyte counts (LC) (R=-0.56, P<0.001), neutrophil count (NC) (R=-0.585, P<0.001) and eosinophil counts (EC) (R=-0.299, P=0.01). Conclusion：With following parameters such as age >67.5 years, IL2R >793.5U/mL, CRP >30.7ng/mL, ferroprotein >2252μg/L, WBC>9.5*10^9/L or NC >7.305*10^9/L, the progress of COVID-19 to critical stage should be closely observed and possibly prevented. Inflammation is closely related to severity of COVID-19, and IL-6, TNFα and IL-8 might be promising therapeutic targets.","2020-02-27","Jing Gong; Hui Dong; Song Qing Xia; Yi Zhao Huang; Dingkun Wang; Yan Zhao; Wenhua Liu; Shenghao Tu; Mingmin Zhang; Qi Wang; Fuer Lu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20025643"],["qtrquuvv","7d3dcbbc2b0741fc544693d7b3fc185f0bd50eac","medrxiv","Open-source analytics tools for studying the COVID-19 coronavirus outbreak","10.1101/2020.02.25.20027433",null,null,"medrvix","To provide convenient access to epidemiological data on the coronavirus outbreak, we developed an R package, nCov2019 (https://github.com/GuangchuangYu/nCov2019). Besides detailed real-time statistics, it offers access to three data sources with detailed daily statistics from December 1, 2019, for 43 countries and more than 500 Chinese cities. We also developed a web app (http://www.bcloud.org/e/) with interactive plots and simple time-series forecasts. These analytics tools could be useful in informing the public and studying how this and similar viruses spread in populous countries.","2020-02-27","Tianzhi Wu; Xijin Ge; Guangchuang Yu; Erqiang Hu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20027433"],["aoqyx8mk","160ecaaa5a766c289fc2f6b5499f0dfe7aab971c","medrxiv","Stochastic discrete epidemic modeling of COVID-19 transmission in the Province of Shaanxi incorporating public health intervention and case importation","10.1101/2020.02.25.20027615",null,null,"medrvix","Before the lock-down of Wuhan/Hubei/China, on January 23rd 2020, a large number of individuals infected by COVID-19 moved from the epicenter Wuhan and the Hubei province due to the Spring Festival, resulting in an epidemic in the other provinces including the Shaanxi province. The epidemic scale in Shaanxi was comparatively small and with half of cases being imported from the epicenter. Based on the complete epidemic data including the symptom onset time and transmission chains, we calculate the control reproduction number (1.48-1.69) in Xian. We could also compute the time transition, for each imported or local case, from the latent, to infected, to hospitalized compartment, as well as the effective reproduction number. This calculation enables us to revise our early deterministic transmission model to a stochastic discrete epidemic model with case importation and parameterize it. Our model-based analyses reveal that the newly generated infections decay to zero quickly; the cumulative number of case-driven quarantined individuals via contact tracing stabilize at a manageable level, indicating that the intervention strategies implemented in the Shaanxi province have been effective. Risk analyses, important for the consideration of resumption of work, show that a large second outbreak is expected if the level of case importation remains at the same level as between January 10th and February 4th 2020. However, if the case importation decreases by 30%, 60% and 90%, the second outbreak if happening will be of small-scale assuming contact tracing and quarantine/isolation remain as effective as before. Finally, we consider the effects of intermittent inflow with a Poisson distribution on the likelihood of multiple outbreaks. We believe the developed methodology and stochastic model provide an important model framework for the evaluation of revising travel restriction rules in the consideration of resuming social-economic activities while managing the disease control with potential case importation.","2020-02-29","Sanyi Tang; Biao Tang; Nicola Luigi Bragazzi; Fan Xia; Tangjuan Li; Sha He; Pengyu Ren; Xia Wang; Zhihang Peng; Yanni Xiao; Jianhong Wu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20027615"],["7s9ot4vq","875b7c463f00772fa0dc18ada678bc1ff16a4274","medrxiv","Comorbidity and its impact on 1,590 patients with COVID-19 in China: A Nationwide Analysis","10.1101/2020.02.25.20027664",null,null,"medrvix","Objective: To evaluate the spectrum of comorbidities and its impact on the clinical outcome in patients with coronavirus disease 2019 (COVID-19). Design: Retrospective case studies Setting: 575 hospitals in 31 province/autonomous regions/provincial municipalities across China Participants: 1,590 laboratory-confirmed hospitalized patients. Data were collected from November 21st, 2019 to January 31st, 2020. Main outcomes and measures: Epidemiological and clinical variables (in particular, comorbidities) were extracted from medical charts. The disease severity was categorized based on the American Thoracic Society guidelines for community-acquired pneumonia. The primary endpoint was the composite endpoints, which consisted of the admission to intensive care unit (ICU), or invasive ventilation, or death. The risk of reaching to the composite endpoints was compared among patients with COVID-19 according to the presence and number of comorbidities. Results: Of the 1,590 cases, the mean age was 48.9 years. 686 patients (42.7%) were females. 647 (40.7%) patients were managed inside Hubei province, and 1,334 (83.9%) patients had a contact history of Wuhan city. Severe cases accounted for 16.0% of the study population. 131 (8.2%) patients reached to the composite endpoints. 399 (25.1%) reported having at least one comorbidity. 269 (16.9%), 59 (3.7%), 30 (1.9%), 130 (8.2%), 28 (1.8%), 24 (1.5%), 21 (1.3%), 18 (1.1%) and 3 (0.2%) patients reported having hypertension, cardiovascular diseases, cerebrovascular diseases, diabetes, hepatitis B infections, chronic obstructive pulmonary disease, chronic kidney diseases, malignancy and immunodeficiency, respectively. 130 (8.2%) patients reported having two or more comorbidities. Patients with two or more comorbidities had significantly escalated risks of reaching to the composite endpoint compared with those who had a single comorbidity, and even more so as compared with those without (all P<0.05). After adjusting for age and smoking status, patients with COPD (HR 2.681, 95%CI 1.424-5.048), diabetes (HR 1.59, 95%CI 1.03-2.45), hypertension (HR 1.58, 95%CI 1.07-2.32) and malignancy (HR 3.50, 95%CI 1.60-7.64) were more likely to reach to the composite endpoints than those without. As compared with patients without comorbidity, the HR (95%CI) was 1.79 (95%CI 1.16-2.77) among patients with at least one comorbidity and 2.59 (95%CI 1.61-4.17) among patients with two or more comorbidities. Conclusion: Comorbidities are present in around one fourth of patients with COVID-19 in China, and predispose to poorer clinical outcomes.","2020-02-27","Wei-jie Guan; Wen-hua Liang; Yi Zhao; Heng-rui Liang; Zi-sheng Chen; Yi-min Li; Xiao-qing Liu; Ru-chong Chen; Chun-li Tang; Tao Wang; Chun-quan Ou; Li Li; Ping-yan Chen; Ling Sang; Wei Wang; Jian-fu Li; Cai-chen Li; Li-min Ou; Bo Cheng; Shan Xiong; Zheng-yi Ni; Yu Hu; Jie Xiang; Lei Liu; Hong Shan; Chun-liang Lei; Yi-xiang Peng; Li Wei; Yong Liu; Ya-hua Hu; Peng Peng; Jian-ming Wang; Ji-yang Liu; Zhong Chen; Gang Li; Zhi-jian Zheng; Shao-qin Qiu; Jie Luo; Chang-jiang Ye; Shao-yong Zhu; Lin-ling Cheng; Feng Ye; Shi-yue Li; Jin-ping Zheng; Nuo-fu Zhang; Nan-shan Zhong; Jian-xing He",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20027664"],["gvjiexfm",null,"medrxiv","Age-dependent risks of Incidence and Mortality of COVID-19 in Hubei Province and Other Parts of China","10.1101/2020.02.25.20027672",null,null,"medrvix","New coronavirus SARS-CoV-2 poses a big challenge for global public health in early 2020. Coronavirus Disease 2019 (COVID-19) caused by the virus rapidly spreads all over the world and takes thousands of lives in just two months. It is critical to refine the incidence and mortality risks of COVID-19 for the effective management of the general public and patients in the outbreak. In this report, we investigate the incidence and mortality risks of the infection by analyzing the age composition of 5319 infected patients, 76 fatal cases, and 1,144,648 individuals of the general public in China. Our result shows a relatively low incidence risk for young people but a very high mortality risk for seniors. Notably, mortality risk could be as high as 0.48 for people older than 80 years. Furthermore, our study suggests that a good medical service can effectively reduce the mortality rate of the viral infection to 1% or less.","2020-02-27","Hongdou Li; Shuang Wang; Fan Zhong; Wuyin Bao; Yipeng Li; Lei Liu; Hongyan Wang; Yungang He",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20027672"],["65plab8b","e046a06f262d74d025272ad697d358c4afb5d9bd","medrxiv","Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection","10.1101/2020.02.25.20027755",null,null,"medrvix","Background: Coronavirus Disease 2019 (COVID-19) caused by Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a global threat to public health. Aiming to construct an efficient screening pattern, we comprehensively evaluated the performances of RT-PCR and chest CT in diagnosing COVID-19. Methods: The records including demographics, RT-PCR, and CT from 87 confirmed COVID-19 cases and 481 exclusion cases were collected. The diagnostic accuracy of the pharyngeal swab RT-PCR, CT, combination with the second pharyngeal swab RT-PCR or with CT were evaluated individually. Besides, all the stool RT-PCR results were plotted by time to explore the value of stool RT-PCR. Findings: Combination of RT-PCR and CT has the higher sensitivity (91.9%,79/86) than RT-PCR alone (78.2%，68/87) or CT alone (66.7%, 54 of 81) or combination of two RT-PCR tests (86.2%,75/87). There was good agreement between RT-PCR and CT (kappa-value, 0.430). In 34 COVID-19 cases with inconsistent results, 94.1% (n=32) are mild infection, 62.5% of which (20/32) showed positive RT-PCR. 46.7% (35/75) COVID-19 patients had at least one positive stool during the course. Two cases had positive stool earlier than the pharyngeal swabs. Importantly, one patient had consecutive positive stool but negative pharyngeal swabs. Interpretation: Combination of RT-PCR and CT with the highest sensitivity is an optimal pattern to screen COVID-19. RT-PCR is superior to CT in diagnosing mild infections. Stool RT-PCR should be considered as an item for improving discovery rate and hospital discharge. This study shed light for optimizing scheme of screening and monitoring of SARS-CoV-2 infection.","2020-02-27","Guanmin Jiang; Xiaoshuai Ren; Yan Liu; Hongtao Chen; Wei Liu; Zhaowang Guo; Yaqin Zhang; Chaoqun Chen; Jianhui Zhou; Qiang Xiao; Hong Shan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20027755"],["cr0s5d1j","0eee9760aee0ca2fdf8d1d215c3689d5f8d84df5","medrxiv","Prevalence and clinical features of 2019 novel coronavirus disease (COVID-19) in the Fever Clinic of a teaching hospital in Beijing: a single-center, retrospective study","10.1101/2020.02.25.20027763",null,null,"medrvix","Background With the spread of COVID-19 from Wuhan, Hubei Province to other areas of the country, medical staff in Fever Clinics faced the challenge of identifying suspected cases among patients with respiratory infections manifested with fever. We aimed to describe the prevalence and clinical features of COVID-19 as compared to pneumonias of other etiologies in a Fever Clinic in Beijing. Methods In this single-center, retrospective study, 342 cases of pneumonia were diagnosed in Fever Clinic in Peking University Third Hospital between January 21 to February 15, 2020. From these patients, 88 were reviewed by panel discussion as possible or probable cases of COVID-19, and received 2019-nCoV detection by RT-PCR. COVID-19 was confirmed by positive 2019-nCoV in 19 cases, and by epidemiological, clinical and CT features in 2 cases (the COVID-19 Group, n=21), while the remaining 67 cases served as the non-COVID-19 group. Demographic and epidemiological data, symptoms, laboratory and lung CT findings were collected, and compared between the two groups. Findings The prevalence of COVID-19 in all pneumonia patients during the study period was 6.14% (21/342). Compared with the non-COVID-19 group, more patients with COVID-19 had an identified epidemiological history (90.5% versus 32.8%, P<0.001). The COVID-19 group had lower WBC [5.19×10^9/L (±1.47) versus 7.21×10^9/L (±2.94), P<0.001] and neutrophil counts [3.39×10^9/L (±1.48) versus 5.38×10^9/L (±2.85), P<0.001] in peripheral blood. However, the percentage and count of lymphocytes were not different. On lung CT scans, involvement of 4 or more lobes was more common in the COVID-19 group (45% versus 16.4%, P=0.008). Interpretation In the period of COVID-19 epidemic outside Hubei Province, the prevalence of COVID-19 in patients with pneumonia visiting to our Fever Clinic in Beijing was 6.14%. Epidemiological evidence was important for prompt case finding, and lower blood WBC and neutrophil counts may be useful for differentiation from pneumonia of other etiologies.","2020-02-27","Ying Liang; Jingjin Liang; Qingtao Zhou; Xiaoguang Li; Fei Lin; Zhonghua Deng; Biying Zhang; Lu Li; Xiaohua Wang; Hong Zhu; Qingbian Ma; Xiaomei Tong; Jie Xu; Yongchang Sun",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20027763"],["a12708qd",null,"medrxiv","Genomic variations of SARS-CoV-2 suggest multiple outbreak sources of transmission","10.1101/2020.02.25.20027953",null,null,"medrvix","We examined 169 genomes of SARS-CoV-2 and found that they can be classified into two major genotypes, Type I and Type II. Type I can be further divided into Type IA and IB. Our phylogenetic analysis showed that the Type IA resembles the ancestral SARS-CoV-2 most. Type II was likely evolved from Type I and predominant in the infections. Our results suggest that Type II SARS-CoV-2 was the source of the outbreak in the Wuhan Huanan market and it was likely originated from a super-spreader. The outbreak caused by the Type I virus should have occurred somewhere else, because the patients had no direct link to the market. Furthermore, by analyzing three genomic sites that distinguish Type I and Type II strains, we found that synonymous changes at two of the three sites confer higher protein translational efficiencies in Type II strains than in Type I strains, which might explain why Type II strains are predominant, implying that Type II is more contagious (transmissible) than Type I. These findings could be valuable for the current epidemic prevention and control.","2020-02-26","Liangsheng Zhang; Jian-Rong Yang; Zhenguo Zhang; Zhenguo Lin",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20027953"],["lad31adx","eca1cf50bd67a7efbcaf78864f5f5b3b850243d6","medrxiv","Clinical Data on Hospital Environmental Hygiene Monitoring and Medical Staff Protection during the Coronavirus Disease 2019 Outbreak","10.1101/2020.02.25.20028043",null,null,"medrvix","Background: The outbreak of coronavirus disease 2019 (COVID-19) has placed unprecedented challenges on hospital environmental hygiene and medical staff protection. It is crucial to assess hospital environmental hygiene to understand the most important environmental issues for controlling the spread of COVID-19 in hospitals. Objective: To detect the presence of COVID-19 in the samples from the area at risk of contamination in the First Hospital of Jilin University. Methods: Viruses in the air were collected by natural sedimentation and air particle sampler methods. Predetermined environmental surfaces were sampled using swabs at seven o'clock in the morning before disinfection. The real-time reverse-transcription PCR method was used to detect the existence of COVID-19 pathogens. Results: Viruses could be detected on the surfaces of the nurse station in the isolation area with suspected patients and in the air of the isolation ward with an intensive care patient. Conclusion: Comprehensive monitoring of hospital environmental hygiene during pandemic outbreaks is conducive to the refinement of hospital infection control. It is of great significance to ensure the safety of medical treatment and the quality of hospital infection control through the monitoring of environmental hygiene.","2020-02-27","Yanfang Jiang; Haifeng Wang; Yukun Chen; Jiaxue He; Liguo Chen; Yong Liu; Xinyuan Hu; Ang Li; Siwen Liu; Peng Zhang; Hongyan Zou; Shucheng Hua",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.20028043"],["i55tm3hh","a2a6e262098539eb875a26800d9f6d3d0d5d1875","biorxiv","An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design","10.1101/2020.02.25.963546",null,null,"biorxiv","AbstractThe 2013-2016 West Africa EBOV epidemic was the biggest EBOV outbreak to date. An analysis of virus-specific CD8+ T-cell immunity in 30 survivors showed that 26 of those individuals had a CD8+ response to at least one EBOV protein. The dominant response (25/26 subjects) was specific to the EBOV nucleocapsid protein (NP). It has been suggested that epitopes on the EBOV NP could form an important part of an effective T-cell vaccine for Ebola Zaire. We show that a 9-amino-acid peptide NP44-52 (YQVNNLEEI) located in a conserved region of EBOV NP provides protection against morbidity and mortality after mouse adapted EBOV challenge. A single vaccination in a C57BL/6 mouse using an adjuvanted microsphere peptide vaccine formulation containing NP44-52 is enough to confer immunity in mice. Our work suggests that a peptide vaccine based on CD8+ T-cell immunity in EBOV survivors is conceptually sound and feasible. Nucleocapsid proteins within COVID-19 contain multiple class I epitopes with predicted HLA restrictions consistent with broad population coverage. A similar approach to a CTL vaccine design may be possible for that virus.","2020-02-27","CV Herst; S Burkholz; J Sidney; A Sette; PE Harris; S Massey; T Brasel; E Cunha-Neto; DS Rosa; WCH Chao; R Carback; T Hodge; L Wang; S Ciotlos; P Lloyd; R Rubsamen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.963546"],["2hyiped2","86b6b0c1b2777541feb83116bcb7a5cb12a52310","biorxiv","Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2","10.1101/2020.02.25.964775",null,null,"biorxiv","AbstractCoronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the ‘2019-nCoV_N2, N3’ of USA and the ‘ORF1ab’ of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.","2020-02-27","Yu Jin Jung; Gun-Soo Park; Jun Hye Moon; Keunbon Ku; Seung-Hwa Beak; Seil Kim; Edmond Changkyun Park; Daeui Park; Jong-Hwan Lee; Cheol Woo Byeon; Joong Jin Lee; Jin-Soo Maeng; Seong Jun Kim; Seung Il Kim; Bum-Tae Kim; Min Jun Lee; Hong Gi Kim",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.964775"],["e9vq3fe3","949feecec7c10e71532e860f7f1230bd696e8234","biorxiv","Epitope-based peptide vaccine design and target site characterization against novel coronavirus disease caused by SARS-CoV-2","10.1101/2020.02.25.965434",null,null,"biorxiv","AbstractThe outbreak of the 2019 novel coronavirus (SARS-CoV-2) has infected thousands of people with a large number of deaths across 26 countries. The sudden appearance of the virus leads to the limited existing therapies for SARS-CoV-2. Therefore, vaccines and antiviral medicines are in desperate need. This study took immune-informatics approaches to identify B- and T-cell epitopes for surface glycoprotein (S) of SARS-CoV-2, followed by estimating their antigenicity and interactions with the human leukocyte antigen (HLA) alleles. We identified four B cell epitopes, two MHC class-I and nine MHC class-II binding T-cell epitopes, which showed highly antigenic features. Allergenicity, toxicity and physiochemical properties analysis confirmed the specificity and selectivity of epitopes. The stability and safety of epitopes were confirmed by digestion analysis. No mutations were observed in all the selected B- and T-cell epitopes across all isolates from different locations worldwide. Epitopes were thus identified and some of them can be potential candidates for vaccine development.","2020-02-27","Lin Li; Ting Sun; Yufei He; Wendong Li; Yubo Fan; Jing Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.965434"],["qefiw4ho","dbefc8ad2a3de5d1696b7e604de8bce1da2ea8cd","biorxiv","Screening of FDA-approved drugs using a MERS-CoV clinical isolate from South Korea identifies potential therapeutic options for COVID-19","10.1101/2020.02.25.965582",null,null,"biorxiv","AbstractIn 2015, the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) reached the Republic of Korea through nosocomial transmission and was the largest epidemic outside of the Arabian Peninsula. To date, despite various strategies to identify CoV interventions, only limited therapeutic options are available. To address these unmet medical needs, we used a South Korean MERS-CoV clinical isolate and screened 5,406 compounds, including United States Food and Drug Administration (FDA)-approved drugs and bioactive molecules, for their activity against the isolate. The primary assay confirmed 221 hits by dose-response curve analysis and identified 54 hits with a therapeutic index (TI) greater than 6. Time-of-addition studies with 12 FDA-approved drugs demonstrated that 8 and 4 therapeutics act on the early and late stages of the viral life cycle, respectively. Among the drugs were e.g., three cardiotonic agents (ouabain, digitoxin, digoxin) with a TI greater than 100, an anti-malaria drug (atovaquone; TI >34), an inhalable corticosteroid (ciclesonide; TI >6), etc. Together, our results identify potential therapeutic options for treating MERS-CoV infections and could provide a basis for agents against a wider range of coronavirus-related illnesses, including the currently emerging Coronavirus Disease 2019 (COVID-19) outbreak.","2020-03-02","Meehyun Ko; So Young Chang; Soo Young Byun; Inhee Choi; Anne-Laure Pham Hung d’Alexandry d’Orengiani; David Shum; Ji-Young Min; Marc P. Windisch",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.25.965582"],["v0ln3wfa","baabfb35a321ea12028160e0d2c1552a2fda2dd5","medrxiv","Clinical Features of COVID-19 Related Liver Damage","10.1101/2020.02.26.20026971",null,null,"medrvix","BACKGROUND: A recent outbreak of SARS-CoV-2 infection occurs mainly in China, with rapidly increasing the number of cases (namely COVID-19). Abnormal liver functions are frequently present in these patients, here we aimed to clarify the clinical features of COVID-19-related liver damage to provide some references for the clinical treatment. METHODS: In this retrospective, single-center study, we included all confirmed COVID-19 cases in Shanghai Public Health Clinical Center from January 20 to January 31, 2020. The outcomes were followed up until February 19, 2020. A total of 148 cases were analyzed for clinical features, laboratory parameters (including liver function tests), medications and the length of stay. FINDINGS: Of 148 confirmed SARS-CoV-2-infected patients, 49.3% were females and 50.7% were males. The median age was 50.5 years (interquartile range, 36-64). Patients had clinical manifestations of fever (70.1%), cough (45.3%), expectoration (26.7%) at admission. 75 patients (50.7%) showed abnormal liver functions at admission. Patients (n = 75) who had elevated liver function index were more likely to have a moderate-high degree fever (44% vs 27.4%; p = 0.035) and significantly present in male patients (62.67% vs 38.36%; p = 0.005). The numbers of CD4+ and CD8+ T cells were significantly lower in abnormal liver function group than those in normal liver function group. There was no statistical difference in prehospital medications between normal and abnormal liver function groups, while the utilization rate of lopinavir/ritonavir after admission was significantly higher in patients with emerging liver injury than that in patients with normal liver functions. Importantly, the emerging abnormal liver functions after admission caused a prolonged length of stay. INTERPRETATION: SARS-CoV-2 may cause the liver function damage and the Lopinavir/ritonavir should be applied carefully for the treatment of COVID-19.","2020-02-27","Zhenyu Fan; Liping Chen; Jun Li; Cheng Tian; Yajun Zhang; Shaoping Huang; Zhanju Liu; Jilin Cheng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20026971"],["r7yyffgt",null,"medrxiv","The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis","10.1101/2020.02.26.20026989",null,null,"medrvix","IMPORTANCE: COVID-19-infected pneumonia patients with severe immune abnormalities and risk of cytokine release syndrome. The definition, prevention, and treatment of COVID-19-infected pneumonia in critically ill patients with cytokine release syndrome symptoms is an important problem.","2020-02-27","Wenjun Wang; Jianxing He; puyi Lie; liyan Huang; Sipei Wu; yongping lin; xiaoqing liu",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20026989"],["m0d72ntg","7344c6de7ce2258e74ce131bed72e6ad465f0ce8","medrxiv","Relations of parameters for describing the epidemic of COVID―19 by the Kermack―McKendrick model","10.1101/2020.02.26.20027797",null,null,"medrvix","In order to quantitatively characterize the epidemic of COVID―19, useful relations among parameters describing an epidemic in general are derived based on the Kermack-McKendrick model. The first relation is 1/τgrow=1/τtrans−1/τinf, where τgrow is the time constant of the exponential growth of an epidemic, τtrans is the time for a pathogen to be transmitted from one patient to uninfected person, and the infectious time τinf is the time during which the pathogen keeps its power of transmission. The second relation p(∞) ≈1−exp(−(R0−1)/0.60) is the relation between p(∞), the final size of the disaster defined by the ratio of the total infected people to the population of the society,and the basic reproduction number, R0, which is the number of persons infected by the transmission of the pathogen from one infected person during the infectious time. The third relation 1/τend=1/τinf−(1−p(∞))/τtrans gives the decay time constant τend at the ending stage of the epidemic. Derived relations are applied to influenza in Japan in 2019 for characterizing the epidemic.","2020-03-03","Toshihisa Tomie",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20027797"],["pytu6sy0","c10661c9b35e068691b879661bcebeba3bd6aad9","medrxiv","The infection evidence of SARS-COV-2 in ocular surface： a single-center cross-sectional study","10.1101/2020.02.26.20027938",null,null,"medrvix","Purpose: The aim of this study was to identify whether SARS-COV-2 infected in ocular surface. Methods: Cross-sectional study of patients presenting for who received a COVID-19 diagnosis, from December 30, 2019 to February 7, 2020, at Tongji hospital, Tongji medical college, Huazhong University of Science and Technology. Demographics, temperature was recorded, blood routine test (Rt), chest Computed Tomography (CT) were took intermittently, and SARS-COV-2 real-time reverse-transcriptase-polymerase-chain-reaction (RT-PCR) assay were arranged for the nasopharyngeal and conjunctival swab samples. Results: A total of 102 patients (48 Male [50%] and 54 Female [50%]) with clinical symptoms, Rt, and chest Computed Tomography (CT) abnormalities were identified with a clinical diagnosis of COVID-19. Patients had a mean [SD] gestational age of 57.63 [14.90] years. Of a total of 102 patients identified, 72 patients (36 men [50%] and 36 women [50%]; mean [SD] age, 58.68 [14.81] years) confirmed by laboratory diagnosis with SARS-COV-2 RT-PCR assay. Only two patients (2.78%) with conjunctivitis was identified from 72 patients with a laboratory confirmed COVID-19. However, SARS-COV-2 RNA fragments was found in ocular discharges by SARS-COV-2 RT-PCR only in one patient with conjunctivitis. Conclusions: Although we suspect the incidence of SARS-COV-2 infection through the ocular surface is extremely low, the nosocomial infection of SARS-CoV-2 through the eyes after occupational exposure is a potential route. The inefficient diagnostic method and the sampling time lag may contribute to the lower positive rate of conjunctival swab samples of SARS-COV-2. Therefore, to lower the SARS-COV-2 nosocomial infection, the protective goggles should be wore in all the health care workers.","2020-02-26","Xufang Sun; Xian Zhang; Xuhui Chen; Liwen Chen; Chaohua Deng; Xiaojing Zou; Weiyong Liu; Huimin Yu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20027938"],["hk491mr6","46634d03aa169aab9c372746ebaf3aaa65d7c7d9","medrxiv","Case fatality rate of novel coronavirus disease 2019 in China","10.1101/2020.02.26.20028076",null,null,"medrvix","Abstract Background: A pandemic of coronavirus disease 2019 (COVID-19) which have caused more than 80 thousand persons infected globally is still ongoing. This study aims to calculate its case fatality rate (CFR). Methods: The method, termed as converged CFR calculation, was based on the formula of dividing the number of known deaths by the number of confirmed cases T days before, where T was an average time period from case confirmation to death. It was found that supposing a T, if it was smaller (bigger) than the true T, calculated CFRs would gradually increase (decrease) to infinitely near the true T with time went on. According to the law, the true T value could be determined by trends of daily CFRs calculated with different assumed T values (left of true T is decreasing, right is increasing). Then the CFR could be calculated. Results: CFR of COVID-19 in China except Hubei Province was 0.8% to 0.9%. So far, the CFR had accurately predicted the death numbers more than 3 weeks. CFR in Hubei of China was 5.4% by which the calculated death number corresponded with the reported number for 2 weeks. Conclusion: The method could be used for CFR calculating while pandemics are still ongoing. Dynamic monitoring of the daily CFRs trends could help outbreak-controller to have a clear vision in the timeliness of the case confirmation.","2020-02-26","Rui Qi; Chao Ye; Xiang-rong Qin; Xue-Jie Yu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028076"],["011k6mm0","c954675ee859e2f7b8f352a398a67469b50f05de","medrxiv","Evaluation of the clinical characteristics of suspected or confirmed cases of COVID-19 during home care with isolation: A new retrospective analysis based on O2O","10.1101/2020.02.26.20028084",null,null,"medrvix","Summary Background The recent outbreak of the novel coronavirus in December 2019 (COVID-19) has activated top-level response nationwide. We developed a new treatment model based on the online-to-offline (O2O) model for the home isolated patients, because in the early stages the medical staff were insufficient to cope with so many patients. Methods In this single-centered, retrospective study, we enrolled 48 confirmed/suspected COVID-19 patients who underwent home isolation in Wuhan between January 6 and January 31, 2020. By WeChat and online document editing all patients were treated with medical observation scale. The clinical indications such as Fever, Muscle soreness, Dyspnea and Lack of strength were collected with this system led by medical staff in management, medicine, nursing, rehabilitation and psychology. Findings The mean(SD) age of 48 patients was 39.08(13.88) years, 35(72.9%) were women. Compared with non-hospitalized patients, inpatients were older(≥8805;70years, 2.4% vs 33.3%, P<0.04). All inpatients had fever, 50% inpatients had coughs and showed infiltration in both lungs at the time of diagnosis. 33.3% inpatients exhibited negative changes in their CT results at initial diagnosis. The body temperature of non-hospitalized patients with mild symptoms returned to normal by day 4-5. While dyspnea peaked on day 6 for non-hospitalized patients with mild symptoms, it persisted in hospitalized patients and exacerbated over time. The lack of strength and muscle soreness were both back to normal by day 4 for non-hospitalized patients. Interpretation Monitoring the trends of symptoms is more important for identifying severe cases. Excessive laboratory data and physical examination are not necessary for the evaluation of patients with mild symptoms. The system we developed is the first to convert the subjective symptoms of patients into objective scores. This type of O2O, subjective-to-objective strategy may be used in regions with similar highly infectious diseases to minimize the possibility of infection among medical staff.","2020-02-29","Hui Xu; Sufang Huang; Shangkun Liu; Juan Deng; Bo Jiao; Ling Ai; Yaru Xiao; Li Yan; Shusheng Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028084"],["96wkqutc","2536b83acf84368d7c13be81fe07aa0575115da7","medrxiv","Estimation of country-level basic reproductive ratios for novel Coronavirus (COVID-19) using synthetic contact matrices","10.1101/2020.02.26.20028167",null,null,"medrvix","The outbreak of novel coronavirus (COVID-19) has the potential for global spread, infecting large numbers in all countries. In this case, estimating the country-specific basic reproductive ratio is a vital first step in public-health planning. The basic reproductive ratio (R0) is determined by both the nature of pathogen and the network of contacts through which the disease can spread - with this network determined by socio-demographics including age-structure and household composition. Here we focus on the age-structured transmission within the population, using data from China to inform age-dependent susceptibility and synthetic age-mixing matrices to inform the contact network. This allows us to determine the country-specific basic reproductive ratio as a multiplicative scaling of the value from China. We predict that R0 will be highest across Eastern Europe and Japan, and lowest across Africa, Central America and South-Western Asia. This pattern is largely driven by the ratio of children to older adults in each country and the observed propensity of clinical cases in the elderly.","2020-02-27","Joe Hilton; Matt J Keeling",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028167"],["fcmzdcuh","a625e5d6fc8956c231c500a40e887e248a69de61","medrxiv","Clinical characteristics of 82 death cases with COVID-19","10.1101/2020.02.26.20028191",null,null,"medrvix","Background A recently developing pneumonia caused by SARS-CoV-2 was originated in Wuhan, China, and has quickly spread across the world. We reported the clinical characteristics of 82 death cases with COVID-19 in a single center. Methods Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital′s electronic medical records according to previously designed standardized data collection forms. Findings All patients were local residents of Wuhan, and the great proportion of them were diagnosed as severe illness when admitted. Most of the death cases were male (65.9%). More than half of dead patients were older than 60 years (80.5%) and the median age was 72.5 years. The bulk of death cases had comorbidity (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), following by sepsis syndrome/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhage, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On the admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of >5 (94.5%), high systemic immune-inflammation index of >500 (89.2%), increased C-reactive protein level (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%). A high level of IL-6 (>10 pg/ml) was observed in all detected patients. Median time from initial symptom to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p=0.002), alanine aminotransferase (p=0.037) and time from initial symptom to death were interestingly observed. Conclusion Older males with comorbidities are more likely to develop severe disease, even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but either virus itself or cytokine release storm mediated damage to other organ including cardiac, renal, hepatic, and hemorrhage should be taken seriously as well.","2020-02-27","Bicheng Zhang; Xiaoyang Zhou; Yanru Qiu; Fan Feng; Jia Feng; Yifan Jia; Hengcheng Zhu; Ke Hu; Jiasheng Liu; Zaiming Liu; Shihong Wang; Yiping Gong; Chenliang Zhou; Ting Zhu; Yanxiang Cheng; Zhichao Liu; Hongping Deng; Fenghua Tao; Yijun Ren; Biheng Cheng; Ling Gao; Xiongfei Wu; Lilei Yu; Zhixin Huang; Zhangfan Mao; Qibin Song; Bo Zhu; Jun Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028191"],["87pamw52","7aa49aa2b76ea92b9265ad9d5a9d9ef0ea585b24","medrxiv","Community responses during the early phase of the COVID-19 epidemic in Hong Kong: risk perception, information exposure and preventive measures","10.1101/2020.02.26.20028217",null,null,"medrvix","Background: Community responses are important for outbreak management during the early phase when non-pharmaceutical interventions are the major preventive options. Therefore, this study aims to examine the psychological and behavioral responses of the community during the early phase of the COVID-19 epidemic in Hong Kong. Method: A cross-sectional online survey was launched within 36 hours after confirmed COVID-19 cases were first reported. Councilors of all 452 district council constituency areas were approached for survey dissemination. Respondent demographics, anxiety level, risk perception, sources to retrieve COVID-19 information, actual adoption and perceived efficacy of precautionary measures were collected. Result: Analysis from 1715 complete responses indicated high perceived susceptibility (89%) and high perceived severity (97%). Most respondents were worried about COVID-19 (97%), and had their daily routines disrupted (slightly/greatly: 98%). The anxiety level, measured by the Hospital Anxiety and Depression Scale, was borderline abnormal (9.01). Nearly all respondents were alert to the disease progression (99.5%). The most trusted information sources were doctors (84%), followed by broadcast (57%) and newspaper (54%), but they were not common information sources (doctor: 5%; broadcast: 34%; newspaper: 40%). Only 16% respondents found official websites reliable. Enhanced personal hygiene practices and travel avoidance to China were frequently adopted (>77%) and considered effective (>90%). The adoption of social-distancing measures was lower (39%-88%), and their drivers for greater adoption include: being female (adjusted odds ratio [aOR]:1.27), living in the New Territories (aOR:1.32-1.55), perceived as having good understanding of COVID-19 (aOR:1.84) and being more anxious (aOR:1.07). Discussion: Risk perception towards COVID-19 in the community was high. Most respondents are alert to the disease progression, and adopt self-protective measures. This study contributes by examining the psycho-behavioral responses of hosts, in addition to the largely studied mechanistic aspects, during the early phase of the current COVID-19 epidemic. The timely psychological and behavioral assessment of the community is useful to inform subsequent interventions and risk communication strategies as the epidemic progresses.","2020-02-27","Kin On Kwok; Kin Kit Li; Ho Hin Chan; Yuan Yuan Yi; Arthur Tang; Wan In Wei; Yeung Shan Wong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028217"],["dlivkn8l","111b9a6e91c938696fcdb4cb128b8ae739dbe11c","medrxiv","Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China","10.1101/2020.02.26.20028225",null,null,"medrvix","Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2. Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.","2020-02-27","Pengfei Cui; Zhe Chen; Tian Wang; Jun Dai; Jinjin Zhang; Ting Ding; Jingjing Jiang; Jia Liu; Cong Zhang; Wanying Shan; Sheng Wang; Yueguang Rong; Jiang Chang; Xiaoping Miao; Xiangyi Ma; Shixuan Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028225"],["yk9u4fvp","15e5dd0eb126bfe384c25ba551317eb9ba2ad1ef","medrxiv","Perceptions of the Adult US Population regarding the Novel Coronavirus Outbreak","10.1101/2020.02.26.20028308",null,null,"medrvix","Background: COVID-19 outbreak is spreading globally. Although the risk of infection in the US is currently low, it is important to understand the public perception of risk and trust in sources of information to better inform public health messaging. In this study, we surveyed the adult US population to understand their risk perceptions about the COVID-19 outbreak. Methods and Findings: We used an online platform to survey 718 adults in the US in early February 2020 using a questionnaire that we developed. Our sample was fairly similar to the general adult US population in terms of age, gender, race, ethnicity and education. We found that 69% of the respondents wanted the scientific/public health leadership (either the CDC Director or NIH Director) to lead the US response to COVID-19 outbreak as compared to 14% who wanted the political leadership (either the president or the Congress) to lead the response. Risk perception was low (median score of 5 out of 10) with the respondents trusting health professionals and health officials for information on COVID-19. Majority of the respondents were in favor of strict infection prevention policies to control the outbreak. Conclusion: Given our results, the public health/scientific leadership should be at the forefront of the COVID-19 response to promote trust.","2020-02-27","SarahAnn M McFadden; Amyn A Malik; Obianuju G Aguolu; Kathryn S Willebrand; Saad B Omer",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028308"],["cv3qgno3","0fbb18050e29ca78191625d42576b1c574027377","medrxiv","Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP","10.1101/2020.02.26.20028373",null,null,"medrvix","The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.","2020-02-29","Yinhua Zhang; Nelson Odiwuor; Jin Xiong; Luo Sun; Raphael Ohuru Nyaruaba; Hongping Wei; Nathan A Tanner",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028373"],["px0twvhs","4bd697fdded7ee65154e28def586a7e9b60e7078","medrxiv","Transmission characteristics of the COVID-19 outbreak in China: a study driven by data","10.1101/2020.02.26.20028431",null,null,"medrvix","The COVID-19 outbreak has been a serious public health threat worldwide. We use individually documented case descriptions of COVID-19 from China (excluding Hubei Province) to estimate the distributions of the generation time, incubation period, and periods from symptom onset to isolation and to diagnosis. The recommended 14-day quarantine period may lead to a 6.7% failure for quarantine. We recommend a 22-day quarantine period. The mean generation time is 3.3 days and the mean incubation period is 7.2 days. It took 3.7 days to isolate and 6.6 days to diagnose a patient after his/her symptom onset. Patients may become infectious on average 3.9 days before showing major symptoms. This makes contact tracing and quarantine ineffective. The basic reproduction number is estimated to be 1.54 with contact tracing, quarantine and isolation, mostly driven by super spreaders.","2020-03-01","Meili Li; Pian Chen; Qianqian Yuan; Baojun Song; Junling Ma",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028431"],["onkdv388","54ae20515fcef8417e255b19df1a5bf6f2ab40bb","medrxiv","Modelling the coronavirus disease (COVID-19) outbreak on the Diamond Princess ship using the public surveillance data from January 20 to February 20, 2020","10.1101/2020.02.26.20028449",null,null,"medrvix","The novel coronavirus disease 2019 (COVID-19) outbreak on the Diamond Princess ship has caused over 634 cases as of February 20, 2020. We model the transmission process on the ship with a stochastic model and estimate the basic reproduction number at 2.2 (95%CI: 2.1−2.4). We estimate a large dispersion parameter than other coronaviruses, which implies that the virus is difficult to go extinction. The epidemic doubling time is at 4.6 days (95%CI: 3.0−9.3), and thus timely actions were crucial. The lesson learnt on the ship is generally applicable in other settings.","2020-02-29","Shi Zhao; Peihua Cao; Daozhou Gao; Zian Zhuang; Marc Chong; Yongli Cai; Jinjun Ran; Kai Wang; Yijun Lou; Weiming Wang; Lin Yang; Daihai He; Maggie H Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028449"],["k260c04b",null,"medrxiv","The Effects of \"Fangcang, Huoshenshan, and Leishenshan\" Makeshift Hospitals and Temperature on the Mortality of COVID-19","10.1101/2020.02.26.20028472",null,null,"medrvix","Background In December 2019, a novel coronavirus disease (COVID-19) broke out in Wuhan, China, however, the factors affecting the mortality remain unclear. Methods Thirty-two days of data that were shared by China National Health Commission and China Weather Net were collected using standard forms. The difference in the mortality of confirmed and severe cases before and after the use of Fangcang, Huoshenshan, and Leishenshan makeshift hospitals (MSHs) was tested using Mann-Whitney U test. We also studied whether air temperature (AT) could affect the above outcomes of COVID-19 cases by performing Spearman analysis. Results The mortality of confirmed cases was significantly decreased both in Wuhan (U = 1, P < 0.001) and Hubei (U = 0, P < 0.001), while in non-Hubei regions, as a contrast, the mortality of confirmed cases remained unchanged (U = 40, P = 0.139). However, another eight days later, changes in the mortality in non-Hubei regions also became significant (U = 73, P = 0.039). Mortality of confirmed cases was found to be significantly correlated with temperature both in Wuhan (r = -0.441, P = 0.012) and Hubei (r = -0.440, P = 0.012). Conclusions Our findings indicated that both the use of MSHs and the rise of AT were beneficial to the survival of COVID-19 cases.","2020-02-29","Yuwen Cai; Tianlun Huang; Xin Liu; Gaosi Xu",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028472"],["zriuh5q5","691da9d5576aa959e22114a1ebb35548efd276a0","medrxiv","Infection Dynamics of Coronavirus Disease 2019 (Covid-19) Modeled with the Integration of the Eyring Rate Process Theory and Free Volume Concept","10.1101/2020.02.26.20028571",null,null,"medrvix","The Eyrings rate process theory and free volume concept, two very popular theories in chemistry and physics fields, are employed to treat infectious disease transmissions. The susceptible individ- uals are assumed to move stochastically from one place to another. The virus particle transmission rate is assumed to obey the Eyring rate process theory and also controlled by how much free volume available in a system. The transmission process is considered to be a sequential chemical reaction, and the concentrations or fractions of four epidemiological compartments, the susceptible, the exposed, the infected, and the removed, can be derived and calculated. The obtained equations show that the basic reproduction number, R0, is not a constant, dependent on the volume frac- tion of virus particles, virus particle size, and virus particle packing structure, the energy barrier associated with susceptible individuals, and environment temperature. The developed models are applied to treat coronavirus disease 2019 (Covid-19) transmission and make predictions on peak time, peak infected, and R0. Our work provides a simple and straightforward approach to estimate how infection diseases evolve and how many people may be infected.","2020-02-29","Tian Hao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028571"],["nsp2t0i6",null,"medrxiv","Heart injury signs are associated with higher and earlier mortality in coronavirus disease 2019 (COVID-19)","10.1101/2020.02.26.20028589",null,null,"medrvix","Importance: Heart injury can be easily induced by viral infection such as adenovirus and enterovirus. However, whether coronavirus disease 2019 (COVID-19) causes heart injury and hereby impacts mortality has not yet been fully evaluated. Objective: To explore whether heart injury occurs in COVID-19 on admission and hereby aggravates mortality later. Design, Setting, and Participants A single-center retrospective cohort study including 188 COVID-19 patients admitted from December 25, 2019 to January 27, 2020 in Wuhan Jinyintan Hospital, China; follow up was completed on February 11, 2020. Exposures: High levels of heart injury indicators on admission (hs-TNI; CK; CK-MB; LDH; α-HBDH). Main Outcomes and Measures: Mortality in hospital and days from admission to mortality (survival days). Results: Of 188 patients with COVID-19, the mean age was 51.9 years (standard deviation: 14.26; range: 21~83 years) and 119 (63.3%) were male. Increased hs-TnI levels on admission tended to occur in older patients and patients with comorbidity (especially hypertension). High hs-TnI on admission (≥ 6.126 pg/mL), even within the clinical normal range (0~28 pg/mL), already can be associated with higher mortality. High hs-TnI was associated with increased inflammatory levels (neutrophils, IL-6, CRP, and PCT) and decreased immune levels (lymphocytes, monocytes, and CD4+ and CD8+ T cells). CK was not associated with mortality. Increased CK-MB levels tended to occur in male patients and patients with current smoking. High CK-MB on admission was associated with higher mortality. High CK-MB was associated with increased inflammatory levels and decreased lymphocytes. Increased LDH and α-HBDH levels tended to occur in older patients and patients with hypertension. Both high LDH and α-HBDH on admission were associated with higher mortality. Both high LDH and α-HBDH were associated with increased inflammatory levels and decreased immune levels. hs-TNI level on admission was negatively correlated with survival days (r= -0.42, 95% CI= -0.64~-0.12, P=0.005). LDH level on admission was negatively correlated with survival days (r= -0.35, 95% CI= -0.59~-0.05, P=0.022). Conclusions and Relevance: Heart injury signs arise in COVID-19, especially in older patients, patients with hypertension and male patients with current smoking. COVID-19 virus might attack heart via inducing inflammatory storm. High levels of heart injury indicators on admission are associated with higher mortality and shorter survival days. COVID-19 patients with signs of heart injury on admission must be early identified and carefully managed by cardiologists, because COVID-19 is never just confined to respiratory injury.","2020-02-29","Chaomin Wu; Xianglin Hu; Jianxin Song; Chunling Du; Jie Xu; Dong Yang; Dechang Chen; Ming Zhong; Jinjun Jiang; Weining Xiong; Ke Lang; Yuye Zhang; Guohua Shi; Lei Xu; Yuanlin Song; Xin Zhou; Ming Wei; Junhua Zheng",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.20028589"],["fvig79k3","11875dfda6cd04d5032611edf0e340582ceb620f","biorxiv","Predictions for the binding domain and potential new drug targets of 2019-nCoV","10.1101/2020.02.26.961938",null,null,"biorxiv","AbstractAn outbreak of new SARS-like viral in Wuhan, China has been named 2019-nCoV. The current state of the epidemic is increasingly serious, and there has been the urgent necessity to develop an effective new drug. In previous studies, it was found that the conformation change in CTD1 was the region where SARS-CoV bound to human ACE2. Although there are mutations of the 2019-nCoV, the binding energy of ACE2 remains high. The surface glycoprotein of 2019-nCoV was coincident with the CTD1 region of the S-protein by comparing the I-TASSER prediction model with the actual SARS model, which suggests that 2019-nCoV may bind to the ACE2 receptor through conformational changes. Furthermore, site prediction on the surface glycoprotein of 2019-nCoV suggests some core amino acid area may be a novel drug target against 2019-nCoV.","2020-03-02","Zehua Zeng; Zhi Luo; Hongwu Du",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.961938"],["wx7nheay","313dd0869f169624e78d770963eacb0d1641cafa","biorxiv","Structure of Mpro from COVID-19 virus and discovery of its inhibitors","10.1101/2020.02.26.964882",null,null,"biorxiv","SUMMARYA new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan1-4. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Mpro is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus5,6. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus Mpro in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of Mpro. Six of these inhibit Mpro with IC50 values ranging from 0.67 to 21.4 μM. Ebselen also exhibited strong antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available.","2020-02-27","Zhenming Jin; Xiaoyu Du; Yechun Xu; Yongqiang Deng; Meiqin Liu; Yao Zhao; Bing Zhang; Xiaofeng Li; Leike Zhang; Chao Peng; Yinkai Duan; Jing Yu; Lin Wang; Kailin Yang; Fengjiang Liu; Rendi Jiang; Xinglou Yang; Tian You; Xiaoce Liu; Xiuna Yang; Fang Bai; Hong Liu; Xiang Liu; Luke W. Guddat; Wenqing Xu; Gengfu Xiao; Chengfeng Qin; Zhengli Shi; Hualiang Jiang; Zihe Rao; Haitao Yang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.964882"],["egntml7e","0a32446730827ad8152c6a61e4738e4e0b231412","biorxiv","Artesunate interacts with Vitamin D receptor to reverse mouse model of sepsis-induced immunosuppression via enhancing autophagy","10.1101/2020.02.26.966143",null,null,"biorxiv","AbstractBackground and PurposeImmunosuppression is the predominant cause of mortality for sepsis due to failure to eradicate invading pathogens. Unfortunately, no effective and specific drugs capable of reversing immunosuppression are available for clinical use. Increasing evidence implicates vitamin D receptor (VDR) involved in sepsis-induced immunosuppression. Herein, artesunate (AS) was discovered to reverse sepsis-induced immunosuppression and its molecular mechanism is investigated.Experimental ApproachEffect of artesunate on sepsis-induced immunosuppression was investigated in mice and in vitro. VDR was predicted to be an interacted candidate of AS by bioinformatics predict, then identified using PCR and immunoblotting. VDR, ATG16L1 and NF-κB p65 were modified to investigate the alteration of AS’s effect on pro-inflammatory cytokines release, bacteria clearance and autophagy activities in sepsis-induced immunosuppression.Key ResultsAS significantly reduced the mortality of cecal ligation and puncture (CLP)-induced sepsis immunosuppression mice challenged with Pseudomonas Aeruginosa, and enhanced proinflammatory cytokines release and bacterial clearance to reverse sepsis-induced immunosuppression in vivo and in vitro. Mechanically, AS interacted with VDR thereby inhibited the nuclear translocation of VDR, then influencing ATG16L1 transcription and subsequent autophagy activity. In addition, AS inhibited physical interaction between VDR and NF-κB p65 in LPS tolerance macrophages, then promoted nuclear translocation of NF-κB p65, which activated the transcription of NF-κB p65 target genes such as pro-inflammatory cytokines.Conclusion and ImplicationsOur findings provide an evidence that AS interacted with VDR to reverse sepsis-induced immunosuppression in an autophagy and NF-κB dependent way, highlighting a novel approach for sepsis treatment and drug repurposing of AS in the future.","2020-02-27","Shenglan Shang; Jiaqi Wu; Xiaoli Li; Xin Liu; Pan Li; Chunli Zheng; Yonghua Wang; Songqing Liu; Jiang Zheng; Hong Zhou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.966143"],["7tkvl894","35604e3a5d38d6794543372e4dd9ccf1d8d9cd2a","biorxiv","CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design","10.1101/2020.02.26.967026",null,null,"biorxiv","AbstractThe emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.","2020-03-02","Hayden C. Metsky; Catherine A. Freije; Tinna-Solveig F. Kosoko-Thoroddsen; Pardis C. Sabeti; Cameron Myhrvold",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.26.967026"],["jadzias9","ae61300f4e7a12a68baf114f199eb51940c7aad3","medrxiv","Prediction of the Epidemic of COVID-19 Based on Quarantined Surveillance in China","10.1101/2020.02.27.20027169",null,null,"medrvix","Background and Objective: To predict the epidemic of COVID-19 based on quarantined surveillance from real world in China by modified SEIR model different from the previous simply mathematical model. Design and Methods: We forecasted the epidemic of COVID-19 based on current clinical and epidemiological data and built a modified SEIR model to consider both the infectivity during incubation period and the influence on the epidemic from strict quarantined measures. Results: The peak time of the curve for the infected newly diagnosed as COVID-19 should substantially present on Feb.5,2020(in non-Hubei areas) and Feb.19,2020(in Hubei. It is estimated that the peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and in Hubei province on Mar.10,2020,and the total number of the patients diagnosed as COVID-19 is 18,000 in non-Hubei areas and 78,000-96,000 in Hubei.The Chinese COVID-19 epidemic can be completely controlled in May,2020. Conclusions: COVID-19 is only a local outbreak in Hubei Province,China.It can be probably avoided the pandemic of global SARS-CoV-2 cases rise with the great efforts by Chinese government and its people.","2020-02-29","Rui Li; Wenliang Lu; Xifei Yang; Peihua Feng; Ozarina Muqimova; Xiaoping Chen; Gang Wei",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20027169"],["2zm4dq0u","2e41aee6651cf4cab4edf5aa67cc401c2bef9d67","medrxiv","Sex differences in clinical findings among patients with coronavirus disease 2019 (COVID-19) and severe condition","10.1101/2020.02.27.20027524",null,null,"medrvix","Objective: To compare the sex differences in the clinical findings among patients with severe coronavirus disease 2019 (COVID-19). Methods: We retrospectively collected data of 47 patients diagnosed as severe type of COVID-19 from February 8 to 22, 2020, including demographics, illness history, physical examination, laboratory test, management, and compared differences between men and women. Results: Of the 47 patients, 28 (59.6%) were men. The median age was 62 years, and 30 (63.8%) had comorbidities. The initial symptoms were mainly fever (34 [72.3%]), cough (36 [76.6%]), myalgia (5 [10.6%]) and fatigue (7 [14.9%]). Procalcitonin level was higher in men than in women (0.08 vs. 0.04ng/ml, p=0.002). N-terminal-pro brain natriuretic peptide increased in 16 (57.1%) men and 5 (26.3%) women (p=0.037). Five men (17.9%) had detected positive influenza A antibody, but no women. During 2-week admission, 5 (17.9%) men and 1 (5.3%) woman were reclassified into the critical type due to deterioration. Mortality was 3.6% in men and 0 in women respectively. Four (21.1%) women and one man (3.6%) recovered and discharged from hospital. Conclusion: Sex differences may exist in COVID-19 patients of severe type. Men are likely to have more complicated clinical condition and worse in-hospital outcomes as compared to women.","2020-02-29","Jing Li; Yinghua Zhang; Fang Wang; Bing Liu; Hui Li; Guodong Tang; Zhigang Chang; Aihua Liu; Chunyi Fu; Jing Gao; Jing Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20027524"],["owjgs6ja","2d0285c90976dfa43d58eca6b7d1d7ef29994fc2","medrxiv","Therapeutic effects of dipyridamole on COVID-19 patients with coagulation dysfunction","10.1101/2020.02.27.20027557",null,null,"medrvix","The human coronavirus HCoV-19 infection can cause acute respiratory distress syndrome (ARDS), hypercoagulability, hypertension, extrapulmonary multiorgan dysfunction. Effective antiviral and anti-coagulation agents with safe clinical profiles are urgently needed to improve the overall prognosis. We screened an FDA approved drug library and found that an anticoagulant agent dipyridamole (DIP) suppressed HCoV-19 replication at an EC50 of 100 nM in vitro. It also elicited potent type I interferon responses and ameliorated lung pathology in a viral pneumonia model. In analysis of twelve HCoV-19 infected patients with prophylactic anti-coagulation therapy, we found that DIP supplementation was associated with significantly increased platelet and lymphocyte counts and decreased D-dimer levels in comparison to control patients. Two weeks after initiation of DIP treatment, 3 of the 6 severe cases (60%) and all 4 of the mild cases (100%) were discharged from the hospital. One critically ill patient with extremely high levels of D-dimer and lymphopenia at the time of receiving DIP passed away. All other patients were in clinical remission. In summary, HCoV-19 infected patients could potentially benefit from DIP adjunctive therapy by reducing viral replication, suppressing hypercoagulability and enhancing immune recovery. Larger scale clinical trials of DIP are needed to validate these therapeutic effects.","2020-02-29","Xiaoyan Liu; Zhe Li; Shuai Liu; Zhanghua Chen; Zhiyao Zhao; Yi-you Huang; Qingling Zhang; Jun Wang; Yinyi Shi; Yanhui Xu; Jing Sun; Huifang Xian; Rongli Fang; Fan Bai; Changxing Ou; Bei Xiong; Andrew M Lew; Jun Cui; Hui Huang; Jincun Zhao; Xuechuan Hong; Yuxia Zhang; Fulin Zhou; Hai-Bin Luo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20027557"],["s9j21zsy","bd2f7613f5818fc82f76c7d68bdac08825957fb9","medrxiv","A machine learning-based model for survival prediction in patients with severe COVID-19 infection","10.1101/2020.02.27.20028027",null,null,"medrvix","The sudden increase of COVID-19 cases is putting a high pressure on healthcare services worldwide. At the current stage, fast, accurate and early clinical assessment of the disease severity is vital. To support decision making and logistical planning in healthcare systems, this study leverages a database of blood samples from 404 infected patients in the region of Wuhan, China to identify crucial predictive biomarkers of disease severity. For this purpose, machine learning tools selected three biomarkers that predict the survival of individual patients with more than 90% accuracy: lactic dehydrogenase (LDH), lymphocyte and high-sensitivity C-reactive protein (hs-CRP). In particular, relatively high levels of LDH alone seem to play a crucial role in distinguishing the vast majority of cases that require immediate medical attention. This finding is consistent with current medical knowledge that high LDH levels are associated with tissue breakdown occurring in various diseases, including pulmonary disorders such as pneumonia. Overall, this paper suggests a simple and operable formula to quickly predict patients at the highest risk, allowing them to be prioritised and potentially reducing the mortality rate.","2020-03-01","Li Yan; Hai-Tao Zhang; Jorge Goncalves; Yang Xiao; Maolin Wang; Yuqi Guo; Chuan Sun; Xiuchuan Tang; Liang Jin; Mingyang Zhang; Xiang Huang; Ying Xiao; Haosen Cao; Yanyan Chen; Tongxin Ren; Fang Wang; Yaru Xiao; Sufang Huang; Xi Tan; Niannian Huang; Bo Jiao; Yong Zhang; Ailin Luo; Laurent Mombaerts; Junyang Jin; Zhiguo Cao; Shusheng Li; Hui Xu; Ye Yuan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20028027"],["xiynfpuw","264a816e7099246c13a5dad3a581a474d85d50bd","medrxiv","Hypokalemia and Clinical Implications in Patients with Coronavirus Disease 2019 (COVID-19)","10.1101/2020.02.27.20028530",null,null,"medrvix","BACKGROUND: SARS-CoV-2 has caused a series of COVID-19 globally. SARS-CoV-2 binds angiotensin I converting enzyme 2 (ACE2) of renin-angiotensin system (RAS) and causes prevalent hypokalemia METHODS: The patients with COVID-19 were classified into severe hypokalemia, hypokalemia, and normokalemia group. The study aimed to determine the relationship between hypokalemia and clinical features, the underlying causes and clinical implications of hypokalemia. RESULTS: By Feb 15, 2020, 175 patients with COVID-19 (92 women and 83 men; median age, 46 [IQR, 34-54] years) were admitted to hospital in Wenzhou, China, consisting 39 severe hypokalemia-, 69 hypokalemia-, and 67 normokalemia patients. Gastrointestinal symptoms were not associated with hypokalemia among 108 hypokalemia patients (P>0.05). Body temperature, CK, CK-MB, LDH, and CRP were significantly associated with the severity of hypokalemia (P<0.01). 93% of severe and critically ill patients had hypokalemia which was most common among elevated CK, CK-MB, LDH, and CRP. Urine K+ loss was the primary cause of hypokalemia. severe hypokalemia patients was given 3 g/day, adding up to an average of 34 (SD=4) g potassium during hospital stay. The exciting finding was that patients responded well to K+ supplements when they were inclined to recovery. CONCLUSIONS: Hypokalemia is prevailing in patients with COVID-19. The correction of hypokalemia is challenging because of continuous renal K+ loss resulting from the degradation of ACE2. The end of urine K+ loss indicates a good prognosis and may be a reliable, in-time, and sensitive biomarker directly reflecting the end of adverse effect on RAS system.","2020-02-29","dong chen; Xiaokuni Li; qifa song; Chenchan Hu; Feifei Su; Jianyi Dai",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20028530"],["ry3m6x8c","75e68869f9b65bca661e768402395d9dede6de2c","medrxiv","Modeling the Epidemic Dynamics and Control of COVID-19 Outbreak in China","10.1101/2020.02.27.20028639",null,null,"medrvix","The coronavirus disease 2019 (COVID-19) is rapidly spreading over China and more than 30 countries in last two months. COVID-19 has multiple characteristics distinct from other infectious diseases, including a high infectivity during incubation, time delay between real dynamics and daily observed case numbers, and the effects from multiple quarantine and control measures. We develop a model SUQC to adequately characterizes the dynamics of COVID-19 and explicitly model the control by artificial measures, which is more suitable for analysis than other existing epidemic models. The SUQC model is applied to the daily released data of the confirmed infected to analyze the outbreak of COVID-19 in Wuhan, Hubei (excluding Wuhan), China (excluding Hubei) and four first-tier cities of China. We find that, before January 30, 2020, all these regions except Beijing have a reproductive number R>1, and after January 30, all regions have a reproductive number R<1, indicating the effectiveness of the quarantine and control measures in inhibiting COVID-19. The confirmation rate of Wuhan is 0.0643, significantly lower than 0.1914 of Hubei (excluding Wuhan) and 0.2189 of China (excluding Hubei), but increases to 0.3229 after Feb 12th when clinical diagnosis was adopted. The un-quarantined infected individuals in Wuhan on February 12, 2020 is as high as 3,509 and decreases to 334 on February 21th, 2020. After fitting the model with recent data, we predict that the end times of COVID-19 of Wuhan and Hubei are around late-March, of China (excluding Hubei) around mid-March, and of the four tier-one cities before March 2020. A total of 80,511 individuals of the whole country are infected, among which 49,510 are from Wuhan, 17,679 from Hubei(excluding Wuhan), and the rest 13,322 from other regions of China (excluding Hubei). We suggest the rigorous quarantine and control measures should be kept before March in Beijing, Shanghai, Guangzhou and Shenzhen, and before late-March in Hubei. The model can also be useful to predict the trend of epidemic and provide quantitative guide for other counties in a high risk of outbreak, such as South Korea, Japan and Iran.","2020-02-29","Shilei Zhao; Hua Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20028639"],["rbnfh89u","b579c3547bba2a33057373d57b7c05f37dd4cfc3","medrxiv","Evaluation of Enzyme-Linked Immunoassay and Colloidal Gold- Immunochromatographic Assay Kit for Detection of Novel Coronavirus (SARS-Cov-2) Causing an Outbreak of Pneumonia (COVID-19)","10.1101/2020.02.27.20028787",null,null,"medrvix","Abstract BACKGROUND: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Travel-associated cases have also been reported in other countries. The number of cases has increased rapidly but laboratory diagnosis is limited. METHODS: We collect two groups of cases diagnosed with COVID-19 for experiments. One group collected 63 samples for Enzyme-linked immunosorbent assay (ELISA) IgG and IgM antibodies. The other group collected 91 plasma samples for colloidal gold-immunochromatographic assay (GICA). RESULTS: The sensitivity of the combined ELISA IgM and ELISA IgG detection was 55/63 ( 87.3%), The sensitivity of the combined GICA IgM and GICA IgG detection was 75/91 ( 82.4%), Both methods are negative for healthy controls, specificity of 100% .There is no significant difference between the sensitivity of between ELISA and GICA (IgM+ IgG). CONCLUSIONS: ELISA and GICA for specific IgM and IgG antibodies are conventional serological assays, they are simple, fast, and safe, the results can be used for clinical reference, and the huge clinical diagnosis and treatment pressure can be greatly relieved.","2020-03-01","Jie Xiang; Mingzhe Yan; Hongze Li; Ting Liu; Chenyao Lin; Shuang Huang; Changxin Shen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20028787"],["z7a3g6e8","905603e9db862f9b4b42a95f51e9c8d0015d1dff","medrxiv","Transmission potential of COVID-19 in South Korea","10.1101/2020.02.27.20028829",null,null,"medrvix","Since the first identified individual of 2019 novel coronavirus (COVID-19) infection on Jan 20, 2020 in South Korea, the number of confirmed cases rapidly increased. As of Feb 26, 2020, 1,261 cases of COVID-19 including 12 deaths were confirmed in South Korea. Using the incidence data of COVID-19, we estimate the reproduction number at 1.5 (95% CI: 1.4-1.6), which indicates sustained transmission and support the implementation of social distancing measures to rapidly control the outbreak.","2020-02-29","Eunha Shim; Amna Tariq; Wongyeong Choi; Yiseul Lee; Gerardo Chowell",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20028829"],["hgn6o5yf",null,"medrxiv","A simple ecological model captures the transmission pattern of the coronavirus COVID-19 outbreak in China","10.1101/2020.02.27.20028928",null,null,"medrvix","The rapid spread of the 2019 novel coronavirus (COVID-19), initially reported in the city of Wuhan in China, and quickly transmitted to the entire nation and beyond, has become an international public health emergency. Estimating the final number of infection cases and the turning point (time with the fastest spreading rate) is crucial to assessing and improving the national and international control measures currently being applied. In this paper we develop a simple model based on infectious growth with a time-varying infection rate, and estimate the final number of infections and the turning point using data updated daily from 3 February 2020, when China escalated its initial public health measures, to 10 February. Our model provides an extremely good fit to the existing data and therefore a reasonable estimate of the time-varying infection rate that has largely captured the transmission pattern of this epidemic outbreak. Our estimation suggests that (i) the final number of infections in China could reach 78,000 with an upper 95% confidence limit of 88,880; (ii) the turning point of the fastest spread was on the 4th or the 5th of February; and (iii) the projected period for the end of the outbreak (i.e., when 95% of the final predicted number of infection is reached) will be the 24th of February, with an upper 95% confidence limit on the 19th of March. This suggests that the current control measures in China are excellent, and more than sufficient to contain the spread of this highly infectious novel coronavirus, and that the application of such measures could be considered internationally for the global control of this outbreak.","2020-02-29","Feng Zhang; Jinmei Zhang; Menglan Cao; Cang Hui",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20028928"],["9hh17k86","68ed7bb581b0cf32116ee46767c08e3180b05c34","medrxiv","Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China","10.1101/2020.02.27.20029009",null,null,"medrvix","Background Although the outbreak of Coronavirus disease 2019 (COVID-19) has caused over 2200 deaths in China, there was no study about death yet. We aimed to describe the clinical characteristics of non-survivors with COVID-19. Methods For this retrospective, single-center study, we included 36 non-survivors with COVID-19 in the Fifth Hospital of Wuhan. Cases were confirmed by real-time RT-PCR between Jan 21 and Feb 10, 2020 according to the recommended protocol. The epidemiological, demographic, clinical, laboratory, radiological and treatment data were collected and analyzed. Outcomes were followed up until Feb 14, 2020. This study was approved by the ethics commissions of the Fifth Hospital of Wuhan, with a waiver of informed consent due to a public health outbreak investigation. Results We included 36 patients who died from COVID-19. The mean age of the patients was 69.22 years (SD 9.64, range 50-90). 25(69.44%) patients were males, and 11 (30.56%) female. 26 (72.22%) patients had chronic diseases, mainly including hypertension, cardiovascular disease and diabetes. Patients had common clinical symptoms of fever (34 [94.44%] patients), cough (28 [77.78%] patients), shortness of breath (21 [58.33%] patients), and fatigue (17 [47.22%] patient). Chest computed tomographic scans showed that 31 (96.88%) patients had bilateral pneumonia. Lymphopenia occurred in 24 patients (70.59%), decreased albumin (30.18, [SD, 4.76]) in 25 patients (80.65%), elevated D-dimer (8.64 [IQR, 2.39-20]) in 27 patients (100%), and elevated lactate dehydrogenase (502.5 U/L [IQR, 410-629]) in 26 patients (100%). Nearly all of the patients have elevated CRP (106.3 mg/L [IQR, 60.83-225.3]), PCT (0.61 ng/ml [IQR, 0.16-2.10]) and IL-6 (100.6 pg/ml [IQR, 51.51-919.5]). Most patients received antiviral therapy and antibiotic therapy, and more than half of patients received glucocorticoid therapy (25 [69.44%]). All the patients had acute respiratory distress syndrome (ARDS). The median time from onset to ARDS was 11 days. One (2.78%) patient presented with acute renal injury. The median time from onset to death was 17 days. Interpretation Lots of patients died from COVID-19 till now. The median survival time of these non-survivors from onset to death was about 2 weeks. Most patients were older males with comorbidities. They finally progressed to ARDS. The median time from onset to ARDS was 11 days. Gradually decreased lymphocytes and increased inflammation biomarkers were common, and need to be monitored in the routine treatment.","2020-02-29","Ying Huang; Rui Yang; Ying Xu; Ping Gong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.20029009"],["618glydc","1b9fed2824e97db8cb35912549dd2e9cde7b6c18","biorxiv","Strategies for vaccine design for corona virus using Immunoinformatics techniques","10.1101/2020.02.27.967422",null,null,"biorxiv","ABSTRACTThe cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores −725.0 Kcal/mole and −786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study.","2020-03-02","Anamika Basu; Anasua Sarkar; Ujjwal Maulik",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.967422"],["eaglecq7","92c020779e68e82d9a5f5898435e618caa63a099","biorxiv","TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection","10.1101/2020.02.27.967588",null,null,"biorxiv","AbstractFaced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the “coronavirus” should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.","2020-03-02","Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Chunli Zhang; Yiqing Xue; Guangming Tan; Gang Niu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.967588"],["1f9kjtmm",null,"biorxiv","The SARS-CoV-2 receptor ACE2 expression of maternal-fetal interface and fetal organs by single cell transcriptome study","10.1101/2020.02.27.967760",null,null,"biorxiv","Recent studies have demonstrated that SARS-CoV-2 cell entry depends on both ACE2 and TMPRSS2 genes (DOI: 10.1016/j.cell.2020.02.052), but our current work only focus on ACE2, which is insufficient to support the conclusion of this paper. So the authors have withdrawn their manuscript whilst they perform additional experiments and analysis to test some of their conclusions further. Therefore, the authors do not wish this work to be cited as reference for the project.","2020-02-27","Mengmeng Li; Liang Chen; Chenglong Xiong; Xiangjie Li",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.967760"],["9ruhvpbv","ae66f22f886d3b019934ae237c11a5fb53dbefb2","biorxiv","Prediction of receptorome for human-infecting virome","10.1101/2020.02.27.967885",null,null,"biorxiv","AbstractThe virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome.","2020-02-28","Zheng Zhang; Sifan Ye; Aiping Wu; Taijiao Jiang; Yousong Peng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.967885"],["mp3a9c9u","7445c94870da80c81cd17f6a4c1358004956ee36","biorxiv","Molecular Dynamics Simulations Indicate the SARS-CoV-2 Mpro Is Not a Viable Target for Small-Molecule Inhibitors Design","10.1101/2020.02.27.968008",null,null,"biorxiv","AbstractThe coronavirus outbreak took place in December 2019 and continues to spread worldwide. In the absence of an effective vaccine, inhibitor repurposing may seem a fruitful attempt. Here, we compared Mpros from SARS-CoV-2 and SARS-CoV. Despite a high level of sequence similarity, the binding sites of analysed proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, the analysis of the pockets’ conformational changes during the simulation time indicates their flexibility, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the SARS-CoV-2 Mpro with respect to mutations of the binding cavity and adjacent flexible loops indicates that the protein’s mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.","2020-03-02","Maria Bzówka; Karolina Mitusińska; Agata Raczyńska; Aleksandra Samol; Jack Tuszyński; Artur Góra",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.968008"],["3h1o0oz3","6d0127f985edbe088bc279865bef25a31f54a066","biorxiv","Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13","10.1101/2020.02.27.969006",null,null,"biorxiv","AbstractThe novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.","2020-03-02","Longxian Lv; Gaolei Li; Jinhui Chen; Xinle Liang; Yudong Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.27.969006"],["2vdvswda","24e17488d399c436305c819953beae2961214771","medrxiv","Epidemiological and clinical features of COVID-19 patients with and without pneumonia in Beijing, China","10.1101/2020.02.28.20028068",null,null,"medrvix","Background:SARS-CoV-2-caused coronavirus disease (COVID-19) is posinga large casualty. The features of COVID-19patients withand without pneumonia,SARS-CoV-2 transmissibility in asymptomatic carriers, and factors predicting disease progression remain unknown. Methods: We collected information on clinical characteristics, exposure history, andlaboratory examinations of all laboratory-confirmed COVID-19 patients admitted to PLA General Hospital. Cox regression analysis was applied to identify prognostic factors. The last follow-up was February 18, 2020. Results:We characterized 55 consecutive COVID-19 patients. The mean incubation was 8.42(95% confidence interval [CI], 6.55-10.29) days. The mean SARS-CoV-2-positive duration from first positive test to clearance was 9.71(95%CI, 8.21-11.22) days. COVID-19 course was approximately 2 weeks. Asymptomatic carriers might transmit SARS-CoV-2. Compared with patients without pneumonia, those with pneumonia were 15 years older and had a higher rate of hypertension, higher frequencies of having a fever and cough, and higher levels of interleukin-6 (14.61 vs. 8.06pg/mL, P=0.040), B lymphocyte proportion (13.0% vs.10.0%, P=0.024), low account (<190/μL) of CD8+ T cells (33.3% vs. 0, P=0.019). Multivariate Cox regression analysis indicated that circulating interleukin-6 andlactate independently predicted COVID-19 progression, with a hazard ratio (95%CI) of 1.052 (1.000-1.107) and 1.082 (1.013-1.155), respectively. During disease course,T lymphocytes were generally lower,neutrophils higher, in pneumonia patients than in pneumonia-free patients. CD8+ lymphocytes did not increase at the 20th days after illness onset. Conclusion: The epidemiological features areimportant for COVID-19 prophylaxis. Circulating interleukin-6 and lactateare independent prognostic factors. CD8+ T cell exhaustion might be critical in the development of COVID-19.","2020-03-03","Penghui Yang; Yibo Ding; Zhe Xu; Rui Pu; Ping Li; Jin Yan; Jiluo Liu; Fanping Meng; Lei Huang; Lei Shi; Tianjun Jiang; Enqiang Qin; Min Zhao; Dawei Zhang; Peng Zhao; Lingxiang Yu; Zhaohai Wang; Zhixian Hong; Zhaohui Xiao; Qing Xi; Dexi Zhao; Peng Yu; Caizhong Zhu; Zhu Chen; Shaogeng Zhang; Junsheng Ji; Guangwen Cao; Fusheng Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.20028068"],["73gxq2w6","54cfc3c68e1e4832fee5b3294e5673e37978ebdf","medrxiv","Risk factors related to hepatic injury in patients with corona virus disease 2019","10.1101/2020.02.28.20028514",null,null,"medrvix","Aims: Corona virus disease 2019 (COVID-19) has rapidly become the most severe public health issue all over the world. Despite respiratory symptoms, hepatic injury has also been observed in clinical settings. This study aimed to investigate the risk factors involved with hepatic injury in the patients with COVID-19. Methods: A total of 85 hospitalized patients who were diagnosed with COVID-19 in Beijing Youan Hospital were retrospectively analyzed. According to liver function, they were divided into ALT normal group (n=52) and ALT elevation group (n=33). Clinical features and laboratory data were compared between the two groups. The independent risk factors for liver injury were analyzed. Results: There were 33 patients with hepatic injury in our study, accounting for 38.8% (33/85). The patients in ALT elevation group were older than those in ALT normal group. The levels of lactic acid, CRP, myoglobin, and neutrophils were significantly higher in ALT elevation group. The lymphocytes and albumin were significantly lower in ALT elevation group. The proportion of severe and critical patients in ALT elevation group was significantly higher. Multivariate logistic regression analysis showed CRP ≥20 mg/L and lymphocyte count< 1.1*10^9/L were independently related to hepatic injury. Conclusions: Lymphopenia and CRP may serve as the risk factors related to hepatic injury in patients with COVID-19, which might be related to inflammatory cytokine storm in liver injury. Early detection and timely treatment of hepatic injury in patients with COVID-19 are necessary.","2020-03-03","Lu Li; Shuang Li; Manman Xu; Pengfei Yu; Sujun Zheng; Zhongping Duan; Jing Liu; Yu Chen; Junfeng Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.20028514"],["3wvcuboc","d75c48f51b2a5c26a41fe615339e5947d5e9cb30","medrxiv","Analysis of epidemiological characteristics of coronavirus 2019 infection and preventive measures in Shenzhen China: a heavy population city","10.1101/2020.02.28.20028555",null,null,"medrvix","Coronavirus 2019 infection (COVID-19) outbroke in Wuhan, Hubei and spread to all provinces in China and other countries. Shenzhen ranked the top cities outside Wuhan with reported 416 confirmed cases by February 20, 2020. Here, we analyzed the epidemiological characteristics of COVID-19 in Shenzhen and potential link to the preventive strategies for the whole city and inside hospitals. Based on the daily new cases, the epidemic of COVID-19 in Shenzhen can be classified into three phases: the slow increase phase from January 19 to January 28, the rapid increase and plateau phase from January 29 to February 5 and the decline phase since February 6. In the three phases, the number of patients from Hubei decreased, and the number of familial clustering cases increased. The newly diagnosed COVID-19 cases reached its peak around January 31, which was 7 days after the peak date of cases arrival at Shenzhen. A series of early preventive strategies were implemented since January 19, which included detection of body temperature at all entrances of main traffic and buildings, outpatients service specially for patients with fever in all main hospitals in Shenzhen. All the patients with fever were screened with nasal or throat swab PCR detection of coronavirus 2019, Chest CT and blood lymphocyte counting in order to find out early case of COVID-19. Observation wards were established in every main hospital and a designated hospital was responsible for admission and medical care of all confirmed cases. Protection procedure was established for all medical staff involved in the screening and care of suspected and confirmed cases. 14 days isolated observation of all subjects arrived at Shenzhen from Hubei was implemented in February 2. After the implementation of all these strategies and measures, the COVID-19 cases started to decline since February 6. There were almost no community transmission and nosocomial infection occurred in Shenzhen. In conclusion, in situation of major outbreak of respiratory infectious disease, such as COVID-19, in nearby province of Hubei, Shenzhen, a high population density, high proportion of external population and high mobility city, has to face the imported cases and risk of spreading the outbreak into Shenzhen city. The implementation of early preventive strategies and measures in Shenzhen were successful in early identification of COVID-19 cases and prevented major outbreak occurred in Shenzhen. Early identification of imported cases, prevention of family clustering transmission, preventive measures in the public area and very strict infection control procedure in hospital setting are crucial for the successful control of outbreak in Shenzhen.","2020-03-03","Kai Yang; Lingwei Wang; Furong Li; Dandan Chen; Xi Li; Chen Qiu; Rongchang Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.20028555"],["py38bel4","8575c06c64fdd0bb43efc46cd994f557698c755f","medrxiv","Clinical significance of IgM and IgG test for diagnosis of highly suspected COVID-19 infection","10.1101/2020.02.28.20029025",null,null,"medrvix","Quick, simple and accurate diagnosis of suspected COVID-19 is very important for the screening and therapy of patients. Although several methods were performed in clinical practice, however, the IgM and IgG diagnostic value evaluation was little performed. 57 suspected COVID-19 infection patients were enrolled in our study. 24 patients with positive and 33 patients with negative nucleic acid test. The positive rate of COVID-19 nucleic acid was 42.10%. The positive detection rate of combination of IgM and IgG for patients with COVID-19 negative and positive nucleic acid test was 72.73% and 87.50%. The results were significantly higher than the nucleic acid or IgM, IgG single detection. hsCRP in the COVID-19 nucleic acid negative group showed significantly higher than the positive groups (P=0.0298). AST in the COVID-19 IgM negative group showed significantly lower than the positive groups (P=0.0365). We suggest a quick, simple, accurate aided detection method for the suspected patients and on-site screening in close contact with the population.","2020-03-03","Xingwang Jia; Pengjun Zhang; Yaping Tian; Junli Wang; Huadong Zeng; Jun Wang; Liu Jiao; Zeyan Chen; Lijun Zhang; Haihong He; Kunlun He; Yajie Liu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.20029025"],["p5ecpcll","a5039336577ac0fed15b784adb912df2a6a143f3","medrxiv","Analysis on the Clinical Characteristics of 36 Cases of Novel Coronavirus Pneumonia in Kunming","10.1101/2020.02.28.20029173",null,null,"medrvix","Objective: To analyze the clinical characteristics of patients with novel coronavirus pneumonia in Kunming City, and to study the correlation between nutritional status and immune function. Methods: Clinical data of 36 patients with novel coronavirus pneumonia in isolation area of Kunming Third People's Hospital from January 31 to February 15, 2020 were collected, and the basic situation, clinical characteristics, laboratory examination and CT imaging characteristics were analyzed. Serum albumin (ALB), prealbumin (PAB), hypersensitive c-reactive protein (hs-crp), CD3T cells, CD4T cells, CD8T cells and normal control group were analyzed. A simple linear regression analysis of the relationship between proalbumin and T cell subpopulation counts in the blood of patients. Results: (1) The patients with new coronavirus pneumonia in Kunming were mainly of common type. (2) 50% of the patients' first symptoms were fever and cough; (3) The total number of white blood cells in peripheral blood was normal or decreased in 23 cases (79%), and the lymphocyte count decreased in 5 cases (13.89%), without anemia. Hypersensitive c-reactive protein increased in 19 (52.78%) cases, and procalcitonin increased in 1 case. Albumin decreased in 5 cases (13.89%), proalbumin decreased in 15 cases (41.67%), alanine transaminase increased slightly in 4 cases (11.11%), alanine transaminase increased slightly in 4 cases (11.11%), total bilirubin increased slightly in 11 cases (30.56%), and renal function and blood coagulation were normal. Absolute value of CD3+T cells is with a decrease in 21 cases (58.3%), CD4+T in 28 cases (77.8%), CD8+T in 17 cases (47.2%), and CD4+/ CD8+ inverse in 6 cases (16.7%). (4) The prealbumin, CD3 T cells, CD4 T cells and CD8 T cells in the new coronavirus pneumonia group were significantly lower than those in the normal control group, and the hypersensitive c-reactive protein was higher than that in the normal control group. (5) The levels of PAB in the serum of the patients were linearly correlated with hs-crp, CD3 T cells, CD4 T cells and CD8 T cells, and the correlation coefficients were -0.474, 0.558, 0.467 and 0.613, respectively, showing statistical differences. Conclusion: The clinical characteristics of the novel coronavirus pneumonia in Kunming are different from those in Wuhan. The changes of serum proalbumin and T cell subsets are relatively obvious. Changes in serum proalbumin may contribute to the early warning of novel coronavirus pneumonia. The nutritional status of patients with common and mild pneumonia should be considered.","2020-03-01","Haiyan Fu; Hongjuan Li; Xiaoqing Tang; Xiang Li; Jie Shen; Yujun Zhou; Bing Xu; Yu Luo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.20029173"],["6uaj8fb7","2af91bbb625d983d5ad3aadb9e5f1d82268a0f1f","medrxiv","Highly ACE2 Expression in Pancreas May Cause Pancreas Damage After SARS-CoV-2 Infection","10.1101/2020.02.28.20029181",null,null,"medrvix","The ongoing outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) started in the end of 2019 in China has triggered a global public health crisis. Previous studies have shown that SARS-CoV-2 infects cells by binding angiotensin-converting enzyme 2 (ACE2), which is the same as SARS-CoV. The expression and distribution of ACE2 in the pancreas are unknown. At the same time, the injury of pancreas after SARS-CoV-2 infection has not been concerned. Here, we collected public datasets (bulk RNA-seq and single-cell RNA-seq) to indicate the expression and the distribution of ACE2 in pancreas (in both exocrine glands and islets). And further, clinical data including mild and severe patients with COVID-19 demonstrated there existed mild pancreatitis. In the 67 severe cases, 11 patients (16.41%) showed elevated levels of both amylase and lipase, and 5 patients (7.46%) showed imaging alterations. Only one patient (1.85%) showed elevated levels of both amylase and lipase in 54 mild cases, without imaging changes. Our study revealed the phenomenon and possible cause of mild pancreatic injury in patients with COVID-19. This suggests that pancreatitis after SARS-CoV-2 infection should also be paid attention in clinical work.","2020-03-03","Furong Liu; Xin Long; Wenbin Zou; Minghao Fang; Wenjuan Wu; Wei Li; Bixiang Zhang; Wanguang Zhang; Xiaoping Chen; Zhanguo Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.20029181"],["zr34wsae","530f3ba1a2c044305d73123fd8d2246f63120bc3","medrxiv","The Impact of the COVID-19 Outbreak on the Medical Treatment of Chinese Children with Chronic Kidney Disease (CKD)：A Multicenter Cross-section Study in the Context of a Public Health Emergency of International Concern","10.1101/2020.02.28.20029199",null,null,"medrvix","Objective: To investigate the impact of the COVID-19 outbreak on the medical advice seeking of Chinese children with chronic kidney disease (CKD). Materials and Methods: An anonymous online questionnaire survey was conducted in 17 pediatric nephropathy diagnosis and treatment centers in China. The questions collected basic information on the patients and their parents and data on changes in the approach to medical treatment and their needs in the context of the outbreak etc. This is a Multicenter Cross-section Study. Results: A total of 735 valid questionnaires were collected. 555 patients (75.5%) and their parents said that the outbreak had a significant influence on their medical treatment: 264 patients (47.6%) said that it would be delayed by 2 to 4 weeks and 199 patients (35.9%) by 4 to 8 weeks. 510 patients (84.16%) hoped to get in touch with specialists through online consultation, and 528 patients (84.5%) hoped that online consultation could be implemented and that medication could be delivered to them.. A total of 458 patients (62.3%) said that their greatest concern was that the CKD would be aggravated or that they would experience a relapse; only 203 patients were infected by 2019-nCoV. A total of 313 patients (42.5%) experienced anxiety and thus required the intervention of psychologists. Conclusion: The COVID-19 outbreak has affected the medical treatment of children with CKD. Online consultation, medication delivery and psychological counselling are the greatest needs reported by patients and their families and could especially provide solutions for the management of low income children with CKD in remote rural areas in the context of the COVID-19 epidemic.","2020-03-03","Gaofu Zhang; Haiping Yang; Aihua Zhang; Qian Shen; Li Wang; Zhijuan Li; Yuhong Li; Lijun Zhao; Yue Du; Liangzhong Sun; Bo Zhao; Hongtao Zhu; Haidong Fu; Xiaoyan Li; Xiaojie Gao; Sheng Hao; Juanjuan Ding; Zongwen Chen; Zhiquan Xu; Xiaorong Liu; Daoqi Wu; Mingsi Gao; Mo Wang; Qiu Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.20029199"],["p3dex2to",null,"medrxiv","Closed environments facilitate secondary transmission of coronavirus disease 2019 (COVID-19)","10.1101/2020.02.28.20029272",null,null,"medrvix","Commissioned by the Minister of the Ministry of Health, Labour, and Welfare of Japan, we collected secondary transmission data with the aim of identifying high risk transmission settings. We show that closed environments contribute to secondary transmission of COVID-19 and promote superspreading events. Closed environments are consistent with large-scale COVID-19 transmission events such as that of the ski chalet-associated cluster in France and the church- and hospital-associated clusters in South Korea. Our findings are also consistent with the declining incidence of COVID-19 cases in China, as gathering in closed environments was prohibited in the wake of the rapid spread of the disease. Reduction of unnecessary close contact in closed environments may help prevent large case clusters and superspreading events.","2020-03-03","Hiroshi Nishiura; Hitoshi Oshitani; Tetsuro Kobayashi; Tomoya Saito; Tomimasa Sunagawa; Tamano Matsui; Takaji Wakita; Motoi Suzuki",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.20029272"],["5hkd80eh","da3aa20131ac2805c0d9e1b29f094683479ab5b7","biorxiv","Ruler elements in chromatin remodelers set nucleosome array spacing and phasing","10.1101/2020.02.28.969618",null,null,"biorxiv","Arrays of regularly spaced nucleosomes dominate chromatin and are often phased by alignment to reference sites like active promoters. How the distances between nucleosomes (spacing), and between phasing sites and nucleosomes are determined remains unclear, and specifically, how ATP dependent chromatin remodelers impact these features. Here, we used genome-wide reconstitution to probe how Saccharomyces cerevisiae ATP dependent remodelers generate phased arrays of regularly spaced nucleosomes. We find that remodelers bear a functional element named the ‘ruler’ that determines spacing and phasing in a remodeler-specific way. We use structure-based mutagenesis to identify and tune the ruler element residing in the Nhp10 and Arp8 modules of the INO80 remodeler complex. Generally, we propose that a remodeler ruler regulates nucleosome sliding direction bias in response to (epi)genetic information. This finally conceptualizes how remodeler-mediated nucleosome dynamics determine stable steady-state nucleosome positioning relative to other nucleosomes, DNA bound factors, DNA ends and DNA sequence elements.","2020-02-29","Elisa Oberbeckmann; Vanessa Niebauer; Shinya Watanabe; Lucas Farnung; Manuela Moldt; Andrea Schmid; Patrick Cramer; Craig L. Peterson; Sebastian Eustermann; Karl-Peter Hopfner; Philipp Korber",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.969618"],["48mtdwuv","3d498994d143a4ed7cc381700a19f2790d46bb36","biorxiv","Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models","10.1101/2020.02.28.969782",null,null,"biorxiv","SUMMARYIn the analysis of genomic sequence data, so-called “alignment free” approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification1,2,3,4. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers5. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes6,7,8. This can be seen as an analogy of the “bag-of-words” model successfully employed in natural language processing and computer vision for document and image classification9,10. Feature extraction techniques from natural language processing have previously been analogized to genomics data11; however, the “bag-of-words” approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of “bag-of-words” methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, Vorpal, allows the construction of interpretable linear models with clustered, representative ‘degenerate’ K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae, hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus, and human-host vs. non-human host in Influenza A. The capacity of this approach for in silico identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at https://github.com/mriglobal/vorpal.","2020-03-02","Phillip Davis; John Bagnoli; David Yarmosh; Alan Shteyman; Lance Presser; Sharon Altmann; Shelton Bradrick; Joseph A. Russell",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.969782"],["2uvibr2j",null,"biorxiv","Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches","10.1101/2020.02.28.970343",null,null,"biorxiv","Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available to cure COVID-19. Therefore, an effective vaccine against SARS-COV-2 is crucially needed. This study was conducted to design an effective multiepitope based vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as targets and different epitopes (B-cell, T-cell and IFN-γ inducing) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the world population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and eleven MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the MEV. The antigenicity, allergenicity, physiochemical properties and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic, antigenic, stable and flexible. Molecular docking followed by molecular dynamics (MD) simulation analysis, demonstrated a stable and strong binding affinity of MEV with human pathogenic toll-like receptors (TLR), TLR3 and TLR8. Codon optimization and in silico cloning of MEV ensured increased expression in the Escherichia coli K-12 system. Designed MEV in present study could be a potential candidate for further vaccine production process against COVID-19. However, to ensure its safety and immunogenic profile, the proposed MEV needs to be experimentally validated.","2020-03-02","Muhammad Tahir ul Qamar; Abdur Rehman; Usman Ali Ashfaq; Muhammad Qasim Awan; Israr Fatima; Farah Shahid; Ling-Ling Chen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.28.970343"],["0ga5rel6","0b460e73926eb107001c95c0dfeae2362b4251b0","medrxiv","Precautions are Needed for COVID-19 Patients with Coinfection of Common Respiratory Pathogens","10.1101/2020.02.29.20027698",null,null,"medrvix","Background: With the ongoing outbreak of Coronavirus Disease 2019 (COVID-19), infected patients within and beyond the epidemic area, Wuhan, China, showed different epidemiological and clinical characteristics. There is a paucity of data concerning coinfection with other common respiratory pathogens in COVID-19 patients outside of Wuhan. Methods: We conducted a double-centre study recruiting 68 patients with severe acute respiratory coronavirus 2 (SARS-CoV-2) infection confirmed by nucleic acid testing in Qingdao and Wuhan from January 17 to February 16, 2020. Indirect immunofluorescence was performed to detect the specific IgM antibody against common respiratory pathogens in collected acute phase serum. Results: Of the 68 patients with SARS-CoV-2 infection, 30 (44.12%) were from Qingdao. The median age of Qingdao and Wuhan patients were 50 (IQR: 37-59) and 31 (IQR: 28-38) years, respectively, and the majority of patients were female in Qingdao (60.00%) and Wuhan (55.26%). Among COVID-19 patients in Qingdao, 24 (80.00%) of them had IgM antibodies against at least one respiratory pathogen, whereas only one (2.63%) of the patients in Wuhan had positive results for serum IgM antibody detection (P<0.0001). The most common respiratory pathogens detected in Qingdao COVID-19 patients were influenza virus A (60.00%) and influenza virus B (53.33%), followed by mycoplasma pneumoniae (23.33%) and legionella pneumophila (20.00%). While the pattern for coinfection in patients with community-acquired pneumonia in Qingdao was quite different, with a positive rate of only 20.90%. Interpretation: We reported a large proportion of COVID-19 patients with coinfection of seasonal respiratory pathogens in Qingdao, northeast China, which differed greatly from the patients in Wuhan, central China. Precautions are needed when dealing with COVID-19 patients beyond the epidemic centre who have coinfection with other respiratory pathogens. We highly recommend adding SARS-CoV-2 to routine diagnostic testing in capable hospitals to prevent misdetection of the virus.","2020-03-03","Quansheng Xing; Guoju Li; Yuhan Xing; Ting Chen; Wenjie Li; Wei Ni; Kai Deng; Ruqin Gao; Changzheng Chen; Yang Gao; Qiang Li; Guiling Yu; Jianning Tong; Wei Li; Guiliang Hao; Yue Sun; Ai Zhang; Qin Wu; Zipu Li; Silin Pan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20027698"],["s43mous6","14783283da3482bf2aab4e6100fe4ea5c29d8db7","medrxiv","Vicarious traumatization in the general public, members, and non-members of medical teams aiding in COVID-19 control","10.1101/2020.02.29.20029322",null,null,"medrvix","Since December 2019, more than 79,000 people have been diagnosed with infection of the Corona Virus Disease 2019 (COVID-19). A large number of medical staff were dispersed for Wuhan city and Hubei province to aid COVID-19 control. Psychological stress, especially vicarious traumatization (VT) caused by the COVID-19 pandemic, should not be ignored. To address this concern, the study employed a total of 214 general public (GP) and 526 nurses to evaluate VT scores via a mobile app-based questionnaire. Results showed that the VT scores slightly increased across periods of aiding COVID-19 control, although no statistical difference was noted (P = 0.083). However, the study found lower scores for VT in nurses [median = 69; interquartile range (IQR) = 56-85] than those of the GP (median = 75.5; IQR = 62-88.3) (P = 0.017). In addition, the VT scores for front-line nurses (FLNs; median = 64; IQR = 52-75), including scores for physiological and psychological responses, were significantly lower than those of non-front-line nurses (nFLNs; median = 75.5; IQR = 63-92) (P < 0.001). Interestingly, the VT scores of the GP were significantly higher than those of the FLNs (P < 0.001). However, no statistical difference was observed compared with those of nFLNs (P > 0.05). Importantly, nFLNs are more likely to suffer from VT, which might be related to two factors, namely, gender [odds ratio (OR) = 3.1717; 95% confidence interval (CI) = 4.247-18.808; P = 0.002] and fertility [OR = 2.072; 95%CI = 0.626-24.533; P = 0.039]. Therefore, increased attention should be paid to the psychological problems of the medical staff, especially nFLNs, and GP under the situation of the spread and control of COVID-19. Early strategies that aim to prevent and treat VT in medical staff and GP are extremely necessary.","2020-03-03","Zhenyu Li; Jingwu Ge; Meiling Yang; Jianping Feng; Mei Qiao; Riyue Jiang; Jiangjiang Bi; Gaofeng Zhan; Xiaolin Xu; Long Wang; Qin Zhou; Chenliang Zhou; Yinbing Pan; Shijiang Liu; Haiwei Zhang; Jianjun Yang; Bin Zhu; Yimin Hu; Kenji Hashimoto; Yan Jia; Haofei Wang; Rong Wang; Cunming Liu; Chun Yang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029322"],["9lissxsy","5241581d168b90832f88103bd40c8db2d843a6ea","medrxiv","Association of Cardiovascular Manifestations with In-hospital Outcomes in Patients with COVID-19: A Hospital Staff Data","10.1101/2020.02.29.20029348",null,null,"medrvix","Background: The outbreaks of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) remain a huge threat to the public health worldwide. Clinical data is limited up to now regarding the risk factors in favor of severe conversion of non-severe case with COVID-19. Aims: This study analyzed a hospital staff data to figure out general clinical features of COVID-19 in terms of the association of cardiovascular manifestations (CVMs) with in-hospital outcomes of COVID-19 cases. Methods: Retrospective, single-center case series of 41 consecutive hospitalized health staff with confirmed COVID-19 were collected at the Central Hospital of Wuhan in Wuhan, China, from January 15 to January 24, 2020. Epidemiological, demographic, clinical, laboratory, radiological, treatment data and in-hospital adverse events were collected and analyzed. Final date of follow-up was March 3, 2020. A comparative study was applied between cases with CVMs and those without CVMs. Results: Of all, clinicians and clinical nurses accounted for 80.5%, while 87.8% of all had history of patient contact. The population was presented with a mean age of 39.1 +- 9.2 and less comorbidities than community population. The three most frequent symptoms of COVID-19 cases analyzed were fever (82.9%), myalgia or fatigue (80.5%) and cough (63.4%). While, the three most frequent initial symptoms were myalgia or fatigue (80.5%), fever (73.2%) and cough (41.5%). There were 95.1% cases featured as non-severe course of disease according to the official standard in China. Patients with CVMs and those without CVMs accounted for 58.5% and 41.5%, respectively. Compared with cases without CVMs, patients with CVMs were presented with lower baseline lymphocyte count (0.99 +- 0.43 and 1.55 +- 0.61, P<0.001), more who had at least once positive nucleic acid detection of throat swab during admission (50.0% and 11.8%, P=0.011), and more received oxygen support (79.2% and 23.5%, P<0.001). The rate of in-hospital adverse events was significantly higher in patients with CVMs group (75.0% and 23.5%, P=0.001). Multivariable logistic regression model indicated that, coexisting with CVMs in COVID-19 patients was not independently associated with in-hospital adverse events. Conclusions: Most of hospital staff with COVID-19 had history of patient contact, featured non-severe course of disease. Cases with CVMs suffered from more in-hospital adverse events than those without CVMs. But concomitant CVMs were not independently associated with in-hospital adverse events in COVID-19 patients.","2020-03-03","Ru Liu; Xiaoyan Ming; Ou Xu; Jianli Zhou; Hui Peng; Ning Xiang; Jiaming Zhang; Hong Zhu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029348"],["qs7dsxn5","f87eec8279bc79900914c5689bb42f49f41aace5","medrxiv","The spatiotemporal estimation of the dynamic risk and the international transmission of 2019 Novel Coronavirus (COVID-19) outbreak: A global perspective","10.1101/2020.02.29.20029413",null,null,"medrvix","An ongoing novel coronavirus SARS-CoV-2 pneumonia infection outbreak called COVID-19 started in Wuhan, Hubei Province, China, in December 2019. It both spread rapidly to all provinces in China and started spreading around the world quickly through international human movement from January 2020. Currently, the spatiotemporal epidemic transmission patterns, prediction models, and possible risk analysis for the future are insufficient for COVID-19 but we urgently need relevant information, particularly from the global perspective. We have developed a novel two-stage simulation model to simulate the spatiotemporal changes in the number of COVID-19 cases and estimate the future worldwide risk. Based on the connectivity of countries to China and the country's medical and epidemic prevention capabilities, different scenarios are generated to analyze the possible transmission throughout the world and use this information to evaluate each country's vulnerability to and the dynamic risk of COVID-19. Countries' vulnerability to the COVID-19 outbreak from China is calculated for 63 countries around the world. Taiwan, South Korea, Hong Kong, and Japan are the most vulnerable areas. The relationship between each country's vulnerability and days before the first imported case occurred shows a very high exponential decrease. The cumulative number of cases in each country also has a linear relationship with vulnerability, which can compare and quantify the initial epidemic prevention capabilities to various countries' management strategies. In total, 1,000 simulation results of future cases around the world are generated for the spatiotemporal risk assessment. According to the simulation results of this study, if there is no specific medicine for it, it will likely form a global pandemic. This method can be used as a preliminary risk assessment of the spatiotemporal spread for a new global epidemic. * Note: This study was completed on February 15, 2020.","2020-03-03","Yuan-Chien Lin; Wan-Ju Chi; Yu-Ting Lin; Chun-Yeh Lai",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029413"],["m9icky9z","c12ad8b411c24fe215be91d1f990645743e55ab5","medrxiv","An epidemiological forecast model and software assessing interventions on COVID-19 epidemic in China","10.1101/2020.02.29.20029421",null,null,"medrvix","We develop a health informatics toolbox that enables public health workers to timely analyze and evaluate the time-course dynamics of the novel coronavirus (COVID-19) infection using the public available data from the China CDC. This toolbox is built upon a hierarchical epidemiological model in which two observed time series of daily proportions of infected and removed cases are emitted from the underlying infection dynamics governed by a Markov SIR infectious disease process. We extend the SIR model to incorporate various types of time-varying quarantine protocols, including government-level macro isolation policies and community-level micro inspection measures. We develop a calibration procedure for under-reported infected cases. This toolbox provides forecast, in both online and offline forms, of turning points of interest, including the time when daily infected proportion becomes smaller than the previous ones and the time when daily infected proportions becomes smaller than that of daily removed proportion, as well as the ending time of the epidemic. An R software is made available for the public, and examples on the use of this software are illustrated. Some possible extensions of our novel epidemiological models are discussed.","2020-03-03","Peter X Song; Lili Wang; Yiwang Zhou; Jie He; Bin Zhu; Fei Wang; Lu Tang; Marisa Eisenberg",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029421"],["b8eezxv3",null,"medrxiv","ddPCR: a more sensitive and accurate tool for SARS-CoV-2 detection in low viral load specimens","10.1101/2020.02.29.20029439",null,null,"medrvix","Background: Real-Time PCR (RT-PCR) is widely used as the gold standard for clinical detection of SARS-CoV-2. However, due to the low viral load in patient throat and the limitation of RT-PCR, significant numbers of false negative reports are inevitable, which should not be ignored. Methods: We explored the feasibility of droplet digital PCR (ddPCR) to detect SARS-CoV-2 from 57 clinical pharyngeal swab samples and compared with RT-PCR in terms of the sensitivity and accuracy. Among 57 samples, all of which were reported as negative nucleic acid by officially approved clinical RT-PCR detection, 43 samples were collected from suspected patients with fever in clinic, and 14 were from supposed convalescents who were about to discharge after treatment. The experiment was double-blind. Results: The lower limit of detection of the optimized ddPCR is at least 500 times lower than that of RT-PCR. The overall accuracy of ddPCR for clinical detection is 94.3 %. 33 out of 35 negative pharyngeal swab samples checked by RT-PCR were correctly judged by ddPCR based on the follow-up investigation. In addition, 9 out of 14 (64.2 %) supposed convalescents with negative nucleic acid test twice by RT-PCR were positive by ddPCR detection. Conclusions: ddPCR shows superiority for clinical detection of SARS-CoV-2 to reduce the false negatives, which could be a powerful complement to the current standard RT-PCR. Before the ddPCR to be approved for diagnosis, the current clinical practice that the convalescent continues to be quarantined for 2 weeks is reasonable and necessary.","2020-03-06","Tao Suo; Xinjin Liu; Ming Guo; Jiangpeng Feng; Wenjia Hu; Yang Yang; Qiuhan Zhang; Xin Wang; Muhanmmad Sajid; Dong Guo; Zhixiang Huang; Liping Deng; Tielong Chen; Fang Liu; Ke Xu; Yuan Liu; Qi Zhang; Yingle Liu; Yong Xiong; Guozhong Guo; Yu Chen; Ke Lan",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029439"],["ndn6iwre","1ad1d4b84aea4ceaf05c62a1ad04e7150f7f4684","medrxiv","68 Consecutive patients assessed for COVID-19 infection; experience from a UK regional infectious disease unit","10.1101/2020.02.29.20029462",null,null,"medrvix","Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29th Jan 2020 and 24th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. Final microbiological diagnoses included SARS-CoV-2 (COVID-19), Mycoplasma pneumonia, influenza A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.","2020-03-06","Nicholas Easom; Peter Moss; Gavin Barlow; Anda Samson; Tom Taynton; Kate Adams; Monica Ivan; Phillipa Burns; Kavitha Gajee; Kirstine Eastick; Patrick Lillie",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029462"],["g41pd9uz","7852aafdfb9e59e6af78a47af796325434f8922a","medrxiv","Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients","10.1101/2020.02.29.20029520",null,null,"medrvix","Background: Although the SARS-CoV-2 viral load detection of respiratory specimen has been widely used for novel coronavirus disease (COVID-19) diagnosis, it is undeniable that serum SARS-CoV-2 nucleic acid (RNAaemia) could be detected in a fraction of the COVID-19 patients. However, it is not clear that if the incidence of RNAaemia could be correlated with the occurrence of cytokine storm or with the specific class of patients. Methods: This study enrolled 48 patients with COVID-19 admitted to the General Hospital of Central Theater Command, PLA, a designated hospital in Wuhan, China. The patients were divided into three groups according to the Diagnosis and Treatment of New Coronavirus Pneumonia (version 6) published by the National Health Commission of China. The clinical and laboratory data were collected. The serum viral load detection and serum IL-6 levels were determined. Except for routine statistical analysis, Generalized Linear Models (GLMs) analysis was used to establish a patient status prediction model based on real-time RT-PCR Ct value. Findings: The Result showed that cases with RNAaemia were exclusively confirmed in critically ill patients group and appeared to reflect the illness severity. Further more, the inflammatory cytokine IL-6 levels were significantly elevated in critically ill patients, which is almost 10-folds higher than those in other patients. More importantly, the extremely high IL-6 level was closely correlated with the incidence of RNAaemia (R=0.902) and the vital signs of COVID-19 patients (R= -0.682). Interpretation: Serum SARS-CoV-2 viral load (RNAaemia) is strongly associated with cytokine storm and can be used to predict the poor prognosis of COVID-19 patients. Moreover, our results strongly suggest that cytokine IL-6 should be considered as a therapeutic target in critically ill patients with excessive inflammatory response.","2020-03-03","Xiaohua Chen; Binghong Zhao; Yueming Qu; Yurou Chen; Jie Xiong; Yong Feng; Dong Men; Qianchuan Huang; Ying Liu; Bo Yang; Jinya Ding; Feng Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029520"],["3kp3ptbn","0bd443591b5e10934beef050b516afeca6b668fe","medrxiv","A simple model to assess Wuhan lock-down effect and region efforts during COVID-19 epidemic in China Mainland","10.1101/2020.02.29.20029561",null,null,"medrvix","Since COVID-19 emerged in early December, 2019 in Wuhan and swept across China Mainland, a series of large-scale public health interventions, especially Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement, have been taken by the government to control the epidemic. Based on Baidu Migration data and the confirmed cases data, we identified two key factors affecting the later (e.g February 27, 2020) cumulative confirmed cases in non-Wuhan region (y). One is the sum travelers from Wuhan during January 20 to January 26 (x1), which had higher infected probability but lower transmission ability because the human-to-human transmission risk of COVID-19 was confirmed and announced on January 20. The other is the seed cases from Wuhan before January 19, which had higher transmission ability and could be represented with the confirmed cases before January 29 (x2) due to a mean 10-day delay between infection and detection. A simple yet effective regression model then was established as follow: y= 70.0916+0.0054*x1+2.3455*x2 (n = 44, R2 = 0.9330, P<10-7). Even the lock-down date only delay or in advance 3 days, the estimated confirmed cases by February 27 in non-Wuhan region will increase 35.21% or reduce 30.74% - 48.59%. Although the above interventions greatly reduced the human mobility, Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement do have a determining effect on the ongoing spread of COVID-19 across China Mainland. The strategy adopted by China has changed the fast-rising curve of newly diagnosed cases, the international community should learn from lessons of Wuhan and experience from China. Efforts of 29 Provinces and 44 prefecture-level cities against COVID-19 were also assessed preliminarily according to the interpretive model. Big data has played and will continue playing an important role in public health.","2020-03-03","Yuan zheming; Chen Yuan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029561"],["2s4ifz7i","b8bb4db131a25b1bbb30d4205b16dc9ef988d22f","medrxiv","Machine learning-based CT radiomics model for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection: A multicenter study","10.1101/2020.02.29.20029603",null,null,"medrvix","Abstract Objectives To develop and test machine learning-based CT radiomics models for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection. Design Cross-sectional Setting Multicenter Participants A total of 52 patients with laboratory-confirmed SARS-CoV-2 infection and their initial CT images were enrolled from 5 designated hospitals in Ankang, Lishui, Zhenjiang, Lanzhou, and Linxia between January 23, 2020 and February 8, 2020. As of February 20, patients remained in hospital or with non-findings in CT were excluded. Therefore, 31 patients with 72 lesion segments were included in the final analysis. Intervention CT radiomics models based on logistic regression (LR) and random forest (RF) were developed on features extracted from pneumonia lesions in training and inter-validation datasets. The predictive performance was further evaluated in test dataset on lung lobe- and patients-level. Main outcomes Short-term hospital stay (≤10 days) and long-term hospital stay (>10 days). Results The CT radiomics models based on 6 second-order features were effective in discriminating short- and long-term hospital stay in patients with pneumonia associated with SARS-CoV-2 infection, with areas under the curves of 0.97 (95%CI 0.83-1.0) and 0.92 (95%CI 0.67-1.0) by LR and RF, respectively, in the test dataset. The LR model showed a sensitivity and specificity of 1.0 and 0.89, and the RF model showed similar performance with sensitivity and specificity of 0.75 and 1.0 in test dataset. Conclusions The machine learning-based CT radiomics models showed feasibility and accuracy for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection.","2020-03-03","Xiaolong Qi; Zicheng Jiang; QIAN YU; Chuxiao Shao; Hongguang Zhang; Hongmei Yue; Baoyi Ma; Yuancheng Wang; Chuan Liu; Xiangpan Meng; Shan Huang; Jitao Wang; Dan Xu; Junqiang Lei; Guanghang Xie; Huihong Huang; Jie Yang; Jiansong Ji; Hongqiu Pan; Shengqiang Zou; Shenghong Ju",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.20029603"],["1yka7ej9","38e796bbec2f90b1b802c14dc922102e96f6361e","biorxiv","The within-host viral kinetics of SARS-CoV-2","10.1101/2020.02.29.965418",null,null,"biorxiv","AbstractIn this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.","2020-03-02","Chentong Li; Jinhu Xu; Jiawei Liu; Yicang Zhou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.965418"],["0d77ojnb","07e833d0917cace550853f72923856d0fe1a7120","biorxiv","The 2019 coronavirus (SARS-CoV-2) surface protein (Spike) S1 Receptor Binding Domain undergoes conformational change upon heparin binding","10.1101/2020.02.29.971093",null,null,"biorxiv","AbstractMany pathogens take advantage of the dependence of the host on the interaction of hundreds of extracellular proteins with the glycosaminoglycans heparan sulfate to regulate homeostasis and use heparan sulfate as a means to adhere and gain access to cells. Moreover, mucosal epithelia such as that of the respiratory tract are protected by a layer of mucin polysaccharides, which are usually sulfated. Consequently, the polydisperse, natural products of heparan sulfate and the allied polysaccharide, heparin have been found to be involved and prevent infection by a range of viruses including S-associated coronavirus strain HSR1. Here we use surface plasmon resonance and circular dichroism to measure the interaction between the SARS-CoV-2 Spike S1 protein receptor binding domain (SARS-CoV-2 S1 RBD) and heparin. The data demonstrate an interaction between the recombinant surface receptor binding domain and the polysaccharide. This has implications for the rapid development of a first-line therapeutic by repurposing heparin and for next-generation, tailor-made, GAG-based antivirals.","2020-03-02","Courtney Mycroft-West; Dunhao Su; Stefano Elli; Scott Guimond; Gavin Miller; Jeremy Turnbull; Edwin Yates; Marco Guerrini; David Fernig; Marcelo Lima; Mark Skidmore",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.971093"],["jmrg4oeb","2ea102f58147dab02e4dea90eb90dbc67149f678","biorxiv","Mutations, Recombination and Insertion in the Evolution of 2019-nCoV","10.1101/2020.02.29.971101",null,null,"biorxiv","AbstractBackgroundThe 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear.MethodsWe collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses.ResultsSystematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site.ConclusionsOur studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion.One Sentence SummaryNovel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.","2020-03-02","Aiping Wu; Peihua Niu; Lulan Wang; Hangyu Zhou; Xiang Zhao; Wenling Wang; Jingfeng Wang; Chengyang Ji; Xiao Ding; Xianyue Wang; Roujian Lu; Sarah Gold; Saba Aliyari; Shilei Zhang; Ellee Vikram; Angela Zou; Emily Lenh; Janet Chen; Fei Ye; Na Han; Yousong Peng; Haitao Guo; Guizhen Wu; Taijiao Jiang; Wenjie Tan; Genhong Cheng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.971101"],["gzubfnqg",null,"biorxiv","An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12","10.1101/2020.02.29.971127",null,null,"biorxiv","AbstractSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has received global attention due to the recent outbreak in China. In this work, we report a CRISPR-Cas12 based diagnostic tool to detect synthetic SARS-CoV-2 RNA sequences in a proof-of-principle evaluation. The test proved to be sensitive, rapid, and potentially portable. These key traits of the CRISPR method are critical for virus detection in regions that lack resources to use the currently available methods.","2020-03-02","Curti Lucia; Pereyra-Bonnet Federico; Gimenez Carla Alejandra",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.02.29.971127"],["tne83uu0","7e9cd4bbf0fba1cc0bcded40041fd01b9dfb683f","medrxiv","Epidemiologic Characteristics of COVID-19 in Guizhou, China","10.1101/2020.03.01.20028944",null,null,"medrvix","At the end of 2019, a coronavirus disease 2019 (COVID-19) outbroke in Wuhan, China, and spread to Guizhou province on January of 2020. To acquire the epidemiologic characteristics of COVID-19 in Guizhou, China, we collected data on 162 laboratory-confirmed cases related to COVID-19. We described the demographic characteristics of the cases and estimated the incubation period, serial interval and basic reproduction number. We also presented two representative case studies in Guizhou province -- Case Study 1 was an example of asymptomatic carrier; and Case Study 2 was an example of a large and complex infection chain that involved four different districts spanning three provinces and eight families. With an estimation of 8 days incubation period and 6 days serial interval, our results indicate that there may exist infectiousness during the incubation period for 2019-nCoV. This increases the difficulty of screening or identifying cases related to COVID-19.","2020-03-06","Kaike Ping",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20028944"],["hhsfq8bz","f19e2df932f90f387fe71533f3802875d9042160","medrxiv","Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study","10.1101/2020.03.01.20029074",null,null,"medrvix","Background: Coronavirus disease-2019 (COVID-19) is a rapidly escalating epidemic caused by SARS-CoV-2. Identification of a simple and effective indicator to assess disease severity and outcome is urgently needed. Methods: This study retrospectively analyzed dynamic changes of lymphocyte percentage (LYM%) in 15 death cases, 15 severe cases as well as 40 moderate cases of COVID-19 patients. Next, prognostic role of lymphopenia in COVID-19 were verified in 92 hospitalized cases. Results: Our results from death and severe cases showed that LYM% in blood test were inversely associated with the progression and severity of COVID-19. LYM% in patients with moderate COVID-19 remained higher than 20% 10-12 days after symptom onset. In contrast, LYM% was lower than 20% in severe cases. However, LYM% in severe cases was higher than 5% 17-19 days after the onset of the disease, while it fell below 5% in death cases. Therefore, we established a reliable Time from symptom onset-LYM% model (TLM), which could be used to evaluate disease severity and predict the outcomes of hospitalized patients with COVID-19. Conclusion: Lymphopenia can be used to indicate clinical course, treatment effect and outcomes of COVID-19 patients.","2020-03-03","Li Tan; Qi Wang; Duanyang Zhang; Jinya Ding; Qianchuan Huang; Yi-Quan Tang; Qiongshu Wang; Hongming Miao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029074"],["nsrm0axa","510ecab01c66bc2a911abcf67451ec482ed18a05","medrxiv","Epidemiological and clinical features of 2019-nCoV acute respiratory disease cases in Chongqing municipality, China: a retrospective, descriptive, multiple-center study","10.1101/2020.03.01.20029397",null,null,"medrvix","Background：In January 19, 2020, first case of 2019 novel coronavirus (2019-nCoV) pneumonia (COVID-19) was confirmed in Chongqing municipality, China. Methods：In this retrospective, descriptive, multiple-center study, total of 267 patients with COVID-19 confirmed by real-time RT-PCR in Chongqing from Jan 19 to Feb 16, 2020 were recruited. Epidemiological, demographic, clinical, radiological characteristics, laboratory examinations, and treatment regimens were collected on admission. Clinical outcomes were followed up until Feb 16, 2020. Results：267 laboratory-confirmed COVID-19 patients admitted to 3 designated-hospitals in Chongqing provincial municipality from January 19 to February 16, 2020 were enrolled and categorized on admission. 217 (81.27%) and 50 (18.73%) patients were categorized into non-severe and severe subgroups, respectively. The median age of patients was 48.0 years (IQR, 35.0-65.0), with 129 (48.3%) of the patients were more than 50 years of age. 149 (55.8%) patients were men. Severe patients were significantly older (median age, 71.5 years [IQR, 65.8-77.0] vs 43.0 years [IQR, 32.5-57.0]) and more likely to be male (110 [50.7%] vs 39 [78.0%]) and have coexisting disorders (15 [30.0%] vs 26 [12.0%]). 41 (15.4%) patients had a recent travel to Hubei province, and 139 (52.1%) patients had a history of contact with patients from Hubei. On admission, the most common symptoms of COVID-19 were fever 225(84.3%), fatigue (208 [77.9%]), dry cough (189 [70.8%]), myalgia or arthralgia (136 [50.9%]). Severe patients were more likely to present dyspnea (17 [34.0%] vs 26 [12.0%]) and confusion (10 [20.0%] vs 15 [6.9%]). Rales (32 [12.0%]) and wheezes (20 [7.5%]) are not common noted for COVID-19 patients, especially for the non-severe (11 [5.1%], 10 [4.6%]). 118 (44.2%). Most severe patients demonstrated more laboratory abnormalities. 231 (86.5%), 61 (22.8%) patients had lymphopenia, leukopenia and thrombocytopenia, respectively. CD4+T cell counts decrease was observed in 77.1 % of cases, especially in the severe patients (45, 100%). 53.1% patients had decreased CD+3 T cell counts, count of CD8+T cells was lower than the normal range in part of patients (34.4%). More severe patients had lower level of CD4+ T cells and CD+3 T cells (45 [100.0%] vs 29[56.9%], 31 [68.9%] vs 20 [39.2%]). Most patients had normal level of IL-2, IL-4, TNF-α and INF-γ, while high level of IL-6 and IL-17A was common in COVID-19 patients (47 [70.1%], 35 [52.2%]). Level of IL-6, IL-17A and TNF-α was remarkably elevated in severe patients (32 [84.2%] vs 15 [51.7%], 25 [65.8%] vs 10 [34.5%], 17 [44.7%] vs 5 [17.2%]). All patients received antiviral therapy (267, 100%). A portion of severe patients (38, 76.0%) received systemic corticosteroid therapy. Invasive mechanical ventilation in prone position, non-invasive mechanical ventilation, high-flow nasal cannula oxygen therapy was adopted only in severe patients with respiratory failure (5[10.0%], 35[70.0%], 12[24.0%]). Traditional Chinese medicine was adopted to most of severe patients (43,86.0%). Conclusion:Our study firstly demonstrated the regional disparity of COVID-19 in Chongqing municipality and further thoroughly compared the differences between severe and non-severe patients. The 28-day mortality of COVID-19 patients from 3 designed hospitals of Chongqing is 1.5%, lower than that of Hubei province and mainland China including Hubei province. However, the 28-mortality of severe patients was relatively high, with much higher when complications occurred. Notably, the 28-mortality of critically severe patients complicated with severe ARDS is considerably as high as 44.4%. Therefore, early diagnosis and intensive care of critically severe COVID-19 cases, especially those combined with ARDS, will be considerably essential to reduce mortality.","2020-03-03","Di Qi; Xiaofeng Yan; Xumao Tang; Junnan Peng; Qian Yu; Longhua Feng; Guodan Yuan; An Zhang; Yaokai Chen; Jing Yuan; Xia Huang; Xianxiang Zhang; Peng Hu; Yuyan Song; Chunfang Qian; Qiangzhong Sun; Daoxin Wang; Jin Tong; Jianglin Xiang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029397"],["iwsa760n",null,"medrxiv","Systematic Review of the Registered Clinical Trials of Coronavirus Disease 2019 (COVID-19)","10.1101/2020.03.01.20029611",null,null,"medrvix","Background: Since the outbreak of coronavirus disease 2019 (COVID-19), many researchers in China have immediately carried out clinical research scheme of the COVID-19. But, there is still a lack of systematic review of registered clinical trials. Therefore, we conducted a systematic review of the clinical trials of COVID-19 to summarize the characteristics of the COVID-19 registered clinical trials. Methods: This study is based on the recommendations of the PRISMA in the Cochrane handbook. The databases from the Chinese Clinical Registration Center and the ClinicalTrials.gov were searched to collect the registered clinical trials of COVID-19. The retrieval inception date is February 9, 2020. Two researchers independently selected the literature based on inclusion and exclusion criteria, extracted data and evaluated the risk of bias. Results: A total of 75 registered clinical trials (63 interventional studies and 12 observational studies) of COVID-19 were obtained. A majority of clinical trials were sponsored by Chinese hospitals. Only 11 trials have begun to recruit patients, and none of the registered clinical trials had been completed; 34 trials were early clinical exploratory trials or in a pre-experiment stage, 15 trials belonged to phrase Ⅲ and 4 trials were phrase Ⅳ. The methods of intervention included traditional Chinese medicine involving 26 trials, Western medicine involving 30 trials, and integrated traditional Chinese medicine and Western medicine involving 19 trials. The subjects were mainly non-critical adult patients (≥ 18 years old). The median sample size of the trials was 100 (IQR: 60 - 200), and the median execute time of the trials was 179 d (IQR: 94 - 366 d). The main outcomes were clinical observation and examinations. Overall, both the methodology quality of interventional trials and observational studies were low. Conclusions: Disorderly and intensive clinical trials of COVID-19 using traditional Chinese medicine and western medicine are ongoing or will being carried out in China. However, based on the low methodology quality and small sample size and long studies execute time, we will not be able to obtain reliable, high-quality clinical evidence about COVID-19 treatment in the near future. Improving the quality of study design, prioritizing promising drugs, and using different designs and statistical methods are worth advocating and recommending for the clinical trials of COVID-19 in China.","2020-03-03","Rui-fang Zhu; Ru-lu Gao; Sue-Ho Robert; Jin-ping Gao; Shi-gui Yang; Changtai Zhu",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029611"],["3e1ji2mw","4bc43084ccdb8d3bb704ca577473169cd1a64f9c","medrxiv","Risk estimation and prediction by modeling the transmission of the novel coronavirus (COVID-19) in mainland China excluding Hubei province","10.1101/2020.03.01.20029629",null,null,"medrvix","Background: In December 2019, an outbreak of novel coronavirus disease (COVID-19) emerged in Wuhan, China and has swiftly spread to other parts of China and a number of oversea countries. Our aim is to evaluate the effectiveness of the evolution of interventions and self-protection measures, estimate the risk of partial lifting control measures and predict the epidemic trend of the virus in mainland China excluding Hubei province based on the published data and a novel mathematical model. Methods: A novel COVID-19 transmission dynamic model incorporating the intervention measures implemented in China is proposed. We parameterize the model by using the Markov Chain Monte Carlo (MCMC) method and estimate the control reproduction number Rc, as well as the effective daily reproduction ratio Re(t), of the disease transmission in mainland China excluding Hubei province. Results: The estimation outcomes indicate that the control reproduction number is 3.36 (95% CI 3.20-3.64) and Re(t) has dropped below 1 since January 31st, 2020, which implies that the containment strategies implemented by the Chinese government in mainland China excluding Hubei province are indeed effective and magnificently suppressed COVID-19 transmission. Moreover, our results show that relieving personal protection too early may lead to the spread of disease for a longer time and more people would be infected, and may even cause epidemic or outbreak again. By calculating the effective reproduction ratio, we proved that the contact rate should be kept at least less than 30% of the normal level by April, 2020. Conclusions: To ensure the epidemic ending rapidly, it is necessary to maintain the current integrated restrict interventions and self-protection measures, including travel restriction, quarantine of entry, contact tracing followed by quarantine and isolation and reduction of contact, like wearing masks, etc. People should be fully aware of the real-time epidemic situation and keep sufficient personal protection until April. If all the above conditions are met, the outbreak is expected to be ended by April in mainland China apart from Hubei province.","2020-03-06","Hui Wan; Jing-an Cui; Guo-Jing Yang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029629"],["4k1i6y98","5f72fd41d7ba8d4d7de7aa1f83fc5c90ffa50b03","medrxiv","How does the outbreak of 2019-nCoV spread in mainland China? A retrospective analysis of the dynamic transmission routes","10.1101/2020.03.01.20029645",null,null,"medrvix","The fourth outbreak of the Coronaviruses, known as the 2019-nCoV, has occurred in Wuhan city of Hubei province in China in December 2019. We propose a time-varying sparse vector autoregressive (VAR) model to retrospectively analyze and visualize the dyamic transmission routes of this outbreak in mainland China over January 31 - February 19, 2020. Our results demonstrate that the influential inter-province routes from Hubei have become unidentifiable since February 4, whereas the self-transmission in each province was accelerating over February 4-15. From February 16, all routes became less detectable, and no influential transmissions could be identified on February 18 and 19. Such evidence supports the effectiveness of government interventions, including the travel restrictions in Hubei. Implications of our results suggest that in addition to the origin of the outbreak, virus preventions are of crucial importance in provinces with the largest migrant workers percentages (e.g., Jiangxi, Henan and Anhui) to controlling the spread of 2019-nCoV.","2020-03-06","Xiandeng Jiang; Le Chang; Yanlin Shi",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029645"],["kbd2h4l0",null,"medrxiv","The potential role of IL-6 in monitoring severe case of coronavirus disease 2019","10.1101/2020.03.01.20029769",null,null,"medrvix","Abstract. Background: The outbreak of coronavirus disease 2019 (COVID-19) in Wuhan City, China spreads rapidly since December, 2019. Most patients show mild symptoms, but some of them develop into severe disease. There is currently no specific medication. The purpose of this study is to to explore changes of markers in peripheral blood of severe COVID-19 patients, which may be of value in disease monitoring. Methods Clinical data of patients with nonsevere and severe type COVID-19 diagnosed by laboratory test in our institution were collected. The relationship between peripheral blood cells and cytokines, clinical manifestation and outcome was analyzed. Results A total of 69 severe type COVID-19 patients were included. On admission, the median age of severe cases was 56-year old, with 52.17% female patient. The most common symptoms were fever (79.72%), cough (63.77%), shortness of breath (57.97%) and fatigue (50.72%). Diarrhea is less common. The most common comorbidity is hypertension. Upon admission, the proportion of bilateral pulmonary involvement and interstitial abnormalities evidenced by chest computed tomography (CT) imaging in severe cases was 60.87% and 27.54%, respectively. Compared with patients with nonsevere disease, those with severe disease showed obvious lymphocytopenia. Elevated level of lactate dehydrogenase (LDH), C-reactive protein (CRP), ferritin and D-dimer was found in most cases. Two patients (2.9%) needed transfer to the intensive care unit. Baseline immunological parameters and most of the inflammatory parameters were basically within the normal range. However, baseline interleukin-6 (IL-6) was significantly increased in severe type, which was closely related to the maximal body temperature during hospitalization and to CT findings. Baseline IL-6 was also significantly related to the increase of baseline level of CRP, LDH, ferritin and D-dimer. The increase of baseline IL-6 level suggests that it may positively correlate with the severity of COVID-19. Among the 30 severe type patients whose IL-6 was assessed before and after treatment, significant decrease in IL-6 and improved CT assessment was found in 25 patients after treatment. Whereas the IL-6 level was further increased in 3 cases, which was closely related to disease progression. It is suggested that IL-6 may be used as a marker for disease monitoring in severe COVID-19 patients. Conclusions On admission, the baseline level of IL-6, CRP, LDH and ferritin was closely related to the severity of COVID-19, and the elevated IL-6 was significantly related to the clinical manifestation of severe type patients. The decrease of IL-6 was closely related to treatment effectiveness, while the increase of IL-6 indicated disease exacerbation. Collectively, the dynamic change of IL-6 level can be used as a marker for disease monitoring in patients with severe COVID-19.","2020-03-06","Tao Liu; Jieying Zhang; Yuhui Yang; Hong Ma; Zhengyu Li; Jiaoyue Zhang; Ji Cheng; Xiaoyun Zhang; Yanxia Zhao; Zihan Xia; Liling Zhang; Gang Wu; Jianhua Yi",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029769"],["a5x5ga60","c8437a45bfb84fb206fe03fd18d28858bae32651","medrxiv","Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China","10.1101/2020.03.01.20029785",null,null,"medrvix","The outbreak of the novel coronavirus disease 2019 (COVID-19) infection began in December 2019 in Wuhan, and rapidly spread to many provinces in China. The number of cases has increased markedly in Anhui, but information on the clinical characteristics of patients is limited. We reported 75 patients with COVID-19 in the First Affiliated Hospital of USTC from Jan 21 to Feb 16, 2020, Hefei, Anhui Province, China. COVID-19 infection was confirmed by real-time RT-PCR of respiratory nasopharyngeal swab samples. Epidemiological, clinical and laboratory data were collected and analyzed. Of the 75 patients with COVID-19, 61 (81.33%) had a direct or indirect exposure history to Wuhan. Common symptoms at onset included fever (66 [88.0%] of 75 patients) and dry cough (62 [82.67%]). Of the patients without fever, cough could be the only or primary symptom. The most prominent laboratory abnormalities were lymphopenia, decreased percentage of lymphocytes (LYM%), decreased CD4+ and CD8+ T cell counts, elevated C-reactive protein (CRP) and lactate dehydrogenase (LDH). Patients with elevated interleukin 6 (IL-6) showed significant decreases in the LYM%, CD4+ and CD8+ T cell counts. Besides, the percentage of neutrophils, CRP, LDH and Procalcitonin levels increased significantly. We concluded that COVID-19 could cause different degrees of hematological abnormalities and damage of internal organs. Hematological profiles including LYM, LDH, CRP and IL-6 could be indicators of diseases severity and evaluation of treatment effectiveness. Antiviral treatment requires a comprehensive and supportive approach. Further targeted therapy should be determined based on individual clinical manifestations and laboratory indicators.","2020-03-06","Zonghao Zhao; Jiajia Xie; Ming Yin; Yun Yang; Hongliang He; Tengchuan Jin; Wenting Li; Xiaowu Zhu; Jing Xu; Changcheng Zhao; Lei Li; Yi Li; Hylemariam Mihiretie Mengist; Ayesha Zahid; Ziqin Yao; Chengchao Ding; Yingjie Qi; Yong Gao; Xiaoling Ma",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029785"],["38f8ftmh","2fe6b550f737baa47a5f2c8ab64cc3d9271c308a","medrxiv","Evaluation of the potential incidence of COVID-19 and effectiveness of contention measures in Spain: a data-driven approach","10.1101/2020.03.01.20029801",null,null,"medrvix","Our society is currently experiencing an unprecedented challenge, managing and containing an outbreak of a new coronavirus disease known as COVID-19. While China - were the outbreak started - seems to have been able to contain the growth of the epidemic, different outbreaks are nowadays being detected in multiple countries. Much is currently unknown about the natural history of the disease, such as a possible asymptomatic spreading or the role of age in both the susceptibility and mortality of the disease. Nonetheless, authorities have to take action and implement contention measures, even if not everything is known. To facilitate this task, we have studied the effect of different containment strategies that can be put into effect. Our work specifically refers to the situation in Spain as of February 28th, 2020, where a few dozens of cases have been detected. We implemented an SEIR-metapopulation model that allows tracing explicitly the spatial spread of the disease through data-driven stochastic simulations. Our results are in line with the most recent recommendations from the World Health Organization, namely, that the best strategy is the early detection and isolation of individuals with symptoms, followed by interventions and public recommendations aimed at reducing the transmissibility of the disease, which although not efficacious for disease eradication, would produce as a second-order effect a delay of several days in the raise of the number of infected cases","2020-03-06","Alberto Aleta; Yamir Moreno",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029801"],["7reqkx3h","d7111b44f2d13b24a11d55c8be2d2bbe6b245eac","medrxiv","COVID-19 Epidemic Outside China: 34 Founders and Exponential Growth","10.1101/2020.03.01.20029819",null,null,"medrvix","Background: In December 2019, pneumonia infected with a novel coronavirus burst in Wuhan, China. Now the situation is almost controlled in China but is worse outside China. We aimed to build a mathematical model to capture the global trend of epidemics outside China. Methods: In this retrospective, outside-China diagnosis number reported from Jan 21 to Feb 28, 2020 was downloaded from WHO website. We develop a simple regression model on these numbers: log10 (Nt+34)=0.0515*t+2.075 where Nt is the total diagnosed patient till the ith day, t=1 at Feb 1. Findings: Based on this model, we estimate that there have been about 34 unobserved founder patients at the beginning of spread outside China. The global trend is approximately exponential, with the rate of 10 folds every 19 days.","2020-03-03","Yi Li; Meng Liang; Xianhong Yin; Xiaoyu Liu; Meng Hao; Zixin Hu; Yi Wang; Li Jin",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029819"],["j58f1lwa","f8360b0e64d444215d21a61e056b070f12417e53","medrxiv","Preliminary epidemiological analysis on children and adolescents with novel coronavirus disease 2019 outside Hubei Province, China: an observational study utilizing crowdsourced data","10.1101/2020.03.01.20029884",null,null,"medrvix","Background: The outbreak of coronavirus disease 2019 (COVID-19) continues to expand across the world. Though both the number of cases and mortality rate in children and adolescents is reported to be low in comparison to adults, limited data has been reported on the outbreak with respect to pediatric patients. To elucidate information, we utilized crowdsourced data to perform a preliminary epidemiologic analysis of pediatric patients with COVID-19 Methods: In this observational study, data was collected from two open-access, line list crowdsourced online databases. Pediatric cases of COVID-19 were defined as patients ≤19 years of age with a laboratory confirmed diagnosis. The primary outcomes were case counts and cumulative case counts. Secondary outcomes included days between symptoms onset and first medical care and days between first medical care and reporting. Tertiary outcomes were rate of travel to Wuhan, rate of infected family members and rates of symptoms. Results: A total of 82 patients were included. The median age was 10 [IQR: 5-15] years. Patients from mainland China (outside Hubei) accounted for 46.3% of cases, while the remaining 53.7% of cases were international. Males and females accounted for 52.4% and 32.9% of cases, respectively, with the remaining 14.6% being designated as unknown. A male skew persisted across subgroup analyses by age group (p=1.0) and location (inside/outside China) (p=0.22). While the number of reported international cases has been steadily increasing over the study period, the number of reported cases in China rapidly decreased from the start point. The median reporting delay was 3 [IQR: 2-4.8] days. The median delay between symptom onset and first seeking medical care was 1 [IQR: 0-3.25] day. In international cases, time to first seeking medical care was a median of 2.5 days longer than in China (p=0.04). When clinical features were reported, fever was the most common presentation (68.0%), followed by cough (36.0%). Conclusions: The number of reported international pediatric COVID-19 cases is rapidly increasing. COVID-19 infections are, to-date, more common in males than females in both the children and adolescent age groups. Additionally, this male predominance remains the case both inside and outside of China. Crowdsourced data enabled early analysis of epidemiologic variables in pediatric patients with COVID-19. Further data sharing is required to enable analyses that are required to understand the course of this infection in children.","2020-03-06","Brandon Michael Henry; Maria Helena S Oliveira",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.20029884"],["6om1y33o","a891d059692c951d1fb0fb36914fe451f973684a","biorxiv","Kallikrein 13: a new player in coronaviral infections","10.1101/2020.03.01.971499",null,null,"biorxiv","ABSTRACTHuman coronavirus HKU1 (HCoV-HKU1) is associated with respiratory disease and is prevalent worldwide, but in vitro model for virus replication is lacking. Interaction between the coronaviral spike (S) protein and its receptor is the major determinant of virus tissue and host specificity, but virus entry is a complex process requiring a concerted action of multiple cellular elements. Here, we show that KLK13 is required for the infection of the human respiratory epithelium and is sufficient to mediate the entry of HCoV-HKU1 to non-permissive RD cells. We also demonstrated HCoV-HKU1 S protein cleavage by KLK13 in the S1/S2 region, proving that KLK13 is the priming enzyme for this virus. Summarizing, we show for the first time that protease distribution and specificity predetermines the tissue and cell specificity of the virus and may also regulate interspecies transmission. It is also of importance that presented data may be relevant for the emerging coronaviruses, including SARS-CoV-2 and may help to understand the differences in their zoonotic potential.","2020-03-02","Aleksandra Milewska; Katherine Falkowski; Magdalena Kalinska; Ewa Bielecka; Antonina Naskalska; Pawel Mak; Adam Lesner; Marek Ochman; Maciej Urlik; Jan Potempa; Tomasz Kantyka; Krzysztof Pyrc",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.01.971499"],["yu89n6rq","d810f78f8850d9386abc6fdd8d652f2f85191753","medrxiv","The effect of human mobility and control measures on the COVID-19 epidemic in China","10.1101/2020.03.02.20026708",null,null,"medrvix","The ongoing COVID-19 outbreak has expanded rapidly throughout China. Major behavioral, clinical, and state interventions are underway currently to mitigate the epidemic and prevent the persistence of the virus in human populations in China and worldwide. It remains unclear how these unprecedented interventions, including travel restrictions, have affected COVID-19 spread in China. We use real-time mobility data from Wuhan and detailed case data including travel history to elucidate the role of case importation on transmission in cities across China and ascertain the impact of control measures. Early on, the spatial distribution of COVID-19 cases in China was well explained by human mobility data. Following the implementation of control measures, this correlation dropped and growth rates became negative in most locations, although shifts in the demographics of reported cases are still indicative of local chains of transmission outside Wuhan. This study shows that the drastic control measures implemented in China have substantially mitigated the spread of COVID-19.","2020-03-06","Moritz U.G. Kraemer; Chia-Hung Yang; Bernardo Gutierrez; Chieh-Hsi Wu; Brennan Klein; David M. Pigott; Louis du Plessis; Nuno R Faria; Ruoran Li; William P. Hanage; John S Brownstein; Maylis Layan; Alessandro Vespignani; Huaiyu Tian; Christopher Dye; Simon Cauchemez; Oliver Pybus; Samuel V Scarpino",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20026708"],["ngbfiws5","d85d142716f7f85cd27b54fc38a9e2968cf14346","medrxiv","Effects of weather-related social distancing on city-scale transmission of respiratory viruses","10.1101/2020.03.02.20027599",null,null,"medrvix","BACKGROUND: Unusually high snowfall in western Washington State in February 2019 led to widespread school and workplace closures. We assessed the impact of social distancing caused by this extreme weather event on the transmission of respiratory viruses. METHODS: Residual specimens from patients evaluated for acute respiratory illness at hospitals in the Seattle metropolitan area were screened for a panel of respiratory viruses. Transmission models were fit to each virus, with disruption of contact rates and care-seeking informed by data on local traffic volumes and hospital admissions. RESULTS: Disruption in contact patterns reduced effective contact rates during the intervention period by 16% to 95%, and cumulative disease incidence through the remainder of the season by 3% to 9%. Incidence reductions were greatest for viruses that were peaking when the disruption occurred and least for viruses in early epidemic phase. CONCLUSION: High-intensity, short-duration social distancing measures may substantially reduce total incidence in a respiratory virus epidemic if implemented near the epidemic peak.","2020-03-03","Michael L Jackson; Gregory R Hart; Denise J McCulloch; Amanda Adler; Elisabeth Brandstetter; Kairsten Fay; Peter Han; Kirsten Lacombe; Jover Lee; Thomas Sibley; Deborah A Nickerson; Mark Rieder; Lea Starita; Janet A Englund; Trevor Bedford; Helen Chu; Michael Famulare",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20027599"],["rlsdznmv",null,"medrxiv","Monitoring Disease Transmissibility of 2019 Novel Coronavirus Disease in Zhejiang, China","10.1101/2020.03.02.20028704",null,null,"medrvix","We monitored the transmissibility of 2019 novel coronavirus disease in Zhejiang accounting the transmissions from imported cases. Even though Zhejiang is one of the worst-affected provinces, an interruption of disease transmission (i.e. instantaneous reproduction numbers <1) was observed in early/mid-February after an early social-distancing response to the outbreak.","2020-03-05","Ka Chun Chong; Wei Cheng; Shi Zhao; Feng Ling; Kirran N Mohammad; Maggie Haitian Wang; Benny Chung-ying Zee; Lesley Wei; Xi Xiong; Hengyan Liu; Jingxuan Wang; Enfu Chen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20028704"],["qlkt5fzp","70cc2e5152d3dc4d44494124ff556c9bbe9e6f41","medrxiv","Clinical Characteristics of Patients with Severe Pneumonia Caused by the 2019 Novel Coronavirus in Wuhan, China","10.1101/2020.03.02.20029306",null,null,"medrvix","Abstract Background: A new virus broke out in Wuhan, Hubei, China, and was later named 2019 novel coronavirus (2019-nCoV). The clinical characteristics of severe pneumonia caused by 2019-nCoV are still not clear. Objectives: The aim of this study was to explore the clinical characteristics and risk factors of the severe pneumonia caused by the 2019-nCoV in Wuhan, China. Method: The study included patients hospitalized at the central hospital of Wuhan who had been diagnosed with a pneumonia caused by the novel coronavirus. Clinical features, chronic co-morbidities, demographic data, laboratory examinations, and chest computed tomography (CT) scans were reviewed through electronic medical records. SPSS was used for data analysis to explore the clinical characteristics and risk factors of the patients with the severe pneumonia. Results: A total of 110 patients diagnosed with 2019 novel coronavirus pneumonia were included in the study, including 38 with severe pneumonia and 72 with non-severe pneumonia. Statistical analysis showed that advanced age, an increase of D-dimer, and a decrease of lymphocytes were characteristics of the patients with severe pneumonia. Moreover, in the early stage of the disease, chest CT scans of patients with the severe pneumonia showed the illness can progress rapidly. Conclusions: Advanced age, lymphocyte decline, and D-dimer elevation are important characteristics of patients with severe pneumonia. Clinicians should focus on these characteristics to identify high-risk patients at an early stage.","2020-03-03","Yafei Wang; Ying Zhou; Zhen Yang; Dongping Xia; Shuang Geng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20029306"],["4nmc356g","f82372b109168b411344f537210c63cc6ed323ed","medrxiv","The Seattle Flu Study: a multi-arm community-based prospective study protocol for assessing influenza prevalence, transmission, and genomic epidemiology","10.1101/2020.03.02.20029595",null,null,"medrvix","Introduction. Influenza epidemics and pandemics cause significant morbidity and mortality. An effective response to a potential pandemic requires the infrastructure to rapidly detect, characterize, and potentially contain new and emerging influenza strains at a population level. The objective of this study is to use data gathered simultaneously from community and hospital sites to develop a model of how influenza enters and spreads in a population. Methods and Analysis. Starting in the 2018-19 season, we have been enrolling individuals with acute respiratory illness from community sites throughout the Seattle metropolitan area, including clinics, childcare facilities, Seattle-Tacoma International Airport, workplaces, college campuses, and homeless shelters. At these sites, we collect clinical data and mid-nasal swabs from individuals with at least two acute respiratory symptoms. Additionally, we collect residual nasal swabs and data from individuals who seek care for respiratory symptoms at four regional hospitals. Samples are tested using a multiplex molecular assay, and influenza whole genome sequencing is performed for samples with influenza detected. Geospatial mapping and computational modeling platforms are in development to characterize the regional spread of influenza and other respiratory pathogens. Ethics and Dissemination. The study was approved by the University of Washington's Institutional Review Board. Results will be disseminated through talks at conferences, peer-reviewed publications, and on the study website (www.seattleflu.org).","2020-03-06","Helen Y. Chu; Michael Boeckh; Janet A. Englund; Michael Famulare; Barry R. Lutz; Deborah A Nickerson; Mark J. Rieder; Lea M Starita; Amanda Adler; Elisabeth Brandstetter; Chris D. Frazar; Peter D. Han; Reena K. Gularti; James Hadfield; Michael L. Jackson; Anahita Kiavand; Louise E. Kimball; Kirsten Lacombe; Jennifer Logue; Victoria Lyon; Kira L. Newman; Thomas R. Sibley; Monica L. Zigman Suschsland; Caitlin Wolf; Jay Shendure; Trevor Bedford",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20029595"],["b10yokyu","5796fdc80a85b6e634a279c57711356055b65880","medrxiv","Close contacts and household transmission of SARS-CoV-2 in China: a content analysis based on local Heath Commissions' public disclosures.","10.1101/2020.03.02.20029868",null,null,"medrvix","Public disclosure from Chinese local Health Commissions show that male and mid-aged index patients are the major source of virus from outside to the households. This implies that two important actions should be taken in disease control. First considers the kindergarten and school closure time. Local kindergarten and primary schools should postpone their admission and regular operation, to at least not earlier than local secondary schools. Second, workplace hygiene is of particular important in the next phase of disease control. As travel restrictions within China are being gradually lifted, we suggest that workplace hygiene should be taken serious care in the next month or two.","2020-03-06","xiaoke Xu; Xiaofan Liu; Ye Wu; Sheikh Taslim ALI",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20029868"],["c3dxfet9","ebed882da10cbf669cbb86802e9fa07a4a33ef91","medrxiv","Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome","10.1101/2020.03.02.20029975",null,null,"medrvix","The outbreak of Coronavirus Disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, December 2019, and continuously poses a serious threat to public health. Our previous study has shown that cytokine storm occurred during SARS-CoV-2 infection, while the detailed role of cytokines in the disease severity and progression remained unclear due to the limited case number. In this study, we examined 48 cytokines in the plasma samples from 53 COVID-19 cases, among whom 34 were severe cases, and the others moderate. Results showed that 14 cytokines were significantly elevated upon admission in COVID-19 cases. Moreover, IP-10, MCP-3, and IL-1ra were significantly higher in severe cases, and highly associated with the PaO2/FaO2 and Murray score. Furthermore, the three cytokines were independent predictors for the progression of COVID-19, and the combination of IP-10, MCP-3 and IL-1ra showed the biggest area under the curve (AUC) of the receiver-operating characteristics (ROC) calculations. Serial detection of IP-10, MCP-3 and IL-1ra in 14 severe cases showed that the continuous high levels of these cytokines were associated with disease deterioration and fatal outcome. In conclusion, we report three cytokines that closely associated with disease severity and outcome of COVID-19. These findings add to our understanding of the immunopathologic mechanisms of SARS-CoV-2 infection, which suggested novel therapeutic targets and strategy.","2020-03-06","Yang Yang; Chenguang Shen; Jinxiu Li; Jing Yuan; Minghui Yang; Fuxiang Wang; Guobao Li; Yanjie Li; Li Xing; Ling Peng; Jinli Wei; Mengli Cao; Haixia Zheng; Weibo Wu; Rongrong Zou; Delin Li; Zhixiang Xu; Haiyan Wang; Mingxia Zhang; Zheng Zhang; Lei Liu; Yingxia Liu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20029975"],["zav5gksq","c8df44a3612e85e267351e936ddeb8fc5867afa1","medrxiv","The timing of one-shot interventions for epidemic control","10.1101/2020.03.02.20030007",null,null,"medrvix","The apparent early success in China's large-scale intervention to control the COVID-19 epidemic has led to interest in whether other countries can replicate it as well as concerns about a resurgence of the epidemic if or when China relaxes the interventions. In this paper we look at the impact of a single short-term intervention on an epidemic. We see that if an intervention cannot be sustained long-term, it has the greatest impact if it is imposed once infection levels have become large enough that there is an appreciable number of infections present. For minimising the total number infected it should start close to the peak so that there is no rebound once the intervention is stopped, while to minimise the peak prevalence, it should start earlier, allowing two peaks of comparable size rather than one very large peak. In populations with distinct subgroups, synchronized interventions are less effective than targeting the interventions in each sub-population separately. We do not attempt to clearly determine what makes an intervention sustainable or not. We believe that is a policy question. If an intervention is sustainable, it should be kept in place. Our intent is to offer insight into how best to time an intervention whose impact on society is too great to maintain.","2020-03-06","Francesco Di Lauro; István Z Kiss; Joel Miller",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030007"],["wtdzad90","d0b33d173f1d32662e2ad8d7e85181f9b23f010c","medrxiv","Estimation of COVID-19 outbreak size in Italy based on international case exportations","10.1101/2020.03.02.20030049",null,null,"medrvix","Italy is currently experiencing an epidemic of COVID-19 which emerged in the Lombardy region . During the interval between February 25-29, 2020, we identified 46 cases of COVID-19 reported in 21 countries in Europe, Africa, North America, and South America which were either in individuals with recent travel from Italy, or who had presumed infection by a traveler from Italy 2. In six cases, in four of the affected countries (Switzerland, France, Austria, Croatia), land travel was a likely route of introduction, or was documented to have been the route of introduction. We used air travel volume between Italian cities and cities in other countries as an index of connectedness, using data available from the International Air Transport Association (IATA) for February 2015 (2.61 million total departing international air passengers from Italy). We used the methods of Fraser et al. to estimate the size of the underlying epidemic in Italy necessary in order for these cases to be observed with a reasonable probability. To estimate the time at risk of COVID-19 exposure for travelers departing Italy, we obtained data from the United Nations World Tourism Organization (UNWTO) for the proportion of international travelers that are non-residents of Italy (63%) and the average length of stay of tourists to Italy (3.4 days), and assumed the Italian epidemic began one month prior to February 29, 2020. We also performed sensitivity analyses in which we included outbound travel to all countries regardless of reported case importations, inflated travel volumes by 35%, to account for the relative increase in flight numbers from 2015-2019, and excluded cases in bordering countries and which were documented to have been introduced by overland travel. When all cases were considered we estimated a true outbreak size of 3971 cases (95% CI 2907-5297), as compared to a reported case count of 1128 on February 29, 2020, suggesting non-identification of 72% (61-79%) of cases. In sensitivity analyses, outbreak sizes varied from 1552 to 4533 cases (implying non-identification of 27-75% of cases). We recently used similar methods to estimate a much larger epidemic size in Iran, with a far greater degree of under-reporting, based on many fewer exported cases. The reason for this difference relates to the relatively high volume of travel from Italy, relative to Iran. In summary, we suggest that the numerous COVID-19 case exportations from Italy in recent days suggest an epidemic that is larger than official case counts suggest, and which is approximately on a par with that currently occurring in South Korea, which reports 3526 cases (and fewer deaths) as of February 29, 2020.","2020-03-06","Ashleigh Tuite; Victoria Ng; Erin Rees; David Fisman",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030049"],["xfxf1vse",null,"medrxiv","Forecasting the Cumulative Number of COVID-19 Deaths in China: a Boltzmann Function-based Modeling Study","10.1101/2020.03.02.20030064",null,null,"medrvix","The COVID-19 outbreak is on-going in China. Here we estimated the potential total numbers of COVID-19 deaths in China, outside Hubei (in China), Hubei Province, Wuhan City and outside Wuhan (in Hubei) by Boltzmann function-based analyses, which are 3342 (95% CI, 3214, 3527), 111 (109, 114), 3245 (3100, 3423), 2613 (2498, 2767) and 627 (603, 654), respectively. The results may help to evaluate the severity of COVID-19 outbreaks and facilitate timely mental service for the families of passed patients.","2020-03-03","Cheng Long; Qi Ying; Xinmiao Fu; Zhongyan Li; Yuanyuan Gao",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030064"],["4f84pc83","3b181a741d4beafda3ba3a8fc68239493d49f6aa","medrxiv","Estimation of local novel coronavirus (COVID-19) cases in Wuhan, China from off-site reported cases and population flow data from different sources","10.1101/2020.03.02.20030080",null,null,"medrvix","Backgrounds: In December 2019, a novel coronavirus (COVID-19) pneumonia hit Wuhan, Hubei Province, China and spread to the rest of China and overseas. The emergence of this virus coincided with the Spring Festival Travel Rush in China. It is possible to estimate total number of cases of COVID-19 in Wuhan, by 23 January 2020, given the cases reported in other cities and population flow data between cities. Methods: We built a model to estimate the total number of cases in Wuhan by 23 January 2020, based on the number of cases detected outside Wuhan city in China, with the assumption that if the same screening effort used in other cities applied in Wuhan. We employed population flow data from different sources between Wuhan and other cities/regions by 23 January 2020. The number of total cases was determined by the maximum log likelihood estimation. Findings: From overall cities/regions data, we predicted 1326 (95% CI: 1177, 1484), 1151 (95% CI: 1018, 1292) and 5277 (95% CI: 4732, 5859) as total cases in Wuhan by 23 January 2020, based on different source of data from Changjiang Daily newspaper, Tencent, and Baidu. From separate cities/regions data, we estimated 1059 (95% CI: 918, 1209), 5214 (95% CI: 4659, 5808) as total cases in Wuhan in Wuhan by 23 January 2020, based on different sources of population flow data from Tencent and Baidu. Conclusion: Sources of population follow data and methods impact the estimates of local cases in Wuhan before city lock down. Keyword: COVID-19; mobility; pneumonia; transportation; outbreaks","2020-03-02","Zian Zhuang; Peihua Cao; Shi Zhao; Yijun Lou; Weiming Wang; Shu Yang; Lin Yang; Daihai He",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030080"],["va4t15mb","267729947ca478946d5a4bffb8e13d50c3545120","medrxiv","Rapid Detection of SARS-CoV-2 Using Reverse transcription RT-LAMP method","10.1101/2020.03.02.20030130",null,null,"medrvix","Corona Virus Disease 2019 (COVID-19) is a recently emerged life-threatening disease caused by SARS-CoV-2. Real- time fluorescent PCR (RT-PCR) is the clinical standard for SARS-CoV-2 nucleic acid detection. To detect SARS-CoV-2 early and control the disease spreading on time, a faster and more convenient method for SARS-CoV-2 nucleic acid detecting, RT-LAMP method (reverse transcription loop-mediated isothermal amplification) was developed. RNA reverse transcription and nucleic acid amplification were performed in one step at 63 ℃ isothermal conditions, and the results can be obtained within 30 minutes. ORF1ab gene, E gene and N gene were detected at the same time. ORF1ab gene was very specific and N gene was very sensitivity, so they can guarantee both sensitivity and specificity for SARS-CoV-2. The sensitivity of RT-LAMP assay is similar to RT-PCR, and specificity was 99% as detecting 208 clinical specimens. The RT-LAMP assay reported here has the advantages of rapid amplification, simple operation, and easy detection, which is useful for the rapid and reliable clinical diagnosis of SARS-CoV-2.","2020-03-02","Weihua Yang; Xiaofei Dang; Qingxi Wang; Mingjie Xu; Qianqian Zhao; Yunying Zhou; Huailong Zhao; Li Wang; Yihui Xu; Jun Wang; Shuyi Han; Min Wang; Fengyan Pei; Yunshan Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030130"],["i7tjno0w","b53dd68cad40cbcd698a848f53853e5f03d44a1f","medrxiv","Validity of Wrist and Forehead Temperature in Temperature Screening in the General Population During the Outbreak of 2019 Novel Coronavirus: a prospective real-world study","10.1101/2020.03.02.20030148",null,null,"medrvix","Aims: Temperature screening is important in the population during the outbreak of 2019 Novel Coronavirus (COVID-19). This study aimed to compare the accuracy and precision of wrist and forehead temperature with tympanic temperature under different circumstances. Methods: We performed a prospective observational study in a real-life population. We consecutively collected wrist and forehead temperatures in Celsius (C) using a non-contact infrared thermometer (NCIT). We also measured the tympanic temperature using a tympanic thermometers (IRTT) and defined fever as a tympanic temperature ≥37.3C. Results: We enrolled a total of 528 participants including 261 indoor and 267 outdoor participants. We divided outdoor participants into four types according to their means of transportation to the hospital as walk, bicycle, electric vehicle, car, and inside the car. Under different circumstance, the mean difference ranged from -1.72 to -0.56C in different groups for the forehead measurements, and -0.96 to -0.61C for the wrist measurements. Both measurements had high fever screening abilities in inpatients (wrist: AUC 0.790; 95% CI: 0.725-0.854, P <0.001; forehead: AUC 0.816; 95% CI: 0.757-0.876, P <0.001). The cut-off value of wrist measurement for detecting tympanic temperature ≥37.3C was 36.2C with a 86.4% sensitivity and a 67.0% specificity, and the best threshold of forehead measurement was also 36.2C with a 93.2% sensitivity and a 60.0% specificity. Conclusions: Wrist measurement is more stable than forehead measurement under different circumstance. Both measurements have great fever screening abilities for indoor patients. The cut-off value of both measurements was 36.2C.","2020-03-06","Ge Chen; Jiarong Xie; Guangli Dai; Peijun Zheng; Xiaqing Hu; Hongpeng Lu; Lei Xu; Xueqin Chen; Xiaomin Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030148"],["llqpfhwg","ff067164497bcfbd9145be223dcd2b05f159dd63","medrxiv","Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019","10.1101/2020.03.02.20030189",null,null,"medrvix","Summary Background The novel coronavirus SARS-CoV-2 is a newly emerging virus. The antibody response in infected patient remains largely unknown, and the clinical values of antibody testing have not been fully demonstrated. Methods A total of 173 patients with confirmed SARS-CoV-2 infection were enrolled. Their serial plasma samples (n = 535) collected during the hospitalization period were tested for total antibodies (Ab), IgM and IgG against SARS-CoV-2 using immunoassays. The dynamics of antibodies with the progress and severity of disease was analyzed. Findings Among 173 patients, the seroconversion rate for Ab, IgM and IgG was 93.1% (161/173), 82.7% (143/173) and 64.7% (112/173), respectively. Twelve patients who had not seroconverted were those only blood samples at the early stage of illness were collected. The seroconversion sequentially appeared for Ab, IgM and then IgG, with a median time of 11, 12 and 14 days, respectively. The presence of antibodies was < 40% among patients in the first 7 days of illness, and then rapidly increased to 100.0%, 94.3% and 79.8% for Ab, IgM and IgG respectively since day 15 after onset. In contrast, the positive rate of RNA decreased from 66.7% (58/87) in samples collected before day 7 to 45.5% (25/55) during days 15 to 39. Combining RNA and antibody detections significantly improved the sensitivity of pathogenic diagnosis for COVID-19 patients (p < 0.001), even in early phase of 1-week since onset (p = 0.007). Moreover, a higher titer of Ab was independently associated with a worse clinical classification (p = 0.006). Interpretation The antibody detection offers vital clinical information during the course of SARS-CoV-2 infection. The findings provide strong empirical support for the routine application of serological testing in the diagnosis and management of COVID-19 patients.","2020-03-03","Juanjuan Zhao; Quan Yuan; Haiyan Wang; Wei Liu; Xuejiao Liao; Yingying Su; Xin Wang; Jing Yuan; Tingdong Li; Jinxiu Li; Shen Qian; Congming Hong; Fuxiang Wang; Yingxia Liu; Zhaoqin Wang; Qing He; Zhiyong Li; Bin He; Tianying Zhang; Shengxiang Ge; Lei Liu; Jun Zhang; Ningshao Xia; Zheng Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030189"],["dvrstz6f",null,"medrxiv","Estimating the burden of United States workers exposed to infection or disease: a key factor in containing risk of COVID-19 infection","10.1101/2020.03.02.20030288",null,null,"medrvix","Introduction: With the global spread of COVID-19, there is a compelling public health interest in quantifying who is at increased risk of disease. Occupational characteristics, such as interfacing with the public and being in close quarters with other workers, not only put workers at high risk for disease, but also make them a nexus of disease transmission to the community. This can further be exacerbated through presenteeism, the term used to describe the act of coming to work despite being symptomatic for disease. Understanding which occupational groups are exposed to infection and disease in the workplace can help to inform public health risk response and management for COVID-19, and subsequent infectious disease outbreaks. Methods: To estimate the burden of United States workers exposed to infection and disease in the workplace, national employment data (by Standard Occupational Classification) maintained by the Bureau of Labor Statistics (BLS) was merged with BLS O*NET survey data, which ranks occupations with particular physical, ergonomic, and structural exposures. For this analysis, occupations reporting exposure to infection or disease more than once a month was the focus. Results: Based on our analyses, approximately 10% (14.4 M) of United States workers are employed in occupations where exposure to disease or infection occurs at least once per week. Approximately 18.4% (26.7 M) of all United States workers are employed in occupations where exposure to disease or infection occurs at least once per month. While the majority of exposed workers are employed in healthcare sectors, other occupational sectors also have high proportions of exposed workers. These include protective service occupations (e.g. police officers, correctional officers, firefighters), office and administrative support occupations (e.g. couriers and messengers, patient service representatives), education occupations (e.g. preschool and daycare teachers), community and social services occupations (community health workers, social workers, counselors), and even construction and extraction occupations (e.g. plumbers, septic tank installers, elevator repair). Conclusions: The large number of persons employed in a wide variety of occupations with frequent exposure to infection and disease underscore the importance of all workplaces developing risk response plans for COVID-19. This work also serves as an important reminder that the workplace is a key locus for public health interventions, which could protect both workers and the communities they serve.","2020-03-06","Marissa G Baker; Trevor K Peckham; Noah S. Seixas",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030288"],["etl9uwt7","7fb7c0c48b30e66dfaea5fdec16d999ed52d1ee0","medrxiv","Preliminary estimating the reproduction number of the coronavirus disease (COVID-19) outbreak in Republic of Korea and Italy by 5 March 2020","10.1101/2020.03.02.20030312",null,null,"medrvix","The novel coronavirus disease 2019 (COVID-19) outbreak and Italy has caused 6088 cases and 41 deaths in Republic of Korea and 3144 cases and 107 death in Italy by 5 March 2020. We modeled the transmission process in Republic of Korea and Italy with a stochastic model and estimated the basic reproduction number R0 as 2.6 (95% CI: 2.3-2.9) or 3.2 (95% CI: 2.9-3.5) in Republic of Korea, under the assumption that the exponential growth starting on 31 January or 5 February 2020, and 2.6 (95% CI: 2.3-2.9) or 3.3 (95% CI: 3.0-3.6) in Italy, under the assumption that the exponential growth starting on 5 February or 10 February 2020. Estimates of dispersion term (k) were 10 (95% CI: 5-56) or 22 (95% CI: 8-61) in Republic of Korea, and 13 (95% CI: 5-61) or 37 (95% CI: 13-61) in Italy, and all of which imply few super-spreading events.","2020-03-06","Zian Zhuang; Shi Zhao; Qianying Lin; Peihua Cao; Yijun Lou; Lin Yang; Shu Yang; Daihai He; Li Xiao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030312"],["zclymg50","e5163021a1b88e2c2335cca27fbfcf883f870830","medrxiv","Preliminary estimation of the novel coronavirus disease (COVID-19) cases in Iran: a modelling analysis based on overseas cases and air travel data","10.1101/2020.03.02.20030320",null,null,"medrvix","As of 1 March 2020, Iran has reported 987 COVID-19 cases and including 54 associated deaths. At least six neighboring countries (Bahrain, Iraq, Kuwait, Oman, Afghanistan and Pakistan) have reported imported COVID-19 cases from Iran. We used air travel data and the cases from Iran to other Middle East countries and estimated 16533 (95% CI: 5925, 35538) COVID-19 cases in Iran by 25 February, before UAE and other Gulf Cooperation Council countries suspended inbound and outbound flights from Iran.","2020-03-06","Zian Zhuang; Shi Zhao; Qianying Lin; Peihua Cao; Yijun Lou; Lin Yang; Daihai He",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030320"],["jir7n19b","49ac69f362c27acbc6de0c5cbb640267e7a1e797","medrxiv","Clinical features and outcomes of 221 patients with COVID-19 in Wuhan, China","10.1101/2020.03.02.20030452",null,null,"medrvix","Rationale: In late December 2019, an outbreak of acute respiratory illness, now officially named as COVID-19, or coronavirus disease 2019, emerged in Wuhan, China, now spreading across the whole country and world. More data were needed to understand the clinical characteristics of the disease. Objectives: To study the epidemiology, clinical features and outcomes of patients with COVID-19. Methods: we performed a single center, retrospective case series study in 221 patients with laboratory confirmed SARS-CoV-2 pneumonia at a university hospital. Measurements and Main Results: The median age was 55.0 years and 48.9% were male and only 8 (3.6%) patients had a history of exposure to the Huanan Seafood Market. Compared to the non-severe pneumonia patients, the median age of the severe patients was significantly older, and they were more likely to have chronic comorbidities. Most common symptoms in severe patients were high fever, anorexia and dyspnea. On admission, 33.0% patients showed leukopenia and 73.8% showed lymphopenia. In addition, the severe patients suffered a higher rate of co-infections with bacteria or fungus and they were more likely to developing complications. As of February 15, 2020, 19.0% patients had been discharged and 5.4% patients died. 80% of severe cases received ICU care, and 52.3% of them transferred to the general wards due to relieved symptoms, and the mortality rate of severe patients in ICU was 20.5%. Conclusions: The COVID-19 epidemic spreads rapidly by human-to-human transmission. Patients with elder age, chronic comorbidities, blood leukocyte/lymphocyte count, procalcitonin level, co-infection and severe complications might increase the risk of poor clinical outcomes. Keywords: coronavirus disease 2019; clinical features; outcomes; severe patients","2020-03-06","Guqin Zhang; Chang Hu; Linjie Luo; Fang Fang; Yongfeng Chen; Jianguo Li; Zhiyong Peng; Huaqin Pan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.20030452"],["xq1cxrvf","c15f8c23eedbedf8d48bbc88b8225ae9c9d234bf","biorxiv","Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2","10.1101/2020.03.02.968388",null,null,"biorxiv","ABSTRACTSevere Acute Respiratory Syndrome Coronavirus 2 is rapidly spreading around the world. There is no existing vaccine or proven drug to prevent infections and stop virus proliferation. Although this virus is similar to human and animal SARS- and MERS-CoVs the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies and antivirals. We applied high-throughput protein production and structure determination pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures. Here we report the high-resolution crystal structure of endoribonuclease Nsp15/NendoU from SARS-CoV-2 – a virus causing current world-wide epidemics. We compare this structure with previously reported models of Nsp15 from SARS and MERS coronaviruses.","2020-03-03","Youngchang Kim; Robert Jedrzejczak; Natalia I. Maltseva; Michael Endres; Adam Godzik; Karolina Michalska; Andrzej Joachimiak",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.968388"],["fu3cl7lq","2eac7de1c407d9eab13a6d538142cd21a814ee7b","biorxiv","Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia","10.1101/2020.03.02.968818",null,null,"biorxiv","AbstractRapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia’s first case of COVID-2019.","2020-03-05","Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.968818"],["cr22vp8b","7607a4ad84452e29998a80f44bfd6bb2f5f68a7f","biorxiv","Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM","10.1101/2020.03.02.972927",null,null,"biorxiv","AbstractSince December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.","2020-03-05","Chuang Liu; Yang Yang; Yuanzhu Gao; Chenguang Shen; Bin Ju; Congcong Liu; Xian Tang; Jinli Wei; Xiaomin Ma; Weilong Liu; Shuman Xu; Yingxia Liu; Jing Yuan; Jing Wu; Zheng Liu; Zheng Zhang; Peiyi Wang; Lei Liu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.972927"],["xdsi7oye",null,"biorxiv","Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient","10.1101/2020.03.02.972935",null,null,"biorxiv","ABSTRACTThe etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.","2020-03-03","Jennifer Harcourt; Azaibi Tamin; Xiaoyan Lu; Shifaq Kamili; Senthil Kumar. Sakthivel; Janna Murray; Krista Queen; Ying Tao; Clinton R. Paden; Jing Zhang; Yan Li; Anna Uehara; Haibin Wang; Cynthia Goldsmith; Hannah A. Bullock; Lijuan Wang; Brett Whitaker; Brian Lynch; Rashi Gautam; Craig Schindewolf; Kumari G. Lokugamage; Dionna Scharton; Jessica A. Plante; Divya Mirchandani; Steven G. Widen; Krishna Narayanan; Shinji Makino; Thomas G. Ksiazek; Kenneth S. Plante; Scott C. Weaver; Stephen Lindstrom; Suxiang Tong; Vineet D. Menachery; Natalie J. Thornburg",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.972935"],["vorczspu","bad4192bbc6d41c066bc947eaeeafc53024c98be","biorxiv","Evidence for RNA editing in the transcriptome of 2019 Novel Coronavirus","10.1101/2020.03.02.973255",null,null,"biorxiv","AbstractThe 2019-nCoV outbreak has become a global health risk. Editing by host deaminases is an innate restriction process to counter viruses, and it is not yet known whether it operates against coronaviruses. Here we analyze RNA sequences from bronchoalveolar lavage fluids derived from two Wuhan patients. We identify nucleotide changes that may be signatures of RNA editing: Adenosine-to-Inosine changes from ADAR deaminases and Cytosine-to-Uracil changes from APOBEC ones. A mutational analysis of genomes from different strains of human-hosted Coronaviridae reveals patterns similar to the RNA editing pattern observed in the 2019-nCoV transcriptomes. Our results suggest that both APOBECs and ADARs are involved in Coronavirus genome editing, a process that may shape the fate of both virus and patient.","2020-03-03","Salvatore Di Giorgio; Filippo Martignano; Maria Gabriella Torcia; Giorgio Mattiuz; Silvestro G. Conticello",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.973255"],["fhqbw32a","1d2947d5c0addcbbcfa71d87ea074a8b6ff2a973","biorxiv","Speed and strength of an epidemic intervention","10.1101/2020.03.02.974048",null,null,"biorxiv","AbstractAn epidemic can be characterized by its speed (i.e., the exponential growth rate r) and strength (i.e., the reproductive number ℛ). Disease modelers have historically placed much more emphasis on strength, in part because the effectiveness of an intervention strategy is typically evaluated on this scale. Here, we develop a mathematical framework for this classic, strength-based paradigm and show that there is a corresponding speed-based paradigm which can provide complementary insights. In particular, we note that r = 0 is a threshold for disease spread, just like ℛ = 1, and show that we can measure the speed and strength of an intervention on the same scale as the speed and strength of an epidemic, respectively. We argue that, just as the strength-based paradigm provides the clearest insight into certain questions, the speed-based paradigm provides the clearest view in other cases. As an example, we show that evaluating the prospects of “test-and-treat” interventions against the human immunodeficiency virus (HIV) can be done more clearly on the speed than strength scale, given uncertainty in the proportion of HIV spread that happens early in the course of infection. We suggest that disease modelers should avoid over-emphasizing the reproductive number at the expense of the exponential growth rate, but instead look at these as complementary measures.","2020-03-03","Jonathan Dushoff; Sang Woo Park",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.974048"],["dbowa5bt","7221cb14c12070a78e4ee5abe9484c40289fceea","biorxiv","A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19","10.1101/2020.03.02.974139",null,null,"biorxiv","SummaryThe unprecedented epidemic of pneumonia caused by a novel coronavirus, HCoV-19, in China and beyond has caused public health concern at a global scale. Although bats are regarded as the most likely natural hosts for HCoV-191,2, the origins of the virus remain unclear. Here, we report a novel bat-derived coronavirus, denoted RmYN02, identified from a metagenomics analysis of samples from 227 bats collected from Yunnan Province in China between May and October, 2019. RmYN02 shared 93.3% nucleotide identity with HCoV-19 at the scale of the complete virus genome and 97.2% identity in the 1ab gene in which it was the closest relative of HCoV-19. In contrast, RmYN02 showed low sequence identity (61.3%) to HCoV-19 in the receptor binding domain (RBD) and might not bind to angiotensin-converting enzyme 2 (ACE2). Critically, however, and in a similar manner to HCoV-19, RmYN02 was characterized by the insertion of multiple amino acids at the junction site of the S1 and S2 subunits of the Spike (S) protein. This provides strong evidence that such insertion events can occur in nature. Together, these data suggest that HCoV-19 originated from multiple naturally occurring recombination events among those viruses present in bats and other wildlife species.","2020-03-05","Hong Zhou; Xing Chen; Tao Hu; Juan Li; Hao Song; Yanran Liu; Peihan Wang; Di Liu; Jing Yang; Edward C. Holmes; Alice C. Hughes; Yuhai Bi; Weifeng Shi",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.974139"],["jjraqr85","20f2a638e2c9a1917b7c001c371335da3d288505","biorxiv","Partial RdRp sequences offer a robust method for Coronavirus subgenus classification","10.1101/2020.03.02.974311",null,null,"biorxiv","AbstractThe recent reclassification of the Riboviria, and the introduction of multiple new taxonomic categories including both subfamilies and subgenera for coronaviruses (family Coronaviridae, subfamily Orthocoronavirinae) represents a major shift in how official classifications are used to designate specific viral lineages. While the newly defined subgenera provide much-needed standardisation for commonly cited viruses of public health importance, no method has been proposed for the assignment of subgenus based on partial sequence data, or for sequences that are divergent from the designated holotype reference genomes. Here, we describe the genetic variation of a partial region of the coronavirus RNA-dependent RNA polymerase (RdRp), which is one of the most used partial sequence loci for both detection and classification of coronaviruses in molecular epidemiology. We infer Bayesian phylogenies from more than 7000 publicly available coronavirus sequences and examine clade groupings relative to all subgenus holotype sequences. Our phylogenetic analyses are largely coherent with genome-scale analyses based on designated holotype members for each subgenus. Distance measures between sequences form discrete clusters between taxa, offering logical threshold boundaries that can attribute subgenus or indicate sequences that are likely to belong to unclassified subgenera both accurately and robustly. We thus propose that partial RdRp sequence data of coronaviruses is sufficient for the attribution of subgenus-level taxonomic classifications and we supply the R package, “MyCoV”, which provides a method for attributing subgenus and assessing the reliability of the attribution.Importance StatementThe analysis of polymerase chain reaction amplicons derived from biological samples is the most common modern method for detection and classification of infecting viral agents, such as Coronaviruses. Recent updates to the official standard for taxonomic classification of Coronaviruses, however, may leave researchers unsure as to whether the viral sequences they obtain by these methods can be classified into specific viral taxa due to variations in the sequences when compared to type strains. Here, we present a plausible method for defining genetic dissimilarity cut-offs that will allow researchers to state which taxon their virus belongs to and with what level of certainty. To assist in this, we also provide the R package ‘MyCoV’ which classifies user generated sequences.","2020-03-06","David A Wilkinson; Lea Joffrin; Camille Lebarbenchon; Patrick Mavingui",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.02.974311"],["hnx213kp","3a73b880acd8983f87b210ff7fee434fd6ecc8c8","medrxiv","Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts","10.1101/2020.03.03.20028423",null,null,"medrvix","Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control. Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk. Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 21 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days. Interpretation Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.","2020-03-04","Qifang Bi; Yongsheng Wu; Shujiang Mei; Chenfei Ye; Xuan Zou; Zhen Zhang; Xiaojian Liu; Lan Wei; Shaun A Truelove; Tong Zhang; Wei Gao; Cong Cheng; Xiujuan Tang; Xiaoliang Wu; Yu Wu; Binbin Sun; Suli Huang; Yu Sun; Juncen Zhang; Ting Ma; Justin Lessler; Teijian Feng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20028423"],["jtwb17u8","0f8261c63cdeeb5c2bf9294eba2954e1ca71b244","medrxiv","Effect of non-pharmaceutical interventions for containing the COVID-19 outbreak in China","10.1101/2020.03.03.20029843",null,null,"medrvix","Background: The COVID-19 outbreak containment strategies in China based on non-pharmaceutical interventions (NPIs) appear to be effective. Quantitative research is still needed however to assess the efficacy of different candidate NPIs and their timings to guide ongoing and future responses to epidemics of this emerging disease across the World. Methods: We built a travel network-based susceptible-exposed-infectious-removed (SEIR) model to simulate the outbreak across cities in mainland China. We used epidemiological parameters estimated for the early stage of outbreak in Wuhan to parameterise the transmission before NPIs were implemented. To quantify the relative effect of various NPIs, daily changes of delay from illness onset to the first reported case in each county were used as a proxy for the improvement of case identification and isolation across the outbreak. Historical and near-real time human movement data, obtained from Baidu location-based service, were used to derive the intensity of travel restrictions and contact reductions across China. The model and outputs were validated using daily reported case numbers, with a series of sensitivity analyses conducted. Results: We estimated that there were a total of 114,325 COVID-19 cases (interquartile range [IQR] 76,776 - 164,576) in mainland China as of February 29, 2020, and these were highly correlated (p<0.001, R2=0.86) with reported incidence. Without NPIs, the number of COVID-19 cases would likely have shown a 67-fold increase (IQR: 44 - 94), with the effectiveness of different interventions varying. The early detection and isolation of cases was estimated to prevent more infections than travel restrictions and contact reductions, but integrated NPIs would achieve the strongest and most rapid effect. If NPIs could have been conducted one week, two weeks, or three weeks earlier in China, cases could have been reduced by 66%, 86%, and 95%, respectively, together with significantly reducing the number of affected areas. However, if NPIs were conducted one week, two weeks, or three weeks later, the number of cases could have shown a 3-fold, 7-fold, and 18-fold increase across China, respectively. Results also suggest that the social distancing intervention should be continued for the next few months in China to prevent case numbers increasing again after travel restrictions were lifted on February 17, 2020. Conclusion: The NPIs deployed in China appear to be effectively containing the COVID-19 outbreak, but the efficacy of the different interventions varied, with the early case detection and contact reduction being the most effective. Moreover, deploying the NPIs early is also important to prevent further spread. Early and integrated NPI strategies should be prepared, adopted and adjusted to minimize health, social and economic impacts in affected regions around the World.","2020-03-06","Shengjie Lai; Nick W Ruktanonchai; Liangcai Zhou; Olivia Prosper; Wei Luo; Jessica R Floyd; Amy Wesolowski; Mauricio Santillana; Chi Zhang; Xiangjun Du; Hongjie Yu; Andrew J Tatem",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20029843"],["66ulqu11","3028628066ec2401f3981f4e70c5b1acd4cef573","medrxiv","Transmission interval estimates suggest pre-symptomatic spread of COVID-19","10.1101/2020.03.03.20029983",null,null,"medrvix","Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods: We collected outbreak information from Singapore and Tianjin, China, reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65,2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring. Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread.","2020-03-06","Lauren Tindale; Michelle Coombe; Jessica E Stockdale; Emma Garlock; Wing Yin Venus Lau; Manu Saraswat; Yen-Hsiang Brian Lee; Louxin Zhang; Dongxuan Chen; Jacco Wallinga; Caroline Colijn",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20029983"],["5ciaonf0","462cbb326ccd8587cae7a3538c8c6712d9013698","medrxiv","Epidemiological and clinical features of 291 cases with coronavirus disease 2019 in areas adjacent to Hubei, China: a double-center observational study","10.1101/2020.03.03.20030353",null,null,"medrvix","Abstract Background: The clinical outcomes of COVID-19 patients in Hubei and other areas are different. We aim to investigate the epidemiological and clinical characteristics of patient with COVID-19 in Hunan which is adjacent to Hubei. Methods: In this double-center, observational study, we recruited all consecutive patients with laboratory confirmed COVID-19 from January 23 to February 14, 2020 in two designated hospitals in Hunan province, China. Epidemiological and clinical data from patients' electronic medical records were collected and compared between mild, moderate and severe/critical group in detail. Clinical outcomes were followed up to February 20, 2020. Findings: 291 patients with COVID-19 were categorized into mild group (10.0%), moderate group (72.8%) and severe/critical group (17.2%). The median age of all patients was 46 years (49.8% were male). 86.6% patients had an indirect exposure history. The proportion of patients that had been to Wuhan in severe/critical group (48.0% vs 17.2%, p=0.006) and moderate group (43.4% vs 17.2%, p=0.007) were higher than mild group. Fever (68.7%), cough (60.5%), and fatigue (31.6%) were common symptoms especially for severe and critical patients. Typical lung imaging finding were bilateral and unilateral ground glass opacity or consolidation. Leukopenia, lymphopenia and eosinopenia occurred in 36.1%, 22.7% and 50.2% patients respectively. Increased fibrinogen was detected in 45 of 58 (77.6%) patients with available results. 29 of 44 (65.9%) or 22 of 40 (55.0%) patients were positive in Mycoplasma pneumonia or Chlamydia pneumonia antibody test respectively. Compared with mild or moderate group, severe/critical group had a relative higher level of neutrophil, Neutrophil-to-Lymphocyte Ratio, h-CRP, ESR, CK, CK-MB, LDH, D-dimer, and a lower level of lymphocyte, eosinophils, platelet, HDL and sodium (all p<0.01). Most patients received antiviral therapy and Chinese Medicine therapy. As of February 20, 2020, 159 (54.6%) patients were discharged and 2 (0.7%) patients died during hospitalization. The median length of hospital stay in discharged patients was 12 days (IQR: 10-15). Interpretation: The epidemiological and clinical characteristics of COVID-19 patients in Hunan is different from patients in Wuhan. The proportion of patients that had been to Wuhan in severe/critical group and moderate group were higher than mild group. Laboratory and imaging examination can assist in the diagnosis and classification of COVID-19 patients.","2020-03-06","Xu Chen; Fang Zheng; Yanhua Qing; Shuizi Ding; Danhui Yang; Cheng Lei; Zhilan Yin; Xianglin Zhou; Dixuan Jiang; Qi Zuo; Jun He; Jianlei Lv; Ping Chen; Yan Chen; Hong Peng; Honghui Li; Yuanlin Xie; Jiyang Liu; Zhiguo Zhou; Hong Luo",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030353"],["cv32u0ox","e560eb28bb3f35099d2632f80adaa14436516474","medrxiv","Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients","10.1101/2020.03.03.20030437",null,null,"medrvix","Background: Viral clearance is one important indicator for the recovery of SARS-CoV-2 infected patients. Suboptimal T and B cell responses can delay viral clearance in MERS and SARS patients. The role of leukomonocytes in viral clearance of COVID-19 patients is not yet well defined.Methods: From January 26 to February 28, 2020, an observational study was launched at Zhongnan Hospital of Wuhan University, Wuhan, China. We enrolled 25 laboratory-confirmed COVID-19 patients, whose throat-swab specimens were tested positive for SARS-CoV-2 infection by qRT-PCR. We comprehensively analyzed clinical records, counts of lymphocyte subsets including CD3+, CD4+, CD8+ T cells, B cells and NK cells in the patients who successfully cleared SARS-CoV-2, and compared to those that failed to, after a standardized treatment of 8-14 days. Findings: In 25 enrolled COVID-19 patients, lymphopeniawas a common feature. After the treatment, 14 patients were tested negative for SARS-CoV-2. The patients that cleared the infection had restored the numbers of CD3+, CD4+, CD8+ T cellsand B cells as compared to the still viral RNA positive patients, while the recovered patients had a higher count of leukomonocytes. Conclusions: By comparison of leukomonocytes counts in COVID-19 patients at different stages of the disease, we found that CD3+, CD4+, CD8+ T cells and B cells appear to play important roles in viral clearance. The restoration of leukomonocytes counts from peripheral blood can be used as prognosis for the recovery of an COVID-19 infection. We propose that restoration of leukomonocytes counts can be added to the COVID-19 diagnostic guidanceas a criterion for releasing and discharging patients.","2020-03-06","Xiaoping Chen; Jiaxin Ling; Pingzheng Mo; Yongxi Zhang; Qunqun Jiang; Zhiyong Ma; Qian Cao; Wenjia Hu; Shi Zou; Liangjun Chen; Lei Yao; Mingqi Luo; Tielong Chen; Liping Deng; Ke Liang; Shihui Song; Rongrong Yang; Ruiying Zheng; Shicheng Gao; Xien Gui; Hengning Ke; Wei Hou; Åke Lundkvist; Yong Xiong",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030437"],["y8hzi1gl",null,"medrxiv","Modelling-based evaluation of the effect of quarantine control by the Chinese government in the coronavirus disease 2019 outbreak","10.1101/2020.03.03.20030445",null,null,"medrvix","The novel coronavirus disease 2019 (COVID-19) epidemic, which was first identified in Wuhan, China in December 2019, has rapidly spread all over China and across the world. By the end of February 2020, the epidemic outside Hubei province in China has been well controlled, yet the next wave of transmission in other countries may have just begun. A retrospective modeling of the transmission dynamics would provide insights into the epidemiological characteristics of the disease and evaluation of the effectiveness of the strict measures that have been taken by central and local governments of China. Using a refined susceptible-exposed-infectious-removed (SEIR) transmission model and a new strategy of model fitting, we were able to estimate model parameters in a dynamic manner. The resulting parameter estimation can well reflect the prevention policy scenarios. Our simulation results with different degrees of government control suggest that the strictly enforced quarantine and travel ban have significantly decreased the otherwise uncontrollable spread of the disease. Our results suggest similar measures should be considered by other countries that are of high risk of COVID-19 outbreak.","2020-03-06","Xinkai Zhou; Zhigui Wu; Ranran Yu; Shanni Cao; Wen Fang; Zhen Jiang; Fang Yuan; Chao Yan; Dijun Chen",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030445"],["944pn0k9","9701a8c529cd8c18124da4cd61c1165a64b50281","medrxiv","Evolving Epidemiology and Impact of Non-pharmaceutical Interventions on the Outbreak of Coronavirus Disease 2019 in Wuhan, China","10.1101/2020.03.03.20030593",null,null,"medrvix","BACKGROUND We described the epidemiological features of the coronavirus disease 2019 (Covid-19) outbreak, and evaluated the impact of non-pharmaceutical interventions on the epidemic in Wuhan, China. METHODS Individual-level data on 25,961 laboratory-confirmed Covid-19 cases reported through February 18, 2020 were extracted from the municipal Notifiable Disease Report System. Based on key events and interventions, we divided the epidemic into four periods: before January 11, January 11-22, January 23 - February 1, and February 2-18. We compared epidemiological characteristics across periods and different demographic groups. We developed a susceptible-exposed-infectious-recovered model to study the epidemic and evaluate the impact of interventions. RESULTS The median age of the cases was 57 years and 50.3% were women. The attack rate peaked in the third period and substantially declined afterwards across geographic regions, sex and age groups, except for children (age <20) whose attack rate continued to increase. Healthcare workers and elderly people had higher attack rates and severity risk increased with age. The effective reproductive number dropped from 3.86 (95% credible interval 3.74 to 3.97) before interventions to 0.32 (0.28 to 0.37) post interventions. The interventions were estimated to prevent 94.5% (93.7 to 95.2%) infections till February 18. We found that at least 59% of infected cases were unascertained in Wuhan, potentially including asymptomatic and mild-symptomatic cases. CONCLUSIONS Considerable countermeasures have effectively controlled the Covid-19 outbreak in Wuhan. Special efforts are needed to protect vulnerable populations, including healthcare workers, elderly and children. Estimation of unascertained cases has important implications on continuing surveillance and interventions.","2020-03-06","Chaolong Wang; Li Liu; Xingjie Hao; Huan Guo; Qi Wang; Jiao Huang; Na He; Hongjie Yu; Xihong Lin; An Pan; Sheng Wei; Tangchun Wu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030593"],["d4dnz2lk","9c33486a49de4aea64ce61c0a2c21a88c316b6a8","medrxiv","SOCRATES: An online tool leveraging a social contact data sharing initiative to assess mitigation strategies for COVID-19","10.1101/2020.03.03.20030627",null,null,"medrvix","Objective: Establishing a social contact data sharing initiative and an interactive tool to assess mitigation strategies for COVID-19. Results: We organized data sharing of published social contact surveys via online repositories and formatting guidelines. We analyzed this social contact data in terms of weighted social contact matrices, next generation matrices, relative incidence and R0. We incorporated location-specific isolation measures (e.g. school closure or telework) and capture their effect on transmission dynamics. All methods have been implemented in an online application based on R Shiny and applied to COVID-19 with age-specific susceptibility and infectiousness. Using our online tool with the available social contact data, we illustrate that social distancing could have a considerable impact on reducing transmission for COVID-19. The effect itself depends on assumptions made about disease-specific characteristics and the choice of intervention(s). Keywords: social contact data, user interface, transmission dynamics, infectious diseases, epidemics, social distancing, behavioral changes, data sharing initiative, open-source, COVID-19","2020-03-06","Lander Willem; Thang Van Hoang; Sebastian Funk; Pietro Coletti; Philippe Beutels; Niel Hens",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030627"],["5imbomop","594d23a412649df7dbfef5347a2a0c208ed4417d","medrxiv","Immunodepletion with Hypoxemia: A Potential High Risk Subtype of Coronavirus Disease 2019","10.1101/2020.03.03.20030650",null,null,"medrvix","Background The outbreak of COVID-2019 is becoming a global public health emergency. Although its basic clinical features have been reported, the dynamic characteristics of immune system in COVID-2019 patients, especially those critical patients with refractory hypoxemia, are not yet well understood. We aim to describe the dynamic characteristics of immune system in 3 critical patients with refractory hypoxemia, and discuss the relationship between hypoxemia severity and immune cell levels, and the changes of gut microbes of COVID-2019 patient. Methods This is a retrospective study from 3 patients with 2019-nCoV infection admitted to Renmin Hospital of Wuhan University, a COVID-2019 designated hospital in Wuhan, from January 31 to February 6, 2020. All patients were diagnosed and classified based on the Diagnosis and Treatment of New Coronavirus Pneumonia (6th edition) published by the National Health Commission of China4. We recorded the epidemiological history, demographic features, clinical characteristics, symptoms and signs, treatment and clinical outcome in detail. Blood samples were collected and we determined the expression levels of immune cells (CD3+ T cells, CD4+ T cells, CD8+ T cells, CD19+ B cells, and CD16+56+ NK cells) in different time points. Nanopore Targeted Sequencing was used to determine the alterations of gut microbiota homeostasis. Results Apart from the clinical features described previously4, we found that four patients had decreased immune cells and refractory hypoxemia during the hospitalization, and the severity of hypoxemia was strongly correlated to the expression levels of immune cells. Additionally, we found that the proportion of probiotics was significantly reduced, such as Bifidobacterium, Lactobacillus, and Eubacterium, and the proportion of conditioned pathogenic bacteria was significantly increased, such as Corynebacterium of Actinobacteria and Ruthenibacterium of Firmicutes. Notably, all patients died. Conclusions We discussed the dynamic characteristics of host immune system and the imbalance of gut microbiota in 3 critical patients with COVID-2019. Hypoxemia severity was closely related with host immune cell levels, and the vicious circle between immune disorder and gut microbiota imbalance may be a high risk of fatal pneumonia. To the best of our knowledge, this is the first study which revealing that immunodepletion with refractory hypoxemia is a potential high risk subtype of COVID-2019 and the vicious circle between immune disorder and gut dysbiosis may be a high risk of fatal pneumonia.","2020-03-06","Lilei Yu; Yongqing Tong; Gaigai Shen; Aisi Fu; Yanqiu Lai; Xiaoya Zhou; Yuan Yuan; Yuhong Wang; Yuchen Pan; Zhiyao Yu; Yan Li; Tiangang Liu; Hong Jiang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030650"],["9ecuca17","f879c69d61f14d7efc51023258d1361bc681d403","medrxiv","Clinical findings in critical ill patients infected with SARS-Cov-2 in Guangdong Province, China: a multi-center, retrospective, observational study","10.1101/2020.03.03.20030668",null,null,"medrvix","Abstract Background In December 2019, human infection with a novel coronavirus, known as SARS-CoV-2, was identified in Wuhan, China. The mortality of critical illness was high in Wuhan. Information about critically ill patients with SARS-CoV-2 infection outside of Wuhan is scarce. We aimed to provide the clinical features, treatment, and prognosis of the critically ill patients with SARS-CoV-2 infection in Guangdong Province. Methods In this multi-centered, retrospective, observational study, we enrolled critically ill patients with SARS-CoV-2 pneumonia who were admitted to the intensive care unit (ICU) in Guangdong Province. Demographic data, symptoms, laboratory findings, comorbidities, treatments, and prognosis were collected. Data were compared between patients with and without intubation. Results Forty-five critically ill patients with SARS-CoV-2 pneumonia were identified in 7 ICUs in Guangdong Province. The mean age was 56.7 years, and 29 patients (64.4%) were men. The most common symptoms at the onset of illness were high fever and cough. Majority of patients presented with lymphopenia and elevated lactate dehydrogenase. Treatment with antiviral drugs was initiated in all the patients. Thirty-seven patients (82.2%) had developed acute respiratory distress syndrome, and 13 (28.9%) septic shock. A total of 20 (44.4%) patients required intubation and 9 (20%) required extracorporeal membrane oxygenation. As of February 28th 2020, only one patient (2.2%) had died and half of them had discharged of ICU. Conclusions Infection with SARS-CoV-2 in critical illness is characterized by fever, lymphopenia, acute respiratory failure and multiple organ dysfunction. Compared with critically ill patients infected with SARS-CoV-2 in Wuhan, the mortality of critically ill patients in Guangdong Province was relatively low. These data provide some general understandings and experience for the critical patients with SARS-CoV-2 outside of Wuhan.","2020-03-06","Yonghao Xu; Zhiheng Xu; Xuesong Liu; Lihua Cai; Haichong Zheng; Yongbo Huang; Lixin Zhou; Linxi Huang; Yun Lin; Liehua Deng; Jianwei Li; Sibei Chen; Dongdong Liu; Zhimin Lin; Liang Zhou; Weiqun He; Xiaoqing Liu; Yimin Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030668"],["6t9kl3hq","e0a53c258ee0483666329ccab591c0753ff49531","medrxiv","Key Points of Clinical and CT Imaging Features of 2019 Novel Coronavirus (2019-nCoV) Imported Pneumonia Based On 21 Cases Analysis","10.1101/2020.03.03.20030775",null,null,"medrvix","Background and Objective: WHO Director-General declared that the 2019-nCoV outbreak constitutes a Public Health Emergency of International Concern,and the outbreak is still on-going.Chest CT had been a key component of the diagnostic workup for patients with suspected infection. In this retrospective study, we attempt to summarize and analyze the chest CT features of 2019-nCov infections, and to identify the typical features to improved the diagnostic accuracy of new coronavirus pneumonia (NCP). Methods:Chest CT scans and Clinical data of 21 patients confirmed NCP in our hospital were enrolled.These patients were divided into mild and sever group according to clinical manifestations described by the 6th clinical practice guideline of NCP in China. Main clinical and chest CT features were analyzed and identify. Results: Fever (85.7%) and cough (80.9%) were the two main symptoms of NCP patients.More significantly higher incidence (85.7%) of shortness of breath in the severe cases. Multiple lesions in both lungs and with incidence of GGO(100%),vascular enlargement (76.5%) and cobblestone/reticular pattern(70.6%) were the major feature.The incidence of consolidation, mixed pattern and vascular enlargement features were up to 100% in the severe group, significantly higher than that of patients in mild group. In addition, the incidence of air-bronchogram, dilated bronchi with thickened wall and fibrosis in the severe group was significantly higher than that in the mild group. Conclusions: Fever and cough are the typical clinical features of NCP patients, and chest CT mainly manifested as multiple lesions in both lungs, often accompanied by GGO, vascular enlargement and cobblestone/reticular pattern.Changes in these main CT features can indicate development of the disease.","2020-03-06","wenxiu Wu; zhifeng xu; yabin Jin; aizhen Pan",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030775"],["zhdjtv4j","0ab795fc615df6457551a8e231dce1f268eef9d2","medrxiv","Caution: The clinical characteristics of COVID-19 patients at admission are changing","10.1101/2020.03.03.20030833",null,null,"medrvix","Background With the emergence of 4rd generation transmission, the prevention and treatment of the novel coronavirus disease 2019 (COVID-19) has entered a new period. We aimed to report several changes in the clinical characteristics at admission of patients with COVID-19. Methods Clinical records and laboratory results of patients suffering from COVID-19 were retrospectively reviewed and matched with the admission dates to analyze the changes in characteristics at the onset of illness. Results Of the 89 affected patients, 31 [34.8%] patients were admitted from January 16 to 22, and 58 [65.2%] were admitted from January 23 to 29. Patients were admitted with more systemic symptoms, such as fever (21 [67.7%] of 31), fatigue (13 [41.9%] of 31), and myalgia (7 [22.6%] of 31), before January 23. More patients (10 [32.3%] of 31) admitted before January 23 had a small amount of sputum production compared with a smaller proportion (4 [6.9%] of 58) of the patients admitted after January 23. Other symptoms, such as cough, nausea, diarrhea, and chest tightness, were not significantly different between the two groups. In addition, the group admitted before January 23 had a larger proportion of patients with reduced lymphocyte (13 [54.2%] of 24), CD3 (11 [54.4%] of 21), and CD8 (9 [42.9%] of 21) counts and elevated serum amyloid A (SAA, 18 [75%] of 24). Conclusions The initial symptoms of recently infected patients seem more insidious, indicating that the new coronavirus may gradually evolve into a virus similar to influenza and latent in asymptomatic carriers for a long time.","2020-03-06","Zhaowei Chen; Jijia Hu; Zongwei Zhang; Shan Jiang; Tao Wang; Zhengli Shi; Zhan Zhang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030833"],["1mu1z4xd","5bb89950ec5a06e2b7f69b2a9c4213dda19b1ab0","medrxiv","Prediction of New Coronavirus Infection Based on a Modified SEIR Model","10.1101/2020.03.03.20030858",null,null,"medrvix","BACKGROUND The outbreak of the new coronavirus infection in Wuhan City, Hubei Province in December 2019, poses a huge threat to China and even global public health security. Respiratory droplets and contact transmission are the main routes of transmission of new coronaviruses. Compared with SARS and Ebola viruses, new coronavirus infections are infectious during the incubation period. Traditional SEIR (susceptibility-exposure-infection-Removal) There are some differences in conditions for the prediction of the epidemic trend of new coronavirus infection. The outbreak of the new coronavirus infection coincided with the Spring Festival before and after the Chinese Spring Festival.It is necessary to make appropriate optimization and amendments to the traditional model to meet the actual evolution of the epidemic situation. METHODS The traditional SEIR model assumes that the virus-infected person is not infectious during the incubation period and that the infected person did not take isolation measures during the illness. The transmission of the new coronavirus no longer meets the basic assumptions of the classical kinetic system. Therefore, this article first establishes a modified SEIR model. Predict and analyze the changing trend of the epidemic situation, then estimate the parameters involved in the infection dynamics model, and then use Matlab to simulate the established dynamic equations based on public data and analyze the results. Recommendations for universal prevention and control of infectious diseases. RESULTS The first case of new coronavirus infection was confirmed in Wuhan on December 8, 2019. When Wuhan City took no action, assuming the average daily number of contacts per infected person k = 5, the number of infected persons will reach about 2,384,803 people; If wuhan adopts the measures of sealing the city on January 22, 2020, under the premise of k=2, the number of infected people decreases by 19,773 compared with that on January 23, and there is no significant change in the time when the number of infected people reaches the peak. Under the premise of k = 1, the number of infected persons was reduced by 14,330 compared with the closure on January 23, and the time to reach the peak of the number of infected persons was reduced by 2 days. If Wuhan City is closed for one day, the number of infected persons will increase from 106,145 to 130,626 under the premise of k = 2; the number of infected persons will increase from 74,369 to 92,010 under the premise of k = 1. CONCLUSIONS Comparing the number of confirmed diagnoses actually notified by the department with the number of infected people obtained from the simulation of the model, it can be seen that the city closure measures adopted by the Wuhan Municipal Government on January 23 and the first-level response measures adopted by the country are effective for the epidemic Prevention and control play a vital role. Wearing a mask when going out and avoiding close contact with people can effectively reduce the infection rate.","2020-03-06","Zhou Tang; Xianbin Li; Houqiang Li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030858"],["itp1hnql",null,"medrxiv","Psychological impact of the coronavirus disease 2019 (COVID-19) outbreak on healthcare workers in China","10.1101/2020.03.03.20030874",null,null,"medrvix","Introduction Since the outbreak of coronavirus disease 2019 (COVID-19), more than 3000 (including clinical diagnosis) healthcare workers (HCWs) have been infected with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China. This study is aimed to investigate the risk perception and immediate psychological state of HCWs in the early stage of the COVID-19 epidemic. Methods This study utilized a cross-sectional survey designed on a convenient sample of 4357 HCWs in China. Its data were collected using anonymous structured questionnaires distributed through social software. 6 questions were set to evaluate the participants' risk perception of COVID-19, and a General Health Questionnaire was used to identify the participants' immediate psychological status. Descriptive statistics were used for data analysis. Risk perception and psychological status were compared by demographic characteristics and COVID-19 exposure experiences. Result A total of 4,600 questionnaires were distributed, and 4,357 qualified ones (94.7%) were collected. The main concerns of HCWs are: infection of colleagues (72.5%), infection of family members (63.9%), protective measures (52.3%) and medical violence (48.5%). And 39.1% of the HCWs had psychological distress, especially working in Wuhan, participating in frontline treatments, having been isolated and having family members or colleagues infected. Conclusions The finding indicating that, faced with the COVID-19 epidemic, HCWs, especially in Wuhan, were worried about the risks of infection and protective measures, resulting in psychological distress, so further actions should be taken.","2020-03-06","Yuhong Dai; Guangyuan Hu; Huihua Xiong; Hong Qiu; Xianglin Yuan",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.20030874"],["qeo5dfxg",null,"biorxiv","Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)","10.1101/2020.03.03.962332",null,null,"biorxiv","AbstractA new coronavirus SARS-CoV-2, recently discovered in Wuhan, China, has caused over 74000 infection cases and 2000 deaths. Due to the rapidly growing cases and the unavailability of specific therapy, there is a desperate need for vaccines to combat the epidemic of SARS-CoV-2. In the present study, we performed an in silico approach based on the available virus genome to identify the antigenic B-cell epitopes and human-leukocyte-antigen (HLA) restricted T-cell epitopes. A total of 61 B-cell epitopes were initially identified, 19 of which with higher potential immunogenicity were used for vaccine design. 499 T-cell epitopes were predicted that showed affinity with the 34 most popular HLA alleles in Chinese population. Based on these epitopes, 30 vaccine candidates were designed and inspected against safety risks, including potential toxicity, human homologous, pharmaceutical peptides and bioactive peptides. Majority of vaccine peptides contained both B-cell and T-cell epitopes, which may interact with the most prevalent HLA alleles accounting for ~99% of Chinese population. Docking analysis showed stable hydrogen bonds of epitopes with their corresponding HLA alleles. In conclusion, these putative antigenic peptides may elicit the resistance response to the viral infection. In vitro and in vivo experiments are required to validate the effectiveness of these peptide vaccine.","2020-03-03","Ye Feng; Min Qiu; Shengmei Zou; Yun Li; Kai Luo; Rongchang Chen; Yingqiang Sun; Kui Wang; Xinlei Zhuang; Shanshan Zhang; Shuqing Chen; Fan Mo",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.962332"],["qmg58cgr","8349823092836fe397a59e38615d1491423dbe70","biorxiv","AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2","10.1101/2020.03.03.972133",null,null,"biorxiv","AbstractThe focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at https://github.com/tbwxmu/2019-nCov. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2.","2020-03-08","Bowen Tang; Fengming He; Dongpeng Liu; Meijuan Fang; Zhen Wu; Dong Xu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.972133"],["3ybko13r","0fa18c47543c3ce69a3c9e4d77958c4d3691b1ed","biorxiv","Significance of hydrophobic and charged sequence similarities in sodium-bile acid cotransporter and vitamin D-binding protein macrophage activating factor","10.1101/2020.03.03.975524",null,null,"biorxiv","AbstractSodium-bile acid cotransporter, also denominated sodium-taurocholate cotransporting polypeptide (NTCP) is an integral membrane protein with multiple hydrophobic transmembrane domains. The third extracellular domain of NTCP presents a stretch of nine aminoacids (KGIVISLVL) that is characterized by pronounced hydrophobicity and serves as receptor for a protein, preS1, showing the hydrophobic epta-peptide sequence NPLGFFP. Vitamin D-binding protein macrophage activating factor (DBP-MAF) is a multifunctional protein that is characterized by two hydrophobic regions able to bind fatty acids and vitamin D, respectively. Here we demonstrate that NTCP and DBP-MAF show significant sequence similarities as far as hydrophobic stretches of aminoacids are concerned. Alignment of the sequence of seven aminoacids preceding the 157-KGIVISLVL-165 stretch of NTCP shows four aminoacids that are identical to those of the corresponding sequence of DBP-MAF, and two that are conserved substitutions. In addition, in the sequence of DBP-MAF that is aligned with the sequence YKGIVISLVL of NTCP, there are two contiguous negatively charged aminoacids (ED) and, in the preceding epta-peptide sequence, there are three negatively charged aminoacids (D-ED), whereas in the corresponding sequence of NTCP there are only two (D--D) that are not contiguous. This concentration of negatively charged aminoacids may be involved in binding of protein inserts characterized by high density of positively charges residues. The alternating hydrophobic and electrostatic interactions described in this paper may help elucidating the biological roles of these proteins as far as protein-protein interactions are concerned.","2020-03-05","Ibne Raihan Zunaid; Stefania Pacini; Marco Ruggiero",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.03.975524"],["ti9b1etu","3afd5fba7dc182ddfa769c0d766134b525581005","medrxiv","Transmission and clinical characteristics of coronavirus disease 2019 in 104 outside-Wuhan patients, China","10.1101/2020.03.04.20026005",null,null,"medrvix","Background: Cases with coronavirus disease 2019 (COVID-19) emigrated from Wuhan escalated the risk of spreading in other cities. This report focused on the outside-Wuhan patients to assess the transmission and clinical characteristics of this illness. Methods: Contact investigation was conducted on each patient who admitted to the assigned hospitals in Hunan Province (geographically adjacent to Wuhan) from Jan 22, 2020 to Feb 12, 2020. Demographic, clinical, laboratory and radiological characteristics, medication therapy and outcomes were collected and analyzed. Patients were confirmed by PCR test. Results: Of the 104 patients, 48 (46.15%) were imported cases and 56 (53.85%) were indigenous cases; 93 (89.42%) had a definite contact history with infections. Family clusters were the major body of patients. Transmission along the chain of 3 &ldquo:generations” was observed. Mean age was 43 (rang, 8-84) years (including 3 children) and 49 (47.12%) were male. Most patients had typical symptoms, 5 asymptomatic infections were found and 2 of them infected their relatives. The median incubation period was 6 (rang, 1-32) days, of 8 patients ranged from 18 to 32 days. Just 9 of 16 severe patients required ICU care. Until Feb 12, 2020, 40 (38.46%) discharged and 1 (0.96%) died. For the antiviral treatment, 80 (76.92%) patients received traditional Chinese medicine therapy. Conclusions: Family but not community transmission occupied the main body of infections in the two centers. Asymptomatic transmission demonstrated here warned us that it may bring more risk to the spread of COVID-19. The incubation period of 8 patients exceeded 14 days.","2020-03-06","Chengfeng Qiu; Qian Xiao; Xin Liao; Ziwei Deng; Huiwen Liu; Yuanlu Shu; Dinghui Zhou; Ye Deng; Hongqiang Wang; Xiang Zhao; Jianliang Zhou; Jin Wang; Zhihua Shi; Long Da",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20026005"],["s4d442kx","3deb6393bd43f440a69c84da9ed61986b1781d36","medrxiv","Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses","10.1101/2020.03.04.20029538",null,null,"medrvix","The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.","2020-03-06","Ming Wang; Aisi Fu; Ben Hu; Yongqing Tong; Ran Liu; Jiashuang Gu; Jianghao Liu; Wen Jiang; Gaigai Shen; Wanxu Zhao; Dong Men; Lilei Yu; Zixin Deng; Yan Li; Tiangang Liu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20029538"],["dmud2zf7","3fe2552d3d8e6c9ead2c41e30fbb69e917277f53","medrxiv","Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China","10.1101/2020.03.04.20030395",null,null,"medrvix","Background: Since mid-December 2019, a cluster of pneumonia-like diseases caused by a novel coronavirus, now designated COVID-19 by the WHO, emerged in Wuhan city and rapidly spread throughout China. Here we identify the clinical characteristics of COVID-19 in a cohort of patients in Shanghai. Methods: Cases were confirmed by real-time RT-PCR and were analysed for demographic, clinical, laboratory and radiological features. Results: Of 198 patients, the median duration from disease onset to hospital admission was 4 days. The mean age of the patients was 50.1 years, and 51.0% patients were male. The most common symptom was fever. Less than half of the patients presented with respiratory systems including cough, sputum production, itchy or sore throat, shortness of breath, and chest congestion. 5.6% patients had diarrhoea. On admission, T lymphocytes were decreased in 45.8% patients. Ground glass opacity was the most common radiological finding on chest computed tomography. 9.6% were admitted to the ICU because of the development of organ dysfunction. Compared with patients not treated in ICU, patients treated in the ICU were older, had longer waiting time to admission, fever over 38.5o C, dyspnoea, reduced T lymphocytes, elevated neutrophils and organ failure. Conclusions: In this single centre cohort of COVID-19 patients, the most common symptom was fever, and the most common laboratory abnormality was decreased blood T cell counts. Older age, male, fever over 38.5oC, symptoms of dyspnoea, and underlying comorbidity, were the risk factors most associated with severity of disease. Key words: 2019 novel coronavirus; acute respiratory infection; risk factors for disease severity","2020-03-06","Min Cao; Dandan Zhang; Youhua Wang; Yunfei Lu; Xiangdong Zhu; Ying Li; Honghao Xue; Yunxiao Lin; Min Zhang; Yiguo Sun; Zongguo Yang; Jia Shi; Yi Wang; Chang Zhou; Yidan Dong; Ping Liu; Steven M Dudek; Zhen Xiao; Hongzhou Lu; Longping Peng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20030395"],["na8odvj7","4380f0251c595d6cb551e643198d2dacd3c6746c","medrxiv","Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing","10.1101/2020.03.04.20030916",null,null,"medrvix","Corona Virus Disease 2019 (COVID-19) has spread rapidly to more than 70 countries and regions overseas and over 80000 cases have been infected, resulting in more than three thousand deaths. Rapid diagnosis of patients remains a bottleneck in containing the progress of the epidemic. We used automated chemiluminescent immunoassay to detect serum IgM and IgG antibodies to 2019-nCoV of 736 subjects. COVID-19 patients were becoming reactive(positive) for specific antibodies from 7-12 days after the onset of morbidity. Specific IgM and IgG increased with the progression of the disease. The areas under the ROC curves of IgM and IgG were 0.988 and 1.000, respectively. Specific antibody detection has good sensitivity and specificity. Detection of specific antibodies in patients with fever can be a good distinction between COVID-19 and other diseases, so as to be a complement to nucleic acid diagnosis to early diagnosis of suspected cases.","2020-03-06","Jin Zhang; Jianhua Liu; Na Li; Yong Liu; Rui Ye; Xiaosong Qin; Rui Zheng",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20030916"],["rscduazd","e066fdbb4a9338a57477424cb66fd4d75b2fd66d","medrxiv","Comparison of severe and non-severe COVID-19 pneumonia: review and meta-analysis","10.1101/2020.03.04.20030965",null,null,"medrvix","Objective: To compare the difference between severe and non-severe COVID-19 pneumonia and figure out the potential symptoms lead to severity. Methods: Articles from PubMed, Embase, Cochrane database, and google up-to 24 February 2020 were systematically reviewed. Eighteen Literatures were identified with cases of COVID-19 pneumonia. The extracted data includes clinical symptoms, age, gender, sample size and region et al were systematic reviewed and meta analyzed. Results: 14 eligible studies including 1,424 patients were analyzed. Symptoms like fever (89.2%), cough (67.2%), fatigue (43.6%) were common, dizziness, hemoptysis, abdominal pain and conjunctival congestion/conjunctivitis were rare. Polypnea/dyspnea in severe patients were significantly higher than non-severe (42.7% vs.16.3%, P<0.0001). Fever and diarrhea were higher in severe patients(p=0.0374and0.0267). Further meta-analysis showed incidence of fever(OR1.70,95%CI 1.01-2.87), polypnea/dyspnea(OR3.53, 95%CI 1.95-6.38) and diarrhea(OR1.80,95%CI 1.06-3.03) was higher in severe patients, which meant the severe risk of patients with fever, polypnea/dyspnea, diarrhea were 1.70, 3.53, 1.80 times higher than those with no corresponding symptoms. Conclusions: Fever, cough and fatigue are common symptoms in COVID-19 pneumonia. Compared with non-severe patients, the symptoms as fever, polypnea/dyspnea and diarrhea are potential symptoms lead to severity.","2020-03-09","Weiping Ji; Jing Zhang; Gautam Bishnu; Xudong Du; Xinxin Chen; Hui Xu; Xiaoling Guo; Zhenzhai Cai; Xian Shen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20030965"],["sozs02x0","1a12cc7d49d8521bec5d447ebb413cbcf5aca8f4","medrxiv","Study of the mental health status of medical personnel dealing with new coronavirus pneumonia","10.1101/2020.03.04.20030973",null,null,"medrvix","Objective: This paper studied the relationship between personality traits and mental health conditions of medical personnel to provide a basis and reference for the implementation of targeted education on mental health. Methods: A self-report inventory, the Symptom Checklist-90 (SCL-90), was used to investigate the mental health status of 548 medical personnel dealing with the new coronavirus pneumonia in eight provinces and cities of China. Results: The overall mean SCL-90 score and mean values of factors (somatization, obsessive-compulsive, anxiety, phobic anxiety, and psychoticism) of the medical personnel were significantly higher than in the norm group (p < 0.05), while their average interpersonal sensitivity score was significantly lower (p < 0.01). In addition, personal factors affecting the mental health status of medical personnel were identified. ( all p < 0.05). Conclusion: The overall mental health status of medical personnel responding to new coronavirus pneumonia is generally higher than that of the norm group in China. The results of this study should contribute to measures to alleviate the psychological pressures on medical personnel dealing with the new coronavirus epidemic in China","2020-03-06","ning sun; jun xing; jun xu; ling shu geng; qian yu li",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20030973"],["dop8knqn",null,"medrxiv","Case fatality risk of novel coronavirus diseases 2019 in China","10.1101/2020.03.04.20031005",null,null,"medrvix","Objective The outbreak of novel coronavirus disease 2019 (COVID-19) imposed a substantial health burden in mainland China and remains a global epidemic threat. Our objectives are to assess the case fatality risk (CFR) among COVID-19 patients detected in mainland China, stratified by clinical category and age group. Method We collected individual information on laboratory-confirmed COVID-19 cases from publicly available official sources from December 29, 2019 to February 23, 2020. We explored the risk factors associated with mortality. We used methods accounting for right-censoring and survival analyses to estimate the CFR among detected cases. Results Of 12,863 cases reported outside Hubei, we obtained individual records for 9,651 cases, including 62 deaths and 1,449 discharged cases. The deceased were significantly older than discharged cases (median age: 77 vs 39 years, p<0.001). 58% (36/62) were male. Older age (OR 1.18 per year; 95%CI: 1.14 to 1.22), being male (OR 2.02; 95%CI: 1.02 to 4.03), and being treated in less developed economic regions (e.g., West and Northeast vs. East, OR 3.93; 95%CI: 1.74 to 8.85) were mortality risk factors. The estimated CFR was 0.89-1.24% among all cases. The fatality risk among critical patients was 2-fold higher than that among severe and critical patients, and 24-fold higher than that among moderate, severe and critical patients. Conclusions Our estimates of CFR based on laboratory-confirmed cases ascertained outside of Hubei suggest that COVID-19 is not as severe as severe acute respiratory syndrome and Middle East respiratory syndrome, but more similar to the mortality risk of 2009 H1N1 influenza pandemic in hospitalized patients. The fatality risk of COVID-19 is higher in males and increases with age. Our study improves the severity assessment of the ongoing epidemic and can inform the COVID-19 outbreak response in China and beyond.","2020-03-06","Xiaowei Deng; Juan Yang; Wei Wang; Xiling Wang; Jiaxin Zhou; Zhiyuan Chen; Jing Li; Yinzi Chen; Han Yan; Juanjuan Zhang; Yongli Zhang; Yan Wang; Qi Qiu; Hui Gong; Xianglin Wei; Lili Wang; Kaiyuan Sun; Peng Wu; Marco Ajelli; Benjamin J. Cowling; Cecile Viboud; Hongjie Yu",null,null,null,"False","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031005"],["vg1vvm6f","dd14a222d0d516266debe1c487e32ad13c105291","medrxiv","Clinical characteristics of 101 non-surviving hospitalized patients with COVID-19: A single center, retrospective study","10.1101/2020.03.04.20031039",null,null,"medrvix","Abstract Rationale: COVID-19 have deprived many lives in Wuhan, China and caused global concerns. Few studies focused on clinical characteristics non-survivors hospitalized with COVID-19. Objectives: We aimed to clarify the clinical characteristics of died patients with COVID-19 and tracked the causes of the rapid disease progression and death. Methods: 101 non-surviving patients with confirmed COVID-19 were included in this retrospective study. Clinical data were collected and compared between non-survivors who died within 3 days and after 3 days of admission. Multivariable regression analysis was used to analyze risk factors associated with rapid disease progression and death. Measurements and Main Results: Among included patients, median age was 71 years (IQR, 59-80), 60 (59.41%) were men. 82 (81.19%) had one or more comorbidities including hypertension (59 [58.42%]), diabetes (22 [21.78%]) etc. 100 (99.01%) suffered respiratory failure, 53 (52.48%) developed acute cardiac injury, acute kidney and liver injury occurred in 23 (22.77%) and 18 (17.82%) patients, respectively. Multivariable regression analysis showed that elevated high sensitivity troponin I (OR 2.68, 95%CI 1.31-5.49, P=0.007), neutrophils (OR 1.14, 95%CI 1.01-1.28, P=0.033) and depressed oxygen saturation (OR 0.94, 95%CI 0.89-0.99, P=0.027) on admission were associated with rapid death of patients with COVID-19. Conclusions: Older patients (>70 years) with comorbidities had a steeply increased risk of death with COVID-19. Respiratory failure, acute cardiac and kidney injury played a crucial role in the death of patients. Elevated high sensitivity troponin, neutrophils and depressed oxygen saturation predicted the rapid death of patients.","2020-03-06","Qiao Shi; Kailiang Zhao; Jia Yu; Jiarui Feng; Kaiping Zhao; Xiaoyi Zhang; Xiaoyan Chen; Peng Hu; Yupu Hong; Man Li; Fang Liu; Chen Chen; Weixing Wang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031039"],["4ep7nf3q","f47af9fe364fd9bac5e061142208d801667c5aca","medrxiv","How to differentiate COVID-19 pneumonia from heart failure with computed tomography at initial medical contact during epidemic period","10.1101/2020.03.04.20031047",null,null,"medrvix","OBJECTIVES To compare chest CT findings in heart failure with those of Corona Virus Disease 2019 (COVID-19) pneumonia. BACKGROUND During epidemic period, chest computed tomography (CT) has been highly recommended for screening patients with suspected COVID-19. However, the comparison of CT imaging between heart failure and COVID-19 pneumonia has not been fully elucidated. METHODS Patients with heart failure (n=12), COVID-19 pneumonia (n=12) and one patient with both diseases were retrospectively enrolled. Clinical information and imaging of chest CT were collected and analyzed. RESULTS There was no difference of ground glass opacity (GGO), consolidation, crazy paving pattern, lobes affected and septal thickening between heart failure and COVID-19 pneumonia. However, less rounded morphology (8.3% vs. 67%, p=0.003), more peribronchovascular thickening (75% vs. 33%, p=0.041) and fissural thickening (33% vs. 0%, p=0.028), less peripheral distribution (33% vs. 92%, p=0.003) were found in heart failure group than that in COVID-19 group. Importantly, there were also more patients with upper pulmonary vein enlargement (75% vs. 8.3%, p=0.001), subpleural effusion and cardiac enlargement in heart failure group than that in COVID-19 group (50% vs. 0%, p=0.005, separately). Besides, more fibrous lesions were found in COVID-19 group although there was no statistical difference (25% vs. 0%, P=0.064) CONCLUSIONS Although there are some overlaps of CT imaging between heart failure and COVID-19, CT is still a useful tool in differentiating COVID-19 pneumonia.","2020-03-06","Zhaowei Zhu; Jianjun Tang; Xiangping Chai; Zhenfei Fang; Qiming Liu; Xinqun Hu; Danyan Xu; Jia He; Liang Tang; Shi Tai; Yuzhi Wu; Shenghua Zhou",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031047"],["mrsya6wz","29b3eaeacecf2bfc95b9a493dbb1ba4b2323dd46","medrxiv","Adjusted age-specific case fatality ratio during the COVID-19 epidemic in Hubei, China, January and February 2020","10.1101/2020.03.04.20031104",null,null,"medrvix","The coronavirus disease 2019 (COVID-19) epidemic that originated in Wuhan, China has spread to more than 60 countries. We estimated the age-specific case fatality ratio (CFR) by fitting a transmission model to data from China, accounting for underreporting of cases and the time delay to death. Overall CFR among all infections was 1.6% (1.4-1.8%) and increased considerably for the elderly, highlighting the expected burden for populations with further expansion of the COVID-19 epidemic around the globe.","2020-03-06","Julien Riou; Anthony Hauser; Michel J Counotte; Christian L Althaus",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031104"],["f4hj35dr","734779fad93249a2f6fede6afd10eeff7b37919b","medrxiv","Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period","10.1101/2020.03.04.20031112",null,null,"medrvix","There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.","2020-03-06","Stephen M Kissler; Christine Tedijanto; Edward Goldstein; Yonatan H. Grad; Marc Lipsitch",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031112"],["bpnqz1cn","ee902c8e20385bed8b5fccb85d4e53996069cc0c","medrxiv","Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection","10.1101/2020.03.04.20031120",null,null,"medrvix","BACKGROUND The outbreak of a novel coronavirus (SARS-CoV-2, previously provisionally named 2019 novel coronavirus or 2019-nCoV) since December 2019 in Wuhan, China, has become an emergency of major international concern. Apart from the respiratory system, it is unclear whether SARS-CoV-2 can also directly infect other tissues such as the kidney or induce acute renal failure. METHODS We conducted a retrospective analysis of estimated glomerular filtration rate (eGFR) along with other clinical parameters from 85 patients with laboratory-confirmed COVID-19 admitted to a hospital in Wuhan from January 17, 2020 to March 3, 2020. Kidney tissues from six patients with postmortem examinations in the other hospital were analyzed by Hematoxylin and Eosin (H&E) and in situ expression of viral nucleocaspid protein (NP) antigen, immune cell markers (CD8, CD68 and CD56) and the complement C5b-9 was detected by immunohistochemistry. RESULTS 27.06% (23/85) patients exhibited acute renal failure (ARF). The eldery patients and cases with comorbidities such as hypertension and heart failure more easily developed ARF (65.22% vs 24.19%, p< 0.001; 69.57% vs 11.29%, p< 0.001, respectively). H&E staining demonstrated kidney tissues from postmortems have severe acute tubular necrosis and lymphocyte infiltration. Immunohistochemistry showed that SARS-CoV-2 NP antigen was accumulated in kidney tubules. Viral infection not only induces CD68+ macrophages infiltrated into tubulointerstitium, but also enhances complement C5b-9 deposition on tubules. CONCLUSIONS SARS-CoV-2 induces ARF in COVID-19 patients. Viruses directly infect human kidney tubules to induce acute tubular damage. The viruses not only have direct cytotoxicity, but also initiate CD68+ macrophage together with complement C5b-9 deposition to mediate tubular pathogenesis.","2020-03-06","Bo Diao; Chenhui Wang; Rongshuai Wang; Zeqing Feng; Yingjun Tan; Huiming Wang; Changsong Wang; Liang Liu; Ying Liu; Yueping Liu; Gang Wang; Zilin Yuan; Liang Ren; Yuzhang Wu; Yongwen Chen",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031120"],["utlponna","f40eaec892778166e4f8f8eeb576893e82a591be","medrxiv","The impact of social distancing and epicenter lockdown on the COVID-19 epidemic in mainland China: A data-driven SEIQR model study","10.1101/2020.03.04.20031187",null,null,"medrvix","The outbreak of coronavirus disease 2019 (COVID-19) which originated in Wuhan, China, constitutes a public health emergency of international concern with a very high risk of spread and impact at the global level. We developed data-driven susceptible-exposed-infectious-quarantine-recovered (SEIQR) models to simulate the epidemic with the interventions of social distancing and epicenter lockdown. Population migration data combined with officially reported data were used to estimate model parameters, and then calculated the daily exported infected individuals by estimating the daily infected ratio and daily susceptible population size. As of Jan 01, 2020, the estimated initial number of latently infected individuals was 380.1 (95%-CI: 379.8~381.0). With 30 days of substantial social distancing, the reproductive number in Wuhan and Hubei was reduced from 2.2 (95%-CI: 1.4~3.9) to 1.58 (95%-CI: 1.34~2.07), and in other provinces from 2.56 (95%-CI: 2.43~2.63) to 1.65 (95%-CI: 1.56~1.76). We found that earlier intervention of social distancing could significantly limit the epidemic in mainland China. The number of infections could be reduced up to 98.9%, and the number of deaths could be reduced by up to 99.3% as of Feb 23, 2020. However, earlier epicenter lockdown would partially neutralize this favorable effect. Because it would cause in situ deteriorating, which overwhelms the improvement out of the epicenter. To minimize the epidemic size and death, stepwise implementation of social distancing in the epicenter city first, then in the province, and later the whole nation without the epicenter lockdown would be practical and cost-effective.","2020-03-06","Yuzhen Zhang; Bin Jiang; Jiamin Yuan; Yanyun Tao",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031187"],["rbsv58fo","bfed3ce6a6df2d442d96d0cab07d4bc1f2347255","medrxiv","Seattle Flu Study - Swab and Send: Study Protocol for At-Home Surveillance Methods to Estimate the Burden of Respiratory Pathogens on a City-Wide Scale","10.1101/2020.03.04.20031211",null,null,"medrvix","Introduction. While seasonal influenza and other respiratory pathogens cause significant morbidity and mortality each year, the community-based burden of these infections remains incompletely understood. Understanding the prevalence, epidemiology, and transmission dynamics of respiratory pathogen infections among community-dwelling individuals is essential during pandemic and epidemic settings and for developing pandemic-preparedness infrastructure. Methods and Analysis. We present the protocol for a novel, city-wide home-based cross-sectional study in the Seattle Metropolitan area, utilizing rapid delivery systems for self-collection of a nasal swab and return to the laboratory for respiratory pathogen testing. All participation takes place electronically, including recruitment, consent, and data collection. Within 48 hours of participants self-reporting respiratory symptoms, a nasal swab kit is delivered to the household via a courier service. Demographic and illness characteristics are collected at the time of sample collection and recovery and behavioral information collected one week later. Specimens are tested in the laboratory for multiple respiratory pathogens, and results are available on a public website for participants. Ethics and Dissemination. The study was approved by the University of Washington Institutional Review Board (Protocol #00006181). Results will be disseminated through peer-reviewed publications, talks at conferences, and on the Study Website (www.seattleflu.org).","2020-03-07","Ashley E Kim; Elisabeth Brandstetter; Chelsey Graham; Jessica Heimonen; Audrey Osterbind; Denise J McCulloch; Peter D Han; Lea M Starita; Deborah A Nickerson; Margaret M Van de Loo; Jennifer Mooney; Mark J Rieder; Misja Ilcisin; Kairsten A Fay; Jover Lee; Thomas R Sibley; Trevor Bedford; Janet A Englund; Michael Boeckh; Helen Y Chu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031211"],["myh73pe6","b696d208705fcb1925693c5f0d118733bb557ea6","medrxiv","Exploring diseases/traits and blood proteins causally related to expression of ACE2, the putative receptor of 2019-nCov: A Mendelian Randomization analysis","10.1101/2020.03.04.20031237",null,null,"medrvix","The novel coronavirus 2019-nCoV has caused major outbreaks in many parts of the world. A better understanding of the pathophysiology of COVID-19 is urgently needed. Clinically, it is important to identify who may be susceptible to infection and identify treatments for the disease. There is good evidence that ACE2 is a receptor for 2019-nCoV, and studies also suggested that high expression of ACE2 may increase susceptibility to infection. Here we conducted a phenome-wide Mendelian randomization (MR) study to prioritize diseases/traits and blood proteins that may be causally linked to ACE2 expression in the lung. Expression data was based on GTEx. We also explored drug candidates whose targets overlapped with the top-ranked proteins in MR analysis, as these drugs could potentially alter ACE2 expression and may be clinically relevant. Notably, MR is much less vulnerable to confounding and reverse causality compared to observational studies. The most consistent finding was a tentative causal association between diabetes-related traits and increased ACE2 expression. Based on one of the largest GWAS on type II diabetes (T2DM) to date (N=898,130), we found that T2DM is causally linked to raised ACE2 expression (beta=0.1835, 95% CI 0.0853-0.2817; p=2.49E-4; GSMR method). Significant associations (at nominal level; p<0.05) was also observed across multiple datasets, with different analytic methods, and for both type I and II diabetes. Other diseases/traits having nominal significant associations with increased ACE2 included inflammatory bowel disease, (ER+) breast and lung cancers, asthma, smoking and elevated ALT, among others. We also uncovered a number of plasma/serum proteins potentially linked to altered ACE2 expression, and the top enriched pathways included cytokine-cytokine-receptor interaction, VEGF signaling, JAK-STAT signaling etc. We also explored drugs that target some of the top-ranked proteins in the MR analysis. In conclusion, the current MR analysis reveals diseases/traits and blood proteins that may causally affect ACE2 expression, which in turn may influence susceptibility to the infection. The proteome-wide MR analysis may shed light on the molecular mechanisms underlying ACE2 expression, and may help guide drug repositioning in the future. Nevertheless, we stress that further studies are required to verify our findings due to various limitations and the exploratory nature of some analyses.","2020-03-08","Shitao Rao; Alexandria Lau; Hon-Cheong So",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031237"],["ufcvecwo","374a5295716649a80dca87097ad9ac902d344db6","medrxiv","Outcome reporting from protocols of clinical trials of Coronavirus Disease 2019 (COVID-19): a review","10.1101/2020.03.04.20031401",null,null,"medrvix","Objectives: To examine heterogeneity of outcomes in protocols of clinical trials of Coronavirus Disease 2019 (COVID-19) and to identify outcomes for prioritization in developing a core outcome set (COS) in this field. Design: This study is a review. Data sources: Databases of ICMJE-accepted clinical trial registry platform were searched on February 14, 2020. Eligibility Criteria: Randomized controlled trials (RCTs) and non-RCTs of COVID-19 were considered.Conditions of patients include common type, severe type or critical type. Interventions include traditional Chinese medicine (TCM) and Western medicine. We excluded trials that for discharged patients, psychological intervention and complications of COVID-19. Data extraction and synthesis: The general information and outcomes, outcome measurement instruments and measurement times were extracted. The results were analysed by descriptive analysis. Results: 19 registry platforms were searched. A total of 97 protocols were included from 160 protocols. For protocols of TCM clinical trials, 76 outcomes from 16 outcome domains were reported, and almost half (34/76, 44.74%) of outcomes were reported only once; the most frequently reported outcome was time of SARS-CoV-2 RNA turns to negative. 27 (27/76, 35.53%) outcomes were provided one or more outcome measurement instruments. 10 outcomes were provided one or more measurement time frame. For protocols of western medicine clinical trials, 126 outcomes from 17 outcome domains were reported; almost half (62/126, 49.21%) of outcomes were reported only once; the most frequently reported outcome was proportion of patients with negative SARS-CoV-2. 27 outcomes were provided one or more outcome measurement instruments. 40 (40/126, 31.75%) outcomes were provided one or more measurement time frame. Conclusion: Outcome reporting in protocols of clinical trials of COVID-19 is inconsistent. Thus, developing a core outcome set is necessary. Keywords: Outcomes; clinical trials, COVID-19; review.","2020-03-08","Ruijin Qiu; Xuxu Wei; Mengzhu Zhao; Changming Zhong; Chen Zhao; Jiayuan Hu; Min Li; Ya Huang; Songjie Han; Tianmai He; Jing Chen; Hongcai Shang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.20031401"],["4ihv80au","7bdceb3b947ec934199d0c97fb9ec5a0e2bac776","biorxiv","Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses","10.1101/2020.03.04.975995",null,null,"biorxiv","AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.","2020-03-04","Yonghua Wu",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.975995"],["wynyrumi","6726b273f345838661eaa39e8adb5df4e1815a9a","biorxiv","In silico study of the spike protein from SARS-CoV-2 interaction with ACE2: similarity with SARS-CoV, hot-spot analysis and effect of the receptor polymorphism","10.1101/2020.03.04.976027",null,null,"biorxiv","AbstractThe spread of the COVID-19 caused by the SARS-CoV-2 outbreak has been growing since its first identification in December 2019. The publishing of the first SARS-CoV-2 genome made a valuable source of data to study the details about its phylogeny, evolution, and interaction with the host. Protein-protein binding assays have confirmed that Angiotensin-converting enzyme 2 (ACE2) is more likely to be the cell receptor via which the virus invades the host cell. In the present work, we provide an insight into the interaction of the viral spike Receptor Binding Domain (RBD) from different coronavirus isolates with host ACE2 protein. We used homology-based protein-protein docking, binding energy estimation, and decomposition, aiming to predict both qualitative and quantitative aspects of the interaction. Using in silico structural modelling, we brought additional evidence that the interface segment of the spike protein RBD might be acquired by SARS-CoV-2 via a complex evolutionary process rather than mutation accumulation. We also highlighted the relevance of Q493 and P499 amino acid residues of SARS-CoV-2 RBD for binding to hACE2 and maintaining the stability of the interface. Finally, we studied the impact of eight different variants located at the interaction surface of ACE2, on the complex formation with SARS-CoV-2 RBD. We found that none of them is likely to disrupt the interaction with the viral RBD of SARS-CoV-2.","2020-03-07","Houcemeddine Othman; Zied Bouslama; Jean-Tristan Brandenburg; Jorge da Rocha; Yosr Hamdi; Kais Ghedira; Najet-Srairi Abid; Scott Hazelhurst",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.976027"],["erdity4m","5cd48b8ab4ef66ca582df97a293d22a5088fa568","biorxiv","Cryo-electron microscopy structure of the SADS-CoV spike glycoprotein provides insights into an evolution of unique coronavirus spike proteins","10.1101/2020.03.04.976258",null,null,"biorxiv","AbstractThe current outbreak of Coronavirus Disease 2019 (COVID-19) by a novel betacoronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has aroused great public health concern. Coronavirus has a history of causing epidemics in human and animals. In 2017 an outbreak in piglets by a novel coronavirus was emerged designated as swine acute diarrhea syndrome coronavirus (SADS-CoV) which is originated from the same genus of horseshoe bats (Rhinolophus) as Severe Acute Respiratory Syndrome CoV (SARS-CoV) having a broad species tropism. In addition to human cells, it can also infect cell lines from diverse species. Coronavirus host range is determined by its spike glycoprotein (S). Given the importance of S protein in viral entry to cells and host immune responses, here we report the cryo-EM structure of the SADS-CoV S in the prefusion conformation at a resolution of 3.55 Å. Our study reveals that SADS-CoV S structure takes an intra-subunit quaternary packing mode where the NTD and CTD from the same subunit pack together by facing each other. The comparison of NTD and CTD with that of the other four genera suggests the evolutionary process of the SADS-CoV S. Moreover, SADS-CoV S has several characteristic structural features, such as more compact architecture of S trimer, and masking of epitopes by glycan shielding, which may facilitate viral immune evasion. These data provide new insights into the evolutionary relationships of SADS-CoV S and would extend our understanding of structural and functional diversity, which will facilitate to vaccine development.","2020-03-07","Hongxin Guan; Youwang Wang; Abdullah F.U.H. Saeed; Jinyu Li; Syed Sajid Jan; Vanja Perčulija; Yu Li; Ping Zhu; Songying Ouyang",null,null,null,"True","biorxiv_medrxiv","https://doi.org/10.1101/2020.03.04.976258"]],"plotOptions":{"displayType":"table","customPlotOptions":{},"pivotColumns":null,"pivotAggregation":null,"xColumns":null,"yColumns":null},"columnCustomDisplayInfos":{},"aggType":"","isJsonSchema":true,"removedWidgets":[],"aggSchema":[],"schema":[{"name":"cord_uid","type":"\"string\"","metadata":"{}"},{"name":"sha","type":"\"string\"","metadata":"{}"},{"name":"source_x","type":"\"string\"","metadata":"{}"},{"name":"title","type":"\"string\"","metadata":"{}"},{"name":"doi","type":"\"string\"","metadata":"{}"},{"name":"pmcid","type":"\"string\"","metadata":"{}"},{"name":"pubmed_id","type":"\"string\"","metadata":"{}"},{"name":"license","type":"\"string\"","metadata":"{}"},{"name":"abstract","type":"\"string\"","metadata":"{}"},{"name":"publish_time","type":"\"string\"","metadata":"{}"},{"name":"authors","type":"\"string\"","metadata":"{}"},{"name":"journal","type":"\"string\"","metadata":"{}"},{"name":"Microsoft Academic Paper ID","type":"\"string\"","metadata":"{}"},{"name":"WHO #Covidence","type":"\"string\"","metadata":"{}"},{"name":"has_full_text","type":"\"string\"","metadata":"{}"},{"name":"full_text_file","type":"\"string\"","metadata":"{}"},{"name":"url","type":"\"string\"","metadata":"{}"}],"aggError":"","aggData":[],"addedWidgets":{},"metadata":{},"dbfsResultPath":null,"type":"table","aggOverflow":false,"aggSeriesLimitReached":false,"arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .table-result-container {\n    max-height: 300px;\n    overflow: auto;\n  }\n  table, th, td {\n    border: 1px solid black;\n    border-collapse: collapse;\n  }\n  th, td {\n    padding: 5px;\n  }\n  th {\n    text-align: left;\n  }\n</style><div class='table-result-container'><table class='table-result'><thead style='background-color: white'><tr><th>cord_uid</th><th>sha</th><th>source_x</th><th>title</th><th>doi</th><th>pmcid</th><th>pubmed_id</th><th>license</th><th>abstract</th><th>publish_time</th><th>authors</th><th>journal</th><th>Microsoft Academic Paper ID</th><th>WHO #Covidence</th><th>has_full_text</th><th>full_text_file</th><th>url</th></tr></thead><tbody><tr><td>vho70jcx</td><td>f056da9c64fbf00a4645ae326e8a4339d015d155</td><td>biorxiv</td><td>SIANN: Strain Identification by Alignment to Near Neighbors</td><td>10.1101/001727</td><td>null</td><td>null</td><td>biorxiv</td><td>Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.</td><td>2014-01-10</td><td>Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/001727</td></tr><tr><td>i9tbix2v</td><td>daf32e013d325a6feb80e83d15aabc64a48fae33</td><td>biorxiv</td><td>Spatial epidemiology of networked metapopulation: An overview</td><td>10.1101/003889</td><td>null</td><td>null</td><td>biorxiv</td><td>An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.</td><td>2014-06-04</td><td>Lin WANG; Xiang Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/003889</td></tr><tr><td>62gfisc6</td><td>f33c6d94b0efaa198f8f3f20e644625fa3fe10d2</td><td>biorxiv</td><td>Sequencing of the human IG light chain loci from a hydatidiform mole BAC library reveals locus-specific signatures of genetic diversity</td><td>10.1101/006866</td><td>null</td><td>null</td><td>biorxiv</td><td>Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.</td><td>2014-07-03</td><td>Corey T Watson; Karyn Meltz Steinberg; Tina A Graves-Lindsay; Rene L Warren; Maika Malig; Jacqueline E Schein; Richard K Wilson; Rob Holt; Evan Eichler; Felix Breden</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/006866</td></tr><tr><td>058r9486</td><td>4da8a87e614373d56070ed272487451266dce919</td><td>biorxiv</td><td>Bayesian mixture analysis for metagenomic community profiling.</td><td>10.1101/007476</td><td>null</td><td>null</td><td>biorxiv</td><td>Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix</td><td>2014-07-25</td><td>Sofia Morfopoulou; Vincent Plagnol</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/007476</td></tr><tr><td>wich35l7</td><td>eccef80cfbe078235df22398f195d5db462d8000</td><td>biorxiv</td><td>Mapping a viral phylogeny onto outbreak trees to improve host transmission inference</td><td>10.1101/010389</td><td>null</td><td>null</td><td>biorxiv</td><td>Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.</td><td>2014-11-11</td><td>Stephen P Velsko; Jonathan E Allen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/010389</td></tr><tr><td>z3tgnzth</td><td>c41fdb2efd6d61384a92a84cbba3f8233629a41b</td><td>biorxiv</td><td>The infant airway microbiome in health and disease impacts later asthma development</td><td>10.1101/012070</td><td>null</td><td>null</td><td>biorxiv</td><td>The nasopharynx (NP) is a reservoir for microbes associated with acute respiratory illnesses (ARI). The development of asthma is initiated during infancy, driven by airway inflammation associated with infections. Here, we report viral and bacterial community profiling of NP aspirates across a birth cohort, capturing all lower respiratory illnesses during their first year. Most infants were initially colonized with Staphylococcus or Corynebacterium before stable colonization with Alloiococcus or Moraxella, with transient incursions of Streptococcus, Moraxella or Haemophilus marking virus-associated ARIs. Our data identify the NP microbiome as a determinant for infection spread to the lower airways, severity of accompanying inflammatory symptoms, and risk for future asthma development. Early asymptomatic colonization with Streptococcus was a strong asthma predictor, and antibiotic usage disrupted asymptomatic colonization patterns.</td><td>2014-12-02</td><td>Shu Mei Teo; Danny Mok; Kym Pham; Merci Kusel; Michael Serralha; Niamh Troy; Barbara J Holt; Belinda J Hales; Michael L Walker; Elysia Hollams; Yury H Bochkov; Kristine Grindle; Sebastian L Johnston; James E Gern; Peter D Sly; Patrick G Holt; Kathryn E Holt; Michael Inouye</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/012070</td></tr><tr><td>1xxrnpg3</td><td>1dd898b5ca1ae70ec0e3cad89fc87a165002a99e</td><td>biorxiv</td><td>Using heterogeneity in the population structure of U.S. swine farms to compare transmission models for porcine epidemic diarrhoea</td><td>10.1101/017178</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTIn 2013, U.S. swine producers were confronted with the disruptive emergence of porcine epidemic diarrhoea (PED). Movement of animals among farms is hypothesised to have played a role in the spread of PED among farms. Via this or other mechanisms, the rate of spread may also depend on the geographic density of farms and climate. To evaluate such effects on a large scale, we analyse state-level counts of outbreaks with variables describing the distribution of farm sizes and types, aggregate flows of animals among farms, and an index of climate. Our first main finding is that it is possible for a correlation analysis to be sensitive to transmission model parameters. This finding is based on a global sensitivity analysis of correlations on simulated data that included a biased and noisy observation model based on the available PED data. Our second main finding is that flows are significantly associated with the reports of PED outbreaks. This finding is based on correlations of pairwise relationships and regression modeling of total and weekly outbreak counts. These findings illustrate how variation in population structure may be employed along with observational data to improve understanding of disease spread.</td><td>2015-03-27</td><td>Eamon B. O’Dea; Harry Snelson; Shweta Bansal</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/017178</td></tr><tr><td>8ilzm51q</td><td>33565294e6bc67fb7ee14dcae6cfdb08148f4ea5</td><td>biorxiv</td><td>Big city, small world: Density, contact rates, and transmission of dengue across Pakistan.</td><td>10.1101/018481</td><td>null</td><td>null</td><td>biorxiv</td><td>Macroscopic descriptions of populations commonly assume that encounters between individuals are well mixed; i.e., each individual has an equal chance of coming into contact with any other individual. Relaxing this assumption can be challenging though, due to the difficulty of acquiring detailed knowledge about the non-random nature of encounters. Here, we fitted a mathematical model of dengue virus transmission to spatial time series data from Pakistan and compared maximum-likelihood estimates of “mixing parameters” when disaggregating data across an urban-rural gradient. We show that dynamics across this gradient are subject not only to differing transmission intensities but also to differing strengths of nonlinearity due to differences in mixing. We furthermore show that neglecting spatial variation in mixing can lead to substantial underestimates of the level of effort needed to control a pathogen with vaccines or other control efforts. We complement this analysis with relevant contemporary environmental drivers of dengue.</td><td>2015-04-27</td><td>Moritz U. G. Kraemer; T. Alex Perkins; Derek A.T. Cummings; Rubeena Zakar; Simon I. Hay; David L. Smith; Robert C. Reiner</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/018481</td></tr><tr><td>wafvnbdu</td><td>3461d71f6890f7e5ba53bf168be3945cdb16d901</td><td>biorxiv</td><td>MERS-CoV recombination: implications about the reservoir and potential for adaptation</td><td>10.1101/020834</td><td>null</td><td>null</td><td>biorxiv</td><td>Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology.</td><td>2015-06-12</td><td>Gytis Dudas; Andrew Rambaut</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/020834</td></tr><tr><td>4xocqn6o</td><td>1f9d3f9a1a0e8db6a086e0a2b5ba50cf9f235dae</td><td>biorxiv</td><td>On the causes of evolutionary transition:transversion bias</td><td>10.1101/027722</td><td>null</td><td>null</td><td>biorxiv</td><td>A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the \"conservative transitions\" hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.</td><td>2015-09-28</td><td>Arlin Stoltzfus; Ryan W. Norris</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/027722</td></tr><tr><td>q5iylws9</td><td>01e3b313e78a352593be2ff64927192af66619b5</td><td>biorxiv</td><td>Viruses are a dominant driver of protein adaptation in mammals</td><td>10.1101/029397</td><td>null</td><td>null</td><td>biorxiv</td><td>Viruses interact with hundreds to thousands of proteins in mammals, yet adaptation against viruses has only been studied in a few proteins specialized in antiviral defense. Whether adaptation to viruses typically involves only specialized antiviral proteins or affects a broad array of proteins is unknown. Here, we analyze adaptation in ~1,300 virus-interacting proteins manually curated from a set of 9,900 proteins conserved across mammals. We show that viruses (i) use the more evolutionarily constrained proteins from the cellular functions they hijack and that (ii) despite this high constraint, virus-interacting proteins account for a high proportion of all protein adaptation in humans and other mammals. Adaptation is elevated in virus-interacting proteins across all functional categories, including both immune and non-immune functions. Our results demonstrate that viruses are one of the most dominant drivers of evolutionary change across mammalian and human proteomes.</td><td>2015-10-18</td><td>David Enard; Le Cai; Carina Gwenapp; Dmitri A Petrov</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/029397</td></tr><tr><td>nljskxut</td><td>null</td><td>biorxiv</td><td>What's in my pot? Real-time species identification on the MinION</td><td>10.1101/030742</td><td>null</td><td>null</td><td>biorxiv</td><td>Whole genome sequencing on next-generation instruments provides an unbiased way to identify the organisms present in complex metagenomic samples. However, the time-to-result can be protracted because of fixed-time sequencing runs and cumbersome bioinformatics workflows. This limits the utility of the approach in settings where rapid species identification is crucial, such as in the quality control of food-chain components, or in during an outbreak of an infectious disease. Here we present What′s in my Pot? (WIMP), a laboratory and analysis workflow in which, starting with an unprocessed sample, sequence data is generated and bacteria, viruses and fungi present in the sample are classified to subspecies and strain level in a quantitative manner, without prior knowledge of the sample composition, in approximately 3.5 hours. This workflow relies on the combination of Oxford Nanopore Technologies′ MinION ™ sensing device with a real-time species identification bioinformatics application.</td><td>2015-11-06</td><td>Sissel Juul; Fernando Izquierdo; Adam Hurst; Xiaoguang Dai; Amber Wright; Eugene Kulesha; Roger Pettett; Daniel J Turner</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/030742</td></tr><tr><td>lfm6erzy</td><td>3b73c8abb7528c958cec0bae1890828af007b599</td><td>biorxiv</td><td>How domain growth is implemented determines the long term behaviour of a cell population through its effect on spatial correlations</td><td>10.1101/041509</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractDomain growth plays an important role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations.</td><td>2016-02-26</td><td>Robert J. H. Ross; R. E. Baker; C. A. Yates</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/041509</td></tr><tr><td>lcpp5fim</td><td>8cfb8dfc2f8b2c6e1d60ff48da02de6670694869</td><td>biorxiv</td><td>Productive infection of field strains of avian coronavirus infectious bronchitis virus in chicken peripheral blood-derived monocyte</td><td>10.1101/041558</td><td>null</td><td>null</td><td>biorxiv</td><td>The avian coronavirus infectious bronchitis virus (IBV) typically infects the respiratory tract of chickens, but can also spread to other organs. However, the mechanisms of virus dissemination are presently unclear. We show that peripheral blood-derived monocytes/macrophages from chickens (chPBMC) are productively infected by clinical strains of IBV, accompanied by induction of apoptosis. Our data suggest that chPBMCs play a role in the dissemination of IBV, and may be important for viral pathogenesis.</td><td>2016-02-26</td><td>Yueting Zhang; Gary Whittaker</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/041558</td></tr><tr><td>gjxumrmm</td><td>6499f5c865c3db3bdfc1e969ea940a80d547a5bb</td><td>biorxiv</td><td>Zika virus outbreak in the Americas: Is Aedes albopictus an overlooked culprit?</td><td>10.1101/044594</td><td>null</td><td>null</td><td>biorxiv</td><td>Summary / AbstractCodon usage patterns of viruses reflect a series of evolutionary changes that enable viruses to shape their survival rates and fitness toward the external environment and, most importantly, their hosts. In the present study, we employed multiple codon usage analysis indices to determine genotype specific codon usage patterns of Zika virus (ZIKV) strains from the current outbreak and those reported previously. Several genotype specific and common codon usage traits were noted in ZIKV coding sequences, indicative of independent evolutionary origins from a common ancestor. The overall influence of natural selection was found to be more profound than that of mutation pressure and acting on specific set of viral genes belonging to ZIKV strains of Asian genotype from the recent outbreak. Furthermore, an interplay of codon adaptation and deoptimization have been observed in ZIKV genomes. The collective findings of codon analysis in association with the geographical data of Aedes populations in the Americas suggests that ZIKV have evolved a dynamic set of codon usage patterns in order to maintain a successful replication and transmission chain within multiple hosts and vectors.</td><td>2016-03-30</td><td>Azeem Mehmood Butt; Izza Nasrullah; Raheel Qamar; Yigang Tong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/044594</td></tr><tr><td>csokkcqq</td><td>a59906b732bf4a489e282c3e4f499d4166c622e7</td><td>biorxiv</td><td>Evaluating performance of metagenomic characterization algorithms using in silico datasets generated with FASTQSim</td><td>10.1101/046532</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackgroundIn silico bacterial, viral, and human truth datasets were generated to evaluate available metagenomics algorithms. Sequenced datasets include background organisms, creating ambiguity in the true source organism for each read. Bacterial and viral datasets were created with even and staggered coverage to evaluate organism identification, read mapping, and gene identification capabilities of available algorithms. These truth datasets are provided as a resource for the development and refinement of metagenomic algorithms. Algorithm performance on these truth datasets can inform decision makers on strengths and weaknesses of available algorithms and how the results may be best leveraged for bacterial and viral organism identification and characterization.Source organisms were selected to mirror communities described in the Human Microbiome Project as well as the emerging pathogens listed by the National Institute of Allergy and Infectious Diseases. The six in silico datasets were used to evaluate the performance of six leading metagenomics algorithms: MetaScope, Kraken, LMAT, MetaPhlAn, MetaCV, and MetaPhyler.ResultsAlgorithms were evaluated on runtime, true positive organisms identified to the genus and species levels, false positive organisms identified to genus and species level, read mapping, relative abundance estimation, and gene calling. No algorithm out performed the others in all categories, and the algorithm or algorithms of choice strongly depends on analysis goals. MetaPhlAn excels for bacteria and LMAT for viruses. The algorithms were ranked by overall performance using a normalized weighted sum of the above metrics, and MetaScope emerged as the overall winner, followed by Kraken and LMAT.ConclusionsSimulated FASTQ datasets with well-characterized truth data about microbial community composition reveal numerous insights about the relative strengths and weaknesses of the metagenomics algorithms evaluated. The simulated datasets are available to download from the Sequence Read Archive (SRP062063).</td><td>2016-03-31</td><td>Anna Shcherbina; Darrell O. Ricke; Nelson Chiu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/046532</td></tr><tr><td>08mep8hm</td><td>99dab788e984601ee4efcd1138926caf10c00dcf</td><td>biorxiv</td><td>Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension</td><td>10.1101/048801</td><td>null</td><td>null</td><td>biorxiv</td><td>Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (fs) and monovalent salt concentrations (Cs). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log Cm)α with α = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation.</td><td>2016-04-15</td><td>Naoto Hori; Natalia A. Denesyuk; D. Thirumalai</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/048801</td></tr><tr><td>ejqq2wmi</td><td>97339c9d365437d54ea33be1a1b634d35f460208</td><td>biorxiv</td><td>Fractional Dosing of Yellow Fever Vaccine to Extend Supply: A Modeling Study</td><td>10.1101/053421</td><td>null</td><td>null</td><td>biorxiv</td><td>BackgroundThe ongoing yellow fever (YF) epidemic in Angola strains the global vaccine supply, prompting WHO to adopt dose sparing for its vaccination campaign in Kinshasa in July-August 2016. Although a 5-fold fractional-dose vaccine is similar to standard-dose vaccine in safety and immunogenicity, efficacy is untested. There is an urgent need to ensure the robustness of fractional-dose vaccination by elucidating the conditions under which dose fractionation would reduce transmission.MethodsWe estimate the effective reproductive number for YF in Angola using disease natural history and case report data. With simple mathematical models of YF transmission, we calculate the infection attack rate (IAR, the proportion of population infected over the course of an epidemic) under varying levels of transmissibility and five-fold fractional-dose vaccine efficacy for two vaccination scenarios: (i) random vaccination in a hypothetical population that is completely susceptible; (ii) the Kinshasa vaccination campaign in July-August 2016 with different age cutoff for fractional-dose vaccines.FindingsWe estimate the effective reproductive number early in the Angola outbreak was between 5·2 and 7·1. If vaccine action is all-or-nothing (i.e. a proportion VE of vaccinees receives complete and the remainder receive no protection), n-fold fractionation can dramatically reduce IAR as long as efficacy VE exceeds 1/n. This benefit threshold becomes more stringent if vaccine action is leaky (i.e. the susceptibility of each vaccinee is reduced by a factor that is equal to the vaccine efficacy VE). The age cutoff for fractional-dose vaccines chosen by the WHO for the Kinshasa vaccination campaign (namely, 2 years) provides the largest reduction in IAR if the efficacy of five-fold fractional-dose vaccines exceeds 20%.InterpretationDose fractionation is a very effective strategy for reducing infection attack rate that would be robust with a large margin for error in case fractional-dose VE is lower than expected.FundingNIH-MIDAS, HMRF-Hong Kong</td><td>2016-05-14</td><td>Joseph T. Wu; Corey M. Peak; Gabriel M. Leung; Marc Lipsitch</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/053421</td></tr><tr><td>2eh0bhee</td><td>1c0b25c8e3c89243fd8797e2e964e12689bc8556</td><td>biorxiv</td><td>50-valent inactivated rhinovirus vaccine is broadly immunogenic in rhesus macaques</td><td>10.1101/053967</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAs the predominant etiological agent of the common cold, human rhinovirus (HRV) is the leading cause of human infectious disease. Early studies showed monovalent formalin-inactivated HRV vaccine can be protective, and virus-neutralizing antibodies (nAb) correlated with protection. However, co-circulation of many HRV types discouraged further vaccine efforts. We approached this problem straightforwardly. We tested the hypothesis that increasing virus input titers in polyvalent inactivated HRV vaccine will result in broad nAb responses. Here, we show that serum nAb against many rhinovirus types can be induced by polyvalent, inactivated HRVs plus alhydrogel (alum) adjuvant. Using formulations up to 25-valent in mice and 50-valent in rhesus macaques, HRV vaccine immunogenicity was related to sufficient quantity of input antigens, and valency was not a major factor for potency or breadth of the response. We for the first time generated a vaccine capable of inducing nAb responses to numerous and diverse HRV types.</td><td>2016-05-17</td><td>Sujin Lee; Minh Trang Nguyen; Michael G. Currier; Joe B. Jenkins; Elizabeth A. Strobert; Adriana E. Kajon; Ranjna Madan-Lala; Yury A. Bochkov; James E. Gern; Krishnendu Roy; Xiaoyan Lu; Dean D. Erdman; Paul Spearman; Martin L. Moore</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/053967</td></tr><tr><td>76i8yvb4</td><td>d1b26388670660f66249b55ece2beccee4c18d1a</td><td>biorxiv</td><td>Evidence for reassortment of highly divergent novel rotaviruses from bats in Cameroon, without evidence for human interspecies transmissions</td><td>10.1101/054072</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBats are an important reservoir for pathogenic human respiratory and hemorrhagic viruses but only little is known about bat viruses causing gastroenteritis in humans, including rotavirus A strains (RVA). Only three RVA strains have been reported in bats in Kenya (straw-colored fruit bat) and in China (lesser horseshoe and a stoliczka’s trident bat), being highly divergent from each other. To further elucidate the potential of bat RVAs to cause gastroenteritis in humans we started by investigating the genetic diversity of RVAs in fecal samples from 87 straw-colored fruit bats living in close contact with humans in Cameroon using metagenomics. Five samples contained significant numbers of RVA Illumina reads, sufficient to obtain their (near) complete genomes. A single RVA strain showed a close phylogenetic relationship with the Kenyan bat RVA strain in six gene segments, including VP7 (G25), whereas the other gene segments represented novel genotypes as ratified by the RCWG. The 4 other RVA strains were highly divergent from known strains (but very similar among each other) possessing all novel genotypes. Only the VP7 and VP4 genes showed a significant variability representing multiple novel G and P genotypes, indicating the frequent occurrence of reassortment events.Comparing these bat RVA strains with currently used human RVA screening primers indicated that several of the novel VP7 and VP4 segments would not be detected in routine epidemiological screening studies. Therefore, novel VP6 based screening primers matching both human and bat RVAs were developed and used to screen samples from 25 infants with gastroenteritis living in close proximity with the studied bat population. Although RVA infections were identified in 36% of the infants, Sanger sequencing did not indicate evidence of interspecies transmissions.This study identified multiple novel bat RVA strains, but further epidemiological studies in humans will have to assess if these viruses have the potential to cause gastroenteritis in humans.</td><td>2016-05-18</td><td>Claude Kwe Yinda; Mark Zeller; Nádia Conceição-Neto; Piet Maes; Ward Deboutte; Leen Beller; Elisabeth Heylen; Stephen Mbigha Ghogomu; Marc Van Ranst; Jelle Matthijnssens</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/054072</td></tr><tr><td>omtmy9z6</td><td>null</td><td>biorxiv</td><td>The ATP synthase subunit β (ATP5B) is an entry factor for the hepatitis E virus</td><td>10.1101/060434</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractHepatitis E occurs sporadically and as outbreaks due to contamination of drinking water. The causative agent, hepatitis E virus (HEV) is a hepatotropic non-enveloped RNA virus, which grows poorly in vitro. Consequently, many aspects of HEV biology are poorly characterized, including its cellular receptor and entry mechanism(s). Previous studies from our laboratory have shown that heparan sulfate proteoglycans (HSPGs) act as attachment factors for the virus. In the absence of purified high titer infectious virus, we have used hepatitis E virus-like particles (HEV-LPs) expressed and purified from E. coli to identify HEV entry factor(s) on liver cells in culture. Using a pull down and mass spectrometric approach, we identified the ATP synthase subunit β (ATP5B) to bind the HEV capsid protein. Its role in the entry of HEV was then validated using antibody and siRNA mediated approaches, and infectious HEV from the stools of a hepatitis E patient. Though ATP synthase is largely a mitochondrial protein, the cell surface expressed form of ATP5B is implicated in other viral infections.</td><td>2016-06-23</td><td>Zulfazal Ahmed; Prasida Holla; Imran Ahmad; Shahid Jameel</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/060434</td></tr><tr><td>2cerplno</td><td>53442eacc3f233078507fa37b78267399e8c1e3b</td><td>biorxiv</td><td>Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection</td><td>10.1101/061788</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTZika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes.</td><td>2016-07-01</td><td>Arunachalam Ramaiah; Deisy Contreras; Vineela Gangalapudi; Masumi Sameer Padhye; Jie Tang; Vaithilingaraja Arumugaswami</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/061788</td></tr><tr><td>t3tm6f9p</td><td>c04e10858012bdd666d50c0d69c3d1e7224ccbea</td><td>biorxiv</td><td>Design: An assay based on single-polypeptide-chain heterodimeric A2AR/D2R and non-oligomerized fusions for in vivo analysis of their allosteric receptor-receptor interactions</td><td>10.1101/065250</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackgroundThe adenosine A2A receptor (A2AR) heteromerizes with the dopamine D2 receptor (D2R). In order to explore their functional interaction, we engineered previously stable single-polypeptide-chain (sc) A2AR/D2LR: whether the molecular entity of the striatal A2AR/D2R antagonism, i.e., scA2AR/D2Rs are just A2AR/D2R with the antagonism, remains unresolved.New MethodTo further clarify the heteromerization through the scA2AR/D2LR, we here designed supramolecularly ‘exclusive’ monomers and dimers, using the Cε2 domain of IgE-Fc or apoproteins of the bacterial light-harvesting antenna complex.ResultsA concept of the recptor protein assembly regulation, i.e., the selective monomer/non-obligate dimer formation was obtained. Although none of these new fusions were constructed or tested, we could aim at obtaining heterodimer-specific agents, using the scA2AR/D2R. Whether the resulting designs were explained feasibly and rationally was addressed. The structure and function of the non-obligate dimer were here discussed through scA2AR/D2R, focusing on the procedure of the membrane protein design and methods for transient protein-protein interactions.Summary and OutlookGiven that upon being expressed and allosteric regulation occurs regardless of specific signal to non-specific noise (S/N) ratio, the supramolecular designs, allowing us to express selectively monomer/non-obligate dimer of class A GPCR, are experimentally testable and will be used to confirm in vivo that such low S/N ratio interaction between A2AR and D2LR functions in the dopamine neurotransmission in the striatum.Abbreviations:A2ARadenosine A2A receptor3HA-A2ARA2AR tagged with a triple HA epitopeBchlbacteriochlorophyllBRETbioluminescence resonance energy transferCcarboxy-terminalCDcluster of differentiationD2LR and D2SRthe long and short form of dopamine D2 receptor, respectivelyFabantigen binding fragmentFcFc fragmentFcεRIhigh affinity receptor for IgEFRETfluorescence resonance energy transferG4San amino acid sequence consisting of a four-glycine-repeat followed by a serine residueGABAγ-aminobutyric acidGABABGABA type B receptorGPCRG protein-coupled receptorGttransducinHAhemagglutininHIVhuman immunodeficiency virusICintracellular loopsIgimmunoglobulinLHlight-harvesting antenna complexmAbmonoclonal antibodyMrmolecular weightNamino-terminalPDParkinson’s diseasePSphotosystemRCreaction centerRlucRenilla luciferasescsingle-chainTMtransmembrane3Dthree-dimensional</td><td>2016-07-26</td><td>Toshio Kamiya; Takashi Masuko; Dasiel Oscar Borroto-Escuela; Haruo Okado; Hiroyasu Nakata</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/065250</td></tr><tr><td>57sp9d9l</td><td>5570cb9ff2905fda60cfd6e94f3c32774b765265</td><td>biorxiv</td><td>The genome of the crustacean Parhyale hawaiensis: a model for animal development, regeneration, immunity and lignocellulose digestion</td><td>10.1101/065789</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe amphipod crustacean Parhyale hawaiensis is a blossoming model system for studies of developmental mechanisms and more recently regeneration. We have sequenced the genome allowing annotation of all key signaling pathways, transcription factors, and non-coding RNAs that will enhance ongoing functional studies. Parhyale is a member of the Malacostraca clade, which includes crustacean food crop species. We analysed the immunity related genes of Parhyale as an important comparative system for these species, where immunity related aquaculture problems have increased as farming has intensified. We also find that Parhyale and other species within Multicrustacea contain the enzyme sets necessary to perform lignocellulose digestion (“wood eating”), suggesting this ability may predate the diversification of this lineage. Our data provide an essential resource for further development of Parhyale as an experimental model. The first malacostracan genome will underpin ongoing comparative work in food crop species and research investigating lignocellulose as an energy source.</td><td>2016-07-25</td><td>Damian Kao; Alvina G. Lai; Evangelia Stamataki; Silvana Rosic; Nikolaos Konstantinides; Erin Jarvis; Alessia Di Donfrancesco; Natalia Pouchkina-Stantcheva; Marie Sémon; Marco Grillo; Heather Bruce; Suyash Kumar; Igor Siwanowicz; Andy Le; Andrew Lemire; Michael B. Eisen; Cassandra Extavour; William E. Browne; Carsten Wolff; Michalis Averof; Nipam H. Patel; Peter Sarkies; Anastasios Pavlopoulos; A. Aziz Aboobaker</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/065789</td></tr><tr><td>tiyjmih7</td><td>45f184e736c4a857bc0e3df0d55962f743c1493a</td><td>biorxiv</td><td>Similar ratios of introns to intergenic sequence across animal genomes</td><td>10.1101/068627</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractOne central goal of genome biology is to understand how the usage of the genome differs between organisms. Our knowledge of genome composition, needed for downstream inferences, is critically dependent on gene annotations, yet problems associated with gene annotation and assembly errors are usually ignored in comparative genomics. Here we analyze the genomes of 68 species across 12 animal phyla and some single-cell eukaryotes for general trends in genome composition and transcription, taking into account problems of gene annotation. We show that, regardless of genome size, the ratio of introns to intergenic sequence is comparable across essentially all animals, with nearly all deviations dominated by increased intergenic sequence. Genomes of model organisms have ratios much closer to 1:1, suggesting that the majority of published genomes of non-model organisms are underannotated and consequently omit substantial numbers of genes, with likely negative impact on evolutionary interpretations. Finally, our results also indicate that most animals transcribe half or more of their genomes arguing against differences in genome usage between animal groups, and also suggesting that the transcribed portion is more dependent on genome size than previously thought.</td><td>2016-08-09</td><td>Warren R. Francis; Gert Wörheide</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/068627</td></tr><tr><td>30kwl4rj</td><td>ffbf8ea9948d73572fd052a74afa01b19e6758a3</td><td>biorxiv</td><td>Planning horizon affects prophylactic decision-making and epidemic dynamics</td><td>10.1101/069013</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractHuman behavior can change the spread of infectious disease. There is limited understanding of how the time in the future over which individuals make a behavioral decision, their planning horizon, affects epidemic dynamics. We developed an agent-based model (along with an ODE analog) to explore the decision-making of self-interested individuals on adopting prophylactic behavior. The decision-making process incorporates prophylaxis efficacy and disease prevalence with individuals' payoffs and planning horizon. Our results show that for short and long planning horizons individuals do not consider engaging in prophylactic behavior. In contrast, individuals adopt prophylactic behavior when considering intermediate planning horizons. Such adoption, however, is not always monotonically associated with the prevalence of the disease, depending on the perceived protection efficacy and the disease parameters. Adoption of prophylactic behavior reduces the peak size while prolonging the epidemic and potentially generates secondary waves of infection. These effects can be made stronger by increasing the behavioral decision frequency or distorting an individual’s perceived risk of infection.</td><td>2016-08-12</td><td>Luis G. Nardin; Craig R. Miller; Benjamin J. Ridenhour; Stephen M. Krone; Paul Joyce; Bert O. Baumgaertner</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/069013</td></tr><tr><td>1znuxoj5</td><td>fd3fc2c49f5cc27e4262261d0c9045911d65cb6e</td><td>biorxiv</td><td>XRN1 is a Species-Specific Virus Restriction Factor in Yeasts</td><td>10.1101/069799</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractIn eukaryotes, the degradation of cellular mRNAs is accomplished by Xrn1p and the cytoplasmic exosome. Because viral RNAs often lack canonical caps or poly-A tails, they can also be vulnerable to degradation by these host exonucleases. Yeast lack sophisticated mechanisms of innate and adaptive immunity, but do use RNA degradation as an antiviral defense mechanism. One model is that the RNA of yeast viruses is subject to degradation simply as a side effect of the intrinsic exonuclease activity of proteins involved in RNA metabolism. Contrary to this model, we find a highly refined, species-specific relationship between Xrn1p and the “L-A” totiviruses of different Saccharomyces yeast species. We show that the gene XRN1 has evolved rapidly under positive natural selection in Saccharomyces yeast, resulting in high levels of Xrn1p protein sequence divergence from one yeast species to the next. We also show that these sequence differences translate to differential interactions with the L-A virus, where Xrn1p from S. cerevisiae is most efficient at controlling the L-A virus that chronically infects S. cerevisiae, and Xrn1p from S. kudriavzevii is most efficient at controlling the L-A-like virus that we have discovered within S. kudriavzevii. All Xrn1p orthologs are equivalent in their interaction with another virus-like parasite, the Ty1 retrotransposon. Thus, the activity of Xrn1p against totiviruses is not simply an incidental consequence of the enzymatic activity of Xrn1p, but rather Xrn1p co-evolves with totiviruses to maintain its potent antiviral activity and limit viral propagation in Saccharomyces yeasts. Consistent with this, we demonstrated that Xrn1p physically interacts with the Gag protein encoded by the L-A virus, suggesting a host-virus interaction that is more complicated than just Xrn1p-mediated nucleolytic digestion of viral RNAs.</td><td>2016-08-16</td><td>Paul A. Rowley; Brandon Ho; Sarah Bushong; Arlen Johnson; Sara L. Sawyer</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/069799</td></tr><tr><td>pxpujrhx</td><td>44a440cc1c135c938d2216ea672f8ef4f9c01296</td><td>biorxiv</td><td>Genome-wide Prediction of microRNAs in Zika virus Genomes Reveals Possible Interactions with Human Genes Involved in the Nervous System Development</td><td>10.1101/070656</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractZika virus (ZIKV) is a member of the family Flaviviridae. In 2015, ZIKV triggered a large epidemic in Brazil and spread across Latin America. In November of that year, the Brazilian Ministry of Health reported a 20-fold increase in cases of neonatal microcephaly, which corresponds geographically and temporally to the ZIKV outbreak. ZIKV was isolated from the brain tissue of a fetus diagnosed with microcephaly, and recent studies in mice models revealed that ZIKV infection may cause brain defects by influencing brain cell developments. Unfortunately, the mechanisms by which ZIKV alters neurophysiological development remain unknown. MicroRNAs (miRNAs) are small noncoding RNAs that regulate post-transcriptional gene expression by translational repression. In order to gain insight into the possible role of ZIKV-mediated miRNA signaling dysfunction in brain-tissue development, we computationally predicted new miRNAs encoded by the ZIKV genome and their effective hybridization with transcripts from human genes previously shown to be involved in microcephalia. The results of these studies suggest a possible role of these miRNAs on the expression of human genes associated with this disease. Besides, a new ZIKV miRNA was predicted in the 3’stem loop (3’ SL) of the 3’untranslated region (3’UTR) of the ZIKV genome, suggesting the role of the 3’UTR of flaviviruses as a source of miRNAs.</td><td>2016-08-21</td><td>Juan Cristina; Natalia Echeverría; Fabiana Gambaro; Alvaro Fajardo; Pilar Moreno</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/070656</td></tr><tr><td>2j4z5rp8</td><td>9b5f5119bbfbded3245acc37859cefde967458e7</td><td>biorxiv</td><td>Containing Emerging Epidemics: a Quantitative Comparison of Quarantine and Symptom Monitoring</td><td>10.1101/072652</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTStrategies for containing an emerging infectious disease outbreak must be non-pharmaceutical when drugs or vaccines for the pathogen do not yet exist or are unavailable. The success of these non-pharmaceutical strategies will depend not only on the effectiveness of quarantine or other isolation measures but also on the epidemiological characteristics of the infection. However, there is currently no systematic framework to assess the relationship between different containment strategies and the natural history and epidemiological dynamics of the pathogen. Here, we compare the effectiveness of quarantine and symptom monitoring, implemented via contact tracing, in controlling epidemics using an agent-based branching model. We examine the relationship between epidemic containment and the disease dynamics of symptoms and infectiousness for seven case study diseases with diverse natural histories including Ebola, Influenza A, and Severe Acute Respiratory Syndrome (SARS). We show that the comparative effectiveness of symptom monitoring and quarantine depends critically on the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. The benefit of quarantine over symptom monitoring is generally maximized for fast-course diseases, but we show the conditions under which symptom monitoring alone can control certain outbreaks. This quantitative framework can guide policy-makers on how best to use non-pharmaceutical interventions to contain emerging outbreaks and prioritize research during an outbreak of a novel pathogen.SIGNIFICANCEQuarantine and symptom monitoring of contacts with suspected exposure to an infectious disease are key interventions for the control of emerging epidemics; however, there does not yet exist a quantitative framework for comparing the control performance of each. Here, we use a mathematical model of seven case study diseases to show how the choice of intervention is influenced by the natural history of the infectious disease, its inherent transmissibility, and the intervention feasibility in the particular healthcare setting. We use this information to identify the most important characteristics of the disease and setting that need to be characterized for an emerging pathogen in order to make an informed decision between quarantine and symptom monitoring.</td><td>2016-08-31</td><td>Corey M Peak; Lauren M Childs; Yonatan H Grad; Caroline O Buckee</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/072652</td></tr><tr><td>kxe65hd7</td><td>null</td><td>biorxiv</td><td>Zika virus infection as a cause of congenital brain abnormalities and Guillain-Barré syndrome: systematic review</td><td>10.1101/073098</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackgroundThe World Health Organization stated in March 2016 that there was scientific consensus that the mosquito-borne Zika virus was a cause of the neurological disorder Guillain-Barré syndrome and of microcephaly and other congenital brain abnormalities, based on rapid evidence assessments. Decisions about causality require systematic assessment to guide public health actions. The objectives of this study were: to update and re-assess the evidence for causality through a rapid and systematic review about links between Zika virus infection and a) congenital brain abnormalities, including microcephaly, in the foetuses and offspring of pregnant women and b) Guillain-Barré syndrome in any population; and to describe the process and outcomes of an expert assessment of the evidence about causality.Methods and findingsThe study had three linked components. First, in February 2016, we developed a causality framework that defined questions about the relationship between Zika virus infection and each of the two clinical outcomes in 10 dimensions; temporality, biological plausibility, strength of association, alternative explanations, cessation, dose-response, animal experiments, analogy, specificity and consistency. Second, we did a systematic review (protocol number CRD42016036693). We searched multiple online sources up to May 30, 2016 to find studies that directly addressed either outcome and any causality dimension, used methods to expedite study selection, data extraction and quality assessment, and summarised evidence descriptively. Third, a multidisciplinary panel of experts assessed the review findings and reached consensus on causality. We found 1091 unique items up to May 30, 2016. For congenital brain abnormalities, including microcephaly, we included 72 items; for eight of 10 causality dimensions (all except dose-response relationship and specificity) we found that more than half the relevant studies supported a causal association with Zika virus infection. For Guillain-Barré syndrome, we included 36 items, of which more than half the relevant studies supported a causal association in seven of ten dimensions (all except dose-response relationship, specificity and animal experimental evidence). Articles identified non-systematically from May 30-July 29, 2016 strengthened the review findings. The expert panel concluded that: a) the most likely explanation of available evidence from outbreaks of Zika virus infection and clusters of microcephaly is that Zika virus infection during pregnancy is a cause of congenital brain abnormalities including 61 microcephaly; and b) the most likely explanation of available evidence from outbreaks of Zika virus infection and Guillain-Barré syndrome is that Zika virus infection is a trigger of Guillain-Barré syndrome. The expert panel recognised that Zika virus alone may not be sufficient to cause either congenital brain abnormalities or Guillain-Barré syndrome but agreed that the evidence was sufficient to recommend increased public health measures. Weaknesses are the limited assessment of the role of dengue virus and other possible co-factors, the small number of comparative epidemiological studies, and the difficulty in keeping the review up to date with the pace of publication of new research.ConclusionsRapid and systematic reviews with frequent updating and open dissemination are now needed, both for appraisal of the evidence about Zika virus infection and for the next public health threats that will emerge. This rapid systematic review found sufficient evidence to say that Zika virus is a cause of congenital abnormalities and is a trigger of Guillain-Barré situation.</td><td>2016-09-02</td><td>Fabienne Krauer; Maurane Riesen; Ludovic Reveiz; Olufemi T Oladapo; Ruth Martínez-Vega; Teegwendé V. Porgo; Anina Haefliger; Nathalie J Broutet; Nicola Low</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/073098</td></tr><tr><td>8elmjre9</td><td>8f4b98da50277e6c9516aef0227f5cca9af6b5e8</td><td>biorxiv</td><td>Identification of quercetin from fruits to immediately fight Zika</td><td>10.1101/074559</td><td>null</td><td>null</td><td>biorxiv</td><td>Zika virus is spread mainly by the bite of an infected mosquito, which can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika in the near future, but unfortunately so far there is no vaccine or medicine for Zika. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the key role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. In response to the current global health emergency triggered by the Zika outbreak, we successfully obtained several active forms of Zika NS2B-NS3pro and further attempted to discover its inhibitors from eatable plants and traditional herbal medicines to immediately fight Zika. Here, for the first time, we discovered that quercetin, a flavonoid extensively existing in many fruits and vegetables, effectively inhibits Zika NS2B-NS3pro. We further quantify its inhibitory activity with IC50 of 26 uM; and Ki of 23 uM. As quercetin has been extensively found in fruits, vegetables, leaves and grains, our discovery would benefit the public to immediately fight Zika.</td><td>2016-09-11</td><td>Amrita Roy; Liangzhong Lim; Jianxing Song</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/074559</td></tr><tr><td>l82aakrk</td><td>f9b108c052ded463773a9838a8157ac7d5ba6d1a</td><td>biorxiv</td><td>Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations</td><td>10.1101/078113</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTZika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short β-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors.</td><td>2016-09-28</td><td>Amrita Roy; Liangzhong Lim; Shagun Srivastava; Jianxing Song</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/078113</td></tr><tr><td>gfiukh90</td><td>5b27a16123f46b5aaeda3171d22e075dcbd36ee6</td><td>biorxiv</td><td>Identification of a Zika NS2B-NS3pro pocket susceptible to allosteric inhibition by small molecules including qucertin rich in edible plants</td><td>10.1101/078543</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTIt has been recently estimated that one-third of the world population will be infected by Zika virus, but unfortunately so far there is no vaccine or medicine available. In particular, the special concern on the vaccine treatment to Zika and Dengue arising from antibody-dependent enhancement strongly emphasizes the irreplaceable role of its NS2B-NS3 protease (NS2B-NS3pro) as a target for anti-Zika drug discovery/design due to its absolutely-essential role in viral replication. Very recently we identified two small molecules inhibit Zika NS2B-NS3pro in non-competitive mode, with Ki values of 0.57 and 2.02 µM respective for p-Nitrophenyl-p-guanidino benzoate and qucertin. Here, by molecular docking, we show that although one is designed compound while another is a natural product, both molecules bind to the same pocket on the back of the substrate-binding pocket of Zika NS2B-NS3pro. As the two inhibitors fundamentally differ from cn-716, the only known peptidomimetic boronic acid inhibitor in both structure scaffolds and inhibitory modes, our discovery might open up a new avenue for the future development of allosteric inhibitors, which is highly demanded to achieve therapeutic inhibition of flaviviral NS2B-NS3pro complexes. Furthermore, as qucertin is abundant in many vegetables and fruits such caper, lovage, tea and red onion, our results should benefit the public to immediately fight Zika virus.</td><td>2016-10-01</td><td>Liangzhong Lim; Amrita Roy; Jianxing Song</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/078543</td></tr><tr><td>mkoyc7wh</td><td>null</td><td>biorxiv</td><td>Comparative analysis of variation and selection in the HCV genome</td><td>10.1101/078584</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractGenotype 1 of the hepatitis C virus (HCV) is the most prevalent of the variants of this virus. Its two main subtypes, HCV-1a and HCV-1b, are associated to differences in epidemic features and risk groups, despite sharing similar features in most biological properties. We have analyzed the impact of positive selection on the evolution of these variants using complete genome coding regions, and compared the levels of genetic variability and the distribution of positively selected sites. We have also compared the distributions of positively selected and conserved sites considering different factors such as RNA secondary structure, the presence of different epitopes (antibody, CD4 and CD8), and secondary protein structure. Less than 10% of the genome was found to be under positive selection, and purifying selection was the main evolutionary force in both subtypes. We found differences in the number of positively selected sites between subtypes in several genes (Core, HVR2 in E2, P7, helicase in NS3 and NS4a). Heterozygosity values in positively selected sites and the rate of non-synonymous substitutions were significantly higher in subtype HCV-1b. Logistic regression analyses revealed that similar selective forces act at the genome level in both subtypes: RNA secondary structure and CD4 T-cell epitopes are associated with conservation, while CD8 T-cell epitopes are associated with positive selection in both subtypes. These results indicate that similar selective constraints are acting along HCV-1a and HCV-1b genomes, despite some differences in the distribution of positively selected sites at independent genes.</td><td>2016-09-30</td><td>Juan Ángel Patiño-Galindo; Fernando González-Candelas</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/078584</td></tr><tr><td>lj0ar2v5</td><td>28b107243576723248ad4053261000311a22f134</td><td>biorxiv</td><td>Genetic, transcriptome, proteomic and epidemiological evidence for blood brain barrier disruption and polymicrobial brain invasion as determinant factors in Alzheimer’s disease</td><td>10.1101/080333</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractMultiple pathogens have been detected in Alzheimer’s disease (AD) brains. A bioinformatics approach was used to assess relationships between pathogens and AD genes (GWAS), the AD hippocampal transcriptome and plaque or tangle proteins. Host/pathogen interactomes (C.albicans, C.Neoformans, Bornavirus, B.Burgdorferri, cytomegalovirus, Ebola virus, HSV-1, HERV-W, HIV-1, Epstein-Barr, hepatitis C, influenza, C.Pneumoniae, P.Gingivalis, H.Pylori, T.Gondii, T.Cruzi) significantly overlap with misregulated AD hippocampal genes, with plaque and tangle proteins and, except Bornavirus, Ebola and HERV-W, with AD genes. Upregulated AD hippocampal genes match those upregulated by multiple bacteria, viruses, fungi or protozoa in immunocompetent blood cells. AD genes are enriched in bone marrow and immune locations and in GWAS datasets reflecting pathogen diversity, suggesting selection for pathogen resistance. The age of AD patients implies resistance to infections afflicting the younger. APOE4 protects against malaria and hepatitis C, and immune/inflammatory gain of function applies to APOE4, CR1, TREM2 and presenilin variants. 30/78 AD genes are expressed in the blood brain barrier (BBB), which is disrupted by AD risk factors (ageing, alcohol, aluminium, concussion, cerebral hypoperfusion, diabetes, homocysteine, hypercholesterolaemia, hypertension, obesity, pesticides, pollution, physical inactivity, sleep disruption and smoking). The BBB and AD benefit from statins, NSAIDs, oestrogen, melatonin and the Mediterranean diet. Polymicrobial involvement is supported by the upregulation of pathogen sensors/defenders (bacterial, fungal, viral) in the AD brain, blood or CSF. Cerebral pathogen invasion permitted by BBB inadequacy, activating a hyper-efficient immune/inflammatory system, betaamyloid and other antimicrobial defence may be responsible for AD which may respond to antibiotic, antifungal or antiviral therapy.</td><td>2016-10-12</td><td>C.J. Carter</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/080333</td></tr><tr><td>w49x7kca</td><td>e6fe63419fe399f9c77513c1c11a043db006a2f3</td><td>biorxiv</td><td>Accurate prediction of human essential genes using only nucleotide composition and association information</td><td>10.1101/084129</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThree groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a λ-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier’s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (https://cefg.uestc.edu.cn/Pheg), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the λ-interval Z curve method could be effectively extended to classification issues of other DNA elements.</td><td>2016-10-28</td><td>Feng-Biao Biao; Chuan Dong; Hong-Li Hua; Shuo Liu; Hao Luo; Hong-Wan Zhang; Yan-Ting Jin; Kai-Yue Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/084129</td></tr><tr><td>phmm596b</td><td>dff716159738acd6bd53b4be12af45358a58752e</td><td>biorxiv</td><td>MYCOPLASMA FAUCIUM AND BREAST CANCER</td><td>10.1101/089128</td><td>null</td><td>null</td><td>biorxiv</td><td>Viruses and bacteria are the cause of a large number of different human diseases. It is believed that some of them may even contribute to the development of cancer. The present work is dedicated to the identification of mycoplasmas in patients with breast cancer. Mycoplasmas may participate in the development of several human diseases including chronic fatigue syndrome, acquired immunodeficiency syndrome, atypical pneumonia, etc. Moreover, there is a reason to believe that mycoplasma can participate in the development of cancer, leukemia and lymphoma. DNA samples from blood, saliva and tumor tissues of the Oncology Institute of Moldova patients diagnosed with breast cancer were analyzed. Mycoplasma testing was performed using nested PCR method. For Mycoplasma spp. detection, we used primers from the region of the 16S-23S RNA genes. The identification of Mycoplasma faucium, Mycoplasma salivarium and Mycoplasma orale was performed by nested PCR with primers for RNA polymerase beta subunit gene corresponding to mycoplasma. M.faucium and M.salivarius was found in saliva at about 100%, and M.orale at a frequency of about 50%. Only M.faucium was found with the frequency of about 60% in the tissue of the patients. Moreover, a fairly high rate of detection of mycoplasma is observed both in the cases when primers for RNA polymerase gene and primers for 16S-23S RNA were used. We found M.faucium in tumor tissues of patients diagnosed with breast cancer. It is known that mycoplasmas are able to stimulate the synthesis of certain cytokines, which act as mitogenes on the cell. We assume that M.faucium can stimulate the mitogenes synthesis in breast tissues (e.g., cytokines) which, in turn, stimulate cell division and thus participate in the initiation of breast cancer.</td><td>2016-11-22</td><td>Valentin Mitin; Lidia Tumanova; Natalia Botnariuc</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/089128</td></tr><tr><td>iedh762s</td><td>8c900409a679dc1f16ef13c73500cbb53605b683</td><td>biorxiv</td><td>Lung epithelial cells have virus-specific and shared gene expression responses to infection by diverse respiratory viruses</td><td>10.1101/090936</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe severity and outcome of respiratory viral infections is partially determined by the cellular response mounted by infected lung epithelial cells. Disease prevention and treatment is dependent on our understanding of the shared and unique responses elicited by diverse viruses, yet few studies compare host responses to different viruses while controlling other experimental parameters. We compared changes in gene expression of murine lung epithelial cells infected individually by three respiratory viruses causing mild (rhinovirus, RV1B), moderate (coronavirus, MHV-1), and severe (influenza A virus, PR8) disease in mice. RV1B infection caused numerous gene expression changes, but the differential effect peaked at 12 hours post-infection. PR8 altered an intermediate number of genes whose expression continued to change through 24 hours. MHV-1 had comparatively few effects on host gene expression. The viruses elicited highly overlapping responses in antiviral genes, though MHV-1 induced a lower type I interferon response than the other two viruses. Signature genes were identified for each virus and included host defense genes for PR8, tissue remodeling genes for RV1B, and transcription factors for MHV-1. Our comparative approach identified universal and specific transcriptional signatures of virus infection that can be used to discover mechanisms of pathogenesis in the respiratory tract.</td><td>2016-12-02</td><td>James T. VanLeuven; Benjamin J. Ridenhour; Craig R. Miller; Tanya A. Miura</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/090936</td></tr><tr><td>2179jaes</td><td>6aa2716c00d89f4b760a9167b15d6203303ae6b9</td><td>biorxiv</td><td>Vitrification after multiple rounds of sample application and blotting improves particle density on cryo-electron microscopy grids</td><td>10.1101/094623</td><td>null</td><td>null</td><td>biorxiv</td><td>Single particle cryo-electron microscopy (cryoEM) is becoming widely adopted as a tool for structural characterization of biomolecules at near-atomic resolution. Vitrification of the sample to obtain a dense distribution of particles within a single field of view remains a major bottleneck for the success of such experiments. Here, we describe a simple and cost-effective method to increase the density of frozen-hydrated particles on grids with holey carbon support films. It relies on performing multiple rounds of sample application and blotting prior to plunge freezing in liquid ethane. We show that this approach is generally applicable and significantly increases particle density for a range of samples, such as small protein complexes, viruses and filamentous assemblies. The method is versatile, easy to implement, minimizes sample requirements and can enable characterization of samples that would otherwise resist structural studies using single particle cryoEM.</td><td>2016-12-15</td><td>Joost Snijder; Andrew J. Borst; Annie Dosey; Alexandra C. Walls; Anika Burrell; Vijay S. Reddy; Justin M. Kollman; David Veesler</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/094623</td></tr><tr><td>ahdz5078</td><td>8eb99586e9103599ab2f58cc33ca94cd0b60ae53</td><td>biorxiv</td><td>Epidemiological and ecological modelling reveal diversity in upper respiratory tract microbial population structures from a cross-sectional community swabbing study</td><td>10.1101/099069</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTRespiratory tract infections (RTI) are responsible for over 4 million deaths per year worldwide with pathobiont carriage a required precursor to infection. Through a cross-sectional community-based nasal self-swabbing study we sought to determine carriage epidemiology for respiratory pathogens amongst bacteria (Streptococcus pneumoniae, Haemophilus influenza, Moraxella catarrhalis, Staphylococcus aureus, Pseudomonas aeruginosa and Neisseria meningitidis) and viruses (RSV, Influenza viruses A and B, Rhinovirus/Enterovirus, Coronavirus, Parainfluenza viruses 1-3 and Adenovirus (ADV)). Carriage of bacterial and viral species was shown to vary with participant age, recent RTI and the presence of other species. The spatial structure of microbial respiratory communities was less nested (more disordered) in the young (0-4 years) and those with recent RTI. Species frequency distributions were flatter than random expectation in young individuals (X2 = 20.42, p = 0.002), indicating spatial clumping of species consistent with facilitative relationships amongst them. Deviations from a neutral model of ecological niches were observed for samples collected in the summer and from older individuals (those aged 5-17, 18-64 and ≥65 years) but not in samples collected from winter, younger individuals (those aged 0-4 years), individuals with recent RTI and individuals without recent RTI, demonstrating the importance of both neutral and niche processes in respiratory community assembly. The application of epidemiological methods and ecological theory to sets of respiratory tract samples has yielded novel insights into the factors that drive microbial community composition, such as seasonality and age, as well as species patterns and interactions within the nose.</td><td>2017-01-09</td><td>Abigail L. Coughtrie; Denise E. Morris; Rebecca Anderson; Nelupha Begum; David W. Cleary; Saul N. Faust; Johanna M. Jefferies; Alex R. Kraaijeveld; Michael V. Moore; Mark A. Mullee; Paul J. Roderick; Andrew Tuck; Robert N. Whittaker; Ho Ming Yuen; C. Patrick Doncaster; Stuart C. Clarke</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/099069</td></tr><tr><td>pl7t3hca</td><td>e0fd364797eb02a280e25e2649dfe6d6c0d33e05</td><td>biorxiv</td><td>Spike-based phylogenetically defined clades within the Alphacoronavirus 1 species</td><td>10.1101/101774</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractTaxonomic classification for the Coronaviridae can be challenging, due to the wide host tropism and highly variable genome of the viruses in this Family. Within the Alphacoronavirus genus, there is a single species Alphacoronavirus 1 that encompasses several biologically distinct viruses of distinct animal species. Here, we carried out phylogenetic analysis of members of the Alphacoronavirus genus, focusing on the viral spike gene, which is a primary driver of viral tropism and pathogenesis. We identify two distinct clades (A and B) within the Alphacoronavirus 1 species. Alphacoronavirus 1 clade A encompasses serotype I FCoV and CCoV, and Alphacoronavirus 1 clade B, encompasses serotype II FCoV and CCoV and TGEV-like viruses. We propose this clade designation, along with the newly proposed Alphacoronavirus 2 species, as an improved way to classify the diverse Alphacoronavirus genus.</td><td>2017-01-19</td><td>Gary R. Whittaker; Nicole M. André; Jean Kaoru Millet</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/101774</td></tr><tr><td>km0b1z64</td><td>d418cdee18b07a0adfb553d43b4a08f7736fa9f3</td><td>biorxiv</td><td>Metagenomic Sequencing Detects Respiratory Pathogens in Hematopoietic Cellular Transplant Patients</td><td>10.1101/102798</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTRATIONALECurrent microbiologic diagnostics often fail to identify the etiology of lower respiratory tract infections (LRTI) in hematopoietic cellular transplant recipients (HCT), which precludes the implementation of targeted therapies.OBJECTIVESTo address the need for improved LRTI diagnostics, we evaluated the utility of metagenomic next generation sequencing (mNGS) of bronchoalveolar lavage (BAL) to detect microbial pathogens in HCT patients with acute respiratory illnesses.METHODSWe enrolled 22 post-HCT adults ages 19-69 years with acute respiratory illnesses who underwent BAL at the University of Michigan between January 2012 and May 2013. mNGS was performed on BAL fluid to detect microbes and simultaneously assess the host transcriptional response. Results were compared against conventional microbiologic assays.MEASUREMENTS & MAIN RESULTSmNGS demonstrated 100% sensitivity for detecting respiratory microbes (human metapneumovirus, respiratory syncytial virus, Stenotrophomonas maltophilia, human herpesvirus 6 and cytomegalovirus) when compared to standard testing. Previously unrecognized LRTI pathogens were identified in six patients for whom standard testing was negative (human coronavirus 229E, human rhinovirus A, Corynebacterium propinquum and Streptococcus mitis); findings were confirmed by independent PCR and 16S rRNA sequencing. Relative to patients without infection, patients with infection had increased expression of immunity related genes (p=0.022) and significantly lower diversity of their respiratory microbiome (p=0.017).CONCLUSIONSCompared to conventional diagnostics, mNGS enhanced detection of pathogens in BAL fluid from HCT patients. Furthermore, our results suggest that combining unbiased microbial pathogen detection with assessment of host gene biomarkers of immune response may hold promise for enhancing the diagnosis of post-HCT respiratory infections.</td><td>2017-01-24</td><td>C Langelier; MS Zinter; K Kalantar; GA Yanik; S Christenson; B Odonovan; C White; M Wilson; A Sapru; CC Dvorak; S Miller; CY Chiu; JL DeRisi</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/102798</td></tr><tr><td>ois9fiuq</td><td>null</td><td>biorxiv</td><td>The immunomodulatory CEA cell adhesion molecule 6 (CEACAM6/CD66c) is a candidate receptor for the influenza A virus</td><td>10.1101/104026</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractTo establish a productive infection in host cells, viruses often use one or multiple host membrane glycoprotein as their receptors. For Influenza A virus (IAV) such a glycoprotein receptor has not been described, to date. Here we show that IAV is using the host membrane glycoprotein CD66c as a receptor for entry into human epithelial lung cells. Neuraminidase (NA), a viral spike protein binds to CD66c on the cell surface during IAV entry into the host cells. Lung cells overexpressing CD66c showed an increase in virus binding and subsequent entry into the cell. Upon comparison, CD66c demonstrated higher binding capacity than other membrane glycoproteins (EGFR and DC-SIGN) reported earlier to facilitate IAV entry into host cells. siRNA mediated knockdown of CD66c from lung cells inhibited virus binding on cell surface and entry into cells. Blocking CD66c by antibody on the cell surface resulted in decreased virus entry. We found CD66c is a specific glycoprotein receptor for influenza A virus that did not affect entry of non-IAV RNA virus (Hepatitis C virus). Finally, IAV pre-incubated with recombinant CD66c protein when administered intranasally in mice showed decreased cytopathic effects in mice lungs. This publication is the first to report CD66c (CEACAM6) as a glycoprotein receptor for Influenza A virus.Significance StatementCells are enclosed by a semipermeable membrane that allows selective exchange of biomolecules between cells and their surroundings. A set of specialized proteins in this semipermeable membrane, work like gatekeepers to the cell and regulate entry of these biomolecules. One class of such surface proteins is termed as receptors. Viruses bind to one or more of these receptors and manipulate gatekeepers for their own successful entry into host-cells. A membrane protein that influenza A virus (Flu virus) uses for entry into the cells was not discovered till date. This study reports for the first time, a receptor for influenza A virus, that was sought after by researchers for decades. The viral receptor is a promising target that can be used to inhibit virus entry into host cells.</td><td>2017-01-30</td><td>Shah Kamranur Rahman; Mairaj Ahmed Ansari; Pratibha Gaur; Imtiyaz Ahmad; Chandrani Chakravarty; Dileep Kumar Verma; Sanjay Chhibber; Naila Nehal; Shanmugaapriya Sellathanby; Dagmar Wirth; Gulam Waris; Sunil K. Lal</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/104026</td></tr><tr><td>9pa51d9x</td><td>95754ec84bdee0637765a5ee34983a5f6cc86eed</td><td>biorxiv</td><td>PGC-1α coordinates with Bcl-2 to control cell cycle in U251 cells through reducing ROS</td><td>10.1101/111567</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractB-cell lymphoma 2 (Bcl-2) has a dual function, acting both as an oncogene and an anti-tumor gene. It is well known that Bcl-2 exerts its tumor promoting function through the mitochondrial pathway. However, the mechanism by which Bcl-2 suppresses tumor formation is not well understood. We have previously shown that Bcl-2 inhibits cell cycle progression from the G0/G1 to the S phase after serum starvation, and that quiescent Bcl-2 expressing cells maintained a significant lower level of mitochondrial reactive oxygen species (ROS) than the control cells. Based on the fact that ROS mediate cell cycle progression, and are controlled by peroxisome proliferator-activated receptor-γ co-activator 1α (PGC-1α), a key molecule induced by prolonged starvation and involved in mitochondrial metabolism, we hypothesized that PGC-1α might be related with the cell cycle function of Bcl-2. Here, we showed that PGC-1α was upregulated upon Bcl-2 overexpression and downregulated following Bcl-2 knockdown during serum starvation. Knockdown of PGC-1α activated Bcl-2 expression. Taken together, our results suggest that after serum depletion, PGC-la might coordinate with Bcl-2 to reduce ROS, which in turn delay cell cycle progression.Summary statementPGC-1α coordinate with Bcl-2 delay cell cycle progression to reduce ROS after serum depletion in human glioma U251 cells.</td><td>2017-02-24</td><td>Kun Yao; Xufeng Fu; Du Xing; Yan Li; Bing Han; Chen Zexi; Shanshan Yang; Ran Wei; Jiaqi Zhou; Qinghua Cui</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/111567</td></tr><tr><td>mudv1ejd</td><td>b4fc44eabb1f640124fff98990c85dc5c9984aee</td><td>biorxiv</td><td>The translational landscape of Zika virus during infection of mammalian and insect cells</td><td>10.1101/112904</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractZika virus is a single-stranded, positive-sense RNA virus of the family Flaviviridae, which has recently undergone a rapid expansion among humans in the Western Hemisphere. Here, we report a high-resolution map of ribosomal occupancy of the Zika virus genome during infection of mammalian and insect cells, obtained by ribosome profiling. In contrast to some other flaviviruses such as West Nile, we find no evidence for substantial frameshift-induced ribosomal drop-off during translation of the viral polyprotein, indicating that Zika virus must use alternative mechanisms to downregulate levels of catalytically active viral polymerase. We also show that high levels of ribosome-protected fragments map in-frame to two previously overlooked upstream open reading frames (uORFs) initiating at CUG and UUG codons, with likely consequences for the efficiency of polyprotein expression. Curiously, in African isolates of Zika virus, the two uORFs are fused in-frame into a single uORF. A parallel RNA-Seq analysis reveals the 5′ end position of the subgenomic flavivirus RNA in mammalian and insect cells. Together, these provide the first analysis of flavivirus gene expression by ribosome profiling.Author SummaryRecent Zika virus outbreaks have been associated with congenital diseases and neurological complications. An enhanced understanding of the molecular biology of this pathogen may contribute towards the development of improved treatment and control methods. We present a single-codon resolution analysis of Zika virus translation in mammalian and mosquito cells using ribosome profiling. The analysis revealed two hitherto uncharacterized uORFs in the 5′ leader of Zika virus Brazilian isolate PE243, both of which are occupied by ribosomes during infection. In contrast, these two uORFs are fused into a single uORF in African isolates. This observation provides a new avenue for further investigations into potential factors involved in the emergence of Zika virus from a rarely detected pathogen into a major epidemic.</td><td>2017-03-02</td><td>Nerea Irigoyen; Adam M. Dinan; Luke W. Meredith; Ian Goodfellow; Ian Brierley; Andrew E. Firth</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/112904</td></tr><tr><td>d0y1hs88</td><td>null</td><td>biorxiv</td><td>Myeloablation-associated deletion of ORF4 in a human coronavirus 229E infection</td><td>10.1101/113423</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractWe describe metagenomic next-generation sequencing (mNGS) of a human coronavirus 229E from a patient with AML and persistent upper respiratory symptoms, who underwent hematopoietic cell transplantation (HCT). mNGS revealed a 548-nucleotide deletion, which comprised the near entirety of the ORF4 gene, and no minor allele variants were detected to suggest a mixed infection. As part of her pre-HCT conditioning regimen, the patient received myeloablative treatment with cyclophosphamide and 12 Gy total body irradiation. Iterative sequencing and RT-PCR confirmation of 4 respiratory samples over the 4-week peritransplant period revealed that the pre-conditioning strain contained an intact ORF4 gene, while the deletion strain appeared just after conditioning and persisted over a 2.5-week period. This sequence represents one of the largest genomic deletions detected in a human RNA virus and describes large-scale viral mutation associated with myeloablation for HCT.</td><td>2017-03-03</td><td>Alexander L. Greninger; Gregory Pepper; Ryan C. Shean; Anne Cent; Isabel Palileo; Jane M. Kuypers; Joshua T. Schiffer; Keith R. Jerome</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/113423</td></tr><tr><td>0abutzb3</td><td>d91120d80916351170b34715ea4ab141f02c51fb</td><td>biorxiv</td><td>Prophage genomics reveals patterns in phage genome organization and replication</td><td>10.1101/114819</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractTemperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host’s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts’ translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution.</td><td>2017-03-07</td><td>Han Suh Kang; Katelyn McNair; Daniel A. Cuevas; Barbara A. Bailey; Anca M. Segall; Robert A. Edwards</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/114819</td></tr><tr><td>jopeo9gs</td><td>0cb9c296684ca5e71462d825cab2827854a01544</td><td>biorxiv</td><td>p53 is not necessary for DUX4 pathology</td><td>10.1101/118315</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractsFSHD is a genetically dominant myopathy caused by mutations that cause expression of the normally silent DUX4 gene. This transcription factor has been shown to interfere with myogenesis when misexpressed at very low levels in myoblasts, and to cause cell death when overexpressed at high levels. A previous report using adeno-associated virus to deliver high levels of DUX4 to mouse skeletal muscle demonstrated severe pathology that was suppressed on a p53 knockout background, implying that DUX4 acted through the p53 pathway. Here, we investigate the p53-dependence of DUX4 using both in vitro cellular and the transgenic iDUX4[2.7] mouse models. We find that inhibiting p53 has no effect on the cytoxicity of DUX4 in vitro. When crossed onto the p53 null background, we find no suppression of the male-specific lethality or skin phenotypes of the DUX4 transgene, and find that primary myoblasts from this mouse are still killed by DUX4 expression. These data challenge the notion that the p53 pathway is central to the pathogenicity of DUX4.Summary StatementDUX4 is thought to mediate cytopathology through p53. Here, DUX4 is shown to kill primary myoblasts and promote pathological phenotypes in the iDUX4[2.7] mouse model on the p53-null background, calling into question this notion.</td><td>2017-03-19</td><td>Darko Bosnakovski; Erik A. Toso; Olivia O. Recht; Anja Cucak; Abhinav K Jain; Michelle C. Barton; Michael Kyba</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/118315</td></tr><tr><td>4it5c9n2</td><td>f122e125c2cd3580b1a38e27c780b99f671fd78f</td><td>biorxiv</td><td>An amino acid motif in HLA-DRβ1 distinguishes patients with uveitis in juvenile idiopathic arthritis</td><td>10.1101/140954</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractUveitis is a visually-debilitating disorder that affects up to 30% of children with juvenile idiopathic arthritis (JIA). To identify genetic susceptibility loci for uveitis in JIA, we conducted a genome-wide association study comparing 192 JIA-associated uveitis cases with 330 JIA individuals without uveitis. Two cohorts of JIA patients underwent genotyping and quality control. We used an HLA-specific imputation panel to impute HLA-specific amino acids and HLA types, and identified the amino acid serine at position 11 (serine-11) in HLA- DRB1 as associated to increased risk of uveitis (OR = 2.60, p = 5.43×10−10). The signal at serine-11 was female-specific (interaction of sex and serine-11, p = 0.0096). Serine-11 resides in the YST-motif (positions 10-12) in the peptide binding groove of HLA-DRB1. Quantitative binding affinity predictions revealed peptide-binding preferences that distinguish HLA-DRB1 allotypes with the YST-motif. Our findings highlight a genetically distinct, sexually-dimorphic feature of JIA-associated uveitis.</td><td>2017-05-22</td><td>A.J.W. Haasnoot; M.W. Schilham; S.S.M. Kamphuis; P.C.E. Hissink Muller; A. Heiligenhaus; D. Foell; R.A. Ophoff; T.R.D.J. Radstake; A.I. Den Hollander; T.H.C.M. Reinards; S. Hiddingh; N. Schalij-Delfos; E.P.A.H. Hoppenreijs; M.A.J. van Rossum; C. Wouters; R.K. Saurenmann; N. Wulffraat; R. ten Cate; J.H. de Boer; S.L. Pulit; J.J.W. Kuiper</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/140954</td></tr><tr><td>pt0kes0s</td><td>88e3b43319399c339186c02c44d369228c35eef4</td><td>biorxiv</td><td>An anti-cancer binary system activated by bacteriophage HK022 Integrase</td><td>10.1101/147736</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTCancer gene therapy is a great promising tool for cancer therapeutics due to the specific targeting based on the cancerous gene expression background. Binary systems based on site-specific recombination are one of the most effective potential approaches for cancer gene therapy. In these systems, a cancer specific promoter expresses a site-specific recombinase/integrase that in turn controls the expression of a toxin gene. In the current study, we have developed a new HK022 bacteriophage Integrase (Int) based binary system activating a Diphtheria toxin (DTA) gene expression specifically in cancer cells. We have demonstrated the efficiency, and the high specificity of the system in vitro in cell cultures and in vivo in a lung cancer mouse model. Strikingly, different apoptotic and anti-apoptotic factors demonstrated a remarkable efficacy killing capability of the Int-based binary system compared to the conventional hTERT-DTA mono system in the LLC-Kat lung cancer mice model; we observed that the active hTERT promoter down regulation by the transcription factors Mad-1 is the cornerstone of this phenomenon. The new Int-based binary system offers advantages over already known counterparts and may therefore be developed into a safer and efficient cancer treatment technology.</td><td>2017-06-12</td><td>Amer Elias; Itay Spector; Natasha Gritsenko; Yael Zilberstein; Rena Gorovits; Gali Prag; Mikhail Kolot</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/147736</td></tr><tr><td>9e3z9xdd</td><td>0b4b4e5bb8d0d5167eec1e203b5dad283bd364a5</td><td>biorxiv</td><td>Doubling healthy lifespan using drug synergy</td><td>10.1101/153205</td><td>null</td><td>null</td><td>biorxiv</td><td>SummaryPharmacological interventions that target human ageing would extend individual healthspan and result in dramatic economic benefits to rapidly ageing societies worldwide. For such interventions to be contemplated they need to comprise drugs that are efficacious when given to adults and for which extensive human safety data are available. Here we show that dramatic lifespan extension can be achieved in C.elegans by targeting multiple, evolutionarily conserved ageing pathways using drugs that are already in human use. By targeting multiple synergistic ageing pathways, we are able to slow ageing rate, double lifespan and improves healthspan while minimize developmental and fitness trade-offs. Moreover, we established that there is no synergistic benefit in a daf-2 or daf-7 background, implying the involvement of the TGFβ and IGF pathways in this synergy. Employing lipidomics and transcriptomics analysis we found lipid metabolism to be affected resulting in increased monounsaturated fatty acids (MUFA) and decrease membrane peroxidation index. Our best drug combination showed a conserved lifespan extension in fruit flies. To the best of our knowledge, this is the largest lifespan effect ever reported for any adult-onset drug treatment in C. elegans. This drug-repurposing approach, using drugs already approved for humans to target multiple conserved aging pathways simultaneously, could lead to interventions that prevent age-related diseases and overall frailty in a rapidly ageing population.</td><td>2017-06-21</td><td>Tesfahun Dessale; Krishna Chaithanya Batchu; Diogo Barardo; Ng Li Fang; Vanessa Yuk Man Lam; Linfan Xiao; Markus R. Wenk; Nicholas S. Tolwinski; Jan Gruber</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/153205</td></tr><tr><td>9y8r277c</td><td>90e28ff462882ca7a9329beb879f73c2e99430e4</td><td>biorxiv</td><td>Evolutionary proteomics uncovers ciliary signaling components</td><td>10.1101/153437</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTCilia are organelles specialized for movement and signaling. To infer when during animal evolution signaling pathways became associated with cilia, we characterized the proteomes of cilia from three organisms: sea urchins, sea anemones and choanoflagellates. From these ciliomes, we identified 437 high confidence ciliary candidate proteins conserved in mammals, including known regulators of Hh, GPCR and TRP channel signaling. The phylogenetic profiles of their ciliary association indicate that the Hh and GPCR pathways were linked to cilia before the origin of bilateria and TRP channels before the origin of animals. We demonstrated that some of the candidates not previously implicated in ciliary biology localized to cilia and further investigated ENKUR, a TRP channel-interacting protein that we identified in the cilia of all three organisms. In animals, ENKUR is expressed by cells with motile cilia, ENKUR localizes to cilia in diverse organisms and, in both Xenopus laevis and mice, ENKUR is required for patterning the left/right axis. Moreover, mutation of ENKUR causes situs inversus in humans. Thus, proteomic profiling of cilia from diverse eukaryotes defines a conserved ciliary proteome, reveals ancient connections to Hh, GPCR and TRP channel signaling, and uncovers a novel ciliary protein that controls vertebrate development and human disease.</td><td>2017-06-22</td><td>Monika Abedin Sigg; Tabea Menchen; Jeffery Johnson; Chanjae Lee; Semil P. Choksi; Galo Garcia; Henriette Busengdal; Gerard Dougherty; Petra Pennekamp; Claudius Werner; Fabian Rentzsch; Nevan Krogan; John B. Wallingford; Heymut Omran; Jeremy F. Reiter</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/153437</td></tr><tr><td>7xmn8yvc</td><td>null</td><td>biorxiv</td><td>Mapping the drivers of within-host pathogen evolution using massive data sets</td><td>10.1101/155242</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractDifferences among hosts, resulting from genetic variation in the immune system or heterogeneity in drug treatment, can impact within-host pathogen evolution. Identifying such interactions can potentially be achieved through genetic association studies. However, extensive and correlated genetic population structure in hosts and pathogens presents a substantial risk of confounding analyses. Moreover, the multiple testing burden of interaction scanning can potentially limit power. To address these problems, we have developed a Bayesian approach for detecting host influences on pathogen evolution that makes use of vast existing data sets of pathogen diversity to improve power and control for stratification. The approach models key processes, including recombination and selection, and identifies regions of the pathogen genome affected by host factors. Using simulations and empirical analysis of drug-induced selection on the HIV-1 genome we demonstrate the power of the method to recover known associations and show greatly improved precision-recall characteristics compared to other approaches. We build a high-resolution map of HLA-induced selection in the HIV-1 genome, identifying novel epitope-allele combinations.</td><td>2017-06-25</td><td>Duncan S. Palmer; Isaac Turner; Sarah Fidler; John Frater; Philip Goulder; Dominique Goedhals; Kuan-Hsiang Gary Huang; Annette Oxenius; Rodney Phillips; Roger Shapiro; Cloete van Vuuren; Angela R. McLean; Gil McVean</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/155242</td></tr><tr><td>jhwjh12k</td><td>7a51b191fd6575831585eee67a0ac67f209d8fcb</td><td>biorxiv</td><td>Nuclease escape elements protect messenger RNA against cleavage by multiple viral endonucleases</td><td>10.1101/155457</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTDuring lytic Kaposi’s sarcoma-associated herpesvirus (KSHV) infection, the viral endonu-clease SOX promotes widespread degradation of cytoplasmic messenger RNA (mRNA). However, select mRNAs, including the transcript encoding interleukin-6 (IL-6), escape SOX-induced cleavage. IL-6 escape is mediated through a 3’ UTR RNA regulatory element that overrides the SOX targeting mechanism. Here, we reveal that this protective RNA element functions to broadly restrict cleavage by a range of homologous and non-homologous viral endonucleases. However, it does not impede cleavage by cellular endonucleases. The IL-6 protective sequence may be representative of a larger class of nuclease escape elements, as we identified a similar protective element in the GADD45B mRNA. The IL-6 and GADD45B-derived elements display similarities in their sequence, putative structure, and several associated RNA binding proteins. However, the overall composition of their ribonucleoprotein complexes appears distinct, leading to differences in the breadth of nucleases restricted. These findings highlight how RNA elements can selectively control transcript abundance in the background of widespread virus-induced mRNA degradation.AUTHOR SUMMARYThe ability of viruses to control the host gene expression environment is crucial to promote viral infection. Many viruses express factors that reduce host gene expression through widespread mRNA decay. However, some mRNAs escape this fate, like the transcript encoding the immunoregulatory cytokine IL-6 during KSHV infection. IL-6 escape relies on an RNA regulatory element located in its 3’UTR and involves the recruitment of a protective protein complex. Here, we show that this escape extends beyond KSHV to a variety of related and unrelated viral endonucleases. However, the IL-6 element does not protect against cellular endonucleases, revealing for the first time a virus-specific nuclease escape element. We identified a related escape element in the GADD45B mRNA, which displays several similarities with the IL-6 element. However, these elements assemble a largely distinct complex of proteins, leading to differences in the breadth of their protective capacity. Collectively, these findings reveal how a putative new class of RNA elements function to control RNA fate in the background of widespread mRNA degradation by viral endonucleases.</td><td>2017-06-26</td><td>Mandy Muller; Britt A. Glaunsinger</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/155457</td></tr><tr><td>h89scli5</td><td>b5161b031c7f720562e94735a018d1c3c8be3ae5</td><td>biorxiv</td><td>Quantifying the Risk and Cost of Active Monitoring for Infectious Diseases</td><td>10.1101/156497</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractDuring outbreaks of deadly emerging pathogens (e.g., Ebola, MERS-CoV) and bioterror threats (e.g., smallpox), actively monitoring potentially infected individuals aims to limit disease transmission and morbidity. Guidance issued by CDC on active monitoring was a cornerstone of its response to the West Africa Ebola outbreak. There are limited data on how to balance the costs and performance of this important public health activity. We present a framework that estimates the risks and costs of specific durations of active monitoring for pathogens of significant public health concern. We analyze data from New York City’s Ebola active monitoring program over a 16-month period in 2014-2016. For monitored individuals, we identified unique durations of active monitoring that minimize expected costs for those at “low (but not zero) risk” and “some or high risk”: 21 and 31 days, respectively. Extending our analysis to smallpox and MERS-CoV, we found that the optimal length of active monitoring relative to the median incubation period was reduced compared to Ebola due to less variable incubation periods. Active monitoring can save lives but is expensive. Resources can be most effectively allocated by using exposure-risk categories to modify the duration or intensity of active monitoring.</td><td>2017-06-28</td><td>Nicholas G Reich; Justin Lessler; Jay K Varma; Neil M Vora</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/156497</td></tr><tr><td>iisqysqm</td><td>138d4617f4f4984bec9cf2c7540fed533d9e4d6e</td><td>biorxiv</td><td>Automated collection of pathogen-specific diagnostic data for real-time syndromic epidemiological studies</td><td>10.1101/157156</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractHealth-care and public health professionals rely on accurate, real-time monitoring of infectious diseases for outbreak preparedness and response. Early detection of outbreaks is improved by systems that are pathogen-specific. We describe a system, FilmArray® Trend, for rapid disease reporting that is syndrome-based but pathogen-specific. Results from a multiplex molecular diagnostic test are sent directly to a cloud database. www.syndromictrends.com presents these data in near real-time. Trend preserves patient privacy by removing or obfuscating patient identifiers. We summarize the respiratory pathogen results, for 20 organisms from 344,000 patient samples acquired as standard of care testing over the last four years from 20 clinical laboratories in the United States. The majority of pathogens show influenza-like seasonality, rhinovirus has fall and spring peaks and adenovirus and bacterial pathogens show constant detection over the year. Interestingly, the rate of pathogen co-detections, on average 7.7%, matches predictions based on the relative abundance of organisms present.</td><td>2017-07-31</td><td>Lindsay Meyers; Christine C. Ginocchio; Aimie N. Faucett; Frederick S. Nolte; Per H. Gesteland; Amy Leber; Diane Janowiak; Virginia Donovan; Jennifer Dien Bard; Silvia Spitzer; Kathleen A. Stellrecht; Hossein Salimnia; Rangaraj Selvarangan; Stefan Juretschko; Judy A. Daly; Jeremy C. Wallentine; Kristy Lindsey; Franklin Moore; Sharon L. Reed; Maria Aguero-Rosenfeld; Paul D. Fey; Gregory A. Storch; Steve J. Melnick; Christine C. Robinson; Jennifer F. Meredith; Camille V. Cook; Robert K. Nelson; Jay D. Jones; Samuel V. Scarpino; Benjamin M. Althouse; Kirk M. Ririe; Bradley A. Malin; Mark A. Poritz</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/157156</td></tr><tr><td>j5gjipdn</td><td>null</td><td>biorxiv</td><td>Self-assembled Star-shaped Chiroplasmonic Gold Nanoparticles for Ultrasensitive Chiro-immunosensor of Viruses</td><td>10.1101/162412</td><td>null</td><td>null</td><td>biorxiv</td><td>Near field optics and optical tunneling light-matter interaction in the superstructure of chiral nanostructures and semiconductor quantum dots exhibit strong optical rotation activity that may open a new window for chiral-based bioanalytes detection. Herein we report an ultrasensitive, chiro-immunosensor using superstructure of chiral gold nanohybrids (CAu NPs) and quantum dots (QDs). Self-assembling techniques were employed to create asymmetric plasmonic chiral nanostructures for extending the spectral range of circular dichroism (CD) response for obtaining superior plasmonic resonant coupling with the QDs excitonic state; this may help to achieve lower the limit of detection (LOD) values. As a result, the designed probe exhibited avian influenza A (H5N1) viral concentration at picomolar level, a significant improvement in sensitivity in comparison to a non-assembled CAu NPs based chiroassay. The practicability of the proposed sensing system was successfully demonstrated on several virus cultures including, avian influenza A (H4N6) virus, fowl adenovirus and coronavirus in blood samples. The results of our study highlights that exciton-plasmon interaction changes chirality and the self-assembled nanostructures are an efficient strategy for enhancing the sensitivity of plasmonic nanosensors.</td><td>2017-07-12</td><td>Syed Rahin Ahmed; Éva Nagy; Suresh Neethirajan</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/162412</td></tr><tr><td>jzzhpw7h</td><td>3b22eecad8a582436c52284a4db2198a98a94e18</td><td>biorxiv</td><td>DUSP1 regulates apoptosis and cell migration, but not the JIP1-protected cytokine response, during Respiratory Syncytial Virus and Sendai Virus infection</td><td>10.1101/163360</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe host antiviral response involves the induction of interferons and proinflammatory cytokines, but also the activation of cell death pathways, including apoptosis, to limit viral replication and spreading. This host defense is strictly regulated to eliminate the infection while limiting tissue damage that is associated with virus pathogenesis. Post-translational modifications, most notably phosphorylation, are key regulators of the antiviral defense implying an important role of protein phosphatases. Here, we investigated the role of the dual-specificity phosphatase 1 (DUSP1) in the host defense against human respiratory syncytial virus (RSV), a pathogenic virus of the Pneumoviridae family, and Sendai virus (SeV), a model virus being developed as a vector for anti-RSV vaccine. We found that DUSP1 is upregulated before being subjected to proteasomal degradation. DUSP1 does not inhibit the antiviral response, but negatively regulates virus-induced JNK/p38 MAPK phosphorylation. Interaction with the JNK-interacting protein 1 scaffold protein prevents dephosphorylation of JNK by DUSP1, likely explaining that AP-1 activation and downstream cytokine production are protected from DUSP1 inhibition. Importantly, DUSP1 promotes SeV-induced apoptosis and suppresses cell migration in RSV-infected cells. Collectively, our data unveil a previously unrecognized selective role of DUSP1 in the regulation of tissue damage and repair during infections by RSV and SeV.</td><td>2017-07-13</td><td>Alexa C. Robitaille; Elise Caron; Nicolas Zucchini; Espérance Mukawera; Damien Adam; Mélissa K. Mariani; Anaïs Gélinas; Audray Fortin; Emmanuelle Brochiero; Nathalie Grandvaux</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/163360</td></tr><tr><td>yqa7ggqg</td><td>5d9e490190986584d799e2828c6fd9679e5ae159</td><td>biorxiv</td><td>The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics</td><td>10.1101/167999</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat Pteropus alecto (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that P. alecto ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.Significance StatementBats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection.</td><td>2017-07-24</td><td>Pamela C. De La Cruz-Rivera; Mohammed Kanchwala; Hanquan Liang; Ashwani Kumar; Lin-Fa Wang; Chao Xing; John W. Schoggins</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/167999</td></tr><tr><td>8xcplab3</td><td>fb7371ad31cd501854660876a282060402082537</td><td>biorxiv</td><td>MERS-CoV spillover at the camel-human interface</td><td>10.1101/173211</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic virus from camels causing significant mortality and morbidity in humans in the Arabian Peninsula. The epidemiology of the virus remains poorly understood, and while case-based and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential. Here we use existing MERS-CoV sequence data to explore its phylodynamics in two of its known major hosts, humans and camels. We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that human outbreaks in the Arabian peninsula are driven by seasonally varying zoonotic transfer of viruses from camels. Without heretofore unseen evolution of host tropism, MERS-CoV is unlikely to become endemic in humans.</td><td>2017-08-10</td><td>Gytis Dudas; Luiz Max Carvalho; Andrew Rambaut; Trevor Bedford; Ali M. Somily; Mazin Barry; Sarah S. Al Subaie; Abdulaziz A. BinSaeed; Fahad A. Alzamil; Waleed Zaher; Theeb Al Qahtani; Khaldoon Al Jerian; Scott J.N. McNabb; Imad A. Al-Jahdali; Ahmed M. Alotaibi; Nahid A. Batarfi; Matthew Cotten; Simon J. Watson; Spela Binter; Paul Kellam</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/173211</td></tr><tr><td>v7asfaxc</td><td>656ef218f7dcea5555a59cdc29d0f9ec8d5c2a82</td><td>biorxiv</td><td>MERS-CoV NSP16 necessary for IFN resistance and viral pathogenesis</td><td>10.1101/173286</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractCoronaviruses encode a mix of highly conserved and novel genes as well as genetic elements necessary for infection and pathogenesis, raising the possibility for common targets for attenuation and therapeutic design. In this study, we focus on the highly conserved nonstructural protein (NSP) 16, a viral 2’O methyl-transferase (MTase) that encodes critical functions in immune modulation and infection. Using reverse genetics, we disrupted a key motif in the conserved KDKE motif of MERS NSP16 (D130A) and evaluated the effect on viral infection and pathogenesis. While the absence of 2’O MTase activity had only marginal impact on propagation and replication in Vero cells, the MERS dNSP16 mutant demonstrated significant attenuation relative to control both in primary human airway cultures and in vivo. Further examination indicated the MERS dNSP16 mutant had a type I IFN based attenuation and was partially restored in the absence of IFIT molecules. Importantly, the robust attenuation permitted use of MERS dNSP16 as a live attenuated vaccine platform protecting from challenge with a mouse adapted MERS-CoV strain. These studies demonstrate the importance of the conserved 2’O MTase activity for CoV pathogenesis and highlight NSP16 as a conserved universal target for rapid live attenuated vaccine design in an expanding CoV outbreak setting.SignificanceCoronavirus emergence in both human and livestock represents a significant threat to global public health, as evidenced by the sudden emergence of SARS-CoV, MERS-CoV, PEDV and swine delta coronavirus in the 21st century. These studies describe an approach that effectively targets the highly conserved 2’O methyl-transferase activity of coronaviruses for attenuation. With clear understanding of the IFN/IFIT based mechanism, NSP16 mutants provide a suitable target for a live attenuated vaccine platform as well as therapeutic development for both current and future emergent CoV strains. Importantly, other approaches targeting other conserved pan-coronavirus functions have not yet proven effective against MERS-CoV, illustrating the broad applicability of targeting viral 2’O MTase function across coronaviruses.</td><td>2017-08-08</td><td>Vineet D. Menachery; Lisa E. Gralinski; Hugh D. Mitchell; Kenneth H. Dinnon; Sarah R. Leist; Boyd L. Yount; Rachel L. Graham; Eileen T. McAnarney; Kelly G. Stratton; Adam S. Cockrell; Kari Debbink; Amy C. Sims; Katrina M. Waters; Ralph S. Baric</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/173286</td></tr><tr><td>vqtx98xm</td><td>f126b7f9c8ba02a0f872b51cf95d08086e0bd4ac</td><td>biorxiv</td><td>Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations</td><td>10.1101/175562</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3′→5′exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.IMPORTANCEUnique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness.</td><td>2017-08-11</td><td>Kevin W. Graepel; Xiaotao Lu; James Brett Case; Nicole R. Sexton; Everett Clinton Smith; Mark R. Denison</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/175562</td></tr><tr><td>pgfdy23h</td><td>7116ddab89281e7641fb659019c90ee3715f5039</td><td>biorxiv</td><td>The impact of persistent bacterial bronchitis on the pulmonary microbiome of children</td><td>10.1101/181982</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractPersistent bacterial bronchitis is a leading cause of chronic wet cough in young children. This study aimed to characterise the respiratory bacterial microbiota of healthy children and to assess the impact of the changes associated with the development of persistent bacterial bronchitis.Blind, protected brushings were obtained from 20 healthy controls and 24 children with persistent bacterial bronchitis, with an additional directed sample obtained from persistent bacterial bronchitis patients. DNA was extracted, quantified using a 16S rRNA gene quantitative PCR assay prior to microbial community analysis by 16S rRNA gene sequencing.No significant difference in bacterial diversity or community composition (R2 = 0.01, P = 0.36) was observed between paired blind and non-blind brushes, showing that blind brushings are a valid means of accessing the airway microbiota. This has important implications for collecting lower respiratory samples from healthy children. A significant decrease in bacterial diversity (P < 0.001) and change in community composition (R2 = 0.08, P = 0.004) was observed between controls and patients. Bacterial communities within patients with PBB were dominated by Proteobacteria, and indicator species analysis showed that Haemophilus and Neisseria were significantly associated with the patient group. In 15 (52.9%) cases the dominant organism by sequencing was not identified by standard routine clinical culture.The bacteria present in the lungs of patients with persistent bacterial bronchitis were less diverse in terms of richness and evenness. The results validate the clinical diagnosis, and suggest that more attention to bacterial communities in children with chronic cough may lead to more rapid recognition of this condition with earlier treatment and reduction in disease burden.</td><td>2017-08-29</td><td>Leah Cuthbertson; Vanessa Craven; Lynne Bingle; William O.C.M. Cookson; Mark L. Everard; Miriam F. Moffatt</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/181982</td></tr><tr><td>p3b4yk5d</td><td>3d690de86ca6e200f4f9cac61b004e47ad753aae</td><td>biorxiv</td><td>Mouse hepatitis virus nsp14 exoribonuclease activity is required for resistance to innate immunity</td><td>10.1101/182196</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTCoronaviruses (CoV) are positive-sense RNA viruses that infect numerous mammalian and avian species and are capable of causing severe and lethal disease in humans. CoVs encode several innate immune antagonists that interact with the host innate immune response to facilitate efficient viral replication. CoV non-structural protein 14 (nsp14) encodes 3'-to-5' exoribonuclease activity (ExoN), which performs a proofreading function and is required for high-fidelity replication. Outside of the order Nidovirales, arenaviruses are the only RNA viruses that encode an ExoN, which functions to degrade dsRNA replication intermediates. In this study, we tested the hypothesis that CoV ExoN may also function to antagonize the innate immune response. We demonstrate that viruses lacking ExoN activity [ExoN(-)] are sensitive to cellular pretreatment with interferon beta (IFN-β) in a dose-dependent manner. In addition, ExoN(-) virus replication was attenuated in wild-type bone marrow-derived macrophages (BMMs) and partially restored in interferon alpha/beta receptor deficient (IFNAR-/-) BMMs. ExoN(-) virus replication did not result in IFN-β gene expression, and in the presence of an IFN-β-mediated antiviral state, ExoN(-) viral RNA levels were not substantially reduced relative to untreated. However, ExoN(-) virus generated from IFN-β pretreated cells had reduced specific infectivity and decreased relative fitness, suggesting that ExoN(-) virus generated during an antiviral state is less viable to establish a subsequent infection. Overall, our data suggest MHV ExoN activity is required for resistance to the innate immune response and antiviral mechanisms affecting the viral RNA sequence and/or an RNA modification act on viruses lacking ExoN activity.</td><td>2017-08-29</td><td>James Brett Case; Yize Li; Ruth Elliott; Xiaotao Lu; Kevin W. Graepel; Nicole R. Sexton; Everett Clinton Smith; Susan R. Weiss; Mark R. Denison</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/182196</td></tr><tr><td>g55d5ijx</td><td>08a22278486e12768ce186677a6a89663d24586f</td><td>biorxiv</td><td>Complemented palindrome small RNAs first discovered from SARS coronavirus</td><td>10.1101/185876</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractIn this study, we reported for the first time the existence of complemented palindrome small RNAs (cpsRNAs) and proposed cpsRNAs and palindrome small RNAs (psRNAs) as a novel class of small RNAs. The first discovered cpsRNA UCUUUAACAAGCUUGUUAAAGA from SARS coronavirus named SARS-CoV-cpsR-22 contained 22 nucleotides perfectly matching its reverse complementary sequence. Further sequence analysis supported that SARS-CoV-cpsR-22 originated from bat betacoronavirus. The results of RNAi experiments showed that one 19-nt segment of SARS-CoV-cpsR-22 significantly induced cell apoptosis. These results suggested that SARS-CoV-cpsR-22 could play a role in SARS-CoV infection or pathogenicity. The discovery of psRNAs and cpsRNAs paved the way to find new markers for pathogen detection and reveal the mechanisms in the infection or pathogenicity from a different point of view. The discovery of psRNAs and cpsRNAs also broaden the understanding of palindrome motifs in animal of plant genomes.</td><td>2017-09-07</td><td>Chang Liu; Ze Chen; Wenyuan Shen; Deshui Yu; Siyu Li; Yue Hu; Haishuo Ji; Wenjun Bu; Qingsong Wang; Shan Gao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/185876</td></tr><tr><td>vok1dni9</td><td>090b6c8b3df30bc248221869f673a2d970caa1b9</td><td>medrxiv</td><td>Quantifying the success of measles vaccination campaigns in the Rohingya refugee camps</td><td>10.1101/19008417</td><td>null</td><td>null</td><td>medrvix</td><td>In the wake of the Rohingya population's mass migration from Myanmar, one of the world's largest refugee settlements was constructed in Cox's Bazar, Bangladesh to accommodate nearly 900,000 new refugees. Refugee populations are particularly vulnerable to infectious disease outbreaks due to many population and environmental factors. A large measles outbreak, with over 2,500 cases, occurred among the Rohingya population between September and December 2017. Here, we estimate key epidemiological parameters and use a dynamic mathematical model of measles transmission to evaluate the effectiveness of the reactive vaccination campaigns in the refugee camps. We also estimate the potential for subsequent outbreaks under different vaccination coverage scenarios. Our modeling results highlight the success of the vaccination campaigns in rapidly curbing transmission and emphasize the public health importance of maintaining high levels of vaccination in this population, where high birth rates and historically low vaccination coverage rates create suitable conditions for future measles outbreaks.</td><td>2019-10-05</td><td>Taylor Chin; Caroline O. Buckee; Ayesha S. Mahmud</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/19008417</td></tr><tr><td>jwesa12u</td><td>10525ac89e46be4cb9cb9fd1131d28411a902047</td><td>medrxiv</td><td>Using Digital Surveillance Tools for Near Real-Time Mapping of the Risk of International Infectious Disease Spread: Ebola as a Case Study</td><td>10.1101/19011940</td><td>null</td><td>null</td><td>medrvix</td><td>In our increasingly interconnected world, it is crucial to understand the risk of an outbreak originating in one country or region and spreading to the rest of the world. Digital disease surveillance tools such as ProMED and HealthMap have the potential to serve as important early warning systems as well as complement the field surveillance during an ongoing outbreak. Here we present a flexible statistical model that uses data produced from digital surveillance tools (ProMED and HealthMap) to forecast short term incidence trends in a spatially explicit manner. The model was applied to data collected by ProMED and HealthMap during the 2013-2016 West African Ebola epidemic. The model was able to predict each instance of international spread 1 to 4 weeks in advance. Our study highlights the potential and limitations of using publicly available digital surveillance data for assessing outbreak dynamics in real-time.</td><td>2019-11-15</td><td>Sangeeta Bhatia; Britta Lassmann; Emily Cohn; Malwina Carrion; Moritz U.G. Kraemer; Mark Herringer; John Brownstein; Larry Madoff; Anne Cori; Pierre Nouvellet</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/19011940</td></tr><tr><td>npd2qf7m</td><td>44b6eea3629b4e13595dc49813ee39a0d20dc5f2</td><td>biorxiv</td><td>Evolutionary rate shifts suggest species-specific adaptation events in HIV-1 and SIV</td><td>10.1101/190769</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe process of molecular adaptation following a cross-species virus transmission event is currently poorly understood. Here, we identified 137 protein sites that experienced deceleration in their rate of evolution along the HIV-1/SIV phylogeny, likely indicating gain-of-function and consequent adaptation. The majority of such events occurred in parallel to cross-species transmission events and varied between HIV-1 groups, indicating independent adaptation strategies. The evolutionary rate decelerations we found were particularly prominent in accessory proteins that counteract host antiviral restriction factors, suggesting that these factors are a major barrier to viral adaptation to a new host. Surprisingly, we observed that the non-pandemic HIV-1 group O, derived from gorillas, exhibited more rate deceleration events than the pandemic group M, derived from chimpanzees. We suggest that the species barrier is higher when the genetic distance of the hosts increases. Our approach paves the way for subsequent studies on cross-species transfers in other major pathogens.</td><td>2017-09-19</td><td>Maoz Gelbart; Adi Stern</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/190769</td></tr><tr><td>6riyqn4k</td><td>aa5a99e67d3e887b3fa88575fb0d4a662d0a60ce</td><td>biorxiv</td><td>Modeling Epidemics: A Primer and Numerus Software Implementation</td><td>10.1101/191601</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractEpidemiological models are dominated by SEIR (Susceptible, Exposed, Infected and Removed) dynamical systems formulations and their elaborations. These formulations can be continuous or discrete, deterministic or stochastic, or spatially homogeneous or heterogeneous, the latter often embracing a network formulation. Here we review the continuous and discrete deterministic and discrete stochastic formulations of the SEIR dynamical systems models, and we outline how they can be easily and rapidly constructed using the Numerus Model Builder, a graphically-driven coding platform. We also demonstrate how to extend these models to a metapopulation setting using both the Numerus Model Builder network and geographical mapping tools.</td><td>2017-09-22</td><td>Wayne M. Getz; Richard Salter; Oliver Muellerklein; Hyun S. Yoon; Krti Tallam</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/191601</td></tr><tr><td>uw76y16g</td><td>9b17a9fa6be11d8b3dbed9f3dea2516020083bd3</td><td>biorxiv</td><td>Fast Virome Explorer: A Pipeline for Virus and Phage Identification and Abundance Profiling in Metagenomics Data</td><td>10.1101/196998</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractIdentifying viruses and phages in a metagenomics sample has important implication in improving human health, preventing viral outbreaks, and developing personalized medicine. With the rapid increase in data files generated by next generation sequencing, existing tools for identifying and annotating viruses and phages in metagenomics samples suffer from expensive running time. In this paper, we developed a stand-alone pipeline, FastViromeExplorer, for rapid identification and abundance quantification of viruses and phages in big metagenomic data. Both real and simulated data validated FastViromeExplorer as a reliable tool to accurately identify viruses and their abundances in large data, as well as in a time efficient manner.</td><td>2017-10-02</td><td>Saima Sultana Tithi; Roderick V. Jensen; Liqing Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/196998</td></tr><tr><td>ifla4aix</td><td>7395de48b2402e5e1561a93b99e8aae8728c6ff1</td><td>biorxiv</td><td>RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain and is required for ubiquitination</td><td>10.1101/200410</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTTRIM25 is a novel RNA-binding protein and a member of the Tripartite Motif (TRIM) family of E3 ubiquitin ligases, which plays a pivotal role in the innate immune response. Almost nothing is known about its RNA-related roles in cell biology. Furthermore, its RNA-binding domain has not been characterized. Here, we reveal that RNA-binding activity of TRIM25 is mediated by its PRY/SPRY domain, which we postulate to be a novel RNA-binding domain. Using CLIP-seq and SILAC-based co-immunoprecipitation assays, we uncover TRIM25’s endogenous RNA targets and protein binding partners. Finally, we show that the RNA-binding activity of TRIM25 is important for its ubiquitin ligase function. These results reveal new insights into the molecular roles and characteristics of RNA-binding E3 ubiquitin ligases and demonstrate that RNA could be an essential factor for their biological functions.</td><td>2017-10-09</td><td>Nila Roy Choudhury; Gregory Heikel; Maryia Trubitsyna; Peter Kubik; Jakub Stanislaw Nowak; Shaun Webb; Sander Granneman; Christos Spanos; Juri Rappsilber; Alfredo Castello; Gracjan Michlewski</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/200410</td></tr><tr><td>v4spn75m</td><td>null</td><td>biorxiv</td><td>IFITM proteins inhibit HIV-1 protein synthesis</td><td>10.1101/201525</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractInterferon induced transmembrane proteins (IFITMs) inhibit the cellular entry of a broad range of viruses, but it has been suspected that for HIV-1 IFITMs may also inhibit a post-integration replicative step. We show that IFITM expression reduces HIV-1 viral protein synthesis by preferentially excluding viral mRNA transcripts from translation and thereby restricts viral production. Codon-optimization of proviral DNA rescues viral translation, implying that IFITM-mediated restriction requires recognition of viral RNA elements. In addition, we find that expression of the viral accessory protein Nef can help overcome the IFITM-mediated inhibition of virus production. Our studies identify a novel role for IFITMs in inhibiting HIV replication at the level of translation, but show that the effects can be overcome by the lentiviral protein Nef.</td><td>2017-10-11</td><td>Wing-Yiu Jason Lee; Chen Liang; Richard D Sloan</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/201525</td></tr><tr><td>43cxzudl</td><td>null</td><td>biorxiv</td><td>Delving Below the Species Level To Characterize the Ecological Diversity within the Global Virome: An Exploration of West Nile Virus</td><td>10.1101/2019.12.12.874214</td><td>null</td><td>null</td><td>biorxiv</td><td>SummaryEfforts to describe the diversity of viruses have largely focused on classifying viruses at the species level. However, substantial ecological diversity, both in virulence level and host range, is known within virus species. Here we demonstrate a proof of concept for easily discovering ecological diversity within a virus species taxon. We have focused on the West Nile Virus to take advantage of its broad host range in nature. We produced a genome-based phylogeny of world diversity of WNV and then used Ecotype Simulation 2 to hypothesize demarcation of genomes into 69 putative ecotypes (ecologically distinct populations), based only on clustering of genome sequences. Then we looked for evidence of ecological divergence among ecotypes based on differences in host bird associations within the Connecticut-New York region. Our results indicated significant heterogeneity among ecotypes for their associations with different bird hosts. Ecological diversity within other zoonotic viruses could be easily discovered using this approach. Opportunities for extending this line of research to human associations of virus ecotypes are limited by missing geographic metadata on human samples.</td><td>2019-12-13</td><td>Tong Kong; Kelly Mei; Ammie Wang; Danny Krizanc; Frederick M. Cohan</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.12.874214</td></tr><tr><td>2fcl6f4i</td><td>97027e08cb413795cf19064338efd049fe5d491b</td><td>biorxiv</td><td>Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts</td><td>10.1101/2019.12.13.876037</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAdenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes.</td><td>2019-12-13</td><td>Alexander M. Price; Katharina E. Hayer; Daniel P. Depledge; Angus C. Wilson; Matthew D. Weitzman</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.13.876037</td></tr><tr><td>bqwmx5dy</td><td>93f67ffe7803061de9b19c4dfa346b3aa97aa4eb</td><td>biorxiv</td><td>Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction</td><td>10.1101/2019.12.15.877092</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSingle particle cryo-EM is a powerful method for studying proteins and other biological macromolecules. Many of these molecules comprise regions with varying structural properties including disorder, flexibility, and partial occupancy. These traits make computational 3D reconstruction from 2D images challenging. Detergent micelles and lipid nanodiscs, used to keep membrane proteins in solution, are common examples of locally disordered structures that can negatively affect existing iterative refinement algorithms which assume rigidity (or spatial uniformity). We introduce a cross-validation approach to derive non-uniform refinement, an algorithm that automatically regularizes 3D density maps during iterative refinement to account for spatial variability, yielding dramatically improved resolution and 3D map quality. We find that in common iterative refinement methods, regularization using spatially uniform filtering operations can simultaneously over- and under-regularize local regions of a 3D map. In contrast, non-uniform refinement removes noise in disordered regions while retaining signal useful for aligning particle images. Our results include state-of-the-art resolution 3D reconstructions of multiple membrane proteins with molecular weight as low as 90kDa. These results demonstrate that higher resolutions and improved 3D density map quality can be achieved even for small membrane proteins, an important use case for single particle cryo-EM, both in structural biology and drug discovery. Non-uniform refinement is implemented in the cryoSPARC software package and has already been used successfully in several notable structural studies.</td><td>2019-12-16</td><td>Ali Punjani; Haowei Zhang; David J. Fleet</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.15.877092</td></tr><tr><td>eohjkqc4</td><td>null</td><td>biorxiv</td><td>Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses</td><td>10.1101/2019.12.16.875872</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractRNA viruses rapidly mutate, which can result in increased virulence, increased escape from vaccine protection, and false negative detection results. Targeted detection methods have a limited ability to detect unknown viruses and often provide insufficient data to detect coinfections or identify antigenic variants. Random, deep sequencing is a method that can more fully detect and characterize RNA viruses and is often coupled with molecular techniques or culture methods for viral enrichment. Viral culture coupled with third-generation sequencing were tested for the ability to detect and characterize RNA viruses. Cultures of bovine viral diarrhea virus, canine distemper virus, epizootic hemorrhagic disease virus, infectious bronchitis virus, two influenza A viruses, and porcine respiratory and reproductive syndrome virus were sequenced on the MinION platform using a random, reverse primer in a strand-switching reaction, coupled with PCR-based barcoding. Reads were taxonomically classified and used for reference-based sequence building using a stock personal computer. This method accurately detected and identified complete coding sequence genomes with a minimum of 20× coverage depth for all seven viruses, including a sample containing two viruses. Each lineage-typing region had at least 26× coverage depth for all viruses. Furthermore, analyzing the canine distemper virus sample through a pipeline devoid of canine distemper virus reference sequences modeled the ability of this protocol to detect unknown viruses. These results show the ability of this technique to detect and characterize dsRNA, negative- and positive-sense ssRNA, nonsegmented, and segmented RNA viruses.</td><td>2019-12-16</td><td>Kelsey T. Young; Kevin K. Lahmers; Holly S. Sellers; David E. Stallknecht; Rebecca L. Poulson; Jerry T. Saliki; S. Mark Tompkins; Ian Padykula; Chris Siepker; Elizabeth W. Howerth; Michelle Todd; James B. Stanton</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.16.875872</td></tr><tr><td>t2azgyp8</td><td>b0899a264af548a89d649154aec569889717b295</td><td>medrxiv</td><td>Stochastic challenges to interrupting helminth transmission</td><td>10.1101/2019.12.17.19013490</td><td>null</td><td>null</td><td>medrvix</td><td>Predicting the effect of different programmes designed to control both the morbidity induced by helminth infections and parasite transmission is greatly facilitated by the use of mathematical models of transmission and control impact. In such models, it is essential to account for as many sources of uncertainty -- natural, or otherwise -- to ensure robustness in prediction and to accurately depict variation around an expected outcome. In this paper, we investigate how well the standard deterministic models match the predictions made using individual-based stochastic simulations. We also explore how well concepts which derive from deterministic models, such as ‘breakpoints’ in transmission, apply in the stochastic world. Employing an individual based stochastic model framework we also investigate how transmission and control are affected by the migration of infected people into a defined community. To give our study focus we consider the control of soil-transmitted helminths (STH) by mass drug administration (MDA), though our methodology is readily applicable to the other helminth species such as the schistosome parasites and the filarial worms. We show it is possible to define a ‘stochastic breakpoint’ where much noise surrounds the expected deterministic breakpoint. We also discuss the concept of the ‘interruption of transmission’ independent of the ‘breakpoint’ concept where analyses of model behaviour illustrate the current limitations of deterministic models to account for the ‘fade-out’ or transmission extinction behaviour in simulations. The analyses based on migration confirm a relationship between the infected human migration rate per unit of time and the death rate of infective stages that are released into the free-living environment (eggs or larvae depending on the STH species) that create the reservoir of infection which in turn determines the likelihood that control activities aim at chemotherapeutic treatment of the human host will eliminate transmission. The development of a new stochastic simulation code for STH in the form of a publicly-available open-source python package which includes features to incorporate many population stratifications, different control interventions including mass drug administration (with defined frequency, coverage levels and compliance patterns) and inter-village human migration is also described.</td><td>2019-12-18</td><td>Robert J Hardwick; Marleen Werkman; James E Truscott; Roy M Anderson</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.17.19013490</td></tr><tr><td>bi49e7ne</td><td>a555fbeeffc44dfdb5087c83395f0f743654cbba</td><td>biorxiv</td><td>A novel totivirus alters gene expression and vacuolar morphology in Malassezia cells and induces a TLR3-mediated inflammatory immune response</td><td>10.1101/2019.12.17.880526</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractMost fungal viruses have been identified in plant pathogens, whereas the presence of viral particles in human pathogenic fungi is less well studied. In the present study, we observed extrachromosomal double-stranded RNA (dsRNA) segments in various clinical isolates of Malassezia species. Malassezia is the most dominant fungal genus on the human skin surface, and species in this group are considered to be etiological factors in various skin diseases including dandruff, seborrheic dermatitis, and atopic dermatitis. We identified novel dsRNA segments and our sequencing results revealed that the virus, named MrV40, belongs to the Totiviridae family and contains an additional satellite dsRNA segment encoding a novel protein. The transcriptome of virus-infected M. restricta cells was compared to that of virus-free cells, and the results showed that transcripts involved in ribosomal biosynthesis were down regulated and those involved in energy production and programmed cell death were increased in abundance. Moreover, transmission electron microscopy revealed significantly larger vacuoles for virus-infected M. restricta cells, indicating that MrV40 infection dramatically altered M. restricta physiology. Our analysis also revealed that a viral nucleic acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells (BMDCs) and this result suggests that a viral element contributes to the pathogenesis of Malassezia.ImportanceMalassezia is the most dominant fungal genus on the human skin surface and is associated with various skin diseases including dandruff and seborrheic dermatitis. Among Malassezia species, M. restricta is the most widely observed species on the human skin. In the current study, we identified a novel dsRNA virus, named MrV40, in M. restricta and characterized the sequences and structure of the viral genome along with an independent satellite dsRNA viral segment. Moreover, we found altered expression of genes involved in ribosomal synthesis and programmed cell death, indicating that virus infection altered the physiology of the fungal host cells. Our data also showed that the viral nucleic acid from MrV40 induces a TLR3-mediated inflammatory immune response in bone marrow-derived dendritic cells (BMDCs), indicating that a viral element likely contributes to the pathogenesis of Malassezia. This is the first study to identify and characterize a novel mycovirus in Malassezia.</td><td>2019-12-19</td><td>Minji Park; Yong-Joon Cho; Donggyu Kim; Chul-Su Yang; Shi Mun Lee; Thomas L. Dawson; Satoshi Nakamizo; Kenji Kabashima; Yang Won Lee; Won Hee Jung</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.17.880526</td></tr><tr><td>e5p4metz</td><td>8c223fd63ba6a924ecd3df8d32bcf0556758cb95</td><td>biorxiv</td><td>Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications</td><td>10.1101/2019.12.18.880849</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial (E. coli) and archaeal (H. volcanii, P. furiosus) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism.</td><td>2019-12-19</td><td>Felix Grünberger; Robert Knüppel; Michael Jüttner; Martin Fenk; Andreas Borst; Robert Reichelt; Winfried Hausner; Jörg Soppa; Sebastien Ferreira-Cerca; Dina Grohmann</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.18.880849</td></tr><tr><td>qw2tsrxm</td><td>73d80c8f5780d70bd8d343188c56e898e91557b6</td><td>biorxiv</td><td>Ca2+ ions promote fusion of Middle East Respiratory Syndrome coronavirus with host cells and increase infectivity</td><td>10.1101/2019.12.18.881391</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractMiddle East respiratory syndrome coronavirus (MERS-CoV) is a major emerging zoonotic infectious disease. Since its first outbreak in 2012, the virus has repeatedly transmitted from camels to humans with 2,468 confirmed cases, causing 851 deaths. To date, there are no efficacious drugs and vaccines against MERS-CoV, increasing its potential to cause a public health emergency. A critical step in the life cycle of MERS-CoV is the fusion with the host cell with its spike (S) protein as main determinant of viral entry. Proteolytic cleavage of S exposes its fusion peptide (FP), which initiates membrane fusion. Previous studies on the related severe acute respiratory syndrome coronavirus (SARS-CoV) FP have shown that calcium (Ca2+) plays an important role for fusogenic activity via a Ca2+ binding pocket with conserved glutamic acid (E) and aspartic acid (D) residues. SARS-CoV and MERS-CoV FP share a high sequence homology and here, we investigated whether Ca2+ is required for MERS-CoV fusion by substituting E and D residues in the MERS-CoV FP with neutrally charged alanines. Upon verifying mutant cell surface expression and proteolytic cleavage, we tested the mutants ability to mediate infection of pseudo-particles (PPs) on host cells without and with Ca2+. Our results demonstrate that intracellular Ca2+ enhances MERS-CoV WT PPs infection by approximately two-fold and that E891 is a crucial residue for Ca2+ interaction. Electron spin resonance revealed that this enhancement could be attributed to Ca2+ increasing MERS-CoV FP fusion-relevant membrane ordering. Intriguingly, isothermal calorimetry titration showed that MERS-CoV FP binds one Ca2+, as opposed to SARS-CoV FP which binds two. Our data suggests that there are significant differences in FP-Ca2+ interactions of MERS-CoV and SARS-CoV FP despite their high sequence similarity and that the number of Ca2+ ions interacting with the FP has implications on the fusion dynamics of the virus.</td><td>2019-12-19</td><td>Marco R. Straus; Tiffany Tang; Alex L. Lai; Annkatrin Flegel; Miya Bidon; Jack H. Freed; Susan Daniel; Gary R. Whittaker</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.18.881391</td></tr><tr><td>7a5nxxar</td><td>63781701d3aac3b1640caae24b2c8c2a06c8e6ab</td><td>biorxiv</td><td>Comparison of alternative models of human movement and the spread of disease</td><td>10.1101/2019.12.19.882175</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractPredictive models for the spatial spread of infectious diseases has received much attention in recent years as tools for the management of infectious diseas outbreaks. Prominently, various versions of the so-called gravity model, borrowed from transportation theory, have been used. However, the original literature suggests that the model has some potential misspecifications inasmuch as it fails to capture higher-order interactions among population centers. The fields of economics, geography and network sciences holds alternative formulations for the spatial coupling within and among conurbations. These includes Stouffer’s rank model, Fotheringham’s competing destinations model and the radiation model of Simini et al. Since the spread of infectious disease reflects mobility through the filter of age-specific susceptibility and infectivity and since, moreover, disease may alter spatial behavior, it is essential to confront with epidemiological data on spread. To study their relative merit we, accordingly, fit variants of these models to the uniquely detailed dataset of prevaccination measles in the 954 cities and towns of England and Wales over the years 1944-65 and compare them using a consistent likelihood framework. We find that while the gravity model is a reasonable first approximation, both Stouffer’s rank model, an extended version of the radiation model and the Fotheringham competing destinations model provide significantly better fits, Stouffer’s model being the best. Through a new method of spatially disaggregated likelihoods we identify areas of relatively poorer fit, and show that it is indeed in densely-populated conurbations that higher order spatial interactions are most important. Our main conclusion is that it is premature to narrow in on a single class of models for predicting spatial spread of infectious disease. The supplemental materials contain all code for reproducing the results and applying the methods to other data sets.Author summaryThe ability to predict how infectious disease will spread is of great importance in the face of the numerous emergent and re-emergent pathogens that currently threatening human well-being. We identified a variety of alternative models that predict human mobility as as a function of population distribution across a landscape. These consider some models that account for pair-wise interactions between population centers, as well as some that allow for higher-order interactions. We trained the models using a uniquely rich spatiotemporal data set on pre-vaccination measles in England and wales (1944-65), which comprises more than a million records from 954 cities and towns. Likelihood rankings of the different models reveal strong evidence for higher-order interactions in the form of competition among cities as destinations for travelers and, thus, dilution of spatial transmission. The currently most commonly used so-called ‘gravity’ models were far from the best in capturing spatial disease dynamics.</td><td>2019-12-19</td><td>Ottar N. Bjørnstad; Bryan T. Grenfell; Cecile Viboud; Aaron A. King</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.19.882175</td></tr><tr><td>8b2ookby</td><td>d83e3c028de1950c4f8dedae21eb90f90c4ed6c3</td><td>biorxiv</td><td>Translational profiling of macrophages infected with Leishmania donovani identifies mTOR- and eIF4A-sensitive immune-related transcripts</td><td>10.1101/2019.12.20.884338</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe protozoan parasite Leishmania donovani (L. donovani) causes visceral leishmaniasis, a chronic infection which is fatal when untreated. While previous studies showed that L. donovani reprograms transcription to subvert host cell functions, it remains unclear whether the parasite also alters host mRNA translation to establish a successful infection. To assess this, we compared transcriptome-wide translation in primary mouse macrophages infected with L. donovani promastigotes or amastigotes using polysome-profiling. This identified ample selective changes in translation (3,127 transcripts) which were predicted to target central cellular functions by inducing synthesis of proteins related to chromatin remodeling and RNA metabolism while inhibiting those related to intracellular trafficking and antigen presentation. Parallel quantification of protein and mRNA levels for a set of transcripts whose translation was activated upon L. donovani infection (Papbpc1, Eif2ak2, and Tgfb) confirmed, as indicated by polysome-profiling, increased protein levels despite largely unaltered mRNA levels. Mechanistic in silico analyses suggested activated translation depending on the kinase mTOR (e.g. Pabpc1) and the RNA helicase eIF4A (e.g. Tgfb) during infection. Accordingly, treatment with mTOR inhibitors torin-1 or rapamycin reversed L. donovani-induced PABPC1 without affecting corresponding transcript levels. Similarly, the production of TGF-β decreased in presence of the eIF4A inhibitor silvestrol despite unaltered Tgfb mRNA levels. Consistent with parasite modulation of host eIF4A-sensitive translation to promote infection, silvestrol suppressed L. donovani replication within macrophages. In contrast, parasite survival was favored under mTOR inhibition. In summary, infection-associated changes in translation of mTOR- and eIF4A-sensitive mRNAs contribute to modulate mRNA metabolism and immune responses in L. donovani-infected macrophages. Although the net outcome of such translation programs favours parasite propagation, individual translation programs appear to have opposing roles during L. donovani infection, thereby suggesting their selective targeting as key for therapeutic effects.Author SummaryFine-tuning the efficiency of mRNA translation into proteins allows cells to tailor their responses to stress without the need for synthesizing new mRNA molecules. It is well established that the protozoan parasite Leishmania donovani alters transcription of specific genes to subvert host cell functions. However, discrepancies between transcriptomic and proteomic data suggest that post-transcriptional regulatory mechanisms also contribute to modulate host gene expression programs during L. donovani infection. Herein, we report that one third of protein-coding mRNAs expressed in macrophages are differentially translated upon infection with L. donovani. Our computational analyses reveal that subsets of mRNAs encoding functionally related proteins share the same directionality of translational regulation, which is likely to impact metabolic and microbicidal activity of infected cells. We also show that upregulated translation of transcripts that encode central regulators of mRNA metabolism and inflammation is sensitive to the activation of mTOR or eIF4A during infection. Finally, we observe that inhibition of eIF4A activity reduces parasite survival within macrophages while selective blockade of mTOR has the opposite effect. Thus, our study points to a dual role for translational control of host gene expression during L. donovani infection and suggests that novel regulatory nodes could be targeted for therapeutic intervention.</td><td>2019-12-20</td><td>Visnu Chaparro; Louis-Philippe Leroux; Laia Masvidal; Julie Lorent; Tyson E. Graber; Aude Zimmermann; Guillermo Arango Duque; Albert Descoteaux; Tommy Alain; Ola Larsson; Maritza Jaramillo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.20.884338</td></tr><tr><td>ahs2wdus</td><td>48df5b06710467cbe5b41dd851e946fc936ecf15</td><td>biorxiv</td><td>Low basal expression and slow induction of IFITM3 puts immune cells at risk of influenza A infection</td><td>10.1101/2019.12.20.885590</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe interferon-induced transmembrane protein, IFITM3, has been shown to restrict influenza virus infection in murine and in vitro settings for ten years, but no explanation has been found to explain why this virus infection is so highly contagious and infects most individuals it comes in contact with. We confirm that the expression level of IFITM3 plays a role in determining the level of viral infection through manipulation of IFITM3 levels with interferon (IFN) stimulation and overexpression systems. Low basal expression may put some immune cells, including lymphocytes and lung-resident macrophages, at risk of influenza virus infection. Investigating the induction of IFITM3 by IFN, we find a strong preference for Type I IFN in IFITM3 induction in both cell lines and primary human cells. While myeloid cells can increase expression following stimulation by Type I IFN, lymphocytes show minimal induction of IFITM3 following IFN stimulation, suggesting that they are always at risk of viral infection. Surprisingly, we found that the time it takes for maximal induction of IFITM3 is relatively slow for an interferon-stimulated gene at around 36 hours. Low basal expression and slow induction of IFITM3 could increase the risk of influenza virus infection in selected immune cells.ImportanceInfluenza virus infection remains one of the top ten threats to global health, causing significant deaths and hospitalisations across the world each year. Understanding mechanisms for controlling influenza virus infection remain a priority. The interferon-induced transmembrane protein IFITM3 can restrict influenza infection by limiting replication of the virus. The precise mechanisms of how IFITM3 reduced replication of influenza are unknown, although it is predicted to prevent release of viral contents into the cytosol by preventing pore formation on the endosomal compartments where it is suggested to reside. Here we have shown that the expression level of IFITM3 is important in determining the control of influenza virus infection. We find an expression pattern for IFITM3 that varies based on cell type, tissue locality, differentiation state and cell naivety, all of which highlights cells that may be at the highest risk of influenza infection.</td><td>2019-12-23</td><td>Dannielle Wellington; Zixi Yin; Liwei Zhang; Jessica Forbester; Kerry Kite; Henry Laurenson-Schafer; Shokouh Makvandi-Nejad; Boquan Jin; Emma Bowes; Krishnageetha Manoharan; David Maldonado-Perez; Clare Verill; Ian Humphreys; Tao Dong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.20.885590</td></tr><tr><td>ts91ivwf</td><td>106f10eaae4fb67aca5b2bd737ce2bf1c27fa243</td><td>biorxiv</td><td>Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community</td><td>10.1101/2019.12.21.885921</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractMost emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare—but detectable—bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens.</td><td>2019-12-23</td><td>Daniel J. Becker; Kelly A. Speer; Alexis M. Brown; M. Brock Fenton; Alex D. Washburne; Sonia Altizer; Daniel G. Streicker; Raina K. Plowright; Vladimir E. Chizhikov; Nancy B. Simmons; Dmitriy V. Volokhov</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2019.12.21.885921</td></tr><tr><td>2vecg9op</td><td>09c9fcabc66a106e01ef42247cbd86b6d85bd67f</td><td>biorxiv</td><td>Mono-ADP-ribosylation by ARTD10 restricts Chikungunya virus replication by interfering with the proteolytic activity of nsP2</td><td>10.1101/2020.01.07.896977</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA subset of intracellular mono-ADP-ribosyltransferases diphtheria toxin-like (ARTDs, aka mono-PARPs) is induced by type I interferons. Some of these mono-ARTDs feature antiviral activity while certain RNA viruses, including Chikungunya virus (CHIKV), encode mono-ADP-ribosylhydrolases, suggesting a role for mono-ADP-ribosylation (MARylation) in host-virus conflicts. CHIKV expresses four non-structural proteins (nsP1-nsP4), with nsP3 containing a macrodomain that hydrolyzes and thereby reverses protein MARylation in vitro and in cells. This de-MARylation activity is essential as hydrolase inactivating mutations result in replication defective virus. However, the substrates of MARylation during CHIKV infection are unknown and thus it is unclear how the macrodomain contributes to virus replication and how mono-ARTD-dependent MARylation confers antiviral immunity. We identified ARTD10 and ARTD12 as restriction factors for CHIKV replication in a catalytic activity-dependent manner. CHIKV replication requires processing of the non-structural polyprotein nsP1-4 by the nsP2-encoded protease and the assembly of the four individual nsPs into a functional replication complex. Expression of ARTD10 and ARTD12 resulted in a reduction of processed nsPs. Similarly, MAR hydrolase inactive CHIKV replicon mutants revealed a decrease in processed nsPs, comparable to an nsP2 protease defective mutant. This suggested that the macrodomain contributes to nsP2 protease activity. In support, a hydrolase-deficient virus was complemented by a protease-deficient virus. We hypothesized that MARylation regulates the proteolytic function of nsP2. Indeed, we found that nsP2 is MARylated by ARTD10. This inhibited nsP2 protease activity, thereby preventing polyprotein processing and consequently virus replication. This inhibition was antagonized by the MAR hydrolase activity of nsP3. Together, our findings provide a mechanistic explanation for the need of the viral MAR hydrolase for efficient replication of CHIKV.Author SummaryInfectious diseases still pose major health threats. Especially fast evolving viruses find ever new strategies to manipulate the immune response. With climate warming and increased human mobility vector-borne pathogens like Chikungunya virus (CHIKV) spread and cause world-wide epidemics. Beyond the acute phase, CHIKV patients regularly suffer from chronic rheumatism. This entails a decline in life quality and an economic burden. To date no drugs are approved and the mode of pathogenesis remains elusive. Here we describe a mechanistic function of the CHIKV nsP3 macrodomain. We found that the viral nsP2 is mono-ADP-ribosylated interfering with its auto-proteolytic function. The nsP3 macrodomain removes this modification and restores the protease activity that is essential for replication. Because macrodomains are highly conserved they might represent broad antiviral targets.</td><td>2020-01-08</td><td>Sarah Krieg; Fabian Pott; Laura Eckei; Maud Verheirstraeten; Mareike Bütepage; Barbara Lippok; Christine Goffinet; Bernhard Lüscher; Patricia Verheugd</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.07.896977</td></tr><tr><td>2f9nc2to</td><td>b80d9de7750d627a0189fcd2dfd82c7efcb1c079</td><td>biorxiv</td><td>An interaction network of RNA-binding proteins involved in Drosophila oogenesis</td><td>10.1101/2020.01.08.899146</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractDuring Drosophila oogenesis, the localization and translational regulation of maternal transcripts relies on RNA-binding proteins (RBPs). Many of these RBPs localize several mRNAs and may have additional direct interaction partners to regulate their functions. Using immunoprecipitation from whole Drosophila ovaries coupled to mass spectrometry, we examined protein-protein associations of 6 GFP-tagged RBPs expressed at physiological levels. Analysis of the interaction network and further validation in human cells allowed us to identify 26 previously unknown associations, besides recovering several well characterized interactions. We identified interactions between RBPs and several splicing factors, providing links between nuclear and cytoplasmic events of mRNA regulation. Additionally, components of the translational and RNA decay machineries were selectively co-purified with some baits, suggesting a mechanism for how RBPs may regulate maternal transcripts. Given the evolutionary conservation of the studied RBPs, the interaction network presented here provides the foundation for future functional and structural studies of mRNA localization across metazoans.</td><td>2020-01-09</td><td>Prashali Bansal; Johannes Madlung; Kristina Schaaf; Boris Macek; Fulvia Bono</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.08.899146</td></tr><tr><td>8iet67u3</td><td>e18620d5cff022d18aa23da2fa4634331fa54860</td><td>biorxiv</td><td>Altered polarization of PAR-2 signaling during airway epithelial remodeling</td><td>10.1101/2020.01.09.900555</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTBackgroundProtease-activated receptor 2 (PAR-2) is activated by proteases involved in allergy and triggers airway epithelial secretion and inflammation. PAR-2 is normally expressed basolaterally in differentiated nasal ciliated cells.ObjectiveWe tested if epithelial remodeling during diseases characterized by loss of cilia and squamous metaplasia may alter PAR-2 polarization.MethodsEndogenous PAR-2 responses were measured by live cell calcium and cilia imaging, measurement of fluid secretion, and quantification of cytokines. We utilized airway squamous cell lines, primary differentiated air-liquid interface cultures, and tissue explants. Cells were exposed to disease-related modifiers that alter epithelial morphology, including IL-13, cigarette smoke condensate, and retinoic acid deficiency. We used concentrations and exposure times that altered epithelial morphology without causing breakdown of the epithelial barrier, likely reflecting early disease states.ResultsPAR-2 signaling in airway squamous cells activated calcium and inflammatory responses. Squamous cells cultured at air liquid interface (ALI) responded to PAR-2 agonists applied both apically and basolaterally. Primary well-differentiated nasal epithelial ALI cultures responded only to basolateral PAR-2 stimulation. Primary cultures exposed to IL-13, cigarette smoke condensate, or reduced retinoic acid responded to both apical and basolateral PAR-2 stimulation. Nasal polyp tissue, but not control middle turbinate, exhibited apical calcium responses to PAR-2 stimulation. However, isolated ciliated cells from both polyp and turbinate maintained basolateral PAR-2 polarization.ConclusionsSquamous metaplasia and/or loss of cilia enhances apical PAR-2 responses. Altered PAR-2 polarization in dedifferentiated or remodeled epithelia may contribute to increased sensitivity to inhaled protease allergens in inflammatory airway diseases.</td><td>2020-01-09</td><td>Ryan M. Carey; Jenna R. Freund; Benjamin M. Hariri; Nithin D. Adappa; James N. Palmer; Robert J. Lee</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.09.900555</td></tr><tr><td>wtarrhf0</td><td>0624a12abfe85c8b5070850d912a2db4cd453236</td><td>biorxiv</td><td>TALC: Transcription Aware Long Read Correction</td><td>10.1101/2020.01.10.901728</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTMotivationLong-read sequencing technologies are invaluable for determining complex RNA transcript architectures but are error-prone. Numerous “hybrid correction” algorithms have been developed for genomic data that correct long reads by exploiting the accuracy and depth of short reads sequenced from the same sample. These algorithms are not suited for correcting more complex transcriptome sequencing data.ResultsWe have created a novel algorithm called TALC (Transcription Aware Long Read Correction) which models changes in RNA expression and isoform representation in a weighted De-Bruijn graph to correct long reads from transcriptome studies. We show that transcription aware correction by TALC improves the accuracy of the whole spectrum of downstream RNA-seq applications and is thus necessary for transcriptome analyses that use long read technology.Availability and ImplementationTALC is implemented in C++ and available at https://gitlab.igh.cnrs.fr/lbroseus/TALC.Contactwilliam.ritchie@igh.cnrs.fr</td><td>2020-01-11</td><td>Lucile Broseus; Aubin Thomas; Andrew J. Oldfield; Dany Severac; Emeric Dubois; William Ritchie</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.10.901728</td></tr><tr><td>ahrrphm8</td><td>0015023cc06b5362d332b3baf348d11567ca2fbb</td><td>biorxiv</td><td>The RNA pseudoknots in foot-and-mouth disease virus are dispensable for genome replication but essential for the production of infectious virus</td><td>10.1101/2020.01.10.901801</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe positive stranded RNA genomes of picornaviruses comprise a single large open reading frame flanked by 5′ and 3′ untranslated regions (UTRs). Foot-and-mouth disease virus (FMDV) has an unusually large 5′ UTR (1.3 kb) containing five structural domains. These include the internal ribosome entry site (IRES), which facilitates initiation of translation, and the cis-acting replication element (cre). Less well characterised structures are a 5′ terminal 360 nucleotide stem-loop, a variable length poly-C-tract of approximately 100-200 nucleotides and a series of two to four tandemly repeated pseudoknots (PKs). We investigated the structures of the PKs by selective 2′ hydroxyl acetylation analysed by primer extension (SHAPE) analysis and determined their contribution to genome replication by mutation and deletion experiments. SHAPE and mutation experiments confirmed the importance of the previously predicted PK structures for their function. Deletion experiments showed that although PKs are not essential for replication, they provide genomes with a competitive advantage. However, although replicons and full-length genomes lacking all PKs were replication competent, no infectious virus was rescued from genomes containing less than one PK copy. This is consistent with our earlier report describing the presence of putative packaging signals in the PK region.</td><td>2020-01-11</td><td>Joseph C. Ward; Lidia Lasecka-Dykes; Chris Neil; Oluwapelumi Adeyemi; Sarah Gold; Niall McLean; Caroline Wright; Morgan R. Herod; David Kealy; Emma Warner; Donald P. King; Tobias J. Tuthill; David J. Rowlands; Nicola J. Stonehouse</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.10.901801</td></tr><tr><td>8kced06y</td><td>f8e8a9c6f5662c1739be031e6134284ef08ea8b2</td><td>biorxiv</td><td>Programmable low-cost DNA-based platform for viral RNA detection</td><td>10.1101/2020.01.12.902452</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractViral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ∼24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.</td><td>2020-01-16</td><td>Lifeng Zhou; Arun Richard Chandrasekaran; Jibin Abraham Punnoose; Gaston Bonenfant; Stephon Charles; Oksana Levchenko; Pheonah Badu; Cassandra Cavaliere; Cara T. Pager; Ken Halvorsen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.12.902452</td></tr><tr><td>sjicr24s</td><td>de3ae7971891058db2f4b36cd216531e6f533332</td><td>biorxiv</td><td>Crystal structure of the giant panda MHC class I complex: first insights into the viral peptide presentation profile in the bear family</td><td>10.1101/2020.01.15.908608</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe viral cytotoxic T lymphocyte (CTL) epitope peptides presented by classical MHC-I molecules require the assembly of a peptide-MHC-I-β2m (aka pMHC-I) trimolecular complex for TCR recognition, which is the critical activation link for triggering antiviral T cell immunity. Ursidae includes 5 genera and 8 species; however, research on T cell immunology in this family, especially structural immunology, is lacking. In this study, the structure of the key trimolecular complex pMHC-1 (aka pAime-128), which binds a peptide from canine distemper virus, was solved for the first time using giant panda as a representative species of Ursidae. The structural characteristics of the giant panda pMHC-I complex, including the unique pockets in the peptide-binding groove (PBG), were analyzed in detail. Comparing the panda pMHC-I to others in the bear family and extending the comparison to other mammals revealed distinct features. The interaction between MHC-I and β2m, the features of pAime-128 involved in TCR docking and CD8 binding, the anchor sites in the PBG, and the CTL epitopes of potential viruses that infect pandas were concretely clarified. Unique features of pMHC-I viral antigen presentation in the panda were revealed by solving the three-dimensional structure of pAime-128. The distinct characteristics of pAime-128 indicate an unusual event that emerged during the evolution of the MHC system in the bear family. These results provide a new platform for research on panda CTL immunity and the design of vaccines for application in the bear family.IMPORTANCEUrsidae includes 5 genera and 8 species; however, the study of its immunology, especially structural immunology, is extremely rare to date. In this paper, we first crystallized the key complex pMHC-I, taking the giant panda as its representative species. Structural characteristics of the giant panda pMHC-I complexes, contains the unique pockets of PBG were analyzed in detail. Comparison of the panda pMHC-I in the bear family and other mammals, almost definite features was displayed. Meanwhile, the interaction between HC and LV, the unique features of pMHC-I in the CD8 binding and TCR docking, validation of anchor site in the PBG, and epitopes of potential viruses infected with the pandas, were concretely clarified. These unique characteristics of pMHC-I clearly indicate an unusual situation during the evolution of MHC molecules in the endangered pandas. These results also provide a novel platform for further study of panda T cell immunology and vaccines.</td><td>2020-01-16</td><td>Hongyu Yuan; Lizhen Ma; Lijie Zhang; Xiaoying Li; Chun Xia</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.15.908608</td></tr><tr><td>bf4qpsy7</td><td>26cb6703ca72bf97887abbc29a40b1bd9d7890f4</td><td>biorxiv</td><td>Hotspots of aberrant enhancer activity in fibrolamellar carcinoma reveal molecular mechanisms of oncogenesis and intrinsic drug resistance</td><td>10.1101/2020.01.18.911297</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractFibrolamellar carcinoma (FLC) is a rare, therapeutically intractable liver cancer that disproportionately affects youth. Although FLC tumors exhibit a distinct gene expression profile, the causative transcriptional mechanisms remain unclear. Here we used chromatin run-on sequencing to discover approximately 7,000 enhancers and 141 enhancer hotspots activated in FLC relative to non-malignant liver. Detailed bioinformatic analyses revealed aberrant ERK/MEK signaling and candidate master transcriptional regulators. We also defined the genes most strongly associated with aberrant FLC enhancer activity, including CA12 and SLC16A14. Treatment of FLC cell models with inhibitors of CA12 or SLC16A14 independently reduced cell viability and/or significantly enhanced the effect of MEK inhibitor cobimetinib. These findings highlight new molecular targets for drug development as well as novel drug combination approaches.</td><td>2020-01-18</td><td>Timothy A. Dinh; Ramja Sritharan; F. Donelson Smith; Adam B. Francisco; Rosanna K. Ma; Rodica P. Bunaciu; Matt Kanke; Charles G. Danko; Andrew P. Massa; John D. Scott; Praveen Sethupathy</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.18.911297</td></tr><tr><td>v4mbry22</td><td>8b88d01563538e4abd29c3ba493c534af30664cc</td><td>biorxiv</td><td>A mathematical model for simulating the transmission of Wuhan novel Coronavirus</td><td>10.1101/2020.01.19.911669</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAs reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.</td><td>2020-01-19</td><td>Tianmu Chen; Jia Rui; Qiupeng Wang; Zeyu Zhao; Jing-An Cui; Ling Yin</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.19.911669</td></tr><tr><td>3qds5j1p</td><td>null</td><td>biorxiv</td><td>Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China</td><td>10.1101/2020.01.20.913368</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractDetailed genomic and structure-based analysis of a new coronavirus, namely 2019-nCoV, showed that the new virus is a new type of bat coronavirus and is genetically fairly distant from the human SARS coronavirus. Structure analysis of the spike (S) protein of this new virus showed that its S protein only binds weakly to the ACE2 receptor on human cells whereas the human SARS coronavirus exhibits strongly affinity to the ACE receptor. These findings suggest that the new virus does not readily transmit between humans and should theoretically not able to cause very serious human infection. These data are important to guide design of infection control policy and inform the public on the nature of threat imposed by 2019-nCov when results of direct laboratory tests on this virus are not expected to be available in the near future.</td><td>2020-01-21</td><td>Ning Dong; Xuemei Yang; Lianwei Ye; Kaichao Chen; Edward Wai-Chi Chan; Mengsu Yang; Sheng Chen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.20.913368</td></tr><tr><td>t9bt70f2</td><td>a05179c57d5e1cdddc839ff3c0b70a614efbb6cb</td><td>biorxiv</td><td>Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm</td><td>10.1101/2020.01.21.914044</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe recent outbreak of pneumonia in Wuhan, China caused by the 2019 Novel Coronavirus (2019-nCoV) emphasizes the importance of detecting novel viruses and predicting their risks of infecting people. In this report, we introduced the VHP (Virus Host Prediction) to predict the potential hosts of viruses using deep learning algorithm. Our prediction suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially the severe acute respiratory syndrome coronavirus (SARS-CoV), Bat SARS-like Coronaviruses and the Middle East respiratory syndrome coronavirus (MERS-CoV). Based on our prediction, compared to the Coronaviruses infecting other vertebrates, bat coronaviruses are assigned with more similar infectivity patterns with 2019-nCoVs. Furthermore, by comparing the infectivity patterns of all viruses hosted on vertebrates, we found mink viruses show a closer infectivity pattern to 2019-nCov. These consequences of infectivity pattern analysis illustrate that bat and mink may be two candidate reservoirs of 2019-nCov.These results warn us to beware of 2019-nCoV and guide us to further explore the properties and reservoir of it.One Sentence SummaryIt is of great value to identify whether a newly discovered virus has the risk of infecting human. Guo et al. proposed a virus host prediction method based on deep learning to detect what kind of host a virus can infect with DNA sequence as input. Applied to the Wuhan 2019 Novel Coronavirus, our prediction demonstrated that several vertebrate-infectious coronaviruses have strong potential to infect human. This method will be helpful in future viral analysis and early prevention and control of viral pathogens.</td><td>2020-01-24</td><td>Qian Guo; Mo Li; Chunhui Wang; Peihong Wang; Zhencheng Fang; Jie tan; Shufang Wu; Yonghong Xiao; Huaiqiu Zhu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.21.914044</td></tr><tr><td>f1ixbzx8</td><td>403d885a33958731c4607ea632453e25a89e2b57</td><td>biorxiv</td><td>Novel ionophores active against La Crosse virus identified through rapid antiviral screening</td><td>10.1101/2020.01.21.914929</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBunyaviruses are significant human pathogens, causing diseases ranging from hemorrhagic fevers to encephalitis. Among these viruses, La Crosse virus (LACV), a member of the California serogroup, circulates in the eastern and midwestern United States. While LACV infection is often asymptomatic, dozens of cases of encephalitis are reported yearly. Unfortunately, no antivirals have been approved to treat LACV infection. Here, we developed a method to rapidly test potential antivirals against LACV infection. From this screen, we identified several potential antiviral molecules, including known antivirals. Additionally, we identified many novel antivirals that exhibited antiviral activity without affecting cellular viability. Valinomycin, a potassium ionophore, was among our top targets. We found that valinomycin exhibited potent anti-LACV activity in multiple cell types in a dose-dependent manner. Valinomycin did not affect particle stability or infectivity, suggesting that it may preclude virus replication by altering cellular potassium ions, a known determinant of LACV entry. We extended these results to other ionophores and found that the antiviral activity of valinomycin extended to other viral families including bunyaviruses (Rift Valley fever virus, Keystone virus), enteroviruses (Coxsackievirus, rhinovirus), flavirivuses (Zika), and coronaviruses (229E and MERS-CoV). In all viral infections, we observed significant reductions in virus titer in valinomycin-treated cells. In sum, we demonstrate the importance of potassium ions to virus infection, suggesting a potential therapeutic target to disrupt virus replication.ImportanceNo antivirals are approved for the treatment of bunyavirus infection. The ability to rapidly screen compounds and identify novel antivirals is one means to accelerate drug discovery for viruses with no approved treatments. We used this approach to screen hundreds of compounds against La Crosse virus, an emerging bunyavirus that causes significant disease, including encephalitis. We identified several known and previously unidentified antivirals. We focused on a potassium ionophore, valinomycin, due to its promising in vitro antiviral activity. We demonstrate that valinomycin, as well as a selection of other ionophores, exhibits activity against La Crosse virus as well as several other distantly related bunyaviruses. We finally observe that valinomycin has activity against a wide array of human viral pathogens, suggesting that disrupting potassium ion homeostasis with valinomycin may be a potent host pathway to target to quell virus infection.</td><td>2020-01-23</td><td>Zachary J. Sandler; Michelle N. Vu; Vineet D. Menachery; Bryan C. Mounce</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.21.914929</td></tr><tr><td>o47v5vgw</td><td>null</td><td>biorxiv</td><td>Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin</td><td>10.1101/2020.01.22.914952</td><td>null</td><td>null</td><td>biorxiv</td><td>Since the SARS outbreak 18 years ago, a large number of severe acute respiratory syndrome related coronaviruses (SARSr-CoV) have been discovered in their natural reservoir host, bats1-4. Previous studies indicated that some of those bat SARSr-CoVs have the potential to infect humans5-7. Here we report the identification and characterization of a novel coronavirus (nCoV-2019) which caused an epidemic of acute respiratory syndrome in humans, in Wuhan, China. The epidemic, started from December 12th, 2019, has caused 198 laboratory confirmed infections with three fatal cases by January 20th, 2020. Full-length genome sequences were obtained from five patients at the early stage of the outbreak. They are almost identical to each other and share 79.5% sequence identify to SARS-CoV. Furthermore, it was found that nCoV-2019 is 96% identical at the whole genome level to a bat coronavirus. The pairwise protein sequence analysis of seven conserved non-structural proteins show that this virus belongs to the species of SARSr-CoV. The nCoV-2019 virus was then isolated from the bronchoalveolar lavage fluid of a critically ill patient, which can be neutralized by sera from several patients. Importantly, we have confirmed that this novel CoV uses the same cell entry receptor, ACE2, as SARS-CoV.</td><td>2020-01-23</td><td>Peng Zhou; Xing-Lou Yang; Xian-Guang Wang; Ben Hu; Lei Zhang; Wei Zhang; Hao-Rui Si; Yan Zhu; Bei Li; Chao-Lin Huang; Hui-Dong Chen; Jing Chen; Yun Luo; Hua Guo; Ren-Di Jiang; Mei-Qin Liu; Ying Chen; Xu-Rui Shen; Xi Wang; Xiao-Shuang Zheng; Kai Zhao; Quan-Jiao Chen; Fei Deng; Lin-Lin Liu; Bing Yan; Fa-Xian Zhan; Yan-Yi Wang; Geng-Fu Xiao; Zheng-Li Shi</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.22.914952</td></tr><tr><td>tyhwtt7j</td><td>9daed5e675ac113d1939056dcebbb1f32ec0d7ec</td><td>biorxiv</td><td>Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</td><td>10.1101/2020.01.22.915660</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractOver the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</td><td>2020-01-22</td><td>Michael Letko; Vincent Munster</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.22.915660</td></tr><tr><td>mt8h4rk5</td><td>6cb02c7565f6f74a3d165e14196de5e9e87d2d04</td><td>medrxiv</td><td>Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions</td><td>10.1101/2020.01.23.20018549</td><td>null</td><td>null</td><td>medrvix</td><td>Since first identified, the epidemic scale of the recently emerged novel coronavirus (2019-nCoV) in Wuhan, China, has increased rapidly, with cases arising across China and other countries and regions. using a transmission model, we estimate a basic reproductive number of 3.11 (95%CI, 2.39-4.13); 58-76% of transmissions must be prevented to stop increasing; Wuhan case ascertainment of 5.0% (3.6-7.4); 21022 (11090-33490) total infections in Wuhan 1 to 22 January.</td><td>2020-01-24</td><td>Jonathan M Read; Jessica RE Bridgen; Derek AT Cummings; Antonia Ho; Chris P Jewell</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.23.20018549</td></tr><tr><td>px70bft2</td><td>24ff82a421f1a33db3ebaa22652bd62e0cb77c57</td><td>biorxiv</td><td>Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</td><td>10.1101/2020.01.23.916395</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackgroundsAn ongoing outbreak of a novel coronavirus (2019-nCoV) pneumonia hit a major city of China, Wuhan, December 2019 and subsequently reached other provinces/regions of China and countries. We present estimates of the basic reproduction number, R0, of 2019-nCoV in the early phase of the outbreak.MethodsAccounting for the impact of the variations in disease reporting rate, we modelled the epidemic curve of 2019-nCoV cases time series, in mainland China from January 10 to January 24, 2020, through the exponential growth. With the estimated intrinsic growth rate (γ), we estimated R0 by using the serial intervals (SI) of two other well-known coronavirus diseases, MERS and SARS, as approximations for the true unknown SI.FindingsThe early outbreak data largely follows the exponential growth. We estimated that the mean R0 ranges from 2.24 (95%CI: 1.96-2.55) to 3.58 (95%CI: 2.89-4.39) associated with 8-fold to 2-fold increase in the reporting rate. We demonstrated that changes in reporting rate substantially affect estimates of R0.ConclusionThe mean estimate of R0 for the 2019-nCoV ranges from 2.24 to 3.58, and significantly larger than 1. Our findings indicate the potential of 2019-nCoV to cause outbreaks.</td><td>2020-01-24</td><td>Shi Zhao; Qianyin Lin; Jinjun Ran; Salihu S Musa; Guangpu Yang; Weiming Wang; Yijun Lou; Daozhou Gao; Lin Yang; Daihai He; Maggie H Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.23.916395</td></tr><tr><td>59rcogfc</td><td>fe66306680e5be9820e2cc3742b663a782eb1b31</td><td>biorxiv</td><td>Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China</td><td>10.1101/2020.01.23.916726</td><td>null</td><td>null</td><td>biorxiv</td><td>We present a timely evaluation of the Chinese 2019-nCov epidemic in its initial phase, where 2019-nCov demonstrates comparable transmissibility but lower fatality rates than SARS and MERS. A quick diagnosis that leads to case isolation and integrated interventions will have a major impact on its future trend. Nevertheless, as China is facing its Spring Festival travel rush and the epidemic has spread beyond its borders, further investigation on its potential spatiotemporal transmission pattern and novel intervention strategies are warranted.</td><td>2020-01-25</td><td>Mingwang Shen; Zhihang Peng; Yanni Xiao; Lei Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.23.916726</td></tr><tr><td>jjkf5gg3</td><td>671b44ae9bec89cacf7ae9e7b2a79649a9b66773</td><td>biorxiv</td><td>Pattern of early human-to-human transmission of Wuhan 2019-nCoV</td><td>10.1101/2020.01.23.917351</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTOn December 31, 2019, the World Health Organization was notified about a cluster of pneumonia of unknown aetiology in the city of Wuhan, China. Chinese authorities later identified a new coronavirus (2019-nCoV) as the causative agent of the outbreak. As of January 23, 2020, 655 cases have been confirmed in China and several other countries. Understanding the transmission characteristics and the potential for sustained human-to-human transmission of 2019-nCoV is critically important for coordinating current screening and containment strategies, and determining whether the outbreak constitutes a public health emergency of international concern (PHEIC). We performed stochastic simulations of early outbreak trajectories that are consistent with the epidemiological findings to date. We found the basic reproduction number, R0, to be around 2.2 (90% high density interval 1.4—3.8), indicating the potential for sustained human-to-human transmission. Transmission characteristics appear to be of a similar magnitude to severe acute respiratory syndrome-related coronavirus (SARS-CoV) and the 1918 pandemic influenza. These findings underline the importance of heightened screening, surveillance and control efforts, particularly at airports and other travel hubs, in order to prevent further international spread of 2019-nCoV.</td><td>2020-01-24</td><td>Julien Riou; Christian L. Althaus</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.23.917351</td></tr><tr><td>j99qg2a0</td><td>e4cb4af2f4ec6f023096afcf7b3d4410584e1393</td><td>biorxiv</td><td>Designing Effective small interfering RNA for Post-Transcriptional Silencing of Human GREM1: A Comprehensive Bioinformatics Approach</td><td>10.1101/2020.01.23.917559</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTHuman gremlin-1 is a physiologically versatile signaling molecule that has been associated with several human diseases including cancer. The ability of gremlin-1 to induce fibrosis in organs and transduce angiogenesis makes it a target for cancer therapy. RNAi-based therapy has proven to be very efficient and specific in tumor growth inhibition. The efficacy and specificity of siRNA-mediated gene silencing depends on the designing approaches. Here, empirical guidelines for siRNA design and comprehensive target site availability analysis were used to select effective siRNA from a plethora of potential candidates designed using several computation algorithms. Then, the selected siRNA candidates were subjected to stringent similarity searches in order to obtain siRNA candidates with reduced off-target effects (high specificity). The best candidates were compared to experimentally successful gremlin-1 siRNAs in order to predict the silencing potency of the selected siRNAs. siRNA-6 (sense strand: 5’-CCAAGAAAUUCACUACCAU-3’), siRNA-7 (sense strand: 5’-CCAUGAUGGUCACACUCAA-3’) and siRNA-47 (sense strand: 5’-GGCCCAGCACAAUGACUCA-3’) were predicted to be highly effective siRNA candidates for gremlin-1 silencing. These siRNAs can be considered for RNAi-based therapy because off-target effects are predicted to be minimal.</td><td>2020-01-24</td><td>Asiedu Ebenezer</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.23.917559</td></tr><tr><td>u2w2bb2p</td><td>a3116a1ea2e157c8855023d968cc0f0bd6c9846f</td><td>biorxiv</td><td>The 2019-new coronavirus epidemic: evidence for virus evolution</td><td>10.1101/2020.01.24.915157</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThere is concern about a new coronavirus, the 2019-nCoV, as a global public health threat. In this article, we provide a preliminary evolutionary and molecular epidemiological analysis of this new virus. A phylogenetic tree has been built using the 15 available whole genome sequence of 2019-nCoV and 12 whole genome sequences highly similar sequences available in gene bank (5 from SARS, 2 from MERS and 5 from Bat SARS-like Coronavirus). FUBAR analysis shows that the Nucleocapsid and the Spike Glycoprotein has some sites under positive pressure while homology modelling helped to explain some molecular and structural differences between the viruses. The phylogenetic tree showed that 2019.nCoV significantly clustered with Bat SARS-like Coronavirus sequence isolated in 2015, whereas structural analysis revealed mutation in S and nucleocapsid proteins. From these results, 2019nCoV could be considered a coronavirus distinct from SARS virus, probably transmitted from bats or another host where mutations conferred upon it the ability to infect humans.</td><td>2020-01-24</td><td>Domenico Benvenuto; Marta Giovannetti; Alessandra Ciccozzi; Silvia Spoto; Silvia Angeletti; Massimo Ciccozzi</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.24.915157</td></tr><tr><td>qpaig5tm</td><td>b6e92eda6c26b456165422d4e5552b81754c4a82</td><td>biorxiv</td><td>2019-20 Wuhan coronavirus outbreak: Intense surveillance is vital for preventing sustained transmission in new locations</td><td>10.1101/2020.01.24.919159</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe outbreak of pneumonia originating in Wuhan, China, has generated 830 confirmed cases, including 26 deaths, as of 24 January 2020. The virus (2019-nCoV) has spread elsewhere in China and to other countries, including South Korea, Thailand, Japan and USA. Fortunately, there has not yet been evidence of sustained human-to-human transmission outside of China. Here we assess the risk of sustained transmission whenever the coronavirus arrives in other countries. Data describing the times from symptom onset to hospitalisation for 47 patients infected in the current outbreak are used to generate an estimate for the probability that an imported case is followed by sustained human-to-human transmission. Under the assumptions that the imported case is representative of the patients in China, and that the 2019-nCoV is similarly transmissible to the SARS coronavirus, the probability that an imported case is followed by sustained human-to-human transmission is 0.37. However, if the mean time from symptom onset to hospitalisation can be halved by intense surveillance, then the probability that an imported case leads to sustained transmission is only 0.005. This emphasises the importance of current surveillance efforts in countries around the world, to ensure that the ongoing outbreak will not become a large global epidemic.</td><td>2020-01-25</td><td>R.N. Thompson</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.24.919159</td></tr><tr><td>5jai745w</td><td>bb97f90ff5bfd41f2f1148c5cbdb257fdba535df</td><td>biorxiv</td><td>Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China</td><td>10.1101/2020.01.24.919183</td><td>null</td><td>null</td><td>biorxiv</td><td>Emerging and re-emerging infectious diseases, such as SARS, MERS, Zika and highly pathogenic influenza present a major threat to public health1–3. Despite intense research effort, how, when and where novel diseases appear are still the source of considerable uncertainly. A severe respiratory disease was recently reported in the city of Wuhan, Hubei province, China. At the time of writing, at least 62 suspected cases have been reported since the first patient was hospitalized on December 12nd 2019. Epidemiological investigation by the local Center for Disease Control and Prevention (CDC) suggested that the outbreak was associated with a sea food market in Wuhan. We studied seven patients who were workers at the market, and collected bronchoalveolar lavage fluid (BALF) from one patient who exhibited a severe respiratory syndrome including fever, dizziness and cough, and who was admitted to Wuhan Central Hospital on December 26th 2019. Next generation metagenomic RNA sequencing4 identified a novel RNA virus from the family Coronaviridae designed WH-Human-1 coronavirus (WHCV).Phylogenetic analysis of the complete viral genome (29,903 nucleotides) revealed that WHCV was most closely related (89.1% nucleotide similarity similarity) to a group of Severe Acute Respiratory Syndrome (SARS)-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China and that have a history of genomic recombination. This outbreak highlights the ongoing capacity of viral spill-over from animals to cause severe disease in humans.</td><td>2020-01-25</td><td>Fan Wu; Su Zhao; Bin Yu; Yan-Mei Chen; Wen Wang; Yi Hu; Zhi-Gang Song; Zhao-Wu Tao; Jun-Hua Tian; Yuan-Yuan Pei; Ming-Li Yuan; Yu-Ling Zhang; Fa-Hui Dai; Yi Liu; Qi-Min Wang; Jiao-Jiao Zheng; Lin Xu; Edward C. Holmes; Yong-Zhen Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.24.919183</td></tr><tr><td>hab8h5gz</td><td>null</td><td>biorxiv</td><td>From SARS-CoV to Wuhan 2019-nCoV Outbreak: Similarity of Early Epidemic and Prediction of Future Trends</td><td>10.1101/2020.01.24.919241</td><td>null</td><td>null</td><td>biorxiv</td><td>This manuscript has been withdrawn as it was submitted without the full consent of all the authors. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.</td><td>2020-01-25</td><td>Zeliang Chen; Wenjun Zhang; Yi Lu; Cheng Guo; Zhongmin Guo; Conghui Liao; Xi Zhang; Yi Zhang; Xiaohu Han; Qianlin Li; W. Ian Lipkin; Jiahai Lu</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.24.919241</td></tr><tr><td>9d8k7b23</td><td>e29b287d326be9c2c0a7dcd7a81cb49cc2125360</td><td>biorxiv</td><td>Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes</td><td>10.1101/2020.01.24.919282</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, Aedes aegypti, and its microbiome, which has been shown to influence virus transmission success. We examined the ‘metavirome’ of four populations of Ae. aegypti mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in Ae. aegypti that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.ImportanceSince 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks.</td><td>2020-01-25</td><td>Sean M. Boyles; Carla N. Mavian; Esteban Finol; Maria Ukhanova; Caroline J. Stephenson; Gabriela Hamerlinck; Seokyoung Kang; Caleb Baumgartner; Mary Geesey; Israel Stinton; Kate Williams; Derrick K. Mathias; Mattia Prosperi; Volker Mai; Marco Salemi; Eva A. Buckner; John A. Lednicky; Adam R. Rivers; Rhoel R. Dinglasan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.24.919282</td></tr><tr><td>t1iagum7</td><td>null</td><td>biorxiv</td><td>MRCA time and epidemic dynamics of the 2019 novel coronavirus</td><td>10.1101/2020.01.25.919688</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe 2019 novel coronavirus (2019-nCoV) have emerged from Wuhan, China. Studying the epidemic dynamics is crucial for further surveillance and control of the outbreak. We employed a Bayesian framework to infer the time-calibrated phylogeny and the epidemic dynamics represented by the effective reproductive number (Re) changing over time from 33 genomic sequences available from GISAID. The time of the most recent common ancestor (MRCA) was December 17, 2019 (95% HPD: December 7, 2019 – December 23, 2019). The median estimate of Re shifted from 1.6 to 1.1 on around January 1, 2020. This study provides an early insight of the 2019-nCoV epidemic. However, due to limited amount of data, one should be cautious when interpreting the results at this stage.</td><td>2020-01-26</td><td>Chi Zhang; Mei Wang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.25.919688</td></tr><tr><td>3e2soc6w</td><td>abcfffafab399149d4adadd6bb458c4994e2025d</td><td>biorxiv</td><td>Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China</td><td>10.1101/2020.01.25.919787</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTRationaleSeveral studies have estimated basic production number of novel coronavirus pneumonia (NCP). However, the time-varying transmission dynamics of NCP during the outbreak remain unclear.ObjectivesWe aimed to estimate the basic and time-varying transmission dynamics of NCP across China, and compared them with SARS.MethodsData on NCP cases by February 7, 2020 were collected from epidemiological investigations or official websites. Data on severe acute respiratory syndrome (SARS) cases in Guangdong Province, Beijing and Hong Kong during 2002-2003 were also obtained. We estimated the doubling time, basic reproduction number (R0) and time-varying reproduction number (Rt) of NCP and SARS.Measurements and main resultsAs of February 7, 2020, 34,598 NCP cases were identified in China, and daily confirmed cases decreased after February 4. The doubling time of NCP nationwide was 2.4 days which was shorter than that of SARS in Guangdong (14.3 days), Hong Kong (5.7 days) and Beijing (12.4 days). The R0 of NCP cases nationwide and in Wuhan were 4.5 and 4.4 respectively, which were higher than R0 of SARS in Guangdong (R0=2.3), Hongkong (R0=2.3), and Beijing (R0=2.6). The Rt for NCP continuously decreased especially after January 16 nationwide and in Wuhan. The R0 for secondary NCP cases in Guangdong was 0.6, and the Rt values were less than 1 during the epidemic.ConclusionsNCP may have a higher transmissibility than SARS, and the efforts of containing the outbreak are effective. However, the efforts are needed to persist in for reducing time-varying reproduction number below one.At a Glance CommentaryScientific Knowledge on the SubjectSince December 29, 2019, pneumonia infection with 2019-nCoV, now named as Novel Coronavirus Pneumonia (NCP), occurred in Wuhan, Hubei Province, China. The disease has rapidly spread from Wuhan to other areas. As a novel virus, the time-varying transmission dynamics of NCP remain unclear, and it is also important to compare it with SARS.What This Study Adds to the FieldWe compared the transmission dynamics of NCP with SARS, and found that NCP has a higher transmissibility than SARS. Time-varying production number indicates that rigorous control measures taken by governments are effective across China, and persistent efforts are needed to be taken for reducing instantaneous reproduction number below one.</td><td>2020-01-26</td><td>Tao Liu; Jianxiong Hu; Jianpeng Xiao; Guanhao He; Min Kang; Zuhua Rong; Lifeng Lin; Haojie Zhong; Qiong Huang; Aiping Deng; Weilin Zeng; Xiaohua Tan; Siqing Zeng; Zhihua Zhu; Jiansen Li; Dexin Gong; Donghua Wan; Shaowei Chen; Lingchuan Guo; Yan Li; Limei Sun; Wenjia Liang; Tie Song; Jianfeng He; Wenjun Ma</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.25.919787</td></tr><tr><td>ykofrn9i</td><td>cbc05d14c57b91081970a232ab83bc993f998fe2</td><td>medrxiv</td><td>Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</td><td>10.1101/2020.01.26.20018754</td><td>null</td><td>null</td><td>medrvix</td><td>The geographic spread of 2019 novel coronavirus (COVID-19) infections from the epicenter of Wuhan, China, has provided an opportunity to study the natural history of the recently emerged virus. Using publicly available event-date data from the ongoing epidemic, the present study investigated the incubation period and other time intervals that govern the epidemiological dynamics of COVID-19 infections. Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution. The mean time from illness onset to hospital admission (for treatment and/or isolation) was estimated at 3-4 days without truncation and at 5-9 days when right truncated. Based on the 95th percentile estimate of the incubation period, we recommend that the length of quarantine should be at least 14 days. The median time delay of 13 days from illness onset to death (17 days with right truncation) should be considered when estimating the COVID-19 case fatality risk.</td><td>2020-01-28</td><td>Natalie M. Linton; Tetsuro Kobayashi; Yichi Yang; Katsuma Hayashi; Andrei R. Akhmetzhanov; Sung-mok Jung; Baoyin Yuan; Ryo Kinoshita; Hiroshi Nishiura</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.26.20018754</td></tr><tr><td>n93cm6ws</td><td>null</td><td>medrxiv</td><td>Epidemiological identification of a novel infectious disease in real time: Analysis of the atypical pneumonia outbreak in Wuhan, China, 2019-20</td><td>10.1101/2020.01.26.20018887</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: Virological tests indicate that a novel coronavirus is the most likely explanation for the 2019-20 pneumonia outbreak in Wuhan, China. We demonstrate that non-virological descriptive characteristics could have determined that the outbreak is caused by a novel pathogen in advance of virological testing. Methods: Characteristics of the ongoing outbreak were collected in real time from two medical social media sites. These were compared against characteristics of ten existing pathogens that can induce atypical pneumonia. The probability that the current outbreak is due to \"Disease X\" (i.e., previously unknown etiology) as opposed to one of the known pathogens was inferred, and this estimate was updated as the outbreak continued. Results: The probability that Disease X is driving the outbreak was assessed as over 32% on 31 December 2019, one week before virus identification. After some specific pathogens were ruled out by laboratory tests on 5 Jan 2020, the inferred probability of Disease X was over 59%. Conclusions: We showed quantitatively that the emerging outbreak of atypical pneumonia cases is consistent with causation by a novel pathogen. The proposed approach, that uses only routinely-observed non-virological data, can aid ongoing risk assessments even before virological test results become available. Keywords: Epidemic; Causation; Bayes' theorem; Diagnosis; Prediction; Statistical model</td><td>2020-01-28</td><td>Sung-mok Jung; Ryo Kinoshita; Robin N. Thompson; Katsuma Hayashi; Natalie M. Linton; Yichi Yang; Andrei R. Akhmetzhanov; Hiroshi Nishiura</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.26.20018887</td></tr><tr><td>4pgish5x</td><td>68e509d42a7d3349550512d80eca4e2b2123613f</td><td>biorxiv</td><td>Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCov</td><td>10.1101/2020.01.26.919985</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA novel coronavirus (2019-nCov) was identified in Wuhan, Hubei Province, China in December of 2019. This new coronavirus has resulted in thousands of cases of lethal disease in China, with additional patients being identified in a rapidly growing number internationally. 2019-nCov was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-Cov. Here based on the public database and the state-of-the-art single-cell RNA-Seq technique, we analyzed the ACE2 RNA expression profile in the normal human lungs. The result indicates that the ACE2 virus receptor expression is concentrated in a small population of type II alveolar cells (AT2). Surprisingly, we found that this population of ACE2-expressing AT2 also highly expressed many other genes that positively regulating viral reproduction and transmission. A comparison between eight individual samples demonstrated that the Asian male one has an extremely large number of ACE2-expressing cells in the lung. This study provides a biological background for the epidemic investigation of the 2019-nCov infection disease, and could be informative for future anti-ACE2 therapeutic strategy development.</td><td>2020-01-26</td><td>Yu Zhao; Zixian Zhao; Yujia Wang; Yueqing Zhou; Yu Ma; Wei Zuo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.26.919985</td></tr><tr><td>k2qj6o7j</td><td>null</td><td>biorxiv</td><td>Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event</td><td>10.1101/2020.01.26.920249</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackgroundA novel coronavirus (2019-nCoV) associated with human to human transmission and severe human infection has been recently reported from the city of Wuhan in China. Our objectives were to characterize the genetic relationships of the 2019-nCoV and to search for putative recombination within the subgenus of sarbecovirus.MethodsPutative recombination was investigated by RDP4 and Simplot v3.5.1 and discordant phylogenetic clustering in individual genomic fragments was confirmed by phylogenetic analysis using maximum likelihood and Bayesian methods.ResultsOur analysis suggests that the 2019-nCoV although closely related to BatCoV RaTG13 sequence throughout the genome (sequence similarity 96.3%), shows discordant clustering with the Bat-SARS-like coronavirus sequences. Specifically, in the 5’-part spanning the first 11,498 nucleotides and the last 3’-part spanning 24,341-30,696 positions, 2019-nCoV and RaTG13 formed a single cluster with Bat-SARS-like coronavirus sequences, whereas in the middle region spanning the 3’-end of ORF1a, the ORF1b and almost half of the spike regions, 2019-nCoV and RaTG13 grouped in a separate distant lineage within the sarbecovirus branch.ConclusionsThe levels of genetic similarity between the 2019-nCoV and RaTG13 suggest that the latter does not provide the exact variant that caused the outbreak in humans, but the hypothesis that 2019-nCoV has originated from bats is very likely. We show evidence that the novel coronavirus (2019-nCov) is not-mosaic consisting in almost half of its genome of a distinct lineage within the betacoronavirus. These genomic features and their potential association with virus characteristics and virulence in humans need further attention.</td><td>2020-01-27</td><td>D. Paraskevis; E.G. Kostaki; G. Magiorkinis; G. Panayiotakopoulos; S. Tsiodras</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.26.920249</td></tr><tr><td>u8goc7io</td><td>12fac9aedb1a09a3922a3c084ce4723708e463d6</td><td>medrxiv</td><td>The incubation period of 2019-nCoV infections among travellers from Wuhan, China</td><td>10.1101/2020.01.27.20018986</td><td>null</td><td>null</td><td>medrvix</td><td>Currently, a novel coronavirus 2019-nCoV causes an outbreak of viral pneumonia in Wuhan, China. Little is known about its epidemiological characteristics. Using the travel history and symptom onset of 88 confirmed cases that were detected outside Wuhan, we estimate the mean incubation period to be 6.4 (5.6 - 7.7, 95% CI) days, ranging from 2.1 to 11.1 days (2.5th to 97.5th percentile). These values help to inform case definitions for 2019-nCoV and appropriate durations for quarantine.</td><td>2020-01-28</td><td>Jantien A. Backer; Don Klinkenberg; Jacco Wallinga</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.27.20018986</td></tr><tr><td>2ygtr69i</td><td>029c1c588047f1d612a219ee15494d2d19ff7439</td><td>biorxiv</td><td>Protective Population Behavior Change in Outbreaks of Emerging Infectious Disease</td><td>10.1101/2020.01.27.921536</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTDuring outbreaks of emerging infections, the lack of effective drugs and vaccines increases reliance on non-pharmacologic public health interventions and behavior change to limit human-to-human transmission. Interventions that increase the speed with which infected individuals remove themselves from the susceptible population are paramount, particularly isolation and hospitalization. Ebola virus disease (EVD), Severe Acute Respiratory Syndrome (SARS), and Middle East Respiratory Syndrome (MERS) are zoonotic viruses that have caused significant recent outbreaks with sustained human-to-human transmission. This investigation quantified changing mean removal rates (MRR) and days from symptom onset to hospitalization (DSOH) of infected individuals from the population in seven different outbreaks of EVD, SARS, and MERS, to test for statistically significant differences in these metrics between outbreaks. We found that epidemic week and viral serial interval were correlated with the speed with which populations developed and maintained health behaviors in each outbreak.</td><td>2020-01-28</td><td>Evans K. Lodge; Annakate M. Schatz; John M. Drake</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.27.921536</td></tr><tr><td>zgdbvfi4</td><td>06a1002f9fbea7179ac3572843f66b14568af6e4</td><td>biorxiv</td><td>Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation</td><td>10.1101/2020.01.27.921627</td><td>null</td><td>null</td><td>biorxiv</td><td>Abstract2019-nCov has caused more than 80 deaths as of 27 January 2020 in China, and infection cases have been reported in more than 10 countries. However, there is no approved drug to treat the disease. 2019-nCov Mpro is a potential drug target to combat the virus. We built homology models based on SARS Mpro structures, and docked 1903 small molecule drugs to the models. Based on the docking score and the 3D similarity of the binding mode to the known Mpro ligands, 4 drugs were selected for binding free energy calculations. Both MM/GBSA and SIE methods voted for nelfinavir, with the binding free energy of −24.69±0.52 kcal/mol and −9.42±0.04 kcal/mol, respectively. Therefore, we suggested that nelfinavir might be a potential inhibitor against 2019-nCov Mpro.</td><td>2020-01-28</td><td>Zhijian Xu; Cheng Peng; Yulong Shi; Zhengdan Zhu; Kaijie Mu; Xiaoyu Wang; Weiliang Zhu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.27.921627</td></tr><tr><td>ely200x3</td><td>aef0734eb2538d3624d650a4ae20c3f66cace5f2</td><td>biorxiv</td><td>Chronic hM3Dq signaling in microglia ameliorates neuroinflammation in male mice</td><td>10.1101/2020.01.27.921809</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractMicroglia express muscarinic G protein-coupled receptors (GPCRs) that sense cholinergic activity and are activated by acetylcholine to potentially regulate microglial functions. Knowledge about how distinct types of muscarinic GPCR signaling regulate microglia function in vitro and in vivo is still poor, partly due to the fact that some of these receptors are also present in astrocytes and neurons. We generated mice expressing the hM3Dq Designer Receptor Exclusively Activated by Designer Drugs (DREADD) selectively in microglia to investigate the role of muscarinic M3Gq-linked signaling. We show that activation of hM3Dq using clozapine N-oxide (CNO) elevated intracellular calcium levels and increased phagocytosis of FluoSpheres in vitro. Acute treatment with CNO in vivo did not affect male mouse behavior, however chronic CNO treatment decreased sickness behavior triggered by lipopolysaccharide (LPS) treatment. Interestingly, whereas acute treatment with CNO increased synthesis of cytokine mRNA, chronic treatment attenuated LPS-induced cytokine mRNA changes in the brain, likely explaining the improvement in sickness behavior by chronic hM3Dq activation. No effect of CNO was observed in DREADD-negative mice. These results suggest that chronic activation of M3 muscarinic receptors (the hM3Dq progenitor) in microglia, and potentially other Gq-coupled GPCRs, preconditions microglia to decrease their response to further immunological challenges. Our results indicate that hM3Dq can be a useful tool to modulate neuroinflammation and study microglial immunological memory in vivo, which may be applicable for manipulations of neuroinflammation in neurodegenerative and psychiatric diseases.HighlightsMicroglial function was manipulated by specific activation of hM3Dq signaling.Chronic hM3Dq activation prevented LPS-induced sickness behavior in mice.Microglial hM3Dq signaling modulated expression of pro-inflammatory cytokines.</td><td>2020-01-28</td><td>William Binning; Aja E. Hogan-Cann; Diana Yae Sakae; Matthew Maksoud; Valeriy Ostapchenko; Mohammed Al-Onaizi; Sara Matovic; Wei-Yang Lu; Marco A. M. Prado; Wataru Inoue; Vania F. Prado</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.27.921809</td></tr><tr><td>si486lqi</td><td>754315299d847600d6c5d414665c728d40bf731d</td><td>biorxiv</td><td>Breaking down of the healthcare system: Mathematical modelling for controlling the novel coronavirus (2019-nCoV) outbreak in Wuhan, China</td><td>10.1101/2020.01.27.922443</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA novel coronavirus pneumonia initially identified in Wuhan, China and provisionally named 2019-nCoV has surged in the public. In anticipation of substantial burdens on healthcare system following this human-to-human spread, we aim to scrutinise the currently available information and evaluate the burden of healthcare systems during this outbreak in Wuhan. We applied a modified SIR model to project the actual number of infected cases and the specific burdens on isolation wards and intensive care units, given the scenarios of different diagnosis rates as well as different public health intervention efficacy. Our estimates suggest the actual number of infected cases could be much higher than the reported, with estimated 26,701 cases (as of 28th January 2020) assuming 50% diagnosis rate if no public health interventions were implemented. The estimated burdens on healthcare system could be largely reduced if at least 70% efficacy of public health intervention is achieved.</td><td>2020-01-28</td><td>Wai-Kit Ming; Jian Huang; Casper J. P. Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.27.922443</td></tr><tr><td>x22rc60j</td><td>30b5c7b8faf95265f67ae59f4686eaf9b2772893</td><td>medrxiv</td><td>Estimated effectiveness of traveller screening to prevent international spread of 2019 novel coronavirus (2019-nCoV)</td><td>10.1101/2020.01.28.20019224</td><td>null</td><td>null</td><td>medrvix</td><td>Traveller screening is being used to limit further global spread of 2019 novel coronavirus (nCoV) following its recent emergence. Here, we analyze the expected impact of different travel screening programs given remaining uncertainty around the values of key nCoV life history and epidemiological parameters. Even under best-case assumptions, we estimate that screening will miss around half of infected travellers. Breaking down the factors leading to screening successes and failures, we find that most cases missed by screening are fundamentally undetectable, because they have not yet developed symptoms and are unaware they were exposed. These findings emphasize the need for measures to track travellers who become ill after being missed by a travel screening program. We make our model available for interactive use so stakeholders can explore scenarios of interest using the most up-to-date information. We hope these findings contribute to evidence-based policy to combat the spread of nCoV, and to prospective planning to mitigate future emerging pathogens.</td><td>2020-01-30</td><td>Katelyn Gostic; Ana C. R. Gomez; Riley O. Mummah; Adam J. Kucharski; James O. Lloyd-Smith</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.28.20019224</td></tr><tr><td>mbdj3r0m</td><td>957e71389292e0881dff2bbe0e87fe6bf8b2d381</td><td>medrxiv</td><td>Risk for Transportation of 2019 Novel Coronavirus (COVID-19) from Wuhan to Cities in China</td><td>10.1101/2020.01.28.20019299</td><td>null</td><td>null</td><td>medrvix</td><td>On January 23, 2020, China quarantined Wuhan to contain an emerging coronavirus (COVID-19). We estimated the probability of transportation of COVID-19 from Wuhan to 369 cities in China before the quarantine. The expected risk is >50% in 130 (95% CI 89-190) cities and >99% in the 4 largest metropolitan areas of China.</td><td>2020-01-30</td><td>Zhanwei Du; Ling Wang; Simon Cauchemez; Xiaoke Xu; Xianwen Wang; Benjamin J Cowling; Lauren Ancel Meyers</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.28.20019299</td></tr><tr><td>635lbedk</td><td>d49f8ba7212183bc1b7f2cb6dd709dd197303b3d</td><td>biorxiv</td><td>Spreading predictability in complex networks</td><td>10.1101/2020.01.28.922757</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSpreading dynamics analysis is an important and interesting topic since it has many applications such as rumor or disease controlling, viral marketing and information recommending. Many state-of-the-art researches focus on predicting infection scale or threshold. Few researchers pay attention to the predicting of infection nodes from a snapshot. With developing of precision marketing, recommending and, controlling, how to predict infection nodes precisely from snapshot becomes a key issue in spreading dynamics analysis. In this paper, a probability based prediction model is presented so as to estimate the infection nodes from a snapshot of spreading. Experimental results on synthetic and real networks demonstrate that the model proposed could predict the infection nodes precisely in the sense of probability.</td><td>2020-01-28</td><td>Na Zhao; Jian Wang; Yong Yu; Jun-Yan Zhao; Duan-Bing Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.28.922757</td></tr><tr><td>iu2ziccm</td><td>c6ec1380338cb2467dc6fc14add09ca5bea18411</td><td>biorxiv</td><td>Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening</td><td>10.1101/2020.01.28.922922</td><td>null</td><td>null</td><td>biorxiv</td><td>Abstract2019 Novel Coronavirus (2019-nCoV) is a virus identified as the cause of the outbreak of pneumonia first detected in Wuhan, China. Investigations on the transmissibility, severity, and other features associated with this virus are ongoing. Currently, there is no vaccine or therapeutic antibody to prevent the infection, and more time is required to develop an effective immune strategy against the pathogen. In contrast, specific inhibitors targeting the key protease involved in replication and proliferation of the virus are the most effective means to alleviate the epidemic. The main protease of SARS-CoV is essential for the life cycle of the virus, which showed 96.1% of similarity with the main proteaseof 2019-nCoV, is considered to be an attractive target for drug development. In this study, we have identified 4 small molecular drugs with high binding capacity with SARS-CoV main protease by high-throughput screening based on the 8,000 clinical drug libraries, all these drugs have been widely used in clinical applications with guaranteed safety, which may serve as promising candidates to treat the infection of 2019-nCoV.</td><td>2020-01-29</td><td>Yan Li; Jinyong Zhang; Ning Wang; Haibo Li; Yun Shi; Gang Guo; Kaiyun Liu; Hao Zeng; Quanming Zou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.28.922922</td></tr><tr><td>lzda8kyn</td><td>null</td><td>biorxiv</td><td>Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</td><td>10.1101/2020.01.28.923011</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe newly identified 2019 novel coronavirus (2019-nCoV) has caused more than 800 laboratory-confirmed human infections, including 25 deaths, posing a serious threat to human health. Currently, however, there is no specific antiviral treatment or vaccine. Considering the relatively high identity of receptor binding domain (RBD) in 2019-nCoV and SARS-CoV, it is urgent to assess the cross-reactivity of anti-SARS-CoV antibodies with 2019-nCoV spike protein, which could have important implications for rapid development of vaccines and therapeutic antibodies against 2019-nCoV. Here, we report for the first time that a SARS-CoV-specific human monoclonal antibody, CR3022, could bind potently with 2019-nCoV RBD (KD of 6.3 nM). The epitope of CR3022 does not overlap with the ACE2 binding site within 2019-nCoV RBD. Therefore, CR3022 has the potential to be developed as candidate therapeutics, alone or in combination with other neutralizing antibodies, for the prevention and treatment of 2019-nCoV infections. Interestingly, some of the most potent SARS-CoV-specific neutralizing antibodies (e.g., m396, CR3014) that target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike protein, indicating that the difference in the RBD of SARS-CoV and 2019-nCoV has a critical impact for the cross-reactivity of neutralizing antibodies, and that it is still necessary to develop novel monoclonal antibodies that could bind specifically to 2019-nCoV RBD.</td><td>2020-01-28</td><td>Xiaolong Tian; Cheng Li; Ailing Huang; Shuai Xia; Sicong Lu; Zhengli Shi; Lu Lu; Shibo Jiang; Zhenlin Yang; Yanling Wu; Tianlei Ying</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.28.923011</td></tr><tr><td>x443k65a</td><td>f24d3b4b4af138be06b7452b7acefc8948bc1056</td><td>biorxiv</td><td>Beware of asymptomatic transmission: Study on 2019-nCoV prevention and control measures based on extended SEIR model</td><td>10.1101/2020.01.28.923169</td><td>null</td><td>null</td><td>biorxiv</td><td>SummaryBackgroundThe 2019 new coronavirus, “2019-nCoV”, was discovered from Wuhan Viral Pneumonia cases in December 2019, and was named by the World Health Organization on January 12, 2020. In the early stage, people knows little about the 2019-nCoV virus was not clear, and the spread period was encountering China’s annual spring migration, which made the epidemic spread rapidly from Wuhan to almost all provinces in China.MethodsThis study builds a SEIRD model that considers the movement of people across regions, revealing the effects of three measures on controlling the spread of the epidemic.Based on MATLAB R2017a, computational experiments were performed to simulate the epidemic prevention and control measures.FindingsThe research results show that current prevention and control measures in China are very necessary. This study further validates the concerns of international and domestic experts regarding asymptomatic transmission (E-status).InterpretationThe results of this study are applicable to explore the impact of the implementation of relevant measures on the prevention and control of epidemic spread, and to identify key individuals that may exist during the spread of the epidemic.</td><td>2020-01-28</td><td>Peng Shao; Yingji Shan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.28.923169</td></tr><tr><td>rr5qhsam</td><td>2afff5fe89a3e3ec270a41c2b929dfeec68d7d50</td><td>medrxiv</td><td>Real time estimation of the risk of death from novel coronavirus (2019-nCoV) infection: Inference using exported cases</td><td>10.1101/2020.01.29.20019547</td><td>null</td><td>null</td><td>medrvix</td><td>The exported cases of 2019 novel coronavirus (2019-nCoV) infection who were confirmed in other countries provide a chance to estimate the cumulative incidence and confirmed case fatality risk (cCFR) in China. Knowledge of the cCFR is critical to characterize the severity and understand pandemic potential of 2019-nCoV in the early stage of epidemic. Using the exponential growth rate of the incidence, the present study statistically estimated the cCFR and the basic reproduction number, i.e., the average number of secondary cases generated by a single primary case in a naive population. As of 24 January 2020, with 23 exported cases, and estimating the growth rate from 8 December 2019 (scenario 1) and using the data since growth of exported cases (scenario 2), the cumulative incidence in China was estimated at 5433 cases (95% confidence interval (CI): 3883, 7160) and 17780 cases (95% CI: 9646, 28724), respectively. The latest estimates of the cCFR were 4.6% (95% CI: 3.1-6.6) for scenario 1 and 7.7% (95% CI: 4.9-11.3%) for scenario 2, respectively. The basic reproduction number was estimated to be 2.2 (95% CI: 2.1, 2.3) and 3.7 (95% CI: 3.1, 4.3) for scenarios 1 and 2, respectively. Based on the results, we note that current 2019-nCoV epidemic has a substation potential to cause a pandemic. The proposed approach can provide insights into early risk assessment using only publicly available data.</td><td>2020-02-02</td><td>Sung-mok Jung; Andrei R. Akhmetzhanov; Katsuma Hayashi; Natalie M. Linton; Yichi Yang; Baoyin Yuan; Tetsuro Kobayashi; Ryo Kinoshita; Hiroshi Nishiura</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.29.20019547</td></tr><tr><td>yd7tu7u4</td><td>c04c7fb330a409a00f67040dde0f83b3da88eacb</td><td>biorxiv</td><td>Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines</td><td>10.1101/2020.01.29.924100</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractStarting from December 2019, a novel coronavirus, named 2019-nCoV, was found to cause Severe Acute Respiratory (SARI) symptoms and rapid pandemic in China. With the hope to identify candidate drugs for 2019-nCoV, we adopted a computational approach to screen for available commercial medicines which may function as inhibitors for the Mpro of 2019-nCoV. Up to 10 commercial medicines that may form hydrogen bounds to key residues within the binding pocket of 2019-nCoV Mpro were identified, which may have higher mutation tolerance than lopinavir/ritonavir and may also function as inhibitors for other coronaviruses with similar Mpro binding sites and pocket structures.</td><td>2020-01-29</td><td>Xin Liu; Xiu-Jie Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.29.924100</td></tr><tr><td>ac00tai9</td><td>b9dbb79c9e06164c3b7fefce67c11c5d6caf1fcd</td><td>biorxiv</td><td>Interpretable detection of novel human viruses from genome sequencing data</td><td>10.1101/2020.01.29.925354</td><td>null</td><td>null</td><td>biorxiv</td><td>Viruses evolve extremely quickly, so reliable methods for viral host prediction are necessary to safeguard biosecurity and biosafety alike. Novel human-infecting viruses are difficult to detect with standard bioinformatics workflows. Here, we predict whether a virus can infect humans directly from next-generation sequencing reads. We show that deep neural architectures significantly outperform both shallow machine learning and standard, homology-based algorithms, cutting the error rates in half and generalizing to taxonomic units distant from those presented during training. We propose a new approach for convolutional filter visualization to disentangle the information content of each nucleotide from its contribution to the final classification decision. Nucleotide-resolution maps of the learned associations between pathogen genomes and the infectious phenotype can be used to detect virulence-related genes in novel agents, as we show here for the SARS-CoV-2 coronavirus, unknown before it caused a COVID-19 pandemic in 2020.</td><td>2020-01-30</td><td>Jakub M Bartoszewicz; Anja Seidel; Bernhard Y Renard</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.29.925354</td></tr><tr><td>xxcx7hg5</td><td>acb0e11e5763dbf7588ec435751a2ab705a783f1</td><td>biorxiv</td><td>Insights into Cross-species Evolution of Novel Human Coronavirus 2019-nCoV and Defining Immune Determinants for Vaccine Development</td><td>10.1101/2020.01.29.925867</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTNovel Coronavirus (nCoV) outbreak in the city of Wuhan, China during December 2019, has now spread to various countries across the globe triggering a heightened containment effort. This human pathogen is a member of betacoronavirus genus carrying 30 kilobase of single positive-sense RNA genome. Understanding the evolution, zoonotic transmission, and source of this novel virus would help accelerating containment and prevention efforts. The present study reported detailed analysis of 2019-nCoV genome evolution and potential candidate peptides for vaccine development. This nCoV genotype might have been evolved from a bat-CoV by accumulating non-synonymous mutations, indels, and recombination events. Structural proteins Spike (S), and Membrane (M) had extensive mutational changes, whereas Envelope (E) and Nucleocapsid (N) proteins were very conserved suggesting differential selection pressures exerted on 2019-nCoV during evolution. Interestingly, 2019-nCoV Spike protein contains a 39 nucleotide sequence insertion relative to SARS-like bat-SL-CoVZC45/2017. Furthermore, we identified eight high binding affinity (HBA) CD4 T-cell epitopes in the S, E, M and N proteins, which can be commonly recognized by HLA-DR alleles of Asia and Asia-Pacific Region population. These immunodominant epitopes can be incorporated in universal subunit CoV vaccine. Diverse HLA types and variations in the epitope binding affinity may contribute to the wide range of immunopathological outcomes of circulating virus in humans. Our findings emphasize the requirement for continuous surveillance of CoV strains in live animal markets to better understand the viral adaptation to human host and to develop practical solutions to prevent the emergence of novel pathogenic CoV strains.</td><td>2020-01-30</td><td>Arunachalam Ramaiah; Vaithilingaraja Arumugaswami</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.29.925867</td></tr><tr><td>90tccg03</td><td>a6af28a7f0a0d8d1e4d5b06f5ae38412298bbcb4</td><td>medrxiv</td><td>Direct Measurement of Rates of Asymptomatic Infection and Clinical Care-Seeking for Seasonal Coronavirus</td><td>10.1101/2020.01.30.20019612</td><td>null</td><td>null</td><td>medrvix</td><td>The pandemic potential of the novel coronavirus (nCoV) that emerged in Wuhan, China, during December 2019 is strongly tied to the number and contagiousness of undocumented human infections. Here we present findings from a proactive longitudinal sampling study of acute viral respiratory infections that documents rates of asymptomatic infection and clinical care seeking for seasonal coronavirus. We find that the majority of infections are asymptomatic by most symptom definitions and that only 4% of individuals experiencing a seasonal coronavirus infection episode sought medical care for their symptoms. These numbers indicate that a very high percentage of seasonal coronavirus infections are undocumented and provide a reference for understanding the spread of the emergent nCoV.</td><td>2020-02-03</td><td>Jeffrey Shaman; Marta Galanti</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.20019612</td></tr><tr><td>81j4347d</td><td>null</td><td>medrxiv</td><td>Modelling the epidemic trend of the 2019-nCOV outbreak in Hubei Province, China</td><td>10.1101/2020.01.30.20019828</td><td>null</td><td>null</td><td>medrvix</td><td>As of 8am 30th January (Beijing Time) 2020, Approximate 8000 cases across the world have been confirmed. It's necessary to simulate epidemic trend of the 2019-nCOV outbreak in Hubei Province, the hardest-hit area. By SEIR simulation, the predicted epidemic peak in Hubei will be within 28th January 2020 to 7th February 2020, up to 7000-9000 infectious cases in total. The estimate above was based on some assumptions and limitations exited.</td><td>2020-02-02</td><td>Lizhe Ai</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.20019828</td></tr><tr><td>mm4ngrla</td><td>f17c6ac4986aecba2dc22f67b32f2d16b369117b</td><td>medrxiv</td><td>Transmission and epidemiological characteristics of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infected Pneumonia (COVID-19): preliminary evidence obtained in comparison with 2003-SARS</td><td>10.1101/2020.01.30.20019836</td><td>null</td><td>null</td><td>medrvix</td><td>Objectives: Latest epidemic data of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infected Pneumonia (COVID-19) was collected and a detailed statistical analysis was carried out to make comparison with 2003-SARS in order to provide scientific reference for the prevention and control of COVID-19. Methods: The information of COVID-19 and 2003-SARS from websites of NHCPRC and the World Health Organization was collected, and then the transmission dynamics of the two kinds of infectious diseases were analyzed. The information of 853 confirmed COVID-19 patients obtained from the website of health committees of 18 provinces. A descriptive epidemiological analysis method was employed to carefully analyze the epidemic characteristics. Subsequently, the COVID-19 epidemic data in Wuhan and other inland regions of China was analyzed separately and compared. A multivariate function model was constructed based on the confirmed COVID-19 case data. Results: The growth rate of new cases and deaths of COVID-19 were significantly faster than those of 2003-SARS. The number of confirmed cases in Wuhan and other inland areas both showed increasing trends. 853 confirmed COVID-19 cases aged 1 months to 94 years and the average age was (45.05 ± 17.22) years. The gender ratio (M: F) was 1.12: 1. Conclusions: The fatality rate of COVID-19 is lower than that of 2003-SARS and the cure rate is higher. The age of COVID-19 patients is mainly concentrated in the 30-50 years old (60.61%). The harm of the first-generation COVID-19 patients is higher than that of secondary cases.</td><td>2020-02-02</td><td>Rongqiang Zhang; Hui Liu; Fengying Li; Bei Zhang; Qiling Liu; Xiangwen Li; Limei Luo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.20019836</td></tr><tr><td>e85xdrcw</td><td>null</td><td>medrxiv</td><td>The impact of transmission control measures during the first 50 days of the COVID-19 epidemic in China</td><td>10.1101/2020.01.30.20019844</td><td>null</td><td>null</td><td>medrvix</td><td>Respiratory illness caused by a novel coronavirus (COVID-19) appeared in China during December 2019. Attempting to contain infection, China banned travel to and from Wuhan city on 23 January and implemented a national emergency response. Here we evaluate the spread and control of the epidemic based on a unique synthesis of data including case reports, human movement and public health interventions. The Wuhan shutdown slowed the dispersal of infection to other cities by an estimated 2.91 days (95%CI: 2.54-3.29), delaying epidemic growth elsewhere in China. Other cities that implemented control measures pre-emptively reported 33.3% (11.1-44.4%) fewer cases in the first week of their outbreaks (13.0; 7.1-18.8) compared with cities that started control later (20.6; 14.5-26.8). Among interventions investigated here, the most effective were suspending intra-city public transport, closing entertainment venues and banning public gatherings. The national emergency response delayed the growth and limited the size of the COVID-19 epidemic and, by 19 February (day 50), had averted hundreds of thousands of cases across China.</td><td>2020-02-02</td><td>Huaiyu Tian; Yonghong Liu; Yidan Li; Chieh-Hsi Wu; Bin Chen; Moritz U. G. Kraemer; Bingying Li; Jun Cai; Bo Xu; Qiqi Yang; Ben Wang; Peng Yang; Yujun Cui; Yimeng Song; Pai Zheng; Quanyi Wang; Ottar N Bjornstad; Ruifu Yang; Bryan Grenfell; Oliver Pybus; Christopher Dye</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.20019844</td></tr><tr><td>p5aj5k2g</td><td>0ff4ad5359ee2df3568605385a8616d8da66bf2c</td><td>medrxiv</td><td>Reconciling early-outbreak estimates of the basic reproductive number and its uncertainty: framework and applications to the novel coronavirus (SARS-CoV-2) outbreak</td><td>10.1101/2020.01.30.20019877</td><td>null</td><td>null</td><td>medrvix</td><td>A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number Ro -- the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of Ro vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of Ro across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate $r$, the mean generation interval $\\bar G$, and the generation-interval dispersion $\\kappa$. We then apply our framework to early estimates of Ro for the SARS-CoV-2 outbreak. We show that many early Ro estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of Ro, including the shape of the generation-interval distribution, in efforts to estimate Ro at the outset of an epidemic.</td><td>2020-02-02</td><td>Sang Woo Park; Benjamin M. Bolker; David Champredon; David J.D. Earn; Michael Li; Joshua S. Weitz; Bryan T. Grenfell; Jonathan Dushoff</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.20019877</td></tr><tr><td>ua6aeg5a</td><td>18dc6a67045d2687d2b5c11c85797f42824ed243</td><td>biorxiv</td><td>Evolution and variation of 2019-novel coronavirus</td><td>10.1101/2020.01.30.926477</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackgroundThe current outbreak caused by novel coronavirus (2019-nCoV) in China has become a worldwide concern. As of 28 January 2020, there were 4631 confirmed cases and 106 deaths, and 11 countries or regions were affected.MethodsWe downloaded the genomes of 2019-nCoVs and similar isolates from the Global Initiative on Sharing Avian Influenza Database (GISAID and nucleotide database of the National Center for Biotechnology Information (NCBI). Lasergene 7.0 and MEGA 6.0 softwares were used to calculate genetic distances of the sequences, to construct phylogenetic trees, and to align amino acid sequences. Bayesian coalescent phylogenetic analysis, implemented in the BEAST software package, was used to calculate the molecular clock related characteristics such as the nucleotide substitution rate and the most recent common ancestor (tMRCA) of 2019-nCoVs.ResultsAn isolate numbered EPI_ISL_403928 showed different phylogenetic trees and genetic distances of the whole length genome, the coding sequences (CDS) of ployprotein (P), spike protein (S), and nucleoprotein (N) from other 2019-nCoVs. There are 22, 4, 2 variations in P, S, and N at the level of amino acid residues. The nucleotide substitution rates from high to low are 1·05 × 10−2 (nucleotide substitutions/site/year, with 95% HPD interval being 6.27 × 10−4 to 2.72 × 10−2) for N, 5.34 × 10−3 (5.10 × 10−4, 1.28 × 10−2) for S, 1.69 × 10−3 (3.94 × 10−4, 3.60 × 10−3) for P, 1.65 × 10−3 (4.47 × 10−4, 3.24 × 10−3) for the whole genome, respectively. At this nucleotide substitution rate, the most recent common ancestor (tMRCA) of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.ConclusionOur analysis suggests that at least two different viral strains of 2019-nCoV are involved in this outbreak that might occur a few months earlier before it was officially reported.</td><td>2020-01-30</td><td>Chenglong Xiong; Lufang Jiang; Yue Chen; Qingwu Jiang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.926477</td></tr><tr><td>nfpw89fu</td><td>null</td><td>biorxiv</td><td>Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes</td><td>10.1101/2020.01.30.926881</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractDeubiquitinating enzymes (DUBs) are responsible for removing ubiquitin (Ub) from its protein conjugates. DUBs have been implicated as attractive therapeutic targets in the treatment of viral diseases, neurodegenerative disorders and cancer. The lack of selective chemical tools for the exploration of these enzymes significantly impairs the determination of their roles in both normal and pathological states. Commercially available fluorogenic substrates are based on the C-terminal Ub motif or contain Ub coupled to a fluorophore (Z-LRGG-AMC, Ub-AMC); therefore, these substrates suffer from lack of selectivity. By using a hybrid combinatorial substrate library (HyCoSuL) and a defined P2 library containing a wide variety of nonproteinogenic amino acids, we established a full substrate specificity profile for two DUBs—MERS PLpro and human UCH-L3. Based on these results, we designed and synthesized Ub-based substrates and activity-based probes (ABPs) containing selected unnatural amino acids located in the C-terminal Ub motif. Biochemical analysis and cell-based experiments confirmed the activity and selectivity of engineered Ub-based substrates and probes. Using this approach, we propose that for any protease that recognizes Ub and Ub-like substrates, a highly active and selective unnatural substrate or probe can be engineered.</td><td>2020-01-31</td><td>Wioletta Rut; Mikołaj Żmudziński; Scott J. Snipas; Miklos Bekes; Tony T. Huang; Marcin Drag</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.926881</td></tr><tr><td>dpcpg5c1</td><td>ffc9bc4ac6548cb1c03c2c6f088ab1c1c2067c6f</td><td>biorxiv</td><td>Nucleotide Analogues as Inhibitors of Viral Polymerases</td><td>10.1101/2020.01.30.927574</td><td>null</td><td>null</td><td>biorxiv</td><td>SummaryCoronaviruses such as the newly discovered virus from Wuhan, China, 2019-nCoV, and the viruses that cause SARS and MERS, have resulted in regional and global public health emergencies. Based on our molecular insight that the hepatitis C virus and the coronavirus use a similar viral genome replication mechanism, we reasoned that the FDA-approved drug EPCLUSA (Sofosbuvir/Velpatasvir) for the treatment of hepatitis C will also inhibit the above coronaviruses, including 2019-nCoV. To develop broad spectrum anti-viral agents, we further describe a novel strategy to design and synthesize viral polymerase inhibitors, by combining the ProTide Prodrug approach used in the development of Sofosbuvir with the use of 3’-blocking groups that we have previously built into nucleotide analogues that function as polymerase terminators.</td><td>2020-01-31</td><td>Jingyue Ju; Shiv Kumar; Xiaoxu Li; Steffen Jockusch; James J. Russo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.927574</td></tr><tr><td>vpcx2t3w</td><td>null</td><td>biorxiv</td><td>The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes</td><td>10.1101/2020.01.30.927806</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSince December 2019, a newly identified coronavirus (2019 novel coronavirus, 2019-nCov) is causing outbreak of pneumonia in one of largest cities, Wuhan, in Hubei province of China and has draw significant public health attention. The same as severe acute respiratory syndrome coronavirus (SARS-CoV), 2019-nCov enters into host cells via cell receptor angiotensin converting enzyme II (ACE2). In order to dissect the ACE2-expressing cell composition and proportion and explore a potential route of the 2019-nCov infection in digestive system infection, 4 datasets with single-cell transcriptomes of lung, esophagus, gastric, ileum and colon were analyzed. The data showed that ACE2 was not only highly expressed in the lung AT2 cells, esophagus upper and stratified epithelial cells but also in absorptive enterocytes from ileum and colon. These results indicated along with respiratory systems, digestive system is a potential routes for 2019-nCov infection. In conclusion, this study has provided the bioinformatics evidence of the potential route for infection of 2019-nCov in digestive system along with respiratory tract and may have significant impact for our healthy policy setting regards to prevention of 2019-nCoV infection.</td><td>2020-01-31</td><td>Hao Zhang; Zijian Kang; Haiyi Gong; Da Xu; Jing Wang; Zifu Li; Xingang Cui; Jianru Xiao; Tong Meng; Wang Zhou; Jianmin Liu; Huji Xu</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.927806</td></tr><tr><td>1qniriu0</td><td>6ae20454d1a9f228864de24660c2460becbc8151</td><td>biorxiv</td><td>Machine intelligence design of 2019-nCoV drugs</td><td>10.1101/2020.01.30.927889</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractWuhan coronavirus, called 2019-nCoV, is a newly emerged virus that infected more than 9692 people and leads to more than 213 fatalities by January 30, 2020. Currently, there is no effective treatment for this epidemic. However, the viral protease of a coronavirus is well-known to be essential for its replication and thus is an effective drug target. Fortunately, the sequence identity of the 2019-nCoV protease and that of severe-acute respiratory syndrome virus (SARS-CoV) is as high as 96.1%. We show that the protease inhibitor binding sites of 2019-nCoV and SARS-CoV are almost identical, which means all potential anti-SARS-CoV chemotherapies are also potential 2019-nCoV drugs. Here, we report a family of potential 2019-nCoV drugs generated by a machine intelligence-based generative network complex (GNC). The potential effectiveness of treating 2019-nCoV by using some existing HIV drugs is also analyzed.</td><td>2020-02-04</td><td>Kaifu Gao; Duc Duy Nguyen; Rui Wang; Guo-Wei Wei</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.30.927889</td></tr><tr><td>2wshgzjk</td><td>a01672035e0b9d21f9a934b9c9071610513325f2</td><td>medrxiv</td><td>Effectiveness of airport screening at detecting travellers infected with 2019-nCoV</td><td>10.1101/2020.01.31.20019265</td><td>null</td><td>null</td><td>medrvix</td><td>As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available.</td><td>2020-02-02</td><td>Billy Quilty; Sam Clifford; Stefan Flasche; Rosalind M Eggo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.31.20019265</td></tr><tr><td>gmi1ewc2</td><td>a44334e676e43c1889d282b0f0a8365f0f1e0c52</td><td>medrxiv</td><td>Early dynamics of transmission and control of COVID-19: a mathematical modelling study</td><td>10.1101/2020.01.31.20019901</td><td>null</td><td>null</td><td>medrvix</td><td>Background: An outbreak of the novel coronavirus SARS-CoV-2 has led to 46,997 confirmed cases as of 13th February 2020. Understanding the early transmission dynamics of the infection and evaluating the effectiveness of control measures is crucial for assessing the potential for sustained transmission to occur in new areas. Methods: We combined a stochastic transmission model with data on cases of novel coronavirus disease (COVID-19) in Wuhan and international cases that originated in Wuhan to estimate how transmission had varied over time during January and February 2020. Based on these estimates, we then calculated the probability that newly introduced cases might generate outbreaks in other areas. Findings: We estimated that the median daily reproduction number, Rt , declined from 2.35 (95% CI: 1.15-4.77) one week before travel restrictions were introduced on 23rd January to 1.05 (95% CI: 0.413-2.39) one week after. Based on our estimates of Rt,we calculated that in locations with similar transmission potential as Wuhan in early January, once there are at least four independently introduced cases, there is a more than 50% chance the infection will establish within that population. Interpretation: Our results show that COVID-19 transmission likely declined in Wuhan during late January 2020, coinciding with the introduction of control measures. As more cases arrive in international locations with similar transmission potential to Wuhan pre-control, it is likely many chains of transmission will fail to establish initially, but may still cause new outbreaks eventually.</td><td>2020-02-02</td><td>Adam J Kucharski; Timothy W Russell; Charlie Diamond; Yang Liu; John Edmunds; Sebastian Funk; Rosalind M Eggo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.31.20019901</td></tr><tr><td>gxstlzuk</td><td>0c27c0ddaa4761f6155838df81a88d24619720f8</td><td>biorxiv</td><td>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</td><td>10.1101/2020.01.31.928796</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTSummaryGenome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.AvailabilityAvailable online: https://www.genomedetective.com/app/typingtool/cov*Contactkoen@emweb.be and deoliveira@ukzn.ac.zaSupplementary informationSupplementary data is available online.</td><td>2020-02-02</td><td>Sara Cleemput; Wim Dumon; Vagner Fonseca; Wasim Abdool Karim; Marta Giovanetti; Luiz Carlos Alcantara; Koen Deforche; Tulio de Oliveira</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.31.928796</td></tr><tr><td>rv26qfmf</td><td>eae11a3f01b79e76a968bfb84886432f04584596</td><td>biorxiv</td><td>The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells</td><td>10.1101/2020.01.31.929042</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe emergence of a novel, highly pathogenic coronavirus, 2019-nCoV, in China, and its rapid national and international spread pose a global health emergency. Coronaviruses use their spike proteins to select and enter target cells and insights into nCoV-2019 spike (S)-driven entry might facilitate assessment of pandemic potential and reveal therapeutic targets. Here, we demonstrate that 2019-nCoV-S uses the SARS-coronavirus receptor, ACE2, for entry and the cellular protease TMPRSS2 for 2019-nCoV-S priming. A TMPRSS2 inhibitor blocked entry and might constitute a treatment option. Finally, we show that the serum form a convalescent SARS patient neutralized 2019-nCoV-S-driven entry. Our results reveal important commonalities between 2019-nCoV and SARS-coronavirus infection, which might translate into similar transmissibility and disease pathogenesis. Moreover, they identify a target for antiviral intervention.One sentence summaryThe novel 2019 coronavirus and the SARS-coronavirus share central biological properties which can guide risk assessment and intervention.</td><td>2020-01-31</td><td>Markus Hoffmann; Hannah Kleine-Weber; Nadine Krüger; Marcel Müller; Christian Drosten; Stefan Pöhlmann</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.31.929042</td></tr><tr><td>a1mcnta5</td><td>null</td><td>biorxiv</td><td>Highly Distinguished Amino Acid Sequences of 2019-nCoV (Wuhan Coronavirus)</td><td>10.1101/2020.01.31.929497</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractUsing a method for pathogen screening in DNA synthesis orders, we have identified a number of amino acid sequences that distinguish 2019-nCoV (Wuhan Coronavirus) from all other known viruses in Coronaviridae. We find three main regions of unique sequence: two in the 1ab polyprotein QHO60603.1, one in surface glycoprotein QHO60594.1.</td><td>2020-02-02</td><td>Jacob Beal; Thomas Mitchell; Daniel Wyschogrod; Jeff Manthey; Adam Clore</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.31.929497</td></tr><tr><td>cszqykpu</td><td>ff54e3e961a72eb1d2500166809b3651b2f98cf6</td><td>biorxiv</td><td>Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model</td><td>10.1101/2020.01.31.929547</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe infection of a novel coronavirus found in Wuhan of China (2019-nCoV) is rapidly spreading, and the incidence rate is increasing worldwide. Due to the lack of effective treatment options for 2019-nCoV, various strategies are being tested in China, including drug repurposing. In this study, we used our pretrained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV. The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). Interestingly, lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd < 1000 nM. In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction. Overall, we suggest that the list of antiviral drugs identified by the MT-DTI model should be considered, when establishing effective treatment strategies for 2019-nCoV.</td><td>2020-02-02</td><td>Bo Ram Beck; Bonggun Shin; Yoonjung Choi; Sungsoo Park; Keunsoo Kang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.31.929547</td></tr><tr><td>vbgf50os</td><td>9e94f9379fd74fcacc4f3a57e03cbe9035efee8e</td><td>biorxiv</td><td>Molecular Modeling Evaluation of the Binding Effect of Ritonavir, Lopinavir and Darunavir to Severe Acute Respiratory Syndrome Coronavirus 2 Proteases</td><td>10.1101/2020.01.31.929695</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThree anti-HIV drugs, ritonavir, lopinavir and darunavir, might have therapeutic effect on coronavirus disease 2019 (COVID-19). In this study, the structure models of two severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) proteases, coronavirus endopeptidase C30 (CEP_C30) and papain like viral protease (PLVP), were built by homology modeling. Ritonavir, lopinavir and darunavir were then docked to the models, respectively, followed by energy minimization of the protease-drug complexes. In the simulations, ritonavir can bind to CEP_C30 most suitably, and induce significant conformation changes of CEP_C30; lopinavir can also bind to CEP_C30 suitably, and induce significant conformation changes of CEP_C30; darunavir can bind to PLVP suitably with slight conformation changes of PLVP. It is suggested that the therapeutic effect of ritonavir and lopinavir on COVID-19 may be mainly due to their inhibitory effect on CEP_C30, while ritonavir may have stronger efficacy; the inhibitory effect of darunavir on SARS-CoV-2 and its potential therapeutic effect may be mainly due to its inhibitory effect on PLVP.</td><td>2020-02-03</td><td>Shen Lin; Runnan Shen; Jingdong He; Xinhao Li; Xushun Guo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.01.31.929695</td></tr><tr><td>aeuy92bx</td><td>a8e53c77bc72f467723a7d5b8020875c0e96f8aa</td><td>biorxiv</td><td>Potent neutralization of 2019 novel coronavirus by recombinant ACE2-Ig</td><td>10.1101/2020.02.01.929976</td><td>null</td><td>null</td><td>biorxiv</td><td>Abstract2019-nCoV, which is a novel coronavirus emerged in Wuhan, China, at the end of 2019, has caused at least infected 11,844 as of Feb 1, 2020. However, there is no specific antiviral treatment or vaccine currently. Very recently report had suggested that novel CoV would use the same cell entry receptor, ACE2, as the SARS-CoV. In this report, we generated a novel recombinant protein by connecting the extracellular domain of human ACE2 to the Fc region of the human immunoglobulin IgG1. An ACE2 mutant with low catalytic activity was also used in the study. The fusion proteins were then characterized. Both fusion proteins has high affinity binding to the receptor-binding domain (RBD) of SARS-CoV and 2019-nCoV and exerted desired pharmacological properties. Moreover, fusion proteins potently neutralized SARS-CoV and 2019-nCoV in vitro. As these fusion proteins exhibit cross-reactivity against coronaviruses, they could have potential applications for diagnosis, prophylaxis, and treatment of 2019-nCoV.</td><td>2020-02-02</td><td>Changhai Lei; Wenyan Fu; Kewen Qian; Tian Li; Sheng Zhang; Min Ding; Shi Hu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.01.929976</td></tr><tr><td>eecyduzq</td><td>836190d366ba8e34c3a5df8a24d17a36fd503505</td><td>biorxiv</td><td>Fast assessment of human receptor-binding capability of 2019 novel coronavirus (2019-nCoV)</td><td>10.1101/2020.02.01.930537</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe outbreaks of 2002/2003 SARS, 2012/2015 MERS and 2019/2020 Wuhan respiratory syndrome clearly indicate that genome evolution of an animal coronavirus (CoV) may enable it to acquire human transmission ability, and thereby to cause serious threats to global public health. It is widely accepted that CoV human transmission is driven by the interactions of its spike protein (S-protein) with human receptor on host cell surface; so, quantitative evaluation of these interactions may be used to assess the human transmission capability of CoVs. However, quantitative methods directly using viral genome data are still lacking. Here, we perform large-scale protein-protein docking to quantify the interactions of 2019-nCoV S-protein receptor-binding domain (S-RBD) with human receptor ACE2, based on experimental SARS-CoV S-RBD-ACE2 complex structure. By sampling a large number of thermodynamically probable binding conformations with Monte Carlo algorithm, this approach successfully identified the experimental complex structure as the lowest-energy receptor-binding conformations, and hence established an experiment-based strength reference for evaluating the receptor-binding affinity of 2019-nCoV via comparison with SARS-CoV. Our results show that this binding affinity is about 73% of that of SARS-CoV, supporting that 2019-nCoV may cause human transmission similar to that of SARS-CoV. Thus, this study presents a method for rapidly assessing the human transmission capability of a newly emerged CoV and its mutant strains, and demonstrates that post-genome analysis of protein-protein interactions may provide early scientific guidance for viral prevention and control.</td><td>2020-02-03</td><td>Qiang Huang; Andreas Herrmann</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.01.930537</td></tr><tr><td>uqls3p01</td><td>null</td><td>biorxiv</td><td>Genomic variance of the 2019-nCoV coronavirus</td><td>10.1101/2020.02.02.931162</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThere is rising global concern for the recently emerged novel Coronavirus (2019-nCov). Full genomic sequences have been released by the worldwide scientific community in the last few weeks in order to understand the evolutionary origin and molecular characteristics of this virus. Taking advantage of all the genomic information currently available, we constructed a phylogenetic tree including also representatives of other coronaviridae, such as Bat coronavirus (BCoV) and SARS. We confirm high sequence similarity (>99%) between all sequenced 2019-nCoVs genomes available, with the closest BCoV sequence sharing 96.2% sequence identity, confirming the notion of a zoonotic origin of 2019-nCoV. Despite the low heterogeneity of the 2019-nCoV genomes, we could identify at least two hyper-variable genomic hotspots, one of which is responsible for a Serine/Leucine variation in the viral ORF8-encoded protein. Finally, we perform a full proteomic comparison with other coronaviridae, identifying key aminoacidic differences to be considered for antiviral strategies deriving from previous anti-coronavirus approaches.</td><td>2020-02-04</td><td>Carmine Ceraolo; Federico M. Giorgi</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.02.931162</td></tr><tr><td>rj9k9awy</td><td>null</td><td>medrxiv</td><td>Evidence and characteristics of human-to-human transmission of SARS-CoV-2</td><td>10.1101/2020.02.03.20019141</td><td>null</td><td>null</td><td>medrvix</td><td>Background: On December 31, 2019, an outbreak of COVID-19 in humans was reported in Wuhan, and then spread fast to other provinces, China. We analyzed data from field investigations and genetic sequencing to describe the evidence and characteristics of human-to-human transmission in Guangdong Province. Methods: A confirmed COVID-19 case was defined if a suspected case was verified with positive of SARS-CoV-2 in throat swabs, nasal swabs, bronchoalveolar lavage fluid (BALF), or endotracheal aspirates by real-time reverse transcriptase polymerase chain reaction assay (RT-PCR) or genetic sequencing. Field investigations were conducted for each confirmed case. Clinical and demographic data of confirmed cases were collected from medical records. Exposure and travel history were obtained by interview. Results: A total of 1,151 confirmed cases were identified as of February 10, 2020 in Guangdong Province, China. Of them, 697 (60.1%) cases were from 234 cluster infections. Two hundred and fourteen (18.6%) were secondary cases, in which 144 cases were from family cluster infections. With the epidemic continuing, although familial cluster events were dominated, community cluster events increased with a nosocomial event. The whole genomes within the same family cluster infections were identical, and presented a few unique single nucleotide variants (SNVs) compared with SARS-CoV-2 identified on December 2019 in Wuhan. Conclusions: We observed evident human-to-human transmissions of SARS-CoV-2 in Guangdong, China. Although most of them were from family cluster infections, community and nosocomial infections were increasing. Our findings indicate that human-to-human transmission risks are transferring from family to community in Guangdong Province.</td><td>2020-02-05</td><td>Min Kang; Jie Wu; Wenjun Ma; Jianfeng He; Jing Lu; Tao Liu; Baisheng Li; Shujiang Mei; Feng Ruan; Lifeng Lin; Lirong Zou; Changwen Ke; Haojie Zhong; Yingtao Zhang; Xuguang Chen; Zhe Liu; Qi Zhu; Jianpeng Xiao; Jianxiang Yu; Jianxiong Hu; Weilin Zeng; Xing Li; Yuhuang Liao; Xiujuan Tang; Songjian Xiao; Ying Wang; Yingchao Song; Xue Zhuang; Lijun Liang; Siqing Zeng; Guanhao He; Peng Lin; Huihong Deng; Tie Song</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.20019141</td></tr><tr><td>vspnuxz9</td><td>a1bff76ce360e8990b0a4ee2a5228a6e6e63d9c1</td><td>medrxiv</td><td>Serial interval of novel coronavirus (2019-nCoV) infections</td><td>10.1101/2020.02.03.20019497</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To estimate the serial interval of novel coronavirus (COVID-19) from information on 28 infector-infectee pairs. Methods: We collected dates of illness onset for primary cases (infectors) and secondary cases (infectees) from published research articles and case investigation reports. We subjectively ranked the credibility of the data and performed analyses on both the full dataset (n=28) and a subset of pairs with highest certainty in reporting (n=18). In addition, we adjusting for right truncation of the data as the epidemic is still in its growth phase. Results: Accounting for right truncation and analyzing all pairs, we estimated the median serial interval at 4.0 days (95% credible interval [CrI]: 3.1, 4.9). Limiting our data to only the most certain pairs, the median serial interval was estimated at 4.6 days (95% CrI: 3.5, 5.9). Conclusions: The serial interval of COVID-19 is shorter than its median incubation period. This suggests that a substantial proportion of secondary transmission may occur prior to illness onset. The COVID-19 serial interval is also shorter than the serial interval of severe acute respiratory syndrome (SARS), indicating that calculations made using the SARS serial interval may introduce bias.</td><td>2020-02-13</td><td>Hiroshi Nishiura; Natalie M Linton; Andrei R. Akhmetzhanov</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.20019497</td></tr><tr><td>mjhez5im</td><td>5d3612cd95331a2f0c43ef9d03ce583b5ff8c995</td><td>medrxiv</td><td>Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</td><td>10.1101/2020.02.03.20020206</td><td>null</td><td>null</td><td>medrvix</td><td>The 2019-nCoV is reported to share the same entry (ACE2) as SARS-CoV according to the updated findings. Analyzing the distribution and expression level of the route of coronavirus may help reveal underlying mechanisms of viral susceptibility and post-infection modulation. In this study, we found that the expression of ACE2 in healthy populations and patients with underlying diseases was not significantly different, suggesting relatively similar susceptibility, which was consistent with current clinical observations. Moreover, based on the expression of ACE2 in smoking individuals, we inferred that long-term smoking might be a risk factor for 2019-nCoV. Analyzing the ACE2 in SARS-CoV infected cells suggested that ACE2 was more than just a receptor but also participated in post-infection regulation, including immune response, cytokine secretion, and viral genome replication. We also constructed Protein-protein interaction (PPI) networks and identified hub genes in viral activity and cytokine secretion. Our findings could explain the clinical symptoms so far and help clinicians and researchers understand the pathogenesis and design therapeutic strategies for 2019-nCoV.</td><td>2020-02-05</td><td>Xiang He; Lei Zhang; Qin Ran; Anying Xiong; Junyi Wang; Dehong Wu; Feng Chen; Guoping Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.20020206</td></tr><tr><td>f3h74j1n</td><td>null</td><td>medrxiv</td><td>Estimation of the asymptomatic ratio of novel coronavirus infections (COVID-19)</td><td>10.1101/2020.02.03.20020248</td><td>null</td><td>null</td><td>medrvix</td><td>A total of 565 Japanese citizens were evacuated from Wuhan, China to Japan. All passengers were screened for symptoms and also undertook reverse transcription polymerase chain reaction testing, identifying 5 asymptomatic and 7 symptomatic passengers testing positive for 2019-nCoV. We show that the screening result is suggestive of the asymptomatic ratio at 41.6%.</td><td>2020-02-11</td><td>Hiroshi Nishiura; Tetsuro Kobayashi; Takeshi Miyama; Ayako Suzuki; Sungmok Jung; Katsuma Hayashi; Ryo Kinoshita; Yichi Yang; Baoyin Yuan; Andrei R. Akhmetzhanov; Natalie M Linton</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.20020248</td></tr><tr><td>b4mdiont</td><td>21100aba41a4bfb48d7dc37f1bf5dbb38bf3867a</td><td>medrxiv</td><td>Network-based Drug Repurposing for Human Coronavirus</td><td>10.1101/2020.02.03.20020263</td><td>null</td><td>null</td><td>medrvix</td><td>Human Coronaviruses (HCoVs), including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle east respiratory syndrome coronavirus (MERS-CoV), and 2019 novel coronavirus (2019-nCoV), lead global epidemics with high morbidity and mortality. However, there are currently no effective drugs targeting 2019-nCoV. Drug repurposing, represented as an effective drug discovery strategy from existing drugs, could shorten the time and reduce the cost compared to de novo drug discovery. In this study, we present an integrative, antiviral drug repurposing methodology implementing a systems pharmacology-based network medicine platform, quantifying the interplay between the HCoV-host interactome and drug targets in the human protein-protein interaction network. Phylogenetic analyses of 15 HCoV whole genomes reveal that 2019-nCoV has the highest nucleotide sequence identity with SARS-CoV (79.7%) among the six other known pathogenic HCoVs. Specifically, the envelope and nucleocapsid proteins of 2019-nCoV are two evolutionarily conserved regions, having the sequence identities of 96% and 89.6%, respectively, compared to SARS-CoV. Using network proximity analyses of drug targets and known HCoV-host interactions in the human protein-protein interactome, we computationally identified 135 putative repurposable drugs for the potential prevention and treatment of HCoVs. In addition, we prioritized 16 potential anti-HCoV repurposable drugs (including melatonin, mercaptopurine, and sirolimus) that were further validated by enrichment analyses of drug-gene signatures and HCoV-induced transcriptomics data in human cell lines. Finally, we showcased three potential drug combinations (including sirolimus plus dactinomycin, mercaptopurine plus melatonin, and toremifene plus emodin) captured by the Complementary Exposure pattern: the targets of the drugs both hit the HCoV-host subnetwork, but target separate neighborhoods in the human protein-protein interactome network. In summary, this study offers powerful network-based methodologies for rapid identification of candidate repurposable drugs and potential drug combinations toward future clinical trials for HCoVs.</td><td>2020-02-05</td><td>Yadi Zhou; Yuan Hou; Jiayu Shen; Yin Huang; William Martin; Feixiong Cheng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.20020263</td></tr><tr><td>4sw1hrnf</td><td>5c525cc04075b7c82e52b1e80d96bb394b1a0907</td><td>medrxiv</td><td>Getting to zero quickly in the 2019-nCov epidemic with vaccines or rapid testing</td><td>10.1101/2020.02.03.20020271</td><td>null</td><td>null</td><td>medrvix</td><td>Any plan for stopping the ongoing 2019-nCov epidemic must be based on a quantitative understanding of the proportion of the at-risk population that needs to be protected by effective control measures in order for transmission to decline sufficiently and quickly enough for the epidemic to end. Using an SEIR-type transmission model, we contrasted two alternate strategies by modeling the proportion of the population that needs to be protected from infection by one-time vaccination (assuming 100% effectiveness) or by testing with isolation and treatment of individuals within six, 24, or 48 hours of symptom onset. If R is currently 2.2, vaccination at the herd immunity coverage of 55% would drive R just below 1, but transmission could persist for years. Over 80% of coverage is required to end the epidemic in 6 months with population-wide vaccination. The epidemic could be ended in just under a year if testing with isolation and treatment reached 80% of symptomatically infected patients within 24 hours of symptom onset (assuming 10% asymptomatic transmission). The epidemic could be ended in six months if testing with isolation and treatment reached 90% of symptomatic patients. If 90% of symptomatic patients could be tested within six hours of symptoms appearing, the epidemic could be ended in under four months.</td><td>2020-02-05</td><td>Gerardo Chowell; Ranu Dhillon; Devabhaktuni Srikrishna</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.20020271</td></tr><tr><td>bnv38la3</td><td>b51d1c7287ac75790a52752489577d33d522898b</td><td>medrxiv</td><td>Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</td><td>10.1101/2020.02.03.20020289</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of pneumonia caused by the 2019 Novel Coronavirus (2019-nCoV) was reported in Wuhan City, China. However, the clinical symptoms varied in different reports. Based on the results of inter-group difference test, we found that the incidence of diarrhea differed in three recent reports. As 2019-nCoV utilizes the same cell entry receptor ACE2 as severe acute respiratory syndrome coronavirus (SARS-CoV) and ACE2 tightly controls intestinal inflammation, to trace the route of infection mediated by 2019-nCoV, we used the single-cell RNA sequencing data for analysis. We found that the ACE2 mRNA was highly expressed in the healthy human small intestine rather than the lung. Besides, single-cell RNA sequencing data showed that ACE2 was significantly elevated in the proximal and distal enterocytes, where the small intestinal epithelium is exposed to the foreign pathogen. Thus, we suspect that ACE2-expressing small intestinal epithelium cells might be vulnerable to 2019-nCoV infection when people eat infected wild animals and diarrhea may serve as an indicator for infection, suggesting that clinicians should pay more attention to patients with diarrhea during the outbreak of pneumonia.</td><td>2020-02-11</td><td>Weicheng Liang; Zhijie Feng; Shitao Rao; Cuicui Xiao; Zexiao Lin; Qi Zhang; Wei Qi</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.20020289</td></tr><tr><td>xo5u02d7</td><td>6c91b00faa16142426820016ad3a7847fc77c8e4</td><td>biorxiv</td><td>Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</td><td>10.1101/2020.02.03.931766</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA newly identified coronavirus, 2019-nCoV, has been posing significant threats to public health since December 2019. ACE2, the host cell receptor for severe acute respiratory syndrome coronavirus (SARS), has recently been demonstrated in mediating 2019-nCoV infection. Interestingly, besides the respiratory system, substantial proportion of SARS and 2019-nCoV patients showed signs of various degrees of liver damage, the mechanism and implication of which have not yet been determined. Here, we performed an unbiased evaluation of cell type specific expression of ACE2 in healthy liver tissues using single cell RNA-seq data of two independent cohorts, and identified specific expression in cholangiocytes. The results indicated that virus might directly bind to ACE2 positive cholangiocytes but not necessarily hepatocytes. This finding suggested the liver abnormalities of SARS and 2019-nCoV patients may not be due to hepatocyte damage, but cholangiocyte dysfunction and other causes such as drug induced and systemic inflammatory response induced liver injury. Our findings indicate that special care of liver dysfunction should be installed in treating 2019-nCoV patients during the hospitalization and shortly after cure.</td><td>2020-02-04</td><td>Xiaoqiang Chai; Longfei Hu; Yan Zhang; Weiyu Han; Zhou Lu; Aiwu Ke; Jian Zhou; Guoming Shi; Nan Fang; Jia Fan; Jiabin Cai; Jue Fan; Fei Lan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.931766</td></tr><tr><td>cetdqgff</td><td>bb8e3d331bb2975e1c644c8729b842797da2d626</td><td>biorxiv</td><td>Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</td><td>10.1101/2020.02.03.932350</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAs of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus, within Betacoronavirus. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.</td><td>2020-02-04</td><td>Gurjit S. Randhawa; Maximillian P.M. Soltysiak; Hadi El Roz; Camila P.E. de Souza; Kathleen A. Hill; Lila Kari</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.932350</td></tr><tr><td>7i52vltp</td><td>181b7b57851e6f58a601b68e613d10c10616f774</td><td>biorxiv</td><td>Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies</td><td>10.1101/2020.02.03.933226</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the available SARS-CoV-2 sequences (as of 9 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.</td><td>2020-02-04</td><td>Syed Faraz Ahmed; Ahmed A. Quadeer; Matthew R. McKay</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.03.933226</td></tr><tr><td>f5yjz0zp</td><td>1b531d266b4933bf8040739ed199914655b075d6</td><td>medrxiv</td><td>Population movement, city closure and spatial transmission of the 2019-nCoV infection in China</td><td>10.1101/2020.02.04.20020339</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of pneumonia caused by a novel coronavirus (2019-nCoV) in Wuhan City of China obtained global concern, the population outflow from Wuhan has contributed to spatial expansion in other parts of China. We examined the effects of population outflow from Wuhan on the 2019-nCoV transmission in other provinces and cities of China, as well as the impacts of the city closure in Wuhan. We observed a significantly positive association between population movement and the number of cases. Further analysis revealed that if the city closure policy was implemented two days earlier, 1420 (95% CI: 1059, 1833) cases could be prevented, and if two days later, 1462 (95% CI: 1090, 1886) more cases would be possible. Our findings suggest that population movement might be one important trigger of the 2019-nCoV infection transmission in China, and the policy of city closure is effective to prevent the epidemic.</td><td>2020-02-05</td><td>Siqi Ai; Guanghu Zhu; Fei Tian; Huan Li; Yuan Gao; Yinglin Wu; Qiyong Liu; Hualiang Lin</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.20020339</td></tr><tr><td>xv42jvd6</td><td>e3f19f88e7e3d1279b4d93e4db041e44a8d35dbb</td><td>medrxiv</td><td>Epidemiological parameter review and comparative dynamics of influenza, respiratory syncytial virus, rhinovirus, human coronavirus, and adenovirus</td><td>10.1101/2020.02.04.20020404</td><td>null</td><td>null</td><td>medrvix</td><td>Influenza-like illness (ILI) accounts for a large burden of annual morbidity and mortality worldwide. A finer-grained knowledge of the parameters and dynamics of the viruses commonly underlying ILI is needed for modeling, diagnostic, and intervention efforts. We conducted an extensive literature review for epidemiological parameter values for influenza, respiratory syncytial virus (RSV), rhinovirus, human coronavirus (HCoV), and adenovirus. We also developed a deterministic SEIR model for ILI, and derived an expression for R0. We here report ranges and means for parameters for these five common viruses.</td><td>2020-02-05</td><td>Julie Spencer; Deborah P Shutt; Sarah K Moser; Hannah Clegg; Helen J Wearing; Harshini Mukundan; Carrie A Manore</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.20020404</td></tr><tr><td>kctgr529</td><td>c9533866dd77deaa65ee1015b57bca5486f93326</td><td>medrxiv</td><td>The impact of traffic isolation in Wuhan on the spread of 2019-nCov</td><td>10.1101/2020.02.04.20020438</td><td>null</td><td>null</td><td>medrvix</td><td>The 2019-nCoV outbreak occurred near the Chinese Spring Festival transport period in Wuhan. As an important transportation center, the migration of Wuhan accelerated the spread of 2019-nCoV across mainland China. Based on the cumulative Baidu migration index (CBMI), we first analyzed the proportion of Wuhan's migrant population to other cities. Our results confirm that there is a significant correlation between the export population of Wuhan and reported cases in various regions. We subsequently found that the mortality rate in Hubei Province was much higher than that in other regions of mainland China, while the investigation of potential cases in Wuhan was far behind other provinces in Mainland China, which indicates the effectiveness of early isolation.</td><td>2020-02-05</td><td>Gehui Jin; Jiayu Yu; Liyuan Han; Shiwei Duan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.20020438</td></tr><tr><td>8kdtpwbv</td><td>164b0678afc42f1923aa100b624ca969a53fb3b2</td><td>medrxiv</td><td>Forecasting the Wuhan coronavirus (2019-nCoV) epidemics using a simple (simplistic) model - update (Feb. 8, 2020)</td><td>10.1101/2020.02.04.20020461</td><td>null</td><td>null</td><td>medrvix</td><td>Confirmed infection cases in mainland China were analyzed using the data up to January 28, 2020 (first 13 days of reliable confirmed cases). For the first period the cumulative number of cases followed an exponential function. However, from January 28, we discerned a downward deviation from the exponential growth. This slower-than-exponential growth was also confirmed by a steady decline of the effective reproduction number. A backtrend analysis suggested the original basic reproduction number R0 to be about 2.4 to 2.5. As data become available, we subsequently analyzed them during three consecutive periods obtaining a sequence of model predictions. All available data up were processed the same way. We used a simple logistic growth model that fitted very well with all data. Using this model and the three sets of data, we estimated maximum cases as about 21,000, 28,000 and 35,000 cases refining these predictions in near-real time. With slightly different approach (linearization in time) the estimate of maximum cases was even higher (about 65,000). Although the estimates of maximum cases increase as more data were reported all models show reaching a peak in mid-February in contrast to the unconfined exponential growth. These predictions do not account for any possible other secondary sources of infection.</td><td>2020-02-05</td><td>Slav W Hermanowicz</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.20020461</td></tr><tr><td>iat80b4l</td><td>013d9d1cba8a54d5d3718c229b812d7cf91b6c89</td><td>medrxiv</td><td>Assessing spread risk of Wuhan novel coronavirus within and beyond China, January-April 2020: a travel network-based modelling study</td><td>10.1101/2020.02.04.20020479</td><td>null</td><td>null</td><td>medrvix</td><td>Background: A novel coronavirus (2019-nCoV) emerged in Wuhan City, China, at the end of 2019 and has caused an outbreak of human-to-human transmission with a Public Health Emergency of International Concern declared by the World Health Organization on January 30, 2020. Aim: We aimed to estimate the potential risk and geographic range of Wuhan novel coronavirus (2019-nCoV) spread within and beyond China from January through to April, 2020. Methods: A series of domestic and international travel network-based connectivity and risk analyses were performed, by using de-identified and aggregated mobile phone data, air passenger itinerary data, and case reports. Results: The cordon sanitaire of Wuhan is likely to have occurred during the latter stages of peak population numbers leaving the city before Lunar New Year (LNY), with travellers departing into neighbouring cities and other megacities in China. We estimated that 59,912 air passengers, of which 834 (95% UI: 478 - 1349) had 2019-nCoV infection, travelled from Wuhan to 382 cities outside of mainland China during the two weeks prior to the lockdown of Wuhan. The majority of these cities were in Asia, but major hubs in Europe, the US and Australia were also prominent, with strong correlation seen between predicted importation risks and reported cases. Because significant spread has already occurred, a large number of airline travellers (3.3 million under the scenario of 75% travel reduction from normal volumes) may be required to be screened at origin high-risk cities in China and destinations across the globe for the following three months of February to April, 2020 to effectively limit spread beyond its current extent. Conclusion: Further spread of 2019-nCoV within China and international exportation is likely to occur. All countries, especially vulnerable regions, should be prepared for efforts to contain the 2019-nCoV infection.</td><td>2020-02-05</td><td>Shengjie Lai; Isaac Bogoch; Nick Ruktanonchai; Alexander Watts; Xin Lu; Weizhong Yang; Hongjie Yu; Kamran Khan; Andrew J Tatem</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.20020479</td></tr><tr><td>9fd5a49o</td><td>158e546658e42ab0a89d174bc9facb437a034df6</td><td>medrxiv</td><td>Using predicted imports of 2019-nCoV cases to determine locations that may not be identifying all imported cases</td><td>10.1101/2020.02.04.20020495</td><td>null</td><td>null</td><td>medrvix</td><td>Cases from the ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (2019-nCoV) exported from mainland China can lead to self-sustained outbreaks in other populations. Internationally imported cases are currently being reported in several different locations. Early detection of imported cases is critical for containment of the virus. Based on air travel volume estimates from Wuhan to international destinations and using a generalized linear regression model we identify locations which may potentially have undetected internationally imported cases.</td><td>2020-02-05</td><td>Pablo M De Salazar; Rene Niehus; Aimee Taylor; Caroline O Buckee; Marc Lipsitch</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.20020495</td></tr><tr><td>ecu579el</td><td>136ee862ad53b606c8f2bea917b9705f5079e88e</td><td>medrxiv</td><td>Risk assessment of novel coronavirus COVID-19 outbreaks outside China</td><td>10.1101/2020.02.04.20020503</td><td>null</td><td>null</td><td>medrvix</td><td>We developed a computational tool to assess the risk of novel coronavirus outbreaks outside China. We estimate the dependence of the risk of a major outbreak in a country from imported cases on key parameters such as: (i) the evolution of the cumulative number of cases in mainland China outside the closed areas; (ii) the connectivity of the destination country with China, including baseline travel frequencies, the effect of travel restrictions, and the efficacy of entry screening at destination; (iii) the efficacy of control measures in the destination country (expressed by the local reproduction number Rloc). We found that in countries with low connectivity to China but with relatively high Rloc, the most beneficial control measure to reduce the risk of outbreaks is a further reduction in their importation number either by entry screening or travel restrictions. Countries with high connectivity but low Rloc benefit the most from policies that further reduce Rloc. Countries in the middle should consider a combination of such policies. Risk assessments were illustrated for selected groups of countries from America, Asia and Europe, and we investigated how their risks depend on those parameters, and how the risk is increasing in time as the number of cases in China is growing.</td><td>2020-02-05</td><td>Peter Boldog; Tamas Tekeli; Zsolt Vizi; Attila Denes; Ferenc Bartha; Gergely Rost</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.20020503</td></tr><tr><td>mtv80pjo</td><td>8a1fde8c65e439496ac5810504de23ef77312f28</td><td>biorxiv</td><td>Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1</td><td>10.1101/2020.02.04.933135</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAs the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, careful analysis of its transmission and cellular mechanisms is sorely needed. In this report, we re-analyzed the computational approaches and findings presented in two recent manuscripts by Ji et al. (https://doi.org/10.1002/jmv.25682) and by Pradhan et al. (https://doi.org/10.1101/2020.01.30.927871), which concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions shared a unique similarity to HIV-1. Results from our re-implementation of the analyses, built on larger-scale datasets using state-of-the-art bioinformatics methods and databases, do not support the conclusions proposed by these manuscripts. Based on our analyses and existing data of coronaviruses, we concluded that the intermediate hosts of 2019-nCoV are more likely to be mammals and birds than snakes, and that the “novel insertions” observed in the spike protein are naturally evolved from bat coronaviruses.</td><td>2020-02-08</td><td>Chengxin Zhang; Wei Zheng; Xiaoqiang Huang; Eric W. Bell; Xiaogen Zhou; Yang Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.933135</td></tr><tr><td>6ojmmmuj</td><td>74b00f19c3af87d1081644f02490ba250f57b7ca</td><td>biorxiv</td><td>Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach</td><td>10.1101/2020.02.04.934232</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackgroundNew endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a “public-health emergency of international concern” due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.MethodSeveral techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.ResultsExtensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.ConclusionT cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.</td><td>2020-02-11</td><td>Miyssa I. Abdelmageed; Abdelrahman H. Abdelmoneim; Mujahed I. Mustafa; Nafisa M. Elfadol; Naseem S. Murshed; Shaza W. Shantier; Abdelrafie M. Makhawi</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.04.934232</td></tr><tr><td>uqkcbs2s</td><td>59eab95c43fdea01481fdbf9bae45dfe28ffc693</td><td>medrxiv</td><td>Bulk and single-cell transcriptomics identify tobacco-use disparity in lung gene expression of ACE2, the receptor of 2019-nCov</td><td>10.1101/2020.02.05.20020107</td><td>null</td><td>null</td><td>medrvix</td><td>In current severe global emergency situation of 2019-nCov outbreak, it is imperative to identify vulnerable and susceptible groups for effective protection and care. Recently, studies found that 2019-nCov and SARS-nCov share the same receptor, ACE2. In this study, we analyzed five large-scale bulk transcriptomic datasets of normal lung tissue and two single-cell transcriptomic datasets to investigate the disparities related to race, age, gender and smoking status in ACE2 gene expression and its distribution among cell types. We didn't find significant disparities in ACE2 gene expression between racial groups (Asian vs Caucasian), age groups (>60 vs <60) or gender groups (male vs female). However, we observed significantly higher ACE2 gene expression in former smoker's lung compared to non-smoker's lung. Also, we found higher ACE2 gene expression in Asian current smokers compared to non-smokers but not in Caucasian current smokers, which may indicate an existence of gene-smoking interaction. In addition, we found that ACE2 gene is expressed in specific cell types related to smoking history and location. In bronchial epithelium, ACE2 is actively expressed in goblet cells of current smokers and club cells of non-smokers. In alveoli, ACE2 is actively expressed in remodelled AT2 cells of former smokers. Together, this study indicates that smokers especially former smokers may be more susceptible to 2019-nCov and have infection paths different with non-smokers. Thus, smoking history may provide valuable information in identifying susceptible population and standardizing treatment regimen.</td><td>2020-02-11</td><td>Guoshuai Cai</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.05.20020107</td></tr><tr><td>xtmozkf6</td><td>f5d9ab798adf8face1655d476644ebb75fbfef18</td><td>medrxiv</td><td>ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</td><td>10.1101/2020.02.05.20020545</td><td>null</td><td>null</td><td>medrvix</td><td>Respiratory disease caused by the 2019 novel coronavirus (2019-nCoV) pneumonia first emerged in Wuhan, Hubei Province, China, in December 2019 and spread rapidly to other provinces and other countries. Angiotensin-converting enzyme 2 (ACE2) is the receptor for SARS-CoV and has been suggested to be also the receptor for 2019-nCoV. Paradoxically, ACE2 expression in the lung protects mice from SARS-CoV spike protein induced lung injury by attenuating the renin-angiotensin system. In the intestine, ACE2 also suppresses intestinal inflammation by maintaining amino acid homeostasis, antimicrobial peptide expression and ecology of the gut microbiome. Upon analysis of single cell-RNA sequencing data from control subjects and those with colitis or inflammatory bowel disease (IBD), we found that ACE2 expression in the colonocytes was positively associated with genes regulating viral infection, innate and cellular immunity, but was negatively associated with viral transcription, protein translation, humoral immunity, phagocytosis and complement activation. In summary, we suggest that ACE2 may play dual roles in mediating the susceptibility and immunity of 2019-nCoV infection.</td><td>2020-02-07</td><td>Jun Wang; Shanmeizi Zhao; Ming Liu; Zhiyao Zhao; Yiping Xu; Ping Wang; Meng Lin; Yanhui Xu; Bing Huang; Xiaoyu Zuo; Zhanghua Chen; Fan Bai; Jun Cui; Andrew M Lew; Jincun Zhao; Yan Zhang; Haibin Luo; Yuxia Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.05.20020545</td></tr><tr><td>vpouyyev</td><td>5f6a410a4ba086b296100a8a5a10df59eb337c90</td><td>medrxiv</td><td>Epidemic doubling time of the COVID-19 epidemic by Chinese province</td><td>10.1101/2020.02.05.20020750</td><td>null</td><td>null</td><td>medrvix</td><td>COVID-19 epidemic doubling time by Chinese province was increasing from January 20 through February 9, 2020. Yet, the harmonic mean doubling time was relatively short, ranging from 1.4 (Hunan, 95% CI, 1.2-2.0) to 3.0 (Xinjiang, 95% CI, 2.0-4.9) days, with an estimate of 2.5 days (95% CI, 2.4-2.7) for Hubei.</td><td>2020-02-06</td><td>Kamalich Muniz-Rodriguez; Gerardo Chowell; Chi-Hin Cheung; Dongyu Jia; Po-Ying Lai; Yiseul Lee; Manyun Liu; Sylvia K. Ofori; Kimberlyn M. Roosa; Lone Simonsen; Cecile G. Viboud; Isaac Chun-Hai Fung</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.05.20020750</td></tr><tr><td>4i97fpzc</td><td>8762adc9311e5803d1de438493772269acfe7fa3</td><td>medrxiv</td><td>Preparedness and vulnerability of African countries against introductions of 2019-nCoV</td><td>10.1101/2020.02.05.20020792</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus (2019-nCoV) epidemic has spread to 23 countries from China. Local cycles of transmission already occurred in 7 countries following case importation. No African country has reported cases yet. The management and control of 2019-nCoV introductions heavily relies on the public health capacity of a country. Here we evaluate the preparedness and vulnerability of African countries against their risk of importation of 2019-nCoV. We used data on air travel volumes departing from airports in the infected provinces in China and directed to Africa to estimate the risk of introduction per country. We determined the countries capacity to detect and respond to cases with two indicators: preparedness, using the WHO International Health Regulation Monitoring and Evaluation Framework; and vulnerability, with the Infectious Disease Vulnerability Index. Countries were clustered according to the Chinese regions contributing the most to their risk. Findings: Countries at the highest importation risk (Egypt, Algeria, Republic of South Africa) have moderate to high capacity to respond to outbreaks. Countries at moderate risk (Nigeria, Ethiopia, Sudan, Angola, Tanzania, Ghana, Kenya) have variable capacity and high vulnerability. Three clusters of countries are identified that share the same exposure to the risk originating from the provinces of Guangdong, Fujian, and Beijing, respectively. Interpretation: Several countries in Africa are stepping up their preparedness to detect and cope with 2019-nCoV importations. Resources and intensified surveillance and capacity capacity should be urgently prioritized towards countries at moderate risk that may be ill-prepared to face the importation and to limit onward transmission.</td><td>2020-02-07</td><td>Marius Gilbert; Giulia Pullano; Francesco Pinotti; Eugenio Valdano; Chiara Poletto; Pierre-Yves Boelle; Eric D'Ortenzio; Yazdan Yazdanpanah; Serge Paul Eholie; Mathias Altmann; Bernardo Gutierrez; Moritz U. G. Kraemer; Vittoria Colizza</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.05.20020792</td></tr><tr><td>jawwrsa5</td><td>null</td><td>biorxiv</td><td>A database resource for Genome-wide dynamics analysis of Coronaviruses on a historical and global scale</td><td>10.1101/2020.02.05.920009</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe recent outbreak of a new zoonotic origin Coronavirus has ring the bell for the potential spread of epidemic Coronavirus crossing the species. With the urgent needs to assist the control of the Coronavirus spread and to provide valuable scientific information, we developed a coronavirus database (CoVdb), an online genomics and proteomics analysis platform. Based on public available coronavirus genomic information, the database annotates the genome of every strain and identifies 780 possible ORFs of all strains available in Genebank. In addition, the comprehensive evaluation of all the published genomes of Coronavirus strains, including population genetics analysis, functional analysis and structural analysis on a historical and global scale were presented in the CoVdb. In the database, the researcher can easily obtain the basic information of a Coronavirus gene with the distribution of the gene among strains, conserved or high mutation regions, possible subcellular location and topology of the gene. Moreover, sliding windows for population genetics analysis results is provided, thereby facilitating genetics and evolutional analysis at the genomic level. CoVdb can be accessed freely at http://covdb.popgenetics.net.</td><td>2020-02-07</td><td>Zhenglin Zhu; Kaiwen Meng; Geng Meng</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.05.920009</td></tr><tr><td>7vrm081c</td><td>73c8af41cfdbf52c0dfba37727e3b94cb56b495e</td><td>biorxiv</td><td>The Essential Facts of Wuhan Novel Coronavirus Outbreak in China and Epitope-based Vaccine Designing against COVID-19</td><td>10.1101/2020.02.05.935072</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractWuhan Novel Coronavirus disease (COVID-19) outbreak has become a global outbreak which has raised the concern of scientific community to design and discover a definitive cure against this deadly virus which has caused deaths of numerous infected people upon infection and spreading. To date, no antiviral therapy or vaccine is available which can effectively combat the infection caused by this virus. This study was conducted to design possible epitope-based subunit vaccines against the SARS-CoV-2 virus using the approaches of reverse vaccinology and immunoinformatics. Upon continual computational experimentation three possible vaccine constructs were designed and one vaccine construct was selected as the best vaccine based on molecular docking study which is supposed to effectively act against SARS-CoV-2. Later, molecular dynamics simulation and in silico codon adaptation experiments were carried out in order to check biological stability and find effective mass production strategy of the selected vaccine. Hopefully, this study will contribute to uphold the present efforts of the researches to secure a definitive treatment against this lethal virus.</td><td>2020-02-11</td><td>Bishajit Sarkar; Md. Asad Ullah; Fatema Tuz Johora; Masuma Afrin Taniya; Yusha Araf</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.05.935072</td></tr><tr><td>py6qu4tl</td><td>dca8ced82157924ed86c698a7dd482be81b4b266</td><td>biorxiv</td><td>Teicoplanin potently blocks the cell entry of 2019-nCoV</td><td>10.1101/2020.02.05.935387</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSince December 2019, the outbreak of a new coronavirus, named 2019-nCoV, has greatly threatened the public health in China and raised great concerns worldwide. No specific treatment for this infection is currently available. We previously reported that teicoplanin, a glycopeptide antibiotic which has routinely been used in the clinic to treat bacterial infection with low toxicity, significantly inhibits the invasion of cells by Ebola virus, SARS-CoV and MERS-CoV, via specifically inhibiting the activity of cathepsin L. Here, we tested the efficacy of teicoplanin against 2019-nCoV virus infection and found that teicoplanin potently prevents the entrance of 2019-nCoV-Spike-pseudoviruses into the cytoplasm, with an IC50 of 1.66 μM. Although the inhibitory effect upon the replication of wildtype viruses ex vivo and in vivo remains to be determined, our preliminary result indicates that the potential antiviral activity of teicoplanin could be applied for the treatment of 2019-nCoV virus infection.</td><td>2020-02-13</td><td>Junsong Zhang; Xiancai Ma; Fei Yu; Jun Liu; Fan Zou; Ting Pan; Hui Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.05.935387</td></tr><tr><td>g5wpa2ee</td><td>0aff00101d5ccc6592987185ab833f95d842f98b</td><td>biorxiv</td><td>Potentially highly potent drugs for 2019-nCoV</td><td>10.1101/2020.02.05.936013</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe World Health Organization (WHO) has declared the 2019 novel coronavirus (2019-nCoV) infection outbreak a global health emergency. Currently, there is no effective anti-2019-nCoV medication. The sequence identity of the 3CL proteases of 2019-nCoV and SARS is 96%, which provides a sound foundation for structural-based drug repositioning (SBDR). Based on a SARS 3CL protease X-ray crystal structure, we construct a 3D homology structure of 2019-nCoV 3CL protease. Based on this structure and existing experimental datasets for SARS 3CL protease inhibitors, we develop an SBDR model based on machine learning and mathematics to screen 1465 drugs in the DrugBank that have been approved by the U.S. Food and Drug Administration (FDA). We found that many FDA approved drugs are potentially highly potent to 2019-nCoV.</td><td>2020-02-13</td><td>Duc Duy Nguyen; Kaifu Gao; Jiahui Chen; Rui Wang; Guo-Wei Wei</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.05.936013</td></tr><tr><td>ra3t6kmm</td><td>c85f571a674c7fed0ccb9176e9cf9f3d3659ca32</td><td>medrxiv</td><td>Analysis of the epidemic growth of the early 2019-nCoV outbreak using internationally confirmed cases</td><td>10.1101/2020.02.06.20020941</td><td>null</td><td>null</td><td>medrvix</td><td>Background: On January 23, 2020, a quarantine was imposed on travel in and out of Wuhan, where the 2019 novel coronavirus (2019-nCoV) outbreak originated from. Previous analyses estimated the basic epidemiological parameters using symptom onset dates of the confirmed cases in Wuhan and outside China. Methods: We obtained information on the 46 coronavirus cases who traveled from Wuhan before January 23 and have been subsequently confirmed in Hong Kong, Japan, Korea, Macau, Singapore, and Taiwan as of February 5, 2020. Most cases have detailed travel history and disease progress. Compared to previous analyses, an important distinction is that we used this data to informatively simulate the infection time of each case using the symptom onset time, previously reported incubation interval, and travel history. We then fitted a simple exponential growth model with adjustment for the January 23 travel ban to the distribution of the simulated infection time. We used a Bayesian analysis with diffuse priors to quantify the uncertainty of the estimated epidemiological parameters. We performed sensitivity analysis to different choices of incubation interval and the hyperparameters in the prior specification. Results: We found that our model provides good fit to the distribution of the infection time. Assuming the travel rate to the selected countries and regions is constant over the study period, we found that the epidemic was doubling in size every 2.9 days (95% credible interval [CrI], 2 days--4.1 days). Using previously reported serial interval for 2019-nCoV, the estimated basic reproduction number is 5.7 (95% CrI, 3.4--9.2). The estimates did not change substantially if we assumed the travel rate doubled in the last 3 days before January 23, when we used previously reported incubation interval for severe acute respiratory syndrome (SARS), or when we changed the hyperparameters in our prior specification. Conclusions: Our estimated epidemiological parameters are higher than an earlier report using confirmed cases in Wuhan. This indicates the 2019-nCoV could have been spreading faster than previous estimates.</td><td>2020-02-09</td><td>Qingyuan Zhao; Yang Chen; Dylan S Small</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.06.20020941</td></tr><tr><td>tovfd9lw</td><td>dfb0fedbeed56bd2b795a67faab28295afc14c96</td><td>medrxiv</td><td>Clinical characteristics of 2019 novel coronavirus infection in China</td><td>10.1101/2020.02.06.20020974</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since December 2019, acute respiratory disease (ARD) due to 2019 novel coronavirus (2019-nCoV) emerged in Wuhan city and rapidly spread throughout China. We sought to delineate the clinical characteristics of these cases. Methods: We extracted the data on 1,099 patients with laboratory-confirmed 2019-nCoV ARD from 552 hospitals in 31 provinces/provincial municipalities through January 29th, 2020. Results: The median age was 47.0 years, and 41.90% were females. Only 1.18% of patients had a direct contact with wildlife, whereas 31.30% had been to Wuhan and 71.80% had contacted with people from Wuhan. Fever (87.9%) and cough (67.7%) were the most common symptoms. Diarrhea is uncommon. The median incubation period was 3.0 days (range, 0 to 24.0 days). On admission, ground-glass opacity was the typical radiological finding on chest computed tomography (50.00%). Significantly more severe cases were diagnosed by symptoms plus reverse-transcriptase polymerase-chain-reaction without abnormal radiological findings than non-severe cases (23.87% vs. 5.20%, P<0.001). Lymphopenia was observed in 82.1% of patients. 55 patients (5.00%) were admitted to intensive care unit and 15 (1.36%) succumbed. Severe pneumonia was independently associated with either the admission to intensive care unit, mechanical ventilation, or death in multivariate competing-risk model (sub-distribution hazards ratio, 9.80; 95% confidence interval, 4.06 to 23.67). Conclusions: The 2019-nCoV epidemic spreads rapidly by human-to-human transmission. Normal radiologic findings are present among some patients with 2019-nCoV infection. The disease severity (including oxygen saturation, respiratory rate, blood leukocyte/lymphocyte count and chest X-ray/CT manifestations) predict poor clinical outcomes.</td><td>2020-02-09</td><td>Wei-jie Guan; Zheng-yi Ni; Yu Hu; Wen-hua Liang; Chun-quan Ou; Jian-xing He; Lei Liu; Hong Shan; Chun-liang Lei; David SC Hui; Bin Du; Lan-juan Li; Guang Zeng; Kowk-Yung Yuen; Ru-chong Chen; Chun-li Tang; Tao Wang; Ping-yan Chen; Jie Xiang; Shi-yue Li; Jin-lin Wang; Zi-jing Liang; Yi-xiang Peng; Li Wei; Yong Liu; Ya-hua Hu; Peng Peng; Jian-ming Wang; Ji-yang Liu; Zhong Chen; Gang Li; Zhi-jian Zheng; Shao-qin Qiu; Jie Luo; Chang-jiang Ye; Shao-yong Zhu; Nan-shan Zhong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.06.20020974</td></tr><tr><td>77nsidzc</td><td>4c24ee4a86de1d5207d93ce39132f7b660cf82e0</td><td>biorxiv</td><td>Freshwater monitoring by nanopore sequencing</td><td>10.1101/2020.02.06.936302</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTClean freshwater lies at the heart of human society and monitoring its quality is paramount. In addition to chemical controls, traditional microbiological water tests focus on the detection of specific bacterial pathogens. The direct tracing of all aquatic DNA poses a more profound alternative. Yet, this has hitherto been underused due to challenges in cost and logistics. Here we present a simple, fast, inexpensive and comprehensive freshwater diagnostics workflow centred around portable nanopore DNA sequencing. Using defined bacterial compositions and spatiotemporal microbiota from surface water of an example river in Cambridge (UK), our study shows how nanopore sequencing can be readily integrated for the assessment of aquatic bacterial diversity and pollution. We provide a computational benchmark that features more than ten taxonomic classification tools to derive guidelines for bacterial DNA analyses with nanopore data. Through complementary physicochemical measurements, we find that nanopore metagenomics can depict fine temporal gradients along the main hydrological axis of an urban-rural interface and yield high-resolution pathogen maps that address concerns of public health.</td><td>2020-02-07</td><td>Lara Urban; Andre Holzer; J Jotautas Baronas; Michael Hall; Philipp Braeuninger-Weimer; Michael J Scherm; Daniel J Kunz; Surangi N Perera; Daniel E Martin-Herranz; Edward T Tipper; Susannah J Salter; Maximilian R Stammnitz</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.06.936302</td></tr><tr><td>q14x0i2c</td><td>ac35eeb6eaaf382fe0e123719f7dc409046e5d87</td><td>medrxiv</td><td>Incorporating Human Movement Data to Improve Epidemiological Estimates for 2019-nCoV</td><td>10.1101/2020.02.07.20021071</td><td>null</td><td>null</td><td>medrvix</td><td>Estimating the key epidemiological features of the novel coronavirus (2019-nCoV) epidemic proves to be challenging, given incompleteness and delays in early data reporting, in particular, the severe under-reporting bias in the epicenter, Wuhan, Hubei Province, China. As a result, the current literature reports widely varying estimates. We developed an alternative geo-stratified debiasing estimation framework by incorporating human mobility with case reporting data in three stratified zones, i.e., Wuhan, Hubei Province excluding Wuhan, and mainland China excluding Hubei. We estimated the latent infection ratio to be around 0.12% (18,556 people) and the basic reproduction number to be 3.24 in Wuhan before the city's lockdown on January 23, 2020. The findings based on this debiasing framework have important implications to prioritization of control and prevention efforts.</td><td>2020-02-09</td><td>Zhidong Cao; Qingpeng Zhang; Xin Lu; Dirk Pfeiffer; Lei Wang; Hongbing Song; Tao Pei; Zhongwei Jia; Daniel Dajun Zeng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.20021071</td></tr><tr><td>kevrp8rg</td><td>d5f14e63b86239f491c9dd1749cff01dc4d633cb</td><td>medrxiv</td><td>Networks of information token recurrences derived from genomic sequences may reveal hidden patterns in epidemic outbreaks: A case study of the 2019-nCoV coronavirus.</td><td>10.1101/2020.02.07.20021139</td><td>null</td><td>null</td><td>medrvix</td><td>Profiling the genetic evolution and dynamic spreading of viruses is a crucial task when responding to epidemic outbreaks. We aim to devise novel ways to model, visualise and analyse the temporal dynamics of epidemic outbreaks in order to help researchers and other people involved in crisis response to make well-informed and targeted decisions about from which geographical locations and time periods more genetic samples may be required to fully understand the outbreak. Our approach relies on the application of Transcendental Information Cascades to a set of temporally ordered nucleotide sequences and we apply it to real-world data that was collected during the currently ongoing outbreak of the novel 2019-nCoV coronavirus. We assess information-theoretic and network-theoretic measures that characterise the resulting complex network and suggest touching points and temporal pathways that are of interest for deeper investigation by geneticists and epidemiologists.</td><td>2020-02-11</td><td>Markus Luczak-Roesch</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.20021139</td></tr><tr><td>45g12waw</td><td>36a5f6d55d7c5f67d4344e36da0a72856ad3dda0</td><td>medrxiv</td><td>The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated</td><td>10.1101/2020.02.07.20021154</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus (2019-nCoV) is a recently emerged human pathogen that has spread widely since January 2020. Initially, the basic reproductive number, R0, was estimated to be 2.2 to 2.7. Here we provide a new estimate of this quantity. We collected extensive individual case reports and estimated key epidemiology parameters, including the incubation period. Integrating these estimates and high-resolution real-time human travel and infection data with mathematical models, we estimated that the number of infected individuals during early epidemic double every 2.4 days, and the R0 value is likely to be between 4.7 and 6.6. We further show that quarantine and contact tracing of symptomatic individuals alone may not be effective and early, strong control measures are needed to stop transmission of the virus.</td><td>2020-02-11</td><td>Steven Sanche; Yen Ting Lin; Chonggang Xu; Ethan Romero-Severson; Nick Hengartner; Ruian Ke</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.20021154</td></tr><tr><td>7axgotby</td><td>ba92c589f077b55e9cd13264d619c2ffe9d3ce7a</td><td>medrxiv</td><td>Transmission Dynamics of 2019-nCoV in Malaysia</td><td>10.1101/2020.02.07.20021188</td><td>null</td><td>null</td><td>medrvix</td><td>This paper focuses on the formulation of a deterministic 2019-nCov transmission model by considering the exposed and recovered populations with immunity. The scenario of the simulation is depicted based on the patient zero in Malaysia. The transmission model is found to be able to predict the next confirmed case given a single case introduced in a fully susceptible population. The mathematical model is developed based on the SEIR model which has susceptible, exposed, infectious and recovered populations. The system of equations which were obtained were solved numerically and the simulation results were analyzed. The analysis includes the impact of the disease if no control is taken.</td><td>2020-02-11</td><td>Jane Labadin; Boon Hao Hong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.20021188</td></tr><tr><td>y4we4rrk</td><td>null</td><td>medrxiv</td><td>Tracking the spread of novel coronavirus (2019-nCoV) based on big data</td><td>10.1101/2020.02.07.20021196</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus (2019-nCoV) appeared in Wuhan in late 2019 have infected 34,598 people, and killed 723 among them until 8th February 2020. The new virus has spread to at least 316 cities (until 1st February 2020) in China. We used the traffic flow data from Baidu Map, and number of air passengers who left Wuhan from 1st January to 26th January, to quantify the potential infectious people. We developed multiple linear models with local population and air passengers as predicted variables to explain the variance of confirmed cases in every city across China. We found the contribution of air passengers from Wuhan was decreasing gradually, but the effect of local population was increasing, indicating the trend of local transmission. However, the increase of local transmission is slow during the early stage of novel coronavirus, due to the super strict control measures carried out by government agents and communities.</td><td>2020-02-11</td><td>Xumao Zhao; Xiang Liu; Xinhai Li</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.20021196</td></tr><tr><td>ptbk69rk</td><td>3e5633a9a76a8c03e4ec57e4659872a63f045117</td><td>biorxiv</td><td>Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group</td><td>10.1101/2020.02.07.937862</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe present outbreak of lower respiratory tract infections, including respiratory distress syndrome, is the third spillover, in only two decades, of an animal coronavirus to humans resulting in a major epidemic. Here, the Coronavirus Study Group (CSG) of the International Committee on Taxonomy of Viruses, which is responsible for developing the official classification of viruses and taxa naming (taxonomy) of the Coronaviridae family, assessed the novelty of the human pathogen tentatively named 2019-nCoV. Based on phylogeny, taxonomy and established practice, the CSG formally recognizes this virus as a sister to severe acute respiratory syndrome coronaviruses (SARS-CoVs) of the species Severe acute respiratory syndrome-related coronavirus and designates it as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). To facilitate communication, the CSG further proposes to use the following naming convention for individual isolates: SARS-CoV-2/Isolate/Host/Date/Location. The spectrum of clinical manifestations associated with SARS-CoV-2 infections in humans remains to be determined. The independent zoonotic transmission of SARS-CoV and SARS-CoV-2 highlights the need for studying the entire (virus) species to complement research focused on individual pathogenic viruses of immediate significance. This research will improve our understanding of virus-host interactions in an ever-changing environment and enhance our preparedness for future outbreaks.</td><td>2020-02-11</td><td>Alexander E. Gorbalenya; Susan C. Baker; Ralph S. Baric; Raoul J. de Groot; Christian Drosten; Anastasia A. Gulyaeva; Bart L. Haagmans; Chris Lauber; Andrey M Leontovich; Benjamin W. Neuman; Dmitry Penzar; Stanley Perlman; Leo L.M. Poon; Dmitry Samborskiy; Igor A. Sidorov; Isabel Sola; John Ziebuhr</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.937862</td></tr><tr><td>kh06egig</td><td>null</td><td>biorxiv</td><td>Exploring the coronavirus epidemic using the new WashU Virus Genome Browser</td><td>10.1101/2020.02.07.939124</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSince its debut in mid-December, 2019, the novel coronavirus (2019-nCoV) has rapidly spread from its origin in Wuhan, China, to several countries across the globe, leading to a global health crisis. As of February 7, 2020, 44 strains of the virus have been sequenced and uploaded to NCBI’s GenBank [1], providing insight into the virus’s evolutionary history and pathogenesis. Here, we present the WashU Virus Genome Browser, a web-based portal for viewing virus genomic data. The browser is home to 16 complete 2019-nCoV genome sequences, together with hundreds of related viral sequences including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and Ebola virus. In addition, the browser features unique customizability, supporting user-provided upload of novel viral sequences in various formats. Sequences can be viewed in both a track-based representation as well as a phylogenetic tree-based view, allowing the user to easily compare sequence features across multiple strains. The WashU Virus Genome Browser inherited many features and track types from the WashU Epigenome Browser, and additionally incorporated a new type of SNV track to address the specific needs of viral research. Our Virus Browser portal can be accessed at https://virusgateway.wustl.edu, and documentation is available at https://virusgateway.readthedocs.io/.</td><td>2020-02-11</td><td>Jennifer A. Flynn; Deepak Purushotham; Mayank NK Choudhary; Xiaoyu Zhuo; Changxu Fan; Gavriel Matt; Daofeng Li; Ting Wang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.939124</td></tr><tr><td>dnxhtbxn</td><td>adb09e4e5c7331b2aa661b3d3bb0a643e00d11bc</td><td>biorxiv</td><td>Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019</td><td>10.1101/2020.02.07.939207</td><td>null</td><td>null</td><td>biorxiv</td><td>SUMMARYA novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019.</td><td>2020-02-13</td><td>Matthew C. Wong; Sara J. Javornik Cregeen; Nadim J. Ajami; Joseph F. Petrosino</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.939207</td></tr><tr><td>25uy4lrw</td><td>null</td><td>biorxiv</td><td>The Pathogenicity of SARS-CoV-2 in hACE2 Transgenic Mice</td><td>10.1101/2020.02.07.939389</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSevere acute respiratory syndrome CoV-2 (SARS-CoV-2) caused the Corona Virus Disease 2019 (COVID-19) cases in China has become a public health emergency of international concern (PHEIC). Based on angiotensin converting enzyme 2 (ACE2) as cell entry receptor of SARS-CoV, we used the hACE2 transgenic mice infected with SARS-CoV-2 to study the pathogenicity of the virus. Weight loss and virus replication in lung were observed in hACE2 mice infected with SARS-CoV-2. The typical histopathology was interstitial pneumonia with infiltration of significant lymphocytes and monocytes in alveolar interstitium, and accumulation of macrophages in alveolar cavities. Viral antigens were observed in the bronchial epithelial cells, alveolar macrophages and alveolar epithelia. The phenomenon was not found in wild type mice with SARS-CoV-2 infection. The pathogenicity of SARS-CoV-2 in hACE2 mice was clarified and the Koch’s postulates were fulfilled as well, and the mouse model may facilitate the development of therapeutics and vaccines against SARS-CoV-2.</td><td>2020-02-11</td><td>Linlin Bao; Wei Deng; Baoying Huang; Hong Gao; Jiangning Liu; Lili Ren; Qiang Wei; Pin Yu; Yanfeng Xu; Feifei Qi; Yajin Qu; Fengdi Li; Qi Lv; Wenling Wang; Jing Xue; Shuran Gong; Mingya Liu; Guanpeng Wang; Shunyi Wang; Zhiqi Song; Linna Zhao; Peipei Liu; Li Zhao; Fei Ye; Huijuan Wang; Weimin Zhou; Na Zhu; Wei Zhen; Haisheng Yu; Xiaojuan Zhang; Li Guo; Lan Chen; Conghui Wang; Ying Wang; Xinming Wang; Yan Xiao; Qiangming Sun; Hongqi Liu; Fanli Zhu; Chunxia Ma; Lingmei Yan; Mengli Yang; Jun Han; Wenbo Xu; Wenjie Tan; Xiaozhong Peng; Qi Jin; Guizhen Wu; Chuan Qin</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.07.939389</td></tr><tr><td>a8ig607t</td><td>2cc809ed4d5c3640ee2351fdb1876e8ff6c01b51</td><td>medrxiv</td><td>Feasibility of controlling 2019-nCoV outbreaks by isolation of cases and contacts</td><td>10.1101/2020.02.08.20021162</td><td>null</td><td>null</td><td>medrvix</td><td>Background: To assess the viability of isolation and contact tracing to control onwards transmission from imported cases of 2019-nCoV. Methods: We developed a stochastic transmission model, parameterised to the 2019-nCoV outbreak. We used the model to quantify the potential effectiveness of contact tracing and isolation of cases at controlling a 2019 nCoV-like pathogen. We considered scenarios that varied in: the number of initial cases; the basic reproduction number R0; the delay from symptom onset to isolation; the probability contacts were traced; the proportion of transmission that occurred before symptom onset, and the proportion of subclinical infections. We assumed isolation prevented all further transmission in the model. Outbreaks were deemed controlled if transmission ended within 12 weeks or before 5000 cases in total. We measured the success of controlling outbreaks using isolation and contact tracing, and quantified the weekly maximum number of cases traced to measure feasibility of public health effort. Findings: While simulated outbreaks starting with only 5 initial cases, R0 of 1.5 and little transmission before symptom onset could be controlled even with low contact tracing probability, the prospects of controlling an outbreak dramatically dropped with the number of initial cases, with higher R0, and with more transmission before symptom onset. Across different initial numbers of cases, the majority of scenarios with an R0 of 1.5 were controllable with under 50% of contacts successfully traced. For R0 of 2.5 and 3.5, more than 70% and 90% of contacts respectively had to be traced to control the majority of outbreaks. The delay between symptom onset and isolation played the largest role in determining whether an outbreak was controllable for lower values of R0. For higher values of R0 and a large initial number of cases, contact tracing and isolation was only potentially feasible when less than 1% of transmission occurred before symptom onset. Interpretation: We found that in most scenarios contact tracing and case isolation alone is unlikely to control a new outbreak of 2019-nCov within three months. The probability of control decreases with longer delays from symptom onset to isolation, fewer cases ascertained by contact tracing, and increasing transmission before symptoms. This model can be modified to reflect updated transmission characteristics and more specific definitions of outbreak control to assess the potential success of local response efforts.</td><td>2020-02-11</td><td>Joel Hellewell; Sam Abbott; Amy Gimma; Nikos I Bosse; Christopher I Jarvis; Timothy W Russell; James D Munday; Adam J Kucharski; W John Edmunds; CMMID nCoV working group; Sebastian Funk; Rosalind M Eggo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.20021162</td></tr><tr><td>0qwwycnc</td><td>null</td><td>medrxiv</td><td>Caution on Kidney Dysfunctions of 2019-nCoV Patients</td><td>10.1101/2020.02.08.20021212</td><td>null</td><td>null</td><td>medrvix</td><td>Until 24:00 of February 7th 2020, 31774 laboratory-confirmed cases of novel coronavirus (2019-nCoV) infection have been reported, including 6101 severe cases in critical conditions and 722 deaths. The critical and urgent need at this moment is to find an effective treatment strategy with available means to prevent these thousands of severe inpatients from worsening and dying. It has been recently known that the 2019-nCoV shares a common cellular mechanism with the severe acute respiratory syndrome-associated coronavirus (SARS-CoV). Thus, we surveyed a previous retrospective case study on SARS which showed that acute renal impairment was uncommon in SARS but carried a formidably high mortality (91.7%, 33 of 36 cases). Here we report an ongoing case study on kidney functions in 59 patients infected by 2019-nCoV (including 28 diagnosed as severe cases and 3 deaths). 63% (32/51) of the patients exhibited proteinuria, indicative of renal impairment. 19% (11/59) and 27% (16/59) of the patients had an elevated level of plasma creatinine and urea nitrogen respectively. The computerized tomography (CT) scan showed radiographic abnormalities of the kidneys in 100% (27/27) of the patients. Together, these multiple lines of evidence point to the idea that renal impairment is common in 2019-nCov patients, which may be one of the major causes of the illness by the virus infection and also may contribute to multi-organ failure and death eventually. Therefore, we strongly suggest exercising a high degree of caution in monitoring the kidney functions of 2019-nCoV patients and, very importantly, that applying potential interventions including continuous renal replacement therapies (CRRT) for protecting kidney functions as early as possible, particular for those with plasma creatinine rising, is key to preventing fatality.</td><td>2020-02-12</td><td>Zhen Li; Ming Wu; Jie Guo; Jiwei Yao; Xiang Liao; Siji Song; Min Han; Jiali Li; Guangjie Duan; Yuanxiu Zhou; Xiaojun Wu; Zhansong Zhou; Taojiao Wang; Ming Hu; Xianxiang Chen; Yu Fu; Chong Lei; Hailong Dong; Yi Zhou; Hongbo Jia; Xiaowei Chen; Junan Yan</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.20021212</td></tr><tr><td>juiomzb3</td><td>5bbc428de7fc2f10c24d8439edda26d84b319687</td><td>medrxiv</td><td>Estimation of the Time-Varying Reproduction Number of COVID-19 Outbreak in China</td><td>10.1101/2020.02.08.20021253</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The 2019-nCoV outbreak in Wuhan, China has attracted world-wide attention. As of February 11, 2020, a total of 44730 cases of novel coronavirus-infected pneumonia associated with COVID-19 were confirmed by the National Health Commission of China. Methods: Three approaches, namely Poisson likelihood-based method (ML), exponential growth rate-based method (EGR) and stochastic Susceptible-Infected-Removed dynamic model-based method (SIR), were implemented to estimate the basic and controlled reproduction numbers. Results: A total of 71 chains of transmission together with dates of symptoms onset and 67 dates of infections were identified among 5405 confirmed cases outside Hubei as reported by February 2, 2020. Based on this information, we find the serial interval having an average of 4.41 days with a standard deviation of 3.17 days and the infectious period having an average of 10.91 days with a standard deviation of 3.95 days. Conclusions: The controlled reproduction number is declining. It is lower than one in most regions of China, but is still larger than one in Hubei Province. Sustained efforts are needed to further reduce the Rc to below one in order to end the current epidemic.</td><td>2020-02-11</td><td>Chong You; Yuhao Deng; Wenjie Hu; Jiarui Sun; Qiushi Lin; Feng Zhou; Cheng Heng Pang; Yuan Zhang; Zhengchao Chen; Xiao-Hua Zhou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.20021253</td></tr><tr><td>2g8rfzf4</td><td>5cfb44ecdfb61b2898b6cf7315b1e810874a9560</td><td>medrxiv</td><td>Assessing the plausibility of subcritical transmission of 2019-nCoV in the United States</td><td>10.1101/2020.02.08.20021311</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract: The 2019-nCoV outbreak has raised concern of global spread. While person-to-person transmission within the Wuhan district has led to a large outbreak, the transmission potential outside of the region remains unclear. Here we present a simple approach for determining whether the upper limit of the confidence interval for the reproduction number exceeds one for transmission in the United States, which would allow endemic transmission. As of February 7, 2020, the number of cases in the United states support subcritical transmission, rather than ongoing transmission. However, this conclusion can change if pre-symptomatic cases resulting from human-to-human transmission have not yet been identified.</td><td>2020-02-11</td><td>Seth Blumberg; Thomas M Lietman; Travis C Porco</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.20021311</td></tr><tr><td>pm8usc55</td><td>null</td><td>biorxiv</td><td>The insert sequence in SARS-CoV-2 enhances spike protein cleavage by TMPRSS</td><td>10.1101/2020.02.08.926006</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAt the end of 2019, the SARS-CoV-2 induces an ongoing outbreak of pneumonia in China1, even more spread than SARS-CoV infection2. The entry of SARS-CoV into host cells mainly depends on the cell receptor (ACE2) recognition and spike protein cleavage-induced cell membrane fusion3,4. The spike protein of SARS-CoV-2 also binds to ACE2 with a similar affinity, whereas its spike protein cleavage remains unclear5,6. Here we show that an insertion sequence in the spike protein of SARS-CoV-2 enhances the cleavage efficiency, and besides pulmonary alveoli, intestinal and esophagus epithelium were also the target tissues of SARS-CoV-2. Compared with SARS-CoV, we found a SPRR insertion in the S1/S2 protease cleavage sites of SARS-CoV-2 spike protein increasing the cleavage efficiency by the protein sequence aligment and furin score calculation. Additionally, the insertion sequence facilitates the formation of an extended loop which was more suitable for protease recognition by the homology modeling and molicular docking. Furthermore, the single-cell transcriptomes identified that ACE2 and TMPRSSs are highly coexpressed in AT2 cells of lung, along with esophageal upper epithelial cells and absorptive enterocytes. Our results provide the bioinformatics evidence for the increased spike protein cleavage of SARS-CoV-2 and indicate its potential target cells.</td><td>2020-02-11</td><td>Tong Meng; Hao Cao; Hao Zhang; Zijian Kang; Da Xu; Haiyi Gong; Jing Wang; Zifu Li; Xingang Cui; Huji Xu; Haifeng Wei; Xiuwu Pan; Rongrong Zhu; Jianru Xiao; Wang Zhou; Liming Cheng; Jianmin Liu</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.926006</td></tr><tr><td>hwvr0v8c</td><td>329dfdd2fbe4b3c510c5329219bfd33764ab1710</td><td>biorxiv</td><td>Domestication reprogrammed the budding yeast life cycle</td><td>10.1101/2020.02.08.939314</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractDomestication of plants and animals is the foundation for feeding the world population. We report that domestication of the model yeast S. cerevisiae reprogrammed its life cycle entirely. We tracked growth, gamete formation and cell survival across many environments for nearly 1000 genome sequenced isolates and found a remarkable dichotomy between domesticated and wild yeasts. Wild yeasts near uniformly trigger meiosis and sporulate when encountering nutrient depletions, whereas domestication relaxed selection on sexual reproduction and favoured survival as quiescent cells. Domestication also systematically enhanced fermentative over respiratory traits while decreasing stress tolerance. We show that this yeast domestication syndrome was driven by aneuploidies and gene function losses that emerged independently in multiple domesticated lineages during the specie’s recent evolutionary history. We found domestication to be the most dramatic event in budding yeast evolution, raising questions on how much domestication has distorted our understanding of this key model species.</td><td>2020-02-09</td><td>Matteo De Chiara; Benjamin Barré; Karl Persson; Amadi Onyetuga Chioma; Agurtzane Irizar; Joseph Schacherer; Jonas Warringer; Gianni Liti</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.939314</td></tr><tr><td>xetzg7gp</td><td>17307ab6eda64e1538da204c02114dd36a9b7fd6</td><td>biorxiv</td><td>The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2</td><td>10.1101/2020.02.08.939553</td><td>null</td><td>null</td><td>biorxiv</td><td>The outbreak of the 2019 Novel Coronavirus (SARS-CoV-2) rapidly spread from Wuhan, China to more than 150 countries, areas or territories, causing staggering number of infections and deaths. A systematic profiling of the immune vulnerability landscape of SARS-CoV-2, which can bring critical insights into the immune clearance mechanism, peptide vaccine development, and antiviral antibody development, is lacking. In this study, we investigated the potential of the SARS-CoV-2 viral proteins to induce class I and II MHC presentation and to form linear antibody epitopes. We created an online database to broadly share the predictions as a resource for the research community. Using this resource, we showed that genetic variations in SARS-CoV-2, though still few for the moment, already follow the pattern of mutations in related coronaviruses, and could alter the immune vulnerability landscape of this virus. Importantly, we discovered evidence that SARS-CoV-2, along with related coronaviruses, used mutations to evade attack from the human immune system. Overall, we present an immunological resource for SARS-CoV-2 that could promote both therapeutic development and mechanistic research.</td><td>2020-02-12</td><td>James Zhu; Jiwoong Kim; Xue Xiao; Yunguan Wang; Danni Luo; Ran Chen; Lin Xu; He Zhang; Guanghua Xiao; John Schoggins; Xiaowei Zhan; Tao Wang; Yang Xie</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.939553</td></tr><tr><td>emr0eh0i</td><td>a22ceac8d5e1cfaf2e79557d3c794458d0e1cdbe</td><td>biorxiv</td><td>Identification of a Pangolin Niche for a 2019-nCoV-like Coronavirus via an Extensive Meta-metagenomic Search</td><td>10.1101/2020.02.08.939660</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractIn numerous instances, tracking the biological significance of a nucleic acid sequence can be augmented through the identification of environmental niches in which the sequence of interest is present. Many metagenomic datasets are now available, with deep sequencing of samples from diverse biological niches. While any individual metagenomic dataset can be readily queried using web-based tools, meta-searches through all such datasets are less accessible. In this brief communication, we demonstrate such a meta-meta-genomic approach, examining close matches to the Wuhan coronavirus 2019-nCoV in all high-throughput sequencing datasets in the NCBI Sequence Read Archive accessible with the keyword “virome”. In addition to the homology to bat coronaviruses observed in descriptions of the 2019-nCoV sequence (F. Wu et al. 2020, Nature, doi.org/10.1038/s41586-020-2008-3; P. Zhou et al. 2020, Nature, doi.org/10.1038/s41586-020-2012-7), we note a strong homology to numerous sequence reads in a metavirome dataset generated from the lungs of deceased Pangolins reported by Liu et al. (Viruses 11:11, 2019, http://doi.org/10.3390/v11110979). Our observations are relevant to discussions of the derivation of 2019-nCoV and illustrate the utility and limitations of meta-metagenomic search tools in effective and rapid characterization of potentially significant nucleic acid sequences.ImportanceMeta-metagenomic searches allow for high-speed, low-cost identification of potentially significant biological niches for sequences of interest.</td><td>2020-02-14</td><td>Lamia Wahba; Nimit Jain; Andrew Z. Fire; Massa J. Shoura; Karen L. Artiles; Matthew J. McCoy; Dae-Eun Jeong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.939660</td></tr><tr><td>mm2sp48e</td><td>b970b7a807f37678f91d4b989ffdb7d7606d754d</td><td>biorxiv</td><td>Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</td><td>10.1101/2020.02.08.939892</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSince December 2019, a novel coronavirus named 2019 coronavirus (2019-nCoV) has emerged in Wuhan of China and spread to several countries worldwide within just one month. Apart from fever and respiratory complications, acute kidney injury has been observed in some patients with 2019-nCoV. In a short period of time, angiotensin converting enzyme II (ACE2), have been proposed to serve as the receptor for the entry of 2019-nCoV, which is the same for severe acute respiratory syndrome coronavirus (SARS). To investigate the possible cause of kidney damage in 2019-nCoV patients, we used both published kidney and bladder cell atlas data and an independent unpublished kidney single cell RNA-Seq data generated in-house to evaluate ACE2 gene expressions in all cell types in healthy kidneys and bladders.Our results showed the enriched expression of all subtypes of proximal tubule cells of kidney and low but detectable levels of expression in bladder epithelial cells. These results indicated the urinary system is a potential route for 2019-nCoV infection, along with the respiratory system and digestion system. Our findings suggested the kidney abnormalities of SARS and 2019-nCoV patients may be due to proximal tubule cells damage and subsequent systematic inflammatory response induced kidney injury. Beyond that, laboratory tests of viruses and related indicators in urine may be needed in some special patients of 2019-nCoV.</td><td>2020-02-18</td><td>Wei Lin; Longfei Hu; Yan Zhang; Joshua D. Ooi; Ting Meng; Peng Jin; Xiang Ding; Longkai Peng; Lei Song; Zhou Xiao; Xiang Ao; Xiangcheng Xiao; Qiaoling Zhou; Ping Xiao; Jue Fan; Yong Zhong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.08.939892</td></tr><tr><td>f87h5qh6</td><td>5c31897d01f3edc7f58a0f03fceec2373fcfdc3d</td><td>medrxiv</td><td>The effect of travel restrictions on the spread of the 2019 novel coronavirus (2019-nCoV) outbreak</td><td>10.1101/2020.02.09.20021261</td><td>null</td><td>null</td><td>medrvix</td><td>Motivated by the rapid spread of a novel coronavirus (2019-nCoV) in Mainland China, we use a global metapopulation disease transmission model to project the impact of both domestic and international travel limitations on the national and international spread of the epidemic. The model is calibrated on the evidence of internationally imported cases before the implementation of the travel quarantine of Wuhan. By assuming a generation time of 7.5 days, the reproduction number is estimated to be 2.4 [90% CI 2.2-2.6]. The median estimate for number of cases before the travel ban implementation on January 23, 2020 is 58,956 [90% CI 40,759 - 87,471] in Wuhan and 3,491 [90% CI 1,924 - 7,360] in other locations in Mainland China. The model shows that as of January 23, most Chinese cities had already received a considerable number of infected cases, and the travel quarantine delays the overall epidemic progression by only 3 to 5 days. The travel quarantine has a more marked effect at the international scale, where we estimate the number of case importations to be reduced by 80% until the end of February. Modeling results also indicate that sustained 90% travel restrictions to and from Mainland China only modestly affect the epidemic trajectory unless combined with a 50% or higher reduction of transmission in the community.</td><td>2020-02-11</td><td>Matteo Chinazzi; Jessica T. Davis; Marco Ajelli; Corrado Gioannini; Maria Litvinova; Stefano Merler; Ana Pastore y Piontti; Luca Rossi; Kaiyuan Sun; Cécile Viboud; Xinyue Xiong; Hongjie Yu; M. Elizabeth Halloran; Ira M. Longini; Alessandro Vespignani</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.09.20021261</td></tr><tr><td>6u9q0ox9</td><td>2913d91f13fd59c698f68ba63008d8e0550c0607</td><td>medrxiv</td><td>Spatially Explicit Modeling of 2019-nCoV Epidemic Trend based on Mobile Phone Data in Mainland China</td><td>10.1101/2020.02.09.20021360</td><td>null</td><td>null</td><td>medrvix</td><td>IAs of February 11, 2020, all prefecture-level cities in mainland China have reported confirmed cases of 2019 novel coronavirus (2019-nCoV), but the city-level epidemical dynamics is unknown. The aim of this study is to model the current dynamics of 2019-nCoV at city level and predict the trend in the next 30 days under three possible scenarios in mainland China. We developed a spatially explicit epidemic model to consider the unique characteristics of the virus transmission in individual cities. Our model considered that the rate of virus transmission among local residents is different from those with Wuhan travel history due to the self-isolation policy. We introduced a decay rate to quantify the effort of each city to gradually control the disease spreading. We used mobile phone data to obtain the number of individuals in each city who have travel history to Wuhan. This city-level model was trained using confirmed cases up to February 10, 2020 and validated by new confirmed cases on February 11, 2020. We used the trained model to predict the future dynamics up to March 12, 2020 under different scenarios: the current trend maintained, control efforts expanded, and person-to-person contact increased due to work resuming. We estimated that the total infections in mainland China would be 72172, 54348, and 149774 by March 12, 2020 under each scenario respectively. Under the current trend, all cities will show the peak point of daily new infections by February 21. This date can be advanced to February 14 with control efforts expanded or postponed to February 26 under pressure of work resuming. Except Wuhan that cannot eliminate the disease by March 12, our model predicts that 95.4%, 100%, and 75.7% cities will have no new infections by the end of February under three scenarios. The spatial pattern of our prediction could help the government allocate resources to cities that have a more serious epidemic in the next 30 days.</td><td>2020-02-11</td><td>Xiaolin Zhu; Aiyin Zhang; Shuai Xu; Pengfei Jia; Xiaoyue Tan; Jiaqi Tian; Tao Wei; Zhenxian Quan; Jiali Yu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.09.20021360</td></tr><tr><td>5dd89gnm</td><td>d2609c4d8837558f4b8f32170422ce1cd12425a6</td><td>medrxiv</td><td>Assessing the Tendency of 2019-nCoV (COVID-19) Outbreak in China</td><td>10.1101/2020.02.09.20021444</td><td>null</td><td>null</td><td>medrvix</td><td>Since December 8, 2019, the spread of COVID-19 is increasing every day. It is particularly important to predict the trend of the epidemic for the timely adjustment of the economy and industries. We proposed a Flow-SEHIR model in this paper, based on which we further analyzed the trends of 2019-nCoV (COVID-19) in China. The results show that the basic reproductive numbers R0 of COVID-19 is 3.56 (95% CI: 2.31-4.81). The number of daily confirmed new cases reaches the inflection point on Feb. 6-10 outside Hubei. For the maximum of infected cases number, the predicted peak value in China except Hubei was estimated to be 13806 (95% CI: 11926-15845). The peak arrival time is on March 3-9. The temporal number of patients in most areas of China outside Hubei will peak from March 12 to March 15. The peak values of more than 73.5% provinces or regions in China will be controlled within 1000. According to Flow-SEHIR model and estimations from the data of evacuation of nationals from Wuhan, the peak cumulative number of patients in Hubei was estimated to be 403481 (95% CI: 143284-1166936).</td><td>2020-02-11</td><td>Qinghe Liu; Zhicheng Liu; Deqiang Li; Zefei Gao; Junkai Zhu; Junyan Yang; Qiao Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.09.20021444</td></tr><tr><td>uqgrjjwp</td><td>dcd7a1235ea74e3ef71d051103bf8a64c3c8f457</td><td>medrxiv</td><td>The lockdown of Hubei Province causing different transmission dynamics of the novel coronavirus (2019-nCoV) in Wuhan and Beijing</td><td>10.1101/2020.02.09.20021477</td><td>null</td><td>null</td><td>medrvix</td><td>Background: After the outbreak of novel coronavirus (2019-nCoV) starting in late 2019, a number of researchers have reported the predicted the virus transmission dynamics. However, under the strict control policy the novel coronavirus does not spread naturally outside Hubei Province, and none of the prediction closes to the real situation. Methods and findings: We used the traditional SEIR model, fully estimated the effect of control measures, to predict the virus transmission in Wuhan, the capital city of Hubei Province, and Beijing. We forecast that the outbreak of 2019-nCoV would reach its peak around March 6±10 in Wuhan and March 20±16 in Beijing, respectively. The infectious population in Beijing would be much less (only 0.3%) than those in Wuhan at the peak of this transmission wave. The number of confirmed cases in cities inside Hubei Province grow exponentially, whereas those in cities outside the province increase linearly. Conclusions: The unprecedented province lockdown substantially suspends the national and global outbreak of 2019-nCoV.</td><td>2020-02-11</td><td>Xinhai Li; Xumao Zhao; Yuehua Sun</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.09.20021477</td></tr><tr><td>er3zmcz2</td><td>815745bf1b522d33fd7371cc9a6561a2a93ef87e</td><td>medrxiv</td><td>Simulating the infected population and spread trend of 2019-nCov under different policy by EIR model</td><td>10.1101/2020.02.10.20021519</td><td>null</td><td>null</td><td>medrvix</td><td>Chinese government has taken strong measures in response to the epidemic of new coronavirus (2019-nCoV) from Jan.23, 2020. The number of confirmed infected individuals are still increasing rapidly. Estimating the accurate infected population and the future trend of epidemic spreading under control measures is significant and urgent. There have been reports external icon of spread from an infected patient with no symptoms to a close contact, which means the incubation individuals may has the possibility of infectiousness. However, the traditional transmission model, Susceptible-Exposed-Infectious-Recovered (SEIR) model, assumes that the exposed individual is being infected but without infectiousness. Thus, the estimating infected populations based on SEIR model from the existing literatures seems too far more than the official reported data. Here, we inferred that the epidemic could be spread by exposed (incubation) individuals. Then, we provide a new Exposed-identified-Recovered (EIR) model, and simulated the epidemic spreading processes from free propagation phase to extremely control phase. Then, we estimate of the size of the epidemic and forecast the future development of the epidemics under strong prevention interventions. According to the spread characters of 2019-nCov, we construct a novel EIR compartment system dynamics model. This model integrates two phases of the epidemic spreading: before intervention and after intervention. We assume that 2019-nCov is firstly spread without intervention then the government started to take strong quarantine measures. Use the latest reported official data, we estimate the basic parameters of the model and the basic reproduction number of 2019-nCov. Then, based on this model, we simulate the future spread of the epidemics. Both the infected population and the spreading trend of 2019-nCov under different prevention policy scenarios are estimated. The epidemic spreading trends under different quarantine rate and action starting date of prevention policy are simulated and compared.</td><td>2020-02-12</td><td>Hao Xiong; Huili Yan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.20021519</td></tr><tr><td>pd70i3d8</td><td>48656efc59191537073975938f25f201524971af</td><td>medrxiv</td><td>Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</td><td>10.1101/2020.02.10.20021584</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Severe ill patients with 2019 novel coronavirus (2019-nCoV) infection progressed rapidly to acute respiratory failure. We aimed to select the most useful prognostic factor for severe illness incidence. Methods: The study prospectively included 61 patients with 2019-nCoV infection treated at Beijing Ditan Hospital from January 13, 2020 to January 31, 2020. Prognostic factor of severe illness was selected by the LASSO COX regression analyses, to predict the severe illness probability of 2019-CoV pneumonia. The predictive accuracy was evaluated by concordance index, calibration curve, decision curve and clinical impact curve. Results: The neutrophil-to-lymphocyte ratio (NLR) was identified as the independent risk factor for severe illness in patients with 2019-nCoV infection. The NLR had a c-index of 0.807 (95% confidence interval, 0.676-0.38), the calibration curves fitted well, and the decision curve and clinical impact curve showed that the NLR had superior standardized net benefit. In addition, the incidence of severe illness was 9.1% in age ≥ 50 and NLR < 3.13 patients, and half of patients with age ≥ 50 and NLR ≥ 3.13 would develop severe illness. Based on the risk stratification of NLR with age, the study developed a 2019-nCoV pneumonia management process. Conclusions: The NLR was the early identification of risk factors for 2019-nCoV severe illness. Patients with age ≥ 50 and NLR ≥ 3.13 facilitated severe illness, and they should rapidly access to intensive care unit if necessary.</td><td>2020-02-12</td><td>Jingyuan Liu; Yao Liu; Pan Xiang; Lin Pu; Haofeng Xiong; Chuansheng Li; Ming Zhang; Jianbo Tan; Yanli Xu; Rui Song; Meihua Song; Lin Wang; Wei Zhang; Bing Han; Li Yang; Xiaojing Wang; Guiqin Zhou; Ting Zhang; Ben Li; Yanbin Wang; Zhihai Chen; Xianbo Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.20021584</td></tr><tr><td>172tl3fb</td><td>null</td><td>medrxiv</td><td>Epidemiological and clinical features of the 2019 novel coronavirus outbreak in China</td><td>10.1101/2020.02.10.20021675</td><td>null</td><td>null</td><td>medrvix</td><td>Our manuscript was based on surveillance cases of COVID-19 identified before January 26, 2020. As of February 20, 2020, the total number of confirmed cases in mainland China has reached 18 times of the number in our manuscript. While the methods and the main conclusions in our original analyses remain solid, we decided to withdraw this preprint for the time being, and will replace it with a more up-to-date version shortly. Should you have any comments or suggestions, please feel free to contact the corresponding author.</td><td>2020-02-11</td><td>Yang Yang; Qingbin Lu; Mingjin Liu; Yixing Wang; Anran Zhang; Neda Jalali; Natalie Dean; Ira Longini; M. Elizabeth Halloran; Bo Xu; Xiaoai Zhang; Liping Wang; Wei Liu; Liqun Fang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.20021675</td></tr><tr><td>zbqbogwu</td><td>06c1b3535b83251cf92c01258b5048beeab7a460</td><td>medrxiv</td><td>Beyond R0: the importance of contact tracing when predicting epidemics</td><td>10.1101/2020.02.10.20021725</td><td>null</td><td>null</td><td>medrvix</td><td>The basic reproductive number --- R0 --- is one of the most common and most commonly misapplied numbers in public health. Nevertheless, estimating R0 for every transmissible pathogen, emerging or endemic, remains a priority for epidemiologists the world over. Although often used to compare outbreaks and forecast pandemic risk, this single number belies the complexity that two different pathogens can exhibit, even when they have the same R0. Here, we show how predicting outbreak size requires both an estimate of R0 and an estimate of the heterogeneity in the number of secondary infections. To facilitate rapid determination of outbreak risk, we propose a reformulation of a classic result from random network theory that relies on contact tracing data to simultaneously determine the first moment (R0) and the higher moments (representing the heterogeneity) in the distribution of secondary infections. Further, we show how this framework is robust in the face of the typically limited amount of data for emerging pathogens. Lastly, we demonstrate that without data on the heterogeneity in secondary infections for emerging pathogens like 2019-nCoV, the uncertainty in outbreak size ranges dramatically, in the case of 2019-nCoV from 5-40% of susceptible individuals. Taken together, our work highlights the critical need for contact tracing during emerging infectious disease outbreaks and the need to look beyond R0 when predicting epidemic size.</td><td>2020-02-12</td><td>Laurent Hébert-Dufresne; Benjamin M. Althouse; Samuel V. Scarpino; Antoine Allard</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.20021725</td></tr><tr><td>5td4p2gd</td><td>null</td><td>medrxiv</td><td>Statistical Inference for Coronavirus Infected Patients in Wuhan</td><td>10.1101/2020.02.10.20021774</td><td>null</td><td>null</td><td>medrvix</td><td>Importance: The new coronavirus outbreak has seriously affected the quality of life in China. Wuhan is the disaster area, where the number of cases has increased rapidly. However, the current measures of infected patients in Wuhan are still underestimated. Objective: To estimate the overall infected patients in Wuhan from several sampled data. The correct estimated infected patients can be helpful for the government to arrange the needed beds in hospital wards to meet the actual needs. Design: We proposed to use the sampling survey to estimate the overall infected patients in Wuhan. The sampling survey is a kind of non-comprehensive survey. It selected some units from all the survey objects to carry out the survey and made the estimation and inference to all the survey objects. Sampling surveys can obtain information that reflects the overall situation, although it is not a comprehensive survey. Setting: We estimated the overall infection rate in Wenzhou city, which has a better data collection system. Simultaneously, another different samples of Wuhan tourists to Singapore will be used to validate the infection rate in Wenzhou city. Combined these two samples, we give the estimation of the number of infected patients in Wuhan and other prefecture-level cities in Hubei Province. Participants: The number of people who returned from Wuhan to Wenzhou was selected from the daily notification of the pneumonia epidemic caused by a new coronavirus infection in the city. Exposures for observational studies: The daily rate of the pneumonia epidemic caused by the new coronavirus infection in Wenzhou City. The numerator is the number of people diagnosed and whether each person diagnosed had a history of living in Wuhan. The denominator is the total number of people returning to Wenzhou from Wuhan. Based on this rate, it is reasonable to predict the number of the infected patients.</td><td>2020-02-12</td><td>Yongdao Zhou; Jianghu Dong</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.20021774</td></tr><tr><td>o7pz0rst</td><td>eec5b4ce2de8d9d00be664e6af79cac476991bbf</td><td>medrxiv</td><td>Distribution of the 2019-nCoV Epidemic and Correlation with Population Emigration from Wuhan, China</td><td>10.1101/2020.02.10.20021824</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUNDS: The ongoing new coronavirus (2019-nCoV) pneumonia outbreak is spreading in China and has not reached its peak. Five millions of people had emigrated from Wuhan before the city lockdown, which potentially represent a source of virus spreaders. Case distribution and its correlation with population emigration from Wuhan in early epidemic are of great importance for early warning and prevention of future outbreak. METHODS: The officially reported cases of 2019-nCoV pneumonia were collected as of January 30, 2020. Time and location information of these cases were extracted analyzed with ArcGIS and WinBUGS. Population migration data of Wuhan City and Hubei province were extracted from Baidu Qianxi and analyzed for their correlation with case number. FINDINGS: The 2019-nCoV pneumonia cases were predominantly distributed in Hubei and other provinces of South China. Hot spot provinces included Sichuan and Yunnan provinces that are adjacent to Hubei. While Wuhan city has the highest number of cases, the time risk is relatively stable. Numbers of cases in some cities are relatively low, but the time risks are continuously rising. The case numbers of different provinces and cities of Hubei province were highly correlated with the emigrated populations from Wuhan. Lockdown of 19 cities of Hubei province, and implementation of nationwide control measures efficiently prevented the exponential growth of case number. INTERPRETATION: Population emigrated from Wuhan was the main infection source for other cities and provinces. Some cities with low case number but were in rapid increase. Due to the upcoming Spring Festival return transport wave, understanding of the trends of risks in different regions is of great significance for preparedness for both individuals and institutions.</td><td>2020-02-12</td><td>Zeliang Chen; Qi Zhang; Yi Lu; Xi Zhang; Wenjun Zhang; Cheng Guo; Conghui Liao; Qianlin Li; Xiaohu Han; Jiahai Lu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.20021824</td></tr><tr><td>qng05jry</td><td>15ad534bb2ec7bc4b0ee83ddac11a1b837d743c6</td><td>medrxiv</td><td>Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</td><td>10.1101/2020.02.10.20021832</td><td>null</td><td>null</td><td>medrvix</td><td>Background: To explore the cellular immunity and cytokines status of NCP patients and to predict the correlation between the cellular immunity levels, cytokines and the severity of patients. Methods: 123 NCP patients were divided into mild and severe groups. Peripheral blood was collected, lymphocyte subsets and cytokines were detected. Correlation analysis was performed on the lymphocyte subsets and cytokines, and the differences between the indexes of the two groups were analyzed. Results: 102 mild and 21 severe patients were included. Lymphocyte subsets were reduced in two groups. The proportion of CD8 + T reduction in the mild and severe group was 28.43% and 61.9%, respectively; The proportion of B cell reduction was 25.49% and 28.57%; The proportion of NK cell reduction was 34.31% and 47.62%; The detection value of IL-6 was 0 in 55.88% of the mild group, mild group has a significantly lower proportion of patients with IL-6 higher than normal than severe group; There was no significant linear correlation between the lymphocyte subsets and cytokines, while significant differences were noticed between the two groups in CD4 + T, CD8 + T, IL-6 and IL-10. Conclusions: Low levels of CD4+T and CD8+T are common in severe NCP. IL-6 and IL-10 levels were higher in severe patients. T cell subsets and cytokines can be used as one of the basis for predicting the transition from mild to severe. Large number of samples are still needed to confirm the \"warning value\" of CD4 + T, CD8 + T IL-6 and IL-10.</td><td>2020-02-12</td><td>Suxin Wan; Qingjie Yi; Shibing Fan; Jinglong Lv; Xianxiang Zhang; Lian Guo; Chunhui Lang; Qing Xiao; Kaihu Xiao; Zhengjun Yi; Mao Qiang; Jianglin Xiang; Bangshuo Zhang; Yongping Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.20021832</td></tr><tr><td>7n8p9okf</td><td>83323659de2f2d5ed3ca6ada7cf137a8a17bb014</td><td>biorxiv</td><td>Alpha-ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication</td><td>10.1101/2020.02.10.936898</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate. Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs. In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases. Six crystal structures of protease:inhibitor complexes were determined as part of this study. Compounds synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic. Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera. The best near-equipotent inhibitors, 11u (P2 = cyclopentylmethyl) and 11r (P2 = cyclohexylmethyl), display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures. In Huh7 cells, 11r exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.</td><td>2020-02-10</td><td>Linlin Zhang; Daizong Lin; Yuri Kusov; Yong Nian; Qingjun Ma; Jiang Wang; Albrecht von Brunn; Pieter Leyssen; Kristina Lanko; Johan Neyts; Adriaan de Wilde; Eric J. Snijder; Hong Liu; Rolf Hilgenfeld</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.936898</td></tr><tr><td>z3x0c37y</td><td>7f8c673d136b9bc0aeda55e49cc72aacaa5688de</td><td>biorxiv</td><td>Structural genomics and interactomics of 2019 Wuhan novel coronavirus, 2019-nCoV, indicate evolutionary conserved functional regions of viral proteins</td><td>10.1101/2020.02.10.942136</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractDuring its first month, the recently emerged 2019 Wuhan novel coronavirus (2019-nCoV) has already infected many thousands of people in mainland China and worldwide and took hundreds of lives. However, the swiftly spreading virus also caused an unprecedentedly rapid response from the research community facing the unknown health challenge of potentially enormous proportions. Unfortunately, the experimental research to understand the molecular mechanisms behind the viral infection and to design a vaccine or antivirals is costly and takes months to develop. To expedite the advancement of our knowledge we leverage the data about the related coronaviruses that is readily available in public databases, and integrate these data into a single computational pipeline. As a result, we provide a comprehensive structural genomics and interactomics road-maps of 2019-nCoV and use these information to infer the possible functional differences and similarities with the related SARS coronavirus. All data are made publicly available to the research community at http://korkinlab.org/wuhan</td><td>2020-02-14</td><td>Hongzhu Cui; Ziyang Gao; Ming Liu; Senbao Lu; Sun Mo; Winnie Mkandawire; Oleksandr Narykov; Suhas Srinivasan; Dmitry Korkin</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.942136</td></tr><tr><td>xa6kwguo</td><td>7ab9f9fcea519ebce527c3ede8091beedbb26ad9</td><td>biorxiv</td><td>Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses</td><td>10.1101/2020.02.10.942185</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe 2019 novel coronavirus (2019-nCoV) is currently causing a widespread outbreak centered on Hubei province, China and is a major public health concern. Taxonomically 2019-nCoV is closely related to SARS-CoV and SARS-related bat coronaviruses, and it appears to share a common receptor with SARS-CoV (ACE-2). Here, we perform structural modeling of the 2019-nCoV spike glycoprotein. Our data provide support for the similar receptor utilization between 2019-nCoV and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains, which we predict to be proteolytically-sensitive. We suggest this loop confers fusion activation and entry properties more in line with MERS-CoV and other coronaviruses, and that the presence of this structural loop in 2019-nCoV may affect virus stability and transmission.</td><td>2020-02-18</td><td>Javier A. Jaimes; Nicole M. André; Jean K. Millet; Gary R. Whittaker</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.942185</td></tr><tr><td>bgdcm1i1</td><td>8c210df7072bc88fa7d2da72b25c84430c16f5e8</td><td>biorxiv</td><td>Recombination and lineage-specific mutations led to the emergence of SARS-CoV-2</td><td>10.1101/2020.02.10.942748</td><td>null</td><td>null</td><td>biorxiv</td><td>The recent outbreak of a new coronavirus (SARS-CoV-2) in Wuhan, China, underscores the need for understanding the evolutionary processes that drive the emergence and adaptation of zoonotic viruses in humans. Here, we show that recombination in betacoronaviruses, including human-infecting viruses like SARS-CoV and MERS-CoV, frequently encompasses the Receptor Binding Domain (RBD) in the Spike gene. We find that this common process likely led to a recombination event at least 11 years ago in an ancestor of the SARS-CoV-2 involving the RBD. As a result of this recombination event, SARS-CoV and SARS-CoV-2 share a similar genotype in RBD, including two insertions (positions 432-436 and 460-472), and alleles 427N and 436Y. Both 427N and 436Y belong to a helix that interacts with the human ACE2 receptor. Ancestral state analyses revealed that SARS-CoV-2 differentiated from its most recent common ancestor with RaTG13 by accumulating a significant number of amino acid changes in the RBD. In sum, we propose a two-hit scenario in the emergence of the SARS-CoV-2 virus whereby the SARS-CoV-2 ancestors in bats first acquired genetic characteristics of SARS-CoV by incorporation of a SARS-like RBD through recombination before 2009, and subsequently, the lineage that led to SARS-CoV-2 accumulated further, unique changes specifically in the RBD.</td><td>2020-02-18</td><td>Juan Angel Patino-Galindo; Ioan Filip; Mohammed AlQuraishi; Raul Rabadan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.942748</td></tr><tr><td>mpb4fy16</td><td>35ac99bfeb665ce0a2ae361dba1161e69eb8b7d5</td><td>biorxiv</td><td>ΦX174 Attenuation by Whole Genome Codon Deoptimization</td><td>10.1101/2020.02.10.942847</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractNatural selection acting on synonymous mutations in protein-coding genes influences genome composition and evolution. In viruses, introducing synonymous mutations in genes encoding structural proteins can drastically reduce viral growth, providing a means to generate potent, live attenuated vaccine candidates. However, an improved understanding of what compositional features are under selection and how combinations of synonymous mutations affect viral growth is needed to predictably attenuate viruses and make them resistant to reversion. We systematically recoded all non-overlapping genes of the bacteriophage ΦX174 with codons rarely used in its E. coli host. The fitness of recombinant viruses decreases as additional deoptimizing mutations are made to the genome, although not always linearly, and not consistently across genes. Combining deoptimizing mutations may reduce viral fitness more or less than expected from the effect size of the constituent mutations and we point out difficulties in untangling correlated compositional features. We test our model by optimizing the same genes and find that the relationship between codon usage and fitness does not hold for optimization, suggesting that wild-type ΦX174 is at a fitness optimum. This work highlights the need to better understand how selection acts on patterns of synonymous codon usage across the genome and provides a convenient system to investigate the genetic determinants of virulence.</td><td>2020-02-11</td><td>James T. Van Leuven; Martina M. Ederer; Katelyn Burleigh; LuAnn Scott; Randall A. Hughes; Vlad Codrea; Andrew D. Ellington; Holly Wichman; Craig Miller</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.10.942847</td></tr><tr><td>6r6zwfoy</td><td>2cfe7a4cc1fceed0ad7ffbd6244a61e0b2c3ac8b</td><td>medrxiv</td><td>Facemask shortage and the coronavirus disease (COVID-19) outbreak: Reflection on public health measures</td><td>10.1101/2020.02.11.20020735</td><td>null</td><td>null</td><td>medrvix</td><td>To the best of our knowledge, this is the first study to investigate the facemask shortage during the novel coronavirus pneumonia (COVID-19) outbreak in China. We have summarized in detail the management strategies implemented by the Chinese governments during the outbreaks. By considering three scenarios for the outbreak development, we simulated the facemasks availability from late-December 2019 to late-April 2020 and estimated the duration of sufficient facemask supplies. Our findings showed that if the COVID-19 outbreak occurred only in Wuhan city or Hubei province, facemask shortage would not appear with the existing public health measures. However, if the outbreak occurred in the whole of China, a shortage of facemask could be substantial assuming no alternative public health measures. Supplies of facemasks in the whole of China would have been sufficient for both healthcare workers and the general population if the COVID-19 outbreak only occurred in Wuhan city or Hubei province. However, if the outbreak occurred in the whole of China, facemask supplies in China could last for 5 days if under the existing public health measures and a shortage of 853 million facemasks is expected by 30 Apr 2020. Assuming a gradually decreased import volume, we estimated that dramatic increase in productivity (42.7 times of the usual level) is needed to mitigate the facemask crisis by the end of April. In light of the COVID-19 outbreak in China, a shortage of facemasks and other medical resources can considerably compromise the efficacy of public health measures. Effective public health measures should also consider the adequacy and affordability of medical resources. Global collaboration should be strengthened to prevent the development of a global pandemic from a regional epidemic via easing the medical resources crisis in the affected countries.</td><td>2020-02-12</td><td>Huailiang Wu; Jian Huang; Casper JP Zhang; Zonglin He; Wai-kit Ming</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20020735</td></tr><tr><td>5xki1ulf</td><td>8b88901d0bb033af3a18ebcdf7f2aa7110b21504</td><td>medrxiv</td><td>Evaluating the accuracy of different respiratory specimens in the laboratory diagnosis and monitoring the viral shedding of 2019-nCoV infections</td><td>10.1101/2020.02.11.20021493</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The outbreak of novel coronavirus pneumonia (NCP) caused by 2019-nCoV spread rapidly, and elucidation the diagnostic accuracy of different respiratory specimens is crucial for the control and treatment of this diseases. Methods: Respiratory samples including nasal swabs, throat swabs, sputum and bronchoalveolar lavage fluid (BALF) were collected from Guangdong CDC confirmed NCP patients, and viral RNAs were detected using a CFDA approved detection kit. Results were analyzed in combination with sample collection date and clinical information. Finding: Except for BALF, the sputum possessed the highest positive rate (74.4%~88.9%), followed by nasal swabs (53.6%~73.3%) for both severe and mild cases during the first 14 days after illness onset (d.a.o). For samples collected ≥ 15 d.a.o, sputum and nasal swabs still possessed a high positive rate ranging from 42.9%~61.1%. The positive rate of throat swabs collected ≥ 8 d.a.o was low, especially in samples from mild cases. Viral RNAs could be detected in all the lower respiratory tract of severe cases, but not the mild cases. CT scan of cases 02, 07 and 13 showed typical viral pneumonia with ground glass opacity, while no viral RNAs were detected in first three or all the upper respiratory samples. Interpretation: Sputum is most accurate for laboratory diagnosis of NCP, followed by nasal swabs. Detection of viral RNAs in BLAF is necessary for diagnosis and monitoring of viruses in severe cases. CT scan could serve as an important make up for the diagnosis of NCP. Funding National Science and Technology Major Project, Sanming Project of Medicine and China Postdoctoral Science Foundation.</td><td>2020-02-12</td><td>Yang Yang; Minghui Yang; Chenguang Shen; Fuxiang Wang; Jing Yuan; Jinxiu Li; Mingxia Zhang; Zhaoqin Wang; Li Xing; Jinli Wei; Ling Peng; Gary Wong; Haixia Zheng; Mingfeng Liao; Kai Feng; Jianming Li; Qianting Yang; Juanjuan Zhao; Zheng Zhang; Lei Liu; Yingxia Liu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20021493</td></tr><tr><td>w5t1ni4l</td><td>null</td><td>medrxiv</td><td>Ophthalmologic evidence against the interpersonal transmission of 2019 novel coronavirus through conjunctiva</td><td>10.1101/2020.02.11.20021956</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The emerging 2019 novel coronavirus (2019−nCoV) has pushed several countries into state of emergency all over the world. The possible transmission of 2019−nCoV by conjunctiva is controversial and has substantial public health implications. Methods: A retrospective cohort study was initiated to investigate the possible transmission of 2019−nCoV through aerosol contact with conjunctiva. We enrolled 67 cases of confirmed or suspected cases of novel coronavirus pneumonia (NCP) during 17−28 Jan 2020. Nasopharyngeal and conjunctival swabs were collected for real time RT-PCR analysis to detect 2019−nCoV. Results: 63 patients were identified as laboratory−confirmed NCP and the remaining four were suspected NCP. Conjunctival swab samples from one NCP patient yielded positive PCR results and two NCP patients yielded probable positive PCR results. None of the three patients had ocular symptoms. The only one NCP patient with conjunctivitis as the first symptom had negative conjunctival sac 2019−nCoV test. Conjunctival swab samples from the four suspected cases of NCIP were negative. Conclusion: 2019−nCoV can be detected in the conjunctival sac of patients with NCP. Through clinical analysis, viral transmission via the conjunctival route was not supported by the data. Good clinical protection can effectively cut off the transmission path.</td><td>2020-02-12</td><td>Yunyun Zhou; Yuyang Zeng; Yongqing Tong; ChangZheng Chen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20021956</td></tr><tr><td>w4zad1zt</td><td>null</td><td>medrxiv</td><td>Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</td><td>10.1101/2020.02.11.20022053</td><td>null</td><td>null</td><td>medrvix</td><td>An increasing number of cases of novel coronavirus pneumonia (NCP) infected with 2019-nCoV have been identified in Wuhan and other cities in China, since December 2019. We analyzed data on the 17 confirmed cases in Dazhou to provide the epidemiologic characteristics of NCP outside Wuhan. Among them, 12 patients were still quarantined in the hospital, 5 patients were discharged NCP patients according to the national standards. Compared with non-discharged NCP patients, the discharged NCP patients had younger ages. Moreover, discharged NCP patients had higher heart rate, lymphocytes levels and monocytes levels than non-discharged NCP patients on admission to the hospital. Notably, all of 17 patients had abnormal increased C-reactive protein levels, and 16 patients had abnormal computed tomography images. This study provided some information that younger age, higher lymphocytes levels and monocytes levels at the diagnoses of 2019-nCoV may contributed to faster recovery and better therapeutic outcome.</td><td>2020-02-12</td><td>Jie Li; Shilin Li; Yurui Cai; Qin Liu; Xue Li; Zhaoping Zeng; Yanpeng Chu; Fangcheng Zhu; Fanxin Zeng</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20022053</td></tr><tr><td>1f5hn3ig</td><td>null</td><td>medrxiv</td><td>Primary Care Practitioners' Response to 2019 Novel Coronavirus Outbreak in China</td><td>10.1101/2020.02.11.20022095</td><td>null</td><td>null</td><td>medrvix</td><td>None</td><td>2020-02-12</td><td>Zhijie Xu; Yi Qian; Lizheng Fang; Mi Yao</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20022095</td></tr><tr><td>qbemfymp</td><td>004f0f8bb66cf446678dc13cf2701feec4f36d76</td><td>medrxiv</td><td>Healthcare-resource-adjusted vulnerabilities towards the 2019-nCoV epidemic across China</td><td>10.1101/2020.02.11.20022111</td><td>null</td><td>null</td><td>medrvix</td><td>We integrate the human movement and healthcare resource data to identify cities with high vulnerability towards the 2019-nCoV epidemic with respect to available health resources. The results inform public health responses in multiple ways.</td><td>2020-02-12</td><td>Hanchu Zhou; Jianan Yang; Kaichen Tang; Qingpeng Zhang; zhidong cao; Dirk Pfeiffer; Daniel Dajun Zeng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20022111</td></tr><tr><td>ssdx8xwi</td><td>8bcbb998e7067bab377471601a3e72e50c2fe24f</td><td>medrxiv</td><td>Data-Based Analysis, Modelling and Forecasting of the COVID-19 outbreak</td><td>10.1101/2020.02.11.20022186</td><td>null</td><td>null</td><td>medrvix</td><td>Since the first suspected case of coronavirus disease-2019 (COVID-19) on December 1st, 2019, in Wuhan, Hubei Province, China, a total of 40,235 confirmed cases and 909 deaths have been reported in China up to February 10, 2020, evoking fear locally and internationally. Here, based on the publicly available epidemiological data for Hubei, China from January 11 to February 10, 2020, we provide estimates of the main epidemiological parameters. In particular, we provide an estimation of the case fatality and case recovery ratios, along with their 90% confidence intervals as the outbreak evolves. On the basis of a Susceptible-Infected-Recovered-Dead (SIDR) model, we provide estimations of the basic reproduction number (R0), and the per day infection mortality and recovery rates. By calibrating the parameters of the SIRD model to the reported data, we also attempt to forecast the evolution of the of the outbreak at the epicenter three weeks ahead, i.e. until February 29. As the number of infected individuals, especially of those with asymptomatic or mild courses, is suspected to be much higher than the official numbers, which can be considered only as a subset of the actual numbers of infected and recovered cases in the total population, we have repeated the calculations under a second scenario that considers twenty times the number of confirmed infected cases and forty times the number of recovered, leaving the number of deaths unchanged. Based on the reported data, the expected value of R0 as computed considering the period from the 11th of January until the 18th of January, using the official counts of confirmed cases was found to be ~4.6, while the one computed under the second scenario was found to be ~3.2. Thus, based on the SIRD simulations, the estimated average value of R0 was found to be ~2.6 based on confirmed cases and ~2 based on the second scenario. Our forecasting flashes a note of caution for the presently unfolding outbreak in China. Based on the official counts for confirmed cases, the simulations suggest that the cumulative number of infected could reach 180,000 (with lower bound of 45,000) by February 29. Regarding the number of deaths, simulations forecast that on the basis of the up to the 10th of February reported data, the death toll might exceed 2,700 (as a lower bound) by February 29. Our analysis further reveals a significant decline of the case fatality ratio from January 26 to which various factors may have contributed, such as the severe control measures taken in Hubei, China (e.g. quarantine and hospitalization of infected individuals), but mainly because of the fact that the actual cumulative numbers of infected and recovered cases in the population most likely are much higher than the reported ones. Thus, in a scenario where we have taken twenty times the confirmed number of infected and forty times the confirmed number of recovered cases, the case fatality ratio is around 0.15% in the total population. Importantly, based on this scenario, simulations suggest a slow down of the outbreak in Hubei at the end of February.</td><td>2020-02-13</td><td>Cleo Anastassopoulou; Lucia Russo; Athanasios Tsakris; Constantinos Siettos</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20022186</td></tr><tr><td>dbwfn27p</td><td>fbc41a8e025cd6eb2d7c156aad3b6923af2349c4</td><td>medrxiv</td><td>Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCoV, in the nasal tissue</td><td>10.1101/2020.02.11.20022228</td><td>null</td><td>null</td><td>medrvix</td><td>A novel coronavirus (2019-nCoV) was first identified in Wuhan, Hubei Province, and then spreads to the other Provinces of China. WHO decides to determine a Public Health Emergency of International Concern (PHEIC) of 2019-nCoV. 2019-nCov was reported to share the same receptor, Angiotensin-converting enzyme 2 (ACE2), with SARS-Cov. Here based on the public single-cell RNA-Seq datasets, we analyzed the ACE2 RNA expression profile in the tissues at different locations of the respiratory tract. The result indicates that the ACE2 expression appears in nasal epithelial cells. We found that the size of this population of ACE2-expressing nasal epithelial cells is comparable with the size of the population of ACE2-expression type II alveolar cells (AT2) in the Asian sample reported by Yu Zhao et al. We further detected 2019-nCoV by polymerase chain reaction (PCR) from the nasal-swab and throat-swab of seven suspected cases. We found that 2019-nCoV tends to have a higher concentration in the nasal-swab comparing to the throat-swab, which could attribute to the ACE2-expressing nasal epithelial cells. We hope this study could be informative for virus-prevention strategy development, especially the treatment of nasal mucus.</td><td>2020-02-13</td><td>CHAO WU; Shufa Zheng; Yu Chen; Min Zheng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20022228</td></tr><tr><td>tzdgc3dw</td><td>a726e1844e86c7c17b8b4461dd4c29ad3094c792</td><td>medrxiv</td><td>Lockdown may partially halt the spread of 2019 novel coronavirus in Hubei province, China</td><td>10.1101/2020.02.11.20022236</td><td>null</td><td>null</td><td>medrvix</td><td>We present a timely evaluation of the impact of lockdown on the 2019-nCov epidemic in Hubei province, China. The implementation appears to be effective in reducing about 60% of new infections and deaths, and its effect also appears to be sustainable even after its removal. Delaying its implementation reduces its effectiveness. However, the direct economic cost of such a lockdown remains to be seen and whether the model is replicable in other Chinese regions remains a matter of further investigation.</td><td>2020-02-13</td><td>Mingwang Shen; Zhihang Peng; Yuming Guo; Yanni Xiao; Lei Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.20022236</td></tr><tr><td>prqeufwg</td><td>fc15f433048e4d9464964930c36a1e7057ba1088</td><td>biorxiv</td><td>Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation</td><td>10.1101/2020.02.11.944462</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 Å-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis.</td><td>2020-02-15</td><td>Daniel Wrapp; Nianshuang Wang; Kizzmekia S. Corbett; Jory A. Goldsmith; Ching-Lin Hsieh; Olubukola Abiona; Barney S. Graham; Jason S. McLellan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.944462</td></tr><tr><td>cv36vc8i</td><td>4d1090e239baa1ea54af521f4b8bd8b5ab51fd35</td><td>biorxiv</td><td>A Generalized Discrete Dynamic Model for Human Epidemics</td><td>10.1101/2020.02.11.944728</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA discrete dynamic model for human epidemics was developed in present study. The model included major parameters as transmission strength and its decline parameters, mean incubation period, hospitalization time, non-hospitalization daily mortality, non-hospitalization daily recovery rate, and hospitalization proportion, etc. Sensitivity analysis of the model indicated the total cumulative cases significantly increased with initial transmission strength, hospitalization time. The total cumulative cases significantly decreased with transmission strength’s decline and hospitalization proportion, and linearly decreased with non-hospitalization daily mortality and non-hospitalization daily recovery rate. In a certain range, the total cumulative cases significantly increased with mean incubation period. Sensitivity analysis demonstrated that dynamic change of transmission strength is one of the most important and controllable factors. In addition, reducing the delay for hospitalization is much effective in weakening disease epidemic. Non-hospitalization recovery rate is of importance for enhancing immunity to recover from the disease.</td><td>2020-02-12</td><td>Wenjun Zhang; Zeliang Chen; Yi Lu; Zhongmin Guo; Yanhong Qi; Guoling Wang; Jiahai Lu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.11.944728</td></tr><tr><td>7t41wvw3</td><td>d3c2e2839498c613ee95739dce7052109750362c</td><td>medrxiv</td><td>Long-Term Persistence of IgG Antibodies in SARS-CoV Infected Healthcare Workers</td><td>10.1101/2020.02.12.20021386</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND: The ongoing worldwide outbreak of the 2019-nCoV is markedly similar to the severe acute respiratory syndrome (SARS) outbreak 17 years ago. During the 2002-2003 SARS outbreak, healthcare workers formed a special population of patients. Although virus-specific IgG play important roles in virus neutralization and prevention against future infection, limited information is available regarding the long term persistence of IgG after infection with SARS-like coronavirus. METHODS: A long-term prospective cohort study followed 34 SARS-CoV-infected healthcare workers from a hospital with clustered infected cases during the 2002-2003 SARS outbreak in Guangzhou, China, with a 13-year follow-up. Serum samples were collected annually from 2003-2015. Twenty SARS-CoV-infected and 40 non-infected healthcare workers were enrolled in 2015, and their serum samples were collected. All sera were tested for IgG antibodies with ELISA using whole virus and a recombinant nucleocapsid protein of SARS-CoV, as a diagnostic antigen. RESULTS: Anti SARS-CoV IgG was found to persist for up to 12 years. IgG titers typically peaked in 2004, declining rapidly from 2004-2006, and then continued to decline at a slower rate. IgG titers in SARS-CoV-infected healthcare workers remained at a significantly high level until 2015. Patients treated with corticosteroids at the time of infection were found to have lower IgG titers than those without. CONCLUSIONS: IgG antibodies against SARS-CoV can persist for at least 12 years. The presence of SARS-CoV IgG might provide protection against SARS-CoV and other betacoronavirus. This study provides valuable information regarding humoral immune responses against SARS-CoV and the 2019-nCoV.</td><td>2020-02-14</td><td>Xiaoqin Guo; Zhongmin Guo; Chaohui Duan; Zeliang chen; Guoling Wang; Yi Lu; Mengfeng Li; Jiahai Lu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20021386</td></tr><tr><td>bhm2un6v</td><td>d34a47b592e9c6e84b5b4115474349acbc0fb925</td><td>medrxiv</td><td>Statistics based predictions of coronavirus 2019-nCoV spreading in mainland China</td><td>10.1101/2020.02.12.20021931</td><td>null</td><td>null</td><td>medrvix</td><td>Background. The epidemic outbreak cased by coronavirus 2019-nCoV is of great interest to researches because of the high rate of spread of the infection and the significant number of fatalities. A detailed scientific analysis of the phenomenon is yet to come, but the public is already interested in the questions of the duration of the epidemic, the expected number of patients and deaths. For long time predictions, the complicated mathematical models are necessary which need many efforts for unknown parameters identification and calculations. In this article, some preliminary estimates will be presented. Objective. Since the reliable long time data are available only for mainland China, we will try to predict the epidemic characteristics only in this area. We will estimate some of the epidemic characteristics and present the most reliable dependences for victim numbers, infected and removed persons versus time. Methods. In this study we use the known SIR model for the dynamics of an epidemic, the known exact solution of the linear equations and statistical approach developed before for investigation of the children disease, which occurred in Chernivtsi (Ukraine) in 1988-1989. Results. The optimal values of the SIR model parameters were identified with the use of statistical approach. The numbers of infected, susceptible and removed persons versus time were predicted. Conclusions. Simple mathematical model was used to predict the characteristics of the epidemic caused by coronavirus 2019-nCoV in mainland China. The further research should focus on updating the predictions with the use of fresh data and using more complicated mathematical models.</td><td>2020-02-13</td><td>Igor Nesteruk</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20021931</td></tr><tr><td>lvjveaob</td><td>5fb2cce255dd56b944ae52cec664bd7f4f222c66</td><td>medrxiv</td><td>Estimating the daily trend in the size of the COVID-19 infected population in Wuhan</td><td>10.1101/2020.02.12.20022277</td><td>null</td><td>null</td><td>medrvix</td><td>There has been an outbreak of coronavirus disease (COVID-19) in Wuhan city, Hubei province, China since December 2019. Cases have been exported to other parts of China and more than 20 countries. We provide estimates of the daily trend in the size of the epidemic in Wuhan based on detailed information of 10,940 confirmed cases outside Hubei province.</td><td>2020-02-13</td><td>Qiushi Lin; Taojun Hu; Xiao-Hua Zhou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022277</td></tr><tr><td>j60j7e9b</td><td>5585d267f9b505b068f9f72bff5f12c248472f0a</td><td>medrxiv</td><td>Epidemic size of novel coronavirus-infected pneumonia in the Epicenter Wuhan: using data of five-countries' evacuation action</td><td>10.1101/2020.02.12.20022285</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since late December 2019, novel coronavirus-infected pneumonia (NCP) emerged in Wuhan, Hubei province, China. Meanwhile, NCP rapidly spread from China to other countries, and several countries' government rush to evacuate their citizens from Wuhan. We analyzed the infection rate of the evacuees and extrapolated the results in Wuhan's NCP incidence estimation. Methods: We collected the total number and confirmed cases of 2019-nCov infection in the evacuation of Korea, Japan, Germany, Singapore, and France and estimated the infection rate of the 2019 novel coronavirus (2019-nCov) among people who were evacuated from Wuhan with a meta-analysis. NCP incidence of Wuhan was indirectly estimated based on data of evacuation. Results: From Jan 29 to Feb 2, 2020, 1916 people have been evacuated from Wuhan, among them 17 have been confirmed 2019-nCov infected. The infection rate is estimated to be 1.1% (95% CI 0.4%-3.1%) using one group meta-analysis method with random effect model. We then estimated that almost 110,000 (95% CI: 40,000-310,000) people were infected with 2019-nCov in Wuhan around Feb 2, 2020, assuming the infection risk of evacuees is close to Chinese citizens in Wuhan. Conclusions: At the beginning of the outbreak, incidence of NCP may be vastly underestimated. Our result emphasizes that 2019-nCov has proposed a huge public health threats in Wuhan. We need to respond more rapidly, take large-scale public health interventions and draconian measures to limiting population mobility and control the epidemic.</td><td>2020-02-13</td><td>Hongxin Zhao; Sailimai Man; Bo Wang; Yi Ning</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022285</td></tr><tr><td>78qyuk5x</td><td>null</td><td>medrxiv</td><td>Clinical diagnosis of 8274 samples with 2019-novel coronavirus in Wuhan</td><td>10.1101/2020.02.12.20022327</td><td>null</td><td>null</td><td>medrvix</td><td>Background: 2019-Novel coronavirus (2019-nCoV) outbreaks create challenges for hospital laboratories because thousands of samples must be evaluated each day. Sample types, interpretation methods, and corresponding laboratory standards must be established. The possibility of other infections should be assessed to provide a basis for clinical classification, isolation, and treatment. Accordingly, in the present study, we evaluated the testing methods for 2019-nCoV and co-infections. Methods: We used a fluorescence-based quantitative PCR kit urgently distributed by the Chinese CDC to detect 8274 close contacts in the Wuhan region against two loci on the 2019-nCoV genome. We also analyzed 613 patients with fever who underwent multiple tests for 13 respiratory pathogens; 316 subjects were also tested for 2019-nCoV. Findings: Among the 8274 subjects, 2745 (33.2%) had 2019-nCoV infection; 5277 (63.8%) subjects showed negative results in the 2019-nCoV nucleic acid test (non-019-nCoV); and 252 cases (3.0%) because only one target was positive, the diagnosis was not definitive. Sixteen patients who originally had only one positive target were re-examined a few days later; 14 patients (87.5%) were finally defined as 2019-nCoV-positive, and 2 (12.5%) were finally defined as negative. The positive rates of nCoV-NP and nCovORF1ab were 34.7% and 34.7%, respectively. nCoV-NP-positive only and nCovORF1ab-positive cases accounted for 1.5% and 1.5%, respectively. In the 316 patients with multiple respiratory pathogens, 104 were positive for 2019-nCov and 6/104 had co-infection with coronavirus (3/104), influenza A virus (2/104), rhinovirus (2/104), and influenza A H3N2 (1/104); the remaining 212 patients had influenza A virus (11/202), influenza A H3N2 (11/202), rhinovirus (10/202), respiratory syncytial virus (7/202), influenza B virus (6/202), metapneumovirus (4/202), and coronavirus (2/202). Interpretation: Clinical testing methods for 2019-nCoV require improvement. Importantly, 5.8% of 2019-nCoV infected and 18.4% of non-2019-nCoV-infected patients had other pathogen infections. It is important to treat combined infections and perform rapid screening to avoid cross-contamination of patients. A test that quickly and simultaneously screens as many pathogens as possible is needed.</td><td>2020-02-13</td><td>Ming Wang; Qing Wu; Wanzhou Xu; Bin Qiao; Jingwei Wang; Hongyun Zheng; Shupeng Jiang; Junchi Mei; Zegang Wu; Yayun Deng; Fangyuan Zhou; Wei Wu; Yan Zhang; Zhihua Lv; Jingtao Huang; Xiaoqian Guo; Lina Feng; Zunen Xia; Di Li; Zhiliang Xu; Tiangang Liu; Pingan Zhang; Yongqing Tong; Yan Li</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022327</td></tr><tr><td>vv1wjen0</td><td>409c431edf54d022463f40f2d31a0dfab42f5324</td><td>medrxiv</td><td>Can Search Query Forecast successfully in China's 2019-nCov pneumonia?</td><td>10.1101/2020.02.12.20022400</td><td>null</td><td>null</td><td>medrvix</td><td>Recently the novel coronavirus (2019-nCov) pneumonia outbreak in China then the world, and the Number of infections and death continues to increases. Search Query performs well in forecasting the epidemics. It is still a question whether search engine data can forecast the drift and the inflexion in 2019-nCov pneumonia. Based on the Baidu Search Index, we propose three prediction models: composite Index, composite Index with filtering and suspected NCP(Novel Coronavirus Pneumonia). The result demonstrates that the predictive model of composite index with filtering performs the best while the model of suspected NCP has the highest forecast error. We further predict the out-of-the-set NCP confirmed cases and monitor that the next peak of new diagnoses will occur on February 16th and 17th.</td><td>2020-02-18</td><td>Li Xiaoxuan; Wu Qi; Lv Benfu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022400</td></tr><tr><td>s5w8rf6t</td><td>c672f5b0d7c0a40f7cc1bdf079066d72eb8f64ec</td><td>medrxiv</td><td>ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</td><td>10.1101/2020.02.12.20022418</td><td>null</td><td>null</td><td>medrvix</td><td>In December 2019 and January 2020, novel coronavirus (2019-nCoV) - infected pneumonia (NCIP) occurred in Wuhan, and has already posed a serious threat to public health. ACE2 (Angiotensin Converting Enzyme 2) has been shown to be one of the major receptors that mediate the entry of 2019-nCoV into human cells, which also happens in severe acute respiratory syndrome coronavirus (SARS). Several researches have indicated that some patients have abnormal renal function or even kidney damage in addition to injury in respiratory system, and the related mechanism is unknown. This arouses our interest in whether coronavirus infection will affect the urinary and male reproductive systems. Here in this study, we used the online datasets to analyze ACE2 expression in different human organs. The results indicate that ACE2 highly expresses in renal tubular cells, Leydig cells and cells in seminiferous ducts in testis. Therefore, virus might directly bind to such ACE2 positive cells and damage the kidney and testicular tissue of patients. Our results indicate that renal function evaluation and special care should be performed in 2019-nCoV patients during clinical work, because of the kidney damage caused by virus and antiviral drugs with certain renal toxicity. In addition, due to the potential pathogenicity of the virus to testicular tissues, clinicians should pay attention to the risk of testicular lesions in patients during hospitalization and later clinical follow-up, especially the assessment and appropriate intervention in young patients' fertility.</td><td>2020-02-13</td><td>Caibin Fan; Kai Li; Yanhong Ding; Wei Lu Lu; Jianqing Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022418</td></tr><tr><td>y7d3ux28</td><td>11f96d05db6854ef95312aa3a4736724ce1f02d6</td><td>medrxiv</td><td>Interventions targeting air travellers early in the pandemic may delay local outbreaks of SARS-CoV-2</td><td>10.1101/2020.02.12.20022426</td><td>null</td><td>null</td><td>medrvix</td><td>Background: We evaluated if interventions aimed at air travellers can delay establishment of a SARS-CoV-2 outbreak in a previously unaffected country. Methods: For countries with no sustained SARS-CoV-2 transmission and with no shared border with affected regions we simulated arriving infected air travellers. We assessed the effectiveness of syndromic screening at departure and/or arrival & traveller sensitisation to the COVID-2019-like symptoms with the aim to trigger rapid self-isolation and reporting on symptom onset to enable contact tracing. We assumed that syndromic screening would reduce the number of infected arrivals and that traveller sensitisation reduce the average number of secondary cases. We report the minimal expected delay achievable in 50% (75% & 97.5%) of simulations. In the simulations we account for uncertainty in the number of secondary cases in the absence of air traveller targeted interventions and the arrival times of infected cases and also present sensitivity analyses on arrival rates of infected travellers and the effectiveness of traveller sensitisation. Results: Under baseline assumptions exit and entry screening combined with traveller sensitisation can delay a local SARS-CoV-2 outbreak by at least 83 (75% of simulations: at least 36, 97.5% 8) days while there is no more than 1 infected traveller per week. The benefit of entry screening is small if exit screening is effective: the combination of only exit screening and traveller sensitisation can delay an outbreak by at least 76 (75%: 33, 97.5%: 7) days. With increasing rates of infected travellers, less effective sensitisation or without screening these delays shrink rapidly to a week or less. Conclusion: Syndromic screening and traveller sensitisation in combination could delay outbreaks in yet unaffected countries and support local containment efforts, but only if infected traveller numbers are very low.</td><td>2020-02-13</td><td>Samuel J Clifford; Carl A B Pearson; Petra Klepac; Kevin Van Zandvoort; Billy J Quilty; Rosalind M Eggo; Stefan Flasche</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022426</td></tr><tr><td>kfsuiz5t</td><td>null</td><td>medrxiv</td><td>Early epidemiological assessment of the transmission potential and virulence of coronavirus disease 2019 (COVID-19) in Wuhan City: China, January-February, 2020</td><td>10.1101/2020.02.12.20022434</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since the first cluster of cases was identified in Wuhan City, China, in December, 2019, coronavirus disease 2019 (COVID-19) has rapidly spread across China, causing multiple introductions in 109 countries/territories/areas as of March 10th. Despite the scarcity of publicly available data, scientists around the world have made strides in estimating the magnitude of the epidemic, the basic reproduction number, and transmission patterns. Recently more evidence suggests that a substantial fraction of the infected individuals with the novel coronavirus show little if any symptoms, which suggest the need to reassess the transmission potential of this emerging disease. In this study, we derive estimates of the transmissibility and virulence of COVID-19 in Wuhan City, China, by reconstructing the underlying transmission dynamics using multiple data sources. Methods: We employ statistical methods and publicly available epidemiological datasets to jointly derive estimates of transmissibility and severity associated with the novel coronavirus. For this purpose, the daily series of laboratory-confirmed COVID-19 cases and deaths in Wuhan City and epidemiological data of Japanese evacuees from Wuhan City on board government-chartered flights were integrated into our analysis. Results: Our posterior estimates of basic reproduction number (R) in Wuhan City, China in 2019-2020 reached values as high as 5.20 (95%CrI: 5.04-5.47) and the enhanced public health intervention after January 23rd in 2020 was associated with a declined R at 0.58 (95%CrI: 0.51-0.64), with the total number of infections (i.e. cumulative infections) estimated at 1905526 (95%CrI: 1350283-2655936) in Wuhan City, raising the proportion of infected individuals to 19.1% (95%CrI: 13.5-26.6%). We also found that most recent crude infection fatality ratio (IFR) and time-delay adjusted IFR is estimated to be 0.04% (95% CrI: 0.03-0.06%) and 0.12% (95%CrI: 0.08-0.17%), which is several orders of magnitude smaller than the crude CFR estimated at 4.19% Conclusions: We have estimated key epidemiological parameters of the transmissibility and virulence of COVID-19 in Wuhan, China during January-February, 2020 using an ecological modelling approach. The power of our approach lies in the ability to infer epidemiological parameters with quantified uncertainty from partial observations collected by surveillance systems.</td><td>2020-02-13</td><td>Kenji Mizumoto; Katsushi Kagaya; Gerardo Chowell</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022434</td></tr><tr><td>zvngy7zz</td><td>90b5ecf991032f3918ad43b252e17d1171b4ea63</td><td>medrxiv</td><td>The role of absolute humidity on transmission rates of the COVID-19 outbreak</td><td>10.1101/2020.02.12.20022467</td><td>null</td><td>null</td><td>medrvix</td><td>A novel coronavirus (COVID-19) was identified in Wuhan, Hubei Province, China, in December 2019 and has caused over 40,000 cases worldwide to date. Previous studies have supported an epidemiological hypothesis that cold and dry (low absolute humidity) environments facilitate the survival and spread of droplet-mediated viral diseases, and warm and humid (high absolute humidity) environments see attenuated viral transmission (i.e., influenza). However, the role of absolute humidity in transmission of COVID-19 has not yet been established. Here, we examine province-level variability of the basic reproductive numbers of COVID-19 across China and find that changes in weather alone (i.e., increase of temperature and humidity as spring and summer months arrive in the North Hemisphere) will not necessarily lead to declines in COVID-19 case counts without the implementation of extensive public health interventions.</td><td>2020-02-17</td><td>Wei Luo; Maimuna S Majumder; Dianbo Liu; Canelle Poirier; Kenneth D Mandl; Marc Lipsitch; Mauricio Santillana</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022467</td></tr><tr><td>x4qdiln9</td><td>05d99c07db59b6948e39bfa62c2cbbf62944059a</td><td>medrxiv</td><td>A spatial model of CoVID-19 transmission in England and Wales: early spread and peak timing</td><td>10.1101/2020.02.12.20022566</td><td>null</td><td>null</td><td>medrvix</td><td>Background: An outbreak of a novel coronavirus, named CoVID-19, was first reported in China on 31 December 2019. As of 9 February 2020, cases have been reported in 25 countries, including probable cases of human-to-human transmission in England. Methods: We adapted an existing national-scale metapopulation model to capture the spread of CoVID-19 in England and Wales. We used 2011 census data to capture population sizes and population movement, together with parameter estimates from the current outbreak in China. Results: We predict that a CoVID-19 outbreak will peak 126 to 147 days (~4 months) after the start of person-to-person transmission in England and Wales in the absence of controls, assuming biological parameters remain unchanged. Therefore, if person-to-person transmission persists from February, we predict the epidemic peak would occur in June. The starting location has minimal impact on peak timing, and model stochasticity varies peak timing by 10 days. Incorporating realistic parameter uncertainty leads to estimates of peak time ranging from 78 days to 241 days after person-to-person transmission has been established. Seasonal changes in transmission rate substantially impact the timing and size of the epidemic peak, as well as the total attack rate. Discussion: We provide initial estimates of the potential course of CoVID-19 in England and Wales in the absence of control measures. These results can be refined with improved estimates of epidemiological parameters, and permit investigation of control measures and cost effectiveness analyses. Seasonal changes in transmission rate could shift the timing of the peak into winter months, which will have important implications for healthcare capacity planning.</td><td>2020-02-14</td><td>Leon Danon; Ellen Brooks-Pollock; Mick Bailey; Matt J Keeling</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.20022566</td></tr><tr><td>anls8gri</td><td>3579a2b0053bef55f93716e444173727a6c94c6d</td><td>biorxiv</td><td>Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus</td><td>10.1101/2020.02.12.945576</td><td>null</td><td>null</td><td>biorxiv</td><td>Pathogenic human coronavirus infections, such as severe acute respiratory syndrome CoV (SARS-CoV) and Middle East respiratory syndrome CoV (MERS-CoV), cause high morbidity and mortality 1,2. Recently, a severe pneumonia-associated respiratory syndrome caused by a new coronavirus was reported at December 2019 (2019-nCoV) in the city Wuhan, Hubei province, China3–5, which was also named as pneumonia-associated respiratory syndrome (PARS)6. Up to 9th of February 2020, at least 37, 251 cases have been reported with 812 fatal cases according to the report from China CDC. However, the immune mechanism that potential orchestrated acute mortality from patients of 2019-nCoV is still unknown. Here we show that after the 2019-nCoV infection, CD4+T lymphocytes are rapidly activated to become pathogenic T helper (Th) 1 cells and generate GM-CSF etc. The cytokines environment induces inflammatory CD14+CD16+ monocytes with high expression of IL-6 and accelerates the inflammation. These aberrant and excessive immune cells may enter the pulmonary circulation in huge numbers and play an immune damaging role to causing lung functional disability and quick mortality. Our results demonstrate that excessive non-effective host immune responses by pathogenic T cells and inflammatory monocytes may associate with severe lung pathology. Therefore, we suggest that monoclonal antibody that targets the GM-CSF or interleukin 6 receptor may potentially curb immunopathology caused by 2019-nCoV and consequently win more time for virus clearance.</td><td>2020-02-20</td><td>Yonggang Zhou; Binqing Fu; Xiaohu Zheng; Dongsheng Wang; Changcheng Zhao; Yingjie qi; Rui Sun; Zhigang Tian; Xiaoling Xu; Haiming Wei</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.945576</td></tr><tr><td>8p1agcm2</td><td>f6f1ec12291f743de1504003929ea654217f2af6</td><td>biorxiv</td><td>Candidate targets for immune responses to 2019-Novel Coronavirus (nCoV): sequence homology- and bioinformatic-based predictions</td><td>10.1101/2020.02.12.946087</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTEffective countermeasures against the recent emergence and rapid expansion of the 2019-Novel Coronavirus (2019-nCoV) require the development of data and tools to understand and monitor viral spread and immune responses. However, little information about the targets of immune responses to 2019-nCoV is available. We used the Immune Epitope Database and Analysis Resource (IEDB) resource to catalog available data related to other coronaviruses, including SARS-CoV, which has high sequence similarity to 2019-nCoV, and is the best-characterized coronavirus in terms of epitope responses. We identified multiple specific regions in 2019-nCoV that have high homology to SARS virus. Parallel bionformatic predictions identified a priori potential B and T cell epitopes for 2019-nCoV. The independent identification of the same regions using two approaches reflects the high probability that these regions are targets for immune recognition of 2019-nCoV.ONE SENTENCE SUMMARYWe identified potential targets for immune responses to 2019-nCoV and provide essential information for understanding human immune responses to this virus and evaluation of diagnostic and vaccine candidates.</td><td>2020-02-20</td><td>Alba Grifoni; John Sidney; Yun Zhang; Richard H Scheuermann; Bjoern Peters; Alessandro Sette</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.946087</td></tr><tr><td>er6lz09f</td><td>4b56993a4d0f091e70ef5dec2e5a61c3043f752e</td><td>biorxiv</td><td>DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes</td><td>10.1101/2020.02.12.946772</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractMotivationAccurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm.ResultsWe trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours.AvailabilityThe repository of this project will be published.Contactdongsi@uw.eduSupplementary informationSupplementary data will be available at Bioinformatics online.</td><td>2020-02-13</td><td>Jonas Pfab; Dong Si</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.12.946772</td></tr><tr><td>z8ix4kz7</td><td>38bab3f17b186bd8ee289e5d135bb7d500ef500b</td><td>medrxiv</td><td>Probabilistic reconstruction of measles transmission clusters from routinely collected surveillance data</td><td>10.1101/2020.02.13.20020891</td><td>null</td><td>null</td><td>medrvix</td><td>Pockets of susceptibility resulting from spatial or social heterogeneity in vaccine coverage can drive measles outbreaks, as cases imported into such pockets are likely to cause further transmission and lead to large transmission clusters. Characterising the dynamics of transmission is essential for identifying which individuals and regions might be most at risk. As data from detailed contact tracing investigations are not available in many settings, we combined age, location, genotype, and onset date of cases in order to probabilistically reconstruct the importation status and transmission clusters within a newly developed R package called o2geosocial. We compared our inferred cluster size distributions to 737 transmission clusters identified through detailed contact-tracing in the United States between 2001 and 2016. We were able to reconstruct the importation status of the cases and found good agreement between the inferred and reference clusters. The results were improved when the contact-tracing investigations were used to set the importation status before running the model. Spatial heterogeneity in vaccine coverage is difficult to measure directly. Our approach was able to highlight areas with potential for local transmission using a minimal number of variables and could be applied to assess the intensity of ongoing transmission in a region.</td><td>2020-02-15</td><td>Alexis Robert; Adam J Kucharski; Paul A Gastañaduy; Prabasaj Paul; Sebastian Funk</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.20020891</td></tr><tr><td>lmoekp1m</td><td>7ed2c0786eef3d39fa0435b461aa34ab454a4753</td><td>medrxiv</td><td>Understanding the present status and forecasting of COVID―19 in Wuhan</td><td>10.1101/2020.02.13.20022251</td><td>null</td><td>null</td><td>medrvix</td><td>The present status of COVID―19 is analyzed and the end of the disease is forecasted. The peak of the epidemic is different in three regions, Wuhan, Hubei province except Wuhan, and mainland China except Hubei. In two regions except Wuhan, the peak of the epidemic passed ten days ago. If the trend until February 11 does not change, the disease may end by the end of February. In Wuhan, the epidemic reached a peak but the reported number of newly infected patients fluctuates largely. We need to know the reason for the big fluctuation to forecast the end of the disease.</td><td>2020-02-14</td><td>Toshihisa Tomie</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.20022251</td></tr><tr><td>nqxo3f45</td><td>8b71ce92ec03e066eca7792bfd17e2f210d50ce3</td><td>medrxiv</td><td>Optimizing diagnostic strategy for novel coronavirus pneumonia, a multi-center study in Eastern China</td><td>10.1101/2020.02.13.20022673</td><td>null</td><td>null</td><td>medrvix</td><td>COVID-19 caused by a novel coronavirus SARS-CoV-2 emerged in Wuhan, Hubei province since December 2019, and caused a rapid outbreak throughout China and globally. Cities outside Hubei are also facing great challenge and require implementing of effective and feasible strategy in precision diagnosing novel coronavirus pneumonia (NCP). We described a multicenter prospective study on diagnostic strategy of suspected NCP patients from January 22nd to February 9th, 2020 in Eastern China cities. Nasopharyngeal swabs were collected from the patients. The epidemiological characteristics, clinical symptoms, laboratory assessments, and computed tomographic (CT) scans were obtained. Pathogen screen were performed including RT-PCR, multiplex PCR, rapid flu antigen tests and mNGS. We enrolled 53 suspected NCP patients, among whom 20 were laboratory-confirmed. Fourteen (70%) and 3 (15%) patients were positive for the first and second SARS-CoV-2 RT-PCR test, respectively. All NCP patients were positive for mNGS. Chest CT images and the symptoms of early stage NCP patients were similar to other viral pneumonia patients. We identified 11 of 20 co-infections in NCP cases, including regular respiratory virus, fungi and bacteria synchronously. Genomic analysis showed that 8 of 10 cases had no mutation in virus genome, while 2 cases had only one single mutation in N gene. Our study discovered that a combination of chest CT, SARS-CoV-2 RT-PCR and multi-plex PCR is recommended in regions outside Hubei province. Co-infection of other pathogens with SARS-CoV-2 exists and should be acknowledged. Repeated sampling, change of specimen type or metagenomics sequencing could further facilitate during critical clinical cases.</td><td>2020-02-17</td><td>Jing-Wen Ai; Hao-Cheng Zhang; Teng Xu; Jing Wu; Mengqi Zhu; Yi-Qi Yu; Han-Yue Zhang; Zhongliang Shen; Yang Li; Xian Zhou; Guo-Qing Zang; Jie Xu; Wen-Jing Chen; Yong-Jun Li; De-Sheng Xie; Ming-Zhe Zhou; Jing-Ying Sun; Jia-Zhen Chen; Wen-Hong Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.20022673</td></tr><tr><td>h0h0d93t</td><td>adb3a6501ad731eda95d0a1a182a793a0dcd58b0</td><td>medrxiv</td><td>Quantifying bias of COVID-19 prevalence and severity estimates in Wuhan, China that depend on reported cases in international travelers</td><td>10.1101/2020.02.13.20022707</td><td>null</td><td>null</td><td>medrvix</td><td>Risk of COVID-19 infection in Wuhan has been estimated using imported case counts of international travelers, often under the assumption that all cases in travelers are ascertained. Recent work indicates variation among countries in detection capacity for imported cases. Singapore has historically had very strong epidemiological surveillance and contact-tracing capacity and has shown in the COVID-19 epidemic evidence of a high sensitivity of case detection. We therefore used a Bayesian modeling approach to estimate the relative imported case detection capacity for other countries compared to that of Singapore. We estimate that the global ability to detect imported cases is 38% (95% HPDI 22% - 64%) of Singapore′s capacity. Equivalently, an estimate of 2.8 (95% HPDI 1.5 - 4.4) times the current number of imported cases, could have been detected, if all countries had had the same detection capacity as Singapore. Using the second component of the Global Health Security index to stratify likely case-detection capacities, we found that the ability to detect imported cases relative to Singapore among high surveillance locations is 40% (95% HPDI 22% - 67%), among intermediate surveillance locations it is 37% (95% HPDI 18% - 68%), and among low surveillance locations it is 11% (95% HPDI 0% - 42%). Using a simple mathematical model, we further find that treating all travelers as if they were residents (rather than accounting for the brief stay of some of these travelers in Wuhan) can modestly contribute to underestimation of prevalence as well. We conclude that estimates of case counts in Wuhan based on assumptions of perfect detection in travelers may be underestimated by several fold, and severity correspondingly overestimated by several fold. Undetected cases are likely in countries around the world, with greater risk in countries of low detection capacity and high connectivity to the epicenter of the outbreak.</td><td>2020-02-14</td><td>Rene Niehus; Pablo M De Salazar; Aimee Taylor; Marc Lipsitch</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.20022707</td></tr><tr><td>60qmiwjm</td><td>null</td><td>medrxiv</td><td>Analysis of meteorological conditions and prediction of epidemic trend of 2019-nCoV infection in 2020</td><td>10.1101/2020.02.13.20022715</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To investigate the meteorological condition for incidence and spread of 2019-nCoV infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease. Methods: The meteorological factors during the outbreak period of the novel coronavirus pneumonia in Wuhan in 2019 were collected and analyzed, and were confirmed with those of Severe Acute Respiratory Syndrome (SARS) in China in 2003. Data of patients infected with 2019-nCoV and SARS coronavirus were collected from WHO website and other public sources. Results: This study found that the suitable temperature range for 2019-nCoV coronavirus survival is (13-24 degree Celsius), among which 19 degree Celsius lasting about 60 days is conducive to the spread between the vector and humans; the humidity range is 50%-80%, of which about 75% humidity is conducive to the survival of the coronavirus; the suitable precipitation range is below 30 mm/ month. Cold air and continuous low temperature over one week are helpful for the elimination of the virus. The prediction results show that with the approach of spring, the temperature in north China gradually rises, and the coronavirus spreads to middle and high latitudes along the temperature line of 13-18 degree Celsius. The population of new coronavirus infections is concentrated in Beijing, Tianjin, Hebei, Jiangsu, Zhejiang, Shanghai and other urban agglomerations. Starting from May 2020, the Beijing-Tianjin-Hebei urban agglomeration, the Central China Zhengzhou-Wuhan urban agglomeration, the eastern Jiangsu-Zhejiang-Shanghai urban agglomeration, and the southern Pearl River Delta urban agglomeration are all under a high temperature above 24 degree Celsius, which is not conducive to the survival and reproduction of coronaviruses, so the epidemic is expected to end. Conclusions: A wide range of continuous warm and dry weather is conducive to the survival of 2019-nCoV. The coming of spring, in addition to the original Wuhan-Zhengzhou urban agglomeration in central China, means that the prevention and control measures in big cities located in mid-latitude should be strengthened, especially the monitoring of transportation hubs. The Pearl River Delta urban agglomeration is a concentrated area of population in south China, with a faster temperature rise than those in mid-high latitudes, and thus the prevention in this area should be prioritized. From a global perspective, cities with a mean temperature below 24 degree Celsius are all high-risk cities for 2019-nCoV transmission before June.</td><td>2020-02-18</td><td>Jin Bu; Dong-Dong Peng; Hui Xiao; Qian Yue; Yan Han; Yu Lin; Gang Hu; Jing Chen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.20022715</td></tr><tr><td>3p2dl8yf</td><td>1218f278a4f8d83dac14b23c8f698062812ef9d5</td><td>medrxiv</td><td>Potential impact of seasonal forcing on a SARS-CoV-2 pandemic</td><td>10.1101/2020.02.13.20022806</td><td>null</td><td>null</td><td>medrvix</td><td>A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing more than 80,000 confirmed infections and 2,700 fatalities (as of Feb 27, 2020). Imported cases and transmission clusters of various sizes have been reported globally suggesting a pandemic is likely. Here, we explore how seasonal variation in transmissibility could modulate a SARS-CoV-2 pandemic. Data from routine diagnostics show a strong and consistent seasonal variation of the four endemic coronaviruses (229E, HKU1, NL63, OC43) and we parameterize our model for SARS-CoV-2 using these data. The model allows for many subpopulations of different size with variable parameters. Simulations of different scenarios show that plausible parameters result in a small peak in early 2020 in temperate regions of the Northern Hemisphere and a larger peak in winter 2020/2021. Variation in transmission and migration rates can result in substantial variation in prevalence between regions. While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. The likely aggregated effect of seasonal variation, infection control measures and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.</td><td>2020-02-17</td><td>Richard A Neher; Robert Dyrdak; Valentin Druelle; Emma B Hodcroft; Jan Albert</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.20022806</td></tr><tr><td>ibx89gqw</td><td>null</td><td>medrxiv</td><td>Estimating the distribution of the incubation period of 2019 novel coronavirus (COVID-19) infection between travelers to Hubei, China and non-travelers</td><td>10.1101/2020.02.13.20022822</td><td>null</td><td>null</td><td>medrvix</td><td>Objectives: Amid the continuing spread of the novel coronavirus (COVID-19), the incubation period of COVID-19 should be regularly re-assessed as more information is available upon the increase in reported cases. The present work estimated the distribution of incubation periods of patients infected in and outside Hubei province of China. Methods: Clinical data were collected from the individual cases reported by the media as they were not fully available on the official pages of the Chinese health authorities. MLE was used to estimate the distributions of the incubation period. Results: It was found that the incubation period of patients with no travel history to Hubei was longer and more volatile. Conclusion: It is recommended that the duration of quarantine should be extended to at least 3 weeks.</td><td>2020-02-18</td><td>Char Leung</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.20022822</td></tr><tr><td>hlhfx3fm</td><td>47b07518dc0f608e53198bb5f8a95caa1c164fe5</td><td>medrxiv</td><td>A simple laboratory parameter facilitates early identification of COVID-19 patients</td><td>10.1101/2020.02.13.20022830</td><td>null</td><td>null</td><td>medrvix</td><td>The total number of COVID-19 patients since the outbreak of this infection in Wuhan, China has reached 40000 and are still growing. To facilitate triage or identification of the large number of COVID-19 patients from other patients with similar symptoms in designated fever clinics, we set to identify a practical marker that could be conveniently utilized by first-line health-care workers in clinics. To do so, we performed a case-control study by analyzing clinical and laboratory findings between PCR-confirmed SARS-CoV-2 positive patients (n=52) and SARS-CoV-2 negative patients (n=53). The patients in two cohorts all had similar symptoms, mainly fever and respiratory symptoms. The rates of patients with leukocyte counts (normal or decreased number) or lymphopenia (two parameters suggested by current National and WHO COVID-19 guidelines) had no differences between these two cohorts, while the rate of eosinopenia (decreased number of eosinophils) in SARS-CoV-2 positive patients (79%) was much higher than that in SARS-CoV-2 negative patients (36%). When the symptoms were combined with eosinopenia, this combination led to a diagnosis sensitivity and specificity of 79% and 64%, respectively, much higher than 48% and 53% when symptoms were combined with leukocyte counts (normal or decreased number) and/ or lymphopenia. Thus, our analysis reveals that eosinopenia may be a potentially more reliable laboratory predictor for SARS-CoV-2 infection than leukocyte counts and lymphopenia recommended by the current guidelines.</td><td>2020-02-17</td><td>Qilin Li; Xiuli Ding; Geqing Xia; Zhi Geng; Fenghua Chen; Lin Wang; Zheng Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.20022830</td></tr><tr><td>5f42du0b</td><td>3319392ca8bd2b8f2e2d00bccdb80deaa8a77c2e</td><td>biorxiv</td><td>Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China</td><td>10.1101/2020.02.13.945485</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe ongoing outbreak of viral pneumonia in China and beyond is associated with a novel coronavirus, provisionally termed 2019-nCoV. This outbreak has been tentatively associated with a seafood market in Wuhan, China, where the sale of wild animals may be the source of zoonotic infection. Although bats are likely reservoir hosts for 2019-nCoV, the identity of any intermediate host facilitating transfer to humans is unknown. Here, we report the identification of 2019-nCoV related coronaviruses in pangolins (Manis javanica) seized in anti-smuggling operations in southern China. Metagenomic sequencing identified pangolin associated CoVs that belong to two sub-lineages of 2019-nCoV related coronaviruses, including one very closely related to 2019-nCoV in the receptor-binding domain. The discovery of multiple lineages of pangolin coronavirus and their similarity to 2019-nCoV suggests that pangolins should be considered as possible intermediate hosts for this novel human virus and should be removed from wet markets to prevent zoonotic transmission.</td><td>2020-02-18</td><td>Tommy Tsan-Yuk Lam; Marcus Ho-Hin Shum; Hua-Chen Zhu; Yi-Gang Tong; Xue-Bing Ni; Yun-Shi Liao; Wei Wei; William Yiu-Man Cheung; Wen-Juan Li; Lian-Feng Li; Gabriel M Leung; Edward C. Holmes; Yan-Ling Hu; Yi Guan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.13.945485</td></tr><tr><td>rdgyej4p</td><td>null</td><td>medrxiv</td><td>Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</td><td>10.1101/2020.02.14.20021535</td><td>null</td><td>null</td><td>medrvix</td><td>Background:A Novel pneumonia associated with the 2019 coronavirus infected pneumonia (NCIP) suddenly broke out in Wuhan, China in December 2019. 37287 confirmed cases and 813 death case in China (Until 8th/Feb/2019) have been reported in just fortnight. Although this risky pneumonia with high infection rates and high mortality rates need to be resolved immediately, major gaps in our knowledge of clinical characters of it were still not be established. The aim of this study is to summaries and analysis the clinical characteristics of 2019-nCoV pneumonia. Methods: Literature have been systematically performed a search on PubMed, Embase, Web of Science, GreyNet International, and The Cochrane Library from inception up to February 8, 2020. The Newcastle-Ottawa Scale was used to assess quality, and publication bias was analyzed by Egger test. In the single-arm meta-analysis, A fix-effects model was used to obtain a pooled incidence rate. We conducted subgroup analysis according to geographic region and research scale. Results: A total of nine studies including 356 patients were included in this study, the mean age was 52.4 years and 221 (62.1%) were male. The pooled incidences rate of symptoms as follows: pharyngalgia (12.2%, 95% CI: 0.087-0.167), diarrhea (9.2%, 95% CI: 0.062-0.133) and headache (8.9%, 95% CI: 0.063-0.125). Meanwhile, 5.7% (95% CI: 0.027-0.114) of patients were found without any symptoms although they were diagnosed by RT-PCR. In the terms of CT imaging examination, the most of patients showed bilateral mottling or ground-glass opacity, 8.6% (95% CI: 0.048-0.148) of patients with crazy-paving pattern, and 11.5% (95% CI: 0.064-0.197) of patients without obvious CT imaging presentations. The pooled incidence of mortality was 8.9% (95% CI: 0.062-0.126). Conclusions: To our knowledge, this is the first evidence-based medicine research to further elaborate the clinical characteristics of NCIP, which is beneficial to the next step of prevention and treatment.</td><td>2020-02-17</td><td>Kai Qian; Yi Deng; Yonghang Tai; Jun Peng; Hao Peng; Lihong Jiang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.14.20021535</td></tr><tr><td>j1yqn9xg</td><td>f4d76fb8161f4cee6987d7d412798e1276348858</td><td>medrxiv</td><td>Assessing the impact of reduced travel on exportation dynamics of novel coronavirus infection (COVID-19)</td><td>10.1101/2020.02.14.20022897</td><td>null</td><td>null</td><td>medrvix</td><td>The impact of the drastic reduction in travel volume within mainland China in January and February 2020 was quantified with respect to reports of novel coronavirus (COVID-19) infections outside China. Data on confirmed cases diagnosed outside China were analyzed using statistical models to estimate the impact of travel reduction on three epidemiological outcome measures: (i) the number of exported cases, (ii) the probability of a major epidemic, and (iii) the time delay to a major epidemic. From 28 January to 7 February 2020, we estimated that 226 exported cases (95% confidence interval: 86, 449) were prevented, corresponding to a 70.4% reduction in incidence compared to the counterfactual scenario. The reduced probability of a major epidemic ranged from 7% to 20% in Japan, which resulted in a median time delay to a major epidemic of two days. Depending on the scenario, the estimated delay may be less than one day. As the delay is small, the decision to control travel volume through restrictions on freedom of movement should be balanced between the resulting estimated epidemiological impact and predicted economic fallout.</td><td>2020-02-17</td><td>Asami Anzai; Tetsuro Kobayashi; Natalie M. Linton; Ryo Kinoshita; Katsuma Hayashi; Ayako Suzuki; Yichi Yang; Sungmok Jung; Takeshi Miyama; Andrei R. Akhmetzhanov; Hiroshi Nishiura</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.14.20022897</td></tr><tr><td>6f6t558e</td><td>b1b9d81726e56c2781ce6b63ae5015cf2f314c6f</td><td>medrxiv</td><td>Estimating the Efficacy of Traffic Blockage and Quarantine for the Epidemic Caused by 2019-nCoV (COVID-19)</td><td>10.1101/2020.02.14.20022913</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since the 2019-nCoV (COVID-19) outbreaks in Wuhan, China, the cumulative number of confirmed cases is increasing every day, and a large number of populations all over the world are at risk. The quarantine and traffic blockage can alleviate the risk of the epidemic and the infections, henceforth evaluating the efficacy of such actions is essential to inform policy makers and raise the public awareness of the importance of self-isolation and quarantine. Method: We collected confirmed case data and the migration data, and introduced the quarantine factor and traffic blockage factor to the Flow-SEIR model. By varying the quarantine factor and traffic blockage factor, we simulated the change of the peak number and arrival time of infections, then the efficacy of these two intervation measures can be analyzed in our simulation. In our study, the self-protection at home is also included in quarantine. Results: In the simulated results, the quarantine and traffic blockage are effective for epidemic control. For Hubei province, the current quarantine factor is estimaed to be 0.405, which means around 40.5% of suceptibles who are close contacting with are in quarantine, and the current traffic blockage factor is estimaed to be 0.66, which indicates around 34% of suceptibles who had flowed out from Hubei. For the other provinces outside Hubei, the current quarantine factor is estimated to be 0.285, and the current traffic blockage factor is estimated to be 0.26. With the quarantine and traffic blockage factor increasing, the number of infections decrease dramatically. We also simulated the start dates of quarantine and traffic blockage at four time points, the simulated results show that the early of warning is also effective for epidemic containing. However, provincial level traffic blockage can only alleviate 21.06% - 22.38% of the peak number of infections. In general, the quarantine is much more effective than the traffic blockage control. Conclusion: Both of quarantine and traffic blockage are effective ways to control the spread of COVID-19. However, the eff icacy of quarantine is found to be much stronger than that of traffic blockage. Considering traffic blockage may also cause huge losses of economy, we propose to gradually deregulate the traffic blockage, and improve quarantine instead. Also, there might be a large number of asymptomatic carriers of COVID-19, the quarantine should be continued for a long time until the epidemic is totally under control.</td><td>2020-02-17</td><td>Deqiang Li; Zhicheng Liu; Qinghe Liu; Zefei Gao; Junkai Zhu; Junyan Yang; Qiao Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.14.20022913</td></tr><tr><td>4ah705nc</td><td>f4f4b045dce8ff5ff1f5aaa48a6bdbaaa7b05179</td><td>medrxiv</td><td>A new transmission route for the propagation of the SARS-CoV-2 coronavirus</td><td>10.1101/2020.02.14.20022939</td><td>null</td><td>null</td><td>medrvix</td><td>We explore here how variation in the SARS-CoV-2 virus tropism could influence epidemic spread. We use a compartmental model fit to the existing data. The model indicates that Wuhan quarantine measures were effective but that alternative virus forms (gut tropism) and a second propagation route (through environment) was present. For Singapore and Shenzhen region the secondary route does not seem to be active yet. Adequate prevention measures taking into account both routes should be implemented.</td><td>2020-02-18</td><td>Antoine Danchin; Tuen Wai Patrick Ng; Gabriel TURINICI</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.14.20022939</td></tr><tr><td>u2uswgy3</td><td>080660f20f078c10524f6186bca263327094acbb</td><td>medrxiv</td><td>A deep learning algorithm using CT images to screen for Corona Virus Disease (COVID-19)</td><td>10.1101/2020.02.14.20023028</td><td>null</td><td>null</td><td>medrvix</td><td>To control the spread of Corona Virus Disease (COVID-19), screening large numbers of suspected cases for appropriate quarantine and treatment is a priority. Pathogenic laboratory testing is the diagnostic gold standard but it is time consuming with significant false negative results. Fast and accurate diagnostic methods are urgently needed to combat the disease. Based on COVID-19 radiographical changes in CT images, we aimed to develop a deep learning method that could extract COVID-19's graphical features in order to provide a clinical diagnosis ahead of the pathogenic test, thus saving critical time for disease control. Methods:We collected 1,119 CT images of pathogen-confirmed COVID-19 cases along with those previously diagnosed with typical viral pneumonia. We modified the Inception transfer-learning model to establish the algorithm, followed by internal and external validation. Results: The internal validation achieved a total accuracy of 89.5% with specificity of 0.88 and sensitivity of 0.87. The external testing dataset showed a total accuracy of 79.3% with specificity of 0.83 and sensitivity of 0.67. In addition, in 54 COVID-19 images that first two nucleic acid test results were negative, 46 were predicted as COVID-19 positive by the algorithm, with the accuracy of 85.2%. Conclusion: These results demonstrate the proof-of-principle for using artificial intelligence to extract radiological features for timely and accurate COVID-19 diagnosis.</td><td>2020-02-17</td><td>Shuai Wang; Bo Kang; Jinlu Ma; Xianjun Zeng; Mingming Xiao; Jia Guo; Mengjiao Cai; Jingyi Yang; Yaodong Li; Xiangfei Meng; Bo Xu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.14.20023028</td></tr><tr><td>rgzmpoxv</td><td>f7891683eecd12b3908862fc7ef1d341ca7a293c</td><td>medrxiv</td><td>The Efficacy of Contact Tracing for the Containment of the 2019 Novel Coronavirus (COVID-19).</td><td>10.1101/2020.02.14.20023036</td><td>null</td><td>null</td><td>medrvix</td><td>Contact tracing is a central public health response to infectious disease outbreaks, especially in the early stages of an outbreak when specific treatments are limited. Importation of novel Coronavirus (COVID-19) from China and elsewhere into the United Kingdom highlights the need to understand the impact of contact tracing as a control measure. Using detailed survey information on social encounters coupled to predictive models, we investigate the likely efficacy of the current UK definition of a close contact (within 2 meters for 15 minutes or more) and the distribution of secondary cases that may go untraced. Taking recent estimates for COVID-19 transmission, we show that less than 1 in 5 cases will generate any subsequent untraced cases, although this comes at a high logistical burden with an average of 36.1 individuals (95th percentiles 0-182) traced per case. Changes to the definition of a close contact can reduce this burden, but with increased risk of untraced cases; we estimate that any definition where close contact requires more than 4 hours of contact is likely to lead to uncontrolled spread.</td><td>2020-02-17</td><td>Matt J Keeling; T. Deirdre Hollingsworth; Jonathan M Read</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.14.20023036</td></tr><tr><td>3nipr212</td><td>0eda8331214ca028350b07e2953702f3078a105e</td><td>medrxiv</td><td>Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (COVID-19)</td><td>10.1101/2020.02.14.20023127</td><td>null</td><td>null</td><td>medrvix</td><td>Background Estimation of the fraction and contagiousness of undocumented novel coronavirus (COVID-19) infections is critical for understanding the overall prevalence and pandemic potential of this disease. Many mild infections are typically not reported and, depending on their contagiousness, may support stealth transmission and the spread of documented infection. Methods Here we use observations of reported infection and spread within China in conjunction with mobility data, a networked dynamic metapopulation model and Bayesian inference, to infer critical epidemiological characteristics associated with the emerging coronavirus, including the fraction of undocumented infections and their contagiousness. Results We estimate 86% of all infections were undocumented (95% CI: [82%-90%]) prior to the Wuhan travel shutdown (January 23, 2020). Per person, these undocumented infections were 52% as contagious as documented infections ([44%-69%]) and were the source of infection for two-thirds of documented cases. Our estimate of the reproductive number (2.23; [1.77-3.00]) aligns with earlier findings; however, after travel restrictions and control measures were imposed this number falls considerably. Conclusions A majority of COVID-19 infections were undocumented prior to implementation of control measures on January 23, and these undocumented infections substantially contributed to virus transmission. These findings explain the rapid geographic spread of COVID-19 and indicate containment of this virus will be particularly challenging. Our findings also indicate that heightened awareness of the outbreak, increased use of personal protective measures, and travel restriction have been associated with reductions of the overall force of infection; however, it is unclear whether this reduction will be sufficient to stem the virus spread.</td><td>2020-02-17</td><td>Ruiyun Li; Sen Pei; Bin Chen; Yimeng Song; Tao Zhang; Wan Yang; Jeffrey Shaman</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.14.20023127</td></tr><tr><td>rcbw54xc</td><td>57e01ad2a4961cd5cc6a3733f5f8c013a8946f3c</td><td>medrxiv</td><td>A model simulation study on effects of intervention measures in Wuhan COVID-19 epidemic</td><td>10.1101/2020.02.14.20023168</td><td>null</td><td>null</td><td>medrvix</td><td>Background: In the beginning of January 2020, new unknown virus pneumonia cases started to emerge in local hospitals in Wuhan, China. This virus epidemic quickly became a public health emergency of international concern by the WHO. Enormous amount of medical supplies as well as healthcare personals from other provinces were mobilized to support Wuhan. This current work tent to help people understanding how infectious disease spread and the purpose and consequences of various efforts based on simulation model. Method: a simulation model was created using known parameters. R0 set to 3 and mean incubation time to be 7.5days. the epidemic was divided to 3 periods. Simulation would run 50 times to mimic different patient0 status. Personal activity index was used to mimic different level of control measures. 141427709 simulated patients were created. Cumulation number of patients at the end of period 1 (day50) is 2868.7 ± 1739.0. Total infected patients could be 913396.5 ± 559099.9 by the end of period 2 (day70) in free transmission state. And at day90, total patients number is 913396.5 ± 559099.9. Conclusion: COVID-19 is a novel severe respiratory disease. This will put great burden on the shoulder of healthcare workers as well as on medical hardware and supplements. Current strict control measures help to contain disease from spreading. An early detecting, reporting and fast reacting system needs to be setup to prevent future unknown infectious disease.</td><td>2020-02-18</td><td>Guopeng ZHOU; Chunhua CHI</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.14.20023168</td></tr><tr><td>y9sel25v</td><td>null</td><td>medrxiv</td><td>Estimate number of individuals infected with the 2019-novel coronavirus in South Korea due to the influx of international students from countries with virus risk: a simulation study</td><td>10.1101/2020.02.15.20023234</td><td>null</td><td>null</td><td>medrvix</td><td>Background: In March 2020, overall, 37,000 international students from the country at risk of the 2019-novel coronavirus (COVID-19) infection will arrive in Seoul, South Korea. Individuals from the country at risk of COVID-19 infection have been included in a home-quarantine program, but the efficacy of the program is uncertain. Methods: To estimate the possible number of infected individuals within the large influx of international students, we used a deterministic compartmental model for epidemic and perform a simulation-based search of different rates of compliance with home-quarantine. Results: Under the home-quarantine program, the total number of the infected individuals would reach 24-53 from March 17-March 20, 50-86 from March 18-March 16, and 234-343 from March 4-March 23 with the arrival of 0.1%, 0.2%, and 1% of pre-infectious individuals, in Seoul, South Korea, respectively. Our findings indicated when incoming international students showed strict compliance with quarantine, epidemics were less likely to occur in Seoul, South Korea. Conclusion: To mitigate possible epidemics, additional efforts to improve the compliance of home-quarantine are warranted along with other containment policies.</td><td>2020-02-19</td><td>Sukhyun Ryu; Sheikh Taslim Ali; Jun-sik Lim; Byung Chul Chun</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.15.20023234</td></tr><tr><td>1rq612hm</td><td>91098f6fe46a21565bf0cb06fe960cdb2c3f5e38</td><td>medrxiv</td><td>The role of institutional trust in preventive and treatment-seeking behaviors during the 2019 novel coronavirus (2019-nCoV) outbreak among residents in Hubei, China</td><td>10.1101/2020.02.15.20023333</td><td>null</td><td>null</td><td>medrvix</td><td>Background Since December 2019, pneumonia associated with the 2019 novel coronavirus (2019-nCoV) has emerged in Wuhan, China. The exponential increase of the confirmed number of cases of 2019n-CoV is of great concern to the global community. The fears and panic among residents in the epicenters have prompted diverse responses, which are understudied. During such a crisis, community trust and support for the government and health authorities are important to contain the outbreak. We aimed to investigate the influence of institutional trust on public responses to the 2019-nCoV outbreak. Methods An anonymous Internet-based, cross-sectional survey was administered on January 29, 2020. The study population comprised all residents currently residing or working in the province of Hubei, where Wuhan is the capital city. The level of trust in information provision and preventive instructions, individual preventive behaviors and treatment-seeking behaviors were queried. Findings The majority of the participants expressed a great extent of trust in the information and preventive instructions provided by the central government than by the local government. A high uptake of 2019-nCoV preventive measures was found, particularly among people who had been placed under quarantine. Being under quarantine (adjusted odds ratio [OR] = 2.35, 95% confidence interval [CI] 1.80 to 3.08) and having a high institutional trust score (OR = 2.23, 95% CI 1.96 to 2.53) were both strong and significant determinants of higher preventive behavior scores. The majority of study participants (85.7%, n = 3,640) reported that they would seek hospital treatment if they suspected themselves to have been infected with 2019 n-CoV. Few of the participants from Wuhan (16.6%, n = 475) and those participants who were under quarantine (13.8%, n = 550) expressed an unwillingness to seek hospital treatment. Similarly, being under quarantine (OR = 2.36, 95% CI 1.80 to 3.09) and having a high institutional trust score (OR = 2.20, 95% CI 1.96 to 2.49) were two strong significant determinants of hospital treatment-seeking. Interpretation The results of this study suggest that institutional trust is an important factor influencing adequate preventive behavior and seeking formal medical care during an outbreak. In view of the 2019-nCoV being highly pathogenic and extremely contagious, our findings also underscore the importance of public health intervention to reach individuals with poor adherence to preventive measures and who are reluctant to seek treatment at formal health services. Funding National Key R&D Program of China, Ningbo Health Branding Subject Fund, Sanming Project of Medicine in Shenzhen, K.C. Wong Magna Fund in Ningbo University, National Natural Science Foundation of China, Fundamental Research Funds for the Central Universities, China Postdoctoral Science Foundation, and Natural Science Basic Research Program of Shanxi Province. Keywords: 2019-nCoV; institutional trust; preventive behaviors</td><td>2020-02-21</td><td>Li Ping Wong; Qun hong Wu; Xi Chen; Zhuo Chen; Haridah Alias; Mingwang Shen; Jing cen Hu; Shiwei Duan; Jin jie Zhang; Liyuan Han</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.15.20023333</td></tr><tr><td>ehm4fgos</td><td>317943dec06b88c1c62ef0ecd832f0d644b1d417</td><td>medrxiv</td><td>Evaluating new evidence in the early dynamics of the novel coronavirus COVID-19 outbreak in Wuhan, China with real time domestic traffic and potential asymptomatic transmissions</td><td>10.1101/2020.02.15.20023440</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus (COVID-19), first detected in Wuhan, China in December 2019, has spread to 28 countries/regions with over 43,000 confirmed cases. Much about this outbreak is still unknown. At this early stage of the epidemic, it is important to investigate alternative sources of information to understand its dynamics and spread. With updated real time domestic traffic, this study aims to integrate recent evidence of international evacuees extracted from Wuhan between Jan. 29 and Feb. 2, 2020 to infer the dynamics of the COVD-19 outbreak in Wuhan. In addition, a modified SEIR model was used to evaluate the empirical support for the presence of asymptomatic transmissions. Based on the data examined, this study found little evidence for the presence of asymptomatic transmissions. However, it is still too early to rule out its presence conclusively due to sample size and other limitations. The updated basic reproductive number was found to be 2.12 on average with a 95% credible interval of [2.04, 2.18]. It is smaller than previous estimates probably because the new estimate factors in the social and non-pharmaceutical mitigation implemented in Wuhan through the evacuee dataset. Detailed predictions of infected individuals exported both domestically and internationally were produced. The estimated case confirmation rate has been low but has increased steadily to 23.37% on average. The findings of this study depend on the validity of the underlying assumptions, and continuing work is needed, especially in monitoring the current infection status of Wuhan residents.</td><td>2020-02-18</td><td>Can Zhou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.15.20023440</td></tr><tr><td>3r0nqzgu</td><td>73998a3f565b661816d48aa7c20fa5ae617864d5</td><td>medrxiv</td><td>Profiling ACE2 expression in colon tissue of healthy adults and colorectal cancer patients by single-cell transcriptome analysis</td><td>10.1101/2020.02.15.20023457</td><td>null</td><td>null</td><td>medrvix</td><td>A newly identified novel coronavirus (2019-nCoV) has caused numerous acute respiratory syndrome cases in Wuhan China from December 2019 to Feb 2020. Its fast spreading to other provinces in China and overseas is very likely causing a pandemic. Since the novel coronavirus has been reported to be capable of endangering thousands of lives, it is extremely important to find out how the coronavirus is transmitted in human organs. Apart from fever and respiratory complications, gastrointestinal symptoms are observed in some patients with 2019-nCoV but the significance remains undetermined. The cell receptor angiotensin covering enzyme II (ACE2), which is the major receptor of SARS-nCoV, has been reported to be a cellular entry receptor of 2019-nCoV as well. Here, to more precisely explore the potential pathogen transmission route of the 2019-nCoV infections in the gastrointestinal tract, we analyzed the ACE2 RNA expression profile in the colon tissue of healthy adults and colorectal cancer patients of our cohort and other databases. The data indicates that ACE2 is mainly expressed in epithelial cells of the colon. The expression of ACE2 is gradually increased from healthy control, adenoma to colorectal cancer patients in our cohort as well as in the external Asian datasets. According to the expression profile of ACE2 in colon epithelial cells, we speculate adenoma and colorectal cancer patients are more likely to be infected with 2019-nCoV than healthy people. Our data may provide a theoretical basis for the classification and management of future 2019-nCoV susceptibility people in clinical application.</td><td>2020-02-23</td><td>Haoyan Chen; Baoqin Xuan; Yuqing Yan; Xiaoqiang Zhu; Chaoqin Shen; Gang Zhao; Linhua Ji; Danhua Xu; Hua Xiong; TaChung Yu; Xiaobo Li; Qiang Liu; Yingxuan Chen; Yun Cui; Jie Hong; Jing-Yuan Fang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.15.20023457</td></tr><tr><td>b4kzgubs</td><td>f9c6a656d0ce352cc1fa0df8d83e6d4a1f53d7c2</td><td>medrxiv</td><td>Estimating the Case Fatality Risk of COVID-19 using Cases from Outside China</td><td>10.1101/2020.02.15.20023499</td><td>null</td><td>null</td><td>medrvix</td><td>There is large uncertainty around the case fatality risk (CFR) for COVID-19 in China. Therefore, we considered symptomatic cases outside of China (countries/settings with 20+ cases) and the proportion who are in intensive care units (4.0%, 14/349 on 13 February 2020). Given what is known about CFRs for ICU patients with severe respiratory conditions from a meta-analysis, we estimated a CFR of 1.37% (95%CI: 0.57% to 3.22%) for COVID-19 cases outside of China.</td><td>2020-02-18</td><td>Nick Wilson; Amanda Kvalsvig; Lucy Telfar Barnard; Michael Baker</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.15.20023499</td></tr><tr><td>9aegg5sd</td><td>1df9ce60e358f6cef0be3f6e15fe2a9c6f93f39c</td><td>biorxiv</td><td>Multivariate Analyses of Codon Usage of SARS-CoV-2 and other betacoronaviruses</td><td>10.1101/2020.02.15.950568</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractCoronavirus disease 2019 (COVID-19) is a global health concern as it continues to spread within China and beyond. The causative agent of this disease, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belongs to the genus Betacoronavirus which also includes severe acute respiratory syndrome related coronavirus (SARSr-CoV) and Middle East respiratory syndrome related coronavirus (MERSr-CoV). Codon usage of viral genes are believed to be subjected to different selection pressures in different host environments. Previous studies on codon usage of influenza A viruses can help identify viral host origins and evolution trends, however, similar studies on coronaviruses are lacking. In this study, global correspondence analysis (CA), within-group correspondence analysis (WCA) and between-group correspondence analysis (BCA) were performed among different genes in coronavirus viral sequences. The amino acid usage pattern of SARS-CoV-2 was generally found similar to bat and human SARSr-CoVs. However, we found greater synonymous codon usage differences between SARS-CoV-2 and its phylogenetic relatives on spike and membrane genes, suggesting these two genes of SARS-CoV-2 are subjected to different evolutionary pressures.</td><td>2020-02-20</td><td>Haogao Gu; Daniel Chu; Malik Peiris; Leo L.M. Poon</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.15.950568</td></tr><tr><td>m87tapjp</td><td>10138816c67a562b55bc9c2b882b62c81fc4b062</td><td>medrxiv</td><td>Epidemic analysis of COVID-19 in China by dynamical modeling</td><td>10.1101/2020.02.16.20023465</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of the novel coronavirus (2019-nCoV) epidemic has attracted world- wide attention. Herein, we propose a mathematical model to analyzes this epidemic, based on a dynamic mechanism that incorporating the intrinsic impact of hidden la- tent and infectious cases on the entire process of transmission. Meanwhile, this model is validated by data correlation analysis, predicting the recent public data, and back- tracking, as well as sensitivity analysis. The dynamical model reveals the impact of various measures on the key parameters of the epidemic. According to the public data of NHCs from 01/20 to 02/09, we predict the epidemic peak and possible end time for 5 different regions. The epidemic in Beijing and Shanghai, Mainland/Hubei and Hubei/Wuhan, are expected to end before the end of February, and before mid- March respectively. The model indicates that, the outbreak in Wuhan is predicted to be ended in the early April. As a result, more effective policies and more efforts on clinical research are demanded. Moreover, through the backtracking simulation, we infer that the outbreak of the epidemic in Mainland/Hubei, Hubei/Wuhan, and Wuhan can be dated back to the end of December 2019 or the beginning of January 2020.</td><td>2020-02-18</td><td>Liangrong Peng; Wuyue Yang; Dongyan Zhang; Changjing Zhuge; Liu Hong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023465</td></tr><tr><td>9j5yda3k</td><td>b117016a887f05386acc4387dab5bf8c769cf90e</td><td>medrxiv</td><td>Simulating and Forecasting the Cumulative Confirmed Cases of SARS-CoV-2 in China by Boltzmann Function-based Regression Analyses</td><td>10.1101/2020.02.16.20023564</td><td>null</td><td>null</td><td>medrvix</td><td>An ongoing outbreak of atypical pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is hitting Wuhan City and has spread to other provinces/cities of China and overseas. It very urgent to forecast the future course of the outbreak. Here, we provide an estimate of the potential total number of confirmed cases in mainland China by applying Boltzmann-function based regression analyses. We found that the cumulative number of confirmed cases from Jan 21 to Feb 14, 2020 for mainland China, Hubei Province, Wuhan City and other provinces were all well fitted with the Boltzmann function (R2 being close to 0.999). The potential total number of confirmed cases in the above geographic regions were estimated at 95% confidence interval (CI) as 79589 (71576, 93855), 64817 (58223, 77895), 46562 (40812, 57678) and 13956 (12748, 16092), respectively. Notably, our results suggest that the number of daily new confirmed cases of SARS-CoV-2 in mainland China (including Hubei Province) will become minimal between Feb 28 and Mar 10, 2020, with 95% CI. In addition, we found that the data of cumulative confirmed cases of 2003 SARS-CoV in China and Worldwide were also well fitted to the Boltzmann function. To our knowledge this is the first study revealing that the Boltzmann function is suitable to simulate epidemics. The estimated potential total number of confirmed cases and key dates for the SARS-CoV-2 outbreak may provide certain guidance for governments, organizations and citizens to optimize preparedness and response efforts.</td><td>2020-02-18</td><td>Xinmiao Fu; Qi Ying; Tieyong Zeng; Tao Long; Yan Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023564</td></tr><tr><td>ndnkhj42</td><td>8bd8158e4a47c65b12a462b49b4b8b4af1488dc3</td><td>medrxiv</td><td>Estimation of the final size of the coronavirus epidemic by the logistic model</td><td>10.1101/2020.02.16.20023606</td><td>null</td><td>null</td><td>medrvix</td><td>In the note, the logistic growth regression model is used for the estimation of the final size and its peak time of the coronavirus epidemic. Based on available data estimation the final size will be about 90 000 cases and the peak time was on 10 Feb 2020.</td><td>2020-02-18</td><td>milan batista</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023606</td></tr><tr><td>oknxic4q</td><td>d069dfb7f0aefcdc2c890a1bbe773ebd26b01a55</td><td>medrxiv</td><td>Utilize State Transition Matrix Model to Predict the Novel Corona Virus Infection Peak and Patient Distribution</td><td>10.1101/2020.02.16.20023614</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since December 2019, a pneumonia caused by the 2019 novel coronavirus (2019-nCoV) has broken out in Wuhan, Hubei province, China. The continuous rising of infected cases has imposed overwhelming pressure on public health decision and medical resource allocation in China. We managed to forecast the infection peak time in Hubei province and the severe and critical case distribution. Methods: We used data resource according to cases reported by the National Health Commission of the People's Republic of China (Jan 25, 2019, to Feb 28, 2020) as the training set to deduce the arrival of the peak infection time and the number of severe and critical cases in Wuhan on subsequent days. Medical observation, discharge, infected, non-Severe, infected and severe, cure and death data were collected and analyzed. Using this state transition matrix model, we will be able predict when the inflection peak time (the maximum open infection cases) in Hubei Province will occur. Also, we can use this model to predict the patient distribution (severe, non-severe) to better allocate medical resource. Under relative pessimistic scenario, the inflection peak time is April 6-April 14. The numbers of critically ill and critically ill patients will lie between 8300-9800 and 2200-2700, respectively. Results: In very optimistic scenarios (daily NCC decay rate of -10%), the peak time of open inflection cases will arrive around February 23-February 26. At the same time, there will be a peak in the numbers of severely ill and critically ill patients, between 6800-7200 and 1800-2000, respectively. In a relative optimistic scenario (daily NCC decay rate of -5%), the inflection case peak time will arrive around February 28-March 2. The numbers of critically ill and critically ill patients will lie between 7100-7800 and 1900-2200, respectively. In a relatively pessimistic scenario (daily NCC decay rate of -1%), the inflection peak time does not arrive around the end of March. Estimated time is April 6-April 14. The numbers of critically ill and critically ill patients will lie between 8300-9800 and 2200-2700, respectively. We are using the diagnosis rate, mortality rate, cure rate as the 2/8 data. There should be room for improvement, if these metrics continue to improve. In that case, the peak time will arrive earlier than our estimation. Also, the severe and critical case ratios are likely to decline as the virus becomes less toxic and medical conditions improve. If that happens, the peak numbers will be lower than predicted above. Conclusion: We can infer that we are still not close to the end of this outbreak and the number of critically ill patients is still climbing. Assisting critical care resources in Hubei province requires the government to consider further tilt, and it is vital to make reasonable management of doctors and medical assistance systems to curb the transmission trend.</td><td>2020-02-19</td><td>Ke Wu; Junhua Zheng; Jian Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023614</td></tr><tr><td>c800ynvc</td><td>3da1a03cb675f463b0d8e30744aa33512241dd29</td><td>medrxiv</td><td>SEIR Transmission dynamics model of 2019 nCoV coronavirus with considering the weak infectious ability and changes in latency duration</td><td>10.1101/2020.02.16.20023655</td><td>null</td><td>null</td><td>medrvix</td><td>Pneumonia patients of 2019-ncov in latent period are not easy to be effectively quarantined, but there is evidence that they have strong infectious ability. Here, the infectious ability of patients during the latent period is slightly less than that of the infected patients was assumed. We established a new SEIR propagation dynamics model, that considered the weak transmission ability of the incubation period, the variation of the incubation period length, and the government intervention measures to track and isolate comprehensively. Based on the raw epidemic data of China from January 23, 2020 to February 10, 2020, the dynamic parameters of the new present SEIR model are fitted. Through the Euler integration algorithm to solve the model, the effect of infectious ability of incubation patients on the theoretical estimation of the present SEIR model was analyzed, and the occurrence time of peak number in China was predicted.</td><td>2020-02-20</td><td>Pengpeng Shi; Shengli Cao; Peihua Feng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023655</td></tr><tr><td>9xmeyrtu</td><td>7325363d061f0f4699e29bfa544438e14e7c207d</td><td>medrxiv</td><td>Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</td><td>10.1101/2020.02.16.20023671</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The dynamic changes of lymphocyte subsets and cytokines profiles of patients with novel coronavirus disease (COVID-19) and their correlation with the disease severity remain unclear. Method: Peripheral blood samples were longitudinally collected from 40 confirmed COVID-19 patients and examined for lymphocyte subsets by flow cytometry and cytokine profiles by specific immunoassays. Findings: Of the 40 COVID-19 patients enrolled, 13 severe cases showed significant and sustained decreases in lymphocyte counts but increases in neutrophil counts than 27 mild cases. Further analysis demonstrated significant decreases in the counts of T cells, especially CD8 + T cells, as well as increases in IL-6, IL-10, IL-2 and IFN-γ levels in the peripheral blood in the severe cases compared to those in the mild cases. T cell counts and cytokine levels in severe COVID-19 patients who survived the disease gradually recovered at later time points to levels that were comparable to those of the mild cases. Moreover, the neutrophil-to-CD8+ T cell ratio (N8R) were identified as the most powerful prognostic factor affecting the prognosis for severe COVID-19. Conclusion: The degree of lymphopenia and a proinflammatory cytokine storm is higher in severe COVID-19 patients than in mild cases, and is associated with the disease severity. N8R may serve as a useful prognostic factor for early identification of severe COVID-19 cases.</td><td>2020-02-18</td><td>Jing Liu; Sumeng Li; Jia Liu; Boyun Liang; Xiaobei Wang; Hua Wang; Wei Li; Qiaoxia Tong; Jianhua Yi; Lei Zhao; Lijuan Xiong; Chunxia Guo; Jin Tian; Jinzhuo Luo; Jinghong Yao; Ran Pang; Hui Shen; Cheng Peng; Ting Liu; Qian Zhang; Jun Wu; Ling Xu; Sihong Lu; Baoju Wang; Zhihong Weng; Chunrong Han; Huabing Zhu; Ruxia Zhou; Helong Zhou; Xiliu Chen; Pian Ye; Bin Zhu; Shengsong He; Yongwen He; Shenghua Jie; Ping Wei; Jianao Zhang; Yinping Lu; Weixian Wang; Li Zhang; Ling Li; Fengqin Zhou; Jun Wang; Ulf Dittmer; Mengji Lu; Yu Hu; Dongliang Yang; Xin Zheng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023671</td></tr><tr><td>fugb778l</td><td>6445688d3f59cb7e02f2a8b28450cdf118a8a373</td><td>medrxiv</td><td>Contacts in context: large-scale setting-specific social mixing matrices from the BBC Pandemic project</td><td>10.1101/2020.02.16.20023754</td><td>null</td><td>null</td><td>medrvix</td><td>Social mixing patterns are crucial in driving transmission of infectious diseases and informing public health interventions to contain their spread. Age-specific social mixing is often inferred from surveys of self-recorded contacts which by design often have a very limited number of participants. In addition, such surveys are rare, so public health interventions are often evaluated by considering only one such study. Here we report detailed population contact patterns for United Kingdom based self-reported contact data from over 36,000 volunteers that participated in the massive citizen science project BBC Pandemic. The amount of data collected allows us generate fine-scale age-specific population contact matrices by context (home, work, school, other) and type (conversational or physical) of contact that took place. These matrices are highly relevant for informing prevention and control of new outbreaks, and evaluating strategies that reduce the amount of mixing in the population (such as school closures, social distancing, or working from home). In addition, they finally provide the possibility to use multiple sources of social mixing data to evaluate the uncertainty that stems from social mixing when designing public health interventions.</td><td>2020-02-19</td><td>Petra Klepac; Adam J Kucharski; Andrew JK Conlan; Stephen Kissler; Maria Tang; Hannah Fry; Julia R Gog</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023754</td></tr><tr><td>asyrijyg</td><td>null</td><td>medrxiv</td><td>Study on SARS-COV-2 transmission and the effects of control measures in China</td><td>10.1101/2020.02.16.20023770</td><td>null</td><td>null</td><td>medrvix</td><td>Objective To reconstruct the transmission trajectory of SARS-COV-2 and analyze the effects of control measures in China. Methods Python 3.7.1 was used to write a SEIR class to model the epidemic procedure and a back propagation class to estimate the initial true infected number. The epidemic area in China was divided into three parts, Wuhan city, Hubei province (except Wuhan) and China (except Hubei) based on the different transmission pattern. A limitation factor for the medical resource was imposed to model the infected but not quarantined. Credible data source from Baidu Qianxi were used to assess the number of infected cases migrated from Wuhan to other areas. Results Basic reproduction number, R0, was 3.6 in the very early stage. The true infected number was 4508 in our model in Wuhan before January 22, 2020. By January 22 2020, it was estimated that 1764 infected cases migrated from Wuhan to other cities in Hubei province. Effective reproductive number, R, gradually decreased from 3.6 (Wuhan, stage 1), 3.4 (Hubei except Wuhan, stage 1) and 3.3 (China except Hubei, stage 1) to 0.67 (Wuhan, stage 4), 0.83 (Hubei except Wuhan, stage 2) and 0.63 (China except Hubei, stage 2), respectively. Especially after January 23, 2020 when Wuhan City was closed, the infected number showed a turning point in Wuhan. By early April, there would be 42073, 21342 and 13384 infected cases in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively, and there would be 2179, 633 and 107 death in Wuhan, Hubei (except Wuhan) and China (except Hubei) respectively. Conclusion A series of control measures in China have effectively prevented the spread of COVID-19, and the epidemic will end in early April.</td><td>2020-02-18</td><td>Bo Zhang; Hongwei Zhou; Fang Zhou</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023770</td></tr><tr><td>32vs98ya</td><td>e62e453735dc2022f4d81aa9b22769de30d0bb90</td><td>medrxiv</td><td>When will the battle against novel coronavirus end in Wuhan: a SEIR modeling analysis</td><td>10.1101/2020.02.16.20023804</td><td>null</td><td>null</td><td>medrvix</td><td>Recent outbreak of 2019-nCoV in Wuhan raised serious public health concerns. By February 15, 2020 in Wuhan, the total number of confirmed infection cases has reached 37,914, and the number of deaths has reached 1123, accounting for 56.9% of the total confirmed cases and 73.7% of the total deaths in China. People are eager to know when the epidemic will be completely controlled and when people's work and life will be on the right track. In this study we analyzed the epidemic dynamics and trend of 2019-nCoV in Wuhan by using the data after the closure of Wuhan city till February 12, 2020 based on the SEIR modeling method. The optimal parameters were estimated as R0=1.44 (interquartile range: 1.40-1.47)，TI=14 (interquartile range: 14-14) and TE=3.0 (interquartile range: 2.8-3.1). Based on these parameters, the number of infected individuals in Wuhan city may reach the peak around February 19 at about 45,000 people. Once entering March, the epidemic would gradually decline, and end around the late March. It is worth noting that the above prediction is based on the assumption that the number of susceptible population N = 200,000 will not increase. If the epidemic situation is not properly controlled, the peak of infected number can be further increased and the peak time will be a little postponed. It was expected that the epidemic would subside in early March, and disappear gradually towards the late March.</td><td>2020-02-18</td><td>lianglu zhang; kangkang wan; jing chen; changming lu; lanlan dong; zhicheng wu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023804</td></tr><tr><td>jljjqs6m</td><td>33bd85ea9ca61c4d0674c43ae5b901b271b46f1e</td><td>medrxiv</td><td>Fractal kinetics of COVID-19 pandemic</td><td>10.1101/2020.02.16.20023820</td><td>null</td><td>null</td><td>medrvix</td><td>We give an update to the original paper posted on 2/17/20 -- now (as of 3/1/20) the China deaths are rapidly decreasing, and we find an exponential decline to the power law similar to the that predicted by the network model of \\citet{vazquez_polynomial_2006}. At the same time, we see non-China deaths increasing rapidly, and similar to the early behavior of the China statistics. Thus, we see three stages of the spread of the disease in terms of number of deaths: exponential growth, power-law behavior, and then exponential decline in the daily rate. (Original abstract) The novel coronavirus (COVID-19) continues to grow rapidly in China and is spreading in other parts of the world. The classic epidemiological approach in studying this growth is to quantify a reproduction number and infection time, and this is the approach followed by many studies on the epidemiology of this disease. However, this assumption leads to exponential growth, and while the growth rate is high, it is not following exponential behavior. One approach that is being used is to simply keep adjusting the reproduction number to match the dynamics. Other approaches use rate equations such as the SEIR and logistical models. Here we show that the current growth closely follows power-law kinetics, indicative of an underlying fractal or small-world network of connections between susceptible and infected individuals. Positive deviations from this growth law might indicate either a failure of the current containment efforts while negative deviations might indicate the beginnings of the end of the pandemic. We cannot predict the ultimate extent of the pandemic but can get an estimate of the growth of the disease.</td><td>2020-02-20</td><td>Anna L. Ziff; Robert M. Ziff</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023820</td></tr><tr><td>z5i8hyja</td><td>fbebe4b66073c44cface2e842754bce26e3e2913</td><td>medrxiv</td><td>Risk map of the novel coronavirus (2019-nCoV) in China: proportionate control is needed</td><td>10.1101/2020.02.16.20023838</td><td>null</td><td>null</td><td>medrvix</td><td>Background China is running a national level antivirus campaign against the novel coronavirus (2019-nCoV). Strict control measures are being enforced in either the populated areas and remote regions. While the virus is closed to be under control, tremendous economic loss has been caused. Methods and findings We assessed the pandemic risk of 2019-nCoV for all cities/regions in China using the random forest algorithm, taking into account the effect of five factors: the accumulative and increased numbers of confirmed cases, total population, population density, and GDP. We defined four levels of the risk, corresponding to the four response levels to public health emergencies in China. The classification system has good consistency among cities in China, as the error rate of the confusion matrix is 1.58%. Conclusions The pandemic risk of 2019-nCoV is dramatically different among the 442 cities/regions. We recommend to adopt proportionate control policy according to the risk level to reduce unnecessary economic loss.</td><td>2020-02-18</td><td>Xinhai Li; Xumao Zhao; Yingqiang Lou; Yuehua Sun</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023838</td></tr><tr><td>bsgqsdbx</td><td>d4cfb1fc4fc53abbc1d4b0ca60276c6af6632c3c</td><td>medrxiv</td><td>Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</td><td>10.1101/2020.02.16.20023903</td><td>null</td><td>null</td><td>medrvix</td><td>Background Since late December, 2019, an outbreak of pneumonia cases caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, and continued to spread throughout China and across the globe. To date, few data on immunologic features of Coronavirus Disease 2019 (COVID-19) have been reported. Methods In this single-centre retrospective study, a total of 21 patients with pneumonia who were laboratory-confirmed to be infected with SARS-CoV-2 in Wuhan Tongji hospital were included from Dec 19, 2019 to Jan 27, 2020. The immunologic characteristics as well as their clinical, laboratory, radiological features were compared between 11 severe cases and 10 moderate cases. Results Of the 21 patients with COVID-19, only 4 (19%) had a history of exposure to the Huanan seafood market. 7 (33.3%) patients had underlying conditions. The average age of severe and moderate cases was 63.9 and 51.4 years, 10 (90.9%) severe cases and 7 (70.0%) moderate cases were male. Common clinical manifestations including fever (100%, 100%), cough (70%, 90%), fatigue (100%, 70%) and myalgia (50%, 30%) in severe cases and moderate cases. PaO2/FiO2 ratio was significantly lower in severe cases (122.9) than moderate cases (366.2). Lymphocyte counts were significantly lower in severe cases (7000 million/L) than moderate cases (11000 million/L). Alanine aminotransferase, lactate dehydrogenase levels, high-sensitivity C-reactive protein and ferritin were significantly higher in severe cases (41.4 U/L, 567.2 U/L, 135.2 mg/L and 1734.4 ug/L) than moderate cases (17.6 U/L, 234.4 U/L, 51.4 mg/L and 880.2 ug /L). IL-2R, TNF-α and IL-10 concentrations on admission were significantly higher in severe cases (1202.4 pg/mL, 10.9 pg/mL and 10.9 pg/mL) than moderate cases (441.7 pg/mL, 7.5 pg/mL and 6.6 pg/mL). Absolute number of total T lymphocytes, CD4+T cells and CD8+T cells decreased in nearly all the patients, and were significantly lower in severe cases (332.5, 185.6 and 124.3 million/L) than moderate cases (676.5, 359.2 and 272.0 million/L). The expressions of IFN-γ by CD4+T cells tended to be lower in severe cases (14.6%) than moderate cases (23.6%). Conclusion The SARS-CoV-2 infection may affect primarily T lymphocytes, particularly CD4+T cells, resulting in significant decrease in number as well as IFN-γ production, which may be associated with disease severity. Together with clinical characteristics, early immunologic indicators including diminished T lymphocytes and elevated cytokines may serve as potential markers for prognosis in COVID-19.</td><td>2020-02-19</td><td>Guang Chen; Di Wu; Wei Guo; Yong Cao; Da Huang; Hongwu Wang; Tao Wang; Xiaoyun Zhang; Huilong Chen; Haijing Yu; Xiaoping Zhang; Minxia Zhang; Shiji Wu; Jianxin Song; Tao Chen; Meifang Han; Shusheng Li; Xiaoping Luo; Jianping Zhao; Qin Ning</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.20023903</td></tr><tr><td>tjxtifao</td><td>56a81431a148344381a1da80c877885f3f3d6d59</td><td>biorxiv</td><td>Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites</td><td>10.1101/2020.02.16.946699</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAlternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of Toxoplasma gondii and Plasmodium falciparum. We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies.</td><td>2020-02-17</td><td>V Vern Lee; Louise M. Judd; Aaron R. Jex; Kathryn E. Holt; Christopher J. Tonkin; Stuart A. Ralph</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.946699</td></tr><tr><td>nhq0oq8y</td><td>a388cca261e8af7936cf3b79c35f43442f65706b</td><td>biorxiv</td><td>SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</td><td>10.1101/2020.02.16.951723</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSARS-CoV-2 and SARS-CoV share a common human receptor ACE2. Protein-protein interaction structure modeling indicates that spike-RBD of the two viruses also has similar overall binding conformation and binding free energy to ACE2. In vitro assays using recombinant ACE2 proteins and ACE2 expressing cells confirmed the two coronaviruses’ similar binding affinities to ACE2. The above studies provide experimental supporting evidences and possible explanation for the high transmissibility observed in the SARS-CoV-2 outbreak. Potent ACE2-blocking SARS-CoV neutralizing antibodies showed limited cross-binding and neutralizing activities to SARS-CoV-2. ACE2-non-blocking SARS-CoV RBD antibodies, though with weaker neutralizing activities against SARS-CoV, showed positive cross-neutralizing activities to SARS-CoV-2 with an unknown mechanism. These findings suggest a trade-off between the efficacy and spectrum for therapeutic antibodies to different coronaviruses, and hence highlight the possibilities and challenges in developing broadly protecting antibodies and vaccines against SARS-CoV-2 and its future mutants.</td><td>2020-02-20</td><td>Chunyun Sun; Long Chen; Ji Yang; Chunxia Luo; Yanjing Zhang; Jing Li; Jiahui Yang; Jie Zhang; Liangzhi Xie</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.951723</td></tr><tr><td>rymj6lkj</td><td>22f3be8bbf6d792549b7a6088df7440e425cb34f</td><td>biorxiv</td><td>Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses</td><td>10.1101/2020.02.16.951913</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe new coronavirus (2019-nCoV) outbreak from December 2019 in Wuhan, Hubei, China, has been declared a global public health emergency. Angiotensin I converting enzyme 2 (ACE2), is the host receptor by 2019-nCov to infect human cells. Although ACE2 is reported to be expressed in lung, liver, stomach, ileum, kidney and colon, its expressing levels are rather low, especially in the lung. 2019-nCoV may use co-receptors/auxiliary proteins as ACE2 partner to facilitate the virus entry. To identify the potential candidates, we explored the single cell gene expression atlas including 119 cell types of 13 human tissues and analyzed the single cell co-expression spectrum of 51 reported RNA virus receptors and 400 other membrane proteins. Consistent with other recent reports, we confirmed that ACE2 was mainly expressed in lung AT2, liver cholangiocyte, colon colonocytes, esophagus keratinocytes, ileum ECs, rectum ECs, stomach epithelial cells, and kidney proximal tubules. Intriguingly, we found that the candidate co-receptors, manifesting the most similar expression patterns with ACE2 across 13 human tissues, are all peptidases, including ANPEP, DPP4 and ENPEP. Among them, ANPEP and DPP4 are the known receptors for human CoVs, suggesting ENPEP as another potential receptor for human CoVs. We also conducted “CellPhoneDB” analysis to understand the cell crosstalk between CoV-targets and their surrounding cells across different tissues. We found that macrophages frequently communicate with the CoVs targets through chemokine and phagocytosis signaling, highlighting the importance of tissue macrophages in immune defense and immune pathogenesis.</td><td>2020-02-21</td><td>Furong Qi; Shen Qian; Shuye Zhang; Zheng Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.16.951913</td></tr><tr><td>x2bl5lcn</td><td>null</td><td>medrxiv</td><td>Estimating the case fatality ratio of the COVID-19 epidemic in China</td><td>10.1101/2020.02.17.20023630</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Corona Virus Disease 2019 (COVID-19) due to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan city and rapidly spread throughout China since late December 2019. Crude case fatality ratio (CFR) with dividing the number of known deaths by the number of confirmed cases does not represent the true CFR and might be off by orders of magnitude. We aim to provide a precise estimate of the CFR of COVID-19 using statistical models at the early stage of the epidemic. Methods: We extracted data from the daily released epidemic report published by the National Health Commission P. R. China from 20 Jan 2020, to 1 March 2020. Competing risk model was used to obtain the cumulative hazards for death, cure, and cure-death hazard ratio. Then the CFR was estimated based on the slope of the last piece in joinpoint regression model, which reflected the most recent trend of the epidemic. Results: As of 1 March 2020, totally 80,369 cases were diagnosed as COVID-19 in China. The CFR of COVID-19 were estimated to be 70.9% (95% CI: 66.8%-75.6%) during Jan 20-Feb 2, 20.2% (18.6%-22.1%) during Feb 3-14, 6.9% (6.4%-7.4%) during Feb 15-23, 1.5% (1.4%-1.6%) during Feb 24-March 1 in Hubei province, and 20.3% (17.0%-25.3%) during Jan 20-28, 1.9% (1.8%-2.1%) during Jan 29-Feb 12, 0.9% (0.8%-1.1%) during Feb 13-18, 0.4% (0.4%-0.5%) during Feb 19-March 1 in other areas of China, respectively. Conclusions: Based on analyses of public data, we found that the CFR in Hubei was much higher than that of other regions in China, over 3 times in all estimation. The CFR would follow a downwards trend based on our estimation from recently released data. Nevertheless, at early stage of outbreak, CFR estimates should be viewed cautiously because of limited data source on true onset and recovery time.</td><td>2020-02-20</td><td>Xing Wang; Zihui Ma; Yi Ning; Chen Chen; Rujin Chen; Qiwen Chen; Heng Zhang; Chunming Li; Yan He; Tao Wang; Cheng Tong; Junqing Wu; Yuyan Li; Handong Ma; Shaodian Zhang; Hongxin Zhao</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20023630</td></tr><tr><td>7odpslba</td><td>fe685aa676e739bd52ba2585a7e5b27c55e2d0d6</td><td>medrxiv</td><td>Evidence for gastrointestinal infection of SARS-CoV-2</td><td>10.1101/2020.02.17.20023721</td><td>null</td><td>null</td><td>medrvix</td><td>The new coronavirus (SARS-CoV-2) outbreak originating from Wuhan, China, poses a threat to global health. While it's evident that the virus invades respiratory tract and transmits from human to human through airway, other viral tropisms and transmission routes remain unknown. We tested viral RNA in stool from 73 SARS-CoV-2-infected hospitalized patients using rRT-PCR. 53.42% of the patients tested positive in stool. 23.29% of the patients remained positive in feces even after the viral RNA decreased to undetectable level in respiratory tract. The viral RNA was also detected in gastrointestinal tissues. Furthermore, gastric, duodenal and rectal epithelia showed positive immunofluorescent staining of viral host receptor ACE2 and viral nucleocapsid protein in a case of SARS-CoV-2 infection. Our results provide evidence for gastrointestinal infection of SARS-CoV-2, highlighting its potential fecal-oral transmission route.</td><td>2020-02-20</td><td>Fei Xiao; Meiwen Tang; Xiaobin Zheng; Chunna Li; Jianzhong He; Zhongsi Hong; Siwen Huang; Zhenyi Zhang; Xianqi Lin; Zhaoxiong Fang; Renxu Lai; Shoudeng Chen; Jing Liu; Jin Huang; Jinyu Xia; Zhonghe Li; Guanmin Jiang; Ye Liu; Xiaofeng Li; Hong Shan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20023721</td></tr><tr><td>q6pz6l2b</td><td>0cad40f50e41144e8b64744251515eca8cfac5f7</td><td>medrxiv</td><td>The reproductive number R0 of COVID-19 based on estimate of a statistical time delay dynamical system</td><td>10.1101/2020.02.17.20023747</td><td>null</td><td>null</td><td>medrvix</td><td>In this paper, we estimate the reproductive number R0 of COVID-19 based on Wallinga and Lipsitch framework {11} and a novel statistical time delay dynamic system. We use the observed data reported in CCDC's paper to estimate distribution of the generation interval of the infection and apply the simulation results from the time delay dynamic system as well as released data from CCDC to fit the growth rate. The conclusion is: Based our Fudan-CCDC model, the growth rate r of COVID-19 is almost in [0.30, 0.32] which is larger than the growth rate 0.1 estimated by CCDC {9}, and the reproductive number R0 of COVID-19 is estimated by 3.25≤ R0 ≤3.4 if we simply use R=1+r*Tc with Tc=7.5, which is bigger than that of SARS. Some evolutions and predictions are listed.</td><td>2020-02-20</td><td>Nian Shao; Jin Cheng; Wenbin Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20023747</td></tr><tr><td>nav88l73</td><td>null</td><td>medrxiv</td><td>COVID-19 in a Designated Infectious Diseases HospitalOutside Hubei Province,China</td><td>10.1101/2020.02.17.20024018</td><td>null</td><td>null</td><td>medrvix</td><td>Background A new type of novel coronavirus infection (COVID-19) occurred in Wuhan, Hubei Province. Previous investigations reported patients in Wuhan city often progressed into severe or critical and had a high mortality rate.The clinical characteristics of affected patients outside the epicenter of Hubei province are less well understood. Methods All confirmed COVID-19 case treated in the Third People's Hospital of Shenzhen,from January 11, 2020 to February 6, 2020, were included in this study. We analyzed the epidemiological and clinical features of these cases to better inform patient management in normal hospital settings. Results Among the 298 confirmed cases, 233(81.5%) had been to Hubei while 42(14%) had not clear epidemiological history. Only 192(64%) cases presented with fever as initial symptom. The lymphocyte count decreased in 38% patients after admission. The number (percent) of cases classified as non-severe and severe was 240(80.6%) and 58(19.4%) respectively. Thirty-two patients (10.7%) needed ICU care. Compared to the non-severe cases, severe cases were associated with older age, underlying diseases, as well as higher levels of CRP, IL-6 and ESR. The median (IRQ) duration of positive viral test were 14(10-19). Slower clearance of virus was associated with higher risk of progression to severe clinical condition. As of February 14, 2020, 66(22.1%) patients were discharged and the overall mortality rate remains 0. Conclusions In a designated hospital outside the Hubei Province, COVID-19 patients were mainly characterized by mild symptoms and could be effectively manage by properly using the existing hospital system.</td><td>2020-02-19</td><td>Qingxian Cai; Deliang Huang; Pengcheng Ou; Hong Yu; Zhibin Zhu; Zhang Xia; Yinan Su; Zhenghua Ma; Yiming Zhang; Zhiwei Li; Qing He; Yang Fu; Lei Liu; Jun Chen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20024018</td></tr><tr><td>s12vwz8m</td><td>dac1b1607ae72b9509ab26367e0d55016e8132a8</td><td>medrxiv</td><td>Epidemic Situation of Novel Coronavirus Pneumonia in China mainland</td><td>10.1101/2020.02.17.20024034</td><td>null</td><td>null</td><td>medrvix</td><td>[Objective] Analyze the occurrence of novel coronavirus pneumonia(NCP) in China mainland, explore the epidemiological rules, and evaluate the effect of prevention and control. [Methods] From December 1, 2019 to February 14, 2020, Analysis of 66,492 confirmed cases of NCP in China mainland. [Results] From December 1, 2019 to February 14, 2020, a total of 66,492 cases of NCP were confirmed in China mainland, a total of 54,406 cases were confirmed in Hubei Province, a total of 37,914 cases were confirmed in Wuhan city. On February 5, 2020, the number of suspected cases of NCP in China mainland reached a maximum of 5,328. Since then, the suspected cases have shown a significant downward trend. On February 4, 2020, the number of confirmed cases has gradually decreased in China mainland since then. On February 12, 2020, the number of confirmed cases of NCP in China mainland increased explosively to 15,152, and then began to decline. From February 3, 2020, except for Hubei Province, In China mainland, the number of newly confirmed cases of NCP has continued to decline; From December 1, 2019 to February 14, 2020, a total of 1,523 cases of NCP deaths in China mainland, a cumulative cure of 8096 cases, and mortality and cure rate were 2.29% (1523/66492) and 12.18% (8096/66492) respectively; Starting from January 27, 2020, The spread index of NCP gradually declined, and the extinction index of NCP rose little by little from January 29, 2020. [Conclusion] Starting from February 5, 2020, the number of suspected cases of the NCP is gradually decrease in China mainland. From February 3, 2020, the number of newly confirmed cases of NCP is continuous decline in China mainland except Hubei Province, This shows that the control of the transmission of NCP has achieved. Judged from the number of confirmed cases of NCP, controlling the outbreak in China mainland is to control the epidemic in Hubei province, and the key to controlling the outbreak in Hubei province is to control the epidemic in Wuhan city; Judging from the gradual increase in the number and cure rate of the NCP, and the gradual decline in the number of deaths and mortality, the trend of outbreak control and treatment is getting better; A decrease of the NCP spread index indicates a slowdown in the spread of the virus, but the cumulative confirmed cases is still increase, so the epidemic will continue for some time. However, the turning point of the epidemic in mainland China has not yet occurred. On February 12, 2020, the turning point of the epidemic in China mainland except Hubei province, has turned up.</td><td>2020-02-18</td><td>Liu youbin; Li bao hong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20024034</td></tr><tr><td>nnkholfe</td><td>541e7d329fa466fe7b961263666386dddec75894</td><td>medrxiv</td><td>A quantitative framework to define the end of an outbreak: application to Ebola Virus Disease</td><td>10.1101/2020.02.17.20024042</td><td>null</td><td>null</td><td>medrvix</td><td>Declaring the end of an outbreak is an important step in controlling infectious disease outbreaks. An objective estimation of the probability of cases arising in the future is important to reduce the risk of post-declaration flare-ups. We developed a simulation-based model to quantify that probability. We tested it on simulated Ebola Virus Disease (EVD) data and found this probability was most sensitive to the instantaneous reproduction number, the reporting rate, and the delay between symptom onset and recovery or death of the last detected case. For EVD, our results suggest that the current WHO criterion of 42 days since the outcome of the last detected case is too short and very sensitive to underreporting. The 90 days of enhanced surveillance period after the end-of-outbreak declaration is therefore crucial to capture potential flare-ups of cases. Hence, we suggest a shift to a preliminary end-of-outbreak declaration after 63 days from the symptom onset day of the last detected case. This should be followed by a 90-day enhanced surveillance, after which the official end-of-outbreak can be declared. This corresponds to less than 5% probability of flare ups in most of the scenarios examined. Our quantitative framework could be adapted to define end-of-outbreak criteria for other infectious diseases.</td><td>2020-02-20</td><td>Bimandra A Djaafara; Natsuko Imai; Esther Hamblion; Benido Impouma; Christl A Donnelly; Anne Cori</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20024042</td></tr><tr><td>2w3bx6p8</td><td>a6d2875c8b70a41ea815511417546a0689816964</td><td>medrxiv</td><td>Estimating number of global importations of COVID-19 from Wuhan, risk of transmission outside mainland China and COVID-19 introduction index between countries outside mainland China</td><td>10.1101/2020.02.17.20024075</td><td>null</td><td>null</td><td>medrvix</td><td>Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China. Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity. Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country. Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.</td><td>2020-02-20</td><td>Haoyang Sun; Borame Lee Dickens; Mark Chen; Alex Richard Cook; Hannah Eleanor Clapham</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20024075</td></tr><tr><td>mlkxcn5k</td><td>f0e75c4697317cbd0e3a5cdd722fe0526595db64</td><td>medrxiv</td><td>Clinical features and progression of acute respiratory distress syndrome in coronavirus disease 2019</td><td>10.1101/2020.02.17.20024166</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) results in a cluster of coronavirus disease 2019 (COVID-19). We reported the clinical characteristics of COVID-19 patients with acute respiratory distress syndrome (ARDS), and further investigated the treatment and progression of ARDS in COVID-19. Methods: This study enrolled 109 patients with COVID-19 admitted to the Central Hospital of Wuhan, a designated hospital in Wuhan, China, from January 2 to February 1, 2020. Patients were followed up to February 12, 2020. The clinical data were collected from the electronic medical records. The differences in the treatment and progression with the time and the severity of ARDS were determined. Findings: Among 109 patients, mean age was 55 years, and 59 patients were male. With a median 15 days (range, 4 to 30 days) follow-up period, 31 patients (28.4%) died, while 78 (71.6%) survived and discharged. Of all patients, 53 (48.6%) developed ARDS. Compared to non-ARDS patients, ARDS patients were elder (mean age, 61 years vs. 49 years), and more likely to have the coexistent conditions, including diabetes (20.8% vs. 1.8%), cerebrovascular disease (11.3% vs. 0%), and chronic kidney disease (15.1% vs. 3.6%). Compared to mild ARDS patients, those with moderate and severe ARDS had higher mortality rates. No significant effect of antivirus, glucocorticoid, or immunoglobulin treatment on survival was observed in patients with ARDS. Interpretation: The mortality rate increased with the severity of ARDS in COVID-19, and the effects of current therapies on the survival for these patients were not satisfactory, which needs more attention from clinicians. Funding: Health and Family Planning Commission of Wuhan Municipality.</td><td>2020-02-20</td><td>Yanli Liu; Wenwu Sun; Jia Li; Liangkai Chen; Yujun Wang; Lijuan Zhang; Li Yu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20024166</td></tr><tr><td>t2x4yfrv</td><td>341fe593ea84b66223d50924bddbba252c37bc4e</td><td>medrxiv</td><td>Tracking and Predicting COVID-19 Epidemic in China Mainland</td><td>10.1101/2020.02.17.20024257</td><td>null</td><td>null</td><td>medrvix</td><td>By proposing a varying coefficient Susceptible-Infected-Removal model (vSIR), we track the epidemic of COVID-19 in 30 provinces in China and 15 cities in Hubei province, the epicenter of the outbreak. It is found that the spread of COVID-19 has been significantly slowing down within the two weeks from January 27 to February 10th with 87.0% and 84.3% reductions in the reproduction number R0 among the 30 provinces and 15 Hubei cities, respectively. This suggests the extreme control measures implemented since January 23, which include cutting off Wuhan and many other cities and towns, a great public awareness and high level of self isolation at home, have contributed to a substantial decline in the reproductivity of the COVID-19 in China. We predict that Hubei province will reach its peak between February 20 and 22, 2020, and if the removal rate can be increased to 0.1, the epidemic outside Hubei province will end in May 2020, and inside Hubei in early June.</td><td>2020-02-20</td><td>Haoxuan Sun; Yumou Qiu; Han Yan; Yaxuan Huang; Yuru Zhu; Song Xi Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.20024257</td></tr><tr><td>t9f87kk0</td><td>a8be6ba53508e988c2c8c153478ff615b0d77687</td><td>biorxiv</td><td>Modulation of metabolic functions through Cas13d-mediated gene knockdown in liver</td><td>10.1101/2020.02.17.945014</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractRNA knockdown in vivo carries significant potential for disease modelings and therapies. Despite the emerging approaches of CRISPR/Cas9-mediated permanent knock out of targeted genes, strategies targeting RNA for disruption are advantageous in the treatment of acquired metabolic disorders when permanent modification of the genome DNA is not appropriate, and RNA virus infection diseases when pathogenic DNA is not available (such as SARS-Cov-2 and MERS infections). Recently, Cas13d, a family of RNA-targeting CRISPR effectors, has been shown to accomplish robust down-regulation of cellular RNAs in mammalian cells in vitro. Among the various Cas13d subtypes, CasRx (RfxCas13d) showed the most potent RNA knockdown efficiency in HEK293T cells. However, the RNA-targeting activity of Cas13d still needs to be verified in vivo. In this study, the CasRx system was demonstrated to efficiently and functionally knock down genes related to metabolism functions, including Pten, Pcsk9 and lncLstr, in mouse hepatocytes. CasRx-mediated simultaneous knockdown of multiple genes was also achieved by sgRNA arrays, providing a useful strategy to modulate complex metabolism networks. Moreover, the AAV (adeno-associated virus)-mediated delivery of CasRx and Pcsk9 sgRNAs into mouse liver successfully decreased serum PCSK9, resulting in significant reduction of serum cholesterol levels. Importantly, CasRx-mediated knockdown of Pcsk9 is reversible and Pcsk9 could be repeatedly down-regulated, providing an effective strategy to reversibly modulate metabolic genes. The present work supplies a successful proof-of-concept trial that suggests efficient and regulatory knockdown of target metabolic genes for a designed metabolism modulation in the liver.</td><td>2020-02-19</td><td>Bingbing He; Wenbo Peng; Jia Huang; Hang Zhang; Yingsi Zhou; Xiali Yang; Jing Liu; Zhijie Li; Chunlong Xu; Mingxing Xue; Hui Yang; Pengyu Huang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.945014</td></tr><tr><td>mw0wb8a8</td><td>null</td><td>biorxiv</td><td>Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins</td><td>10.1101/2020.02.17.951335</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe outbreak of 2019-nCoV in the central Chinese city of Wuhan at the end of 2019 poses unprecedent public health challenges to both China and the rest world1. The new coronavirus shares high sequence identity to SARS-CoV and a newly identified bat coronavirus2. While bats may be the reservoir host for various coronaviruses, whether 2019-nCoV has other hosts is still ambiguous. In this study, one coronavirus isolated from Malayan pangolins showed 100%, 98.2%, 96.7% and 90.4% amino acid identity with 2019-nCoV in the E, M, N and S genes, respectively. In particular, the receptor-binding domain of the S protein of the Pangolin-CoV is virtually identical to that of 2019-nCoV, with one amino acid difference. Comparison of available genomes suggests 2019-nCoV might have originated from the recombination of a Pangolin-CoV-like virus with a Bat-CoV-RaTG13-like virus. Infected pangolins showed clinical signs and histopathological changes, and the circulating antibodies reacted with the S protein of 2019-nCoV. The isolation of a coronavirus that is highly related to 2019-nCoV in the pangolins suggests that these animals have the potential to act as the intermediate host of 2019-nCoV. The newly identified coronavirus in the most-trafficked mammal could represent a continuous threat to public health if wildlife trade is not effectively controlled.</td><td>2020-02-20</td><td>Kangpeng Xiao; Junqiong Zhai; Yaoyu Feng; Niu Zhou; Xu Zhang; Jie-Jian Zou; Na Li; Yaqiong Guo; Xiaobing Li; Xuejuan Shen; Zhipeng Zhang; Fanfan Shu; Wanyi Huang; Yu Li; Ziding Zhang; Rui-Ai Chen; Ya-Jiang Wu; Shi-Ming Peng; Mian Huang; Wei-Jun Xie; Qin-Hui Cai; Fang-Hui Hou; Yahong Liu; Wu Chen; Lihua Xiao; Yongyi Shen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.951335</td></tr><tr><td>sty7myg4</td><td>b705170c7527f5bc6842d90bf443fa9f9ee00bf9</td><td>biorxiv</td><td>Structure of dimeric full-length human ACE2 in complex with B0AT1</td><td>10.1101/2020.02.17.951848</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAngiotensin-converting enzyme 2 (ACE2) is the surface receptor for SARS coronavirus (SARS-CoV), directly interacting with the spike glycoprotein (S protein). ACE2 is also suggested to be the receptor for the new coronavirus (2019-nCoV), which is causing a serious epidemic in China manifested with severe respiratory syndrome. B0AT1 (SLC6A19) is a neutral amino acid transporter whose surface expression in intestinal cells requires ACE2. Here we present the 2.9 Å resolution cryo-EM structure of full-length human ACE2 in complex with B0AT1. The complex, assembled as a dimer of ACE2-B0AT1 heterodimers, exhibits open and closed conformations due to the shifts of the peptidase domains (PDs) of ACE2. A newly resolved Collectrin-like domain (CLD) on ACE2 mediates homo-dimerization. Structural modelling suggests that the ACE2-B0AT1 complex can bind two S proteins simultaneously, providing important clues to the molecular basis for coronavirus recognition and infection.</td><td>2020-02-18</td><td>Renhong Yan; Yuanyuan Zhang; Yaning Li; Lu Xia; Qiang Zhou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.951848</td></tr><tr><td>os758lsd</td><td>null</td><td>biorxiv</td><td>Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</td><td>10.1101/2020.02.17.951939</td><td>null</td><td>null</td><td>biorxiv</td><td>A recombinant adenovirus vaccine against the SARS Coronavirus (SARS-CoV) was constructed, which contains fragments from the S, N, and Orf8 genes. Rhesus Macaques immunized with the recombinant adenovirus generated antigen-specific humoral and cellular response. Furthermore, the vaccine provided significant protection against subsequent live SARS-CoV challenge. In contrast, three out of four monkeys immunized with placebo suffered severe alveolar damage and pulmonary destruction.</td><td>2020-02-21</td><td>Yiyou Chen; Qiang Wei; Ruobing Li; Hong Gao; Hua Zhu; Wei Deng; Linlin Bao; Wei Tong; Zhe Cong; Hong Jiang; Chuan Qin</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.951939</td></tr><tr><td>xhachzby</td><td>cc5e36a8d22f6b1708b1c43764806b1722aa1754</td><td>biorxiv</td><td>X-ray Structure of Main Protease of the Novel Coronavirus SARS-CoV-2 Enables Design of α-Ketoamide Inhibitors</td><td>10.1101/2020.02.17.952879</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA novel coronavirus has been identified as the causative agent of a massive outbreak of atypical pneumonia originating at Wuhan, Hubei province, China. Involved in the formation of the coronavirus replication complex, the viral main protease (Mpro, also called 3CLpro) represents an attractive target for therapy. We determined the crystal structure of the unliganded Mpro at 1.75 Å resolution and used this structure to guide optimization of a series of alpha-ketoamide inhibitors. The main goal of the optimization efforts was improvement of the pharmacokinetic properties of the compounds. We further describe 1.95- and 2.20-Å crystal structures of the complex between the enzyme and the most potent alpha-ketoamide optimized this way. These structures will form the basis for further development of these compounds to antiviral drugs.</td><td>2020-02-20</td><td>Linlin Zhang; Daizong Lin; Xinyuanyuan Sun; Katharina Rox; Rolf Hilgenfeld</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.952879</td></tr><tr><td>p4q64ksa</td><td>1b4ab28a9391d516e1753874c73d2d244104f825</td><td>biorxiv</td><td>Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019</td><td>10.1101/2020.02.17.952895</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells1. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar2, but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1–specific neutralizing antibodies to inhibit SARS-CoV-23. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS)4-6, the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection7,8. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride9,10) or the PBM (SB2035805), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo. Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi-identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed.</td><td>2020-02-21</td><td>Intikhab alam; Allan Kamau; Maxat Kulmanov; Stefan T. Arold; Arnab Pain; Takashi Gojobori; Carlos M. Duarte</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.952895</td></tr><tr><td>hicqy5sj</td><td>null</td><td>biorxiv</td><td>Molecular mechanism of evolution and human infection with the novel coronavirus (2019-nCoV)</td><td>10.1101/2020.02.17.952903</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSince December, 2019, an outbreak of pneumonia caused by the new coronavirus (2019-nCoV) has hit the city of Wuhan in the Hubei Province. With the continuous development of the epidemic, it has become a national public health crisis and calls for urgent antiviral treatments or vaccines. The spike protein on the coronavirus envelope is critical for host cell infection and virus vitality. Previous studies showed that 2019-nCoV is highly homologous to human SARS-CoV and attaches host cells though the binding of the spike receptor binding domain (RBD) domain to the angiotensin-converting enzyme II (ACE2). However, the molecular mechanisms of 2019-nCoV binding to human ACE2 and evolution of 2019-nCoV remain unclear. In this study, we have extensively studied the RBD-ACE2 complex, spike protein, and free RBD systems of 2019-nCoV and SARS-CoV using protein-protein docking and molecular dynamics (MD) simulations. It was shown that the RBD-ACE2 binding free energy for 2019-nCoV is significantly lower than that for SARS-CoV, which is consistent with the fact that 2019-nCoV is much more infectious than SARS-CoV. In addition, the spike protein of 2019-nCoV shows a significantly lower free energy than that of SARS-CoV, suggesting that 2019-nCoV is more stable and able to survive a higher temperature than SARS-CoV. This may also provide insights into the evolution of 2019-nCoV because SARS-like coronaviruses are thought to have originated in bats that are known to have a higher body-temperature than humans. It was also revealed that the RBD of 2019-nCoV is much more flexible especially near the binding site and thus will have a higher entropy penalty upon binding ACE2, compared to the RBD of SARS-CoV. That means that 2019-nCoV will be much more temperature-sensitive in terms of human infection than SARS-CoV. With the rising temperature, 2019-nCoV is expected to decrease its infection ability much faster than SARS-CoV, and get controlled more easily. The present findings are expected to be helpful for the disease prevention and control as well as drug and vaccine development of 2019-nCoV.</td><td>2020-02-21</td><td>Jiahua He; Huanyu Tao; Yumeng Yan; Sheng-You Huang; Yi Xiao</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.17.952903</td></tr><tr><td>ic2h01gm</td><td>6186828135d9e5e7e50f66567505d02e910b3ee2</td><td>medrxiv</td><td>Association between 2019-nCoV transmission and N95 respirator use</td><td>10.1101/2020.02.18.20021881</td><td>null</td><td>null</td><td>medrvix</td><td>2019-nCoV had caused pneumonia outbreak in Wuhan. Existing evidence have confirmed the human-to-human transmission of 2019-nCoV. We retrospectively collected infection data from 2 January to 22 January at six departments from Zhongnan Hospital of Wuhan University. In our study, we found N95 respirators, disinfection and hand washing can help to reduce the risk of 2019-nCoV infection in medical staffs. Our results call for re-emphasizing strict occupational protection code in battling this novel contagious disease. The risk of 2019-nCoV infection was higher in the open area than in the quarantined area. N95 may be more effective for 2019-nCoV infections.</td><td>2020-02-19</td><td>Xinghuan Wang; Zhenyu Pan; Zhenshun Cheng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20021881</td></tr><tr><td>ahxnhutv</td><td>db142fe07544dcf0e454f8a9f39452f0aa92a6ca</td><td>medrxiv</td><td>Kidney impairment is associated with in-hospital death of COVID-19 patients</td><td>10.1101/2020.02.18.20023242</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Information on kidney impairment in patients with coronavirus disease 2019 (COVID-19) is limited. This study aims to assess the prevalence and impact of abnormal urine analysis and kidney dysfunction in hospitalized COVID-19 patients in Wuhan. Method: We conducted a consecutive cohort study of COVID-19 patients admitted in a tertiary teaching hospital with 3 branches following a major outbreak in Wuhan in 2020. Hematuria, proteinuria, serum creatinine concentration and other clinical parameters were extracted from the electronic hospitalization databases and laboratory databases. Incidence rate for acute kidney injury (AKI) was examined during the study period. Association between kidney impairment and in-hospital death was analyzed. Results: We included 710 consecutive COVID19 patients, 89 (12.3%) of whom died in hospital. The median age of the patients was 63 years (inter quartile range, 51-71), including 374 men and 336 women. On admission, 44% of patients have proteinuria hematuria and 26.9% have hematuria, and the prevalence of elevated serum creatinine and blood urea nitrogen were 15.5% and 14.1% respectively. During the study period, AKI occurred in 3.2% patients. Kaplan-Meier analysis demonstrated that patients with kidney impairment have higher risk for in-hospital death. Cox proportional hazard regression confirmed that elevated serum creatinine, elevated urea nitrogen, AKI, proteinuria and hematuria was an independent risk factor for in-hospital death after adjusting for age, sex, disease severity, leukocyte count and lymphocyte count. Conclusion: The prevalence of kidney impairment (hematuria, proteinuria and kidney dysfunction) in hospitalized COVID-19 patients was high. After adjustment for confounders, kidney impairment indicators were associated with higher risk of in-hospital death. Clinicians should increase their awareness of kidney impairment in hospitalized COVID-19 patients.</td><td>2020-02-20</td><td>Yichun Cheng; Ran Luo; Kun Wang; Meng Zhang; Zhixiang Wang; Lei Dong; Junhua Li; Ying Yao; Shuwang Ge; Gang Xu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20023242</td></tr><tr><td>uo7fb10k</td><td>b9548530d954b407b35fb266e2b4192402c982a1</td><td>medrxiv</td><td>Phase adjusted estimation of the number of 2019 novel coronavirus cases in Wuhan, China</td><td>10.1101/2020.02.18.20024281</td><td>null</td><td>null</td><td>medrvix</td><td>An outbreak of clusters of viral pneumonia due to a novel coronavirus (2019-nCoV / SARS-CoV-2) happened in Wuhan, Hubei Province in China in December 2019. Since the outbreak, several groups reported estimated R0 of Coronavirus Disease 2019 (COVID-19) and generated valuable prediction for the early phase of this outbreak. After implementation of strict prevention and control measures in China, new estimation is needed. An infectious disease dynamics SEIR (Susceptible, Exposed, Infectious and Removed) model was applied to estimate the epidemic trend in Wuhan, China under two assumptions of Rt. In the first assumption, Rt was assumed to maintain over 1. The estimated number of infections would continue to increase throughout February without any indication of dropping with Rt = 1.9, 2.6 or 3.1. The number of infections would reach 11,044, 70,258 and 227,989, respectively, by 29 February 2020. In the second assumption, Rt was assumed to gradually decrease at different phases from high level of transmission (Rt = 3.1, 2.6 and 1.9) to below 1 (Rt = 0.9 or 0.5) owing to increasingly implemented public heath intervention. Several phases were divided by the dates when various levels of prevention and control measures were taken in effect in Wuhan. The estimated number of infections would reach the peak in late February, which is 58,077-84,520 or 55,869-81,393. Whether or not the peak of the number of infections would occur in February 2020 may be an important index for evaluating the sufficiency of the current measures taken in China. Regardless of the occurrence of the peak, the currently strict measures in Wuhan should be continuously implemented and necessary strict public health measures should be applied in other locations in China with high number of COVID-19 cases, in order to reduce Rt to an ideal level and control the infection.</td><td>2020-02-23</td><td>Huwen Wang; Zezhou Wang; Yinqiao Dong; Ruijie Chang; Chen Xu; Xiaoyue Yu; Shuxian Zhang; Lhakpa Tsamlag; Meili Shang; Jinyan Huang; Ying Wang; Gang Xu; Tian Shen; Xinxin Zhang; Yong Cai</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024281</td></tr><tr><td>nzynerfu</td><td>36521caf90f471c9da1a4e84f8562440d73ead9a</td><td>medrxiv</td><td>Estimation of the epidemic properties of the 2019 novel coronavirus: A mathematical modeling study</td><td>10.1101/2020.02.18.20024315</td><td>null</td><td>null</td><td>medrvix</td><td>Background The 2019 novel Coronavirus (COVID-19) emerged in Wuhan, China in December 2019 and has been spreading rapidly in China. Decisions about its pandemic threat and the appropriate level of public health response depend heavily on estimates of its basic reproduction number and assessments of interventions conducted in the early stages of the epidemic. Methods We conducted a mathematical modeling study using five independent methods to assess the basic reproduction number (R0) of COVID-19, using data on confirmed cases obtained from the China National Health Commission for the period 10th January to 8th February. We analyzed the data for the period before the closure of Wuhan city (10th January to 23rd January) and the post-closure period (23rd January to 8th February) and for the whole period, to assess both the epidemic risk of the virus and the effectiveness of the closure of Wuhan city on spread of COVID-19. Findings Before the closure of Wuhan city the basic reproduction number of COVID-19 was 4.38 (95% CI: 3.63-5.13), dropping to 3.41 (95% CI: 3.16-3.65) after the closure of Wuhan city. Over the entire epidemic period COVID-19 had a basic reproduction number of 3.39 (95% CI: 3.09-3.70), indicating it has a very high transmissibility. Interpretation COVID-19 is a highly transmissible virus with a very high risk of epidemic outbreak once it emerges in metropolitan areas. The closure of Wuhan city was effective in reducing the severity of the epidemic, but even after closure of the city and the subsequent expansion of that closure to other parts of Hubei the virus remained extremely infectious. Emergency planners in other cities should consider this high infectiousness when considering responses to this virus.</td><td>2020-02-20</td><td>Jinghua Li; Yijing Wang; Stuart Gilmour; Mengying Wang; Daisuke Yoneoka; Ying Wang; Xinyi You; Jing Gu; Chun Hao; Liping Peng; Zhicheng Du; Dong Roman Xu; Yuantao Hao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024315</td></tr><tr><td>yj25zkx6</td><td>4609a2dd3568a7241548b7303e0d836e33d4d5ca</td><td>medrxiv</td><td>Reduction and Functional Exhaustion of T Cells in Patients with Coronavirus Disease 2019 (COVID-19)</td><td>10.1101/2020.02.18.20024364</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND The outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has posed great threat to human health, which has been declared a public health emergency of international concern (PHEIC) by the WHO. T cells play a critical role in antiviral immunity but their numbers and functional state in COVID-19 patients remain largely unclear. METHODS We retrospectively reviewed the counts of total T cells, CD4+, CD8+ T cell subsets, and serum cytokine concentration from inpatient data of 522 patients with laboratory-confirmed COVID-19, admitted into two hospitals in Wuhan from December 2019 to January 2020, and 40 healthy controls, who came to the hospitals for routine physical examination. In addition, the expression of T cell exhaustion markers PD-1 and Tim-3 were measured by flow cytometry in the peripheral blood of 14 COVID-19 cases. RESULTS The number of total T cells, CD4+ and CD8+ T cells were dramatically reduced in COVID-19 patients, especially among elderly patients (≥60 years of age) and in patients requiring Intensive Care Unit (ICU) care. Counts of total T cells, CD8+T cells or CD4+T cells lower than 800/μL, 300/μL, or 400/μL, respectively, are negatively correlated with patient survival. Statistical analysis demonstrated that T cell numbers are negatively correlated to serum IL-6, IL-10 and TNF-α concentration, with patients in decline period showing reduced IL-6, IL-10 and TNF-α concentrations and restored T cell counts. Finally, T cells from COVID-19 patients have significantly higher levels of the exhausted marker PD-1 as compared to health controls. Moreover, increasing PD-1 and Tim-3 expression on T cells could be seen as patients progressed from prodromal to overtly symptomatic stages, further indicative of T cell exhaustion. CONCLUSIONS T cell counts are reduced significantly in COVID-19 patients, and the surviving T cells appear functionally exhausted. Non-ICU patients, with total T cells, CD8+T cells CD4+T cells counts lower than 800/μL, 300/μL, and 400/μL, respectively, may still require aggressive intervention even in the immediate absence of more severe symptoms due to a high risk for further deterioration in condition.</td><td>2020-02-20</td><td>Bo Diao; Chenhui Wang; Yingjun Tan; Xiewan Chen; Ying Liu; Lifeng Ning; Li Chen; Min Li; Yueping Liu; Gang Wang; Zilin Yuan; Zeqing Feng; Yuzhang Wu; Yongwen Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024364</td></tr><tr><td>j0nm444m</td><td>fc7a6b5d1852c5ecce2d20fd0d73d5f957ed7055</td><td>medrxiv</td><td>Effective containment explains sub-exponential growth in confirmed cases of recent COVID-19 outbreak in Mainland China</td><td>10.1101/2020.02.18.20024414</td><td>null</td><td>null</td><td>medrvix</td><td>The recent outbreak of COVID-19 in Mainland China is characterized by a distinctive algebraic, sub-exponential increase of confirmed cases with time during the early phase of the epidemic, contrasting an initial exponential growth expected for an unconstrained outbreak with sufficiently large reproduction rate. Although case counts vary significantly between affected provinces in Mainland China, the scaling law t^μ is surprisingly universal, with a range of exponents μ = 2.1 ± 0.3. The universality of this behavior indicates that, in spite of social, regional, demographical, geographical, and socio-economical heterogeneities of affected Chinese provinces, this outbreak is dominated by fundamental mechanisms that are not captured by standard epidemiological models. We show that the observed scaling law is a direct consequence of containment policies that effectively deplete the susceptible population. To this end we introduce a parsimonious model that captures both, quarantine of symptomatic infected individuals as well as population wide isolation in response to mitigation policies or behavioral changes. For a wide range of parameters, the model reproduces the observed scaling law in confirmed cases and explains the observed exponents. Quantitative fits to empirical data permit the identification of peak times in the number of asymptomatic or oligo-symptomatic, unidentified infected individuals, as well as estimates of local variations in the basic reproduction number. The model implies that the observed scaling law in confirmed cases is a direct signature of effective contaiment strategies and/or systematic behavioral changes that affect a substantial fraction of the susceptible population. These insights may aid the implementation of containment strategies in potential export induced COVID-19 secondary outbreaks elsewhere or similar future outbreaks of other emergent infectious diseases.</td><td>2020-02-20</td><td>Benjamin F Maier; Dirk Brockmann</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024414</td></tr><tr><td>28ddm9um</td><td>ba1c4c0de19352d0b9ed7db6199803b621f0a8f2</td><td>medrxiv</td><td>Psychological responses, behavioral changes and public perceptions during the early phase of the COVID-19 outbreak in China: a population based cross-sectional survey</td><td>10.1101/2020.02.18.20024448</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To investigate psychological and behavioral responses to the threat of SARS-CoV-2 infections and their associations with public perceptions in China Design: Cross sectional population-based telephone survey via random digital dialing between 1 and 10 February, 2020 Setting: Wuhan (the epicentre and quarantined city), and Shanghai (a typical major city with close transportation link with Wuhan) Participants: Random sample of 510 residents in Wuhan and 501 residents in Shanghai aged above 18 Main outcome measures: Anxiety (measured by the 7-item generalized anxiety disorder [GAD-7] scale), recommended and avoidance behaviors (engaged in all six behaviors such as increasing surface cleaning and reducing going out). Results: The prevalence rates of moderate or severe anxiety (score ≥10 on GAD-7) were 32.7% (n=167) among Wuhan participants and 20.4% (n=102) among Shanghai participants. 78.6% (n=401) of Wuhan participants and 63.9% (n=320) of Shanghai participants had carried out all six precautionary behaviors. For both measures, Wuhan participants were more responsive to the outbreak (p<0.001). Controlling for personal characteristics, logistic regression results suggested that risks of moderate or severe anxiety were positively associated with perceived susceptibility (odds ratio 1.6, 95% confidence interval 1.3-1.8) and severity of the disease (1.6, 1.4-1.9) and confusion about information reliability (1.6, 1.5-1.9). Having confidence in taking measures to protect oneself against the disease was associated with a lower risk (0.6, 0.5-0.7). The strongest predictor of behavioral change was perceived severity (1.2, 1.1-1.4), followed by confusion about information reliability (1.1, 1.0-1.3). Conclusions: Psychological and behavioral responses to COVID-19 have been dramatic during the rising phase of the outbreak. Our results support efforts for timely dissemination of accurate and reliable information to address the high anxiety level.</td><td>2020-02-20</td><td>Mengcen Qian; Qianhui Wu; Peng Wu; Zhiyuan Hou; Yuxia Liang; Benjamin J Cowling; Hongjie Yu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024448</td></tr><tr><td>od8s0zhm</td><td>537009c0181c3269f40753f75ddca4d17ae8abae</td><td>medrxiv</td><td>Estimating the cure rate and case fatality rate of the ongoing epidemic COVID-19</td><td>10.1101/2020.02.18.20024513</td><td>null</td><td>null</td><td>medrvix</td><td>The epidemic caused by the novel coronavirus COVID-19 in Wuhan at the end of 2019 has become an urgent public event of worldwide concern. However, due to the changing data of the epidemic, there is no scientific estimate of the cure rate and case fatality rate of the epidemic. This study proposes a method to estimate the cure rate and case fatality rate of COVID-19. The ratio of cumulative discharges on a given day to the sum of cumulative discharges on a given day and cumulative deaths before j days is used to estimate the cure rate. Moreover, the case fatality ratio can also be estimated. After simulation calculations, j is statistically appropriate when it is 8-10, and it is also clinically appropriate. When j is 9, based on the available data, it is inferred that the cure rate of this epidemic is about 93% and the case fatality rate is about 7%. This method of estimating the cure rate can be used to evaluate the effectiveness of treatment in different medical schemes and different regions, and has great value and significance for decision-making in the epidemic.</td><td>2020-02-20</td><td>Ying Diao; Xiaoyun Liu; Tao Wang; Xiaofei Zeng; Chen Dong; Changlong Zhou; Yuanming Zhang; Xuan She; Dingfu Liu; Zhongli Hu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024513</td></tr><tr><td>aoi4iqkf</td><td>9bf1677e5f822110e736076a4ba6960a80f2f348</td><td>medrxiv</td><td>Clinical characteristics of 50466 patients with 2019-nCoV infection</td><td>10.1101/2020.02.18.20024539</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: We aim to summarize reliable evidences of evidence-based medicine for the treatment and prevention of the 2019 novel coronavirus (2019-nCoV) by analyzing all the published studies on the clinical characteristics of patients with 2019-nCoV. Methods: PubMed, Cochrane Library, Embase, and other databases were searched. Several studies on the clinical characteristics of 2019-nCoV infection were collected for Meta-analysis. Results: Ten studies were included in Meta-analysis, including a total number of 50466 patients with 2019-nCoV infection. Meta-analysis shows that, among these patients, the incidence of fever was 89.1%, the incidence of cough was 72.2%, and the incidence of muscle soreness or fatigue was 42.5%. The incidence of acute respiratory distress syndrome (ARDS) was 14.8%, the incidence of abnormal chest computer tomography (CT) was 96.6%, the percentage of severe cases in all infected cases was 18.1%, and the case fatality rate of patients with 2019-nCoV infection was 4.3%. Conclusion: Fever and cough are the most common symptoms in patients with 2019-nCoV infection, and most of these patients have abnormal chest CT examination. Several people have muscle soreness or fatigue as well as ARDS. Diarrhea, hemoptysis, headache, sore throat, shock, and other symptoms only occur in a small number of patients. The case fatality rate of patients with 2019-nCoV infection is lower than that of Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS).</td><td>2020-02-23</td><td>Pengfei Sun; Shuyan Qie; Zongjan Liu; Jizhen Ren; Jianing Jianing Xi</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024539</td></tr><tr><td>tc8eru1w</td><td>672ca9b6e50e26c49fc7be2ef3b4f685badf9db1</td><td>medrxiv</td><td>Modeling and Prediction of the 2019 Coronavirus Disease Spreading in China Incorporating Human Migration Data</td><td>10.1101/2020.02.18.20024570</td><td>null</td><td>null</td><td>medrvix</td><td>This study integrates the daily intercity migration data with the classic Susceptible-Exposed-Infected-Removed (SEIR) model to construct a new model suitable for describing the dynamics of epidemic spreading of Coronavirus Disease 2019 (COVID-19) in China. Daily intercity migration data for 367 cities in China are collected from Baidu Migration, a mobile-app based human migration tracking data system. Historical data of infected, recovered and death cases from official source are used for model fitting. The set of model parameters obtained from best data fitting using a constrained nonlinear optimization procedure is used for estimation of the dynamics of epidemic spreading in the coming weeks. Our results show that the number of infections in most cities in China will peak between mid February to early March 2020, with about 0.8%, less than 0.1% and less than 0.01% of the population eventually infected in Wuhan, Hubei Province and the rest of China, respectively.</td><td>2020-02-20</td><td>Choujun Zhan; Chi K. Tse; Yuxia Fu; Zhikang Lai; Haijun Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024570</td></tr><tr><td>toizrp89</td><td>a09c1ad0f05bbcc256cd667ad7411e6690134370</td><td>medrxiv</td><td>Association of Population Migration and Coronavirus Disease 2019 Epidemic Control</td><td>10.1101/2020.02.18.20024661</td><td>null</td><td>null</td><td>medrvix</td><td>Background and Objective To analyze the impact of different patterns of migration flow in two cities, Hefei and Shenzhen, on the epidemic and disease control of Coronavirus Disease 2019 (COVID-19), in order to provide insight for making differentiated controlling policies. Methods We collected demographic and epidemiological information of confirmed COVID-19 cases in Hefei and Shenzhen between January 19 and February 11, 2020, from data officially published by the provincial and municipal Centers for Disease Control and Prevention (CDC). From these data we calculated basic reproduction number R0 to reflect the rate of spread of COVID-19 in these cities. Aggregated data of population migration during the same period was extracted from Baidu Migration. The change of R0 in the two cites were analyzed and compared. Spearman correlation analysis between R0 and population inflow from epidemic focus were performed. Results A total of 157 confirmed cases was identified in Hefei by 24:00 February 11, 2020, with an average age of 44.4±15.6 years, 74 female (47.1%) and 386 confirmed cases were identified in Shenzhen, with an average age of 45.15±17.99 years, 202 female (52.3%). Significant difference in the proportion of imported cases between the two cities was observed (Hefei vs Shenzhen, 24.2% vs 74.9%, p=0.000). Before January 31 2020, during the initial stage of the Level 1 Response to Major Public Health Emergencies, there was no significant association observed in Shenzhen between R0 and the proportion of population inflow from the epidemic focus (P =0.260, r=-0.452); meanwhile in Hefei, such association was strong (P =0.000, r=1.0). However, after the initial stage of response, the situation reversed. A weak association was observed in Shenzhen between be R0 and the proportion of population inflow from the epidemic focus (P=0.073, r=0.536) but not in Hefei (P =0.498, r=0.217). Conclusion Following Level 1 Response, consistent decline of R0 of COVID-19 was observed in both Hefei and Shenzhen. Different patterns of disease spread were observed in the two cities, driven by different patterns of population migration. This indicated that population migration should be taken into consideration when we set controlling policy of a novel infectious disease.</td><td>2020-02-20</td><td>Yu Ding; Sihui Luo; Xueying Zheng; Ping Ling; Tong Yue; Zhirong Liu; Jianping Weng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.20024661</td></tr><tr><td>2inlyd0t</td><td>cb3974f33a1d54369d630e62052d603148bb12e7</td><td>biorxiv</td><td>Are pangolins the intermediate host of the 2019 novel coronavirus (2019-nCoV) ?</td><td>10.1101/2020.02.18.954628</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe outbreak of 2019-nCoV pneumonia (COVID-19) in the city of Wuhan, China has resulted in more than 70,000 laboratory confirmed cases, and recent studies showed that 2019-nCoV (SARS-CoV-2) could be of bat origin but involve other potential intermediate hosts. In this study, we assembled the genomes of coronaviruses identified in sick pangolins. The molecular and phylogenetic analyses showed that pangolin Coronaviruses (pangolin-CoV) are genetically related to both the 2019-nCoV and bat Coronaviruses but do not support the 2019-nCoV arose directly from the pangolin-CoV. Our study also suggested that pangolin be natural host of Betacoronavirus, with a potential to infect humans. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. Conservation of wildlife and limits of the exposures of humans to wildlife will be important to minimize the spillover risks of coronaviruses from wild animals to humans.</td><td>2020-02-20</td><td>Ping Liu; Jing-Zhe Jiang; Xiu-Feng Wan; Yan Hua; Xiaohu Wang; Fanghui Hou; Jing Chen; Jiejian Zou; Jinping Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.954628</td></tr><tr><td>ukh84gvu</td><td>c2af4db8591a8ef84229eb08359c0a50efa24544</td><td>biorxiv</td><td>Structure and immune recognition of the porcine epidemic diarrhea virus spike protein</td><td>10.1101/2020.02.18.955195</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractPorcine epidemic diarrhea virus is an alphacoronavirus responsible for significant morbidity and mortality in pigs. A key determinant of viral tropism and entry, the PEDV spike protein is a key target for the host antibody response and a good candidate for a protein-based vaccine immunogen. We used electron microscopy to evaluate the PEDV spike structure, as well as pig polyclonal antibody responses to viral infection. The structure of the PEDV spike reveals a configuration similar to that of HuCoV-NL63. Several PEDV protein-protein interfaces are mediated by non-protein components including a glycan at Asn264 and two bound palmitoleic acid molecules. The polyclonal antibody response to PEDV infection shows a dominance of epitopes in the S1 region. This structural and immune characterization provides new insights into coronavirus spike stability determinants and explores the immune landscape of viral spike proteins.</td><td>2020-02-19</td><td>Robert N. Kirchdoerfer; Mahesh Bhandari; Olnita Martini; Leigh M. Sewall; Sandhya Bangaru; Kyoung-Jin Yoon; Andrew B. Ward</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.955195</td></tr><tr><td>deu7lrxq</td><td>null</td><td>biorxiv</td><td>CasRx-mediated RNA targeting prevents choroidal neovascularization in a mouse model of age-related macular degeneration</td><td>10.1101/2020.02.18.955286</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe smallest Cas13 family protein, CasRx, has a high cleavage activity and targeting specificity, offering attractive opportunity for therapeutic applications. Here we report that delivery of CasRx by adeno-associated virus via intravitreal injection could efficiently knockdown Vegfa transcripts and significantly reduce the area of laser-induced choroidal neovascularization in a mouse model of age-related macular degeneration. Thus, RNA-targeting CRISPR system could be used for in vivo gene therapy.</td><td>2020-02-20</td><td>Changyang Zhou; Xinde Hu; Cheng Tang; Wenjia Liu; Shaoran Wang; Yingsi Zhou; Qimeng Zhao; Qiyu Bo; Linyu Shi; Xiaodong Sun; Haibo Zhou; Hui Yang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.18.955286</td></tr><tr><td>chhemr7s</td><td>null</td><td>medrxiv</td><td>Early Phylogenetic Estimate Of The Effective Reproduction Number Of 2019-nCoV</td><td>10.1101/2020.02.19.20024851</td><td>null</td><td>null</td><td>medrvix</td><td>To reconstruct the evolutionary dynamics of the 2019 novel coronavirus, 52 2019−nCOV genomes available on 04 February 2020 at GISAID were analysed. The two models used to estimate the reproduction number (coalescent−based exponential growth and a birth−death skyline method) indicated an estimated mean evolutionary rate of 7.8 x 10−4 subs/site/year (range 1.1x10−4−15x10−4). The estimated R value was 2.6 (range 2.1−5.1), and increased from 0.8 to 2.4 in December 2019. The estimated mean doubling time of the epidemic was between 3.6 and 4.1 days. This study proves the usefulness of phylogeny in supporting the surveillance of emerging new infections even as the epidemic is growing.</td><td>2020-02-23</td><td>Alessia Lai; Annalisa Bergna; Carla Acciarri; Massimo Galli; Gianguglielmo Zehender</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20024851</td></tr><tr><td>5ge7ozpd</td><td>null</td><td>medrxiv</td><td>Comparative study of the lymphocyte change between COVID-19 and non-COVID-19 pneumonia cases suggesting uncontrolled inflammation might not be the main reason of tissue injury</td><td>10.1101/2020.02.19.20024885</td><td>null</td><td>null</td><td>medrvix</td><td>The authors have withdrawn this manuscript because the statistical methods in our manuscript need to be modified and we are going to improve the statistical methods and try to give more precise model. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.</td><td>2020-02-23</td><td>Yishan Zheng; Zhen Huang; Guoping Ying; Xia Zhang; Wei Ye; Zhiliang Hu; Chunmei Hu; Hongxia Wei; Yi Zeng; Yun Chi; Cong Cheng; Feishen Lin; Hu Lu; Lingyan Xiao; Yan Song; Chunming Wang; Yongxiang Yi; Lei Dong</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20024885</td></tr><tr><td>fmymklz6</td><td>f3ff1ecae96700f41b83d2a034a3a959428388b0</td><td>medrxiv</td><td>The cross-sectional study of hospitalized coronavirus disease 2019 patients in Xiangyang, Hubei province</td><td>10.1101/2020.02.19.20025023</td><td>null</td><td>null</td><td>medrvix</td><td>Summary Objective To describe the epidemiological and clinical characteristics of the Coronavirus Disease 2019 (COVID-19) hospitalized patients and to offer suggestions to the urgent needs of COVID-19 prevention, diagnosis and treatment. Methods We included 102 confirmed COVID-19 cases hospitalized in Xiangyang No.1 peoples hospital, Hubei, China until Feb 9th, 2020. Demographic data, laboratory findings and chest computed tomographic (CT) images were obtained and analyzed. Findings All cases were confirmed by real-time RT-PCR, including 52 males and 50 females with a mean age of 50.38 years (SD 16.86). Incubation time ranged from one to twenty days with a mean period of 8.09 days (SD 4.99). Fever (86[84.3%] of 102 patients), cough (58[57%]), fatigue (28[27%]), shortness of breath (24[23%]), diarrhea (15[15%]), expectoration (13[12%]), inappetence (11[10%]) were common clinical manifestations. We observed a decreased blood leukocyte count and lymphopenia in 21 (20.6%) and 56 (54.9%) patients, respectively. There were 66 (68%) of 97 patients with elevated C-reactive protein levels and 49 (57.6%) of 85 with increased erythrocytes sedimentation rate. Higher levels of procalcitonin and ferritin were observed in 19 (25.3%) of 75 and 12 (92.3%) of 13 patients, respectively. Eight patients were admitted to intensive care unit (ICU), six developed respiratory failure, three had multiple organ failure and three died. The cumulative positivity rate over three rounds of real-time RT-PCR was 96%. One-hundred patients were found with typical radiological abnormalities in two rounds of chest CT scans, indicating a 98% consistency with real-time RT-PCR results. Interpretation Most COVID-19 patients in Xiangyang were secondary cases without sex difference, and the rate of severe cases and death was low. Middle-to-old-age individuals were more susceptible to the virus infection and the subsequent development of severe/fatal consequences. The average incubation period was longer among our patients. We recommend prolonging the quarantine period to three weeks. Three times real-time RT-PCR plus two times CT scans is a practical clinical diagnosis strategy at present and should be used to increase the accuracy of diagnosis, thereby controlling the source of infection more effectively. Key Words SARS-CoV-2; COVID-19; epidemiological and clinical features; diagnosis</td><td>2020-02-23</td><td>Jinwei Ai; Junwen Chen; Yong Wang; Xiaoyun Liu; Wufeng Fan; Gaojing Qu; Meiling Zhang; Shengduo Polo Pei; Bowen Tang; Shuai Yuan; Yang Li; Lisha Wang; Guoxin Huang; Bin Pei</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025023</td></tr><tr><td>dbzrd23n</td><td>eb8ac60527db35b10881cb4fd86b8a6e21983d02</td><td>medrxiv</td><td>A descriptive study of the impact of diseases control and prevention on the epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai, lessons learned for metropolis epidemics prevention</td><td>10.1101/2020.02.19.20025031</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To describe and evaluate the impact of diseases control and prevention on epidemics dynamics and clinical features of SARS-CoV-2 outbreak in Shanghai. Design: A retrospective descriptive study Setting: China Participants: Epidemiology information was collected from publicly accessible database. 265 patients admitted to Shanghai Public Health Center with confirmed COVID-19 were enrolled for clinical features analysis. Main outcome measure: Prevention and control measures taken by Shanghai government, epidemiological, demographic, clinical, laboratory and radiology data were collected. Weibull distribution, Chi-square test, Fisher's exact test, t test or Mann-Whitney U test were used in statistical analysis. Results: COVID-19 transmission rate within Shanghai had reduced over 99% than previous speculated, and the exponential growth has been stopped so far. Epidemic was characterized by the first stage mainly composed of imported cases and the second stage where >50% of cases were local. The incubation period was 6.4 (95% CI 5.3 to 7.6) days and the mean onset-admission interval was 5.5 days (95% CI, 5.1 to 5.9). Median time for COVID-19 progressed to severe diseases were 8.5 days (IQR: 4.8-11.0 days). By February 11th, proportion of patients being mild, moderate, severe and critically ill were 1.9%(5/265), 89.8%(238/265), 3.8%(10/265), 4.5%(12/265), respectively; 47 people in our cohort were discharged, and 1 patient died. Conclusion: Strict controlling of the transmission rate at the early stage of an epidemic in metropolis can quickly prohibit the spread of the diseases. Controlling local clusters is the key to prevent outbreaks from imported cases. Most COVID-19 severe cases progressed within 14 days of disease onset. Multiple systemic laboratory abnormalities had been observed before significant respiratory dysfunction. Keyword: COVID-19, SARS-CoV-2, epidemics dynamics, diseases control, clinical features</td><td>2020-02-23</td><td>Hongzhou Lu; Jingwen Ai; Yinzhong Shen; Yang Li; Tao Li; Xian Zhou; Haocheng Zhang; Qiran Zhang; Yun Ling; Sheng Wang; Hongping Qu; Yuan Gao; Yingchuan Li; Kanglong Yu; Duming Zhu; Hecheng Zhu; Rui Tian; Mei Zeng; Qiang Li; Yuanlin Song; Xiangyang Li; Jinfu Xu; Jie Xu; Enqiang Mao; Bijie Hu; Xin Li; Lei Zhu; Wenhong Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025031</td></tr><tr><td>hpid3fzc</td><td>null</td><td>medrxiv</td><td>Trends and prediction in daily incidence of novel coronavirus infection in China, Hubei Province and Wuhan City: an application of Farr law</td><td>10.1101/2020.02.19.20025148</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The recent outbreak of novel coronavirus (2019-nCoV) has infected tens of thousands of patients in China. Studies have forecasted future trends of the incidence of 2019-nCoV infection, but appeared unsuccessful. Farr law is a classic epidemiology theory/practice for predicting epidemics. Therefore, we used and validated a model based on Farr law to predict the daily-incidence of 2019-nCoV infection in China and 2 regions of high-incidence. Methods: We extracted the 2019-nCoV incidence data of China, Hubei Province and Wuhan City from websites of the Chinese and Hubei health commissions. A model based on Farr law was developed using the data available on Feb. 8, 2020, and used to predict daily-incidence of 2019-nCoV infection in China, Hubei Province and Wuhan City afterward. Results: We observed 50,995 (37001 on or before Feb. 8) incident cases in China from January 16 to February 15, 2020. The daily-incidence has peaked in China, Hubei Providence and Wuhan City, but with different downward slopes. If no major changes occur, our model shows that the daily-incidence of 2019-nCoV will drop to single-digit by February 25 for China and Hubei Province, but by March 8 for Wuhan city. However, predicted 75% confidence intervals of daily-incidence in all 3 regions of interest had an upward trend. The predicted trends overall match the prospectively-collected data, confirming usefulness of these models. Conclusions: This study shows the daily-incidence of 2019-nCoV in China, Hubei Province and Wuhan City has reached the peak and was decreasing. However, there is a possibility of upward trend.</td><td>2020-02-23</td><td>Jie Xu; Yajiao Cheng; Xiaoling Yuan; Wei V. Li; Lanjing Zhang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025148</td></tr><tr><td>6kpgt70s</td><td>712f1a74a5083d690832eed5fd9ff16a04434d8d</td><td>medrxiv</td><td>Rapid Detection of Novel Coronavirus (COVID-19) by Reverse Transcription-Loop-Mediated Isothermal Amplification</td><td>10.1101/2020.02.19.20025155</td><td>null</td><td>null</td><td>medrvix</td><td>Novel Corona virus (COVID-19 or 2019-nCoV) is an emerging global health concern that requires a rapid diagnostic test. Quantitative reverse transcription PCR (qRT-PCR) is currently the standard for COVID-19 detection; however, Reverse Transcription Loop-Mediated Isothermal Amplification (RT-LAMP) may allow for faster and cheaper field based testing at point-of-risk. The objective of this study was to develop a rapid screening diagnostic test that could be completed in under 30 minutes. Simulated patient samples were generated by spiking serum, urine, saliva, oropharyngeal swabs, and nasopharyngeal swabs with a portion of the COVID-19 nucleic sequence. The samples were tested using RT-LAMP as well as by conventional qRT-PCR. Specificity of the RT-LAMP was evaluated by also testing against other related coronaviruses. RT-LAMP specifically detected COVID-19 in simulated patient samples. This test was performed in under 30 minutes. This approach could be used for monitoring of exposed individuals or potentially aid with screening efforts in the field and potential ports of entry.</td><td>2020-02-24</td><td>Laura E Lamb; Sarah N Bartolone; Elijah Ward; Michael B Chancellor</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025155</td></tr><tr><td>02qvv8le</td><td>null</td><td>medrxiv</td><td>Estimating the risk of 2019 Novel Coronavirus death during the course of the outbreak in China, 2020</td><td>10.1101/2020.02.19.20025163</td><td>null</td><td>null</td><td>medrvix</td><td>Since the first case of Novel Coronavirus (2019-nCov) was identified in December 2019 in Wuhan City, China, the number of cases continues to grow across China and multiple cases have been exported to other countries. The cumulative number of reported deaths is at 637 as of February 7, 2020. Here we statistically estimated the time-delay adjusted death risk for Wuhan as well as for China excluding Wuhan to interpret the current severity of the epidemic in China. We found that the latest estimates of the death risk in Wuhan could be as high as 20% in the epicenter of the epidemic whereas we estimate it ~1% in the relatively mildly-affected areas. Because the elevated death risk estimates are likely associated with a breakdown of the medical/health system, enhanced public health interventions including social distancing and movement restrictions should be effectively implemented to bring the epidemic under control.</td><td>2020-02-23</td><td>Kenji Mizumoto; Gerardo Chowell</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025163</td></tr><tr><td>qg2o0ug4</td><td>3c94f582610453b6c1389c3a5d6ad57bcf0ba033</td><td>medrxiv</td><td>Clinical characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China</td><td>10.1101/2020.02.19.20025239</td><td>null</td><td>null</td><td>medrvix</td><td>Summary Background The pneumonia caused by the 2019 novel coronavirus (SARS-CoV-2) is a highly infectious disease, which was occurred in Wuhan, Hubei Province, China in December 2019. As of February 13, 2020, a total of 59883 cases of COVID-19 in China have been confirmed and 1368 patients have died from the disease. However, the clinical characteristics of the dyed patients were still not clearly clarified. This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality. Methods The clinical records, laboratory findings and radiologic assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data. Findings The mean age of the dead was 71.48 years, the average course of the disease was 10.56 days, all patients eventually died of respiratory failure. All of those who died had underlying diseases, the most common of which was hypertension (16/25, 64%), followed by diabetes (10/25, 40%), heart diseases (8/25, 32%), kidney diseases (5/25, 20%), cerebral infarction (4/25, 16%), chronic obstructive pulmonary disease (COPD, 2/25, 8%), malignant tumors (2/25, 8%) and acute pancreatitis (1/25, 4%). The most common organ damage outside the lungs was the heart, followed by kidney and liver. In the patients' last examination before death, white blood cell and neutrophil counts were elevated in 17 patients (17/25, 68%) and 18 patients (18/25, 72%), lymphocyte counts were decreased in 22 patients (22/25, 88%). Most patients' PCT, CRP and SAA levels were elevated, the percentages were 90.5% (19/21), 85% (19/20) and 100% (21/21) respectively. The levels of the last test of neutrophils (15/16, 93.8%), PCT (11/11, 100%), CRP (11/13, 84.6%), cTnI (8/9, 88.9%), D-Dimer (11/12, 91.6%) and LDH (9/9, 100%) were increased as compared to the first test, while the levels of lymphocytes were decreased (14/16, 87.5%). Interpretation The age and underlying diseases (hypertension, diabetes, etc.) were the most important risk factors for death of COVID-19 pneumonia. Bacterial infections may play an important role in promoting the death of patients. Malnutrition was common to severe patients. Multiple organ dysfunction can be observed, the most common organ damage was lung, followed by heart, kidney and liver. The rising of neutrophils, SAA, PCT, CRP, cTnI, D-Dimer and LDH levels can be used as indicators of disease progression, as well as the decline of lymphocytes counts.</td><td>2020-02-25</td><td>Xun Li; Luwen Wang; Shaonan Yan; Fan Yang; Longkui Xiang; Jiling Zhu; Bo Shen; Zuojiong Gong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025239</td></tr><tr><td>goqgv2cc</td><td>b463aa228141ea5c920755e57bf9c49a55adf141</td><td>medrxiv</td><td>A Note on COVID-19 Diagnosis Number Prediction Model in China</td><td>10.1101/2020.02.19.20025262</td><td>null</td><td>null</td><td>medrvix</td><td>Importance: To predict the diagnosed COVID-19 patients and the trend of the epidemic in China. It may give the public some scientific information to ease the fear of the epidemic. Objective: In December 2019, pneumonia infected with the novel coronavirus burst in Wuhan, China. We aimed to use a mathematical model to predict number of diagnosed patients in future to ease anxiety on the emergent situation. Design: According to all diagnosis number from WHO website and combining with the transmission mode of infectious diseases, the mathematical model was fitted to predict future trend of outbreak. Setting: Our model was based on the epidemic situation in China, which could provide referential significance for disease prediction in other countries, and provide clues for prevention and intervention of relevant health authorities. Participants: In this retrospective, all diagnosis number from Jan 21 to Feb 10, 2020 reported from China was included and downloaded from WHO website. Main Outcome(s) and Measure(s): We develop a simple but accurate formula to predict the next day diagnosis number:N_i/N_(i-1) =[(N_(i-1)/N_(i-2) )]^α，where Ni is the total diagnosed patient till the ith day, and α was estimated as 0.904 at Feb 10. Results: Based on this model, it is predicted that the rate of disease infection will decrease exponentially. The total number of infected people is limited; thus, the disease will have limited impact. However, new diagnosis will last to March. Conclusions and Relevance: Through the establishment of our model, we can better predict the trend of the epidemic in China.</td><td>2020-02-23</td><td>Yi Li; Xianhong Yin; Meng Liang; Xiaoyu Liu; Meng Hao; Yi Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025262</td></tr><tr><td>368pz09b</td><td>1e45cd5feb7928bbc8bcd524d9c6b7adeae08a6b</td><td>medrxiv</td><td>SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</td><td>10.1101/2020.02.19.20025288</td><td>null</td><td>null</td><td>medrvix</td><td>Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.</td><td>2020-02-23</td><td>Lunwen Wang; Xun Li; Hui Chen; Shaonan Yan; Yan Li; Dong Li; Zuojiong Gong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025288</td></tr><tr><td>j0ufth5d</td><td>4e2bc97a6164191ba3d53abfa49ccd973aa80326</td><td>medrxiv</td><td>Early Prediction of Disease Progression in 2019 Novel Coronavirus Pneumonia Patients Outside Wuhan with CT and Clinical Characteristics</td><td>10.1101/2020.02.19.20025296</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To determine the predictive value of CT and clinical characteristics for short-term disease progression in patients with 2019 novel coronavirus pneumonia (NCP). Materials and Methods: 224 patients with confirmed 2019 novel coronavirus (COVID-19) infection outside Wuhan who had chest CT examinations were retrospectively screened. Clinical data were obtained from electronic medical records. CT images were reviewed and scored for lesion distribution, lobe and segment involvement, ground-glass opacities, consolidation, and interstitial thickening. All included patients with moderate NCP were observed for at least 14 days from admission to determine whether they exacerbated to severe NCP (progressive group) or not (stable group). CT and clinical characteristics between the two groups were compared, and multivariate logistic regression and sensitivity analyses were performed to identify the risk factors for developing severe NCP. Results: A total of 141 patients with moderate NCP were included, of which 15 (10.6%) patients developed severe NCP during hospitalization and assigned to the progressive group. Multivariate logistic regression analysis showed that higher neutrophil-to-lymphocyte ratio (NLR) (odds ratio [OR] and 95% confidence interval [CI], 1.26 [1.04-1.53]; P = 0.018) and CT severity score (OR and 95% CI, 1.25 [1.08-1.46]; P = 0.004) on admission were independent predictors for progression to severe NCP, and sensitivity analysis confirmed the consistent results in nonimported patients but not in imported patients. However, no significant difference in lung involvement was found on CT between imported and nonimported patients (all P > 0.05). Patients who were admitted more than 4 days from symptom onset tended to have more severe lung involvement. Spearman correlation analysis showed the close association between CT severity score and inflammatory indexes (r = 0.17~0.47, all P < 0.05). Conclusion: CT severity score was associated with inflammatory levels and higher NLR and CT severity score on admission were independent risk factors for short-term progression in patients with NCP outside Wuhan. Furthermore, early admission and surveillance by CT should be recommended to improve clinical outcomes.</td><td>2020-02-23</td><td>Zhichao Feng; Qizhi Yu; Shanhu Yao; Lei Luo; Junhong Duan; Zhimin Yan; Min Yang; Hongpei Tan; Mengtian Ma; Ting Li; Dali Yi; Ze Mi; Huafei Zhao; Yi Jiang; Zhenhu He; Huiling Li; Wei Nie; Yin Liu; Jing Zhao; Muqing Luo; Xuanhui Liu; Pengfei Rong; Wei Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025296</td></tr><tr><td>8o3l3rsf</td><td>null</td><td>medrxiv</td><td>Effectiveness of control strategies for Coronavirus Disease 2019: a SEIR dynamic modeling study</td><td>10.1101/2020.02.19.20025387</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since its first cases occurrence in Wuhan, China, the Coronavirus Disease 2019 (COVID-19) has been spreading rapidly to other provinces and neighboring countries. A series of intervention strategies have been implemented, but didn't stop its spread. Methods: Two mathematical models have been developed to simulate the current epidemic situation in the city of Wuhan and in other parts of China. Special considerations were given to the mobility of people for the estimation and forecast the number of asymptomatic infections, symptomatic infections, and the infections of super-spreading events (Isse). Findings: The basic reproductive number (R0) was calculated for the period between 18 January 2020 and 16 February 2020: R0 declined from 5.75 to 1.69 in Wuhan and from 6.22 to 1.67 in the entire country (not including the Wuhan area). At the same time, Wuhan is estimated to observe a peak in the number of confirmed cases around 6 February 2020. The number of infected individuals in the entire country (not including the Wuhan area) peaked around February 3. The results also show that the peak of new asymptomatic cases per day in Wuhan occurred on February 6, and the peak of new symptomatic infections have occurred on February 3. Concurrently, while the number of confirmed cases nationwide would continue to decline, the number of real-time COVID-19 inpatients in Wuhan has reached a peak of 13,030 on February 14 before it decreases. The model further shows that the COVID-19 cases will gradually wane by the end of April 2020, both in Wuhan and the other parts of China. The number of confirmed cases would reach the single digit on March 27 in Wuhan and March 19 in the entire country. The five cities with top risk index in China with the exclusion of Wuhan are: Huanggang, Xiaogan, Jingzhou, Chongqing, and Xiangyang city. Interpretations: Although the national peak time has been reached, a significant proportion of asymptomatic patients and the infections of super-spreading events (Isse) still exist in the population, indicating the potential difficulty for the prevention and control of the disease. As the Return-to-Work tide is approaching and upgrading, further measures (e.g., escalatory quarantine, mask wearing when going out, and sit apart when taking vehicles) will be particularly crucial to stop the COVID-19 in other cities outside of Wuhan.</td><td>2020-02-23</td><td>Jinhua Pan; Ye Yao; Zhixi Liu; Mengyi Li; Ying Wang; Weizhen Dong; Haidong Kan; Weibing Wang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025387</td></tr><tr><td>h5pit94p</td><td>null</td><td>medrxiv</td><td>Generalized anxiety disorder, depressive symptoms and sleep quality during COVID-19 epidemic in China: a web-based cross-sectional survey</td><td>10.1101/2020.02.19.20025395</td><td>null</td><td>null</td><td>medrvix</td><td>Background: China has been severely affected by COVID-19 (Corona Virus Disease 2019) since December, 2019. This study aimed to assess the population mental health burden during the epidemic, and to explore the potential influence factors. Methods: Using a web-based cross-sectional survey, we collected data from 7,236 self-selected volunteers assessed with demographic information, COVID-19 related knowledge, Generalized Anxiety Disorder-7 (GAD-7), Center for Epidemiology Scale for Depression (CES-D), and Pittsburgh Sleep Quality Index (PSQI). Logistic regressions were used to identify influence factors associated with mental health problem. Results: Of the total sample analyzed, the overall prevalence of GAD, depressive symptoms, and sleep quality were 35.1%, 20.1%, and 18.2%, respectively. Young people reported a higher prevalence of GAD and depressive symptoms than older people (P<0.001). Compared with other occupational group, healthcare workers have the highest rate of poor sleep quality (P<0.001). Multivariate logistic regression showed that age (< 35 years) and times to focus on the COVID-19 (≥ 3 hours per day) were associated with GAD, and healthcare workers were associated with poor sleep quality. Conclusions: Our study identified a major mental health burden of the public during COVID-19 epidemic in China. Young people, people who spent too much time on the epidemic, and healthcare workers were at high risk for mental illness. Continuous surveillance and monitoring of the psychological consequences for outbreaks should become routine as part of preparedness efforts worldwide.</td><td>2020-02-23</td><td>Yeen Huang; Ning Zhao</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025395</td></tr><tr><td>st5vs6gq</td><td>7bfc745e4edcef81527e38d69589e7bcb5f725c2</td><td>medrxiv</td><td>The serial interval of COVID-19 from publicly reported confirmed cases</td><td>10.1101/2020.02.19.20025452</td><td>null</td><td>null</td><td>medrvix</td><td>As a novel coronavirus (COVID-19) continues to emerge throughout China and threaten the globe, its transmission characteristics remain uncertain. Here, we analyze the serial intervals-the time period between the onset of symptoms in an index (infector) case and the onset of symptoms in a secondary (infectee) case-of 468 infector-infectee pairs with confirmed COVID-19 cases reported by health departments in 18 Chinese provinces between January 21, 2020, and February 8, 2020. The reported serial intervals range from -11 days to 20 days, with a mean of 3.96 days (95% confidence interval: 3.53-4.39), a standard deviation of 4.75 days (95% confidence interval: 4.46-5.07), and 12.6% of reports indicating pre-symptomatic transmission.</td><td>2020-02-23</td><td>Zhanwei Du; Xiaoke Xu; Ye Wu; Lin Wang; Benjamin J Cowling; Lauren Ancel Meyers</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.20025452</td></tr><tr><td>ao7bkcv5</td><td>aa507fcd3876f8c422993c8dd13847b1b8e77471</td><td>biorxiv</td><td>Pangolin homology associated with 2019-nCoV</td><td>10.1101/2020.02.19.950253</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractTo explore potential intermediate host of a novel coronavirus is vital to rapidly control continuous COVID-19 spread. We found genomic and evolutionary evidences of the occurrence of 2019-nCoV-like coronavirus (named as Pangolin-CoV) from dead Malayan Pangolins. Pangolin-CoV is 91.02% and 90.55% identical at the whole genome level to 2019-nCoV and BatCoV RaTG13, respectively. Pangolin-CoV is the lowest common ancestor of 2019-nCoV and RaTG13. The S1 protein of Pangolin-CoV is much more closely related to 2019-nCoV than RaTG13. Five key amino-acid residues involved in the interaction with human ACE2 are completely consistent between Pangolin-CoV and 2019-nCoV but four amino-acid mutations occur in RaTG13. It indicates Pangolin-CoV has similar pathogenic potential to 2019-nCoV, and would be helpful to trace the origin and probable intermediate host of 2019-nCoV.</td><td>2020-02-20</td><td>Tao Zhang; Qunfu Wu; Zhigang Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.950253</td></tr><tr><td>j6y806qu</td><td>9f5625d9287d8c6729215a20add99c8cd4ba08cb</td><td>biorxiv</td><td>Potential T-cell and B-cell Epitopes of 2019-nCoV</td><td>10.1101/2020.02.19.955484</td><td>null</td><td>null</td><td>biorxiv</td><td>As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data.Significance StatementThe novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV.</td><td>2020-02-21</td><td>Ethan Fast; Russ B. Altman; Binbin Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.955484</td></tr><tr><td>dc8mbllr</td><td>null</td><td>biorxiv</td><td>Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor</td><td>10.1101/2020.02.19.956235</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA novel and highly pathogenic coronavirus (2019-nCoV) has caused an outbreak in Wuhan city, Hubei province of China since December 2019, and soon spread nationwide and spilled over to other countries around the world. To better understand the initial step of infection at atomic-level, we determined the crystal structure of the 2019-nCoV spike receptor-binding domain (RBD) bound with the cell receptor ACE2 at 2.45 Å resolution. The overall ACE2-binding mode of the 2019-nCoV RBD is nearly identical to that of the SARS-CoV RBD, which also utilizes ACE2 as the cell receptor. Structural analysis identified residues in 2019-nCoV RBD critical for ACE2 binding, and majority of which are either highly conserved or shared similar side chain properties with those in the SARS-CoV RBD. Such similarity in structure and sequence strongly argue for a convergent evolution between 2019-nCoV and SARS-CoV RBD for improved binding to ACE2 despite of being segregated in different genetic lineages in the betacoronavirus genus. The epitopes of two SARS-CoV antibodies targeting the RBD are also analyzed with the 2019-nCoV RBD, providing insights into future identification of cross-reactive antibodies.</td><td>2020-02-20</td><td>Jun Lan; Jiwan Ge; Jinfang Yu; Sisi Shan; Huan Zhou; Shilong Fan; Qi Zhang; Xuanling Shi; Qisheng Wang; Linqi Zhang; Xinquan Wang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.956235</td></tr><tr><td>yaow639d</td><td>null</td><td>biorxiv</td><td>Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein</td><td>10.1101/2020.02.19.956581</td><td>null</td><td>null</td><td>biorxiv</td><td>SUMMARYThe recent emergence of a novel coronavirus associated with an ongoing outbreak of pneumonia (Covid-2019) resulted in infections of more than 72,000 people and claimed over 1,800 lives. Coronavirus spike (S) glycoprotein trimers promote entry into cells and are the main target of the humoral immune response. We show here that SARS-CoV-2 S mediates entry in VeroE6 cells and in BHK cells transiently transfected with human ACE2, establishing ACE2 as a functional receptor for this novel coronavirus. We further demonstrate that the receptor-binding domains of SARS-CoV-2 S and SARS-CoV S bind with similar affinities to human ACE2, which correlates with the efficient spread of SARS-CoV-2 among humans. We found that the SARS-CoV-2 S glycoprotein harbors a furin cleavage site at the boundary between the S1/S2 subunits, which is processed during biogenesis and sets this virus apart from SARS-CoV and other SARS-related CoVs. We determined a cryo-electron microscopy structure of the SARS-CoV-2 S ectodomain trimer, demonstrating spontaneous opening of the receptor-binding domain, and providing a blueprint for the design of vaccines and inhibitors of viral entry. Finally, we demonstrate that SARS-CoV S murine polyclonal sera potently inhibited SARS-CoV-2 S-mediated entry into target cells, thereby indicating that cross-neutralizing antibodies targeting conserved S epitopes can be elicited upon vaccination.</td><td>2020-02-20</td><td>Alexandra C. Walls; Young-Jun Park; M. Alexandra Tortorici; Abigail Wall; Andrew T. McGuire; David Veesler</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.956581</td></tr><tr><td>elo7kfun</td><td>3a6544f6f247366d4c12167f7fba743e9d981c63</td><td>biorxiv</td><td>Structural basis for the recognition of the 2019-nCoV by human ACE2</td><td>10.1101/2020.02.19.956946</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAngiotensin-converting enzyme 2 (ACE2) has been suggested to be the cellular receptor for the new coronavirus (2019-nCoV) that is causing the coronavirus disease 2019 (COVID-19). Like other coronaviruses such as the SARS-CoV, the 2019-nCoV uses the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) to engage ACE2. We most recently determined the structure of the full-length human ACE2 in complex with a neutral amino acid transporter B0AT1. Here we report the cryo-EM structure of the full-length human ACE2 bound to the RBD of the 2019-nCoV at an overall resolution of 2.9 Å in the presence of B0AT1. The local resolution at the ACE2-RBD interface is 3.5 Å, allowing analysis of the detailed interactions between the RBD and the receptor. Similar to that for the SARS-CoV, the RBD of the 2019-nCoV is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues. Pairwise comparison reveals a number of variations that may determine the different affinities between ACE2 and the RBDs from these two related viruses.</td><td>2020-02-20</td><td>Renhong Yan; Yuanyuan Zhang; Yingying Guo; Lu Xia; Qiang Zhou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.956946</td></tr><tr><td>99sbhbuo</td><td>70c7260fc0702bd4a554421ac68972c991313e56</td><td>biorxiv</td><td>Mucin 4 Protects Female Mice from Coronavirus Pathogenesis</td><td>10.1101/2020.02.19.957118</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractUsing incipient lines of the Collaborative Cross (CC), a murine genetic reference population, we previously identified a quantitative trait loci (QTL) associated with low SARS-CoV titer. In this study, we integrated sequence information and RNA expression of genes within the QTL to identify mucin 4 (Muc4) as a high priority candidate for controlling SARS-CoV titer in the lung. To test this hypothesis, we infected Muc4-/- mice and found that female, but not male, Muc4-/- mice developed more weight loss and disease following infection with SARS-CoV. Female Muc4-/- mice also had more difficulty breathing despite reduced lung pathology; however, no change in viral titers was observed. Comparing across viral families, studies with chikungunya virus, a mosquito-borne arthralgic virus, suggests that Muc4’s impact on viral pathogenesis may be widespread. Although not confirming the original titer QTL, our data identifies a role for Muc4 in the SARS-CoV disease and viral pathogenesis.ImportanceGiven the recent emergence of SARS-CoV-2, this work suggest that Muc4 expression plays a protective role in female mice not conserved in male mice following SARS-CoV infection. With the SARS-CoV-2 outbreak continuing, treatments that modulate or enhance Muc4 activity may provide an avenue for treatment and improved outcomes. In addition, the work highlights the importance of studying host factors including host genetics and biological sex as key parameters influencing infection and disease outcomes.</td><td>2020-02-20</td><td>Jessica A. Plante; Kenneth S. Plante; Lisa E. Gralinski; Anne Beall; Martin T. Ferris; Daniel Bottomly; Richard Green; Shannon K. McWeeney; Mark T. Heise; Ralph S. Baric; Vineet D. Menachery</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.19.957118</td></tr><tr><td>bojhhrvb</td><td>null</td><td>medrxiv</td><td>Novel Coronavirus 2019 (Covid-19) epidemic scale estimation: topological network-based infection dynamic model</td><td>10.1101/2020.02.20.20023572</td><td>null</td><td>null</td><td>medrvix</td><td>Backgrounds: An ongoing outbreak of novel coronavirus pneumonia (Covid-19) hit Wuhan and hundreds of cities, 29 territories globally. We present a method for scale estimation in dynamic while most of the researchers used static parameters. Methods: We use historical data and the SEIR model for important parameters assumption. And according to the timeline, we use dynamic parameters for infection topology network building. Also, the migration data is used for the Non-Wuhan area estimation which can be cross-validated for the Wuhan model. All data are from the public. Results: The estimated number of infections is 61,596 (95%CI: 58,344.02-64,847.98) by 25 Jan in Wuhan. And the estimation number of the imported cases from Wuhan of Guangzhou was 170 (95%CI: 161.27-179.26), infection scale in Guangzhou is 315 (95%CI: 109.20-520.79), while the imported cases are 168 and the scale of the infection is 339 published by the authority. Conclusions: Using dynamic network models and dynamic parameters for different time periods is an effective way of infection scale modeling.</td><td>2020-02-23</td><td>Keke Tang; Yining Huang; Meilian Chen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20023572</td></tr><tr><td>bxzprauu</td><td>373b3b78e296f0046af3506452c6934b71084b58</td><td>medrxiv</td><td>COVID-19 in Wuhan: Immediate Psychological Impact on 5062 Health Workers</td><td>10.1101/2020.02.20.20025338</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND: The outbreak of COVID-19 has laid unprecedented psychological stress on health workers (HWs). We aimed to assess the immediate psychological impact on HWs at Tongji Hospital in Wuhan, China. METHODS: We conducted a single-center, cross-sectional survey of HWs via online questionnaires between February 8th and 10th, 2020. We evaluated stress, depression and anxiety by Impact of Event Scale-Revised (IES-R), Patient Health Questionnaire-9 (PHQ-9), and Generalized Anxiety Disorder 7-item (GAD-7), respectively. We also designed a questionnaire to assess the effect of psychological protective measures taken by Tongji Hospital. Multivariate logistic regression was used to identify predictors of acute stress, depression, and anxiety. RESULTS: We received 5062 completed questionnaires (response rate, 77.1 percent). 1509 (29.8 percent), 681 (13.5 percent) and 1218 (24.1 percent) HWs reported stress, depression and anxiety symptoms. Women (hazard ratio[HR], 1.31; P=0.032), years of working> 10 years (HR, 2.02; P<0.001), concomitant chronic diseases (HR, 1.51; P<0.001), history of mental disorders (HR, 3.27; P<0.001), and family members or relatives confirmed or suspected (HR, 1.23; P=0.030) were risk factors for stress, whereas care provided by hospital and department administrators(odds ratio [OR], 0.76; P=0.024) and full coverage of all departments with protective measures (OR, 0.69; P=0.004) were protective factors. CONCLUSIONS: Women and those who have more than 10 years of working, concomitant chronic diseases, history of mental disorders, and family members or relatives confirmed or suspected are susceptible to stress, depression and anxiety among HWs during the COVID-19 pandemic. Psychological protective measures implemented by the hospital could be helpful.</td><td>2020-02-23</td><td>Zhou Zhu; Shabei Xu; Hui Wang; Zheng Liu; Jianhong Wu; Guo Li; Jinfeng Miao; Chenyan Zhang; Yuan Yang; Wenzhe Sun; Suiqiang Zhu; Yebin Fan; Junbo Hu; Jihong Liu; Wei Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025338</td></tr><tr><td>eu3rry7p</td><td>f854d7b750541d65c48df953a65028a50ce57881</td><td>medrxiv</td><td>ACP risk grade: a simple mortality index for patients with confirmed or suspected severe acute respiratory syndrome coronavirus 2 disease (COVID-19) during the early stage of outbreak in Wuhan, China</td><td>10.1101/2020.02.20.20025510</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) disease (COVID-19) outbreaks in Wuhan, China, healthcare systems capacities in highly endemic areas have been overwhelmed. Approaches to efficient management are urgently needed and key to a quicker control of the outbreaks and casualties. We aimed to characterize the clinical features of hospitalized patients with confirmed or suspected COVID-19, and develop a mortality risk index for COVID-19 patients. Methods: In this retrospective one-centre cohort study, we included all the confirmed or suspected COVID-19 patients hospitalized in a COVID-19-designated hospital from January 21 to February 5, 2020. Demographic, clinical, laboratory, radiological and clinical outcome data were collected from the hospital information system, nursing records and laboratory reports. Results: Of 577 patients with at least one post-admission evaluation, the median age was 55 years (interquartile range [IQR], 39 - 66); 254 (44.0%) were men; 22.8% (100/438) were severe pneumonia on admission, and 37.7% (75/199) patients were SARS-CoV-2 positive. The clinical, laboratory and radiological data were comparable between positive and negative SARS-CoV-2 patients. During a median follow-up of 8.4 days (IQR, 5.8 - 12.0), 39 patients died with a 12-day cumulative mortality of 8.7% (95% CI, 5.9% to 11.5%). A simple mortality risk index (called ACP index), composed of Age and C-reactive Protein, was developed. By applying the ACP index, patients were categorized into three grades. The 12-day cumulative mortality in grade three (age ≥ 60 years and CRP ≥ 34 mg/L) was 33.2% (95% CI, 19.8% to 44.3%), which was significantly higher than those of grade two (age ≥ 60 years and CRP < 34 mg/L; age < 60 years and CRP ≥ 34 mg/L; 5.6% [95% CI, 0 to 11.3%]) and grade one (age < 60 years and CRP < 34 mg/L, 0%) (P <0.001), respectively. Conclusion: The ACP index can predict COVID-19 related short-term mortality, which may be a useful and convenient tool for quickly establishing a COVID-19 hierarchical management system that can greatly reduce the medical burden and therefore mortality in highly endemic areas.</td><td>2020-02-23</td><td>Jiatao Lu; Shufang Hu; Rong Fan; Zhihong Liu; Xueru Yin; Qiongya Wang; Qingquan Lv; Zhifang Cai; Haijun Li; Yuhai Hu; Ying Han; Hongping Hu; Wenyong Gao; Shibo Feng; Qiongfang Liu; Hui Li; Jian Sun; Jie Peng; Xuefeng Yi; Zixiao Zhou; Yabing Guo; Jinlin Hou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025510</td></tr><tr><td>yqiggzk9</td><td>null</td><td>medrxiv</td><td>Clinical characteristics of 51 patients discharged from hospital with COVID-19 in Chongqing，China</td><td>10.1101/2020.02.20.20025536</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract Background：Since December 2019, Severe acute respiratory syndrome coronavirus 2（SARS-CoV-2）-infected disease (Coronavirus Disease 2019，COVID-19) emerged in Wuhan , China，and rapidly spread throughout China，even throughout the world. We try to describe the epidemiological and clinical characteristics of COVID-19 in non-Wuhan area，and explore its effective treatment. Methods：Retrospective, single-center case series of the 51 hospitalized patients with confirmed COVID-19 at Chongqing University Three Gorges Hospital in Chongqing, China, from January 20 to February 3, 2020；The discharge time was from January 29 to February 11, 2020. The main results and indicators of epidemiology, demography, clinical manifestation, laboratory examination, imaging data and treatment data of 51 patients with covid-19 were collected and analyzed. The changes of blood routine and biochemical indexes at discharge and admission were compared. Compare the clinical characteristics of severe patients (including severe and critical patients) and non- severe patients (general patients). Results： Of 51 hospitalized patients with COVID-19, the median age was 45 years (interquartile range, 34-51; range, 16-68 years) and 32 (62.7%) were men.43（84.3%）patients had been to Wuhan or Other Hubei areas outside Wuhan，and 4（7.7%） patients had a clear contact history of COVID-19 patients before the onset of the disease, and 4（7.7%） patients had no clear epidemiological history of COVID-19.Common symptoms included fever (43 [84.3%]), cough (38 [74.5%]) and fatigue (22 [43.1%]). Lymphopenia was observed in 26 patients (51.0%), and elevated C-reactive protein level in 32 patients (62.7%). Ground-glass opacity was the typical radiological finding on chest computed tomography (41 [80.4%])，Local consolidation of pneumonia in some patients(17 [33.3%]).Most of the patients were treated with traditional Chinese medicine decoction (28 [54.9%])，all of them received aerosol inhalation of recombinant human interferon a-1b for injection and oral antiviral therapy with Lopinavir and Ritonavir tablets (51 [100%]); Most of the patients were given Bacillus licheniformis capsules regulated intestinal flora treatment (44 [86.3%]). 10 patients (19.6%) received short-term (3-5 days) glucocorticoid treatment. Compared with non-severe patients (n = 44), severe patients (n = 7) were older (median age, 52 years vs 44 years), had a higher proportion of diabetes mellitus (4 [57.1%] vs 0 [0.0%]), most of them needed antibiotic treatment (7 [100%] vs 4 [9.1%], most of them needed nutritional diet (6 [85.7%) vs 0 [0.0%], and were more likely to have dyspnea (6 [85.7%] vs 5 [11.4%])，most of them needed noninvasive mechanical ventilation (6 [85.7%] vs 0 [0.0%]). Except one patient died, the remaining 50 patients were discharged according to the discharge standard, the common clinical symptoms disappeared basically, the lymphocyte increased significantly (P=0.008), CRP decreased significantly (P <0.001). The median length of stay was 12 days (IQR, 9-13). Conclusion：In 51 single center cases confirmed as COVID-19 and discharged from the hospital, 13.7% of the patients were severe. The main clinical symptoms of patients with COVID-19 were fever, cough and asthenia，Some patients had obvious dyspnea. They had clinical laboratory and radiologic characteristics. There is no specific drug treatment for the disease. For the treatment of COVID-19, in addition to oxygen inhalation and antiviral treatment, attention should be paid to the dialectical treatment of traditional Chinese medicine, regulation of intestinal flora, nutritional support treatment and other comprehensive treatment.</td><td>2020-02-23</td><td>liu lei; Gao Jian-ya</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025536</td></tr><tr><td>4pe9o59g</td><td>8d6d3be5f9353eb3358b44e646c83f1a90861b4d</td><td>medrxiv</td><td>The efficacy of convalescent plasma for the treatment of severe influenza</td><td>10.1101/2020.02.20.20025593</td><td>null</td><td>null</td><td>medrvix</td><td>Background. Administration of convalescent plasma may be of clinical benefit for treatment of severe acute viral respiratory infections. However, no clear evidence exists to support or oppose convalescent plasma use in clinical practice. We conducted a systematic review and meta-analysis to assess the evidence of randomized controlled trials (RCTs) in the convalescent plasma for the treatment of severe influenza. Methods. Healthcare databases were searched in February 2020. All records were screened against the eligibility criteria. Data extraction and risk of bias assessments were undertaken. The primary outcome was case fatality rates by influenza. Results. We identified 5 RCTs of severe influenza. The pooled analyses showed no evidence for a reduction in mortality (Odds Ratio (OR) 1.06; p = 0.87; I2 = 35%). We also found non significant reductions in days in ICU and hospital, and days on mechanical ventilation. There seemed to have a biological benefit of increasing HAI titer levels and decreasing influenza B virus loads and cytokines after convalescent plasma treatment. No serious adverse events was reported between two groups. Studies were commonly of low risk of bias with high quality. Conclusions. Convalescent plasma appears safe but may not reduce mortality in severe influenza. This therapy should be studied within the context of a well designed clinical trial for treatment of SARS Cov 2 infection.</td><td>2020-02-23</td><td>Zhiheng Xu; Jianmeng Zhou; Yongbo Huang; Xuesong Liu; Yonghao Xu; Sibei Chen; Dongdong Liu; Zhimin Lin; Xiaoqing Liu; Yimin Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025593</td></tr><tr><td>rmia5w3x</td><td>6fcf4019b0be24071210f97eda4baba53a6a79de</td><td>medrxiv</td><td>Clinical and pathological characteristics of 2019 novel coronavirus disease (COVID-19): a systematic review</td><td>10.1101/2020.02.20.20025601</td><td>null</td><td>null</td><td>medrvix</td><td>Importance: In 2002-2003, a severe pulmonary infectious disease occurred in Guangdong, China. The disease was caused by SARS-CoV, 17 years apart, also happen in China, and SARS-CoV-2, this epidemic has posed a significant hazard to people health both China and the whole world. Objective: Summarized the latest epidemiological changes, clinical manifestations, auxiliary examination and pathological characteristics of COVID-19. Evidence Review: PubMed database were searched from 2019 to 2020 using the index terms novel coronavirus or COVID-19 or 2019-nCoV or SARS-CoV-2 and synonyms. Articles that reported clinical characteristics, laboratory results, imageological diagnosis and pathologic condition were included and were retrospectively reviewed for these cases. This paper adopts the method of descriptive statistics. Results: 34 COVID-19-related articles were eligible for this systematic review;Four of the articles were related to pathology. We found that Fever (86.0%), cough (63.9%) and Malaise/Fatigue (34.7%) were the most common symptoms in COVID-19. But in general, the clinical symptoms and signs of COVID-19 were not obvious. Compared with SARS, COVID-19 was transmitted in a more diverse way. The mortality rates of COVID-19 were 2.5%, and the overall infection rate of healthcare worker of COVID-19 was 3.9%. We also found that the pathological features of COVID-19 have greatly similar with SARS, which manifested as ARDS. But the latest pathological examination of COVID-19 revealed the obvious mucinous secretions in the lungs. Interpretation: The clinical and pathological characteristics of SARS and COVID-19 in China are very similar, but also difference. The latest finds of pathological examination on COVID-19 may upend existing treatment schemes, so the early recognition of disease by healthcare worker is very important.</td><td>2020-02-25</td><td>mao yaqian; Wei Lin; Junping Wen; Gang Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025601</td></tr><tr><td>ofoqk100</td><td>a36bfdd1c9a1666401269fb0c08a3c2922bd6812</td><td>medrxiv</td><td>Clinical Characteristics of 24 Asymptomatic Infections with COVID-19 Screened among Close Contacts in Nanjing, China</td><td>10.1101/2020.02.20.20025619</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Previous studies have showed clinical characteristics of patients with the 2019 novel coronavirus disease (COVID-19) and the evidence of person-to-person transmission. Limited data are available for asymptomatic infections. This study aims to present the clinical characteristics of 24 cases with asymptomatic infection screened from close contacts and to show the transmission potential of asymptomatic COVID-19 virus carriers. Methods: Epidemiological investigations were conducted among all close contacts of COVID-19 patients (or suspected patients) in Nanjing, Jiangsu Province, China, from Jan 28 to Feb 9, 2020, both in clinic and in community. Asymptomatic carriers were laboratory-confirmed positive for the COVID-19 virus by testing the nucleic acid of the pharyngeal swab samples. Their clinical records, laboratory assessments, and chest CT scans were reviewed. Findings: None of the 24 asymptomatic cases presented any obvious symptoms before nucleic acid screening. Five cases (20.8%) developed symptoms (fever, cough, fatigue and etc.) during hospitalization. Twelve (50.0%) cases showed typical CT images of ground-glass chest and five (20.8%) presented stripe shadowing in the lungs. The remaining seven (29.2%) cases showed normal CT image and had no symptoms during hospitalization. These seven cases were younger (median age: 14.0 years; P = 0.012) than the rest. None of the 24 cases developed severe COVID-19 pneumonia or died. The median communicable period, defined as the interval from the first day of positive nucleic acid tests to the first day of continuous negative tests, was 9.5 days (up to 21 days among the 24 asymptomatic cases). Through epidemiological investigation, we observed a typical asymptomatic transmission to the cohabiting family members, which even caused severe COVID-19 pneumonia. Interpretation: The asymptomatic carriers identified from close contacts were prone to be mildly ill during hospitalization. However, the communicable period could be up to three weeks and the communicated patients could develop severe illness. These results highlighted the importance of close contact tracing and longitudinally surveillance via virus nucleic acid tests. Further isolation recommendation and continuous nucleic acid tests may also be recommended to the patients discharged.</td><td>2020-02-23</td><td>Zhiliang Hu; Ci Song; Chuanjun Xu; Guangfu Jin; Yaling Chen; Xin Xu; Hongxia Ma; Wei Chen; Yuan Lin; Yishan Zheng; Jianming Wang; zhibin hu; Yongxiang Yi; Hongbing Shen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025619</td></tr><tr><td>kqanoog7</td><td>null</td><td>medrxiv</td><td>Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</td><td>10.1101/2020.02.20.20025841</td><td>null</td><td>null</td><td>medrvix</td><td>We report the kinetics of the immune response in relation to clinical and virological features of a patient with mild-to-moderate coronavirus disease-19 (COVID-19) requiring hospitalisation. Increased antibody-secreting cells, follicular T-helper cells, activated CD4+ and CD8+ T-cells and IgM/IgG SARS-CoV-2-binding antibodies were detected in blood, prior to symptomatic recovery. These immunological changes persisted for at least 7 days following full resolution of symptoms, indicating substantial anti-viral immunity in this non-severe COVID-19.</td><td>2020-02-23</td><td>Irani Thevarajan; Thi HO Nguyen; Marios Koutsakos; Julian Druce; Leon Caly; Carolien E van de Sandt; Xiaoxiao Jia; Suellen Nicholson; Mike Catton; Benjamin Cowie; Steven Tong; Sharon Lewin; Katherine Kedzierska</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025841</td></tr><tr><td>atnz63pk</td><td>207dcb6e3cd43cdea91d29a15be1ee34068bd54e</td><td>medrxiv</td><td>Estimating the Asymptomatic Proportion of 2019 Novel Coronavirus onboard the Princess Cruises Ship, 2020</td><td>10.1101/2020.02.20.20025866</td><td>null</td><td>null</td><td>medrvix</td><td>The potential infectiousness of asymptomatic COVID-19 cases together with a substantial fraction of asymptomatic infections among all infections, have been highlighted in clinical studies. We conducted statistical modeling analysis to derive the delay-adjusted asymptomatic proportion of the positive COVID-19 infections onboard the Princess Cruises ship along with the timeline of infections. We estimated the asymptomatic proportion at 17.9% (95% CrI: 15.5%-20.2%), with most of the infections occurring before the start of the 2-week quarantine.</td><td>2020-02-23</td><td>Kenji Mizumoto; Katsushi Kagaya; Alexander Zarebski; Gerardo Chowell</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025866</td></tr><tr><td>s7uqawbd</td><td>bd8f57ea4aadeda075d557f566a1909d68e658c4</td><td>medrxiv</td><td>Rapid colorimetric detection of COVID-19 coronavirus using a reverse tran-scriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic plat-form: iLACO</td><td>10.1101/2020.02.20.20025874</td><td>null</td><td>null</td><td>medrvix</td><td>The recent outbreak of a novel coronavirus SARS-CoV-2 (also known as 2019-nCoV) threatens global health, given serious cause for concern. SARS-CoV-2 is a human-to-human pathogen that caused fever, severe respiratory disease and pneumonia (known as COVID-19). By press time, more than 70,000 infected people had been confirmed worldwide. SARS-CoV-2 is very similar to the severe acute respiratory syndrome (SARS) coronavirus broke out 17 years ago. However, it has increased transmissibility as compared with the SARS-CoV, e.g. very often infected individuals without any symptoms could still transfer the virus to others. It is thus urgent to develop a rapid, accurate and onsite diagnosis methods in order to effectively identify these early infects, treat them on time and control the disease spreading. Here we developed an isothermal LAMP based method-iLACO (isothermal LAMP based method for COVID-19) to amplify a fragment of the ORF1ab gene using 6 primers. We assured the species-specificity of iLACO by comparing the sequences of 11 related viruses by BLAST (including 7 similar coronaviruses, 2 influenza viruses and 2 normal coronaviruses). The sensitivity is comparable to Taqman based qPCR detection method, detecting synthesized RNA equivalent to 10 copies of 2019-nCoV virus. Reaction time varied from 15-40 minutes, depending on the loading of virus in the collected samples. The accuracy, simplicity and versatility of the new developed method suggests that iLACO assays can be conveniently applied with for 2019-nCoV threat control, even in those cases where specialized molecular biology equipment is not available.</td><td>2020-02-24</td><td>Lin Yu; Shanshan Wu; Xiaowen Hao; Xuelong Li; Xiyang Liu; Shenglong Ye; Heng Han; Xue Dong; Xin Li; Jiyao Li; Jianmin Liu; Na Liu; Wanzhong Zhang; Vicent Pelechano; Wei-Hua Chen; Xiushan Yin</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025874</td></tr><tr><td>86opxdjd</td><td>ba3efcd6b74e55327fd7db470d824fc18943f30e</td><td>medrxiv</td><td>Evaluating the impact of international airline suspensions on the early global spread of COVID-19</td><td>10.1101/2020.02.20.20025882</td><td>null</td><td>null</td><td>medrvix</td><td>Global airline networks play a key role in the global importation of emerging infectious diseases. Detailed information on air traffic between international airports has been demonstrated to be useful in retrospectively validating and prospectively predicting case emergence in other countries. In this paper, we use a well-established metric known as effective distance on the global air traffic data from IATA to quantify risk of emergence for different countries as a consequence of direct importation from China, and compare it against arrival times for the first 24 countries. Using this model trained on official first reports from WHO, we estimate time of arrival (ToA) for all other countries. We then incorporate data on airline suspensions to recompute the effective distance and assess the effect of such cancellations in delaying the estimated arrival time for all other countries. Finally we use the infectious disease vulnerability indices to explain some of the estimated reporting delays.</td><td>2020-02-23</td><td>Aniruddha Adiga; Srinivasan Venkatramanan; James Schlitt; Akhil Peddireddy; Allan Dickerman; Andrei Bura; Andrew Warren; Brian D Klahn; Chunhong Mao; Dawen Xie; Dustin Machi; Erin Raymond; Fanchao Meng; Golda Barrow; Henning Mortveit; Jiangzhuo Chen; Jim Walke; Joshua Goldstein; Mandy L Wilson; Mark Orr; Przemyslaw Porebski; Pyrros A Telionis; Richard Beckman; Stefan Hoops; Stephen Eubank; Young Yun Baek; Bryan Lewis; Madhav Marathe; Chris Barrett</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025882</td></tr><tr><td>w9a0jopu</td><td>9896bc65559e6406d0d3cb35e9b01b953d61959d</td><td>medrxiv</td><td>From Isolation to Coordination: How Can Telemedicine Help Combat the COVID-19 Outbreak?</td><td>10.1101/2020.02.20.20025957</td><td>null</td><td>null</td><td>medrvix</td><td>The rapid spread of Coronavirus disease 2019 (COVID-19) presents China with a critical challenge. As normal capacity of the Chinese hospitals is exceeded, healthcare professionals struggling to manage this unprecedented crisis face the difficult question of how best to coordinate the medical resources used in highly separated locations. Responding rapidly to this crisis, the National Telemedicine Center of China (NTCC), located in Zhengzhou, Henan Province, has established the Emergency Telemedicine Consultation System (ETCS), a telemedicine-enabled outbreak alert and response network. ETCS is built upon a doctor-to-doctor (D2D) approach, in which health services can be accessed remotely through terminals across hospitals. The system architecture of ETCS comprises three major architectural layers: (1) telemedicine service platform layer, (2) telemedicine cloud layer, and (3) telemedicine service application layer. Our ETCS has demonstrated substantial benefits in terms of the effectiveness of consultations and remote patient monitoring, multidisciplinary care, and prevention education and training.</td><td>2020-02-23</td><td>Yunkai Zhai; Yichuan Wang; Minhao Zhang; Jody Hoffer Gittell; Shuai Jiang; Baozhan Chen; Fangfang Cui; Xianying He; Jie Zhao; Xiaojun Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025957</td></tr><tr><td>4gaa14ly</td><td>ae6fc64042b050df93ebb8f8045892952f18510f</td><td>medrxiv</td><td>Assessing the impact of a symptom-based mass screening and testing intervention during a novel infectious disease outbreak: The case of COVID-19</td><td>10.1101/2020.02.20.20025973</td><td>null</td><td>null</td><td>medrvix</td><td>A symptom-based mass screening and testing intervention (MSTI) can identify a large fraction of infected individuals during an infectious disease outbreak. China is currently using this strategy for the COVID-19 outbreak. However, MSTI might lead to increased transmission if not properly implemented. We investigate under which conditions MSTI is beneficial.</td><td>2020-02-23</td><td>Yang Ge; Brian Kenneth McKay; Shengzhi Sun; Feng Zhang; Andreas Handel</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025973</td></tr><tr><td>1s0exznp</td><td>null</td><td>medrxiv</td><td>Generation of antibodies against COVID-19 virus for development of diagnostic tools</td><td>10.1101/2020.02.20.20025999</td><td>null</td><td>null</td><td>medrvix</td><td>The COVID-19 China coronavirus started in Dec 2019 was challenged by the lack of accurate serological diagnostic tool for this deadly disease to quickly identify and isolate the infected patients. The generation of COVID-19-specific antibodies is essential for such tasks. Here we report that polyclonal and monoclonal antibodies were generated by immunizing animals with synthetic peptides corresponding to different areas of Nucleoprotein (N) of COVID-19. The specificities of the COVID-19 antibodies were assessed by Western Blot analysis against NPs from COVID-19, MERS and SARS. Antibodies were used for immunohistochemistry staining of the tissue sections from COVID-19 infected patient, as a potential diagnostic tool. A Sandwich ELISA kit was quickly assembled for quantitation of the virus/NP of COVID-19 concentrations in the vaccine preparations. Development of POCT is also aggressively undergoing.</td><td>2020-02-25</td><td>Maohua Li; Ronghua Jin; Ya Peng; Cuiyan Wang; Wenlin Ren; Fudong Lv; Sitao Gong; Feng Fang; Qianyun Wang; Jianli Li; Tong Shen; Hunter Sun; Lei Zhou; Yali Cui; Hao Song; Le Sun</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.20025999</td></tr><tr><td>bnnt05fn</td><td>d01e580a45c457c7c200cd884e746b4d836e2ea9</td><td>biorxiv</td><td>Vulnerabilities in coronavirus glycan shields despite extensive glycosylation</td><td>10.1101/2020.02.20.957472</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSevere acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) coronaviruses (CoVs) are zoonotic pathogens with high fatality rates and pandemic potential. Vaccine development has focussed on the principal target of the neutralizing humoral immune response, the spike (S) glycoprotein, which mediates receptor recognition and membrane fusion. Coronavirus S proteins are extensively glycosylated viral fusion proteins, encoding around 69-87 N-linked glycosylation sites per trimeric spike. Using a multifaceted structural approach, we reveal a specific area of high glycan density on MERS S that results in the formation of under-processed oligomannose-type glycan clusters, which was absent on SARS and HKU1 CoVs. We provide a comparison of the global glycan density of coronavirus spikes with other viral proteins including HIV-1 envelope, Lassa virus glycoprotein complex, and influenza hemagglutinin, where glycosylation plays a known role in shielding immunogenic epitopes. Consistent with the ability of the antibody-mediated immune response to effectively target and neutralize coronaviruses, we demonstrate that the glycans of coronavirus spikes are not able to form an efficacious high-density global shield to thwart the humoral immune response. Overall, our data reveal how differential organisation of viral glycosylation across class I viral fusion proteins influence not only individual glycan compositions but also the immunological pressure across the viral protein surface.</td><td>2020-02-21</td><td>Yasunori Watanabe; Zachary T. Berndsen; Jayna Raghwani; Gemma E. Seabright; Joel D. Allen; Jason S. McLellan; Ian A. Wilson; Thomas A. Bowden; Andrew B. Ward; Max Crispin</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.957472</td></tr><tr><td>535lw99y</td><td>78e49fdb6f0aa9924a5b510341d52b618fff0ca6</td><td>biorxiv</td><td>A Multiscale and Comparative Model for Receptor Binding of 2019 Novel Coronavirus and the Implication of its Life Cycle in Host Cells</td><td>10.1101/2020.02.20.958272</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe respiratory syndrome caused by a new type of coronavirus has been emerging from China and caused more than 1000 death globally since December 2019. This new virus, called 2019 novel coronavirus (2019-nCoV) uses the same receptor called Angiotensinconverting enzyme 2 (ACE2) to attack humans as the coronavirus that caused the severe acute respiratory syndrome (SARS) seventeen years ago. Both viruses recognize ACE2 through the spike proteins (S-protein) on their surfaces. It was found that the S-protein from the SARS coronavirus (SARS-CoV) bind stronger to ACE2 than 2019-nCoV. However, function of a bio-system is often under kinetic, rather than thermodynamic, control. To address this issue, we constructed a structural model for complex formed between ACE2 and the S-protein from 2019-nCoV, so that the rate of their association can be estimated and compared with the binding of S-protein from SARS-CoV by a multiscale simulation method. Our simulation results suggest that the association of new virus to the receptor is slower than SARS, which is consistent with the experimental data obtained very recently. We further integrated this difference of association rate between virus and receptor into a mathematical model which describes the life cycle of virus in host cells and its interplay with the innate immune system. Interestingly, we found that the slower association between virus and receptor can result in longer incubation period, while still maintaining a relatively higher level of viral concentration in human body. Our computational study therefore provides, from the molecular level, one possible explanation that the new disease by far spread much faster than SARS.</td><td>2020-02-21</td><td>Zhaoqian Su; Yinghao Wu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.958272</td></tr><tr><td>0vjs2w3l</td><td>9aef9e1c22896809852ec6c94e413aff33b3d9cf</td><td>biorxiv</td><td>Inactivating porcine coronavirus before nuclei acid isolation with the temperature higher than 56 °C damages its genome integrity seriously</td><td>10.1101/2020.02.20.958785</td><td>null</td><td>null</td><td>biorxiv</td><td>Abstract2019-Novel Coronavirus (2019-nCoV) is the pathogen of Corona Virus Disease 2019. Nucleic acid detection of 2019-nCoV is one of the key indicators for clinical diagnosis. However, the positive rate is only 30-50%. Currently, fluorescent quantitative RT-PCR technology is mainly used to detect 2019-nCoV. According to “The Laboratory Technical Guidelines for Detection 2019-nCoV (Fourth Edition)” issued by National Health and Commission of China and “The Experts’ Consensus on Nucleic Acid Detection of 2019-nCoV” released by Chinese Society of Laboratory Medicine, the human samples must be placed under 56°C or higher to inactivate the viruses in order to keep the inspectors from virus infection before the nucleic acids were isolated as the template of qRT-PCR. In this study, we demonstrated that the virus inactivation treatment disrupts its genome integrity seriously when using porcine epidemic diarrhea virus (vaccine), a kind of coronavirus, as a model. Our results showed that only 50.11% of the detectable viral templates left after the inactivation of 56 °C for 30 minutes and only 3.36% left after the inactivation of 92 °C for 5 minutes when the samples were preserved by Hank’s solution, one of an isotonic salt solutions currently suggested. However, the detectable templates of viral nucleic acids can be unchanged after the samples were incubated at 56 °C or higher if the samples were preserved with an optimized solution to protect the RNA from being disrupted. We therefore highly recommend to carry out systematic investigation on the impact of high temperature inactivation on the integrity of 2019-nCoV genome and develop a sample preservation solution to protect the detectable templates of 2019-nCoV nucleic acids from high temperature inactivation damage.</td><td>2020-02-22</td><td>Qingxin Zhang; Qingshun Zhao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.20.958785</td></tr><tr><td>u6b8iwr0</td><td>233978852b45429db49d2aabe1c9c9bfe73b1884</td><td>medrxiv</td><td>CoVID-19 in Japan: What could happen in the future?</td><td>10.1101/2020.02.21.20026070</td><td>null</td><td>null</td><td>medrvix</td><td>COVID-19 has been impacting on the whole world critically and constantly Since December 2019. We have independently developed a novel statistical time delay dynamic model on the basis of the distribution models from CCDC. Based only on the numbers of confirmed cases in different regions in China, the model can clearly reveal that the containment of the epidemic highly depends on early and effective isolation. We apply the model on the epidemic in Japan and conclude that there could be a rapid outbreak in Japan if no effective quarantine measures are carried out immediately.</td><td>2020-02-23</td><td>Nian Shao; Hanshuang Pan; Xingjie Li; Weijia Li; Shufen Wang; Yan Xuan; Yue Yan; Yu Jiang; Keji Liu; Yu Chen; Boxi Xu; Xinyue Luo; Christopher Y. Shen; Min Zhong; Xiang Xu; Xu Chen; Shuai Lu; Guanghong Ding; Jin Cheng; Wenbin Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026070</td></tr><tr><td>1mxjklgx</td><td>091a8e9a61e19e88caeb039f0e3888d111b20439</td><td>medrxiv</td><td>Epidemiological characteristics of 1212 COVID-19 patients in Henan, China</td><td>10.1101/2020.02.21.20026112</td><td>null</td><td>null</td><td>medrvix</td><td>Based on publicly released data for 1212 patients, we investigated the epidemiological characteristics of COVID-19 in Henan of China. The following findings are obtained: 1) COVID-19 patients in Henan show gender (55% vs 45%) and age (81% aged between 21 and 60) preferences, possible causes were explored; 2) Statistical analysis on 483 patients reveals that the estimated average, mode and median incubation periods are 7.4, 4 and 7 days; Incubation periods of 92% patients were no more than 14 days; 3) The epidemic of COVID-19 in Henan has undergone three stages and showed high correlations with the numbers of patients that recently return from Wuhan; 4) Network analysis on the aggregate outbreak phenomena of COVID-19 revealed that 208 cases were clustering infected, and various people's Hospital are the main force in treating patients. The related investigations have potential implications for the prevention and control of COVID-19.</td><td>2020-02-23</td><td>Pei Wang; Junan Lu; Yanyu Jin; Mengfan Zhu; Lingling Wang; Shunjie Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026112</td></tr><tr><td>aoyyk5fl</td><td>eaf0b485f290fa884dd18de71be79d86de20eb31</td><td>medrxiv</td><td>Public Exposure to Live Animals, Behavioural Change, and Support in Containment Measures in response to COVID-19 Outbreak: a population-based cross sectional survey in China</td><td>10.1101/2020.02.21.20026146</td><td>null</td><td>null</td><td>medrvix</td><td>Background In response to the COVID-19 outbreak, we aimed to investigate behavioural change on exposure to live animals before and during the outbreak, and public support and confidence for governmental containment measures. Methods A population-based cross-sectional telephone survey via random dialing was conducted in Wuhan (the epicentre) and Shanghai (an affected city with imported cases) between 1 and 10 February, 2020. 510 residents in Wuhan and 501 residents in Shanghai were randomly sampled. Differences of outcome measures were compared before and during the outbreak, and between two cities. Findings Proportion of respondents visiting wet markets at usual was 23.3% (119/510) in Wuhan and 20.4% (102/501) in Shanghai. During the outbreak, it decreased to 3.1% (16) in Wuhan (p<0.001), and 4.4% (22) in Shanghai (p<0.001). Proportion of those consuming wild animal products declined from 10.2% (52) to 0.6% (3) in Wuhan (p<0.001), and from 5.2% (26) to 0.8% (4) in Shanghai (p<0.001). 79.0% (403) of respondents in Wuhan and 66.9% (335) of respondents in Shanghai supported permanent closure of wet markets (P<0.001). 95% and 92% of respondents supported banning wild animal trade and quarantining Wuhan, and 75% were confident towards containment measures. Females and the more educated were more supportive for the above containment measures. Interpretation The public responded quickly to the outbreak, and reduced exposure to live animals, especially in Wuhan. With high public support in containment measures, better regulation of wet markets and healthy diets should be promoted.</td><td>2020-02-23</td><td>Zhiyuan Hou; Leesa Lin; Liang Lu; Fanxing Du; Mengcen Qian; Yuxia Liang; Juanjuan Zhang; Hongjie Yu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026146</td></tr><tr><td>t05ummop</td><td>d12b31daa935e9043a9f526a2547b6216ada7cde</td><td>medrxiv</td><td>2019 Novel Coronavirus can be detected in urine, blood, anal swabs and oropharyngeal swabs samples</td><td>10.1101/2020.02.21.20026179</td><td>null</td><td>null</td><td>medrvix</td><td>We tested samples collected from nine patients diagnosed with coronavirus disease 2019 (COVID-19). The virus was found in urine, blood, anal swabs and oropharyngeal swabs. It is the first time for SARS-CoV-2 found in urine, though no urinary irritation was found.</td><td>2020-02-25</td><td>Liang Peng; Jing Liu; Wenxiong Xu; Qiumin Luo; Keji Deng; Bingliang Lin; Zhiliang Gao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026179</td></tr><tr><td>z4u017sa</td><td>null</td><td>medrxiv</td><td>Comparison of throat swabs and sputum specimens for viral nucleic acid detection in 52 cases of novel coronavirus (SARS-Cov-2) infected pneumonia (COVID-19)</td><td>10.1101/2020.02.21.20026187</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract Background: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Diagnostic test based on real-time reverse transcription polymerase chain reaction assay (qRT-PCR) was the main means of confirmation, and sample collection was mostly throat swabs, which was easy to miss the diagnosis. It is necessary to seek specimen types with higher detection efficiency and accuracy. Methods: Paired specimens of throat swabs and sputum were obtained from 54 cases, and RNA was extracted and tested for 2019-nCoV (equated with SARS-CoV-2) by qRT-PCR assay. Results: The positive rates of 2019-nCoV from sputum specimens and throat swabs were 76.9% and 44.2%, respectively. Sputum specimens showed a significantly higher positive rate than throat swabs in detecting viral nucleic acid using qRT-PCR assay (P=0.001). Conclusions: The detection rates of 2019-nCoV from sputum specimens are significantly higher than throat swabs. We suggest that sputum would benefit for the detection of 2019-nCoV in patients who produce sputum. The results can facilitate the selection of specimens and increase the accuracy of diagnosis.</td><td>2020-02-23</td><td>Chenyao Lin; Jie Xiang; Mingzhe Yan; Hongze Li; Shuang Huang; Changxin Shen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026187</td></tr><tr><td>2vvqczj7</td><td>null</td><td>medrxiv</td><td>Epidemiological Development of Novel Coronavirus Pneumonia in China and Its Forecast</td><td>10.1101/2020.02.21.20026229</td><td>null</td><td>null</td><td>medrvix</td><td>ABSTRACT BACKGROUND AND OBJECTIVE The novel coronavirus (SARS-Cov-2) infected coronavirus disease 2019 (COVID-19) was broken out in Wuhan and Hubei province for more than a month. It severely threats people's health of thousands in Chin and even other countries. In order to prevent its wide spread, it is necessary to understand the development of the epidemic with precise mathematical language. METHODS The various data of novel coronavirus pneumonia were collected from the official websites of the National Health Committee of the People's Republic of China. According to epidemic and administrative division, three groups were divided to analyze the data, Hubei Province (including Wuhan), nationwide without Hubei and Henan Province. With classic SIR models, the fitting epidemiological curves of incidence have made, and basic reproduction number (R0) was also calculated as well. Therefore the disease's infection intensity, peak time and the epidemiological end time can be deduced.</td><td>2020-02-26</td><td>Shan shan Wu; Pan pan Sun; Rui ling Li; Liang Zhao; Yan li Wang; Li fang Jiang; Jin Bo Deng</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026229</td></tr><tr><td>ragcpbl6</td><td>87b89b038b28d2ba0bcfd2203845bf15f922c1b5</td><td>medrxiv</td><td>Evolving epidemiology of novel coronavirus diseases 2019 and possible interruption of local transmission outside Hubei Province in China: a descriptive and modeling study</td><td>10.1101/2020.02.21.20026328</td><td>null</td><td>null</td><td>medrvix</td><td>Background The COVID-19 epidemic originated in Wuhan City of Hubei Province in December 2019 and has spread throughout China. Understanding the fast evolving epidemiology and transmission dynamics of the outbreak beyond Hubei would provide timely information to guide intervention policy. Methods We collected individual information on 8,579 laboratory-confirmed cases from official publically sources reported outside Hubei in mainland China, as of February 17, 2020. We estimated the temporal variation of the demographic characteristics of cases and key time-to-event intervals. We used a Bayesian approach to estimate the dynamics of the net reproduction number (Rt) at the provincial level. Results The median age of the cases was 44 years, with an increasing of cases in younger age groups and the elderly as the epidemic progressed. The delay from symptom onset to hospital admission decreased from 4.4 days (95%CI: 0.0-14.0) until January 27 to 2.6 days (0.0-9.0) from January 28 to February 17. The mean incubation period was estimated at 5.2 days (1.8-12.4) and the mean serial interval at 5.1 days (1.3-11.6). The epidemic dynamics in provinces outside Hubei was highly variable, but consistently included a mix of case importations and local transmission. We estimate that the epidemic was self-sustained for less than three weeks with Rt reaching peaks between 1.40 (1.04-1.85) in Shenzhen City of Guangdong Province and 2.17 (1.69-2.76) in Shandong Province. In all the analyzed locations (n=10) Rt was estimated to be below the epidemic threshold since the end of January. Conclusion Our findings suggest that the strict containment measures and movement restrictions in place may contribute to the interruption of local COVID-19 transmission outside Hubei Province. The shorter serial interval estimated here implies that transmissibility is not as high as initial estimates suggested.</td><td>2020-02-23</td><td>Juanjuan Zhang; Maria Litvinova; Wei Wang; Yan Wang; Xiaowei Deng; Xinghui Chen; Mei Li; Wen Zheng; Lan Yi; Xinhua Chen; Qianhui Wu; Yuxia Liang; Xiling Wang; Juan Yang; Kaiyuan Sun; Ira M. Longini; M. Elizabeth Halloran; Peng Wu; Benjamin J. Cowling; Stefano Merler; Cecile Viboud; Alessandro Vespignani; Marco Ajelli; Hongjie Yu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026328</td></tr><tr><td>vw3h1m5y</td><td>b842a645116edb86df997aa960699a7513f2196a</td><td>medrxiv</td><td>Real-time monitoring the transmission potential of COVID-19 in Singapore, February 2020</td><td>10.1101/2020.02.21.20026435</td><td>null</td><td>null</td><td>medrvix</td><td>The ongoing COVID-19 epidemic that spread widely in China since December 2019 is now generating local transmission in multiple countries including Singapore as of March 5, 2020. This highlights the need to monitor in real time the transmission potential of COVID-19. In Singapore, four major COVID-19 case clusters have emerged thus far. Here we estimate the effective reproduction number, Rt, of COVID-19 in Singapore from the publicly available daily case series of imported and autochthonous cases by date of symptoms onset, after adjusting the local cases for reporting delays. We also derive the reproduction number from the distribution of cluster sizes using a branching process analysis. The effective reproduction number peaked with a mean value ~1.0 around February 6-12, 2020 and declined thereafter. As of March 5, 2020, our most recent estimate of Rt is at 0.9 (95% CI: 0.7,1.0) while an estimate of the overall R based on cluster size distribution is at 0.7 (95% CI: 0.5, 1.0). The trajectory of the reproduction number in Singapore underscore the significant effects of containment efforts in Singapore while at the same time suggest the need to sustain social distancing and active case finding efforts to stomp out all active chains of transmission.</td><td>2020-02-25</td><td>Amna Tariq; Yiseul Lee; Kimberlyn Roosa; Seth Blumberg; Ping Yan; Stefan Ma; Gerardo Chowell</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026435</td></tr><tr><td>eedk54xf</td><td>29bc34aad802ab6b35074f23c631540a84b16d0d</td><td>medrxiv</td><td>Trends in Transmissibility of 2019 Novel Coronavirus-infected Pneumonia in Wuhan and 29 Provinces in China</td><td>10.1101/2020.02.21.20026468</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The 2019 novel coronavirus infected pneumonia (COVID- 19) represents a significant public health threat. The COVID-19 emerged in December 2019 in Wuhan, China and rapidly spread to other regions and countries. The variation in transmission patterns and disease spread in regard to time or among different locations, partially reflecting the public health in- tervention effects, remains to be quantified. As most transmissibility-related epidemic parameters are unknown, we sought, with minimal assumptions, to estimate real-time transmissibility and forecast new cases using dynamic modelling. Method: Using the cases reported from the National Health Commission of China and transportation data, including the total number of travelling hours through railway, airplane, and car outbound from Wuhan, we have built a time-series model to estimate real-time underlying transmission rates of newly generated cases sequentially from January 20, 2020 to Feb 13, 2020 in Wuhan, Hubei province and other 28 provinces in China. We quantified the instantaneous transmission rate and relative reproduction number (R_t) of COVID-19, and evaluated whether public health intervention affected the case transmissibility in each province. Based on the current estimates, we have predicted the trend of disease spread with a high level of certainty. Findings: We estimated that R_t declined from the range of 4 to 5 towards 1 and remained below unity, while there was an initial growth followed by a decline in a shorter period in Hubei and other provinces. The ratio of transmission rates decreased dramatically from January 23 to 27 likely due to the rigorous public health intervention implemented by the government beginning on January 23, 2020. The mean duration of the infectious period was 6 to 9 days. We have predicted the trend of infection sizes which be- came stable in provinces around February 19 to 24, 2020, and the date of containment would be one-week later in Wuhan. Interpretation: Public health interventions implemented at both the so- cial and personal levels are effective in preventing outbreaks of COVID-19 in Wuhan and other provinces. Model prediction results suggested that COVID- 19 will be contained around the end of February 2020 in China. Keywords: Coronavirus, COVID-19, transmissibility, dynamic reproduction number R, statistical modelling, pneumonia outbreak</td><td>2020-02-25</td><td>Huazhen Lin; Wei Liu; Hong Gao; Jinyu Nie; Qiao Fan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026468</td></tr><tr><td>jydxk6j1</td><td>null</td><td>medrxiv</td><td>Estimating the serial interval of the novel coronavirus disease (COVID-19): A statistical analysis using the public data in Hong Kong from January 16 to February 15, 2020</td><td>10.1101/2020.02.21.20026559</td><td>null</td><td>null</td><td>medrvix</td><td>Backgrounds: The emerging virus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has caused a large outbreak of novel coronavirus disease (COVID-19) in Wuhan, China since December 2019. Based on the publicly available surveillance data, we identified 21 transmission chains in Hong Kong and estimated the serial interval (SI) of COVID-19. Methods: Index cases were identified and reported after symptoms onset, and contact tracing was conducted to collect the data of the associated secondary cases. An interval censored likelihood framework is adopted to fit a Gamma distribution function to govern the SI of COVID-19. Findings: Assuming a Gamma distributed model, we estimated the mean of SI at 4.4 days (95%CI: 2.9−6.7) and SD of SI at 3.0 days (95%CI: 1.8−5.8) by using the information of all 21 transmission chains in Hong Kong. Conclusion: The SI of COVID-19 may be shorter than the preliminary estimates in previous works. Given the likelihood that SI could be shorter than the incubation period, pre-symptomatic transmission may occur, and extra efforts on timely contact tracing and quarantine are recommended in combating the COVID-19 outbreak.</td><td>2020-02-25</td><td>Shi Zhao; Daozhou Gao; Zian Zhuang; Marc Chong; Yongli Cai; Jinjun Ran; Peihua Cao; Kai Wang; Yijun Lou; Weiming Wang; Lin Yang; Daihai He; Maggie Wang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.20026559</td></tr><tr><td>if9lar1c</td><td>null</td><td>biorxiv</td><td>Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform</td><td>10.1101/2020.02.21.959817</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractReverse genetics has been an indispensable tool revolutionising our insights into viral pathogenesis and vaccine development. Large RNA virus genomes, such as from Coronaviruses, are cumbersome to clone and to manipulate in E. coli hosts due to size and occasional instability1-3. Therefore, an alternative rapid and robust reverse genetics platform for RNA viruses would benefit the research community. Here we show the full functionality of a yeast-based synthetic genomics platform for the genetic reconstruction of diverse RNA viruses, including members of the Coronaviridae, Flaviviridae and Paramyxoviridae families. Viral subgenomic fragments were generated using viral isolates, cloned viral DNA, clinical samples, or synthetic DNA, and reassembled in one step in Saccharomyces cerevisiae using transformation associated recombination (TAR) cloning to maintain the genome as a yeast artificial chromosome (YAC). T7-RNA polymerase has been used to generate infectious RNA, which was then used to rescue viable virus. Based on this platform we have been able to engineer and resurrect chemically-synthetized clones of the recent epidemic SARS-CoV-24 in only a week after receipt of the synthetic DNA fragments. The technical advance we describe here allows to rapidly responding to emerging viruses as it enables the generation and functional characterization of evolving RNA virus variants - in real-time - during an outbreak.</td><td>2020-02-21</td><td>Tran Thi Nhu Thao; Fabien Labroussaa; Nadine Ebert; Philip V’kovski; Hanspeter Stalder; Jasmine Portmann; Jenna Kelly; Silvio Steiner; Melle Holwerda; Annika Kratzel; Mitra Gultom; Laura Laloli; Linda Hüsser; Manon Wider; Stephanie Pfaender; Dagny Hirt; Valentina Cippà; Silvia Crespo-Pomar; Simon Schröder; Doreen Muth; Daniela Niemeyer; Marcel A. Müller; Christian Drosten; Ronald Dijkman; Joerg Jores; Volker Thiel</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.959817</td></tr><tr><td>v0yzur5h</td><td>cc41c51293ed24a57ddf467e462ec9399463ee33</td><td>biorxiv</td><td>No more business as usual: agile and effective responses to emerging pathogen threats require open data and open analytics</td><td>10.1101/2020.02.21.959973</td><td>null</td><td>null</td><td>biorxiv</td><td>The current state of much of the Wuhan pneumonia virus (COVID-19) research shows a regrettable lack of data sharing and considerable analytical obfuscation. This impedes global research cooperation, which is essential for tackling public health emergencies, and requires unimpeded access to data, analysis tools, and computational infrastructure. Here we show that community efforts in developing open analytical software tools over the past ten years, combined with national investments into scientific computational infrastructure, can overcome these deficiencies and provide an accessible platform for tackling global health emergencies in an open and transparent manner. Specifically, we use all COVID-19 genomic data available in the public domain so far to (1) underscore the importance of access to raw data and to (2) demonstrate that existing community efforts in curation and deployment of biomedical software can reliably support rapid, reproducible research during global health crises. All our analyses are fully documented at https://github.com/galaxyproject/SARS-CoV-2.</td><td>2020-02-22</td><td>Anton Nekrutenko; Sergei L Kosakovsky Pond</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.959973</td></tr><tr><td>ppl3xnm9</td><td>null</td><td>biorxiv</td><td>Molecular Survey for Selected Viral Pathogens in Wild Leopard Cats (Prionailurus bengalensis) in Taiwan with an Emphasis on the Spatial and Temporal Dynamics of Carnivore Protoparvovirus 1</td><td>10.1101/2020.02.21.960492</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe leopard cat (Prionailurus bengalensis) has been listed as an endangered species under the Wildlife Conservation Act in Taiwan since 2009. In this study, we targeted viral pathogens, included carnivore protoparvovirus 1 (CPPV-1), feline leukemia virus (FeLV), feline immunodeficiency virus (FIV), coronavirus (CoV), and canine morbillivirus (CMV), using molecular screening. The spatial and temporal dynamics of the target pathogens were evaluated. Through sequencing and phylogenetic analysis, we aimed to clarify the phylogenetic relationship of isolated viral pathogens between leopard cats and domestic carnivores. Samples from 23 and 29 leopard cats that were live-trapped and found dead, respectively, were collected from Miaoli County from 2015 to 2019 in northwestern Taiwan. CPPV-1 and coronavirus were detected in leopard cats. The prevalence (95% confidence interval) of CPPV-1, and CoV was 63.5% (50.4%–76.6%) and 8.8% (0%–18.4%), respectively. The majority of sequences of each CPPV-1 strain amplified from Taiwanese leopard cats and domestic carnivores were identical. All the amplified CoV sequences from leopard cats were identified as feline coronavirus. The spatial and temporal aggregation of CPPV-1 infection in leopard cats was not determined in the sampling area, which indicated a wide distribution of CPPV-1 in the leopard cat habitat. We consider sympatric domestic carnivores to be the probable primary reservoir for the pathogens identified. We strongly recommend establishing efforts to manage CPPV-1 and FCoV in the leopard cat habitat, with an emphasis on vaccination programs and population control measures for free-roaming dogs and cats.IMPORTANCEThe leopard cat (Prionailurus bengalensis) is an endangered species in Taiwan. The effects of infectious diseases on the wildlife population have increasingly been recognized. In this study, we targeted highly pathogenic viral pathogens in wild cat species, included carnivore protoparvovirus 1 (CPPV-1), feline leukemia virus (FeLV), feline immunodeficiency virus (FIV), coronavirus (CoV), and canine morbillivirus (CMV), using molecular screening. Furthermore, we collected the epidemiological and phylogenetic data to understand the spatial and temporal dynamics of the target pathogens in the wild leopard cat population and identified the possible origin of target pathogens. Based on our study, we consider sympatric domestic carnivores to be the probable primary reservoir for the pathogens identified. Our study provides a deeper understanding related to the distribution of target viral pathogens in the wild leopard cats. The information is essential for leopard cat conservation and pathogen management.</td><td>2020-02-23</td><td>Chen-Chih Chen; Ai-Mei Chang; Wan-Jhen Chen; Po-Jen Chang; Yu-Ching Lai; Hsu-Hsun Lee</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.21.960492</td></tr><tr><td>o3hytzwu</td><td>54d2fd5689ac3255e8041c36b5fb333a3bd3f9b2</td><td>medrxiv</td><td>A Novel Method for the Estimation of a Dynamic Effective Reproduction Number (Dynamic-R) in the CoViD-19 Outbreak</td><td>10.1101/2020.02.22.20023267</td><td>null</td><td>null</td><td>medrvix</td><td>The CoViD-19 outbreak has escalated to a pandemic in the last few weeks, with no signs of stopping. Pharmaceutical solutions based upon virologic studies, at this point, remain inconclusive. In contrast, this paper looks towards epidemiological models during this phase of viral growth, in particular, by providing a responsive, timely model of the R value based on the previous few days' results. Such an R value, although bearing less statistical precision due to limited sampling, could allow R to become a more effective, responsive standalone measure of infectious transmission. It demonstrates that the R value can be used as a dynamic, time-dependent indicator without the use of curve-fitting, and also estimates the most recent R-value of the CoViD-19 outbreak to be about 4.29, based on the data from the previous 3 days.</td><td>2020-02-25</td><td>Yi Chen Chong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.20023267</td></tr><tr><td>y24qtqjj</td><td>4f8d24c531d2c334969e09e4b5aed66dcc925c4b</td><td>medrxiv</td><td>Association of radiologic findings with mortality of patients infected with 2019 novel coronavirus in Wuhan, China</td><td>10.1101/2020.02.22.20024927</td><td>null</td><td>null</td><td>medrvix</td><td>Radiologic characteristics of 2019 novel coronavirus (2019-nCoV) infected pneumonia (NCIP) which had not been fully understood are especially important for diagnosing and predicting prognosis. We retrospective studied 27 consecutive patients who were confirmed NCIP, the clinical characteristics and CT image findings were collected, and the association of radiologic findings with mortality of patients was evaluated. 27 patients included 12 men and 15 women, with median age of 60 years (IQR 47-69). 17 patients discharged in recovered condition and 10 patients died in hospital. The median age of mortality group was higher compared to survival group (68 (IQR 63-73) vs 55 (IQR 35-60), P = 0.003). The comorbidity rate in mortality group was significantly higher than in survival group (80% vs 29%, P = 0.018). The predominant CT characteristics consisted of ground glass opacity (67%), bilateral sides involved (86%), both peripheral and central distribution (74%), and lower zone involvement (96%). The median CT score of mortality group was higher compared to survival group (30 (IQR 7-13) vs 12 (IQR 11-43), P = 0.021), with more frequency of consolidation (40% vs 6%, P = 0.047) and air bronchogram (60% vs 12%, P = 0.025). An optimal cutoff value of a CT score of 24.5 had a sensitivity of 85.6% and a specificity of 84.5% for the prediction of mortality. 2019-nCoV was more likely to infect elderly people with chronic comorbidities. CT findings of NCIP were featured by predominant ground glass opacities mixed with consolidations, mainly peripheral or combined peripheral and central distributions, bilateral and lower lung zones being mostly involved. A simple CT scoring method was capable to predict mortality.</td><td>2020-02-23</td><td>Mingli Yuan; Wen Yin; Zhaowu Tao; Weijun Tan; Yi Hu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.20024927</td></tr><tr><td>pi5uq90z</td><td>5a9164999237a46d3d6aed53bd59b17d91a9e6d0</td><td>medrxiv</td><td>SARS-CoV-2 transmission in cancer patients of a tertiary hospital in Wuhan</td><td>10.1101/2020.02.22.20025320</td><td>null</td><td>null</td><td>medrvix</td><td>In December 2019, an outbreak of atypical pneumonia known as 2019 novel coronavirus disease (COVID-19) occurred in Wuhan, China. This new type of pneumonia is characterized by rapid human-to-human transmission. Among the different disease types, cancer patients are often recalled to the hospital for treatment and disease surveillance, and the majority of cancer treatments such as chemotherapy and radiotherapy are immunosuppressive. This prompts us to consider if cancer patients were at an elevated risk of SARS-CoV-2 infection. A total of 1,524 cancer patients who were managed at our tertiary cancer institution-Zhongnan hospital of Wuhan University were reviewed during the period of Dec 30, 2019 to Feb 17, 2020. It was found that cancer patients had an estimated 2-fold increased risk of COVID-19 than the general population. We identified twelve patients who were infected with SARS-CoV-2, with two recorded deaths (16.7%), albeit one patient passed away from a COVID-19 unrelated cause. Interestingly, only five of these patients were ongoing treatment at the time of contracting the virus, suggesting that hospital visitation was the likely factor contributing to the elevated incidence in cancer patients. Moreover, we also observed that the incidence of severe COVID-19 was not higher than in the general population. Consequently, for cancer patients who require treatment, proper isolation protocols must be in place to mitigate the risk of SARS-CoV-2 infection.</td><td>2020-02-25</td><td>Jing Yu; Wen Ouyang; Melvin L.K. Chua; Conghua Xie</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.20025320</td></tr><tr><td>hth8f5sn</td><td>04030bba3035a58c7725ae267973206f6eb6c0b4</td><td>medrxiv</td><td>Development and Evaluation of A CRISPR-based Diagnostic For 2019-novel Coronavirus</td><td>10.1101/2020.02.22.20025460</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The recent outbreak of infections by the 2019 novel coronavirus (2019-nCoV), the third zoonotic CoV has raised great public health concern. The demand for rapid and accurate diagnosis of this novel pathogen brought significant clinical and technological challenges. Currently, metagenomic next-generation sequencing (mNGS) and reverse-transcription PCR (RT-PCR) are the most widely used molecular diagnostics. Methods: 2019-nCoV infections were confirmed in 52 specimens by mNGS. Genomic information was analyzed and used for the design and development of an isothermal, CRISPR-based diagnostic for this novel virus. The diagnostic performance of CRISPR-nCoV was assessed and compared across three technology platforms (mNGS, RT-PCR and CRISPR). Results: 2019-nCoVs sequenced in our study were conserved with the Wuhan strain, and shared certain genetic similarity with SARS-CoV. A high degree of variation in the level of viral RNA was observed in clinical specimens. CRISPR-nCoV demonstrated a near single-copy sensitivity and great clinical sensitivity with a shorter turn-around time than RT-PCR. Conclusion: CRISPR-nCoV presents as a promising diagnostic option for the emerging pathogen.</td><td>2020-02-23</td><td>Tieying Hou; Weiqi Zeng; Minling Yang; Wenjing Chen; Lili Ren; Jingwen Ai; Ji Wu; Yalong Liao; Xuejing Gou; Yongjun Li; Xiaorui Wang; Hang Su; Jianwei Wang; Bing Gu; Teng Xu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.20025460</td></tr><tr><td>qc1mpzli</td><td>null</td><td>medrxiv</td><td>Temperature significant change COVID-19 Transmission in 429 cities</td><td>10.1101/2020.02.22.20025791</td><td>null</td><td>null</td><td>medrvix</td><td>Background There is no evidence supporting that temperature changes COVID-19 transmission. Methods We collected the cumulative number of confirmed cases of all cities and regions affected by COVID-19 in the world from January 20 to February 4, 2020, and calculated the daily means of the average, minimum and maximum temperatures in January. Then, restricted cubic spline function and generalized linear mixture model were used to analyze the relationships. Results There were in total 24,232 confirmed cases in China and 26 overseas countries. In total, 16,480 cases (68.01%) were from Hubei Province. The lgN rose as the average temperature went up to a peak of 8.72℃ and then slowly declined. The apexes of the minimum temperature and the maximum temperature were 6.70℃ and 12.42℃ respectively. The curves shared similar shapes. Under the circumstance of lower temperature, every 1℃ increase in average, minimum and maximum temperatures led to an increase of the cumulative number of cases by 0.83, 0.82 and 0.83 respectively. In the single-factor model of the higher-temperature group, every 1℃ increase in the minimum temperature led to a decrease of the cumulative number of cases by 0.86. Conclusion The study found that, to certain extent, temperature could significant change COVID-19 transmission, and there might be a best temperature for the viral transmission, which may partly explain why it first broke out in Wuhan. It is suggested that countries and regions with a lower temperature in the world adopt the strictest control measures to prevent future reversal.</td><td>2020-02-25</td><td>Mao Wang; Aili Jiang; Lijuan Gong; Lina Luo; Wenbin Guo; Chuyi Li; Jing Zheng; Chaoyong Li; Bixing Yang; Jietong Zeng; Youping Chen; Ke Zheng; Hongyan Li</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.20025791</td></tr><tr><td>niw61l9r</td><td>7fd337f16780aba8f1e599ce7516dc9b1d80a546</td><td>medrxiv</td><td>Neurological Manifestations of Hospitalized Patients with COVID-19 in Wuhan, China: a retrospective case series study</td><td>10.1101/2020.02.22.20026500</td><td>null</td><td>null</td><td>medrvix</td><td>OBJECTIVE: To study the neurological manifestations of patients with coronavirus disease 2019 (COVID-19). DESIGN: Retrospective case series SETTING: Three designated COVID-19 care hospitals of the Union Hospital of Huazhong University of Science and Technology in Wuhan, China. PARTICIPANTS: Two hundred fourteen hospitalized patients with laboratory confirmed diagnosis of severe acute respiratory syndrome from coronavirus 2 (SARS-CoV-2) infection. Data were collected from 16 January 2020 to 19 February 2020. MAIN OUTCOME MEASURES: Clinical data were extracted from electronic medical records and reviewed by a trained team of physicians. Neurological symptoms fall into three categories: central nervous system (CNS) symptoms or diseases (headache, dizziness, impaired consciousness, ataxia, acute cerebrovascular disease, and epilepsy), peripheral nervous system (PNS) symptoms (hypogeusia, hyposmia, hypopsia, and neuralgia), and skeletal muscular symptoms. Data of all neurological symptoms were checked by two trained neurologists. RESULTS: Of 214 patients studied, 88 (41.1%) were severe and 126 (58.9%) were non-severe patients. Compared with non-severe patients, severe patients were older (58.7 ± 15.0 years vs 48.9 ± 14.7 years), had more underlying disorders (42 [47.7%] vs 41 [32.5%]), especially hypertension (32 [36.4%] vs 19 [15.1%]), and showed less typical symptoms such as fever (40 [45.5%] vs 92 [73%]) and cough (30 [34.1%] vs 77 [61.1%]). Seventy-eight (36.4%) patients had neurologic manifestations. More severe patients were likely to have neurologic symptoms (40 [45.5%] vs 38 [30.2%]), such as acute cerebrovascular diseases (5 [5.7%] vs 1 [0.8%]), impaired consciousness (13 [14.8%] vs 3 [2.4%]) and skeletal muscle injury (17 [19.3%] vs 6 [4.8%]). CONCLUSION: Compared with non-severe patients with COVID-19, severe patients commonly had neurologic symptoms manifested as acute cerebrovascular diseases, consciousness impairment and skeletal muscle symptoms.</td><td>2020-02-25</td><td>Ling Mao; Mengdie Wang; Shanghai Chen; Quanwei He; Jiang Chang; Candong Hong; Yifan Zhou; David Wang; Yanan Li; Huijuan Jin; Bo Hu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.20026500</td></tr><tr><td>tb37l9yp</td><td>null</td><td>medrxiv</td><td>A Peptide-based Magnetic Chemiluminescence Enzyme Immunoassay for Serological Diagnosis of Corona Virus Disease 2019 (COVID-19)</td><td>10.1101/2020.02.22.20026617</td><td>null</td><td>null</td><td>medrvix</td><td>A respiratory illness has been spreading rapidly in China, since its outbreak in Wuhan city, Hubei province in December 2019. The illness was caused by a novel coronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Clinical manifestations related to SARS-CoV-2 infection ranged from no symptom to fatal pneumonia. World Health Organization (WHO) named the diseases associated with SARS-CoV-2 infection as COVID-19. Real time RT-PCR is the only laboratory test available till now to confirm the infection. However, the accuracy of real time RT-PCR depends on many factors, including sampling location and of methods, quality of RNA extraction and training of operators etc. Variations in these factors might significantly lower the sensitivity of the detection. We developed a peptide-based luminescent immunoassay to detect IgG and IgM. Cut-off value of this assay was determined by the detection of 200 healthy sera and 167 sera from patients infected with other pathogens than SARS-CoV-2. To evaluate the performance of this assay, we detected IgG and IgM in the 276 sera from confirmed patients. The positive rate of IgG and IgM were 71.4% (197/276) and 57.2% (158/276) respectively. By combining with real time RT-PCR detection, this assay might help to enhance the accuracy of diagnosis of SARS-CoV-2 infection.</td><td>2020-02-25</td><td>Xuefei Cai; Juan Chen; Jieli Hu; Quanxin Long; Haijun Deng; Kai Fan; Pu Liao; Beizhong Liu; Guicheng Wu; Yaokai Chen; Zhijie Li; Kun Wang; Xiaoli Zhang; Wenguang Tian; Jianglin Xiang; Hongxin Du; Jing Wang; Yuan Hu; Ni Tang; Yong Lin; Jihua Ren; Luyi Huang; Jie Wei; Chunyang Gan; Yanmeng Chen; Qingzhu Gao; Amei Chen; Changlong He; Daoxin Wang; Peng Hu; Fachun Zhou; Ailong Huang; Ping Liu; Deqiang Wang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.20026617</td></tr><tr><td>sjod3q3j</td><td>eeae21c913759e8cf4c345e561457fe7ef84480f</td><td>medrxiv</td><td>Applying chemical reaction transition theory to predict the latent transmission dynamics of coronavirus outbreak in China</td><td>10.1101/2020.02.22.20026815</td><td>null</td><td>null</td><td>medrvix</td><td>The recent outbreak of the Covid-19 suggests a rather long latent phase that precludes public health officials to predict the pandemic transmission on time. Here we apply mass action laws and chemical transition theory to propose a kinetic model that accounts for viral transmission dynamics at the latent phase. This model is useful for authorities to make early preventions and control measurements that stop the spread of a deadly new virus.</td><td>2020-02-25</td><td>Peng Xu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.20026815</td></tr><tr><td>dqxfcwyu</td><td>eb7fc5004191e8fce0d6e4ff98de6865872e665f</td><td>biorxiv</td><td>Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2</td><td>10.1101/2020.02.22.951178</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractCoronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the β-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2.</td><td>2020-02-27</td><td>Tamina Park; Sang-Yeop Lee; Seil Kim; Mi Jeong Kim; Hong Gi Kim; Sangmi Jun; Seung Il Kim; Bum Tae Kim; Edmond Changkyun Park; Daeui Park</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.951178</td></tr><tr><td>nbu6bsb3</td><td>7207df5b61e93fa5a2b6e720c91aed05270e81be</td><td>biorxiv</td><td>A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2</td><td>10.1101/2020.02.22.961268</td><td>null</td><td>null</td><td>biorxiv</td><td>1AbstractThe ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.</td><td>2020-02-27</td><td>Zhen Zhao; Haodong Cui; Wenxing Song; Xiaoling Ru; Wenhua Zhou; Xuefeng Yu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.22.961268</td></tr><tr><td>ne0qwv56</td><td>8b2e50eb3ea84225580fbdbccccb4fcd3f062feb</td><td>medrxiv</td><td>Intrinsic growth rules of patients infected, dead and cured with 2019 novel coronavirus in mainland China</td><td>10.1101/2020.02.23.20024802</td><td>null</td><td>null</td><td>medrvix</td><td>Background An outbreak of a novel coronavirus (SARS-CoV-2)-infected pneumonia (COVID-19) was first occurred in Wuhan, China, in December 2019 and then spread rapidly to other regions. It has been declared that at least one confirmed case infected by SARS-CoV-2 was found in each province of China by late January 2020. Methods We collected the time series data of the cumulative number of confirmed infected, dead, and cured cases from the health commissions in 31 provinces in mainland China. A simple descriptive model in a logistic form was formulated to infer the intrinsic epidemic rules of COVID-19. Furthermore, we compared the intrinsic epidemic rules of COVID-19 in Hubei with that of the severe acute respiratory syndrome (SARS) in Beijing, which was obtained from the Ministry of Public Health of China in 2003. Results The feature of data of three population groups in all the provinces could be well captured (R-square>0.95) by the proposed descriptive model with slight variation among different provinces. By comparing fitted parameters, we found that the fitted curve of the infected case is the earliest to be saturated and has the lowest semi-saturation period in both COVID-19 and SARS. While all three population groups of SARS are later to saturate and have a longer semi-saturation period than that of COVID-19. Conclusions Our model is robust and stable to depict the intrinsic growth rule for the cumulative number of confirmed infected, dead, and cured cases in 30 provinces, which could depict the epidemic situation in mainland China very well. Despite the virus caused SARS (SARS-CoV) and the virus caused COVID-19 (SARS-CoV-2) are homologous, their epidemiologic characteristics are quite different. The duration of the outbreak would be shorter for COVID-19.</td><td>2020-02-25</td><td>Chuanliang Han; Yimeng Liu; Saini Yang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.20024802</td></tr><tr><td>oly7a3c2</td><td>null</td><td>medrxiv</td><td>Conjunctival polymerase chain reaction-tests of 2019 novel coronavirus in patients in Shenyang,China</td><td>10.1101/2020.02.23.20024935</td><td>null</td><td>null</td><td>medrvix</td><td>Purpose: The 2019 novel coronavirus(COVID-19) mainly transmitted by person-to-person through inhalation of respiratory droplets. We report the laboratory results of conjunctival PCR-tests and some clinical features of these patients in shenyang China. Design: This is a cross-sectional non-randomized study. Subjects: The study include 14 confirmly diagnosed cases, 16 suspected cases and some medical observed patients. Methods: All patients with diagnosed and suspected COVID-19 were admitted to a designated hospital in Shenyang,China. We collected conjunctival samples of these patients to do the laboratory tests by real time RT-PCR.Medical observed patients were enrolled if they had clinical symptoms. Then we analysed the PCR results and clinical data from eletronic medical records in order to find some relationships. Main Outcome Measures: Clinical condition and PCR results of conjunctival swabs compared with other specimens. Results: One of the identified case coverted from suspected case without typical clinical symptoms. Twenty-two medical observed cases were removed because none of them converted to identified cases.One of the suspected converted to identified case recently.The included cases in our study are imported cases with less underlying diseases and the severity of their infection was relatively moderate. All the conjunctival results of PCR-test were negative. Two cases had typical clinical symptoms but were finally confirmed by repeated pharynx swab tests. Conclusion: Conjunctiva may be a transmission way of COVID-19. And ocular conjunctival swabs in combination with PCR test could be a non-invasive,convenient and feasible diagnostic method for identifying the infection of COVID-19. Emphasis on the false-negative results is vital.</td><td>2020-02-25</td><td>li Xu; Xinyue Zhang; Wei Song; Baijun Sun; Jinping Mu; Xue Dong; Bing Wang</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.20024935</td></tr><tr><td>8l1vfsbc</td><td>bbba5ff9ab9eccef99a0bbc6ea557a373158bbcd</td><td>medrxiv</td><td>The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</td><td>10.1101/2020.02.23.20026690</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus SARS-CoV-2, etiological agent of recently named Coronavirus infected disease (COVID-19) by WHO, has caused more than 2, 000 deaths worldwide since its emergency in Wuhan City, Hubei province, China, in December, 2019. The symptoms of COVID-19 varied from modest, mild to acute respiratory distress syndrome (ARDS), and the latter of which is generally associated with deregulated immune cytokine production; however, we currently know little as to the interplay between the extent of clinical symptoms and the compositions of lung immune microenvironment. Here, we comprehensively characterized the lung immune microenvironment with the bronchoalveolar lavage fluid (BALF) from 3 severe and 3 mild COVID-19 patients and 8 previously reported healthy lung controls through single-cell RNA sequence (scRNA-seq) combined with TCR-seq. Our data shows that monocyte-derived FCN1+ macrophages, whereas notFABP4+ alveolar macrophages that represent a predominant macrophage subset in BALF from patients with mild diseases, overwhelm in the severely damaged lungs from patients with ARDS. These cells are highly inflammatory and enormous chemokine producers implicated in cytokine storm. Furthermore, the formation of tissue resident, highly expanded clonal CD8+ T cells in the lung microenvironment of mild symptom patients suggests a robust adaptive immune response connected to a better control of COVID-19. This study first reported the cellular atlas of lung bronchoalveolar immune microenvironment in COVID-19 patients at the single-cell resolution, and unveiled the potential immune mechanisms underlying disease progression and protection in COVID-19.</td><td>2020-02-26</td><td>Minfeng Liao; Yang Liu; Jin Yuan; Yanling Wen; Gang Xu; Juanjuan Zhao; Lin Chen; Jinxiu Li; Xin Wang; Fuxiang Wang; Lei Liu; Shuye Zhang; Zheng Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.20026690</td></tr><tr><td>jxtch47t</td><td>a9c58042555c0d56b8e94bc5558db0c3dd697dcc</td><td>medrxiv</td><td>Epidemiologic and Clinical Characteristics of 91 Hospitalized Patients with COVID-19 in Zhejiang, China: A retrospective, multi-centre case series</td><td>10.1101/2020.02.23.20026856</td><td>null</td><td>null</td><td>medrvix</td><td>Background Recent studies have focused initial clinical and Epidemiologic characteristics on the COVID-19, mainly revealing situation in Wuhan, Hubei. Aim To reveal more data on the epidemiologic and clinical characteristics of COVID-19 patients outside of Wuhan, in Zhejiang, China. Design Retrospective case series. Methods 88 cases of laboratory-confirmed and 3 cases of clinical-confirmed COVID-19 were admitted to five hospitals in Zhejiang province, China. Data were collected from 20 January 2020 to 11 February 2020. Results Of all 91 patients, 88 (96.70%) were laboratory-confirmed COVID-19 with throat swab samples that tested positive for SARS-Cov-2 while 3 (3.30%) were clinical-diagnosed COVID-19 cases. The median age of the patients was 50 (36.5-57) years, and female accounted for 59.34%. In this sample 40 (43.96%) patients had contracted the diseases from local cases, 31 (34.07%) patients had been to Wuhan/Hubei, 8 (8.79%) cases had contacted with people from Wuhan, 11 (12.09%) cases were confirmed aircraft transmission. In particular within the city of Ningbo, 60.52% cases can be traced back to an event held in a temple. The most common symptoms were fever (71.43%), cough (60.44%) and fatigue (43.96%). The median of incubation period was 6 (IQR, 3-8) days and the median time from first visit to a doctor to confirmed diagnosis was 1 (1-2) days. According to the Chest computed tomography scans, 67.03% cases had bilateral pneumonia. Conclusions Social activity cluster, family cluster and travel by airplane were how COVID-19 patients get transmitted and could be rapidly diagnosed COVID-19 in Zhejiang.</td><td>2020-02-25</td><td>Guo-Qing Qian; Nai-Bin Yang; Feng Ding; Ada Hoi Yan Ma; Zong-Yi Wang; Yue-Fei Shen; Chun-Wei Shi; Xiang Lian; Jin-Guo Chu; Lei Chen; Zhi-Yu Wang; Da-Wei Ren; Guo-Xiang Li; Xue-Qin Chen; Hua-Jiang Shen; Xiao-Min Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.20026856</td></tr><tr><td>oeqf7uk9</td><td>bf39a81c9aab9e0569f33d93d5e203d320dac024</td><td>medrxiv</td><td>Gender differences in patients with COVID-19: Focus on severity and mortality</td><td>10.1101/2020.02.23.20026864</td><td>null</td><td>null</td><td>medrvix</td><td>Importance: The recent outbreak of Novel Coronavirus (SARS-CoV-2) Disease (COVID-19) has put the world on alert, that is reminiscent of the SARS outbreak seventeen years ago. Objective: We aim to compare the severity and mortality between male and female patients with both COVID-19 and SARS, to explore the most useful prognostic factors for individualized assessment. Design, Setting, and Participants: We extracted the data from a case series of 43 hospitalized patients we treated, a public data set of the first 37 cases died of COVID-19 in Wuhan city and 1019 survived patients from six cities in China. We also analyzed the data of 524 patients with SARS, including 139 deaths, from Beijing city in early 2003. Main Outcomes and Measures: Severity and mortality. Results: Older age and high number of comorbidities were associated with higher severity and mortality in patients with both COVID-19 and SARS. The percentages of older age (≥65 years) were much higher in the deceased group than in the survived group in patients with both COVID-19 (83.8 vs. 13.2, P<0.001) and SARS (37.4 vs. 4.9, P<0.001). In the case series, men tend to be more serious than women (P=0.035), although age was comparable between men and women. In the public data set, age was also comparable between men and women in the deceased group or the survived group in patients with COVID-19. Meanwhile, gender distribution was exactly symmetrical in the 1019 survivors of COVID-19. However, the percentage of male were higher in the deceased group than in the survived group (70.3 vs. 50.0, P=0.015). The gender role in mortality was also observed in SARS patients. Survival analysis showed that men (hazard ratio [95% CI] 1.47 [1.05-2.06, P= 0.025) had a significantly higher mortality rate than women in patients with SARS. Conclusions and Relevance: Older age and male gender are risk factors for worse outcome in patients with COVID. While men and women have the same susceptibility to both SARS-CoV-2 and SARS-CoV, men may be more prone to have higher severity and mortality independent of age and susceptibility.</td><td>2020-02-25</td><td>Jian-Min Jin; Peng Bai; Wei He; Fei Wu; Xiao-Fang Liu; De-Min Han; Shi Liu; Jin-Kui Yang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.20026864</td></tr><tr><td>mlyg4hj0</td><td>null</td><td>medrxiv</td><td>Mental health status and coping strategy of medical workers in China during The COVID-19 outbreak</td><td>10.1101/2020.02.23.20026872</td><td>null</td><td>null</td><td>medrvix</td><td>The authors have withdrawn their manuscript whilst they perform additional experiments to test some of their conclusions further. Therefore, the authors do not wish this work to be cited as reference for the project. If you have any questions, please contact the corresponding author.</td><td>2020-02-25</td><td>Chen Siyu; Min Xia; Weiping Wen; Liqian Cui; Weiqiang Yang; Shaokun Liu; Jiahua Fan Fan; Huijun Yue; Shangqing Tang; Bingjie Tang; Xiaoling Li; Lin Chen; Zili Qin; Kexing Lv; Xueqin Guo; Yu Lin; Yihui Wen; Wenxiang Gao; Ying Zheng; Wei Xu; Yun Li; Yang Xu; Li Ling; Wenbin Lei</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.20026872</td></tr><tr><td>zqsy6r4i</td><td>6bdd6867205d9b09f833f01c65f1bbfdae469164</td><td>medrxiv</td><td>Deep learning Enables Accurate Diagnosis of Novel Coronavirus (COVID-19) with CT images</td><td>10.1101/2020.02.23.20026930</td><td>null</td><td>null</td><td>medrvix</td><td>Background A novel coronavirus (COVID-19) has emerged recently as an acute respiratory syndrome. The outbreak was originally reported in Wuhan, China, but has subsequently been spread world-widely. As the COVID-19 continues to spread rapidly across the world, computed tomography (CT) has become essentially important for fast diagnoses. Thus, it is urgent to develop an accurate computer-aided method to assist clinicians to identify COVID-19-infected patients by CT images. Materials and Methods We collected chest CT scans of 88 patients diagnosed with the COVID-19 from hospitals of two provinces in China, 101 patients infected with bacteria pneumonia, and 86 healthy persons for comparison and modeling. Based on the collected dataset, a deep learning-based CT diagnosis system (DeepPneumonia) was developed to identify patients with COVID-19. Results The experimental results showed that our model can accurately identify the COVID-19 patients from others with an excellent AUC of 0.99 and recall (sensitivity) of 0.93. In addition, our model was capable of discriminating the COVID-19 infected patients and bacteria pneumonia-infected patients with an AUC of 0.95, recall (sensitivity) of 0.96. Moreover, our model could localize the main lesion features, especially the ground-glass opacity (GGO) that is of great help to assist doctors in diagnosis. The diagnosis for a patient could be finished in 30 seconds, and the implementation on Tianhe-2 supercompueter enables a parallel executions of thousands of tasks simultaneously. An online server is available for online diagnoses with CT images by http://biomed.nscc-gz.cn/server/Ncov2019. Conclusions The established models can achieve a rapid and accurate identification of COVID-19 in human samples, thereby allowing identification of patients.</td><td>2020-02-25</td><td>Ying Song; Shuangjia Zheng; Liang Li; Xiang Zhang; Xiaodong Zhang; Ziwang Huang; Jianwen Chen; Huiying Zhao; Yusheng Jie; Ruixuan Wang; Yutian Chong; Jun Shen; Yunfei Zha; Yuedong Yang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.20026930</td></tr><tr><td>jtsxpjhc</td><td>e4701bdf5d414ea8ca38a4d8d3f69540805293b0</td><td>medrxiv</td><td>Clinical features and laboratory inspection of novel coronavirus pneumonia (COVID-19) in Xiangyang, Hubei</td><td>10.1101/2020.02.23.20026963</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since December 2019, a novel coronavirus pneumonia (COVID-19) rapidly spread in China, reached multiple continents currently.We aimed to reveal the infectious characteristics of COVID-19 that provide more information for the research of novel coronavirus. Methods: We performed a retrospective study on the clinical characteristics of 128 COVID-19 cases with laboratory-confirmed from Xiangyang No 1 Hospitalad during January 2020 to 16 February 2020. Results: Female patients account for 53.1%. The aged below 20 years that accounts for 1.6% of overall patients. The aged in 21~50, 51~65, over 66 years were accounts for 44.5%, 35.1%,18.8%, respectively. In the difference age spectrum, all severe groups compared with non-severe groups were difference significantly ( P < 0.01 ). Fever ( 89.8% ) and Cough ( 67.2% ) were common clinical symptoms. The rate of patients with sore throats (14.1%) was rare. The rate of chest computed tomography scan showing ground glass opacity in overall, non-severe, severe groups were 63.3%, 60.7%, 76.2%, respectively. White blood cell counts in the normal range of overall patients, but severe group patients were increased significantly ( P < 0.01). Lymphocytes of overall patients were decreased. Alanine transaminase (ALT) and aspartate transaminase (AST) in the normal range of overall patients, but its were elevated in the severe group. Creatinine (CR) and blood urea nitrogen (BUN) of overall patients in the normal range. C-reactive protein (CRP) level of all patients were increased markedly, but it in the severe group was significantly higher than that in the non-severe group ( P < 0.01 ). Conclusions: Our data provide more information that advanced age, lower lymphocytes levels at the diagnosed COVID-19 patients may be a risk factor for unfavourable prognosis. The white blood cells and C-reactive protein level elevated in severe COVID-19 patients may be accompanying bacterial infection. 2019-nCov may be carries a risk factor of impaired liver and kidney function.</td><td>2020-02-25</td><td>Weiliang Cao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.20026963</td></tr><tr><td>25va2cvt</td><td>09b6706748f0c1ae0da436ac2dfac9052b84e4ea</td><td>biorxiv</td><td>Cryo-EM structures of HKU2 and SADS-CoV spike glycoproteins and insights into coronavirus evolution</td><td>10.1101/2020.02.23.961912</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.</td><td>2020-02-24</td><td>Jinfang Yu; Shuyuan Qiao; Runyu Guo; Xinquan Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.23.961912</td></tr><tr><td>nqn1qzcu</td><td>1d882d90109fb4496088342cf2bcaa4fec40cf9f</td><td>medrxiv</td><td>Inferring transmission trees to guide targeting of interventions against visceral leishmaniasis and post-kala-azar dermal leishmaniasis</td><td>10.1101/2020.02.24.20023325</td><td>null</td><td>null</td><td>medrvix</td><td>Understanding of spatiotemporal transmission of infectious diseases has improved significantly in recent years. Advances in Bayesian inference methods for individual-level geo-located epidemiological data have enabled reconstruction of transmission trees and quantification of disease spread in space and time, while accounting for uncertainty in missing data. However, these methods have rarely been applied to endemic diseases or ones in which asymptomatic infection plays a role, for which novel estimation methods are required. Here, we develop such methods to analyse longitudinal incidence data on visceral leishmaniasis (VL), and its sequela, post-kala-azar dermal leishmaniasis (PKDL), in a highly endemic community in Bangladesh. Incorporating recent data on infectiousness of VL and PKDL, we show that while VL cases drive transmission when incidence is high, the contribution of PKDL increases significantly as VL incidence declines (reaching 55% in this setting). Transmission is highly focal: >85% of mean distances from inferred infectors to their secondary VL cases were <300m, and estimated average times from infector onset to secondary case infection were <4 months for 90% of VL infectors, but up to 2.75yrs for PKDL infectors. Estimated numbers of secondary VL cases per VL and PKDL case varied from 0-6 and were strongly correlated with the infector's duration of symptoms. Counterfactual simulations suggest that prevention of PKDL could have reduced VL incidence by up to a quarter. These results highlight the need for prompt detection and treatment of PKDL to achieve VL elimination in the Indian subcontinent and provide quantitative estimates to guide spatiotemporally-targeted interventions against VL.</td><td>2020-02-25</td><td>Lloyd A. C. Chapman; Simon E. F. Spencer; Timothy M. Pollington; Chris P. Jewell; Dinesh Mondal; Jorge Alvar; T. Deirdre Hollingsworth; Mary M. Cameron; Caryn Bern; Graham F. Medley</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20023325</td></tr><tr><td>f21dknmb</td><td>01d162d7fae6aaba8e6e60e563ef4c2fca7b0e18</td><td>medrxiv</td><td>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</td><td>10.1101/2020.02.24.20025437</td><td>null</td><td>null</td><td>medrvix</td><td>Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the \"coronavirus\" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.</td><td>2020-02-26</td><td>Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Yiqing Xue; Chunli Zhang; Guangming Tan; Gang Niu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20025437</td></tr><tr><td>r31pgnu0</td><td>47f65bff75b7752a38ccbb540fbd74955b3addea</td><td>medrxiv</td><td>Demonstration of four entities of appendicitis in China through studying cluster/outbreak</td><td>10.1101/2020.02.24.20026021</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To differentiate different entities of appendicitis through studying cluster/outbreak, ascertain common setting of cluster/outbreak, and provide the epidemiological evidences of infectious etiology of appendicitis. Background: Differential diagnosis and management for perforated appendicitis and non-perforated appendicitis and the infectious etiology of appendicitis are current hot topics. Methods: Field investigation for Tibetan students were carried out and reports published in English and Chinese medical journals were reviewed. Results: The literature review included 473 patients in 7 cluster/outbreaks of appendicitis in 6 provinces and autonomous regions. All the clusters/outbreaks occurred in group living units. We found two classic entities of appendicitis with natural history from non perforated appendicitis to perforated appendicitis and two entities of non-perforated appendicitis . In classic entities, one may represent majority of sporadic patients and the other may represent partial sporadic patients with obvious gastrointestinal manifestation. In entities of non-perforated appendicitis, one was identical to features of sporadic non-perforated appendicitis and the other one is identical to the following Tibetan students and associated with Fusobacterium. The field investigation for 120 Tibetan students with appendicitis showed that the resected appendices exhibited diffuse or focal hemorrhages and infiltration by eosinophils and by lymphocytes. Most patients had normal body temperature, white blood cell count and neutrophil count. This is a new entity of appendicitis. The clusters/outbreaks of appendicitis showed the features of infectious disease in epidemiology. The entity of perforated appendicitis was not found. Conclusion: Studying cluster/outbreak is a good method to differentiate different entities of appendicitis and infectious etiology.</td><td>2020-02-26</td><td>Yi-Tian Guo; De-qiang Ye; Gui-fang Yang; Guo-zhen Liu; Xiao-chen Cui; Shi-Yun Tan; Yi Guo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20026021</td></tr><tr><td>g3asp1ps</td><td>99a46ba28f5811c112876ec77f570d6c10778db6</td><td>medrxiv</td><td>WeChat, a Chinese social media, may early detect the SARS-CoV-2 outbreak in 2019</td><td>10.1101/2020.02.24.20026682</td><td>null</td><td>null</td><td>medrvix</td><td>We plotted daily data on the frequencies of keywords related to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from WeChat, a Chinese social media. Using 'Feidian', Chinese abbreviation for SARS, may detect the SARS-CoV-2 outbreak in 2019 two weeks earlier. WeChat offered a new approach to early detect disease outbreaks.</td><td>2020-02-26</td><td>Wenjun Wang; Yikai Wang; Xin Zhang; Yaping Li; Xiaoli Jia; Shuangsuo Dang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20026682</td></tr><tr><td>rr7r7ef0</td><td>0fd300aefb704c20f32152b97b6194015f1c74e7</td><td>medrxiv</td><td>Characterizing the transmission and identifying the control strategy for COVID-19 through epidemiological modeling</td><td>10.1101/2020.02.24.20026773</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of the novel coronavirus disease, COVID-19, originating from Wuhan, China in early December, has infected more than 70,000 people in China and other countries and has caused more than 2,000 deaths. As the disease continues to spread, the biomedical society urgently began identifying effective approaches to prevent further outbreaks. Through rigorous epidemiological analysis, we characterized the fast transmission of COVID-19 with a basic reproductive number 5.6 and proved a sole zoonotic source to originate in Wuhan. No changes in transmission have been noted across generations. By evaluating different control strategies through predictive modeling and Monte carlo simulations, a comprehensive quarantine in hospitals and quarantine stations has been found to be the most effective approach. Government action to immediately enforce this quarantine is highly recommended.</td><td>2020-02-25</td><td>Ke K. Zhang; Linglin Xie; Lauren Lawless; Huijuan Zhou; Guannan Gao; Chengbin Xue</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20026773</td></tr><tr><td>tkvgzbuk</td><td>5b1a2b3207e3f674b94f6f271b92c29f3a8a6754</td><td>medrxiv</td><td>Clinical and radiographic features of cardiac injury in patients with 2019 novel coronavirus pneumonia</td><td>10.1101/2020.02.24.20027052</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To investigate the correlation between clinical characteristics and cardiac injury of COVID-2019 pneumonia. Methods: In this retrospective, single-center study, 41 consecutive corona virus disease 2019 (COVID-2019) patients (including 2 deaths) of COVID-2019 in Beijing Youan Hospital, China Jan 21 to Feb 03, 2020, were involved in this study. The high risk factors of cardiac injury in different COVID-2019 patients were analyzed. Computed tomographic (CT) imaging of epicardial adipose tissue (EAT) has been used to demonstrate the cardiac inflammation of COVID-2019. Results：Of the 41 COVID-2019 patients, 2 (4.88%), 32 (78.05%), 4 (9.75%) and 3 (7.32%) patients were clinically diagnosed as light, mild, severe and critical cases, according to the 6th guidance issued by the National Health Commission of China. 10 (24.4%) patients had underlying complications, such as hypertension, CAD, type 2 diabetes mellites and tumor. The peak value of TnI in critical patients is 40-fold more than normal value. 2 patients in the critical group had the onset of atrial fibrillation, and the peak heart rates reached up to 160 bpm. CT scan showed low EAT density in severe and critical patients. Conclusion: Our results indicated that cardiac injury of COVID-2019 was rare in light and mild patients, while common in severe and critical patients. Therefore, the monitoring of the heart functions of COVID-2019 patients and applying potential interventions for those with abnormal cardiac injury related characteristics, is vital to prevent the fatality.</td><td>2020-02-27</td><td>Hui Hui; Yingqian Zhang; Xin Yang; Xi Wang; Bingxi He; Li Li; Hongjun Li; Jie Tian; Yundai Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20027052</td></tr><tr><td>b2znv6pa</td><td>d9eeeb81d17be6a722b11081f13576de197d249f</td><td>medrxiv</td><td>2019 novel coronavirus disease in hemodialysis (HD) patients: Report from one HD center in Wuhan, China</td><td>10.1101/2020.02.24.20027201</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of COVID-19 originated in Wuhan has become a global epidemic of contagious diseases, which poses a serious threat to human life and health, especially for those with underlined diseases. However, Impacts of COVID-19 epidemic on HD center and HD patients are still unknown. In this report, we reviewed the whole course of the epidemic emerged in the HD center of Renmin Hospital, Wuhan University from January 14, 2020, the day the first case was confirmed, to February 17, 2020, the day the epidemic extinction. There are totally 37 cases among 230 HD patients and 4 cases among 33 staff being diagnosed with COVID-19. The epidemiology, clinical presentation and immune profile of dialysis patients contracted COVID-19 were further studied. We found that the two key measures we took in response to the epidemic, one was upgrading level of prevention and protection on January 21 and the other one starting universal screening, isolating, and distributing the infected cases on February 4, were effective in the epidemic control. No new COVID-19 case had been diagnosed since February 13. During the epidemic, 7 HD patients died, including 6 with COVID-19 and 1 without COVID-19. The presumed causes of death were not directly related to pneumonia, but due to cardiovascular and cerebrovascular diseases, hyperkalemia, etc. Most of the leukocytes in the peripheral blood of the HD patients infected with COVID-19 decreased, and the CT images of the chest mostly showed the ground glass like changes on the right side. The symptoms of most of the patients were mild, and there were no cases admitted to ICU. The frequency of lymphocytes in PBMCs and the serum level of inflammatory cytokines were assessed in HD patients contracted COVID-19 or not, non-HD COVID-19 patients, as well as healthy volunteers. The results showed that lymphocytes of T cell, Th cells, killer T cells, as well as NK cells in PBMCs of HD patients decreased significantly than other groups. HD patients with COVID-19 also displayed remarkable lower serum level of inflammatory cytokines than other COVID-19 patients. Our study indicates that HD patients are the highly susceptible population and HD centers are high risk area in the outbreak of COVID-19 epidemic. Measures of prevention, protection, screening, isolation, and distribution are essential in the epidemic management and should be taken in the early stage. HD Patients with COVID-19 are mostly clinical mild and unlikely progress to severe pneumonia due to the impaired cellular immune function and incapability of mounting cytokines storm. More attention should be paid to prevent cardiovascular events, which may be the collateral impacts of COVID-19 epidemic on HD patients.</td><td>2020-02-25</td><td>Yiqiong Ma; Bo Diao; Xifeng Lv; Jili Zhu; Wei Liang; Lei Liu; Wenduo Bu; Huiling Cheng; Sihao Zhang; Lianhua Yang; Ming Shi; Guohua Ding; Bo Shen; Huiming Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20027201</td></tr><tr><td>9tgnx7du</td><td>6f53b9dff58818839c45410d11b58f0f3e327d8a</td><td>medrxiv</td><td>Estimation of risk factors for COVID-19 mortality - preliminary results</td><td>10.1101/2020.02.24.20027268</td><td>null</td><td>null</td><td>medrvix</td><td>Since late December 2019 a new epidemic outbreak has emerged from Whuhan, China. Rapidly the new coronavirus has spread worldwide. China CDC has reported results of a descriptive exploratory analysis of all cases diagnosed until the 11th February 2020, presenting the epidemiologic curves and geo-temporal spread of COVID-19 along with case fatality rate according to some baseline characteristics, such as age, gender and several well-established high prevalence comorbidities. Despite this, we intend to increase even further the predictive value of that manuscript by presenting the odds ratio for mortality due to COVID-19 adjusted for the presence of those comorbidities and baseline characteristics such as age and gender. Besides, we present a way to determine the risk of each particular patient, given his characteristics. We found that age is the variable that presents higher risk of COVID-19 mortality, where 60 or older patients have an OR = 18.8161 (CI95%[7.1997; 41.5517]). Regarding comorbidities, cardiovascular disease appears to be the riskiest (OR=12.8328 CI95%[10.2736; 15.8643], along with chronic respiratory disease (OR=7.7925 CI95%[5.5446; 10.4319]). Males are more likely to die from COVID-19 (OR=1.8518 (CI95%[1.5996; 2.1270]). Some limitations such as the lack of information about the correct prevalence of gender per age or about comorbidities per age and gender or the assumption of independence between risk factors are expected to have a small impact on results. A final point of paramount importance is that the equation presented here can be used to determine the probability of dying from COVID-19 for a particular patient, given its age interval, gender and comorbidities associated.</td><td>2020-02-25</td><td>Francisco Caramelo; Nuno Ferreira; Barbara Oliveiros</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20027268</td></tr><tr><td>ad3tct5b</td><td>53326420d7eb607538a248c0da27d3b092f651e2</td><td>medrxiv</td><td>Lessons learnt from 288 COVID-19 international cases: importations over time, effect of interventions, underdetection of imported cases</td><td>10.1101/2020.02.24.20027326</td><td>null</td><td>null</td><td>medrvix</td><td>288 cases have been confirmed out of China from January 3 to February 13, 2020. We collected and synthesized all available information on these cases from official sources and media. We analyzed importations that were successfully isolated and those leading to onward transmission. We modeled their number over time, in relation to the origin of travel (Hubei province, other Chinese provinces, other countries) and interventions. We characterized importations timeline to assess the rapidity of isolation, and epidemiologically linked clusters to estimate the rate of detection. We found a rapid exponential growth of importations from Hubei, combined with a slower growth from the other areas. We predicted a rebound of importations from South East Asia in the upcoming weeks. Time from travel to detection has considerably decreased since the first importation, however 6 cases out of 10 were estimated to go undetected. Countries outside China should be prepared for the possible emergence of several undetected clusters of chains of local transmissions.</td><td>2020-02-25</td><td>Francesco Pinotti; Laura Di Domenico; Ernesto Ortega; Marco Mancastroppa; Giulia Pullano; Eugenio Valdano; Pierre-Yves Boelle; Chiara Poletto; Vittoria Colizza</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20027326</td></tr><tr><td>of9wlhga</td><td>ffbd7555a337706238c211197b221795e4e35146</td><td>medrxiv</td><td>Estimation of COVID-2019 burden and potential for international dissemination of infection from Iran</td><td>10.1101/2020.02.24.20027375</td><td>null</td><td>null</td><td>medrvix</td><td>The Coronavirus Disease 2019 (COVID-19) epidemic began in Wuhan, China in late 2019 and continues to spread globally, with exported cases confirmed in 28 countries at the time of writing. During the interval between February 19 and 23, 2020, Iran reported its first 43 cases with eight deaths. Three exported cases originating in Iran were identified, suggesting a underlying burden of disease in that country than is indicated by reported cases. A large epidemic in Iran could further fuel global dissemination of COVID-19. We sought to estimate COVID-19 outbreak size in Iran based on known exported case counts and air travel links between Iran and other countries, and to anticipate where infections originating in Iran may spread to next. We assessed interconnectivity between Iran and other countries using using International Air Transport Association (IATA) data. We used the methods of Fraser et al. to estimate the size of the underlying epidemic that would result in cases being observed in the United Arab Emirates (UAE), Lebanon, and Canada. Time at risk estimates were based on a presumed 6 week epidemic age, and length of stay data for visitors to Iran derived from the United Nations World Tourism Organization (UNWTO). We evaluated the relationship between the strength of travel links with Iran, and destination country rankings on the Infectious Disease Vulnerability Index (IDVI), a validated metric that estimates the capacity of a country to respond to an infectious disease outbreak . Scores range between 0-1, with higher scores reflecting greater capacity to manage infectious outbreaks. UAE, Lebanon, and Canada ranked 3rd, 21st, and 31st, respectively, for outbound air travel volume from Iran in February 2019. We estimated that 18,300 (95% confidence interval: 3770 to 53,470) COVID-19 cases would have had to occur in Iran, assuming an outbreak duration of 1.5 months in the country, in order to observe these three internationally exported cases reported at the time of writing. Results were robust under varying assumptions about undiagnosed case numbers in Syria, Azerbaijan and Iraq. Even if it were assumed that all cases were identified in all countries with certainty, the \"best case\" outbreak size was substantial (1820, 95% CI: 380-5320 cases), and far higher than reported case counts. Given the low volumes of air travel to countries with identified cases of COVID-19 with origin in Iran (such as Canada), it is likely that Iran is currently experiencing a COVID-19 epidemic of significant size for such exportations to be occurring. This is concerning, both for public health in Iran itself, and because of the high likelihood for outward dissemination of the epidemic to neighbouring countries with lower capacity to respond to infectious diseases epidemics.</td><td>2020-02-25</td><td>Ashleigh R. Tuite; Isaac Bogoch; Ryan Sherbo; Alexander Watts; David N. Fisman; Kamran Khan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20027375</td></tr><tr><td>vkgnwxzc</td><td>c1ae608c7ffb926a0f50a6a34c0780983274ea74</td><td>medrxiv</td><td>Estimate the incubation period of coronavirus 2019 (COVID-19)</td><td>10.1101/2020.02.24.20027474</td><td>null</td><td>null</td><td>medrvix</td><td>Motivation: Wuhan pneumonia is an acute infectious disease caused by the 2019 novel coronavirus (COVID-19). It is being treated as a Class A infectious disease though it was classified as Class B according to the Infectious Disease Prevention Act of China. Accurate estimation of the incubation period of the coronavirus is essential to the prevention and control. However, it remains unclear about its exact incubation period though it is believed that symptoms of COVID-19 can appear in as few as 2 days or as long as 14 or even more after exposure. The accurate incubation period calculation requires original chain-of-infection data that may not be fully available in the Wuhan regions. In this study, we aim to accurately calculate the incubation period of COVID-19 by taking advantage of the chain-of-infection data, which is well-documented and epidemiologically informative, outside the Wuhan regions. Methods: We acquired and collected officially reported COVID-19 data from 10 regions in China except for Hubei province. To achieve the accurate calculation of the incubation period, we only involved the officially confirmed cases with a clear history of exposure and time of onset. We excluded those without relevant epidemiological descriptions, working or living in Wuhan for a long time, or hard to determine the possible exposure time. We proposed a Monte Caro simulation approach to estimate the incubation of COVID-19 as well as employed nonparametric ways. We also employed manifold learning and related statistical analysis to decipher the incubation relationships between different age/gender groups. Result: The incubation period of COVID-19 did not follow general incubation distributions such as lognormal, Weibull, and Gamma distributions. We estimated that the mean and median of its incubation were 5.84 and 5.0 days via bootstrap and proposed Monte Carlo simulations. We found that the incubation periods of the groups with age>=40 years and age<40 years demonstrated a statistically significant difference. The former group had a longer incubation period and a larger variance than the latter. It further suggested that different quarantine time should be applied to the groups for their different incubation periods. Our machine learning analysis also showed that the two groups were linearly separable. incubation of COVID-19 along with previous statistical analysis. Our results further indicated that the incubation difference between males and females did not demonstrate a statistical significance.</td><td>2020-02-29</td><td>Henry Han</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20027474</td></tr><tr><td>mopydeq5</td><td>7b5be48b7769ccad2f44a8c54efcea5281a6c633</td><td>medrxiv</td><td>Spread and control of COVID-19 in China and their associations with population movement, public health emergency measures, and medical resources</td><td>10.1101/2020.02.24.20027623</td><td>null</td><td>null</td><td>medrvix</td><td>ABSTRACT BACKGROUND The COVID-19 epidemic, first emerged in Wuhan during December 2019, has spread globally. While the mass population movement for Chinese New Year has significantly influenced spreading the disease, little direct evidence exists about the relevance to epidemic and its control of population movement from Wuhan, local emergency response, and medical resources in China. METHODS Spearman's correlation analysis was performed between official data of confirmed COVID-19 cases from Jan 20th to Feb 19th, 2020 and real-time travel data and health resources data. RESULTS There were 74,675 confirmed COVID-19 cases in China by Feb 19th, 2020. The overall fatality rate was 2.84%, much higher in Hubei than in other regions (3.27% vs 0.73%). The index of population inflow from Hubei was positively correlated with total (Provincial r=0.9159, p<0.001; City r=0.6311, p<0.001) and primary cases (Provincial r=0.8702, p<0.001; City r=0.6358, p<0.001). The local health emergency measures (eg, city lockdown and traffic control) were associated with reduced infections nationwide. Moreover, the number of public health employees per capita was inversely correlated with total cases (r=-0.6295, p<0.001) and infection rates (r=-0.4912, p<0.01). Similarly, cities with less medical resources had higher fatality (r=-0.4791, p<0.01) and lower cure rates (r=0.5286, p<0.01) among the confirmed cases. CONCLUSIONS The spread of the COVID-19 in China in its early phase was attributed primarily to population movement from Hubei, and effective governmental health emergency measures and adequate medical resources played important roles in subsequent control of epidemic and improved prognosis of affected individuals.</td><td>2020-02-27</td><td>Songmin Ying; Fei Li; Xinwei Geng; Zhouyang Li; Xufei Du; Haixia Chen; Sisi Chen; Min Zhang; Zhehua Shao; Yinfang Wu; Madiha Zahra Syeda; Fugui Yan; Luanqing Che; Bin Zhang; Jian Lou; Shaobin Wang; Zhengming Chen; Wen Li; Ye Shen; Zhihua Chen; Huahao Shen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20027623</td></tr><tr><td>eoxrh2sl</td><td>d7701711485f822c0f43df58b48faf6a433028c4</td><td>medrxiv</td><td>Transmission potential of the novel coronavirus (COVID-19) onboard the Diamond Princess Cruises Ship, 2020</td><td>10.1101/2020.02.24.20027649</td><td>null</td><td>null</td><td>medrvix</td><td>An outbreak of COVID-19 developed aboard the Princess Cruises Ship during January-February 2020. Using mathematical modeling and time-series incidence data describing the trajectory of the outbreak among passengers and crew members, we characterize how the transmission potential varied over the course of the outbreak. Our estimate of the mean reproduction number in the confined setting reached values as high as ~11, which is higher than mean estimates reported from community-level transmission dynamics in China and Singapore (approximate range: 1.1-7). Our findings suggest that Rt decreased substantially compared to values during the early phase after the Japanese government implemented an enhanced quarantine control. Most recent estimates of Rt reached values largely below the epidemic threshold, indicating that a secondary outbreak of the novel coronavirus was unlikely to occur aboard the Diamond Princess Ship.</td><td>2020-02-27</td><td>Kenji Mizumoto; Gerardo Chowell</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.20027649</td></tr><tr><td>0n5bxp93</td><td>null</td><td>biorxiv</td><td>IFN-λ4 increases the risk of gastrointestinal infections and malaria in Malian children</td><td>10.1101/2020.02.24.962688</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTGenetic polymorphisms within the IFNL3/IFNL4 genomic region encoding type III interferons have been strongly associated with impaired clearance of hepatitis C virus (HCV) infection. We hypothesized that this association might extend to the immune response to other pathogens as well. In a cohort of 914 Malian children enrolled at birth, we analyzed episodes of malaria, gastrointestinal and respiratory infections in relation to two genetic polymorphisms functionally affecting type III interferons - rs368234815 (IFN-λ4) and rs4803217 (IFN-λ3), using information for 30,626 clinic visits from birth through 1-5 years of follow-up. Compared to children with the rs368234815-TT/TT genotype (IFN-λ4-Null), each copy of the rs368234815-dG allele was associated with an earlier first episode of a gastrointestinal infection (p=0.004), malaria (p=0.044) or respiratory infection (p=0.045). The risk of ever experiencing an infection during the follow-up was also significantly increased with each copy of the rs368234815-dG allele – for gastrointestinal infections (OR=1.5, 95%CI (1.11-2.03), p=0.0079) and malaria (OR=1.32, 95%CI (1.04-1.68, p=0.022), but not respiratory infections (p=0.58). IFNL4-rs368234815 and IFNL3-rs4803217 were in moderate linkage disequilibrium in this population (r2=0.78). All the associations for rs4803217 were weaker and lost significance after adjusting for rs368234815, implicating IFN-λ4 and not IFN-λ3 as the primary cause of these associations. Thus, our results suggest the role of IFN-λ4 in several infections in young children.</td><td>2020-02-25</td><td>Ludmila Prokunina-Olsson; Robert L. Morrison; Adeola Obajemu; Almahamoudou Mahamar; Sungduk Kim; Oumar Attaher; Oscar Florez-Vargas; Youssoufa Sidibe; Olusegun O. Onabajo; Amy A. Hutchinson; Michelle Manning; Jennifer Kwan; Nathan Brand; Alassane Dicko; Michal Fried; Paul S. Albert; Sam M. Mbulaiteye; Patrick E. Duffy</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.962688</td></tr><tr><td>epy7774j</td><td>c8b6f2752a842dd9eb9c50a82112748ef10ba259</td><td>biorxiv</td><td>Increasing Host Cellular Receptor—Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection</td><td>10.1101/2020.02.24.963348</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe ongoing outbreak of a new coronavirus (2019-nCoV) causes an epidemic of acute respiratory syndrome in humans. 2019-nCoV rapidly spread to national regions and multiple other countries, thus, pose a serious threat to public health. Recent studies show that spike (S) proteins of 2019-nCoV and SARS-CoV may use the same host cell receptor called angiotensin-converting enzyme 2 (ACE2) for entering into host cells. The affinity between ACE2 and 2019-nCoV S is much higher than ACE2 binding to SARS-CoV S protein, explaining that why 2019-nCoV seems to be more readily transmitted from the human to human. Here, we reported that ACE2 can be significantly upregulated after infection of various viruses including SARS-CoV and MERS-CoV. Basing on findings here, we propose that coronavirus infection can positively induce its cellular entry receptor to accelerate their replication and spread, thus drugs targeting ACE2 expression may be prepared for the future emerging infectious diseases caused by this cluster of viruses.</td><td>2020-02-27</td><td>Pei-Hui Wang; Yun Cheng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.24.963348</td></tr><tr><td>z4q0eyjo</td><td>null</td><td>medrxiv</td><td>Deep learning-based model for detecting 2019 novel coronavirus pneumonia on high-resolution computed tomography: a prospective study</td><td>10.1101/2020.02.25.20021568</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Computed tomography (CT) is the preferred imaging method for diagnosing 2019 novel coronavirus (COVID19) pneumonia. Our research aimed to construct a system based on deep learning for detecting COVID-19 pneumonia on high resolution CT, relieve working pressure of radiologists and contribute to the control of the epidemic. Methods: For model development and validation, 46,096 anonymous images from 106 admitted patients, including 51 patients of laboratory confirmed COVID-19 pneumonia and 55 control patients of other diseases in Renmin Hospital of Wuhan University (Wuhan, Hubei province, China) were retrospectively collected and processed. Twenty-seven consecutive patients undergoing CT scans in Feb, 5, 2020 in Renmin Hospital of Wuhan University were prospectively collected to evaluate and compare the efficiency of radiologists against 2019-CoV pneumonia with that of the model. Findings: The model achieved a per-patient sensitivity of 100%, specificity of 93.55%, accuracy of 95.24%, PPV of 84.62%, and NPV of 100%; a per-image sensitivity of 94.34%, specificity of 99.16%, accuracy of 98.85%, PPV of 88.37%, and NPV of 99.61% in retrospective dataset. For 27 prospective patients, the model achieved a comparable performance to that of expert radiologist. With the assistance of the model, the reading time of radiologists was greatly decreased by 65%. Conclusion: The deep learning model showed a comparable performance with expert radiologist, and greatly improve the efficiency of radiologists in clinical practice. It holds great potential to relieve the pressure of frontline radiologists, improve early diagnosis, isolation and treatment, and thus contribute to the control of the epidemic.</td><td>2020-02-26</td><td>Jun Chen; Lianlian Wu; Jun Zhang; Liang Zhang; Dexin Gong; Yilin Zhao; Shan Hu; Yonggui Wang; Xiao Hu; Biqing Zheng; Kuo Zhang; Huiling Wu; Zehua Dong; Youming Xu; Yijie Zhu; Xi Chen; Lilei Yu; Honggang Yu</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20021568</td></tr><tr><td>acmfhbtm</td><td>da5ecffba8c5d1b7afee057ff61149fb976501ce</td><td>medrxiv</td><td>Highland of COVID-19 outside Hubei: epidemic characteristics, control and projections of Wenzhou, China</td><td>10.1101/2020.02.25.20024398</td><td>null</td><td>null</td><td>medrvix</td><td>In late December 2019, Chinese authorities reported a cluster of pneumonia cases of unknown aetiology in Wuhan, China. Wenzhou, as a southeast coastal city with the most cases outside Hubei Province, its policy control and epidemic projections have certain references for national and worldwide epidemic prevention and control. The maximum number of illness onset in Wenzhou appeared on January 26 (38 cases). Since January 27, the incidence per day in Wenzhou has continued to decline is likely to be due to isolation of people return to Wenzhou, implementation of region quarantine and suspension of public transportation system. Under the current control efforts, we estimate that the daily incidence by March 3-9, 2020 will drop to 0 in Wenzhou using SEIR model. The total number of affected people is 538.</td><td>2020-02-29</td><td>Liangde Xu; Jian Yuan; Yaru Zhang; Guosi Zhang; Fan Lu; Jianzhong Su; Jia Qu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20024398</td></tr><tr><td>dy28eo89</td><td>bca15ad98e1d08c6a62c9c0486de84bc15a9f661</td><td>medrxiv</td><td>Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</td><td>10.1101/2020.02.25.20024711</td><td>null</td><td>null</td><td>medrvix</td><td>Background. The clinical presentation of 2019 Novel Coronavirus (2019-nCov) infected pneumonia (NCIP) resembles that of other etiologies of community-acquired pneumonia (CAP). We aimed to identify clinical laboratory features to distinguish NCIP from CAP. Methods. We compared the ability of the hematological and biochemical features of 84 patients with NCIP at hospital admission and 316 patients with CAP. Parameters independently predictive of NCIP were calculated by multivariate logistic regression. The receiver operating characteristic (ROC) curves were generated and the area under the ROC curve (AUC) was measured to evaluate the discriminative ability. Results. Most hematological and biochemical indexes of patients with NCIP were significantly different from patients with CAP. Nine laboratory parameters were identified to be highly predictive of a diagnosis of NCIP by multivariate analysis. The AUCs demonstrated good discriminatory ability for red cell distribution width (RDW) with an AUC of 0.88 and Hemoglobin (HGB) with an AUC of 0.82. Red blood cell (RBC), albumin (ALB), eosinophil (EO), hematocrit (HCT), alkaline phosphatase (ALP), and white blood cell (WBC) had fair discriminatory ability. Combinations of any two parameters performed better than did the RDW alone. Conclusions. Routine laboratory examinations may be helpful for the diagnosis of NCIP. Application of laboratory tests may help to optimize the use of isolation rooms for patients when they present with unexplained febrile respiratory illnesses.</td><td>2020-02-25</td><td>Yunbao Pan; Guangming Ye; Xiantao Zeng; Guohong Liu; Xiaojiao Zeng; Xianghu Jiang; Jin Zhao; Liangjun Chen; Shuang Guo; Qiaoling Deng; Xiaoyue Hong; Ying Yang; Yirong Li; Xinghuan Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20024711</td></tr><tr><td>370fqkat</td><td>c8d206a4f9af0709b6e9ee90c4d854d482cb0784</td><td>medrxiv</td><td>Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</td><td>10.1101/2020.02.25.20025643</td><td>null</td><td>null</td><td>medrvix</td><td>Aim: The new coronavirus pneumonia (COVID-19) outbreaking at the end of 2019 is highly contagious. Crude mortality rate reached 49% in critical patients. Inflammation matters on disease progression. This study analyzed blood inflammation indicators among mild, severe and critical patients, helping to identify severe or critical patients early. Methods: In this cross-sectional study, 100 patients were included and divided to mild, severe or critical groups. Correlation of peripheral blood inflammation-related indicators with disease criticality was analyzed. Cut-off values for critically ill patients were speculated through the ROC curve. Results：Significantly, disease severity were associated with age (R=-0.564, P<0.001), interleukin-2 receptor (IL2R) (R=-0.534, P<0.001), interleukin-6 (IL-6) (R=-0.535, P<0.001), interleukin-8 (IL-8) (R=-0.308, P<0.001), interleukin-10 (IL-10) (R=-0.422, P<0.001), tumor necrosis factor α (TNFα) (R=-0.322, P<0.001), C-reactive protein (CRP) (R=-0.604, P<0.001), ferroprotein (R=-0.508, P<0.001), procalcitonin (R=-0.650, P<0.001), white cell counts (WBC) (R=-0.54, P<0.001), lymphocyte counts (LC) (R=-0.56, P<0.001), neutrophil count (NC) (R=-0.585, P<0.001) and eosinophil counts (EC) (R=-0.299, P=0.01). Conclusion：With following parameters such as age >67.5 years, IL2R >793.5U/mL, CRP >30.7ng/mL, ferroprotein >2252μg/L, WBC>9.5*10^9/L or NC >7.305*10^9/L, the progress of COVID-19 to critical stage should be closely observed and possibly prevented. Inflammation is closely related to severity of COVID-19, and IL-6, TNFα and IL-8 might be promising therapeutic targets.</td><td>2020-02-27</td><td>Jing Gong; Hui Dong; Song Qing Xia; Yi Zhao Huang; Dingkun Wang; Yan Zhao; Wenhua Liu; Shenghao Tu; Mingmin Zhang; Qi Wang; Fuer Lu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20025643</td></tr><tr><td>qtrquuvv</td><td>7d3dcbbc2b0741fc544693d7b3fc185f0bd50eac</td><td>medrxiv</td><td>Open-source analytics tools for studying the COVID-19 coronavirus outbreak</td><td>10.1101/2020.02.25.20027433</td><td>null</td><td>null</td><td>medrvix</td><td>To provide convenient access to epidemiological data on the coronavirus outbreak, we developed an R package, nCov2019 (https://github.com/GuangchuangYu/nCov2019). Besides detailed real-time statistics, it offers access to three data sources with detailed daily statistics from December 1, 2019, for 43 countries and more than 500 Chinese cities. We also developed a web app (http://www.bcloud.org/e/) with interactive plots and simple time-series forecasts. These analytics tools could be useful in informing the public and studying how this and similar viruses spread in populous countries.</td><td>2020-02-27</td><td>Tianzhi Wu; Xijin Ge; Guangchuang Yu; Erqiang Hu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20027433</td></tr><tr><td>aoqyx8mk</td><td>160ecaaa5a766c289fc2f6b5499f0dfe7aab971c</td><td>medrxiv</td><td>Stochastic discrete epidemic modeling of COVID-19 transmission in the Province of Shaanxi incorporating public health intervention and case importation</td><td>10.1101/2020.02.25.20027615</td><td>null</td><td>null</td><td>medrvix</td><td>Before the lock-down of Wuhan/Hubei/China, on January 23rd 2020, a large number of individuals infected by COVID-19 moved from the epicenter Wuhan and the Hubei province due to the Spring Festival, resulting in an epidemic in the other provinces including the Shaanxi province. The epidemic scale in Shaanxi was comparatively small and with half of cases being imported from the epicenter. Based on the complete epidemic data including the symptom onset time and transmission chains, we calculate the control reproduction number (1.48-1.69) in Xian. We could also compute the time transition, for each imported or local case, from the latent, to infected, to hospitalized compartment, as well as the effective reproduction number. This calculation enables us to revise our early deterministic transmission model to a stochastic discrete epidemic model with case importation and parameterize it. Our model-based analyses reveal that the newly generated infections decay to zero quickly; the cumulative number of case-driven quarantined individuals via contact tracing stabilize at a manageable level, indicating that the intervention strategies implemented in the Shaanxi province have been effective. Risk analyses, important for the consideration of resumption of work, show that a large second outbreak is expected if the level of case importation remains at the same level as between January 10th and February 4th 2020. However, if the case importation decreases by 30%, 60% and 90%, the second outbreak if happening will be of small-scale assuming contact tracing and quarantine/isolation remain as effective as before. Finally, we consider the effects of intermittent inflow with a Poisson distribution on the likelihood of multiple outbreaks. We believe the developed methodology and stochastic model provide an important model framework for the evaluation of revising travel restriction rules in the consideration of resuming social-economic activities while managing the disease control with potential case importation.</td><td>2020-02-29</td><td>Sanyi Tang; Biao Tang; Nicola Luigi Bragazzi; Fan Xia; Tangjuan Li; Sha He; Pengyu Ren; Xia Wang; Zhihang Peng; Yanni Xiao; Jianhong Wu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20027615</td></tr><tr><td>7s9ot4vq</td><td>875b7c463f00772fa0dc18ada678bc1ff16a4274</td><td>medrxiv</td><td>Comorbidity and its impact on 1,590 patients with COVID-19 in China: A Nationwide Analysis</td><td>10.1101/2020.02.25.20027664</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To evaluate the spectrum of comorbidities and its impact on the clinical outcome in patients with coronavirus disease 2019 (COVID-19). Design: Retrospective case studies Setting: 575 hospitals in 31 province/autonomous regions/provincial municipalities across China Participants: 1,590 laboratory-confirmed hospitalized patients. Data were collected from November 21st, 2019 to January 31st, 2020. Main outcomes and measures: Epidemiological and clinical variables (in particular, comorbidities) were extracted from medical charts. The disease severity was categorized based on the American Thoracic Society guidelines for community-acquired pneumonia. The primary endpoint was the composite endpoints, which consisted of the admission to intensive care unit (ICU), or invasive ventilation, or death. The risk of reaching to the composite endpoints was compared among patients with COVID-19 according to the presence and number of comorbidities. Results: Of the 1,590 cases, the mean age was 48.9 years. 686 patients (42.7%) were females. 647 (40.7%) patients were managed inside Hubei province, and 1,334 (83.9%) patients had a contact history of Wuhan city. Severe cases accounted for 16.0% of the study population. 131 (8.2%) patients reached to the composite endpoints. 399 (25.1%) reported having at least one comorbidity. 269 (16.9%), 59 (3.7%), 30 (1.9%), 130 (8.2%), 28 (1.8%), 24 (1.5%), 21 (1.3%), 18 (1.1%) and 3 (0.2%) patients reported having hypertension, cardiovascular diseases, cerebrovascular diseases, diabetes, hepatitis B infections, chronic obstructive pulmonary disease, chronic kidney diseases, malignancy and immunodeficiency, respectively. 130 (8.2%) patients reported having two or more comorbidities. Patients with two or more comorbidities had significantly escalated risks of reaching to the composite endpoint compared with those who had a single comorbidity, and even more so as compared with those without (all P<0.05). After adjusting for age and smoking status, patients with COPD (HR 2.681, 95%CI 1.424-5.048), diabetes (HR 1.59, 95%CI 1.03-2.45), hypertension (HR 1.58, 95%CI 1.07-2.32) and malignancy (HR 3.50, 95%CI 1.60-7.64) were more likely to reach to the composite endpoints than those without. As compared with patients without comorbidity, the HR (95%CI) was 1.79 (95%CI 1.16-2.77) among patients with at least one comorbidity and 2.59 (95%CI 1.61-4.17) among patients with two or more comorbidities. Conclusion: Comorbidities are present in around one fourth of patients with COVID-19 in China, and predispose to poorer clinical outcomes.</td><td>2020-02-27</td><td>Wei-jie Guan; Wen-hua Liang; Yi Zhao; Heng-rui Liang; Zi-sheng Chen; Yi-min Li; Xiao-qing Liu; Ru-chong Chen; Chun-li Tang; Tao Wang; Chun-quan Ou; Li Li; Ping-yan Chen; Ling Sang; Wei Wang; Jian-fu Li; Cai-chen Li; Li-min Ou; Bo Cheng; Shan Xiong; Zheng-yi Ni; Yu Hu; Jie Xiang; Lei Liu; Hong Shan; Chun-liang Lei; Yi-xiang Peng; Li Wei; Yong Liu; Ya-hua Hu; Peng Peng; Jian-ming Wang; Ji-yang Liu; Zhong Chen; Gang Li; Zhi-jian Zheng; Shao-qin Qiu; Jie Luo; Chang-jiang Ye; Shao-yong Zhu; Lin-ling Cheng; Feng Ye; Shi-yue Li; Jin-ping Zheng; Nuo-fu Zhang; Nan-shan Zhong; Jian-xing He</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20027664</td></tr><tr><td>gvjiexfm</td><td>null</td><td>medrxiv</td><td>Age-dependent risks of Incidence and Mortality of COVID-19 in Hubei Province and Other Parts of China</td><td>10.1101/2020.02.25.20027672</td><td>null</td><td>null</td><td>medrvix</td><td>New coronavirus SARS-CoV-2 poses a big challenge for global public health in early 2020. Coronavirus Disease 2019 (COVID-19) caused by the virus rapidly spreads all over the world and takes thousands of lives in just two months. It is critical to refine the incidence and mortality risks of COVID-19 for the effective management of the general public and patients in the outbreak. In this report, we investigate the incidence and mortality risks of the infection by analyzing the age composition of 5319 infected patients, 76 fatal cases, and 1,144,648 individuals of the general public in China. Our result shows a relatively low incidence risk for young people but a very high mortality risk for seniors. Notably, mortality risk could be as high as 0.48 for people older than 80 years. Furthermore, our study suggests that a good medical service can effectively reduce the mortality rate of the viral infection to 1% or less.</td><td>2020-02-27</td><td>Hongdou Li; Shuang Wang; Fan Zhong; Wuyin Bao; Yipeng Li; Lei Liu; Hongyan Wang; Yungang He</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20027672</td></tr><tr><td>65plab8b</td><td>e046a06f262d74d025272ad697d358c4afb5d9bd</td><td>medrxiv</td><td>Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</td><td>10.1101/2020.02.25.20027755</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Coronavirus Disease 2019 (COVID-19) caused by Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has become a global threat to public health. Aiming to construct an efficient screening pattern, we comprehensively evaluated the performances of RT-PCR and chest CT in diagnosing COVID-19. Methods: The records including demographics, RT-PCR, and CT from 87 confirmed COVID-19 cases and 481 exclusion cases were collected. The diagnostic accuracy of the pharyngeal swab RT-PCR, CT, combination with the second pharyngeal swab RT-PCR or with CT were evaluated individually. Besides, all the stool RT-PCR results were plotted by time to explore the value of stool RT-PCR. Findings: Combination of RT-PCR and CT has the higher sensitivity (91.9%,79/86) than RT-PCR alone (78.2%，68/87) or CT alone (66.7%, 54 of 81) or combination of two RT-PCR tests (86.2%,75/87). There was good agreement between RT-PCR and CT (kappa-value, 0.430). In 34 COVID-19 cases with inconsistent results, 94.1% (n=32) are mild infection, 62.5% of which (20/32) showed positive RT-PCR. 46.7% (35/75) COVID-19 patients had at least one positive stool during the course. Two cases had positive stool earlier than the pharyngeal swabs. Importantly, one patient had consecutive positive stool but negative pharyngeal swabs. Interpretation: Combination of RT-PCR and CT with the highest sensitivity is an optimal pattern to screen COVID-19. RT-PCR is superior to CT in diagnosing mild infections. Stool RT-PCR should be considered as an item for improving discovery rate and hospital discharge. This study shed light for optimizing scheme of screening and monitoring of SARS-CoV-2 infection.</td><td>2020-02-27</td><td>Guanmin Jiang; Xiaoshuai Ren; Yan Liu; Hongtao Chen; Wei Liu; Zhaowang Guo; Yaqin Zhang; Chaoqun Chen; Jianhui Zhou; Qiang Xiao; Hong Shan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20027755</td></tr><tr><td>cr0s5d1j</td><td>0eee9760aee0ca2fdf8d1d215c3689d5f8d84df5</td><td>medrxiv</td><td>Prevalence and clinical features of 2019 novel coronavirus disease (COVID-19) in the Fever Clinic of a teaching hospital in Beijing: a single-center, retrospective study</td><td>10.1101/2020.02.25.20027763</td><td>null</td><td>null</td><td>medrvix</td><td>Background With the spread of COVID-19 from Wuhan, Hubei Province to other areas of the country, medical staff in Fever Clinics faced the challenge of identifying suspected cases among patients with respiratory infections manifested with fever. We aimed to describe the prevalence and clinical features of COVID-19 as compared to pneumonias of other etiologies in a Fever Clinic in Beijing. Methods In this single-center, retrospective study, 342 cases of pneumonia were diagnosed in Fever Clinic in Peking University Third Hospital between January 21 to February 15, 2020. From these patients, 88 were reviewed by panel discussion as possible or probable cases of COVID-19, and received 2019-nCoV detection by RT-PCR. COVID-19 was confirmed by positive 2019-nCoV in 19 cases, and by epidemiological, clinical and CT features in 2 cases (the COVID-19 Group, n=21), while the remaining 67 cases served as the non-COVID-19 group. Demographic and epidemiological data, symptoms, laboratory and lung CT findings were collected, and compared between the two groups. Findings The prevalence of COVID-19 in all pneumonia patients during the study period was 6.14% (21/342). Compared with the non-COVID-19 group, more patients with COVID-19 had an identified epidemiological history (90.5% versus 32.8%, P<0.001). The COVID-19 group had lower WBC [5.19×10^9/L (±1.47) versus 7.21×10^9/L (±2.94), P<0.001] and neutrophil counts [3.39×10^9/L (±1.48) versus 5.38×10^9/L (±2.85), P<0.001] in peripheral blood. However, the percentage and count of lymphocytes were not different. On lung CT scans, involvement of 4 or more lobes was more common in the COVID-19 group (45% versus 16.4%, P=0.008). Interpretation In the period of COVID-19 epidemic outside Hubei Province, the prevalence of COVID-19 in patients with pneumonia visiting to our Fever Clinic in Beijing was 6.14%. Epidemiological evidence was important for prompt case finding, and lower blood WBC and neutrophil counts may be useful for differentiation from pneumonia of other etiologies.</td><td>2020-02-27</td><td>Ying Liang; Jingjin Liang; Qingtao Zhou; Xiaoguang Li; Fei Lin; Zhonghua Deng; Biying Zhang; Lu Li; Xiaohua Wang; Hong Zhu; Qingbian Ma; Xiaomei Tong; Jie Xu; Yongchang Sun</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20027763</td></tr><tr><td>a12708qd</td><td>null</td><td>medrxiv</td><td>Genomic variations of SARS-CoV-2 suggest multiple outbreak sources of transmission</td><td>10.1101/2020.02.25.20027953</td><td>null</td><td>null</td><td>medrvix</td><td>We examined 169 genomes of SARS-CoV-2 and found that they can be classified into two major genotypes, Type I and Type II. Type I can be further divided into Type IA and IB. Our phylogenetic analysis showed that the Type IA resembles the ancestral SARS-CoV-2 most. Type II was likely evolved from Type I and predominant in the infections. Our results suggest that Type II SARS-CoV-2 was the source of the outbreak in the Wuhan Huanan market and it was likely originated from a super-spreader. The outbreak caused by the Type I virus should have occurred somewhere else, because the patients had no direct link to the market. Furthermore, by analyzing three genomic sites that distinguish Type I and Type II strains, we found that synonymous changes at two of the three sites confer higher protein translational efficiencies in Type II strains than in Type I strains, which might explain why Type II strains are predominant, implying that Type II is more contagious (transmissible) than Type I. These findings could be valuable for the current epidemic prevention and control.</td><td>2020-02-26</td><td>Liangsheng Zhang; Jian-Rong Yang; Zhenguo Zhang; Zhenguo Lin</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20027953</td></tr><tr><td>lad31adx</td><td>eca1cf50bd67a7efbcaf78864f5f5b3b850243d6</td><td>medrxiv</td><td>Clinical Data on Hospital Environmental Hygiene Monitoring and Medical Staff Protection during the Coronavirus Disease 2019 Outbreak</td><td>10.1101/2020.02.25.20028043</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The outbreak of coronavirus disease 2019 (COVID-19) has placed unprecedented challenges on hospital environmental hygiene and medical staff protection. It is crucial to assess hospital environmental hygiene to understand the most important environmental issues for controlling the spread of COVID-19 in hospitals. Objective: To detect the presence of COVID-19 in the samples from the area at risk of contamination in the First Hospital of Jilin University. Methods: Viruses in the air were collected by natural sedimentation and air particle sampler methods. Predetermined environmental surfaces were sampled using swabs at seven o'clock in the morning before disinfection. The real-time reverse-transcription PCR method was used to detect the existence of COVID-19 pathogens. Results: Viruses could be detected on the surfaces of the nurse station in the isolation area with suspected patients and in the air of the isolation ward with an intensive care patient. Conclusion: Comprehensive monitoring of hospital environmental hygiene during pandemic outbreaks is conducive to the refinement of hospital infection control. It is of great significance to ensure the safety of medical treatment and the quality of hospital infection control through the monitoring of environmental hygiene.</td><td>2020-02-27</td><td>Yanfang Jiang; Haifeng Wang; Yukun Chen; Jiaxue He; Liguo Chen; Yong Liu; Xinyuan Hu; Ang Li; Siwen Liu; Peng Zhang; Hongyan Zou; Shucheng Hua</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.20028043</td></tr><tr><td>i55tm3hh</td><td>a2a6e262098539eb875a26800d9f6d3d0d5d1875</td><td>biorxiv</td><td>An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</td><td>10.1101/2020.02.25.963546</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe 2013-2016 West Africa EBOV epidemic was the biggest EBOV outbreak to date. An analysis of virus-specific CD8+ T-cell immunity in 30 survivors showed that 26 of those individuals had a CD8+ response to at least one EBOV protein. The dominant response (25/26 subjects) was specific to the EBOV nucleocapsid protein (NP). It has been suggested that epitopes on the EBOV NP could form an important part of an effective T-cell vaccine for Ebola Zaire. We show that a 9-amino-acid peptide NP44-52 (YQVNNLEEI) located in a conserved region of EBOV NP provides protection against morbidity and mortality after mouse adapted EBOV challenge. A single vaccination in a C57BL/6 mouse using an adjuvanted microsphere peptide vaccine formulation containing NP44-52 is enough to confer immunity in mice. Our work suggests that a peptide vaccine based on CD8+ T-cell immunity in EBOV survivors is conceptually sound and feasible. Nucleocapsid proteins within COVID-19 contain multiple class I epitopes with predicted HLA restrictions consistent with broad population coverage. A similar approach to a CTL vaccine design may be possible for that virus.</td><td>2020-02-27</td><td>CV Herst; S Burkholz; J Sidney; A Sette; PE Harris; S Massey; T Brasel; E Cunha-Neto; DS Rosa; WCH Chao; R Carback; T Hodge; L Wang; S Ciotlos; P Lloyd; R Rubsamen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.963546</td></tr><tr><td>2hyiped2</td><td>86b6b0c1b2777541feb83116bcb7a5cb12a52310</td><td>biorxiv</td><td>Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2</td><td>10.1101/2020.02.25.964775</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractCoronavirus disease 2019 (COVID-19) is newly emerging human infectious diseases, which is caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV). Within two months of the outbreak, more than 80,000 cases of COVID-19 have been confirmed worldwide. Since the human to human transmission occurred easily and the human infection is rapidly increasing, the sensitive and early diagnosis is essential to prevent the global outbreak. Recently, World Health Organization (WHO) announced various primer and probe sets for SARS-CoV-2 previously developed in China, Germany, Hong Kong, Japan, Thailand, and USA. In this study, we compared the ability to detect SARS-CoV-2 RNA among the seven primer-probe sets for N gene and the three primer-probe sets for Orf1 gene. The result of the comparative analysis represented that the ‘2019-nCoV_N2, N3’ of USA and the ‘ORF1ab’ of China are the most sensitive primer-probe sets for N and Orf1 genes, respectively. Therefore, the appropriate combination from ORF1ab (China), 2019-nCoV_N2, N3 (USA), and NIID_2019-nCOV_N (Japan) sets should be selected for the sensitive and reliable laboratory confirmation of SARS-CoV-2.</td><td>2020-02-27</td><td>Yu Jin Jung; Gun-Soo Park; Jun Hye Moon; Keunbon Ku; Seung-Hwa Beak; Seil Kim; Edmond Changkyun Park; Daeui Park; Jong-Hwan Lee; Cheol Woo Byeon; Joong Jin Lee; Jin-Soo Maeng; Seong Jun Kim; Seung Il Kim; Bum-Tae Kim; Min Jun Lee; Hong Gi Kim</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.964775</td></tr><tr><td>e9vq3fe3</td><td>949feecec7c10e71532e860f7f1230bd696e8234</td><td>biorxiv</td><td>Epitope-based peptide vaccine design and target site characterization against novel coronavirus disease caused by SARS-CoV-2</td><td>10.1101/2020.02.25.965434</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe outbreak of the 2019 novel coronavirus (SARS-CoV-2) has infected thousands of people with a large number of deaths across 26 countries. The sudden appearance of the virus leads to the limited existing therapies for SARS-CoV-2. Therefore, vaccines and antiviral medicines are in desperate need. This study took immune-informatics approaches to identify B- and T-cell epitopes for surface glycoprotein (S) of SARS-CoV-2, followed by estimating their antigenicity and interactions with the human leukocyte antigen (HLA) alleles. We identified four B cell epitopes, two MHC class-I and nine MHC class-II binding T-cell epitopes, which showed highly antigenic features. Allergenicity, toxicity and physiochemical properties analysis confirmed the specificity and selectivity of epitopes. The stability and safety of epitopes were confirmed by digestion analysis. No mutations were observed in all the selected B- and T-cell epitopes across all isolates from different locations worldwide. Epitopes were thus identified and some of them can be potential candidates for vaccine development.</td><td>2020-02-27</td><td>Lin Li; Ting Sun; Yufei He; Wendong Li; Yubo Fan; Jing Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.965434</td></tr><tr><td>qefiw4ho</td><td>dbefc8ad2a3de5d1696b7e604de8bce1da2ea8cd</td><td>biorxiv</td><td>Screening of FDA-approved drugs using a MERS-CoV clinical isolate from South Korea identifies potential therapeutic options for COVID-19</td><td>10.1101/2020.02.25.965582</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractIn 2015, the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) reached the Republic of Korea through nosocomial transmission and was the largest epidemic outside of the Arabian Peninsula. To date, despite various strategies to identify CoV interventions, only limited therapeutic options are available. To address these unmet medical needs, we used a South Korean MERS-CoV clinical isolate and screened 5,406 compounds, including United States Food and Drug Administration (FDA)-approved drugs and bioactive molecules, for their activity against the isolate. The primary assay confirmed 221 hits by dose-response curve analysis and identified 54 hits with a therapeutic index (TI) greater than 6. Time-of-addition studies with 12 FDA-approved drugs demonstrated that 8 and 4 therapeutics act on the early and late stages of the viral life cycle, respectively. Among the drugs were e.g., three cardiotonic agents (ouabain, digitoxin, digoxin) with a TI greater than 100, an anti-malaria drug (atovaquone; TI >34), an inhalable corticosteroid (ciclesonide; TI >6), etc. Together, our results identify potential therapeutic options for treating MERS-CoV infections and could provide a basis for agents against a wider range of coronavirus-related illnesses, including the currently emerging Coronavirus Disease 2019 (COVID-19) outbreak.</td><td>2020-03-02</td><td>Meehyun Ko; So Young Chang; Soo Young Byun; Inhee Choi; Anne-Laure Pham Hung d’Alexandry d’Orengiani; David Shum; Ji-Young Min; Marc P. Windisch</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.25.965582</td></tr><tr><td>v0ln3wfa</td><td>baabfb35a321ea12028160e0d2c1552a2fda2dd5</td><td>medrxiv</td><td>Clinical Features of COVID-19 Related Liver Damage</td><td>10.1101/2020.02.26.20026971</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND: A recent outbreak of SARS-CoV-2 infection occurs mainly in China, with rapidly increasing the number of cases (namely COVID-19). Abnormal liver functions are frequently present in these patients, here we aimed to clarify the clinical features of COVID-19-related liver damage to provide some references for the clinical treatment. METHODS: In this retrospective, single-center study, we included all confirmed COVID-19 cases in Shanghai Public Health Clinical Center from January 20 to January 31, 2020. The outcomes were followed up until February 19, 2020. A total of 148 cases were analyzed for clinical features, laboratory parameters (including liver function tests), medications and the length of stay. FINDINGS: Of 148 confirmed SARS-CoV-2-infected patients, 49.3% were females and 50.7% were males. The median age was 50.5 years (interquartile range, 36-64). Patients had clinical manifestations of fever (70.1%), cough (45.3%), expectoration (26.7%) at admission. 75 patients (50.7%) showed abnormal liver functions at admission. Patients (n = 75) who had elevated liver function index were more likely to have a moderate-high degree fever (44% vs 27.4%; p = 0.035) and significantly present in male patients (62.67% vs 38.36%; p = 0.005). The numbers of CD4+ and CD8+ T cells were significantly lower in abnormal liver function group than those in normal liver function group. There was no statistical difference in prehospital medications between normal and abnormal liver function groups, while the utilization rate of lopinavir/ritonavir after admission was significantly higher in patients with emerging liver injury than that in patients with normal liver functions. Importantly, the emerging abnormal liver functions after admission caused a prolonged length of stay. INTERPRETATION: SARS-CoV-2 may cause the liver function damage and the Lopinavir/ritonavir should be applied carefully for the treatment of COVID-19.</td><td>2020-02-27</td><td>Zhenyu Fan; Liping Chen; Jun Li; Cheng Tian; Yajun Zhang; Shaoping Huang; Zhanju Liu; Jilin Cheng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20026971</td></tr><tr><td>r7yyffgt</td><td>null</td><td>medrxiv</td><td>The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</td><td>10.1101/2020.02.26.20026989</td><td>null</td><td>null</td><td>medrvix</td><td>IMPORTANCE: COVID-19-infected pneumonia patients with severe immune abnormalities and risk of cytokine release syndrome. The definition, prevention, and treatment of COVID-19-infected pneumonia in critically ill patients with cytokine release syndrome symptoms is an important problem.</td><td>2020-02-27</td><td>Wenjun Wang; Jianxing He; puyi Lie; liyan Huang; Sipei Wu; yongping lin; xiaoqing liu</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20026989</td></tr><tr><td>m0d72ntg</td><td>7344c6de7ce2258e74ce131bed72e6ad465f0ce8</td><td>medrxiv</td><td>Relations of parameters for describing the epidemic of COVID―19 by the Kermack―McKendrick model</td><td>10.1101/2020.02.26.20027797</td><td>null</td><td>null</td><td>medrvix</td><td>In order to quantitatively characterize the epidemic of COVID―19, useful relations among parameters describing an epidemic in general are derived based on the Kermack-McKendrick model. The first relation is 1/τgrow=1/τtrans−1/τinf, where τgrow is the time constant of the exponential growth of an epidemic, τtrans is the time for a pathogen to be transmitted from one patient to uninfected person, and the infectious time τinf is the time during which the pathogen keeps its power of transmission. The second relation p(∞) ≈1−exp(−(R0−1)/0.60) is the relation between p(∞), the final size of the disaster defined by the ratio of the total infected people to the population of the society,and the basic reproduction number, R0, which is the number of persons infected by the transmission of the pathogen from one infected person during the infectious time. The third relation 1/τend=1/τinf−(1−p(∞))/τtrans gives the decay time constant τend at the ending stage of the epidemic. Derived relations are applied to influenza in Japan in 2019 for characterizing the epidemic.</td><td>2020-03-03</td><td>Toshihisa Tomie</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20027797</td></tr><tr><td>pytu6sy0</td><td>c10661c9b35e068691b879661bcebeba3bd6aad9</td><td>medrxiv</td><td>The infection evidence of SARS-COV-2 in ocular surface： a single-center cross-sectional study</td><td>10.1101/2020.02.26.20027938</td><td>null</td><td>null</td><td>medrvix</td><td>Purpose: The aim of this study was to identify whether SARS-COV-2 infected in ocular surface. Methods: Cross-sectional study of patients presenting for who received a COVID-19 diagnosis, from December 30, 2019 to February 7, 2020, at Tongji hospital, Tongji medical college, Huazhong University of Science and Technology. Demographics, temperature was recorded, blood routine test (Rt), chest Computed Tomography (CT) were took intermittently, and SARS-COV-2 real-time reverse-transcriptase-polymerase-chain-reaction (RT-PCR) assay were arranged for the nasopharyngeal and conjunctival swab samples. Results: A total of 102 patients (48 Male [50%] and 54 Female [50%]) with clinical symptoms, Rt, and chest Computed Tomography (CT) abnormalities were identified with a clinical diagnosis of COVID-19. Patients had a mean [SD] gestational age of 57.63 [14.90] years. Of a total of 102 patients identified, 72 patients (36 men [50%] and 36 women [50%]; mean [SD] age, 58.68 [14.81] years) confirmed by laboratory diagnosis with SARS-COV-2 RT-PCR assay. Only two patients (2.78%) with conjunctivitis was identified from 72 patients with a laboratory confirmed COVID-19. However, SARS-COV-2 RNA fragments was found in ocular discharges by SARS-COV-2 RT-PCR only in one patient with conjunctivitis. Conclusions: Although we suspect the incidence of SARS-COV-2 infection through the ocular surface is extremely low, the nosocomial infection of SARS-CoV-2 through the eyes after occupational exposure is a potential route. The inefficient diagnostic method and the sampling time lag may contribute to the lower positive rate of conjunctival swab samples of SARS-COV-2. Therefore, to lower the SARS-COV-2 nosocomial infection, the protective goggles should be wore in all the health care workers.</td><td>2020-02-26</td><td>Xufang Sun; Xian Zhang; Xuhui Chen; Liwen Chen; Chaohua Deng; Xiaojing Zou; Weiyong Liu; Huimin Yu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20027938</td></tr><tr><td>hk491mr6</td><td>46634d03aa169aab9c372746ebaf3aaa65d7c7d9</td><td>medrxiv</td><td>Case fatality rate of novel coronavirus disease 2019 in China</td><td>10.1101/2020.02.26.20028076</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract Background: A pandemic of coronavirus disease 2019 (COVID-19) which have caused more than 80 thousand persons infected globally is still ongoing. This study aims to calculate its case fatality rate (CFR). Methods: The method, termed as converged CFR calculation, was based on the formula of dividing the number of known deaths by the number of confirmed cases T days before, where T was an average time period from case confirmation to death. It was found that supposing a T, if it was smaller (bigger) than the true T, calculated CFRs would gradually increase (decrease) to infinitely near the true T with time went on. According to the law, the true T value could be determined by trends of daily CFRs calculated with different assumed T values (left of true T is decreasing, right is increasing). Then the CFR could be calculated. Results: CFR of COVID-19 in China except Hubei Province was 0.8% to 0.9%. So far, the CFR had accurately predicted the death numbers more than 3 weeks. CFR in Hubei of China was 5.4% by which the calculated death number corresponded with the reported number for 2 weeks. Conclusion: The method could be used for CFR calculating while pandemics are still ongoing. Dynamic monitoring of the daily CFRs trends could help outbreak-controller to have a clear vision in the timeliness of the case confirmation.</td><td>2020-02-26</td><td>Rui Qi; Chao Ye; Xiang-rong Qin; Xue-Jie Yu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028076</td></tr><tr><td>011k6mm0</td><td>c954675ee859e2f7b8f352a398a67469b50f05de</td><td>medrxiv</td><td>Evaluation of the clinical characteristics of suspected or confirmed cases of COVID-19 during home care with isolation: A new retrospective analysis based on O2O</td><td>10.1101/2020.02.26.20028084</td><td>null</td><td>null</td><td>medrvix</td><td>Summary Background The recent outbreak of the novel coronavirus in December 2019 (COVID-19) has activated top-level response nationwide. We developed a new treatment model based on the online-to-offline (O2O) model for the home isolated patients, because in the early stages the medical staff were insufficient to cope with so many patients. Methods In this single-centered, retrospective study, we enrolled 48 confirmed/suspected COVID-19 patients who underwent home isolation in Wuhan between January 6 and January 31, 2020. By WeChat and online document editing all patients were treated with medical observation scale. The clinical indications such as Fever, Muscle soreness, Dyspnea and Lack of strength were collected with this system led by medical staff in management, medicine, nursing, rehabilitation and psychology. Findings The mean(SD) age of 48 patients was 39.08(13.88) years, 35(72.9%) were women. Compared with non-hospitalized patients, inpatients were older(≥8805;70years, 2.4% vs 33.3%, P<0.04). All inpatients had fever, 50% inpatients had coughs and showed infiltration in both lungs at the time of diagnosis. 33.3% inpatients exhibited negative changes in their CT results at initial diagnosis. The body temperature of non-hospitalized patients with mild symptoms returned to normal by day 4-5. While dyspnea peaked on day 6 for non-hospitalized patients with mild symptoms, it persisted in hospitalized patients and exacerbated over time. The lack of strength and muscle soreness were both back to normal by day 4 for non-hospitalized patients. Interpretation Monitoring the trends of symptoms is more important for identifying severe cases. Excessive laboratory data and physical examination are not necessary for the evaluation of patients with mild symptoms. The system we developed is the first to convert the subjective symptoms of patients into objective scores. This type of O2O, subjective-to-objective strategy may be used in regions with similar highly infectious diseases to minimize the possibility of infection among medical staff.</td><td>2020-02-29</td><td>Hui Xu; Sufang Huang; Shangkun Liu; Juan Deng; Bo Jiao; Ling Ai; Yaru Xiao; Li Yan; Shusheng Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028084</td></tr><tr><td>96wkqutc</td><td>2536b83acf84368d7c13be81fe07aa0575115da7</td><td>medrxiv</td><td>Estimation of country-level basic reproductive ratios for novel Coronavirus (COVID-19) using synthetic contact matrices</td><td>10.1101/2020.02.26.20028167</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of novel coronavirus (COVID-19) has the potential for global spread, infecting large numbers in all countries. In this case, estimating the country-specific basic reproductive ratio is a vital first step in public-health planning. The basic reproductive ratio (R0) is determined by both the nature of pathogen and the network of contacts through which the disease can spread - with this network determined by socio-demographics including age-structure and household composition. Here we focus on the age-structured transmission within the population, using data from China to inform age-dependent susceptibility and synthetic age-mixing matrices to inform the contact network. This allows us to determine the country-specific basic reproductive ratio as a multiplicative scaling of the value from China. We predict that R0 will be highest across Eastern Europe and Japan, and lowest across Africa, Central America and South-Western Asia. This pattern is largely driven by the ratio of children to older adults in each country and the observed propensity of clinical cases in the elderly.</td><td>2020-02-27</td><td>Joe Hilton; Matt J Keeling</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028167</td></tr><tr><td>fcmzdcuh</td><td>a625e5d6fc8956c231c500a40e887e248a69de61</td><td>medrxiv</td><td>Clinical characteristics of 82 death cases with COVID-19</td><td>10.1101/2020.02.26.20028191</td><td>null</td><td>null</td><td>medrvix</td><td>Background A recently developing pneumonia caused by SARS-CoV-2 was originated in Wuhan, China, and has quickly spread across the world. We reported the clinical characteristics of 82 death cases with COVID-19 in a single center. Methods Clinical data on 82 death cases laboratory-confirmed as SARS-CoV-2 infection were obtained from a Wuhan local hospital′s electronic medical records according to previously designed standardized data collection forms. Findings All patients were local residents of Wuhan, and the great proportion of them were diagnosed as severe illness when admitted. Most of the death cases were male (65.9%). More than half of dead patients were older than 60 years (80.5%) and the median age was 72.5 years. The bulk of death cases had comorbidity (76.8%), including hypertension (56.1%), heart disease (20.7%), diabetes (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%). Respiratory failure remained the leading cause of death (69.5%), following by sepsis syndrome/MOF (28.0%), cardiac failure (14.6%), hemorrhage (6.1%), and renal failure (3.7%). Furthermore, respiratory, cardiac, hemorrhage, hepatic, and renal damage were found in 100%, 89%, 80.5%, 78.0%, and 31.7% of patients, respectively. On the admission, lymphopenia (89.2%), neutrophilia (74.3%), and thrombocytopenia (24.3%) were usually observed. Most patients had a high neutrophil-to-lymphocyte ratio of >5 (94.5%), high systemic immune-inflammation index of >500 (89.2%), increased C-reactive protein level (100%), lactate dehydrogenase (93.2%), and D-dimer (97.1%). A high level of IL-6 (>10 pg/ml) was observed in all detected patients. Median time from initial symptom to death was 15 days (IQR 11-20), and a significant association between aspartate aminotransferase (p=0.002), alanine aminotransferase (p=0.037) and time from initial symptom to death were interestingly observed. Conclusion Older males with comorbidities are more likely to develop severe disease, even die from SARS-CoV-2 infection. Respiratory failure is the main cause of COVID-19, but either virus itself or cytokine release storm mediated damage to other organ including cardiac, renal, hepatic, and hemorrhage should be taken seriously as well.</td><td>2020-02-27</td><td>Bicheng Zhang; Xiaoyang Zhou; Yanru Qiu; Fan Feng; Jia Feng; Yifan Jia; Hengcheng Zhu; Ke Hu; Jiasheng Liu; Zaiming Liu; Shihong Wang; Yiping Gong; Chenliang Zhou; Ting Zhu; Yanxiang Cheng; Zhichao Liu; Hongping Deng; Fenghua Tao; Yijun Ren; Biheng Cheng; Ling Gao; Xiongfei Wu; Lilei Yu; Zhixin Huang; Zhangfan Mao; Qibin Song; Bo Zhu; Jun Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028191</td></tr><tr><td>87pamw52</td><td>7aa49aa2b76ea92b9265ad9d5a9d9ef0ea585b24</td><td>medrxiv</td><td>Community responses during the early phase of the COVID-19 epidemic in Hong Kong: risk perception, information exposure and preventive measures</td><td>10.1101/2020.02.26.20028217</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Community responses are important for outbreak management during the early phase when non-pharmaceutical interventions are the major preventive options. Therefore, this study aims to examine the psychological and behavioral responses of the community during the early phase of the COVID-19 epidemic in Hong Kong. Method: A cross-sectional online survey was launched within 36 hours after confirmed COVID-19 cases were first reported. Councilors of all 452 district council constituency areas were approached for survey dissemination. Respondent demographics, anxiety level, risk perception, sources to retrieve COVID-19 information, actual adoption and perceived efficacy of precautionary measures were collected. Result: Analysis from 1715 complete responses indicated high perceived susceptibility (89%) and high perceived severity (97%). Most respondents were worried about COVID-19 (97%), and had their daily routines disrupted (slightly/greatly: 98%). The anxiety level, measured by the Hospital Anxiety and Depression Scale, was borderline abnormal (9.01). Nearly all respondents were alert to the disease progression (99.5%). The most trusted information sources were doctors (84%), followed by broadcast (57%) and newspaper (54%), but they were not common information sources (doctor: 5%; broadcast: 34%; newspaper: 40%). Only 16% respondents found official websites reliable. Enhanced personal hygiene practices and travel avoidance to China were frequently adopted (>77%) and considered effective (>90%). The adoption of social-distancing measures was lower (39%-88%), and their drivers for greater adoption include: being female (adjusted odds ratio [aOR]:1.27), living in the New Territories (aOR:1.32-1.55), perceived as having good understanding of COVID-19 (aOR:1.84) and being more anxious (aOR:1.07). Discussion: Risk perception towards COVID-19 in the community was high. Most respondents are alert to the disease progression, and adopt self-protective measures. This study contributes by examining the psycho-behavioral responses of hosts, in addition to the largely studied mechanistic aspects, during the early phase of the current COVID-19 epidemic. The timely psychological and behavioral assessment of the community is useful to inform subsequent interventions and risk communication strategies as the epidemic progresses.</td><td>2020-02-27</td><td>Kin On Kwok; Kin Kit Li; Ho Hin Chan; Yuan Yuan Yi; Arthur Tang; Wan In Wei; Yeung Shan Wong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028217</td></tr><tr><td>dlivkn8l</td><td>111b9a6e91c938696fcdb4cb128b8ae739dbe11c</td><td>medrxiv</td><td>Clinical features and sexual transmission potential of SARS-CoV-2 infected female patients: a descriptive study in Wuhan, China</td><td>10.1101/2020.02.26.20028225</td><td>null</td><td>null</td><td>medrvix</td><td>Background: As of March 2, 2020, SARS-CoV-2 has infected more than 80174 people and caused 2915 deaths in China. This virus rapidly spreads to 56 countries worldwide. Thus, in order to effectively block its transmission, it is urgent to uncover all the possible transmission routes of SARS-CoV-2. Methods: From January 28 to February 18, 2020, 35 female patients diagnosed with COVID-19 in Tongji Hospital were included in this descriptive study. The gynecologic history, clinical characteristics, laboratory findings and chest computed tomography (CT) of all patients were recorded in detail. To examine whether there is sexual transmission through vaginal from female to her partner, we employed real-time polymerase chain reaction testing (RT-PCR) to detect SARS-CoV-2 in vaginal environment (including vaginal discharge, cervical or vaginal residual exfoliated cells) and anal swab samples, and inquired recent sexual behaviors from the patients. Findings: The age range of the 35 patients with COVID-19 was 37-88 years. Over 50% patients infected with SARS-CoV-2 had chronic diseases. We tested the vaginal environment and anal swabs from the 35 female patients with COVID-19 and found that only an anal swab sample from one patient was positive for SARS-CoV-2. All the samples from vaginal environment were negative for SARS-CoV-2. The infection rate of the patients' sexual partner was 42.9%. Additionally, two female patients admitted having sex with their partners during a possible infection incubation period, while one patient's partner was uninfected and the other patient's partner was diagnosed with COVID-19 (after the diagnosis of the female patient). Conclusion: No positive RT-PCR result was found in the vaginal environment perhaps due to the lack of ACE2 expression, which is the receptor of SARS-CoV-2, in the vagina and cervix tissues (human protein atlas). The results from this study show no evidence of transmission of SARS-CoV-2 through vaginal sex from female to her partner. However, the risk of infection of non vaginal sex and other intimate contacts during vaginal sex should not be ignored.</td><td>2020-02-27</td><td>Pengfei Cui; Zhe Chen; Tian Wang; Jun Dai; Jinjin Zhang; Ting Ding; Jingjing Jiang; Jia Liu; Cong Zhang; Wanying Shan; Sheng Wang; Yueguang Rong; Jiang Chang; Xiaoping Miao; Xiangyi Ma; Shixuan Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028225</td></tr><tr><td>yk9u4fvp</td><td>15e5dd0eb126bfe384c25ba551317eb9ba2ad1ef</td><td>medrxiv</td><td>Perceptions of the Adult US Population regarding the Novel Coronavirus Outbreak</td><td>10.1101/2020.02.26.20028308</td><td>null</td><td>null</td><td>medrvix</td><td>Background: COVID-19 outbreak is spreading globally. Although the risk of infection in the US is currently low, it is important to understand the public perception of risk and trust in sources of information to better inform public health messaging. In this study, we surveyed the adult US population to understand their risk perceptions about the COVID-19 outbreak. Methods and Findings: We used an online platform to survey 718 adults in the US in early February 2020 using a questionnaire that we developed. Our sample was fairly similar to the general adult US population in terms of age, gender, race, ethnicity and education. We found that 69% of the respondents wanted the scientific/public health leadership (either the CDC Director or NIH Director) to lead the US response to COVID-19 outbreak as compared to 14% who wanted the political leadership (either the president or the Congress) to lead the response. Risk perception was low (median score of 5 out of 10) with the respondents trusting health professionals and health officials for information on COVID-19. Majority of the respondents were in favor of strict infection prevention policies to control the outbreak. Conclusion: Given our results, the public health/scientific leadership should be at the forefront of the COVID-19 response to promote trust.</td><td>2020-02-27</td><td>SarahAnn M McFadden; Amyn A Malik; Obianuju G Aguolu; Kathryn S Willebrand; Saad B Omer</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028308</td></tr><tr><td>cv3qgno3</td><td>0fbb18050e29ca78191625d42576b1c574027377</td><td>medrxiv</td><td>Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</td><td>10.1101/2020.02.26.20028373</td><td>null</td><td>null</td><td>medrvix</td><td>The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.</td><td>2020-02-29</td><td>Yinhua Zhang; Nelson Odiwuor; Jin Xiong; Luo Sun; Raphael Ohuru Nyaruaba; Hongping Wei; Nathan A Tanner</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028373</td></tr><tr><td>px0twvhs</td><td>4bd697fdded7ee65154e28def586a7e9b60e7078</td><td>medrxiv</td><td>Transmission characteristics of the COVID-19 outbreak in China: a study driven by data</td><td>10.1101/2020.02.26.20028431</td><td>null</td><td>null</td><td>medrvix</td><td>The COVID-19 outbreak has been a serious public health threat worldwide. We use individually documented case descriptions of COVID-19 from China (excluding Hubei Province) to estimate the distributions of the generation time, incubation period, and periods from symptom onset to isolation and to diagnosis. The recommended 14-day quarantine period may lead to a 6.7% failure for quarantine. We recommend a 22-day quarantine period. The mean generation time is 3.3 days and the mean incubation period is 7.2 days. It took 3.7 days to isolate and 6.6 days to diagnose a patient after his/her symptom onset. Patients may become infectious on average 3.9 days before showing major symptoms. This makes contact tracing and quarantine ineffective. The basic reproduction number is estimated to be 1.54 with contact tracing, quarantine and isolation, mostly driven by super spreaders.</td><td>2020-03-01</td><td>Meili Li; Pian Chen; Qianqian Yuan; Baojun Song; Junling Ma</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028431</td></tr><tr><td>onkdv388</td><td>54ae20515fcef8417e255b19df1a5bf6f2ab40bb</td><td>medrxiv</td><td>Modelling the coronavirus disease (COVID-19) outbreak on the Diamond Princess ship using the public surveillance data from January 20 to February 20, 2020</td><td>10.1101/2020.02.26.20028449</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus disease 2019 (COVID-19) outbreak on the Diamond Princess ship has caused over 634 cases as of February 20, 2020. We model the transmission process on the ship with a stochastic model and estimate the basic reproduction number at 2.2 (95%CI: 2.1−2.4). We estimate a large dispersion parameter than other coronaviruses, which implies that the virus is difficult to go extinction. The epidemic doubling time is at 4.6 days (95%CI: 3.0−9.3), and thus timely actions were crucial. The lesson learnt on the ship is generally applicable in other settings.</td><td>2020-02-29</td><td>Shi Zhao; Peihua Cao; Daozhou Gao; Zian Zhuang; Marc Chong; Yongli Cai; Jinjun Ran; Kai Wang; Yijun Lou; Weiming Wang; Lin Yang; Daihai He; Maggie H Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028449</td></tr><tr><td>k260c04b</td><td>null</td><td>medrxiv</td><td>The Effects of \"Fangcang, Huoshenshan, and Leishenshan\" Makeshift Hospitals and Temperature on the Mortality of COVID-19</td><td>10.1101/2020.02.26.20028472</td><td>null</td><td>null</td><td>medrvix</td><td>Background In December 2019, a novel coronavirus disease (COVID-19) broke out in Wuhan, China, however, the factors affecting the mortality remain unclear. Methods Thirty-two days of data that were shared by China National Health Commission and China Weather Net were collected using standard forms. The difference in the mortality of confirmed and severe cases before and after the use of Fangcang, Huoshenshan, and Leishenshan makeshift hospitals (MSHs) was tested using Mann-Whitney U test. We also studied whether air temperature (AT) could affect the above outcomes of COVID-19 cases by performing Spearman analysis. Results The mortality of confirmed cases was significantly decreased both in Wuhan (U = 1, P < 0.001) and Hubei (U = 0, P < 0.001), while in non-Hubei regions, as a contrast, the mortality of confirmed cases remained unchanged (U = 40, P = 0.139). However, another eight days later, changes in the mortality in non-Hubei regions also became significant (U = 73, P = 0.039). Mortality of confirmed cases was found to be significantly correlated with temperature both in Wuhan (r = -0.441, P = 0.012) and Hubei (r = -0.440, P = 0.012). Conclusions Our findings indicated that both the use of MSHs and the rise of AT were beneficial to the survival of COVID-19 cases.</td><td>2020-02-29</td><td>Yuwen Cai; Tianlun Huang; Xin Liu; Gaosi Xu</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028472</td></tr><tr><td>zriuh5q5</td><td>691da9d5576aa959e22114a1ebb35548efd276a0</td><td>medrxiv</td><td>Infection Dynamics of Coronavirus Disease 2019 (Covid-19) Modeled with the Integration of the Eyring Rate Process Theory and Free Volume Concept</td><td>10.1101/2020.02.26.20028571</td><td>null</td><td>null</td><td>medrvix</td><td>The Eyrings rate process theory and free volume concept, two very popular theories in chemistry and physics fields, are employed to treat infectious disease transmissions. The susceptible individ- uals are assumed to move stochastically from one place to another. The virus particle transmission rate is assumed to obey the Eyring rate process theory and also controlled by how much free volume available in a system. The transmission process is considered to be a sequential chemical reaction, and the concentrations or fractions of four epidemiological compartments, the susceptible, the exposed, the infected, and the removed, can be derived and calculated. The obtained equations show that the basic reproduction number, R0, is not a constant, dependent on the volume frac- tion of virus particles, virus particle size, and virus particle packing structure, the energy barrier associated with susceptible individuals, and environment temperature. The developed models are applied to treat coronavirus disease 2019 (Covid-19) transmission and make predictions on peak time, peak infected, and R0. Our work provides a simple and straightforward approach to estimate how infection diseases evolve and how many people may be infected.</td><td>2020-02-29</td><td>Tian Hao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028571</td></tr><tr><td>nsp2t0i6</td><td>null</td><td>medrxiv</td><td>Heart injury signs are associated with higher and earlier mortality in coronavirus disease 2019 (COVID-19)</td><td>10.1101/2020.02.26.20028589</td><td>null</td><td>null</td><td>medrvix</td><td>Importance: Heart injury can be easily induced by viral infection such as adenovirus and enterovirus. However, whether coronavirus disease 2019 (COVID-19) causes heart injury and hereby impacts mortality has not yet been fully evaluated. Objective: To explore whether heart injury occurs in COVID-19 on admission and hereby aggravates mortality later. Design, Setting, and Participants A single-center retrospective cohort study including 188 COVID-19 patients admitted from December 25, 2019 to January 27, 2020 in Wuhan Jinyintan Hospital, China; follow up was completed on February 11, 2020. Exposures: High levels of heart injury indicators on admission (hs-TNI; CK; CK-MB; LDH; α-HBDH). Main Outcomes and Measures: Mortality in hospital and days from admission to mortality (survival days). Results: Of 188 patients with COVID-19, the mean age was 51.9 years (standard deviation: 14.26; range: 21~83 years) and 119 (63.3%) were male. Increased hs-TnI levels on admission tended to occur in older patients and patients with comorbidity (especially hypertension). High hs-TnI on admission (≥ 6.126 pg/mL), even within the clinical normal range (0~28 pg/mL), already can be associated with higher mortality. High hs-TnI was associated with increased inflammatory levels (neutrophils, IL-6, CRP, and PCT) and decreased immune levels (lymphocytes, monocytes, and CD4+ and CD8+ T cells). CK was not associated with mortality. Increased CK-MB levels tended to occur in male patients and patients with current smoking. High CK-MB on admission was associated with higher mortality. High CK-MB was associated with increased inflammatory levels and decreased lymphocytes. Increased LDH and α-HBDH levels tended to occur in older patients and patients with hypertension. Both high LDH and α-HBDH on admission were associated with higher mortality. Both high LDH and α-HBDH were associated with increased inflammatory levels and decreased immune levels. hs-TNI level on admission was negatively correlated with survival days (r= -0.42, 95% CI= -0.64~-0.12, P=0.005). LDH level on admission was negatively correlated with survival days (r= -0.35, 95% CI= -0.59~-0.05, P=0.022). Conclusions and Relevance: Heart injury signs arise in COVID-19, especially in older patients, patients with hypertension and male patients with current smoking. COVID-19 virus might attack heart via inducing inflammatory storm. High levels of heart injury indicators on admission are associated with higher mortality and shorter survival days. COVID-19 patients with signs of heart injury on admission must be early identified and carefully managed by cardiologists, because COVID-19 is never just confined to respiratory injury.</td><td>2020-02-29</td><td>Chaomin Wu; Xianglin Hu; Jianxin Song; Chunling Du; Jie Xu; Dong Yang; Dechang Chen; Ming Zhong; Jinjun Jiang; Weining Xiong; Ke Lang; Yuye Zhang; Guohua Shi; Lei Xu; Yuanlin Song; Xin Zhou; Ming Wei; Junhua Zheng</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.20028589</td></tr><tr><td>fvig79k3</td><td>11875dfda6cd04d5032611edf0e340582ceb620f</td><td>biorxiv</td><td>Predictions for the binding domain and potential new drug targets of 2019-nCoV</td><td>10.1101/2020.02.26.961938</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAn outbreak of new SARS-like viral in Wuhan, China has been named 2019-nCoV. The current state of the epidemic is increasingly serious, and there has been the urgent necessity to develop an effective new drug. In previous studies, it was found that the conformation change in CTD1 was the region where SARS-CoV bound to human ACE2. Although there are mutations of the 2019-nCoV, the binding energy of ACE2 remains high. The surface glycoprotein of 2019-nCoV was coincident with the CTD1 region of the S-protein by comparing the I-TASSER prediction model with the actual SARS model, which suggests that 2019-nCoV may bind to the ACE2 receptor through conformational changes. Furthermore, site prediction on the surface glycoprotein of 2019-nCoV suggests some core amino acid area may be a novel drug target against 2019-nCoV.</td><td>2020-03-02</td><td>Zehua Zeng; Zhi Luo; Hongwu Du</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.961938</td></tr><tr><td>wx7nheay</td><td>313dd0869f169624e78d770963eacb0d1641cafa</td><td>biorxiv</td><td>Structure of Mpro from COVID-19 virus and discovery of its inhibitors</td><td>10.1101/2020.02.26.964882</td><td>null</td><td>null</td><td>biorxiv</td><td>SUMMARYA new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan1-4. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Mpro is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus5,6. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus Mpro in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of Mpro. Six of these inhibit Mpro with IC50 values ranging from 0.67 to 21.4 μM. Ebselen also exhibited strong antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available.</td><td>2020-02-27</td><td>Zhenming Jin; Xiaoyu Du; Yechun Xu; Yongqiang Deng; Meiqin Liu; Yao Zhao; Bing Zhang; Xiaofeng Li; Leike Zhang; Chao Peng; Yinkai Duan; Jing Yu; Lin Wang; Kailin Yang; Fengjiang Liu; Rendi Jiang; Xinglou Yang; Tian You; Xiaoce Liu; Xiuna Yang; Fang Bai; Hong Liu; Xiang Liu; Luke W. Guddat; Wenqing Xu; Gengfu Xiao; Chengfeng Qin; Zhengli Shi; Hualiang Jiang; Zihe Rao; Haitao Yang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.964882</td></tr><tr><td>egntml7e</td><td>0a32446730827ad8152c6a61e4738e4e0b231412</td><td>biorxiv</td><td>Artesunate interacts with Vitamin D receptor to reverse mouse model of sepsis-induced immunosuppression via enhancing autophagy</td><td>10.1101/2020.02.26.966143</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackground and PurposeImmunosuppression is the predominant cause of mortality for sepsis due to failure to eradicate invading pathogens. Unfortunately, no effective and specific drugs capable of reversing immunosuppression are available for clinical use. Increasing evidence implicates vitamin D receptor (VDR) involved in sepsis-induced immunosuppression. Herein, artesunate (AS) was discovered to reverse sepsis-induced immunosuppression and its molecular mechanism is investigated.Experimental ApproachEffect of artesunate on sepsis-induced immunosuppression was investigated in mice and in vitro. VDR was predicted to be an interacted candidate of AS by bioinformatics predict, then identified using PCR and immunoblotting. VDR, ATG16L1 and NF-κB p65 were modified to investigate the alteration of AS’s effect on pro-inflammatory cytokines release, bacteria clearance and autophagy activities in sepsis-induced immunosuppression.Key ResultsAS significantly reduced the mortality of cecal ligation and puncture (CLP)-induced sepsis immunosuppression mice challenged with Pseudomonas Aeruginosa, and enhanced proinflammatory cytokines release and bacterial clearance to reverse sepsis-induced immunosuppression in vivo and in vitro. Mechanically, AS interacted with VDR thereby inhibited the nuclear translocation of VDR, then influencing ATG16L1 transcription and subsequent autophagy activity. In addition, AS inhibited physical interaction between VDR and NF-κB p65 in LPS tolerance macrophages, then promoted nuclear translocation of NF-κB p65, which activated the transcription of NF-κB p65 target genes such as pro-inflammatory cytokines.Conclusion and ImplicationsOur findings provide an evidence that AS interacted with VDR to reverse sepsis-induced immunosuppression in an autophagy and NF-κB dependent way, highlighting a novel approach for sepsis treatment and drug repurposing of AS in the future.</td><td>2020-02-27</td><td>Shenglan Shang; Jiaqi Wu; Xiaoli Li; Xin Liu; Pan Li; Chunli Zheng; Yonghua Wang; Songqing Liu; Jiang Zheng; Hong Zhou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.966143</td></tr><tr><td>7tkvl894</td><td>35604e3a5d38d6794543372e4dd9ccf1d8d9cd2a</td><td>biorxiv</td><td>CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design</td><td>10.1101/2020.02.26.967026</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe emergence and outbreak of SARS-CoV-2, the causative agent of COVID-19, has rapidly become a global concern and has highlighted the need for fast, sensitive, and specific tools to surveil circulating viruses. Here we provide assay designs and experimental resources, for use with CRISPR-based nucleic acid detection, that could be valuable for ongoing surveillance. We provide assay designs for detection of 67 viral species and subspecies, including: SARS-CoV-2, phylogenetically-related viruses, and viruses with similar clinical presentation. The designs are outputs of algorithms that we are developing for rapidly designing nucleic acid detection assays that are comprehensive across genomic diversity and predicted to be highly sensitive and specific. Of our design set, we experimentally screened 4 SARS-CoV-2 designs with a CRISPR-Cas13 detection system and then extensively tested the highest-performing SARS-CoV-2 assay. We demonstrate the sensitivity and speed of this assay using synthetic targets with fluorescent and lateral flow detection. Moreover, our provided protocol can be extended for testing the other 66 provided designs. Assay designs are available at https://adapt.sabetilab.org/.</td><td>2020-03-02</td><td>Hayden C. Metsky; Catherine A. Freije; Tinna-Solveig F. Kosoko-Thoroddsen; Pardis C. Sabeti; Cameron Myhrvold</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.26.967026</td></tr><tr><td>jadzias9</td><td>ae61300f4e7a12a68baf114f199eb51940c7aad3</td><td>medrxiv</td><td>Prediction of the Epidemic of COVID-19 Based on Quarantined Surveillance in China</td><td>10.1101/2020.02.27.20027169</td><td>null</td><td>null</td><td>medrvix</td><td>Background and Objective: To predict the epidemic of COVID-19 based on quarantined surveillance from real world in China by modified SEIR model different from the previous simply mathematical model. Design and Methods: We forecasted the epidemic of COVID-19 based on current clinical and epidemiological data and built a modified SEIR model to consider both the infectivity during incubation period and the influence on the epidemic from strict quarantined measures. Results: The peak time of the curve for the infected newly diagnosed as COVID-19 should substantially present on Feb.5,2020(in non-Hubei areas) and Feb.19,2020(in Hubei. It is estimated that the peak of the curve of the cumulative confirmed cases will appear in non-Hubei areas on Mar.3,2020 and in Hubei province on Mar.10,2020,and the total number of the patients diagnosed as COVID-19 is 18,000 in non-Hubei areas and 78,000-96,000 in Hubei.The Chinese COVID-19 epidemic can be completely controlled in May,2020. Conclusions: COVID-19 is only a local outbreak in Hubei Province,China.It can be probably avoided the pandemic of global SARS-CoV-2 cases rise with the great efforts by Chinese government and its people.</td><td>2020-02-29</td><td>Rui Li; Wenliang Lu; Xifei Yang; Peihua Feng; Ozarina Muqimova; Xiaoping Chen; Gang Wei</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20027169</td></tr><tr><td>2zm4dq0u</td><td>2e41aee6651cf4cab4edf5aa67cc401c2bef9d67</td><td>medrxiv</td><td>Sex differences in clinical findings among patients with coronavirus disease 2019 (COVID-19) and severe condition</td><td>10.1101/2020.02.27.20027524</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To compare the sex differences in the clinical findings among patients with severe coronavirus disease 2019 (COVID-19). Methods: We retrospectively collected data of 47 patients diagnosed as severe type of COVID-19 from February 8 to 22, 2020, including demographics, illness history, physical examination, laboratory test, management, and compared differences between men and women. Results: Of the 47 patients, 28 (59.6%) were men. The median age was 62 years, and 30 (63.8%) had comorbidities. The initial symptoms were mainly fever (34 [72.3%]), cough (36 [76.6%]), myalgia (5 [10.6%]) and fatigue (7 [14.9%]). Procalcitonin level was higher in men than in women (0.08 vs. 0.04ng/ml, p=0.002). N-terminal-pro brain natriuretic peptide increased in 16 (57.1%) men and 5 (26.3%) women (p=0.037). Five men (17.9%) had detected positive influenza A antibody, but no women. During 2-week admission, 5 (17.9%) men and 1 (5.3%) woman were reclassified into the critical type due to deterioration. Mortality was 3.6% in men and 0 in women respectively. Four (21.1%) women and one man (3.6%) recovered and discharged from hospital. Conclusion: Sex differences may exist in COVID-19 patients of severe type. Men are likely to have more complicated clinical condition and worse in-hospital outcomes as compared to women.</td><td>2020-02-29</td><td>Jing Li; Yinghua Zhang; Fang Wang; Bing Liu; Hui Li; Guodong Tang; Zhigang Chang; Aihua Liu; Chunyi Fu; Jing Gao; Jing Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20027524</td></tr><tr><td>owjgs6ja</td><td>2d0285c90976dfa43d58eca6b7d1d7ef29994fc2</td><td>medrxiv</td><td>Therapeutic effects of dipyridamole on COVID-19 patients with coagulation dysfunction</td><td>10.1101/2020.02.27.20027557</td><td>null</td><td>null</td><td>medrvix</td><td>The human coronavirus HCoV-19 infection can cause acute respiratory distress syndrome (ARDS), hypercoagulability, hypertension, extrapulmonary multiorgan dysfunction. Effective antiviral and anti-coagulation agents with safe clinical profiles are urgently needed to improve the overall prognosis. We screened an FDA approved drug library and found that an anticoagulant agent dipyridamole (DIP) suppressed HCoV-19 replication at an EC50 of 100 nM in vitro. It also elicited potent type I interferon responses and ameliorated lung pathology in a viral pneumonia model. In analysis of twelve HCoV-19 infected patients with prophylactic anti-coagulation therapy, we found that DIP supplementation was associated with significantly increased platelet and lymphocyte counts and decreased D-dimer levels in comparison to control patients. Two weeks after initiation of DIP treatment, 3 of the 6 severe cases (60%) and all 4 of the mild cases (100%) were discharged from the hospital. One critically ill patient with extremely high levels of D-dimer and lymphopenia at the time of receiving DIP passed away. All other patients were in clinical remission. In summary, HCoV-19 infected patients could potentially benefit from DIP adjunctive therapy by reducing viral replication, suppressing hypercoagulability and enhancing immune recovery. Larger scale clinical trials of DIP are needed to validate these therapeutic effects.</td><td>2020-02-29</td><td>Xiaoyan Liu; Zhe Li; Shuai Liu; Zhanghua Chen; Zhiyao Zhao; Yi-you Huang; Qingling Zhang; Jun Wang; Yinyi Shi; Yanhui Xu; Jing Sun; Huifang Xian; Rongli Fang; Fan Bai; Changxing Ou; Bei Xiong; Andrew M Lew; Jun Cui; Hui Huang; Jincun Zhao; Xuechuan Hong; Yuxia Zhang; Fulin Zhou; Hai-Bin Luo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20027557</td></tr><tr><td>s9j21zsy</td><td>bd2f7613f5818fc82f76c7d68bdac08825957fb9</td><td>medrxiv</td><td>A machine learning-based model for survival prediction in patients with severe COVID-19 infection</td><td>10.1101/2020.02.27.20028027</td><td>null</td><td>null</td><td>medrvix</td><td>The sudden increase of COVID-19 cases is putting a high pressure on healthcare services worldwide. At the current stage, fast, accurate and early clinical assessment of the disease severity is vital. To support decision making and logistical planning in healthcare systems, this study leverages a database of blood samples from 404 infected patients in the region of Wuhan, China to identify crucial predictive biomarkers of disease severity. For this purpose, machine learning tools selected three biomarkers that predict the survival of individual patients with more than 90% accuracy: lactic dehydrogenase (LDH), lymphocyte and high-sensitivity C-reactive protein (hs-CRP). In particular, relatively high levels of LDH alone seem to play a crucial role in distinguishing the vast majority of cases that require immediate medical attention. This finding is consistent with current medical knowledge that high LDH levels are associated with tissue breakdown occurring in various diseases, including pulmonary disorders such as pneumonia. Overall, this paper suggests a simple and operable formula to quickly predict patients at the highest risk, allowing them to be prioritised and potentially reducing the mortality rate.</td><td>2020-03-01</td><td>Li Yan; Hai-Tao Zhang; Jorge Goncalves; Yang Xiao; Maolin Wang; Yuqi Guo; Chuan Sun; Xiuchuan Tang; Liang Jin; Mingyang Zhang; Xiang Huang; Ying Xiao; Haosen Cao; Yanyan Chen; Tongxin Ren; Fang Wang; Yaru Xiao; Sufang Huang; Xi Tan; Niannian Huang; Bo Jiao; Yong Zhang; Ailin Luo; Laurent Mombaerts; Junyang Jin; Zhiguo Cao; Shusheng Li; Hui Xu; Ye Yuan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20028027</td></tr><tr><td>xiynfpuw</td><td>264a816e7099246c13a5dad3a581a474d85d50bd</td><td>medrxiv</td><td>Hypokalemia and Clinical Implications in Patients with Coronavirus Disease 2019 (COVID-19)</td><td>10.1101/2020.02.27.20028530</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND: SARS-CoV-2 has caused a series of COVID-19 globally. SARS-CoV-2 binds angiotensin I converting enzyme 2 (ACE2) of renin-angiotensin system (RAS) and causes prevalent hypokalemia METHODS: The patients with COVID-19 were classified into severe hypokalemia, hypokalemia, and normokalemia group. The study aimed to determine the relationship between hypokalemia and clinical features, the underlying causes and clinical implications of hypokalemia. RESULTS: By Feb 15, 2020, 175 patients with COVID-19 (92 women and 83 men; median age, 46 [IQR, 34-54] years) were admitted to hospital in Wenzhou, China, consisting 39 severe hypokalemia-, 69 hypokalemia-, and 67 normokalemia patients. Gastrointestinal symptoms were not associated with hypokalemia among 108 hypokalemia patients (P>0.05). Body temperature, CK, CK-MB, LDH, and CRP were significantly associated with the severity of hypokalemia (P<0.01). 93% of severe and critically ill patients had hypokalemia which was most common among elevated CK, CK-MB, LDH, and CRP. Urine K+ loss was the primary cause of hypokalemia. severe hypokalemia patients was given 3 g/day, adding up to an average of 34 (SD=4) g potassium during hospital stay. The exciting finding was that patients responded well to K+ supplements when they were inclined to recovery. CONCLUSIONS: Hypokalemia is prevailing in patients with COVID-19. The correction of hypokalemia is challenging because of continuous renal K+ loss resulting from the degradation of ACE2. The end of urine K+ loss indicates a good prognosis and may be a reliable, in-time, and sensitive biomarker directly reflecting the end of adverse effect on RAS system.</td><td>2020-02-29</td><td>dong chen; Xiaokuni Li; qifa song; Chenchan Hu; Feifei Su; Jianyi Dai</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20028530</td></tr><tr><td>ry3m6x8c</td><td>75e68869f9b65bca661e768402395d9dede6de2c</td><td>medrxiv</td><td>Modeling the Epidemic Dynamics and Control of COVID-19 Outbreak in China</td><td>10.1101/2020.02.27.20028639</td><td>null</td><td>null</td><td>medrvix</td><td>The coronavirus disease 2019 (COVID-19) is rapidly spreading over China and more than 30 countries in last two months. COVID-19 has multiple characteristics distinct from other infectious diseases, including a high infectivity during incubation, time delay between real dynamics and daily observed case numbers, and the effects from multiple quarantine and control measures. We develop a model SUQC to adequately characterizes the dynamics of COVID-19 and explicitly model the control by artificial measures, which is more suitable for analysis than other existing epidemic models. The SUQC model is applied to the daily released data of the confirmed infected to analyze the outbreak of COVID-19 in Wuhan, Hubei (excluding Wuhan), China (excluding Hubei) and four first-tier cities of China. We find that, before January 30, 2020, all these regions except Beijing have a reproductive number R>1, and after January 30, all regions have a reproductive number R<1, indicating the effectiveness of the quarantine and control measures in inhibiting COVID-19. The confirmation rate of Wuhan is 0.0643, significantly lower than 0.1914 of Hubei (excluding Wuhan) and 0.2189 of China (excluding Hubei), but increases to 0.3229 after Feb 12th when clinical diagnosis was adopted. The un-quarantined infected individuals in Wuhan on February 12, 2020 is as high as 3,509 and decreases to 334 on February 21th, 2020. After fitting the model with recent data, we predict that the end times of COVID-19 of Wuhan and Hubei are around late-March, of China (excluding Hubei) around mid-March, and of the four tier-one cities before March 2020. A total of 80,511 individuals of the whole country are infected, among which 49,510 are from Wuhan, 17,679 from Hubei(excluding Wuhan), and the rest 13,322 from other regions of China (excluding Hubei). We suggest the rigorous quarantine and control measures should be kept before March in Beijing, Shanghai, Guangzhou and Shenzhen, and before late-March in Hubei. The model can also be useful to predict the trend of epidemic and provide quantitative guide for other counties in a high risk of outbreak, such as South Korea, Japan and Iran.</td><td>2020-02-29</td><td>Shilei Zhao; Hua Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20028639</td></tr><tr><td>rbnfh89u</td><td>b579c3547bba2a33057373d57b7c05f37dd4cfc3</td><td>medrxiv</td><td>Evaluation of Enzyme-Linked Immunoassay and Colloidal Gold- Immunochromatographic Assay Kit for Detection of Novel Coronavirus (SARS-Cov-2) Causing an Outbreak of Pneumonia (COVID-19)</td><td>10.1101/2020.02.27.20028787</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract BACKGROUND: In December 2019, a novel coronavirus (SARS-CoV-2) infected pneumonia (COVID-19) occurred in Wuhan, China. Travel-associated cases have also been reported in other countries. The number of cases has increased rapidly but laboratory diagnosis is limited. METHODS: We collect two groups of cases diagnosed with COVID-19 for experiments. One group collected 63 samples for Enzyme-linked immunosorbent assay (ELISA) IgG and IgM antibodies. The other group collected 91 plasma samples for colloidal gold-immunochromatographic assay (GICA). RESULTS: The sensitivity of the combined ELISA IgM and ELISA IgG detection was 55/63 ( 87.3%), The sensitivity of the combined GICA IgM and GICA IgG detection was 75/91 ( 82.4%), Both methods are negative for healthy controls, specificity of 100% .There is no significant difference between the sensitivity of between ELISA and GICA (IgM+ IgG). CONCLUSIONS: ELISA and GICA for specific IgM and IgG antibodies are conventional serological assays, they are simple, fast, and safe, the results can be used for clinical reference, and the huge clinical diagnosis and treatment pressure can be greatly relieved.</td><td>2020-03-01</td><td>Jie Xiang; Mingzhe Yan; Hongze Li; Ting Liu; Chenyao Lin; Shuang Huang; Changxin Shen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20028787</td></tr><tr><td>z7a3g6e8</td><td>905603e9db862f9b4b42a95f51e9c8d0015d1dff</td><td>medrxiv</td><td>Transmission potential of COVID-19 in South Korea</td><td>10.1101/2020.02.27.20028829</td><td>null</td><td>null</td><td>medrvix</td><td>Since the first identified individual of 2019 novel coronavirus (COVID-19) infection on Jan 20, 2020 in South Korea, the number of confirmed cases rapidly increased. As of Feb 26, 2020, 1,261 cases of COVID-19 including 12 deaths were confirmed in South Korea. Using the incidence data of COVID-19, we estimate the reproduction number at 1.5 (95% CI: 1.4-1.6), which indicates sustained transmission and support the implementation of social distancing measures to rapidly control the outbreak.</td><td>2020-02-29</td><td>Eunha Shim; Amna Tariq; Wongyeong Choi; Yiseul Lee; Gerardo Chowell</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20028829</td></tr><tr><td>hgn6o5yf</td><td>null</td><td>medrxiv</td><td>A simple ecological model captures the transmission pattern of the coronavirus COVID-19 outbreak in China</td><td>10.1101/2020.02.27.20028928</td><td>null</td><td>null</td><td>medrvix</td><td>The rapid spread of the 2019 novel coronavirus (COVID-19), initially reported in the city of Wuhan in China, and quickly transmitted to the entire nation and beyond, has become an international public health emergency. Estimating the final number of infection cases and the turning point (time with the fastest spreading rate) is crucial to assessing and improving the national and international control measures currently being applied. In this paper we develop a simple model based on infectious growth with a time-varying infection rate, and estimate the final number of infections and the turning point using data updated daily from 3 February 2020, when China escalated its initial public health measures, to 10 February. Our model provides an extremely good fit to the existing data and therefore a reasonable estimate of the time-varying infection rate that has largely captured the transmission pattern of this epidemic outbreak. Our estimation suggests that (i) the final number of infections in China could reach 78,000 with an upper 95% confidence limit of 88,880; (ii) the turning point of the fastest spread was on the 4th or the 5th of February; and (iii) the projected period for the end of the outbreak (i.e., when 95% of the final predicted number of infection is reached) will be the 24th of February, with an upper 95% confidence limit on the 19th of March. This suggests that the current control measures in China are excellent, and more than sufficient to contain the spread of this highly infectious novel coronavirus, and that the application of such measures could be considered internationally for the global control of this outbreak.</td><td>2020-02-29</td><td>Feng Zhang; Jinmei Zhang; Menglan Cao; Cang Hui</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20028928</td></tr><tr><td>9hh17k86</td><td>68ed7bb581b0cf32116ee46767c08e3180b05c34</td><td>medrxiv</td><td>Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</td><td>10.1101/2020.02.27.20029009</td><td>null</td><td>null</td><td>medrvix</td><td>Background Although the outbreak of Coronavirus disease 2019 (COVID-19) has caused over 2200 deaths in China, there was no study about death yet. We aimed to describe the clinical characteristics of non-survivors with COVID-19. Methods For this retrospective, single-center study, we included 36 non-survivors with COVID-19 in the Fifth Hospital of Wuhan. Cases were confirmed by real-time RT-PCR between Jan 21 and Feb 10, 2020 according to the recommended protocol. The epidemiological, demographic, clinical, laboratory, radiological and treatment data were collected and analyzed. Outcomes were followed up until Feb 14, 2020. This study was approved by the ethics commissions of the Fifth Hospital of Wuhan, with a waiver of informed consent due to a public health outbreak investigation. Results We included 36 patients who died from COVID-19. The mean age of the patients was 69.22 years (SD 9.64, range 50-90). 25(69.44%) patients were males, and 11 (30.56%) female. 26 (72.22%) patients had chronic diseases, mainly including hypertension, cardiovascular disease and diabetes. Patients had common clinical symptoms of fever (34 [94.44%] patients), cough (28 [77.78%] patients), shortness of breath (21 [58.33%] patients), and fatigue (17 [47.22%] patient). Chest computed tomographic scans showed that 31 (96.88%) patients had bilateral pneumonia. Lymphopenia occurred in 24 patients (70.59%), decreased albumin (30.18, [SD, 4.76]) in 25 patients (80.65%), elevated D-dimer (8.64 [IQR, 2.39-20]) in 27 patients (100%), and elevated lactate dehydrogenase (502.5 U/L [IQR, 410-629]) in 26 patients (100%). Nearly all of the patients have elevated CRP (106.3 mg/L [IQR, 60.83-225.3]), PCT (0.61 ng/ml [IQR, 0.16-2.10]) and IL-6 (100.6 pg/ml [IQR, 51.51-919.5]). Most patients received antiviral therapy and antibiotic therapy, and more than half of patients received glucocorticoid therapy (25 [69.44%]). All the patients had acute respiratory distress syndrome (ARDS). The median time from onset to ARDS was 11 days. One (2.78%) patient presented with acute renal injury. The median time from onset to death was 17 days. Interpretation Lots of patients died from COVID-19 till now. The median survival time of these non-survivors from onset to death was about 2 weeks. Most patients were older males with comorbidities. They finally progressed to ARDS. The median time from onset to ARDS was 11 days. Gradually decreased lymphocytes and increased inflammation biomarkers were common, and need to be monitored in the routine treatment.</td><td>2020-02-29</td><td>Ying Huang; Rui Yang; Ying Xu; Ping Gong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.20029009</td></tr><tr><td>618glydc</td><td>1b9fed2824e97db8cb35912549dd2e9cde7b6c18</td><td>biorxiv</td><td>Strategies for vaccine design for corona virus using Immunoinformatics techniques</td><td>10.1101/2020.02.27.967422</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores −725.0 Kcal/mole and −786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study.</td><td>2020-03-02</td><td>Anamika Basu; Anasua Sarkar; Ujjwal Maulik</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.967422</td></tr><tr><td>eaglecq7</td><td>92c020779e68e82d9a5f5898435e618caa63a099</td><td>biorxiv</td><td>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</td><td>10.1101/2020.02.27.967588</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractFaced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the “coronavirus” should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity. However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.</td><td>2020-03-02</td><td>Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Chunli Zhang; Yiqing Xue; Guangming Tan; Gang Niu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.967588</td></tr><tr><td>1f9kjtmm</td><td>null</td><td>biorxiv</td><td>The SARS-CoV-2 receptor ACE2 expression of maternal-fetal interface and fetal organs by single cell transcriptome study</td><td>10.1101/2020.02.27.967760</td><td>null</td><td>null</td><td>biorxiv</td><td>Recent studies have demonstrated that SARS-CoV-2 cell entry depends on both ACE2 and TMPRSS2 genes (DOI: 10.1016/j.cell.2020.02.052), but our current work only focus on ACE2, which is insufficient to support the conclusion of this paper. So the authors have withdrawn their manuscript whilst they perform additional experiments and analysis to test some of their conclusions further. Therefore, the authors do not wish this work to be cited as reference for the project.</td><td>2020-02-27</td><td>Mengmeng Li; Liang Chen; Chenglong Xiong; Xiangjie Li</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.967760</td></tr><tr><td>9ruhvpbv</td><td>ae66f22f886d3b019934ae237c11a5fb53dbefb2</td><td>biorxiv</td><td>Prediction of receptorome for human-infecting virome</td><td>10.1101/2020.02.27.967885</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome.</td><td>2020-02-28</td><td>Zheng Zhang; Sifan Ye; Aiping Wu; Taijiao Jiang; Yousong Peng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.967885</td></tr><tr><td>mp3a9c9u</td><td>7445c94870da80c81cd17f6a4c1358004956ee36</td><td>biorxiv</td><td>Molecular Dynamics Simulations Indicate the SARS-CoV-2 Mpro Is Not a Viable Target for Small-Molecule Inhibitors Design</td><td>10.1101/2020.02.27.968008</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe coronavirus outbreak took place in December 2019 and continues to spread worldwide. In the absence of an effective vaccine, inhibitor repurposing may seem a fruitful attempt. Here, we compared Mpros from SARS-CoV-2 and SARS-CoV. Despite a high level of sequence similarity, the binding sites of analysed proteins show major differences in both shape and size indicating that repurposing SARS drugs for COVID-19 may be futile. Furthermore, the analysis of the pockets’ conformational changes during the simulation time indicates their flexibility, which dashes hopes for rapid and reliable drug design. Conversely, structural stability of the SARS-CoV-2 Mpro with respect to mutations of the binding cavity and adjacent flexible loops indicates that the protein’s mutability will pose a further challenge to the rational design of small-molecule inhibitors. However, few residues contribute significantly to the protein stability and thus can be considered as key anchoring residues for Mpro inhibitor design.</td><td>2020-03-02</td><td>Maria Bzówka; Karolina Mitusińska; Agata Raczyńska; Aleksandra Samol; Jack Tuszyński; Artur Góra</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.968008</td></tr><tr><td>3h1o0oz3</td><td>6d0127f985edbe088bc279865bef25a31f54a066</td><td>biorxiv</td><td>Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13</td><td>10.1101/2020.02.27.969006</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.</td><td>2020-03-02</td><td>Longxian Lv; Gaolei Li; Jinhui Chen; Xinle Liang; Yudong Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.27.969006</td></tr><tr><td>2vdvswda</td><td>24e17488d399c436305c819953beae2961214771</td><td>medrxiv</td><td>Epidemiological and clinical features of COVID-19 patients with and without pneumonia in Beijing, China</td><td>10.1101/2020.02.28.20028068</td><td>null</td><td>null</td><td>medrvix</td><td>Background:SARS-CoV-2-caused coronavirus disease (COVID-19) is posinga large casualty. The features of COVID-19patients withand without pneumonia,SARS-CoV-2 transmissibility in asymptomatic carriers, and factors predicting disease progression remain unknown. Methods: We collected information on clinical characteristics, exposure history, andlaboratory examinations of all laboratory-confirmed COVID-19 patients admitted to PLA General Hospital. Cox regression analysis was applied to identify prognostic factors. The last follow-up was February 18, 2020. Results:We characterized 55 consecutive COVID-19 patients. The mean incubation was 8.42(95% confidence interval [CI], 6.55-10.29) days. The mean SARS-CoV-2-positive duration from first positive test to clearance was 9.71(95%CI, 8.21-11.22) days. COVID-19 course was approximately 2 weeks. Asymptomatic carriers might transmit SARS-CoV-2. Compared with patients without pneumonia, those with pneumonia were 15 years older and had a higher rate of hypertension, higher frequencies of having a fever and cough, and higher levels of interleukin-6 (14.61 vs. 8.06pg/mL, P=0.040), B lymphocyte proportion (13.0% vs.10.0%, P=0.024), low account (<190/μL) of CD8+ T cells (33.3% vs. 0, P=0.019). Multivariate Cox regression analysis indicated that circulating interleukin-6 andlactate independently predicted COVID-19 progression, with a hazard ratio (95%CI) of 1.052 (1.000-1.107) and 1.082 (1.013-1.155), respectively. During disease course,T lymphocytes were generally lower,neutrophils higher, in pneumonia patients than in pneumonia-free patients. CD8+ lymphocytes did not increase at the 20th days after illness onset. Conclusion: The epidemiological features areimportant for COVID-19 prophylaxis. Circulating interleukin-6 and lactateare independent prognostic factors. CD8+ T cell exhaustion might be critical in the development of COVID-19.</td><td>2020-03-03</td><td>Penghui Yang; Yibo Ding; Zhe Xu; Rui Pu; Ping Li; Jin Yan; Jiluo Liu; Fanping Meng; Lei Huang; Lei Shi; Tianjun Jiang; Enqiang Qin; Min Zhao; Dawei Zhang; Peng Zhao; Lingxiang Yu; Zhaohai Wang; Zhixian Hong; Zhaohui Xiao; Qing Xi; Dexi Zhao; Peng Yu; Caizhong Zhu; Zhu Chen; Shaogeng Zhang; Junsheng Ji; Guangwen Cao; Fusheng Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.20028068</td></tr><tr><td>73gxq2w6</td><td>54cfc3c68e1e4832fee5b3294e5673e37978ebdf</td><td>medrxiv</td><td>Risk factors related to hepatic injury in patients with corona virus disease 2019</td><td>10.1101/2020.02.28.20028514</td><td>null</td><td>null</td><td>medrvix</td><td>Aims: Corona virus disease 2019 (COVID-19) has rapidly become the most severe public health issue all over the world. Despite respiratory symptoms, hepatic injury has also been observed in clinical settings. This study aimed to investigate the risk factors involved with hepatic injury in the patients with COVID-19. Methods: A total of 85 hospitalized patients who were diagnosed with COVID-19 in Beijing Youan Hospital were retrospectively analyzed. According to liver function, they were divided into ALT normal group (n=52) and ALT elevation group (n=33). Clinical features and laboratory data were compared between the two groups. The independent risk factors for liver injury were analyzed. Results: There were 33 patients with hepatic injury in our study, accounting for 38.8% (33/85). The patients in ALT elevation group were older than those in ALT normal group. The levels of lactic acid, CRP, myoglobin, and neutrophils were significantly higher in ALT elevation group. The lymphocytes and albumin were significantly lower in ALT elevation group. The proportion of severe and critical patients in ALT elevation group was significantly higher. Multivariate logistic regression analysis showed CRP ≥20 mg/L and lymphocyte count< 1.1*10^9/L were independently related to hepatic injury. Conclusions: Lymphopenia and CRP may serve as the risk factors related to hepatic injury in patients with COVID-19, which might be related to inflammatory cytokine storm in liver injury. Early detection and timely treatment of hepatic injury in patients with COVID-19 are necessary.</td><td>2020-03-03</td><td>Lu Li; Shuang Li; Manman Xu; Pengfei Yu; Sujun Zheng; Zhongping Duan; Jing Liu; Yu Chen; Junfeng Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.20028514</td></tr><tr><td>3wvcuboc</td><td>d75c48f51b2a5c26a41fe615339e5947d5e9cb30</td><td>medrxiv</td><td>Analysis of epidemiological characteristics of coronavirus 2019 infection and preventive measures in Shenzhen China: a heavy population city</td><td>10.1101/2020.02.28.20028555</td><td>null</td><td>null</td><td>medrvix</td><td>Coronavirus 2019 infection (COVID-19) outbroke in Wuhan, Hubei and spread to all provinces in China and other countries. Shenzhen ranked the top cities outside Wuhan with reported 416 confirmed cases by February 20, 2020. Here, we analyzed the epidemiological characteristics of COVID-19 in Shenzhen and potential link to the preventive strategies for the whole city and inside hospitals. Based on the daily new cases, the epidemic of COVID-19 in Shenzhen can be classified into three phases: the slow increase phase from January 19 to January 28, the rapid increase and plateau phase from January 29 to February 5 and the decline phase since February 6. In the three phases, the number of patients from Hubei decreased, and the number of familial clustering cases increased. The newly diagnosed COVID-19 cases reached its peak around January 31, which was 7 days after the peak date of cases arrival at Shenzhen. A series of early preventive strategies were implemented since January 19, which included detection of body temperature at all entrances of main traffic and buildings, outpatients service specially for patients with fever in all main hospitals in Shenzhen. All the patients with fever were screened with nasal or throat swab PCR detection of coronavirus 2019, Chest CT and blood lymphocyte counting in order to find out early case of COVID-19. Observation wards were established in every main hospital and a designated hospital was responsible for admission and medical care of all confirmed cases. Protection procedure was established for all medical staff involved in the screening and care of suspected and confirmed cases. 14 days isolated observation of all subjects arrived at Shenzhen from Hubei was implemented in February 2. After the implementation of all these strategies and measures, the COVID-19 cases started to decline since February 6. There were almost no community transmission and nosocomial infection occurred in Shenzhen. In conclusion, in situation of major outbreak of respiratory infectious disease, such as COVID-19, in nearby province of Hubei, Shenzhen, a high population density, high proportion of external population and high mobility city, has to face the imported cases and risk of spreading the outbreak into Shenzhen city. The implementation of early preventive strategies and measures in Shenzhen were successful in early identification of COVID-19 cases and prevented major outbreak occurred in Shenzhen. Early identification of imported cases, prevention of family clustering transmission, preventive measures in the public area and very strict infection control procedure in hospital setting are crucial for the successful control of outbreak in Shenzhen.</td><td>2020-03-03</td><td>Kai Yang; Lingwei Wang; Furong Li; Dandan Chen; Xi Li; Chen Qiu; Rongchang Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.20028555</td></tr><tr><td>py38bel4</td><td>8575c06c64fdd0bb43efc46cd994f557698c755f</td><td>medrxiv</td><td>Clinical significance of IgM and IgG test for diagnosis of highly suspected COVID-19 infection</td><td>10.1101/2020.02.28.20029025</td><td>null</td><td>null</td><td>medrvix</td><td>Quick, simple and accurate diagnosis of suspected COVID-19 is very important for the screening and therapy of patients. Although several methods were performed in clinical practice, however, the IgM and IgG diagnostic value evaluation was little performed. 57 suspected COVID-19 infection patients were enrolled in our study. 24 patients with positive and 33 patients with negative nucleic acid test. The positive rate of COVID-19 nucleic acid was 42.10%. The positive detection rate of combination of IgM and IgG for patients with COVID-19 negative and positive nucleic acid test was 72.73% and 87.50%. The results were significantly higher than the nucleic acid or IgM, IgG single detection. hsCRP in the COVID-19 nucleic acid negative group showed significantly higher than the positive groups (P=0.0298). AST in the COVID-19 IgM negative group showed significantly lower than the positive groups (P=0.0365). We suggest a quick, simple, accurate aided detection method for the suspected patients and on-site screening in close contact with the population.</td><td>2020-03-03</td><td>Xingwang Jia; Pengjun Zhang; Yaping Tian; Junli Wang; Huadong Zeng; Jun Wang; Liu Jiao; Zeyan Chen; Lijun Zhang; Haihong He; Kunlun He; Yajie Liu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.20029025</td></tr><tr><td>p5ecpcll</td><td>a5039336577ac0fed15b784adb912df2a6a143f3</td><td>medrxiv</td><td>Analysis on the Clinical Characteristics of 36 Cases of Novel Coronavirus Pneumonia in Kunming</td><td>10.1101/2020.02.28.20029173</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To analyze the clinical characteristics of patients with novel coronavirus pneumonia in Kunming City, and to study the correlation between nutritional status and immune function. Methods: Clinical data of 36 patients with novel coronavirus pneumonia in isolation area of Kunming Third People's Hospital from January 31 to February 15, 2020 were collected, and the basic situation, clinical characteristics, laboratory examination and CT imaging characteristics were analyzed. Serum albumin (ALB), prealbumin (PAB), hypersensitive c-reactive protein (hs-crp), CD3T cells, CD4T cells, CD8T cells and normal control group were analyzed. A simple linear regression analysis of the relationship between proalbumin and T cell subpopulation counts in the blood of patients. Results: (1) The patients with new coronavirus pneumonia in Kunming were mainly of common type. (2) 50% of the patients' first symptoms were fever and cough; (3) The total number of white blood cells in peripheral blood was normal or decreased in 23 cases (79%), and the lymphocyte count decreased in 5 cases (13.89%), without anemia. Hypersensitive c-reactive protein increased in 19 (52.78%) cases, and procalcitonin increased in 1 case. Albumin decreased in 5 cases (13.89%), proalbumin decreased in 15 cases (41.67%), alanine transaminase increased slightly in 4 cases (11.11%), alanine transaminase increased slightly in 4 cases (11.11%), total bilirubin increased slightly in 11 cases (30.56%), and renal function and blood coagulation were normal. Absolute value of CD3+T cells is with a decrease in 21 cases (58.3%), CD4+T in 28 cases (77.8%), CD8+T in 17 cases (47.2%), and CD4+/ CD8+ inverse in 6 cases (16.7%). (4) The prealbumin, CD3 T cells, CD4 T cells and CD8 T cells in the new coronavirus pneumonia group were significantly lower than those in the normal control group, and the hypersensitive c-reactive protein was higher than that in the normal control group. (5) The levels of PAB in the serum of the patients were linearly correlated with hs-crp, CD3 T cells, CD4 T cells and CD8 T cells, and the correlation coefficients were -0.474, 0.558, 0.467 and 0.613, respectively, showing statistical differences. Conclusion: The clinical characteristics of the novel coronavirus pneumonia in Kunming are different from those in Wuhan. The changes of serum proalbumin and T cell subsets are relatively obvious. Changes in serum proalbumin may contribute to the early warning of novel coronavirus pneumonia. The nutritional status of patients with common and mild pneumonia should be considered.</td><td>2020-03-01</td><td>Haiyan Fu; Hongjuan Li; Xiaoqing Tang; Xiang Li; Jie Shen; Yujun Zhou; Bing Xu; Yu Luo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.20029173</td></tr><tr><td>6uaj8fb7</td><td>2af91bbb625d983d5ad3aadb9e5f1d82268a0f1f</td><td>medrxiv</td><td>Highly ACE2 Expression in Pancreas May Cause Pancreas Damage After SARS-CoV-2 Infection</td><td>10.1101/2020.02.28.20029181</td><td>null</td><td>null</td><td>medrvix</td><td>The ongoing outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) started in the end of 2019 in China has triggered a global public health crisis. Previous studies have shown that SARS-CoV-2 infects cells by binding angiotensin-converting enzyme 2 (ACE2), which is the same as SARS-CoV. The expression and distribution of ACE2 in the pancreas are unknown. At the same time, the injury of pancreas after SARS-CoV-2 infection has not been concerned. Here, we collected public datasets (bulk RNA-seq and single-cell RNA-seq) to indicate the expression and the distribution of ACE2 in pancreas (in both exocrine glands and islets). And further, clinical data including mild and severe patients with COVID-19 demonstrated there existed mild pancreatitis. In the 67 severe cases, 11 patients (16.41%) showed elevated levels of both amylase and lipase, and 5 patients (7.46%) showed imaging alterations. Only one patient (1.85%) showed elevated levels of both amylase and lipase in 54 mild cases, without imaging changes. Our study revealed the phenomenon and possible cause of mild pancreatic injury in patients with COVID-19. This suggests that pancreatitis after SARS-CoV-2 infection should also be paid attention in clinical work.</td><td>2020-03-03</td><td>Furong Liu; Xin Long; Wenbin Zou; Minghao Fang; Wenjuan Wu; Wei Li; Bixiang Zhang; Wanguang Zhang; Xiaoping Chen; Zhanguo Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.20029181</td></tr><tr><td>zr34wsae</td><td>530f3ba1a2c044305d73123fd8d2246f63120bc3</td><td>medrxiv</td><td>The Impact of the COVID-19 Outbreak on the Medical Treatment of Chinese Children with Chronic Kidney Disease (CKD)：A Multicenter Cross-section Study in the Context of a Public Health Emergency of International Concern</td><td>10.1101/2020.02.28.20029199</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To investigate the impact of the COVID-19 outbreak on the medical advice seeking of Chinese children with chronic kidney disease (CKD). Materials and Methods: An anonymous online questionnaire survey was conducted in 17 pediatric nephropathy diagnosis and treatment centers in China. The questions collected basic information on the patients and their parents and data on changes in the approach to medical treatment and their needs in the context of the outbreak etc. This is a Multicenter Cross-section Study. Results: A total of 735 valid questionnaires were collected. 555 patients (75.5%) and their parents said that the outbreak had a significant influence on their medical treatment: 264 patients (47.6%) said that it would be delayed by 2 to 4 weeks and 199 patients (35.9%) by 4 to 8 weeks. 510 patients (84.16%) hoped to get in touch with specialists through online consultation, and 528 patients (84.5%) hoped that online consultation could be implemented and that medication could be delivered to them.. A total of 458 patients (62.3%) said that their greatest concern was that the CKD would be aggravated or that they would experience a relapse; only 203 patients were infected by 2019-nCoV. A total of 313 patients (42.5%) experienced anxiety and thus required the intervention of psychologists. Conclusion: The COVID-19 outbreak has affected the medical treatment of children with CKD. Online consultation, medication delivery and psychological counselling are the greatest needs reported by patients and their families and could especially provide solutions for the management of low income children with CKD in remote rural areas in the context of the COVID-19 epidemic.</td><td>2020-03-03</td><td>Gaofu Zhang; Haiping Yang; Aihua Zhang; Qian Shen; Li Wang; Zhijuan Li; Yuhong Li; Lijun Zhao; Yue Du; Liangzhong Sun; Bo Zhao; Hongtao Zhu; Haidong Fu; Xiaoyan Li; Xiaojie Gao; Sheng Hao; Juanjuan Ding; Zongwen Chen; Zhiquan Xu; Xiaorong Liu; Daoqi Wu; Mingsi Gao; Mo Wang; Qiu Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.20029199</td></tr><tr><td>p3dex2to</td><td>null</td><td>medrxiv</td><td>Closed environments facilitate secondary transmission of coronavirus disease 2019 (COVID-19)</td><td>10.1101/2020.02.28.20029272</td><td>null</td><td>null</td><td>medrvix</td><td>Commissioned by the Minister of the Ministry of Health, Labour, and Welfare of Japan, we collected secondary transmission data with the aim of identifying high risk transmission settings. We show that closed environments contribute to secondary transmission of COVID-19 and promote superspreading events. Closed environments are consistent with large-scale COVID-19 transmission events such as that of the ski chalet-associated cluster in France and the church- and hospital-associated clusters in South Korea. Our findings are also consistent with the declining incidence of COVID-19 cases in China, as gathering in closed environments was prohibited in the wake of the rapid spread of the disease. Reduction of unnecessary close contact in closed environments may help prevent large case clusters and superspreading events.</td><td>2020-03-03</td><td>Hiroshi Nishiura; Hitoshi Oshitani; Tetsuro Kobayashi; Tomoya Saito; Tomimasa Sunagawa; Tamano Matsui; Takaji Wakita; Motoi Suzuki</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.20029272</td></tr><tr><td>5hkd80eh</td><td>da3aa20131ac2805c0d9e1b29f094683479ab5b7</td><td>biorxiv</td><td>Ruler elements in chromatin remodelers set nucleosome array spacing and phasing</td><td>10.1101/2020.02.28.969618</td><td>null</td><td>null</td><td>biorxiv</td><td>Arrays of regularly spaced nucleosomes dominate chromatin and are often phased by alignment to reference sites like active promoters. How the distances between nucleosomes (spacing), and between phasing sites and nucleosomes are determined remains unclear, and specifically, how ATP dependent chromatin remodelers impact these features. Here, we used genome-wide reconstitution to probe how Saccharomyces cerevisiae ATP dependent remodelers generate phased arrays of regularly spaced nucleosomes. We find that remodelers bear a functional element named the ‘ruler’ that determines spacing and phasing in a remodeler-specific way. We use structure-based mutagenesis to identify and tune the ruler element residing in the Nhp10 and Arp8 modules of the INO80 remodeler complex. Generally, we propose that a remodeler ruler regulates nucleosome sliding direction bias in response to (epi)genetic information. This finally conceptualizes how remodeler-mediated nucleosome dynamics determine stable steady-state nucleosome positioning relative to other nucleosomes, DNA bound factors, DNA ends and DNA sequence elements.</td><td>2020-02-29</td><td>Elisa Oberbeckmann; Vanessa Niebauer; Shinya Watanabe; Lucas Farnung; Manuela Moldt; Andrea Schmid; Patrick Cramer; Craig L. Peterson; Sebastian Eustermann; Karl-Peter Hopfner; Philipp Korber</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.969618</td></tr><tr><td>48mtdwuv</td><td>3d498994d143a4ed7cc381700a19f2790d46bb36</td><td>biorxiv</td><td>Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models</td><td>10.1101/2020.02.28.969782</td><td>null</td><td>null</td><td>biorxiv</td><td>SUMMARYIn the analysis of genomic sequence data, so-called “alignment free” approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification1,2,3,4. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers5. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes6,7,8. This can be seen as an analogy of the “bag-of-words” model successfully employed in natural language processing and computer vision for document and image classification9,10. Feature extraction techniques from natural language processing have previously been analogized to genomics data11; however, the “bag-of-words” approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of “bag-of-words” methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, Vorpal, allows the construction of interpretable linear models with clustered, representative ‘degenerate’ K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae, hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus, and human-host vs. non-human host in Influenza A. The capacity of this approach for in silico identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at https://github.com/mriglobal/vorpal.</td><td>2020-03-02</td><td>Phillip Davis; John Bagnoli; David Yarmosh; Alan Shteyman; Lance Presser; Sharon Altmann; Shelton Bradrick; Joseph A. Russell</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.969782</td></tr><tr><td>2uvibr2j</td><td>null</td><td>biorxiv</td><td>Designing of a next generation multiepitope based vaccine (MEV) against SARS-COV-2: Immunoinformatics and in silico approaches</td><td>10.1101/2020.02.28.970343</td><td>null</td><td>null</td><td>biorxiv</td><td>Coronavirus disease 2019 (COVID-19) associated pneumonia caused by severe acute respiratory coronavirus 2 (SARS-COV-2) was first reported in Wuhan, China in December 2019. Till date, no vaccine or completely effective drug is available to cure COVID-19. Therefore, an effective vaccine against SARS-COV-2 is crucially needed. This study was conducted to design an effective multiepitope based vaccine (MEV) against SARS-COV-2. Seven antigenic proteins were taken as targets and different epitopes (B-cell, T-cell and IFN-γ inducing) were predicted. Highly antigenic and overlapping epitopes were shortlisted. Selected epitopes indicated significant interactions with the HLA-binding alleles and 99.29% coverage of the world population. Finally, 505 amino acids long MEV was designed by connecting sixteen MHC class I and eleven MHC class II epitopes with suitable linkers and adjuvant. Linkers and adjuvant were added to enhance the immunogenicity response of the MEV. The antigenicity, allergenicity, physiochemical properties and structural details of MEV were analyzed in order to ensure safety and immunogenicity. MEV construct was non-allergenic, antigenic, stable and flexible. Molecular docking followed by molecular dynamics (MD) simulation analysis, demonstrated a stable and strong binding affinity of MEV with human pathogenic toll-like receptors (TLR), TLR3 and TLR8. Codon optimization and in silico cloning of MEV ensured increased expression in the Escherichia coli K-12 system. Designed MEV in present study could be a potential candidate for further vaccine production process against COVID-19. However, to ensure its safety and immunogenic profile, the proposed MEV needs to be experimentally validated.</td><td>2020-03-02</td><td>Muhammad Tahir ul Qamar; Abdur Rehman; Usman Ali Ashfaq; Muhammad Qasim Awan; Israr Fatima; Farah Shahid; Ling-Ling Chen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.28.970343</td></tr><tr><td>0ga5rel6</td><td>0b460e73926eb107001c95c0dfeae2362b4251b0</td><td>medrxiv</td><td>Precautions are Needed for COVID-19 Patients with Coinfection of Common Respiratory Pathogens</td><td>10.1101/2020.02.29.20027698</td><td>null</td><td>null</td><td>medrvix</td><td>Background: With the ongoing outbreak of Coronavirus Disease 2019 (COVID-19), infected patients within and beyond the epidemic area, Wuhan, China, showed different epidemiological and clinical characteristics. There is a paucity of data concerning coinfection with other common respiratory pathogens in COVID-19 patients outside of Wuhan. Methods: We conducted a double-centre study recruiting 68 patients with severe acute respiratory coronavirus 2 (SARS-CoV-2) infection confirmed by nucleic acid testing in Qingdao and Wuhan from January 17 to February 16, 2020. Indirect immunofluorescence was performed to detect the specific IgM antibody against common respiratory pathogens in collected acute phase serum. Results: Of the 68 patients with SARS-CoV-2 infection, 30 (44.12%) were from Qingdao. The median age of Qingdao and Wuhan patients were 50 (IQR: 37-59) and 31 (IQR: 28-38) years, respectively, and the majority of patients were female in Qingdao (60.00%) and Wuhan (55.26%). Among COVID-19 patients in Qingdao, 24 (80.00%) of them had IgM antibodies against at least one respiratory pathogen, whereas only one (2.63%) of the patients in Wuhan had positive results for serum IgM antibody detection (P<0.0001). The most common respiratory pathogens detected in Qingdao COVID-19 patients were influenza virus A (60.00%) and influenza virus B (53.33%), followed by mycoplasma pneumoniae (23.33%) and legionella pneumophila (20.00%). While the pattern for coinfection in patients with community-acquired pneumonia in Qingdao was quite different, with a positive rate of only 20.90%. Interpretation: We reported a large proportion of COVID-19 patients with coinfection of seasonal respiratory pathogens in Qingdao, northeast China, which differed greatly from the patients in Wuhan, central China. Precautions are needed when dealing with COVID-19 patients beyond the epidemic centre who have coinfection with other respiratory pathogens. We highly recommend adding SARS-CoV-2 to routine diagnostic testing in capable hospitals to prevent misdetection of the virus.</td><td>2020-03-03</td><td>Quansheng Xing; Guoju Li; Yuhan Xing; Ting Chen; Wenjie Li; Wei Ni; Kai Deng; Ruqin Gao; Changzheng Chen; Yang Gao; Qiang Li; Guiling Yu; Jianning Tong; Wei Li; Guiliang Hao; Yue Sun; Ai Zhang; Qin Wu; Zipu Li; Silin Pan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20027698</td></tr><tr><td>s43mous6</td><td>14783283da3482bf2aab4e6100fe4ea5c29d8db7</td><td>medrxiv</td><td>Vicarious traumatization in the general public, members, and non-members of medical teams aiding in COVID-19 control</td><td>10.1101/2020.02.29.20029322</td><td>null</td><td>null</td><td>medrvix</td><td>Since December 2019, more than 79,000 people have been diagnosed with infection of the Corona Virus Disease 2019 (COVID-19). A large number of medical staff were dispersed for Wuhan city and Hubei province to aid COVID-19 control. Psychological stress, especially vicarious traumatization (VT) caused by the COVID-19 pandemic, should not be ignored. To address this concern, the study employed a total of 214 general public (GP) and 526 nurses to evaluate VT scores via a mobile app-based questionnaire. Results showed that the VT scores slightly increased across periods of aiding COVID-19 control, although no statistical difference was noted (P = 0.083). However, the study found lower scores for VT in nurses [median = 69; interquartile range (IQR) = 56-85] than those of the GP (median = 75.5; IQR = 62-88.3) (P = 0.017). In addition, the VT scores for front-line nurses (FLNs; median = 64; IQR = 52-75), including scores for physiological and psychological responses, were significantly lower than those of non-front-line nurses (nFLNs; median = 75.5; IQR = 63-92) (P < 0.001). Interestingly, the VT scores of the GP were significantly higher than those of the FLNs (P < 0.001). However, no statistical difference was observed compared with those of nFLNs (P > 0.05). Importantly, nFLNs are more likely to suffer from VT, which might be related to two factors, namely, gender [odds ratio (OR) = 3.1717; 95% confidence interval (CI) = 4.247-18.808; P = 0.002] and fertility [OR = 2.072; 95%CI = 0.626-24.533; P = 0.039]. Therefore, increased attention should be paid to the psychological problems of the medical staff, especially nFLNs, and GP under the situation of the spread and control of COVID-19. Early strategies that aim to prevent and treat VT in medical staff and GP are extremely necessary.</td><td>2020-03-03</td><td>Zhenyu Li; Jingwu Ge; Meiling Yang; Jianping Feng; Mei Qiao; Riyue Jiang; Jiangjiang Bi; Gaofeng Zhan; Xiaolin Xu; Long Wang; Qin Zhou; Chenliang Zhou; Yinbing Pan; Shijiang Liu; Haiwei Zhang; Jianjun Yang; Bin Zhu; Yimin Hu; Kenji Hashimoto; Yan Jia; Haofei Wang; Rong Wang; Cunming Liu; Chun Yang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029322</td></tr><tr><td>9lissxsy</td><td>5241581d168b90832f88103bd40c8db2d843a6ea</td><td>medrxiv</td><td>Association of Cardiovascular Manifestations with In-hospital Outcomes in Patients with COVID-19: A Hospital Staff Data</td><td>10.1101/2020.02.29.20029348</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The outbreaks of coronavirus disease 2019 (COVID-19) caused by the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) remain a huge threat to the public health worldwide. Clinical data is limited up to now regarding the risk factors in favor of severe conversion of non-severe case with COVID-19. Aims: This study analyzed a hospital staff data to figure out general clinical features of COVID-19 in terms of the association of cardiovascular manifestations (CVMs) with in-hospital outcomes of COVID-19 cases. Methods: Retrospective, single-center case series of 41 consecutive hospitalized health staff with confirmed COVID-19 were collected at the Central Hospital of Wuhan in Wuhan, China, from January 15 to January 24, 2020. Epidemiological, demographic, clinical, laboratory, radiological, treatment data and in-hospital adverse events were collected and analyzed. Final date of follow-up was March 3, 2020. A comparative study was applied between cases with CVMs and those without CVMs. Results: Of all, clinicians and clinical nurses accounted for 80.5%, while 87.8% of all had history of patient contact. The population was presented with a mean age of 39.1 +- 9.2 and less comorbidities than community population. The three most frequent symptoms of COVID-19 cases analyzed were fever (82.9%), myalgia or fatigue (80.5%) and cough (63.4%). While, the three most frequent initial symptoms were myalgia or fatigue (80.5%), fever (73.2%) and cough (41.5%). There were 95.1% cases featured as non-severe course of disease according to the official standard in China. Patients with CVMs and those without CVMs accounted for 58.5% and 41.5%, respectively. Compared with cases without CVMs, patients with CVMs were presented with lower baseline lymphocyte count (0.99 +- 0.43 and 1.55 +- 0.61, P<0.001), more who had at least once positive nucleic acid detection of throat swab during admission (50.0% and 11.8%, P=0.011), and more received oxygen support (79.2% and 23.5%, P<0.001). The rate of in-hospital adverse events was significantly higher in patients with CVMs group (75.0% and 23.5%, P=0.001). Multivariable logistic regression model indicated that, coexisting with CVMs in COVID-19 patients was not independently associated with in-hospital adverse events. Conclusions: Most of hospital staff with COVID-19 had history of patient contact, featured non-severe course of disease. Cases with CVMs suffered from more in-hospital adverse events than those without CVMs. But concomitant CVMs were not independently associated with in-hospital adverse events in COVID-19 patients.</td><td>2020-03-03</td><td>Ru Liu; Xiaoyan Ming; Ou Xu; Jianli Zhou; Hui Peng; Ning Xiang; Jiaming Zhang; Hong Zhu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029348</td></tr><tr><td>qs7dsxn5</td><td>f87eec8279bc79900914c5689bb42f49f41aace5</td><td>medrxiv</td><td>The spatiotemporal estimation of the dynamic risk and the international transmission of 2019 Novel Coronavirus (COVID-19) outbreak: A global perspective</td><td>10.1101/2020.02.29.20029413</td><td>null</td><td>null</td><td>medrvix</td><td>An ongoing novel coronavirus SARS-CoV-2 pneumonia infection outbreak called COVID-19 started in Wuhan, Hubei Province, China, in December 2019. It both spread rapidly to all provinces in China and started spreading around the world quickly through international human movement from January 2020. Currently, the spatiotemporal epidemic transmission patterns, prediction models, and possible risk analysis for the future are insufficient for COVID-19 but we urgently need relevant information, particularly from the global perspective. We have developed a novel two-stage simulation model to simulate the spatiotemporal changes in the number of COVID-19 cases and estimate the future worldwide risk. Based on the connectivity of countries to China and the country's medical and epidemic prevention capabilities, different scenarios are generated to analyze the possible transmission throughout the world and use this information to evaluate each country's vulnerability to and the dynamic risk of COVID-19. Countries' vulnerability to the COVID-19 outbreak from China is calculated for 63 countries around the world. Taiwan, South Korea, Hong Kong, and Japan are the most vulnerable areas. The relationship between each country's vulnerability and days before the first imported case occurred shows a very high exponential decrease. The cumulative number of cases in each country also has a linear relationship with vulnerability, which can compare and quantify the initial epidemic prevention capabilities to various countries' management strategies. In total, 1,000 simulation results of future cases around the world are generated for the spatiotemporal risk assessment. According to the simulation results of this study, if there is no specific medicine for it, it will likely form a global pandemic. This method can be used as a preliminary risk assessment of the spatiotemporal spread for a new global epidemic. * Note: This study was completed on February 15, 2020.</td><td>2020-03-03</td><td>Yuan-Chien Lin; Wan-Ju Chi; Yu-Ting Lin; Chun-Yeh Lai</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029413</td></tr><tr><td>m9icky9z</td><td>c12ad8b411c24fe215be91d1f990645743e55ab5</td><td>medrxiv</td><td>An epidemiological forecast model and software assessing interventions on COVID-19 epidemic in China</td><td>10.1101/2020.02.29.20029421</td><td>null</td><td>null</td><td>medrvix</td><td>We develop a health informatics toolbox that enables public health workers to timely analyze and evaluate the time-course dynamics of the novel coronavirus (COVID-19) infection using the public available data from the China CDC. This toolbox is built upon a hierarchical epidemiological model in which two observed time series of daily proportions of infected and removed cases are emitted from the underlying infection dynamics governed by a Markov SIR infectious disease process. We extend the SIR model to incorporate various types of time-varying quarantine protocols, including government-level macro isolation policies and community-level micro inspection measures. We develop a calibration procedure for under-reported infected cases. This toolbox provides forecast, in both online and offline forms, of turning points of interest, including the time when daily infected proportion becomes smaller than the previous ones and the time when daily infected proportions becomes smaller than that of daily removed proportion, as well as the ending time of the epidemic. An R software is made available for the public, and examples on the use of this software are illustrated. Some possible extensions of our novel epidemiological models are discussed.</td><td>2020-03-03</td><td>Peter X Song; Lili Wang; Yiwang Zhou; Jie He; Bin Zhu; Fei Wang; Lu Tang; Marisa Eisenberg</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029421</td></tr><tr><td>b8eezxv3</td><td>null</td><td>medrxiv</td><td>ddPCR: a more sensitive and accurate tool for SARS-CoV-2 detection in low viral load specimens</td><td>10.1101/2020.02.29.20029439</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Real-Time PCR (RT-PCR) is widely used as the gold standard for clinical detection of SARS-CoV-2. However, due to the low viral load in patient throat and the limitation of RT-PCR, significant numbers of false negative reports are inevitable, which should not be ignored. Methods: We explored the feasibility of droplet digital PCR (ddPCR) to detect SARS-CoV-2 from 57 clinical pharyngeal swab samples and compared with RT-PCR in terms of the sensitivity and accuracy. Among 57 samples, all of which were reported as negative nucleic acid by officially approved clinical RT-PCR detection, 43 samples were collected from suspected patients with fever in clinic, and 14 were from supposed convalescents who were about to discharge after treatment. The experiment was double-blind. Results: The lower limit of detection of the optimized ddPCR is at least 500 times lower than that of RT-PCR. The overall accuracy of ddPCR for clinical detection is 94.3 %. 33 out of 35 negative pharyngeal swab samples checked by RT-PCR were correctly judged by ddPCR based on the follow-up investigation. In addition, 9 out of 14 (64.2 %) supposed convalescents with negative nucleic acid test twice by RT-PCR were positive by ddPCR detection. Conclusions: ddPCR shows superiority for clinical detection of SARS-CoV-2 to reduce the false negatives, which could be a powerful complement to the current standard RT-PCR. Before the ddPCR to be approved for diagnosis, the current clinical practice that the convalescent continues to be quarantined for 2 weeks is reasonable and necessary.</td><td>2020-03-06</td><td>Tao Suo; Xinjin Liu; Ming Guo; Jiangpeng Feng; Wenjia Hu; Yang Yang; Qiuhan Zhang; Xin Wang; Muhanmmad Sajid; Dong Guo; Zhixiang Huang; Liping Deng; Tielong Chen; Fang Liu; Ke Xu; Yuan Liu; Qi Zhang; Yingle Liu; Yong Xiong; Guozhong Guo; Yu Chen; Ke Lan</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029439</td></tr><tr><td>ndn6iwre</td><td>1ad1d4b84aea4ceaf05c62a1ad04e7150f7f4684</td><td>medrxiv</td><td>68 Consecutive patients assessed for COVID-19 infection; experience from a UK regional infectious disease unit</td><td>10.1101/2020.02.29.20029462</td><td>null</td><td>null</td><td>medrvix</td><td>Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29th Jan 2020 and 24th Feb 2020. Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment. Final microbiological diagnoses included SARS-CoV-2 (COVID-19), Mycoplasma pneumonia, influenza A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir. The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.</td><td>2020-03-06</td><td>Nicholas Easom; Peter Moss; Gavin Barlow; Anda Samson; Tom Taynton; Kate Adams; Monica Ivan; Phillipa Burns; Kavitha Gajee; Kirstine Eastick; Patrick Lillie</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029462</td></tr><tr><td>g41pd9uz</td><td>7852aafdfb9e59e6af78a47af796325434f8922a</td><td>medrxiv</td><td>Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</td><td>10.1101/2020.02.29.20029520</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Although the SARS-CoV-2 viral load detection of respiratory specimen has been widely used for novel coronavirus disease (COVID-19) diagnosis, it is undeniable that serum SARS-CoV-2 nucleic acid (RNAaemia) could be detected in a fraction of the COVID-19 patients. However, it is not clear that if the incidence of RNAaemia could be correlated with the occurrence of cytokine storm or with the specific class of patients. Methods: This study enrolled 48 patients with COVID-19 admitted to the General Hospital of Central Theater Command, PLA, a designated hospital in Wuhan, China. The patients were divided into three groups according to the Diagnosis and Treatment of New Coronavirus Pneumonia (version 6) published by the National Health Commission of China. The clinical and laboratory data were collected. The serum viral load detection and serum IL-6 levels were determined. Except for routine statistical analysis, Generalized Linear Models (GLMs) analysis was used to establish a patient status prediction model based on real-time RT-PCR Ct value. Findings: The Result showed that cases with RNAaemia were exclusively confirmed in critically ill patients group and appeared to reflect the illness severity. Further more, the inflammatory cytokine IL-6 levels were significantly elevated in critically ill patients, which is almost 10-folds higher than those in other patients. More importantly, the extremely high IL-6 level was closely correlated with the incidence of RNAaemia (R=0.902) and the vital signs of COVID-19 patients (R= -0.682). Interpretation: Serum SARS-CoV-2 viral load (RNAaemia) is strongly associated with cytokine storm and can be used to predict the poor prognosis of COVID-19 patients. Moreover, our results strongly suggest that cytokine IL-6 should be considered as a therapeutic target in critically ill patients with excessive inflammatory response.</td><td>2020-03-03</td><td>Xiaohua Chen; Binghong Zhao; Yueming Qu; Yurou Chen; Jie Xiong; Yong Feng; Dong Men; Qianchuan Huang; Ying Liu; Bo Yang; Jinya Ding; Feng Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029520</td></tr><tr><td>3kp3ptbn</td><td>0bd443591b5e10934beef050b516afeca6b668fe</td><td>medrxiv</td><td>A simple model to assess Wuhan lock-down effect and region efforts during COVID-19 epidemic in China Mainland</td><td>10.1101/2020.02.29.20029561</td><td>null</td><td>null</td><td>medrvix</td><td>Since COVID-19 emerged in early December, 2019 in Wuhan and swept across China Mainland, a series of large-scale public health interventions, especially Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement, have been taken by the government to control the epidemic. Based on Baidu Migration data and the confirmed cases data, we identified two key factors affecting the later (e.g February 27, 2020) cumulative confirmed cases in non-Wuhan region (y). One is the sum travelers from Wuhan during January 20 to January 26 (x1), which had higher infected probability but lower transmission ability because the human-to-human transmission risk of COVID-19 was confirmed and announced on January 20. The other is the seed cases from Wuhan before January 19, which had higher transmission ability and could be represented with the confirmed cases before January 29 (x2) due to a mean 10-day delay between infection and detection. A simple yet effective regression model then was established as follow: y= 70.0916+0.0054*x1+2.3455*x2 (n = 44, R2 = 0.9330, P<10-7). Even the lock-down date only delay or in advance 3 days, the estimated confirmed cases by February 27 in non-Wuhan region will increase 35.21% or reduce 30.74% - 48.59%. Although the above interventions greatly reduced the human mobility, Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement do have a determining effect on the ongoing spread of COVID-19 across China Mainland. The strategy adopted by China has changed the fast-rising curve of newly diagnosed cases, the international community should learn from lessons of Wuhan and experience from China. Efforts of 29 Provinces and 44 prefecture-level cities against COVID-19 were also assessed preliminarily according to the interpretive model. Big data has played and will continue playing an important role in public health.</td><td>2020-03-03</td><td>Yuan zheming; Chen Yuan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029561</td></tr><tr><td>2s4ifz7i</td><td>b8bb4db131a25b1bbb30d4205b16dc9ef988d22f</td><td>medrxiv</td><td>Machine learning-based CT radiomics model for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection: A multicenter study</td><td>10.1101/2020.02.29.20029603</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract Objectives To develop and test machine learning-based CT radiomics models for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection. Design Cross-sectional Setting Multicenter Participants A total of 52 patients with laboratory-confirmed SARS-CoV-2 infection and their initial CT images were enrolled from 5 designated hospitals in Ankang, Lishui, Zhenjiang, Lanzhou, and Linxia between January 23, 2020 and February 8, 2020. As of February 20, patients remained in hospital or with non-findings in CT were excluded. Therefore, 31 patients with 72 lesion segments were included in the final analysis. Intervention CT radiomics models based on logistic regression (LR) and random forest (RF) were developed on features extracted from pneumonia lesions in training and inter-validation datasets. The predictive performance was further evaluated in test dataset on lung lobe- and patients-level. Main outcomes Short-term hospital stay (≤10 days) and long-term hospital stay (>10 days). Results The CT radiomics models based on 6 second-order features were effective in discriminating short- and long-term hospital stay in patients with pneumonia associated with SARS-CoV-2 infection, with areas under the curves of 0.97 (95%CI 0.83-1.0) and 0.92 (95%CI 0.67-1.0) by LR and RF, respectively, in the test dataset. The LR model showed a sensitivity and specificity of 1.0 and 0.89, and the RF model showed similar performance with sensitivity and specificity of 0.75 and 1.0 in test dataset. Conclusions The machine learning-based CT radiomics models showed feasibility and accuracy for predicting hospital stay in patients with pneumonia associated with SARS-CoV-2 infection.</td><td>2020-03-03</td><td>Xiaolong Qi; Zicheng Jiang; QIAN YU; Chuxiao Shao; Hongguang Zhang; Hongmei Yue; Baoyi Ma; Yuancheng Wang; Chuan Liu; Xiangpan Meng; Shan Huang; Jitao Wang; Dan Xu; Junqiang Lei; Guanghang Xie; Huihong Huang; Jie Yang; Jiansong Ji; Hongqiu Pan; Shengqiang Zou; Shenghong Ju</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.20029603</td></tr><tr><td>1yka7ej9</td><td>38e796bbec2f90b1b802c14dc922102e96f6361e</td><td>biorxiv</td><td>The within-host viral kinetics of SARS-CoV-2</td><td>10.1101/2020.02.29.965418</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractIn this work, we use a within-host viral dynamic model to describe the SARS-CoV-2 kinetics in host. Chest radiograph score data are used to estimate the parameters of that model. Our result shows that the basic reproductive number of SARS-CoV-2 in host growth is around 3.79. Using the same method we also estimate the basic reproductive number of MERS virus is 8.16 which is higher than SARS-CoV-2. The PRCC method is used to analyze the sensitivities of model parameters and the drug effects on virus growth are also implemented to analyze the model.</td><td>2020-03-02</td><td>Chentong Li; Jinhu Xu; Jiawei Liu; Yicang Zhou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.965418</td></tr><tr><td>0d77ojnb</td><td>07e833d0917cace550853f72923856d0fe1a7120</td><td>biorxiv</td><td>The 2019 coronavirus (SARS-CoV-2) surface protein (Spike) S1 Receptor Binding Domain undergoes conformational change upon heparin binding</td><td>10.1101/2020.02.29.971093</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractMany pathogens take advantage of the dependence of the host on the interaction of hundreds of extracellular proteins with the glycosaminoglycans heparan sulfate to regulate homeostasis and use heparan sulfate as a means to adhere and gain access to cells. Moreover, mucosal epithelia such as that of the respiratory tract are protected by a layer of mucin polysaccharides, which are usually sulfated. Consequently, the polydisperse, natural products of heparan sulfate and the allied polysaccharide, heparin have been found to be involved and prevent infection by a range of viruses including S-associated coronavirus strain HSR1. Here we use surface plasmon resonance and circular dichroism to measure the interaction between the SARS-CoV-2 Spike S1 protein receptor binding domain (SARS-CoV-2 S1 RBD) and heparin. The data demonstrate an interaction between the recombinant surface receptor binding domain and the polysaccharide. This has implications for the rapid development of a first-line therapeutic by repurposing heparin and for next-generation, tailor-made, GAG-based antivirals.</td><td>2020-03-02</td><td>Courtney Mycroft-West; Dunhao Su; Stefano Elli; Scott Guimond; Gavin Miller; Jeremy Turnbull; Edwin Yates; Marco Guerrini; David Fernig; Marcelo Lima; Mark Skidmore</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.971093</td></tr><tr><td>jmrg4oeb</td><td>2ea102f58147dab02e4dea90eb90dbc67149f678</td><td>biorxiv</td><td>Mutations, Recombination and Insertion in the Evolution of 2019-nCoV</td><td>10.1101/2020.02.29.971101</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractBackgroundThe 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV. However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear.MethodsWe collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses.ResultsSystematic analysis of 120 genomic sequences of 2019-nCoV revealed co-circulation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site.ConclusionsOur studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion.One Sentence SummaryNovel 2019-nCoV sequences revealed the evolution and specificity of betacoronavirus with possible mechanisms of enhanced infectivity.</td><td>2020-03-02</td><td>Aiping Wu; Peihua Niu; Lulan Wang; Hangyu Zhou; Xiang Zhao; Wenling Wang; Jingfeng Wang; Chengyang Ji; Xiao Ding; Xianyue Wang; Roujian Lu; Sarah Gold; Saba Aliyari; Shilei Zhang; Ellee Vikram; Angela Zou; Emily Lenh; Janet Chen; Fei Ye; Na Han; Yousong Peng; Haitao Guo; Guizhen Wu; Taijiao Jiang; Wenjie Tan; Genhong Cheng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.971101</td></tr><tr><td>gzubfnqg</td><td>null</td><td>biorxiv</td><td>An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12</td><td>10.1101/2020.02.29.971127</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has received global attention due to the recent outbreak in China. In this work, we report a CRISPR-Cas12 based diagnostic tool to detect synthetic SARS-CoV-2 RNA sequences in a proof-of-principle evaluation. The test proved to be sensitive, rapid, and potentially portable. These key traits of the CRISPR method are critical for virus detection in regions that lack resources to use the currently available methods.</td><td>2020-03-02</td><td>Curti Lucia; Pereyra-Bonnet Federico; Gimenez Carla Alejandra</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.02.29.971127</td></tr><tr><td>tne83uu0</td><td>7e9cd4bbf0fba1cc0bcded40041fd01b9dfb683f</td><td>medrxiv</td><td>Epidemiologic Characteristics of COVID-19 in Guizhou, China</td><td>10.1101/2020.03.01.20028944</td><td>null</td><td>null</td><td>medrvix</td><td>At the end of 2019, a coronavirus disease 2019 (COVID-19) outbroke in Wuhan, China, and spread to Guizhou province on January of 2020. To acquire the epidemiologic characteristics of COVID-19 in Guizhou, China, we collected data on 162 laboratory-confirmed cases related to COVID-19. We described the demographic characteristics of the cases and estimated the incubation period, serial interval and basic reproduction number. We also presented two representative case studies in Guizhou province -- Case Study 1 was an example of asymptomatic carrier; and Case Study 2 was an example of a large and complex infection chain that involved four different districts spanning three provinces and eight families. With an estimation of 8 days incubation period and 6 days serial interval, our results indicate that there may exist infectiousness during the incubation period for 2019-nCoV. This increases the difficulty of screening or identifying cases related to COVID-19.</td><td>2020-03-06</td><td>Kaike Ping</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20028944</td></tr><tr><td>hhsfq8bz</td><td>f19e2df932f90f387fe71533f3802875d9042160</td><td>medrxiv</td><td>Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</td><td>10.1101/2020.03.01.20029074</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Coronavirus disease-2019 (COVID-19) is a rapidly escalating epidemic caused by SARS-CoV-2. Identification of a simple and effective indicator to assess disease severity and outcome is urgently needed. Methods: This study retrospectively analyzed dynamic changes of lymphocyte percentage (LYM%) in 15 death cases, 15 severe cases as well as 40 moderate cases of COVID-19 patients. Next, prognostic role of lymphopenia in COVID-19 were verified in 92 hospitalized cases. Results: Our results from death and severe cases showed that LYM% in blood test were inversely associated with the progression and severity of COVID-19. LYM% in patients with moderate COVID-19 remained higher than 20% 10-12 days after symptom onset. In contrast, LYM% was lower than 20% in severe cases. However, LYM% in severe cases was higher than 5% 17-19 days after the onset of the disease, while it fell below 5% in death cases. Therefore, we established a reliable Time from symptom onset-LYM% model (TLM), which could be used to evaluate disease severity and predict the outcomes of hospitalized patients with COVID-19. Conclusion: Lymphopenia can be used to indicate clinical course, treatment effect and outcomes of COVID-19 patients.</td><td>2020-03-03</td><td>Li Tan; Qi Wang; Duanyang Zhang; Jinya Ding; Qianchuan Huang; Yi-Quan Tang; Qiongshu Wang; Hongming Miao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029074</td></tr><tr><td>nsrm0axa</td><td>510ecab01c66bc2a911abcf67451ec482ed18a05</td><td>medrxiv</td><td>Epidemiological and clinical features of 2019-nCoV acute respiratory disease cases in Chongqing municipality, China: a retrospective, descriptive, multiple-center study</td><td>10.1101/2020.03.01.20029397</td><td>null</td><td>null</td><td>medrvix</td><td>Background：In January 19, 2020, first case of 2019 novel coronavirus (2019-nCoV) pneumonia (COVID-19) was confirmed in Chongqing municipality, China. Methods：In this retrospective, descriptive, multiple-center study, total of 267 patients with COVID-19 confirmed by real-time RT-PCR in Chongqing from Jan 19 to Feb 16, 2020 were recruited. Epidemiological, demographic, clinical, radiological characteristics, laboratory examinations, and treatment regimens were collected on admission. Clinical outcomes were followed up until Feb 16, 2020. Results：267 laboratory-confirmed COVID-19 patients admitted to 3 designated-hospitals in Chongqing provincial municipality from January 19 to February 16, 2020 were enrolled and categorized on admission. 217 (81.27%) and 50 (18.73%) patients were categorized into non-severe and severe subgroups, respectively. The median age of patients was 48.0 years (IQR, 35.0-65.0), with 129 (48.3%) of the patients were more than 50 years of age. 149 (55.8%) patients were men. Severe patients were significantly older (median age, 71.5 years [IQR, 65.8-77.0] vs 43.0 years [IQR, 32.5-57.0]) and more likely to be male (110 [50.7%] vs 39 [78.0%]) and have coexisting disorders (15 [30.0%] vs 26 [12.0%]). 41 (15.4%) patients had a recent travel to Hubei province, and 139 (52.1%) patients had a history of contact with patients from Hubei. On admission, the most common symptoms of COVID-19 were fever 225(84.3%), fatigue (208 [77.9%]), dry cough (189 [70.8%]), myalgia or arthralgia (136 [50.9%]). Severe patients were more likely to present dyspnea (17 [34.0%] vs 26 [12.0%]) and confusion (10 [20.0%] vs 15 [6.9%]). Rales (32 [12.0%]) and wheezes (20 [7.5%]) are not common noted for COVID-19 patients, especially for the non-severe (11 [5.1%], 10 [4.6%]). 118 (44.2%). Most severe patients demonstrated more laboratory abnormalities. 231 (86.5%), 61 (22.8%) patients had lymphopenia, leukopenia and thrombocytopenia, respectively. CD4+T cell counts decrease was observed in 77.1 % of cases, especially in the severe patients (45, 100%). 53.1% patients had decreased CD+3 T cell counts, count of CD8+T cells was lower than the normal range in part of patients (34.4%). More severe patients had lower level of CD4+ T cells and CD+3 T cells (45 [100.0%] vs 29[56.9%], 31 [68.9%] vs 20 [39.2%]). Most patients had normal level of IL-2, IL-4, TNF-α and INF-γ, while high level of IL-6 and IL-17A was common in COVID-19 patients (47 [70.1%], 35 [52.2%]). Level of IL-6, IL-17A and TNF-α was remarkably elevated in severe patients (32 [84.2%] vs 15 [51.7%], 25 [65.8%] vs 10 [34.5%], 17 [44.7%] vs 5 [17.2%]). All patients received antiviral therapy (267, 100%). A portion of severe patients (38, 76.0%) received systemic corticosteroid therapy. Invasive mechanical ventilation in prone position, non-invasive mechanical ventilation, high-flow nasal cannula oxygen therapy was adopted only in severe patients with respiratory failure (5[10.0%], 35[70.0%], 12[24.0%]). Traditional Chinese medicine was adopted to most of severe patients (43,86.0%). Conclusion:Our study firstly demonstrated the regional disparity of COVID-19 in Chongqing municipality and further thoroughly compared the differences between severe and non-severe patients. The 28-day mortality of COVID-19 patients from 3 designed hospitals of Chongqing is 1.5%, lower than that of Hubei province and mainland China including Hubei province. However, the 28-mortality of severe patients was relatively high, with much higher when complications occurred. Notably, the 28-mortality of critically severe patients complicated with severe ARDS is considerably as high as 44.4%. Therefore, early diagnosis and intensive care of critically severe COVID-19 cases, especially those combined with ARDS, will be considerably essential to reduce mortality.</td><td>2020-03-03</td><td>Di Qi; Xiaofeng Yan; Xumao Tang; Junnan Peng; Qian Yu; Longhua Feng; Guodan Yuan; An Zhang; Yaokai Chen; Jing Yuan; Xia Huang; Xianxiang Zhang; Peng Hu; Yuyan Song; Chunfang Qian; Qiangzhong Sun; Daoxin Wang; Jin Tong; Jianglin Xiang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029397</td></tr><tr><td>iwsa760n</td><td>null</td><td>medrxiv</td><td>Systematic Review of the Registered Clinical Trials of Coronavirus Disease 2019 (COVID-19)</td><td>10.1101/2020.03.01.20029611</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since the outbreak of coronavirus disease 2019 (COVID-19), many researchers in China have immediately carried out clinical research scheme of the COVID-19. But, there is still a lack of systematic review of registered clinical trials. Therefore, we conducted a systematic review of the clinical trials of COVID-19 to summarize the characteristics of the COVID-19 registered clinical trials. Methods: This study is based on the recommendations of the PRISMA in the Cochrane handbook. The databases from the Chinese Clinical Registration Center and the ClinicalTrials.gov were searched to collect the registered clinical trials of COVID-19. The retrieval inception date is February 9, 2020. Two researchers independently selected the literature based on inclusion and exclusion criteria, extracted data and evaluated the risk of bias. Results: A total of 75 registered clinical trials (63 interventional studies and 12 observational studies) of COVID-19 were obtained. A majority of clinical trials were sponsored by Chinese hospitals. Only 11 trials have begun to recruit patients, and none of the registered clinical trials had been completed; 34 trials were early clinical exploratory trials or in a pre-experiment stage, 15 trials belonged to phrase Ⅲ and 4 trials were phrase Ⅳ. The methods of intervention included traditional Chinese medicine involving 26 trials, Western medicine involving 30 trials, and integrated traditional Chinese medicine and Western medicine involving 19 trials. The subjects were mainly non-critical adult patients (≥ 18 years old). The median sample size of the trials was 100 (IQR: 60 - 200), and the median execute time of the trials was 179 d (IQR: 94 - 366 d). The main outcomes were clinical observation and examinations. Overall, both the methodology quality of interventional trials and observational studies were low. Conclusions: Disorderly and intensive clinical trials of COVID-19 using traditional Chinese medicine and western medicine are ongoing or will being carried out in China. However, based on the low methodology quality and small sample size and long studies execute time, we will not be able to obtain reliable, high-quality clinical evidence about COVID-19 treatment in the near future. Improving the quality of study design, prioritizing promising drugs, and using different designs and statistical methods are worth advocating and recommending for the clinical trials of COVID-19 in China.</td><td>2020-03-03</td><td>Rui-fang Zhu; Ru-lu Gao; Sue-Ho Robert; Jin-ping Gao; Shi-gui Yang; Changtai Zhu</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029611</td></tr><tr><td>3e1ji2mw</td><td>4bc43084ccdb8d3bb704ca577473169cd1a64f9c</td><td>medrxiv</td><td>Risk estimation and prediction by modeling the transmission of the novel coronavirus (COVID-19) in mainland China excluding Hubei province</td><td>10.1101/2020.03.01.20029629</td><td>null</td><td>null</td><td>medrvix</td><td>Background: In December 2019, an outbreak of novel coronavirus disease (COVID-19) emerged in Wuhan, China and has swiftly spread to other parts of China and a number of oversea countries. Our aim is to evaluate the effectiveness of the evolution of interventions and self-protection measures, estimate the risk of partial lifting control measures and predict the epidemic trend of the virus in mainland China excluding Hubei province based on the published data and a novel mathematical model. Methods: A novel COVID-19 transmission dynamic model incorporating the intervention measures implemented in China is proposed. We parameterize the model by using the Markov Chain Monte Carlo (MCMC) method and estimate the control reproduction number Rc, as well as the effective daily reproduction ratio Re(t), of the disease transmission in mainland China excluding Hubei province. Results: The estimation outcomes indicate that the control reproduction number is 3.36 (95% CI 3.20-3.64) and Re(t) has dropped below 1 since January 31st, 2020, which implies that the containment strategies implemented by the Chinese government in mainland China excluding Hubei province are indeed effective and magnificently suppressed COVID-19 transmission. Moreover, our results show that relieving personal protection too early may lead to the spread of disease for a longer time and more people would be infected, and may even cause epidemic or outbreak again. By calculating the effective reproduction ratio, we proved that the contact rate should be kept at least less than 30% of the normal level by April, 2020. Conclusions: To ensure the epidemic ending rapidly, it is necessary to maintain the current integrated restrict interventions and self-protection measures, including travel restriction, quarantine of entry, contact tracing followed by quarantine and isolation and reduction of contact, like wearing masks, etc. People should be fully aware of the real-time epidemic situation and keep sufficient personal protection until April. If all the above conditions are met, the outbreak is expected to be ended by April in mainland China apart from Hubei province.</td><td>2020-03-06</td><td>Hui Wan; Jing-an Cui; Guo-Jing Yang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029629</td></tr><tr><td>4k1i6y98</td><td>5f72fd41d7ba8d4d7de7aa1f83fc5c90ffa50b03</td><td>medrxiv</td><td>How does the outbreak of 2019-nCoV spread in mainland China? A retrospective analysis of the dynamic transmission routes</td><td>10.1101/2020.03.01.20029645</td><td>null</td><td>null</td><td>medrvix</td><td>The fourth outbreak of the Coronaviruses, known as the 2019-nCoV, has occurred in Wuhan city of Hubei province in China in December 2019. We propose a time-varying sparse vector autoregressive (VAR) model to retrospectively analyze and visualize the dyamic transmission routes of this outbreak in mainland China over January 31 - February 19, 2020. Our results demonstrate that the influential inter-province routes from Hubei have become unidentifiable since February 4, whereas the self-transmission in each province was accelerating over February 4-15. From February 16, all routes became less detectable, and no influential transmissions could be identified on February 18 and 19. Such evidence supports the effectiveness of government interventions, including the travel restrictions in Hubei. Implications of our results suggest that in addition to the origin of the outbreak, virus preventions are of crucial importance in provinces with the largest migrant workers percentages (e.g., Jiangxi, Henan and Anhui) to controlling the spread of 2019-nCoV.</td><td>2020-03-06</td><td>Xiandeng Jiang; Le Chang; Yanlin Shi</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029645</td></tr><tr><td>kbd2h4l0</td><td>null</td><td>medrxiv</td><td>The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</td><td>10.1101/2020.03.01.20029769</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract. Background: The outbreak of coronavirus disease 2019 (COVID-19) in Wuhan City, China spreads rapidly since December, 2019. Most patients show mild symptoms, but some of them develop into severe disease. There is currently no specific medication. The purpose of this study is to to explore changes of markers in peripheral blood of severe COVID-19 patients, which may be of value in disease monitoring. Methods Clinical data of patients with nonsevere and severe type COVID-19 diagnosed by laboratory test in our institution were collected. The relationship between peripheral blood cells and cytokines, clinical manifestation and outcome was analyzed. Results A total of 69 severe type COVID-19 patients were included. On admission, the median age of severe cases was 56-year old, with 52.17% female patient. The most common symptoms were fever (79.72%), cough (63.77%), shortness of breath (57.97%) and fatigue (50.72%). Diarrhea is less common. The most common comorbidity is hypertension. Upon admission, the proportion of bilateral pulmonary involvement and interstitial abnormalities evidenced by chest computed tomography (CT) imaging in severe cases was 60.87% and 27.54%, respectively. Compared with patients with nonsevere disease, those with severe disease showed obvious lymphocytopenia. Elevated level of lactate dehydrogenase (LDH), C-reactive protein (CRP), ferritin and D-dimer was found in most cases. Two patients (2.9%) needed transfer to the intensive care unit. Baseline immunological parameters and most of the inflammatory parameters were basically within the normal range. However, baseline interleukin-6 (IL-6) was significantly increased in severe type, which was closely related to the maximal body temperature during hospitalization and to CT findings. Baseline IL-6 was also significantly related to the increase of baseline level of CRP, LDH, ferritin and D-dimer. The increase of baseline IL-6 level suggests that it may positively correlate with the severity of COVID-19. Among the 30 severe type patients whose IL-6 was assessed before and after treatment, significant decrease in IL-6 and improved CT assessment was found in 25 patients after treatment. Whereas the IL-6 level was further increased in 3 cases, which was closely related to disease progression. It is suggested that IL-6 may be used as a marker for disease monitoring in severe COVID-19 patients. Conclusions On admission, the baseline level of IL-6, CRP, LDH and ferritin was closely related to the severity of COVID-19, and the elevated IL-6 was significantly related to the clinical manifestation of severe type patients. The decrease of IL-6 was closely related to treatment effectiveness, while the increase of IL-6 indicated disease exacerbation. Collectively, the dynamic change of IL-6 level can be used as a marker for disease monitoring in patients with severe COVID-19.</td><td>2020-03-06</td><td>Tao Liu; Jieying Zhang; Yuhui Yang; Hong Ma; Zhengyu Li; Jiaoyue Zhang; Ji Cheng; Xiaoyun Zhang; Yanxia Zhao; Zihan Xia; Liling Zhang; Gang Wu; Jianhua Yi</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029769</td></tr><tr><td>a5x5ga60</td><td>c8437a45bfb84fb206fe03fd18d28858bae32651</td><td>medrxiv</td><td>Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</td><td>10.1101/2020.03.01.20029785</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of the novel coronavirus disease 2019 (COVID-19) infection began in December 2019 in Wuhan, and rapidly spread to many provinces in China. The number of cases has increased markedly in Anhui, but information on the clinical characteristics of patients is limited. We reported 75 patients with COVID-19 in the First Affiliated Hospital of USTC from Jan 21 to Feb 16, 2020, Hefei, Anhui Province, China. COVID-19 infection was confirmed by real-time RT-PCR of respiratory nasopharyngeal swab samples. Epidemiological, clinical and laboratory data were collected and analyzed. Of the 75 patients with COVID-19, 61 (81.33%) had a direct or indirect exposure history to Wuhan. Common symptoms at onset included fever (66 [88.0%] of 75 patients) and dry cough (62 [82.67%]). Of the patients without fever, cough could be the only or primary symptom. The most prominent laboratory abnormalities were lymphopenia, decreased percentage of lymphocytes (LYM%), decreased CD4+ and CD8+ T cell counts, elevated C-reactive protein (CRP) and lactate dehydrogenase (LDH). Patients with elevated interleukin 6 (IL-6) showed significant decreases in the LYM%, CD4+ and CD8+ T cell counts. Besides, the percentage of neutrophils, CRP, LDH and Procalcitonin levels increased significantly. We concluded that COVID-19 could cause different degrees of hematological abnormalities and damage of internal organs. Hematological profiles including LYM, LDH, CRP and IL-6 could be indicators of diseases severity and evaluation of treatment effectiveness. Antiviral treatment requires a comprehensive and supportive approach. Further targeted therapy should be determined based on individual clinical manifestations and laboratory indicators.</td><td>2020-03-06</td><td>Zonghao Zhao; Jiajia Xie; Ming Yin; Yun Yang; Hongliang He; Tengchuan Jin; Wenting Li; Xiaowu Zhu; Jing Xu; Changcheng Zhao; Lei Li; Yi Li; Hylemariam Mihiretie Mengist; Ayesha Zahid; Ziqin Yao; Chengchao Ding; Yingjie Qi; Yong Gao; Xiaoling Ma</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029785</td></tr><tr><td>38f8ftmh</td><td>2fe6b550f737baa47a5f2c8ab64cc3d9271c308a</td><td>medrxiv</td><td>Evaluation of the potential incidence of COVID-19 and effectiveness of contention measures in Spain: a data-driven approach</td><td>10.1101/2020.03.01.20029801</td><td>null</td><td>null</td><td>medrvix</td><td>Our society is currently experiencing an unprecedented challenge, managing and containing an outbreak of a new coronavirus disease known as COVID-19. While China - were the outbreak started - seems to have been able to contain the growth of the epidemic, different outbreaks are nowadays being detected in multiple countries. Much is currently unknown about the natural history of the disease, such as a possible asymptomatic spreading or the role of age in both the susceptibility and mortality of the disease. Nonetheless, authorities have to take action and implement contention measures, even if not everything is known. To facilitate this task, we have studied the effect of different containment strategies that can be put into effect. Our work specifically refers to the situation in Spain as of February 28th, 2020, where a few dozens of cases have been detected. We implemented an SEIR-metapopulation model that allows tracing explicitly the spatial spread of the disease through data-driven stochastic simulations. Our results are in line with the most recent recommendations from the World Health Organization, namely, that the best strategy is the early detection and isolation of individuals with symptoms, followed by interventions and public recommendations aimed at reducing the transmissibility of the disease, which although not efficacious for disease eradication, would produce as a second-order effect a delay of several days in the raise of the number of infected cases</td><td>2020-03-06</td><td>Alberto Aleta; Yamir Moreno</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029801</td></tr><tr><td>7reqkx3h</td><td>d7111b44f2d13b24a11d55c8be2d2bbe6b245eac</td><td>medrxiv</td><td>COVID-19 Epidemic Outside China: 34 Founders and Exponential Growth</td><td>10.1101/2020.03.01.20029819</td><td>null</td><td>null</td><td>medrvix</td><td>Background: In December 2019, pneumonia infected with a novel coronavirus burst in Wuhan, China. Now the situation is almost controlled in China but is worse outside China. We aimed to build a mathematical model to capture the global trend of epidemics outside China. Methods: In this retrospective, outside-China diagnosis number reported from Jan 21 to Feb 28, 2020 was downloaded from WHO website. We develop a simple regression model on these numbers: log10 (Nt+34)=0.0515*t+2.075 where Nt is the total diagnosed patient till the ith day, t=1 at Feb 1. Findings: Based on this model, we estimate that there have been about 34 unobserved founder patients at the beginning of spread outside China. The global trend is approximately exponential, with the rate of 10 folds every 19 days.</td><td>2020-03-03</td><td>Yi Li; Meng Liang; Xianhong Yin; Xiaoyu Liu; Meng Hao; Zixin Hu; Yi Wang; Li Jin</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029819</td></tr><tr><td>j58f1lwa</td><td>f8360b0e64d444215d21a61e056b070f12417e53</td><td>medrxiv</td><td>Preliminary epidemiological analysis on children and adolescents with novel coronavirus disease 2019 outside Hubei Province, China: an observational study utilizing crowdsourced data</td><td>10.1101/2020.03.01.20029884</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The outbreak of coronavirus disease 2019 (COVID-19) continues to expand across the world. Though both the number of cases and mortality rate in children and adolescents is reported to be low in comparison to adults, limited data has been reported on the outbreak with respect to pediatric patients. To elucidate information, we utilized crowdsourced data to perform a preliminary epidemiologic analysis of pediatric patients with COVID-19 Methods: In this observational study, data was collected from two open-access, line list crowdsourced online databases. Pediatric cases of COVID-19 were defined as patients ≤19 years of age with a laboratory confirmed diagnosis. The primary outcomes were case counts and cumulative case counts. Secondary outcomes included days between symptoms onset and first medical care and days between first medical care and reporting. Tertiary outcomes were rate of travel to Wuhan, rate of infected family members and rates of symptoms. Results: A total of 82 patients were included. The median age was 10 [IQR: 5-15] years. Patients from mainland China (outside Hubei) accounted for 46.3% of cases, while the remaining 53.7% of cases were international. Males and females accounted for 52.4% and 32.9% of cases, respectively, with the remaining 14.6% being designated as unknown. A male skew persisted across subgroup analyses by age group (p=1.0) and location (inside/outside China) (p=0.22). While the number of reported international cases has been steadily increasing over the study period, the number of reported cases in China rapidly decreased from the start point. The median reporting delay was 3 [IQR: 2-4.8] days. The median delay between symptom onset and first seeking medical care was 1 [IQR: 0-3.25] day. In international cases, time to first seeking medical care was a median of 2.5 days longer than in China (p=0.04). When clinical features were reported, fever was the most common presentation (68.0%), followed by cough (36.0%). Conclusions: The number of reported international pediatric COVID-19 cases is rapidly increasing. COVID-19 infections are, to-date, more common in males than females in both the children and adolescent age groups. Additionally, this male predominance remains the case both inside and outside of China. Crowdsourced data enabled early analysis of epidemiologic variables in pediatric patients with COVID-19. Further data sharing is required to enable analyses that are required to understand the course of this infection in children.</td><td>2020-03-06</td><td>Brandon Michael Henry; Maria Helena S Oliveira</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.20029884</td></tr><tr><td>6om1y33o</td><td>a891d059692c951d1fb0fb36914fe451f973684a</td><td>biorxiv</td><td>Kallikrein 13: a new player in coronaviral infections</td><td>10.1101/2020.03.01.971499</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTHuman coronavirus HKU1 (HCoV-HKU1) is associated with respiratory disease and is prevalent worldwide, but in vitro model for virus replication is lacking. Interaction between the coronaviral spike (S) protein and its receptor is the major determinant of virus tissue and host specificity, but virus entry is a complex process requiring a concerted action of multiple cellular elements. Here, we show that KLK13 is required for the infection of the human respiratory epithelium and is sufficient to mediate the entry of HCoV-HKU1 to non-permissive RD cells. We also demonstrated HCoV-HKU1 S protein cleavage by KLK13 in the S1/S2 region, proving that KLK13 is the priming enzyme for this virus. Summarizing, we show for the first time that protease distribution and specificity predetermines the tissue and cell specificity of the virus and may also regulate interspecies transmission. It is also of importance that presented data may be relevant for the emerging coronaviruses, including SARS-CoV-2 and may help to understand the differences in their zoonotic potential.</td><td>2020-03-02</td><td>Aleksandra Milewska; Katherine Falkowski; Magdalena Kalinska; Ewa Bielecka; Antonina Naskalska; Pawel Mak; Adam Lesner; Marek Ochman; Maciej Urlik; Jan Potempa; Tomasz Kantyka; Krzysztof Pyrc</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.01.971499</td></tr><tr><td>yu89n6rq</td><td>d810f78f8850d9386abc6fdd8d652f2f85191753</td><td>medrxiv</td><td>The effect of human mobility and control measures on the COVID-19 epidemic in China</td><td>10.1101/2020.03.02.20026708</td><td>null</td><td>null</td><td>medrvix</td><td>The ongoing COVID-19 outbreak has expanded rapidly throughout China. Major behavioral, clinical, and state interventions are underway currently to mitigate the epidemic and prevent the persistence of the virus in human populations in China and worldwide. It remains unclear how these unprecedented interventions, including travel restrictions, have affected COVID-19 spread in China. We use real-time mobility data from Wuhan and detailed case data including travel history to elucidate the role of case importation on transmission in cities across China and ascertain the impact of control measures. Early on, the spatial distribution of COVID-19 cases in China was well explained by human mobility data. Following the implementation of control measures, this correlation dropped and growth rates became negative in most locations, although shifts in the demographics of reported cases are still indicative of local chains of transmission outside Wuhan. This study shows that the drastic control measures implemented in China have substantially mitigated the spread of COVID-19.</td><td>2020-03-06</td><td>Moritz U.G. Kraemer; Chia-Hung Yang; Bernardo Gutierrez; Chieh-Hsi Wu; Brennan Klein; David M. Pigott; Louis du Plessis; Nuno R Faria; Ruoran Li; William P. Hanage; John S Brownstein; Maylis Layan; Alessandro Vespignani; Huaiyu Tian; Christopher Dye; Simon Cauchemez; Oliver Pybus; Samuel V Scarpino</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20026708</td></tr><tr><td>ngbfiws5</td><td>d85d142716f7f85cd27b54fc38a9e2968cf14346</td><td>medrxiv</td><td>Effects of weather-related social distancing on city-scale transmission of respiratory viruses</td><td>10.1101/2020.03.02.20027599</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND: Unusually high snowfall in western Washington State in February 2019 led to widespread school and workplace closures. We assessed the impact of social distancing caused by this extreme weather event on the transmission of respiratory viruses. METHODS: Residual specimens from patients evaluated for acute respiratory illness at hospitals in the Seattle metropolitan area were screened for a panel of respiratory viruses. Transmission models were fit to each virus, with disruption of contact rates and care-seeking informed by data on local traffic volumes and hospital admissions. RESULTS: Disruption in contact patterns reduced effective contact rates during the intervention period by 16% to 95%, and cumulative disease incidence through the remainder of the season by 3% to 9%. Incidence reductions were greatest for viruses that were peaking when the disruption occurred and least for viruses in early epidemic phase. CONCLUSION: High-intensity, short-duration social distancing measures may substantially reduce total incidence in a respiratory virus epidemic if implemented near the epidemic peak.</td><td>2020-03-03</td><td>Michael L Jackson; Gregory R Hart; Denise J McCulloch; Amanda Adler; Elisabeth Brandstetter; Kairsten Fay; Peter Han; Kirsten Lacombe; Jover Lee; Thomas Sibley; Deborah A Nickerson; Mark Rieder; Lea Starita; Janet A Englund; Trevor Bedford; Helen Chu; Michael Famulare</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20027599</td></tr><tr><td>rlsdznmv</td><td>null</td><td>medrxiv</td><td>Monitoring Disease Transmissibility of 2019 Novel Coronavirus Disease in Zhejiang, China</td><td>10.1101/2020.03.02.20028704</td><td>null</td><td>null</td><td>medrvix</td><td>We monitored the transmissibility of 2019 novel coronavirus disease in Zhejiang accounting the transmissions from imported cases. Even though Zhejiang is one of the worst-affected provinces, an interruption of disease transmission (i.e. instantaneous reproduction numbers <1) was observed in early/mid-February after an early social-distancing response to the outbreak.</td><td>2020-03-05</td><td>Ka Chun Chong; Wei Cheng; Shi Zhao; Feng Ling; Kirran N Mohammad; Maggie Haitian Wang; Benny Chung-ying Zee; Lesley Wei; Xi Xiong; Hengyan Liu; Jingxuan Wang; Enfu Chen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20028704</td></tr><tr><td>qlkt5fzp</td><td>70cc2e5152d3dc4d44494124ff556c9bbe9e6f41</td><td>medrxiv</td><td>Clinical Characteristics of Patients with Severe Pneumonia Caused by the 2019 Novel Coronavirus in Wuhan, China</td><td>10.1101/2020.03.02.20029306</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract Background: A new virus broke out in Wuhan, Hubei, China, and was later named 2019 novel coronavirus (2019-nCoV). The clinical characteristics of severe pneumonia caused by 2019-nCoV are still not clear. Objectives: The aim of this study was to explore the clinical characteristics and risk factors of the severe pneumonia caused by the 2019-nCoV in Wuhan, China. Method: The study included patients hospitalized at the central hospital of Wuhan who had been diagnosed with a pneumonia caused by the novel coronavirus. Clinical features, chronic co-morbidities, demographic data, laboratory examinations, and chest computed tomography (CT) scans were reviewed through electronic medical records. SPSS was used for data analysis to explore the clinical characteristics and risk factors of the patients with the severe pneumonia. Results: A total of 110 patients diagnosed with 2019 novel coronavirus pneumonia were included in the study, including 38 with severe pneumonia and 72 with non-severe pneumonia. Statistical analysis showed that advanced age, an increase of D-dimer, and a decrease of lymphocytes were characteristics of the patients with severe pneumonia. Moreover, in the early stage of the disease, chest CT scans of patients with the severe pneumonia showed the illness can progress rapidly. Conclusions: Advanced age, lymphocyte decline, and D-dimer elevation are important characteristics of patients with severe pneumonia. Clinicians should focus on these characteristics to identify high-risk patients at an early stage.</td><td>2020-03-03</td><td>Yafei Wang; Ying Zhou; Zhen Yang; Dongping Xia; Shuang Geng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20029306</td></tr><tr><td>4nmc356g</td><td>f82372b109168b411344f537210c63cc6ed323ed</td><td>medrxiv</td><td>The Seattle Flu Study: a multi-arm community-based prospective study protocol for assessing influenza prevalence, transmission, and genomic epidemiology</td><td>10.1101/2020.03.02.20029595</td><td>null</td><td>null</td><td>medrvix</td><td>Introduction. Influenza epidemics and pandemics cause significant morbidity and mortality. An effective response to a potential pandemic requires the infrastructure to rapidly detect, characterize, and potentially contain new and emerging influenza strains at a population level. The objective of this study is to use data gathered simultaneously from community and hospital sites to develop a model of how influenza enters and spreads in a population. Methods and Analysis. Starting in the 2018-19 season, we have been enrolling individuals with acute respiratory illness from community sites throughout the Seattle metropolitan area, including clinics, childcare facilities, Seattle-Tacoma International Airport, workplaces, college campuses, and homeless shelters. At these sites, we collect clinical data and mid-nasal swabs from individuals with at least two acute respiratory symptoms. Additionally, we collect residual nasal swabs and data from individuals who seek care for respiratory symptoms at four regional hospitals. Samples are tested using a multiplex molecular assay, and influenza whole genome sequencing is performed for samples with influenza detected. Geospatial mapping and computational modeling platforms are in development to characterize the regional spread of influenza and other respiratory pathogens. Ethics and Dissemination. The study was approved by the University of Washington's Institutional Review Board. Results will be disseminated through talks at conferences, peer-reviewed publications, and on the study website (www.seattleflu.org).</td><td>2020-03-06</td><td>Helen Y. Chu; Michael Boeckh; Janet A. Englund; Michael Famulare; Barry R. Lutz; Deborah A Nickerson; Mark J. Rieder; Lea M Starita; Amanda Adler; Elisabeth Brandstetter; Chris D. Frazar; Peter D. Han; Reena K. Gularti; James Hadfield; Michael L. Jackson; Anahita Kiavand; Louise E. Kimball; Kirsten Lacombe; Jennifer Logue; Victoria Lyon; Kira L. Newman; Thomas R. Sibley; Monica L. Zigman Suschsland; Caitlin Wolf; Jay Shendure; Trevor Bedford</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20029595</td></tr><tr><td>b10yokyu</td><td>5796fdc80a85b6e634a279c57711356055b65880</td><td>medrxiv</td><td>Close contacts and household transmission of SARS-CoV-2 in China: a content analysis based on local Heath Commissions' public disclosures.</td><td>10.1101/2020.03.02.20029868</td><td>null</td><td>null</td><td>medrvix</td><td>Public disclosure from Chinese local Health Commissions show that male and mid-aged index patients are the major source of virus from outside to the households. This implies that two important actions should be taken in disease control. First considers the kindergarten and school closure time. Local kindergarten and primary schools should postpone their admission and regular operation, to at least not earlier than local secondary schools. Second, workplace hygiene is of particular important in the next phase of disease control. As travel restrictions within China are being gradually lifted, we suggest that workplace hygiene should be taken serious care in the next month or two.</td><td>2020-03-06</td><td>xiaoke Xu; Xiaofan Liu; Ye Wu; Sheikh Taslim ALI</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20029868</td></tr><tr><td>c3dxfet9</td><td>ebed882da10cbf669cbb86802e9fa07a4a33ef91</td><td>medrxiv</td><td>Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</td><td>10.1101/2020.03.02.20029975</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of Coronavirus Disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerged in Wuhan, December 2019, and continuously poses a serious threat to public health. Our previous study has shown that cytokine storm occurred during SARS-CoV-2 infection, while the detailed role of cytokines in the disease severity and progression remained unclear due to the limited case number. In this study, we examined 48 cytokines in the plasma samples from 53 COVID-19 cases, among whom 34 were severe cases, and the others moderate. Results showed that 14 cytokines were significantly elevated upon admission in COVID-19 cases. Moreover, IP-10, MCP-3, and IL-1ra were significantly higher in severe cases, and highly associated with the PaO2/FaO2 and Murray score. Furthermore, the three cytokines were independent predictors for the progression of COVID-19, and the combination of IP-10, MCP-3 and IL-1ra showed the biggest area under the curve (AUC) of the receiver-operating characteristics (ROC) calculations. Serial detection of IP-10, MCP-3 and IL-1ra in 14 severe cases showed that the continuous high levels of these cytokines were associated with disease deterioration and fatal outcome. In conclusion, we report three cytokines that closely associated with disease severity and outcome of COVID-19. These findings add to our understanding of the immunopathologic mechanisms of SARS-CoV-2 infection, which suggested novel therapeutic targets and strategy.</td><td>2020-03-06</td><td>Yang Yang; Chenguang Shen; Jinxiu Li; Jing Yuan; Minghui Yang; Fuxiang Wang; Guobao Li; Yanjie Li; Li Xing; Ling Peng; Jinli Wei; Mengli Cao; Haixia Zheng; Weibo Wu; Rongrong Zou; Delin Li; Zhixiang Xu; Haiyan Wang; Mingxia Zhang; Zheng Zhang; Lei Liu; Yingxia Liu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20029975</td></tr><tr><td>zav5gksq</td><td>c8df44a3612e85e267351e936ddeb8fc5867afa1</td><td>medrxiv</td><td>The timing of one-shot interventions for epidemic control</td><td>10.1101/2020.03.02.20030007</td><td>null</td><td>null</td><td>medrvix</td><td>The apparent early success in China's large-scale intervention to control the COVID-19 epidemic has led to interest in whether other countries can replicate it as well as concerns about a resurgence of the epidemic if or when China relaxes the interventions. In this paper we look at the impact of a single short-term intervention on an epidemic. We see that if an intervention cannot be sustained long-term, it has the greatest impact if it is imposed once infection levels have become large enough that there is an appreciable number of infections present. For minimising the total number infected it should start close to the peak so that there is no rebound once the intervention is stopped, while to minimise the peak prevalence, it should start earlier, allowing two peaks of comparable size rather than one very large peak. In populations with distinct subgroups, synchronized interventions are less effective than targeting the interventions in each sub-population separately. We do not attempt to clearly determine what makes an intervention sustainable or not. We believe that is a policy question. If an intervention is sustainable, it should be kept in place. Our intent is to offer insight into how best to time an intervention whose impact on society is too great to maintain.</td><td>2020-03-06</td><td>Francesco Di Lauro; István Z Kiss; Joel Miller</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030007</td></tr><tr><td>wtdzad90</td><td>d0b33d173f1d32662e2ad8d7e85181f9b23f010c</td><td>medrxiv</td><td>Estimation of COVID-19 outbreak size in Italy based on international case exportations</td><td>10.1101/2020.03.02.20030049</td><td>null</td><td>null</td><td>medrvix</td><td>Italy is currently experiencing an epidemic of COVID-19 which emerged in the Lombardy region . During the interval between February 25-29, 2020, we identified 46 cases of COVID-19 reported in 21 countries in Europe, Africa, North America, and South America which were either in individuals with recent travel from Italy, or who had presumed infection by a traveler from Italy 2. In six cases, in four of the affected countries (Switzerland, France, Austria, Croatia), land travel was a likely route of introduction, or was documented to have been the route of introduction. We used air travel volume between Italian cities and cities in other countries as an index of connectedness, using data available from the International Air Transport Association (IATA) for February 2015 (2.61 million total departing international air passengers from Italy). We used the methods of Fraser et al. to estimate the size of the underlying epidemic in Italy necessary in order for these cases to be observed with a reasonable probability. To estimate the time at risk of COVID-19 exposure for travelers departing Italy, we obtained data from the United Nations World Tourism Organization (UNWTO) for the proportion of international travelers that are non-residents of Italy (63%) and the average length of stay of tourists to Italy (3.4 days), and assumed the Italian epidemic began one month prior to February 29, 2020. We also performed sensitivity analyses in which we included outbound travel to all countries regardless of reported case importations, inflated travel volumes by 35%, to account for the relative increase in flight numbers from 2015-2019, and excluded cases in bordering countries and which were documented to have been introduced by overland travel. When all cases were considered we estimated a true outbreak size of 3971 cases (95% CI 2907-5297), as compared to a reported case count of 1128 on February 29, 2020, suggesting non-identification of 72% (61-79%) of cases. In sensitivity analyses, outbreak sizes varied from 1552 to 4533 cases (implying non-identification of 27-75% of cases). We recently used similar methods to estimate a much larger epidemic size in Iran, with a far greater degree of under-reporting, based on many fewer exported cases. The reason for this difference relates to the relatively high volume of travel from Italy, relative to Iran. In summary, we suggest that the numerous COVID-19 case exportations from Italy in recent days suggest an epidemic that is larger than official case counts suggest, and which is approximately on a par with that currently occurring in South Korea, which reports 3526 cases (and fewer deaths) as of February 29, 2020.</td><td>2020-03-06</td><td>Ashleigh Tuite; Victoria Ng; Erin Rees; David Fisman</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030049</td></tr><tr><td>xfxf1vse</td><td>null</td><td>medrxiv</td><td>Forecasting the Cumulative Number of COVID-19 Deaths in China: a Boltzmann Function-based Modeling Study</td><td>10.1101/2020.03.02.20030064</td><td>null</td><td>null</td><td>medrvix</td><td>The COVID-19 outbreak is on-going in China. Here we estimated the potential total numbers of COVID-19 deaths in China, outside Hubei (in China), Hubei Province, Wuhan City and outside Wuhan (in Hubei) by Boltzmann function-based analyses, which are 3342 (95% CI, 3214, 3527), 111 (109, 114), 3245 (3100, 3423), 2613 (2498, 2767) and 627 (603, 654), respectively. The results may help to evaluate the severity of COVID-19 outbreaks and facilitate timely mental service for the families of passed patients.</td><td>2020-03-03</td><td>Cheng Long; Qi Ying; Xinmiao Fu; Zhongyan Li; Yuanyuan Gao</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030064</td></tr><tr><td>4f84pc83</td><td>3b181a741d4beafda3ba3a8fc68239493d49f6aa</td><td>medrxiv</td><td>Estimation of local novel coronavirus (COVID-19) cases in Wuhan, China from off-site reported cases and population flow data from different sources</td><td>10.1101/2020.03.02.20030080</td><td>null</td><td>null</td><td>medrvix</td><td>Backgrounds: In December 2019, a novel coronavirus (COVID-19) pneumonia hit Wuhan, Hubei Province, China and spread to the rest of China and overseas. The emergence of this virus coincided with the Spring Festival Travel Rush in China. It is possible to estimate total number of cases of COVID-19 in Wuhan, by 23 January 2020, given the cases reported in other cities and population flow data between cities. Methods: We built a model to estimate the total number of cases in Wuhan by 23 January 2020, based on the number of cases detected outside Wuhan city in China, with the assumption that if the same screening effort used in other cities applied in Wuhan. We employed population flow data from different sources between Wuhan and other cities/regions by 23 January 2020. The number of total cases was determined by the maximum log likelihood estimation. Findings: From overall cities/regions data, we predicted 1326 (95% CI: 1177, 1484), 1151 (95% CI: 1018, 1292) and 5277 (95% CI: 4732, 5859) as total cases in Wuhan by 23 January 2020, based on different source of data from Changjiang Daily newspaper, Tencent, and Baidu. From separate cities/regions data, we estimated 1059 (95% CI: 918, 1209), 5214 (95% CI: 4659, 5808) as total cases in Wuhan in Wuhan by 23 January 2020, based on different sources of population flow data from Tencent and Baidu. Conclusion: Sources of population follow data and methods impact the estimates of local cases in Wuhan before city lock down. Keyword: COVID-19; mobility; pneumonia; transportation; outbreaks</td><td>2020-03-02</td><td>Zian Zhuang; Peihua Cao; Shi Zhao; Yijun Lou; Weiming Wang; Shu Yang; Lin Yang; Daihai He</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030080</td></tr><tr><td>va4t15mb</td><td>267729947ca478946d5a4bffb8e13d50c3545120</td><td>medrxiv</td><td>Rapid Detection of SARS-CoV-2 Using Reverse transcription RT-LAMP method</td><td>10.1101/2020.03.02.20030130</td><td>null</td><td>null</td><td>medrvix</td><td>Corona Virus Disease 2019 (COVID-19) is a recently emerged life-threatening disease caused by SARS-CoV-2. Real- time fluorescent PCR (RT-PCR) is the clinical standard for SARS-CoV-2 nucleic acid detection. To detect SARS-CoV-2 early and control the disease spreading on time, a faster and more convenient method for SARS-CoV-2 nucleic acid detecting, RT-LAMP method (reverse transcription loop-mediated isothermal amplification) was developed. RNA reverse transcription and nucleic acid amplification were performed in one step at 63 ℃ isothermal conditions, and the results can be obtained within 30 minutes. ORF1ab gene, E gene and N gene were detected at the same time. ORF1ab gene was very specific and N gene was very sensitivity, so they can guarantee both sensitivity and specificity for SARS-CoV-2. The sensitivity of RT-LAMP assay is similar to RT-PCR, and specificity was 99% as detecting 208 clinical specimens. The RT-LAMP assay reported here has the advantages of rapid amplification, simple operation, and easy detection, which is useful for the rapid and reliable clinical diagnosis of SARS-CoV-2.</td><td>2020-03-02</td><td>Weihua Yang; Xiaofei Dang; Qingxi Wang; Mingjie Xu; Qianqian Zhao; Yunying Zhou; Huailong Zhao; Li Wang; Yihui Xu; Jun Wang; Shuyi Han; Min Wang; Fengyan Pei; Yunshan Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030130</td></tr><tr><td>i7tjno0w</td><td>b53dd68cad40cbcd698a848f53853e5f03d44a1f</td><td>medrxiv</td><td>Validity of Wrist and Forehead Temperature in Temperature Screening in the General Population During the Outbreak of 2019 Novel Coronavirus: a prospective real-world study</td><td>10.1101/2020.03.02.20030148</td><td>null</td><td>null</td><td>medrvix</td><td>Aims: Temperature screening is important in the population during the outbreak of 2019 Novel Coronavirus (COVID-19). This study aimed to compare the accuracy and precision of wrist and forehead temperature with tympanic temperature under different circumstances. Methods: We performed a prospective observational study in a real-life population. We consecutively collected wrist and forehead temperatures in Celsius (C) using a non-contact infrared thermometer (NCIT). We also measured the tympanic temperature using a tympanic thermometers (IRTT) and defined fever as a tympanic temperature ≥37.3C. Results: We enrolled a total of 528 participants including 261 indoor and 267 outdoor participants. We divided outdoor participants into four types according to their means of transportation to the hospital as walk, bicycle, electric vehicle, car, and inside the car. Under different circumstance, the mean difference ranged from -1.72 to -0.56C in different groups for the forehead measurements, and -0.96 to -0.61C for the wrist measurements. Both measurements had high fever screening abilities in inpatients (wrist: AUC 0.790; 95% CI: 0.725-0.854, P <0.001; forehead: AUC 0.816; 95% CI: 0.757-0.876, P <0.001). The cut-off value of wrist measurement for detecting tympanic temperature ≥37.3C was 36.2C with a 86.4% sensitivity and a 67.0% specificity, and the best threshold of forehead measurement was also 36.2C with a 93.2% sensitivity and a 60.0% specificity. Conclusions: Wrist measurement is more stable than forehead measurement under different circumstance. Both measurements have great fever screening abilities for indoor patients. The cut-off value of both measurements was 36.2C.</td><td>2020-03-06</td><td>Ge Chen; Jiarong Xie; Guangli Dai; Peijun Zheng; Xiaqing Hu; Hongpeng Lu; Lei Xu; Xueqin Chen; Xiaomin Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030148</td></tr><tr><td>llqpfhwg</td><td>ff067164497bcfbd9145be223dcd2b05f159dd63</td><td>medrxiv</td><td>Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</td><td>10.1101/2020.03.02.20030189</td><td>null</td><td>null</td><td>medrvix</td><td>Summary Background The novel coronavirus SARS-CoV-2 is a newly emerging virus. The antibody response in infected patient remains largely unknown, and the clinical values of antibody testing have not been fully demonstrated. Methods A total of 173 patients with confirmed SARS-CoV-2 infection were enrolled. Their serial plasma samples (n = 535) collected during the hospitalization period were tested for total antibodies (Ab), IgM and IgG against SARS-CoV-2 using immunoassays. The dynamics of antibodies with the progress and severity of disease was analyzed. Findings Among 173 patients, the seroconversion rate for Ab, IgM and IgG was 93.1% (161/173), 82.7% (143/173) and 64.7% (112/173), respectively. Twelve patients who had not seroconverted were those only blood samples at the early stage of illness were collected. The seroconversion sequentially appeared for Ab, IgM and then IgG, with a median time of 11, 12 and 14 days, respectively. The presence of antibodies was < 40% among patients in the first 7 days of illness, and then rapidly increased to 100.0%, 94.3% and 79.8% for Ab, IgM and IgG respectively since day 15 after onset. In contrast, the positive rate of RNA decreased from 66.7% (58/87) in samples collected before day 7 to 45.5% (25/55) during days 15 to 39. Combining RNA and antibody detections significantly improved the sensitivity of pathogenic diagnosis for COVID-19 patients (p < 0.001), even in early phase of 1-week since onset (p = 0.007). Moreover, a higher titer of Ab was independently associated with a worse clinical classification (p = 0.006). Interpretation The antibody detection offers vital clinical information during the course of SARS-CoV-2 infection. The findings provide strong empirical support for the routine application of serological testing in the diagnosis and management of COVID-19 patients.</td><td>2020-03-03</td><td>Juanjuan Zhao; Quan Yuan; Haiyan Wang; Wei Liu; Xuejiao Liao; Yingying Su; Xin Wang; Jing Yuan; Tingdong Li; Jinxiu Li; Shen Qian; Congming Hong; Fuxiang Wang; Yingxia Liu; Zhaoqin Wang; Qing He; Zhiyong Li; Bin He; Tianying Zhang; Shengxiang Ge; Lei Liu; Jun Zhang; Ningshao Xia; Zheng Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030189</td></tr><tr><td>dvrstz6f</td><td>null</td><td>medrxiv</td><td>Estimating the burden of United States workers exposed to infection or disease: a key factor in containing risk of COVID-19 infection</td><td>10.1101/2020.03.02.20030288</td><td>null</td><td>null</td><td>medrvix</td><td>Introduction: With the global spread of COVID-19, there is a compelling public health interest in quantifying who is at increased risk of disease. Occupational characteristics, such as interfacing with the public and being in close quarters with other workers, not only put workers at high risk for disease, but also make them a nexus of disease transmission to the community. This can further be exacerbated through presenteeism, the term used to describe the act of coming to work despite being symptomatic for disease. Understanding which occupational groups are exposed to infection and disease in the workplace can help to inform public health risk response and management for COVID-19, and subsequent infectious disease outbreaks. Methods: To estimate the burden of United States workers exposed to infection and disease in the workplace, national employment data (by Standard Occupational Classification) maintained by the Bureau of Labor Statistics (BLS) was merged with BLS O*NET survey data, which ranks occupations with particular physical, ergonomic, and structural exposures. For this analysis, occupations reporting exposure to infection or disease more than once a month was the focus. Results: Based on our analyses, approximately 10% (14.4 M) of United States workers are employed in occupations where exposure to disease or infection occurs at least once per week. Approximately 18.4% (26.7 M) of all United States workers are employed in occupations where exposure to disease or infection occurs at least once per month. While the majority of exposed workers are employed in healthcare sectors, other occupational sectors also have high proportions of exposed workers. These include protective service occupations (e.g. police officers, correctional officers, firefighters), office and administrative support occupations (e.g. couriers and messengers, patient service representatives), education occupations (e.g. preschool and daycare teachers), community and social services occupations (community health workers, social workers, counselors), and even construction and extraction occupations (e.g. plumbers, septic tank installers, elevator repair). Conclusions: The large number of persons employed in a wide variety of occupations with frequent exposure to infection and disease underscore the importance of all workplaces developing risk response plans for COVID-19. This work also serves as an important reminder that the workplace is a key locus for public health interventions, which could protect both workers and the communities they serve.</td><td>2020-03-06</td><td>Marissa G Baker; Trevor K Peckham; Noah S. Seixas</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030288</td></tr><tr><td>etl9uwt7</td><td>7fb7c0c48b30e66dfaea5fdec16d999ed52d1ee0</td><td>medrxiv</td><td>Preliminary estimating the reproduction number of the coronavirus disease (COVID-19) outbreak in Republic of Korea and Italy by 5 March 2020</td><td>10.1101/2020.03.02.20030312</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus disease 2019 (COVID-19) outbreak and Italy has caused 6088 cases and 41 deaths in Republic of Korea and 3144 cases and 107 death in Italy by 5 March 2020. We modeled the transmission process in Republic of Korea and Italy with a stochastic model and estimated the basic reproduction number R0 as 2.6 (95% CI: 2.3-2.9) or 3.2 (95% CI: 2.9-3.5) in Republic of Korea, under the assumption that the exponential growth starting on 31 January or 5 February 2020, and 2.6 (95% CI: 2.3-2.9) or 3.3 (95% CI: 3.0-3.6) in Italy, under the assumption that the exponential growth starting on 5 February or 10 February 2020. Estimates of dispersion term (k) were 10 (95% CI: 5-56) or 22 (95% CI: 8-61) in Republic of Korea, and 13 (95% CI: 5-61) or 37 (95% CI: 13-61) in Italy, and all of which imply few super-spreading events.</td><td>2020-03-06</td><td>Zian Zhuang; Shi Zhao; Qianying Lin; Peihua Cao; Yijun Lou; Lin Yang; Shu Yang; Daihai He; Li Xiao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030312</td></tr><tr><td>zclymg50</td><td>e5163021a1b88e2c2335cca27fbfcf883f870830</td><td>medrxiv</td><td>Preliminary estimation of the novel coronavirus disease (COVID-19) cases in Iran: a modelling analysis based on overseas cases and air travel data</td><td>10.1101/2020.03.02.20030320</td><td>null</td><td>null</td><td>medrvix</td><td>As of 1 March 2020, Iran has reported 987 COVID-19 cases and including 54 associated deaths. At least six neighboring countries (Bahrain, Iraq, Kuwait, Oman, Afghanistan and Pakistan) have reported imported COVID-19 cases from Iran. We used air travel data and the cases from Iran to other Middle East countries and estimated 16533 (95% CI: 5925, 35538) COVID-19 cases in Iran by 25 February, before UAE and other Gulf Cooperation Council countries suspended inbound and outbound flights from Iran.</td><td>2020-03-06</td><td>Zian Zhuang; Shi Zhao; Qianying Lin; Peihua Cao; Yijun Lou; Lin Yang; Daihai He</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030320</td></tr><tr><td>jir7n19b</td><td>49ac69f362c27acbc6de0c5cbb640267e7a1e797</td><td>medrxiv</td><td>Clinical features and outcomes of 221 patients with COVID-19 in Wuhan, China</td><td>10.1101/2020.03.02.20030452</td><td>null</td><td>null</td><td>medrvix</td><td>Rationale: In late December 2019, an outbreak of acute respiratory illness, now officially named as COVID-19, or coronavirus disease 2019, emerged in Wuhan, China, now spreading across the whole country and world. More data were needed to understand the clinical characteristics of the disease. Objectives: To study the epidemiology, clinical features and outcomes of patients with COVID-19. Methods: we performed a single center, retrospective case series study in 221 patients with laboratory confirmed SARS-CoV-2 pneumonia at a university hospital. Measurements and Main Results: The median age was 55.0 years and 48.9% were male and only 8 (3.6%) patients had a history of exposure to the Huanan Seafood Market. Compared to the non-severe pneumonia patients, the median age of the severe patients was significantly older, and they were more likely to have chronic comorbidities. Most common symptoms in severe patients were high fever, anorexia and dyspnea. On admission, 33.0% patients showed leukopenia and 73.8% showed lymphopenia. In addition, the severe patients suffered a higher rate of co-infections with bacteria or fungus and they were more likely to developing complications. As of February 15, 2020, 19.0% patients had been discharged and 5.4% patients died. 80% of severe cases received ICU care, and 52.3% of them transferred to the general wards due to relieved symptoms, and the mortality rate of severe patients in ICU was 20.5%. Conclusions: The COVID-19 epidemic spreads rapidly by human-to-human transmission. Patients with elder age, chronic comorbidities, blood leukocyte/lymphocyte count, procalcitonin level, co-infection and severe complications might increase the risk of poor clinical outcomes. Keywords: coronavirus disease 2019; clinical features; outcomes; severe patients</td><td>2020-03-06</td><td>Guqin Zhang; Chang Hu; Linjie Luo; Fang Fang; Yongfeng Chen; Jianguo Li; Zhiyong Peng; Huaqin Pan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.20030452</td></tr><tr><td>xq1cxrvf</td><td>c15f8c23eedbedf8d48bbc88b8225ae9c9d234bf</td><td>biorxiv</td><td>Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</td><td>10.1101/2020.03.02.968388</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTSevere Acute Respiratory Syndrome Coronavirus 2 is rapidly spreading around the world. There is no existing vaccine or proven drug to prevent infections and stop virus proliferation. Although this virus is similar to human and animal SARS- and MERS-CoVs the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies and antivirals. We applied high-throughput protein production and structure determination pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures. Here we report the high-resolution crystal structure of endoribonuclease Nsp15/NendoU from SARS-CoV-2 – a virus causing current world-wide epidemics. We compare this structure with previously reported models of Nsp15 from SARS and MERS coronaviruses.</td><td>2020-03-03</td><td>Youngchang Kim; Robert Jedrzejczak; Natalia I. Maltseva; Michael Endres; Adam Godzik; Karolina Michalska; Andrzej Joachimiak</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.968388</td></tr><tr><td>fu3cl7lq</td><td>2eac7de1c407d9eab13a6d538142cd21a814ee7b</td><td>biorxiv</td><td>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</td><td>10.1101/2020.03.02.968818</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractRapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia’s first case of COVID-2019.</td><td>2020-03-05</td><td>Jessica E. Manning; Jennifer A. Bohl; Sreyngim Lay; Sophana Chea; Ly Sovann; Yi Sengdoeurn; Seng Heng; Chan Vuthy; Katrina Kalantar; Vida Ahyong; Michelle Tan; Jonathan Sheu; Cristina M. Tato; Joseph L. DeRisi; Laurence Baril; Veasna Duong; Philippe Dussart; Erik A. Karlsson</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.968818</td></tr><tr><td>cr22vp8b</td><td>7607a4ad84452e29998a80f44bfd6bb2f5f68a7f</td><td>biorxiv</td><td>Viral Architecture of SARS-CoV-2 with Post-Fusion Spike Revealed by Cryo-EM</td><td>10.1101/2020.03.02.972927</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSince December 2019, the outbreak of Coronavirus Disease 2019 (COVID-19) spread from Wuhan, China to the world, it has caused more than 87,000 diagnosed cases and more than 3,000 deaths globally. To fight against COVID-19, we carried out research for the near native SARS-CoV-2 and report here our preliminary results obtained. The pathogen of the COVID-19, the native SARS-CoV-2, was isolated, amplified and purified in a BSL-3 laboratory. The whole viral architecture of SARS-CoV-2 was examined by transmission electron microscopy (both negative staining and cryo-EM). We observed that the virion particles are roughly spherical or moderately pleiomorphic. Spikes have nail-like shape towards outside with a long body embedded in the envelope. The morphology of virion observed in our result indicates that the S protein of SARS-CoV-2 is in post-fusion state, with S1 disassociated. This state revealed by cryo-EM first time could provide an important information for the identification and relevant clinical research of this new coronavirus.</td><td>2020-03-05</td><td>Chuang Liu; Yang Yang; Yuanzhu Gao; Chenguang Shen; Bin Ju; Congcong Liu; Xian Tang; Jinli Wei; Xiaomin Ma; Weilong Liu; Shuman Xu; Yingxia Liu; Jing Yuan; Jing Wu; Zheng Liu; Zheng Zhang; Peiyi Wang; Lei Liu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.972927</td></tr><tr><td>xdsi7oye</td><td>null</td><td>biorxiv</td><td>Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient</td><td>10.1101/2020.03.02.972935</td><td>null</td><td>null</td><td>biorxiv</td><td>ABSTRACTThe etiologic agent of the outbreak of pneumonia in Wuhan China was identified as severe acute respiratory syndrome associated coronavirus 2 (SARS-CoV-2) in January, 2020. The first US patient was diagnosed by the State of Washington and the US Centers for Disease Control and Prevention on January 20, 2020. We isolated virus from nasopharyngeal and oropharyngeal specimens, and characterized the viral sequence, replication properties, and cell culture tropism. We found that the virus replicates to high titer in Vero-CCL81 cells and Vero E6 cells in the absence of trypsin. We also deposited the virus into two virus repositories, making it broadly available to the public health and research communities. We hope that open access to this important reagent will expedite development of medical countermeasures.Article SummaryScientists have isolated virus from the first US COVID-19 patient. The isolation and reagents described here will serve as the US reference strain used in research, drug discovery and vaccine testing.</td><td>2020-03-03</td><td>Jennifer Harcourt; Azaibi Tamin; Xiaoyan Lu; Shifaq Kamili; Senthil Kumar. Sakthivel; Janna Murray; Krista Queen; Ying Tao; Clinton R. Paden; Jing Zhang; Yan Li; Anna Uehara; Haibin Wang; Cynthia Goldsmith; Hannah A. Bullock; Lijuan Wang; Brett Whitaker; Brian Lynch; Rashi Gautam; Craig Schindewolf; Kumari G. Lokugamage; Dionna Scharton; Jessica A. Plante; Divya Mirchandani; Steven G. Widen; Krishna Narayanan; Shinji Makino; Thomas G. Ksiazek; Kenneth S. Plante; Scott C. Weaver; Stephen Lindstrom; Suxiang Tong; Vineet D. Menachery; Natalie J. Thornburg</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.972935</td></tr><tr><td>vorczspu</td><td>bad4192bbc6d41c066bc947eaeeafc53024c98be</td><td>biorxiv</td><td>Evidence for RNA editing in the transcriptome of 2019 Novel Coronavirus</td><td>10.1101/2020.03.02.973255</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe 2019-nCoV outbreak has become a global health risk. Editing by host deaminases is an innate restriction process to counter viruses, and it is not yet known whether it operates against coronaviruses. Here we analyze RNA sequences from bronchoalveolar lavage fluids derived from two Wuhan patients. We identify nucleotide changes that may be signatures of RNA editing: Adenosine-to-Inosine changes from ADAR deaminases and Cytosine-to-Uracil changes from APOBEC ones. A mutational analysis of genomes from different strains of human-hosted Coronaviridae reveals patterns similar to the RNA editing pattern observed in the 2019-nCoV transcriptomes. Our results suggest that both APOBECs and ADARs are involved in Coronavirus genome editing, a process that may shape the fate of both virus and patient.</td><td>2020-03-03</td><td>Salvatore Di Giorgio; Filippo Martignano; Maria Gabriella Torcia; Giorgio Mattiuz; Silvestro G. Conticello</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.973255</td></tr><tr><td>fhqbw32a</td><td>1d2947d5c0addcbbcfa71d87ea074a8b6ff2a973</td><td>biorxiv</td><td>Speed and strength of an epidemic intervention</td><td>10.1101/2020.03.02.974048</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractAn epidemic can be characterized by its speed (i.e., the exponential growth rate r) and strength (i.e., the reproductive number ℛ). Disease modelers have historically placed much more emphasis on strength, in part because the effectiveness of an intervention strategy is typically evaluated on this scale. Here, we develop a mathematical framework for this classic, strength-based paradigm and show that there is a corresponding speed-based paradigm which can provide complementary insights. In particular, we note that r = 0 is a threshold for disease spread, just like ℛ = 1, and show that we can measure the speed and strength of an intervention on the same scale as the speed and strength of an epidemic, respectively. We argue that, just as the strength-based paradigm provides the clearest insight into certain questions, the speed-based paradigm provides the clearest view in other cases. As an example, we show that evaluating the prospects of “test-and-treat” interventions against the human immunodeficiency virus (HIV) can be done more clearly on the speed than strength scale, given uncertainty in the proportion of HIV spread that happens early in the course of infection. We suggest that disease modelers should avoid over-emphasizing the reproductive number at the expense of the exponential growth rate, but instead look at these as complementary measures.</td><td>2020-03-03</td><td>Jonathan Dushoff; Sang Woo Park</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.974048</td></tr><tr><td>dbowa5bt</td><td>7221cb14c12070a78e4ee5abe9484c40289fceea</td><td>biorxiv</td><td>A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19</td><td>10.1101/2020.03.02.974139</td><td>null</td><td>null</td><td>biorxiv</td><td>SummaryThe unprecedented epidemic of pneumonia caused by a novel coronavirus, HCoV-19, in China and beyond has caused public health concern at a global scale. Although bats are regarded as the most likely natural hosts for HCoV-191,2, the origins of the virus remain unclear. Here, we report a novel bat-derived coronavirus, denoted RmYN02, identified from a metagenomics analysis of samples from 227 bats collected from Yunnan Province in China between May and October, 2019. RmYN02 shared 93.3% nucleotide identity with HCoV-19 at the scale of the complete virus genome and 97.2% identity in the 1ab gene in which it was the closest relative of HCoV-19. In contrast, RmYN02 showed low sequence identity (61.3%) to HCoV-19 in the receptor binding domain (RBD) and might not bind to angiotensin-converting enzyme 2 (ACE2). Critically, however, and in a similar manner to HCoV-19, RmYN02 was characterized by the insertion of multiple amino acids at the junction site of the S1 and S2 subunits of the Spike (S) protein. This provides strong evidence that such insertion events can occur in nature. Together, these data suggest that HCoV-19 originated from multiple naturally occurring recombination events among those viruses present in bats and other wildlife species.</td><td>2020-03-05</td><td>Hong Zhou; Xing Chen; Tao Hu; Juan Li; Hao Song; Yanran Liu; Peihan Wang; Di Liu; Jing Yang; Edward C. Holmes; Alice C. Hughes; Yuhai Bi; Weifeng Shi</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.974139</td></tr><tr><td>jjraqr85</td><td>20f2a638e2c9a1917b7c001c371335da3d288505</td><td>biorxiv</td><td>Partial RdRp sequences offer a robust method for Coronavirus subgenus classification</td><td>10.1101/2020.03.02.974311</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe recent reclassification of the Riboviria, and the introduction of multiple new taxonomic categories including both subfamilies and subgenera for coronaviruses (family Coronaviridae, subfamily Orthocoronavirinae) represents a major shift in how official classifications are used to designate specific viral lineages. While the newly defined subgenera provide much-needed standardisation for commonly cited viruses of public health importance, no method has been proposed for the assignment of subgenus based on partial sequence data, or for sequences that are divergent from the designated holotype reference genomes. Here, we describe the genetic variation of a partial region of the coronavirus RNA-dependent RNA polymerase (RdRp), which is one of the most used partial sequence loci for both detection and classification of coronaviruses in molecular epidemiology. We infer Bayesian phylogenies from more than 7000 publicly available coronavirus sequences and examine clade groupings relative to all subgenus holotype sequences. Our phylogenetic analyses are largely coherent with genome-scale analyses based on designated holotype members for each subgenus. Distance measures between sequences form discrete clusters between taxa, offering logical threshold boundaries that can attribute subgenus or indicate sequences that are likely to belong to unclassified subgenera both accurately and robustly. We thus propose that partial RdRp sequence data of coronaviruses is sufficient for the attribution of subgenus-level taxonomic classifications and we supply the R package, “MyCoV”, which provides a method for attributing subgenus and assessing the reliability of the attribution.Importance StatementThe analysis of polymerase chain reaction amplicons derived from biological samples is the most common modern method for detection and classification of infecting viral agents, such as Coronaviruses. Recent updates to the official standard for taxonomic classification of Coronaviruses, however, may leave researchers unsure as to whether the viral sequences they obtain by these methods can be classified into specific viral taxa due to variations in the sequences when compared to type strains. Here, we present a plausible method for defining genetic dissimilarity cut-offs that will allow researchers to state which taxon their virus belongs to and with what level of certainty. To assist in this, we also provide the R package ‘MyCoV’ which classifies user generated sequences.</td><td>2020-03-06</td><td>David A Wilkinson; Lea Joffrin; Camille Lebarbenchon; Patrick Mavingui</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.02.974311</td></tr><tr><td>hnx213kp</td><td>3a73b880acd8983f87b210ff7fee434fd6ecc8c8</td><td>medrxiv</td><td>Epidemiology and Transmission of COVID-19 in Shenzhen China: Analysis of 391 cases and 1,286 of their close contacts</td><td>10.1101/2020.03.03.20028423</td><td>null</td><td>null</td><td>medrvix</td><td>Background Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control. Methods The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk. Findings Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 21 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days. Interpretation Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R. Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.</td><td>2020-03-04</td><td>Qifang Bi; Yongsheng Wu; Shujiang Mei; Chenfei Ye; Xuan Zou; Zhen Zhang; Xiaojian Liu; Lan Wei; Shaun A Truelove; Tong Zhang; Wei Gao; Cong Cheng; Xiujuan Tang; Xiaoliang Wu; Yu Wu; Binbin Sun; Suli Huang; Yu Sun; Juncen Zhang; Ting Ma; Justin Lessler; Teijian Feng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20028423</td></tr><tr><td>jtwb17u8</td><td>0f8261c63cdeeb5c2bf9294eba2954e1ca71b244</td><td>medrxiv</td><td>Effect of non-pharmaceutical interventions for containing the COVID-19 outbreak in China</td><td>10.1101/2020.03.03.20029843</td><td>null</td><td>null</td><td>medrvix</td><td>Background: The COVID-19 outbreak containment strategies in China based on non-pharmaceutical interventions (NPIs) appear to be effective. Quantitative research is still needed however to assess the efficacy of different candidate NPIs and their timings to guide ongoing and future responses to epidemics of this emerging disease across the World. Methods: We built a travel network-based susceptible-exposed-infectious-removed (SEIR) model to simulate the outbreak across cities in mainland China. We used epidemiological parameters estimated for the early stage of outbreak in Wuhan to parameterise the transmission before NPIs were implemented. To quantify the relative effect of various NPIs, daily changes of delay from illness onset to the first reported case in each county were used as a proxy for the improvement of case identification and isolation across the outbreak. Historical and near-real time human movement data, obtained from Baidu location-based service, were used to derive the intensity of travel restrictions and contact reductions across China. The model and outputs were validated using daily reported case numbers, with a series of sensitivity analyses conducted. Results: We estimated that there were a total of 114,325 COVID-19 cases (interquartile range [IQR] 76,776 - 164,576) in mainland China as of February 29, 2020, and these were highly correlated (p<0.001, R2=0.86) with reported incidence. Without NPIs, the number of COVID-19 cases would likely have shown a 67-fold increase (IQR: 44 - 94), with the effectiveness of different interventions varying. The early detection and isolation of cases was estimated to prevent more infections than travel restrictions and contact reductions, but integrated NPIs would achieve the strongest and most rapid effect. If NPIs could have been conducted one week, two weeks, or three weeks earlier in China, cases could have been reduced by 66%, 86%, and 95%, respectively, together with significantly reducing the number of affected areas. However, if NPIs were conducted one week, two weeks, or three weeks later, the number of cases could have shown a 3-fold, 7-fold, and 18-fold increase across China, respectively. Results also suggest that the social distancing intervention should be continued for the next few months in China to prevent case numbers increasing again after travel restrictions were lifted on February 17, 2020. Conclusion: The NPIs deployed in China appear to be effectively containing the COVID-19 outbreak, but the efficacy of the different interventions varied, with the early case detection and contact reduction being the most effective. Moreover, deploying the NPIs early is also important to prevent further spread. Early and integrated NPI strategies should be prepared, adopted and adjusted to minimize health, social and economic impacts in affected regions around the World.</td><td>2020-03-06</td><td>Shengjie Lai; Nick W Ruktanonchai; Liangcai Zhou; Olivia Prosper; Wei Luo; Jessica R Floyd; Amy Wesolowski; Mauricio Santillana; Chi Zhang; Xiangjun Du; Hongjie Yu; Andrew J Tatem</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20029843</td></tr><tr><td>66ulqu11</td><td>3028628066ec2401f3981f4e70c5b1acd4cef573</td><td>medrxiv</td><td>Transmission interval estimates suggest pre-symptomatic spread of COVID-19</td><td>10.1101/2020.03.03.20029983</td><td>null</td><td>null</td><td>medrvix</td><td>Background: As the COVID-19 epidemic is spreading, incoming data allows us to quantify values of key variables that determine the transmission and the effort required to control the epidemic. We determine the incubation period and serial interval distribution for transmission clusters in Singapore and in Tianjin. We infer the basic reproduction number and identify the extent of pre-symptomatic transmission. Methods: We collected outbreak information from Singapore and Tianjin, China, reported from Jan.19-Feb.26 and Jan.21-Feb.27, respectively. We estimated incubation periods and serial intervals in both populations. Results: The mean incubation period was 7.1 (6.13, 8.25) days for Singapore and 9 (7.92, 10.2)days for Tianjin. Both datasets had shorter incubation periods for earlier-occurring cases. The mean serial interval was 4.56 (2.69, 6.42) days for Singapore and 4.22 (3.43, 5.01) for Tianjin. We inferred that early in the outbreaks, infection was transmitted on average 2.55 and 2.89days before symptom onset (Singapore, Tianjin). The estimated basic reproduction number for Singapore was 1.97 (1.45, 2.48) secondary cases per infective; for Tianjin it was 1.87 (1.65,2.09) secondary cases per infective. Conclusions: Estimated serial intervals are shorter than incubation periods in both Singapore and Tianjin, suggesting that pre-symptomatic transmission is occurring. Shorter serial intervals lead to lower estimates of R0, which suggest that half of all secondary infections should be prevented to control spread.</td><td>2020-03-06</td><td>Lauren Tindale; Michelle Coombe; Jessica E Stockdale; Emma Garlock; Wing Yin Venus Lau; Manu Saraswat; Yen-Hsiang Brian Lee; Louxin Zhang; Dongxuan Chen; Jacco Wallinga; Caroline Colijn</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20029983</td></tr><tr><td>5ciaonf0</td><td>462cbb326ccd8587cae7a3538c8c6712d9013698</td><td>medrxiv</td><td>Epidemiological and clinical features of 291 cases with coronavirus disease 2019 in areas adjacent to Hubei, China: a double-center observational study</td><td>10.1101/2020.03.03.20030353</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract Background: The clinical outcomes of COVID-19 patients in Hubei and other areas are different. We aim to investigate the epidemiological and clinical characteristics of patient with COVID-19 in Hunan which is adjacent to Hubei. Methods: In this double-center, observational study, we recruited all consecutive patients with laboratory confirmed COVID-19 from January 23 to February 14, 2020 in two designated hospitals in Hunan province, China. Epidemiological and clinical data from patients' electronic medical records were collected and compared between mild, moderate and severe/critical group in detail. Clinical outcomes were followed up to February 20, 2020. Findings: 291 patients with COVID-19 were categorized into mild group (10.0%), moderate group (72.8%) and severe/critical group (17.2%). The median age of all patients was 46 years (49.8% were male). 86.6% patients had an indirect exposure history. The proportion of patients that had been to Wuhan in severe/critical group (48.0% vs 17.2%, p=0.006) and moderate group (43.4% vs 17.2%, p=0.007) were higher than mild group. Fever (68.7%), cough (60.5%), and fatigue (31.6%) were common symptoms especially for severe and critical patients. Typical lung imaging finding were bilateral and unilateral ground glass opacity or consolidation. Leukopenia, lymphopenia and eosinopenia occurred in 36.1%, 22.7% and 50.2% patients respectively. Increased fibrinogen was detected in 45 of 58 (77.6%) patients with available results. 29 of 44 (65.9%) or 22 of 40 (55.0%) patients were positive in Mycoplasma pneumonia or Chlamydia pneumonia antibody test respectively. Compared with mild or moderate group, severe/critical group had a relative higher level of neutrophil, Neutrophil-to-Lymphocyte Ratio, h-CRP, ESR, CK, CK-MB, LDH, D-dimer, and a lower level of lymphocyte, eosinophils, platelet, HDL and sodium (all p<0.01). Most patients received antiviral therapy and Chinese Medicine therapy. As of February 20, 2020, 159 (54.6%) patients were discharged and 2 (0.7%) patients died during hospitalization. The median length of hospital stay in discharged patients was 12 days (IQR: 10-15). Interpretation: The epidemiological and clinical characteristics of COVID-19 patients in Hunan is different from patients in Wuhan. The proportion of patients that had been to Wuhan in severe/critical group and moderate group were higher than mild group. Laboratory and imaging examination can assist in the diagnosis and classification of COVID-19 patients.</td><td>2020-03-06</td><td>Xu Chen; Fang Zheng; Yanhua Qing; Shuizi Ding; Danhui Yang; Cheng Lei; Zhilan Yin; Xianglin Zhou; Dixuan Jiang; Qi Zuo; Jun He; Jianlei Lv; Ping Chen; Yan Chen; Hong Peng; Honghui Li; Yuanlin Xie; Jiyang Liu; Zhiguo Zhou; Hong Luo</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030353</td></tr><tr><td>cv32u0ox</td><td>e560eb28bb3f35099d2632f80adaa14436516474</td><td>medrxiv</td><td>Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</td><td>10.1101/2020.03.03.20030437</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Viral clearance is one important indicator for the recovery of SARS-CoV-2 infected patients. Suboptimal T and B cell responses can delay viral clearance in MERS and SARS patients. The role of leukomonocytes in viral clearance of COVID-19 patients is not yet well defined.Methods: From January 26 to February 28, 2020, an observational study was launched at Zhongnan Hospital of Wuhan University, Wuhan, China. We enrolled 25 laboratory-confirmed COVID-19 patients, whose throat-swab specimens were tested positive for SARS-CoV-2 infection by qRT-PCR. We comprehensively analyzed clinical records, counts of lymphocyte subsets including CD3+, CD4+, CD8+ T cells, B cells and NK cells in the patients who successfully cleared SARS-CoV-2, and compared to those that failed to, after a standardized treatment of 8-14 days. Findings: In 25 enrolled COVID-19 patients, lymphopeniawas a common feature. After the treatment, 14 patients were tested negative for SARS-CoV-2. The patients that cleared the infection had restored the numbers of CD3+, CD4+, CD8+ T cellsand B cells as compared to the still viral RNA positive patients, while the recovered patients had a higher count of leukomonocytes. Conclusions: By comparison of leukomonocytes counts in COVID-19 patients at different stages of the disease, we found that CD3+, CD4+, CD8+ T cells and B cells appear to play important roles in viral clearance. The restoration of leukomonocytes counts from peripheral blood can be used as prognosis for the recovery of an COVID-19 infection. We propose that restoration of leukomonocytes counts can be added to the COVID-19 diagnostic guidanceas a criterion for releasing and discharging patients.</td><td>2020-03-06</td><td>Xiaoping Chen; Jiaxin Ling; Pingzheng Mo; Yongxi Zhang; Qunqun Jiang; Zhiyong Ma; Qian Cao; Wenjia Hu; Shi Zou; Liangjun Chen; Lei Yao; Mingqi Luo; Tielong Chen; Liping Deng; Ke Liang; Shihui Song; Rongrong Yang; Ruiying Zheng; Shicheng Gao; Xien Gui; Hengning Ke; Wei Hou; Åke Lundkvist; Yong Xiong</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030437</td></tr><tr><td>y8hzi1gl</td><td>null</td><td>medrxiv</td><td>Modelling-based evaluation of the effect of quarantine control by the Chinese government in the coronavirus disease 2019 outbreak</td><td>10.1101/2020.03.03.20030445</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus disease 2019 (COVID-19) epidemic, which was first identified in Wuhan, China in December 2019, has rapidly spread all over China and across the world. By the end of February 2020, the epidemic outside Hubei province in China has been well controlled, yet the next wave of transmission in other countries may have just begun. A retrospective modeling of the transmission dynamics would provide insights into the epidemiological characteristics of the disease and evaluation of the effectiveness of the strict measures that have been taken by central and local governments of China. Using a refined susceptible-exposed-infectious-removed (SEIR) transmission model and a new strategy of model fitting, we were able to estimate model parameters in a dynamic manner. The resulting parameter estimation can well reflect the prevention policy scenarios. Our simulation results with different degrees of government control suggest that the strictly enforced quarantine and travel ban have significantly decreased the otherwise uncontrollable spread of the disease. Our results suggest similar measures should be considered by other countries that are of high risk of COVID-19 outbreak.</td><td>2020-03-06</td><td>Xinkai Zhou; Zhigui Wu; Ranran Yu; Shanni Cao; Wen Fang; Zhen Jiang; Fang Yuan; Chao Yan; Dijun Chen</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030445</td></tr><tr><td>944pn0k9</td><td>9701a8c529cd8c18124da4cd61c1165a64b50281</td><td>medrxiv</td><td>Evolving Epidemiology and Impact of Non-pharmaceutical Interventions on the Outbreak of Coronavirus Disease 2019 in Wuhan, China</td><td>10.1101/2020.03.03.20030593</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND We described the epidemiological features of the coronavirus disease 2019 (Covid-19) outbreak, and evaluated the impact of non-pharmaceutical interventions on the epidemic in Wuhan, China. METHODS Individual-level data on 25,961 laboratory-confirmed Covid-19 cases reported through February 18, 2020 were extracted from the municipal Notifiable Disease Report System. Based on key events and interventions, we divided the epidemic into four periods: before January 11, January 11-22, January 23 - February 1, and February 2-18. We compared epidemiological characteristics across periods and different demographic groups. We developed a susceptible-exposed-infectious-recovered model to study the epidemic and evaluate the impact of interventions. RESULTS The median age of the cases was 57 years and 50.3% were women. The attack rate peaked in the third period and substantially declined afterwards across geographic regions, sex and age groups, except for children (age <20) whose attack rate continued to increase. Healthcare workers and elderly people had higher attack rates and severity risk increased with age. The effective reproductive number dropped from 3.86 (95% credible interval 3.74 to 3.97) before interventions to 0.32 (0.28 to 0.37) post interventions. The interventions were estimated to prevent 94.5% (93.7 to 95.2%) infections till February 18. We found that at least 59% of infected cases were unascertained in Wuhan, potentially including asymptomatic and mild-symptomatic cases. CONCLUSIONS Considerable countermeasures have effectively controlled the Covid-19 outbreak in Wuhan. Special efforts are needed to protect vulnerable populations, including healthcare workers, elderly and children. Estimation of unascertained cases has important implications on continuing surveillance and interventions.</td><td>2020-03-06</td><td>Chaolong Wang; Li Liu; Xingjie Hao; Huan Guo; Qi Wang; Jiao Huang; Na He; Hongjie Yu; Xihong Lin; An Pan; Sheng Wei; Tangchun Wu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030593</td></tr><tr><td>d4dnz2lk</td><td>9c33486a49de4aea64ce61c0a2c21a88c316b6a8</td><td>medrxiv</td><td>SOCRATES: An online tool leveraging a social contact data sharing initiative to assess mitigation strategies for COVID-19</td><td>10.1101/2020.03.03.20030627</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: Establishing a social contact data sharing initiative and an interactive tool to assess mitigation strategies for COVID-19. Results: We organized data sharing of published social contact surveys via online repositories and formatting guidelines. We analyzed this social contact data in terms of weighted social contact matrices, next generation matrices, relative incidence and R0. We incorporated location-specific isolation measures (e.g. school closure or telework) and capture their effect on transmission dynamics. All methods have been implemented in an online application based on R Shiny and applied to COVID-19 with age-specific susceptibility and infectiousness. Using our online tool with the available social contact data, we illustrate that social distancing could have a considerable impact on reducing transmission for COVID-19. The effect itself depends on assumptions made about disease-specific characteristics and the choice of intervention(s). Keywords: social contact data, user interface, transmission dynamics, infectious diseases, epidemics, social distancing, behavioral changes, data sharing initiative, open-source, COVID-19</td><td>2020-03-06</td><td>Lander Willem; Thang Van Hoang; Sebastian Funk; Pietro Coletti; Philippe Beutels; Niel Hens</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030627</td></tr><tr><td>5imbomop</td><td>594d23a412649df7dbfef5347a2a0c208ed4417d</td><td>medrxiv</td><td>Immunodepletion with Hypoxemia: A Potential High Risk Subtype of Coronavirus Disease 2019</td><td>10.1101/2020.03.03.20030650</td><td>null</td><td>null</td><td>medrvix</td><td>Background The outbreak of COVID-2019 is becoming a global public health emergency. Although its basic clinical features have been reported, the dynamic characteristics of immune system in COVID-2019 patients, especially those critical patients with refractory hypoxemia, are not yet well understood. We aim to describe the dynamic characteristics of immune system in 3 critical patients with refractory hypoxemia, and discuss the relationship between hypoxemia severity and immune cell levels, and the changes of gut microbes of COVID-2019 patient. Methods This is a retrospective study from 3 patients with 2019-nCoV infection admitted to Renmin Hospital of Wuhan University, a COVID-2019 designated hospital in Wuhan, from January 31 to February 6, 2020. All patients were diagnosed and classified based on the Diagnosis and Treatment of New Coronavirus Pneumonia (6th edition) published by the National Health Commission of China4. We recorded the epidemiological history, demographic features, clinical characteristics, symptoms and signs, treatment and clinical outcome in detail. Blood samples were collected and we determined the expression levels of immune cells (CD3+ T cells, CD4+ T cells, CD8+ T cells, CD19+ B cells, and CD16+56+ NK cells) in different time points. Nanopore Targeted Sequencing was used to determine the alterations of gut microbiota homeostasis. Results Apart from the clinical features described previously4, we found that four patients had decreased immune cells and refractory hypoxemia during the hospitalization, and the severity of hypoxemia was strongly correlated to the expression levels of immune cells. Additionally, we found that the proportion of probiotics was significantly reduced, such as Bifidobacterium, Lactobacillus, and Eubacterium, and the proportion of conditioned pathogenic bacteria was significantly increased, such as Corynebacterium of Actinobacteria and Ruthenibacterium of Firmicutes. Notably, all patients died. Conclusions We discussed the dynamic characteristics of host immune system and the imbalance of gut microbiota in 3 critical patients with COVID-2019. Hypoxemia severity was closely related with host immune cell levels, and the vicious circle between immune disorder and gut microbiota imbalance may be a high risk of fatal pneumonia. To the best of our knowledge, this is the first study which revealing that immunodepletion with refractory hypoxemia is a potential high risk subtype of COVID-2019 and the vicious circle between immune disorder and gut dysbiosis may be a high risk of fatal pneumonia.</td><td>2020-03-06</td><td>Lilei Yu; Yongqing Tong; Gaigai Shen; Aisi Fu; Yanqiu Lai; Xiaoya Zhou; Yuan Yuan; Yuhong Wang; Yuchen Pan; Zhiyao Yu; Yan Li; Tiangang Liu; Hong Jiang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030650</td></tr><tr><td>9ecuca17</td><td>f879c69d61f14d7efc51023258d1361bc681d403</td><td>medrxiv</td><td>Clinical findings in critical ill patients infected with SARS-Cov-2 in Guangdong Province, China: a multi-center, retrospective, observational study</td><td>10.1101/2020.03.03.20030668</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract Background In December 2019, human infection with a novel coronavirus, known as SARS-CoV-2, was identified in Wuhan, China. The mortality of critical illness was high in Wuhan. Information about critically ill patients with SARS-CoV-2 infection outside of Wuhan is scarce. We aimed to provide the clinical features, treatment, and prognosis of the critically ill patients with SARS-CoV-2 infection in Guangdong Province. Methods In this multi-centered, retrospective, observational study, we enrolled critically ill patients with SARS-CoV-2 pneumonia who were admitted to the intensive care unit (ICU) in Guangdong Province. Demographic data, symptoms, laboratory findings, comorbidities, treatments, and prognosis were collected. Data were compared between patients with and without intubation. Results Forty-five critically ill patients with SARS-CoV-2 pneumonia were identified in 7 ICUs in Guangdong Province. The mean age was 56.7 years, and 29 patients (64.4%) were men. The most common symptoms at the onset of illness were high fever and cough. Majority of patients presented with lymphopenia and elevated lactate dehydrogenase. Treatment with antiviral drugs was initiated in all the patients. Thirty-seven patients (82.2%) had developed acute respiratory distress syndrome, and 13 (28.9%) septic shock. A total of 20 (44.4%) patients required intubation and 9 (20%) required extracorporeal membrane oxygenation. As of February 28th 2020, only one patient (2.2%) had died and half of them had discharged of ICU. Conclusions Infection with SARS-CoV-2 in critical illness is characterized by fever, lymphopenia, acute respiratory failure and multiple organ dysfunction. Compared with critically ill patients infected with SARS-CoV-2 in Wuhan, the mortality of critically ill patients in Guangdong Province was relatively low. These data provide some general understandings and experience for the critical patients with SARS-CoV-2 outside of Wuhan.</td><td>2020-03-06</td><td>Yonghao Xu; Zhiheng Xu; Xuesong Liu; Lihua Cai; Haichong Zheng; Yongbo Huang; Lixin Zhou; Linxi Huang; Yun Lin; Liehua Deng; Jianwei Li; Sibei Chen; Dongdong Liu; Zhimin Lin; Liang Zhou; Weiqun He; Xiaoqing Liu; Yimin Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030668</td></tr><tr><td>6t9kl3hq</td><td>e0a53c258ee0483666329ccab591c0753ff49531</td><td>medrxiv</td><td>Key Points of Clinical and CT Imaging Features of 2019 Novel Coronavirus (2019-nCoV) Imported Pneumonia Based On 21 Cases Analysis</td><td>10.1101/2020.03.03.20030775</td><td>null</td><td>null</td><td>medrvix</td><td>Background and Objective: WHO Director-General declared that the 2019-nCoV outbreak constitutes a Public Health Emergency of International Concern,and the outbreak is still on-going.Chest CT had been a key component of the diagnostic workup for patients with suspected infection. In this retrospective study, we attempt to summarize and analyze the chest CT features of 2019-nCov infections, and to identify the typical features to improved the diagnostic accuracy of new coronavirus pneumonia (NCP). Methods:Chest CT scans and Clinical data of 21 patients confirmed NCP in our hospital were enrolled.These patients were divided into mild and sever group according to clinical manifestations described by the 6th clinical practice guideline of NCP in China. Main clinical and chest CT features were analyzed and identify. Results: Fever (85.7%) and cough (80.9%) were the two main symptoms of NCP patients.More significantly higher incidence (85.7%) of shortness of breath in the severe cases. Multiple lesions in both lungs and with incidence of GGO(100%),vascular enlargement (76.5%) and cobblestone/reticular pattern(70.6%) were the major feature.The incidence of consolidation, mixed pattern and vascular enlargement features were up to 100% in the severe group, significantly higher than that of patients in mild group. In addition, the incidence of air-bronchogram, dilated bronchi with thickened wall and fibrosis in the severe group was significantly higher than that in the mild group. Conclusions: Fever and cough are the typical clinical features of NCP patients, and chest CT mainly manifested as multiple lesions in both lungs, often accompanied by GGO, vascular enlargement and cobblestone/reticular pattern.Changes in these main CT features can indicate development of the disease.</td><td>2020-03-06</td><td>wenxiu Wu; zhifeng xu; yabin Jin; aizhen Pan</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030775</td></tr><tr><td>zhdjtv4j</td><td>0ab795fc615df6457551a8e231dce1f268eef9d2</td><td>medrxiv</td><td>Caution: The clinical characteristics of COVID-19 patients at admission are changing</td><td>10.1101/2020.03.03.20030833</td><td>null</td><td>null</td><td>medrvix</td><td>Background With the emergence of 4rd generation transmission, the prevention and treatment of the novel coronavirus disease 2019 (COVID-19) has entered a new period. We aimed to report several changes in the clinical characteristics at admission of patients with COVID-19. Methods Clinical records and laboratory results of patients suffering from COVID-19 were retrospectively reviewed and matched with the admission dates to analyze the changes in characteristics at the onset of illness. Results Of the 89 affected patients, 31 [34.8%] patients were admitted from January 16 to 22, and 58 [65.2%] were admitted from January 23 to 29. Patients were admitted with more systemic symptoms, such as fever (21 [67.7%] of 31), fatigue (13 [41.9%] of 31), and myalgia (7 [22.6%] of 31), before January 23. More patients (10 [32.3%] of 31) admitted before January 23 had a small amount of sputum production compared with a smaller proportion (4 [6.9%] of 58) of the patients admitted after January 23. Other symptoms, such as cough, nausea, diarrhea, and chest tightness, were not significantly different between the two groups. In addition, the group admitted before January 23 had a larger proportion of patients with reduced lymphocyte (13 [54.2%] of 24), CD3 (11 [54.4%] of 21), and CD8 (9 [42.9%] of 21) counts and elevated serum amyloid A (SAA, 18 [75%] of 24). Conclusions The initial symptoms of recently infected patients seem more insidious, indicating that the new coronavirus may gradually evolve into a virus similar to influenza and latent in asymptomatic carriers for a long time.</td><td>2020-03-06</td><td>Zhaowei Chen; Jijia Hu; Zongwei Zhang; Shan Jiang; Tao Wang; Zhengli Shi; Zhan Zhang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030833</td></tr><tr><td>1mu1z4xd</td><td>5bb89950ec5a06e2b7f69b2a9c4213dda19b1ab0</td><td>medrxiv</td><td>Prediction of New Coronavirus Infection Based on a Modified SEIR Model</td><td>10.1101/2020.03.03.20030858</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND The outbreak of the new coronavirus infection in Wuhan City, Hubei Province in December 2019, poses a huge threat to China and even global public health security. Respiratory droplets and contact transmission are the main routes of transmission of new coronaviruses. Compared with SARS and Ebola viruses, new coronavirus infections are infectious during the incubation period. Traditional SEIR (susceptibility-exposure-infection-Removal) There are some differences in conditions for the prediction of the epidemic trend of new coronavirus infection. The outbreak of the new coronavirus infection coincided with the Spring Festival before and after the Chinese Spring Festival.It is necessary to make appropriate optimization and amendments to the traditional model to meet the actual evolution of the epidemic situation. METHODS The traditional SEIR model assumes that the virus-infected person is not infectious during the incubation period and that the infected person did not take isolation measures during the illness. The transmission of the new coronavirus no longer meets the basic assumptions of the classical kinetic system. Therefore, this article first establishes a modified SEIR model. Predict and analyze the changing trend of the epidemic situation, then estimate the parameters involved in the infection dynamics model, and then use Matlab to simulate the established dynamic equations based on public data and analyze the results. Recommendations for universal prevention and control of infectious diseases. RESULTS The first case of new coronavirus infection was confirmed in Wuhan on December 8, 2019. When Wuhan City took no action, assuming the average daily number of contacts per infected person k = 5, the number of infected persons will reach about 2,384,803 people; If wuhan adopts the measures of sealing the city on January 22, 2020, under the premise of k=2, the number of infected people decreases by 19,773 compared with that on January 23, and there is no significant change in the time when the number of infected people reaches the peak. Under the premise of k = 1, the number of infected persons was reduced by 14,330 compared with the closure on January 23, and the time to reach the peak of the number of infected persons was reduced by 2 days. If Wuhan City is closed for one day, the number of infected persons will increase from 106,145 to 130,626 under the premise of k = 2; the number of infected persons will increase from 74,369 to 92,010 under the premise of k = 1. CONCLUSIONS Comparing the number of confirmed diagnoses actually notified by the department with the number of infected people obtained from the simulation of the model, it can be seen that the city closure measures adopted by the Wuhan Municipal Government on January 23 and the first-level response measures adopted by the country are effective for the epidemic Prevention and control play a vital role. Wearing a mask when going out and avoiding close contact with people can effectively reduce the infection rate.</td><td>2020-03-06</td><td>Zhou Tang; Xianbin Li; Houqiang Li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030858</td></tr><tr><td>itp1hnql</td><td>null</td><td>medrxiv</td><td>Psychological impact of the coronavirus disease 2019 (COVID-19) outbreak on healthcare workers in China</td><td>10.1101/2020.03.03.20030874</td><td>null</td><td>null</td><td>medrvix</td><td>Introduction Since the outbreak of coronavirus disease 2019 (COVID-19), more than 3000 (including clinical diagnosis) healthcare workers (HCWs) have been infected with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in China. This study is aimed to investigate the risk perception and immediate psychological state of HCWs in the early stage of the COVID-19 epidemic. Methods This study utilized a cross-sectional survey designed on a convenient sample of 4357 HCWs in China. Its data were collected using anonymous structured questionnaires distributed through social software. 6 questions were set to evaluate the participants' risk perception of COVID-19, and a General Health Questionnaire was used to identify the participants' immediate psychological status. Descriptive statistics were used for data analysis. Risk perception and psychological status were compared by demographic characteristics and COVID-19 exposure experiences. Result A total of 4,600 questionnaires were distributed, and 4,357 qualified ones (94.7%) were collected. The main concerns of HCWs are: infection of colleagues (72.5%), infection of family members (63.9%), protective measures (52.3%) and medical violence (48.5%). And 39.1% of the HCWs had psychological distress, especially working in Wuhan, participating in frontline treatments, having been isolated and having family members or colleagues infected. Conclusions The finding indicating that, faced with the COVID-19 epidemic, HCWs, especially in Wuhan, were worried about the risks of infection and protective measures, resulting in psychological distress, so further actions should be taken.</td><td>2020-03-06</td><td>Yuhong Dai; Guangyuan Hu; Huihua Xiong; Hong Qiu; Xianglin Yuan</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.20030874</td></tr><tr><td>qeo5dfxg</td><td>null</td><td>biorxiv</td><td>Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</td><td>10.1101/2020.03.03.962332</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractA new coronavirus SARS-CoV-2, recently discovered in Wuhan, China, has caused over 74000 infection cases and 2000 deaths. Due to the rapidly growing cases and the unavailability of specific therapy, there is a desperate need for vaccines to combat the epidemic of SARS-CoV-2. In the present study, we performed an in silico approach based on the available virus genome to identify the antigenic B-cell epitopes and human-leukocyte-antigen (HLA) restricted T-cell epitopes. A total of 61 B-cell epitopes were initially identified, 19 of which with higher potential immunogenicity were used for vaccine design. 499 T-cell epitopes were predicted that showed affinity with the 34 most popular HLA alleles in Chinese population. Based on these epitopes, 30 vaccine candidates were designed and inspected against safety risks, including potential toxicity, human homologous, pharmaceutical peptides and bioactive peptides. Majority of vaccine peptides contained both B-cell and T-cell epitopes, which may interact with the most prevalent HLA alleles accounting for ~99% of Chinese population. Docking analysis showed stable hydrogen bonds of epitopes with their corresponding HLA alleles. In conclusion, these putative antigenic peptides may elicit the resistance response to the viral infection. In vitro and in vivo experiments are required to validate the effectiveness of these peptide vaccine.</td><td>2020-03-03</td><td>Ye Feng; Min Qiu; Shengmei Zou; Yun Li; Kai Luo; Rongchang Chen; Yingqiang Sun; Kui Wang; Xinlei Zhuang; Shanshan Zhang; Shuqing Chen; Fan Mo</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.962332</td></tr><tr><td>qmg58cgr</td><td>8349823092836fe397a59e38615d1491423dbe70</td><td>biorxiv</td><td>AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2</td><td>10.1101/2020.03.03.972133</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at https://github.com/tbwxmu/2019-nCov. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2.</td><td>2020-03-08</td><td>Bowen Tang; Fengming He; Dongpeng Liu; Meijuan Fang; Zhen Wu; Dong Xu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.972133</td></tr><tr><td>3ybko13r</td><td>0fa18c47543c3ce69a3c9e4d77958c4d3691b1ed</td><td>biorxiv</td><td>Significance of hydrophobic and charged sequence similarities in sodium-bile acid cotransporter and vitamin D-binding protein macrophage activating factor</td><td>10.1101/2020.03.03.975524</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractSodium-bile acid cotransporter, also denominated sodium-taurocholate cotransporting polypeptide (NTCP) is an integral membrane protein with multiple hydrophobic transmembrane domains. The third extracellular domain of NTCP presents a stretch of nine aminoacids (KGIVISLVL) that is characterized by pronounced hydrophobicity and serves as receptor for a protein, preS1, showing the hydrophobic epta-peptide sequence NPLGFFP. Vitamin D-binding protein macrophage activating factor (DBP-MAF) is a multifunctional protein that is characterized by two hydrophobic regions able to bind fatty acids and vitamin D, respectively. Here we demonstrate that NTCP and DBP-MAF show significant sequence similarities as far as hydrophobic stretches of aminoacids are concerned. Alignment of the sequence of seven aminoacids preceding the 157-KGIVISLVL-165 stretch of NTCP shows four aminoacids that are identical to those of the corresponding sequence of DBP-MAF, and two that are conserved substitutions. In addition, in the sequence of DBP-MAF that is aligned with the sequence YKGIVISLVL of NTCP, there are two contiguous negatively charged aminoacids (ED) and, in the preceding epta-peptide sequence, there are three negatively charged aminoacids (D-ED), whereas in the corresponding sequence of NTCP there are only two (D--D) that are not contiguous. This concentration of negatively charged aminoacids may be involved in binding of protein inserts characterized by high density of positively charges residues. The alternating hydrophobic and electrostatic interactions described in this paper may help elucidating the biological roles of these proteins as far as protein-protein interactions are concerned.</td><td>2020-03-05</td><td>Ibne Raihan Zunaid; Stefania Pacini; Marco Ruggiero</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.03.975524</td></tr><tr><td>ti9b1etu</td><td>3afd5fba7dc182ddfa769c0d766134b525581005</td><td>medrxiv</td><td>Transmission and clinical characteristics of coronavirus disease 2019 in 104 outside-Wuhan patients, China</td><td>10.1101/2020.03.04.20026005</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Cases with coronavirus disease 2019 (COVID-19) emigrated from Wuhan escalated the risk of spreading in other cities. This report focused on the outside-Wuhan patients to assess the transmission and clinical characteristics of this illness. Methods: Contact investigation was conducted on each patient who admitted to the assigned hospitals in Hunan Province (geographically adjacent to Wuhan) from Jan 22, 2020 to Feb 12, 2020. Demographic, clinical, laboratory and radiological characteristics, medication therapy and outcomes were collected and analyzed. Patients were confirmed by PCR test. Results: Of the 104 patients, 48 (46.15%) were imported cases and 56 (53.85%) were indigenous cases; 93 (89.42%) had a definite contact history with infections. Family clusters were the major body of patients. Transmission along the chain of 3 &ldquo:generations” was observed. Mean age was 43 (rang, 8-84) years (including 3 children) and 49 (47.12%) were male. Most patients had typical symptoms, 5 asymptomatic infections were found and 2 of them infected their relatives. The median incubation period was 6 (rang, 1-32) days, of 8 patients ranged from 18 to 32 days. Just 9 of 16 severe patients required ICU care. Until Feb 12, 2020, 40 (38.46%) discharged and 1 (0.96%) died. For the antiviral treatment, 80 (76.92%) patients received traditional Chinese medicine therapy. Conclusions: Family but not community transmission occupied the main body of infections in the two centers. Asymptomatic transmission demonstrated here warned us that it may bring more risk to the spread of COVID-19. The incubation period of 8 patients exceeded 14 days.</td><td>2020-03-06</td><td>Chengfeng Qiu; Qian Xiao; Xin Liao; Ziwei Deng; Huiwen Liu; Yuanlu Shu; Dinghui Zhou; Ye Deng; Hongqiang Wang; Xiang Zhao; Jianliang Zhou; Jin Wang; Zhihua Shi; Long Da</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20026005</td></tr><tr><td>s4d442kx</td><td>3deb6393bd43f440a69c84da9ed61986b1781d36</td><td>medrxiv</td><td>Nanopore target sequencing for accurate and comprehensive detection of SARS-CoV-2 and other respiratory viruses</td><td>10.1101/2020.03.04.20029538</td><td>null</td><td>null</td><td>medrvix</td><td>The ongoing novel coronavirus pneumonia COVID-19 outbreak in Wuhan, China, has engendered numerous cases of infection and death. COVID-19 diagnosis relies upon nucleic acid detection; however, current recommended methods exhibit high false-negative rates, low sensitivity, and cannot identify other respiratory virus infections, thereby resulting patient misdiagnosis and impeding epidemic containment. Combining the advantages of target amplification and long-read, real-time nanopore sequencing, we developed nanopore target sequencing (NTS) to detect SARS-CoV-2 and other respiratory viruses simultaneously within 6-10 h. Parallel testing with approved qPCR kits of SARS-CoV-2 and NTS using 61 nucleic acid samples from suspected COVID-19 cases confirmed that NTS identified more infected patients as positive, and could also monitor for mutated nucleic acid sequence or other respiratory virus infection in the test sample. NTS is thus suitable for contemporary COVID-19 diagnosis; moreover, this platform can be further extended for diagnosing other viruses or pathogens.</td><td>2020-03-06</td><td>Ming Wang; Aisi Fu; Ben Hu; Yongqing Tong; Ran Liu; Jiashuang Gu; Jianghao Liu; Wen Jiang; Gaigai Shen; Wanxu Zhao; Dong Men; Lilei Yu; Zixin Deng; Yan Li; Tiangang Liu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20029538</td></tr><tr><td>dmud2zf7</td><td>3fe2552d3d8e6c9ead2c41e30fbb69e917277f53</td><td>medrxiv</td><td>Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</td><td>10.1101/2020.03.04.20030395</td><td>null</td><td>null</td><td>medrvix</td><td>Background: Since mid-December 2019, a cluster of pneumonia-like diseases caused by a novel coronavirus, now designated COVID-19 by the WHO, emerged in Wuhan city and rapidly spread throughout China. Here we identify the clinical characteristics of COVID-19 in a cohort of patients in Shanghai. Methods: Cases were confirmed by real-time RT-PCR and were analysed for demographic, clinical, laboratory and radiological features. Results: Of 198 patients, the median duration from disease onset to hospital admission was 4 days. The mean age of the patients was 50.1 years, and 51.0% patients were male. The most common symptom was fever. Less than half of the patients presented with respiratory systems including cough, sputum production, itchy or sore throat, shortness of breath, and chest congestion. 5.6% patients had diarrhoea. On admission, T lymphocytes were decreased in 45.8% patients. Ground glass opacity was the most common radiological finding on chest computed tomography. 9.6% were admitted to the ICU because of the development of organ dysfunction. Compared with patients not treated in ICU, patients treated in the ICU were older, had longer waiting time to admission, fever over 38.5o C, dyspnoea, reduced T lymphocytes, elevated neutrophils and organ failure. Conclusions: In this single centre cohort of COVID-19 patients, the most common symptom was fever, and the most common laboratory abnormality was decreased blood T cell counts. Older age, male, fever over 38.5oC, symptoms of dyspnoea, and underlying comorbidity, were the risk factors most associated with severity of disease. Key words: 2019 novel coronavirus; acute respiratory infection; risk factors for disease severity</td><td>2020-03-06</td><td>Min Cao; Dandan Zhang; Youhua Wang; Yunfei Lu; Xiangdong Zhu; Ying Li; Honghao Xue; Yunxiao Lin; Min Zhang; Yiguo Sun; Zongguo Yang; Jia Shi; Yi Wang; Chang Zhou; Yidan Dong; Ping Liu; Steven M Dudek; Zhen Xiao; Hongzhou Lu; Longping Peng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20030395</td></tr><tr><td>na8odvj7</td><td>4380f0251c595d6cb551e643198d2dacd3c6746c</td><td>medrxiv</td><td>Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</td><td>10.1101/2020.03.04.20030916</td><td>null</td><td>null</td><td>medrvix</td><td>Corona Virus Disease 2019 (COVID-19) has spread rapidly to more than 70 countries and regions overseas and over 80000 cases have been infected, resulting in more than three thousand deaths. Rapid diagnosis of patients remains a bottleneck in containing the progress of the epidemic. We used automated chemiluminescent immunoassay to detect serum IgM and IgG antibodies to 2019-nCoV of 736 subjects. COVID-19 patients were becoming reactive(positive) for specific antibodies from 7-12 days after the onset of morbidity. Specific IgM and IgG increased with the progression of the disease. The areas under the ROC curves of IgM and IgG were 0.988 and 1.000, respectively. Specific antibody detection has good sensitivity and specificity. Detection of specific antibodies in patients with fever can be a good distinction between COVID-19 and other diseases, so as to be a complement to nucleic acid diagnosis to early diagnosis of suspected cases.</td><td>2020-03-06</td><td>Jin Zhang; Jianhua Liu; Na Li; Yong Liu; Rui Ye; Xiaosong Qin; Rui Zheng</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20030916</td></tr><tr><td>rscduazd</td><td>e066fdbb4a9338a57477424cb66fd4d75b2fd66d</td><td>medrxiv</td><td>Comparison of severe and non-severe COVID-19 pneumonia: review and meta-analysis</td><td>10.1101/2020.03.04.20030965</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: To compare the difference between severe and non-severe COVID-19 pneumonia and figure out the potential symptoms lead to severity. Methods: Articles from PubMed, Embase, Cochrane database, and google up-to 24 February 2020 were systematically reviewed. Eighteen Literatures were identified with cases of COVID-19 pneumonia. The extracted data includes clinical symptoms, age, gender, sample size and region et al were systematic reviewed and meta analyzed. Results: 14 eligible studies including 1,424 patients were analyzed. Symptoms like fever (89.2%), cough (67.2%), fatigue (43.6%) were common, dizziness, hemoptysis, abdominal pain and conjunctival congestion/conjunctivitis were rare. Polypnea/dyspnea in severe patients were significantly higher than non-severe (42.7% vs.16.3%, P<0.0001). Fever and diarrhea were higher in severe patients(p=0.0374and0.0267). Further meta-analysis showed incidence of fever(OR1.70,95%CI 1.01-2.87), polypnea/dyspnea(OR3.53, 95%CI 1.95-6.38) and diarrhea(OR1.80,95%CI 1.06-3.03) was higher in severe patients, which meant the severe risk of patients with fever, polypnea/dyspnea, diarrhea were 1.70, 3.53, 1.80 times higher than those with no corresponding symptoms. Conclusions: Fever, cough and fatigue are common symptoms in COVID-19 pneumonia. Compared with non-severe patients, the symptoms as fever, polypnea/dyspnea and diarrhea are potential symptoms lead to severity.</td><td>2020-03-09</td><td>Weiping Ji; Jing Zhang; Gautam Bishnu; Xudong Du; Xinxin Chen; Hui Xu; Xiaoling Guo; Zhenzhai Cai; Xian Shen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20030965</td></tr><tr><td>sozs02x0</td><td>1a12cc7d49d8521bec5d447ebb413cbcf5aca8f4</td><td>medrxiv</td><td>Study of the mental health status of medical personnel dealing with new coronavirus pneumonia</td><td>10.1101/2020.03.04.20030973</td><td>null</td><td>null</td><td>medrvix</td><td>Objective: This paper studied the relationship between personality traits and mental health conditions of medical personnel to provide a basis and reference for the implementation of targeted education on mental health. Methods: A self-report inventory, the Symptom Checklist-90 (SCL-90), was used to investigate the mental health status of 548 medical personnel dealing with the new coronavirus pneumonia in eight provinces and cities of China. Results: The overall mean SCL-90 score and mean values of factors (somatization, obsessive-compulsive, anxiety, phobic anxiety, and psychoticism) of the medical personnel were significantly higher than in the norm group (p < 0.05), while their average interpersonal sensitivity score was significantly lower (p < 0.01). In addition, personal factors affecting the mental health status of medical personnel were identified. ( all p < 0.05). Conclusion: The overall mental health status of medical personnel responding to new coronavirus pneumonia is generally higher than that of the norm group in China. The results of this study should contribute to measures to alleviate the psychological pressures on medical personnel dealing with the new coronavirus epidemic in China</td><td>2020-03-06</td><td>ning sun; jun xing; jun xu; ling shu geng; qian yu li</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20030973</td></tr><tr><td>dop8knqn</td><td>null</td><td>medrxiv</td><td>Case fatality risk of novel coronavirus diseases 2019 in China</td><td>10.1101/2020.03.04.20031005</td><td>null</td><td>null</td><td>medrvix</td><td>Objective The outbreak of novel coronavirus disease 2019 (COVID-19) imposed a substantial health burden in mainland China and remains a global epidemic threat. Our objectives are to assess the case fatality risk (CFR) among COVID-19 patients detected in mainland China, stratified by clinical category and age group. Method We collected individual information on laboratory-confirmed COVID-19 cases from publicly available official sources from December 29, 2019 to February 23, 2020. We explored the risk factors associated with mortality. We used methods accounting for right-censoring and survival analyses to estimate the CFR among detected cases. Results Of 12,863 cases reported outside Hubei, we obtained individual records for 9,651 cases, including 62 deaths and 1,449 discharged cases. The deceased were significantly older than discharged cases (median age: 77 vs 39 years, p<0.001). 58% (36/62) were male. Older age (OR 1.18 per year; 95%CI: 1.14 to 1.22), being male (OR 2.02; 95%CI: 1.02 to 4.03), and being treated in less developed economic regions (e.g., West and Northeast vs. East, OR 3.93; 95%CI: 1.74 to 8.85) were mortality risk factors. The estimated CFR was 0.89-1.24% among all cases. The fatality risk among critical patients was 2-fold higher than that among severe and critical patients, and 24-fold higher than that among moderate, severe and critical patients. Conclusions Our estimates of CFR based on laboratory-confirmed cases ascertained outside of Hubei suggest that COVID-19 is not as severe as severe acute respiratory syndrome and Middle East respiratory syndrome, but more similar to the mortality risk of 2009 H1N1 influenza pandemic in hospitalized patients. The fatality risk of COVID-19 is higher in males and increases with age. Our study improves the severity assessment of the ongoing epidemic and can inform the COVID-19 outbreak response in China and beyond.</td><td>2020-03-06</td><td>Xiaowei Deng; Juan Yang; Wei Wang; Xiling Wang; Jiaxin Zhou; Zhiyuan Chen; Jing Li; Yinzi Chen; Han Yan; Juanjuan Zhang; Yongli Zhang; Yan Wang; Qi Qiu; Hui Gong; Xianglin Wei; Lili Wang; Kaiyuan Sun; Peng Wu; Marco Ajelli; Benjamin J. Cowling; Cecile Viboud; Hongjie Yu</td><td>null</td><td>null</td><td>null</td><td>False</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031005</td></tr><tr><td>vg1vvm6f</td><td>dd14a222d0d516266debe1c487e32ad13c105291</td><td>medrxiv</td><td>Clinical characteristics of 101 non-surviving hospitalized patients with COVID-19: A single center, retrospective study</td><td>10.1101/2020.03.04.20031039</td><td>null</td><td>null</td><td>medrvix</td><td>Abstract Rationale: COVID-19 have deprived many lives in Wuhan, China and caused global concerns. Few studies focused on clinical characteristics non-survivors hospitalized with COVID-19. Objectives: We aimed to clarify the clinical characteristics of died patients with COVID-19 and tracked the causes of the rapid disease progression and death. Methods: 101 non-surviving patients with confirmed COVID-19 were included in this retrospective study. Clinical data were collected and compared between non-survivors who died within 3 days and after 3 days of admission. Multivariable regression analysis was used to analyze risk factors associated with rapid disease progression and death. Measurements and Main Results: Among included patients, median age was 71 years (IQR, 59-80), 60 (59.41%) were men. 82 (81.19%) had one or more comorbidities including hypertension (59 [58.42%]), diabetes (22 [21.78%]) etc. 100 (99.01%) suffered respiratory failure, 53 (52.48%) developed acute cardiac injury, acute kidney and liver injury occurred in 23 (22.77%) and 18 (17.82%) patients, respectively. Multivariable regression analysis showed that elevated high sensitivity troponin I (OR 2.68, 95%CI 1.31-5.49, P=0.007), neutrophils (OR 1.14, 95%CI 1.01-1.28, P=0.033) and depressed oxygen saturation (OR 0.94, 95%CI 0.89-0.99, P=0.027) on admission were associated with rapid death of patients with COVID-19. Conclusions: Older patients (>70 years) with comorbidities had a steeply increased risk of death with COVID-19. Respiratory failure, acute cardiac and kidney injury played a crucial role in the death of patients. Elevated high sensitivity troponin, neutrophils and depressed oxygen saturation predicted the rapid death of patients.</td><td>2020-03-06</td><td>Qiao Shi; Kailiang Zhao; Jia Yu; Jiarui Feng; Kaiping Zhao; Xiaoyi Zhang; Xiaoyan Chen; Peng Hu; Yupu Hong; Man Li; Fang Liu; Chen Chen; Weixing Wang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031039</td></tr><tr><td>4ep7nf3q</td><td>f47af9fe364fd9bac5e061142208d801667c5aca</td><td>medrxiv</td><td>How to differentiate COVID-19 pneumonia from heart failure with computed tomography at initial medical contact during epidemic period</td><td>10.1101/2020.03.04.20031047</td><td>null</td><td>null</td><td>medrvix</td><td>OBJECTIVES To compare chest CT findings in heart failure with those of Corona Virus Disease 2019 (COVID-19) pneumonia. BACKGROUND During epidemic period, chest computed tomography (CT) has been highly recommended for screening patients with suspected COVID-19. However, the comparison of CT imaging between heart failure and COVID-19 pneumonia has not been fully elucidated. METHODS Patients with heart failure (n=12), COVID-19 pneumonia (n=12) and one patient with both diseases were retrospectively enrolled. Clinical information and imaging of chest CT were collected and analyzed. RESULTS There was no difference of ground glass opacity (GGO), consolidation, crazy paving pattern, lobes affected and septal thickening between heart failure and COVID-19 pneumonia. However, less rounded morphology (8.3% vs. 67%, p=0.003), more peribronchovascular thickening (75% vs. 33%, p=0.041) and fissural thickening (33% vs. 0%, p=0.028), less peripheral distribution (33% vs. 92%, p=0.003) were found in heart failure group than that in COVID-19 group. Importantly, there were also more patients with upper pulmonary vein enlargement (75% vs. 8.3%, p=0.001), subpleural effusion and cardiac enlargement in heart failure group than that in COVID-19 group (50% vs. 0%, p=0.005, separately). Besides, more fibrous lesions were found in COVID-19 group although there was no statistical difference (25% vs. 0%, P=0.064) CONCLUSIONS Although there are some overlaps of CT imaging between heart failure and COVID-19, CT is still a useful tool in differentiating COVID-19 pneumonia.</td><td>2020-03-06</td><td>Zhaowei Zhu; Jianjun Tang; Xiangping Chai; Zhenfei Fang; Qiming Liu; Xinqun Hu; Danyan Xu; Jia He; Liang Tang; Shi Tai; Yuzhi Wu; Shenghua Zhou</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031047</td></tr><tr><td>mrsya6wz</td><td>29b3eaeacecf2bfc95b9a493dbb1ba4b2323dd46</td><td>medrxiv</td><td>Adjusted age-specific case fatality ratio during the COVID-19 epidemic in Hubei, China, January and February 2020</td><td>10.1101/2020.03.04.20031104</td><td>null</td><td>null</td><td>medrvix</td><td>The coronavirus disease 2019 (COVID-19) epidemic that originated in Wuhan, China has spread to more than 60 countries. We estimated the age-specific case fatality ratio (CFR) by fitting a transmission model to data from China, accounting for underreporting of cases and the time delay to death. Overall CFR among all infections was 1.6% (1.4-1.8%) and increased considerably for the elderly, highlighting the expected burden for populations with further expansion of the COVID-19 epidemic around the globe.</td><td>2020-03-06</td><td>Julien Riou; Anthony Hauser; Michel J Counotte; Christian L Althaus</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031104</td></tr><tr><td>f4hj35dr</td><td>734779fad93249a2f6fede6afd10eeff7b37919b</td><td>medrxiv</td><td>Projecting the transmission dynamics of SARS-CoV-2 through the post-pandemic period</td><td>10.1101/2020.03.04.20031112</td><td>null</td><td>null</td><td>medrvix</td><td>There is an urgent need to project how transmission of the novel betacoronavirus SARS-CoV-2 will unfold in coming years. These dynamics will depend on seasonality, the duration of immunity, and the strength of cross-immunity to/from the other human coronaviruses. Using data from the United States, we measured how these factors affect transmission of human betacoronaviruses HCoV-OC43 and HCoV-HKU1. We then built a mathematical model to simulate transmission of SARS-CoV-2 through the year 2025. We project that recurrent wintertime outbreaks of SARS-CoV-2 will probably occur after an initial pandemic wave. We summarize the full range of plausible transmission scenarios and identify key data still needed to distinguish between them, most importantly longitudinal serological studies to determine the duration of immunity to SARS-CoV-2.</td><td>2020-03-06</td><td>Stephen M Kissler; Christine Tedijanto; Edward Goldstein; Yonatan H. Grad; Marc Lipsitch</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031112</td></tr><tr><td>bpnqz1cn</td><td>ee902c8e20385bed8b5fccb85d4e53996069cc0c</td><td>medrxiv</td><td>Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</td><td>10.1101/2020.03.04.20031120</td><td>null</td><td>null</td><td>medrvix</td><td>BACKGROUND The outbreak of a novel coronavirus (SARS-CoV-2, previously provisionally named 2019 novel coronavirus or 2019-nCoV) since December 2019 in Wuhan, China, has become an emergency of major international concern. Apart from the respiratory system, it is unclear whether SARS-CoV-2 can also directly infect other tissues such as the kidney or induce acute renal failure. METHODS We conducted a retrospective analysis of estimated glomerular filtration rate (eGFR) along with other clinical parameters from 85 patients with laboratory-confirmed COVID-19 admitted to a hospital in Wuhan from January 17, 2020 to March 3, 2020. Kidney tissues from six patients with postmortem examinations in the other hospital were analyzed by Hematoxylin and Eosin (H&E) and in situ expression of viral nucleocaspid protein (NP) antigen, immune cell markers (CD8, CD68 and CD56) and the complement C5b-9 was detected by immunohistochemistry. RESULTS 27.06% (23/85) patients exhibited acute renal failure (ARF). The eldery patients and cases with comorbidities such as hypertension and heart failure more easily developed ARF (65.22% vs 24.19%, p< 0.001; 69.57% vs 11.29%, p< 0.001, respectively). H&E staining demonstrated kidney tissues from postmortems have severe acute tubular necrosis and lymphocyte infiltration. Immunohistochemistry showed that SARS-CoV-2 NP antigen was accumulated in kidney tubules. Viral infection not only induces CD68+ macrophages infiltrated into tubulointerstitium, but also enhances complement C5b-9 deposition on tubules. CONCLUSIONS SARS-CoV-2 induces ARF in COVID-19 patients. Viruses directly infect human kidney tubules to induce acute tubular damage. The viruses not only have direct cytotoxicity, but also initiate CD68+ macrophage together with complement C5b-9 deposition to mediate tubular pathogenesis.</td><td>2020-03-06</td><td>Bo Diao; Chenhui Wang; Rongshuai Wang; Zeqing Feng; Yingjun Tan; Huiming Wang; Changsong Wang; Liang Liu; Ying Liu; Yueping Liu; Gang Wang; Zilin Yuan; Liang Ren; Yuzhang Wu; Yongwen Chen</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031120</td></tr><tr><td>utlponna</td><td>f40eaec892778166e4f8f8eeb576893e82a591be</td><td>medrxiv</td><td>The impact of social distancing and epicenter lockdown on the COVID-19 epidemic in mainland China: A data-driven SEIQR model study</td><td>10.1101/2020.03.04.20031187</td><td>null</td><td>null</td><td>medrvix</td><td>The outbreak of coronavirus disease 2019 (COVID-19) which originated in Wuhan, China, constitutes a public health emergency of international concern with a very high risk of spread and impact at the global level. We developed data-driven susceptible-exposed-infectious-quarantine-recovered (SEIQR) models to simulate the epidemic with the interventions of social distancing and epicenter lockdown. Population migration data combined with officially reported data were used to estimate model parameters, and then calculated the daily exported infected individuals by estimating the daily infected ratio and daily susceptible population size. As of Jan 01, 2020, the estimated initial number of latently infected individuals was 380.1 (95%-CI: 379.8~381.0). With 30 days of substantial social distancing, the reproductive number in Wuhan and Hubei was reduced from 2.2 (95%-CI: 1.4~3.9) to 1.58 (95%-CI: 1.34~2.07), and in other provinces from 2.56 (95%-CI: 2.43~2.63) to 1.65 (95%-CI: 1.56~1.76). We found that earlier intervention of social distancing could significantly limit the epidemic in mainland China. The number of infections could be reduced up to 98.9%, and the number of deaths could be reduced by up to 99.3% as of Feb 23, 2020. However, earlier epicenter lockdown would partially neutralize this favorable effect. Because it would cause in situ deteriorating, which overwhelms the improvement out of the epicenter. To minimize the epidemic size and death, stepwise implementation of social distancing in the epicenter city first, then in the province, and later the whole nation without the epicenter lockdown would be practical and cost-effective.</td><td>2020-03-06</td><td>Yuzhen Zhang; Bin Jiang; Jiamin Yuan; Yanyun Tao</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031187</td></tr><tr><td>rbsv58fo</td><td>bfed3ce6a6df2d442d96d0cab07d4bc1f2347255</td><td>medrxiv</td><td>Seattle Flu Study - Swab and Send: Study Protocol for At-Home Surveillance Methods to Estimate the Burden of Respiratory Pathogens on a City-Wide Scale</td><td>10.1101/2020.03.04.20031211</td><td>null</td><td>null</td><td>medrvix</td><td>Introduction. While seasonal influenza and other respiratory pathogens cause significant morbidity and mortality each year, the community-based burden of these infections remains incompletely understood. Understanding the prevalence, epidemiology, and transmission dynamics of respiratory pathogen infections among community-dwelling individuals is essential during pandemic and epidemic settings and for developing pandemic-preparedness infrastructure. Methods and Analysis. We present the protocol for a novel, city-wide home-based cross-sectional study in the Seattle Metropolitan area, utilizing rapid delivery systems for self-collection of a nasal swab and return to the laboratory for respiratory pathogen testing. All participation takes place electronically, including recruitment, consent, and data collection. Within 48 hours of participants self-reporting respiratory symptoms, a nasal swab kit is delivered to the household via a courier service. Demographic and illness characteristics are collected at the time of sample collection and recovery and behavioral information collected one week later. Specimens are tested in the laboratory for multiple respiratory pathogens, and results are available on a public website for participants. Ethics and Dissemination. The study was approved by the University of Washington Institutional Review Board (Protocol #00006181). Results will be disseminated through peer-reviewed publications, talks at conferences, and on the Study Website (www.seattleflu.org).</td><td>2020-03-07</td><td>Ashley E Kim; Elisabeth Brandstetter; Chelsey Graham; Jessica Heimonen; Audrey Osterbind; Denise J McCulloch; Peter D Han; Lea M Starita; Deborah A Nickerson; Margaret M Van de Loo; Jennifer Mooney; Mark J Rieder; Misja Ilcisin; Kairsten A Fay; Jover Lee; Thomas R Sibley; Trevor Bedford; Janet A Englund; Michael Boeckh; Helen Y Chu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031211</td></tr><tr><td>myh73pe6</td><td>b696d208705fcb1925693c5f0d118733bb557ea6</td><td>medrxiv</td><td>Exploring diseases/traits and blood proteins causally related to expression of ACE2, the putative receptor of 2019-nCov: A Mendelian Randomization analysis</td><td>10.1101/2020.03.04.20031237</td><td>null</td><td>null</td><td>medrvix</td><td>The novel coronavirus 2019-nCoV has caused major outbreaks in many parts of the world. A better understanding of the pathophysiology of COVID-19 is urgently needed. Clinically, it is important to identify who may be susceptible to infection and identify treatments for the disease. There is good evidence that ACE2 is a receptor for 2019-nCoV, and studies also suggested that high expression of ACE2 may increase susceptibility to infection. Here we conducted a phenome-wide Mendelian randomization (MR) study to prioritize diseases/traits and blood proteins that may be causally linked to ACE2 expression in the lung. Expression data was based on GTEx. We also explored drug candidates whose targets overlapped with the top-ranked proteins in MR analysis, as these drugs could potentially alter ACE2 expression and may be clinically relevant. Notably, MR is much less vulnerable to confounding and reverse causality compared to observational studies. The most consistent finding was a tentative causal association between diabetes-related traits and increased ACE2 expression. Based on one of the largest GWAS on type II diabetes (T2DM) to date (N=898,130), we found that T2DM is causally linked to raised ACE2 expression (beta=0.1835, 95% CI 0.0853-0.2817; p=2.49E-4; GSMR method). Significant associations (at nominal level; p<0.05) was also observed across multiple datasets, with different analytic methods, and for both type I and II diabetes. Other diseases/traits having nominal significant associations with increased ACE2 included inflammatory bowel disease, (ER+) breast and lung cancers, asthma, smoking and elevated ALT, among others. We also uncovered a number of plasma/serum proteins potentially linked to altered ACE2 expression, and the top enriched pathways included cytokine-cytokine-receptor interaction, VEGF signaling, JAK-STAT signaling etc. We also explored drugs that target some of the top-ranked proteins in the MR analysis. In conclusion, the current MR analysis reveals diseases/traits and blood proteins that may causally affect ACE2 expression, which in turn may influence susceptibility to the infection. The proteome-wide MR analysis may shed light on the molecular mechanisms underlying ACE2 expression, and may help guide drug repositioning in the future. Nevertheless, we stress that further studies are required to verify our findings due to various limitations and the exploratory nature of some analyses.</td><td>2020-03-08</td><td>Shitao Rao; Alexandria Lau; Hon-Cheong So</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031237</td></tr><tr><td>ufcvecwo</td><td>374a5295716649a80dca87097ad9ac902d344db6</td><td>medrxiv</td><td>Outcome reporting from protocols of clinical trials of Coronavirus Disease 2019 (COVID-19): a review</td><td>10.1101/2020.03.04.20031401</td><td>null</td><td>null</td><td>medrvix</td><td>Objectives: To examine heterogeneity of outcomes in protocols of clinical trials of Coronavirus Disease 2019 (COVID-19) and to identify outcomes for prioritization in developing a core outcome set (COS) in this field. Design: This study is a review. Data sources: Databases of ICMJE-accepted clinical trial registry platform were searched on February 14, 2020. Eligibility Criteria: Randomized controlled trials (RCTs) and non-RCTs of COVID-19 were considered.Conditions of patients include common type, severe type or critical type. Interventions include traditional Chinese medicine (TCM) and Western medicine. We excluded trials that for discharged patients, psychological intervention and complications of COVID-19. Data extraction and synthesis: The general information and outcomes, outcome measurement instruments and measurement times were extracted. The results were analysed by descriptive analysis. Results: 19 registry platforms were searched. A total of 97 protocols were included from 160 protocols. For protocols of TCM clinical trials, 76 outcomes from 16 outcome domains were reported, and almost half (34/76, 44.74%) of outcomes were reported only once; the most frequently reported outcome was time of SARS-CoV-2 RNA turns to negative. 27 (27/76, 35.53%) outcomes were provided one or more outcome measurement instruments. 10 outcomes were provided one or more measurement time frame. For protocols of western medicine clinical trials, 126 outcomes from 17 outcome domains were reported; almost half (62/126, 49.21%) of outcomes were reported only once; the most frequently reported outcome was proportion of patients with negative SARS-CoV-2. 27 outcomes were provided one or more outcome measurement instruments. 40 (40/126, 31.75%) outcomes were provided one or more measurement time frame. Conclusion: Outcome reporting in protocols of clinical trials of COVID-19 is inconsistent. Thus, developing a core outcome set is necessary. Keywords: Outcomes; clinical trials, COVID-19; review.</td><td>2020-03-08</td><td>Ruijin Qiu; Xuxu Wei; Mengzhu Zhao; Changming Zhong; Chen Zhao; Jiayuan Hu; Min Li; Ya Huang; Songjie Han; Tianmai He; Jing Chen; Hongcai Shang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.20031401</td></tr><tr><td>4ihv80au</td><td>7bdceb3b947ec934199d0c97fb9ec5a0e2bac776</td><td>biorxiv</td><td>Strong evolutionary convergence of receptor-binding protein spike between COVID-19 and SARS-related coronaviruses</td><td>10.1101/2020.03.04.975995</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractCoronavirus Disease 2019 (COVID-19) and severe acute respiratory syndrome (SARS)-related coronaviruses (e.g., 2019-nCoV and SARS-CoV) are phylogenetically distantly related, but both are capable of infecting human hosts via the same receptor, angiotensin-converting enzyme 2, and cause similar clinical and pathological features, suggesting their phenotypic convergence. Yet, the molecular basis that underlies their phenotypic convergence remains unknown. Here, we used a recently developed molecular phyloecological approach to examine the molecular basis leading to their phenotypic convergence. Our genome-level analyses show that the spike protein, which is responsible for receptor binding, has undergone significant Darwinian selection along the branches related to 2019-nCoV and SARS-CoV. Further examination shows an unusually high proportion of evolutionary convergent amino acid sites in the receptor binding domain (RBD) of the spike protein between COVID-19 and SARS-related CoV clades, leading to the phylogenetic uniting of their RBD protein sequences. In addition to the spike protein, we also find the evolutionary convergence of its partner protein, ORF3a, suggesting their possible co-evolutionary convergence. Our results demonstrate a strong adaptive evolutionary convergence between COVID-19 and SARS-related CoV, possibly facilitating their adaptation to similar or identical receptors. Finally, it should be noted that many observed bat SARS-like CoVs that have an evolutionary convergent RBD sequence with 2019-nCoV and SARS-CoV may be pre-adapted to human host receptor ACE2, and hence would be potential new coronavirus sources to infect humans in the future.</td><td>2020-03-04</td><td>Yonghua Wu</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.975995</td></tr><tr><td>wynyrumi</td><td>6726b273f345838661eaa39e8adb5df4e1815a9a</td><td>biorxiv</td><td>In silico study of the spike protein from SARS-CoV-2 interaction with ACE2: similarity with SARS-CoV, hot-spot analysis and effect of the receptor polymorphism</td><td>10.1101/2020.03.04.976027</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe spread of the COVID-19 caused by the SARS-CoV-2 outbreak has been growing since its first identification in December 2019. The publishing of the first SARS-CoV-2 genome made a valuable source of data to study the details about its phylogeny, evolution, and interaction with the host. Protein-protein binding assays have confirmed that Angiotensin-converting enzyme 2 (ACE2) is more likely to be the cell receptor via which the virus invades the host cell. In the present work, we provide an insight into the interaction of the viral spike Receptor Binding Domain (RBD) from different coronavirus isolates with host ACE2 protein. We used homology-based protein-protein docking, binding energy estimation, and decomposition, aiming to predict both qualitative and quantitative aspects of the interaction. Using in silico structural modelling, we brought additional evidence that the interface segment of the spike protein RBD might be acquired by SARS-CoV-2 via a complex evolutionary process rather than mutation accumulation. We also highlighted the relevance of Q493 and P499 amino acid residues of SARS-CoV-2 RBD for binding to hACE2 and maintaining the stability of the interface. Finally, we studied the impact of eight different variants located at the interaction surface of ACE2, on the complex formation with SARS-CoV-2 RBD. We found that none of them is likely to disrupt the interaction with the viral RBD of SARS-CoV-2.</td><td>2020-03-07</td><td>Houcemeddine Othman; Zied Bouslama; Jean-Tristan Brandenburg; Jorge da Rocha; Yosr Hamdi; Kais Ghedira; Najet-Srairi Abid; Scott Hazelhurst</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.976027</td></tr><tr><td>erdity4m</td><td>5cd48b8ab4ef66ca582df97a293d22a5088fa568</td><td>biorxiv</td><td>Cryo-electron microscopy structure of the SADS-CoV spike glycoprotein provides insights into an evolution of unique coronavirus spike proteins</td><td>10.1101/2020.03.04.976258</td><td>null</td><td>null</td><td>biorxiv</td><td>AbstractThe current outbreak of Coronavirus Disease 2019 (COVID-19) by a novel betacoronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has aroused great public health concern. Coronavirus has a history of causing epidemics in human and animals. In 2017 an outbreak in piglets by a novel coronavirus was emerged designated as swine acute diarrhea syndrome coronavirus (SADS-CoV) which is originated from the same genus of horseshoe bats (Rhinolophus) as Severe Acute Respiratory Syndrome CoV (SARS-CoV) having a broad species tropism. In addition to human cells, it can also infect cell lines from diverse species. Coronavirus host range is determined by its spike glycoprotein (S). Given the importance of S protein in viral entry to cells and host immune responses, here we report the cryo-EM structure of the SADS-CoV S in the prefusion conformation at a resolution of 3.55 Å. Our study reveals that SADS-CoV S structure takes an intra-subunit quaternary packing mode where the NTD and CTD from the same subunit pack together by facing each other. The comparison of NTD and CTD with that of the other four genera suggests the evolutionary process of the SADS-CoV S. Moreover, SADS-CoV S has several characteristic structural features, such as more compact architecture of S trimer, and masking of epitopes by glycan shielding, which may facilitate viral immune evasion. These data provide new insights into the evolutionary relationships of SADS-CoV S and would extend our understanding of structural and functional diversity, which will facilitate to vaccine development.</td><td>2020-03-07</td><td>Hongxin Guan; Youwang Wang; Abdullah F.U.H. Saeed; Jinyu Li; Syed Sajid Jan; Vanja Perčulija; Yu Li; Ping Zhu; Songying Ouyang</td><td>null</td><td>null</td><td>null</td><td>True</td><td>biorxiv_medrxiv</td><td>https://doi.org/10.1101/2020.03.04.976258</td></tr></tbody></table>Showing the first 542 rows.</div>"]}}],"execution_count":0},{"cell_type":"markdown","source":["--> 3. Extract the underlying RDD."],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"00433d9d-6fb6-4b5e-abf3-64d85a9a2e2d"}}},{"cell_type":"code","source":["rdd_cord = df_cord.rdd\n\nrdd_cord.take(2)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"fb85ccdc-4af2-46f1-9b79-833bc23587f3"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">Out[16]: [Row(cord_uid=&#39;vho70jcx&#39;, sha=&#39;f056da9c64fbf00a4645ae326e8a4339d015d155&#39;, source_x=&#39;biorxiv&#39;, title=&#39;SIANN: Strain Identification by Alignment to Near Neighbors&#39;, doi=&#39;10.1101/001727&#39;, pmcid=None, pubmed_id=None, license=&#39;biorxiv&#39;, abstract=&#39;Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.&#39;, publish_time=&#39;2014-01-10&#39;, authors=&#39;Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy&#39;, journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text=&#39;True&#39;, full_text_file=&#39;biorxiv_medrxiv&#39;, url=&#39;https://doi.org/10.1101/001727&#39;),\n Row(cord_uid=&#39;i9tbix2v&#39;, sha=&#39;daf32e013d325a6feb80e83d15aabc64a48fae33&#39;, source_x=&#39;biorxiv&#39;, title=&#39;Spatial epidemiology of networked metapopulation: An overview&#39;, doi=&#39;10.1101/003889&#39;, pmcid=None, pubmed_id=None, license=&#39;biorxiv&#39;, abstract=&#39;An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.&#39;, publish_time=&#39;2014-06-04&#39;, authors=&#39;Lin WANG; Xiang Li&#39;, journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text=&#39;True&#39;, full_text_file=&#39;biorxiv_medrxiv&#39;, url=&#39;https://doi.org/10.1101/003889&#39;)]</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">Out[16]: [Row(cord_uid=&#39;vho70jcx&#39;, sha=&#39;f056da9c64fbf00a4645ae326e8a4339d015d155&#39;, source_x=&#39;biorxiv&#39;, title=&#39;SIANN: Strain Identification by Alignment to Near Neighbors&#39;, doi=&#39;10.1101/001727&#39;, pmcid=None, pubmed_id=None, license=&#39;biorxiv&#39;, abstract=&#39;Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.&#39;, publish_time=&#39;2014-01-10&#39;, authors=&#39;Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy&#39;, journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text=&#39;True&#39;, full_text_file=&#39;biorxiv_medrxiv&#39;, url=&#39;https://doi.org/10.1101/001727&#39;),\n Row(cord_uid=&#39;i9tbix2v&#39;, sha=&#39;daf32e013d325a6feb80e83d15aabc64a48fae33&#39;, source_x=&#39;biorxiv&#39;, title=&#39;Spatial epidemiology of networked metapopulation: An overview&#39;, doi=&#39;10.1101/003889&#39;, pmcid=None, pubmed_id=None, license=&#39;biorxiv&#39;, abstract=&#39;An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.&#39;, publish_time=&#39;2014-06-04&#39;, authors=&#39;Lin WANG; Xiang Li&#39;, journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text=&#39;True&#39;, full_text_file=&#39;biorxiv_medrxiv&#39;, url=&#39;https://doi.org/10.1101/003889&#39;)]</div>"]}}],"execution_count":0},{"cell_type":"markdown","source":["--> 4. Form a new RDD which contains the pair (cord id, abstract)."],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"e4f1b148-6180-4654-a330-64f7f1c237f2"}}},{"cell_type":"code","source":["slic_rdd = rdd_cord.map(lambda row: (row.cord_uid, row.abstract))\n\nslic_rdd.take(2)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"532e9957-81a2-46dc-83b2-061c52083704"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">Out[22]: [(&#39;vho70jcx&#39;,\n  &#39;Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.&#39;),\n (&#39;i9tbix2v&#39;,\n  &#39;An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.&#39;)]</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">Out[22]: [(&#39;vho70jcx&#39;,\n  &#39;Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.&#39;),\n (&#39;i9tbix2v&#39;,\n  &#39;An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.&#39;)]</div>"]}}],"execution_count":0},{"cell_type":"markdown","source":["--> 5. Convert the RDD you created in Step 4 into an RDD which consists of pairs (cord id, list of words in abstract)."],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"0d5ca4b3-d8f1-4fbb-8cce-ba7219018332"}}},{"cell_type":"code","source":["slic_rdd = slic_rdd.filter(lambda line: line[1] is not None).map(lambda line: (line[0], line[1].split()))\n\nslic_rdd.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"3ab2e039-7f6e-4b37-a8f1-8bc5182092df"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">Out[23]: [(&#39;vho70jcx&#39;,\n  [&#39;Next-generation&#39;,\n   &#39;sequencing&#39;,\n   &#39;is&#39;,\n   &#39;increasingly&#39;,\n   &#39;being&#39;,\n   &#39;used&#39;,\n   &#39;to&#39;,\n   &#39;study&#39;,\n   &#39;samples&#39;,\n   &#39;composed&#39;,\n   &#39;of&#39;,\n   &#39;mixtures&#39;,\n   &#39;of&#39;,\n   &#39;organisms,&#39;,\n   &#39;such&#39;,\n   &#39;as&#39;,\n   &#39;in&#39;,\n   &#39;clinical&#39;,\n   &#39;applications&#39;,\n   &#39;where&#39;,\n   &#39;the&#39;,\n   &#39;presence&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#39;pathogen&#39;,\n   &#39;at&#39;,\n   &#39;very&#39;,\n   &#39;low&#39;,\n   &#39;abundance&#39;,\n   &#39;may&#39;,\n   &#39;be&#39;,\n   &#39;highly&#39;,\n   &#39;important.&#39;,\n   &#39;We&#39;,\n   &#39;present&#39;,\n   &#39;an&#39;,\n   &#39;analytical&#39;,\n   &#39;method&#39;,\n   &#39;(SIANN:&#39;,\n   &#39;Strain&#39;,\n   &#39;Identification&#39;,\n   &#39;by&#39;,\n   &#39;Alignment&#39;,\n   &#39;to&#39;,\n   &#39;Near&#39;,\n   &#39;Neighbors)&#39;,\n   &#39;specifically&#39;,\n   &#39;designed&#39;,\n   &#39;to&#39;,\n   &#39;rapidly&#39;,\n   &#39;detect&#39;,\n   &#39;a&#39;,\n   &#39;set&#39;,\n   &#39;of&#39;,\n   &#39;target&#39;,\n   &#39;organisms&#39;,\n   &#39;in&#39;,\n   &#39;mixed&#39;,\n   &#39;samples&#39;,\n   &#39;that&#39;,\n   &#39;achieves&#39;,\n   &#39;a&#39;,\n   &#39;high&#39;,\n   &#39;degree&#39;,\n   &#39;of&#39;,\n   &#39;species-&#39;,\n   &#39;and&#39;,\n   &#39;strain-specificity&#39;,\n   &#39;by&#39;,\n   &#39;aligning&#39;,\n   &#39;short&#39;,\n   &#39;sequence&#39;,\n   &#39;reads&#39;,\n   &#39;to&#39;,\n   &#39;the&#39;,\n   &#39;genomes&#39;,\n   &#39;of&#39;,\n   &#39;near&#39;,\n   &#39;neighbor&#39;,\n   &#39;organisms,&#39;,\n   &#39;as&#39;,\n   &#39;well&#39;,\n   &#39;as&#39;,\n   &#39;that&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;target.&#39;,\n   &#39;Empirical&#39;,\n   &#39;benchmarking&#39;,\n   &#39;alongside&#39;,\n   &#39;the&#39;,\n   &#39;current&#39;,\n   &#39;state-of-the-art&#39;,\n   &#39;methods&#39;,\n   &#39;shows&#39;,\n   &#39;an&#39;,\n   &#39;extremely&#39;,\n   &#39;high&#39;,\n   &#39;Positive&#39;,\n   &#39;Predictive&#39;,\n   &#39;Value,&#39;,\n   &#39;even&#39;,\n   &#39;at&#39;,\n   &#39;very&#39;,\n   &#39;low&#39;,\n   &#39;abundances&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;target&#39;,\n   &#39;organism&#39;,\n   &#39;in&#39;,\n   &#39;a&#39;,\n   &#39;mixed&#39;,\n   &#39;sample.&#39;,\n   &#39;SIANN&#39;,\n   &#39;is&#39;,\n   &#39;available&#39;,\n   &#39;as&#39;,\n   &#39;an&#39;,\n   &#39;Illumina&#39;,\n   &#39;BaseSpace&#39;,\n   &#39;app,&#39;,\n   &#39;as&#39;,\n   &#39;well&#39;,\n   &#39;as&#39;,\n   &#39;through&#39;,\n   &#39;Signature&#39;,\n   &#39;Science,&#39;,\n   &#39;LLC.&#39;,\n   &#39;SIANN&#39;,\n   &#39;results&#39;,\n   &#39;are&#39;,\n   &#39;presented&#39;,\n   &#39;in&#39;,\n   &#39;a&#39;,\n   &#39;streamlined&#39;,\n   &#39;report&#39;,\n   &#39;designed&#39;,\n   &#39;to&#39;,\n   &#39;be&#39;,\n   &#39;comprehensible&#39;,\n   &#39;to&#39;,\n   &#39;the&#39;,\n   &#39;non-specialist&#39;,\n   &#39;user,&#39;,\n   &#39;providing&#39;,\n   &#39;a&#39;,\n   &#39;powerful&#39;,\n   &#39;tool&#39;,\n   &#39;for&#39;,\n   &#39;rapid&#39;,\n   &#39;species&#39;,\n   &#39;detection&#39;,\n   &#39;in&#39;,\n   &#39;a&#39;,\n   &#39;mixed&#39;,\n   &#39;sample.&#39;,\n   &#39;By&#39;,\n   &#39;focusing&#39;,\n   &#39;on&#39;,\n   &#39;a&#39;,\n   &#39;set&#39;,\n   &#39;of&#39;,\n   &#39;(customizable)&#39;,\n   &#39;target&#39;,\n   &#39;organisms&#39;,\n   &#39;and&#39;,\n   &#39;their&#39;,\n   &#39;near&#39;,\n   &#39;neighbors,&#39;,\n   &#39;SIANN&#39;,\n   &#39;can&#39;,\n   &#39;operate&#39;,\n   &#39;quickly&#39;,\n   &#39;and&#39;,\n   &#39;with&#39;,\n   &#39;low&#39;,\n   &#39;computational&#39;,\n   &#39;requirements&#39;,\n   &#39;while&#39;,\n   &#39;delivering&#39;,\n   &#39;highly&#39;,\n   &#39;accurate&#39;,\n   &#39;results.&#39;]),\n (&#39;i9tbix2v&#39;,\n  [&#39;An&#39;,\n   &#39;emerging&#39;,\n   &#39;disease&#39;,\n   &#39;is&#39;,\n   &#39;one&#39;,\n   &#39;infectious&#39;,\n   &#39;epidemic&#39;,\n   &#39;caused&#39;,\n   &#39;by&#39;,\n   &#39;a&#39;,\n   &#39;newly&#39;,\n   &#39;transmissible&#39;,\n   &#39;pathogen,&#39;,\n   &#39;which&#39;,\n   &#39;has&#39;,\n   &#39;either&#39;,\n   &#39;appeared&#39;,\n   &#39;for&#39;,\n   &#39;the&#39;,\n   &#39;first&#39;,\n   &#39;time&#39;,\n   &#39;or&#39;,\n   &#39;already&#39;,\n   &#39;existed&#39;,\n   &#39;in&#39;,\n   &#39;human&#39;,\n   &#39;populations,&#39;,\n   &#39;having&#39;,\n   &#39;the&#39;,\n   &#39;capacity&#39;,\n   &#39;to&#39;,\n   &#39;increase&#39;,\n   &#39;rapidly&#39;,\n   &#39;in&#39;,\n   &#39;incidence&#39;,\n   &#39;as&#39;,\n   &#39;well&#39;,\n   &#39;as&#39;,\n   &#39;geographic&#39;,\n   &#39;range.&#39;,\n   &#39;Adapting&#39;,\n   &#39;to&#39;,\n   &#39;human&#39;,\n   &#39;immune&#39;,\n   &#39;system,&#39;,\n   &#39;emerging&#39;,\n   &#39;diseases&#39;,\n   &#39;may&#39;,\n   &#39;trigger&#39;,\n   &#39;large-scale&#39;,\n   &#39;pandemic&#39;,\n   &#39;spreading,&#39;,\n   &#39;such&#39;,\n   &#39;as&#39;,\n   &#39;the&#39;,\n   &#39;transnational&#39;,\n   &#39;spreading&#39;,\n   &#39;of&#39;,\n   &#39;SARS,&#39;,\n   &#39;the&#39;,\n   &#39;global&#39;,\n   &#39;outbreak&#39;,\n   &#39;of&#39;,\n   &#39;A(H1N1),&#39;,\n   &#39;and&#39;,\n   &#39;the&#39;,\n   &#39;recent&#39;,\n   &#39;potential&#39;,\n   &#39;invasion&#39;,\n   &#39;of&#39;,\n   &#39;avian&#39;,\n   &#39;influenza&#39;,\n   &#39;A(H7N9).&#39;,\n   &#39;To&#39;,\n   &#39;study&#39;,\n   &#39;the&#39;,\n   &#39;dynamics&#39;,\n   &#39;mediating&#39;,\n   &#39;the&#39;,\n   &#39;transmission&#39;,\n   &#39;of&#39;,\n   &#39;emerging&#39;,\n   &#39;diseases,&#39;,\n   &#39;spatial&#39;,\n   &#39;epidemiology&#39;,\n   &#39;of&#39;,\n   &#39;networked&#39;,\n   &#39;metapopulation&#39;,\n   &#39;provides&#39;,\n   &#39;a&#39;,\n   &#39;valuable&#39;,\n   &#39;modeling&#39;,\n   &#39;framework,&#39;,\n   &#39;which&#39;,\n   &#39;takes&#39;,\n   &#39;spatially&#39;,\n   &#39;distributed&#39;,\n   &#39;factors&#39;,\n   &#39;into&#39;,\n   &#39;consideration.&#39;,\n   &#39;This&#39;,\n   &#39;review&#39;,\n   &#39;elaborates&#39;,\n   &#39;the&#39;,\n   &#39;latest&#39;,\n   &#39;progresses&#39;,\n   &#39;on&#39;,\n   &#39;the&#39;,\n   &#39;spatial&#39;,\n   &#39;metapopulation&#39;,\n   &#39;dynamics,&#39;,\n   &#39;discusses&#39;,\n   &#39;empirical&#39;,\n   &#39;and&#39;,\n   &#39;theoretical&#39;,\n   &#39;findings&#39;,\n   &#39;that&#39;,\n   &#39;verify&#39;,\n   &#39;the&#39;,\n   &#39;validity&#39;,\n   &#39;of&#39;,\n   &#39;networked&#39;,\n   &#39;metapopulations,&#39;,\n   &#39;and&#39;,\n   &#39;the&#39;,\n   &#39;application&#39;,\n   &#39;in&#39;,\n   &#39;evaluating&#39;,\n   &#39;the&#39;,\n   &#39;effectiveness&#39;,\n   &#39;of&#39;,\n   &#39;disease&#39;,\n   &#39;intervention&#39;,\n   &#39;strategies&#39;,\n   &#39;as&#39;,\n   &#39;well.&#39;]),\n (&#39;62gfisc6&#39;,\n  [&#39;Germline&#39;,\n   &#39;variation&#39;,\n   &#39;at&#39;,\n   &#39;immunoglobulin&#39;,\n   &#39;gene&#39;,\n   &#39;(IG)&#39;,\n   &#39;loci&#39;,\n   &#39;is&#39;,\n   &#39;critical&#39;,\n   &#39;for&#39;,\n   &#39;pathogen-mediated&#39;,\n   &#39;immunity,&#39;,\n   &#39;but&#39;,\n   &#39;establishing&#39;,\n   &#39;complete&#39;,\n   &#39;reference&#39;,\n   &#39;sequences&#39;,\n   &#39;in&#39;,\n   &#39;these&#39;,\n   &#39;regions&#39;,\n   &#39;is&#39;,\n   &#39;problematic&#39;,\n   &#39;because&#39;,\n   &#39;of&#39;,\n   &#39;segmental&#39;,\n   &#39;duplications&#39;,\n   &#39;and&#39;,\n   &#39;somatically&#39;,\n   &#39;rearranged&#39;,\n   &#39;source&#39;,\n   &#39;DNA.&#39;,\n   &#39;We&#39;,\n   &#39;sequenced&#39;,\n   &#39;BAC&#39;,\n   &#39;clones&#39;,\n   &#39;from&#39;,\n   &#39;the&#39;,\n   &#39;essentially&#39;,\n   &#39;haploid&#39;,\n   &#39;hydatidiform&#39;,\n   &#39;mole,&#39;,\n   &#39;CHM1,&#39;,\n   &#39;across&#39;,\n   &#39;the&#39;,\n   &#39;light&#39;,\n   &#39;chain&#39;,\n   &#39;IG&#39;,\n   &#39;loci,&#39;,\n   &#39;kappa&#39;,\n   &#39;(IGK)&#39;,\n   &#39;and&#39;,\n   &#39;lambda&#39;,\n   &#39;(IGL),&#39;,\n   &#39;creating&#39;,\n   &#39;single&#39;,\n   &#39;haplotype&#39;,\n   &#39;representations&#39;,\n   &#39;of&#39;,\n   &#39;these&#39;,\n   &#39;regions.&#39;,\n   &#39;The&#39;,\n   &#39;IGL&#39;,\n   &#39;haplotype&#39;,\n   &#39;is&#39;,\n   &#39;1.25Mb&#39;,\n   &#39;of&#39;,\n   &#39;contiguous&#39;,\n   &#39;sequence&#39;,\n   &#39;with&#39;,\n   &#39;four&#39;,\n   &#39;novel&#39;,\n   &#39;V&#39;,\n   &#39;gene&#39;,\n   &#39;and&#39;,\n   &#39;one&#39;,\n   &#39;novel&#39;,\n   &#39;C&#39;,\n   &#39;gene&#39;,\n   &#39;alleles&#39;,\n   &#39;and&#39;,\n   &#39;an&#39;,\n   &#39;11.9kbp&#39;,\n   &#39;insertion.&#39;,\n   &#39;The&#39;,\n   &#39;IGK&#39;,\n   &#39;haplotype&#39;,\n   &#39;consists&#39;,\n   &#39;of&#39;,\n   &#39;two&#39;,\n   &#39;644kbp&#39;,\n   &#39;proximal&#39;,\n   &#39;and&#39;,\n   &#39;466kbp&#39;,\n   &#39;distal&#39;,\n   &#39;contigs&#39;,\n   &#39;separated&#39;,\n   &#39;by&#39;,\n   &#39;a&#39;,\n   &#39;gap&#39;,\n   &#39;also&#39;,\n   &#39;present&#39;,\n   &#39;in&#39;,\n   &#39;the&#39;,\n   &#39;reference&#39;,\n   &#39;genome&#39;,\n   &#39;sequence.&#39;,\n   &#39;Our&#39;,\n   &#39;effort&#39;,\n   &#39;added&#39;,\n   &#39;an&#39;,\n   &#39;additional&#39;,\n   &#39;49kbp&#39;,\n   &#39;of&#39;,\n   &#39;unique&#39;,\n   &#39;sequence&#39;,\n   &#39;extending&#39;,\n   &#39;into&#39;,\n   &#39;this&#39;,\n   &#39;gap.&#39;,\n   &#39;The&#39;,\n   &#39;IGK&#39;,\n   &#39;haplotype&#39;,\n   &#39;contains&#39;,\n   &#39;six&#39;,\n   &#39;novel&#39;,\n   &#39;V&#39;,\n   &#39;gene&#39;,\n   &#39;and&#39;,\n   &#39;one&#39;,\n   &#39;novel&#39;,\n   &#39;J&#39;,\n   &#39;gene&#39;,\n   &#39;alleles&#39;,\n   &#39;and&#39;,\n   &#39;a&#39;,\n   &#39;16.7kbp&#39;,\n   &#39;region&#39;,\n   &#39;with&#39;,\n   &#39;increased&#39;,\n   &#39;sequence&#39;,\n   &#39;identity&#39;,\n   &#39;between&#39;,\n   &#39;the&#39;,\n   &#39;two&#39;,\n   &#39;IGK&#39;,\n   &#39;contigs,&#39;,\n   &#39;exhibiting&#39;,\n   &#39;signatures&#39;,\n   &#39;of&#39;,\n   &#39;interlocus&#39;,\n   &#39;gene&#39;,\n   &#39;conversion.&#39;,\n   &#39;Our&#39;,\n   &#39;data&#39;,\n   &#39;facilitated&#39;,\n   &#39;the&#39;,\n   &#39;first&#39;,\n   &#39;comparison&#39;,\n   &#39;of&#39;,\n   &#39;nucleotide&#39;,\n   &#39;diversity&#39;,\n   &#39;between&#39;,\n   &#39;the&#39;,\n   &#39;light&#39;,\n   &#39;and&#39;,\n   &#39;IG&#39;,\n   &#39;heavy&#39;,\n   &#39;(IGH)&#39;,\n   &#39;chain&#39;,\n   &#39;haplotypes&#39;,\n   &#39;within&#39;,\n   &#39;a&#39;,\n   &#39;single&#39;,\n   &#39;genome,&#39;,\n   &#39;revealing&#39;,\n   &#39;a&#39;,\n   &#39;three&#39;,\n   &#39;to&#39;,\n   &#39;six&#39;,\n   &#39;fold&#39;,\n   &#39;enrichment&#39;,\n   &#39;in&#39;,\n   &#39;the&#39;,\n   &#39;IGH&#39;,\n   &#39;locus,&#39;,\n   &#39;supporting&#39;,\n   &#39;the&#39;,\n   &#39;theory&#39;,\n   &#39;that&#39;,\n   &#39;the&#39;,\n   &#39;heavy&#39;,\n   &#39;chain&#39;,\n   &#39;may&#39;,\n   &#39;be&#39;,\n   &#39;more&#39;,\n   &#39;important&#39;,\n   &#39;in&#39;,\n   &#39;determining&#39;,\n   &#39;antigenic&#39;,\n   &#39;specificity.&#39;]),\n (&#39;058r9486&#39;,\n  [&#39;Deep&#39;,\n   &#39;sequencing&#39;,\n   &#39;of&#39;,\n   &#39;clinical&#39;,\n   &#39;samples&#39;,\n   &#39;is&#39;,\n   &#39;now&#39;,\n   &#39;an&#39;,\n   &#39;established&#39;,\n   &#39;tool&#39;,\n   &#39;for&#39;,\n   &#39;the&#39;,\n   &#39;detection&#39;,\n   &#39;of&#39;,\n   &#39;infectious&#39;,\n   &#39;pathogens,&#39;,\n   &#39;with&#39;,\n   &#39;direct&#39;,\n   &#39;medical&#39;,\n   &#39;applications.&#39;,\n   &#39;The&#39;,\n   &#39;large&#39;,\n   &#39;amount&#39;,\n   &#39;of&#39;,\n   &#39;data&#39;,\n   &#39;generated&#39;,\n   &#39;provides&#39;,\n   &#39;an&#39;,\n   &#39;opportunity&#39;,\n   &#39;to&#39;,\n   &#39;detect&#39;,\n   &#39;species&#39;,\n   &#39;even&#39;,\n   &#39;at&#39;,\n   &#39;very&#39;,\n   &#39;low&#39;,\n   &#39;levels,&#39;,\n   &#39;provided&#39;,\n   &#39;that&#39;,\n   &#39;computational&#39;,\n   &#39;tools&#39;,\n   &#39;can&#39;,\n   &#39;effectively&#39;,\n   &#39;interpret&#39;,\n   &#39;potentially&#39;,\n   &#39;complex&#39;,\n   &#39;metagenomic&#39;,\n   &#39;mixtures.&#39;,\n   &#39;Data&#39;,\n   &#39;interpretation&#39;,\n   &#39;is&#39;,\n   &#39;complicated&#39;,\n   &#39;by&#39;,\n   &#39;the&#39;,\n   &#39;fact&#39;,\n   &#39;that&#39;,\n   &#39;short&#39;,\n   &#39;sequencing&#39;,\n   &#39;reads&#39;,\n   &#39;can&#39;,\n   &#39;match&#39;,\n   &#39;multiple&#39;,\n   &#39;organisms&#39;,\n   &#39;and&#39;,\n   &#39;by&#39;,\n   &#39;the&#39;,\n   &#39;lack&#39;,\n   &#39;of&#39;,\n   &#39;completeness&#39;,\n   &#39;of&#39;,\n   &#39;existing&#39;,\n   &#39;databases,&#39;,\n   &#39;in&#39;,\n   &#39;particular&#39;,\n   &#39;for&#39;,\n   &#39;viral&#39;,\n   &#39;pathogens.&#39;,\n   &#39;This&#39;,\n   &#39;interpretation&#39;,\n   &#39;problem&#39;,\n   &#39;can&#39;,\n   &#39;be&#39;,\n   &#39;formulated&#39;,\n   &#39;statistically&#39;,\n   &#39;as&#39;,\n   &#39;a&#39;,\n   &#39;mixture&#39;,\n   &#39;model,&#39;,\n   &#39;where&#39;,\n   &#39;the&#39;,\n   &#39;species&#39;,\n   &#39;of&#39;,\n   &#39;origin&#39;,\n   &#39;of&#39;,\n   &#39;each&#39;,\n   &#39;read&#39;,\n   &#39;is&#39;,\n   &#39;missing,&#39;,\n   &#39;but&#39;,\n   &#39;the&#39;,\n   &#39;complete&#39;,\n   &#39;knowledge&#39;,\n   &#39;of&#39;,\n   &#39;all&#39;,\n   &#39;species&#39;,\n   &#39;present&#39;,\n   &#39;in&#39;,\n   &#39;the&#39;,\n   &#39;mixture&#39;,\n   &#39;helps&#39;,\n   &#39;with&#39;,\n   &#39;the&#39;,\n   &#39;individual&#39;,\n   &#39;reads&#39;,\n   &#39;assignment.&#39;,\n   &#39;Several&#39;,\n   &#39;analytical&#39;,\n   &#39;tools&#39;,\n   &#39;have&#39;,\n   &#39;been&#39;,\n   &#39;proposed&#39;,\n   &#39;to&#39;,\n   &#39;approximately&#39;,\n   &#39;solve&#39;,\n   &#39;this&#39;,\n   &#39;computational&#39;,\n   &#39;problem.&#39;,\n   &#39;Here,&#39;,\n   &#39;we&#39;,\n   &#39;show&#39;,\n   &#39;that&#39;,\n   &#39;the&#39;,\n   &#39;use&#39;,\n   &#39;of&#39;,\n   &#39;parallel&#39;,\n   &#39;Monte&#39;,\n   &#39;Carlo&#39;,\n   &#39;Markov&#39;,\n   &#39;chains&#39;,\n   &#39;(MCMC)&#39;,\n   &#39;for&#39;,\n   &#39;the&#39;,\n   &#39;exploration&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;species&#39;,\n   &#39;space&#39;,\n   &#39;enables&#39;,\n   &#39;the&#39;,\n   &#39;identification&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;set&#39;,\n   &#39;of&#39;,\n   &#39;species&#39;,\n   &#39;most&#39;,\n   &#39;likely&#39;,\n   &#39;to&#39;,\n   &#39;contribute&#39;,\n   &#39;to&#39;,\n   &#39;the&#39;,\n   &#39;mixture.&#39;,\n   &#39;The&#39;,\n   &#39;added&#39;,\n   &#39;accuracy&#39;,\n   &#39;comes&#39;,\n   &#39;at&#39;,\n   &#39;a&#39;,\n   &#39;cost&#39;,\n   &#39;of&#39;,\n   &#39;increased&#39;,\n   &#39;computation&#39;,\n   &#39;time.&#39;,\n   &#39;Our&#39;,\n   &#39;approach&#39;,\n   &#39;is&#39;,\n   &#39;useful&#39;,\n   &#39;for&#39;,\n   &#39;solving&#39;,\n   &#39;complex&#39;,\n   &#39;mixtures&#39;,\n   &#39;involving&#39;,\n   &#39;several&#39;,\n   &#39;related&#39;,\n   &#39;species.&#39;,\n   &#39;We&#39;,\n   &#39;designed&#39;,\n   &#39;our&#39;,\n   &#39;method&#39;,\n   &#39;specifically&#39;,\n   &#39;for&#39;,\n   &#39;the&#39;,\n   &#39;analysis&#39;,\n   &#39;of&#39;,\n   &#39;deep&#39;,\n   &#39;transcriptome&#39;,\n   &#39;sequencing&#39;,\n   &#39;datasets&#39;,\n   &#39;and&#39;,\n   &#39;with&#39;,\n   &#39;a&#39;,\n   &#39;particular&#39;,\n   &#39;focus&#39;,\n   &#39;on&#39;,\n   &#39;viral&#39;,\n   &#39;pathogen&#39;,\n   &#39;detection,&#39;,\n   &#39;but&#39;,\n   &#39;the&#39;,\n   &#39;principles&#39;,\n   &#39;are&#39;,\n   &#39;applicable&#39;,\n   &#39;more&#39;,\n   &#39;generally&#39;,\n   &#39;to&#39;,\n   &#39;all&#39;,\n   &#39;types&#39;,\n   &#39;of&#39;,\n   &#39;metagenomics&#39;,\n   &#39;mixtures.&#39;,\n   &#39;The&#39;,\n   &#39;work&#39;,\n   &#39;is&#39;,\n   &#39;implemented&#39;,\n   &#39;as&#39;,\n   &#39;a&#39;,\n   &#39;user&#39;,\n   &#39;friendly&#39;,\n   &#39;R&#39;,\n   &#39;package,&#39;,\n   &#39;available&#39;,\n   &#39;from&#39;,\n   &#39;CRAN:&#39;,\n   &#39;http://cran.r-project.org/web/packages/metaMix&#39;]),\n (&#39;wich35l7&#39;,\n  [&#39;Developing&#39;,\n   &#39;methods&#39;,\n   &#39;to&#39;,\n   &#39;reconstruct&#39;,\n   &#39;transmission&#39;,\n   &#39;histories&#39;,\n   &#39;for&#39;,\n   &#39;viral&#39;,\n   &#39;outbreaks&#39;,\n   &#39;could&#39;,\n   &#39;provide&#39;,\n   &#39;critical&#39;,\n   &#39;information&#39;,\n   &#39;to&#39;,\n   &#39;support&#39;,\n   &#39;locating&#39;,\n   &#39;sources&#39;,\n   &#39;of&#39;,\n   &#39;disease&#39;,\n   &#39;transmission.&#39;,\n   &#39;Phylogenetic&#39;,\n   &#39;methods&#39;,\n   &#39;used&#39;,\n   &#39;to&#39;,\n   &#39;measure&#39;,\n   &#39;the&#39;,\n   &#39;degree&#39;,\n   &#39;of&#39;,\n   &#39;relatedness&#39;,\n   &#39;among&#39;,\n   &#39;sequenced&#39;,\n   &#39;viral&#39;,\n   &#39;samples&#39;,\n   &#39;have&#39;,\n   &#39;proven&#39;,\n   &#39;useful&#39;,\n   &#39;in&#39;,\n   &#39;identifying&#39;,\n   &#39;potential&#39;,\n   &#39;outbreak&#39;,\n   &#39;sources.&#39;,\n   &#39;The&#39;,\n   &#39;complex&#39;,\n   &#39;nature&#39;,\n   &#39;of&#39;,\n   &#39;infectious&#39;,\n   &#39;disease,&#39;,\n   &#39;however,&#39;,\n   &#39;makes&#39;,\n   &#39;it&#39;,\n   &#39;difficult&#39;,\n   &#39;to&#39;,\n   &#39;assign&#39;,\n   &#39;a&#39;,\n   &#39;rigorously&#39;,\n   &#39;defined&#39;,\n   &#39;quantitative&#39;,\n   &#39;confidence&#39;,\n   &#39;value&#39;,\n   &#39;assessing&#39;,\n   &#39;the&#39;,\n   &#39;likelihood&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#39;true&#39;,\n   &#39;direct&#39;,\n   &#39;transmission&#39;,\n   &#39;event&#39;,\n   &#39;using&#39;,\n   &#39;genetic&#39;,\n   &#39;data&#39;,\n   &#39;alone.&#39;,\n   &#39;A&#39;,\n   &#39;new&#39;,\n   &#39;method&#39;,\n   &#39;is&#39;,\n   &#39;presented&#39;,\n   &#39;to&#39;,\n   &#39;calculate&#39;,\n   &#39;a&#39;,\n   &#39;confidence&#39;,\n   &#39;value&#39;,\n   &#39;assessing&#39;,\n   &#39;the&#39;,\n   &#39;likelihood&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#39;transmission&#39;,\n   &#39;event&#39;,\n   &#39;using&#39;,\n   &#39;both&#39;,\n   &#39;phylogenetic&#39;,\n   &#39;inference&#39;,\n   &#39;and&#39;,\n   &#39;limited&#39;,\n   &#39;knowledge&#39;,\n   &#39;of&#39;,\n   &#39;incubation&#39;,\n   &#39;and&#39;,\n   &#39;infectious&#39;,\n   &#39;duration&#39;,\n   &#39;times.&#39;,\n   &#39;The&#39;,\n   &#39;method&#39;,\n   &#39;is&#39;,\n   &#39;applied&#39;,\n   &#39;to&#39;,\n   &#39;simulations&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#39;foot&#39;,\n   &#39;and&#39;,\n   &#39;mouth&#39;,\n   &#39;disease&#39;,\n   &#39;(FMD)&#39;,\n   &#39;outbreak&#39;,\n   &#39;to&#39;,\n   &#39;demonstrate&#39;,\n   &#39;how&#39;,\n   &#39;the&#39;,\n   &#39;combination&#39;,\n   &#39;of&#39;,\n   &#39;both&#39;,\n   &#39;phylogenetic&#39;,\n   &#39;and&#39;,\n   &#39;epidemiology&#39;,\n   &#39;data&#39;,\n   &#39;can&#39;,\n   &#39;be&#39;,\n   &#39;used&#39;,\n   &#39;to&#39;,\n   &#39;strengthen&#39;,\n   &#39;the&#39;,\n   &#39;assessment&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;likelihood&#39;,\n   &#39;of&#39;,\n   &#39;direct&#39;,\n   &#39;transmission&#39;,\n   &#39;over&#39;,\n   &#39;methods&#39;,\n   &#39;using&#39;,\n   &#39;just&#39;,\n   &#39;phylogenetic&#39;,\n   &#39;data&#39;,\n   &#39;or&#39;,\n   &#39;infection&#39;,\n   &#39;timing&#39;,\n   &#39;data&#39;,\n   &#39;alone.&#39;,\n   &#39;The&#39;,\n   &#39;method&#39;,\n   &#39;is&#39;,\n   &#39;applied&#39;,\n   &#39;to&#39;,\n   &#39;a&#39;,\n   &#39;previous&#39;,\n   &#39;FMD&#39;,\n   &#39;outbreak&#39;,\n   &#39;to&#39;,\n   &#39;identify&#39;,\n   &#39;areas&#39;,\n   &#39;where&#39;,\n   &#39;over&#39;,\n   &#39;confidence&#39;,\n   &#39;in&#39;,\n   &#39;previously&#39;,\n   &#39;inferred&#39;,\n   &#39;direct&#39;,\n   &#39;transmission&#39;,\n   &#39;may&#39;,\n   &#39;exist.&#39;,\n   &#39;Combining&#39;,\n   &#39;knowledge&#39;,\n   &#39;from&#39;,\n   &#39;viral&#39;,\n   &#39;evolution&#39;,\n   &#39;and&#39;,\n   &#39;epidemiology&#39;,\n   &#39;within&#39;,\n   &#39;a&#39;,\n   &#39;single&#39;,\n   &#39;integrated&#39;,\n   &#39;transmission&#39;,\n   &#39;inference&#39;,\n   &#39;framework&#39;,\n   &#39;is&#39;,\n   &#39;an&#39;,\n   &#39;important&#39;,\n   &#39;approach&#39;,\n   &#39;to&#39;,\n   &#39;assess&#39;,\n   &#39;the&#39;,\n   &#39;potential&#39;,\n   &#39;likelihood&#39;,\n   &#39;of&#39;,\n   &#39;transmission&#39;,\n   &#39;events&#39;,\n   &#39;and&#39;,\n   &#39;makes&#39;,\n   &#39;clear&#39;,\n   &#39;how&#39;,\n   &#39;specific&#39;,\n   &#39;features&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#34;virus&#39;&#34;,\n   &#39;spread&#39;,\n   &#39;through&#39;,\n   &#39;the&#39;,\n   &#39;course&#39;,\n   &#39;of&#39;,\n   &#39;an&#39;,\n   &#39;outbreak&#39;,\n   &#39;will&#39;,\n   &#39;directly&#39;,\n   &#39;determine&#39;,\n   &#39;the&#39;,\n   &#39;potential&#39;,\n   &#39;for&#39;,\n   &#39;confidence&#39;,\n   &#39;in&#39;,\n   &#39;inferred&#39;,\n   &#39;host&#39;,\n   &#39;transmission&#39;,\n   &#39;links.&#39;])]</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">Out[23]: [(&#39;vho70jcx&#39;,\n  [&#39;Next-generation&#39;,\n   &#39;sequencing&#39;,\n   &#39;is&#39;,\n   &#39;increasingly&#39;,\n   &#39;being&#39;,\n   &#39;used&#39;,\n   &#39;to&#39;,\n   &#39;study&#39;,\n   &#39;samples&#39;,\n   &#39;composed&#39;,\n   &#39;of&#39;,\n   &#39;mixtures&#39;,\n   &#39;of&#39;,\n   &#39;organisms,&#39;,\n   &#39;such&#39;,\n   &#39;as&#39;,\n   &#39;in&#39;,\n   &#39;clinical&#39;,\n   &#39;applications&#39;,\n   &#39;where&#39;,\n   &#39;the&#39;,\n   &#39;presence&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#39;pathogen&#39;,\n   &#39;at&#39;,\n   &#39;very&#39;,\n   &#39;low&#39;,\n   &#39;abundance&#39;,\n   &#39;may&#39;,\n   &#39;be&#39;,\n   &#39;highly&#39;,\n   &#39;important.&#39;,\n   &#39;We&#39;,\n   &#39;present&#39;,\n   &#39;an&#39;,\n   &#39;analytical&#39;,\n   &#39;method&#39;,\n   &#39;(SIANN:&#39;,\n   &#39;Strain&#39;,\n   &#39;Identification&#39;,\n   &#39;by&#39;,\n   &#39;Alignment&#39;,\n   &#39;to&#39;,\n   &#39;Near&#39;,\n   &#39;Neighbors)&#39;,\n   &#39;specifically&#39;,\n   &#39;designed&#39;,\n   &#39;to&#39;,\n   &#39;rapidly&#39;,\n   &#39;detect&#39;,\n   &#39;a&#39;,\n   &#39;set&#39;,\n   &#39;of&#39;,\n   &#39;target&#39;,\n   &#39;organisms&#39;,\n   &#39;in&#39;,\n   &#39;mixed&#39;,\n   &#39;samples&#39;,\n   &#39;that&#39;,\n   &#39;achieves&#39;,\n   &#39;a&#39;,\n   &#39;high&#39;,\n   &#39;degree&#39;,\n   &#39;of&#39;,\n   &#39;species-&#39;,\n   &#39;and&#39;,\n   &#39;strain-specificity&#39;,\n   &#39;by&#39;,\n   &#39;aligning&#39;,\n   &#39;short&#39;,\n   &#39;sequence&#39;,\n   &#39;reads&#39;,\n   &#39;to&#39;,\n   &#39;the&#39;,\n   &#39;genomes&#39;,\n   &#39;of&#39;,\n   &#39;near&#39;,\n   &#39;neighbor&#39;,\n   &#39;organisms,&#39;,\n   &#39;as&#39;,\n   &#39;well&#39;,\n   &#39;as&#39;,\n   &#39;that&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;target.&#39;,\n   &#39;Empirical&#39;,\n   &#39;benchmarking&#39;,\n   &#39;alongside&#39;,\n   &#39;the&#39;,\n   &#39;current&#39;,\n   &#39;state-of-the-art&#39;,\n   &#39;methods&#39;,\n   &#39;shows&#39;,\n   &#39;an&#39;,\n   &#39;extremely&#39;,\n   &#39;high&#39;,\n   &#39;Positive&#39;,\n   &#39;Predictive&#39;,\n   &#39;Value,&#39;,\n   &#39;even&#39;,\n   &#39;at&#39;,\n   &#39;very&#39;,\n   &#39;low&#39;,\n   &#39;abundances&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;target&#39;,\n   &#39;organism&#39;,\n   &#39;in&#39;,\n   &#39;a&#39;,\n   &#39;mixed&#39;,\n   &#39;sample.&#39;,\n   &#39;SIANN&#39;,\n   &#39;is&#39;,\n   &#39;available&#39;,\n   &#39;as&#39;,\n   &#39;an&#39;,\n   &#39;Illumina&#39;,\n   &#39;BaseSpace&#39;,\n   &#39;app,&#39;,\n   &#39;as&#39;,\n   &#39;well&#39;,\n   &#39;as&#39;,\n   &#39;through&#39;,\n   &#39;Signature&#39;,\n   &#39;Science,&#39;,\n   &#39;LLC.&#39;,\n   &#39;SIANN&#39;,\n   &#39;results&#39;,\n   &#39;are&#39;,\n   &#39;presented&#39;,\n   &#39;in&#39;,\n   &#39;a&#39;,\n   &#39;streamlined&#39;,\n   &#39;report&#39;,\n   &#39;designed&#39;,\n   &#39;to&#39;,\n   &#39;be&#39;,\n   &#39;comprehensible&#39;,\n   &#39;to&#39;,\n   &#39;the&#39;,\n   &#39;non-specialist&#39;,\n   &#39;user,&#39;,\n   &#39;providing&#39;,\n   &#39;a&#39;,\n   &#39;powerful&#39;,\n   &#39;tool&#39;,\n   &#39;for&#39;,\n   &#39;rapid&#39;,\n   &#39;species&#39;,\n   &#39;detection&#39;,\n   &#39;in&#39;,\n   &#39;a&#39;,\n   &#39;mixed&#39;,\n   &#39;sample.&#39;,\n   &#39;By&#39;,\n   &#39;focusing&#39;,\n   &#39;on&#39;,\n   &#39;a&#39;,\n   &#39;set&#39;,\n   &#39;of&#39;,\n   &#39;(customizable)&#39;,\n   &#39;target&#39;,\n   &#39;organisms&#39;,\n   &#39;and&#39;,\n   &#39;their&#39;,\n   &#39;near&#39;,\n   &#39;neighbors,&#39;,\n   &#39;SIANN&#39;,\n   &#39;can&#39;,\n   &#39;operate&#39;,\n   &#39;quickly&#39;,\n   &#39;and&#39;,\n   &#39;with&#39;,\n   &#39;low&#39;,\n   &#39;computational&#39;,\n   &#39;requirements&#39;,\n   &#39;while&#39;,\n   &#39;delivering&#39;,\n   &#39;highly&#39;,\n   &#39;accurate&#39;,\n   &#39;results.&#39;]),\n (&#39;i9tbix2v&#39;,\n  [&#39;An&#39;,\n   &#39;emerging&#39;,\n   &#39;disease&#39;,\n   &#39;is&#39;,\n   &#39;one&#39;,\n   &#39;infectious&#39;,\n   &#39;epidemic&#39;,\n   &#39;caused&#39;,\n   &#39;by&#39;,\n   &#39;a&#39;,\n   &#39;newly&#39;,\n   &#39;transmissible&#39;,\n   &#39;pathogen,&#39;,\n   &#39;which&#39;,\n   &#39;has&#39;,\n   &#39;either&#39;,\n   &#39;appeared&#39;,\n   &#39;for&#39;,\n   &#39;the&#39;,\n   &#39;first&#39;,\n   &#39;time&#39;,\n   &#39;or&#39;,\n   &#39;already&#39;,\n   &#39;existed&#39;,\n   &#39;in&#39;,\n   &#39;human&#39;,\n   &#39;populations,&#39;,\n   &#39;having&#39;,\n   &#39;the&#39;,\n   &#39;capacity&#39;,\n   &#39;to&#39;,\n   &#39;increase&#39;,\n   &#39;rapidly&#39;,\n   &#39;in&#39;,\n   &#39;incidence&#39;,\n   &#39;as&#39;,\n   &#39;well&#39;,\n   &#39;as&#39;,\n   &#39;geographic&#39;,\n   &#39;range.&#39;,\n   &#39;Adapting&#39;,\n   &#39;to&#39;,\n   &#39;human&#39;,\n   &#39;immune&#39;,\n   &#39;system,&#39;,\n   &#39;emerging&#39;,\n   &#39;diseases&#39;,\n   &#39;may&#39;,\n   &#39;trigger&#39;,\n   &#39;large-scale&#39;,\n   &#39;pandemic&#39;,\n   &#39;spreading,&#39;,\n   &#39;such&#39;,\n   &#39;as&#39;,\n   &#39;the&#39;,\n   &#39;transnational&#39;,\n   &#39;spreading&#39;,\n   &#39;of&#39;,\n   &#39;SARS,&#39;,\n   &#39;the&#39;,\n   &#39;global&#39;,\n   &#39;outbreak&#39;,\n   &#39;of&#39;,\n   &#39;A(H1N1),&#39;,\n   &#39;and&#39;,\n   &#39;the&#39;,\n   &#39;recent&#39;,\n   &#39;potential&#39;,\n   &#39;invasion&#39;,\n   &#39;of&#39;,\n   &#39;avian&#39;,\n   &#39;influenza&#39;,\n   &#39;A(H7N9).&#39;,\n   &#39;To&#39;,\n   &#39;study&#39;,\n   &#39;the&#39;,\n   &#39;dynamics&#39;,\n   &#39;mediating&#39;,\n   &#39;the&#39;,\n   &#39;transmission&#39;,\n   &#39;of&#39;,\n   &#39;emerging&#39;,\n   &#39;diseases,&#39;,\n   &#39;spatial&#39;,\n   &#39;epidemiology&#39;,\n   &#39;of&#39;,\n   &#39;networked&#39;,\n   &#39;metapopulation&#39;,\n   &#39;provides&#39;,\n   &#39;a&#39;,\n   &#39;valuable&#39;,\n   &#39;modeling&#39;,\n   &#39;framework,&#39;,\n   &#39;which&#39;,\n   &#39;takes&#39;,\n   &#39;spatially&#39;,\n   &#39;distributed&#39;,\n   &#39;factors&#39;,\n   &#39;into&#39;,\n   &#39;consideration.&#39;,\n   &#39;This&#39;,\n   &#39;review&#39;,\n   &#39;elaborates&#39;,\n   &#39;the&#39;,\n   &#39;latest&#39;,\n   &#39;progresses&#39;,\n   &#39;on&#39;,\n   &#39;the&#39;,\n   &#39;spatial&#39;,\n   &#39;metapopulation&#39;,\n   &#39;dynamics,&#39;,\n   &#39;discusses&#39;,\n   &#39;empirical&#39;,\n   &#39;and&#39;,\n   &#39;theoretical&#39;,\n   &#39;findings&#39;,\n   &#39;that&#39;,\n   &#39;verify&#39;,\n   &#39;the&#39;,\n   &#39;validity&#39;,\n   &#39;of&#39;,\n   &#39;networked&#39;,\n   &#39;metapopulations,&#39;,\n   &#39;and&#39;,\n   &#39;the&#39;,\n   &#39;application&#39;,\n   &#39;in&#39;,\n   &#39;evaluating&#39;,\n   &#39;the&#39;,\n   &#39;effectiveness&#39;,\n   &#39;of&#39;,\n   &#39;disease&#39;,\n   &#39;intervention&#39;,\n   &#39;strategies&#39;,\n   &#39;as&#39;,\n   &#39;well.&#39;]),\n (&#39;62gfisc6&#39;,\n  [&#39;Germline&#39;,\n   &#39;variation&#39;,\n   &#39;at&#39;,\n   &#39;immunoglobulin&#39;,\n   &#39;gene&#39;,\n   &#39;(IG)&#39;,\n   &#39;loci&#39;,\n   &#39;is&#39;,\n   &#39;critical&#39;,\n   &#39;for&#39;,\n   &#39;pathogen-mediated&#39;,\n   &#39;immunity,&#39;,\n   &#39;but&#39;,\n   &#39;establishing&#39;,\n   &#39;complete&#39;,\n   &#39;reference&#39;,\n   &#39;sequences&#39;,\n   &#39;in&#39;,\n   &#39;these&#39;,\n   &#39;regions&#39;,\n   &#39;is&#39;,\n   &#39;problematic&#39;,\n   &#39;because&#39;,\n   &#39;of&#39;,\n   &#39;segmental&#39;,\n   &#39;duplications&#39;,\n   &#39;and&#39;,\n   &#39;somatically&#39;,\n   &#39;rearranged&#39;,\n   &#39;source&#39;,\n   &#39;DNA.&#39;,\n   &#39;We&#39;,\n   &#39;sequenced&#39;,\n   &#39;BAC&#39;,\n   &#39;clones&#39;,\n   &#39;from&#39;,\n   &#39;the&#39;,\n   &#39;essentially&#39;,\n   &#39;haploid&#39;,\n   &#39;hydatidiform&#39;,\n   &#39;mole,&#39;,\n   &#39;CHM1,&#39;,\n   &#39;across&#39;,\n   &#39;the&#39;,\n   &#39;light&#39;,\n   &#39;chain&#39;,\n   &#39;IG&#39;,\n   &#39;loci,&#39;,\n   &#39;kappa&#39;,\n   &#39;(IGK)&#39;,\n   &#39;and&#39;,\n   &#39;lambda&#39;,\n   &#39;(IGL),&#39;,\n   &#39;creating&#39;,\n   &#39;single&#39;,\n   &#39;haplotype&#39;,\n   &#39;representations&#39;,\n   &#39;of&#39;,\n   &#39;these&#39;,\n   &#39;regions.&#39;,\n   &#39;The&#39;,\n   &#39;IGL&#39;,\n   &#39;haplotype&#39;,\n   &#39;is&#39;,\n   &#39;1.25Mb&#39;,\n   &#39;of&#39;,\n   &#39;contiguous&#39;,\n   &#39;sequence&#39;,\n   &#39;with&#39;,\n   &#39;four&#39;,\n   &#39;novel&#39;,\n   &#39;V&#39;,\n   &#39;gene&#39;,\n   &#39;and&#39;,\n   &#39;one&#39;,\n   &#39;novel&#39;,\n   &#39;C&#39;,\n   &#39;gene&#39;,\n   &#39;alleles&#39;,\n   &#39;and&#39;,\n   &#39;an&#39;,\n   &#39;11.9kbp&#39;,\n   &#39;insertion.&#39;,\n   &#39;The&#39;,\n   &#39;IGK&#39;,\n   &#39;haplotype&#39;,\n   &#39;consists&#39;,\n   &#39;of&#39;,\n   &#39;two&#39;,\n   &#39;644kbp&#39;,\n   &#39;proximal&#39;,\n   &#39;and&#39;,\n   &#39;466kbp&#39;,\n   &#39;distal&#39;,\n   &#39;contigs&#39;,\n   &#39;separated&#39;,\n   &#39;by&#39;,\n   &#39;a&#39;,\n   &#39;gap&#39;,\n   &#39;also&#39;,\n   &#39;present&#39;,\n   &#39;in&#39;,\n   &#39;the&#39;,\n   &#39;reference&#39;,\n   &#39;genome&#39;,\n   &#39;sequence.&#39;,\n   &#39;Our&#39;,\n   &#39;effort&#39;,\n   &#39;added&#39;,\n   &#39;an&#39;,\n   &#39;additional&#39;,\n   &#39;49kbp&#39;,\n   &#39;of&#39;,\n   &#39;unique&#39;,\n   &#39;sequence&#39;,\n   &#39;extending&#39;,\n   &#39;into&#39;,\n   &#39;this&#39;,\n   &#39;gap.&#39;,\n   &#39;The&#39;,\n   &#39;IGK&#39;,\n   &#39;haplotype&#39;,\n   &#39;contains&#39;,\n   &#39;six&#39;,\n   &#39;novel&#39;,\n   &#39;V&#39;,\n   &#39;gene&#39;,\n   &#39;and&#39;,\n   &#39;one&#39;,\n   &#39;novel&#39;,\n   &#39;J&#39;,\n   &#39;gene&#39;,\n   &#39;alleles&#39;,\n   &#39;and&#39;,\n   &#39;a&#39;,\n   &#39;16.7kbp&#39;,\n   &#39;region&#39;,\n   &#39;with&#39;,\n   &#39;increased&#39;,\n   &#39;sequence&#39;,\n   &#39;identity&#39;,\n   &#39;between&#39;,\n   &#39;the&#39;,\n   &#39;two&#39;,\n   &#39;IGK&#39;,\n   &#39;contigs,&#39;,\n   &#39;exhibiting&#39;,\n   &#39;signatures&#39;,\n   &#39;of&#39;,\n   &#39;interlocus&#39;,\n   &#39;gene&#39;,\n   &#39;conversion.&#39;,\n   &#39;Our&#39;,\n   &#39;data&#39;,\n   &#39;facilitated&#39;,\n   &#39;the&#39;,\n   &#39;first&#39;,\n   &#39;comparison&#39;,\n   &#39;of&#39;,\n   &#39;nucleotide&#39;,\n   &#39;diversity&#39;,\n   &#39;between&#39;,\n   &#39;the&#39;,\n   &#39;light&#39;,\n   &#39;and&#39;,\n   &#39;IG&#39;,\n   &#39;heavy&#39;,\n   &#39;(IGH)&#39;,\n   &#39;chain&#39;,\n   &#39;haplotypes&#39;,\n   &#39;within&#39;,\n   &#39;a&#39;,\n   &#39;single&#39;,\n   &#39;genome,&#39;,\n   &#39;revealing&#39;,\n   &#39;a&#39;,\n   &#39;three&#39;,\n   &#39;to&#39;,\n   &#39;six&#39;,\n   &#39;fold&#39;,\n   &#39;enrichment&#39;,\n   &#39;in&#39;,\n   &#39;the&#39;,\n   &#39;IGH&#39;,\n   &#39;locus,&#39;,\n   &#39;supporting&#39;,\n   &#39;the&#39;,\n   &#39;theory&#39;,\n   &#39;that&#39;,\n   &#39;the&#39;,\n   &#39;heavy&#39;,\n   &#39;chain&#39;,\n   &#39;may&#39;,\n   &#39;be&#39;,\n   &#39;more&#39;,\n   &#39;important&#39;,\n   &#39;in&#39;,\n   &#39;determining&#39;,\n   &#39;antigenic&#39;,\n   &#39;specificity.&#39;]),\n (&#39;058r9486&#39;,\n  [&#39;Deep&#39;,\n   &#39;sequencing&#39;,\n   &#39;of&#39;,\n   &#39;clinical&#39;,\n   &#39;samples&#39;,\n   &#39;is&#39;,\n   &#39;now&#39;,\n   &#39;an&#39;,\n   &#39;established&#39;,\n   &#39;tool&#39;,\n   &#39;for&#39;,\n   &#39;the&#39;,\n   &#39;detection&#39;,\n   &#39;of&#39;,\n   &#39;infectious&#39;,\n   &#39;pathogens,&#39;,\n   &#39;with&#39;,\n   &#39;direct&#39;,\n   &#39;medical&#39;,\n   &#39;applications.&#39;,\n   &#39;The&#39;,\n   &#39;large&#39;,\n   &#39;amount&#39;,\n   &#39;of&#39;,\n   &#39;data&#39;,\n   &#39;generated&#39;,\n   &#39;provides&#39;,\n   &#39;an&#39;,\n   &#39;opportunity&#39;,\n   &#39;to&#39;,\n   &#39;detect&#39;,\n   &#39;species&#39;,\n   &#39;even&#39;,\n   &#39;at&#39;,\n   &#39;very&#39;,\n   &#39;low&#39;,\n   &#39;levels,&#39;,\n   &#39;provided&#39;,\n   &#39;that&#39;,\n   &#39;computational&#39;,\n   &#39;tools&#39;,\n   &#39;can&#39;,\n   &#39;effectively&#39;,\n   &#39;interpret&#39;,\n   &#39;potentially&#39;,\n   &#39;complex&#39;,\n   &#39;metagenomic&#39;,\n   &#39;mixtures.&#39;,\n   &#39;Data&#39;,\n   &#39;interpretation&#39;,\n   &#39;is&#39;,\n   &#39;complicated&#39;,\n   &#39;by&#39;,\n   &#39;the&#39;,\n   &#39;fact&#39;,\n   &#39;that&#39;,\n   &#39;short&#39;,\n   &#39;sequencing&#39;,\n   &#39;reads&#39;,\n   &#39;can&#39;,\n   &#39;match&#39;,\n   &#39;multiple&#39;,\n   &#39;organisms&#39;,\n   &#39;and&#39;,\n   &#39;by&#39;,\n   &#39;the&#39;,\n   &#39;lack&#39;,\n   &#39;of&#39;,\n   &#39;completeness&#39;,\n   &#39;of&#39;,\n   &#39;existing&#39;,\n   &#39;databases,&#39;,\n   &#39;in&#39;,\n   &#39;particular&#39;,\n   &#39;for&#39;,\n   &#39;viral&#39;,\n   &#39;pathogens.&#39;,\n   &#39;This&#39;,\n   &#39;interpretation&#39;,\n   &#39;problem&#39;,\n   &#39;can&#39;,\n   &#39;be&#39;,\n   &#39;formulated&#39;,\n   &#39;statistically&#39;,\n   &#39;as&#39;,\n   &#39;a&#39;,\n   &#39;mixture&#39;,\n   &#39;model,&#39;,\n   &#39;where&#39;,\n   &#39;the&#39;,\n   &#39;species&#39;,\n   &#39;of&#39;,\n   &#39;origin&#39;,\n   &#39;of&#39;,\n   &#39;each&#39;,\n   &#39;read&#39;,\n   &#39;is&#39;,\n   &#39;missing,&#39;,\n   &#39;but&#39;,\n   &#39;the&#39;,\n   &#39;complete&#39;,\n   &#39;knowledge&#39;,\n   &#39;of&#39;,\n   &#39;all&#39;,\n   &#39;species&#39;,\n   &#39;present&#39;,\n   &#39;in&#39;,\n   &#39;the&#39;,\n   &#39;mixture&#39;,\n   &#39;helps&#39;,\n   &#39;with&#39;,\n   &#39;the&#39;,\n   &#39;individual&#39;,\n   &#39;reads&#39;,\n   &#39;assignment.&#39;,\n   &#39;Several&#39;,\n   &#39;analytical&#39;,\n   &#39;tools&#39;,\n   &#39;have&#39;,\n   &#39;been&#39;,\n   &#39;proposed&#39;,\n   &#39;to&#39;,\n   &#39;approximately&#39;,\n   &#39;solve&#39;,\n   &#39;this&#39;,\n   &#39;computational&#39;,\n   &#39;problem.&#39;,\n   &#39;Here,&#39;,\n   &#39;we&#39;,\n   &#39;show&#39;,\n   &#39;that&#39;,\n   &#39;the&#39;,\n   &#39;use&#39;,\n   &#39;of&#39;,\n   &#39;parallel&#39;,\n   &#39;Monte&#39;,\n   &#39;Carlo&#39;,\n   &#39;Markov&#39;,\n   &#39;chains&#39;,\n   &#39;(MCMC)&#39;,\n   &#39;for&#39;,\n   &#39;the&#39;,\n   &#39;exploration&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;species&#39;,\n   &#39;space&#39;,\n   &#39;enables&#39;,\n   &#39;the&#39;,\n   &#39;identification&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;set&#39;,\n   &#39;of&#39;,\n   &#39;species&#39;,\n   &#39;most&#39;,\n   &#39;likely&#39;,\n   &#39;to&#39;,\n   &#39;contribute&#39;,\n   &#39;to&#39;,\n   &#39;the&#39;,\n   &#39;mixture.&#39;,\n   &#39;The&#39;,\n   &#39;added&#39;,\n   &#39;accuracy&#39;,\n   &#39;comes&#39;,\n   &#39;at&#39;,\n   &#39;a&#39;,\n   &#39;cost&#39;,\n   &#39;of&#39;,\n   &#39;increased&#39;,\n   &#39;computation&#39;,\n   &#39;time.&#39;,\n   &#39;Our&#39;,\n   &#39;approach&#39;,\n   &#39;is&#39;,\n   &#39;useful&#39;,\n   &#39;for&#39;,\n   &#39;solving&#39;,\n   &#39;complex&#39;,\n   &#39;mixtures&#39;,\n   &#39;involving&#39;,\n   &#39;several&#39;,\n   &#39;related&#39;,\n   &#39;species.&#39;,\n   &#39;We&#39;,\n   &#39;designed&#39;,\n   &#39;our&#39;,\n   &#39;method&#39;,\n   &#39;specifically&#39;,\n   &#39;for&#39;,\n   &#39;the&#39;,\n   &#39;analysis&#39;,\n   &#39;of&#39;,\n   &#39;deep&#39;,\n   &#39;transcriptome&#39;,\n   &#39;sequencing&#39;,\n   &#39;datasets&#39;,\n   &#39;and&#39;,\n   &#39;with&#39;,\n   &#39;a&#39;,\n   &#39;particular&#39;,\n   &#39;focus&#39;,\n   &#39;on&#39;,\n   &#39;viral&#39;,\n   &#39;pathogen&#39;,\n   &#39;detection,&#39;,\n   &#39;but&#39;,\n   &#39;the&#39;,\n   &#39;principles&#39;,\n   &#39;are&#39;,\n   &#39;applicable&#39;,\n   &#39;more&#39;,\n   &#39;generally&#39;,\n   &#39;to&#39;,\n   &#39;all&#39;,\n   &#39;types&#39;,\n   &#39;of&#39;,\n   &#39;metagenomics&#39;,\n   &#39;mixtures.&#39;,\n   &#39;The&#39;,\n   &#39;work&#39;,\n   &#39;is&#39;,\n   &#39;implemented&#39;,\n   &#39;as&#39;,\n   &#39;a&#39;,\n   &#39;user&#39;,\n   &#39;friendly&#39;,\n   &#39;R&#39;,\n   &#39;package,&#39;,\n   &#39;available&#39;,\n   &#39;from&#39;,\n   &#39;CRAN:&#39;,\n   &#39;http://cran.r-project.org/web/packages/metaMix&#39;]),\n (&#39;wich35l7&#39;,\n  [&#39;Developing&#39;,\n   &#39;methods&#39;,\n   &#39;to&#39;,\n   &#39;reconstruct&#39;,\n   &#39;transmission&#39;,\n   &#39;histories&#39;,\n   &#39;for&#39;,\n   &#39;viral&#39;,\n   &#39;outbreaks&#39;,\n   &#39;could&#39;,\n   &#39;provide&#39;,\n   &#39;critical&#39;,\n   &#39;information&#39;,\n   &#39;to&#39;,\n   &#39;support&#39;,\n   &#39;locating&#39;,\n   &#39;sources&#39;,\n   &#39;of&#39;,\n   &#39;disease&#39;,\n   &#39;transmission.&#39;,\n   &#39;Phylogenetic&#39;,\n   &#39;methods&#39;,\n   &#39;used&#39;,\n   &#39;to&#39;,\n   &#39;measure&#39;,\n   &#39;the&#39;,\n   &#39;degree&#39;,\n   &#39;of&#39;,\n   &#39;relatedness&#39;,\n   &#39;among&#39;,\n   &#39;sequenced&#39;,\n   &#39;viral&#39;,\n   &#39;samples&#39;,\n   &#39;have&#39;,\n   &#39;proven&#39;,\n   &#39;useful&#39;,\n   &#39;in&#39;,\n   &#39;identifying&#39;,\n   &#39;potential&#39;,\n   &#39;outbreak&#39;,\n   &#39;sources.&#39;,\n   &#39;The&#39;,\n   &#39;complex&#39;,\n   &#39;nature&#39;,\n   &#39;of&#39;,\n   &#39;infectious&#39;,\n   &#39;disease,&#39;,\n   &#39;however,&#39;,\n   &#39;makes&#39;,\n   &#39;it&#39;,\n   &#39;difficult&#39;,\n   &#39;to&#39;,\n   &#39;assign&#39;,\n   &#39;a&#39;,\n   &#39;rigorously&#39;,\n   &#39;defined&#39;,\n   &#39;quantitative&#39;,\n   &#39;confidence&#39;,\n   &#39;value&#39;,\n   &#39;assessing&#39;,\n   &#39;the&#39;,\n   &#39;likelihood&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#39;true&#39;,\n   &#39;direct&#39;,\n   &#39;transmission&#39;,\n   &#39;event&#39;,\n   &#39;using&#39;,\n   &#39;genetic&#39;,\n   &#39;data&#39;,\n   &#39;alone.&#39;,\n   &#39;A&#39;,\n   &#39;new&#39;,\n   &#39;method&#39;,\n   &#39;is&#39;,\n   &#39;presented&#39;,\n   &#39;to&#39;,\n   &#39;calculate&#39;,\n   &#39;a&#39;,\n   &#39;confidence&#39;,\n   &#39;value&#39;,\n   &#39;assessing&#39;,\n   &#39;the&#39;,\n   &#39;likelihood&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#39;transmission&#39;,\n   &#39;event&#39;,\n   &#39;using&#39;,\n   &#39;both&#39;,\n   &#39;phylogenetic&#39;,\n   &#39;inference&#39;,\n   &#39;and&#39;,\n   &#39;limited&#39;,\n   &#39;knowledge&#39;,\n   &#39;of&#39;,\n   &#39;incubation&#39;,\n   &#39;and&#39;,\n   &#39;infectious&#39;,\n   &#39;duration&#39;,\n   &#39;times.&#39;,\n   &#39;The&#39;,\n   &#39;method&#39;,\n   &#39;is&#39;,\n   &#39;applied&#39;,\n   &#39;to&#39;,\n   &#39;simulations&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#39;foot&#39;,\n   &#39;and&#39;,\n   &#39;mouth&#39;,\n   &#39;disease&#39;,\n   &#39;(FMD)&#39;,\n   &#39;outbreak&#39;,\n   &#39;to&#39;,\n   &#39;demonstrate&#39;,\n   &#39;how&#39;,\n   &#39;the&#39;,\n   &#39;combination&#39;,\n   &#39;of&#39;,\n   &#39;both&#39;,\n   &#39;phylogenetic&#39;,\n   &#39;and&#39;,\n   &#39;epidemiology&#39;,\n   &#39;data&#39;,\n   &#39;can&#39;,\n   &#39;be&#39;,\n   &#39;used&#39;,\n   &#39;to&#39;,\n   &#39;strengthen&#39;,\n   &#39;the&#39;,\n   &#39;assessment&#39;,\n   &#39;of&#39;,\n   &#39;the&#39;,\n   &#39;likelihood&#39;,\n   &#39;of&#39;,\n   &#39;direct&#39;,\n   &#39;transmission&#39;,\n   &#39;over&#39;,\n   &#39;methods&#39;,\n   &#39;using&#39;,\n   &#39;just&#39;,\n   &#39;phylogenetic&#39;,\n   &#39;data&#39;,\n   &#39;or&#39;,\n   &#39;infection&#39;,\n   &#39;timing&#39;,\n   &#39;data&#39;,\n   &#39;alone.&#39;,\n   &#39;The&#39;,\n   &#39;method&#39;,\n   &#39;is&#39;,\n   &#39;applied&#39;,\n   &#39;to&#39;,\n   &#39;a&#39;,\n   &#39;previous&#39;,\n   &#39;FMD&#39;,\n   &#39;outbreak&#39;,\n   &#39;to&#39;,\n   &#39;identify&#39;,\n   &#39;areas&#39;,\n   &#39;where&#39;,\n   &#39;over&#39;,\n   &#39;confidence&#39;,\n   &#39;in&#39;,\n   &#39;previously&#39;,\n   &#39;inferred&#39;,\n   &#39;direct&#39;,\n   &#39;transmission&#39;,\n   &#39;may&#39;,\n   &#39;exist.&#39;,\n   &#39;Combining&#39;,\n   &#39;knowledge&#39;,\n   &#39;from&#39;,\n   &#39;viral&#39;,\n   &#39;evolution&#39;,\n   &#39;and&#39;,\n   &#39;epidemiology&#39;,\n   &#39;within&#39;,\n   &#39;a&#39;,\n   &#39;single&#39;,\n   &#39;integrated&#39;,\n   &#39;transmission&#39;,\n   &#39;inference&#39;,\n   &#39;framework&#39;,\n   &#39;is&#39;,\n   &#39;an&#39;,\n   &#39;important&#39;,\n   &#39;approach&#39;,\n   &#39;to&#39;,\n   &#39;assess&#39;,\n   &#39;the&#39;,\n   &#39;potential&#39;,\n   &#39;likelihood&#39;,\n   &#39;of&#39;,\n   &#39;transmission&#39;,\n   &#39;events&#39;,\n   &#39;and&#39;,\n   &#39;makes&#39;,\n   &#39;clear&#39;,\n   &#39;how&#39;,\n   &#39;specific&#39;,\n   &#39;features&#39;,\n   &#39;of&#39;,\n   &#39;a&#39;,\n   &#34;virus&#39;&#34;,\n   &#39;spread&#39;,\n   &#39;through&#39;,\n   &#39;the&#39;,\n   &#39;course&#39;,\n   &#39;of&#39;,\n   &#39;an&#39;,\n   &#39;outbreak&#39;,\n   &#39;will&#39;,\n   &#39;directly&#39;,\n   &#39;determine&#39;,\n   &#39;the&#39;,\n   &#39;potential&#39;,\n   &#39;for&#39;,\n   &#39;confidence&#39;,\n   &#39;in&#39;,\n   &#39;inferred&#39;,\n   &#39;host&#39;,\n   &#39;transmission&#39;,\n   &#39;links.&#39;])]</div>"]}}],"execution_count":0},{"cell_type":"markdown","source":["--> 6. Now we'll use the positive words available from\n\nhttps://github.com/shekhargulati/sentiment-analysis-python/blob/master/opinion-lexicon-English/positive-words.txt\n\nYou'll need to download the file (click on raw and download the page that appears) and remove the comments from\nthe top of the file on your own machine. Save the new file as positive words.csv You can either add the header word\nto the file at this point, or when you load the file through the UI into Databricks, but you should ensure that:\n(a) the resulting table is called positive words, and\n(b) has a single column, called words"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"1f055198-0490-4d76-90b2-e7b46beb8017"}}},{"cell_type":"code","source":["dbutils.fs.ls(\"/FileStore/tables/positive_words-2.csv\")"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"1d4d7912-f580-48e4-acd6-c4863c941d7e"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">Out[133]: [FileInfo(path=&#39;dbfs:/FileStore/tables/positive_words-2.csv&#39;, name=&#39;positive_words-2.csv&#39;, size=21099)]</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">Out[133]: [FileInfo(path=&#39;dbfs:/FileStore/tables/positive_words-2.csv&#39;, name=&#39;positive_words-2.csv&#39;, size=21099)]</div>"]}}],"execution_count":0},{"cell_type":"code","source":["positive_rdd = sc.textFile(\"/FileStore/tables/positive_words-2.csv\")\npositive_rdd.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"0aa02bf5-8445-4b09-927c-44a82948bfea"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">Out[134]: [&#39;a+&#39;, &#39;abound&#39;, &#39;abounds&#39;, &#39;abundance&#39;, &#39;abundant&#39;]</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">Out[134]: [&#39;a+&#39;, &#39;abound&#39;, &#39;abounds&#39;, &#39;abundance&#39;, &#39;abundant&#39;]</div>"]}}],"execution_count":0},{"cell_type":"markdown","source":["--> 7. Do the same for negative words available from\n\nhttps://github.com/shekhargulati/sentiment-analysis-python/blob/master/opinion-lexicon-English/negative-words.txt"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"a6ee7877-e716-4971-96a0-99a7293f9c1c"}}},{"cell_type":"code","source":["dbutils.fs.ls(\"/FileStore/tables/negative_words.csv\")"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"b1f2ffcb-ba95-47b6-9e9f-00ee0d29fa9f"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">Out[135]: [FileInfo(path=&#39;dbfs:/FileStore/tables/negative_words.csv&#39;, name=&#39;negative_words.csv&#39;, size=49541)]</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">Out[135]: [FileInfo(path=&#39;dbfs:/FileStore/tables/negative_words.csv&#39;, name=&#39;negative_words.csv&#39;, size=49541)]</div>"]}}],"execution_count":0},{"cell_type":"code","source":["negative_rdd = sc.textFile(\"/FileStore/tables/negative_words.csv\")\nnegative_rdd.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"4dfcd15d-2ec8-4cee-988d-22999a625ba5"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"<div class=\"ansiout\">Out[136]: [&#39;2-faced&#39;, &#39;2-faces&#39;, &#39;abnormal&#39;, &#39;abolish&#39;, &#39;abominable&#39;]</div>","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"html","arguments":{}}},"output_type":"display_data","data":{"text/html":["<style scoped>\n  .ansiout {\n    display: block;\n    unicode-bidi: embed;\n    white-space: pre-wrap;\n    word-wrap: break-word;\n    word-break: break-all;\n    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n    font-size: 13px;\n    color: #555;\n    margin-left: 4px;\n    line-height: 19px;\n  }\n</style>\n<div class=\"ansiout\">Out[136]: [&#39;2-faced&#39;, &#39;2-faces&#39;, &#39;abnormal&#39;, &#39;abolish&#39;, &#39;abominable&#39;]</div>"]}}],"execution_count":0},{"cell_type":"code","source":[""],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"485bd4f3-d93f-4283-a683-66ab18db284f"}},"outputs":[],"execution_count":0}],"metadata":{"application/vnd.databricks.v1+notebook":{"notebookName":"W10D2_SentimentAnalysis","dashboards":[],"notebookMetadata":{"pythonIndentUnit":2},"language":"python","widgets":{},"notebookOrigID":274456297658461}},"nbformat":4,"nbformat_minor":0}
